<!DOCTYPE html>
<html lang="en" class="no-js">
    <head>
        <title>Development of 17 polymorphic microsatellite loci from Jeju striped field mouse, Apodemus agrarius chejuensis (Rodentia: Muridae), by 454 pyrosequencing | Hereditas | Full Text</title>

        

    <meta name="citation_abstract" content="The striped field mouse, Apodemus agrarius, is the most common mammal in Korea. Although microsatellite loci for the species have been identified from populations in southwestern China, amplification of those markers for Korean populations have been unsuccessful. The complicated taxonomy of Korean striped field mouse including populations on Jeju Island (A. a chejuensis) necessitates identification of additional molecular markers. We applied 454 pyrosequencing systems to develop a suite of microsatellite markers. Muscle tissue was harvested and sequenced from 30 Jeju striped field mouse specimens which yielded 12,165 reads with a mean length per read of 287 bp. From these reads, we identified 17 microsatellite loci for A. a. chejuensis and tested these new markers against samples of both A. a chejuensis and A. a coreae, the mainland taxon. All 17 loci were amplified successfully for both taxa. Of the total 17 loci, one locus failed to amplify for a population on Heuksan Island. The cross-species transferability was also tested with the allied taxon, A. peninsulae and confirmed successful for 12 loci. These newly developed markers will benefit studies of genetic structure, evolution, and resolving taxonomic problems of striped field mice and allied taxa in Korea."/>

    <meta name="description" content="The striped field mouse, Apodemus agrarius, is the most common mammal in Korea. Although microsatellite loci for the species have been identified from populations in southwestern China, amplification of those markers for Korean populations have been unsuccessful. The complicated taxonomy of Korean striped field mouse including populations on Jeju Island (A. a chejuensis) necessitates identification of additional molecular markers. We applied 454 pyrosequencing systems to develop a suite of microsatellite markers. Muscle tissue was harvested and sequenced from 30 Jeju striped field mouse specimens which yielded 12,165 reads with a mean length per read of 287 bp. From these reads, we identified 17 microsatellite loci for A. a. chejuensis and tested these new markers against samples of both A. a chejuensis and A. a coreae, the mainland taxon. All 17 loci were amplified successfully for both taxa. Of the total 17 loci, one locus failed to amplify for a population on Heuksan Island. The cross-species transferability was also tested with the allied taxon, A. peninsulae and confirmed successful for 12 loci. These newly developed markers will benefit studies of genetic structure, evolution, and resolving taxonomic problems of striped field mice and allied taxa in Korea."/>

    <meta name="journal_id" content="41065"/>

    <meta name="dc.title" content="Development of 17 polymorphic microsatellite loci from Jeju striped field mouse, Apodemus agrarius chejuensis (Rodentia: Muridae), by 454 pyrosequencing"/>

    <meta name="dc.source" content="Hereditas 2018 155:1"/>

    <meta name="dc.format" content="text/html"/>

    <meta name="dc.publisher" content="BioMed Central"/>

    <meta name="dc.date" content="2018-09-26"/>

    <meta name="dc.type" content="Report"/>

    <meta name="dc.language" content="En"/>

    <meta name="dc.copyright" content="2018 The Author(s)"/>

    <meta name="dc.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="dc.description" content="The striped field mouse, Apodemus agrarius, is the most common mammal in Korea. Although microsatellite loci for the species have been identified from populations in southwestern China, amplification of those markers for Korean populations have been unsuccessful. The complicated taxonomy of Korean striped field mouse including populations on Jeju Island (A. a chejuensis) necessitates identification of additional molecular markers. We applied 454 pyrosequencing systems to develop a suite of microsatellite markers. Muscle tissue was harvested and sequenced from 30 Jeju striped field mouse specimens which yielded 12,165 reads with a mean length per read of 287&#160;bp. From these reads, we identified 17 microsatellite loci for A. a. chejuensis and tested these new markers against samples of both A. a chejuensis and A. a coreae, the mainland taxon. All 17 loci were amplified successfully for both taxa. Of the total 17 loci, one locus failed to amplify for a population on Heuksan Island. The cross-species transferability was also tested with the allied taxon, A. peninsulae and confirmed successful for 12 loci. These newly developed markers will benefit studies of genetic structure, evolution, and resolving taxonomic problems of striped field mice and allied taxa in Korea."/>

    <meta name="prism.issn" content="1601-5223"/>

    <meta name="prism.publicationName" content="Hereditas"/>

    <meta name="prism.publicationDate" content="2018-09-26"/>

    <meta name="prism.volume" content="155"/>

    <meta name="prism.number" content="1"/>

    <meta name="prism.section" content="Report"/>

    <meta name="prism.startingPage" content="30"/>

    <meta name="prism.copyright" content="2018 The Author(s)"/>

    <meta name="prism.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="prism.url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0070-8"/>

    <meta name="prism.doi" content="doi:10.1186/s41065-018-0070-8"/>

    <meta name="citation_pdf_url" content="https://hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-018-0070-8"/>

    <meta name="citation_fulltext_html_url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0070-8"/>

    <meta name="citation_journal_title" content="Hereditas"/>

    <meta name="citation_publisher" content="BioMed Central"/>

    <meta name="citation_issn" content="1601-5223"/>

    <meta name="citation_title" content="Development of 17 polymorphic microsatellite loci from Jeju striped field mouse, Apodemus agrarius chejuensis (Rodentia: Muridae), by 454 pyrosequencing"/>

    <meta name="citation_volume" content="155"/>

    <meta name="citation_issue" content="1"/>

    <meta name="citation_publication_date" content="2018/12"/>

    <meta name="citation_online_date" content="2018/09/26"/>

    <meta name="citation_firstpage" content="30"/>

    <meta name="citation_article_type" content="Brief report"/>

    <meta name="citation_fulltext_world_readable" content=""/>

    <meta name="dc.identifier" content="doi:10.1186/s41065-018-0070-8"/>

    <meta name="DOI" content="10.1186/s41065-018-0070-8"/>

    <meta name="citation_doi" content="10.1186/s41065-018-0070-8"/>

    <meta name="dc.creator" content="Han-Na Kim"/>

    <meta name="dc.creator" content="Han-Ul Kim"/>

    <meta name="dc.creator" content="Yeong-Seok Jo"/>

    <meta name="dc.creator" content="Jongwoo Jung"/>

    <meta name="dc.subject" content="Animal Genetics and Genomics"/>

    <meta name="dc.subject" content="Evolutionary Biology"/>

    <meta name="citation_reference" content="citation_title=Superfamily Muroidae; citation_inbook_title=Mammal species world a Taxon Geogr Ref; citation_publication_date=2005; citation_pages=894-1532; citation_id=CR1; citation_author=GG Musser; citation_author=MD Carleton; citation_publisher=Johns Hopkins University Press"/>

    <meta name="citation_reference" content="citation_title=Illustrated encyclopedia of fauna and flora of Korea volume 7, mammals; citation_publication_date=1967; citation_id=CR2; citation_author=PH Won; citation_publisher=Ministry of Education"/>

    <meta name="citation_reference" content="citation_journal_title=Mammalia; citation_title=Genetic differentiation of the Korean striped field mouse, Apodemus agrarius (Muridae, Rodentia), based on microsatellite polymorphism; citation_author=Y Jo, H Kim, JT Baccus, J Jung; citation_volume=81; citation_publication_date=2017; citation_pages=297-307; citation_doi=10.1515/mammalia-2015-0152; citation_id=CR3"/>

    <meta name="citation_reference" content="citation_journal_title=Korean J Zool; citation_title=Geographic variation of morphometric characters among three subspecies of striped field mice, Apodemus agrarius Pallas (Muridae, Rodentia), from Korea; citation_author=HS Koh; citation_volume=29; citation_publication_date=1986; citation_pages=272-282; citation_id=CR4"/>

    <meta name="citation_reference" content="citation_journal_title=Korean J Syst Zool.; citation_title=Systematic studies of Korean rodents: III. Morphometric and chromosomal analyses of striped field mice, Apodemus agrarius chejuensis Jones and Johnson, from Jeju-Do; citation_author=HS Koh; citation_volume=3; citation_publication_date=1987; citation_pages=24-40; citation_id=CR5"/>

    <meta name="citation_reference" content="citation_journal_title=Korean J Syst Zool; citation_title=Systematic studies of Korean rodents. V. Morphometric and chromosomal analyses on island populations of striped field mice (Apodemus agrarius coreae) in southwestern coasts of the Korean peninsula; citation_author=HS Koh; citation_volume=5; citation_publication_date=1989; citation_pages=1-12; citation_id=CR6"/>

    <meta name="citation_reference" content="citation_journal_title=Korean J Syst Zool.; citation_title=Morphometric analyses with eight subspecies of striped field mice, Apodemus agrarius Pallas (Rodentia, Mammalia), in Asia: the taxonomic status of subspecies chejuensis at Cheju island in Korea; citation_author=HS Koh; citation_volume=7; citation_publication_date=1991; citation_pages=179-188; citation_id=CR7"/>

    <meta name="citation_reference" content="citation_journal_title=J Fac Agric Kyushu Univ; citation_title=Taxonomic re-examination of the striped field mouse, Apodemus agrarius coreae and A. a. chejuensis: evidence from crossbreeding experiments; citation_author=HS Oh, T Mori; citation_volume=43; citation_publication_date=1998; citation_pages=143-151; citation_id=CR8"/>

    <meta name="citation_reference" content="citation_journal_title=Korean J Zool.; citation_title=Variation of mitochondrial DNA in two subspecies of striped field mice, Apodemus agrarius coreae and Apodemus agrarius chejuensis, from Korea; citation_author=HS Koh, BS Yoo; citation_volume=35; citation_publication_date=1992; citation_pages=332-338; citation_id=CR9"/>

    <meta name="citation_reference" content="citation_journal_title=Heredity (Edinb); citation_title=The genetic relationships of two subspecies of striped field mice, Apodemus agrarius coreae and Apodemus agrarius chejuensis; citation_author=HS Koh, WJ Lee, TD Kocher; citation_volume=85; citation_publication_date=2000; citation_pages=30-36; citation_doi=10.1046/j.1365-2540.2000.00723.x; citation_id=CR10"/>

    <meta name="citation_reference" content="citation_journal_title=Mitochondrial DNA; citation_title=Migration route estimation of the Jeju striped field mouse Apodemus agrarius chejuensis (Rodentia, Muridae); citation_author=D-J Oh, T-W Kim, M-H Chang, S-H Han, H-S Oh, S-J Kim; citation_volume=24; citation_publication_date=2013; citation_pages=137-144; citation_doi=10.3109/19401736.2012.726619; citation_id=CR11"/>

    <meta name="citation_reference" content="citation_journal_title=Curr Top Microbiol Immunol; citation_title=Hantaviruses: history and overview; citation_author=KM Johnson; citation_volume=256; citation_publication_date=2001; citation_pages=1-14; citation_id=CR12"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Rev Genet; citation_title=Microsatellites: simple sequences with complex evolution; citation_author=H Ellegren; citation_volume=5; citation_publication_date=2004; citation_pages=435-445; citation_doi=10.1038/nrg1348; citation_id=CR13"/>

    <meta name="citation_reference" content="Lopez L, Barreiro R, Fischer M, Koch MA. Mining microsatellite markers from public expressed sequence tags databases for the study of threatened plants. BMC Genomics [Internet]. BMC Genomics; 2015;16. Available from: doi:
                    https://doi.org/10.1186/s12864-015-2031-1
                    
                  
                        "/>

    <meta name="citation_reference" content="citation_journal_title=Mol Ecol; citation_title=How much effort is required to isolate nuclear microsatellites from plants?; citation_author=J Squirrell, PM Hollingsworth, M Woodhead, J Russell, AJ Lowe, M Gibby; citation_volume=12; citation_publication_date=2003; citation_pages=1339-1348; citation_doi=10.1046/j.1365-294X.2003.01825.x; citation_id=CR15"/>

    <meta name="citation_reference" content="citation_journal_title=Conserv Genet; citation_title=Isolation and characterization of fourteen microsatellite loci for striped field mouse (Apodemus agrarius); citation_author=H Wu, XJ Zhan, L Yan, SY Liu, M Li, JC Hu; citation_volume=9; citation_publication_date=2008; citation_pages=1691-1693; citation_doi=10.1007/s10592-008-9535-z; citation_id=CR16"/>

    <meta name="citation_reference" content="citation_journal_title=J Mammal; citation_title=2016 guidelines of the American Society of Mammalogists for the use of wild mammals in research and education; citation_author=RS Sikes; citation_volume=97; citation_publication_date=2016; citation_pages=663-688; citation_doi=10.1093/jmammal/gyw078; citation_id=CR17"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Ecol Resour; citation_title=QDD version 3.1: a user-friendly computer program for microsatellite selection and primer design revisited: experimental validation of variables determining genotyping success rate; citation_author=E Megl&#233;cz, N Pech, A Gilles, V Dubut, P Hingamp, A Trilles; citation_volume=14; citation_publication_date=2014; citation_pages=1302-1313; citation_doi=10.1111/1755-0998.12271; citation_id=CR18"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=Primer3-new capabilities and interfaces; citation_author=A Untergasser, I Cutcutache, T Koressaar, J Ye, BC Faircloth, M Remm; citation_volume=40; citation_publication_date=2012; citation_pages=1-12; citation_doi=10.1093/nar/gks596; citation_id=CR19"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Ecol Resour; citation_title=Arlequin suite ver 3.5: a new series of programs to perform population genetics analyses under Linux and windows; citation_author=L Excoffier, HEL Lischer; citation_volume=10; citation_publication_date=2010; citation_pages=564-567; citation_doi=10.1111/j.1755-0998.2010.02847.x; citation_id=CR20"/>

    <meta name="citation_reference" content="citation_journal_title=J Hered; citation_title=GENEPOP (version 1.2): population genetics software for exact tests and ecumenicism; citation_author=M Raymond, F Rousset; citation_volume=86; citation_publication_date=1995; citation_pages=248-249; citation_doi=10.1093/oxfordjournals.jhered.a111573; citation_id=CR21"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Ecol Resour; citation_title=GENEPOP&#8217;007: a complete re-implementation of the GENEPOP software for windows and Linux; citation_author=F Rousset; citation_volume=8; citation_publication_date=2008; citation_pages=103-106; citation_doi=10.1111/j.1471-8286.2007.01931.x; citation_id=CR22"/>

    <meta name="citation_reference" content="citation_journal_title=Evolution (N Y); citation_title=Analyzing tables of statistical tests; citation_author=WR Rice; citation_volume=43; citation_publication_date=1989; citation_pages=223-225; citation_id=CR23"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=GenALEx 6.5: genetic analysis in excel. Population genetic software for teaching and research-an update; citation_author=R Peakall, PE Smouse; citation_volume=28; citation_publication_date=2012; citation_pages=2537-2539; citation_doi=10.1093/bioinformatics/bts460; citation_id=CR24"/>

    <meta name="citation_author" content="Han-Na Kim"/>

    <meta name="citation_author_institution" content="The Division of EcoCreative, Ewha Womans University, Seoul, Korea"/>

    <meta name="citation_author_institution" content="Animal Resources, National Institute of Biological Resources, Incheon, Korea"/>

    <meta name="citation_author" content="Han-Ul Kim"/>

    <meta name="citation_author_institution" content="The Division of EcoCreative, Ewha Womans University, Seoul, Korea"/>

    <meta name="citation_author" content="Yeong-Seok Jo"/>

    <meta name="citation_author_institution" content="Animal Resources, National Institute of Biological Resources, Incheon, Korea"/>

    <meta name="citation_author" content="Jongwoo Jung"/>

    <meta name="citation_author_institution" content="The Division of EcoCreative, Ewha Womans University, Seoul, Korea"/>

    <meta name="citation_author_institution" content="Department of Science Education, Ewha Woman University, Seoul, Korea"/>



        <meta charset="UTF-8"/>
        <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
        <meta name="viewport" content="width=device-width, initial-scale=1"/>

        <!-- For IE 9 and below -->
<!--[if IE]><link rel="shortcut icon" href=/static/images/favicons//favicon.ico><![endif]-->
<!-- Touch Icons - iOS and Android 2.1+ -->
<link rel="apple-touch-icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>
<!-- Firefox, Chrome, Safari, IE 11+ and Opera -->
<link rel="icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>


        <script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script>
        <script>(function(d){if (sessionStorage.fontsLoaded){d.style.setProperty('--font-family-sans', 'Europa');d.style.setProperty('--font-family-serif', 'Noto Serif')}})(document.documentElement);</script>

        <link rel="stylesheet" media="screen" href=/static/styles/basic/basic-a31ca85bc2.css>
<link rel="stylesheet" media="print" href=/static/styles/bmc/print-3a6240f456.css>

<link rel="stylesheet" class="js-ctm" href=/static/styles/bmc/main-84414d3d47.css
      media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">

        

        <script src="/static/js/app-bundle-8d61e6ef37.js" defer></script>
        
        <script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script>
        
    <script type="text/javascript">
        config = {
            env: 'live',
            site: 'hereditasjournal.biomedcentral.com',
            siteWithPath: 'hereditasjournal.biomedcentral.com' + window.location.pathname,
            twitterHashtag: '',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/s41065-018-0070-8',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central'
        };
    </script>

        
    <script type="text/javascript">
        dataLayer = [{"content":{"article":{"doi":"10.1186/s41065-018-0070-8","articleType":"Brief report","peerReviewType":"Closed","supplement":null,"keywords":"Apodemus agrarius chejuensis, Microsatellites, South Korea, Jeju Island, Pyrosequencing"},"contentInfo":{"imprint":"BioMed Central","title":"Development of 17 polymorphic microsatellite loci from Jeju striped field mouse, Apodemus agrarius chejuensis (Rodentia: Muridae), by 454 pyrosequencing","publishedAt":1537920000000,"publishedAtDate":"2018-09-26","author":["Han-Na Kim","Han-Ul Kim","Yeong-Seok Jo","Jongwoo Jung"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":"CC BY + CC0","openAccess":true},"environment":"live"},"journal":{"siteKey":"hereditasjournal.biomedcentral.com","volume":"155","issue":"1","title":"Hereditas","type":"AJs Life","journalID":41065,"gaCode":"UA-62066850-11","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Brief report","publishingSegment":"Genetics & Heredity (Life)"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}}}]

    </script>

        <script src="https://cdn.cookielaw.org/consent/7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script>
        
    
        
    

        
    
        
            
                <!-- Google Tag Manager -->
                <script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script>
                <!-- End Google Tag Manager -->
            
        
    


        
    
    <link rel="canonical" href="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0070-8"/>
    

        
        
        <meta property="og:url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0070-8"/>
        <meta property="og:type" content="article"/>
        <meta property="og:site_name" content="Hereditas"/>
        <meta property="og:title" content="Development of 17 polymorphic microsatellite loci from Jeju striped field mouse, Apodemus agrarius chejuensis (Rodentia: Muridae), by 454 pyrosequencing"/>
        <meta property="og:description" content="The striped field mouse, Apodemus agrarius, is the most common mammal in Korea. Although microsatellite loci for the species have been identified from populations in southwestern China, amplification of those markers for Korean populations have been unsuccessful. The complicated taxonomy of Korean striped field mouse including populations on Jeju Island (A. a chejuensis) necessitates identification of additional molecular markers. We applied 454 pyrosequencing systems to develop a suite of microsatellite markers. Muscle tissue was harvested and sequenced from 30 Jeju striped field mouse specimens which yielded 12,165 reads with a mean length per read of 287&amp;nbsp;bp. From these reads, we identified 17 microsatellite loci for A. a. chejuensis and tested these new markers against samples of both A. a chejuensis and A. a coreae, the mainland taxon. All 17 loci were amplified successfully for both taxa. Of the total 17 loci, one locus failed to amplify for a population on Heuksan Island. The cross-species transferability was also tested with the allied taxon, A. peninsulae and confirmed successful for 12 loci. These newly developed markers will benefit studies of genetic structure, evolution, and resolving taxonomic problems of striped field mice and allied taxa in Korea."/>
        <meta property="og:image" content="https://images.springer.com/sgw/journals/medium/41065.jpg"/>
    
    </head>

    <body class="journal journal-fulltext"
          
          >
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK"
                            height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><feColorMatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"/></filter></defs><symbol id="icon-arrow-down" viewBox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"/></symbol><symbol id="icon-arrow-left-bullet" viewBox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"/></symbol><symbol id="icon-arrow-right-bullet" viewBox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"/></symbol><symbol id="icon-arrow-right" viewBox="0 0 32 64"><path d="M0 0v64l32-32z"/></symbol><symbol id="icon-close" viewBox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"/></symbol><symbol id="icon-community" viewBox="0 0 60.643 31.01"><mask maskUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><path fill="#FFF" d="M0 31.01h60.643V0H0z" filter="url(#a)"/></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"/></symbol><symbol id="icon-download" viewBox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"/></symbol><symbol id="icon-ext-link" viewBox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"/><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"/><path fill="#676767" d="M9.7 0L16 6.2V0z"/></symbol><symbol id="icon-info-bordered" viewBox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"/><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"/></symbol><symbol id="icon-opr" viewBox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"/><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"/></symbol><symbol id="icon-quotation" viewBox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"/></symbol><symbol id="icon-remove" viewBox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"/><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"/></symbol><symbol id="icon-ribbon" viewBox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"/></symbol><symbol id="icon-rss" viewBox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"/><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"/><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"/></g></symbol><symbol id="icon-search" viewBox="0 0 16 16"><path d="M6.276 12.257C2.815 12.257 0 9.508 0 6.128 0 2.749 2.815 0 6.276 0c3.462 0 6.277 2.749 6.277 6.128.002 3.38-2.815 6.129-6.277 6.129zm0-10.772c-2.622 0-4.755 2.083-4.755 4.643 0 2.562 2.133 4.644 4.755 4.644 2.623 0 4.756-2.082 4.756-4.644 0-2.56-2.134-4.643-4.756-4.643z"/><path d="M16 13.751L13.641 16l-4.875-4.82 2.36-2.248L16 13.751z"/></symbol><symbol id="icon-speech-bubble" viewBox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"/></symbol><symbol id="icon-error" viewBox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"/></symbol><symbol id="icon-info" viewBox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"/></symbol><symbol id="icon-success" viewBox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"/></symbol><symbol id="icon-warning" viewBox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"/></symbol><symbol id="icon-facebook-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"/></symbol><symbol id="icon-facebook" viewBox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"/><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"/></symbol><symbol id="icon-gplus-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M480.5 273.117c.123.076.26.185.416.334.146.152.289.343.432.562.135.211.258.459.359.746.086.286.129.619.129 1.004-.012.7-.164 1.262-.465 1.683a5.374 5.374 0 0 1-.461.562c-.186.176-.375.35-.58.527a2.65 2.65 0 0 0-.322.41.908.908 0 0 0-.176.553c0 .205.059.375.18.508.102.129.199.238.299.336l.662.545a6.48 6.48 0 0 1 1.086 1.119c.293.415.445.955.459 1.625 0 .949-.42 1.789-1.256 2.521-.869.76-2.123 1.15-3.762 1.177-1.373-.015-2.4-.308-3.074-.879-.682-.534-1.023-1.172-1.023-1.917 0-.359.111-.766.336-1.211.215-.443.602-.837 1.172-1.174a6.126 6.126 0 0 1 1.998-.725 15.415 15.415 0 0 1 1.723-.173 3.996 3.996 0 0 1-.377-.596 1.347 1.347 0 0 1-.193-.74c0-.174.023-.317.074-.433a8.83 8.83 0 0 1 .121-.338 5.973 5.973 0 0 1-.633.035c-1.043-.012-1.838-.339-2.381-.979-.574-.603-.861-1.3-.861-2.097 0-.961.404-1.832 1.217-2.616.559-.46 1.135-.757 1.736-.898a8.494 8.494 0 0 1 1.67-.182h3.916l-1.209.704h-1.207v.005l-.005.002zm.754 11.276c0-.498-.162-.932-.484-1.302-.346-.348-.881-.778-1.609-1.29a4.045 4.045 0 0 0-.439-.02c-.102-.016-.354 0-.768.039a8.141 8.141 0 0 0-1.244.275c-.1.036-.24.096-.42.17a2.713 2.713 0 0 0-.553.355 1.985 1.985 0 0 0-.459.592c-.145.25-.215.551-.215.901 0 .687.311 1.253.932 1.698.596.444 1.404.68 2.432.688.928-.014 1.629-.217 2.117-.612.476-.385.71-.886.71-1.494zm-2.729-5.743a1.843 1.843 0 0 0 1.289-.556c.164-.249.271-.506.318-.767a7.32 7.32 0 0 0 .041-.659c0-.772-.195-1.552-.594-2.336a2.611 2.611 0 0 0-.73-.921 1.894 1.894 0 0 0-1.064-.352 1.723 1.723 0 0 0-1.324.64c-.299.437-.438.92-.428 1.457 0 .709.209 1.444.623 2.211.201.356.459.659.775.912.323.248.686.371 1.094.371zm9.698.077h-2.055v-2.056h-.996v2.056h-2.053v.998h2.053v2.052h.996v-2.052h2.055z"/></symbol><symbol id="icon-gplus" viewBox="0 0 24 24"><circle fill="#D84A36" cx="12" cy="12" r="12"/><g fill="#FFF"><path d="M12.012 12.111c-.304-.33-.07-.842.155-1.078.226-.241.639-.56.829-.74.193-.184.66-.605.674-1.868 0 0 .112-1.139-1.127-2.06h.977l1.038-.593h-3.507s-2.083-.067-3.085 1.207c-1 1.276-.966 2.871.114 3.77 1.08.898 2.371.648 2.371.648s-.197.422-.163.775c.036.353.421.849.504.965 0 0-2.386-.011-3.551 1.033-1.165 1.053-1.29 3.097 1.021 3.803 2.312.707 5.303-.055 5.906-2.277 0 0 .262-.908-.189-1.683-.459-.775-1.663-1.571-1.967-1.902zm-2.555-1.83c-.744-1-1.308-2.953 0-3.783 0 0 .955-.579 1.801.266.846.846 1.082 2.277.928 3.003-.15.727-.563.897-.739 1.008-.185.111-1.249.504-1.99-.494zm3.668 5.481c-.035.393-.083 1.254-1.491 1.664-1.409.408-3.826-.264-3.757-1.914 0 0 .006-.698.449-1.089.445-.396 1.094-.794 3.029-.819 0 0 .643.473 1.072.809.427.332.733.952.698 1.349z"/><path d="M16.323 11.055V9.803h-.752v1.252h-1.255v.748h1.255v1.256h.752v-1.256h1.248v-.748z"/></g></symbol><symbol id="icon-linkedin" viewBox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"/><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"/><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"/></g></symbol><symbol id="icon-reddit" viewBox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"/><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"/><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"/><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"/><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"/><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"/><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"/><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"/></symbol><symbol id="icon-twitter-bordered" viewBox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"/><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/></symbol><symbol id="icon-twitter" viewBox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"/><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"/></symbol><symbol id="icon-weibo-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"/><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"/><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"/></symbol><symbol id="icon-weibo" viewBox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"/><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"/><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"/><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"/><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"/><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"/></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner c-banner--info">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner" >
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1"
                 data-gpt-unitpath="/270604982/bmc/hereditasjournal/articles"
                 data-gpt-sizes="728x90,970x90"
                 data-gpt-targeting="pos=LB1;doi=10.1186/s41065-018-0070-8;kwrd=Apodemus agrarius chejuensis, Microsatellites, South Korea, Jeju Island, Pyrosequencing;pmc=L32030,L21001;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-018-0070-8&amp;kwrd=Apodemus agrarius chejuensis, Microsatellites, South Korea, Jeju Island, Pyrosequencing&amp;pmc=L32030,L21001&amp;">
                        <img data-test="gpt-advert-fallback-img"
                             src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-018-0070-8&amp;kwrd=Apodemus agrarius chejuensis, Microsatellites, South Korea, Jeju Island, Pyrosequencing&amp;pmc=L32030,L21001&amp;"
                             alt="Advertisement"
                             width="728"
                             height="90">
                    </a>
                </noscript>
            </div>
        </div>
    </div>

        
         
    <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>


<header class="c-popup-search ctx-publisher-header" id="top" data-test="publisher-header">
    <nav class="c-navbar">
        <div class="u-container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization" />
                <div class="c-logo">
    <a href="https://www.biomedcentral.com" itemprop="url">
        <img src=/static/images/bmc/logos/logo-bmc-white-aj-e3a70ae848.svg alt="BMC" itemprop="logo">
        <div class="c-logo__strapline">
            <img src=/static/images/bmc/logos/logo-bmc-white-strapline-sn-e7b1d6e67f.svg alt="Part of Springer Nature">
        </div>
    </a>
</div>
            </div>
            <button
                type="button"
                class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg"
                id="publisher-header-menu-button"
                data-toggle="collapse"
                data-test="publisher-header-menu-toggle"
                data-toggle-linked="publisher-header"
                data-target="publisher-header-search publisher-header-menu"
                aria-controls="publisher-header-menu publisher-header-search"
                aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


                <ul class="c-navbar__nav">
                    <li class="c-navbar__item">
                        <a data-replace="true"
                           data-placeholder="/placeholder/v1/username"
                           class="c-navbar__link"
                           href="https://www.biomedcentral.com/login"
                           data-test="login-link"
                           data-track="click"
                           data-track-category="Account"
                           data-track-action="Click Login Link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                    <li class="c-navbar__item">
                        <button
                            type="button"
                            class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg"
                            data-toggle="collapse"
                            data-test="header-search-button"
                            data-target="publisher-header-search publisher-header-menu publisher-header-menu-button"
                            aria-controls="publisher-header-search publisher-header-menu"
                            aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true">
                                    <use xlink:href="#icon-search"></use>
                                </svg>
                            </span>
                        </button>
                    </li>
                </ul>
            </div>
        </div>
    </nav>
    <div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field c-form-field--inline" method="GET" action="//www.biomedcentral.com/search" data-track="submit"
        data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label data-track-label="" data-test="global-search">
        <label for="publisherSearch" class="u-visually-hidden">Search all BMC articles</label>
        <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text" value="" placeholder="Search all BMC articles"/>
        <button class="c-button c-button--secondary" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true">
        <use xlink:href="#icon-search"></use>
    </svg>
</button>


        <input type="hidden" name="searchType" value="publisherSearch"/>
    </form>
</div>

            </div>
        </div>
    </div>
</header>


        
            <header class="c-journal-header c-journal-header--hereditas ctx-journal-header">
                <div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="/">
    
        
    
    <span class="c-journal-title__text ">Hereditas</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="u-container">
                        
                            <button type="button"
                                    class="c-navbar__link c-dropdown u-hidden-at-lg"
                                    data-toggle="collapse"
                                    data-test="journal-header-menu"
                                    data-toggle-linked="journal-header"
                                    data-target="journal-menu"
                                    aria-controls="journal-menu"
                                    aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav">
                                    <ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/'>Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/about'>About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href='/articles'>Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/submission-guidelines'>Submission Guidelines</a>
                                            </li>
                                        
                                    </ul>
                                </nav>
                            </div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--ocean-light-blue">
                    
                </div>
            </header>
            
        

        <div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-content-layout c-content-layout--fulltext" data-component="sticky-container">
                <div class="c-content-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3><ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc=""><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Introduction">Introduction</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec5" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results &amp; discussion">Results &amp; discussion</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li></ol></nav>
                </div>

                <main class="c-content-layout__main">
                    <article itemscope itemtype="http://schema.org/ScholarlyArticle">
                        
                        
    <div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
            <li class="c-article-identifiers__item" data-test="article-category">Brief report</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
    </div>

                        <div class="FulltextWrapper"><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">Development of 17 polymorphic microsatellite loci from Jeju striped field mouse, <em xmlns="" class="EmphasisTypeItalic">Apodemus agrarius chejuensis</em> (Rodentia: Muridae), by 454 pyrosequencing</h1></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true"><ul class="u-listReset"><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Han-Na&nbsp;Kim</span><sup><a href="#Aff1" class="AffiliationID">1</a>, <a href="#Aff2" class="AffiliationID">2</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Han-Ul&nbsp;Kim</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Yeong-Seok&nbsp;Jo</span><sup><a href="#Aff2" class="AffiliationID">2</a></sup><a href="mailto:biostan@hotmail.com" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a><a class="Orcid" title="View ORCID ID profile" href="http://orcid.org/0000-0002-0711-3871" target="_blank" rel="noopener"><span class="OrcidIcon" aria-hidden="true"></span><span class="u-srOnly">View ORCID ID profile</span></a> and </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Jongwoo&nbsp;Jung</span><sup><a href="#Aff1" class="AffiliationID">1</a>, <a href="#Aff3" class="AffiliationID">3</a></sup><a href="mailto:jongwoo@ewha.ac.kr" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a></li></ul></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation"><div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">Hereditas</span></span><span class="ArticleCitation_Year">2018</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">155</strong>:30</span></span></div><p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/s41065-018-0070-8">https://doi.org/10.1186/s41065-018-0070-8</a></p><p class="Copyright">© &nbsp;The Author(s)&nbsp;2018</p></div></div><ul class="ArticleHistory u-reset-list"><li class="History HistoryReceived"><strong>Received: </strong><span>24&nbsp;April&nbsp;2018</span></li><li class="History HistoryAccepted"><strong>Accepted: </strong><span>20&nbsp;September&nbsp;2018</span></li><li class="History HistoryOnlineDate"><strong>Published: </strong><span itemprop="datePublished">26&nbsp;September&nbsp;2018</span></li></ul></div>
                        
                        
                        
    

                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2><div class="c-section__content" id="Abstract"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p id="Par1" class="Para">The striped field mouse, <em xmlns="" class="EmphasisTypeItalic">Apodemus agrarius</em>, is the most common mammal in Korea. Although microsatellite loci for the species have been identified from populations in southwestern China, amplification of those markers for Korean populations have been unsuccessful. The complicated taxonomy of Korean striped field mouse including populations on Jeju Island (<em xmlns="" class="EmphasisTypeItalic">A. a chejuensis</em>) necessitates identification of additional molecular markers.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Findings</h3>
                <p id="Par2" class="Para">We applied 454 pyrosequencing systems to develop a suite of microsatellite markers. Muscle tissue was harvested and sequenced from 30 Jeju striped field mouse specimens which yielded 12,165 reads with a mean length per read of 287&nbsp;bp. From these reads, we identified 17 microsatellite loci for <em xmlns="" class="EmphasisTypeItalic">A. a. chejuensis</em> and tested these new markers against samples of both <em xmlns="" class="EmphasisTypeItalic">A. a chejuensis</em> and <em xmlns="" class="EmphasisTypeItalic">A. a coreae</em>, the mainland taxon. All 17 loci were amplified successfully for both taxa. Of the total 17 loci, one locus failed to amplify for a population on Heuksan Island. The cross-species transferability was also tested with the allied taxon, <em xmlns="" class="EmphasisTypeItalic">A. peninsulae</em> and confirmed successful for 12 loci.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusions</h3>
                <p id="Par3" class="Para">These newly developed markers will benefit studies of genetic structure, evolution, and resolving taxonomic problems of striped field mice and allied taxa in Korea.</p>
              </div></div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords"><li class="c-keywords__item" itemprop="about">
                        <em xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="EmphasisTypeItalic">Apodemus agrarius chejuensis</em>, Microsatellites, South Korea, Jeju Island, Pyrosequencing</li></ul></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Introduction" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Introduction">Introduction</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Introduction" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par13" class="Para">With 24 subspecies, the striped field mouse, <em xmlns="" class="EmphasisTypeItalic">Apodemus agrarius</em> (Pallas, 1771) is widely distributed throughout Eurasia [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>]. Striped field mice are the most abundant mammal in Korea with an important place in the ecosystems as a primary prey item for predators and a vector of seed dispersal [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>]. In Korea, four subspecies (<em xmlns="" class="EmphasisTypeItalic">A. a mantchuricus, A. a. chejuensis, A. a. pallenscens</em>, and <em xmlns="" class="EmphasisTypeItalic">A. a. coreae</em>) have been reported, but only <em xmlns="" class="EmphasisTypeItalic">A. a. chejuensis</em> on Jeju Island has a definitive geographic barrier from the other subspecies [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>]. Based on differences in morphology [<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>–<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>], physiology [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>], and mitochondrial DNA [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>–<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>], the taxonomic status of <em xmlns="" class="EmphasisTypeItalic">A. a. chejuensis</em> has changed frequently in rank between species and subspecies. The striped field mouse is the first known vector of Hanta virus, which killed roughly 3000 United Nations soldiers per year during the Korean War, which has attracted great attention by epidemiologists [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>]. Because hemorrhagic fever never occurred in Jeju Island, the island subspecies was used as a control for the isolation of Hanta virus [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par14" class="Para">Simple Sequence Repeats (SSRs, microsatellites) are one of the most common and efficient nuclear DNA markers for population genetics and phylogenetic studies of recently diverged taxa because they are multiallelic, highly polymorphic, abundant and show a codominant mode of inheritance [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>, <span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>]. However, development of microsatellite markers has been costly, ineffective, and time consuming [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>]. The application of next generation sequencing technology has greatly improved these challenges over the past few years [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par15" class="Para">To address ongoing taxonomic debates, the status of <em xmlns="" class="EmphasisTypeItalic">A. a. chejuensis</em> and its relationship to mainland species should be examined using both haplotypic (mitochondrial DNA markers) and genotypic molecular markers (nuclear DNA markers). Wu et al. [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>] developed 14 loci of microsatellite from populations in southwestern China; however, only five of 14 loci were successfully amplified in Korean striped field mouse samples [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>]. Thus, nearly two thirds of the microsatellite markers available for the species failed to amplify in the targeted Korean populations [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>]. Given the low transferability of the existing markers, we hypothesized that a limited proportion of markers developed for <em xmlns="" class="EmphasisTypeItalic">A. a chejuensis</em> will be applicable for the mainland taxon, <em xmlns="" class="EmphasisTypeItalic">A. a coreae</em> assuming their taxonomic status is different. Therefore, we developed 17 microsatellite loci from <em xmlns="" class="EmphasisTypeItalic">A. a. chejuensis</em> for future investigation of genetic properties of the striped field mouse and then tested cross transferability for the markers against <em xmlns="" class="EmphasisTypeItalic">A. a coreae</em> samples from the mainland.</p><section id="Sec2" class="Section2 RenderAsSection2">
                <h3 class="Heading">Sample preparation and genotyping</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par16" class="Para">We used muscle tissues of <em xmlns="" class="EmphasisTypeItalic">A. a. chejuensis</em> collected on Jeju Island to develop microsatellites loci. We collected 15 individuals each at Yongdu-dong (YD), Jeju-si (N33°31′08.72″ E126°29′47.77″) and 15 individuals at Beophwan-dong (BH), Seogwipo-si, Korea (N33°24′42.97″ E126°51′52.74″). We captured mice in Sherman traps following guidelines of the American Society of Mammalogists [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>]. Specimens were deposited in the mammal collection at the National Institute of Biological Resources, Korea (NIBRMM0000105863).</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par17" class="Para">Genomic DNA was extracted using DNeasy Blood &amp; Tissue Kit (Qiagen, Valencia, CA, USA). The isolated DNA was visualized on 1% agarose gel and quantified in NanoDrop 2000 (Thermo Fisher Scientific, Waltham, MA, USA). We used a total amount of over 2&nbsp;μg with concentration of 50&nbsp;ng/μL or higher for 454 pyrosequencing library preparation. Library preparation, amplification, and sequencing were conducted at Macrogen Inc. (Seoul, Korea). Briefly, after DNA quality control steps, isolated genomic DNAs were fragmented using restriction enzymes and tagged them with two multiplex identifiers. After amplification of DNA fragments, the prepared libraries were pooled and sequenced in 454 GS FLX platform (Roche).</p>
              </section><section id="Sec3" class="Section2 RenderAsSection2">
                <h3 class="Heading">Primer design and testing makers</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par18" class="Para">Candidate loci for microsatellites were selected using QDD3 software [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>]. A total of 335 primer pairs were obtained for candidate loci using PRIMER3 [<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>]. Each locus was then evaluated with PCR amplification performed with a total volume of 25&nbsp;μl containing 1.0&nbsp;μl of genomic DNA, 2.5&nbsp;μl of 10× buffer (Takara), 0.7&nbsp;μl of dNTP (2.5&nbsp;mM each), 1.5&nbsp;μl of MgCl<sub>2</sub>, 0.5&nbsp;μl of primers each, and 0.3&nbsp;U of Taq polymerase (Takara r-Taq, Takara). PCR conditions were follows: initial denaturation at 95&nbsp;°C for 5&nbsp;min followed by 35&nbsp;cycles of denaturation at 94&nbsp;°C for 60&nbsp;s, annealing at 45–60&nbsp;°C for 60&nbsp;s, elongation at 72&nbsp;°C for 90&nbsp;s, and a final extension at 72&nbsp;°C for 10&nbsp;min. PCR products were selected and sequenced to confirm whether they contained microsatellite sequences.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par19" class="Para">A M13-tail (FAM: 5’-TTTCCCAGTCACGACGTTG-3′, VIC: 5’-GGAAACAGCTATGACCA-3′, PET: 5’-GCGGATAACAATTTCACACAGG-3′, NED: 5’-TAAAACGACGGCCAGTGC-3′) was added to the 5′ end of each forward primers while a pig tail (GTTTCTT) was added to the 5′ end of each reverse primers. For multiplex PCR, we performed PCR amplifications in a total reaction volume of 16&nbsp;μl containing 5&nbsp;μl of 2× QIAGEN Multiplex PCR master mix (QIAGEN), 0.08&nbsp;μl of M13-tailed forward primer, 0.8&nbsp;μl of pig tailed reverse primer, 0.3&nbsp;μl of templated DNA and 0.16&nbsp;μl of each of fluorescence primer (FAM/VIC/PET/NED). Multiplex PCR cycling conditions were as follows: initial denaturation at 95&nbsp;°C for 15&nbsp;min followed by denaturation at 95&nbsp;°C for 30&nbsp;s, annealing (14&nbsp;cycles at 63&nbsp;°C, 7&nbsp;cycles at 58&nbsp;°C, 20&nbsp;cycles at 55&nbsp;°C) for 90&nbsp;s and 72&nbsp;°C for 30&nbsp;s, with a final elongation at 72&nbsp;°C for 20&nbsp;min.).</p>
              </section><section id="Sec4" class="Section2 RenderAsSection2">
                <h3 class="Heading">Genotyping and population analysis</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par20" class="Para">We genotyped 30 Jeju striped field mice (<em xmlns="" class="EmphasisTypeItalic">A. a chejuensis</em>) from two populations (YD and BH) with the newly designed microsatellites. The amplified fragments were separated in ABI 3730xl DNA analyzer (Applied Biosystem, Inc., Forster City, CA, USA) and GeneMapper software v3.7 (Applied Biosystems) was used for manual scoring.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par21" class="Para">To test for cross-species transferability, we amplified one congener (<em xmlns="" class="EmphasisTypeItalic">Agrarius peninsulae</em>) and one subspecies (<em xmlns="" class="EmphasisTypeItalic">A. a. coreae</em>) using the 17 developed markers with the same PCR conditions. Additionally, we added an <em xmlns="" class="EmphasisTypeItalic">A. agrarius</em> sample collected from Heuksan Island in the cross-species transferability run because Jo et al. [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>] suggested that the <em xmlns="" class="EmphasisTypeItalic">A. agrarius</em> population in Heuksan Island might be a newly diverged subspecies from the mainland populations (<em xmlns="" class="EmphasisTypeItalic">A. a. coreae</em>) in Korea.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par22" class="Para">The following genetic diversity parameters were calculated using ARLEQUIN 3.5 [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>]: the number of alleles at each locus (<em xmlns="" class="EmphasisTypeItalic">A</em>), observed heterozygosity (<em xmlns="" class="EmphasisTypeItalic">H</em><sub>O</sub>), expected heterozygosity (<em xmlns="" class="EmphasisTypeItalic">H</em><sub>E</sub>) and population divergence (<em xmlns="" class="EmphasisTypeItalic">F</em><sub>ST</sub>) between the two populations (BH and YD). Polymorphic information contents (PIC) were also estimated in CENES software (<span class="ExternalRef"><a href="http://www.ufv.br/dbg/genes/genes.htm"><span class="RefSource">http://www.ufv.br/dbg/genes/genes.htm</span></a></span>; Cruz 2007). We tested whether significant deviation from Hardy-Weinberg equilibrium (HWE) within each population across all amplified loci in ARLEQUIN 3.5. Linkage disequilibrium (LD) between pairs of loci within each population was tested using GENEPOP 4.2.1 [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>, <span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>]. All statistical significances of multiple comparisons were adjusted with sequential Bonferroni corrections [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>]. We performed Analysis of Molecular Variance (AMOVA) to hierarchically partition the total genetic variance within and between the two populations in GENALEX v. 6.5 [<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>]. We also carried out principal coordinate analysis (PCoA) based on pairwise F<sub>ST</sub> for all 30 individuals to examine clustering pattern among the individual genotypes in GENALEX v. 6.5 [<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>].</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec5" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Resultsdiscussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Resultsdiscussion">Results &amp; discussion</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Resultsdiscussion" data-test="collapse-section"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par23" class="Para">Four hundred and fifty-four pyrosequencing produced 12,165 reads with a mean length of 287&nbsp;bp. We selected 335 candidate microsatellite loci from the produced reads and confirmed that 17 loci were polymorphic (Table&nbsp;<span class="InternalRef"><a href="#Tab1">1</a></span>).<figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 1</span>
                      <p xmlns="" class="SimplePara">Characterization of 17 microsatellites of <em class="EmphasisTypeItalic">Apodemus agrarius chejuensis</em></p>
                    </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col></col><col></col><col></col><col></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Locus</p></th><th><p xmlns="" class="SimplePara">Repeat motif</p></th><th><p xmlns="" class="SimplePara">Repeat number range</p></th><th><p xmlns="" class="SimplePara">Primer sequence (5′-3′)</p></th><th><p xmlns="" class="SimplePara">Dye</p></th><th><p xmlns="" class="SimplePara">Size range (bp)</p></th></tr></thead><tbody><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">AC2&nbsp;L</p></td><td rowspan="2"><p xmlns="" class="SimplePara">TGTA</p></td><td rowspan="2" char="–"><p xmlns="" class="SimplePara">5–10</p></td><td><p xmlns="" class="SimplePara">F: TTTCCCAGTCACGACGTTGGAACCTCAAAGAAGTGGGCTT</p></td><td rowspan="2"><p xmlns="" class="SimplePara">6FAM</p></td><td rowspan="2" char="–"><p xmlns="" class="SimplePara">213–233</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R: GTTTCTTGGGATCATCAGCAGATAGCAG</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">ACH7</p></td><td rowspan="2"><p xmlns="" class="SimplePara">AACC</p></td><td rowspan="2" char="–"><p xmlns="" class="SimplePara">5–9</p></td><td><p xmlns="" class="SimplePara">F: GCCCTGTACTTACCAACTCCC</p></td><td rowspan="2"><p xmlns="" class="SimplePara">6FAM</p></td><td rowspan="2" char="–"><p xmlns="" class="SimplePara">139–155</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R: GGTAGGTCAATGAGTTGGGTTC</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">ACW7</p></td><td rowspan="2"><p xmlns="" class="SimplePara">CA</p></td><td rowspan="2" char="–"><p xmlns="" class="SimplePara">17–27</p></td><td><p xmlns="" class="SimplePara">F: TTTCCCAGTCACGACGTTGCAAGCTTGGGATCGCAGT</p></td><td rowspan="2"><p xmlns="" class="SimplePara">6FAM</p></td><td rowspan="2" char="–"><p xmlns="" class="SimplePara">211–231</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R: GTTTCTTGGCCCTTCCTGCATACTTTG</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">AC9S</p></td><td rowspan="2"><p xmlns="" class="SimplePara">CA</p></td><td rowspan="2" char="–"><p xmlns="" class="SimplePara">18–25</p></td><td><p xmlns="" class="SimplePara">F: TAAAACGACGGCCAGTGCTTTGTCCTTGCAAACTACCC</p></td><td rowspan="2"><p xmlns="" class="SimplePara">6FAM</p></td><td rowspan="2" char="–"><p xmlns="" class="SimplePara">260–274</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R: GTTTCTTACCAAGATTGTAAGATGGCTGAA</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">ACAQ</p></td><td rowspan="2"><p xmlns="" class="SimplePara">TG</p></td><td rowspan="2" char="–"><p xmlns="" class="SimplePara">15–31</p></td><td><p xmlns="" class="SimplePara">F: TTTCCCAGTCACGACGTTGGCTTGGGATCATGTCCACTT</p></td><td rowspan="2"><p xmlns="" class="SimplePara">6FAM</p></td><td rowspan="2" char="–"><p xmlns="" class="SimplePara">162–194</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R: GTTTCTTGACTGTTTAACTTGACTCTAGATTGTG</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">ACWV</p></td><td rowspan="2"><p xmlns="" class="SimplePara">CCA</p></td><td rowspan="2" char="–"><p xmlns="" class="SimplePara">6–11</p></td><td><p xmlns="" class="SimplePara">F: TTTCCCAGTCACGACGTTGGCTTGGGATCAGTTGCTAGG</p></td><td rowspan="2"><p xmlns="" class="SimplePara">6FAM</p></td><td rowspan="2" char="–"><p xmlns="" class="SimplePara">316–331</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R: GTTTCTTTGTGAACACTGATGTTTATGTTCC</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">ACAV</p></td><td rowspan="2"><p xmlns="" class="SimplePara">AG</p></td><td rowspan="2" char="–"><p xmlns="" class="SimplePara">5–9</p></td><td><p xmlns="" class="SimplePara">F: TTTCCCAGTCACGACGTTGGGATCAAAGCCAGTCCTGAG</p></td><td rowspan="2"><p xmlns="" class="SimplePara">6FAM</p></td><td rowspan="2" char="–"><p xmlns="" class="SimplePara">192–200</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R: GTTTCTTTTCTATCTTTAGGTACTCTGTTTCCC</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">ACU8</p></td><td rowspan="2"><p xmlns="" class="SimplePara">AACC</p></td><td rowspan="2" char="–"><p xmlns="" class="SimplePara">6–18</p></td><td><p xmlns="" class="SimplePara">F: TTGGGATCAATCAGTCAGTCAG</p></td><td rowspan="2"><p xmlns="" class="SimplePara">HEX</p></td><td rowspan="2" char="–"><p xmlns="" class="SimplePara">291–339</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R: TGCATGAATTACTAAACAGAGTATCAA</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">AC9&nbsp;M</p></td><td rowspan="2"><p xmlns="" class="SimplePara">CA</p></td><td rowspan="2" char="–"><p xmlns="" class="SimplePara">4–5</p></td><td><p xmlns="" class="SimplePara">F: GGAAACAGCTATGACCACCACGTTACACACACCATGC</p></td><td rowspan="2"><p xmlns="" class="SimplePara">VIC</p></td><td rowspan="2" char="–"><p xmlns="" class="SimplePara">165–167</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R: GTTTCTTGTGGTACAGGTAAAGCGTGGG</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">AC124</p></td><td rowspan="2"><p xmlns="" class="SimplePara">TG</p></td><td rowspan="2" char="–"><p xmlns="" class="SimplePara">6–8</p></td><td><p xmlns="" class="SimplePara">F: GGAAACAGCTATGACCATTCCCGTCTGAGTGAAGACATG</p></td><td rowspan="2"><p xmlns="" class="SimplePara">VIC</p></td><td rowspan="2" char="–"><p xmlns="" class="SimplePara">278–282</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R: GTTTCTTGCACACTCTGCTTTTTCTTGGG</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">ACEC</p></td><td rowspan="2"><p xmlns="" class="SimplePara">ATGA</p></td><td rowspan="2" char="–"><p xmlns="" class="SimplePara">5–10</p></td><td><p xmlns="" class="SimplePara">F: GGAAACAGCTATGACCATGAGTGCAATGGCAGAAGTC</p></td><td rowspan="2"><p xmlns="" class="SimplePara">VIC</p></td><td rowspan="2" char="–"><p xmlns="" class="SimplePara">148–168</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R: GTTTCTTGGCCTCACGAGCATACAGA</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">ACQT</p></td><td rowspan="2"><p xmlns="" class="SimplePara">AG</p></td><td rowspan="2" char="–"><p xmlns="" class="SimplePara">7–12</p></td><td><p xmlns="" class="SimplePara">F: GGAAACAGCTATGACCATCTTTGTTTGACACCCACCA</p></td><td rowspan="2"><p xmlns="" class="SimplePara">VIC</p></td><td rowspan="2" char="–"><p xmlns="" class="SimplePara">160–170</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R: GTTTCTTGGTTCCTCACTCCCATTCAA</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">ACHZ</p></td><td rowspan="2"><p xmlns="" class="SimplePara">TTTTG</p></td><td rowspan="2" char="–"><p xmlns="" class="SimplePara">7–16</p></td><td><p xmlns="" class="SimplePara">F: GCGGATAACAATTTCACACAGGTCCCTTGAAGAGTGCTCACC</p></td><td rowspan="2"><p xmlns="" class="SimplePara">PET</p></td><td rowspan="2" char="–"><p xmlns="" class="SimplePara">312–357</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R: GTTTCTTCCTGAGGAGACCCATCCTTA</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">AC53</p></td><td rowspan="2"><p xmlns="" class="SimplePara">AAAC</p></td><td rowspan="2" char="–"><p xmlns="" class="SimplePara">7–15</p></td><td><p xmlns="" class="SimplePara">F: GCGGATAACAATTTCACACAGGTTCAGAGGGTATACTGGGGAGG</p></td><td rowspan="2"><p xmlns="" class="SimplePara">PET</p></td><td rowspan="2" char="–"><p xmlns="" class="SimplePara">313–345</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R: GTTTCTTCAGTGTGGGCACTGTGAAAT</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">ACLI</p></td><td rowspan="2"><p xmlns="" class="SimplePara">TG</p></td><td rowspan="2" char="–"><p xmlns="" class="SimplePara">5–6</p></td><td><p xmlns="" class="SimplePara">F: TAAAACGACGGCCAGTGCGGCCACGCTAACAATCTCAT</p></td><td rowspan="2"><p xmlns="" class="SimplePara">NED</p></td><td rowspan="2" char="–"><p xmlns="" class="SimplePara">231–233</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R: GTTTCTTTGTTATGAATGCCCTGACCA</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">AC5S</p></td><td rowspan="2"><p xmlns="" class="SimplePara">AC</p></td><td rowspan="2" char="–"><p xmlns="" class="SimplePara">20–49</p></td><td><p xmlns="" class="SimplePara">F: TAAAACGACGGCCAGTGCTGGGATCAGTTAATTCAAGGC</p></td><td rowspan="2"><p xmlns="" class="SimplePara">NED</p></td><td rowspan="2" char="–"><p xmlns="" class="SimplePara">153–211</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R: GTTTCTTTCTGTAAGGCCAGAGGGCTA</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">ACPJ</p></td><td rowspan="2"><p xmlns="" class="SimplePara">AC</p></td><td rowspan="2" char="–"><p xmlns="" class="SimplePara">10–36</p></td><td><p xmlns="" class="SimplePara">F: TAAAACGACGGCCAGTGCTATCATCCCATAGCAGGCAGA</p></td><td rowspan="2"><p xmlns="" class="SimplePara">NED</p></td><td rowspan="2" char="–"><p xmlns="" class="SimplePara">132–184</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R: GTTTCTTAGCTTCTGTGATGGGATGGT</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                    <p xmlns="" class="SimplePara">F denote forward sequences and R denote revers sequences</p>
                  </div></figure></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par24" class="Para">The number of alleles per locus in the two populations of <em xmlns="" class="EmphasisTypeItalic">A. a. chejuensis</em> varied from 2 to 11 in YD and 2–13 in BH. Ranges of <em xmlns="" class="EmphasisTypeItalic">H</em><sub>E</sub> and <em xmlns="" class="EmphasisTypeItalic">H</em><sub>O</sub> in YD were 0.067–0.905 and 0.067–0.867, respectively. In BH, <em xmlns="" class="EmphasisTypeItalic">H</em><sub>E</sub> and <em xmlns="" class="EmphasisTypeItalic">H</em><sub>O</sub> were 0.067–0.913 and 0.067–0.917, respectively (Table&nbsp;<span class="InternalRef"><a href="#Tab2">2</a></span>). The value of polymorphic information content ranged from 0.067 to 0.856 with a mean of 0.537 in YD. It ranged from 0.062 to 0.872 with a mean of 0.672 in BH. Three loci in YD and four loci in BH were significantly deviated from HWE after Bonferroni correction (<em xmlns="" class="EmphasisTypeItalic">p &lt;</em>&nbsp;0.0001). No linkage disequilibrium was detected at any pair of loci.<figure xmlns="" id="Tab2" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 2</span>
                      <p xmlns="" class="SimplePara">Genetic diversity indices of two populations of <em class="EmphasisTypeItalic">Apodemus agrarius chejuensis</em> in Jeju Island, South Korea</p>
                    </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col></colgroup><thead><tr class="header"><th colspan="3">&nbsp;</th><th colspan="5"><p xmlns="" class="SimplePara">Yongdu-dong (YD)</p></th><th colspan="7"><p xmlns="" class="SimplePara">Beophwan-dong (BH)</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">Locus</p></td><td><p xmlns="" class="SimplePara">n</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">A</em>
                          </p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">H</em>
                            <sub xmlns="http://www.w3.org/1999/xhtml">E</sub>
                          </p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">H</em>
                            <sub xmlns="http://www.w3.org/1999/xhtml">O</sub>
                          </p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">F</em>
                            <sub xmlns="http://www.w3.org/1999/xhtml">IS</sub>
                          </p></td><td><p xmlns="" class="SimplePara">PIC</p></td><td><p xmlns="" class="SimplePara">HWE</p></td><td><p xmlns="" class="SimplePara">n</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">A</em>
                          </p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">H</em>
                            <sub xmlns="http://www.w3.org/1999/xhtml">E</sub>
                          </p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">H</em>
                            <sub xmlns="http://www.w3.org/1999/xhtml">O</sub>
                          </p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">F</em>
                            <sub xmlns="http://www.w3.org/1999/xhtml">IS</sub>
                          </p></td><td><p xmlns="" class="SimplePara">PIC</p></td><td><p xmlns="" class="SimplePara">HWE</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">AC2&nbsp;L</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">5</p></td><td><p xmlns="" class="SimplePara">0.605</p></td><td><p xmlns="" class="SimplePara">0.200</p></td><td><p xmlns="" class="SimplePara">0.677</p></td><td><p xmlns="" class="SimplePara">0.543</p></td><td><p xmlns="" class="SimplePara">*</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">4</p></td><td><p xmlns="" class="SimplePara">0.674</p></td><td><p xmlns="" class="SimplePara">0.467</p></td><td><p xmlns="" class="SimplePara">0.315</p></td><td><p xmlns="" class="SimplePara">0.587</p></td><td><p xmlns="" class="SimplePara">ns</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">ACH7</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">3</p></td><td><p xmlns="" class="SimplePara">0.248</p></td><td><p xmlns="" class="SimplePara">0.267</p></td><td><p xmlns="" class="SimplePara">−0.077</p></td><td><p xmlns="" class="SimplePara">0.227</p></td><td><p xmlns="" class="SimplePara">ns</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">5</p></td><td><p xmlns="" class="SimplePara">0.694</p></td><td><p xmlns="" class="SimplePara">0.667</p></td><td><p xmlns="" class="SimplePara">0.041</p></td><td><p xmlns="" class="SimplePara">0.629</p></td><td><p xmlns="" class="SimplePara">ns</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">ACW7</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">10</p></td><td><p xmlns="" class="SimplePara">0.905</p></td><td><p xmlns="" class="SimplePara">0.692</p></td><td><p xmlns="" class="SimplePara">0.242</p></td><td><p xmlns="" class="SimplePara">0.856</p></td><td><p xmlns="" class="SimplePara">ns</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">8</p></td><td><p xmlns="" class="SimplePara">0.884</p></td><td><p xmlns="" class="SimplePara">0.333</p></td><td><p xmlns="" class="SimplePara">0.633</p></td><td><p xmlns="" class="SimplePara">0.830</p></td><td><p xmlns="" class="SimplePara">*</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">AC9S</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">7</p></td><td><p xmlns="" class="SimplePara">0.779</p></td><td><p xmlns="" class="SimplePara">0.800</p></td><td><p xmlns="" class="SimplePara">−0.028</p></td><td><p xmlns="" class="SimplePara">0.729</p></td><td><p xmlns="" class="SimplePara">ns</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">7</p></td><td><p xmlns="" class="SimplePara">0.867</p></td><td><p xmlns="" class="SimplePara">0.667</p></td><td><p xmlns="" class="SimplePara">0.237</p></td><td><p xmlns="" class="SimplePara">0.814</p></td><td><p xmlns="" class="SimplePara">ns</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">ACAQ</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">11</p></td><td><p xmlns="" class="SimplePara">0.853</p></td><td><p xmlns="" class="SimplePara">0.867</p></td><td><p xmlns="" class="SimplePara">−0.017</p></td><td><p xmlns="" class="SimplePara">0.807</p></td><td><p xmlns="" class="SimplePara">ns</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">9</p></td><td><p xmlns="" class="SimplePara">0.865</p></td><td><p xmlns="" class="SimplePara">0.643</p></td><td><p xmlns="" class="SimplePara">0.264</p></td><td><p xmlns="" class="SimplePara">0.820</p></td><td><p xmlns="" class="SimplePara">ns</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">ACWV</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">4</p></td><td><p xmlns="" class="SimplePara">0.570</p></td><td><p xmlns="" class="SimplePara">0.333</p></td><td><p xmlns="" class="SimplePara">0.431</p></td><td><p xmlns="" class="SimplePara">0.497</p></td><td><p xmlns="" class="SimplePara">ns</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">–</p></td><td><p xmlns="" class="SimplePara">–</p></td><td><p xmlns="" class="SimplePara">–</p></td><td><p xmlns="" class="SimplePara">–</p></td><td><p xmlns="" class="SimplePara">–</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">ACAV</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">2</p></td><td><p xmlns="" class="SimplePara">0.480</p></td><td><p xmlns="" class="SimplePara">0.333</p></td><td><p xmlns="" class="SimplePara">0.314</p></td><td><p xmlns="" class="SimplePara">0.357</p></td><td><p xmlns="" class="SimplePara">ns</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">3</p></td><td><p xmlns="" class="SimplePara">0.549</p></td><td><p xmlns="" class="SimplePara">0.333</p></td><td><p xmlns="" class="SimplePara">0.402</p></td><td><p xmlns="" class="SimplePara">0.421</p></td><td><p xmlns="" class="SimplePara">ns</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">ACU8</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">4</p></td><td><p xmlns="" class="SimplePara">0.628</p></td><td><p xmlns="" class="SimplePara">0.267</p></td><td><p xmlns="" class="SimplePara">0.590</p></td><td><p xmlns="" class="SimplePara">0.557</p></td><td><p xmlns="" class="SimplePara">ns</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">7</p></td><td><p xmlns="" class="SimplePara">0.805</p></td><td><p xmlns="" class="SimplePara">0.800</p></td><td><p xmlns="" class="SimplePara">0.012</p></td><td><p xmlns="" class="SimplePara">0.753</p></td><td><p xmlns="" class="SimplePara">ns</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">AC9&nbsp;M</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">2</p></td><td><p xmlns="" class="SimplePara">0.067</p></td><td><p xmlns="" class="SimplePara">0.067</p></td><td><p xmlns="" class="SimplePara">0.000</p></td><td><p xmlns="" class="SimplePara">0.062</p></td><td><p xmlns="" class="SimplePara">ns</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">2</p></td><td><p xmlns="" class="SimplePara">0.186</p></td><td><p xmlns="" class="SimplePara">0.067</p></td><td><p xmlns="" class="SimplePara">0.650</p></td><td><p xmlns="" class="SimplePara">0.164</p></td><td><p xmlns="" class="SimplePara">ns</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">AC124</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">3</p></td><td><p xmlns="" class="SimplePara">0.191</p></td><td><p xmlns="" class="SimplePara">0.067</p></td><td><p xmlns="" class="SimplePara">0.659</p></td><td><p xmlns="" class="SimplePara">0.175</p></td><td><p xmlns="" class="SimplePara">ns</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">2</p></td><td><p xmlns="" class="SimplePara">0.067</p></td><td><p xmlns="" class="SimplePara">0.067</p></td><td><p xmlns="" class="SimplePara">0.000</p></td><td><p xmlns="" class="SimplePara">0.062</p></td><td><p xmlns="" class="SimplePara">ns</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">ACEC</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">5</p></td><td><p xmlns="" class="SimplePara">0.717</p></td><td><p xmlns="" class="SimplePara">0.467</p></td><td><p xmlns="" class="SimplePara">0.357</p></td><td><p xmlns="" class="SimplePara">0.643</p></td><td><p xmlns="" class="SimplePara">ns</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">5</p></td><td><p xmlns="" class="SimplePara">0.713</p></td><td><p xmlns="" class="SimplePara">0.133</p></td><td><p xmlns="" class="SimplePara">0.818</p></td><td><p xmlns="" class="SimplePara">0.636</p></td><td><p xmlns="" class="SimplePara">*</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">ACQT</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">4</p></td><td><p xmlns="" class="SimplePara">0.476</p></td><td><p xmlns="" class="SimplePara">0.2143</p></td><td><p xmlns="" class="SimplePara">0.559</p></td><td><p xmlns="" class="SimplePara">0.427</p></td><td><p xmlns="" class="SimplePara">ns</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">3</p></td><td><p xmlns="" class="SimplePara">0.570</p></td><td><p xmlns="" class="SimplePara">0.133</p></td><td><p xmlns="" class="SimplePara">0.772</p></td><td><p xmlns="" class="SimplePara">0.485</p></td><td><p xmlns="" class="SimplePara">*</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">ACHZ</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">5</p></td><td><p xmlns="" class="SimplePara">0.740</p></td><td><p xmlns="" class="SimplePara">0.867</p></td><td><p xmlns="" class="SimplePara">−0.178</p></td><td><p xmlns="" class="SimplePara">0.664</p></td><td><p xmlns="" class="SimplePara">ns</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">9</p></td><td><p xmlns="" class="SimplePara">0.826</p></td><td><p xmlns="" class="SimplePara">0.917</p></td><td><p xmlns="" class="SimplePara">−0.115</p></td><td><p xmlns="" class="SimplePara">0.764</p></td><td><p xmlns="" class="SimplePara">ns</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">AC53</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">5</p></td><td><p xmlns="" class="SimplePara">0.802</p></td><td><p xmlns="" class="SimplePara">0.786</p></td><td><p xmlns="" class="SimplePara">0.021</p></td><td><p xmlns="" class="SimplePara">0.737</p></td><td><p xmlns="" class="SimplePara">ns</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">8</p></td><td><p xmlns="" class="SimplePara">0.870</p></td><td><p xmlns="" class="SimplePara">0.917</p></td><td><p xmlns="" class="SimplePara">−0.048</p></td><td><p xmlns="" class="SimplePara">0.821</p></td><td><p xmlns="" class="SimplePara">ns</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">ACLI</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">2</p></td><td><p xmlns="" class="SimplePara">0.198</p></td><td><p xmlns="" class="SimplePara">0.214</p></td><td><p xmlns="" class="SimplePara">−0.083</p></td><td><p xmlns="" class="SimplePara">0.173</p></td><td><p xmlns="" class="SimplePara">ns</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">–</p></td><td><p xmlns="" class="SimplePara">–</p></td><td><p xmlns="" class="SimplePara">–</p></td><td><p xmlns="" class="SimplePara">–</p></td><td><p xmlns="" class="SimplePara">–</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">AC5S</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">8</p></td><td><p xmlns="" class="SimplePara">0.885</p></td><td><p xmlns="" class="SimplePara">0.467</p></td><td><p xmlns="" class="SimplePara">0.473</p></td><td><p xmlns="" class="SimplePara">0.824</p></td><td><p xmlns="" class="SimplePara">*</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">8</p></td><td><p xmlns="" class="SimplePara">0.802</p></td><td><p xmlns="" class="SimplePara">0.533</p></td><td><p xmlns="" class="SimplePara">0.343</p></td><td><p xmlns="" class="SimplePara">0.752</p></td><td><p xmlns="" class="SimplePara">ns</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">ACPJ</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">10</p></td><td><p xmlns="" class="SimplePara">0.887</p></td><td><p xmlns="" class="SimplePara">0.467</p></td><td><p xmlns="" class="SimplePara">0.483</p></td><td><p xmlns="" class="SimplePara">0.843</p></td><td><p xmlns="" class="SimplePara">*</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">13</p></td><td><p xmlns="" class="SimplePara">0.913</p></td><td><p xmlns="" class="SimplePara">0.600</p></td><td><p xmlns="" class="SimplePara">0.351</p></td><td><p xmlns="" class="SimplePara">0.872</p></td><td><p xmlns="" class="SimplePara">*</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                    <p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">n</em> number of individuals, <em class="EmphasisTypeItalic">A</em> number of allele, <em class="EmphasisTypeItalic">H</em><sub xmlns="http://www.w3.org/1999/xhtml">E</sub> expected heterozygosity, <em class="EmphasisTypeItalic">H</em><sub xmlns="http://www.w3.org/1999/xhtml">O</sub> observed heterozygosity, <em class="EmphasisTypeItalic">F</em><sub xmlns="http://www.w3.org/1999/xhtml">IS</sub> inbreeding coefficient, <em class="EmphasisTypeItalic">PIC</em> polymorphic information contents, <em class="EmphasisTypeItalic">HWE</em> Hardy-Weinberg equilibrium, <em class="EmphasisTypeItalic">ns</em> not significant and – uncomputed</p>
                    <p xmlns="" class="SimplePara">*Significant after Bonferroni correction (<em class="EmphasisTypeItalic">p</em> &lt; 0.0029)</p>
                  </div></figure></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par25" class="Para">F<sub>ST</sub> (0.05) estimated between the two Jeju striped field mouse populations was not significantly different from zero (<em xmlns="" class="EmphasisTypeItalic">P</em> &gt; 0.99). The AMOVA results revealed that in the two populations most of genetic variation (88%) was harbored within each population, whereas among population genetic variation was proportionally minor (12%; Fig.&nbsp;<span class="InternalRef"><a href="#Fig1">1</a></span>). Coupled with the small F<sub>ST</sub> value, the AMOVA results suggest that YD and BH are not greatly diverged in part due to high gene flow between the two populations. However, the PCoA analysis showed spatial structure among the two populations. Overall, the genotypes from each population are affiliated with the population from which the genotypes were drawn (Fig.&nbsp;<span class="InternalRef"><a href="#Fig2">2</a></span>). The clustering pattern indicates that the 17 markers developed from our study have sufficient power to separate out the two local populations.<figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO1">
                    <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0070-8/MediaObjects/41065_2018_70_Fig1_HTML.png"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0070-8/MediaObjects/41065_2018_70_Fig1_HTML.png" alt="Fig. 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0070-8/MediaObjects/41065_2018_70_Fig1_HTML.png"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0070-8/MediaObjects/41065_2018_70_Fig1_HTML.png" alt="Fig. 1"></img></noscript></a>
                  </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 1</span>
                      <p xmlns="" class="SimplePara">A pie chart summarizing AMOVA results. Genetic variance was hierarchically partitioned into two levels: 1) among populations (12% of total genetic variance) and 2) within each population (88% of total genetic variance)</p>
                    </div></figcaption></figure><figure xmlns="" class="Figure" id="Fig2"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO2">
                    <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0070-8/MediaObjects/41065_2018_70_Fig2_HTML.png"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0070-8/MediaObjects/41065_2018_70_Fig2_HTML.png" alt="Fig. 2" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0070-8/MediaObjects/41065_2018_70_Fig2_HTML.png"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0070-8/MediaObjects/41065_2018_70_Fig2_HTML.png" alt="Fig. 2"></img></noscript></a>
                  </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 2</span>
                      <p xmlns="" class="SimplePara">PCoA plot for 30 individual genotypes of <em class="EmphasisTypeItalic">A. agrarius chejuensis</em> from two populations. The first two variance components PC1 and PC2 were plotted. BHs are samples from Beophwan-dong and YDs are from Yongdu-dong</p>
                    </div></figcaption></figure></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par26" class="Para">Cross-species amplification examined for a subspecies, <em xmlns="" class="EmphasisTypeItalic">A. a. coreae</em>, and its congeneric species, <em xmlns="" class="EmphasisTypeItalic">A. peninsulae,</em> in South Korea showed high cross-species transferability. All 17 loci were successfully amplified for <em xmlns="" class="EmphasisTypeItalic">A. a. coreae</em> and 12 of 17 loci were successfully amplified for <em xmlns="" class="EmphasisTypeItalic">A. peninsulae</em> (Table&nbsp;<span class="InternalRef"><a href="#Tab3">3</a></span>). Except for the AC53 locus, all microsatellite loci were successfully amplified for individuals collected on Heuksan Island. The cross-species amplification could be an indicator of genetic distance among species. The Jeju island subspecies, <em xmlns="" class="EmphasisTypeItalic">A. a chejuensis</em> has often been regarded a distinct species in Korea [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>]. In contrast, the classification of the population on Heuksan Island has been <em xmlns="" class="EmphasisTypeItalic">A. a. coreae</em> (the same as the mainland populations) in most of the literature despite its high genetic divergence from mainland populations as revealed in a microsatellite analysis [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>]. Our cross-species amplification tests suggested that the subspecies <em xmlns="" class="EmphasisTypeItalic">A. a chejuensis</em> might be more closely related to the mainland populations of <em xmlns="" class="EmphasisTypeItalic">A. a. coreae</em> than the populations on Heuksan Island. Intensive population level studies with these developed markers might elucidate the true phylogenetic relationships among these three taxa.<figure xmlns="" id="Tab3" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 3</span>
                      <p xmlns="" class="SimplePara">Results of cross-species amplifications</p>
                    </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col></col></colgroup><thead><tr class="header"><th rowspan="2"><p xmlns="" class="SimplePara">Loci</p></th><th colspan="2"><p xmlns="" class="SimplePara">Species</p></th></tr><tr class="header"><th><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">Apodemus agrarius coreae</em>
                          </p></th><th><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">Apodemus peninsulae</em>
                          </p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">AC2&nbsp;L</p></td><td><p xmlns="" class="SimplePara">+</p></td><td><p xmlns="" class="SimplePara">+</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">ACH7</p></td><td><p xmlns="" class="SimplePara">+</p></td><td><p xmlns="" class="SimplePara">+</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">ACW7</p></td><td><p xmlns="" class="SimplePara">+</p></td><td><p xmlns="" class="SimplePara">+</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">AC9S</p></td><td><p xmlns="" class="SimplePara">+</p></td><td><p xmlns="" class="SimplePara">+</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">ACAQ</p></td><td><p xmlns="" class="SimplePara">+</p></td><td><p xmlns="" class="SimplePara">+</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">ACWV</p></td><td><p xmlns="" class="SimplePara">+</p></td><td><p xmlns="" class="SimplePara">–</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">ACAV</p></td><td><p xmlns="" class="SimplePara">+</p></td><td><p xmlns="" class="SimplePara">–</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">ACU8</p></td><td><p xmlns="" class="SimplePara">+</p></td><td><p xmlns="" class="SimplePara">+</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">AC9&nbsp;M</p></td><td><p xmlns="" class="SimplePara">+</p></td><td><p xmlns="" class="SimplePara">+</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">AC124</p></td><td><p xmlns="" class="SimplePara">+</p></td><td><p xmlns="" class="SimplePara">+</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">ACEC</p></td><td><p xmlns="" class="SimplePara">+</p></td><td><p xmlns="" class="SimplePara">–</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">ACQT</p></td><td><p xmlns="" class="SimplePara">+</p></td><td><p xmlns="" class="SimplePara">–</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">ACHZ</p></td><td><p xmlns="" class="SimplePara">+</p></td><td><p xmlns="" class="SimplePara">+</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">AC53</p></td><td><p xmlns="" class="SimplePara">+</p></td><td><p xmlns="" class="SimplePara">+</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">ACLI</p></td><td><p xmlns="" class="SimplePara">+</p></td><td><p xmlns="" class="SimplePara">–</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">AC5S</p></td><td><p xmlns="" class="SimplePara">+</p></td><td><p xmlns="" class="SimplePara">+</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">ACPJ</p></td><td><p xmlns="" class="SimplePara">+</p></td><td><p xmlns="" class="SimplePara">+</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                    <p xmlns="" class="SimplePara">Seven individual of each species were tested</p>
                    <p xmlns="" class="SimplePara">Note: + successful PCR amplification; − PCR failed</p>
                  </div></figure></div><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par27" class="Para">Although this species is the main vector of Hanta virus and the most common mammal in the Korean Peninsula, few studies have addressed populations of the striped field mouse in Korea. One of the limitations in the study of <em xmlns="" class="EmphasisTypeItalic">A. agrarius</em> is the deficiency of genetic markers [<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>]. The microsatellite loci developed in this study have proven to be useful for investigating genetic structure, introgression, hybrid, and taxonomic status of <em xmlns="" class="EmphasisTypeItalic">A. agrarius</em> in not only the Korean Peninsula but the entire distribution for the species. The genetic markers might also be used to establish a management strategy for <em xmlns="" class="EmphasisTypeItalic">A. a. chejuensis,</em> the most common endemic rodent on Jeju Island.</p></div></section>
          <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AbbreviationGroup Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Abbreviations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Abbreviations">Abbreviations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Abbreviations"><div class="DefinitionList">
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>A:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par4" class="Para">The number of allele at each locus</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>BH:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par5" class="Para">Beophwan-dong</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>
                    <em class="EmphasisTypeItalic">H</em>
                    <sub xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml">E</sub>
                  :&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par6" class="Para">Expected heterozygosity</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>
                    <em class="EmphasisTypeItalic">H</em>
                    <sub xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml">O</sub>
                  :&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par7" class="Para">Observed heterozygosity</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>HWE:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par8" class="Para">Hardy-Weinberg equilibrium</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>LD:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par9" class="Para">Linkage disequilibrium</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>PIC:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par10" class="Para">Polymorphic information contents</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>SSR:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par11" class="Para">Simple Sequence Repeat</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>YD:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par12" class="Para">Yongdu-dong</p>
                  </dd></dl>
              </div></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments"><h3 xmlns="" class="Heading">Acknowledgments</h3><p xmlns="" class="SimplePara">This study was supported by a grant (NIBR201403201) from the National Institute of Biological Resources (NIBR), funded by the Ministry of Environment (MOE), Republic of Korea. Also, NIBR supported field collections of this study as a part of Wildlife Survey (NIBR20182401). We also thank Drs J. Baccus, P. Schlichting, M. Forstner, anonymous reviewers with the editors Y. Shi and S. Baumgartner for valuable suggestions and comments.</p><div id="FPar1" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Funding</h3>
                <p id="Par28" class="Para">Genetic works were supported by a grant (NIBR201403201) and field collection was supported by Wildlife Survey (NIBR20182401).</p>
              </div><div id="FPar2" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Availability of data and materials</h3>
                <p id="Par29" class="Para">Dataset is provided by an additional file. All specimens are stored in National Institute of Biological Resources, Korea. It’s available after obtaining permission from the agency.</p>
              </div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="FormalPara FormalParaRenderingStyle1"><h3 xmlns="" class="Heading">Authors’ contributions</h3><p xmlns="" class="SimplePara">HNK and JJ conceived the idea of this study. HNK collected samples. HNK and HUK performed lab works. YSJ and JJ wrote the manuscript. All authors read and approved the final manuscript.</p></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Ethics">
              <div id="FPar3" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Ethics approval</h3>
                <p id="Par30" class="Para">Sample collections were followed by guidelines of American Society of Mammalogists. This study was approved by Research Management Committee of National Institute of Biological Resources, Ministry of Environment, Korea.</p>
              </div>
              <div id="FPar4" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Consent for publication</h3>
                <p id="Par31" class="Para">Not applicable.</p>
              </div>
              <div id="FPar5" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Competing interests</h3>
                <p id="Par32" class="Para">The authors declare that they have no competing interests.</p>
              </div>
              <div id="FPar6" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Publisher’s Note</h3>
                <p id="Par33" class="Para">Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
              </div>
            </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="License">
                <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Open Access</strong>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/4.0/"><span class="RefSource">http://creativecommons.org/licenses/by/4.0/</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/publicdomain/zero/1.0/"><span class="RefSource">http://creativecommons.org/publicdomain/zero/1.0/</span></a></span>) applies to the data made available in this article, unless otherwise stated.</p>
              </div></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authors’ Affiliations</a></h2><div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1"><span class="AffiliationNumber">(1)</span><div class="AffiliationText">The Division of EcoCreative, Ewha Womans University, Seoul, 03760, Korea</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff2"><span class="AffiliationNumber">(2)</span><div class="AffiliationText">Animal Resources, National Institute of Biological Resources, Incheon, 22689, Korea</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff3"><span class="AffiliationNumber">(3)</span><div class="AffiliationText">Department of Science Education, Ewha Woman University, Seoul, 03760, Korea</div></div></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper"><li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Musser GG, Carleton MD. Superfamily Muroidae. In: Winson DE, Reeder DM, editors. Mammal species world a Taxon Geogr Ref. 3rd ed. Baltimore: Johns Hopkins University Press; 2005. p. 894–1532.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Superfamily%20Muroidae&amp;author=GG.%20Musser&amp;author=MD.%20Carleton&amp;pages=894-1532&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Won PH. Illustrated encyclopedia of fauna and flora of Korea volume 7, mammals. Seoul: Ministry of Education; 1967.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Illustrated%20encyclopedia%20of%20fauna%20and%20flora%20of%20Korea%20volume%207%2C%20mammals&amp;author=PH.%20Won&amp;publication_year=1967"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Jo Y, Kim H, Baccus JT, Jung J. Genetic differentiation of the Korean striped field mouse, <em class="EmphasisTypeItalic">Apodemus agrarius</em> (Muridae, Rodentia), based on microsatellite polymorphism. Mammalia. 2017;81:297–307.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1515/mammalia-2015-0152"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20differentiation%20of%20the%20Korean%20striped%20field%20mouse%2C%20Apodemus%20agrarius%20%28Muridae%2C%20Rodentia%29%2C%20based%20on%20microsatellite%20polymorphism&amp;author=Y.%20Jo&amp;author=H.%20Kim&amp;author=JT.%20Baccus&amp;author=J.%20Jung&amp;journal=Mammalia&amp;volume=81&amp;pages=297-307&amp;publication_year=2017"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Koh HS. Geographic variation of morphometric characters among three subspecies of striped field mice, <em class="EmphasisTypeItalic">Apodemus agrarius</em> Pallas (Muridae, Rodentia), from Korea. Korean J Zool. 1986;29:272–82.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Geographic%20variation%20of%20morphometric%20characters%20among%20three%20subspecies%20of%20striped%20field%20mice%2C%20Apodemus%20agrarius%20Pallas%20%28Muridae%2C%20Rodentia%29%2C%20from%20Korea&amp;author=HS.%20Koh&amp;journal=Korean%20J%20Zool&amp;volume=29&amp;pages=272-282&amp;publication_year=1986"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Koh HS. Systematic studies of Korean rodents: III. Morphometric and chromosomal analyses of striped field mice, <em class="EmphasisTypeItalic">Apodemus agrarius chejuensis</em> Jones and Johnson, from Jeju-Do. Korean J Syst Zool. 1987;3:24–40.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Systematic%20studies%20of%20Korean%20rodents%3A%20III.%20Morphometric%20and%20chromosomal%20analyses%20of%20striped%20field%20mice%2C%20Apodemus%20agrarius%20chejuensis%20Jones%20and%20Johnson%2C%20from%20Jeju-Do&amp;author=HS.%20Koh&amp;journal=Korean%20J%20Syst%20Zool.&amp;volume=3&amp;pages=24-40&amp;publication_year=1987"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Koh HS. Systematic studies of Korean rodents. V. Morphometric and chromosomal analyses on island populations of striped field mice (<em class="EmphasisTypeItalic">Apodemus agrarius coreae</em>) in southwestern coasts of the Korean peninsula. Korean J Syst Zool. 1989;5:1–12.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Systematic%20studies%20of%20Korean%20rodents.%20V.%20Morphometric%20and%20chromosomal%20analyses%20on%20island%20populations%20of%20striped%20field%20mice%20%28Apodemus%20agrarius%20coreae%29%20in%20southwestern%20coasts%20of%20the%20Korean%20peninsula&amp;author=HS.%20Koh&amp;journal=Korean%20J%20Syst%20Zool&amp;volume=5&amp;pages=1-12&amp;publication_year=1989"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Koh HS. Morphometric analyses with eight subspecies of striped field mice, <em class="EmphasisTypeItalic">Apodemus agrarius</em> Pallas (Rodentia, Mammalia), in Asia: the taxonomic status of subspecies <em class="EmphasisTypeItalic">chejuensis</em> at Cheju island in Korea. Korean J Syst Zool. 1991;7:179–88.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Morphometric%20analyses%20with%20eight%20subspecies%20of%20striped%20field%20mice%2C%20Apodemus%20agrarius%20Pallas%20%28Rodentia%2C%20Mammalia%29%2C%20in%20Asia%3A%20the%20taxonomic%20status%20of%20subspecies%20chejuensis%20at%20Cheju%20island%20in%20Korea&amp;author=HS.%20Koh&amp;journal=Korean%20J%20Syst%20Zool.&amp;volume=7&amp;pages=179-188&amp;publication_year=1991"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Oh HS, Mori T. Taxonomic re-examination of the striped field mouse, <em class="EmphasisTypeItalic">Apodemus agrarius coreae</em> and <em class="EmphasisTypeItalic">A. a. chejuensis</em>: evidence from crossbreeding experiments. J Fac Agric Kyushu Univ. 1998;43:143–51.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Taxonomic%20re-examination%20of%20the%20striped%20field%20mouse%2C%20Apodemus%20agrarius%20coreae%20and%20A.%20a.%20chejuensis%3A%20evidence%20from%20crossbreeding%20experiments&amp;author=HS.%20Oh&amp;author=T.%20Mori&amp;journal=J%20Fac%20Agric%20Kyushu%20Univ&amp;volume=43&amp;pages=143-151&amp;publication_year=1998"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Koh HS, Yoo BS. Variation of mitochondrial DNA in two subspecies of striped field mice, <em class="EmphasisTypeItalic">Apodemus agrarius coreae</em> and <em class="EmphasisTypeItalic">Apodemus agrarius chejuensis</em>, from Korea. Korean J Zool. 1992;35:332–8.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Variation%20of%20mitochondrial%20DNA%20in%20two%20subspecies%20of%20striped%20field%20mice%2C%20Apodemus%20agrarius%20coreae%20and%20Apodemus%20agrarius%20chejuensis%2C%20from%20Korea&amp;author=HS.%20Koh&amp;author=BS.%20Yoo&amp;journal=Korean%20J%20Zool.&amp;volume=35&amp;pages=332-338&amp;publication_year=1992"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Koh HS, Lee WJ, Kocher TD. The genetic relationships of two subspecies of striped field mice, <em class="EmphasisTypeItalic">Apodemus agrarius coreae</em> and <em class="EmphasisTypeItalic">Apodemus agrarius chejuensis</em>. Heredity (Edinb). 2000;85:30–6.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1046/j.1365-2540.2000.00723.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20genetic%20relationships%20of%20two%20subspecies%20of%20striped%20field%20mice%2C%20Apodemus%20agrarius%20coreae%20and%20Apodemus%20agrarius%20chejuensis&amp;author=HS.%20Koh&amp;author=WJ.%20Lee&amp;author=TD.%20Kocher&amp;journal=Heredity%20%28Edinb%29&amp;volume=85&amp;pages=30-36&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Oh D-J, Kim T-W, Chang M-H, Han S-H, Oh H-S, Kim S-J. Migration route estimation of the Jeju striped field mouse <em class="EmphasisTypeItalic">Apodemus agrarius chejuensis</em> (Rodentia, Muridae). Mitochondrial DNA. 2013;24:137–44.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.3109/19401736.2012.726619"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Migration%20route%20estimation%20of%20the%20Jeju%20striped%20field%20mouse%20Apodemus%20agrarius%20chejuensis%20%28Rodentia%2C%20Muridae%29&amp;author=D-J.%20Oh&amp;author=T-W.%20Kim&amp;author=M-H.%20Chang&amp;author=S-H.%20Han&amp;author=H-S.%20Oh&amp;author=S-J.%20Kim&amp;journal=Mitochondrial%20DNA&amp;volume=24&amp;pages=137-144&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Johnson KM. Hantaviruses: history and overview. Curr Top Microbiol Immunol. 2001;256:1–14.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11217399"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Hantaviruses%3A%20history%20and%20overview&amp;author=KM.%20Johnson&amp;journal=Curr%20Top%20Microbiol%20Immunol&amp;volume=256&amp;pages=1-14&amp;publication_year=2001"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ellegren H. Microsatellites: simple sequences with complex evolution. Nat Rev Genet. 2004;5:435–45.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nrg1348"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Microsatellites%3A%20simple%20sequences%20with%20complex%20evolution&amp;author=H.%20Ellegren&amp;journal=Nat%20Rev%20Genet&amp;volume=5&amp;pages=435-445&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lopez L, Barreiro R, Fischer M, Koch MA. Mining microsatellite markers from public expressed sequence tags databases for the study of threatened plants. BMC Genomics [Internet]. BMC Genomics; 2015;16. Available from: doi:<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.1186/s12864-015-2031-1"><span class="RefSource">https://doi.org/10.1186/s12864-015-2031-1</span></a></span>
                        </span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Squirrell J, Hollingsworth PM, Woodhead M, Russell J, Lowe AJ, Gibby M, et al. How much effort is required to isolate nuclear microsatellites from plants? Mol Ecol. 2003;12:1339–48.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1046/j.1365-294X.2003.01825.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=How%20much%20effort%20is%20required%20to%20isolate%20nuclear%20microsatellites%20from%20plants%3F&amp;author=J.%20Squirrell&amp;author=PM.%20Hollingsworth&amp;author=M.%20Woodhead&amp;author=J.%20Russell&amp;author=AJ.%20Lowe&amp;author=M.%20Gibby&amp;journal=Mol%20Ecol&amp;volume=12&amp;pages=1339-1348&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wu H, Zhan XJ, Yan L, Liu SY, Li M, Hu JC, et al. Isolation and characterization of fourteen microsatellite loci for striped field mouse (<em class="EmphasisTypeItalic">Apodemus agrarius</em>). Conserv Genet. 2008;9:1691–3.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s10592-008-9535-z"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Isolation%20and%20characterization%20of%20fourteen%20microsatellite%20loci%20for%20striped%20field%20mouse%20%28Apodemus%20agrarius%29&amp;author=H.%20Wu&amp;author=XJ.%20Zhan&amp;author=L.%20Yan&amp;author=SY.%20Liu&amp;author=M.%20Li&amp;author=JC.%20Hu&amp;journal=Conserv%20Genet&amp;volume=9&amp;pages=1691-1693&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sikes RS. 2016 guidelines of the American Society of Mammalogists for the use of wild mammals in research and education. J Mammal. 2016;97:663–88.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/jmammal/gyw078"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=2016%20guidelines%20of%20the%20American%20Society%20of%20Mammalogists%20for%20the%20use%20of%20wild%20mammals%20in%20research%20and%20education&amp;author=RS.%20Sikes&amp;journal=J%20Mammal&amp;volume=97&amp;pages=663-688&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Meglécz E, Pech N, Gilles A, Dubut V, Hingamp P, Trilles A, et al. QDD version 3.1: a user-friendly computer program for microsatellite selection and primer design revisited: experimental validation of variables determining genotyping success rate. Mol Ecol Resour. 2014;14:1302–13.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/1755-0998.12271"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=QDD%20version%203.1%3A%20a%20user-friendly%20computer%20program%20for%20microsatellite%20selection%20and%20primer%20design%20revisited%3A%20experimental%20validation%20of%20variables%20determining%20genotyping%20success%20rate&amp;author=E.%20Megl%C3%A9cz&amp;author=N.%20Pech&amp;author=A.%20Gilles&amp;author=V.%20Dubut&amp;author=P.%20Hingamp&amp;author=A.%20Trilles&amp;journal=Mol%20Ecol%20Resour&amp;volume=14&amp;pages=1302-1313&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Untergasser A, Cutcutache I, Koressaar T, Ye J, Faircloth BC, Remm M, et al. Primer3-new capabilities and interfaces. Nucleic Acids Res. 2012;40:1–12.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gks596"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Primer3-new%20capabilities%20and%20interfaces&amp;author=A.%20Untergasser&amp;author=I.%20Cutcutache&amp;author=T.%20Koressaar&amp;author=J.%20Ye&amp;author=BC.%20Faircloth&amp;author=M.%20Remm&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=1-12&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Excoffier L, Lischer HEL. Arlequin suite ver 3.5: a new series of programs to perform population genetics analyses under Linux and windows. Mol Ecol Resour. 2010;10:564–7.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1755-0998.2010.02847.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Arlequin%20suite%20ver%203.5%3A%20a%20new%20series%20of%20programs%20to%20perform%20population%20genetics%20analyses%20under%20Linux%20and%20windows&amp;author=L.%20Excoffier&amp;author=HEL.%20Lischer&amp;journal=Mol%20Ecol%20Resour&amp;volume=10&amp;pages=564-567&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Raymond M, Rousset F. GENEPOP (version 1.2): population genetics software for exact tests and ecumenicism. J Hered. 1995;86:248–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/oxfordjournals.jhered.a111573"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=GENEPOP%20%28version%201.2%29%3A%20population%20genetics%20software%20for%20exact%20tests%20and%20ecumenicism&amp;author=M.%20Raymond&amp;author=F.%20Rousset&amp;journal=J%20Hered&amp;volume=86&amp;pages=248-249&amp;publication_year=1995"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Rousset F. GENEPOP’007: a complete re-implementation of the GENEPOP software for windows and Linux. Mol Ecol Resour. 2008;8:103–6.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1471-8286.2007.01931.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=GENEPOP%E2%80%99007%3A%20a%20complete%20re-implementation%20of%20the%20GENEPOP%20software%20for%20windows%20and%20Linux&amp;author=F.%20Rousset&amp;journal=Mol%20Ecol%20Resour&amp;volume=8&amp;pages=103-106&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Rice WR. Analyzing tables of statistical tests. Evolution (N Y). 1989;43:223–5.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Analyzing%20tables%20of%20statistical%20tests&amp;author=WR.%20Rice&amp;journal=Evolution%20%28N%20Y%29&amp;volume=43&amp;pages=223-225&amp;publication_year=1989"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR24" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Peakall R, Smouse PE. GenALEx 6.5: genetic analysis in excel. Population genetic software for teaching and research-an update. Bioinformatics. 2012;28:2537–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/bts460"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=GenALEx%206.5%3A%20genetic%20analysis%20in%20excel.%20Population%20genetic%20software%20for%20teaching%20and%20research-an%20update&amp;author=R.%20Peakall&amp;author=PE.%20Smouse&amp;journal=Bioinformatics&amp;volume=28&amp;pages=2537-2539&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li></ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#TheAuthors" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="TheAuthors">Copyright</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="TheAuthors"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">©&nbsp;The Author(s)&nbsp;2018</div></div></section></div>
                        


                    </article>
                </main>

                <div class="c-content-layout__side u-text-sm">
                    <aside>
                        <div class="p-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="//hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-018-0070-8" data-track="click" data-track-category="PDF Download"
            data-track-action="Click download article PDF"
            data-track-label="10.1186/s41065-018-0070-8">Download PDF</a>
        </p>
    
                            
                            
    <button type="button"
       class="c-button c-button--secondary c-button--full-width c-button--expandable"
       data-toggle="collapse"
       data-target="export-citations"
       data-test="export-citations-button"
       data-sticky-update="true"
       aria-controls="export-citations"
       aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/s41065-018-0070-8"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0070-8?format=refman&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/s41065-018-0070-8"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0070-8?format=refman&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/s41065-018-0070-8"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0070-8?format=refman&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/s41065-018-0070-8"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0070-8?format=endnote&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/s41065-018-0070-8"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0070-8?format=endnote&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/s41065-018-0070-8"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0070-8?format=endnote&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/s41065-018-0070-8"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0070-8?format=bibtex&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/s41065-018-0070-8"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0070-8?format=bibtex&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/s41065-018-0070-8"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0070-8?format=bibtex&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
    </div>

                        </div>
                        
    
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
                                <div id="article-access-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/access-count/10.1186/s41065-018-0070-8 #accessCount-container"></div>
                                <div id="citation-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/citation-count/10.1186/s41065-018-0070-8 #count-container"></div>
                                <div id="altmetric-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/altmetric/10.1186/s41065-018-0070-8 #altmetric-container"></div>
                            </ul>
                        </div>
                        
    <div id="sociaMediaShare" class="p-separator u-js-shown">
        <h3 class="p-section-title">Share this article</h3>
        <ul id="shareButtons" class="c-social-links">
            <li class="c-social-links__item">
                <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-018-0070-8">
                    <span class="u-visually-hidden">Share on Twitter</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-twitter"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-018-0070-8">
                    <span class="u-visually-hidden">Share on Facebook</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-facebook"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click"
                        data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-018-0070-8">
                    <span class="u-visually-hidden">Share on LinkedIn</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-linkedin"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-018-0070-8">
                    <span class="u-visually-hidden">Share on Weibo</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-weibo"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnGooglePlus" class="js-btnShareOnGplus" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Google+" data-track-label="10.1186/s41065-018-0070-8">
                    <span class="u-visually-hidden">Share on Google Plus</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-gplus"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-018-0070-8">
                    <span class="u-visually-hidden">Share on Reddit</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-reddit"></use>
                    </svg>
                </button>
            </li>
        </ul>
    </div>
 
                        
    
        <div class="p-separator u-js-shown">
            <h3 class="p-section-title">See updates</h3>
            <button class="c-crossmark-button" data-target="crossmark" data-test="crossmark" data-track="click" data-track-category="Article Page" data-track-action="Click CrossMark" data-track-label="10.1186/s41065-018-0070-8">
                <img id="crossmark-icon" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" alt="Crossmark logo" />
            </button>
        </div>
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
                                <li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/s41065-018-0070-8" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=738&doi=10.1186%2Fs41065-018-0070-8&volume=155&issue=1&title=Development+of+17+polymorphic+microsatellite+loci+from+Jeju+striped+field+mouse%2C+Apodemus+agrarius+chejuensis+%28Rodentia%3A+Muridae%29%2C+by+454+pyrosequencing&author_name=Han-Na+Kim&start_page=1&end_page=6">Order reprint</a>

                                </li>
                            </ul>
                        </div>
                    </aside>
                    
    <div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">Hereditas</span>
                        </h4>
                        <p class="c-journal-footer__issn">ISSN: 1601-5223</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
                                
                                    <li class="c-journal-footer__contact-item">Submission enquiries: <a href="https://www.editorialmanager.com/here/" target="_blank">Access here and click Contact Us</a></li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a></li>
                                
                            </ul>
                        </div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class='tracker' style='display:none' src='/track/article/10.1186/s41065-018-0070-8' alt=""/>

         
    <footer>
        <div class="ctx-publisher-footer">
    <div class="c-navbar">
        <div class="u-container">
            <div class="c-navbar__logo u-margin-right-xl">
                <a href="https://www.biomedcentral.com">
                    <img src=/static/images/bmc/logos/logo-bmc-aj-594e03e20d.svg alt="BMC logo" data-test="navbar-logo-footer">

                </a>
            </div>
            <div class="c-navbar__content">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


            </div>
        </div>
    </div>
    <div class="c-publisher-footer" data-test="publisher-footer">
        <div class="u-container">
            
            <div class="c-flex c-flex--wrap c-flex--space-between" role="navigation" data-test="publisher-footer-menu">
                <div class="c-flex" aria-label="Publisher footer links">
                    
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/login">Receive BMC newsletters</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/account">Manage article alerts</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/policies">Policies</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/accessibility">Accessibility</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/press-centre">Press center</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/contact-us">Contact us</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/jobs">Careers</a>
                                    </li>
                                
                            </ul>
                        
                    
                </div>
                <div class="u-margin-bottom-lg">
                    <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                    <ul class="c-flex u-reset-list" data-test="footer-social-links">
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://twitter.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Twitter link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Twitter page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-twitter-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://www.facebook.com/BioMedCentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Facebook link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Facebook page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-facebook-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://plus.google.com/106226337984152901734/"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Google Plus link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Google Plus page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-gplus-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="http://www.weibo.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Weibo link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Weibo page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-weibo-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </div>
            <p class="u-reset-margin">
                By using this website, you agree to our
                <a class="u-text-inherit" href="//www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
                <a class="u-text-inherit" href="//www.biomedcentral.com/privacy-statement">Privacy
                    statement</a> and
                <a class="u-text-inherit" href="//www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
                
                    <a class="optanon-toggle-display u-text-inherit" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
                
            </p>
        </div>
    </div>
</div>


        <div class="c-sn-footer c-sn-footer--neutral">
    <div class="u-container">
        <img src=/static/images/bmc/logos/logo-springernature-8bd4e8d82f.svg class="c-sn-footer__logo" alt="Springer Nature Logo" itemprop="logo" >
        <p data-test="copyright">&#169; 2018 BioMed Central Ltd unless otherwise stated. Part of
            <a href="http://www.springernature.com"
               data-track="click"
               data-track-action="Click Springer Nature link"
               data-track-category="Corporate footer"
               itemscope itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>

    </div>

    
    <script type="text/javascript" src="//cdn.pbgrd.com/bmc.js" async="async"></script>
    <noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>
    <!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker -->

<script id="springerlink-event-tracker" src="https://event-tracker.springernature.com/dist/eventTracker.js"></script>
<script type="text/javascript">
    (function(){
        var dataLayer = window.dataLayer[0];
        var doi = dataLayer.content.article.doi;
        var imprint = dataLayer.content.contentInfo.imprint;
        var bpids = dataLayer.session.authentication.authenticationID;
        var publisher;
        var pageType = dataLayer.page.category.pageType;
        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }
        var eventObject = {'content_type' : pageType,'doi' : doi};

        if (imprint === 'BioMed Central') { publisher = 'BMC' }
        else if (imprint === 'SpringerOpen') { publisher = 'SpringerOpen' }
        else { throw new Error('Event Tracker Error: The publisher has an unexpected value') }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }
        new EventTracker({'platform':publisher}).sendEvent('display', eventObject);
    })();
</script>


    </body>
</html>

