<!DOCTYPE html>
<html lang="en" class="no-js">
    <head>
        <title>Molecular markers associated with aluminium tolerance in Sorghum bicolor | Hereditas | Full Text</title>

        

    <meta name="citation_abstract" content="Sorghum (Sorghum bicolor, L. Moench) production in many agro-ecologies is constrained by a variety of stresses, including high levels of aluminium (Al) commonly found in acid soils. Therefore, for such soils, growing Al tolerant cultivars is imperative for high productivity. In this study, molecular markers associated with Al tolerance were identified using a mapping population developed by crossing two contrasting genotypes for this trait. Four SSR (Xtxp34, Sb5_236, Sb6_34, and Sb6_342), one STS (CTG29_3b) and three ISSR (811_1400, 835_200 and 884_200) markers produced alleles that showed significant association with Al tolerance. CTG29_3b, 811_1400, Xtxp34 and Sb5_236 are located on chromosome 3 with the first two markers located close to Alt
                    SB
                  , a locus that underlie the Al tolerance gene (SbMATE) implying that their association with Al tolerance is due to their linkage to this gene. Although CTG29_3b and 811_1400 are located closer to Alt
                    SB
                  , Xtxp34 and Sb5_236 explained higher phenotypic variance of Al tolerance indices. Markers 835_200, 884_200, Sb6_34 and Sb6_342 are located on different chromosomes, which implies the presence of several genes involved in Al tolerance in addition to SbMATE in sorghum. These molecular markers have a high potential for use in breeding for Al tolerance in sorghum."/>

    <meta name="description" content="Sorghum (Sorghum bicolor, L. Moench) production in many agro-ecologies is constrained by a variety of stresses, including high levels of aluminium (Al) commonly found in acid soils. Therefore, for such soils, growing Al tolerant cultivars is imperative for high productivity. In this study, molecular markers associated with Al tolerance were identified using a mapping population developed by crossing two contrasting genotypes for this trait. Four SSR (Xtxp34, Sb5_236, Sb6_34, and Sb6_342), one STS (CTG29_3b) and three ISSR (811_1400, 835_200 and 884_200) markers produced alleles that showed significant association with Al tolerance. CTG29_3b, 811_1400, Xtxp34 and Sb5_236 are located on chromosome 3 with the first two markers located close to Alt
                    SB
                  , a locus that underlie the Al tolerance gene (SbMATE) implying that their association with Al tolerance is due to their linkage to this gene. Although CTG29_3b and 811_1400 are located closer to Alt
                    SB
                  , Xtxp34 and Sb5_236 explained higher phenotypic variance of Al tolerance indices. Markers 835_200, 884_200, Sb6_34 and Sb6_342 are located on different chromosomes, which implies the presence of several genes involved in Al tolerance in addition to SbMATE in sorghum. These molecular markers have a high potential for use in breeding for Al tolerance in sorghum."/>

    <meta name="journal_id" content="41065"/>

    <meta name="dc.title" content="Molecular markers associated with aluminium tolerance in Sorghum bicolor"/>

    <meta name="dc.source" content="Hereditas 2018 155:1"/>

    <meta name="dc.format" content="text/html"/>

    <meta name="dc.publisher" content="BioMed Central"/>

    <meta name="dc.date" content="2018-04-13"/>

    <meta name="dc.type" content="OriginalPaper"/>

    <meta name="dc.language" content="En"/>

    <meta name="dc.copyright" content="2018 The Author(s)"/>

    <meta name="dc.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="dc.description" content="Sorghum (Sorghum bicolor, L. Moench) production in many agro-ecologies is constrained by a variety of stresses, including high levels of aluminium (Al) commonly found in acid soils. Therefore, for such soils, growing Al tolerant cultivars is imperative for high productivity. In this study, molecular markers associated with Al tolerance were identified using a mapping population developed by crossing two contrasting genotypes for this trait. Four SSR (Xtxp34, Sb5_236, Sb6_34, and Sb6_342), one STS (CTG29_3b) and three ISSR (811_1400, 835_200 and 884_200) markers produced alleles that showed significant association with Al tolerance. CTG29_3b, 811_1400, Xtxp34 and Sb5_236 are located on chromosome 3 with the first two markers located close to Alt
                    SB
                  , a locus that underlie the Al tolerance gene (SbMATE) implying that their association with Al tolerance is due to their linkage to this gene. Although CTG29_3b and 811_1400 are located closer to Alt
                    SB
                  , Xtxp34 and Sb5_236 explained higher phenotypic variance of Al tolerance indices. Markers 835_200, 884_200, Sb6_34 and Sb6_342 are located on different chromosomes, which implies the presence of several genes involved in Al tolerance in addition to SbMATE in sorghum. These molecular markers have a high potential for use in breeding for Al tolerance in sorghum."/>

    <meta name="prism.issn" content="1601-5223"/>

    <meta name="prism.publicationName" content="Hereditas"/>

    <meta name="prism.publicationDate" content="2018-04-13"/>

    <meta name="prism.volume" content="155"/>

    <meta name="prism.number" content="1"/>

    <meta name="prism.section" content="OriginalPaper"/>

    <meta name="prism.startingPage" content="20"/>

    <meta name="prism.copyright" content="2018 The Author(s)"/>

    <meta name="prism.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="prism.url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0059-3"/>

    <meta name="prism.doi" content="doi:10.1186/s41065-018-0059-3"/>

    <meta name="citation_pdf_url" content="https://hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-018-0059-3"/>

    <meta name="citation_fulltext_html_url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0059-3"/>

    <meta name="citation_journal_title" content="Hereditas"/>

    <meta name="citation_publisher" content="BioMed Central"/>

    <meta name="citation_issn" content="1601-5223"/>

    <meta name="citation_title" content="Molecular markers associated with aluminium tolerance in Sorghum bicolor"/>

    <meta name="citation_volume" content="155"/>

    <meta name="citation_issue" content="1"/>

    <meta name="citation_publication_date" content="2018/12"/>

    <meta name="citation_online_date" content="2018/04/13"/>

    <meta name="citation_firstpage" content="20"/>

    <meta name="citation_article_type" content="Research"/>

    <meta name="citation_fulltext_world_readable" content=""/>

    <meta name="dc.identifier" content="doi:10.1186/s41065-018-0059-3"/>

    <meta name="DOI" content="10.1186/s41065-018-0059-3"/>

    <meta name="citation_doi" content="10.1186/s41065-018-0059-3"/>

    <meta name="dc.creator" content="Emily Jepkosgei Too"/>

    <meta name="dc.creator" content="Augustino Osoro Onkware"/>

    <meta name="dc.creator" content="Beatrice Ang&#8217;iyo Were"/>

    <meta name="dc.creator" content="Samuel Gudu"/>

    <meta name="dc.creator" content="Anders Carlsson"/>

    <meta name="dc.creator" content="Mulatu Geleta"/>

    <meta name="dc.subject" content="Animal Genetics and Genomics"/>

    <meta name="dc.subject" content="Evolutionary Biology"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Cell and Environ; citation_title=Assessing the phytotoxicity of mononuclear hydroxyl-aluminum; citation_author=TB Kindraide, DR Parker; citation_volume=12; citation_publication_date=1989; citation_pages=479-487; citation_doi=10.1111/j.1365-3040.1989.tb02120.x; citation_id=CR1"/>

    <meta name="citation_reference" content="citation_journal_title=Exp Bot; citation_title=The identification of aluminium-resistance genes provides opportunities for enhancing crop production on acid soils; citation_author=PR Ryan, SD Tyerman, T Sasaki, T Furuichi, Y Yamamoto, WH Zhang, E Delhaize; citation_volume=62; citation_publication_date=2011; citation_pages=9-20; citation_doi=10.1093/jxb/erq272; citation_id=CR2"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Physiol; citation_title=Development of a novel aluminium tolerance phenotyping platform used for comparisons of cereal aluminium tolerance and investigations into rice aluminium tolerance mechanisms; citation_author=AN Famoso, RT Clark, JE Shaff, E Craft, SR Mccouch, LV Kochian; citation_volume=153; citation_publication_date=2010; citation_pages=1678-1691; citation_doi=10.1104/pp.110.156794; citation_id=CR3"/>

    <meta name="citation_reference" content="citation_journal_title=Ann Bot; citation_title=The role of the root apoplast in aluminium-induced inhibition of root elongation and in aluminium resistance of plants: a review; citation_author=WJ Horst, YX Wang, D Eticha; citation_volume=106; citation_publication_date=2010; citation_pages=185-197; citation_doi=10.1093/aob/mcq053; citation_id=CR4"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Cell Tissue Organ Cult; citation_title=Plant improvement for resistance to aluminium in acid soils-a review; citation_author=DA Samac, M Tesfaye; citation_volume=75; citation_publication_date=2003; citation_pages=189-207; citation_doi=10.1023/A:1025843829545; citation_id=CR5"/>

    <meta name="citation_reference" content="citation_journal_title=Bot Rev; citation_title=Aluminium hyper accumulation in angiosperms: a review of its phylogenetic significance; citation_author=S Jansen, MR Broadley, W Robbrecht, E Smets; citation_volume=68; citation_publication_date=2002; citation_pages=235-269; citation_doi=10.1663/0006-8101(2002)068[0235:AHIAAR]2.0.CO;2; citation_id=CR6"/>

    <meta name="citation_reference" content="Kanyanjua SM, Iren I, Wambua S, Nandwa SM. Acidic soils in Kenya: constraints and remedial options. KARI Technical Notes Series No. 2002:11."/>

    <meta name="citation_reference" content="citation_title=Effects of soil properties on bioavailability of aluminium and phosphorus in selected Kenyan and Brazilian soils. PhD thesis, Purdue University; citation_inbook_title=USA; citation_publication_date=2008; citation_id=CR8; citation_author=PA Obura"/>

    <meta name="citation_reference" content="Wortmann CS, Mamo M, Abebe G, Mburu C, Kayuki KC, Letayo E, Xerinda S. Atlas of sorghum (Sorghum bicolor (L.) Moench) production in five countries of eastern Africa. The Board of Regents of the University of Nebraska, Lincoln, USA; 2006."/>

    <meta name="citation_reference" content="citation_journal_title=Adv Agron; citation_title=Strategies of plants to adapt to mineral stresses in problem soils; citation_author=S Hiradate, JF Ma, H Matsumoto; citation_volume=96; citation_publication_date=2007; citation_pages=65-132; citation_doi=10.1016/S0065-2113(07)96004-6; citation_id=CR10"/>

    <meta name="citation_reference" content="citation_journal_title=J Hort Sci Biotechnol; citation_title=Effect of aluminium supply on aluminium uptake, translocation and blueing of Hydrangea macrophylla (Thunb.) Ser. Cultivars in a peat-clay substrate; citation_author=A Naumann, WJ Horst; citation_volume=78; citation_publication_date=2003; citation_pages=463-469; citation_doi=10.1080/14620316.2003.11511651; citation_id=CR11"/>

    <meta name="citation_reference" content="citation_journal_title=Trends Plant Sci; citation_title=Aluminium resistance in plants and the complexing role of organic acids; citation_author=JF Ma, PR Ryan, E Delhaize; citation_volume=6; citation_publication_date=2001; citation_pages=273-278; citation_doi=10.1016/S1360-1385(01)01961-6; citation_id=CR12"/>

    <meta name="citation_reference" content="citation_journal_title=Colloid Surface B; citation_title=Physiological changes in barley plants under combined toxicity of aluminium, copper and cadmium; citation_author=TR Guo, GP Zhang, YH Zhang; citation_volume=57; citation_publication_date=2007; citation_pages=182-188; citation_doi=10.1016/j.colsurfb.2007.01.013; citation_id=CR13"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Genet; citation_title=A gene in the multidrug and toxic compound extrusion (MATE) family confers aluminium resistance in sorghum; citation_author=JV Magalhaes, J Liu, CT Guimaraes, UGP Lana, VMC Alves; citation_volume=39; citation_publication_date=2007; citation_pages=1156-1161; citation_doi=10.1038/ng2074; citation_id=CR14"/>

    <meta name="citation_reference" content="citation_journal_title=J Plant Nutr; citation_title=Effects of aluminium on organic acid, sugar and amino acid composition of the root system of sorghum (Sorghum bicolor, L. Moench); citation_author=J Cambraia, FR Galvani, MM Estevao; citation_volume=6; citation_publication_date=1983; citation_pages=313-322; citation_doi=10.1080/01904168309363092; citation_id=CR15"/>

    <meta name="citation_reference" content="citation_journal_title=J Plant Nutr; citation_title=Aluminium effect on organic acid production and accumulation in sorghum; citation_author=JFC Gon&#231;ales, J Cambraia, PR Mosquim, EF Ara&#250;jo; citation_volume=28; citation_publication_date=2005; citation_pages=507-520; citation_doi=10.1081/PLN-200049202; citation_id=CR16"/>

    <meta name="citation_reference" content="citation_journal_title=Plant J; citation_title=A wheat gene encoding an aluminum-activated malate transporter; citation_author=T Sasaki, Y Yamamoto, B Ezaki, M Katsuhara, S Ahn; citation_volume=37; citation_publication_date=2004; citation_pages=645-653; citation_doi=10.1111/j.1365-313X.2003.01991.x; citation_id=CR17"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Cell Physiol; citation_title=An aluminum-activated citrate transporter in barley; citation_author=J Furukawa, N Yamaji, H Wang, N Mitani, Y Murata; citation_volume=48; citation_publication_date=2007; citation_pages=1081-1091; citation_doi=10.1093/pcp/pcm091; citation_id=CR18"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Soil; citation_title=Growth and yield traits of sorghum grown on acid soil at varied aluminum saturations; citation_author=CI Flores, RB Clark, LM Gourley; citation_volume=106; citation_publication_date=1988; citation_pages=49-57; citation_doi=10.1007/BF02371194; citation_id=CR19"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Breed; citation_title=Classification of Brazilian wheat cultivars for aluminium toxicity in acid soils; citation_author=CAN De Sousa; citation_volume=117; citation_publication_date=1998; citation_pages=217-221; citation_doi=10.1111/j.1439-0523.1998.tb01929.x; citation_id=CR20"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Soil; citation_title=Alfalfa, Medicago sativa L., in highly weathered, acid soils. V. Field performance of alfalfa selected for acid tolerance; citation_author=JH Bouton, ME Sumner; citation_volume=74; citation_publication_date=1983; citation_pages=431-436; citation_doi=10.1007/BF02181360; citation_id=CR21"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Soil; citation_title=Evaluating the role of root citrate exudation as a mechanism of aluminium resistance in maize genotypes; citation_author=ED Mariano, WG Keltjens; citation_volume=256; citation_publication_date=2003; citation_pages=469-479; citation_doi=10.1023/A:1026106714644; citation_id=CR22"/>

    <meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=Genetic diversity for aluminium tolerance in sorghum; citation_author=FF Caniato, CT Guimaraes, RE Schaffert, VMC Alves, LV Kochian, A Boren, PE Klein, JV Magalhaes; citation_volume=114; citation_publication_date=2007; citation_pages=863-876; citation_doi=10.1007/s00122-006-0485-x; citation_id=CR23"/>

    <meta name="citation_reference" content="citation_journal_title=Genetics; citation_title=Comparative mapping of a major aluminium resistance gene in sorghum and other species of the poaceae; citation_author=JV Magalhaes, DF Garvin, Y Wang, ME Sorrels, PE Klein, RE Schaffert, L Li, LV Kochian; citation_volume=167; citation_publication_date=2004; citation_pages=1905-1914; citation_doi=10.1534/genetics.103.023580; citation_id=CR24"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=The relationship between population structure and aluminum tolerance in cultivated sorghum; citation_author=FF Caniato, CT Guimar&#227;es, M Hamblin, C Billot, J-F Rami; citation_volume=6; citation_issue=6; citation_publication_date=2011; citation_pages=e20830; citation_doi=10.1371/journal.pone.0020830; citation_id=CR25"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Plant Biol; citation_title=Two in one sweep: aluminum tolerance and grain yield in P-limited soils are associated to the same genomic region in west African sorghum; citation_author=WL Leiser, HFW Rattunde, E Weltzien, N Cisse, M Abdou, A Diallo, AO Tour&#232;, JV Magalhaes, BIG Haussmann; citation_volume=14; citation_publication_date=2014; citation_pages=206; citation_doi=10.1186/s12870-014-0206-6; citation_id=CR26"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=Association mapping provides insights into the origin and the fine structure of the sorghum aluminum tolerance locus, AltSB; citation_author=FF Caniato, MT Hamblin, CT Guimaraes, Z Zhang, RE Schaffert; citation_volume=9; citation_issue=1; citation_publication_date=2014; citation_pages=e87438; citation_doi=10.1371/journal.pone.0087438; citation_id=CR27"/>

    <meta name="citation_reference" content="citation_title=Comparison of maize populations for aluminium resistance in nutrient solution; citation_inbook_title=Genetic aspects of plant mineral nutrition. Dordrecht; citation_publication_date=1987; citation_pages=189-199; citation_id=CR28; citation_author=R Magnavaca; citation_author=C Gardner; citation_author=R Clark; citation_publisher=Martinus Nijhoff"/>

    <meta name="citation_reference" content="citation_journal_title=Afr Crop Sci J; citation_title=Tolerance to Aluminium toxicity in Tanzanian sorghum genotypes; citation_author=JH Ringo, EE Mneney, AO Onkware, BA Were, EJ Too, JO Owuoche, SO Gudu; citation_volume=18; citation_publication_date=2010; citation_pages=155-164; citation_id=CR29"/>

    <meta name="citation_reference" content="citation_journal_title=Genet Resour Crop Evol; citation_title=Molecular phylogeny of genus Guizotia (Asteraceae) using DNA sequences derived from ITS; citation_author=E Bekele, M Geleta, K Dagne, AL Jones, I Barnes, N Bradman, MG Thomas; citation_volume=54; citation_publication_date=2007; citation_pages=1419-1427; citation_doi=10.1007/s10722-006-9126-0; citation_id=CR30"/>

    <meta name="citation_reference" content="Don R, Cox P, Wainwright B, Baker K, Mattick J. &#8220;Touchdown&#8221; PCR to circumvent spurious priming during gene amplification&quot;. Nucleic Acids Res 1991;19:4008."/>

    <meta name="citation_reference" content="Geleta M, Bryngelsson T. Inter simple sequence repeats (ISSR) analysis of genetic diversity and population genetic structure of Lobelia rhynchopetalum (Hochest.) Hemsel. (Campanulaceae). Hereditas 2009;146:122&#8211;130."/>

    <meta name="citation_reference" content="citation_journal_title=BioTechniques; citation_title=Rapid silver staining and recovery of PCR products separated on polyacrylamide gels; citation_author=CJ Sanguinetti, NE Dias, AJ Simpson; citation_volume=17; citation_publication_date=1994; citation_pages=914-921; citation_id=CR33"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Clustal W and Clustal X version 2.0; citation_author=MA Larkin, G Blackshields, NP Brown; citation_volume=23; citation_publication_date=2007; citation_pages=2947-2948; citation_doi=10.1093/bioinformatics/btm404; citation_id=CR34"/>

    <meta name="citation_reference" content="citation_journal_title=Nucl Acids Symp Ser; citation_title=BioEdit: a user-friendly biological sequence alignment editor and analysis programme of windows 95/98/NT; citation_author=TA Hall; citation_volume=41; citation_publication_date=1999; citation_pages=95-98; citation_id=CR35"/>

    <meta name="citation_reference" content="Miftahudin G, Scoles J, Gustaffson JP. AFLP markers tightly linked to the aluminium-resistance gene Alt3 in rye (Secale cereale L.). Theor Appl Genet 2002;104:626&#8211;631."/>

    <meta name="citation_reference" content="Gallego FJ, Calles B, Benito C. Molecular markers linked to the aluminium resistance gene Alt1 in rye (Secale cereale, L.). Theor Appl Genet 1998;97:1104&#8211;1109."/>

    <meta name="citation_reference" content="citation_journal_title=Euphytica; citation_title=Mapping QTLs for aluminium resistance in maize; citation_author=FE Ninamango-C&#225;rdenas, C Guimar&#227;es, P Martins, S Parentoni, N Carneiro, M Lopes, J Moro, E Paiva; citation_volume=130; citation_publication_date=2003; citation_pages=223-232; citation_doi=10.1023/A:1022867416513; citation_id=CR38"/>

    <meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=Identification of AFLP and microsatellite markers linked with aluminium resistance gene in barley (Hordeum vulgare, L); citation_author=H Raman, JS Moroni, K Sato, BJ Read, BJ Scott; citation_volume=105; citation_publication_date=2002; citation_pages=458-464; citation_doi=10.1007/s00122-002-0934-0; citation_id=CR39"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Cell; citation_title=A bacterial-type ABC transporter is involved in aluminum tolerance in rice; citation_author=CF Huang, N Yamaji, N Mitani, M Yano, Y Nagamura, JF Ma; citation_volume=21; citation_publication_date=2009; citation_pages=655-667; citation_doi=10.1105/tpc.108.064543; citation_id=CR40"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Cell; citation_title=A zinc finger transcription factor ART1 regulates multiple genes implicated in aluminum tolerance in rice; citation_author=N Yamaji, CF Huang, S Nagao, M Yano, Y Sato, Y Nagamura, JF Ma; citation_volume=21; citation_publication_date=2009; citation_pages=3339-3349; citation_doi=10.1105/tpc.109.070771; citation_id=CR41"/>

    <meta name="citation_reference" content="citation_journal_title=Euphytica; citation_title=The development and use of microsatellite markers for genetic analysis and plant breeding with emphasis on bread wheat; citation_author=PK Gupta, RK Varshney; citation_volume=113; citation_publication_date=2000; citation_pages=163-185; citation_doi=10.1023/A:1003910819967; citation_id=CR42"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Breed; citation_title=Detection and mapping of SSRs in rye ESTs from aluminium-stressed roots; citation_author=M Matos, V P&#233;rez-Flores, MV Camacho, B Pernaute, O Pinto-Carnide, C Benito; citation_volume=20; citation_publication_date=2007; citation_pages=103-115; citation_doi=10.1007/s11032-007-9077-y; citation_id=CR43"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Mol Biol; citation_title=A high density genetic map of Sorghum bicolor (L) Moench based on 2926 AFLP, RFLP and SSR markers; citation_author=MA Menz, RR Klein, JE Mullet, JA Obert, NC Unruh, PE Klein; citation_volume=48; citation_publication_date=2002; citation_pages=483-499; citation_doi=10.1023/A:1014831302392; citation_id=CR44"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Physiol; citation_title=Sensitive to proton Rhizotoxicity1, calmodulin binding transcription activator 2, and other transcription factors are involved in aluminum-activated malate Transporter1 expression; citation_author=M Tokizawa, Y Kobayashi, T Saito, M Kobayashi, S Iuchi; citation_volume=167; citation_publication_date=2015; citation_pages=991-1003; citation_doi=10.1104/pp.114.256552; citation_id=CR45"/>

    <meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=Homoeology of rye chromosome arms to wheat; citation_author=T Naranjo, P Fern&#225;ndez-Rueda; citation_volume=82; citation_publication_date=1991; citation_pages=577-586; citation_doi=10.1007/BF00226793; citation_id=CR46"/>

    <meta name="citation_reference" content="citation_journal_title=Genome; citation_title=RFLP mapping of rye chromosome 7R reveals a highly translocated chromosome relative to wheat; citation_author=OA Rognli, KM Devos, CN Chino, RL Harcourt, MD Atkinson, MD Gale; citation_volume=35; citation_publication_date=1992; citation_pages=1026-1031; citation_doi=10.1139/g92-157; citation_id=CR47"/>

    <meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=Chromosomal rearrangements in the rye genome relative to that of wheat; citation_author=KM Devos, MD Atkinson, CN Chinoy, HA Francis, RL Harcourt, RMD Koebner, CJ Liu, P Masojc, DX Xie, MD Gale; citation_volume=85; citation_publication_date=1993; citation_pages=673-680; citation_doi=10.1007/BF00225004; citation_id=CR48"/>

    <meta name="citation_reference" content="Naranjo T, Fern&#225;ndez-Rueda P, Maestra B. Chromosome rearrangements and homoeologous pairing: implications for the introgression of alien genes into wheat. In: Current topics in plant cytogenetics related to plant improvement Lelley T, editor. WUV-Univeit&#228;tsverlag, Austria;1997. p. 198&#8211;205."/>

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci U S A; citation_title=Comparative genetics in the grasses; citation_author=MD Gale, KM Devos; citation_volume=95; citation_publication_date=1998; citation_pages=1971-1974; citation_doi=10.1073/pnas.95.5.1971; citation_id=CR50"/>

    <meta name="citation_reference" content="citation_journal_title=Int J Nat Sci Res; citation_title=Physiological characterization of Kenyan sorghum lines for tolerance to aluminium; citation_author=RK Cheprot, TK Matonyei, KK Maritim, BA Were, OG Dangasuk, AO Onkware, S Gudu; citation_volume=2; citation_publication_date=2014; citation_pages=59-71; citation_id=CR51"/>

    <meta name="citation_reference" content="Fontecha G, Silva-Navas J, Benito C, Metres MA, Espino FJ, Hernandez-Riquer MV, Gallego FJ. Candidate gene identification of an aluminum-activated organic acid transporter gene at the Alt4 locus for aluminum resistance in rye (Secale cereale, L.). Theor Appl Genet 2007;114:249&#8211;260."/>

    <meta name="citation_reference" content="Collins NC, Shirley NJ, Saeed M, Pallotta M, Gustafson JP. An ALMTI gene cluster controlling aluminum resistance at the Atl4 locus of rye (Secale cereale L.). Genetics 2008;179:669&#8211;682."/>

    <meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=Genetic diversity among elite sorghum inbred lines assessed with simple sequence repeats; citation_author=JSC Smith, S Kresovich, MS Hopkins, SE Mitchel, RE Dean, WL Woodman, M Lee, K Porter; citation_volume=40; citation_publication_date=2000; citation_pages=226-232; citation_doi=10.2135/cropsci2000.401226x; citation_id=CR54"/>

    <meta name="citation_reference" content="Kong L, Dong J, Hart GE. Characteristics, linkage-map positions, and allelic differentiation of Sorghum bicolor (L) Moech DNA simple sequence repeats (SSRs). Theor Appl Genet 2000;101:438&#8211;448."/>

    <meta name="citation_author" content="Emily Jepkosgei Too"/>

    <meta name="citation_author_institution" content="Department of Biological Sciences, University of Eldoret, Eldoret, Kenya"/>

    <meta name="citation_author" content="Augustino Osoro Onkware"/>

    <meta name="citation_author_institution" content="Rongo University College, Rongo, Kenya"/>

    <meta name="citation_author" content="Beatrice Ang&#8217;iyo Were"/>

    <meta name="citation_author_institution" content="Department of Biological Sciences, University of Eldoret, Eldoret, Kenya"/>

    <meta name="citation_author" content="Samuel Gudu"/>

    <meta name="citation_author_institution" content="Rongo University College, Rongo, Kenya"/>

    <meta name="citation_author" content="Anders Carlsson"/>

    <meta name="citation_author_institution" content="Department of Plant Breeding, Swedish University of Agricultural Sciences, Alnarp, Sweden"/>

    <meta name="citation_author" content="Mulatu Geleta"/>

    <meta name="citation_author_institution" content="Department of Plant Breeding, Swedish University of Agricultural Sciences, Alnarp, Sweden"/>



        <meta charset="UTF-8"/>
        <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
        <meta name="viewport" content="width=device-width, initial-scale=1"/>

        <!-- For IE 9 and below -->
<!--[if IE]><link rel="shortcut icon" href=/static/images/favicons//favicon.ico><![endif]-->
<!-- Touch Icons - iOS and Android 2.1+ -->
<link rel="apple-touch-icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>
<!-- Firefox, Chrome, Safari, IE 11+ and Opera -->
<link rel="icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>


        <script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script>
        <script>(function(d){if (sessionStorage.fontsLoaded){d.style.setProperty('--font-family-sans', 'Europa');d.style.setProperty('--font-family-serif', 'Noto Serif')}})(document.documentElement);</script>

        <link rel="stylesheet" media="screen" href=/static/styles/basic/basic-a31ca85bc2.css>
<link rel="stylesheet" media="print" href=/static/styles/bmc/print-3a6240f456.css>

<link rel="stylesheet" class="js-ctm" href=/static/styles/bmc/main-84414d3d47.css
      media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">

        

        <script src="/static/js/app-bundle-8d61e6ef37.js" defer></script>
        
        <script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script>
        
    <script type="text/javascript">
        config = {
            env: 'live',
            site: 'hereditasjournal.biomedcentral.com',
            siteWithPath: 'hereditasjournal.biomedcentral.com' + window.location.pathname,
            twitterHashtag: '',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/s41065-018-0059-3',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central'
        };
    </script>

        
    <script type="text/javascript">
        dataLayer = [{"content":{"article":{"doi":"10.1186/s41065-018-0059-3","articleType":"Research","peerReviewType":"Closed","supplement":null,"keywords":"Aluminium tolerance;Mapping population;Molecular markers;Net root length in aluminium;Sorghum bicolor"},"contentInfo":{"imprint":"BioMed Central","title":"Molecular markers associated with aluminium tolerance in Sorghum bicolor","publishedAt":1523577600000,"publishedAtDate":"2018-04-13","author":["Emily Jepkosgei Too","Augustino Osoro Onkware","Beatrice Ang’iyo Were","Samuel Gudu","Anders Carlsson","Mulatu Geleta"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":"CC BY + CC0","openAccess":true},"environment":"live"},"journal":{"siteKey":"hereditasjournal.biomedcentral.com","volume":"155","issue":"1","title":"Hereditas","type":"AJs Life","journalID":41065,"gaCode":"UA-62066850-11","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Research","publishingSegment":"Genetics & Heredity (Life)"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}}}]

    </script>

        <script src="https://cdn.cookielaw.org/consent/7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script>
        
    
        
    

        
    
        
            
                <!-- Google Tag Manager -->
                <script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script>
                <!-- End Google Tag Manager -->
            
        
    


        
    
    <link rel="canonical" href="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0059-3"/>
    

        
        
        <meta property="og:url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0059-3"/>
        <meta property="og:type" content="article"/>
        <meta property="og:site_name" content="Hereditas"/>
        <meta property="og:title" content="Molecular markers associated with aluminium tolerance in Sorghum bicolor"/>
        <meta property="og:description" content="Sorghum (Sorghum bicolor, L. Moench) production in many agro-ecologies is constrained by a variety of stresses, including high levels of aluminium (Al) commonly found in acid soils. Therefore, for such soils, growing Al tolerant cultivars is imperative for high productivity. In this study, molecular markers associated with Al tolerance were identified using a mapping population developed by crossing two contrasting genotypes for this trait. Four SSR (Xtxp34, Sb5_236, Sb6_34, and Sb6_342), one STS (CTG29_3b) and three ISSR (811_1400, 835_200 and 884_200) markers produced alleles that showed significant association with Al tolerance. CTG29_3b, 811_1400, Xtxp34 and Sb5_236 are located on chromosome 3 with the first two markers located close to Alt SB , a locus that underlie the Al tolerance gene (SbMATE) implying that their association with Al tolerance is due to their linkage to this gene. Although CTG29_3b and 811_1400 are located closer to Alt SB , Xtxp34 and Sb5_236 explained higher phenotypic variance of Al tolerance indices. Markers 835_200, 884_200, Sb6_34 and Sb6_342 are located on different chromosomes, which implies the presence of several genes involved in Al tolerance in addition to SbMATE in sorghum. These molecular markers have a high potential for use in breeding for Al tolerance in sorghum."/>
        <meta property="og:image" content="https://images.springer.com/sgw/journals/medium/41065.jpg"/>
    
    </head>

    <body class="journal journal-fulltext"
          
          >
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK"
                            height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><feColorMatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"/></filter></defs><symbol id="icon-arrow-down" viewBox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"/></symbol><symbol id="icon-arrow-left-bullet" viewBox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"/></symbol><symbol id="icon-arrow-right-bullet" viewBox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"/></symbol><symbol id="icon-arrow-right" viewBox="0 0 32 64"><path d="M0 0v64l32-32z"/></symbol><symbol id="icon-close" viewBox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"/></symbol><symbol id="icon-community" viewBox="0 0 60.643 31.01"><mask maskUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><path fill="#FFF" d="M0 31.01h60.643V0H0z" filter="url(#a)"/></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"/></symbol><symbol id="icon-download" viewBox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"/></symbol><symbol id="icon-ext-link" viewBox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"/><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"/><path fill="#676767" d="M9.7 0L16 6.2V0z"/></symbol><symbol id="icon-info-bordered" viewBox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"/><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"/></symbol><symbol id="icon-opr" viewBox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"/><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"/></symbol><symbol id="icon-quotation" viewBox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"/></symbol><symbol id="icon-remove" viewBox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"/><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"/></symbol><symbol id="icon-ribbon" viewBox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"/></symbol><symbol id="icon-rss" viewBox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"/><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"/><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"/></g></symbol><symbol id="icon-search" viewBox="0 0 16 16"><path d="M6.276 12.257C2.815 12.257 0 9.508 0 6.128 0 2.749 2.815 0 6.276 0c3.462 0 6.277 2.749 6.277 6.128.002 3.38-2.815 6.129-6.277 6.129zm0-10.772c-2.622 0-4.755 2.083-4.755 4.643 0 2.562 2.133 4.644 4.755 4.644 2.623 0 4.756-2.082 4.756-4.644 0-2.56-2.134-4.643-4.756-4.643z"/><path d="M16 13.751L13.641 16l-4.875-4.82 2.36-2.248L16 13.751z"/></symbol><symbol id="icon-speech-bubble" viewBox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"/></symbol><symbol id="icon-error" viewBox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"/></symbol><symbol id="icon-info" viewBox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"/></symbol><symbol id="icon-success" viewBox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"/></symbol><symbol id="icon-warning" viewBox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"/></symbol><symbol id="icon-facebook-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"/></symbol><symbol id="icon-facebook" viewBox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"/><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"/></symbol><symbol id="icon-gplus-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M480.5 273.117c.123.076.26.185.416.334.146.152.289.343.432.562.135.211.258.459.359.746.086.286.129.619.129 1.004-.012.7-.164 1.262-.465 1.683a5.374 5.374 0 0 1-.461.562c-.186.176-.375.35-.58.527a2.65 2.65 0 0 0-.322.41.908.908 0 0 0-.176.553c0 .205.059.375.18.508.102.129.199.238.299.336l.662.545a6.48 6.48 0 0 1 1.086 1.119c.293.415.445.955.459 1.625 0 .949-.42 1.789-1.256 2.521-.869.76-2.123 1.15-3.762 1.177-1.373-.015-2.4-.308-3.074-.879-.682-.534-1.023-1.172-1.023-1.917 0-.359.111-.766.336-1.211.215-.443.602-.837 1.172-1.174a6.126 6.126 0 0 1 1.998-.725 15.415 15.415 0 0 1 1.723-.173 3.996 3.996 0 0 1-.377-.596 1.347 1.347 0 0 1-.193-.74c0-.174.023-.317.074-.433a8.83 8.83 0 0 1 .121-.338 5.973 5.973 0 0 1-.633.035c-1.043-.012-1.838-.339-2.381-.979-.574-.603-.861-1.3-.861-2.097 0-.961.404-1.832 1.217-2.616.559-.46 1.135-.757 1.736-.898a8.494 8.494 0 0 1 1.67-.182h3.916l-1.209.704h-1.207v.005l-.005.002zm.754 11.276c0-.498-.162-.932-.484-1.302-.346-.348-.881-.778-1.609-1.29a4.045 4.045 0 0 0-.439-.02c-.102-.016-.354 0-.768.039a8.141 8.141 0 0 0-1.244.275c-.1.036-.24.096-.42.17a2.713 2.713 0 0 0-.553.355 1.985 1.985 0 0 0-.459.592c-.145.25-.215.551-.215.901 0 .687.311 1.253.932 1.698.596.444 1.404.68 2.432.688.928-.014 1.629-.217 2.117-.612.476-.385.71-.886.71-1.494zm-2.729-5.743a1.843 1.843 0 0 0 1.289-.556c.164-.249.271-.506.318-.767a7.32 7.32 0 0 0 .041-.659c0-.772-.195-1.552-.594-2.336a2.611 2.611 0 0 0-.73-.921 1.894 1.894 0 0 0-1.064-.352 1.723 1.723 0 0 0-1.324.64c-.299.437-.438.92-.428 1.457 0 .709.209 1.444.623 2.211.201.356.459.659.775.912.323.248.686.371 1.094.371zm9.698.077h-2.055v-2.056h-.996v2.056h-2.053v.998h2.053v2.052h.996v-2.052h2.055z"/></symbol><symbol id="icon-gplus" viewBox="0 0 24 24"><circle fill="#D84A36" cx="12" cy="12" r="12"/><g fill="#FFF"><path d="M12.012 12.111c-.304-.33-.07-.842.155-1.078.226-.241.639-.56.829-.74.193-.184.66-.605.674-1.868 0 0 .112-1.139-1.127-2.06h.977l1.038-.593h-3.507s-2.083-.067-3.085 1.207c-1 1.276-.966 2.871.114 3.77 1.08.898 2.371.648 2.371.648s-.197.422-.163.775c.036.353.421.849.504.965 0 0-2.386-.011-3.551 1.033-1.165 1.053-1.29 3.097 1.021 3.803 2.312.707 5.303-.055 5.906-2.277 0 0 .262-.908-.189-1.683-.459-.775-1.663-1.571-1.967-1.902zm-2.555-1.83c-.744-1-1.308-2.953 0-3.783 0 0 .955-.579 1.801.266.846.846 1.082 2.277.928 3.003-.15.727-.563.897-.739 1.008-.185.111-1.249.504-1.99-.494zm3.668 5.481c-.035.393-.083 1.254-1.491 1.664-1.409.408-3.826-.264-3.757-1.914 0 0 .006-.698.449-1.089.445-.396 1.094-.794 3.029-.819 0 0 .643.473 1.072.809.427.332.733.952.698 1.349z"/><path d="M16.323 11.055V9.803h-.752v1.252h-1.255v.748h1.255v1.256h.752v-1.256h1.248v-.748z"/></g></symbol><symbol id="icon-linkedin" viewBox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"/><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"/><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"/></g></symbol><symbol id="icon-reddit" viewBox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"/><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"/><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"/><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"/><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"/><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"/><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"/><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"/></symbol><symbol id="icon-twitter-bordered" viewBox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"/><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/></symbol><symbol id="icon-twitter" viewBox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"/><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"/></symbol><symbol id="icon-weibo-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"/><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"/><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"/></symbol><symbol id="icon-weibo" viewBox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"/><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"/><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"/><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"/><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"/><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"/></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner c-banner--info">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner" >
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1"
                 data-gpt-unitpath="/270604982/bmc/hereditasjournal/articles"
                 data-gpt-sizes="728x90,970x90"
                 data-gpt-targeting="pos=LB1;doi=10.1186/s41065-018-0059-3;kwrd=Aluminium tolerance,Mapping population,Molecular markers,Net root length in aluminium,Sorghum bicolor;pmc=L32030,L21001;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-018-0059-3&amp;kwrd=Aluminium tolerance,Mapping population,Molecular markers,Net root length in aluminium,Sorghum bicolor&amp;pmc=L32030,L21001&amp;">
                        <img data-test="gpt-advert-fallback-img"
                             src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-018-0059-3&amp;kwrd=Aluminium tolerance,Mapping population,Molecular markers,Net root length in aluminium,Sorghum bicolor&amp;pmc=L32030,L21001&amp;"
                             alt="Advertisement"
                             width="728"
                             height="90">
                    </a>
                </noscript>
            </div>
        </div>
    </div>

        
         
    <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>


<header class="c-popup-search ctx-publisher-header" id="top" data-test="publisher-header">
    <nav class="c-navbar">
        <div class="u-container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization" />
                <div class="c-logo">
    <a href="https://www.biomedcentral.com" itemprop="url">
        <img src=/static/images/bmc/logos/logo-bmc-white-aj-e3a70ae848.svg alt="BMC" itemprop="logo">
        <div class="c-logo__strapline">
            <img src=/static/images/bmc/logos/logo-bmc-white-strapline-sn-e7b1d6e67f.svg alt="Part of Springer Nature">
        </div>
    </a>
</div>
            </div>
            <button
                type="button"
                class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg"
                id="publisher-header-menu-button"
                data-toggle="collapse"
                data-test="publisher-header-menu-toggle"
                data-toggle-linked="publisher-header"
                data-target="publisher-header-search publisher-header-menu"
                aria-controls="publisher-header-menu publisher-header-search"
                aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


                <ul class="c-navbar__nav">
                    <li class="c-navbar__item">
                        <a data-replace="true"
                           data-placeholder="/placeholder/v1/username"
                           class="c-navbar__link"
                           href="https://www.biomedcentral.com/login"
                           data-test="login-link"
                           data-track="click"
                           data-track-category="Account"
                           data-track-action="Click Login Link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                    <li class="c-navbar__item">
                        <button
                            type="button"
                            class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg"
                            data-toggle="collapse"
                            data-test="header-search-button"
                            data-target="publisher-header-search publisher-header-menu publisher-header-menu-button"
                            aria-controls="publisher-header-search publisher-header-menu"
                            aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true">
                                    <use xlink:href="#icon-search"></use>
                                </svg>
                            </span>
                        </button>
                    </li>
                </ul>
            </div>
        </div>
    </nav>
    <div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field c-form-field--inline" method="GET" action="//www.biomedcentral.com/search" data-track="submit"
        data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label data-track-label="" data-test="global-search">
        <label for="publisherSearch" class="u-visually-hidden">Search all BMC articles</label>
        <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text" value="" placeholder="Search all BMC articles"/>
        <button class="c-button c-button--secondary" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true">
        <use xlink:href="#icon-search"></use>
    </svg>
</button>


        <input type="hidden" name="searchType" value="publisherSearch"/>
    </form>
</div>

            </div>
        </div>
    </div>
</header>


        
            <header class="c-journal-header c-journal-header--hereditas ctx-journal-header">
                <div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="/">
    
        
    
    <span class="c-journal-title__text ">Hereditas</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="u-container">
                        
                            <button type="button"
                                    class="c-navbar__link c-dropdown u-hidden-at-lg"
                                    data-toggle="collapse"
                                    data-test="journal-header-menu"
                                    data-toggle-linked="journal-header"
                                    data-target="journal-menu"
                                    aria-controls="journal-menu"
                                    aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav">
                                    <ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/'>Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/about'>About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href='/articles'>Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/submission-guidelines'>Submission Guidelines</a>
                                            </li>
                                        
                                    </ul>
                                </nav>
                            </div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--ocean-light-blue">
                    
                </div>
            </header>
            
        

        <div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-content-layout c-content-layout--fulltext" data-component="sticky-container">
                <div class="c-content-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3><ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc=""><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Methods">Methods</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec9" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results">Results</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec12" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Discussion">Discussion</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec13" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Conclusion">Conclusion</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li></ol></nav>
                </div>

                <main class="c-content-layout__main">
                    <article itemscope itemtype="http://schema.org/ScholarlyArticle">
                        
                        
    <div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
            <li class="c-article-identifiers__item" data-test="article-category">Research</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
    </div>

                        <div class="FulltextWrapper"><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">Molecular markers associated with aluminium tolerance in <em xmlns="" class="EmphasisTypeItalic">Sorghum bicolor</em></h1></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true"><ul class="u-listReset"><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Emily&nbsp;Jepkosgei&nbsp;Too</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Augustino&nbsp;Osoro&nbsp;Onkware</span><sup><a href="#Aff2" class="AffiliationID">2</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Beatrice&nbsp;Ang’iyo&nbsp;Were</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Samuel&nbsp;Gudu</span><sup><a href="#Aff2" class="AffiliationID">2</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Anders&nbsp;Carlsson</span><sup><a href="#Aff3" class="AffiliationID">3</a></sup> and </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Mulatu&nbsp;Geleta</span><sup><a href="#Aff3" class="AffiliationID">3</a></sup><a href="mailto:Mulatu.Geleta.Dida@slu.se" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a><a class="Orcid" title="View ORCID ID profile" href="http://orcid.org/0000-0002-2293-1178" target="_blank" rel="noopener"><span class="OrcidIcon" aria-hidden="true"></span><span class="u-srOnly">View ORCID ID profile</span></a></li></ul></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation"><div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">Hereditas</span></span><span class="ArticleCitation_Year">2018</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">155</strong>:20</span></span></div><p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/s41065-018-0059-3">https://doi.org/10.1186/s41065-018-0059-3</a></p><p class="Copyright">© &nbsp;The Author(s)&nbsp;2018</p></div></div><ul class="ArticleHistory u-reset-list"><li class="History HistoryReceived"><strong>Received: </strong><span>24&nbsp;August&nbsp;2017</span></li><li class="History HistoryAccepted"><strong>Accepted: </strong><span>3&nbsp;April&nbsp;2018</span></li><li class="History HistoryOnlineDate"><strong>Published: </strong><span itemprop="datePublished">13&nbsp;April&nbsp;2018</span></li></ul></div>
                        
                        
                        
    

                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2><div class="c-section__content" id="Abstract"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p id="Par1" class="Para">Sorghum (<em xmlns="" class="EmphasisTypeItalic">Sorghum bicolor</em>, L. Moench) production in many agro-ecologies is constrained by a variety of stresses, including high levels of aluminium (Al) commonly found in acid soils. Therefore, for such soils, growing Al tolerant cultivars is imperative for high productivity.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Methods</h3>
                <p id="Par111" class="Para">In this study, molecular markers associated with Al tolerance were identified using a mapping population developed by crossing two contrasting genotypes for this trait.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Results</h3>
                <p id="Par112" class="Para">Four SSR (<em xmlns="" class="EmphasisTypeItalic">Xtxp34</em>, <em xmlns="" class="EmphasisTypeItalic">Sb5_236</em>, <em xmlns="" class="EmphasisTypeItalic">Sb6_34</em>, and <em xmlns="" class="EmphasisTypeItalic">Sb6_342</em>), one STS (<em xmlns="" class="EmphasisTypeItalic">CTG29_3b</em>) and three ISSR (<em xmlns="" class="EmphasisTypeItalic">811_1400</em>, <em xmlns="" class="EmphasisTypeItalic">835_200</em> and <em xmlns="" class="EmphasisTypeItalic">884_200</em>) markers produced alleles that showed significant association with Al tolerance. <em xmlns="" class="EmphasisTypeItalic">CTG29_3b, 811_1400</em>, <em xmlns="" class="EmphasisTypeItalic">Xtxp34</em> and <em xmlns="" class="EmphasisTypeItalic">Sb5_</em>236 are located on chromosome 3 with the first two markers located close to <em xmlns="" class="EmphasisTypeItalic">Alt</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">SB</em>
                  </sub>, a locus that underlie the Al tolerance gene (<em xmlns="" class="EmphasisTypeItalic">SbMATE</em>) implying that their association with Al tolerance is due to their linkage to this gene. Although <em xmlns="" class="EmphasisTypeItalic">CTG29_3b</em> and <em xmlns="" class="EmphasisTypeItalic">811_</em>1400 are located closer to <em xmlns="" class="EmphasisTypeItalic">Alt</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">SB</em>
                  </sub>, <em xmlns="" class="EmphasisTypeItalic">Xtxp34</em> and <em xmlns="" class="EmphasisTypeItalic">Sb5_236</em> explained higher phenotypic variance of Al tolerance indices. Markers <em xmlns="" class="EmphasisTypeItalic">835_200</em>, <em xmlns="" class="EmphasisTypeItalic">884_200</em>, <em xmlns="" class="EmphasisTypeItalic">Sb6_34</em> and <em xmlns="" class="EmphasisTypeItalic">Sb6_342</em> are located on different chromosomes, which implies the presence of several genes involved in Al tolerance in addition to S<em xmlns="" class="EmphasisTypeItalic">bMATE</em> in sorghum.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec4">
                <h3 xmlns="" class="Heading">Conclusion</h3>
                <p id="Par113" class="Para">These molecular markers have a high potential for use in breeding for Al tolerance in sorghum.</p>
              </div></div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords"><li class="c-keywords__item" itemprop="about">Aluminium tolerance</li><li class="c-keywords__item" itemprop="about">Mapping population</li><li class="c-keywords__item" itemprop="about">Molecular markers</li><li class="c-keywords__item" itemprop="about">Net root length in aluminium</li><li class="c-keywords__item" itemprop="about">
                <em xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="EmphasisTypeItalic">Sorghum bicolor</em>
              </li></ul></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par2" class="Para">Sorghum (<em xmlns="" class="EmphasisTypeItalic">Sorghum bicolor</em> L. Moench), is a staple cereal in many parts of Africa and Asia where it is grown mostly on small-scale, resource poor holdings. Although it is a hardy cereal, its production is significantly reduced by aluminium (Al) stress, especially in the highly leached acidic soils [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>, <span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>]. Inhibition of root growth is the primary symptom of Al stress in plants [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>, <span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>]. The primary consequence of Al stress is a poorly developed root system that makes the crop vulnerable to drought and limited nutrient uptake [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>] and ultimately reduced crop yields [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>]. In Kenya, over 70% of sorghum is produced in the western and central regions. These areas are characterized by acid soils [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>] with Al percent saturation ranging from 4 to 46% [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>]. Low sorghum grain yield in the country is partly attributed to acid soil stress [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>] including Al toxicity. Therefore, there is the need to deploy Al tolerant sorghum cultivars in order to increase productivity.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par3" class="Para">Plant tolerance to aluminium stress is based on exclusion or internal mechanisms that enable Al to be tolerated once it has entered the plant cells [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>, <span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>]. Some plant species accumulate Al and complexed it with other substances to render it less toxic [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>]. Al tolerant cultivars of sorghum and most other cereals use the Al exclusion mechanisms based mainly on secretion of organic acids, such as citrate, malate and oxalate that chelate Al outside the cells and thereby reduce its availability [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>, <span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>]. When exposed to Al stress, tolerant sorghum varieties reportedly secrete large quantities of citric acid [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>], malic acid and trans-aconitic acids [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>, <span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>]. Aluminium-induced exudation of organic acids in plant roots is mediated by anionic channels in the plasma membrane [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>]. The plant genes involved in the Al-induced acid exudation are members of the aluminium activated malate transporter (ALMT) and the multi-drug and toxin extrusion (MATE) families that encode membrane transporter proteins [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>, <span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>, <span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par4" class="Para">Although significant progress has been made in crop improvement through phenotypic selection for Al tolerance [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>, <span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>–<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>], the testing procedures may be difficult and time consuming due to the effect of genotype by environment interactions for this trait. In this regard, molecular markers based screening procedure can be more efficient than the use of morphological markers in identifying Al tolerant sorghum genotypes.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par5" class="Para">Sorghum has significant genotypic variation for tolerance to Al [<span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>, <span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>], which can be exploited to breed genotypes with superior tolerance to Al stress. Significant progress has been made in developing genomic tools and resources related to aluminium tolerance in sorghum, including the development of some molecular markers that can be used for marker assisted selection [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>, <span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>–<span class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>]. However, additional work has to be done through exploring global sorghum genetic resources to identify additional genomic regions that contribute to this trait and eventually develop sets of molecular markers that can be used to efficiently breed sorghum for Al tolerance.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par6" class="Para">An aluminium tolerance locus referred to as <em xmlns="" class="EmphasisTypeItalic">Alt</em><sub>
                  <em xmlns="" class="EmphasisTypeItalic">SB</em>
                </sub> was identified through comparative mapping procedure in sorghum [<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>]. This locus was later found to underlie <em xmlns="" class="EmphasisTypeItalic">SbMATE,</em> a MATE family gene, which encodes a protein that is responsible for citric acid exudation [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>]. Causative polymorphisms that include insertions and single nucleotide polymorphism (SNPs) that were positively correlated with aluminium tolerance have been identified within this gene [<span class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>]. STS markers designated as <em xmlns="" class="EmphasisTypeItalic">CTG29</em> and <em xmlns="" class="EmphasisTypeItalic">M181</em> have been reported to be closely linked to <em xmlns="" class="EmphasisTypeItalic">Alt</em><sub>
                  <em xmlns="" class="EmphasisTypeItalic">SB</em>
                </sub> at 0.2&nbsp;cM and 0&nbsp;cM, respectively [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>]. However, it was not known whether the sorghum population used in this study relied on <em xmlns="" class="EmphasisTypeItalic">Alt</em><sub>
                  <em xmlns="" class="EmphasisTypeItalic">SB</em>
                </sub> or different Al tolerance gene(s). Conserved gene order in genomic regions harboring Al tolerance loci has been reported in members of the grass family [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>]. Hence, with the availability of saturated genetic maps of major cereals, it is possible to identify markers linked to Al tolerance genes in sorghum that are orthologous to Al tolerance genes in other cereal crops. The present study was conducted to identify molecular markers associated with genes/QTLs that confer tolerance to aluminium toxicity in sorghum.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Methods" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Methods">Methods</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Methods" data-test="collapse-section"><section id="Sec3" class="Section2 RenderAsSection2">
                <h3 class="Heading">Plant material and the development of the mapping population</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par7" class="Para">Two sorghum lines, <em xmlns="" class="EmphasisTypeItalic">Seredo</em> and <em xmlns="" class="EmphasisTypeItalic">ICSR 110,</em> were used to develop a mapping population for the identification of molecular markers associated with aluminium tolerance. <em xmlns="" class="EmphasisTypeItalic">Seredo</em> is a popular but Al-sensitive commercial Kenyan sorghum with medium height and early maturity that is marketed by the Kenya Seed Company (MUSRT). <em xmlns="" class="EmphasisTypeItalic">ICSR 110</em> is an early maturing, medium height, Al-tolerant inbred line developed by International Crops Research Institute for the Semi-Arid Tropics (ICRISAT, Hyderabad, India). <em xmlns="" class="EmphasisTypeItalic">Seredo</em> was used as a pollen recipient from <em xmlns="" class="EmphasisTypeItalic">ICSR 110</em> to develop the hybrid population that was advanced to F<sub>3</sub> by selfing. The mapping population comprised 229&nbsp;F<sub>2:3</sub> progenies derived from 22&nbsp;F<sub>2</sub> plants, which in turn derived from three different F<sub>1</sub> plants.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par8" class="Para">The Al tolerance of parental lines and F<sub>2:3</sub> progenies was evaluated according to the procedures described by Magalhaes et al. [<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>] using the basal nutrient solution of Magnavaca et al. [<span class="CitationRef"><a href="#CR28" data-toggle="citation">28</a></span>]. A concentration of 148&nbsp;μM Al was used to study the effect of aluminium on root growth of the seedlings based on recommendations from previous studies [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>, <span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>]. For this purpose, seeds were sterilized in 1% hypochlorite and germinated on paper.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par9" class="Para">Since individual plants in F<sub>2:3</sub> population are not genetically identical, it was not possible to set up a separate control experiment. Hence, the following procedure was followed to determine the root growth of individual plants under a solution without Al (control solution) and under a solution containing Al, as described in Caniato et al. [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>] and Ringo et al. [<span class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>]. The two solutions have the same composition except that the latter contains 148&nbsp;μM Al. First, the seedlings were given a germination period of 4 days and the root length of each plant, which is referred to as initial root length under control solution (il<sub>c</sub>), was measured. Then, the seedlings were acclimated in a control nutrient solution for 24&nbsp;h (1&nbsp;day) and the root length of each plant, which is referred to as final root length under control solution (fl<sub>c</sub>), was measured. The seedlings were then transferred to a solution containing 148&nbsp;μM Al and allowed to grow for 5 days and the root length of each plant, which is referred to as final root length under Al (fl<sub>Al</sub>), was measured. Based on these measurements, the net root length of each plant under Al (NRL<sub>Al</sub>) was calculated as fl<sub>Al</sub>-fl<sub>c</sub>, whereas the percent relative root growth of each plant under the Al solution as compared to the growth under the control solution (%RRG) was calculated as [(fl<sub>Al</sub>–fl<sub>c</sub>)<sub>5d</sub>/(fl<sub>c</sub>–il<sub>c</sub>)<sub>1d</sub> × 5] × 100. The values of NRL<sub>Al</sub> and %RRG were used to classify the F<sub>2:3</sub> progenies into tolerant or sensitive groups. Leaves were sampled from the parents, F<sub>1</sub> and F<sub>2:3</sub> seedlings for DNA extraction and marker analysis.</p>
              </section><section id="Sec4" class="Section2 RenderAsSection2">
                <h3 class="Heading">DNA extraction</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par10" class="Para">Genomic DNA was extracted separately from leaf tissue of each F<sub>2:3</sub> seedling, parent and F<sub>1</sub> hybrid using a cetyltrimethyl ammonium bromide (CTAB) method, as described in Bekele et al. [<span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>]. In total, DNA was extracted from six parental plants (three <em xmlns="" class="EmphasisTypeItalic">ICSR 110</em> and three <em xmlns="" class="EmphasisTypeItalic">Seredo</em>), three F<sub>1</sub> and 229&nbsp;F<sub>2:3</sub> plants. The DNA quality and concentration were assessed using a Nanodrop® ND-1000 spectrophotometer (Saveen &amp; Werner®, Malmö, Sweden) and by ethidium bromide staining following electrophoresis on 1.5% agarose gel.</p>
              </section><section id="Sec5" class="Section2 RenderAsSection2">
                <h3 class="Heading">PCR amplification of ISSR, SSR and STS markers</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par11" class="Para">Fifty (50) inter-simple sequence repeat (ISSR) primers, twenty four (24) simple sequence repeat (SSR) primer-pairs and two (2) sequence tagged site (STS) primer-pairs were tested on Al tolerant and sensitive parental lines to identify potential markers linked to Al tolerance.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par12" class="Para">The 50 ISSR primers were selected for screening, as they generated clear and well-separated banding pattern suitable for detecting polymorphism. The ISSR amplification reaction was performed in a total volume of 25&nbsp;μl containing 1× reaction buffer (20&nbsp;mM Tris-HCl pH&nbsp;8.55, 16&nbsp;mM (NH<sub>4</sub>)SO<sub>4</sub>, 0.01% Tween®20 and 2&nbsp;mM MgCl<sub>2</sub>), 0.4&nbsp;μM primer, 0.2&nbsp;mM dNTPs, 0.5&nbsp;U <em xmlns="" class="EmphasisTypeItalic">Taq</em> DNA polymerase and 10&nbsp;ng of sample DNA. Amplifications were carried out using GeneAmp® PCR System 9700 (Applied Biosystems) with an initial denaturation step at 94&nbsp;°C for 1&nbsp;min followed by 40&nbsp;cycles of 94&nbsp;°C for 1&nbsp;min, 55&nbsp;°C for 2&nbsp;min and 72&nbsp;°C for 18&nbsp;s; and a final extension step at 72&nbsp;°C for 5&nbsp;min.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par13" class="Para">The 24 SSR markers were selected based on their representation of all sorghum chromosomes and their previously reported high polymorphism. The PCR amplification of SSR loci was performed in a 25&nbsp;μl reaction mixture containing 25&nbsp;ng of template DNA, 1× PCR buffer [20&nbsp;mM Tris-HCl (pH&nbsp;8.55), 16&nbsp;mM (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, 0.01% Tween 20 and 2&nbsp;mM MgCl<sub>2</sub>], 0.3&nbsp;mM dNTPs, 0.25&nbsp;μM of each primer, and 1&nbsp;U Taq DNA polymerase (Saveen &amp; Werner®, Sweden). The PCR reactions employed a touchdown PCR method [<span class="CitationRef"><a href="#CR31" data-toggle="citation">31</a></span>] and were run using an Eppendorf® AG-22331 Thermal Cycler (Hamburg, Germany) after optimizing annealing temperatures for the individual primer-pairs. The amplification profiles consisted of initial denaturation of the template DNA at 95&nbsp;°C for 3&nbsp;min, followed by 10&nbsp;cycles at 95&nbsp;°C for 30&nbsp;s, 60&nbsp;°C or 65&nbsp;°C (depending on the primer-pair) for 30&nbsp;s (with a decrease of 1&nbsp;°C/cycle) and 72&nbsp;°C for 45&nbsp;s; 30&nbsp;cycles at 94&nbsp;°C for 30&nbsp;s, 50&nbsp;°C or 55&nbsp;°C (depending on the primer-pair) for 30&nbsp;s and 72&nbsp;°C for 45&nbsp;s, and a final extension for 20&nbsp;min at 72&nbsp;°C.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par14" class="Para">The two STS markers (<em xmlns="" class="EmphasisTypeItalic">CTG29</em> and <em xmlns="" class="EmphasisTypeItalic">M181</em>) are tightly linked to <em xmlns="" class="EmphasisTypeItalic">Alt</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">SB</em>
                  </sub>, the locus that underlies <em xmlns="" class="EmphasisTypeItalic">SbMATE</em> Al tolerance gene [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>, <span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>]. The primers used for the amplification of these markers were those published by Caniato et al. [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>]. The PCR reactions for STS markers were run in a GeneAmp® PCR System 9700 Thermal Cycler (Applied Biosystems) in a 20&nbsp;μl reaction mixture containing 30&nbsp;ng of DNA template, 1× PCR buffer [20&nbsp;mM Tris-HCl (pH&nbsp;8.55), 16&nbsp;mM (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, 0.01% Tween 20 and 2&nbsp;mM MgCl<sub>2</sub>], 0.5&nbsp;mM dNTPs, 1.9&nbsp;mM MgCl<sub>2</sub>, 0.11&nbsp;μM of each primer and 1&nbsp;U Taq DNA polymerase. The PCR program consisted of an initial DNA denaturation step of 1&nbsp;min at 95&nbsp;°C followed by 30&nbsp;cycles of denaturation at 95&nbsp;°C for 30&nbsp;s, annealing at 55&nbsp;°C (for <em xmlns="" class="EmphasisTypeItalic">M181</em>) or 57&nbsp;°C (for <em xmlns="" class="EmphasisTypeItalic">CTG29</em>) for 1&nbsp;min, extension at 72&nbsp;°C for 1&nbsp;min, followed by a final extension step at 72&nbsp;°C for 5&nbsp;min. In the case of <em xmlns="" class="EmphasisTypeItalic">CTG29</em>, additional 2&nbsp;mM MgCl<sub>2</sub> was used in the reaction mixture, and 40&nbsp;cycles of amplification at annealing temperature of 58&nbsp;°C was used.</p>
              </section><section id="Sec6" class="Section2 RenderAsSection2">
                <h3 class="Heading">Electrophoresis, staining, visualization and polymorphism survey</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par15" class="Para">The PCR products were separated on 1.5% agarose gel containing ethidium bromide to confirm amplification and thereafter visualized and photographed using a UV photo print system (IP-215-SD) fitted with a Sony® XC-ST50CE camera (Saveen &amp; Werner, Sweden). For both ISSR and SSR markers, the PCR products were then electrophoresed on polyacrylamide gels (CleanGel 10% 52S; ETC Electrophorase-technik®, Germany) for better resolution and silver-stained, as described by Geleta and Bryngelsson [<span class="CitationRef"><a href="#CR32" data-toggle="citation">32</a></span>].</p>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par16" class="Para">A marker polymorphism survey of the parents was done and the markers that differentiated the two parents were used to genotype the F2:3 progenies. The F<sub>1</sub>s were included in the analysis as positive controls. Three ISSR primers, four SSR and one STS primer-pairs generated promising markers (Table&nbsp;<span class="InternalRef"><a href="#Tab1">1</a></span>). Two of the three selected polymorphic ISSR markers were converted to sequence characterized amplified region (SCAR) markers as described below.<figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 1</span>
                        <p xmlns="" class="SimplePara">List of primers/primer-pairs used to amplify markers that showed association with Al tolerance in sorghum</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col></col><col></col><col></col><col char="."></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Marker type</p></th><th><p xmlns="" class="SimplePara">Locus name</p></th><th><p xmlns="" class="SimplePara">Primer sequences (5′ - 3′)</p></th><th><p xmlns="" class="SimplePara">Repeat motif</p></th><th><p xmlns="" class="SimplePara">Fragment size range (bp)</p></th><th char="."><p xmlns="" class="SimplePara">T<sub xmlns="http://www.w3.org/1999/xhtml">a</sub> (°C)</p></th></tr></thead><tbody><tr class="noclass"><td rowspan="3"><p xmlns="" class="SimplePara">ISSR</p></td><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">ISSR_811</em>
                              <sup xmlns="http://www.w3.org/1999/xhtml">a</sup>
                            </p></td><td><p xmlns="" class="SimplePara">F/R: GAG AGA GAG AGA GAG AC</p></td><td><p xmlns="" class="SimplePara">–</p></td><td><p xmlns="" class="SimplePara">1400</p></td><td char="."><p xmlns="" class="SimplePara">55</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">ISSR_835</em>
                              <sup xmlns="http://www.w3.org/1999/xhtml">a</sup>
                            </p></td><td><p xmlns="" class="SimplePara">F/R: AGA GAG AGA GAG AGA GCT C</p></td><td><p xmlns="" class="SimplePara">–</p></td><td><p xmlns="" class="SimplePara">Ca. 200</p></td><td char="."><p xmlns="" class="SimplePara">55</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">ISSR_884</em>
                              <sup xmlns="http://www.w3.org/1999/xhtml">a,i</sup>
                            </p></td><td><p xmlns="" class="SimplePara">F/R: HBH ATC AGA GAG AGA GAG AG</p></td><td><p xmlns="" class="SimplePara">–</p></td><td><p xmlns="" class="SimplePara">Ca. 200</p></td><td char="."><p xmlns="" class="SimplePara">55</p></td></tr><tr class="noclass"><td rowspan="4"><p xmlns="" class="SimplePara">SSR</p></td><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Sb5_236</em>
                              <sup xmlns="http://www.w3.org/1999/xhtml">b</sup>
                            </p></td><td><p xmlns="" class="SimplePara">F: GCC AAG AGA AAC ACA AAC AA</p><p xmlns="" class="SimplePara">R: AGC AAT GTA TTT AGG CAA CAC A</p></td><td><p xmlns="" class="SimplePara">(AG)<sub xmlns="http://www.w3.org/1999/xhtml">20</sub></p></td><td><p xmlns="" class="SimplePara">160–208</p></td><td char="."><p xmlns="" class="SimplePara">55</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Sb6_342</em>
                              <sup xmlns="http://www.w3.org/1999/xhtml">b</sup>
                            </p></td><td><p xmlns="" class="SimplePara">F: TGC TTG TGA GAG TGC CTC CCT</p><p xmlns="" class="SimplePara">R: GTG AAC CTG CTG CTT TAG TCG ATG</p></td><td><p xmlns="" class="SimplePara">(AC)<sub xmlns="http://www.w3.org/1999/xhtml">25</sub></p></td><td><p xmlns="" class="SimplePara">270–294</p></td><td char="."><p xmlns="" class="SimplePara">50</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Sb6_34</em>
                              <sup xmlns="http://www.w3.org/1999/xhtml">b</sup>
                            </p></td><td><p xmlns="" class="SimplePara">F: AAC AGC AGT AAT GCC ACA C</p><p xmlns="" class="SimplePara">R: TGA CTT GGT AGA GAA CTT GTC TTC</p></td><td><p xmlns="" class="SimplePara">[(AC)/(CG)]<sub xmlns="http://www.w3.org/1999/xhtml">15</sub></p></td><td><p xmlns="" class="SimplePara">188–208</p></td><td char="."><p xmlns="" class="SimplePara">55</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Xtxp34</em>
                              <sup xmlns="http://www.w3.org/1999/xhtml">c</sup>
                            </p></td><td><p xmlns="" class="SimplePara">F: TGG TTC GTA TCC TTC TCT ACA G</p><p xmlns="" class="SimplePara">R: CAT ATA CCT CCT CGT CGC TC</p></td><td><p xmlns="" class="SimplePara">(CT)<sub xmlns="http://www.w3.org/1999/xhtml">29</sub></p></td><td><p xmlns="" class="SimplePara">365</p></td><td char="."><p xmlns="" class="SimplePara">55</p></td></tr><tr class="noclass"><td rowspan="3"><p xmlns="" class="SimplePara">STS</p></td><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">CTG29</em>
                              <sup xmlns="http://www.w3.org/1999/xhtml">d,e</sup>
                            </p></td><td><p xmlns="" class="SimplePara">F: ATG CAG TAT CTG CAG TAT CAT TT</p><p xmlns="" class="SimplePara">R: AAT CCG TCA GGT CAG CAA TC</p></td><td><p xmlns="" class="SimplePara">–</p></td><td><p xmlns="" class="SimplePara">226–228</p></td><td char="."><p xmlns="" class="SimplePara">57</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">M181</em>
                              <sup xmlns="http://www.w3.org/1999/xhtml">d,e</sup>
                            </p></td><td><p xmlns="" class="SimplePara">F: AAG GCA ACA ACT GAG GCA CT</p><p xmlns="" class="SimplePara">R: TCG CTA GAG TGG TGC AAG AA</p></td><td><p xmlns="" class="SimplePara">–</p></td><td><p xmlns="" class="SimplePara">169–174</p></td><td char="."><p xmlns="" class="SimplePara">55</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">CTG29_3b</em>
                              <sup xmlns="http://www.w3.org/1999/xhtml">f,g</sup>
                            </p></td><td><p xmlns="" class="SimplePara">F: TGG TGA TAT TAT TAA AAC TGT GTT A</p><p xmlns="" class="SimplePara">R: AAT CCG TCA GGT CAG CAA TC</p></td><td><p xmlns="" class="SimplePara">–</p></td><td><p xmlns="" class="SimplePara">200</p></td><td char="."><p xmlns="" class="SimplePara">58</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SCAR</p></td><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">SCAR_811</em>
                              <sup xmlns="http://www.w3.org/1999/xhtml">h</sup>
                            </p></td><td><p xmlns="" class="SimplePara">F: ACG CAA GTT CCG AGG AGA A</p><p xmlns="" class="SimplePara">R: GAG AGA GAG AGA GAG ACA GAG GTT GTC</p></td><td><p xmlns="" class="SimplePara">–</p></td><td><p xmlns="" class="SimplePara">1119</p></td><td char="."><p xmlns="" class="SimplePara">65</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">SCAR_884</em>
                              <sup xmlns="http://www.w3.org/1999/xhtml">h</sup>
                            </p></td><td><p xmlns="" class="SimplePara">F: AGA GAG AGA GAG AGC TCA CAC A</p><p xmlns="" class="SimplePara">R: AGA GAG AGA GAG AGG TGT TTT A</p></td><td><p xmlns="" class="SimplePara">–</p></td><td><p xmlns="" class="SimplePara">226 and 446</p></td><td char="."><p xmlns="" class="SimplePara">61</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara"><sup xmlns="http://www.w3.org/1999/xhtml">a</sup>Source: University of British Columbia, Canada: <span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://www.scribd.com/doc/23812434/UBC"><span class="RefSource">http://www.scribd.com/doc/23812434/UBC</span></a></span></p>
                      <p xmlns="" class="SimplePara"><sup xmlns="http://www.w3.org/1999/xhtml">b</sup>Source: Smith et al. [<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR54" data-toggle="citation">54</a></span>], <sup xmlns="http://www.w3.org/1999/xhtml">c</sup>Source: Kong et al. [<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR55" data-toggle="citation">55</a></span>], <sup xmlns="http://www.w3.org/1999/xhtml">d</sup>Source: Caniato et al. [<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>]; <sup xmlns="http://www.w3.org/1999/xhtml">e</sup>The markers did not differentiate the resistant and sensitive parents in this study. <sup xmlns="http://www.w3.org/1999/xhtml">f</sup>Source of forward primer: this study; <sup xmlns="http://www.w3.org/1999/xhtml">g</sup>Source of reverse primer: Caniato et al. [<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>]; <sup xmlns="http://www.w3.org/1999/xhtml">h</sup>Source: this study. F = Forward; R = Reverse; F/R = used as both forward and reverse primers. <sup xmlns="http://www.w3.org/1999/xhtml">i</sup>In the primer sequence of <em class="EmphasisTypeItalic">ISSR_884</em>, B = C or G or T whereas H = A or C or T</p>
                    </div></figure></div>
              </section><section id="Sec7" class="Section2 RenderAsSection2">
                <h3 class="Heading">Development and analysis of SCAR markers</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par17" class="Para">Two DNA fragments of approximately 1400&nbsp;bp and 200&nbsp;bp long that were amplified by <em xmlns="" class="EmphasisTypeItalic">ISSR_811</em> and <em xmlns="" class="EmphasisTypeItalic">ISSR_884</em> primers, respectively, were identified as co-segregating with aluminium tolerance. In this paper, these fragments are referred to as <em xmlns="" class="EmphasisTypeItalic">811_1400</em> and <em xmlns="" class="EmphasisTypeItalic">884_200</em>, respectively. Both fragments were recovered from gels for cloning and sequencing. The <em xmlns="" class="EmphasisTypeItalic">811_1400</em> fragment was purified from agarose gel using QIAquick®Gel extraction and purification kit (QIAGEN GmbH, Germany) according to the manufacturer’s instructions. The purified DNA was used in the subsequent cloning of the marker fragment.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par18" class="Para">The <em xmlns="" class="EmphasisTypeItalic">884_200</em> fragment was not clearly resolved on agarose gel and hence the PCR products were separated on polyacrylamide gel and then recovered following the procedure described by Sanguinetti [<span class="CitationRef"><a href="#CR33" data-toggle="citation">33</a></span>]. The fragments were extracted from agarose or polyacrylamide gels in 100&nbsp;μl of extraction buffer (10&nbsp;mM Tris-HCl, 50&nbsp;mM KCl, 1.5&nbsp;mM MgCl<sub>2</sub>, 0.1% Triton® X-100, pH&nbsp;9.0) by incubation at 95&nbsp;°C for 20&nbsp;min. One microliter of the extracted DNA solution was used to re-amplify the target fragment using Advantage® HF2 Taq polymerase (Invitrogen) according to the ISSR amplification procedures describe above. An aliquot of the PCR products was analysed on agarose gel to confirm amplification and the rest purified using QIAquick® PCR purification kit (QIAGEN GmbH, Germany). The purified DNA fragments were ligated into pJET1.2/blunt cloning vector following the sticky-end cloning protocol as outlined in the CloneJET<sup>™</sup> PCR Cloning Kit instruction manual (Fermentas®, Life Sciences).</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par19" class="Para">Transformation of chemically competent <em xmlns="" class="EmphasisTypeItalic">Escherichia coli</em> cells (One Shot® TOP10-Invitrogen) was done following the manufacturer’s instructions. Single colonies of transformed cells were picked and analyzed by colony PCR using pJET1.2 primer-pairs to identify clones that carried the fragments of interest. These clones were sub-cultured in liquid Luria Bertini (LB) media containing ampicillin (100&nbsp;μg/ml) for plasmid mini-preparations. Plasmid DNA was purified using QIAprep® spin miniprep kit (QIAGEN GmbH, Germany) as described by the manufacturer.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par20" class="Para">Plasmid DNA harbouring the inserts was digested with <em xmlns="" class="EmphasisTypeItalic">Bgl</em>11 restriction endonuclease to confirm fragment size. Samples that had the desired inserts were sequenced using one of the pJET1.2 primers. The sequence quality was checked using Sequence Scanner v. 1.0 (Applied Biosystems®) and sequence alignment was done with ClustalX v. 2 [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>]. The aligned sequences were manually edited using BioEdit© version 7.0 [<span class="CitationRef"><a href="#CR35" data-toggle="citation">35</a></span>].</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par21" class="Para">The sequences of the <em xmlns="" class="EmphasisTypeItalic">811_1400</em> and <em xmlns="" class="EmphasisTypeItalic">884_200</em> fragments were used to design new primers to amplify SCAR markers <em xmlns="" class="EmphasisTypeItalic">SCAR_811</em> and <em xmlns="" class="EmphasisTypeItalic">SCAR_884</em>, respectively. Primers were designed by extending the ISSR primers so that the 3′-end of the primers contain SNPs that differentiated the tolerant and sensitive sorghum genotypes. Primer3 (an online primer designing program (<span class="ExternalRef"><a href="http://primer3.ut.ee"><span class="RefSource">http://primer3.ut.ee</span></a></span>) was also used to design alternative primers. Different combinations of primers were tested for optimal amplification of the corresponding SCAR markers.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par22" class="Para">The PCR for <em xmlns="" class="EmphasisTypeItalic">SCAR_884</em> was performed in a volume of 50&nbsp;μl containing 50&nbsp;ng of template DNA, 1× PCR buffer, 3.5&nbsp;mM MgCl<sub>2</sub>, 0.1&nbsp;mM dNTPs, 0.5&nbsp;μM of each primer and 1&nbsp;U of Taq DNA polymerase. The reactions were carried out in GeneAmp® PCR System 9700 Thermal Cycler with an initial denaturation step at 94&nbsp;°C for 3&nbsp;min, followed by 30&nbsp;cycles at 94&nbsp;°C for 1&nbsp;min, 61&nbsp;°C for 1&nbsp;min, and 72&nbsp;°C for 2&nbsp;min, followed by final extension step at 72&nbsp;°C for 7&nbsp;min. The reaction mixtures and PCR conditions for <em xmlns="" class="EmphasisTypeItalic">SCAR_811</em> were similar to those of <em xmlns="" class="EmphasisTypeItalic">SCAR_884</em> except that the concentration of MgCl<sub>2</sub> was 2&nbsp;mM and the annealing temperature was 65&nbsp;°C.</p>
              </section><section id="Sec8" class="Section2 RenderAsSection2">
                <h3 class="Heading">Data scoring and statistical analysis</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par23" class="Para">The bands for ISSR, STS and SCAR markers were scored as present (1) or absent (0) whereas the SSR marker fragments were scored as either parental allele (A or B),or heterozygous for both alleles (AB). All phenotypic data were subjected to analysis of variance (ANOVA) using SPSS® and means were separated using Tukey’s test at 5% significance level. The association of individual marker with Al tolerance was tested with one-way analysis of variance and linear regression analysis, with a threshold significance level of <em xmlns="" class="EmphasisTypeItalic">P</em> ≤ 0.05. When a significant difference was found between the genotypic groups, a pairwise comparison was made using Tukey’s test. For those markers that showed significant association with aluminium tolerance when analyzed individually, multiple linear regression analysis was conducted to check for collinearity between the markers. The Chi-square goodness-of-fit test for the Mendelian segregation of the alleles of each marker was also conducted for the F<sub>2:3</sub> population (Table&nbsp;<span class="InternalRef"><a href="#Tab2">2</a></span>).<figure xmlns="" id="Tab2" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 2</span>
                        <p xmlns="" class="SimplePara">Simple and multiple linear regression analysis of 229&nbsp;F<sub xmlns="http://www.w3.org/1999/xhtml">2:3</sub> progeny plants for the association between marker-based genotypes and net root length in aluminium (NRL<sub xmlns="http://www.w3.org/1999/xhtml">Al</sub>) and percent relative root growth (%RRG); and Chi-square goodness- of-fit test for Mendelian segregation of the markers</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col char="."></col><col char="–"></col><col char="."></col><col char="."></col><col char="."></col><col char="."></col><col char="."></col><col char="."></col><col char="."></col><col char="."></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Marker</p></th><th char="."><p xmlns="" class="SimplePara"><sup xmlns="http://www.w3.org/1999/xhtml">d</sup>Chromosome number</p></th><th char="–"><p xmlns="" class="SimplePara"><sup xmlns="http://www.w3.org/1999/xhtml">d</sup>Marker position (bp range)</p></th><th colspan="3" char="."><p xmlns="" class="SimplePara">NRL<sub xmlns="http://www.w3.org/1999/xhtml">Al</sub></p></th><th colspan="3" char="."><p xmlns="" class="SimplePara">%RRG</p></th><th colspan="2" char="."><p xmlns="" class="SimplePara">χ <sup xmlns="http://www.w3.org/1999/xhtml">2</sup> test</p></th></tr><tr class="header"><th>&nbsp;</th><th char=".">&nbsp;</th><th char="–">&nbsp;</th><th char="."><p xmlns="" class="SimplePara">F</p></th><th char="."><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">P</em>-value</p></th><th char="."><p xmlns="" class="SimplePara">R<sup xmlns="http://www.w3.org/1999/xhtml">2</sup></p></th><th char="."><p xmlns="" class="SimplePara">F</p></th><th char="."><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">P</em>-value</p></th><th char="."><p xmlns="" class="SimplePara">R<sup xmlns="http://www.w3.org/1999/xhtml">2</sup></p></th><th char="."><p xmlns="" class="SimplePara">χ<sup xmlns="http://www.w3.org/1999/xhtml">2</sup></p></th><th char="."><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">P</em>-value</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Sb5_236</em>
                            </p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="–"><p xmlns="" class="SimplePara">52,278,272–52,278,445</p></td><td char="."><p xmlns="" class="SimplePara">15.08</p></td><td char="."><p xmlns="" class="SimplePara">0.000</p></td><td char="."><p xmlns="" class="SimplePara">0.110</p></td><td char="."><p xmlns="" class="SimplePara">11.92</p></td><td char="."><p xmlns="" class="SimplePara">0.001</p></td><td char="."><p xmlns="" class="SimplePara">0.050</p></td><td char="."><p xmlns="" class="SimplePara">5.33</p></td><td char="."><p xmlns="" class="SimplePara">0.070</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">Xtxp3</em>4</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="–"><p xmlns="" class="SimplePara">69,704,047–69,704,411</p></td><td char="."><p xmlns="" class="SimplePara">18.75</p></td><td char="."><p xmlns="" class="SimplePara">0.000</p></td><td char="."><p xmlns="" class="SimplePara">0.142</p></td><td char="."><p xmlns="" class="SimplePara">17.07</p></td><td char="."><p xmlns="" class="SimplePara">0.000</p></td><td char="."><p xmlns="" class="SimplePara">0.070</p></td><td char="."><p xmlns="" class="SimplePara">3.94</p></td><td char="."><p xmlns="" class="SimplePara">0.139</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">CTG29_3b</em>
                            </p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="–"><p xmlns="" class="SimplePara">70,939,651–70,939,834</p></td><td char="."><p xmlns="" class="SimplePara">11.42</p></td><td char="."><p xmlns="" class="SimplePara">0.001</p></td><td char="."><p xmlns="" class="SimplePara">0.048</p></td><td char="."><p xmlns="" class="SimplePara">9.49</p></td><td char="."><p xmlns="" class="SimplePara">0.002</p></td><td char="."><p xmlns="" class="SimplePara">0.040</p></td><td char="."><p xmlns="" class="SimplePara">3.60</p></td><td char="."><p xmlns="" class="SimplePara">0.059</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">811_1400</em>
                              <sup xmlns="http://www.w3.org/1999/xhtml">a</sup>
                            </p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="–"><p xmlns="" class="SimplePara">71,473,923–71,475,318</p></td><td char="."><p xmlns="" class="SimplePara">24.88</p></td><td char="."><p xmlns="" class="SimplePara">0.000</p></td><td char="."><p xmlns="" class="SimplePara">0.100</p></td><td char="."><p xmlns="" class="SimplePara">9.78</p></td><td char="."><p xmlns="" class="SimplePara">0.002</p></td><td char="."><p xmlns="" class="SimplePara">0.041</p></td><td char="."><p xmlns="" class="SimplePara">0.14</p></td><td char="."><p xmlns="" class="SimplePara">0.709</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">884_200</em>
                              <sup xmlns="http://www.w3.org/1999/xhtml">b</sup>
                            </p></td><td char="."><p xmlns="" class="SimplePara">6</p></td><td char="–"><p xmlns="" class="SimplePara">42,689,435–42,689,555</p></td><td char="."><p xmlns="" class="SimplePara">4.64</p></td><td char="."><p xmlns="" class="SimplePara">0.032</p></td><td char="."><p xmlns="" class="SimplePara">0.017</p></td><td char="."><p xmlns="" class="SimplePara">5.07</p></td><td char="."><p xmlns="" class="SimplePara">0.025</p></td><td char="."><p xmlns="" class="SimplePara">0.025</p></td><td char="."><p xmlns="" class="SimplePara">3.09</p></td><td char="."><p xmlns="" class="SimplePara">0.079</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Sb6_342</em>
                            </p></td><td char="."><p xmlns="" class="SimplePara">7</p></td><td char="–"><p xmlns="" class="SimplePara">38,823,193–38,823,478</p></td><td char="."><p xmlns="" class="SimplePara">4.49</p></td><td char="."><p xmlns="" class="SimplePara">0.012</p></td><td char="."><p xmlns="" class="SimplePara">0.038</p></td><td char="."><p xmlns="" class="SimplePara">6.06</p></td><td char="."><p xmlns="" class="SimplePara">0.015</p></td><td char="."><p xmlns="" class="SimplePara">0.030</p></td><td char="."><p xmlns="" class="SimplePara">0.36</p></td><td char="."><p xmlns="" class="SimplePara">0.837</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Sb6_34</em>
                            </p></td><td char="."><p xmlns="" class="SimplePara">8</p></td><td char="–"><p xmlns="" class="SimplePara">61,802,592–61,802,795</p></td><td char="."><p xmlns="" class="SimplePara">7.58</p></td><td char="."><p xmlns="" class="SimplePara">0.001</p></td><td char="."><p xmlns="" class="SimplePara">0.063</p></td><td char="."><p xmlns="" class="SimplePara">4.02</p></td><td char="."><p xmlns="" class="SimplePara">0.046</p></td><td char="."><p xmlns="" class="SimplePara">0.020</p></td><td char="."><p xmlns="" class="SimplePara">5.70</p></td><td char="."><p xmlns="" class="SimplePara">0.058</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">835_200</em>
                            </p></td><td char="."><p xmlns="" class="SimplePara">10</p></td><td char="–"><p xmlns="" class="SimplePara">56,491,522–56,491,702</p></td><td char="."><p xmlns="" class="SimplePara">3.61</p></td><td char="."><p xmlns="" class="SimplePara">0.049</p></td><td char="."><p xmlns="" class="SimplePara">0.016</p></td><td char="."><p xmlns="" class="SimplePara">6.85</p></td><td char="."><p xmlns="" class="SimplePara">0.009</p></td><td char="."><p xmlns="" class="SimplePara">0.029</p></td><td char="."><p xmlns="" class="SimplePara">1.50</p></td><td char="."><p xmlns="" class="SimplePara">0.220</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Four markers<sup xmlns="http://www.w3.org/1999/xhtml">c</sup></p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="–">&nbsp;</td><td char="."><p xmlns="" class="SimplePara">15.67</p></td><td char="."><p xmlns="" class="SimplePara">0.000</p></td><td char="."><p xmlns="" class="SimplePara">0.219</p></td><td char="."><p xmlns="" class="SimplePara">7.06</p></td><td char="."><p xmlns="" class="SimplePara">0.000</p></td><td char="."><p xmlns="" class="SimplePara">0.119</p></td><td char=".">&nbsp;</td><td char=".">&nbsp;</td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">All markers</p></td><td char=".">&nbsp;</td><td char="–">&nbsp;</td><td char="."><p xmlns="" class="SimplePara">8.15</p></td><td char="."><p xmlns="" class="SimplePara">0.000</p></td><td char="."><p xmlns="" class="SimplePara">0.251</p></td><td char="."><p xmlns="" class="SimplePara">3.82</p></td><td char="."><p xmlns="" class="SimplePara">0.000</p></td><td char="."><p xmlns="" class="SimplePara">0.122</p></td><td char=".">&nbsp;</td><td char=".">&nbsp;</td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara"><sup xmlns="http://www.w3.org/1999/xhtml">a</sup>Ccorresponds to <em class="EmphasisTypeItalic">SCAR_811;</em> <sup xmlns="http://www.w3.org/1999/xhtml">b</sup>corresponds to <em class="EmphasisTypeItalic">SCAR_884</em>; <sup xmlns="http://www.w3.org/1999/xhtml">c</sup>the four markers on chromosome 3. <sup xmlns="http://www.w3.org/1999/xhtml">d</sup>The chromosome number and the position of the markers within the corresponding chromosomes (in bp range) were identified through BLAST (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi"><span class="RefSource">https://blast.ncbi.nlm.nih.gov/Blast.cgi</span></a></span>) search of the sequences of the markers against the sorghum genome. The GenBank Accession number of chromosomes 3, 6, 7, 8 and 10 are <em class="EmphasisTypeItalic">NC_012872.2</em>, <em class="EmphasisTypeItalic">NC_012875.</em>2, <em class="EmphasisTypeItalic">NC_012876.</em>2, <em class="EmphasisTypeItalic">NC_012877.2</em>, <em class="EmphasisTypeItalic">NC_012879.2</em>, respectively. Note: the position of <em class="EmphasisTypeItalic">SbMATE</em> gene on chromosome 3 (<em class="EmphasisTypeItalic">NC_012872.2</em>) is 71,105,461–71,108,054&nbsp;bp</p>
                    </div></figure></div>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec9" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Results" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Results">Results</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Results" data-test="collapse-section"><section id="Sec10" class="Section2 RenderAsSection2">
                <h3 class="Heading">Response of parental lines and their progenies to aluminium stress</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par24" class="Para">The root growth in the two parental sorghum lines, <em xmlns="" class="EmphasisTypeItalic">Seredo</em> and <em xmlns="" class="EmphasisTypeItalic">ICSR 110,</em> exhibited significantly different responses (<em xmlns="" class="EmphasisTypeItalic">p</em> &lt; 0.05) to the Al stress at 148&nbsp;μM Al (Figs.&nbsp;<span class="InternalRef"><a href="#Fig1">1</a></span> and <span class="InternalRef"><a href="#Fig2">2</a></span>). <em xmlns="" class="EmphasisTypeItalic">ICSR 110</em> was tolerant with only 15% reduction in root growth, whereas <em xmlns="" class="EmphasisTypeItalic">Seredo</em> had 53% reduction in root growth. The net root growth in aluminium and percentage of relative root growth were strongly correlated (<em xmlns="" class="EmphasisTypeItalic">r</em><sup>2</sup> = 0.70). Based on the variation in net growth of the sensitive parent, the F<sub>2:3</sub> progeny plants with net root growth of 4.0&nbsp;cm or less were classified as sensitive. Those that had net root growth ranging between 4.0&nbsp;cm and 5.0&nbsp;cm were classified as moderately tolerant, while progeny plants with a net root growth above 5.0&nbsp;cm were classified as tolerant to Al stress. Based on this criterion, among the 229&nbsp;F<sub>2:3</sub> population, 98, 58 and 73 individuals were tolerant, moderately tolerant and sensitive, respectively.<figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO1">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0059-3/MediaObjects/41065_2018_59_Fig1_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0059-3/MediaObjects/41065_2018_59_Fig1_HTML.gif" alt="Fig. 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0059-3/MediaObjects/41065_2018_59_Fig1_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0059-3/MediaObjects/41065_2018_59_Fig1_HTML.gif" alt="Fig. 1"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 1</span>
                        <p xmlns="" class="SimplePara">Frequency distribution of NRL<sub xmlns="http://www.w3.org/1999/xhtml">Al</sub> of 229&nbsp;F<sub xmlns="http://www.w3.org/1999/xhtml">2:3</sub> progeny derived from a cross between <em class="EmphasisTypeItalic">Seredo</em>, an aluminium sensitive sorghum line, and <em class="EmphasisTypeItalic">ICSR 110</em>, an aluminium tolerant line. The NRL<sub xmlns="http://www.w3.org/1999/xhtml">Al</sub> mean values for <em class="EmphasisTypeItalic">Seredo</em> and <em class="EmphasisTypeItalic">ICSR 110</em> were 3.8 (<em class="EmphasisTypeItalic">n</em> = 40) and 6.0 (<em class="EmphasisTypeItalic">n</em> = 40), respectively as shown by the arrows</p>
                      </div></figcaption></figure><figure xmlns="" class="Figure" id="Fig2"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO2">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0059-3/MediaObjects/41065_2018_59_Fig2_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0059-3/MediaObjects/41065_2018_59_Fig2_HTML.gif" alt="Fig. 2" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0059-3/MediaObjects/41065_2018_59_Fig2_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0059-3/MediaObjects/41065_2018_59_Fig2_HTML.gif" alt="Fig. 2"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 2</span>
                        <p xmlns="" class="SimplePara">Frequency distribution of %RRG of 229&nbsp;F<sub xmlns="http://www.w3.org/1999/xhtml">2:3</sub> progeny derived from a cross between <em class="EmphasisTypeItalic">Seredo,</em> an aluminium sensitive sorghum line, and <em class="EmphasisTypeItalic">ICSR 110,</em> an aluminium tolerant line. The %RRGs of the parental lines are shown by the arrows for comparison</p>
                      </div></figcaption></figure></div>
              </section><section id="Sec11" class="Section2 RenderAsSection2">
                <h3 class="Heading">The molecular markers associated with tolerance to Aluminium toxicity</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par25" class="Para">Ten of the fifty ISSR primers amplified fragments that were polymorphic between the two parental lines. Preliminary analysis of the ten polymorphic markers using 21 Al tolerant and 21 Al sensitive F<sub>2:3</sub> progeny revealed that <em xmlns="" class="EmphasisTypeItalic">ISSR_811</em>, <em xmlns="" class="EmphasisTypeItalic">ISSR_835</em> and <em xmlns="" class="EmphasisTypeItalic">ISSR_884</em> primers amplified three fragments of approximately 1400&nbsp;bp, 200&nbsp;bp and 200&nbsp;bp, respectively, having uneven distribution among aluminium tolerant and sensitive progenies. After genotyping, the remaining 187&nbsp;F<sub>2:3</sub> progeny plants&nbsp;were genotyped with these primers. The analysis of phenotypic and genotypic data from the 229&nbsp;F<sub>2:3</sub> progeny revealed significant association of these markers with tolerance to Al toxicity (Table <span class="InternalRef"><a href="#Tab2">2</a></span>). The <em xmlns="" class="EmphasisTypeItalic">811_1400</em> fragment was specific to <em xmlns="" class="EmphasisTypeItalic">Seredo</em> and found in most Al sensitive progeny. On the other hand, the <em xmlns="" class="EmphasisTypeItalic">835_200</em> and <em xmlns="" class="EmphasisTypeItalic">884_200</em> amplified fragments showed significant association with aluminium tolerance.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par26" class="Para">Basic Local Alignment Search Tool (BLAST) was used to search the GenBank (<span class="ExternalRef"><a href="https://www.ncbi.nlm.nih.gov/genbank/"><span class="RefSource">https://www.ncbi.nlm.nih.gov/genbank/</span></a></span>) and the Gramene (<span class="ExternalRef"><a href="http://www.gramene.org"><span class="RefSource">http://www.gramene.org</span></a></span>) databases to locate the genomic positions of these markers within the sorghum genome. The BLAST searching of <em xmlns="" class="EmphasisTypeItalic">811_1400</em> marker sequence in the Gramene database resulted in a sorghum sequence that share 1386&nbsp;bp (with 99.6% sequence identity) with <em xmlns="" class="EmphasisTypeItalic">811_1400</em> on chromosome 3 at position 71,512,250–71,513,635&nbsp;bp. The corresponding position of this marker on sorghum chromosome 3 (GenBank accession number <em xmlns="" class="EmphasisTypeItalic">NC_012872.2</em>) is 71,473,923–71,475,318&nbsp;bp (Table <span class="InternalRef"><a href="#Tab2">2</a></span>).</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par27" class="Para">Two sequences that shared 116&nbsp;bp and 115&nbsp;bp (both with 100% sequence identity) with <em xmlns="" class="EmphasisTypeItalic">884_200</em> were found through BLAST searching of Gramene database. These sequences are located on chromosome 6 at positions 42,646,793–42,646,908&nbsp;bp and 42,646,902–42,647,016&nbsp;bp, respectively, and the two sequences together cover the whole sequence of the marker <em xmlns="" class="EmphasisTypeItalic">884_200</em>. These sequences have an overlapping sequence with a gene, <em xmlns="" class="EmphasisTypeItalic">Sb06g015330</em>, that codes for putative uncharacterized protein Sb06g015330 (source: UniProtKB/TrEMBL; Acc: C5YFS2_SORBI; cited at <span class="ExternalRef"><a href="http://www.gramene.org"><span class="RefSource">http://www.gramene.org</span></a></span>). The corresponding position of marker <em xmlns="" class="EmphasisTypeItalic">884_200</em> on sorghum chromosome 6 (GenBank accession number <em xmlns="" class="EmphasisTypeItalic">NC_012875.2</em>) is 42,689,435–42,689,555&nbsp;bp (Table <span class="InternalRef"><a href="#Tab2">2</a></span>). Similarly, two sequences that shared 155&nbsp;bp and 72&nbsp;bp (with 92.9 and 98.6% sequence identity, respectively) with <em xmlns="" class="EmphasisTypeItalic">835_200</em> were found through BLAST searching of Gramene database. These sequences are located on chromosome 10 at position 56,232,617–56,232,771&nbsp;bp and 56,232,771–56,232,842&nbsp;bp, respectively. The corresponding position of marker <em xmlns="" class="EmphasisTypeItalic">835_200</em> on sorghum chromosome 10 (GenBank accession number <em xmlns="" class="EmphasisTypeItalic">NC_012879.2</em>) is 56,491,522–56,491,702&nbsp;bp (Table <span class="InternalRef"><a href="#Tab2">2</a></span>). Hence, markers <em xmlns="" class="EmphasisTypeItalic">811_1400</em>, <em xmlns="" class="EmphasisTypeItalic">884_200</em> and <em xmlns="" class="EmphasisTypeItalic">835_200</em> are located on chromosomes 3, 6 and 10, respectively. Of these markers, <em xmlns="" class="EmphasisTypeItalic">811_1400</em> and <em xmlns="" class="EmphasisTypeItalic">884_200</em> were converted into SCAR markers <em xmlns="" class="EmphasisTypeItalic">SCAR_811</em> and <em xmlns="" class="EmphasisTypeItalic">SCAR_884</em>, respectively.</p>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par28" class="Para">The first attempt to develop SCAR marker based on the sequence of <em xmlns="" class="EmphasisTypeItalic">811_1400</em> fragment was by using forward and reverse primers that were developed through extending the sequence of <em xmlns="" class="EmphasisTypeItalic">ISSR_811</em> primer. These primers amplified fragments of the same size in both parents and their progeny. DNA sequencing of the fragments from tolerant and sensitive parents revealed two SNP sites that differentiate the parents (Table&nbsp;<span class="InternalRef"><a href="#Tab3">3</a></span>). One of these SNPs was targeted to design the forward primer in order to get amplified product specific to the sensitive parent, <em xmlns="" class="EmphasisTypeItalic">Seredo</em>. The use of a forward primer that included the SNP site and a reverse primer that was designed by extending the <em xmlns="" class="EmphasisTypeItalic">ISSR_811</em> primer resulted in the development of a polymorphic marker, <em xmlns="" class="EmphasisTypeItalic">SCAR_811</em>. Fig.&nbsp;<span class="InternalRef"><a href="#Fig3">3</a></span> shows the <em xmlns="" class="EmphasisTypeItalic">SCAR_811</em> marker segregation in the F<sub>2:3</sub> progeny and parents. The Chi-square goodness-of-fit test showed Mendelian segregation of this marker among the F<sub>2:3</sub> progeny (χ<sup>2</sup> = 0.14; <em xmlns="" class="EmphasisTypeItalic">P</em> = 0.71; Table <span class="InternalRef"><a href="#Tab2">2</a></span>). The sensitive parental line and most of the sensitive F<sub>2:3</sub> progeny bear <em xmlns="" class="EmphasisTypeItalic">SCAR_811</em> whereas the marker is absent in the tolerant parent <em xmlns="" class="EmphasisTypeItalic">ICSR 110</em> and most of the tolerant progeny showing the association between the <em xmlns="" class="EmphasisTypeItalic">SCAR_811</em> marker and tolerance to Al toxicity. This marker explained 10% of the variation in NRL<sub>Al</sub> (<em xmlns="" class="EmphasisTypeItalic">R</em><sup>2</sup> = 0.10; <em xmlns="" class="EmphasisTypeItalic">P</em> &lt; 0.001) and 4% of %RRG (<em xmlns="" class="EmphasisTypeItalic">R</em><sup>2</sup> = 0.04; <em xmlns="" class="EmphasisTypeItalic">P</em> &lt; 0.01) in the F<sub>2:3</sub> progeny (Table <span class="InternalRef"><a href="#Tab2">2</a></span>).<figure xmlns="" id="Tab3" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 3</span>
                        <p xmlns="" class="SimplePara">Partial DNA sequences of <em class="EmphasisTypeItalic">ICSR 110</em> and <em class="EmphasisTypeItalic">Seredo</em> parental lines sequenced by primers designed based on the DNA sequence of <em class="EmphasisTypeItalic">811_1400</em> marker that was amplified in <em class="EmphasisTypeItalic">Seredo</em> but not in <em class="EmphasisTypeItalic">ICSR 110</em></p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">PL</p></th><th colspan="55"><p xmlns="" class="SimplePara">DNA sequence</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">1<sup xmlns="http://www.w3.org/1999/xhtml">a</sup></p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">A</strong>
                            </p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">G</strong>
                            </p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">C</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">2<sup xmlns="http://www.w3.org/1999/xhtml">b</sup></p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">G</strong>
                            </p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">A</strong>
                            </p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">C</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">PL</em> parental line, <em class="EmphasisTypeItalic">1</em><sup xmlns="http://www.w3.org/1999/xhtml">
                          <em xmlns="" class="EmphasisTypeItalic">a</em>
                        </sup>&nbsp;<em class="EmphasisTypeItalic">ICSR 110</em>; <em class="EmphasisTypeItalic">2</em><sup xmlns="http://www.w3.org/1999/xhtml">
                          <em xmlns="" class="EmphasisTypeItalic">b</em>
                        </sup>&nbsp;<em class="EmphasisTypeItalic">Seredo</em></p>
                      <p xmlns="" class="SimplePara">There are two variable sites that differentiated the two parents as indicated in bold</p>
                    </div></figure><figure xmlns="" class="Figure" id="Fig3"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO3">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0059-3/MediaObjects/41065_2018_59_Fig3_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0059-3/MediaObjects/41065_2018_59_Fig3_HTML.gif" alt="Fig. 3" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0059-3/MediaObjects/41065_2018_59_Fig3_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0059-3/MediaObjects/41065_2018_59_Fig3_HTML.gif" alt="Fig. 3"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 3</span>
                        <p xmlns="" class="SimplePara">Segregation of <em class="EmphasisTypeItalic">SCAR_811</em> marker in aluminium tolerant (lanes 4–18) and sensitive (lanes 22–35) F<sub xmlns="http://www.w3.org/1999/xhtml">2:3</sub> progeny. The F<sub xmlns="http://www.w3.org/1999/xhtml">1</sub>s are on lanes 19–21 and the parental lines <em class="EmphasisTypeItalic">Seredo</em> and <em class="EmphasisTypeItalic">ICSR 110</em> are on lanes 1–3 and 36–38, respectively</p>
                      </div></figcaption></figure></div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par29" class="Para">The primers for <em xmlns="" class="EmphasisTypeItalic">SCAR_884</em> were developed by excluding the first three nucleotides at the 5′-end of <em xmlns="" class="EmphasisTypeItalic">ISSR_884</em> primer and extending the sequence at the 3′-end. The <em xmlns="" class="EmphasisTypeItalic">884_200</em> fragment was specific to <em xmlns="" class="EmphasisTypeItalic">ICSR 110</em> parental line. However, the newly designed <em xmlns="" class="EmphasisTypeItalic">SCAR_884</em> primer-pairs amplified fragments both in <em xmlns="" class="EmphasisTypeItalic">Seredo</em> and <em xmlns="" class="EmphasisTypeItalic">ICSR 110</em> but of different fragment sizes. Sequencing of these fragments revealed that the fragment from <em xmlns="" class="EmphasisTypeItalic">Seredo</em> had a 220&nbsp;bp insertion. All F<sub>1</sub> individuals produced from the <em xmlns="" class="EmphasisTypeItalic">Seredo x ICSR 110</em> crosses had both fragments and hence the <em xmlns="" class="EmphasisTypeItalic">SCAR_884</em> marker is co-dominant and can be used to differentiate homozygotes and heterozygotes at this locus. This marker showed significant association with Al tolerance (<em xmlns="" class="EmphasisTypeItalic">P</em> &lt; 0.05) when analyzed using both NRL<sub>Al</sub> (<em xmlns="" class="EmphasisTypeItalic">R</em><sup>2</sup> = 0.017) and %RRG (<em xmlns="" class="EmphasisTypeItalic">R</em><sup>2</sup> = 0.025) indices.</p>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par30" class="Para">Analysis of the parental lines and some of their F<sub>2:3</sub> progeny using STS markers <em xmlns="" class="EmphasisTypeItalic">M181</em> and <em xmlns="" class="EmphasisTypeItalic">CTG29</em> produced fragments of the same size for each marker, and could not be used to differentiate the tolerant and sensitive lines. Hence, the PCR products of the parental lines as well as five most tolerant and four most sensitive F<sub>2:3</sub> progeny were sequenced for both markers in an attempt to identify useful sequence variation. The DNA sequences of marker <em xmlns="" class="EmphasisTypeItalic">M181</em> were identical for all genotypes sequenced whereas a two base pair <em xmlns="" class="EmphasisTypeItalic">indel</em> (GA/−) that differentiated the sensitive and tolerant parents was observed in the case of <em xmlns="" class="EmphasisTypeItalic">CTG29</em> (Table&nbsp;<span class="InternalRef"><a href="#Tab4">4</a></span>). Based on these sequences, a new dominant marker, designated as <em xmlns="" class="EmphasisTypeItalic">CTG29_3b</em> (Table <span class="InternalRef"><a href="#Tab2">2</a></span>), was developed by designing a new forward primer that included the <em xmlns="" class="EmphasisTypeItalic">indel</em> sequence. The <em xmlns="" class="EmphasisTypeItalic">CTG29_3b</em> marker amplified by this forward primer and the <em xmlns="" class="EmphasisTypeItalic">CTG29</em> reverse primer was present in <em xmlns="" class="EmphasisTypeItalic">ICSR 110</em> and most tolerant F<sub>2:3</sub> progeny and absent in <em xmlns="" class="EmphasisTypeItalic">Seredo</em> and most sensitive F<sub>2:3</sub> progeny. The association of this marker with Al tolerance when analyzed based on NRL<sub>Al</sub> (<em xmlns="" class="EmphasisTypeItalic">R</em><sup>2</sup> = 0.048) and %RRG (<em xmlns="" class="EmphasisTypeItalic">R</em><sup>2</sup> = 0.040) is significant (<em xmlns="" class="EmphasisTypeItalic">P</em> &lt; 0.005) (Table <span class="InternalRef"><a href="#Tab2">2</a></span>).<figure xmlns="" id="Tab4" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 4</span>
                        <p xmlns="" class="SimplePara">Partial DNA sequence of STS locus <em class="EmphasisTypeItalic">CTG29</em> showing indels that differentiated the aluminium tolerant F<sub xmlns="http://www.w3.org/1999/xhtml">2:3</sub> progenies (15r, 15A2, 21a, 15&nbsp;m, 21&nbsp;h, 15v and 42c) and parental line <em class="EmphasisTypeItalic">ICSR 110</em> from aluminium sensitive F<sub xmlns="http://www.w3.org/1999/xhtml">2:3</sub> progeny (60f, 59a, 689, 693) and parental line <em class="EmphasisTypeItalic">Seredo</em></p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Sample code</p></th><th colspan="38"><p xmlns="" class="SimplePara">DNA sequence</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">15r</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">C</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">15A2</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">C</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">21a</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">C</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">15&nbsp;m</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">C</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">21&nbsp;h</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">C</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">15v</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">C</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">42c</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">C</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">ICSR-110</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">C</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Seredo</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">–</p></td><td><p xmlns="" class="SimplePara">–</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">C</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">60f</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">–</p></td><td><p xmlns="" class="SimplePara">–</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">C</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">59a</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">–</p></td><td><p xmlns="" class="SimplePara">–</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">C</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">689</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">–</p></td><td><p xmlns="" class="SimplePara">–</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">C</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">693</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">–</p></td><td><p xmlns="" class="SimplePara">–</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">C</p></td></tr></tbody></table></div></figure></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par31" class="Para">The BLAST searching of <em xmlns="" class="EmphasisTypeItalic">CTG29-3b</em> sequence against the sorghum genome sequence at <span class="ExternalRef"><a href="http://www.gramene.org"><span class="RefSource">http://www.gramene.org</span></a></span> identified a sequence that share 177&nbsp;bp (with 100% sequence similarity) with <em xmlns="" class="EmphasisTypeItalic">CTG29-3b</em> at position 70,979,101–70,979,277&nbsp;bp on chromosome 3, in agreement with previous reports [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>, <span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>]. The corresponding position of this marker on sorghum chromosome 3 (GenBank accession number <em xmlns="" class="EmphasisTypeItalic">NC_012872.2</em>) is 70,939,651–70,939,834&nbsp;bp (Table <span class="InternalRef"><a href="#Tab2">2</a></span>). Overall, single marker analysis revealed that the mean net root length of the different genotypic groups for each of the four dominant markers (<em xmlns="" class="EmphasisTypeItalic">811_1400</em>, <em xmlns="" class="EmphasisTypeItalic">835_200</em>, <em xmlns="" class="EmphasisTypeItalic">884_200</em>, <em xmlns="" class="EmphasisTypeItalic">CTG29_3b</em>) was significantly different (Fig.&nbsp;<span class="InternalRef"><a href="#Fig4">4</a></span>).<figure xmlns="" class="Figure" id="Fig4"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO4">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0059-3/MediaObjects/41065_2018_59_Fig4_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0059-3/MediaObjects/41065_2018_59_Fig4_HTML.gif" alt="Fig. 4" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0059-3/MediaObjects/41065_2018_59_Fig4_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0059-3/MediaObjects/41065_2018_59_Fig4_HTML.gif" alt="Fig. 4"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 4</span>
                        <p xmlns="" class="SimplePara">Net root length of sorghum F<sub xmlns="http://www.w3.org/1999/xhtml">2:3</sub> progeny grown in the presence of 148&nbsp;μM aluminium based on the different genotypic classes for four dominant markers (<em class="EmphasisTypeItalic">811_1400</em>, <em class="EmphasisTypeItalic">835_200</em>, <em class="EmphasisTypeItalic">CTG29_3b</em> and <em class="EmphasisTypeItalic">884_200</em>). For each locus, genotypes with different letters indicated on top of the bars were significantly different from each other in their tolerance to Al toxicity. The error bars represent standard error values</p>
                      </div></figcaption></figure></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par32" class="Para">In the case of SSR, polymorphism was detected in nine out of the 24 loci analyzed for the parents. Data analysis based on the 229&nbsp;F<sub>2:3</sub> progeny together with the parental lines revealed that markers from four of the nine loci (<em xmlns="" class="EmphasisTypeItalic">Xtxp34</em>, <em xmlns="" class="EmphasisTypeItalic">Sb6_34</em>, <em xmlns="" class="EmphasisTypeItalic">Sb5_236</em> and <em xmlns="" class="EmphasisTypeItalic">Sb6_342</em>) showed significant association with both NRL<sub>Al</sub> and %RRG (Table <span class="InternalRef"><a href="#Tab2">2</a></span>; Fig.&nbsp;<span class="InternalRef"><a href="#Fig5">5</a></span>). The mean net root growth of the different genotypic classes, especially the homozygous classes were significantly different (<em xmlns="" class="EmphasisTypeItalic">P</em> ≤ 0.05; Fig. <span class="InternalRef"><a href="#Fig5">5</a></span>). For marker <em xmlns="" class="EmphasisTypeItalic">Xtxp34</em>, the mean net root lengths of the three genotypic classes were distinctly different (Fig. <span class="InternalRef"><a href="#Fig5">5</a></span>). For markers <em xmlns="" class="EmphasisTypeItalic">Sb6_342</em> and <em xmlns="" class="EmphasisTypeItalic">Sb6_34</em>, the heterozygous classes (AB) had mean net root lengths that were intermediate between the homozygous classes and were not significantly different from both homozygous parents (<em xmlns="" class="EmphasisTypeItalic">P</em> &gt; 0.05). Whereas the heterozygous class in marker <em xmlns="" class="EmphasisTypeItalic">Sb5_236</em> had mean net root length that was similar to net root length of the B homozygous class. The NRL<sub>Al</sub> phenotypic variance (<em xmlns="" class="EmphasisTypeItalic">R</em><sup>2</sup> values) explained by these SSR markers were 14% for <em xmlns="" class="EmphasisTypeItalic">Xtxp34</em>, 6% for <em xmlns="" class="EmphasisTypeItalic">Sb6_34</em>, 11% for <em xmlns="" class="EmphasisTypeItalic">Sb5_236</em> and 4% <em xmlns="" class="EmphasisTypeItalic">Sb6_342</em>. For %RRG, these values are 7, 2, 5 and 3%, respectively (Table <span class="InternalRef"><a href="#Tab2">2</a></span>). The four markers on chromosome 3 (<em xmlns="" class="EmphasisTypeItalic">811_1400</em>, <em xmlns="" class="EmphasisTypeItalic">CTG29_3b</em>, <em xmlns="" class="EmphasisTypeItalic">Xtxp34</em> and <em xmlns="" class="EmphasisTypeItalic">Sb5_236</em>) together explained 22% of the variation in NRL<sub>Al</sub> and 12% of the variation in %RRG (<em xmlns="" class="EmphasisTypeItalic">P</em> &lt; 0.001). The order of these markers on chromosome 3 (accession number <em xmlns="" class="EmphasisTypeItalic">NC_012872.2</em>) is <em xmlns="" class="EmphasisTypeItalic">Sb5_236</em>, <em xmlns="" class="EmphasisTypeItalic">Xtxp34</em>, <em xmlns="" class="EmphasisTypeItalic">CTG29_3b</em> and <em xmlns="" class="EmphasisTypeItalic">811_1400</em> with 17,426 kbp, 1235 kbp and 532 kbp between them in that order (Fig.&nbsp;<span class="InternalRef"><a href="#Fig6">6</a></span>). The closest marker to the <em xmlns="" class="EmphasisTypeItalic">SbMATE</em> gene is <em xmlns="" class="EmphasisTypeItalic">CTG29_3b</em> with 166 kbp between them. Overall, the eight markers explained 25% of the variation in NRL<sub>Al</sub> and 12% of the variation in %RRG (<em xmlns="" class="EmphasisTypeItalic">P</em> &lt; 0.001).<figure xmlns="" class="Figure" id="Fig5"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO5">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0059-3/MediaObjects/41065_2018_59_Fig5_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0059-3/MediaObjects/41065_2018_59_Fig5_HTML.gif" alt="Fig. 5" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0059-3/MediaObjects/41065_2018_59_Fig5_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0059-3/MediaObjects/41065_2018_59_Fig5_HTML.gif" alt="Fig. 5"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 5</span>
                        <p xmlns="" class="SimplePara">Net root length of sorghum F<sub xmlns="http://www.w3.org/1999/xhtml">2:3</sub> progeny grown in the presence of 148&nbsp;μM aluminium based on the different genotypic classes of four codominant markers (<em class="EmphasisTypeItalic">Xtxp34</em>, <em class="EmphasisTypeItalic">Sb6_342</em>, <em class="EmphasisTypeItalic">Sb6_34</em> and <em class="EmphasisTypeItalic">Sb5_236</em>). For each locus, genotypes that share the same letter indicated on top of the bars were not significantly different from each other in their tolerance to Al toxicity. The error bars represent standard error values</p>
                      </div></figcaption></figure><figure xmlns="" class="Figure" id="Fig6"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO6">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0059-3/MediaObjects/41065_2018_59_Fig6_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0059-3/MediaObjects/41065_2018_59_Fig6_HTML.gif" alt="Fig. 6" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0059-3/MediaObjects/41065_2018_59_Fig6_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0059-3/MediaObjects/41065_2018_59_Fig6_HTML.gif" alt="Fig. 6"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 6</span>
                        <p xmlns="" class="SimplePara">Diagrammatic representation of the four markers on chromosome 3 (<em class="EmphasisTypeItalic">Sb5_236</em>, <em class="EmphasisTypeItalic">Xtxp34</em>, <em class="EmphasisTypeItalic">CTG29_3b and 811_1400</em>) showing their order and distance between them in Mbp</p>
                      </div></figcaption></figure></div>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec12" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Discussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Discussion">Discussion</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Discussion" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par33" class="Para">Several molecular markers that are linked to aluminium tolerance genes have been identified in different crops. In rye, three AFLP markers [<span class="CitationRef"><a href="#CR36" data-toggle="citation">36</a></span>] that are linked to gene <em xmlns="" class="EmphasisTypeItalic">Alt3</em> and two RAPD markers that were converted to SCAR markers [<span class="CitationRef"><a href="#CR37" data-toggle="citation">37</a></span>], flank <em xmlns="" class="EmphasisTypeItalic">Alt1</em> gene. In maize, RFLP and SSR markers linked to Al tolerance QTLs have been reported [<span class="CitationRef"><a href="#CR38" data-toggle="citation">38</a></span>]. Raman et al. [<span class="CitationRef"><a href="#CR39" data-toggle="citation">39</a></span>] identified AFLP and microsatellite markers linked to the aluminium tolerance gene located on chromosome 4H in barley. The identification of molecular markers linked to Al tolerance in sorghum through genetic linkage, QTL and association mapping can accelerate the development of high yielding varieties that can withstand Al stress [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>, <span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>–<span class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>]. Three ISSR (<em xmlns="" class="EmphasisTypeItalic">811_1400</em>, <em xmlns="" class="EmphasisTypeItalic">835_200</em> and <em xmlns="" class="EmphasisTypeItalic">884_200</em>), one STS (<em xmlns="" class="EmphasisTypeItalic">CTG29_3b</em>) and four SSR (<em xmlns="" class="EmphasisTypeItalic">Xtxp34</em>, <em xmlns="" class="EmphasisTypeItalic">Sb6_34</em>, <em xmlns="" class="EmphasisTypeItalic">Sb5_236</em> and <em xmlns="" class="EmphasisTypeItalic">Sb6_342</em>) markers showed significant association with aluminium tolerance in this study. Four of these markers (<em xmlns="" class="EmphasisTypeItalic">811_1400</em>, <em xmlns="" class="EmphasisTypeItalic">CTG29_3b, Xtxp34</em> and <em xmlns="" class="EmphasisTypeItalic">Sb5_236</em>) are located on chromosome 3 whereas markers <em xmlns="" class="EmphasisTypeItalic">884_200</em>, <em xmlns="" class="EmphasisTypeItalic">Sb6_342, Sb6_34</em> and <em xmlns="" class="EmphasisTypeItalic">835_200</em> are located on chromosomes 6, 7, 8 and 10, respectively.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par34" class="Para">Genes that regulate Al tolerance in plants are shown to have different mode of actions. In rice, two genes that were referred to as sensitive to Al rhizotoxicity1 and 2 (<em xmlns="" class="EmphasisTypeItalic">STAR1</em> and <em xmlns="" class="EmphasisTypeItalic">STAR2</em>) were reported to function as an ATP binding cassette (ABC) transporter and have significant contribution in Al tolerance [<span class="CitationRef"><a href="#CR40" data-toggle="citation">40</a></span>]. Yamaji et al. [<span class="CitationRef"><a href="#CR41" data-toggle="citation">41</a></span>] reported a C2H2-type zinc finger transcription factor, Al resistance transcription factor 1 (<em xmlns="" class="EmphasisTypeItalic">ART1</em>), which regulates the expression of genes related to Al tolerance in rice, including <em xmlns="" class="EmphasisTypeItalic">STAR1</em> and <em xmlns="" class="EmphasisTypeItalic">STAR2</em>. In line with these studies, the present study suggests the involvement of several genes in regulating Al tolerance in sorghum, as the markers that showed significant association with Al tolerance are located on five different chromosomes.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par35" class="Para">To date, only one Al tolerance locus, <em xmlns="" class="EmphasisTypeItalic">Alt</em><sub>
                  <em xmlns="" class="EmphasisTypeItalic">SB</em>
                </sub>, which underlies the <em xmlns="" class="EmphasisTypeItalic">SbMATE</em> gene in sorghum, has been identified and mapped to the terminal region of chromosome 3 and characterized in detail [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>, <span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>, <span class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>]. The <em xmlns="" class="EmphasisTypeItalic">SbMATE</em> gene, codes for aluminium-activated citrate transporter that is highly expressed in root apices of aluminium-tolerant sorghum lines [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>]. Caniato et al. [<span class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>] developed functional markers within the <em xmlns="" class="EmphasisTypeItalic">SbMATE</em> gene that can be used for marker assisted selection for aluminium tolerance. In this study, two STS markers <em xmlns="" class="EmphasisTypeItalic">CTG29</em> and <em xmlns="" class="EmphasisTypeItalic">M181</em> that are located at 0.2&nbsp;cM and 0&nbsp;cM from <em xmlns="" class="EmphasisTypeItalic">Alt</em><sub>
                  <em xmlns="" class="EmphasisTypeItalic">SB</em>
                </sub> [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>] were tested on Al tolerant and Al sensitive parental lines. STS <em xmlns="" class="EmphasisTypeItalic">M181</em> was monomorphic and could not be used in the genotyping of the <em xmlns="" class="EmphasisTypeItalic">Seredo</em> x <em xmlns="" class="EmphasisTypeItalic">ICSR 110</em>&nbsp;F<sub>2:3</sub> progeny. The <em xmlns="" class="EmphasisTypeItalic">CTG29_3b</em> dominant marker that was successfully developed using the nucleotide polymorphic site, showed significant association with Al tolerance and therefore suggests the role of <em xmlns="" class="EmphasisTypeItalic">Alt</em><sub>
                  <em xmlns="" class="EmphasisTypeItalic">SB</em>
                </sub> in contributing to Al tolerance in the population studied.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par36" class="Para">SSRs can be used in cultivar genotyping, genotype identification, genetic diversity assessment and marker-assisted breeding [<span class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>, <span class="CitationRef"><a href="#CR42" data-toggle="citation">42</a></span>]. Matos et al. [<span class="CitationRef"><a href="#CR43" data-toggle="citation">43</a></span>] developed EST-SSRs that are associated with aluminium tolerance in rye. The SSR markers that showed significant association with Al tolerance in the present study are located on chromosomes 3, 7 and 8. SSR marker <em xmlns="" class="EmphasisTypeItalic">Xtxp34</em> has been mapped to chromosome 3 [<span class="CitationRef"><a href="#CR41" data-toggle="citation">41</a></span>, <span class="CitationRef"><a href="#CR44" data-toggle="citation">44</a></span>]. <em xmlns="" class="EmphasisTypeItalic">Sb6_342</em> was mapped to chromosome 7 [<span class="CitationRef"><a href="#CR44" data-toggle="citation">44</a></span>] whereas <em xmlns="" class="EmphasisTypeItalic">Sb5_236</em> was mapped to chromosome 3 [<span class="CitationRef"><a href="#CR41" data-toggle="citation">41</a></span>, <span class="CitationRef"><a href="#CR44" data-toggle="citation">44</a></span>], both of which were confirmed in the present study through BLAST searching of the sorghum genome. Similarly, the SSR marker <em xmlns="" class="EmphasisTypeItalic">Sb6_34</em> was located on chromosome 8 through BLAST search. These results indicate that in addition to <em xmlns="" class="EmphasisTypeItalic">SbMATE</em>, other genes are involved in conferring Al tolerance in sorghum. Hence, there is a need to build on these findings by screening more markers close to the identified SSRs to ascertain the presence of new Al tolerance genes in these regions.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par37" class="Para">The <em xmlns="" class="EmphasisTypeItalic">ISSR_811</em> primer amplified a distinct locus of about1400 bp (<em xmlns="" class="EmphasisTypeItalic">811_1400</em>) that differentiated the two parents and also showed a significant association with aluminium tolerance in the F<sub>2:3</sub> population studied. This marker is located close to <em xmlns="" class="EmphasisTypeItalic">SbMATE</em> gene on chromosome 3 with about 366 kbp between them (Fig. <span class="InternalRef"><a href="#Fig6">6</a></span>), and hence the phenotypic variance of the aluminium tolerance indices explained by this marker is most likely due to <em xmlns="" class="EmphasisTypeItalic">SbMATE</em> gene. However, <em xmlns="" class="EmphasisTypeItalic">811_1400</em> (<em xmlns="" class="EmphasisTypeItalic">SCAR_811</em>) is also located very close to Calmodulin-Binding Transcription Activator 4 (<em xmlns="" class="EmphasisTypeItalic">CAMTA4</em>) on this chromosome with only 7.3 kbp between them. <em xmlns="" class="EmphasisTypeItalic">CAMTA2</em> is an activator of Al inducible <em xmlns="" class="EmphasisTypeItalic">ALMT1</em> expression in <em xmlns="" class="EmphasisTypeItalic">Arabidopsis AtALMT1</em> [<span class="CitationRef"><a href="#CR45" data-toggle="citation">45</a></span>]. Marker <em xmlns="" class="EmphasisTypeItalic">CTG29-3b</em> is located closer to <em xmlns="" class="EmphasisTypeItalic">SbMATE</em> gene (with only 166 kbp between them) than marker <em xmlns="" class="EmphasisTypeItalic">811_1400</em>. However, <em xmlns="" class="EmphasisTypeItalic">811_1400</em> explained more phenotypic variance (<em xmlns="" class="EmphasisTypeItalic">R</em><sup>2</sup> = 0.10) than <em xmlns="" class="EmphasisTypeItalic">CTG29_3b</em> (<em xmlns="" class="EmphasisTypeItalic">R</em><sup>2</sup> = 0.04) in NRL<em xmlns="" class="EmphasisTypeItalic">Al</em>, in this study. This may suggest the role of <em xmlns="" class="EmphasisTypeItalic">CAMTA4</em> in regulating <em xmlns="" class="EmphasisTypeItalic">SbMATE</em> gene in sorghum.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par38" class="Para">The other two markers on chromosome 3 are located relatively far from <em xmlns="" class="EmphasisTypeItalic">SbMATE</em> with 17.4 Mbp (<em xmlns="" class="EmphasisTypeItalic">Xtxp34</em>) and 1.2 Mbp (<em xmlns="" class="EmphasisTypeItalic">Sb5_236</em>) when compared with the locations of <em xmlns="" class="EmphasisTypeItalic">CTG29_3b</em> and <em xmlns="" class="EmphasisTypeItalic">811_1400</em>. However, these markers explained higher phenotypic variance of the aluminium tolerance indices than <em xmlns="" class="EmphasisTypeItalic">CTG29_3b</em> and <em xmlns="" class="EmphasisTypeItalic">811_1400</em>, which may suggest the presence of other genes around the vicinity of these markers that influence Al tolerance in sorghum. It is interesting to note that any pair of these markers on chromosome 3 explained more phenotypic variance than the variance they explained separately (data not shown) suggesting the independence of these markers in their contribution to Al tolerance. The four markers together explained 22% of the variance in NRL<sub>Al</sub> and 12% in %RRG (Table <span class="InternalRef"><a href="#Tab2">2</a></span>), which is higher than the variance explained by any three or two or one of these markers. Hence, combined use of these markers together with markers on other chromosomes is a preferable approach in screening sorghum genetic resources for tolerance to Al toxicity.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par39" class="Para">The homeologous relationships between genomes of various cereals have been established [<span class="CitationRef"><a href="#CR39" data-toggle="citation">39</a></span>, <span class="CitationRef"><a href="#CR46" data-toggle="citation">46</a></span>–<span class="CitationRef"><a href="#CR50" data-toggle="citation">50</a></span>]. Previous studies [<span class="CitationRef"><a href="#CR46" data-toggle="citation">46</a></span>–<span class="CitationRef"><a href="#CR49" data-toggle="citation">49</a></span>] have shown that the long arm of wheat chromosome 4D (4DL) is partially homologous to the proximal portion of the short arm of rye chromosome 7R. Moreover, barley chromosome arm 4HL is homologous to the wheat chromosome arm 4DL [<span class="CitationRef"><a href="#CR39" data-toggle="citation">39</a></span>] and a consensus grass comparative map has shown that rice chromosome 3 is homologous to wheat chromosome four [<span class="CitationRef"><a href="#CR50" data-toggle="citation">50</a></span>]. An Aluminium-activated malate transporter 1-like (<em xmlns="" class="EmphasisTypeItalic">ALMT1</em>-like) gene (LOC110436609) is located at position 43,835,494–43,840,158&nbsp;bp on sorghum chromosome 6 (accession number <em xmlns="" class="EmphasisTypeItalic">Nc_012875.2</em>). Marker <em xmlns="" class="EmphasisTypeItalic">884_200</em> (<em xmlns="" class="EmphasisTypeItalic">SCAR_884</em>) that showed significant association with aluminium tolerance in this study is located at position 42,689,435–42,689,555&nbsp;bp on this chromosome. Since this marker is located close to <em xmlns="" class="EmphasisTypeItalic">ALMT1</em>-like gene with only about 1.1 Mbp between them, and a strong positive correlation between Al induced malate exudation and relative net root growth has already been shown in sorghum [<span class="CitationRef"><a href="#CR51" data-toggle="citation">51</a></span>], the significant association of this marker with aluminium tolerance is likely due to the role of <em xmlns="" class="EmphasisTypeItalic">ALMT</em>1-like gene in Al tolerance. This also suggests that the sorghum <em xmlns="" class="EmphasisTypeItalic">ALMT1</em>-like gene is an orthologue of <em xmlns="" class="EmphasisTypeItalic">ALMTI</em>, as <em xmlns="" class="EmphasisTypeItalic">ALMTI</em> gene that encodes Al activated malate transporter is known to regulate aluminium tolerance in wheat, barley and rye [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>, <span class="CitationRef"><a href="#CR52" data-toggle="citation">52</a></span>, <span class="CitationRef"><a href="#CR53" data-toggle="citation">53</a></span>]. Hence, it is important that this gene is investigated in detail in relation to aluminium tolerance in sorghum.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec13" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Conclusion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Conclusion">Conclusion</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Conclusion" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par40" class="Para">The four markers on chromosome 3 (<em xmlns="" class="EmphasisTypeItalic">811_1400</em> or <em xmlns="" class="EmphasisTypeItalic">SCAR_811</em>, <em xmlns="" class="EmphasisTypeItalic">CTG29_3b</em>, <em xmlns="" class="EmphasisTypeItalic">Xtxp34</em> and <em xmlns="" class="EmphasisTypeItalic">Sb5_236</em>) suggest the role of <em xmlns="" class="EmphasisTypeItalic">Alt</em><sub>
                  <em xmlns="" class="EmphasisTypeItalic">SB</em>
                </sub> locus (<em xmlns="" class="EmphasisTypeItalic">SbMATE</em> gene) in Al tolerance in the sorghum population used in the present study. However, their positions on chromosome 3 in relation to the position of <em xmlns="" class="EmphasisTypeItalic">SbMATE</em> and the phenotypic variance explained by each of these markers suggest the presence of other genes on this chromosome that play a role in aluminium tolerance in sorghum. The presence of four other markers on different chromosomes that showed significant association with aluminium tolerance suggests the presence of additional genes that contribute to this trait in sorghum. Since the eight markers together explained higher phenotypic variance of the two Al tolerance indices than what explained by individual markers or any other combination of markers, the use of all markers in screening sorghum germplasm for Al tolerance is recommended.</p></div></section>
          <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments"><h3 xmlns="" class="Heading">Acknowledgments</h3><p xmlns="" class="SimplePara">This work was supported by the Swedish International Development Agency (SIDA/SAREC) through the BIO-EARN program and the Swedish University of Agricultural Sciences (SLU) through funding from the Swedish Ministry of Foreign Affairs as part of its special allocation on global food security (UD40). The authors are grateful to ICRISAT and MUSRT for providing the sorghum seeds. We confirm that none of the coauthors have conflict of interest to declare.</p></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="FormalPara FormalParaRenderingStyle1"><h3 xmlns="" class="Heading">Authors’ contributions</h3><p xmlns="" class="SimplePara">EJT conducted the lab work and data analysis and drafted the manuscript. SG and BAW were involved in selecting the parental lines and developing the mapping population. OAO was involved in the physiology aspects. AC contributed to the development of molecular markers. MG contributed to the lab work, data analysis and last stage revision of the manuscript. All co-authors contributed to the interpretation of the results and critically reviewed the manuscript at its initial stage and read and approved final version of manuscript for publication.</p></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Ethics">
              <div id="FPar1" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Competing interests</h3>
                <p id="Par41" class="Para">The authors declare that they have no competing interests.</p>
              </div>
              <div id="FPar2" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Publisher’s Note</h3>
                <p id="Par42" class="Para">Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
              </div>
            </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="License">
                <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Open Access</strong>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/4.0/"><span class="RefSource">http://creativecommons.org/licenses/by/4.0/</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/publicdomain/zero/1.0/"><span class="RefSource">http://creativecommons.org/publicdomain/zero/1.0/</span></a></span>) applies to the data made available in this article, unless otherwise stated.</p>
              </div></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authors’ Affiliations</a></h2><div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1"><span class="AffiliationNumber">(1)</span><div class="AffiliationText">Department of Biological Sciences, University of Eldoret, P.O. Box 1125-30100, Eldoret, Kenya</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff2"><span class="AffiliationNumber">(2)</span><div class="AffiliationText">Rongo University College, P.O. Box 103-40404, Rongo, Kenya</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff3"><span class="AffiliationNumber">(3)</span><div class="AffiliationText">Department of Plant Breeding, Swedish University of Agricultural Sciences, P.O. Box 101, SE-230 53&nbsp;Alnarp, Sweden</div></div></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper"><li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kindraide TB, Parker DR. Assessing the phytotoxicity of mononuclear hydroxyl-aluminum. Plant Cell and Environ. 1989;12:479–87.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1365-3040.1989.tb02120.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Assessing%20the%20phytotoxicity%20of%20mononuclear%20hydroxyl-aluminum&amp;author=TB.%20Kindraide&amp;author=DR.%20Parker&amp;journal=Plant%20Cell%20and%20Environ&amp;volume=12&amp;pages=479-487&amp;publication_year=1989"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ryan PR, Tyerman SD, Sasaki T, Furuichi T, Yamamoto Y, Zhang WH, Delhaize E. The identification of aluminium-resistance genes provides opportunities for enhancing crop production on acid soils. Exp Bot. 2011;62:9–20.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/jxb/erq272"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20identification%20of%20aluminium-resistance%20genes%20provides%20opportunities%20for%20enhancing%20crop%20production%20on%20acid%20soils&amp;author=PR.%20Ryan&amp;author=SD.%20Tyerman&amp;author=T.%20Sasaki&amp;author=T.%20Furuichi&amp;author=Y.%20Yamamoto&amp;author=WH.%20Zhang&amp;author=E.%20Delhaize&amp;journal=Exp%20Bot&amp;volume=62&amp;pages=9-20&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Famoso AN, Clark RT, Shaff JE, Craft E, Mccouch SR, Kochian LV. Development of a novel aluminium tolerance phenotyping platform used for comparisons of cereal aluminium tolerance and investigations into rice aluminium tolerance mechanisms. Plant Physiol. 2010;153:1678–91.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1104/pp.110.156794"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20538888"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2923895"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Development%20of%20a%20novel%20aluminium%20tolerance%20phenotyping%20platform%20used%20for%20comparisons%20of%20cereal%20aluminium%20tolerance%20and%20investigations%20into%20rice%20aluminium%20tolerance%20mechanisms&amp;author=AN.%20Famoso&amp;author=RT.%20Clark&amp;author=JE.%20Shaff&amp;author=E.%20Craft&amp;author=SR.%20Mccouch&amp;author=LV.%20Kochian&amp;journal=Plant%20Physiol&amp;volume=153&amp;pages=1678-1691&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Horst WJ, Wang YX, Eticha D. The role of the root apoplast in aluminium-induced inhibition of root elongation and in aluminium resistance of plants: a review. Ann Bot. 2010;106:185–97.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/aob/mcq053"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20237112"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2889789"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20role%20of%20the%20root%20apoplast%20in%20aluminium-induced%20inhibition%20of%20root%20elongation%20and%20in%20aluminium%20resistance%20of%20plants%3A%20a%20review&amp;author=WJ.%20Horst&amp;author=YX.%20Wang&amp;author=D.%20Eticha&amp;journal=Ann%20Bot&amp;volume=106&amp;pages=185-197&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Samac DA, Tesfaye M. Plant improvement for resistance to aluminium in acid soils-a review. Plant Cell Tissue Organ Cult. 2003;75:189–207.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1023/A%3A1025843829545"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Plant%20improvement%20for%20resistance%20to%20aluminium%20in%20acid%20soils-a%20review&amp;author=DA.%20Samac&amp;author=M.%20Tesfaye&amp;journal=Plant%20Cell%20Tissue%20Organ%20Cult&amp;volume=75&amp;pages=189-207&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Jansen S, Broadley MR, Robbrecht W, Smets E. Aluminium hyper accumulation in angiosperms: a review of its phylogenetic significance. Bot Rev. 2002;68:235–69.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1663/0006-8101(2002)068%5B0235%3AAHIAAR%5D2.0.CO%3B2"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Aluminium%20hyper%20accumulation%20in%20angiosperms%3A%20a%20review%20of%20its%20phylogenetic%20significance&amp;author=S.%20Jansen&amp;author=MR.%20Broadley&amp;author=W.%20Robbrecht&amp;author=E.%20Smets&amp;journal=Bot%20Rev&amp;volume=68&amp;pages=235-269&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kanyanjua SM, Iren I, Wambua S, Nandwa SM. Acidic soils in Kenya: constraints and remedial options. KARI Technical Notes Series No. 2002:11.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Kanyanjua%20SM%2C%20Iren%20I%2C%20Wambua%20S%2C%20Nandwa%20SM.%20Acidic%20soils%20in%20Kenya%3A%20constraints%20and%20remedial%20options.%20KARI%20Technical%20Notes%20Series%20No.%202002%3A11."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Obura PA. Effects of soil properties on bioavailability of aluminium and phosphorus in selected Kenyan and Brazilian soils. PhD thesis, Purdue University. In: USA; 2008.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Effects%20of%20soil%20properties%20on%20bioavailability%20of%20aluminium%20and%20phosphorus%20in%20selected%20Kenyan%20and%20Brazilian%20soils.%20PhD%20thesis%2C%20Purdue%20University&amp;author=PA.%20Obura&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wortmann CS, Mamo M, Abebe G, Mburu C, Kayuki KC, Letayo E, Xerinda S. Atlas of sorghum (Sorghum bicolor (L.) Moench) production in five countries of eastern Africa. The Board of Regents of the University of Nebraska, Lincoln, USA; 2006.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Wortmann%20CS%2C%20Mamo%20M%2C%20Abebe%20G%2C%20Mburu%20C%2C%20Kayuki%20KC%2C%20Letayo%20E%2C%20Xerinda%20S.%20Atlas%20of%20sorghum%20%28Sorghum%20bicolor%20%28L.%29%20Moench%29%20production%20in%20five%20countries%20of%20eastern%20Africa.%20The%20Board%20of%20Regents%20of%20the%20University%20of%20Nebraska%2C%20Lincoln%2C%20USA%3B%202006."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hiradate S, Ma JF, Matsumoto H. Strategies of plants to adapt to mineral stresses in problem soils. Adv Agron. 2007;96:65–132.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/S0065-2113(07)96004-6"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Strategies%20of%20plants%20to%20adapt%20to%20mineral%20stresses%20in%20problem%20soils&amp;author=S.%20Hiradate&amp;author=JF.%20Ma&amp;author=H.%20Matsumoto&amp;journal=Adv%20Agron&amp;volume=96&amp;pages=65-132&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Naumann A, Horst WJ. Effect of aluminium supply on aluminium uptake, translocation and blueing of Hydrangea macrophylla (Thunb.) Ser. Cultivars in a peat-clay substrate. J Hort Sci Biotechnol. 2003;78:463–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1080/14620316.2003.11511651"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Effect%20of%20aluminium%20supply%20on%20aluminium%20uptake%2C%20translocation%20and%20blueing%20of%20Hydrangea%20macrophylla%20%28Thunb.%29%20Ser.%20Cultivars%20in%20a%20peat-clay%20substrate&amp;author=A.%20Naumann&amp;author=WJ.%20Horst&amp;journal=J%20Hort%20Sci%20Biotechnol&amp;volume=78&amp;pages=463-469&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ma JF, Ryan PR, Delhaize E. Aluminium resistance in plants and the complexing role of organic acids. Trends Plant Sci. 2001;6:273–8.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/S1360-1385(01)01961-6"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11378470"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Aluminium%20resistance%20in%20plants%20and%20the%20complexing%20role%20of%20organic%20acids&amp;author=JF.%20Ma&amp;author=PR.%20Ryan&amp;author=E.%20Delhaize&amp;journal=Trends%20Plant%20Sci&amp;volume=6&amp;pages=273-278&amp;publication_year=2001"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Guo TR, Zhang GP, Zhang YH. Physiological changes in barley plants under combined toxicity of aluminium, copper and cadmium. Colloid Surface B. 2007;57:182–8.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.colsurfb.2007.01.013"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Physiological%20changes%20in%20barley%20plants%20under%20combined%20toxicity%20of%20aluminium%2C%20copper%20and%20cadmium&amp;author=TR.%20Guo&amp;author=GP.%20Zhang&amp;author=YH.%20Zhang&amp;journal=Colloid%20Surface%20B&amp;volume=57&amp;pages=182-188&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Magalhaes JV, Liu J, Guimaraes CT, Lana UGP, Alves VMC, et al. A gene in the multidrug and toxic compound extrusion (MATE) family confers aluminium resistance in sorghum. Nat Genet. 2007;39:1156–61.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/ng2074"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17721535"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20gene%20in%20the%20multidrug%20and%20toxic%20compound%20extrusion%20%28MATE%29%20family%20confers%20aluminium%20resistance%20in%20sorghum&amp;author=JV.%20Magalhaes&amp;author=J.%20Liu&amp;author=CT.%20Guimaraes&amp;author=UGP.%20Lana&amp;author=VMC.%20Alves&amp;journal=Nat%20Genet&amp;volume=39&amp;pages=1156-1161&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Cambraia J, Galvani FR, Estevao MM. Effects of aluminium on organic acid, sugar and amino acid composition of the root system of sorghum (<em class="EmphasisTypeItalic">Sorghum bicolor</em>, L. Moench). J Plant Nutr. 1983;6:313–22.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1080/01904168309363092"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Effects%20of%20aluminium%20on%20organic%20acid%2C%20sugar%20and%20amino%20acid%20composition%20of%20the%20root%20system%20of%20sorghum%20%28Sorghum%20bicolor%2C%20L.%20Moench%29&amp;author=J.%20Cambraia&amp;author=FR.%20Galvani&amp;author=MM.%20Estevao&amp;journal=J%20Plant%20Nutr&amp;volume=6&amp;pages=313-322&amp;publication_year=1983"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gonçales JFC, Cambraia J, Mosquim PR, Araújo EF. Aluminium effect on organic acid production and accumulation in sorghum. J Plant Nutr. 2005;28:507–20.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1081/PLN-200049202"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Aluminium%20effect%20on%20organic%20acid%20production%20and%20accumulation%20in%20sorghum&amp;author=JFC.%20Gon%C3%A7ales&amp;author=J.%20Cambraia&amp;author=PR.%20Mosquim&amp;author=EF.%20Ara%C3%BAjo&amp;journal=J%20Plant%20Nutr&amp;volume=28&amp;pages=507-520&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sasaki T, Yamamoto Y, Ezaki B, Katsuhara M, Ahn S, et al. A wheat gene encoding an aluminum-activated malate transporter. Plant J. 2004;37:645–53.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1365-313X.2003.01991.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14871306"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20wheat%20gene%20encoding%20an%20aluminum-activated%20malate%20transporter&amp;author=T.%20Sasaki&amp;author=Y.%20Yamamoto&amp;author=B.%20Ezaki&amp;author=M.%20Katsuhara&amp;author=S.%20Ahn&amp;journal=Plant%20J&amp;volume=37&amp;pages=645-653&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Furukawa J, Yamaji N, Wang H, Mitani N, Murata Y, et al. An aluminum-activated citrate transporter in barley. Plant Cell Physiol. 2007;48:1081–91.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/pcp/pcm091"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17634181"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=An%20aluminum-activated%20citrate%20transporter%20in%20barley&amp;author=J.%20Furukawa&amp;author=N.%20Yamaji&amp;author=H.%20Wang&amp;author=N.%20Mitani&amp;author=Y.%20Murata&amp;journal=Plant%20Cell%20Physiol&amp;volume=48&amp;pages=1081-1091&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Flores CI, Clark RB, Gourley LM. Growth and yield traits of sorghum grown on acid soil at varied aluminum saturations. Plant Soil. 1988;106:49–57.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/BF02371194"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Growth%20and%20yield%20traits%20of%20sorghum%20grown%20on%20acid%20soil%20at%20varied%20aluminum%20saturations&amp;author=CI.%20Flores&amp;author=RB.%20Clark&amp;author=LM.%20Gourley&amp;journal=Plant%20Soil&amp;volume=106&amp;pages=49-57&amp;publication_year=1988"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">De Sousa CAN. Classification of Brazilian wheat cultivars for aluminium toxicity in acid soils. Plant Breed. 1998;117:217–21.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1439-0523.1998.tb01929.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Classification%20of%20Brazilian%20wheat%20cultivars%20for%20aluminium%20toxicity%20in%20acid%20soils&amp;author=CAN.%20De%20Sousa&amp;journal=Plant%20Breed&amp;volume=117&amp;pages=217-221&amp;publication_year=1998"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Bouton JH, Sumner ME. Alfalfa, Medicago sativa L., in highly weathered, acid soils. V. Field performance of alfalfa selected for acid tolerance. Plant Soil. 1983;74:431–6.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/BF02181360"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Alfalfa%2C%20Medicago%20sativa%20L.%2C%20in%20highly%20weathered%2C%20acid%20soils.%20V.%20Field%20performance%20of%20alfalfa%20selected%20for%20acid%20tolerance&amp;author=JH.%20Bouton&amp;author=ME.%20Sumner&amp;journal=Plant%20Soil&amp;volume=74&amp;pages=431-436&amp;publication_year=1983"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Mariano ED, Keltjens WG. Evaluating the role of root citrate exudation as a mechanism of aluminium resistance in maize genotypes. Plant Soil. 2003;256:469–79.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1023/A%3A1026106714644"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Evaluating%20the%20role%20of%20root%20citrate%20exudation%20as%20a%20mechanism%20of%20aluminium%20resistance%20in%20maize%20genotypes&amp;author=ED.%20Mariano&amp;author=WG.%20Keltjens&amp;journal=Plant%20Soil&amp;volume=256&amp;pages=469-479&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Caniato FF, Guimaraes CT, Schaffert RE, Alves VMC, Kochian LV, Boren A, Klein PE, Magalhaes JV. Genetic diversity for aluminium tolerance in sorghum. Theor Appl Genet. 2007;114:863–76.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00122-006-0485-x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17252254"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20for%20aluminium%20tolerance%20in%20sorghum&amp;author=FF.%20Caniato&amp;author=CT.%20Guimaraes&amp;author=RE.%20Schaffert&amp;author=VMC.%20Alves&amp;author=LV.%20Kochian&amp;author=A.%20Boren&amp;author=PE.%20Klein&amp;author=JV.%20Magalhaes&amp;journal=Theor%20Appl%20Genet&amp;volume=114&amp;pages=863-876&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR24" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Magalhaes JV, Garvin DF, Wang Y, Sorrels ME, Klein PE, Schaffert RE, Li L, Kochian LV. Comparative mapping of a major aluminium resistance gene in sorghum and other species of the poaceae. Genetics. 2004;167:1905–14.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1534/genetics.103.023580"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15342528"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1471010"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparative%20mapping%20of%20a%20major%20aluminium%20resistance%20gene%20in%20sorghum%20and%20other%20species%20of%20the%20poaceae&amp;author=JV.%20Magalhaes&amp;author=DF.%20Garvin&amp;author=Y.%20Wang&amp;author=ME.%20Sorrels&amp;author=PE.%20Klein&amp;author=RE.%20Schaffert&amp;author=L.%20Li&amp;author=LV.%20Kochian&amp;journal=Genetics&amp;volume=167&amp;pages=1905-1914&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR25" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Caniato FF, Guimarães CT, Hamblin M, Billot C, Rami J-F, et al. The relationship between population structure and aluminum tolerance in cultivated sorghum. PLoS One. 2011;6(6):e20830. <span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.1371/journal.pone.0020830"><span class="RefSource">https://doi.org/10.1371/journal.pone.0020830</span></a></span>.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0020830"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21695088"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3114870"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20relationship%20between%20population%20structure%20and%20aluminum%20tolerance%20in%20cultivated%20sorghum&amp;author=FF.%20Caniato&amp;author=CT.%20Guimar%C3%A3es&amp;author=M.%20Hamblin&amp;author=C.%20Billot&amp;author=J-F.%20Rami&amp;journal=PLoS%20One&amp;volume=6&amp;issue=6&amp;pages=e20830&amp;publication_year=2011&amp;doi=10.1371%2Fjournal.pone.0020830"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR26" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Leiser WL, Rattunde HFW, Weltzien E, Cisse N, Abdou M, Diallo A, Tourè AO, Magalhaes JV, Haussmann BIG. Two in one sweep: aluminum tolerance and grain yield in P-limited soils are associated to the same genomic region in west African sorghum. BMC Plant Biol. 2014;14:206.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/s12870-014-0206-6"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25112843"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4256928"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Two%20in%20one%20sweep%3A%20aluminum%20tolerance%20and%20grain%20yield%20in%20P-limited%20soils%20are%20associated%20to%20the%20same%20genomic%20region%20in%20west%20African%20sorghum&amp;author=WL.%20Leiser&amp;author=HFW.%20Rattunde&amp;author=E.%20Weltzien&amp;author=N.%20Cisse&amp;author=M.%20Abdou&amp;author=A.%20Diallo&amp;author=AO.%20Tour%C3%A8&amp;author=JV.%20Magalhaes&amp;author=BIG.%20Haussmann&amp;journal=BMC%20Plant%20Biol&amp;volume=14&amp;pages=206&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR27" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Caniato FF, Hamblin MT, Guimaraes CT, Zhang Z, Schaffert RE, et al. Association mapping provides insights into the origin and the fine structure of the sorghum aluminum tolerance locus, AltSB. PLoS One. 2014;9(1):e87438. <span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.1371/journal.pone.0087438"><span class="RefSource">https://doi.org/10.1371/journal.pone.0087438</span></a></span>.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0087438"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24498106"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3907521"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Association%20mapping%20provides%20insights%20into%20the%20origin%20and%20the%20fine%20structure%20of%20the%20sorghum%20aluminum%20tolerance%20locus%2C%20AltSB&amp;author=FF.%20Caniato&amp;author=MT.%20Hamblin&amp;author=CT.%20Guimaraes&amp;author=Z.%20Zhang&amp;author=RE.%20Schaffert&amp;journal=PLoS%20One&amp;volume=9&amp;issue=1&amp;pages=e87438&amp;publication_year=2014&amp;doi=10.1371%2Fjournal.pone.0087438"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR28" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Magnavaca R, Gardner C, Clark R. Comparison of maize populations for aluminium resistance in nutrient solution. In: Gabelman H, Loughman BC, editors. Genetic aspects of plant mineral nutrition. Dordrecht. Netherlands: Martinus Nijhoff; 1987. p. 189–99.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/978-94-009-3581-5_17"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparison%20of%20maize%20populations%20for%20aluminium%20resistance%20in%20nutrient%20solution&amp;author=R.%20Magnavaca&amp;author=C.%20Gardner&amp;author=R.%20Clark&amp;pages=189-199&amp;publication_year=1987"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR29" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ringo JH, Mneney EE, Onkware AO, Were BA, Too EJ, Owuoche JO, Gudu SO. Tolerance to Aluminium toxicity in Tanzanian sorghum genotypes. Afr Crop Sci J. 2010;18:155–64.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Tolerance%20to%20Aluminium%20toxicity%20in%20Tanzanian%20sorghum%20genotypes&amp;author=JH.%20Ringo&amp;author=EE.%20Mneney&amp;author=AO.%20Onkware&amp;author=BA.%20Were&amp;author=EJ.%20Too&amp;author=JO.%20Owuoche&amp;author=SO.%20Gudu&amp;journal=Afr%20Crop%20Sci%20J&amp;volume=18&amp;pages=155-164&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR30" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Bekele E, Geleta M, Dagne K, Jones AL, Barnes I, Bradman N, Thomas MG. Molecular phylogeny of genus <em class="EmphasisTypeItalic">Guizotia</em> (Asteraceae) using DNA sequences derived from ITS. Genet Resour Crop Evol. 2007;54:1419–27.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s10722-006-9126-0"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Molecular%20phylogeny%20of%20genus%20Guizotia%20%28Asteraceae%29%20using%20DNA%20sequences%20derived%20from%20ITS&amp;author=E.%20Bekele&amp;author=M.%20Geleta&amp;author=K.%20Dagne&amp;author=AL.%20Jones&amp;author=I.%20Barnes&amp;author=N.%20Bradman&amp;author=MG.%20Thomas&amp;journal=Genet%20Resour%20Crop%20Evol&amp;volume=54&amp;pages=1419-1427&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR31" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Don R, Cox P, Wainwright B, Baker K, Mattick J. “Touchdown” PCR to circumvent spurious priming during gene amplification". Nucleic Acids Res 1991;19:4008.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Don%20R%2C%20Cox%20P%2C%20Wainwright%20B%2C%20Baker%20K%2C%20Mattick%20J.%20%E2%80%9CTouchdown%E2%80%9D%20PCR%20to%20circumvent%20spurious%20priming%20during%20gene%20amplification%22.%20Nucleic%20Acids%20Res%201991%3B19%3A4008."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR32" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Geleta M, Bryngelsson T. Inter simple sequence repeats (ISSR) analysis of genetic diversity and population genetic structure of Lobelia rhynchopetalum (Hochest.) Hemsel. (Campanulaceae). Hereditas 2009;146:122–130.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Geleta%20M%2C%20Bryngelsson%20T.%20Inter%20simple%20sequence%20repeats%20%28ISSR%29%20analysis%20of%20genetic%20diversity%20and%20population%20genetic%20structure%20of%20Lobelia%20rhynchopetalum%20%28Hochest.%29%20Hemsel.%20%28Campanulaceae%29.%20Hereditas%202009%3B146%3A122%E2%80%93130."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR33" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sanguinetti CJ, Dias NE, Simpson AJ. Rapid silver staining and recovery of PCR products separated on polyacrylamide gels. BioTechniques. 1994;17:914–21.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=7840973"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Rapid%20silver%20staining%20and%20recovery%20of%20PCR%20products%20separated%20on%20polyacrylamide%20gels&amp;author=CJ.%20Sanguinetti&amp;author=NE.%20Dias&amp;author=AJ.%20Simpson&amp;journal=BioTechniques&amp;volume=17&amp;pages=914-921&amp;publication_year=1994"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR34" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Larkin MA, Blackshields G, Brown NP. Clustal W and Clustal X version 2.0. Bioinformatics. 2007;23:2947–8.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btm404"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17846036"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Clustal%20W%20and%20Clustal%20X%20version%202.0&amp;author=MA.%20Larkin&amp;author=G.%20Blackshields&amp;author=NP.%20Brown&amp;journal=Bioinformatics&amp;volume=23&amp;pages=2947-2948&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR35" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hall TA. BioEdit: a user-friendly biological sequence alignment editor and analysis programme of windows 95/98/NT. Nucl Acids Symp Ser. 1999;41:95–8.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=BioEdit%3A%20a%20user-friendly%20biological%20sequence%20alignment%20editor%20and%20analysis%20programme%20of%20windows%2095%2F98%2FNT&amp;author=TA.%20Hall&amp;journal=Nucl%20Acids%20Symp%20Ser&amp;volume=41&amp;pages=95-98&amp;publication_year=1999"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR36" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Miftahudin G, Scoles J, Gustaffson JP. AFLP markers tightly linked to the aluminium-resistance gene Alt3 in rye (Secale cereale L.). Theor Appl Genet 2002;104:626–631.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Miftahudin%20G%2C%20Scoles%20J%2C%20Gustaffson%20JP.%20AFLP%20markers%20tightly%20linked%20to%20the%20aluminium-resistance%20gene%20Alt3%20in%20rye%20%28Secale%20cereale%20L.%29.%20Theor%20Appl%20Genet%202002%3B104%3A626%E2%80%93631."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR37" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gallego FJ, Calles B, Benito C. Molecular markers linked to the aluminium resistance gene Alt1 in rye (Secale cereale, L.). Theor Appl Genet 1998;97:1104–1109.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Gallego%20FJ%2C%20Calles%20B%2C%20Benito%20C.%20Molecular%20markers%20linked%20to%20the%20aluminium%20resistance%20gene%20Alt1%20in%20rye%20%28Secale%20cereale%2C%20L.%29.%20Theor%20Appl%20Genet%201998%3B97%3A1104%E2%80%931109."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR38" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ninamango-Cárdenas FE, Guimarães C, Martins P, Parentoni S, Carneiro N, Lopes M, Moro J, Paiva E. Mapping QTLs for aluminium resistance in maize. Euphytica. 2003;130:223–32.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1023/A%3A1022867416513"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Mapping%20QTLs%20for%20aluminium%20resistance%20in%20maize&amp;author=FE.%20Ninamango-C%C3%A1rdenas&amp;author=C.%20Guimar%C3%A3es&amp;author=P.%20Martins&amp;author=S.%20Parentoni&amp;author=N.%20Carneiro&amp;author=M.%20Lopes&amp;author=J.%20Moro&amp;author=E.%20Paiva&amp;journal=Euphytica&amp;volume=130&amp;pages=223-232&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR39" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Raman H, Moroni JS, Sato K, Read BJ, Scott BJ. Identification of AFLP and microsatellite markers linked with aluminium resistance gene in barley (<em class="EmphasisTypeItalic">Hordeum vulgare</em>, L). Theor Appl Genet. 2002;105:458–64.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00122-002-0934-0"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12582551"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Identification%20of%20AFLP%20and%20microsatellite%20markers%20linked%20with%20aluminium%20resistance%20gene%20in%20barley%20%28Hordeum%20vulgare%2C%20L%29&amp;author=H.%20Raman&amp;author=JS.%20Moroni&amp;author=K.%20Sato&amp;author=BJ.%20Read&amp;author=BJ.%20Scott&amp;journal=Theor%20Appl%20Genet&amp;volume=105&amp;pages=458-464&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR40" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Huang CF, Yamaji N, Mitani N, Yano M, Nagamura Y, Ma JF. A bacterial-type ABC transporter is involved in aluminum tolerance in rice. Plant Cell. 2009;21:655–67.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1105/tpc.108.064543"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19244140"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2660611"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20bacterial-type%20ABC%20transporter%20is%20involved%20in%20aluminum%20tolerance%20in%20rice&amp;author=CF.%20Huang&amp;author=N.%20Yamaji&amp;author=N.%20Mitani&amp;author=M.%20Yano&amp;author=Y.%20Nagamura&amp;author=JF.%20Ma&amp;journal=Plant%20Cell&amp;volume=21&amp;pages=655-667&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR41" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yamaji N, Huang CF, Nagao S, Yano M, Sato Y, Nagamura Y, Ma JF. A zinc finger transcription factor ART1 regulates multiple genes implicated in aluminum tolerance in rice. Plant Cell. 2009;21:3339–49.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1105/tpc.109.070771"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19880795"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2782276"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20zinc%20finger%20transcription%20factor%20ART1%20regulates%20multiple%20genes%20implicated%20in%20aluminum%20tolerance%20in%20rice&amp;author=N.%20Yamaji&amp;author=CF.%20Huang&amp;author=S.%20Nagao&amp;author=M.%20Yano&amp;author=Y.%20Sato&amp;author=Y.%20Nagamura&amp;author=JF.%20Ma&amp;journal=Plant%20Cell&amp;volume=21&amp;pages=3339-3349&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR42" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gupta PK, Varshney RK. The development and use of microsatellite markers for genetic analysis and plant breeding with emphasis on bread wheat. Euphytica. 2000;113:163–85.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1023/A%3A1003910819967"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20development%20and%20use%20of%20microsatellite%20markers%20for%20genetic%20analysis%20and%20plant%20breeding%20with%20emphasis%20on%20bread%20wheat&amp;author=PK.%20Gupta&amp;author=RK.%20Varshney&amp;journal=Euphytica&amp;volume=113&amp;pages=163-185&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR43" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Matos M, Pérez-Flores V, Camacho MV, Pernaute B, Pinto-Carnide O, Benito C. Detection and mapping of SSRs in rye ESTs from aluminium-stressed roots. Mol Breed. 2007;20:103–15.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11032-007-9077-y"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Detection%20and%20mapping%20of%20SSRs%20in%20rye%20ESTs%20from%20aluminium-stressed%20roots&amp;author=M.%20Matos&amp;author=V.%20P%C3%A9rez-Flores&amp;author=MV.%20Camacho&amp;author=B.%20Pernaute&amp;author=O.%20Pinto-Carnide&amp;author=C.%20Benito&amp;journal=Mol%20Breed&amp;volume=20&amp;pages=103-115&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR44" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Menz MA, Klein RR, Mullet JE, Obert JA, Unruh NC, Klein PE. A high density genetic map of <em class="EmphasisTypeItalic">Sorghum bicolor</em> (L) Moench based on 2926 AFLP, RFLP and SSR markers. Plant Mol Biol. 2002;48:483–99.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1023/A%3A1014831302392"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11999830"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20high%20density%20genetic%20map%20of%20Sorghum%20bicolor%20%28L%29%20Moench%20based%20on%202926%20AFLP%2C%20RFLP%20and%20SSR%20markers&amp;author=MA.%20Menz&amp;author=RR.%20Klein&amp;author=JE.%20Mullet&amp;author=JA.%20Obert&amp;author=NC.%20Unruh&amp;author=PE.%20Klein&amp;journal=Plant%20Mol%20Biol&amp;volume=48&amp;pages=483-499&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR45" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Tokizawa M, Kobayashi Y, Saito T, Kobayashi M, Iuchi S, et al. Sensitive to proton Rhizotoxicity1, calmodulin binding transcription activator 2, and other transcription factors are involved in aluminum-activated malate Transporter1 expression. Plant Physiol. 2015;167:991–1003.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1104/pp.114.256552"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25627216"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4348791"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Sensitive%20to%20proton%20Rhizotoxicity1%2C%20calmodulin%20binding%20transcription%20activator%202%2C%20and%20other%20transcription%20factors%20are%20involved%20in%20aluminum-activated%20malate%20Transporter1%20expression&amp;author=M.%20Tokizawa&amp;author=Y.%20Kobayashi&amp;author=T.%20Saito&amp;author=M.%20Kobayashi&amp;author=S.%20Iuchi&amp;journal=Plant%20Physiol&amp;volume=167&amp;pages=991-1003&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR46" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Naranjo T, Fernández-Rueda P. Homoeology of rye chromosome arms to wheat. Theor Appl Genet. 1991;82:577–86.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/BF00226793"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24213336"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Homoeology%20of%20rye%20chromosome%20arms%20to%20wheat&amp;author=T.%20Naranjo&amp;author=P.%20Fern%C3%A1ndez-Rueda&amp;journal=Theor%20Appl%20Genet&amp;volume=82&amp;pages=577-586&amp;publication_year=1991"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR47" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Rognli OA, Devos KM, Chino CN, Harcourt RL, Atkinson MD, Gale MD. RFLP mapping of rye chromosome 7R reveals a highly translocated chromosome relative to wheat. Genome. 1992;35:1026–31.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1139/g92-157"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=RFLP%20mapping%20of%20rye%20chromosome%207R%20reveals%20a%20highly%20translocated%20chromosome%20relative%20to%20wheat&amp;author=OA.%20Rognli&amp;author=KM.%20Devos&amp;author=CN.%20Chino&amp;author=RL.%20Harcourt&amp;author=MD.%20Atkinson&amp;author=MD.%20Gale&amp;journal=Genome&amp;volume=35&amp;pages=1026-1031&amp;publication_year=1992"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR48" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Devos KM, Atkinson MD, Chinoy CN, Francis HA, Harcourt RL, Koebner RMD, Liu CJ, Masojc P, Xie DX, Gale MD. Chromosomal rearrangements in the rye genome relative to that of wheat. Theor Appl Genet. 1993;85:673–80.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/BF00225004"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24196035"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Chromosomal%20rearrangements%20in%20the%20rye%20genome%20relative%20to%20that%20of%20wheat&amp;author=KM.%20Devos&amp;author=MD.%20Atkinson&amp;author=CN.%20Chinoy&amp;author=HA.%20Francis&amp;author=RL.%20Harcourt&amp;author=RMD.%20Koebner&amp;author=CJ.%20Liu&amp;author=P.%20Masojc&amp;author=DX.%20Xie&amp;author=MD.%20Gale&amp;journal=Theor%20Appl%20Genet&amp;volume=85&amp;pages=673-680&amp;publication_year=1993"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR49" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Naranjo T, Fernández-Rueda P, Maestra B. Chromosome rearrangements and homoeologous pairing: implications for the introgression of alien genes into wheat. In: Current topics in plant cytogenetics related to plant improvement Lelley T, editor. WUV-Univeitätsverlag, Austria;1997. p. 198–205.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Naranjo%20T%2C%20Fern%C3%A1ndez-Rueda%20P%2C%20Maestra%20B.%20Chromosome%20rearrangements%20and%20homoeologous%20pairing%3A%20implications%20for%20the%20introgression%20of%20alien%20genes%20into%20wheat.%20In%3A%20Current%20topics%20in%20plant%20cytogenetics%20related%20to%20plant%20improvement%20Lelley%20T%2C%20editor.%20WUV-Univeit%C3%A4tsverlag%2C%20Austria%3B1997.%20p.%20198%E2%80%93205."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR50" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gale MD, Devos KM. Comparative genetics in the grasses. Proc Natl Acad Sci U S A. 1998;95:1971–4.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.95.5.1971"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9482816"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC33824"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparative%20genetics%20in%20the%20grasses&amp;author=MD.%20Gale&amp;author=KM.%20Devos&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;volume=95&amp;pages=1971-1974&amp;publication_year=1998"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR51" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Cheprot RK, Matonyei TK, Maritim KK, Were BA, Dangasuk OG, Onkware AO, Gudu S. Physiological characterization of Kenyan sorghum lines for tolerance to aluminium. Int J Nat Sci Res. 2014;2:59–71.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Physiological%20characterization%20of%20Kenyan%20sorghum%20lines%20for%20tolerance%20to%20aluminium&amp;author=RK.%20Cheprot&amp;author=TK.%20Matonyei&amp;author=KK.%20Maritim&amp;author=BA.%20Were&amp;author=OG.%20Dangasuk&amp;author=AO.%20Onkware&amp;author=S.%20Gudu&amp;journal=Int%20J%20Nat%20Sci%20Res&amp;volume=2&amp;pages=59-71&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR52" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Fontecha G, Silva-Navas J, Benito C, Metres MA, Espino FJ, Hernandez-Riquer MV, Gallego FJ. Candidate gene identification of an aluminum-activated organic acid transporter gene at the Alt4 locus for aluminum resistance in rye (Secale cereale, L.). Theor Appl Genet 2007;114:249–260.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Fontecha%20G%2C%20Silva-Navas%20J%2C%20Benito%20C%2C%20Metres%20MA%2C%20Espino%20FJ%2C%20Hernandez-Riquer%20MV%2C%20Gallego%20FJ.%20Candidate%20gene%20identification%20of%20an%20aluminum-activated%20organic%20acid%20transporter%20gene%20at%20the%20Alt4%20locus%20for%20aluminum%20resistance%20in%20rye%20%28Secale%20cereale%2C%20L.%29.%20Theor%20Appl%20Genet%202007%3B114%3A249%E2%80%93260."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR53" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Collins NC, Shirley NJ, Saeed M, Pallotta M, Gustafson JP. An ALMTI gene cluster controlling aluminum resistance at the Atl4 locus of rye (Secale cereale L.). Genetics 2008;179:669–682.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Collins%20NC%2C%20Shirley%20NJ%2C%20Saeed%20M%2C%20Pallotta%20M%2C%20Gustafson%20JP.%20An%20ALMTI%20gene%20cluster%20controlling%20aluminum%20resistance%20at%20the%20Atl4%20locus%20of%20rye%20%28Secale%20cereale%20L.%29.%20Genetics%202008%3B179%3A669%E2%80%93682."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR54" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Smith JSC, Kresovich S, Hopkins MS, Mitchel SE, Dean RE, Woodman WL, Lee M, Porter K. Genetic diversity among elite sorghum inbred lines assessed with simple sequence repeats. Crop Sci. 2000;40:226–32.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2135/cropsci2000.401226x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20among%20elite%20sorghum%20inbred%20lines%20assessed%20with%20simple%20sequence%20repeats&amp;author=JSC.%20Smith&amp;author=S.%20Kresovich&amp;author=MS.%20Hopkins&amp;author=SE.%20Mitchel&amp;author=RE.%20Dean&amp;author=WL.%20Woodman&amp;author=M.%20Lee&amp;author=K.%20Porter&amp;journal=Crop%20Sci&amp;volume=40&amp;pages=226-232&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR55" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kong L, Dong J, Hart GE. Characteristics, linkage-map positions, and allelic differentiation of Sorghum bicolor (L) Moech DNA simple sequence repeats (SSRs). Theor Appl Genet 2000;101:438–448.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Kong%20L%2C%20Dong%20J%2C%20Hart%20GE.%20Characteristics%2C%20linkage-map%20positions%2C%20and%20allelic%20differentiation%20of%20Sorghum%20bicolor%20%28L%29%20Moech%20DNA%20simple%20sequence%20repeats%20%28SSRs%29.%20Theor%20Appl%20Genet%202000%3B101%3A438%E2%80%93448."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li></ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#TheAuthors" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="TheAuthors">Copyright</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="TheAuthors"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">©&nbsp;The Author(s)&nbsp;2018</div></div></section></div>
                        


                    </article>
                </main>

                <div class="c-content-layout__side u-text-sm">
                    <aside>
                        <div class="p-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="//hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-018-0059-3" data-track="click" data-track-category="PDF Download"
            data-track-action="Click download article PDF"
            data-track-label="10.1186/s41065-018-0059-3">Download PDF</a>
        </p>
    
                            
                            
    <button type="button"
       class="c-button c-button--secondary c-button--full-width c-button--expandable"
       data-toggle="collapse"
       data-target="export-citations"
       data-test="export-citations-button"
       data-sticky-update="true"
       aria-controls="export-citations"
       aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/s41065-018-0059-3"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0059-3?format=refman&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/s41065-018-0059-3"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0059-3?format=refman&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/s41065-018-0059-3"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0059-3?format=refman&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/s41065-018-0059-3"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0059-3?format=endnote&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/s41065-018-0059-3"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0059-3?format=endnote&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/s41065-018-0059-3"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0059-3?format=endnote&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/s41065-018-0059-3"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0059-3?format=bibtex&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/s41065-018-0059-3"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0059-3?format=bibtex&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/s41065-018-0059-3"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0059-3?format=bibtex&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
    </div>

                        </div>
                        
    
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
                                <div id="article-access-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/access-count/10.1186/s41065-018-0059-3 #accessCount-container"></div>
                                <div id="citation-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/citation-count/10.1186/s41065-018-0059-3 #count-container"></div>
                                <div id="altmetric-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/altmetric/10.1186/s41065-018-0059-3 #altmetric-container"></div>
                            </ul>
                        </div>
                        
    <div id="sociaMediaShare" class="p-separator u-js-shown">
        <h3 class="p-section-title">Share this article</h3>
        <ul id="shareButtons" class="c-social-links">
            <li class="c-social-links__item">
                <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-018-0059-3">
                    <span class="u-visually-hidden">Share on Twitter</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-twitter"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-018-0059-3">
                    <span class="u-visually-hidden">Share on Facebook</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-facebook"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click"
                        data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-018-0059-3">
                    <span class="u-visually-hidden">Share on LinkedIn</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-linkedin"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-018-0059-3">
                    <span class="u-visually-hidden">Share on Weibo</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-weibo"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnGooglePlus" class="js-btnShareOnGplus" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Google+" data-track-label="10.1186/s41065-018-0059-3">
                    <span class="u-visually-hidden">Share on Google Plus</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-gplus"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-018-0059-3">
                    <span class="u-visually-hidden">Share on Reddit</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-reddit"></use>
                    </svg>
                </button>
            </li>
        </ul>
    </div>
 
                        
    
        <div class="p-separator u-js-shown">
            <h3 class="p-section-title">See updates</h3>
            <button class="c-crossmark-button" data-target="crossmark" data-test="crossmark" data-track="click" data-track-category="Article Page" data-track-action="Click CrossMark" data-track-label="10.1186/s41065-018-0059-3">
                <img id="crossmark-icon" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" alt="Crossmark logo" />
            </button>
        </div>
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
                                <li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/s41065-018-0059-3" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=738&doi=10.1186%2Fs41065-018-0059-3&volume=155&issue=1&title=Molecular+markers+associated+with+aluminium+tolerance+in+Sorghum+bicolor&author_name=Emily+Jepkosgei+Too&start_page=1&end_page=13">Order reprint</a>

                                </li>
                            </ul>
                        </div>
                    </aside>
                    
    <div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">Hereditas</span>
                        </h4>
                        <p class="c-journal-footer__issn">ISSN: 1601-5223</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
                                
                                    <li class="c-journal-footer__contact-item">Submission enquiries: <a href="https://www.editorialmanager.com/here/" target="_blank">Access here and click Contact Us</a></li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a></li>
                                
                            </ul>
                        </div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class='tracker' style='display:none' src='/track/article/10.1186/s41065-018-0059-3' alt=""/>

         
    <footer>
        <div class="ctx-publisher-footer">
    <div class="c-navbar">
        <div class="u-container">
            <div class="c-navbar__logo u-margin-right-xl">
                <a href="https://www.biomedcentral.com">
                    <img src=/static/images/bmc/logos/logo-bmc-aj-594e03e20d.svg alt="BMC logo" data-test="navbar-logo-footer">

                </a>
            </div>
            <div class="c-navbar__content">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


            </div>
        </div>
    </div>
    <div class="c-publisher-footer" data-test="publisher-footer">
        <div class="u-container">
            
            <div class="c-flex c-flex--wrap c-flex--space-between" role="navigation" data-test="publisher-footer-menu">
                <div class="c-flex" aria-label="Publisher footer links">
                    
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/login">Receive BMC newsletters</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/account">Manage article alerts</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/policies">Policies</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/accessibility">Accessibility</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/press-centre">Press center</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/contact-us">Contact us</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/jobs">Careers</a>
                                    </li>
                                
                            </ul>
                        
                    
                </div>
                <div class="u-margin-bottom-lg">
                    <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                    <ul class="c-flex u-reset-list" data-test="footer-social-links">
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://twitter.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Twitter link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Twitter page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-twitter-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://www.facebook.com/BioMedCentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Facebook link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Facebook page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-facebook-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://plus.google.com/106226337984152901734/"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Google Plus link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Google Plus page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-gplus-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="http://www.weibo.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Weibo link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Weibo page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-weibo-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </div>
            <p class="u-reset-margin">
                By using this website, you agree to our
                <a class="u-text-inherit" href="//www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
                <a class="u-text-inherit" href="//www.biomedcentral.com/privacy-statement">Privacy
                    statement</a> and
                <a class="u-text-inherit" href="//www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
                
                    <a class="optanon-toggle-display u-text-inherit" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
                
            </p>
        </div>
    </div>
</div>


        <div class="c-sn-footer c-sn-footer--neutral">
    <div class="u-container">
        <img src=/static/images/bmc/logos/logo-springernature-8bd4e8d82f.svg class="c-sn-footer__logo" alt="Springer Nature Logo" itemprop="logo" >
        <p data-test="copyright">&#169; 2018 BioMed Central Ltd unless otherwise stated. Part of
            <a href="http://www.springernature.com"
               data-track="click"
               data-track-action="Click Springer Nature link"
               data-track-category="Corporate footer"
               itemscope itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>

    </div>

    
    <script type="text/javascript" src="//cdn.pbgrd.com/bmc.js" async="async"></script>
    <noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>
    <!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker -->

<script id="springerlink-event-tracker" src="https://event-tracker.springernature.com/dist/eventTracker.js"></script>
<script type="text/javascript">
    (function(){
        var dataLayer = window.dataLayer[0];
        var doi = dataLayer.content.article.doi;
        var imprint = dataLayer.content.contentInfo.imprint;
        var bpids = dataLayer.session.authentication.authenticationID;
        var publisher;
        var pageType = dataLayer.page.category.pageType;
        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }
        var eventObject = {'content_type' : pageType,'doi' : doi};

        if (imprint === 'BioMed Central') { publisher = 'BMC' }
        else if (imprint === 'SpringerOpen') { publisher = 'SpringerOpen' }
        else { throw new Error('Event Tracker Error: The publisher has an unexpected value') }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }
        new EventTracker({'platform':publisher}).sendEvent('display', eventObject);
    })();
</script>


    </body>
</html>

