<!DOCTYPE html>
<html lang="en" class="no-js">
    <head>
        <title>Genome-wide association study of rust traits in orchardgrass using SLAF-seq technology | Hereditas | Full Text</title>

        

    <meta name="citation_abstract" content="While orchardgrass (Dactylis glomerata L.) is a well-known perennial forage species, rust diseases cause serious reductions in the yield and quality of orchardgrass; however, genetic mechanisms of rust resistance are not well understood in orchardgrass. In this study, a genome-wide association study (GWAS) was performed using specific-locus amplified fragment sequencing (SLAF-seq) technology in orchardgrass. A total of 2,334,889 SLAF tags were generated to produce 2,309,777 SNPs. ADMIXTURE analysis revealed unstructured subpopulations for 33 accessions, indicating that this orchardgrass population could be used for association analysis. Linkage disequilibrium (LD) analysis revealed an average r2 of 0.4 across all SNP pairs, indicating a high extent of LD in these samples. Through GWAS, a total of 4,604 SNPs were found to be significantly (P < 0.01) associated with the rust trait. The bulk analysis discovered a number of 5,211 SNPs related to rust trait. Two candidate genes, including cytochrome P450, and prolamin were implicated in disease resistance through prediction of functional genes surrounding each high-quality SNP (P < 0.01) associated with rust traits based on GWAS analysis and bulk analysis The large number of SNPs associated with rust traits and these two candidate genes may provide the basis for further research on rust resistance mechanisms and marker-assisted selection (MAS) for rust-resistant lineages."/>

    <meta name="description" content="While orchardgrass (Dactylis glomerata L.) is a well-known perennial forage species, rust diseases cause serious reductions in the yield and quality of orchardgrass; however, genetic mechanisms of rust resistance are not well understood in orchardgrass. In this study, a genome-wide association study (GWAS) was performed using specific-locus amplified fragment sequencing (SLAF-seq) technology in orchardgrass. A total of 2,334,889 SLAF tags were generated to produce 2,309,777 SNPs. ADMIXTURE analysis revealed unstructured subpopulations for 33 accessions, indicating that this orchardgrass population could be used for association analysis. Linkage disequilibrium (LD) analysis revealed an average r2 of 0.4 across all SNP pairs, indicating a high extent of LD in these samples. Through GWAS, a total of 4,604 SNPs were found to be significantly (P < 0.01) associated with the rust trait. The bulk analysis discovered a number of 5,211 SNPs related to rust trait. Two candidate genes, including cytochrome P450, and prolamin were implicated in disease resistance through prediction of functional genes surrounding each high-quality SNP (P < 0.01) associated with rust traits based on GWAS analysis and bulk analysis The large number of SNPs associated with rust traits and these two candidate genes may provide the basis for further research on rust resistance mechanisms and marker-assisted selection (MAS) for rust-resistant lineages."/>

    <meta name="journal_id" content="41065"/>

    <meta name="dc.title" content="Genome-wide association study of rust traits in orchardgrass using SLAF-seq technology"/>

    <meta name="dc.source" content="Hereditas 2017 154:1"/>

    <meta name="dc.format" content="text/html"/>

    <meta name="dc.publisher" content="BioMed Central"/>

    <meta name="dc.date" content="2017-02-23"/>

    <meta name="dc.type" content="OriginalPaper"/>

    <meta name="dc.language" content="En"/>

    <meta name="dc.copyright" content="2017 The Author(s)"/>

    <meta name="dc.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="dc.description" content="While orchardgrass (Dactylis glomerata L.) is a well-known perennial forage species, rust diseases cause serious reductions in the yield and quality of orchardgrass; however, genetic mechanisms of rust resistance are not well understood in orchardgrass. In this study, a genome-wide association study (GWAS) was performed using specific-locus amplified fragment sequencing (SLAF-seq) technology in orchardgrass. A total of 2,334,889 SLAF tags were generated to produce 2,309,777 SNPs. ADMIXTURE analysis revealed unstructured subpopulations for 33 accessions, indicating that this orchardgrass population could be used for association analysis. Linkage disequilibrium (LD) analysis revealed an average r2 of 0.4 across all SNP pairs, indicating a high extent of LD in these samples. Through GWAS, a total of 4,604 SNPs were found to be significantly (P&#8201;&lt;&#8201;0.01) associated with the rust trait. The bulk analysis discovered a number of 5,211 SNPs related to rust trait. Two candidate genes, including cytochrome P450, and prolamin were implicated in disease resistance through prediction of functional genes surrounding each high-quality SNP (P&#8201;&lt;&#8201;0.01) associated with rust traits based on GWAS analysis and bulk analysis The large number of SNPs associated with rust traits and these two candidate genes may provide the basis for further research on rust resistance mechanisms and marker-assisted selection (MAS) for rust-resistant lineages."/>

    <meta name="prism.issn" content="1601-5223"/>

    <meta name="prism.publicationName" content="Hereditas"/>

    <meta name="prism.publicationDate" content="2017-02-23"/>

    <meta name="prism.volume" content="154"/>

    <meta name="prism.number" content="1"/>

    <meta name="prism.section" content="OriginalPaper"/>

    <meta name="prism.startingPage" content="5"/>

    <meta name="prism.copyright" content="2017 The Author(s)"/>

    <meta name="prism.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="prism.url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0027-3"/>

    <meta name="prism.doi" content="doi:10.1186/s41065-017-0027-3"/>

    <meta name="citation_pdf_url" content="https://hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-017-0027-3"/>

    <meta name="citation_fulltext_html_url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0027-3"/>

    <meta name="citation_journal_title" content="Hereditas"/>

    <meta name="citation_publisher" content="BioMed Central"/>

    <meta name="citation_issn" content="1601-5223"/>

    <meta name="citation_title" content="Genome-wide association study of rust traits in orchardgrass using SLAF-seq technology"/>

    <meta name="citation_volume" content="154"/>

    <meta name="citation_issue" content="1"/>

    <meta name="citation_publication_date" content="2017/12"/>

    <meta name="citation_online_date" content="2017/02/23"/>

    <meta name="citation_firstpage" content="5"/>

    <meta name="citation_article_type" content="Research"/>

    <meta name="citation_fulltext_world_readable" content=""/>

    <meta name="dc.identifier" content="doi:10.1186/s41065-017-0027-3"/>

    <meta name="DOI" content="10.1186/s41065-017-0027-3"/>

    <meta name="citation_doi" content="10.1186/s41065-017-0027-3"/>

    <meta name="dc.creator" content="Bing Zeng"/>

    <meta name="dc.creator" content="Haidong Yan"/>

    <meta name="dc.creator" content="Xinchun Liu"/>

    <meta name="dc.creator" content="Wenjing Zang"/>

    <meta name="dc.creator" content="Ailing Zhang"/>

    <meta name="dc.creator" content="Sifan Zhou"/>

    <meta name="dc.creator" content="Linkai Huang"/>

    <meta name="dc.creator" content="Jinping Liu"/>

    <meta name="dc.subject" content="Animal Genetics and Genomics"/>

    <meta name="dc.subject" content="Evolutionary Biology"/>

    <meta name="citation_reference" content="citation_title=The genus Dactylis; wealth of wild species: role in plant genome elucidation and improvement; citation_publication_date=2010; citation_id=CR1; citation_author=AV Stewart; citation_author=NW Ellison; citation_publisher=Springer"/>

    <meta name="citation_reference" content="citation_journal_title=Biochem Syst Ecol; citation_title=Genetic diversity analysis and transferability of cereal EST-SSR markers to orchardgrass (Dactylis glomerata L.); citation_author=WG Xie, XQ Zhang, HW Cai, W Liu, Y Peng; citation_volume=38; citation_issue=4; citation_publication_date=2010; citation_pages=740-9; citation_doi=10.1016/j.bse.2010.06.009; citation_id=CR2"/>

    <meta name="citation_reference" content="citation_journal_title=Biochem Syst Ecol; citation_title=Genetic diversity of orchardgrass (Dactylis glomerata L.) germplasms with resistance to rust diseases revealed by Start Codon Targeted (SCoT) markers; citation_author=B Zeng, Y Zhang, LK Huang, XM Jiang, D Luo, GH Yin; citation_volume=54; citation_publication_date=2014; citation_pages=96-102; citation_doi=10.1016/j.bse.2013.12.028; citation_id=CR3"/>

    <meta name="citation_reference" content="citation_journal_title=J Jpan Grassl Sci; citation_title=Differentiation and maintenance of three graminicolous formae speciales of stem rust with clonal lines of orchardgrass, Dactylis glomerata L; citation_author=A Tajimi; citation_volume=20; citation_publication_date=1975; citation_pages=26-30; citation_id=CR4"/>

    <meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=Breeding for rust resistance in orchardgrass by phenotypic and phenotypic-genotypic selection; citation_author=TL Miller, IT Carlson; citation_volume=22; citation_issue=6; citation_publication_date=1982; citation_pages=1218-21; citation_doi=10.2135/cropsci1982.0011183X002200060031x; citation_id=CR5"/>

    <meta name="citation_reference" content="citation_journal_title=Grassl Sci; citation_title=Screening orchardgrass (Dactylis glomerata L.) germplasm for high rust resistance and high summer survival rate in a subtropical climate; citation_author=HD Yan, B Zeng, XQ Zhang, L Cheng, S Miller, LK Huang; citation_volume=59; citation_issue=4; citation_publication_date=2013; citation_pages=205-10; citation_doi=10.1111/grs.12030; citation_id=CR6"/>

    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Natural allelic variation underlying a major fitness trade-off in Arabidopsis thaliana
                           ; citation_author=M Todesco, S Balasubramanian, TT Hu, MB Traw, M Horton, P Epple; citation_volume=465; citation_issue=7298; citation_publication_date=2010; citation_pages=632-6; citation_doi=10.1038/nature09083; citation_id=CR7"/>

    <meta name="citation_reference" content="citation_journal_title=PNAS; citation_title=Association mapping of local climate-sensitive quantitative trait loci in Arabidopsis thaliana
                           ; citation_author=Y Li, Y Huang, J Bergelson, M Nordborg, JO Borevitz; citation_volume=107; citation_issue=49; citation_publication_date=2010; citation_pages=21199-204; citation_doi=10.1073/pnas.1007431107; citation_id=CR8"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Genet; citation_title=Genome-wide association study of leaf architecture in the maize nested association mapping population; citation_author=F Tian, PJ Bradbury, PJ Brown, H Hung, Q Sun, S Flint-Garcia; citation_volume=43; citation_issue=2; citation_publication_date=2011; citation_pages=159-62; citation_doi=10.1038/ng.746; citation_id=CR9"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=Whole genome association mapping of Fusarium head blight resistance in European winter wheat (Triticum aestivum L.); citation_author=S Kollers, B Rodemann, J Ling, V Korzun, E Ebmeyer, O Argillier; citation_volume=8; citation_issue=2; citation_publication_date=2013; citation_doi=10.1371/journal.pone.0057500; citation_id=CR10"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=Genome-wide association mapping for yield and other agronomic traits in an elite breeding population of tropical rice (Oryza sativa); citation_author=H Begum, JE Spindel, A Lalusin, T Borromeo, G Gregorio, J Hernandez; citation_volume=10; citation_issue=3; citation_publication_date=2015; citation_doi=10.1371/journal.pone.0119873; citation_id=CR11"/>

    <meta name="citation_reference" content="citation_journal_title=DNA Res; citation_title=Genome-wide association mapping of salinity tolerance in rice (Oryza sativa); citation_author=V Kumar, A Singh, SVA Mithra, SL Krishnamurthy, SK Parida, S Jain; citation_volume=22; citation_issue=2; citation_publication_date=2015; citation_pages=133-45; citation_doi=10.1093/dnares/dsu046; citation_id=CR12"/>

    <meta name="citation_reference" content="Yang WN, Guo ZL, Huang CL, Duan LF, Chen GX, Jiang N, et al. Combining high-throughput phenotyping and genome-wide association studies to reveal natural genetic variation in rice. Nat Commun. 2014;5."/>

    <meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=Association mapping reveals gene action and interactions in the determination of flowering time in barley; citation_author=S Stracke, G Haseneyer, J-B Veyrieras, HH Geiger, S Sauer, A Graner; citation_volume=118; citation_issue=2; citation_publication_date=2009; citation_pages=259-73; citation_doi=10.1007/s00122-008-0896-y; citation_id=CR14"/>

    <meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=Allelic variation at Psy1-A1 and association with yellow pigment in durum wheat grain; citation_author=A Singh, S Reimer, CJ Pozniak, FR Clarke, JM Clarke, RE Knox; citation_volume=118; citation_issue=8; citation_publication_date=2009; citation_pages=1539-48; citation_doi=10.1007/s00122-009-1001-x; citation_id=CR15"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Breed; citation_title=Development of SNP markers and haplotype analysis of the candidate gene for rhg1, which confers resistance to soybean cyst nematode in soybean; citation_author=YH Li, C Zhang, ZS Gao, MJM Smulders, Z Ma, ZX Liu; citation_volume=24; citation_issue=1; citation_publication_date=2009; citation_pages=63-76; citation_doi=10.1007/s11032-009-9272-0; citation_id=CR16"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Plant Biol; citation_title=Genome-wide association studies for agronomical traits in a world wide spring barley collection; citation_author=RK Pasam, R Sharma, M Malosetti, FA Eeuwijk, G Haseneyer, B Kilian; citation_volume=12; citation_issue=1; citation_publication_date=2012; citation_pages=16; citation_doi=10.1186/1471-2229-12-16; citation_id=CR17"/>

    <meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=Identification of Ug99 stem rust resistance loci in winter wheat germplasm using genome-wide association analysis; citation_author=LX Yu, A Morgounov, R Wanyera, M Keser, SK Singh, M Sorrells; citation_volume=125; citation_issue=4; citation_publication_date=2012; citation_pages=749-58; citation_doi=10.1007/s00122-012-1867-x; citation_id=CR18"/>

    <meta name="citation_reference" content="citation_journal_title=Front Plant Sci; citation_title=Genome-wide association study for crown rust (Puccinia coronata f. sp. avenae) and powdery mildew (Blumeria graminis f. sp. avenae) resistance in an oat (Avena sativa) collection of commercial varieties and landraces; citation_author=G Montilla-Basc&#243;n, N Rispail, J S&#225;nchez-Mart&#237;n, D Rubiales, LAJ Mur, T Langdon; citation_volume=6; citation_publication_date=2015; citation_pages=1-11; citation_doi=10.3389/fpls.2015.00103; citation_id=CR19"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=A high-density genetic map for soybean based on specific length amplified fragment sequencing; citation_author=ZM Qi, L Huang, RS Zhu, DW Xin, CY Liu, X Han; citation_volume=9; citation_issue=11; citation_publication_date=2014; citation_doi=10.1371/journal.pone.0114349; citation_id=CR20"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=SLAF-seq: an efficient method of large-scale de novo SNP discovery and genotyping using high-throughput sequencing; citation_author=XW Sun, DY Liu, XF Zhang, WB Li, H Liu, WG Hong; citation_volume=8; citation_issue=3; citation_publication_date=2013; citation_doi=10.1371/journal.pone.0058700; citation_id=CR21"/>

    <meta name="citation_reference" content="citation_journal_title=Genome Biol; citation_title=Genome-wide association studies in plants: the missing heritability is in the field; citation_author=B Brachi, GP Morris, JO Borevitz; citation_volume=12; citation_issue=10; citation_publication_date=2011; citation_pages=232; citation_doi=10.1186/gb-2011-12-10-232; citation_id=CR22"/>

    <meta name="citation_reference" content="Wang WH, Zhang T, Zhang GX, Wang JY, Han KP, Wang YJ, et al. Genome-wide association study of antibody level response to NDV and IBV in Jinghai yellow chicken based on SLAF-seq technology. J Appl Genet. 2015;1&#8211;9."/>

    <meta name="citation_reference" content="citation_journal_title=BMC Genomics; citation_title=Construction of a high-density genetic map based on large-scale markers developed by specific length amplified fragment sequencing (SLAF-seq) and its application to QTL analysis for isoflavone content in Glycine max
                           ; citation_author=B Li, L Tian, JY Zhang, L Huang, FX Han, SR Yan; citation_volume=15; citation_issue=1; citation_publication_date=2014; citation_pages=1086; citation_doi=10.1186/1471-2164-15-1086; citation_id=CR24"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Genomics; citation_title=An SNP-based saturated genetic map and QTL analysis of fruit-related traits in cucumber using specific-length amplified fragment (SLAF) sequencing; citation_author=QZ Wei, YZ Wang, XD Qin, YX Zhang, ZT Zhang, J Wang; citation_volume=15; citation_issue=1; citation_publication_date=2014; citation_pages=1158; citation_doi=10.1186/1471-2164-15-1158; citation_id=CR25"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Plant Biol; citation_title=Construction of a high-density genetic map for sesame based on large scale marker development by specific length amplified fragment (SLAF) sequencing; citation_author=YX Zhang, LH Wang, HG Xin, DH Li, CX Ma, X Ding; citation_volume=13; citation_issue=1; citation_publication_date=2013; citation_pages=141; citation_doi=10.1186/1471-2229-13-141; citation_id=CR26"/>

    <meta name="citation_reference" content="Xu XW, Xu RX, Zhu BY, Yu T, Qu WQ, Lu L, et al. A high-density genetic map of cucumber derived from Specific Length Amplified Fragment sequencing (SLAF-seq). Front Plant Sci. 2014;5."/>

    <meta name="citation_reference" content="citation_journal_title=Am J Hum Genet; citation_title=Five years of GWAS discovery; citation_author=PM Visscher, MA Brown, MI McCarthy, J Yang; citation_volume=90; citation_issue=1; citation_publication_date=2012; citation_pages=7-24; citation_doi=10.1016/j.ajhg.2011.11.029; citation_id=CR28"/>

    <meta name="citation_reference" content="citation_journal_title=Plant J; citation_title=Loci and candidate gene identification for resistance to Sclerotinia sclerotiorum in soybean (Glycine max L. Merr.) viaassociation and linkage maps; citation_author=X Zhao, YP Han, YH Li, DY Liu, MM Sun, Y Zhao; citation_volume=82; citation_issue=2; citation_publication_date=2015; citation_pages=245-55; citation_doi=10.1111/tpj.12810; citation_id=CR29"/>

    <meta name="citation_reference" content="citation_journal_title=New Phytol; citation_title=Domestication footprints anchor genomic regions of agronomic importance in soybeans; citation_author=Y Han, X Zhao, D Liu, Y Li, DA Lightfoot, Z Yang; citation_volume=209; citation_issue=2; citation_publication_date=2016; citation_pages=871-84; citation_doi=10.1111/nph.13626; citation_id=CR30"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Genom; citation_title=Genetic characteristics of soybean resistance to HG type 0 and HG type 1.2.3.5.7 of the cyst nematode analyzed by genome-wide association mapping; citation_author=YP Han, X Zhao, GL Cao, Y Wang, YH Li, DY Liu; citation_volume=16; citation_issue=1; citation_publication_date=2015; citation_pages=1-11; citation_doi=10.1186/1471-2164-16-1; citation_id=CR31"/>

    <meta name="citation_reference" content="citation_journal_title=Am J Hum Genet; citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses; citation_author=S Purcell, B Neale, K Todd-Brown, L Thomas, MAR Ferreira, D Bender; citation_volume=81; citation_issue=3; citation_publication_date=2007; citation_pages=559-75; citation_doi=10.1086/519795; citation_id=CR32"/>

    <meta name="citation_reference" content="citation_journal_title=Am J Hum Genet; citation_title=Association mapping in structured populations; citation_author=JK Pritchard, M Stephens, NA Rosenberg, P Donnelly; citation_volume=67; citation_issue=1; citation_publication_date=2000; citation_pages=170-81; citation_doi=10.1086/302959; citation_id=CR33"/>

    <meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=Population structure and linkage disequilibrium in US barley germplasm: implications for association mapping; citation_author=MT Hamblin, TJ Close, PR Bhat, S Chao, JG Kling, KJ Abraham; citation_volume=50; citation_issue=2; citation_publication_date=2010; citation_pages=556-66; citation_doi=10.2135/cropsci2009.04.0198; citation_id=CR34"/>

    <meta name="citation_reference" content="citation_journal_title=Appl Microbiol Biotechnol; citation_title=Microbial P450 enzymes in biotechnology; citation_author=VB Urlacher, S Lutz-Wahl, RD Schmid; citation_volume=64; citation_issue=3; citation_publication_date=2004; citation_pages=317-25; citation_doi=10.1007/s00253-003-1514-1; citation_id=CR35"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Physiol; citation_title=Transcript profiling of poplar leaves upon infection with compatible and incompatible strains of the foliar rust Melampsora larici-populina
                           ; citation_author=C Rinaldi, A Kohler, P Frey, F Duchaussoy, N Ningre, A Couloux; citation_volume=144; citation_issue=1; citation_publication_date=2007; citation_pages=347-66; citation_doi=10.1104/pp.106.094987; citation_id=CR36"/>

    <meta name="citation_reference" content="citation_journal_title=Plant J; citation_title=Molecular cloning of a new receptor-like kinase gene encoded at the LrlO disease resistance locus of wheat; citation_author=C Feuillet, G Schachermayr, B Keller; citation_volume=11; citation_issue=1; citation_publication_date=1997; citation_pages=45-52; citation_doi=10.1046/j.1365-313X.1997.11010045.x; citation_id=CR37"/>

    <meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=Identification of molecular markers linked to the Yr15 stripe rust resistance gene of wheat originated in wild emmer wheat, Triticum dicoccoides
                           ; citation_author=G Sun, T Fahima, A Korol, T Turpeinen, A Grama, Y Ronin; citation_volume=95; citation_issue=4; citation_publication_date=1997; citation_pages=622-8; citation_doi=10.1007/s001220050604; citation_id=CR38"/>

    <meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=Microsatellite tagging of the stripe-rust resistance gene YrH52 derived from wild emmer wheat, Triticum dicoccoides, and suggestive negative crossover interference on chromosome 1B; citation_author=JH Peng, T Fahima, MS R&#246;der, YC Li, A Dahan, A Grama; citation_volume=98; citation_issue=6&#8211;7; citation_publication_date=1999; citation_pages=862-72; citation_doi=10.1007/s001220051145; citation_id=CR39"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Rep; citation_title=Cloning and characterization of a calcium binding EF-hand protein gene TaCab1 from wheat and its expression in response to Puccinia striiformis f. sp. tritici and abiotic stresses; citation_author=H Feng, XM Wang, YF Sun, XJ Wang, XM Chen, J Guo; citation_volume=38; citation_issue=6; citation_publication_date=2011; citation_pages=3857-66; citation_doi=10.1007/s11033-010-0501-8; citation_id=CR40"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Evolution; citation_title=The neighbor-joining method: a new method for reconstructing phylogenetic trees; citation_author=N Saitou, M Nei; citation_volume=4; citation_issue=4; citation_publication_date=1987; citation_pages=406-25; citation_id=CR41"/>

    <meta name="citation_reference" content="citation_journal_title=Algorithms Mol Biol; citation_title=On the optimality of the neighbor-joining algorithm; citation_author=K Eickmeyer, P Huggins, L Pachter, R Yoshida; citation_volume=3; citation_issue=1; citation_publication_date=2008; citation_pages=1; citation_doi=10.1186/1748-7188-3-5; citation_id=CR42"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinform; citation_title=Open source clustering software; citation_author=MJL Hoon, S Imoto, J Nolan, S Miyano; citation_volume=20; citation_issue=9; citation_publication_date=2004; citation_pages=1453-4; citation_doi=10.1093/bioinformatics/bth078; citation_id=CR43"/>

    <meta name="citation_reference" content="citation_journal_title=Computational Toxicology: Volume II; citation_title=Principal components analysis; citation_author=D Groth, S Hartmann, S Klie, J Selbig; citation_volume=2013; citation_publication_date=2013; citation_pages=527-47; citation_doi=10.1007/978-1-62703-059-5_22; citation_id=CR44"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinform; citation_title=Haploview: analysis and visualization of LD and haplotype maps; citation_author=JC Barrett, B Fry, JDMJ Maller, MJ Daly; citation_volume=21; citation_issue=2; citation_publication_date=2005; citation_pages=263-5; citation_doi=10.1093/bioinformatics/bth457; citation_id=CR45"/>

    <meta name="citation_reference" content="citation_journal_title=Plant J; citation_title=QTL-seq: rapid mapping of quantitative trait loci in rice by whole genome resequencing of DNA from two bulked populations; citation_author=H Takagi, A Abe, K Yoshida, S Kosugi, S Natsume, C Mitsuoka; citation_volume=74; citation_issue=1; citation_publication_date=2013; citation_pages=174-83; citation_doi=10.1111/tpj.12105; citation_id=CR46"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Fast and accurate long-read alignment with Burrows&#8211;Wheeler transform; citation_author=H Li, R Durbin; citation_volume=25; citation_issue=5; citation_publication_date=2010; citation_pages=1754-60; citation_id=CR47"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=The Sequence Alignment-Map format and SAMtools; citation_author=H Li; citation_volume=25; citation_issue=16; citation_publication_date=2009; citation_pages=2078-9; citation_doi=10.1093/bioinformatics/btp352; citation_id=CR48"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acid Res; citation_title=NCBI Reference Sequence (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins; citation_author=KD Pruitt, T Tatusova, DR Maglott; citation_volume=33; citation_issue=suppl 1; citation_publication_date=2005; citation_pages=D501-4; citation_id=CR49"/>

    <meta name="citation_author" content="Bing Zeng"/>

    <meta name="citation_author_institution" content="Department of Animal Science, Southwest University, Chongqing, China"/>

    <meta name="citation_author" content="Haidong Yan"/>

    <meta name="citation_author_institution" content="Department of Grassland Science, Animal Science and Technology College, Sichuan Agricultural University, Chengdu, China"/>

    <meta name="citation_author" content="Xinchun Liu"/>

    <meta name="citation_author_institution" content="Agricultural College, Sichuan Agricultural University, Chengdu, China"/>

    <meta name="citation_author" content="Wenjing Zang"/>

    <meta name="citation_author_institution" content="College of Life Science, China West Normal University, Nanchong, China"/>

    <meta name="citation_author" content="Ailing Zhang"/>

    <meta name="citation_author_institution" content="Department of Grassland Science, Animal Science and Technology College, Sichuan Agricultural University, Chengdu, China"/>

    <meta name="citation_author" content="Sifan Zhou"/>

    <meta name="citation_author_institution" content="Department of Grassland Science, Animal Science and Technology College, Sichuan Agricultural University, Chengdu, China"/>

    <meta name="citation_author" content="Linkai Huang"/>

    <meta name="citation_author_institution" content="Department of Grassland Science, Animal Science and Technology College, Sichuan Agricultural University, Chengdu, China"/>

    <meta name="citation_author" content="Jinping Liu"/>

    <meta name="citation_author_institution" content="College of Life Science, China West Normal University, Nanchong, China"/>



        <meta charset="UTF-8"/>
        <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
        <meta name="viewport" content="width=device-width, initial-scale=1"/>

        <!-- For IE 9 and below -->
<!--[if IE]><link rel="shortcut icon" href=/static/images/favicons//favicon.ico><![endif]-->
<!-- Touch Icons - iOS and Android 2.1+ -->
<link rel="apple-touch-icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>
<!-- Firefox, Chrome, Safari, IE 11+ and Opera -->
<link rel="icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>


        <script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script>
        <script>(function(d){if (sessionStorage.fontsLoaded){d.style.setProperty('--font-family-sans', 'Europa');d.style.setProperty('--font-family-serif', 'Noto Serif')}})(document.documentElement);</script>

        <link rel="stylesheet" media="screen" href=/static/styles/basic/basic-a31ca85bc2.css>
<link rel="stylesheet" media="print" href=/static/styles/bmc/print-3a6240f456.css>

<link rel="stylesheet" class="js-ctm" href=/static/styles/bmc/main-84414d3d47.css
      media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">

        

        <script src="/static/js/app-bundle-8d61e6ef37.js" defer></script>
        
        <script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script>
        
    <script type="text/javascript">
        config = {
            env: 'live',
            site: 'hereditasjournal.biomedcentral.com',
            siteWithPath: 'hereditasjournal.biomedcentral.com' + window.location.pathname,
            twitterHashtag: '',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/s41065-017-0027-3',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central'
        };
    </script>

        
    <script type="text/javascript">
        dataLayer = [{"content":{"article":{"doi":"10.1186/s41065-017-0027-3","articleType":"Research","peerReviewType":"Closed","supplement":null,"keywords":"Genome-wide association study;Orchardgrass;Rust disease;SLAF-seq"},"contentInfo":{"imprint":"BioMed Central","title":"Genome-wide association study of rust traits in orchardgrass using SLAF-seq technology","publishedAt":1487808000000,"publishedAtDate":"2017-02-23","author":["Bing Zeng","Haidong Yan","Xinchun Liu","Wenjing Zang","Ailing Zhang","Sifan Zhou","Linkai Huang","Jinping Liu"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":"CC BY + CC0","openAccess":true},"environment":"live"},"journal":{"siteKey":"hereditasjournal.biomedcentral.com","volume":"154","issue":"1","title":"Hereditas","type":"AJs Life","journalID":41065,"gaCode":"UA-62066850-11","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Research","publishingSegment":"Genetics & Heredity (Life)"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}}}]

    </script>

        <script src="https://cdn.cookielaw.org/consent/7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script>
        
    
        
    

        
    
        
            
                <!-- Google Tag Manager -->
                <script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script>
                <!-- End Google Tag Manager -->
            
        
    


        
    
    <link rel="canonical" href="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0027-3"/>
    

        
        
        <meta property="og:url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0027-3"/>
        <meta property="og:type" content="article"/>
        <meta property="og:site_name" content="Hereditas"/>
        <meta property="og:title" content="Genome-wide association study of rust traits in orchardgrass using SLAF-seq technology"/>
        <meta property="og:description" content="While orchardgrass (Dactylis glomerata L.) is a well-known perennial forage species, rust diseases cause serious reductions in the yield and quality of orchardgrass; however, genetic mechanisms of rust resistance are not well understood in orchardgrass. In this study, a genome-wide association study (GWAS) was performed using specific-locus amplified fragment sequencing (SLAF-seq) technology in orchardgrass. A total of 2,334,889 SLAF tags were generated to produce 2,309,777 SNPs. ADMIXTURE analysis revealed unstructured subpopulations for 33 accessions, indicating that this orchardgrass population could be used for association analysis. Linkage disequilibrium (LD) analysis revealed an average r2 of 0.4 across all SNP pairs, indicating a high extent of LD in these samples. Through GWAS, a total of 4,604 SNPs were found to be significantly (P &amp;lt; 0.01) associated with the rust trait. The bulk analysis discovered a number of 5,211 SNPs related to rust trait. Two candidate genes, including cytochrome P450, and prolamin were implicated in disease resistance through prediction of functional genes surrounding each high-quality SNP (P &amp;lt; 0.01) associated with rust traits based on GWAS analysis and bulk analysis The large number of SNPs associated with rust traits and these two candidate genes may provide the basis for further research on rust resistance mechanisms and marker-assisted selection (MAS) for rust-resistant lineages."/>
        <meta property="og:image" content="https://images.springer.com/sgw/journals/medium/41065.jpg"/>
    
    </head>

    <body class="journal journal-fulltext"
          
          >
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK"
                            height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><feColorMatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"/></filter></defs><symbol id="icon-arrow-down" viewBox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"/></symbol><symbol id="icon-arrow-left-bullet" viewBox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"/></symbol><symbol id="icon-arrow-right-bullet" viewBox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"/></symbol><symbol id="icon-arrow-right" viewBox="0 0 32 64"><path d="M0 0v64l32-32z"/></symbol><symbol id="icon-close" viewBox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"/></symbol><symbol id="icon-community" viewBox="0 0 60.643 31.01"><mask maskUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><path fill="#FFF" d="M0 31.01h60.643V0H0z" filter="url(#a)"/></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"/></symbol><symbol id="icon-download" viewBox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"/></symbol><symbol id="icon-ext-link" viewBox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"/><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"/><path fill="#676767" d="M9.7 0L16 6.2V0z"/></symbol><symbol id="icon-info-bordered" viewBox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"/><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"/></symbol><symbol id="icon-opr" viewBox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"/><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"/></symbol><symbol id="icon-quotation" viewBox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"/></symbol><symbol id="icon-remove" viewBox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"/><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"/></symbol><symbol id="icon-ribbon" viewBox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"/></symbol><symbol id="icon-rss" viewBox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"/><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"/><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"/></g></symbol><symbol id="icon-search" viewBox="0 0 16 16"><path d="M6.276 12.257C2.815 12.257 0 9.508 0 6.128 0 2.749 2.815 0 6.276 0c3.462 0 6.277 2.749 6.277 6.128.002 3.38-2.815 6.129-6.277 6.129zm0-10.772c-2.622 0-4.755 2.083-4.755 4.643 0 2.562 2.133 4.644 4.755 4.644 2.623 0 4.756-2.082 4.756-4.644 0-2.56-2.134-4.643-4.756-4.643z"/><path d="M16 13.751L13.641 16l-4.875-4.82 2.36-2.248L16 13.751z"/></symbol><symbol id="icon-speech-bubble" viewBox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"/></symbol><symbol id="icon-error" viewBox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"/></symbol><symbol id="icon-info" viewBox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"/></symbol><symbol id="icon-success" viewBox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"/></symbol><symbol id="icon-warning" viewBox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"/></symbol><symbol id="icon-facebook-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"/></symbol><symbol id="icon-facebook" viewBox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"/><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"/></symbol><symbol id="icon-gplus-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M480.5 273.117c.123.076.26.185.416.334.146.152.289.343.432.562.135.211.258.459.359.746.086.286.129.619.129 1.004-.012.7-.164 1.262-.465 1.683a5.374 5.374 0 0 1-.461.562c-.186.176-.375.35-.58.527a2.65 2.65 0 0 0-.322.41.908.908 0 0 0-.176.553c0 .205.059.375.18.508.102.129.199.238.299.336l.662.545a6.48 6.48 0 0 1 1.086 1.119c.293.415.445.955.459 1.625 0 .949-.42 1.789-1.256 2.521-.869.76-2.123 1.15-3.762 1.177-1.373-.015-2.4-.308-3.074-.879-.682-.534-1.023-1.172-1.023-1.917 0-.359.111-.766.336-1.211.215-.443.602-.837 1.172-1.174a6.126 6.126 0 0 1 1.998-.725 15.415 15.415 0 0 1 1.723-.173 3.996 3.996 0 0 1-.377-.596 1.347 1.347 0 0 1-.193-.74c0-.174.023-.317.074-.433a8.83 8.83 0 0 1 .121-.338 5.973 5.973 0 0 1-.633.035c-1.043-.012-1.838-.339-2.381-.979-.574-.603-.861-1.3-.861-2.097 0-.961.404-1.832 1.217-2.616.559-.46 1.135-.757 1.736-.898a8.494 8.494 0 0 1 1.67-.182h3.916l-1.209.704h-1.207v.005l-.005.002zm.754 11.276c0-.498-.162-.932-.484-1.302-.346-.348-.881-.778-1.609-1.29a4.045 4.045 0 0 0-.439-.02c-.102-.016-.354 0-.768.039a8.141 8.141 0 0 0-1.244.275c-.1.036-.24.096-.42.17a2.713 2.713 0 0 0-.553.355 1.985 1.985 0 0 0-.459.592c-.145.25-.215.551-.215.901 0 .687.311 1.253.932 1.698.596.444 1.404.68 2.432.688.928-.014 1.629-.217 2.117-.612.476-.385.71-.886.71-1.494zm-2.729-5.743a1.843 1.843 0 0 0 1.289-.556c.164-.249.271-.506.318-.767a7.32 7.32 0 0 0 .041-.659c0-.772-.195-1.552-.594-2.336a2.611 2.611 0 0 0-.73-.921 1.894 1.894 0 0 0-1.064-.352 1.723 1.723 0 0 0-1.324.64c-.299.437-.438.92-.428 1.457 0 .709.209 1.444.623 2.211.201.356.459.659.775.912.323.248.686.371 1.094.371zm9.698.077h-2.055v-2.056h-.996v2.056h-2.053v.998h2.053v2.052h.996v-2.052h2.055z"/></symbol><symbol id="icon-gplus" viewBox="0 0 24 24"><circle fill="#D84A36" cx="12" cy="12" r="12"/><g fill="#FFF"><path d="M12.012 12.111c-.304-.33-.07-.842.155-1.078.226-.241.639-.56.829-.74.193-.184.66-.605.674-1.868 0 0 .112-1.139-1.127-2.06h.977l1.038-.593h-3.507s-2.083-.067-3.085 1.207c-1 1.276-.966 2.871.114 3.77 1.08.898 2.371.648 2.371.648s-.197.422-.163.775c.036.353.421.849.504.965 0 0-2.386-.011-3.551 1.033-1.165 1.053-1.29 3.097 1.021 3.803 2.312.707 5.303-.055 5.906-2.277 0 0 .262-.908-.189-1.683-.459-.775-1.663-1.571-1.967-1.902zm-2.555-1.83c-.744-1-1.308-2.953 0-3.783 0 0 .955-.579 1.801.266.846.846 1.082 2.277.928 3.003-.15.727-.563.897-.739 1.008-.185.111-1.249.504-1.99-.494zm3.668 5.481c-.035.393-.083 1.254-1.491 1.664-1.409.408-3.826-.264-3.757-1.914 0 0 .006-.698.449-1.089.445-.396 1.094-.794 3.029-.819 0 0 .643.473 1.072.809.427.332.733.952.698 1.349z"/><path d="M16.323 11.055V9.803h-.752v1.252h-1.255v.748h1.255v1.256h.752v-1.256h1.248v-.748z"/></g></symbol><symbol id="icon-linkedin" viewBox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"/><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"/><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"/></g></symbol><symbol id="icon-reddit" viewBox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"/><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"/><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"/><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"/><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"/><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"/><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"/><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"/></symbol><symbol id="icon-twitter-bordered" viewBox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"/><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/></symbol><symbol id="icon-twitter" viewBox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"/><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"/></symbol><symbol id="icon-weibo-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"/><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"/><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"/></symbol><symbol id="icon-weibo" viewBox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"/><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"/><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"/><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"/><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"/><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"/></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner c-banner--info">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner" >
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1"
                 data-gpt-unitpath="/270604982/bmc/hereditasjournal/articles"
                 data-gpt-sizes="728x90,970x90"
                 data-gpt-targeting="pos=LB1;doi=10.1186/s41065-017-0027-3;kwrd=Genome-wide association study,Orchardgrass,Rust disease,SLAF-seq;pmc=L32030,L21001;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-017-0027-3&amp;kwrd=Genome-wide association study,Orchardgrass,Rust disease,SLAF-seq&amp;pmc=L32030,L21001&amp;">
                        <img data-test="gpt-advert-fallback-img"
                             src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-017-0027-3&amp;kwrd=Genome-wide association study,Orchardgrass,Rust disease,SLAF-seq&amp;pmc=L32030,L21001&amp;"
                             alt="Advertisement"
                             width="728"
                             height="90">
                    </a>
                </noscript>
            </div>
        </div>
    </div>

        
         
    <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>


<header class="c-popup-search ctx-publisher-header" id="top" data-test="publisher-header">
    <nav class="c-navbar">
        <div class="u-container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization" />
                <div class="c-logo">
    <a href="https://www.biomedcentral.com" itemprop="url">
        <img src=/static/images/bmc/logos/logo-bmc-white-aj-e3a70ae848.svg alt="BMC" itemprop="logo">
        <div class="c-logo__strapline">
            <img src=/static/images/bmc/logos/logo-bmc-white-strapline-sn-e7b1d6e67f.svg alt="Part of Springer Nature">
        </div>
    </a>
</div>
            </div>
            <button
                type="button"
                class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg"
                id="publisher-header-menu-button"
                data-toggle="collapse"
                data-test="publisher-header-menu-toggle"
                data-toggle-linked="publisher-header"
                data-target="publisher-header-search publisher-header-menu"
                aria-controls="publisher-header-menu publisher-header-search"
                aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


                <ul class="c-navbar__nav">
                    <li class="c-navbar__item">
                        <a data-replace="true"
                           data-placeholder="/placeholder/v1/username"
                           class="c-navbar__link"
                           href="https://www.biomedcentral.com/login"
                           data-test="login-link"
                           data-track="click"
                           data-track-category="Account"
                           data-track-action="Click Login Link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                    <li class="c-navbar__item">
                        <button
                            type="button"
                            class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg"
                            data-toggle="collapse"
                            data-test="header-search-button"
                            data-target="publisher-header-search publisher-header-menu publisher-header-menu-button"
                            aria-controls="publisher-header-search publisher-header-menu"
                            aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true">
                                    <use xlink:href="#icon-search"></use>
                                </svg>
                            </span>
                        </button>
                    </li>
                </ul>
            </div>
        </div>
    </nav>
    <div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field c-form-field--inline" method="GET" action="//www.biomedcentral.com/search" data-track="submit"
        data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label data-track-label="" data-test="global-search">
        <label for="publisherSearch" class="u-visually-hidden">Search all BMC articles</label>
        <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text" value="" placeholder="Search all BMC articles"/>
        <button class="c-button c-button--secondary" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true">
        <use xlink:href="#icon-search"></use>
    </svg>
</button>


        <input type="hidden" name="searchType" value="publisherSearch"/>
    </form>
</div>

            </div>
        </div>
    </div>
</header>


        
            <header class="c-journal-header c-journal-header--hereditas ctx-journal-header">
                <div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="/">
    
        
    
    <span class="c-journal-title__text ">Hereditas</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="u-container">
                        
                            <button type="button"
                                    class="c-navbar__link c-dropdown u-hidden-at-lg"
                                    data-toggle="collapse"
                                    data-test="journal-header-menu"
                                    data-toggle-linked="journal-header"
                                    data-target="journal-menu"
                                    aria-controls="journal-menu"
                                    aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav">
                                    <ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/'>Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/about'>About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href='/articles'>Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/submission-guidelines'>Submission Guidelines</a>
                                            </li>
                                        
                                    </ul>
                                </nav>
                            </div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--ocean-light-blue">
                    
                </div>
            </header>
            
        

        <div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-content-layout c-content-layout--fulltext" data-component="sticky-container">
                <div class="c-content-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3><ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc=""><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results and Discussion">Results and Discussion</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec11" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Method">Method</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec18" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Conclusions">Conclusions</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li></ol></nav>
                </div>

                <main class="c-content-layout__main">
                    <article itemscope itemtype="http://schema.org/ScholarlyArticle">
                        
                        
    <div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
            <li class="c-article-identifiers__item" data-test="article-category">Research</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
    </div>

                        <div class="FulltextWrapper"><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">Genome-wide association study of rust traits in orchardgrass using SLAF-seq technology</h1></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true"><ul class="u-listReset"><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Bing&nbsp;Zeng</span><span class="EqualContributionSymbol">†</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup><a href="mailto:zbin78@163.com" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Haidong&nbsp;Yan</span><span class="EqualContributionSymbol">†</span><sup><a href="#Aff2" class="AffiliationID">2</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Xinchun&nbsp;Liu</span><sup><a href="#Aff3" class="AffiliationID">3</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Wenjing&nbsp;Zang</span><sup><a href="#Aff4" class="AffiliationID">4</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Ailing&nbsp;Zhang</span><sup><a href="#Aff2" class="AffiliationID">2</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Sifan&nbsp;Zhou</span><sup><a href="#Aff2" class="AffiliationID">2</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Linkai&nbsp;Huang</span><sup><a href="#Aff2" class="AffiliationID">2</a></sup><a href="mailto:huanglinkai@sicau.edu.cn" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a> and </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Jinping&nbsp;Liu</span><sup><a href="#Aff4" class="AffiliationID">4</a></sup></li></ul><div class="AffiliationLegend"><div class="AffiliationLegendLine"><span class="EqualContributionSymbol">†</span>Contributed equally</div></div></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation"><div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">Hereditas</span></span><span class="ArticleCitation_Year">2017</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">154</strong>:5</span></span></div><p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/s41065-017-0027-3">https://doi.org/10.1186/s41065-017-0027-3</a></p><p class="Copyright">© &nbsp;The Author(s)&nbsp;2017</p></div></div><ul class="ArticleHistory u-reset-list"><li class="History HistoryReceived"><strong>Received: </strong><span>20&nbsp;August&nbsp;2016</span></li><li class="History HistoryAccepted"><strong>Accepted: </strong><span>24&nbsp;January&nbsp;2017</span></li><li class="History HistoryOnlineDate"><strong>Published: </strong><span itemprop="datePublished">23&nbsp;February&nbsp;2017</span></li></ul></div>
                        
                        
                        
    

                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2><div class="c-section__content" id="Abstract"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p id="Par1" class="Para">While orchardgrass (<em xmlns="" class="EmphasisTypeItalic">Dactylis glomerata</em> L.) is a well-known perennial forage species, rust diseases cause serious reductions in the yield and quality of orchardgrass; however, genetic mechanisms of rust resistance are not well understood in orchardgrass.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Results</h3>
                <p id="Par2" class="Para">In this study, a genome-wide association study (GWAS) was performed using specific-locus amplified fragment sequencing (SLAF-seq) technology in orchardgrass. A total of 2,334,889 SLAF tags were generated to produce 2,309,777 SNPs. ADMIXTURE analysis revealed unstructured subpopulations for 33 accessions, indicating that this orchardgrass population could be used for association analysis. Linkage disequilibrium (LD) analysis revealed an average r<sup>2</sup> of 0.4 across all SNP pairs, indicating a high extent of LD in these samples. Through GWAS, a total of 4,604 SNPs were found to be significantly (<em xmlns="" class="EmphasisTypeItalic">P</em> &lt; 0.01) associated with the rust trait. The bulk analysis discovered a number of 5,211 SNPs related to rust trait. Two candidate genes, including cytochrome P450, and prolamin were implicated in disease resistance through prediction of functional genes surrounding each high-quality SNP (<em xmlns="" class="EmphasisTypeItalic">P</em> &lt; 0.01) associated with rust traits based on GWAS analysis and bulk analysis</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusions</h3>
                <p id="Par3" class="Para">The large number of SNPs associated with rust traits and these two candidate genes may provide the basis for further research on rust resistance mechanisms and marker-assisted selection (MAS) for rust-resistant lineages.</p>
              </div></div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords"><li class="c-keywords__item" itemprop="about">Genome-wide association study</li><li class="c-keywords__item" itemprop="about">Orchardgrass</li><li class="c-keywords__item" itemprop="about">Rust disease</li><li class="c-keywords__item" itemprop="about">SLAF-seq</li></ul></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par22" class="Para">Orchardgrass (<em xmlns="" class="EmphasisTypeItalic">Dactylis glomerata</em> L.) is a cool-season forage grass that is wildly cultivated in all mild, subtropical regions of the world [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>]. Due to its high yield, high nutritional value, and shade tolerance, it has become a valuable pasture species in China [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>]. However, orchardgrass is frequently infected by rust disease, which leads to low yield and poor quality [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par23" class="Para">Several studies on rust disease in orchardgrass have been reported. Tajimi et al. [<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>] studied orchardgrass clonal lines and found that the rusts of orchardgrass, timothy, and ryegrass were closely related and that stem rust was under the genetic control of orchardgrass. Miller and Carlson [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>] evaluated rust resistance in orchardgrass based on both phenotypic performance and a polycross progeny test through phenotypic selection. Yan et al. [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>] investigated 242 germplasm resources of orchardgrass for 2 years, finding that 13 accessions exhibited high resistance to rust disease. However, the phenotype of an organism can be influenced by both its genotype and its environment. To better assess the impact of this disease, additional methods are needed to study the desirable genetic traits for these plants.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par24" class="Para">Genome-wide association studies (GWASs) have become one of the most commonly strategies for identifying genes underlying complex traits in plants. In model species, such as <em xmlns="" class="EmphasisTypeItalic">Arabidopsis thaliana</em>, the potential of GWAS to reveal genetic traits has been demonstrated successfully by the functional validation of the accelerated cell death 6 (ACD6) gene [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>]. This approach has also been used to develop a precise estimation of variation in traits such as flowering time [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>] of <em xmlns="" class="EmphasisTypeItalic">Arabidopsis thaliana</em>, leaf architecture [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>] of maize (<em xmlns="" class="EmphasisTypeItalic">Zea mays</em>), and blight resistance [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>] of wheat (<em xmlns="" class="EmphasisTypeItalic">Triticum aestivum</em> L.).In recent studies on rice (<em xmlns="" class="EmphasisTypeItalic">Oryza sativa</em>), several genes that have large effects on traits involved in determining yield, morphology, and salinity tolerance have been identified [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>–<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>]. In the past few years, flowering time genes in barley (<em xmlns="" class="EmphasisTypeItalic">Hordeum vulgare</em>) [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>], the PsyI-AI locus in wheat [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>], and the rhg-1 gene in soybean (<em xmlns="" class="EmphasisTypeItalic">Glycine max</em>) [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>] have been identified as candidate genes [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>] through GWAS. Rust disease has seriously influenced plant growth and development, so some studies focus on conducting GWAS to plant rust disease. For example, a GWAS was conducted using 232 winter wheat breeding lines to identify loci conferring resistance to Ug99 that is a new race of stem rust and threatens global wheat production, and 12 loci associated with Ug99 resistance were discovered in this study [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>]. A number of 177 oat (<em xmlns="" class="EmphasisTypeItalic">Avena sativa</em>) accessions were evaluated for disease resistance and further genotyped with 15,000 Diversity Arrays Technology (DArT) and 31 simple sequence repeat and markers to disclose association with disease resistance trait and found five markers were associated with rust resistance [<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par25" class="Para">Consequently, several reduced-representation sequencing technologies, including restriction site-associated sequencing (RADseq), double digest RADseq, and two-enzyme genotyping-by-sequencing (GBS), were developed as cost-effective methods for SNP discovery and high-throughput genotyping [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>]. Recently, specific-locus amplified fragment sequencing (SLAF-seq), a more efficient solution for large-scale genotyping, was developed by Sun et al. [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>]. This approach involves several distinguishing characteristics: (1) fine-mapping resolution among founders can be performed given a high-density of genotypes; (2) outcrossing reshuffles variation in the founder genomes, providing some control over population structure effects; (3) joint-linkage mapping identifies low-resolution quantitative trait loci (QTLs) across all recombinant inbred line (RIL) families, and this genetic background can be controlled while performing nested associations for fine mapping; and (4) the use of RILs allows repeated measurement of phenotypes on the same lines in common and different environments [<span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>]. This approach has previously been used successfully to perform GWAS [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par26" class="Para">SLAF-seq is highly automated due to the development of bioinformatics tools and high-throughput sequencing technology applications. SLAF sequencing methods ascertain the uniformity, efficiency, and density of marker development, and they have been applied in several studies. For example, Li et al. [<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>] constructed a high-density soybean genetic map to discover QTLs pertaining to isoflavone content that were consistent across various environments; Wei et al. [<span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>] established a high-density SNP map for cucumber (<em xmlns="" class="EmphasisTypeItalic">Cucumis sativus</em> L.) through SLAF-seq to detect fruit-related QTLs. Additional studies have created high-density genetic maps in order to provide a platform for gene/QTL fine-mapping [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>, <span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>, <span class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>]. These studies showed that SLAF-seq methods are used primarily to construct genetic maps for discovering valuable QTLs; however, for QTL mapping, a large investment of time is needed to establish mapping populations, and the intensive labor required may be a limitation for identification of specific traits. To avoid the limitations of this approach, GWAS is a reliable method used to confirm molecular markers relating to important economic traits. SNP markers that cover the entire genome may be analyzed without the construction of mapping populations [<span class="CitationRef"><a href="#CR28" data-toggle="citation">28</a></span>]. Moreover, several studies have used used SLAF-seq for GWAS [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>, <span class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>–<span class="CitationRef"><a href="#CR31" data-toggle="citation">31</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par27" class="Para">In this study, we use the SLAF-seq technology to perform a GWAS of rust traits in orchardgrass to identify the associated SNPs and predict functional genes. These results will provide a basis for orchardgrass breeding and may be helpful in enhancing rust resistance in orchardgrass.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 SectionTypeDiscussion c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#ResultsandDiscussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="ResultsandDiscussion">Results and Discussion</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="ResultsandDiscussion" data-test="collapse-section"><section id="Sec3" class="Section2 RenderAsSection2">
                <h3 class="Heading">Sequence and quality statistics</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par28" class="Para">A total of 76.74&nbsp;M reads were obtained from this experiment. The average Q3 value was 86.91%, and the average GC content was 46.97%. Details are listed in Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>: Table S1.</p>
              </section><section id="Sec4" class="Section2 RenderAsSection2">
                <h3 class="Heading">Specific-locus amplified fragment sequencing results</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par29" class="Para">In total, 76.74&nbsp;M reads were generated for the 33 genotypes, encompassing 4.30 Gb of the orchardgrass genomic DNA sequence. A total of 2,334,889 SLAF tags were identified throughout the genome (Table&nbsp;<span class="InternalRef"><a href="#Tab1">1</a></span>). These SLAF tags were divided into three types: Marker (polymorphism tags), No ploy (no polymorphism tags), and Repeat (SLAF tags on repeat sequences) (Table&nbsp;<span class="InternalRef"><a href="#Tab1">1</a></span>). PLINK (v1.07) [<span class="CitationRef"><a href="#CR32" data-toggle="citation">32</a></span>] was used to carry out quality control of the data. Those SNPs with low integrity (&lt;85%), low minor allele frequency (&lt;5%), and insufficient reads (&lt;90%) were discarded. After filtration, 2,309,777 SNPs were used for further analysis.<figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 1</span>
                        <p xmlns="" class="SimplePara">The statistical results for each type of SLAF tags</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Type</p></th><th><p xmlns="" class="SimplePara">Marker</p></th><th><p xmlns="" class="SimplePara">NoPoly</p></th><th><p xmlns="" class="SimplePara">Repeat</p></th><th><p xmlns="" class="SimplePara">Total</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">Number</p></td><td><p xmlns="" class="SimplePara">643 008</p></td><td><p xmlns="" class="SimplePara">1 689 409</p></td><td><p xmlns="" class="SimplePara">2 472</p></td><td><p xmlns="" class="SimplePara">2 334 889</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Percent</p></td><td><p xmlns="" class="SimplePara">27.54%</p></td><td><p xmlns="" class="SimplePara">72.36%</p></td><td><p xmlns="" class="SimplePara">0.11%</p></td><td><p xmlns="" class="SimplePara">100%</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">Note: Marker, pleomorphic tags; Nopoly, non-pleomorphic tags; Repeat: SLAF tags on repeat sequence</p>
                    </div></figure>
                        </div>
              </section><section id="Sec5" class="Section2 RenderAsSection2">
                <h3 class="Heading">Phylogenetic analysis</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par30" class="Para">The 33 orchardgrass samples could not be clearly divided into subgroups by the neighbor-joining analysis using MEGA software. However, the results from this phylogenetic analysis showed that the HS (high susceptibility of disease) materials were relatively concentrated, comparing with HR (high level of resistance of disease) materials (Fig.&nbsp;<span class="InternalRef"><a href="#Fig1">1</a></span>).<figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO1">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0027-3/MediaObjects/41065_2017_27_Fig1_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0027-3/MediaObjects/41065_2017_27_Fig1_HTML.gif" alt="Fig. 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0027-3/MediaObjects/41065_2017_27_Fig1_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0027-3/MediaObjects/41065_2017_27_Fig1_HTML.gif" alt="Fig. 1"></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 1</span>
                        <p xmlns="" class="SimplePara">The evolutionary tree of 33 orchardgrass accessions. The branch indicates each material. The red line is HS materials, and blue line is HR materials</p>
                      </div></figcaption></figure>
                        </div>
              </section><section id="Sec6" class="Section2 RenderAsSection2">
                <h3 class="Heading">Group structure and clustering analysis</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par31" class="Para">The best dataset was produced by using a K-value of 2, indicating that our samples probably derived from 2 ancestors. With the maximum membership probability, 5 accessions were assigned to group 1 (G1), while 28 accessions were assigned to group 2 (G2). All 5 accessions from G1 were HR samples, and all 16 HS accessions (16) were included in G2 (Fig.&nbsp;<span class="InternalRef"><a href="#Fig2">2</a></span>).<figure xmlns="" class="Figure" id="Fig2"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO2">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0027-3/MediaObjects/41065_2017_27_Fig2_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0027-3/MediaObjects/41065_2017_27_Fig2_HTML.gif" alt="Fig. 2" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0027-3/MediaObjects/41065_2017_27_Fig2_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0027-3/MediaObjects/41065_2017_27_Fig2_HTML.gif" alt="Fig. 2"></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 2</span>
                        <p xmlns="" class="SimplePara">Two subgroups inferred from ADMIXTURE analysis. The vertical coordinated of each subgroup means the membership coefficients for each accession. The digits of the horizontal coordinate represent the accessions according to Table&nbsp;<span xmlns="http://www.w3.org/1999/xhtml" class="InternalRef"><a href="#Tab3">3</a></span>. Green zone: G1; Blue zone: G2. The blue line means HR materials; The red line means HS materials</p>
                      </div></figcaption></figure>
                        </div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par32" class="Para">We applied a clustering strategy to the samples with ADMIXTURE software. This method has been used with large sample sizes, exhibiting a strong capability to assign individuals into populations. The best dataset (K = 2) showed that our samples probably derive from two ancestors. It is important to use population-based methods to separate accessions from mixed populations into unstructured subpopulations, allowing for association analyses between phenotypes and molecular bands to be conducted in homogeneous subpopulations [<span class="CitationRef"><a href="#CR33" data-toggle="citation">33</a></span>, <span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>]. In this study, the accessions associated with resistant and susceptible traits remained in G1, indicating that this orchardgrass population could be used for association analysis.</p>
              </section><section id="Sec7" class="Section2 RenderAsSection2">
                <h3 class="Heading">PCA analysis</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par33" class="Para">A principal component analysis (PCA) was performed using the 2,309,777 SNPs from all 33 accessions to estimate the clusters of population. Principal Component 1 (PC1) explained 7.94% of the variation in the genotypic data, while PC2 and PC3 explained 5.20% and 2.57% of the variation, respectively. Although there are intermediate accessions that make the groupings less clear, the PCA results indicate that the HS samples can be clustered into one group (Fig.&nbsp;<span class="InternalRef"><a href="#Fig3">3</a></span>).<figure xmlns="" class="Figure" id="Fig3"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO3">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0027-3/MediaObjects/41065_2017_27_Fig3_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0027-3/MediaObjects/41065_2017_27_Fig3_HTML.gif" alt="Fig. 3" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0027-3/MediaObjects/41065_2017_27_Fig3_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0027-3/MediaObjects/41065_2017_27_Fig3_HTML.gif" alt="Fig. 3"></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 3</span>
                        <p xmlns="" class="SimplePara">The PCA analysis of 33 orchardgrass accessions. The red balls mean HS materials; the blue balls mean HR materials</p>
                      </div></figcaption></figure>
                        </div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par34" class="Para">Based on analysis of 2,309,777 SNPs, a neighbor-joining tree was constructed using MEGA software. The results from the phylogenetic analysis showed that the 33 orchardgrass samples could not be clearly divided into two major clusters (Fig.&nbsp;<span class="InternalRef"><a href="#Fig1">1</a></span>), though the 16 HS individuals were able to cluster into one group as shown in the PCA results (Fig.&nbsp;<span class="InternalRef"><a href="#Fig3">3</a></span>). This indicates that there may be no strong relationship between rust resistance and genetic background. Several individuals appeared to be intermediates between the two groups, making the distinction between groups less clear; it may be that we lack a sufficient number of individuals in our sample to appropriately delineate the groups.</p>
              </section><section id="Sec8" class="Section2 RenderAsSection2">
                <h3 class="Heading">Linkage disequilibrium analysis</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par35" class="Para">The 2,309,777 SNPs generated from the 2,334,889 SLAF markers with unknown chromosome information were used to evaluate the extent of LD in the 33 orchardgrass accessions. In total, 9,345,646 pairs of SNPs were detected and an r<sup>2</sup> value was calculated for each pair. The values ranged from 0.0000 to 1.0000, with an average r<sup>2</sup> of 0.4, indicating a substantial amount of LD existing in these SNPs. Of the 9,345,646 pairs, 7,270 (0.08%) were considered to be in LD with strong linkage at r<sup>2</sup> &gt; 0.8 (Additional file <span class="InternalRef"><a href="#MOESM2">2</a></span>).</p>
              </section><section id="Sec9" class="Section2 RenderAsSection2">
                <h3 class="Heading">GWAS analysis and bulk analysis</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par36" class="Para">The association analysis between SNP markers and the rust trait was performed using TASSEL to search for associated tags and allelic variation. After general linear model (GLM) analysis of 2,309,777 SNPs, 4,604 SNPs were significantly (<em xmlns="" class="EmphasisTypeItalic">P</em> &lt; 0.01) associated with the rust trait (Additional file <span class="InternalRef"><a href="#MOESM3">3</a></span>). At a more stringent P-value cutoff of &lt; 0.001, 1,761 SNPs were still associated with the rust trait (Additional file <span class="InternalRef"><a href="#MOESM4">4</a></span>). The bulk analysis was conducted to discover 69,690 SNPs related to the rust trait, and a total number of 5,211 SNPs were filtered when the SNP index is over 0.3 (Additional file <span class="InternalRef"><a href="#MOESM5">5</a></span>).</p>
              </section><section id="Sec10" class="Section2 RenderAsSection2">
                <h3 class="Heading">Bioinformatics analysis of candidate genes</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par37" class="Para">We predicted 555 genes from these 4,604 SNPs in our study (Additional file <span class="InternalRef"><a href="#MOESM6">6</a></span>) through GWAS analysis, and only four candidate genes, including cytochrome P450, Pc68LrkC4, prolamin, and EF-hand Ca<sup>2+</sup>-binding protein (TaCab1), were implicated with disease resistance in the Nonredundant NCBI nucleotide sequences (NT) database, and their predicted molecular functions and biological processes are specified in Table&nbsp;<span class="InternalRef"><a href="#Tab2">2</a></span>.<figure xmlns="" id="Tab2" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 2</span>
                        <p xmlns="" class="SimplePara">Bioinformatics analysis of candidate genes</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></colgroup><colgroup><col char="."></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">SLAF marker</p></th><th char="."><p xmlns="" class="SimplePara">E_value</p></th><th><p xmlns="" class="SimplePara">Nearest gene</p></th><th><p xmlns="" class="SimplePara">Predict Function</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">Marker10747</p></td><td char="."><p xmlns="" class="SimplePara">5.00E-18</p></td><td><p xmlns="" class="SimplePara">Cytochrome P450</p></td><td><p xmlns="" class="SimplePara">Resist pathogenic bacterial</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Marker11029</p></td><td char="."><p xmlns="" class="SimplePara">2.00E-12</p></td><td><p xmlns="" class="SimplePara">Cytochrome P450</p></td><td><p xmlns="" class="SimplePara">Resist pathogenic bacterial</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Marker13729<sup xmlns="http://www.w3.org/1999/xhtml">a</sup>
                                          </p></td><td char="."><p xmlns="" class="SimplePara">2.00E-17</p></td><td><p xmlns="" class="SimplePara">Cytochrome P450</p></td><td><p xmlns="" class="SimplePara">Resist pathogenic bacterial</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Marker1700<sup xmlns="http://www.w3.org/1999/xhtml">a</sup>
                                          </p></td><td char="."><p xmlns="" class="SimplePara">5.00E-13</p></td><td><p xmlns="" class="SimplePara">Cytochrome P450</p></td><td><p xmlns="" class="SimplePara">Resist pathogenic bacterial</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Marker3124</p></td><td char="."><p xmlns="" class="SimplePara">5.00E-18</p></td><td><p xmlns="" class="SimplePara">Cytochrome P450</p></td><td><p xmlns="" class="SimplePara">Resist pathogenic bacterial</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Marker33680</p></td><td char="."><p xmlns="" class="SimplePara">5.00E-18</p></td><td><p xmlns="" class="SimplePara">Cytochrome P450</p></td><td><p xmlns="" class="SimplePara">Resist pathogenic bacterial</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Marker40081</p></td><td char="."><p xmlns="" class="SimplePara">2.00E-11</p></td><td><p xmlns="" class="SimplePara">Cytochrome P450</p></td><td><p xmlns="" class="SimplePara">Resist pathogenic bacterial</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Marker4015</p></td><td char="."><p xmlns="" class="SimplePara">2.00E-27</p></td><td><p xmlns="" class="SimplePara">Cytochrome P450</p></td><td><p xmlns="" class="SimplePara">Resist pathogenic bacterial</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Marker4728</p></td><td char="."><p xmlns="" class="SimplePara">2.00E-27</p></td><td><p xmlns="" class="SimplePara">Cytochrome P450</p></td><td><p xmlns="" class="SimplePara">Resist pathogenic bacterial</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Marker5389</p></td><td char="."><p xmlns="" class="SimplePara">1.00E-08</p></td><td><p xmlns="" class="SimplePara">Cytochrome P450</p></td><td><p xmlns="" class="SimplePara">Resist pathogenic bacterial</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Marker12565</p></td><td char="."><p xmlns="" class="SimplePara">6.00E-22</p></td><td><p xmlns="" class="SimplePara">Pc68LrkC4</p></td><td><p xmlns="" class="SimplePara">Activate chemical reactions</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Marker2312943</p></td><td char="."><p xmlns="" class="SimplePara">8.00E-16</p></td><td><p xmlns="" class="SimplePara">Pc68LrkC4</p></td><td><p xmlns="" class="SimplePara">Activate chemical reactions</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Marker2649</p></td><td char="."><p xmlns="" class="SimplePara">1.00E-24</p></td><td><p xmlns="" class="SimplePara">Prolamin gene</p></td><td><p xmlns="" class="SimplePara">Regulate stripe rust resistance</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Marker10408<sup xmlns="http://www.w3.org/1999/xhtml">a</sup>
                                          </p></td><td char="."><p xmlns="" class="SimplePara">1.00E-08</p></td><td><p xmlns="" class="SimplePara">Prolamin gene</p></td><td><p xmlns="" class="SimplePara">Regulate stripe rust resistance</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Marker21044</p></td><td char="."><p xmlns="" class="SimplePara">8.00E-21</p></td><td><p xmlns="" class="SimplePara">TaCab1</p></td><td><p xmlns="" class="SimplePara">Regulate stripe rust resistance</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">Note: <sup xmlns="http://www.w3.org/1999/xhtml">a</sup>means that makers have also been found in bulk analysis</p>
                    </div></figure>
                        </div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par38" class="Para">Four genes, including cytochrome P450, Pc68LrkC4, prolamin, and TaCab1, were discovered by detecting genes surrounding each candidate SNP (&lt;1&nbsp;Mb). Cytochrome P450 plays an important role in the synthesis of secondary metabolites of fungi and metabolism of foreign compounds [<span class="CitationRef"><a href="#CR35" data-toggle="citation">35</a></span>]. Studies on <em xmlns="" class="EmphasisTypeItalic">Populus tomentosa</em> infected by stripe rust showed that Cytochrome P450 is one of the genes related to germ defense [<span class="CitationRef"><a href="#CR36" data-toggle="citation">36</a></span>]. Pc68LrkC4 was isolated from <em xmlns="" class="EmphasisTypeItalic">Avena sativa</em> and contains a retrotransposon and repetitive DNA linked to a receptor kinase gene. It has been shown that the leaf rust resistance gene Lr10 of <em xmlns="" class="EmphasisTypeItalic">Triticum aestivum</em> encodes an extracellular receptor protein kinase with similar domains [<span class="CitationRef"><a href="#CR37" data-toggle="citation">37</a></span>]. Several studies on prolamin have found that proteins with the same or similar rust resistance are closely related to prolamin, and most of these cluster in dendrograms. For example, wheat stripe rust resistance was related to the genetic distance of prolamin, guessing it might because wheat stripe rust gene and partial encoded gene of prolamin were chained on 1BS [<span class="CitationRef"><a href="#CR38" data-toggle="citation">38</a></span>, <span class="CitationRef"><a href="#CR39" data-toggle="citation">39</a></span>]. In affinity reactions between wheat and stripe rust, the expression of the TaCab1 gene increases significantly. At addition, its expression can be induced by several different biotic stresses. The TaCab1 gene exhibits a marked change in expression after being treated with CaCl2. Researchers have guessed that TaCab1 plays a role in the interaction between wheat and stripe rust via Ca<sup>2+</sup> transport. In addition, TaCab1 gene silencing increases wheat’s resistance to stripe rust. Therefore, researchers have shown that the TaCab1 gene is correlated with susceptibility to stripe rust [<span class="CitationRef"><a href="#CR40" data-toggle="citation">40</a></span>]. In conclusion, these four candidate genes play important roles in disease progression, particularly the TaCab1 gene. This indicates that these candidate genes are likely to be critical in orchardgrass resistance to rust disease.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par39" class="Para">We also associated the results between GWAS analysis and bulk analysis to confirm these four predicted genes; however, two types of analysis only shared three markers representing two predicted genes including Cytochrome P450 and Prolamin gene (Table&nbsp;<span class="InternalRef"><a href="#Tab2">2</a></span>). Therefore, these two genes might be more reliable for regulation mechanism of rust resistance in orchgardgrass.</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec11" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Method" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Method">Method</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Method" data-test="collapse-section"><section id="Sec12" class="Section2 RenderAsSection2">
                <h3 class="Heading">Experimental materials</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par40" class="Para">A total of 33 orchardgrass accessions, with high level of resistance of disease (HR) and high susceptibility of disease (HS), as evaluated by Yan et al. [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>], were chosen for the experiment, including 17 with HR and 16 with HS in 2011 or 2012 (Table&nbsp;<span class="InternalRef"><a href="#Tab3">3</a></span>). The germplasm of orchardgrass used in this study consisted of 33 accessions that originated from Asia, Africa, Europe, and Oceania. For each accession, 10 individuals were randomly, and 0.5&nbsp;g of clean young leaves were selected per plant in 2014 for further DNA extraction.<figure xmlns="" id="Tab3" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 3</span>
                        <p xmlns="" class="SimplePara">The detailed information about accession number, name, origin, and rust resistance in orchardgrass for 2011 and 2012</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Number</p></th><th><p xmlns="" class="SimplePara">Accession name</p></th><th><p xmlns="" class="SimplePara">Origin</p></th><th><p xmlns="" class="SimplePara">2011</p></th><th><p xmlns="" class="SimplePara">2012</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">01819-6</p></td><td><p xmlns="" class="SimplePara">Beijing,China</p></td><td><p xmlns="" class="SimplePara">HS</p></td><td><p xmlns="" class="SimplePara">HS</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">2</p></td><td><p xmlns="" class="SimplePara">01824-2</p></td><td><p xmlns="" class="SimplePara">Beijing,China</p></td><td><p xmlns="" class="SimplePara">HR</p></td><td><p xmlns="" class="SimplePara">HR</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">3</p></td><td><p xmlns="" class="SimplePara">02122-5</p></td><td><p xmlns="" class="SimplePara">Hubei,China</p></td><td><p xmlns="" class="SimplePara">HR</p></td><td><p xmlns="" class="SimplePara">HR</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">4</p></td><td><p xmlns="" class="SimplePara">231469-1</p></td><td><p xmlns="" class="SimplePara">Libya</p></td><td><p xmlns="" class="SimplePara">HS</p></td><td><p xmlns="" class="SimplePara">HS</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">5</p></td><td><p xmlns="" class="SimplePara">2410-2</p></td><td><p xmlns="" class="SimplePara">Xinjiang,China</p></td><td><p xmlns="" class="SimplePara">HS</p></td><td><p xmlns="" class="SimplePara">HS</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">6</p></td><td><p xmlns="" class="SimplePara">2410-6</p></td><td><p xmlns="" class="SimplePara">Xinjiang,China</p></td><td><p xmlns="" class="SimplePara">HS</p></td><td><p xmlns="" class="SimplePara">HS</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">7</p></td><td><p xmlns="" class="SimplePara">2410-7</p></td><td><p xmlns="" class="SimplePara">Xinjiang,China</p></td><td><p xmlns="" class="SimplePara">HS</p></td><td><p xmlns="" class="SimplePara">HS</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">8</p></td><td><p xmlns="" class="SimplePara">287804-1</p></td><td><p xmlns="" class="SimplePara">Spain</p></td><td><p xmlns="" class="SimplePara">HS</p></td><td><p xmlns="" class="SimplePara">HS</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">9</p></td><td><p xmlns="" class="SimplePara">287804-2</p></td><td><p xmlns="" class="SimplePara">Spain</p></td><td><p xmlns="" class="SimplePara">HS</p></td><td><p xmlns="" class="SimplePara">HS</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">10</p></td><td><p xmlns="" class="SimplePara">287804-3</p></td><td><p xmlns="" class="SimplePara">Spain</p></td><td><p xmlns="" class="SimplePara">HS</p></td><td><p xmlns="" class="SimplePara">HS</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">11</p></td><td><p xmlns="" class="SimplePara">287804-4</p></td><td><p xmlns="" class="SimplePara">Spain</p></td><td><p xmlns="" class="SimplePara">HS</p></td><td><p xmlns="" class="SimplePara">HS</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">12</p></td><td><p xmlns="" class="SimplePara">287804-5</p></td><td><p xmlns="" class="SimplePara">Spain</p></td><td><p xmlns="" class="SimplePara">HS</p></td><td><p xmlns="" class="SimplePara">HS</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">13</p></td><td><p xmlns="" class="SimplePara">287804-8</p></td><td><p xmlns="" class="SimplePara">Spain</p></td><td><p xmlns="" class="SimplePara">HS</p></td><td><p xmlns="" class="SimplePara">HS</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">14</p></td><td><p xmlns="" class="SimplePara">292587-1</p></td><td><p xmlns="" class="SimplePara">Israel</p></td><td><p xmlns="" class="SimplePara">HS</p></td><td><p xmlns="" class="SimplePara">HS</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">302884-3</p></td><td><p xmlns="" class="SimplePara">Spain</p></td><td><p xmlns="" class="SimplePara">HS</p></td><td><p xmlns="" class="SimplePara">HS</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">16</p></td><td><p xmlns="" class="SimplePara">308794-1</p></td><td><p xmlns="" class="SimplePara">India</p></td><td><p xmlns="" class="SimplePara">HR</p></td><td><p xmlns="" class="SimplePara">HR</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">17</p></td><td><p xmlns="" class="SimplePara">308794-2</p></td><td><p xmlns="" class="SimplePara">India</p></td><td><p xmlns="" class="SimplePara">HR</p></td><td><p xmlns="" class="SimplePara">HR</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">18</p></td><td><p xmlns="" class="SimplePara">308794-3</p></td><td><p xmlns="" class="SimplePara">India</p></td><td><p xmlns="" class="SimplePara">HR</p></td><td><p xmlns="" class="SimplePara">HR</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">19</p></td><td><p xmlns="" class="SimplePara">308794-5</p></td><td><p xmlns="" class="SimplePara">India</p></td><td><p xmlns="" class="SimplePara">HR</p></td><td><p xmlns="" class="SimplePara">HR</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">20</p></td><td><p xmlns="" class="SimplePara">308794-7</p></td><td><p xmlns="" class="SimplePara">India</p></td><td><p xmlns="" class="SimplePara">HR</p></td><td><p xmlns="" class="SimplePara">HR</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">21</p></td><td><p xmlns="" class="SimplePara">308794-8</p></td><td><p xmlns="" class="SimplePara">India</p></td><td><p xmlns="" class="SimplePara">HR</p></td><td><p xmlns="" class="SimplePara">HR</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">22</p></td><td><p xmlns="" class="SimplePara">325293-2</p></td><td><p xmlns="" class="SimplePara">Russian Federation</p></td><td><p xmlns="" class="SimplePara">HR</p></td><td><p xmlns="" class="SimplePara">HR</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">23</p></td><td><p xmlns="" class="SimplePara">325293-4</p></td><td><p xmlns="" class="SimplePara">Russian Federation</p></td><td><p xmlns="" class="SimplePara">HR</p></td><td><p xmlns="" class="SimplePara">HR</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">24</p></td><td><p xmlns="" class="SimplePara">325293-5</p></td><td><p xmlns="" class="SimplePara">Russian Federation</p></td><td><p xmlns="" class="SimplePara">HR</p></td><td><p xmlns="" class="SimplePara">HR</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">25</p></td><td><p xmlns="" class="SimplePara">325293-6</p></td><td><p xmlns="" class="SimplePara">Russian Federation</p></td><td><p xmlns="" class="SimplePara">HR</p></td><td><p xmlns="" class="SimplePara">HR</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">26</p></td><td><p xmlns="" class="SimplePara">325293-7</p></td><td><p xmlns="" class="SimplePara">Russian Federation</p></td><td><p xmlns="" class="SimplePara">HR</p></td><td><p xmlns="" class="SimplePara">HR</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">27</p></td><td><p xmlns="" class="SimplePara">325293-8</p></td><td><p xmlns="" class="SimplePara">Russian Federation</p></td><td><p xmlns="" class="SimplePara">HR</p></td><td><p xmlns="" class="SimplePara">HR</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">28</p></td><td><p xmlns="" class="SimplePara">578635-7</p></td><td><p xmlns="" class="SimplePara">Morocco</p></td><td><p xmlns="" class="SimplePara">HS</p></td><td><p xmlns="" class="SimplePara">HS</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">29</p></td><td><p xmlns="" class="SimplePara">578635-8</p></td><td><p xmlns="" class="SimplePara">Morocco</p></td><td><p xmlns="" class="SimplePara">HS</p></td><td><p xmlns="" class="SimplePara">HS</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">30</p></td><td><p xmlns="" class="SimplePara">79-118-2</p></td><td><p xmlns="" class="SimplePara">Netherlands</p></td><td><p xmlns="" class="SimplePara">HS</p></td><td><p xmlns="" class="SimplePara">HS</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">31</p></td><td><p xmlns="" class="SimplePara">woronowii(H12)-3</p></td><td><p xmlns="" class="SimplePara">New Zealand</p></td><td><p xmlns="" class="SimplePara">HR</p></td><td><p xmlns="" class="SimplePara">HR</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">32</p></td><td><p xmlns="" class="SimplePara">woronowii(H12)-4</p></td><td><p xmlns="" class="SimplePara">New Zealand</p></td><td><p xmlns="" class="SimplePara">HR</p></td><td><p xmlns="" class="SimplePara">HR</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">33</p></td><td><p xmlns="" class="SimplePara">woronowii(H12)-7</p></td><td><p xmlns="" class="SimplePara">New Zealand</p></td><td><p xmlns="" class="SimplePara">HR</p></td><td><p xmlns="" class="SimplePara">HR</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">Note: <em class="EmphasisTypeItalic">HR</em> high resistance, <em class="EmphasisTypeItalic">HS</em> high sense of disease</p>
                    </div></figure>
                        </div>
              </section><section id="Sec13" class="Section2 RenderAsSection2">
                <h3 class="Heading">Specific-locus amplified fragment sequencing</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par41" class="Para">Total orchardgrass genomic DNA was extracted using the DNeasy Plant Mini Kit (Qiagen USA); the quality and quantity of DNA was then inspected using 0.8% gel electophoresis. The quantified DNA was diluted to 20&nbsp;μg/μL and was stored at −20&nbsp;°C before use.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par42" class="Para">The orchardgrass genomic DNA was analyzed with SLAF-seq [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>]. Sequencing libraries of each accession were constructed through digestion with the restriction enzyme HaeIII, (New England Biolabs, USA). A single nucleotide (A) overhang was added to the digested fragments with Klenow Fragment (3′ → 5´ exo–) (New England Biolabs, USA) and dATP at 37&nbsp;°C, and then duplex tag-labeled sequencing adapters (PAGE purified, Life Technologies, Beijing, China) were ligated to the A-tailed DNA with T4 DNA ligase. The PCR products were purified using Agencourt AMPure XP beads (Beckman Coulter, High Wycombe, UK) and pooled. The pooled sample was separated via electrophoresis in a 2% agarose gel. Fragments with indexes and adaptors from 400 to 450&nbsp;bp were excised and purified using a QIAquick Gel Extraction Kit (QIAGEN, Duesseldorf, Germany). Finally, the gel-purified product was sequenced using the Illumina HiSeq 2500 system (Illumina, Inc., San Diego, CA, USA). The average depth of sequencing was 6.7×. After sequencing, low-quality reads were filtered out (quality score &lt; 20). Reads with double ends were compared with similar sequences that could be labelled as a candidate SLAF to proceed with the next step. SNPs with low minor allele frequencies (&lt;5%) and low call frequencies (&lt;85%) were deleted [<span class="CitationRef"><a href="#CR32" data-toggle="citation">32</a></span>]. Finally, 33 samples and 2,309,777 SNPs remained for genome-wide association analysis (Additional file <span class="InternalRef"><a href="#MOESM7">7</a></span>: Table S2).</p>
              </section><section id="Sec14" class="Section2 RenderAsSection2">
                <h3 class="Heading">Phylogenetic analysis</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par43" class="Para">Based on the SNP genotype data from the 33 orchardgrass samples, a dendrogram was constructed in MEGA5 [<span class="CitationRef"><a href="#CR41" data-toggle="citation">41</a></span>] using the neighbor-joining algorithm [<span class="CitationRef"><a href="#CR42" data-toggle="citation">42</a></span>]. The structure of the orchardgrass population was analyzed using ADMIXTURE software based on SNP genotype data [<span class="CitationRef"><a href="#CR43" data-toggle="citation">43</a></span>]. The pre-defined K, which indicates the number of groups in a population, varied from 1 to 10 in ADMIXTURE models. A K value was selected when the estimate of lnPr(X|K) peaked in the range of 1 to 10 subpopulations. Delta K (<span class="stix">△</span>K), as an ad hoc quantity related to the second order change in the log probability of the data with respect to the number of clusters, was considered as the most probable value of K according to the model choice criteria [<span class="CitationRef"><a href="#CR42" data-toggle="citation">42</a></span>]. A PCA approach with the cluster software [<span class="CitationRef"><a href="#CR44" data-toggle="citation">44</a></span>] was used to cluster the orchardgrass population.</p>
              </section><section id="Sec15" class="Section2 RenderAsSection2">
                <h3 class="Heading">Evaluation of linkage disequilibrium and GWAS analysis</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par44" class="Para">The squared correlations (r<sup>2</sup>) between all combinations of SNPs (2,309,777) were used to evaluate the significance of pairwise linkage disequilibrium (LD) using Haploview [<span class="CitationRef"><a href="#CR45" data-toggle="citation">45</a></span>] (<span class="ExternalRef"><a href="http://www.softpedia.com/get/Science-CAD/Haploview.shtml"><span class="RefSource">http://www.softpedia.com/get/Science-CAD/Haploview.shtml</span></a></span>). Each pair of SNPs was considered to have strong linkage if r<sup>2</sup> &gt; 0.8. Based on SNP genotype data, the GWAS analysis was conducted using a GLM in the TASSEL software [<span class="CitationRef"><a href="#CR43" data-toggle="citation">43</a></span>] as follows:<div id="Equa" class="Equation EquationMathjax"><div class="EquationContent">$$ \mathrm{Y}=\mathrm{X}\upalpha +\mathrm{Q}\upbeta +\mathrm{K}\upmu +\mathrm{e}\mathrm{Y} $$</div></div>
                        </div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par45" class="Para">where y is the phenotype value, Q is the population structure matrix calculated by the ADMIXTURE program, X is the genotype matrix, α is the effect of genotype, β is the effect of population structure, μ is the effect of kinship, and e is the residual error. Finally, each SNP locus was assigned a value associated with related traits.</p>
              </section><section id="Sec16" class="Section2 RenderAsSection2">
                <h3 class="Heading">Bulk analysis</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par46" class="Para">Two groups (HS and HR) were used to conduct a bulk analysis, and the procedures were followed as previously described [<span class="CitationRef"><a href="#CR46" data-toggle="citation">46</a></span>]. To identify potential trait-retaled SNPs, we aligned the short reads obtained from the two DNA bulks to the reference genome (sequences from the HR group were regarded as reference genome) using bwa software [<span class="CitationRef"><a href="#CR47" data-toggle="citation">47</a></span>], and alignment files were converted to SAM/BAM files using SAMtools [<span class="CitationRef"><a href="#CR48" data-toggle="citation">48</a></span>]. SNP-index was calculated for all the SNP positions. We further excluded SNP positions with SNP-index of &lt;0.3 from the two sequences to avoid spurious SNPs called due to alignment errors.</p>
              </section><section id="Sec17" class="Section2 RenderAsSection2">
                <h3 class="Heading">Bioinformatics analysis of candidate regions</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par47" class="Para">The genetic data relating to rust traits of a 500-Kb window surrounding each SNP were downloaded from Ensembl (<span class="ExternalRef"><a href="http://ensemblgenomes.org/"><span class="RefSource">http://ensemblgenomes.org/</span></a></span>) and NCBI (<span class="ExternalRef"><a href="http://www.ncbi.nlm.nih.gov/"><span class="RefSource">http://www.ncbi.nlm.nih.gov/</span></a></span>). The NT databases were used to conduct pathway analysis and functional annotation for predicted genes [<span class="CitationRef"><a href="#CR49" data-toggle="citation">49</a></span>].</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec18" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Conclusions" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Conclusions">Conclusions</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Conclusions" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par48" class="Para">In this study, SLAF-seq technology was used to conduct the GWAS in orchardgrass, and a number of 2,334,889 SLAF tags were produced to generate 2,309,777 SNPs. The GWAS showed that a total number of 4,604 SNPs were significantly (<em xmlns="" class="EmphasisTypeItalic">P</em> &lt; 0.01) associated with the rust trait, while a total number of 5,211 SNPs were filtered by bulk analysis. Based on bioinformatics analysis for GWAS results, four candidate genes, including cytochrome P450, Pc68LrkC4, prolamin, and TaCab1 were predicted involving in disease resistance, and the bulk analysis further confirmed cytochrome P450 and prolamin genes were reliable. Besides, further research should be done to determine how these two genes work for resisting diseases. The results can also provide basic information for MAS of rust-resistant lineages.</p></div></section>
          <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AbbreviationGroup Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Abbreviations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Abbreviations">Abbreviations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Abbreviations"><div class="DefinitionList">
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>ACD6:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par4" class="Para">Cell death 6</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>G1:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par5" class="Para">Group 1</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>G2:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par6" class="Para">Group 2</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>GBS:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par7" class="Para">Genotyping-by-sequencing</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>GLM:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par8" class="Para">General linear model</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>GWAS:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par9" class="Para">Genome-wide association study</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>HR:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par10" class="Para">High level of resistance of disease</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>HS:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par11" class="Para">High susceptibility of disease</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>LD:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par12" class="Para">Linkage disequilibrium</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>MAS:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par13" class="Para">Marker-assisted selection</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>NT:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par14" class="Para">Nucleotide sequences</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>PC:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par15" class="Para">Principal component analysis</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>PC1:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par16" class="Para">Principal Component 1</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>QTLs:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par17" class="Para">Quantitative trait loci</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>RADseq:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par18" class="Para">Restriction site-associated sequencing</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>RIL:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par19" class="Para">Recombinant inbred line</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>SLAF-seq:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par20" class="Para">Specific-locus amplified fragment sequencing</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>TaCab1:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par21" class="Para">EF-hand Ca<sup>2+</sup>-binding protein</p>
                  </dd></dl>
              </div></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments"><h3 xmlns="" class="Heading">Acknowledgments</h3><p xmlns="" class="SimplePara">This work was supported by the National Basic Research Program of China (973 Program) (2014CB138705) and the National Natural Science Foundation of China, NSFC (31101760).</p><div id="FPar1" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Funding</h3>
                <p id="Par49" class="Para">This work was supported by the National Basic Research Program of China (973 Program) (2014CB138705), Science and technology commission of livelihoods and social security project, and construction and demonstration promotion of grass and sheep matching model under whole barn feeding conditions for goats in Chongqing (cstc2016shmszx0473), Agriculture committee project and forage grass laboratory construction project of goat industrial technology system in Chongqing, and Funding project of innovation team construction in chongqing universities: modern production technology of beef (CXTDG201602003). And these four programs contribute to the design of the study, collection, analysis, interpretation of data, and in writing the manuscript.</p>
              </div><div id="FPar2" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Availability of data and materials</h3>
                <p id="Par50" class="Para">We have provided detailed information about materials and method in our manuscript, so we will not provide data and supporting materials in this section.</p>
              </div><div id="FPar3" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Authors’ contributions</h3>
                <p id="Par51" class="Para">BZ and LH conceived and designed the experiments; HY and BZ performed the experiments; HY, XL and BZ analyzed the data; WZ, AZ, SZ, JL and HY contributed reagents/materials/analysis tools; HY, WZ, AZ and SZ wrote the paper. All authors read and approved the final manuscript.</p>
              </div><div id="FPar4" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Competing interests</h3>
                <p id="Par52" class="Para">The authors declare that they have no competing interests.</p>
              </div><div id="FPar5" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Consent for publication</h3>
                <p id="Par53" class="Para">Not applicable.</p>
              </div><div id="FPar6" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Ethics approval and consent to participate</h3>
                <p id="Par54" class="Para">Not applicable.</p>
              </div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="License">
                <p xmlns="" class="SimplePara">
                           <strong class="EmphasisTypeBold">Open Access</strong>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/4.0/"><span class="RefSource">http://creativecommons.org/licenses/by/4.0/</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/publicdomain/zero/1.0/"><span class="RefSource">http://creativecommons.org/publicdomain/zero/1.0/</span></a></span>) applies to the data made available in this article, unless otherwise stated.</p>
              </div><aside class="Appendix" id="App1">
              
                <h3 class="Heading">Additional files</h3><div id="widgetContainer" data-test="figshare-container"></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par55" class="Para">
                  <div class="MediaObject" id="MOESM1">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0027-3/MediaObjects/41065_2017_27_MOESM1_ESM.docx"><span class="caption-number">Additional file 1:</span></a>
                          <span xmlns="" class="SimplePara">(DOCX 21&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM2">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0027-3/MediaObjects/41065_2017_27_MOESM2_ESM.xlsx"><span class="caption-number">Additional file 2:</span></a>
                          <span xmlns="" class="SimplePara">(XLSX 181&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM3">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0027-3/MediaObjects/41065_2017_27_MOESM3_ESM.xlsx"><span class="caption-number">Additional file 3:</span></a>
                          <span xmlns="" class="SimplePara">(XLSX 173&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM4">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0027-3/MediaObjects/41065_2017_27_MOESM4_ESM.xlsx"><span class="caption-number">Additional file 4:</span></a>
                          <span xmlns="" class="SimplePara">(XLSX 79&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM5">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0027-3/MediaObjects/41065_2017_27_MOESM5_ESM.xlsx"><span class="caption-number">Additional file 5:</span></a>
                          <span xmlns="" class="SimplePara">(XLSX 328&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM6">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0027-3/MediaObjects/41065_2017_27_MOESM6_ESM.xlsx"><span class="caption-number">Additional file 6:</span></a>
                          <span xmlns="" class="SimplePara">(XLSX 50&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM7">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0027-3/MediaObjects/41065_2017_27_MOESM7_ESM.docx"><span class="caption-number">Additional file 7:</span></a>
                          <span xmlns="" class="SimplePara">(DOCX 19&nbsp;kb)</span>
                        </div></div>
                  </div>
                </div>
              
            </aside></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authors’ Affiliations</a></h2><div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1"><span class="AffiliationNumber">(1)</span><div class="AffiliationText">Department of Animal Science, Southwest University, Rongchang, Chongqing, 402460, China</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff2"><span class="AffiliationNumber">(2)</span><div class="AffiliationText">Department of Grassland Science, Animal Science and Technology College, Sichuan Agricultural University, Chengdu, 611130, China</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff3"><span class="AffiliationNumber">(3)</span><div class="AffiliationText">Agricultural College, Sichuan Agricultural University, Chengdu, 611130, China</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff4"><span class="AffiliationNumber">(4)</span><div class="AffiliationText">College of Life Science, China West Normal University, Nanchong, 637009, China</div></div></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper"><li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Stewart AV, Ellison NW. The genus <em class="EmphasisTypeItalic">Dactylis</em>; wealth of wild species: role in plant genome elucidation and improvement. New York: Springer; 2010.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20genus%20Dactylis%3B%20wealth%20of%20wild%20species%3A%20role%20in%20plant%20genome%20elucidation%20and%20improvement&amp;author=AV.%20Stewart&amp;author=NW.%20Ellison&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Xie WG, Zhang XQ, Cai HW, Liu W, Peng Y. Genetic diversity analysis and transferability of cereal EST-SSR markers to orchardgrass (<em class="EmphasisTypeItalic">Dactylis glomerata</em> L.). Biochem Syst Ecol. 2010;38(4):740–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.bse.2010.06.009"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20analysis%20and%20transferability%20of%20cereal%20EST-SSR%20markers%20to%20orchardgrass%20%28Dactylis%20glomerata%20L.%29&amp;author=WG.%20Xie&amp;author=XQ.%20Zhang&amp;author=HW.%20Cai&amp;author=W.%20Liu&amp;author=Y.%20Peng&amp;journal=Biochem%20Syst%20Ecol&amp;volume=38&amp;issue=4&amp;pages=740-9&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zeng B, Zhang Y, Huang LK, Jiang XM, Luo D, Yin GH. Genetic diversity of orchardgrass (<em class="EmphasisTypeItalic">Dactylis glomerata</em> L.) germplasms with resistance to rust diseases revealed by Start Codon Targeted (SCoT) markers. Biochem Syst Ecol. 2014;54:96–102.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.bse.2013.12.028"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20of%20orchardgrass%20%28Dactylis%20glomerata%20L.%29%20germplasms%20with%20resistance%20to%20rust%20diseases%20revealed%20by%20Start%20Codon%20Targeted%20%28SCoT%29%20markers&amp;author=B.%20Zeng&amp;author=Y.%20Zhang&amp;author=LK.%20Huang&amp;author=XM.%20Jiang&amp;author=D.%20Luo&amp;author=GH.%20Yin&amp;journal=Biochem%20Syst%20Ecol&amp;volume=54&amp;pages=96-102&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Tajimi A. Differentiation and maintenance of three graminicolous formae speciales of stem rust with clonal lines of orchardgrass, <em class="EmphasisTypeItalic">Dactylis glomerata</em> L. J Jpan Grassl Sci. 1975;20:26–30.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Differentiation%20and%20maintenance%20of%20three%20graminicolous%20formae%20speciales%20of%20stem%20rust%20with%20clonal%20lines%20of%20orchardgrass%2C%20Dactylis%20glomerata%20L&amp;author=A.%20Tajimi&amp;journal=J%20Jpan%20Grassl%20Sci&amp;volume=20&amp;pages=26-30&amp;publication_year=1975"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Miller TL, Carlson IT. Breeding for rust resistance in orchardgrass by phenotypic and phenotypic-genotypic selection. Crop Sci. 1982;22(6):1218–21.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2135/cropsci1982.0011183X002200060031x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Breeding%20for%20rust%20resistance%20in%20orchardgrass%20by%20phenotypic%20and%20phenotypic-genotypic%20selection&amp;author=TL.%20Miller&amp;author=IT.%20Carlson&amp;journal=Crop%20Sci&amp;volume=22&amp;issue=6&amp;pages=1218-21&amp;publication_year=1982"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yan HD, Zeng B, Zhang XQ, Cheng L, Miller S, Huang LK. Screening orchardgrass (<em class="EmphasisTypeItalic">Dactylis glomerata</em> L.) germplasm for high rust resistance and high summer survival rate in a subtropical climate. Grassl Sci. 2013;59(4):205–10.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/grs.12030"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Screening%20orchardgrass%20%28Dactylis%20glomerata%20L.%29%20germplasm%20for%20high%20rust%20resistance%20and%20high%20summer%20survival%20rate%20in%20a%20subtropical%20climate&amp;author=HD.%20Yan&amp;author=B.%20Zeng&amp;author=XQ.%20Zhang&amp;author=L.%20Cheng&amp;author=S.%20Miller&amp;author=LK.%20Huang&amp;journal=Grassl%20Sci&amp;volume=59&amp;issue=4&amp;pages=205-10&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Todesco M, Balasubramanian S, Hu TT, Traw MB, Horton M, Epple P, et al. Natural allelic variation underlying a major fitness trade-off in <em class="EmphasisTypeItalic">Arabidopsis thaliana</em>. Nature. 2010;465(7298):632–6.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nature09083"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20520716"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3055268"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Natural%20allelic%20variation%20underlying%20a%20major%20fitness%20trade-off%20in%20Arabidopsis%20thaliana%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=M.%20Todesco&amp;author=S.%20Balasubramanian&amp;author=TT.%20Hu&amp;author=MB.%20Traw&amp;author=M.%20Horton&amp;author=P.%20Epple&amp;journal=Nature&amp;volume=465&amp;issue=7298&amp;pages=632-6&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li Y, Huang Y, Bergelson J, Nordborg M, Borevitz JO. Association mapping of local climate-sensitive quantitative trait loci in <em class="EmphasisTypeItalic">Arabidopsis thaliana</em>. PNAS. 2010;107(49):21199–204.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.1007431107"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21078970"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3000268"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Association%20mapping%20of%20local%20climate-sensitive%20quantitative%20trait%20loci%20in%20Arabidopsis%20thaliana%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=Y.%20Li&amp;author=Y.%20Huang&amp;author=J.%20Bergelson&amp;author=M.%20Nordborg&amp;author=JO.%20Borevitz&amp;journal=PNAS&amp;volume=107&amp;issue=49&amp;pages=21199-204&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Tian F, Bradbury PJ, Brown PJ, Hung H, Sun Q, Flint-Garcia S, et al. Genome-wide association study of leaf architecture in the maize nested association mapping population. Nat Genet. 2011;43(2):159–62.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/ng.746"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21217756"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20study%20of%20leaf%20architecture%20in%20the%20maize%20nested%20association%20mapping%20population&amp;author=F.%20Tian&amp;author=PJ.%20Bradbury&amp;author=PJ.%20Brown&amp;author=H.%20Hung&amp;author=Q.%20Sun&amp;author=S.%20Flint-Garcia&amp;journal=Nat%20Genet&amp;volume=43&amp;issue=2&amp;pages=159-62&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kollers S, Rodemann B, Ling J, Korzun V, Ebmeyer E, Argillier O, et al. Whole genome association mapping of Fusarium head blight resistance in European winter wheat (<em class="EmphasisTypeItalic">Triticum aestivum</em> L.). PLoS One. 2013;8(2), e57500.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0057500"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23451238"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3579808"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Whole%20genome%20association%20mapping%20of%20Fusarium%20head%20blight%20resistance%20in%20European%20winter%20wheat%20%28Triticum%20aestivum%20L.%29&amp;author=S.%20Kollers&amp;author=B.%20Rodemann&amp;author=J.%20Ling&amp;author=V.%20Korzun&amp;author=E.%20Ebmeyer&amp;author=O.%20Argillier&amp;journal=PLoS%20One&amp;volume=8&amp;issue=2&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Begum H, Spindel JE, Lalusin A, Borromeo T, Gregorio G, Hernandez J, et al. Genome-wide association mapping for yield and other agronomic traits in an elite breeding population of tropical rice (<em class="EmphasisTypeItalic">Oryza sativa</em>). PLoS One. 2015;10(3), e0119873.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0119873"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25785447"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4364887"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20mapping%20for%20yield%20and%20other%20agronomic%20traits%20in%20an%20elite%20breeding%20population%20of%20tropical%20rice%20%28Oryza%20sativa%29&amp;author=H.%20Begum&amp;author=JE.%20Spindel&amp;author=A.%20Lalusin&amp;author=T.%20Borromeo&amp;author=G.%20Gregorio&amp;author=J.%20Hernandez&amp;journal=PLoS%20One&amp;volume=10&amp;issue=3&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kumar V, Singh A, Mithra SVA, Krishnamurthy SL, Parida SK, Jain S, et al. Genome-wide association mapping of salinity tolerance in rice (<em class="EmphasisTypeItalic">Oryza sativa</em>). DNA Res. 2015;22(2):133–45.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/dnares/dsu046"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25627243"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4401324"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20mapping%20of%20salinity%20tolerance%20in%20rice%20%28Oryza%20sativa%29&amp;author=V.%20Kumar&amp;author=A.%20Singh&amp;author=SVA.%20Mithra&amp;author=SL.%20Krishnamurthy&amp;author=SK.%20Parida&amp;author=S.%20Jain&amp;journal=DNA%20Res&amp;volume=22&amp;issue=2&amp;pages=133-45&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yang WN, Guo ZL, Huang CL, Duan LF, Chen GX, Jiang N, et al. Combining high-throughput phenotyping and genome-wide association studies to reveal natural genetic variation in rice. Nat Commun. 2014;5.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Yang%20WN%2C%20Guo%20ZL%2C%20Huang%20CL%2C%20Duan%20LF%2C%20Chen%20GX%2C%20Jiang%20N%2C%20et%20al.%20Combining%20high-throughput%20phenotyping%20and%20genome-wide%20association%20studies%20to%20reveal%20natural%20genetic%20variation%20in%20rice.%20Nat%20Commun.%202014%3B5."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Stracke S, Haseneyer G, Veyrieras J-B, Geiger HH, Sauer S, Graner A, et al. Association mapping reveals gene action and interactions in the determination of flowering time in barley. Theor Appl Genet. 2009;118(2):259–73.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00122-008-0896-y"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18830577"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Association%20mapping%20reveals%20gene%20action%20and%20interactions%20in%20the%20determination%20of%20flowering%20time%20in%20barley&amp;author=S.%20Stracke&amp;author=G.%20Haseneyer&amp;author=J-B.%20Veyrieras&amp;author=HH.%20Geiger&amp;author=S.%20Sauer&amp;author=A.%20Graner&amp;journal=Theor%20Appl%20Genet&amp;volume=118&amp;issue=2&amp;pages=259-73&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Singh A, Reimer S, Pozniak CJ, Clarke FR, Clarke JM, Knox RE, et al. Allelic variation at Psy1-A1 and association with yellow pigment in durum wheat grain. Theor Appl Genet. 2009;118(8):1539–48.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00122-009-1001-x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19319504"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Allelic%20variation%20at%20Psy1-A1%20and%20association%20with%20yellow%20pigment%20in%20durum%20wheat%20grain&amp;author=A.%20Singh&amp;author=S.%20Reimer&amp;author=CJ.%20Pozniak&amp;author=FR.%20Clarke&amp;author=JM.%20Clarke&amp;author=RE.%20Knox&amp;journal=Theor%20Appl%20Genet&amp;volume=118&amp;issue=8&amp;pages=1539-48&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li YH, Zhang C, Gao ZS, Smulders MJM, Ma Z, Liu ZX, et al. Development of SNP markers and haplotype analysis of the candidate gene for rhg1, which confers resistance to soybean cyst nematode in soybean. Mol Breed. 2009;24(1):63–76.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11032-009-9272-0"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Development%20of%20SNP%20markers%20and%20haplotype%20analysis%20of%20the%20candidate%20gene%20for%20rhg1%2C%20which%20confers%20resistance%20to%20soybean%20cyst%20nematode%20in%20soybean&amp;author=YH.%20Li&amp;author=C.%20Zhang&amp;author=ZS.%20Gao&amp;author=MJM.%20Smulders&amp;author=Z.%20Ma&amp;author=ZX.%20Liu&amp;journal=Mol%20Breed&amp;volume=24&amp;issue=1&amp;pages=63-76&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Pasam RK, Sharma R, Malosetti M, van Eeuwijk FA, Haseneyer G, Kilian B, et al. Genome-wide association studies for agronomical traits in a world wide spring barley collection. BMC Plant Biol. 2012;12(1):16.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2229-12-16"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22284310"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3349577"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20studies%20for%20agronomical%20traits%20in%20a%20world%20wide%20spring%20barley%20collection&amp;author=RK.%20Pasam&amp;author=R.%20Sharma&amp;author=M.%20Malosetti&amp;author=FA.%20Eeuwijk&amp;author=G.%20Haseneyer&amp;author=B.%20Kilian&amp;journal=BMC%20Plant%20Biol&amp;volume=12&amp;issue=1&amp;pages=16&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yu LX, Morgounov A, Wanyera R, Keser M, Singh SK, Sorrells M. Identification of Ug99 stem rust resistance loci in winter wheat germplasm using genome-wide association analysis. Theor Appl Genet. 2012;125(4):749–58.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00122-012-1867-x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22534791"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Identification%20of%20Ug99%20stem%20rust%20resistance%20loci%20in%20winter%20wheat%20germplasm%20using%20genome-wide%20association%20analysis&amp;author=LX.%20Yu&amp;author=A.%20Morgounov&amp;author=R.%20Wanyera&amp;author=M.%20Keser&amp;author=SK.%20Singh&amp;author=M.%20Sorrells&amp;journal=Theor%20Appl%20Genet&amp;volume=125&amp;issue=4&amp;pages=749-58&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Montilla-Bascón G, Rispail N, Sánchez-Martín J, Rubiales D, Mur LAJ, Langdon T, et al. Genome-wide association study for crown rust (<em class="EmphasisTypeItalic">Puccinia coronata</em> f. sp. avenae) and powdery mildew (<em class="EmphasisTypeItalic">Blumeria graminis</em> f. sp. avenae) resistance in an oat (<em class="EmphasisTypeItalic">Avena sativa</em>) collection of commercial varieties and landraces. Front Plant Sci. 2015;6:1–11.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.3389/fpls.2015.00103"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20study%20for%20crown%20rust%20%28Puccinia%20coronata%20f.%20sp.%20avenae%29%20and%20powdery%20mildew%20%28Blumeria%20graminis%20f.%20sp.%20avenae%29%20resistance%20in%20an%20oat%20%28Avena%20sativa%29%20collection%20of%20commercial%20varieties%20and%20landraces&amp;author=G.%20Montilla-Basc%C3%B3n&amp;author=N.%20Rispail&amp;author=J.%20S%C3%A1nchez-Mart%C3%ADn&amp;author=D.%20Rubiales&amp;author=LAJ.%20Mur&amp;author=T.%20Langdon&amp;journal=Front%20Plant%20Sci&amp;volume=6&amp;pages=1-11&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Qi ZM, Huang L, Zhu RS, Xin DW, Liu CY, Han X, et al. A high-density genetic map for soybean based on specific length amplified fragment sequencing. PLoS One. 2014;9(11), e114349.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0114349"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20high-density%20genetic%20map%20for%20soybean%20based%20on%20specific%20length%20amplified%20fragment%20sequencing&amp;author=ZM.%20Qi&amp;author=L.%20Huang&amp;author=RS.%20Zhu&amp;author=DW.%20Xin&amp;author=CY.%20Liu&amp;author=X.%20Han&amp;journal=PLoS%20One&amp;volume=9&amp;issue=11&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sun XW, Liu DY, Zhang XF, Li WB, Liu H, Hong WG, et al. SLAF-seq: an efficient method of large-scale de novo SNP discovery and genotyping using high-throughput sequencing. PLoS One. 2013;8(3), e58700.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0058700"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23527008"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3602454"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=SLAF-seq%3A%20an%20efficient%20method%20of%20large-scale%20de%20novo%20SNP%20discovery%20and%20genotyping%20using%20high-throughput%20sequencing&amp;author=XW.%20Sun&amp;author=DY.%20Liu&amp;author=XF.%20Zhang&amp;author=WB.%20Li&amp;author=H.%20Liu&amp;author=WG.%20Hong&amp;journal=PLoS%20One&amp;volume=8&amp;issue=3&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Brachi B, Morris GP, Borevitz JO. Genome-wide association studies in plants: the missing heritability is in the field. Genome Biol. 2011;12(10):232.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/gb-2011-12-10-232"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22035733"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3333769"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20studies%20in%20plants%3A%20the%20missing%20heritability%20is%20in%20the%20field&amp;author=B.%20Brachi&amp;author=GP.%20Morris&amp;author=JO.%20Borevitz&amp;journal=Genome%20Biol&amp;volume=12&amp;issue=10&amp;pages=232&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wang WH, Zhang T, Zhang GX, Wang JY, Han KP, Wang YJ, et al. Genome-wide association study of antibody level response to NDV and IBV in Jinghai yellow chicken based on SLAF-seq technology. J Appl Genet. 2015;1–9.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Wang%20WH%2C%20Zhang%20T%2C%20Zhang%20GX%2C%20Wang%20JY%2C%20Han%20KP%2C%20Wang%20YJ%2C%20et%20al.%20Genome-wide%20association%20study%20of%20antibody%20level%20response%20to%20NDV%20and%20IBV%20in%20Jinghai%20yellow%20chicken%20based%20on%20SLAF-seq%20technology.%20J%20Appl%20Genet.%202015%3B1%E2%80%939."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR24" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li B, Tian L, Zhang JY, Huang L, Han FX, Yan SR, et al. Construction of a high-density genetic map based on large-scale markers developed by specific length amplified fragment sequencing (SLAF-seq) and its application to QTL analysis for isoflavone content in <em class="EmphasisTypeItalic">Glycine max</em>. BMC Genomics. 2014;15(1):1086.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2164-15-1086"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25494922"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4320444"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20high-density%20genetic%20map%20based%20on%20large-scale%20markers%20developed%20by%20specific%20length%20amplified%20fragment%20sequencing%20%28SLAF-seq%29%20and%20its%20application%20to%20QTL%20analysis%20for%20isoflavone%20content%20in%20Glycine%20max%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=B.%20Li&amp;author=L.%20Tian&amp;author=JY.%20Zhang&amp;author=L.%20Huang&amp;author=FX.%20Han&amp;author=SR.%20Yan&amp;journal=BMC%20Genomics&amp;volume=15&amp;issue=1&amp;pages=1086&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR25" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wei QZ, Wang YZ, Qin XD, Zhang YX, Zhang ZT, Wang J, et al. An SNP-based saturated genetic map and QTL analysis of fruit-related traits in cucumber using specific-length amplified fragment (SLAF) sequencing. BMC Genomics. 2014;15(1):1158.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2164-15-1158"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25534138"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4367881"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=An%20SNP-based%20saturated%20genetic%20map%20and%20QTL%20analysis%20of%20fruit-related%20traits%20in%20cucumber%20using%20specific-length%20amplified%20fragment%20%28SLAF%29%20sequencing&amp;author=QZ.%20Wei&amp;author=YZ.%20Wang&amp;author=XD.%20Qin&amp;author=YX.%20Zhang&amp;author=ZT.%20Zhang&amp;author=J.%20Wang&amp;journal=BMC%20Genomics&amp;volume=15&amp;issue=1&amp;pages=1158&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR26" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhang YX, Wang LH, Xin HG, Li DH, Ma CX, Ding X, et al. Construction of a high-density genetic map for sesame based on large scale marker development by specific length amplified fragment (SLAF) sequencing. BMC Plant Biol. 2013;13(1):141.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2229-13-141"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24060091"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3852768"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20high-density%20genetic%20map%20for%20sesame%20based%20on%20large%20scale%20marker%20development%20by%20specific%20length%20amplified%20fragment%20%28SLAF%29%20sequencing&amp;author=YX.%20Zhang&amp;author=LH.%20Wang&amp;author=HG.%20Xin&amp;author=DH.%20Li&amp;author=CX.%20Ma&amp;author=X.%20Ding&amp;journal=BMC%20Plant%20Biol&amp;volume=13&amp;issue=1&amp;pages=141&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR27" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Xu XW, Xu RX, Zhu BY, Yu T, Qu WQ, Lu L, et al. A high-density genetic map of cucumber derived from Specific Length Amplified Fragment sequencing (SLAF-seq). Front Plant Sci. 2014;5.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Xu%20XW%2C%20Xu%20RX%2C%20Zhu%20BY%2C%20Yu%20T%2C%20Qu%20WQ%2C%20Lu%20L%2C%20et%20al.%20A%20high-density%20genetic%20map%20of%20cucumber%20derived%20from%20Specific%20Length%20Amplified%20Fragment%20sequencing%20%28SLAF-seq%29.%20Front%20Plant%20Sci.%202014%3B5."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR28" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Visscher PM, Brown MA, McCarthy MI, Yang J. Five years of GWAS discovery. Am J Hum Genet. 2012;90(1):7–24.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.ajhg.2011.11.029"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22243964"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3257326"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Five%20years%20of%20GWAS%20discovery&amp;author=PM.%20Visscher&amp;author=MA.%20Brown&amp;author=MI.%20McCarthy&amp;author=J.%20Yang&amp;journal=Am%20J%20Hum%20Genet&amp;volume=90&amp;issue=1&amp;pages=7-24&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR29" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhao X, Han YP, Li YH, Liu DY, Sun MM, Zhao Y, et al. Loci and candidate gene identification for resistance to Sclerotinia sclerotiorum in soybean (<em class="EmphasisTypeItalic">Glycine max</em> L. Merr.) viaassociation and linkage maps. Plant J. 2015;82(2):245–55.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/tpj.12810"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25736370"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Loci%20and%20candidate%20gene%20identification%20for%20resistance%20to%20Sclerotinia%20sclerotiorum%20in%20soybean%20%28Glycine%20max%20L.%20Merr.%29%20viaassociation%20and%20linkage%20maps&amp;author=X.%20Zhao&amp;author=YP.%20Han&amp;author=YH.%20Li&amp;author=DY.%20Liu&amp;author=MM.%20Sun&amp;author=Y.%20Zhao&amp;journal=Plant%20J&amp;volume=82&amp;issue=2&amp;pages=245-55&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR30" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Han Y, Zhao X, Liu D, Li Y, Lightfoot DA, Yang Z, et al. Domestication footprints anchor genomic regions of agronomic importance in soybeans. New Phytol. 2016;209(2):871–84.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/nph.13626"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26479264"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Domestication%20footprints%20anchor%20genomic%20regions%20of%20agronomic%20importance%20in%20soybeans&amp;author=Y.%20Han&amp;author=X.%20Zhao&amp;author=D.%20Liu&amp;author=Y.%20Li&amp;author=DA.%20Lightfoot&amp;author=Z.%20Yang&amp;journal=New%20Phytol&amp;volume=209&amp;issue=2&amp;pages=871-84&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR31" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Han YP, Zhao X, Cao GL, Wang Y, Li YH, Liu DY, et al. Genetic characteristics of soybean resistance to HG type 0 and HG type 1.2.3.5.7 of the cyst nematode analyzed by genome-wide association mapping. BMC Genom. 2015;16(1):1–11.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2164-16-1"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20characteristics%20of%20soybean%20resistance%20to%20HG%20type%200%20and%20HG%20type%201.2.3.5.7%20of%20the%20cyst%20nematode%20analyzed%20by%20genome-wide%20association%20mapping&amp;author=YP.%20Han&amp;author=X.%20Zhao&amp;author=GL.%20Cao&amp;author=Y.%20Wang&amp;author=YH.%20Li&amp;author=DY.%20Liu&amp;journal=BMC%20Genom&amp;volume=16&amp;issue=1&amp;pages=1-11&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR32" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MAR, Bender D, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007;81(3):559–75.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1086/519795"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17701901"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1950838"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=PLINK%3A%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&amp;author=S.%20Purcell&amp;author=B.%20Neale&amp;author=K.%20Todd-Brown&amp;author=L.%20Thomas&amp;author=MAR.%20Ferreira&amp;author=D.%20Bender&amp;journal=Am%20J%20Hum%20Genet&amp;volume=81&amp;issue=3&amp;pages=559-75&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR33" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Pritchard JK, Stephens M, Rosenberg NA, Donnelly P. Association mapping in structured populations. Am J Hum Genet. 2000;67(1):170–81.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1086/302959"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10827107"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1287075"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Association%20mapping%20in%20structured%20populations&amp;author=JK.%20Pritchard&amp;author=M.%20Stephens&amp;author=NA.%20Rosenberg&amp;author=P.%20Donnelly&amp;journal=Am%20J%20Hum%20Genet&amp;volume=67&amp;issue=1&amp;pages=170-81&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR34" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hamblin MT, Close TJ, Bhat PR, Chao S, Kling JG, Abraham KJ, et al. Population structure and linkage disequilibrium in US barley germplasm: implications for association mapping. Crop Sci. 2010;50(2):556–66.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2135/cropsci2009.04.0198"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Population%20structure%20and%20linkage%20disequilibrium%20in%20US%20barley%20germplasm%3A%20implications%20for%20association%20mapping&amp;author=MT.%20Hamblin&amp;author=TJ.%20Close&amp;author=PR.%20Bhat&amp;author=S.%20Chao&amp;author=JG.%20Kling&amp;author=KJ.%20Abraham&amp;journal=Crop%20Sci&amp;volume=50&amp;issue=2&amp;pages=556-66&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR35" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Urlacher VB, Lutz-Wahl S, Schmid RD. Microbial P450 enzymes in biotechnology. Appl Microbiol Biotechnol. 2004;64(3):317–25.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00253-003-1514-1"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14716467"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Microbial%20P450%20enzymes%20in%20biotechnology&amp;author=VB.%20Urlacher&amp;author=S.%20Lutz-Wahl&amp;author=RD.%20Schmid&amp;journal=Appl%20Microbiol%20Biotechnol&amp;volume=64&amp;issue=3&amp;pages=317-25&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR36" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Rinaldi C, Kohler A, Frey P, Duchaussoy F, Ningre N, Couloux A, et al. Transcript profiling of poplar leaves upon infection with compatible and incompatible strains of the foliar rust <em class="EmphasisTypeItalic">Melampsora larici-populina</em>. Plant Physiol. 2007;144(1):347–66.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1104/pp.106.094987"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17400708"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1913798"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Transcript%20profiling%20of%20poplar%20leaves%20upon%20infection%20with%20compatible%20and%20incompatible%20strains%20of%20the%20foliar%20rust%20Melampsora%20larici-populina%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=C.%20Rinaldi&amp;author=A.%20Kohler&amp;author=P.%20Frey&amp;author=F.%20Duchaussoy&amp;author=N.%20Ningre&amp;author=A.%20Couloux&amp;journal=Plant%20Physiol&amp;volume=144&amp;issue=1&amp;pages=347-66&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR37" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Feuillet C, Schachermayr G, Keller B. Molecular cloning of a new receptor-like kinase gene encoded at the LrlO disease resistance locus of wheat. Plant J. 1997;11(1):45–52.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1046/j.1365-313X.1997.11010045.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9025301"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Molecular%20cloning%20of%20a%20new%20receptor-like%20kinase%20gene%20encoded%20at%20the%20LrlO%20disease%20resistance%20locus%20of%20wheat&amp;author=C.%20Feuillet&amp;author=G.%20Schachermayr&amp;author=B.%20Keller&amp;journal=Plant%20J&amp;volume=11&amp;issue=1&amp;pages=45-52&amp;publication_year=1997"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR38" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sun G, Fahima T, Korol A, Turpeinen T, Grama A, Ronin Y, et al. Identification of molecular markers linked to the Yr15 stripe rust resistance gene of wheat originated in wild emmer wheat, <em class="EmphasisTypeItalic">Triticum dicoccoides</em>. Theor Appl Genet. 1997;95(4):622–8.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s001220050604"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Identification%20of%20molecular%20markers%20linked%20to%20the%20Yr15%20stripe%20rust%20resistance%20gene%20of%20wheat%20originated%20in%20wild%20emmer%20wheat%2C%20Triticum%20dicoccoides%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=G.%20Sun&amp;author=T.%20Fahima&amp;author=A.%20Korol&amp;author=T.%20Turpeinen&amp;author=A.%20Grama&amp;author=Y.%20Ronin&amp;journal=Theor%20Appl%20Genet&amp;volume=95&amp;issue=4&amp;pages=622-8&amp;publication_year=1997"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR39" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Peng JH, Fahima T, Röder MS, Li YC, Dahan A, Grama A, et al. Microsatellite tagging of the stripe-rust resistance gene YrH52 derived from wild emmer wheat, Triticum dicoccoides, and suggestive negative crossover interference on chromosome 1B. Theor Appl Genet. 1999;98(6–7):862–72.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s001220051145"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Microsatellite%20tagging%20of%20the%20stripe-rust%20resistance%20gene%20YrH52%20derived%20from%20wild%20emmer%20wheat%2C%20Triticum%20dicoccoides%2C%20and%20suggestive%20negative%20crossover%20interference%20on%20chromosome%201B&amp;author=JH.%20Peng&amp;author=T.%20Fahima&amp;author=MS.%20R%C3%B6der&amp;author=YC.%20Li&amp;author=A.%20Dahan&amp;author=A.%20Grama&amp;journal=Theor%20Appl%20Genet&amp;volume=98&amp;issue=6%E2%80%937&amp;pages=862-72&amp;publication_year=1999"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR40" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Feng H, Wang XM, Sun YF, Wang XJ, Chen XM, Guo J, et al. Cloning and characterization of a calcium binding EF-hand protein gene TaCab1 from wheat and its expression in response to <em class="EmphasisTypeItalic">Puccinia striiformis</em> f. sp. <em class="EmphasisTypeItalic">tritici</em> and abiotic stresses. Mol Biol Rep. 2011;38(6):3857–66.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11033-010-0501-8"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21110112"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Cloning%20and%20characterization%20of%20a%20calcium%20binding%20EF-hand%20protein%20gene%20TaCab1%20from%20wheat%20and%20its%20expression%20in%20response%20to%20Puccinia%20striiformis%20f.%20sp.%20tritici%20and%20abiotic%20stresses&amp;author=H.%20Feng&amp;author=XM.%20Wang&amp;author=YF.%20Sun&amp;author=XJ.%20Wang&amp;author=XM.%20Chen&amp;author=J.%20Guo&amp;journal=Mol%20Biol%20Rep&amp;volume=38&amp;issue=6&amp;pages=3857-66&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR41" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Saitou N, Nei M. The neighbor-joining method: a new method for reconstructing phylogenetic trees. Mol Biol Evolution. 1987;4(4):406–25.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20neighbor-joining%20method%3A%20a%20new%20method%20for%20reconstructing%20phylogenetic%20trees&amp;author=N.%20Saitou&amp;author=M.%20Nei&amp;journal=Mol%20Biol%20Evolution&amp;volume=4&amp;issue=4&amp;pages=406-25&amp;publication_year=1987"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR42" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Eickmeyer K, Huggins P, Pachter L, Yoshida R. On the optimality of the neighbor-joining algorithm. Algorithms Mol Biol. 2008;3(1):1.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1748-7188-3-5"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=On%20the%20optimality%20of%20the%20neighbor-joining%20algorithm&amp;author=K.%20Eickmeyer&amp;author=P.%20Huggins&amp;author=L.%20Pachter&amp;author=R.%20Yoshida&amp;journal=Algorithms%20Mol%20Biol&amp;volume=3&amp;issue=1&amp;pages=1&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR43" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">de Hoon MJL, Imoto S, Nolan J, Miyano S. Open source clustering software. Bioinform. 2004;20(9):1453–4.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/bth078"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Open%20source%20clustering%20software&amp;author=MJL.%20Hoon&amp;author=S.%20Imoto&amp;author=J.%20Nolan&amp;author=S.%20Miyano&amp;journal=Bioinform&amp;volume=20&amp;issue=9&amp;pages=1453-4&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR44" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Groth D, Hartmann S, Klie S, Selbig J. Principal components analysis. Computational Toxicology: Volume II. 2013;2013:527–47.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/978-1-62703-059-5_22"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Principal%20components%20analysis&amp;author=D.%20Groth&amp;author=S.%20Hartmann&amp;author=S.%20Klie&amp;author=J.%20Selbig&amp;journal=Computational%20Toxicology%3A%20Volume%20II&amp;volume=2013&amp;pages=527-47&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR45" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Barrett JC, Fry B, Maller JDMJ, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. Bioinform. 2005;21(2):263–5.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/bth457"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Haploview%3A%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&amp;author=JC.%20Barrett&amp;author=B.%20Fry&amp;author=JDMJ.%20Maller&amp;author=MJ.%20Daly&amp;journal=Bioinform&amp;volume=21&amp;issue=2&amp;pages=263-5&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR46" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Takagi H, Abe A, Yoshida K, Kosugi S, Natsume S, Mitsuoka C, et al. QTL-seq: rapid mapping of quantitative trait loci in rice by whole genome resequencing of DNA from two bulked populations. Plant J. 2013;74(1):174–83.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/tpj.12105"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23289725"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=QTL-seq%3A%20rapid%20mapping%20of%20quantitative%20trait%20loci%20in%20rice%20by%20whole%20genome%20resequencing%20of%20DNA%20from%20two%20bulked%20populations&amp;author=H.%20Takagi&amp;author=A.%20Abe&amp;author=K.%20Yoshida&amp;author=S.%20Kosugi&amp;author=S.%20Natsume&amp;author=C.%20Mitsuoka&amp;journal=Plant%20J&amp;volume=74&amp;issue=1&amp;pages=174-83&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR47" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li H, Durbin R. Fast and accurate long-read alignment with Burrows–Wheeler transform. Bioinformatics. 2010;25(5):1754–60.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20long-read%20alignment%20with%20Burrows%E2%80%93Wheeler%20transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;journal=Bioinformatics&amp;volume=25&amp;issue=5&amp;pages=1754-60&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR48" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li H. The Sequence Alignment-Map format and SAMtools. Bioinformatics. 2009;25(16):2078–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btp352"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19505943"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2723002"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20Sequence%20Alignment-Map%20format%20and%20SAMtools&amp;author=H.%20Li&amp;journal=Bioinformatics&amp;volume=25&amp;issue=16&amp;pages=2078-9&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR49" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Pruitt KD, Tatusova T, Maglott DR. NCBI Reference Sequence (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins. Nucleic Acid Res. 2005;33 suppl 1:D501–4.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15608248"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=NCBI%20Reference%20Sequence%20%28RefSeq%29%3A%20a%20curated%20non-redundant%20sequence%20database%20of%20genomes%2C%20transcripts%20and%20proteins&amp;author=KD.%20Pruitt&amp;author=T.%20Tatusova&amp;author=DR.%20Maglott&amp;journal=Nucleic%20Acid%20Res&amp;volume=33&amp;issue=suppl%201&amp;pages=D501-4&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li></ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#TheAuthors" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="TheAuthors">Copyright</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="TheAuthors"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">©&nbsp;The Author(s)&nbsp;2017</div></div></section></div>
                        


                    </article>
                </main>

                <div class="c-content-layout__side u-text-sm">
                    <aside>
                        <div class="p-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="//hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-017-0027-3" data-track="click" data-track-category="PDF Download"
            data-track-action="Click download article PDF"
            data-track-label="10.1186/s41065-017-0027-3">Download PDF</a>
        </p>
    
                            
                            
    <button type="button"
       class="c-button c-button--secondary c-button--full-width c-button--expandable"
       data-toggle="collapse"
       data-target="export-citations"
       data-test="export-citations-button"
       data-sticky-update="true"
       aria-controls="export-citations"
       aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/s41065-017-0027-3"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0027-3?format=refman&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/s41065-017-0027-3"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0027-3?format=refman&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/s41065-017-0027-3"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0027-3?format=refman&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/s41065-017-0027-3"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0027-3?format=endnote&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/s41065-017-0027-3"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0027-3?format=endnote&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/s41065-017-0027-3"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0027-3?format=endnote&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/s41065-017-0027-3"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0027-3?format=bibtex&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/s41065-017-0027-3"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0027-3?format=bibtex&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/s41065-017-0027-3"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0027-3?format=bibtex&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
    </div>

                        </div>
                        
    
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
                                <div id="article-access-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/access-count/10.1186/s41065-017-0027-3 #accessCount-container"></div>
                                <div id="citation-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/citation-count/10.1186/s41065-017-0027-3 #count-container"></div>
                                <div id="altmetric-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/altmetric/10.1186/s41065-017-0027-3 #altmetric-container"></div>
                            </ul>
                        </div>
                        
    <div id="sociaMediaShare" class="p-separator u-js-shown">
        <h3 class="p-section-title">Share this article</h3>
        <ul id="shareButtons" class="c-social-links">
            <li class="c-social-links__item">
                <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-017-0027-3">
                    <span class="u-visually-hidden">Share on Twitter</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-twitter"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-017-0027-3">
                    <span class="u-visually-hidden">Share on Facebook</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-facebook"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click"
                        data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-017-0027-3">
                    <span class="u-visually-hidden">Share on LinkedIn</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-linkedin"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-017-0027-3">
                    <span class="u-visually-hidden">Share on Weibo</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-weibo"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnGooglePlus" class="js-btnShareOnGplus" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Google+" data-track-label="10.1186/s41065-017-0027-3">
                    <span class="u-visually-hidden">Share on Google Plus</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-gplus"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-017-0027-3">
                    <span class="u-visually-hidden">Share on Reddit</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-reddit"></use>
                    </svg>
                </button>
            </li>
        </ul>
    </div>
 
                        
    
        <div class="p-separator u-js-shown">
            <h3 class="p-section-title">See updates</h3>
            <button class="c-crossmark-button" data-target="crossmark" data-test="crossmark" data-track="click" data-track-category="Article Page" data-track-action="Click CrossMark" data-track-label="10.1186/s41065-017-0027-3">
                <img id="crossmark-icon" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" alt="Crossmark logo" />
            </button>
        </div>
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
                                <li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/s41065-017-0027-3" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=738&doi=10.1186%2Fs41065-017-0027-3&volume=154&issue=1&title=Genome-wide+association+study+of+rust+traits+in+orchardgrass+using+SLAF-seq+technology&author_name=Bing+Zeng&start_page=1&end_page=8">Order reprint</a>

                                </li>
                            </ul>
                        </div>
                    </aside>
                    
    <div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">Hereditas</span>
                        </h4>
                        <p class="c-journal-footer__issn">ISSN: 1601-5223</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
                                
                                    <li class="c-journal-footer__contact-item">Submission enquiries: <a href="https://www.editorialmanager.com/here/" target="_blank">Access here and click Contact Us</a></li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a></li>
                                
                            </ul>
                        </div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class='tracker' style='display:none' src='/track/article/10.1186/s41065-017-0027-3' alt=""/>

         
    <footer>
        <div class="ctx-publisher-footer">
    <div class="c-navbar">
        <div class="u-container">
            <div class="c-navbar__logo u-margin-right-xl">
                <a href="https://www.biomedcentral.com">
                    <img src=/static/images/bmc/logos/logo-bmc-aj-594e03e20d.svg alt="BMC logo" data-test="navbar-logo-footer">

                </a>
            </div>
            <div class="c-navbar__content">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


            </div>
        </div>
    </div>
    <div class="c-publisher-footer" data-test="publisher-footer">
        <div class="u-container">
            
            <div class="c-flex c-flex--wrap c-flex--space-between" role="navigation" data-test="publisher-footer-menu">
                <div class="c-flex" aria-label="Publisher footer links">
                    
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/login">Receive BMC newsletters</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/account">Manage article alerts</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/policies">Policies</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/accessibility">Accessibility</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/press-centre">Press center</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/contact-us">Contact us</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/jobs">Careers</a>
                                    </li>
                                
                            </ul>
                        
                    
                </div>
                <div class="u-margin-bottom-lg">
                    <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                    <ul class="c-flex u-reset-list" data-test="footer-social-links">
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://twitter.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Twitter link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Twitter page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-twitter-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://www.facebook.com/BioMedCentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Facebook link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Facebook page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-facebook-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://plus.google.com/106226337984152901734/"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Google Plus link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Google Plus page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-gplus-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="http://www.weibo.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Weibo link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Weibo page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-weibo-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </div>
            <p class="u-reset-margin">
                By using this website, you agree to our
                <a class="u-text-inherit" href="//www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
                <a class="u-text-inherit" href="//www.biomedcentral.com/privacy-statement">Privacy
                    statement</a> and
                <a class="u-text-inherit" href="//www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
                
                    <a class="optanon-toggle-display u-text-inherit" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
                
            </p>
        </div>
    </div>
</div>


        <div class="c-sn-footer c-sn-footer--neutral">
    <div class="u-container">
        <img src=/static/images/bmc/logos/logo-springernature-8bd4e8d82f.svg class="c-sn-footer__logo" alt="Springer Nature Logo" itemprop="logo" >
        <p data-test="copyright">&#169; 2018 BioMed Central Ltd unless otherwise stated. Part of
            <a href="http://www.springernature.com"
               data-track="click"
               data-track-action="Click Springer Nature link"
               data-track-category="Corporate footer"
               itemscope itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>

    </div>

    
    <script type="text/javascript" src="//cdn.pbgrd.com/bmc.js" async="async"></script>
    <noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>
    <!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker -->

<script id="springerlink-event-tracker" src="https://event-tracker.springernature.com/dist/eventTracker.js"></script>
<script type="text/javascript">
    (function(){
        var dataLayer = window.dataLayer[0];
        var doi = dataLayer.content.article.doi;
        var imprint = dataLayer.content.contentInfo.imprint;
        var bpids = dataLayer.session.authentication.authenticationID;
        var publisher;
        var pageType = dataLayer.page.category.pageType;
        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }
        var eventObject = {'content_type' : pageType,'doi' : doi};

        if (imprint === 'BioMed Central') { publisher = 'BMC' }
        else if (imprint === 'SpringerOpen') { publisher = 'SpringerOpen' }
        else { throw new Error('Event Tracker Error: The publisher has an unexpected value') }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }
        new EventTracker({'platform':publisher}).sendEvent('display', eventObject);
    })();
</script>


    </body>
</html>

