<!DOCTYPE html>
<html lang="en" class="no-js">
    <head>
        <title>Genotypic and phenotypic evaluation of off-type grasses in hybrid Bermudagrass [Cynodon dactylon (L.) Pers. x C. transvaalensis Burtt-Davy] putting greens using genotyping-by-sequencing and morphological characterization | Hereditas | Full Text</title>

        

    <meta name="citation_abstract" content="Interspecific hybrid bermudagrass [Cynodon dactylon (L.) Pers. x C. transvaalensis Burtt-Davy] is one of the most widely used grasses on golf courses, with cultivars derived from ‘Tifgreen’ or ‘Tifdwarf’ particularly used for putting greens. Many bermudagrass cultivars established for putting greens can be genetically unstable and lead to the occurrence of undesirable off-type grasses that vary in phenotype. The objective of this research was to genetically and phenotypically differentiate off-type grasses and hybrid cultivars. Beginning in 2013, off-type and desirable hybrid bermudagrass samples were collected from golf course putting greens in the southeastern United States and genetically and phenotypically characterized using genotyping-by-sequencing and morphology. Genotyping-by-sequencing determined that 11% (5) of off-type and desirable samples from putting greens were genetically divergent from standard cultivars such as Champion, MiniVerde, Tifdwarf, TifEagle, and Tifgreen. In addition, genotyping-by-sequencing was unable to genetically distinguish all standard cultivars from one another due to their similar origin and clonal propagation; however, over 90,000 potentially informative nucleotide variants were identified among the triploid hybrid cultivars. Although few genetic differences were found in this research, samples harvested from golf course putting greens had variable morphology and were clustered into three distinct phenotypic groups. The majority of off-type grasses in hybrid bermudagrass putting greens were genetically similar with variable morphological traits. Off-type grasses within golf course putting greens have the potential to compromise putting surface functionality and aesthetics."/>

    <meta name="description" content="Interspecific hybrid bermudagrass [Cynodon dactylon (L.) Pers. x C. transvaalensis Burtt-Davy] is one of the most widely used grasses on golf courses, with cultivars derived from ‘Tifgreen’ or ‘Tifdwarf’ particularly used for putting greens. Many bermudagrass cultivars established for putting greens can be genetically unstable and lead to the occurrence of undesirable off-type grasses that vary in phenotype. The objective of this research was to genetically and phenotypically differentiate off-type grasses and hybrid cultivars. Beginning in 2013, off-type and desirable hybrid bermudagrass samples were collected from golf course putting greens in the southeastern United States and genetically and phenotypically characterized using genotyping-by-sequencing and morphology. Genotyping-by-sequencing determined that 11% (5) of off-type and desirable samples from putting greens were genetically divergent from standard cultivars such as Champion, MiniVerde, Tifdwarf, TifEagle, and Tifgreen. In addition, genotyping-by-sequencing was unable to genetically distinguish all standard cultivars from one another due to their similar origin and clonal propagation; however, over 90,000 potentially informative nucleotide variants were identified among the triploid hybrid cultivars. Although few genetic differences were found in this research, samples harvested from golf course putting greens had variable morphology and were clustered into three distinct phenotypic groups. The majority of off-type grasses in hybrid bermudagrass putting greens were genetically similar with variable morphological traits. Off-type grasses within golf course putting greens have the potential to compromise putting surface functionality and aesthetics."/>

    <meta name="journal_id" content="41065"/>

    <meta name="dc.title" content="Genotypic and phenotypic evaluation of off-type grasses in hybrid Bermudagrass [ Cynodon dactylon (L.) Pers. x C. transvaalensis Burtt-Davy] putting greens using genotyping-by-sequencing and morphological characterization"/>

    <meta name="dc.source" content="Hereditas 2017 155:1"/>

    <meta name="dc.format" content="text/html"/>

    <meta name="dc.publisher" content="BioMed Central"/>

    <meta name="dc.date" content="2017-08-18"/>

    <meta name="dc.type" content="OriginalPaper"/>

    <meta name="dc.language" content="En"/>

    <meta name="dc.copyright" content="2017 The Author(s)"/>

    <meta name="dc.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="dc.description" content="Interspecific hybrid bermudagrass [Cynodon dactylon (L.) Pers. x C. transvaalensis Burtt-Davy] is one of the most widely used grasses on golf courses, with cultivars derived from &#8216;Tifgreen&#8217; or &#8216;Tifdwarf&#8217; particularly used for putting greens. Many bermudagrass cultivars established for putting greens can be genetically unstable and lead to the occurrence of undesirable off-type grasses that vary in phenotype. The objective of this research was to genetically and phenotypically differentiate off-type grasses and hybrid cultivars. Beginning in 2013, off-type and desirable hybrid bermudagrass samples were collected from golf course putting greens in the southeastern United States and genetically and phenotypically characterized using genotyping-by-sequencing and morphology. Genotyping-by-sequencing determined that 11% (5) of off-type and desirable samples from putting greens were genetically divergent from standard cultivars such as Champion, MiniVerde, Tifdwarf, TifEagle, and Tifgreen. In addition, genotyping-by-sequencing was unable to genetically distinguish all standard cultivars from one another due to their similar origin and clonal propagation; however, over 90,000 potentially informative nucleotide variants were identified among the triploid hybrid cultivars. Although few genetic differences were found in this research, samples harvested from golf course putting greens had variable morphology and were clustered into three distinct phenotypic groups. The majority of off-type grasses in hybrid bermudagrass putting greens were genetically similar with variable morphological traits. Off-type grasses within golf course putting greens have the potential to compromise putting surface functionality and aesthetics."/>

    <meta name="prism.issn" content="1601-5223"/>

    <meta name="prism.publicationName" content="Hereditas"/>

    <meta name="prism.publicationDate" content="2017-08-18"/>

    <meta name="prism.volume" content="155"/>

    <meta name="prism.number" content="1"/>

    <meta name="prism.section" content="OriginalPaper"/>

    <meta name="prism.startingPage" content="8"/>

    <meta name="prism.copyright" content="2017 The Author(s)"/>

    <meta name="prism.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="prism.url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0043-3"/>

    <meta name="prism.doi" content="doi:10.1186/s41065-017-0043-3"/>

    <meta name="citation_pdf_url" content="https://hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-017-0043-3"/>

    <meta name="citation_fulltext_html_url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0043-3"/>

    <meta name="citation_journal_title" content="Hereditas"/>

    <meta name="citation_publisher" content="BioMed Central"/>

    <meta name="citation_issn" content="1601-5223"/>

    <meta name="citation_title" content="Genotypic and phenotypic evaluation of off-type grasses in hybrid Bermudagrass [ Cynodon dactylon (L.) Pers. x C. transvaalensis Burtt-Davy] putting greens using genotyping-by-sequencing and morphological characterization"/>

    <meta name="citation_volume" content="155"/>

    <meta name="citation_issue" content="1"/>

    <meta name="citation_publication_date" content="2018/12"/>

    <meta name="citation_online_date" content="2017/08/18"/>

    <meta name="citation_firstpage" content="8"/>

    <meta name="citation_article_type" content="Research"/>

    <meta name="citation_fulltext_world_readable" content=""/>

    <meta name="dc.identifier" content="doi:10.1186/s41065-017-0043-3"/>

    <meta name="DOI" content="10.1186/s41065-017-0043-3"/>

    <meta name="citation_doi" content="10.1186/s41065-017-0043-3"/>

    <meta name="dc.creator" content="Eric H. Reasor"/>

    <meta name="dc.creator" content="James T. Brosnan"/>

    <meta name="dc.creator" content="Margaret E. Staton"/>

    <meta name="dc.creator" content="Thomas Lane"/>

    <meta name="dc.creator" content="Robert N. Trigiano"/>

    <meta name="dc.creator" content="Phillip A. Wadl"/>

    <meta name="dc.creator" content="Joann A. Conner"/>

    <meta name="dc.creator" content="Brian M. Schwartz"/>

    <meta name="dc.subject" content="Animal Genetics and Genomics"/>

    <meta name="dc.subject" content="Evolutionary Biology"/>

    <meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=Nuclear DNA content of thirteen turfgrass species by flow cytometry; citation_author=K Arumuganathan, SP Tallury, ML Fraser, AH Bruneau, R Qu; citation_volume=39; citation_publication_date=1999; citation_pages=1518-1521; citation_id=CR1"/>

    <meta name="citation_reference" content="Beard JB. Turf management for golf courses 2nd ed. Chelsea: Wiley; 2002."/>

    <meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=Ploidy determination in Agrostis using flow cytometry and morphological traits; citation_author=SA Bonos, KA Plumley, WA Meyer; citation_volume=42; citation_publication_date=2002; citation_pages=192-196; citation_id=CR3"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=TASSEL: software for association mapping of complex traits in diverse samples; citation_author=PJ Bradbury, Z Zhang, DE Kroon, M Terry Casstevens, Y Ramdoss, ES Buckler; citation_volume=23; citation_publication_date=2007; citation_pages=2633-2635; citation_doi=10.1093/bioinformatics/btm308; citation_id=CR4"/>

    <meta name="citation_reference" content="citation_title=&#8216;champion&#8217; dwarf&#8217; hybrid bermudagrass. US Plant patent 9888 date issued: 6 may 1997; citation_publication_date=1997; citation_id=CR5; citation_author=RM Brown; citation_author=MA Brown; citation_author=SD Brown"/>

    <meta name="citation_reference" content="citation_journal_title=USGA Green Section Record; citation_title=&#8216;Tifgreen&#8217; (Tifton 328) bermudagrass for golf greens; citation_author=GW Burton; citation_volume=2; citation_publication_date=1964; citation_pages=11-13; citation_id=CR6"/>

    <meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=&#8216;Tifdwarf&#8217; bermudagrass; citation_author=GW Burton; citation_volume=6; citation_publication_date=1966; citation_pages=94; citation_doi=10.2135/cropsci1966.0011183X000600010036x; citation_id=CR7"/>

    <meta name="citation_reference" content="citation_journal_title=USGA Green Section Record; citation_title=&#8216;Tifdwarf&#8217;-a new bermudagrass for golf greens; citation_author=GW Burton, JE Elsner; citation_volume=2; citation_publication_date=1965; citation_pages=8-9; citation_id=CR8"/>

    <meta name="citation_reference" content="citation_journal_title=Euphytica; citation_title=Genetic instability of bermudagrass (Cynodon) cultivars &#8216;Tifgreen&#8217; and &#8216;Tifdwarf&#8217; detected by DAF and ASAP analysis of accessions and off-types; citation_author=G Caetano-Anoll&#233;s; citation_volume=101; citation_publication_date=1998; citation_pages=165-173; citation_doi=10.1023/A:1018392623408; citation_id=CR9"/>

    <meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=The origin of bermudagrass (Cynodon) off-types inferred by DNA amplification fingerprinting; citation_author=G Caetano-Anoll&#233;s, LM Callahan, PM Gresshoff; citation_volume=37; citation_publication_date=1997; citation_pages=81-87; citation_doi=10.2135/cropsci1997.0011183X003700010013x; citation_id=CR10"/>

    <meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=Dinitroaniline-induced genetic changes in bermudagrass; citation_author=LJ Capo-chichi, JM Goatley, W Philley, J Krans, D Davis, A Kato, E Santen; citation_volume=45; citation_publication_date=2005; citation_pages=1504-1510; citation_doi=10.2135/cropsci2004.0187; citation_id=CR11"/>

    <meta name="citation_reference" content="citation_journal_title=Floriculture Ornamental Biotechnol; citation_title=Genetic diversity of warm-season turfgrass: seashore paspalum, bermudagrass, and zoysiagrass revealed by AFLPs; citation_author=Z Chen, ML Wang, C Waltz, P Raymer; citation_volume=3; citation_publication_date=2009; citation_pages=20-24; citation_id=CR12"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species; citation_author=RJ Elshire, JC Glaubitz, G Sun, JA Poland, K Kawamoto, ES Buckler, SE Mitchell; citation_volume=6; citation_publication_date=2011; citation_pages=E19379; citation_doi=10.1371/journal.pone.0019379; citation_id=CR13"/>

    <meta name="citation_reference" content="citation_journal_title=Acta Hortic; citation_title=Recent progresses on turfgrass molecular genetics and biotechnology; citation_author=S Fei; citation_volume=783; citation_publication_date=2008; citation_pages=247-260; citation_id=CR14"/>

    <meta name="citation_reference" content="Fiedler JD, Lanzatella C, Okada M, Jenkins J, Schmutz J, Tobias CM. High-density single nucleotide polymorphism linkage map of lowland switchgrass using genotyping-by-sequencing. Plant Genome. 2015;8. doi:
                    10.3835/plantgenome2014.10.0065
                    
                  ."/>

    <meta name="citation_reference" content="Garrison E, Marth G. Haplotype-based variant detection from short-read sequencing. Cornell University Library; 2012. 
                    https://arxiv.org/abs/1207.3907
                    
                  ."/>

    <meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=TASSEL-GBS: a high capacity genotyping by sequencing analysis pipeline; citation_author=JC Glaubitz, TM Casstevens, F Lu, J Harriman, RJ Elshire, Q Sun, ES Buckler; citation_volume=9; citation_issue=2; citation_publication_date=2014; citation_pages=e90346; citation_doi=10.1371/journal.pone.0090346; citation_id=CR17"/>

    <meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=Registration of &#8216;TifEagle&#8217; bermudagrass; citation_author=WW Hanna, JE Elsner; citation_volume=39; citation_publication_date=1999; citation_pages=1258; citation_doi=10.2135/cropsci1999.0011183X003900040060x; citation_id=CR18"/>

    <meta name="citation_reference" content="citation_journal_title=J Amer Soc Hort Sci; citation_title=Identification of simple sequence repeat markers that differentiate bermudagrass cultivars derived from &#8216;Tifgreen&#8217;; citation_author=KR Harris-Shultz, BM Schwartz, JA Brady; citation_volume=136; citation_publication_date=2011; citation_pages=211-218; citation_id=CR19"/>

    <meta name="citation_reference" content="citation_journal_title=J Amer Soc Hort Sci; citation_title=Development, linkage mapping, and use of microsatellites in bermudagrass; citation_author=KR Harris-Shultz, BM Schwartz, WW Hanna, JA Brady; citation_volume=135; citation_publication_date=2010; citation_pages=511-520; citation_id=CR20"/>

    <meta name="citation_reference" content="citation_journal_title=Agron J; citation_title=Registration of varieties and strains of bermudagrass, III. (Cynodon dactylon (L.) Pers.); citation_author=MA Hein; citation_volume=53; citation_publication_date=1961; citation_pages=276; citation_doi=10.2134/agronj1961.00021962005300040021x; citation_id=CR21"/>

    <meta name="citation_reference" content="citation_title=Manual of the grasses of the United States; citation_publication_date=1971; citation_id=CR22; citation_author=AS Hitchcock; citation_publisher=Dover Publications"/>

    <meta name="citation_reference" content="citation_title=Bermudagrass &#8216;P-18&#8217;. United States plant patent. 12084P3 date issued: 11 September; citation_publication_date=2001; citation_id=CR23; citation_author=HE Kaerwer; citation_author=B Kaerwer"/>

    <meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=DNA polymorphisms at bermudagrass microsatellite loci and their use in genotype fingerprinting; citation_author=TL Kamps, NR Williams, VM Ortega, KC Chamusco, KR Harris-Shultz, BT Scully, CD Chase; citation_volume=51; citation_publication_date=2011; citation_pages=1122-1131; citation_doi=10.2135/cropsci2010.08.0478; citation_id=CR24"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Cell; citation_title=Genetic diversity among Korean bermudagrass (Cynodon spp.) ecotypes characterized by morphological, cytological, and molecular approaches; citation_author=SY Kang, GJ Lee, KB Lim, HJ Lee, IS Park, SJ Chung, JB Kim, DS Kim, HK Rhee; citation_volume=25; citation_publication_date=2008; citation_pages=163-171; citation_id=CR25"/>

    <meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=Genetic variation in Cynodon transvaalensis Burtt-Davy; citation_author=KE Kenworthy, CM Taliaferro, BF Carver, DL Martin, JA Anderson; citation_volume=46; citation_publication_date=2006; citation_pages=2376-2381; citation_doi=10.2135/cropsci2006.02.0075; citation_id=CR26"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Methods; citation_title=Fast gapped-read alignment with bowtie 2; citation_author=B Langmead, SL Salzberg; citation_volume=9; citation_publication_date=2012; citation_pages=357-359; citation_doi=10.1038/nmeth.1923; citation_id=CR27"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=The sequence alignment/map format and SAMtools; citation_author=H Li, B Handsaker, A Wysoker, T Fennell, J Ruan, N Homer, G Marth, G Abecasis, R Durbin; citation_volume=25; citation_publication_date=2009; citation_pages=2078-2079; citation_doi=10.1093/bioinformatics/btp352; citation_id=CR28"/>

    <meta name="citation_reference" content="Lyman GT, Throssell CS, Johnson ME, Stacey GA, Brown CD. Golf course profile describes turfgrass, landscape, and environmental stewardship features. Applied Turfgrass Science. 2007; doi:
                    10.1094/ATS-2007-1107-01-RS
                    
                  ."/>

    <meta name="citation_reference" content="citation_title=Some methods for classification and analysis of multivariate observations. Proceedings of the fifth symposium on math, statistics, and probability; citation_publication_date=1967; citation_id=CR30; citation_author=JB MacQueen; citation_publisher=University of California Press"/>

    <meta name="citation_reference" content="citation_journal_title=Acta Agric Scand Sect B Soil Plant Sci; citation_title=Phenotypic traits and establishment speed of 44 turf bermudagrass accessions; citation_author=S Magni, M Gaetani, L Caturegli, C Leto, T Tuttolomodo, S Bella, G Virga, N Ntoulas, M Volterrani; citation_volume=64; citation_publication_date=2014; citation_pages=722-733; citation_id=CR31"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=Development of high-density genetic maps for barley and wheat using a novel two enzyme genotyping-by sequencing approach; citation_author=JA Poland, PJ Brown, ME Sorrells, JL Jannink; citation_volume=7; citation_publication_date=2012; citation_pages=E32253; citation_doi=10.1371/journal.pone.0032253; citation_id=CR32"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Genome; citation_title=Genotyping-by-sequencing for plant breeding and genetics; citation_author=JA Poland, TW Rife; citation_volume=5; citation_publication_date=2012; citation_pages=92-105; citation_doi=10.3835/plantgenome2012.05.0005; citation_id=CR33"/>

    <meta name="citation_reference" content="citation_journal_title=Ann Bot; citation_title=Genome evolution in the genus Sorghum (Poaceae); citation_author=HJ Price, SL Dillon, G Hodnett, WL Rooney, L Ross, JS Johnston; citation_volume=95; citation_publication_date=2005; citation_pages=219-227; citation_doi=10.1093/aob/mci015; citation_id=CR34"/>

    <meta name="citation_reference" content="citation_journal_title=Amer J Human Genet; citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses; citation_author=S Purcell, B Neale, K Todd-Brown, L Thomas, MAR Ferreira, D Bender, J Maller; citation_volume=81; citation_publication_date=2007; citation_pages=559-575; citation_doi=10.1086/519795; citation_id=CR35"/>

    <meta name="citation_reference" content="citation_journal_title=Planta; citation_title=The genetic and phenotypic variability of interspecific hybrid bermudagrasses (Cynodon dactylon (L.) Pers. x C. transvaalensis Burtt-Davy) used on golf course putting greens; citation_author=EH Reasor, JT Brosnan, RN Trigiano, JE Elsner, GM Henry, BM Schwartz; citation_volume=244; citation_publication_date=2016; citation_pages=761-773; citation_doi=10.1007/s00425-016-2573-8; citation_id=CR36"/>

    <meta name="citation_reference" content="citation_journal_title=Intl Turfgrass Soc Res J; citation_title=Morphological and development comparisons of seven greens quality hybrid bermudagrass [Cynodon dactylon (L.) Pers. X C. transvaalensis Burtt-Davy] cultivars; citation_author=MB Roche, DS Loch; citation_volume=10; citation_publication_date=2005; citation_pages=627-634; citation_id=CR37"/>

    <meta name="citation_reference" content="citation_journal_title=Acta Hort; citation_title=Evaluation of native bermudagrass (Cynodon dactylon) germplasm from Italy for the selection of adapted turfgrass cultivars; citation_author=M Romani, E Piano, AM Carroni, L Pecetti; citation_volume=661; citation_publication_date=2004; citation_pages=381-386; citation_doi=10.17660/ActaHortic.2004.661.52; citation_id=CR38"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Cell Rep; citation_title=Epigentic mechanisms of plant stress responses and adaptation; citation_author=PP Sahu, G Pandey, N Sharma, S Puranik, M Muthamilarason, M Prasad; citation_volume=32; citation_publication_date=2013; citation_pages=1151-1159; citation_doi=10.1007/s00299-013-1462-x; citation_id=CR39"/>

    <meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=Identification of vegetatively propagated turf bermudagrass cultivars using simple sequence repeat markers; citation_author=Z Wang, Y Wu, DL Martin, H Gao, T Samuels, C Tan; citation_volume=50; citation_publication_date=2010; citation_pages=2103-2111; citation_doi=10.2135/cropsci2010.02.0116; citation_id=CR40"/>

    <meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=Differentiation of bermudagrass (Cynodon spp.) genotypes by AFLP analyses; citation_author=LH Zhang, P Ozias-Akins, G Kochert, S Kresovich, R Dean, WW Hanna; citation_volume=98; citation_publication_date=1999; citation_pages=895-902; citation_doi=10.1007/s001220051148; citation_id=CR41"/>

    <meta name="citation_author" content="Eric H. Reasor"/>

    <meta name="citation_author_institution" content="Department of Plant and Soil Sciences, Mississippi State University, Mississippi State, USA"/>

    <meta name="citation_author" content="James T. Brosnan"/>

    <meta name="citation_author_institution" content="Department of Plant Sciences, University of Tennessee, Knoxville, USA"/>

    <meta name="citation_author" content="Margaret E. Staton"/>

    <meta name="citation_author_institution" content="Department of Entomology and Plant Pathology, University of Tennessee, Knoxville, USA"/>

    <meta name="citation_author" content="Thomas Lane"/>

    <meta name="citation_author_institution" content="Department of Entomology and Plant Pathology, University of Tennessee, Knoxville, USA"/>

    <meta name="citation_author" content="Robert N. Trigiano"/>

    <meta name="citation_author_institution" content="Department of Entomology and Plant Pathology, University of Tennessee, Knoxville, USA"/>

    <meta name="citation_author" content="Phillip A. Wadl"/>

    <meta name="citation_author_institution" content="United States Department of Agriculture, Agriculture Research Service, United States Vegetable Laboratory, Charleston, USA"/>

    <meta name="citation_author" content="Joann A. Conner"/>

    <meta name="citation_author_institution" content="Department of Horticulture, University of Georgia, Tifton, USA"/>

    <meta name="citation_author" content="Brian M. Schwartz"/>

    <meta name="citation_author_institution" content="Department of Crop and Soil Sciences, University of Georgia, University of Georgia, Tifton, USA"/>



        <meta charset="UTF-8"/>
        <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
        <meta name="viewport" content="width=device-width, initial-scale=1"/>

        <!-- For IE 9 and below -->
<!--[if IE]><link rel="shortcut icon" href=/static/images/favicons//favicon.ico><![endif]-->
<!-- Touch Icons - iOS and Android 2.1+ -->
<link rel="apple-touch-icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>
<!-- Firefox, Chrome, Safari, IE 11+ and Opera -->
<link rel="icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>


        <script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script>
        <script>(function(d){if (sessionStorage.fontsLoaded){d.style.setProperty('--font-family-sans', 'Europa');d.style.setProperty('--font-family-serif', 'Noto Serif')}})(document.documentElement);</script>

        <link rel="stylesheet" media="screen" href=/static/styles/basic/basic-a31ca85bc2.css>
<link rel="stylesheet" media="print" href=/static/styles/bmc/print-3a6240f456.css>

<link rel="stylesheet" class="js-ctm" href=/static/styles/bmc/main-84414d3d47.css
      media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">

        

        <script src="/static/js/app-bundle-8d61e6ef37.js" defer></script>
        
        <script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script>
        
    <script type="text/javascript">
        config = {
            env: 'live',
            site: 'hereditasjournal.biomedcentral.com',
            siteWithPath: 'hereditasjournal.biomedcentral.com' + window.location.pathname,
            twitterHashtag: '',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/s41065-017-0043-3',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central'
        };
    </script>

        
    <script type="text/javascript">
        dataLayer = [{"content":{"article":{"doi":"10.1186/s41065-017-0043-3","articleType":"Research","peerReviewType":"Closed","supplement":null,"keywords":"Turfgrass;Bermudagrass;Off-types;Genotyping-by-sequencing;Morphology;Putting greens"},"contentInfo":{"imprint":"BioMed Central","title":"Genotypic and phenotypic evaluation of off-type grasses in hybrid Bermudagrass [ Cynodon dactylon (L.) Pers. x C. transvaalensis Burtt-Davy] putting greens using genotyping-by-sequencing and morphological characterization","publishedAt":1503014400000,"publishedAtDate":"2017-08-18","author":["Eric H. Reasor","James T. Brosnan","Margaret E. Staton","Thomas Lane","Robert N. Trigiano","Phillip A. Wadl","Joann A. Conner","Brian M. Schwartz"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":"CC BY + CC0","openAccess":true},"environment":"live"},"journal":{"siteKey":"hereditasjournal.biomedcentral.com","volume":"155","issue":"1","title":"Hereditas","type":"AJs Life","journalID":41065,"gaCode":"UA-62066850-11","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Research","publishingSegment":"Genetics & Heredity (Life)"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}}}]

    </script>

        <script src="https://cdn.cookielaw.org/consent/7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script>
        
    
        
    

        
    
        
            
                <!-- Google Tag Manager -->
                <script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script>
                <!-- End Google Tag Manager -->
            
        
    


        
    
    <link rel="canonical" href="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0043-3"/>
    

        
        
        <meta property="og:url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0043-3"/>
        <meta property="og:type" content="article"/>
        <meta property="og:site_name" content="Hereditas"/>
        <meta property="og:title" content="Genotypic and phenotypic evaluation of off-type grasses in hybrid Bermudagrass [ Cynodon dactylon (L.) Pers. x C. transvaalensis Burtt-Davy] putting greens using genotyping-by-sequencing and morphological characterization"/>
        <meta property="og:description" content="Interspecific hybrid bermudagrass [Cynodon dactylon (L.) Pers. x C. transvaalensis Burtt-Davy] is one of the most widely used grasses on golf courses, with cultivars derived from ‘Tifgreen’ or ‘Tifdwarf’ particularly used for putting greens. Many bermudagrass cultivars established for putting greens can be genetically unstable and lead to the occurrence of undesirable off-type grasses that vary in phenotype. The objective of this research was to genetically and phenotypically differentiate off-type grasses and hybrid cultivars. Beginning in 2013, off-type and desirable hybrid bermudagrass samples were collected from golf course putting greens in the southeastern United States and genetically and phenotypically characterized using genotyping-by-sequencing and morphology. Genotyping-by-sequencing determined that 11% (5) of off-type and desirable samples from putting greens were genetically divergent from standard cultivars such as Champion, MiniVerde, Tifdwarf, TifEagle, and Tifgreen. In addition, genotyping-by-sequencing was unable to genetically distinguish all standard cultivars from one another due to their similar origin and clonal propagation; however, over 90,000 potentially informative nucleotide variants were identified among the triploid hybrid cultivars. Although few genetic differences were found in this research, samples harvested from golf course putting greens had variable morphology and were clustered into three distinct phenotypic groups. The majority of off-type grasses in hybrid bermudagrass putting greens were genetically similar with variable morphological traits. Off-type grasses within golf course putting greens have the potential to compromise putting surface functionality and aesthetics."/>
        <meta property="og:image" content="https://images.springer.com/sgw/journals/medium/41065.jpg"/>
    
    </head>

    <body class="journal journal-fulltext"
          
          >
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK"
                            height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><feColorMatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"/></filter></defs><symbol id="icon-arrow-down" viewBox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"/></symbol><symbol id="icon-arrow-left-bullet" viewBox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"/></symbol><symbol id="icon-arrow-right-bullet" viewBox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"/></symbol><symbol id="icon-arrow-right" viewBox="0 0 32 64"><path d="M0 0v64l32-32z"/></symbol><symbol id="icon-close" viewBox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"/></symbol><symbol id="icon-community" viewBox="0 0 60.643 31.01"><mask maskUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><path fill="#FFF" d="M0 31.01h60.643V0H0z" filter="url(#a)"/></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"/></symbol><symbol id="icon-download" viewBox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"/></symbol><symbol id="icon-ext-link" viewBox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"/><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"/><path fill="#676767" d="M9.7 0L16 6.2V0z"/></symbol><symbol id="icon-info-bordered" viewBox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"/><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"/></symbol><symbol id="icon-opr" viewBox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"/><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"/></symbol><symbol id="icon-quotation" viewBox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"/></symbol><symbol id="icon-remove" viewBox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"/><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"/></symbol><symbol id="icon-ribbon" viewBox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"/></symbol><symbol id="icon-rss" viewBox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"/><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"/><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"/></g></symbol><symbol id="icon-search" viewBox="0 0 16 16"><path d="M6.276 12.257C2.815 12.257 0 9.508 0 6.128 0 2.749 2.815 0 6.276 0c3.462 0 6.277 2.749 6.277 6.128.002 3.38-2.815 6.129-6.277 6.129zm0-10.772c-2.622 0-4.755 2.083-4.755 4.643 0 2.562 2.133 4.644 4.755 4.644 2.623 0 4.756-2.082 4.756-4.644 0-2.56-2.134-4.643-4.756-4.643z"/><path d="M16 13.751L13.641 16l-4.875-4.82 2.36-2.248L16 13.751z"/></symbol><symbol id="icon-speech-bubble" viewBox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"/></symbol><symbol id="icon-error" viewBox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"/></symbol><symbol id="icon-info" viewBox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"/></symbol><symbol id="icon-success" viewBox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"/></symbol><symbol id="icon-warning" viewBox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"/></symbol><symbol id="icon-facebook-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"/></symbol><symbol id="icon-facebook" viewBox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"/><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"/></symbol><symbol id="icon-gplus-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M480.5 273.117c.123.076.26.185.416.334.146.152.289.343.432.562.135.211.258.459.359.746.086.286.129.619.129 1.004-.012.7-.164 1.262-.465 1.683a5.374 5.374 0 0 1-.461.562c-.186.176-.375.35-.58.527a2.65 2.65 0 0 0-.322.41.908.908 0 0 0-.176.553c0 .205.059.375.18.508.102.129.199.238.299.336l.662.545a6.48 6.48 0 0 1 1.086 1.119c.293.415.445.955.459 1.625 0 .949-.42 1.789-1.256 2.521-.869.76-2.123 1.15-3.762 1.177-1.373-.015-2.4-.308-3.074-.879-.682-.534-1.023-1.172-1.023-1.917 0-.359.111-.766.336-1.211.215-.443.602-.837 1.172-1.174a6.126 6.126 0 0 1 1.998-.725 15.415 15.415 0 0 1 1.723-.173 3.996 3.996 0 0 1-.377-.596 1.347 1.347 0 0 1-.193-.74c0-.174.023-.317.074-.433a8.83 8.83 0 0 1 .121-.338 5.973 5.973 0 0 1-.633.035c-1.043-.012-1.838-.339-2.381-.979-.574-.603-.861-1.3-.861-2.097 0-.961.404-1.832 1.217-2.616.559-.46 1.135-.757 1.736-.898a8.494 8.494 0 0 1 1.67-.182h3.916l-1.209.704h-1.207v.005l-.005.002zm.754 11.276c0-.498-.162-.932-.484-1.302-.346-.348-.881-.778-1.609-1.29a4.045 4.045 0 0 0-.439-.02c-.102-.016-.354 0-.768.039a8.141 8.141 0 0 0-1.244.275c-.1.036-.24.096-.42.17a2.713 2.713 0 0 0-.553.355 1.985 1.985 0 0 0-.459.592c-.145.25-.215.551-.215.901 0 .687.311 1.253.932 1.698.596.444 1.404.68 2.432.688.928-.014 1.629-.217 2.117-.612.476-.385.71-.886.71-1.494zm-2.729-5.743a1.843 1.843 0 0 0 1.289-.556c.164-.249.271-.506.318-.767a7.32 7.32 0 0 0 .041-.659c0-.772-.195-1.552-.594-2.336a2.611 2.611 0 0 0-.73-.921 1.894 1.894 0 0 0-1.064-.352 1.723 1.723 0 0 0-1.324.64c-.299.437-.438.92-.428 1.457 0 .709.209 1.444.623 2.211.201.356.459.659.775.912.323.248.686.371 1.094.371zm9.698.077h-2.055v-2.056h-.996v2.056h-2.053v.998h2.053v2.052h.996v-2.052h2.055z"/></symbol><symbol id="icon-gplus" viewBox="0 0 24 24"><circle fill="#D84A36" cx="12" cy="12" r="12"/><g fill="#FFF"><path d="M12.012 12.111c-.304-.33-.07-.842.155-1.078.226-.241.639-.56.829-.74.193-.184.66-.605.674-1.868 0 0 .112-1.139-1.127-2.06h.977l1.038-.593h-3.507s-2.083-.067-3.085 1.207c-1 1.276-.966 2.871.114 3.77 1.08.898 2.371.648 2.371.648s-.197.422-.163.775c.036.353.421.849.504.965 0 0-2.386-.011-3.551 1.033-1.165 1.053-1.29 3.097 1.021 3.803 2.312.707 5.303-.055 5.906-2.277 0 0 .262-.908-.189-1.683-.459-.775-1.663-1.571-1.967-1.902zm-2.555-1.83c-.744-1-1.308-2.953 0-3.783 0 0 .955-.579 1.801.266.846.846 1.082 2.277.928 3.003-.15.727-.563.897-.739 1.008-.185.111-1.249.504-1.99-.494zm3.668 5.481c-.035.393-.083 1.254-1.491 1.664-1.409.408-3.826-.264-3.757-1.914 0 0 .006-.698.449-1.089.445-.396 1.094-.794 3.029-.819 0 0 .643.473 1.072.809.427.332.733.952.698 1.349z"/><path d="M16.323 11.055V9.803h-.752v1.252h-1.255v.748h1.255v1.256h.752v-1.256h1.248v-.748z"/></g></symbol><symbol id="icon-linkedin" viewBox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"/><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"/><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"/></g></symbol><symbol id="icon-reddit" viewBox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"/><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"/><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"/><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"/><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"/><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"/><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"/><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"/></symbol><symbol id="icon-twitter-bordered" viewBox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"/><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/></symbol><symbol id="icon-twitter" viewBox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"/><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"/></symbol><symbol id="icon-weibo-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"/><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"/><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"/></symbol><symbol id="icon-weibo" viewBox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"/><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"/><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"/><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"/><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"/><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"/></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner c-banner--info">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner" >
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1"
                 data-gpt-unitpath="/270604982/bmc/hereditasjournal/articles"
                 data-gpt-sizes="728x90,970x90"
                 data-gpt-targeting="pos=LB1;doi=10.1186/s41065-017-0043-3;kwrd=Turfgrass,Bermudagrass,Off-types,Genotyping-by-sequencing,Morphology,Putting greens;pmc=L32030,L21001;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-017-0043-3&amp;kwrd=Turfgrass,Bermudagrass,Off-types,Genotyping-by-sequencing,Morphology,Putting greens&amp;pmc=L32030,L21001&amp;">
                        <img data-test="gpt-advert-fallback-img"
                             src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-017-0043-3&amp;kwrd=Turfgrass,Bermudagrass,Off-types,Genotyping-by-sequencing,Morphology,Putting greens&amp;pmc=L32030,L21001&amp;"
                             alt="Advertisement"
                             width="728"
                             height="90">
                    </a>
                </noscript>
            </div>
        </div>
    </div>

        
         
    <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>


<header class="c-popup-search ctx-publisher-header" id="top" data-test="publisher-header">
    <nav class="c-navbar">
        <div class="u-container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization" />
                <div class="c-logo">
    <a href="https://www.biomedcentral.com" itemprop="url">
        <img src=/static/images/bmc/logos/logo-bmc-white-aj-e3a70ae848.svg alt="BMC" itemprop="logo">
        <div class="c-logo__strapline">
            <img src=/static/images/bmc/logos/logo-bmc-white-strapline-sn-e7b1d6e67f.svg alt="Part of Springer Nature">
        </div>
    </a>
</div>
            </div>
            <button
                type="button"
                class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg"
                id="publisher-header-menu-button"
                data-toggle="collapse"
                data-test="publisher-header-menu-toggle"
                data-toggle-linked="publisher-header"
                data-target="publisher-header-search publisher-header-menu"
                aria-controls="publisher-header-menu publisher-header-search"
                aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


                <ul class="c-navbar__nav">
                    <li class="c-navbar__item">
                        <a data-replace="true"
                           data-placeholder="/placeholder/v1/username"
                           class="c-navbar__link"
                           href="https://www.biomedcentral.com/login"
                           data-test="login-link"
                           data-track="click"
                           data-track-category="Account"
                           data-track-action="Click Login Link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                    <li class="c-navbar__item">
                        <button
                            type="button"
                            class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg"
                            data-toggle="collapse"
                            data-test="header-search-button"
                            data-target="publisher-header-search publisher-header-menu publisher-header-menu-button"
                            aria-controls="publisher-header-search publisher-header-menu"
                            aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true">
                                    <use xlink:href="#icon-search"></use>
                                </svg>
                            </span>
                        </button>
                    </li>
                </ul>
            </div>
        </div>
    </nav>
    <div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field c-form-field--inline" method="GET" action="//www.biomedcentral.com/search" data-track="submit"
        data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label data-track-label="" data-test="global-search">
        <label for="publisherSearch" class="u-visually-hidden">Search all BMC articles</label>
        <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text" value="" placeholder="Search all BMC articles"/>
        <button class="c-button c-button--secondary" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true">
        <use xlink:href="#icon-search"></use>
    </svg>
</button>


        <input type="hidden" name="searchType" value="publisherSearch"/>
    </form>
</div>

            </div>
        </div>
    </div>
</header>


        
            <header class="c-journal-header c-journal-header--hereditas ctx-journal-header">
                <div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="/">
    
        
    
    <span class="c-journal-title__text ">Hereditas</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="u-container">
                        
                            <button type="button"
                                    class="c-navbar__link c-dropdown u-hidden-at-lg"
                                    data-toggle="collapse"
                                    data-test="journal-header-menu"
                                    data-toggle-linked="journal-header"
                                    data-target="journal-menu"
                                    aria-controls="journal-menu"
                                    aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav">
                                    <ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/'>Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/about'>About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href='/articles'>Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/submission-guidelines'>Submission Guidelines</a>
                                            </li>
                                        
                                    </ul>
                                </nav>
                            </div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--ocean-light-blue">
                    
                </div>
            </header>
            
        

        <div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-content-layout c-content-layout--fulltext" data-component="sticky-container">
                <div class="c-content-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3><ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc=""><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Methods">Methods</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec10" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results">Results</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec13" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Discussion">Discussion</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec16" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Conclusions">Conclusions</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li></ol></nav>
                </div>

                <main class="c-content-layout__main">
                    <article itemscope itemtype="http://schema.org/ScholarlyArticle">
                        
                        
    <div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
            <li class="c-article-identifiers__item" data-test="article-category">Research</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
    </div>

                        <div class="FulltextWrapper"><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">Genotypic and phenotypic evaluation of off-type grasses in hybrid Bermudagrass [<em xmlns="" class="EmphasisTypeItalic">Cynodon dactylon</em> (L.) Pers. x <em xmlns="" class="EmphasisTypeItalic">C. transvaalensis</em> Burtt-Davy] putting greens using genotyping-by-sequencing and morphological characterization</h1></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true"><ul class="u-listReset"><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Eric&nbsp;H.&nbsp;Reasor</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup><a href="mailto:ehr54@msstate.edu" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">James&nbsp;T.&nbsp;Brosnan</span><sup><a href="#Aff2" class="AffiliationID">2</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Margaret&nbsp;E.&nbsp;Staton</span><sup><a href="#Aff3" class="AffiliationID">3</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Thomas&nbsp;Lane</span><sup><a href="#Aff3" class="AffiliationID">3</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Robert&nbsp;N.&nbsp;Trigiano</span><sup><a href="#Aff3" class="AffiliationID">3</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Phillip&nbsp;A.&nbsp;Wadl</span><sup><a href="#Aff4" class="AffiliationID">4</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Joann&nbsp;A.&nbsp;Conner</span><sup><a href="#Aff5" class="AffiliationID">5</a></sup> and </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Brian&nbsp;M.&nbsp;Schwartz</span><sup><a href="#Aff6" class="AffiliationID">6</a></sup></li></ul></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation"><div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">Hereditas</span></span><span class="ArticleCitation_Year">2017</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">155</strong>:8</span></span></div><p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/s41065-017-0043-3">https://doi.org/10.1186/s41065-017-0043-3</a></p><p class="Copyright">© &nbsp;The Author(s)&nbsp;2017</p></div></div><ul class="ArticleHistory u-reset-list"><li class="History HistoryReceived"><strong>Received: </strong><span>25&nbsp;April&nbsp;2017</span></li><li class="History HistoryAccepted"><strong>Accepted: </strong><span>9&nbsp;August&nbsp;2017</span></li><li class="History HistoryOnlineDate"><strong>Published: </strong><span itemprop="datePublished">18&nbsp;August&nbsp;2017</span></li></ul></div>
                        
                        
                        
    

                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2><div class="c-section__content" id="Abstract"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p id="Par1" class="Para">Interspecific hybrid bermudagrass [<em xmlns="" class="EmphasisTypeItalic">Cynodon dactylon</em> (L.) Pers. x <em xmlns="" class="EmphasisTypeItalic">C. transvaalensis</em> Burtt-Davy] is one of the most widely used grasses on golf courses, with cultivars derived from ‘Tifgreen’ or ‘Tifdwarf’ particularly used for putting greens. Many bermudagrass cultivars established for putting greens can be genetically unstable and lead to the occurrence of undesirable off-type grasses that vary in phenotype. The objective of this research was to genetically and phenotypically differentiate off-type grasses and hybrid cultivars. Beginning in 2013, off-type and desirable hybrid bermudagrass samples were collected from golf course putting greens in the southeastern United States and genetically and phenotypically characterized using genotyping-by-sequencing and morphology.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Results</h3>
                <p id="Par2" class="Para">Genotyping-by-sequencing determined that 11% (5) of off-type and desirable samples from putting greens were genetically divergent from standard cultivars such as Champion, MiniVerde, Tifdwarf, TifEagle, and Tifgreen. In addition, genotyping-by-sequencing was unable to genetically distinguish all standard cultivars from one another due to their similar origin and clonal propagation; however, over 90,000 potentially informative nucleotide variants were identified among the triploid hybrid cultivars.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusions</h3>
                <p id="Par3" class="Para">Although few genetic differences were found in this research, samples harvested from golf course putting greens had variable morphology and were clustered into three distinct phenotypic groups. The majority of off-type grasses in hybrid bermudagrass putting greens were genetically similar with variable morphological traits. Off-type grasses within golf course putting greens have the potential to compromise putting surface functionality and aesthetics.</p>
              </div></div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords"><li class="c-keywords__item" itemprop="about">Turfgrass</li><li class="c-keywords__item" itemprop="about">Bermudagrass</li><li class="c-keywords__item" itemprop="about">Off-types</li><li class="c-keywords__item" itemprop="about">Genotyping-by-sequencing</li><li class="c-keywords__item" itemprop="about">Morphology</li><li class="c-keywords__item" itemprop="about">Putting greens</li></ul></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par4" class="Para">The economic impact of the golf industry in the United States (U.S.) in 2011 was estimated to be $176.8 billion with a contribution of approximately 1.98 million jobs (SRI International; <span class="ExternalRef"><a href="http://wearegolf.org/economy/impact"><span class="RefSource">http://wearegolf.org/economy/impact</span></a></span>). Interspecific hybrid bermudagrasses (<em xmlns="" class="EmphasisTypeItalic">Cynodon dactylon</em> (L.) Pers. x <em xmlns="" class="EmphasisTypeItalic">C. transvaalensis</em> Burtt-Davy) are some of the most widely utilized grasses on golf courses throughout tropical, subtropical, and temperate climates due to their desirable turfgrass characteristics of texture, color, and stress tolerance [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>]. Putting greens are a vital aspect of a golf course and in 2007 hybrid bermudagrasses were grown on 80% of putting green acreage in the southern U.S. [<span class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par5" class="Para">‘Tifgreen’ was one of the first interspecific hybrids developed for putting green use [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>, <span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>]. Soon after its commercial release, ‘Tifdwarf’ was selected from a somatic mutation in a ‘Tifgreen’ establishment [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>, <span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>]. Despite being genetically unstable [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>, <span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>], ‘Tifgreen’ and ‘Tifdwarf’ are used on putting surfaces; however, superior mutations have been released as new “ultradwarf” cultivars [<span class="CitationRef"><a href="#CR36" data-toggle="citation">36</a></span>]. Ultradwarf bermudagrass cultivars are classified due to their more diminutive morphology (i.e., internode length and leaf length and width) compared to ‘Tifdwarf’. ‘Champion’ and ‘MiniVerde’ were selected from somatic mutations in established ‘Tifdwarf’ plantings [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>, <span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>], whereas ‘TifEagle’ was a putative mutant from radiation-induced ‘Tifgreen’ or ‘Tifway II’ rhizome [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>, <span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>, <span class="CitationRef"><a href="#CR41" data-toggle="citation">41</a></span>]. In all cases, “ultradwarf” cultivars such as ‘Champion’, ‘MiniVerde’, and ‘TifEagle’ were once identified as off-types.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par6" class="Para">Morphological characterization has been considered a traditional method of studying turfgrass classification and diversity [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>, <span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>, <span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>, <span class="CitationRef"><a href="#CR38" data-toggle="citation">38</a></span>]. Morphological characteristics such as internode length, leaf length, leaf width, and stolon diameter are of particular interest for classification of bermudagrasses [<span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>, <span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>, <span class="CitationRef"><a href="#CR38" data-toggle="citation">38</a></span>] because differences in morphology can differentiate off-types from desirable cultivars [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>, <span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>]. Additionally, Roche and Loch [<span class="CitationRef"><a href="#CR37" data-toggle="citation">37</a></span>] stated that morphological characterization could provide useful information to further research for adaptation and management of different hybrid bermudagrasses. Researchers have used morphological characterizations to compare hybrid bermudagrass cultivars within the ‘Tifgreen’ family [<span class="CitationRef"><a href="#CR31" data-toggle="citation">31</a></span>, <span class="CitationRef"><a href="#CR37" data-toggle="citation">37</a></span>]; however, morphological inconsistencies suggest that molecular techniques are also needed to accurately evaluate hybrid bermudagrass diversity [<span class="CitationRef"><a href="#CR36" data-toggle="citation">36</a></span>].</p><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par7" class="Para">Several methods have been used to explore genotypic differences among off-type grasses and hybrid bermudagrass cultivars. DNA amplification fingerprinting (DAF) and signatures from amplification profiles identified contaminant off-types not related to the ‘Tifgreen’ family, but could not distinguish mutant off-types within the ‘Tifgreen’ family [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>]. Amplified fragment length polymorphisms (AFLPs) studies determined the genetic diversity among several ‘Tifgreen’-derived cultivars. Studies utilizing AFLPs grouped ‘Tifgreen’, ‘Tifdwarf’, ‘TifEagle’, and ‘Champion’ into the same genetic cluster despite the grasses having differing phenotypic characteristics [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>, <span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>, <span class="CitationRef"><a href="#CR41" data-toggle="citation">41</a></span>]. Attempts to use various simple-sequence repeats (SSRs) to identify ‘Tifgreen’-derived hybrid bermudagrass cultivars met with limited success as well [<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>, <span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>]. While SSRs identified ‘TifEagle’ from other ‘Tifgreen’-derived cultivars and identified polymorphisms unique to ‘Tifdwarf’ and ‘MiniVerde’ [<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>, <span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>], SSRs are not able to readily distinguish all ‘Tifgreen’-derived hybrid bermudagrass cultivars from one another nor have they been able to identify weedy off-type grasses from standard cultivars used on golf course putting greens. Fig. <span class="InternalRef"><a href="#Fig1">1</a></span> displays the aesthetic and functional disruptions that off-type grasses in bermudagrass putting greens can create.<figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO1">
                    <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0043-3/MediaObjects/41065_2017_43_Fig1_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0043-3/MediaObjects/41065_2017_43_Fig1_HTML.gif" alt="Fig. 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0043-3/MediaObjects/41065_2017_43_Fig1_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0043-3/MediaObjects/41065_2017_43_Fig1_HTML.gif" alt="Fig. 1"></img></noscript></a>
                  </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 1</span>
                      <p xmlns="" class="SimplePara">
                                    <strong class="EmphasisTypeBold">a</strong> Off-type grasses (lighter in color and noted by red circle) present in an ultradwarf bermudagrass (<em class="EmphasisTypeItalic">Cynodon dactylon</em> (L.) Pers. x <em class="EmphasisTypeItalic">C. transvaalensis</em> Burtt-Davy) putting green. The difference in turfgrass color between desirable and off-type grasses disrupts aesthetic uniformity of the putting surface. <strong class="EmphasisTypeBold">b</strong> Close-up of an off-type grass patch (noted by red circle) present in an ultradwarf bermudagrass putting green. The difference in growth rate between the desirable and off-type grasses has the potential to disrupt the functional uniformity of putting surfaces with off-type infestations. Figure was generated using Keynote (v6.6.2)</p>
                    </div></figcaption></figure>
                     </div><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par8" class="Para">Genotyping-by-sequencing (GBS) is a high-throughput, next generation sequencing method capable of generating large numbers of single nucleotide polymorphism (SNPs) from species with high diversity [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>]. GBS offers several advantages over other molecular marker techniques including the amount data generated and the price per sample [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>, <span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>]. Moreover, GBS allows analysis of a species (e.g., hybrid bermudagrass) for which a complete reference genome sequence is not available. Fiedler et al. [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>] identified over 4600 high-quality SNPs in switchgrass (<em xmlns="" class="EmphasisTypeItalic">Panicum virgatum</em>) using an early draft genome assembly with only half of the assembled DNA contigs scaffolded. Poland et al. [<span class="CitationRef"><a href="#CR32" data-toggle="citation">32</a></span>] used GBS to map over 34,000 SNPs for the Oregon Wolfe Barley (<em xmlns="" class="EmphasisTypeItalic">Hordeum vulgare</em>) reference population and 20,000 SNPs for the Synthetic W9784xOpata85 wheat (<em xmlns="" class="EmphasisTypeItalic">Triticum aestivum</em>) reference population, both of which lacked a complete reference genome sequence. Based on the robustness of the technique and successful use in other grasses without a complete reference genome [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>, <span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>, <span class="CitationRef"><a href="#CR32" data-toggle="citation">32</a></span>, <span class="CitationRef"><a href="#CR33" data-toggle="citation">33</a></span>], we hypothesize that GBS may be able to identify genetic variation among off-types and hybrid bermudagrasses used on putting greens. Therefore, our objectives were to explore the genetic and the phenotypic variation among off-type grasses sampled from hybrid bermudagrass putting greens using GBS and morphological characterization.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Methods" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Methods">Methods</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Methods" data-test="collapse-section"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par9" class="Para">Single replicates of desirable and off-type hybrid bermudagrass samples were harvested in 2013 from putting greens on golf courses in Alabama, Arkansas, Florida, Mississippi, South Carolina, and Tennessee (Table <span class="InternalRef"><a href="#Tab1">1</a></span>). The greenkeeper at each golf course determined samples that were desirable from those that were off-type grasses. Samples were harvested with a 7.5&nbsp;cm diameter tubular plugger (Turf Tec International; Tallahassee, FL, USA) and established using one three node stolon planted in a 64&nbsp;cm<sup>2</sup> pot filled with a peat moss based growing medium (Pro-Mix BX Mycorrhizae; Premier Horticulture, Inc.; Quakertown, PA, USA) in a glasshouse environment at the University of Tennessee (Knoxville, TN, USA; 35.5°N, −83.5°W). Plants were maintained with 24&nbsp;kg N ha<sup>−1</sup> wk.<sup>−1</sup> of a water-soluble complete fertilizer (20&nbsp;N-8.7P-16.6&nbsp;K; Southern Agriculture; Hendersonville, NC, USA), irrigated to promote active growth, and insecticides (abamectin 0.01&nbsp;kg ai ha<sup>−1</sup>, Avid 0.15EC, Syngenta; pymetrozine 0.35&nbsp;kg ai ha<sup>−1</sup>, Endeavor, Syngenta) were applied on a preventive basis.<figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 1</span>
                      <p xmlns="" class="SimplePara">Plant material used in genetic and phenotypic evaluation of off-type grasses in ultradwarf hybrid bermudagrass putting greens (<em class="EmphasisTypeItalic">Cynodon dactylon</em> (L.) Pers. x <em class="EmphasisTypeItalic">C. transvaalensis</em> Burtt-Davy)</p>
                    </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col></col><col></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Sample</p></th><th><p xmlns="" class="SimplePara">Sample Origin<sup xmlns="http://www.w3.org/1999/xhtml">a</sup>
                                       </p></th><th><p xmlns="" class="SimplePara">USA State</p></th><th><p xmlns="" class="SimplePara">Ploidy<sup xmlns="http://www.w3.org/1999/xhtml">b</sup>
                                       </p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">S1</p></td><td><p xmlns="" class="SimplePara">Champion (DS)</p></td><td><p xmlns="" class="SimplePara">TN</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S2</p></td><td><p xmlns="" class="SimplePara">Champion (DS)</p></td><td><p xmlns="" class="SimplePara">MS</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S3</p></td><td><p xmlns="" class="SimplePara">Champion (DS)</p></td><td><p xmlns="" class="SimplePara">TN</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S4</p></td><td><p xmlns="" class="SimplePara">Champion (DS)</p></td><td><p xmlns="" class="SimplePara">TN</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S5</p></td><td><p xmlns="" class="SimplePara">Champion (DS)</p></td><td><p xmlns="" class="SimplePara">TN</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S6</p></td><td><p xmlns="" class="SimplePara">Champion (DS)</p></td><td><p xmlns="" class="SimplePara">TN</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S7</p></td><td><p xmlns="" class="SimplePara">Champion (DS)</p></td><td><p xmlns="" class="SimplePara">TN</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S8</p></td><td><p xmlns="" class="SimplePara">Champion (DS)</p></td><td><p xmlns="" class="SimplePara">MS</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S9</p></td><td><p xmlns="" class="SimplePara">Champion (DS)</p></td><td><p xmlns="" class="SimplePara">TN</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S10</p></td><td><p xmlns="" class="SimplePara">Champion (DS)</p></td><td><p xmlns="" class="SimplePara">MS</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S11</p></td><td><p xmlns="" class="SimplePara">Champion (DS)</p></td><td><p xmlns="" class="SimplePara">AR</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S12</p></td><td><p xmlns="" class="SimplePara">Champion (DS)</p></td><td><p xmlns="" class="SimplePara">TN</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S13</p></td><td><p xmlns="" class="SimplePara">MiniVerde (DS)</p></td><td><p xmlns="" class="SimplePara">FL</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S14</p></td><td><p xmlns="" class="SimplePara">MiniVerde (DS)</p></td><td><p xmlns="" class="SimplePara">FL</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S15</p></td><td><p xmlns="" class="SimplePara">MiniVerde (DS)</p></td><td><p xmlns="" class="SimplePara">TN</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S16</p></td><td><p xmlns="" class="SimplePara">MiniVerde (DS)</p></td><td><p xmlns="" class="SimplePara">TN</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S17</p></td><td><p xmlns="" class="SimplePara">TifEagle (DS)</p></td><td><p xmlns="" class="SimplePara">AL</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S18</p></td><td><p xmlns="" class="SimplePara">TifEagle (DS)</p></td><td><p xmlns="" class="SimplePara">TN</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S19</p></td><td><p xmlns="" class="SimplePara">MiniVerde (OT)</p></td><td><p xmlns="" class="SimplePara">FL</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S20</p></td><td><p xmlns="" class="SimplePara">MiniVerde (OT)</p></td><td><p xmlns="" class="SimplePara">FL</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S21</p></td><td><p xmlns="" class="SimplePara">Champion (OT)</p></td><td><p xmlns="" class="SimplePara">TN</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S22</p></td><td><p xmlns="" class="SimplePara">Champion (OT)</p></td><td><p xmlns="" class="SimplePara">MS</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S23</p></td><td><p xmlns="" class="SimplePara">Champion (OT)</p></td><td><p xmlns="" class="SimplePara">TN</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S24</p></td><td><p xmlns="" class="SimplePara">Champion (OT)</p></td><td><p xmlns="" class="SimplePara">TN</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S25</p></td><td><p xmlns="" class="SimplePara">Champion (OT)</p></td><td><p xmlns="" class="SimplePara">TN</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S26</p></td><td><p xmlns="" class="SimplePara">Champion (OT)</p></td><td><p xmlns="" class="SimplePara">TN</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S27</p></td><td><p xmlns="" class="SimplePara">Champion (OT)</p></td><td><p xmlns="" class="SimplePara">TN</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S28</p></td><td><p xmlns="" class="SimplePara">Champion (OT)</p></td><td><p xmlns="" class="SimplePara">TN</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S29</p></td><td><p xmlns="" class="SimplePara">Champion (OT)</p></td><td><p xmlns="" class="SimplePara">TN</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S30</p></td><td><p xmlns="" class="SimplePara">Champion (OT)</p></td><td><p xmlns="" class="SimplePara">TN</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S31</p></td><td><p xmlns="" class="SimplePara">Champion (OT)</p></td><td><p xmlns="" class="SimplePara">MS</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S32</p></td><td><p xmlns="" class="SimplePara">Champion (OT)</p></td><td><p xmlns="" class="SimplePara">MS</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S33</p></td><td><p xmlns="" class="SimplePara">Champion (OT)</p></td><td><p xmlns="" class="SimplePara">TN</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S34</p></td><td><p xmlns="" class="SimplePara">Champion (OT)</p></td><td><p xmlns="" class="SimplePara">MS</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S35</p></td><td><p xmlns="" class="SimplePara">Champion (OT)</p></td><td><p xmlns="" class="SimplePara">AR</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S36</p></td><td><p xmlns="" class="SimplePara">Champion (OT)</p></td><td><p xmlns="" class="SimplePara">TN</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S37</p></td><td><p xmlns="" class="SimplePara">Champion (OT)</p></td><td><p xmlns="" class="SimplePara">TN</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S38</p></td><td><p xmlns="" class="SimplePara">MiniVerde (OT)</p></td><td><p xmlns="" class="SimplePara">TN</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S39</p></td><td><p xmlns="" class="SimplePara">TifEagle (OT)</p></td><td><p xmlns="" class="SimplePara">AL</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S40</p></td><td><p xmlns="" class="SimplePara">TifEagle (OT)</p></td><td><p xmlns="" class="SimplePara">AL</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S41</p></td><td><p xmlns="" class="SimplePara">MiniVerde (OT)</p></td><td><p xmlns="" class="SimplePara">TN</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S42</p></td><td><p xmlns="" class="SimplePara">MiniVerde (DS)</p></td><td><p xmlns="" class="SimplePara">TN</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S43</p></td><td><p xmlns="" class="SimplePara">MiniVerde (OT)</p></td><td><p xmlns="" class="SimplePara">FL</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S44</p></td><td><p xmlns="" class="SimplePara">MiniVerde (OT)</p></td><td><p xmlns="" class="SimplePara">MS</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S45</p></td><td><p xmlns="" class="SimplePara">MiniVerde (OT)</p></td><td><p xmlns="" class="SimplePara">MS</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S46</p></td><td><p xmlns="" class="SimplePara">MiniVerde (OT)</p></td><td><p xmlns="" class="SimplePara">TN</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S47</p></td><td><p xmlns="" class="SimplePara">TifEagle (DS)</p></td><td><p xmlns="" class="SimplePara">AL</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S48</p></td><td><p xmlns="" class="SimplePara">TifEagle (DS)</p></td><td><p xmlns="" class="SimplePara">AL</p></td><td><p xmlns="" class="SimplePara">NA</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S49</p></td><td><p xmlns="" class="SimplePara">Champion (OT)</p></td><td><p xmlns="" class="SimplePara">TN</p></td><td><p xmlns="" class="SimplePara">NA</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S50</p></td><td><p xmlns="" class="SimplePara">Champion (OT)</p></td><td><p xmlns="" class="SimplePara">TN</p></td><td><p xmlns="" class="SimplePara">NA</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S51</p></td><td><p xmlns="" class="SimplePara">Champion (OT)</p></td><td><p xmlns="" class="SimplePara">GA</p></td><td><p xmlns="" class="SimplePara">NA</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S52</p></td><td><p xmlns="" class="SimplePara">Champion (OT)</p></td><td><p xmlns="" class="SimplePara">GA</p></td><td><p xmlns="" class="SimplePara">NA</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S53</p></td><td><p xmlns="" class="SimplePara">Champion (OT)</p></td><td><p xmlns="" class="SimplePara">TN</p></td><td><p xmlns="" class="SimplePara">NA</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S54</p></td><td><p xmlns="" class="SimplePara">Champion (OT)</p></td><td><p xmlns="" class="SimplePara">TN</p></td><td><p xmlns="" class="SimplePara">NA</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S55</p></td><td><p xmlns="" class="SimplePara">MiniVerde (DS)</p></td><td><p xmlns="" class="SimplePara">TN</p></td><td><p xmlns="" class="SimplePara">NA</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S56</p></td><td><p xmlns="" class="SimplePara">Champion (DS)</p></td><td><p xmlns="" class="SimplePara">SC</p></td><td><p xmlns="" class="SimplePara">NA</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S57</p></td><td><p xmlns="" class="SimplePara">Champion (DS)</p></td><td><p xmlns="" class="SimplePara">GA</p></td><td><p xmlns="" class="SimplePara">NA</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S58</p></td><td><p xmlns="" class="SimplePara">TifEagle (OT)</p></td><td><p xmlns="" class="SimplePara">MS</p></td><td><p xmlns="" class="SimplePara">NA</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S59</p></td><td><p xmlns="" class="SimplePara">Champion (OT)</p></td><td><p xmlns="" class="SimplePara">GA</p></td><td><p xmlns="" class="SimplePara">NA</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S60</p></td><td><p xmlns="" class="SimplePara">TifEagle (DS)</p></td><td><p xmlns="" class="SimplePara">TN</p></td><td><p xmlns="" class="SimplePara">NA</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S61</p></td><td><p xmlns="" class="SimplePara">TifEagle (DS)</p></td><td><p xmlns="" class="SimplePara">MS</p></td><td><p xmlns="" class="SimplePara">NA</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">S62</p></td><td><p xmlns="" class="SimplePara">MiniVerde (DS)</p></td><td><p xmlns="" class="SimplePara">TN</p></td><td><p xmlns="" class="SimplePara">NA</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">CH1–6</p></td><td><p xmlns="" class="SimplePara">Champion (ST)</p></td><td><p xmlns="" class="SimplePara">GA</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">MV1–6</p></td><td><p xmlns="" class="SimplePara">MiniVerde (ST)</p></td><td><p xmlns="" class="SimplePara">GA</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">TD1–6</p></td><td><p xmlns="" class="SimplePara">Tifdwarf (ST)</p></td><td><p xmlns="" class="SimplePara">GA</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">TE1–6</p></td><td><p xmlns="" class="SimplePara">TifEagle (ST)</p></td><td><p xmlns="" class="SimplePara">GA</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">TG1–6</p></td><td><p xmlns="" class="SimplePara">Tifgreen (ST)</p></td><td><p xmlns="" class="SimplePara">GA</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">TW1–6</p></td><td><p xmlns="" class="SimplePara">Tifway (ST)</p></td><td><p xmlns="" class="SimplePara">GA</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">TA1–3</p></td><td><p xmlns="" class="SimplePara">
                                          <em class="EmphasisTypeItalic">C. dactylon</em> (ST)</p></td><td><p xmlns="" class="SimplePara">GA</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;4×&nbsp;=&nbsp;36</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">TB1–3</p></td><td><p xmlns="" class="SimplePara">
                                          <em class="EmphasisTypeItalic">C. dactylon</em> (ST)</p></td><td><p xmlns="" class="SimplePara">GA</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;4×&nbsp;=&nbsp;36</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">DA1–3</p></td><td><p xmlns="" class="SimplePara">
                                          <em class="EmphasisTypeItalic">C. transvaalensis</em> (ST)</p></td><td><p xmlns="" class="SimplePara">GA</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;2×&nbsp;=&nbsp;18</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">DB1–3</p></td><td><p xmlns="" class="SimplePara">
                                          <em class="EmphasisTypeItalic">C. transvaalensis</em> (ST)</p></td><td><p xmlns="" class="SimplePara">GA</p></td><td><p xmlns="" class="SimplePara">2n&nbsp;=&nbsp;2×&nbsp;=&nbsp;18</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                    <p xmlns="" class="SimplePara">Selections included 62 desirable (DS) and off-type (OT) ultradwarf bermudagrasses sampled from golf course putting greens in TN, MS, AR, FL, AL, GA, and SC. Six standard (ST) hybrid bermudagrass cultivars [Champion (CH1–6), MiniVerde (MV1–6), Tifdwarf (TD1–6), TifEagle (TE1–6), Tifgreen (TG1–6), and Tifway (TW1–6)] and two progenitor species [(<em class="EmphasisTypeItalic">C. dactylon</em> (TA1–3 and TB1–3) and <em class="EmphasisTypeItalic">C. transvaalensis</em> (DA1–3 and DB1–3)] were included in the analysis for comparison. Ploidy level was confirmed using flow cytometry</p>
                    <p xmlns="" class="SimplePara">
                                 <sup xmlns="http://www.w3.org/1999/xhtml">a</sup>Desirable and off-type samples were harvested from golf course putting greens. Standard samples were provided by the University of Georgia Coastal Plain Experiment Station in Tifton, GA</p>
                    <p xmlns="" class="SimplePara">
                                 <sup xmlns="http://www.w3.org/1999/xhtml">b</sup>Ploidy was confirmed using flow cytometry. Ploidy was not confirmed for samples with “NA”</p>
                  </div></figure>
                     </div><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par10" class="Para">Ploidy levels were confirmed for each sample included in the GBS assay using flow cytometry at the University of Georgia Coastal Plain Experiment Station (Tifton, GA). Fresh leaf tissue was isolated from samples and chopped using a razor in 300&nbsp;μL of LB01-lysis buffer (15&nbsp;mM Tris, 2&nbsp;mM Na<sub>2</sub>EDTA, 0.5&nbsp;mM spermine-4HCl, 80&nbsp;mM KCl, 20&nbsp;mM NaCl, 0.1% <em xmlns="" class="EmphasisTypeItalic">v</em>/v Triton X-100 pH&nbsp;7.5 and 16&nbsp;mM ß-mercaptoethanol) to release nuclei. Each bermudagrass sample was combined with <em xmlns="" class="EmphasisTypeItalic">Sorghum bicolor</em> cv. BTx623 for a standard genome size comparison. Samples were passed through a 30-μm filter (CellTrics; Partec; Munster, Germany) and then 150&nbsp;μL of RNase and propidium iodide solution (PI/RNase Staining Buffer, BD Biosciences, San Jose, CA, USA) was added. Samples were incubated on ice for 15&nbsp;min and analyzed on an Accuri C6 flow cytometer (BD Biosciences; San Jose, CA, USA). Gating was set by the selection of objects that exhibited a strong correlation between the FL2 and FL3 signals using a flow rate of 14&nbsp;μL per minute and a minimum cell count of 10,000. The mean FL2-A peaks from the signals were determined for <em xmlns="" class="EmphasisTypeItalic">S. bicolar</em> and each unknown hybrid bermudagrass sample using Accuri C6 software (BD Biosciences; San Jose, CA, USA). These mean FL2-A values were then used with <em xmlns="" class="EmphasisTypeItalic">S. bicolar</em> genome size (1.67&nbsp;pg/2C) to calculate the genome size of each unknown hybrid bermudagrass sample [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>].</p><section id="Sec3" class="Section2 RenderAsSection2">
                <h3 class="Heading">Genotyping-by-sequencing</h3>
                <section id="Sec4" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Plant material and DNA isolation</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par11" class="Para">Desirable and off-type samples labeled S1 to S47 in Table <span class="InternalRef"><a href="#Tab1">1</a></span> were included in GBS due to the expense of the analysis. Hybrid bermudagrass cultivars [Champion (CH1–6), MiniVerde (MV1–6), Tifdwarf (TD1–6), TifEagle (TE1–6), Tifgreen (TG1–6), and Tifway (TW1–6)] and progenitor species [<em xmlns="" class="EmphasisTypeItalic">C. dactylon</em> (TA1–3 and TB1–3) and <em xmlns="" class="EmphasisTypeItalic">C. transvaalensis</em> (DA1–3 and DB1–3)] were used as standards in the GBS analysis. Only three biological replicates of the progenitor species were included also due to the expense of the analysis. Plant material for these standard entries was obtained from the University of Georgia Coastal Plain Experiment Station (Tifton, GA, USA).</p>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par12" class="Para">For all samples, plant genomic DNA was isolated from actively growing leaf tissue on a single stolon using the Qiagen DNeasy Plant Mini Kit (Qiagen; Valencia, California, USA) according to the manufacturer’s protocol. DNA concentration was quantified using an intercalating dye (Quant-iT™ PicoGreen dsDNA Asasy Kit; Life Technologies; Carlsbad, CA, USA). DNA working solutions for the GBS protocol had a total volume of 30&nbsp;μL and a concentration ranging from 50 to 105&nbsp;ng μL<sup>−1</sup>.</p>
                </section>
                <section id="Sec5" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Genotyping-by-sequencing analysis</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par13" class="Para">Genotyping-by-sequencing was conducted at the Cornell University Institute for Biotechnology (Ithaca, NY) using the protocol described by Elshire et al. [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>]. <em xmlns="" class="EmphasisTypeItalic">Ape</em>KI restriction enzyme was selected based on optimization trials for the GBS digestion to maximize the number of sampled genomic loci [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>]. Libraries for next-generation sequencing were constructed from DNA samples and multiplexed using Illumina HiSeq 2500 and then Illumina NextSeq 500 to increase read coverage and depth.</p>
                </section>
                <section id="Sec6" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Bioinformatics analysis</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par14" class="Para">The combined Illumina data sets were initially analyzed with the UNEAK pipeline of the Tassel software package [<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>, <span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>]. One of the limitations of the UNEAK pipeline is that its nucleotide variant calling algorithm relies on a diploid model. Bermudagrasses sequenced in our GBS analysis included diploid (2n&nbsp;=&nbsp;2×&nbsp;=&nbsp;18), triploid (2n&nbsp;=&nbsp;3×&nbsp;=&nbsp;27), and tetraploid (2n&nbsp;=&nbsp;4×&nbsp;=&nbsp;36) samples; therefore, an alternative approach was used to call variants. Sequence tags with a predicted variant were extracted from the topm.bin libraries generated during the Tag-Pair-Export phase using the UNEAK Binary-to-text-plugin; then raw reads were mapped with Bowtie2 v2.2.7 to these tags generated from the UNEAK pipeline as a pseudo-reference [<span class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>].</p>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par15" class="Para">The haplotype-based variants caller, Freebayes v1.0.2–15, was used to call variants for each set of samples with the same ploidy level with the correct ploidy level specified with parameter p [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>]. The sorting, indexing, and merging of alignment files was performed with the SAMtools v1.3 package [<span class="CitationRef"><a href="#CR28" data-toggle="citation">28</a></span>]. Multidimensional scaling (MDS) plots were generated from these variants using PLINK v1.9 to illustrate the variation among samples. The bermudagrass samples S19, S28, S30, S32, and S44 were not included because they had less than one million raw-reads [<span class="CitationRef"><a href="#CR35" data-toggle="citation">35</a></span>]. The individual samples for each triploid cultivar were pooled to increase the read depth for each cultivar. The read alignment files were pooled using SAMtools v1.3 package [<span class="CitationRef"><a href="#CR28" data-toggle="citation">28</a></span>] and then the FreeBayes method of determining variants was utilized again. The pooled data was then used in a custom Python script to determine loci that differed between at least two cultivars (<span class="ExternalRef"><a href="http://github.com/statonlab/UDBG_Informative_SNPs/blob/master/find_informative_SNPs.py"><span class="RefSource">github.com/statonlab/UDBG_Informative_SNPs/blob/master/find_informative_SNPs.py</span></a></span>). Any loci with three different genotypes (homozygous for the reference allele, homozygous for the alternate allele, or heterozygous) for at least two cultivars were flagged. Loci were not flagged if heterozygosity differed between or within the two subgenomes in triploid cultivars (i.e., 0/0/1 or 0/1/1). Lastly, each cultivar was filtered to use only individual variants with a read depth greater than or equal to 40. All raw read data has been submitted to NCBI under BioProject accession PRJNA353769.</p>
                </section>
              </section><section id="Sec7" class="Section2 RenderAsSection2">
                <h3 class="Heading">Phenotypic evaluation</h3>
                <section id="Sec8" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Plant materials</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par16" class="Para">Bermudagrass samples labeled S1 to S62, except for S13, S14, S19, S20, and S43 were used in phenotypic evaluation (Table <span class="InternalRef"><a href="#Tab1">1</a></span>). Bermudagrass samples S13, S14, S19, S20, and S43 were excluded from phenotypic evaluation due to their harvest date. The greenkeeper at each golf course determined samples that were desirable from those that were off-type grasses. Samples were harvested with a 7.5&nbsp;cm diameter tubular plugger (Turf Tec International; Tallahassee, FL, USA) and established during the summer of 2013 using one three node stolon planted in a 64&nbsp;cm<sup>2</sup> pot filled with a peat moss based growing medium (Pro-Mix BX Mycorrhizae; Premier Horticulture, Inc.; Quakertown, PA, USA) in a glasshouse environment at the University of Tennessee (Knoxville, TN, USA; 35.5°N, −83.5°W). Plants were maintained with 24&nbsp;kg N ha<sup>−1</sup> wk.<sup>−1</sup> of a water-soluble complete fertilizer (20&nbsp;N-8.7P-16.6&nbsp;K; Southern Agriculture; Hendersonville, NC, USA), irrigated to promote active growth, and insecticides (abamectin 0.01&nbsp;kg ai ha<sup>−1</sup>, Avid 0.15EC, Syngenta; pymetrozine 0.35&nbsp;kg ai ha<sup>−1</sup>, Endeavor, Syngenta) were applied on a preventive basis.</p>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par17" class="Para">For phenotypic evaluation, one three-node stolon of each sample was established in four, 64&nbsp;cm<sup>2</sup> pots filled with a peat moss based growing medium (Pro-Mix BX Mycorrhizae; Premier Horticulture, Inc.; Quakertown, PA, USA) on 7 April 2014. The stolon length at transplanting of the 52 selections ranged from 3.4 to 11.3&nbsp;cm. The plants were maintained as previously described but regular clipping was ceased 2 weeks prior to evaluation.</p>
                </section>
                <section id="Sec9" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Morphological measurements and statistical analysis</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par18" class="Para">Phenotypic evaluation of off-type and desirable samples was conducted by measuring plant morphological characteristics via methods outlined by Roche and Loch [<span class="CitationRef"><a href="#CR37" data-toggle="citation">37</a></span>]. Five parameters were assessed and included internode length and stolon diameter, leaf length and width, and the leaf length:width ratio (LWR). Measurements were made between the third and fourth node and on the outer leaf from the third node using digital calipers (Digimatic Caliper, Model No.CD-6″ CX, Mitutoyo Corporation, Kawasaki-shi, Kanagawa, Japan). The experiment was a completely randomized design with pots replicated four times and morphology measured on three stolons per pot. Morphology was assessed on 3 June 2014 and repeated again on 25 June 2014.</p>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par19" class="Para">All morphological data describing desirable and off-type hybrid bermudagrass samples were analyzed using cluster analysis in SAS Enterprise Guide (Version 6.1, SAS Institute, Cary, NC, USA). K-means clustering algorithm was used to partition the data set into a user-defined number of clusters [<span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>]. Three clusters were determined based on the cubic clustering criterion and the frequency of observations in each cluster [<span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>]. Cluster means and standard deviations for each morphological measurement were then graphed in Prism (Prism 6 for Mac OS X; GraphPad Software, Inc.) to determine statistical differences among cluster means.</p>
                </section>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec10" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Results" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Results">Results</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Results" data-test="collapse-section"><section id="Sec11" class="Section2 RenderAsSection2">
                <h3 class="Heading">Genotyping-by-sequencing</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par20" class="Para">Over 878 million reads were generated through Illumina sequencing, with 271 million from the Illumina HiSeq platform and an additional 606 million from the same libraries run on the Illumina NextSeq platform. Of the 878 million reads, 757 million were determined to be of high quality and were able to be demultiplexed and assigned to individual plant samples. After excluding five samples with less than a million reads, the remaining individual samples had a range of 1.3 (sample S44) million reads to 13.9 million reads (sample S22). The variants yielded from GBS analysis include single nucleotide variants, multiple nucleotide variants, and indels. An initial 1,088,920 total variants were identified with an average read depth of 4.9 sequences for the triploid bermudagrass (<em xmlns="" class="EmphasisTypeItalic">C. dactylon</em> x <em xmlns="" class="EmphasisTypeItalic">C. transvaalensis</em>) samples. GBS analysis identified 347,512 total variants with an average read depth of 9.5 per individual for the two diploid, <em xmlns="" class="EmphasisTypeItalic">C. transvaalensis</em> selections. For the tetraploid, <em xmlns="" class="EmphasisTypeItalic">C. dactylon</em> samples, 587,053 total variants were identified with an average sequence read depth of 7.4 (Fig. <span class="InternalRef"><a href="#Fig2">2</a></span>). Only 136,205 variants were shared among diploid, triploid, and tetraploid species; therefore, the remaining variants are fixed in at least one species (Fig. <span class="InternalRef"><a href="#Fig2">2</a></span>).<figure xmlns="" class="Figure" id="Fig2"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO2">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0043-3/MediaObjects/41065_2017_43_Fig2_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0043-3/MediaObjects/41065_2017_43_Fig2_HTML.gif" alt="Fig. 2" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0043-3/MediaObjects/41065_2017_43_Fig2_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0043-3/MediaObjects/41065_2017_43_Fig2_HTML.gif" alt="Fig. 2"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 2</span>
                        <p xmlns="" class="SimplePara">Venn diagram showing the number of total and shared nucleotide variants from GBS for <em class="EmphasisTypeItalic">Cynodon transvaalensis</em>, <em class="EmphasisTypeItalic">C. dactylon</em>, and <em class="EmphasisTypeItalic">C. dactylon</em> x <em class="EmphasisTypeItalic">C. transvaalensis.</em> Ploidy levels were confirmed using flow cytometry and are noted in parentheses. Figure was generated using Keynote (v.6.6.2)</p>
                      </div></figcaption></figure>
                        </div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par21" class="Para">The majority of samples harvested from golf courses clustered with the hybrid bermudagrass cultivars Champion, MiniVerde, Tifdwarf, TifEagle, and Tifgreen in the MDS plot (Fig. <span class="InternalRef"><a href="#Fig3">3</a></span>). The clustering suggested that samples from golf courses were genetically similar to those hybrid bermudagrass cultivars typically established on golf course putting greens (Fig. <span class="InternalRef"><a href="#Fig3">3</a></span>). Of the 47 unknown samples, only five (~11%) were genetically divergent from the standard cultivars (S4, S16, S31, S33, and S45), as illustrated by the MDS plot (Fig. <span class="InternalRef"><a href="#Fig3">3</a></span>). Hybrid bermudagrass cultivars Champion, MiniVerde, Tifdwarf, TifEagle, and Tifgreen were genetically similar to one another; however, GBS separated these cultivars from ‘Tifway’ hybrid bermudagrass (Fig. <span class="InternalRef"><a href="#Fig3">3</a></span>). Pooling individual cultivar samples yielded a higher average read depth of 31 per variant site per cultivar. Using the pooled data, 675,578 loci were identified as different between at least two cultivars. The majority of these genotype differences were only able to differentiate cultivars within the ‘Tifgreen’-cultivar family from those with different lineage (i.e., ‘Tifway’) (Table <span class="InternalRef"><a href="#Tab2">2</a></span>). Table <span class="InternalRef"><a href="#Tab2">2</a></span> exhibits the number of nucleotide variants with different genotypes between each pair of triploid hybrid bermudagrass cultivars. Variants were included if they were homozygous for the reference allele in one cultivar and homozygous for the alternate allele in the other cultivar or if they were homozygous in one cultivar and heterozygous in the other. ‘Tifway’ has a much larger number of variants compared to the other five cultivars (29,614 variants when compared to ‘MiniVerde’ to 37,802 when compared to ‘Tifdwarf’). The five other cultivars are more similar to each other, with the highest number of identified variants of 4476 between ‘Tifdwarf’ and ‘Tifgreen’.<figure xmlns="" class="Figure" id="Fig3"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO3">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0043-3/MediaObjects/41065_2017_43_Fig3_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0043-3/MediaObjects/41065_2017_43_Fig3_HTML.gif" alt="Fig. 3" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0043-3/MediaObjects/41065_2017_43_Fig3_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0043-3/MediaObjects/41065_2017_43_Fig3_HTML.gif" alt="Fig. 3"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 3</span>
                        <p xmlns="" class="SimplePara">
                                       <strong class="EmphasisTypeBold">a</strong> Multidimensional scaling plot (MDS) of nucleotide variants from 47 desirable and off-type bermudagrasses sampled from golf course putting greens (S1–47), six hybrid bermudagrass (<em class="EmphasisTypeItalic">Cynodon dactylon</em> (L.) Pers. x <em class="EmphasisTypeItalic">C. transvaalensis</em> Burtt-Davy) cultivars [Champion (CH1–6), MiniVerde (MV1–6), Tifdwarf (TD1–6), TifEagle (TE1–6), Tifgreen (TG1–6), and Tifway (TW1–6)], and two progenitor species [<em class="EmphasisTypeItalic">C. dactylon</em> (TA1–3, TB1–3) and <em class="EmphasisTypeItalic">C. transvaalensis</em> (DA1–3, DB1–3)]. Samples S19, S28, S30, S32, and S44 were not included due to lack of read depth. Variants were generated using Freebayes, the MDS plot was calculated in Plink, and plotted in R. The asterisk on the box indicates the zoomed region in (<strong class="EmphasisTypeBold">b</strong>). <strong class="EmphasisTypeBold">b</strong> A blown up view of the boxed in region of (<strong class="EmphasisTypeBold">a</strong>) indicated by an asterisk. Desirable and off-type bermudagrasses that were analyzed by GBS but did not cluster in this region included the following samples: S4, S16, S31, S33, and S45</p>
                      </div></figcaption></figure>
                           <figure xmlns="" id="Tab2" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 2</span>
                        <p xmlns="" class="SimplePara">The number of nucleotide variants with different genotypes between each pair of triploid hybrid bermudagrass (<em class="EmphasisTypeItalic">Cynodon dactylon</em> x <em class="EmphasisTypeItalic">C. transvaalensis</em>) cultivars</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col></col><col></col><col></col><col></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Cultivar</p></th><th colspan="5"><p xmlns="" class="SimplePara">Number of nucleotide variants</p></th></tr><tr class="header"><th>&nbsp;</th><th><p xmlns="" class="SimplePara">Champion</p></th><th><p xmlns="" class="SimplePara">MiniVerde</p></th><th><p xmlns="" class="SimplePara">Tifdwarf</p></th><th><p xmlns="" class="SimplePara">TifEagle</p></th><th><p xmlns="" class="SimplePara">Tifgreen</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">Champion</p></td><td><p xmlns="" class="SimplePara">-</p></td><td><p xmlns="" class="SimplePara">-</p></td><td><p xmlns="" class="SimplePara">-</p></td><td><p xmlns="" class="SimplePara">-</p></td><td><p xmlns="" class="SimplePara">-</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">MiniVerde</p></td><td><p xmlns="" class="SimplePara">4003</p></td><td><p xmlns="" class="SimplePara">-</p></td><td><p xmlns="" class="SimplePara">-</p></td><td><p xmlns="" class="SimplePara">-</p></td><td><p xmlns="" class="SimplePara">-</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Tifdwarf</p></td><td><p xmlns="" class="SimplePara">4086</p></td><td><p xmlns="" class="SimplePara">3404</p></td><td><p xmlns="" class="SimplePara">-</p></td><td><p xmlns="" class="SimplePara">-</p></td><td><p xmlns="" class="SimplePara">-</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">TifEagle</p></td><td><p xmlns="" class="SimplePara">4281</p></td><td><p xmlns="" class="SimplePara">3489</p></td><td><p xmlns="" class="SimplePara">4299</p></td><td><p xmlns="" class="SimplePara">-</p></td><td><p xmlns="" class="SimplePara">-</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Tifgreen</p></td><td><p xmlns="" class="SimplePara">3969</p></td><td><p xmlns="" class="SimplePara">3028</p></td><td><p xmlns="" class="SimplePara">4476</p></td><td><p xmlns="" class="SimplePara">4088</p></td><td><p xmlns="" class="SimplePara">-</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Tifway</p></td><td><p xmlns="" class="SimplePara">35,104</p></td><td><p xmlns="" class="SimplePara">29,614</p></td><td><p xmlns="" class="SimplePara">37,802</p></td><td><p xmlns="" class="SimplePara">36,796</p></td><td><p xmlns="" class="SimplePara">36,838</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">The variants were filtered to loci with a read depth of at least 40 but less than 100 per cultivar. Variants were included if they were homozygous for the reference allele in one cultivar and homozygous for the alternate allele in the other cultivar or if they were homozygous in one cultivar and heterozygous in the other</p>
                    </div></figure>
                        </div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par22" class="Para">The pooled data from variants among triploid cultivars still encompassed individual loci with both very low and very high individual read depths. Low read depth could miss heterozygotes, whereas high read depth could indicate a repetitive region instead of an individual locus. To mitigate read depth issues, a further filter was applied to identify only the most robust variants with a sequence read depth of at least 40, but no more than 100. The upper limit was set to filter reads originating from repetitive elements where detected variation is not likely to be from a single locus. Filtering using these read depths yielded 93,188 variants between at least two genotypes (Table <span class="InternalRef"><a href="#Tab2">2</a></span>).</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par23" class="Para">The MDS plot revealed clear clustering of the diploid (DA1–3 and DB1–3) and tetraploid progenitor species (TA1–3 and TB1–3) apart from the standard hybrid bermudagrass cultivars (CH1–6, MV1–6, TD1–6, TE1–6, TG1–6, and TW1–6) and the majority of samples harvested from putting greens (Fig. <span class="InternalRef"><a href="#Fig3">3a</a></span>). The two progenitor species also clustered separately from one another with the exception of DA2 due to possible contamination during DNA isolation. The clustering demonstrated that GBS was effective for distinguishing diploid, triploid, and tetraploid bermudagrasses. The ability to distinguish among bermudagrass species is likely due to the large number of unshared variants (Fig. <span class="InternalRef"><a href="#Fig2">2</a></span>).</p>
              </section><section id="Sec12" class="Section2 RenderAsSection2">
                <h3 class="Heading">Phenotypic evaluation</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par24" class="Para">The K-means cluster algorithm yielded three clusters containing 14, 26, and 12 samples, respectively. Cluster one contained nine off-type and five desirable samples, cluster two had 12 off-types and 14 desirables, and cluster three had eight off-types and four desirable samples. The cluster analysis overall expected <em xmlns="" class="EmphasisTypeItalic">R</em>
                           <sup>
                    <em xmlns="" class="EmphasisTypeItalic">2</em>
                  </sup> was 0.61 with a cubic clustering criterion of −19.36. Cluster means and standard deviations for each morphological assessment are presented in Fig. <span class="InternalRef"><a href="#Fig4">4</a></span>. Internode length, leaf length, and LWR were the only statistically different morphological parameters among clusters (Fig. <span class="InternalRef"><a href="#Fig4">4</a></span>). A representative hybrid bermudagrass sample from each cluster is illustrated in Fig. <span class="InternalRef"><a href="#Fig5">5</a></span>.<figure xmlns="" class="Figure" id="Fig4"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO4">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0043-3/MediaObjects/41065_2017_43_Fig4_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0043-3/MediaObjects/41065_2017_43_Fig4_HTML.gif" alt="Fig. 4" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0043-3/MediaObjects/41065_2017_43_Fig4_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0043-3/MediaObjects/41065_2017_43_Fig4_HTML.gif" alt="Fig. 4"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 4</span>
                        <p xmlns="" class="SimplePara">Cluster means and standard deviations for internode length, leaf length, leaf length:width ratio, stolon diameter, and leaf width measurements. Morphological parameters were assessed using methods similar to Roche and Loch [<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR37" data-toggle="citation">37</a></span>]. Measurements were made on 52 off-type and desirable hybrid bermudagrass (<em class="EmphasisTypeItalic">Cynodon dactylon</em> (L.) Pers. x <em class="EmphasisTypeItalic">C. transvaalensis</em> Burtt-Davy) samples harvested from golf course putting greens in the southeastern United States. Cluster means and standard deviations were generated from the K-means algorithm in SAS Enterprise 6.1 and graphed using Prism 6.0 for Mac. Statistical differences were determined using standard deviations</p>
                      </div></figcaption></figure>
                           <figure xmlns="" class="Figure" id="Fig5"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO5">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0043-3/MediaObjects/41065_2017_43_Fig5_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0043-3/MediaObjects/41065_2017_43_Fig5_HTML.gif" alt="Fig. 5" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0043-3/MediaObjects/41065_2017_43_Fig5_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0043-3/MediaObjects/41065_2017_43_Fig5_HTML.gif" alt="Fig. 5"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 5</span>
                        <p xmlns="" class="SimplePara">Photographs of bermudagrass samples representative of each morphological cluster. Cluster analysis was performed using a K-means algorithm in SAS Enterprise Guide (Version 6.1, SAS Institute, Cary, NC, USA) with cluster means and standard deviations graphed in Prism (Prism 6 for Mac OS X; GraphPad Software, Inc.) to determine statistical differences. Grasses in cluster one had significantly longer internode lengths than those within clusters two and three. Grasses in cluster three had significantly longer leaves than those in clusters one and two. Figure was generated using Keynote (v6.6.2)</p>
                      </div></figcaption></figure>
                        </div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par25" class="Para">The average internode length for grasses in cluster one (34.6&nbsp;mm) was significantly longer than the grasses within clusters two (21.9&nbsp;mm) and three (24.7&nbsp;mm) (Fig. <span class="InternalRef"><a href="#Fig4">4</a></span>). Grasses in cluster three had significantly longer leaves than those grouped in clusters one and two (Fig. <span class="InternalRef"><a href="#Fig4">4</a></span>). The leaf length mean for cluster three was 29.8&nbsp;mm, compared to 14.9 and 9.9&nbsp;mm for clusters one and two, respectively (Fig. <span class="InternalRef"><a href="#Fig4">4</a></span>). This relationship was also present in LWR among clusters (Fig. <span class="InternalRef"><a href="#Fig3">3</a></span>). Stolon diameter ranged from 0.7 to 0.8&nbsp;mm and leaf width ranged from 2.0 to 2.2&nbsp;mm, with no statistical differences present among clusters for either parameter.</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec13" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Discussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Discussion">Discussion</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Discussion" data-test="collapse-section"><section id="Sec14" class="Section2 RenderAsSection2">
                <h3 class="Heading">Genotyping-by-sequencing</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par26" class="Para">Caetano-Anollés [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>] and Caetano-Anollés et al. [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>] revealed that eight of 16 off-types were genetically divergent from standard cultivars using DAF, leading researchers to conclude that off-types that were not genetically distinct, but were the result of somatic mutations within ‘Tifgreen’ and ‘Tifdwarf’. The inability of GBS, as well as other molecular marker techniques, to distinguish off-types from hybrid bermudagrass cultivars used on putting greens could be the result of aneuploidy within the ‘Tifgreen’-cultivar family (B.M. Schwartz, unpublished data, 2016; [<span class="CitationRef"><a href="#CR36" data-toggle="citation">36</a></span>]). Reasor et al. [<span class="CitationRef"><a href="#CR36" data-toggle="citation">36</a></span>] suggested that aneuploidy could have resulted during the original hybridization of ‘Tifgreen’ or through intense putting green management techniques. It is expected that some variant locations are not going to be sampled by random chance due to the sparse nature of the GBS analysis. This is a limitation of GBS because it cannot determine presence/absence or copy number variations for individual locations that are needed to determine aneuploidy [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>]. There were a total of 93,188 variants shared between at least two genotypes (Table <span class="InternalRef"><a href="#Tab2">2</a></span>); however, our experiment only included six biological replications of each standard cultivar from a single geographic location (Tifton, GA). Additional research and replication of this study with more samples will be needed to ascertain which variants can be used to identify standard hybrid bermudagrass cultivars, specifically hybrid ultradwarf cultivars, from one another.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par27" class="Para">Despite also being a triploid hybrid, ‘Tifway’ bermudagrass has been genetically distinguished from ‘Tifgreen’-cultivar family using SSRs [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>, <span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>, <span class="CitationRef"><a href="#CR40" data-toggle="citation">40</a></span>] and AFLPs [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>, <span class="CitationRef"><a href="#CR41" data-toggle="citation">41</a></span>]. Arumuganathan et al. [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>] reported that ‘Tifway’ had less nuclear DNA content (1.37&nbsp;±&nbsp;0.01&nbsp;pg/2C) than ‘Tifgreen’ (1.61&nbsp;±&nbsp;0.00&nbsp;pg/2C) despite having the same number of chromosomes. Furthermore, Reasor et al. [<span class="CitationRef"><a href="#CR36" data-toggle="citation">36</a></span>] hypothesized that this difference in DNA content could also be a result of aneuploidy in the ‘Tifgreen’-cultivar family of hybrid bermudagrass, which includes the hybrid ultradwarf cultivars. This difference in DNA content could also aid in genetic identification between hybrid bermudagrasses using GBS. In addition to the identification of ‘Tifway’ from other triploid hybrids, the ability of GBS to distinguish diploid, triploid, and tetraploid bermudagrasses align with previous efforts to genetically identify these grasses from one another using AFLPs [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>, <span class="CitationRef"><a href="#CR41" data-toggle="citation">41</a></span>] and SSRs [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>, <span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>, <span class="CitationRef"><a href="#CR40" data-toggle="citation">40</a></span>].</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par28" class="Para">It is not clear why the majority of grasses included in our experiment exhibited variable morphological characteristics while being similar in genotype. The majority of bermudagrass cultivars established on golf course putting greens were selected from other bermudagrass cultivars [<span class="CitationRef"><a href="#CR36" data-toggle="citation">36</a></span>]. The off-type grasses studied in this experiment were also selected from existing cultivars. Differential gene expression driven by epigenetic mechanisms such as DNA methylation, histone modification, and small RNA expression may also be a possible explanation for the genetic similarities among hybrid bermudagrass samples varying in phenotype [<span class="CitationRef"><a href="#CR39" data-toggle="citation">39</a></span>]. Multiple genes control important turfgrass traits and gene expression can be greatly influenced by environment or management practices [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>]. Golf course putting greens are intensely managed surfaces subjected to daily mowing (often at heights of cut ≤3&nbsp;mm), annual aerification and cultivation, as well as treatment with plant growth regulators and silica sand topdressing on a weekly basis. Any of these practices or other environmental influences could cause lasting epigenetic effects that result in the up or down regulation of genes associated with hybrid bermudagrass phenotypic characteristics; however, no research has been conducted on this possibility. Studying changes in gene expression as a result of these maintenance practices could benefit researchers and industry practitioners to better understand how putting green management could potentially lead to the occurrence of phenotypically different off-type grasses in hybrid bermudagrass putting surfaces and generate new hypotheses into how these changes are induced.</p>
              </section><section id="Sec15" class="Section2 RenderAsSection2">
                <h3 class="Heading">Phenotypic evaluation</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par29" class="Para">The internode length of grasses in this experiment align with Magni et al. [<span class="CitationRef"><a href="#CR31" data-toggle="citation">31</a></span>] who reported an internode length range of 15 to 34&nbsp;mm on ultradwarf hybrid bermudagrass cultivars used on putting greens. However, Roche and Loch [<span class="CitationRef"><a href="#CR37" data-toggle="citation">37</a></span>] reported internode lengths of 9.4 to 12.5&nbsp;mm for hybrid bermudagrasses used on putting greens. In our experiment, mean internode length for each cluster was in the uppermost half of the internode length range reported by Magni et al. [<span class="CitationRef"><a href="#CR31" data-toggle="citation">31</a></span>] and greater than the range measured by Roche and Loch [<span class="CitationRef"><a href="#CR37" data-toggle="citation">37</a></span>]. Internode lengths measured in this experiment varied greatly among desirable and off-type grasses as well as grasses measured in other experiments. This is an indication of the amount of phenotypic variability that can occur in individual putting greens as well as from golf course-to-golf course and cultivar-to-cultivar. Differences in internode length within the same putting surface can lead to decreased turfgrass density and reductions in putting surface quality and playability [<span class="CitationRef"><a href="#CR36" data-toggle="citation">36</a></span>]. Similar to the internode length data, leaf length values (and subsequently LWR values) documented in our experiment were far greater than those reported by Roche and Loch [<span class="CitationRef"><a href="#CR37" data-toggle="citation">37</a></span>]. Stolon diameter and leaf width values were similar to those reported by Roche and Loch [<span class="CitationRef"><a href="#CR37" data-toggle="citation">37</a></span>], but less than those reported by Magni et al. [<span class="CitationRef"><a href="#CR31" data-toggle="citation">31</a></span>].</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec16" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Conclusions" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Conclusions">Conclusions</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Conclusions" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par30" class="Para">Off-type grasses reported to have phenotypic differences from standard hybrid bermudagrass cultivars were sampled from golf course putting greens and subjected to GBS and morphological characterization under controlled growth conditions. Genotyping-by-sequencing only distinguished five off-type grasses from standard hybrid bermudagrass cultivars. In addition, GBS failed to completely distinguish standard hybrid bermudagrass cultivars from one another, including ‘Champion’, ‘MiniVerde’, ‘Tifdwarf’, ‘TifEagle’, and ‘Tifgreen’. These results are not unexpected given their common origin. The final bioinformatics analysis did yield 93,188 variants that offer the potential to be useful in distinguishing standard cultivars from one another; however, additional research beyond the scope if this project would be needed to determine which ones are diagnostic. GBS was successful in determining triploid hybrid bermudagrass cultivars from two diploid and tetraploid progenitor samples. Additionally, GBS was also successful in determining triploid hybrid bermudagrass cultivars with lineage to ‘Tifgreen’ from those not developed from ‘Tifgreen’ (e.g., ‘Tifway’). Morphological characteristics varied among sampled grasses that allowed them to be clustered into three distinct phenotypic groups varying predominately in internode and leaf length.</p></div></section>
          <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments"><h3 xmlns="" class="Heading">Acknowledgments</h3><p xmlns="" class="SimplePara">The authors would like to thank Qi Sun, Jeffe Dunne, Sarah Boggess, Anne Hatmaker, Monil Mehta, Laura Poplawski, Sujata Agarwal, Javier Vargas, Tyler Campbell, Jimmy Greenway, Greg Breeden, Daniel Farnsworth, Trevor Mills, Mitchell Riffey, Kelly Arnholt, Jason Burris, Amanda Webb, Rebecca Grantham, Gerald Henry, and John Sorochan for assistance with this project. Mention of trade names or commercial products in this article is solely for the purpose of providing specific information and does not imply recommendation or endorsement by the University of Tennessee Institute of Agriculture or the U.S. Department of Agriculture. USDA is an equal opportunity provider and employer.</p><div id="FPar1" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Funding</h3>
                <p id="Par31" class="Para">The University of Tennessee Institute of Agriculture and the Memphis Area chapter of the Golf Course Superintendents Association of America provided funding for this project.</p>
              </div><div id="FPar2" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Availability of data and materials</h3>
                <p id="Par32" class="Para">All raw sequence data for bermudagrass samples have been submitted to NCBI under BioProject accession PRJNA353769.</p>
              </div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="FormalPara FormalParaRenderingStyle1"><h3 xmlns="" class="Heading">Authors’ contributions</h3><p xmlns="" class="SimplePara">All authors shared responsibility in designing and completing experiments, and preparing the manuscript based on their specific areas of expertise. All authors read and approved the final manuscript.</p></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Ethics">
              <div id="FPar3" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Ethics approval and consent to participate</h3>
                <p id="Par33" class="Para">Not applicable.</p>
              </div>
              <div id="FPar4" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Consent for publication</h3>
                <p id="Par34" class="Para">Not applicable.</p>
              </div>
              <div id="FPar5" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Competing interests</h3>
                <p id="Par35" class="Para">The authors declare that they have no competing interest.</p>
              </div>
              <div id="FPar6" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Publisher’s Note</h3>
                <p id="Par36" class="Para">Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
              </div>
            </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="License">
                <p xmlns="" class="SimplePara">
                           <strong class="EmphasisTypeBold">Open Access</strong>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/4.0/"><span class="RefSource">http://creativecommons.org/licenses/by/4.0/</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/publicdomain/zero/1.0/"><span class="RefSource">http://creativecommons.org/publicdomain/zero/1.0/</span></a></span>) applies to the data made available in this article, unless otherwise stated.</p>
              </div></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authors’ Affiliations</a></h2><div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1"><span class="AffiliationNumber">(1)</span><div class="AffiliationText">Department of Plant and Soil Sciences, Mississippi State University, 117 Dorman Hall, 32 Creelman Street, Mississippi State University, Mississippi State, MS&nbsp;39762, USA</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff2"><span class="AffiliationNumber">(2)</span><div class="AffiliationText">Department of Plant Sciences, University of Tennessee, 2431 Joe Johnson Dr., 252 Ellington Plant Sciences Bldg., Knoxville, TN&nbsp;37996, USA</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff3"><span class="AffiliationNumber">(3)</span><div class="AffiliationText">Department of Entomology and Plant Pathology, University of Tennessee, 2505 E.J. Chapman Dr., 370 Plant Biotechnology Bldg, Knoxville, TN&nbsp;37996, USA</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff4"><span class="AffiliationNumber">(4)</span><div class="AffiliationText">United States Department of Agriculture, Agriculture Research Service, United States Vegetable Laboratory, 2700 Savannah Highway, Charleston, SC&nbsp;29414, USA</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff5"><span class="AffiliationNumber">(5)</span><div class="AffiliationText">Department of Horticulture, University of Georgia, 2360 Rainwater Rd, Tifton, GA&nbsp;31794, USA</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff6"><span class="AffiliationNumber">(6)</span><div class="AffiliationText">Department of Crop and Soil Sciences, University of Georgia, University of Georgia, 2360 Rainwater Rd, Tifton, GA&nbsp;31794, USA</div></div></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper"><li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Arumuganathan K, Tallury SP, Fraser ML, Bruneau AH, Qu R. Nuclear DNA content of thirteen turfgrass species by flow cytometry. Crop Sci. 1999;39:1518–21.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Nuclear%20DNA%20content%20of%20thirteen%20turfgrass%20species%20by%20flow%20cytometry&amp;author=K.%20Arumuganathan&amp;author=SP.%20Tallury&amp;author=ML.%20Fraser&amp;author=AH.%20Bruneau&amp;author=R.%20Qu&amp;journal=Crop%20Sci&amp;volume=39&amp;pages=1518-1521&amp;publication_year=1999"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Beard JB. Turf management for golf courses 2nd ed. Chelsea: Wiley; 2002.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Beard%20JB.%20Turf%20management%20for%20golf%20courses%202nd%20ed.%20Chelsea%3A%20Wiley%3B%202002."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Bonos SA, Plumley KA, Meyer WA. Ploidy determination in <em class="EmphasisTypeItalic">Agrostis</em> using flow cytometry and morphological traits. Crop Sci. 2002;42:192–6.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11756273"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Ploidy%20determination%20in%20Agrostis%20using%20flow%20cytometry%20and%20morphological%20traits&amp;author=SA.%20Bonos&amp;author=KA.%20Plumley&amp;author=WA.%20Meyer&amp;journal=Crop%20Sci&amp;volume=42&amp;pages=192-196&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Bradbury PJ, Zhang Z, Kroon DE, Terry Casstevens M, Ramdoss Y, Buckler ES. TASSEL: software for association mapping of complex traits in diverse samples. Bioinformatics. 2007;23:2633–5.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btm308"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17586829"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=TASSEL%3A%20software%20for%20association%20mapping%20of%20complex%20traits%20in%20diverse%20samples&amp;author=PJ.%20Bradbury&amp;author=Z.%20Zhang&amp;author=DE.%20Kroon&amp;author=M.%20Terry%20Casstevens&amp;author=Y.%20Ramdoss&amp;author=ES.%20Buckler&amp;journal=Bioinformatics&amp;volume=23&amp;pages=2633-2635&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Brown RM, Brown MA, Brown SD (1997) ‘champion’ dwarf’ hybrid bermudagrass. US Plant patent 9888 date issued: 6 may 1997.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=%E2%80%98champion%E2%80%99%20dwarf%E2%80%99%20hybrid%20bermudagrass.%20US%20Plant%20patent%209888%20date%20issued%3A%206%20may%201997&amp;author=RM.%20Brown&amp;author=MA.%20Brown&amp;author=SD.%20Brown&amp;publication_year=1997"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Burton GW. ‘Tifgreen’ (Tifton 328) bermudagrass for golf greens. USGA Green Section Record. 1964;2:11–3.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=%E2%80%98Tifgreen%E2%80%99%20%28Tifton%20328%29%20bermudagrass%20for%20golf%20greens&amp;author=GW.%20Burton&amp;journal=USGA%20Green%20Section%20Record&amp;volume=2&amp;pages=11-13&amp;publication_year=1964"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Burton GW. ‘Tifdwarf’ bermudagrass. Crop Sci. 1966;6:94.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2135/cropsci1966.0011183X000600010036x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=%E2%80%98Tifdwarf%E2%80%99%20bermudagrass&amp;author=GW.%20Burton&amp;journal=Crop%20Sci&amp;volume=6&amp;pages=94&amp;publication_year=1966"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Burton GW, Elsner JE. ‘Tifdwarf’-a new bermudagrass for golf greens. USGA Green Section Record. 1965;2:8–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=%E2%80%98Tifdwarf%E2%80%99-a%20new%20bermudagrass%20for%20golf%20greens&amp;author=GW.%20Burton&amp;author=JE.%20Elsner&amp;journal=USGA%20Green%20Section%20Record&amp;volume=2&amp;pages=8-9&amp;publication_year=1965"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Caetano-Anollés G. Genetic instability of bermudagrass (<em class="EmphasisTypeItalic">Cynodon</em>) cultivars ‘Tifgreen’ and ‘Tifdwarf’ detected by DAF and ASAP analysis of accessions and off-types. Euphytica. 1998;101:165–73.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1023/A%3A1018392623408"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20instability%20of%20bermudagrass%20%28Cynodon%29%20cultivars%20%E2%80%98Tifgreen%E2%80%99%20and%20%E2%80%98Tifdwarf%E2%80%99%20detected%20by%20DAF%20and%20ASAP%20analysis%20of%20accessions%20and%20off-types&amp;author=G.%20Caetano-Anoll%C3%A9s&amp;journal=Euphytica&amp;volume=101&amp;pages=165-173&amp;publication_year=1998"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Caetano-Anollés G, Callahan LM, Gresshoff PM. The origin of bermudagrass (<em class="EmphasisTypeItalic">Cynodon</em>) off-types inferred by DNA amplification fingerprinting. Crop Sci. 1997;37:81–7.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2135/cropsci1997.0011183X003700010013x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20origin%20of%20bermudagrass%20%28Cynodon%29%20off-types%20inferred%20by%20DNA%20amplification%20fingerprinting&amp;author=G.%20Caetano-Anoll%C3%A9s&amp;author=LM.%20Callahan&amp;author=PM.%20Gresshoff&amp;journal=Crop%20Sci&amp;volume=37&amp;pages=81-87&amp;publication_year=1997"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Capo-chichi LJ, Goatley JM, Philley W, Krans J, Davis D, Kato A, Santen E. Dinitroaniline-induced genetic changes in bermudagrass. Crop Sci. 2005;45:1504–10.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2135/cropsci2004.0187"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Dinitroaniline-induced%20genetic%20changes%20in%20bermudagrass&amp;author=LJ.%20Capo-chichi&amp;author=JM.%20Goatley&amp;author=W.%20Philley&amp;author=J.%20Krans&amp;author=D.%20Davis&amp;author=A.%20Kato&amp;author=E.%20Santen&amp;journal=Crop%20Sci&amp;volume=45&amp;pages=1504-1510&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Chen Z, Wang ML, Waltz C, Raymer P. Genetic diversity of warm-season turfgrass: seashore paspalum, bermudagrass, and zoysiagrass revealed by AFLPs. Floriculture Ornamental Biotechnol. 2009;3:20–4.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20of%20warm-season%20turfgrass%3A%20seashore%20paspalum%2C%20bermudagrass%2C%20and%20zoysiagrass%20revealed%20by%20AFLPs&amp;author=Z.%20Chen&amp;author=ML.%20Wang&amp;author=C.%20Waltz&amp;author=P.%20Raymer&amp;journal=Floriculture%20Ornamental%20Biotechnol&amp;volume=3&amp;pages=20-24&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Elshire RJ, Glaubitz JC, Sun G, Poland JA, Kawamoto K, Buckler ES, Mitchell SE. A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species. PLoS One. 2011;6:E19379. doi:<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.1371/journal.pone.0019379"><span class="RefSource">10.1371/journal.pone.0019379</span></a></span>.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0019379"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21573248"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3087801"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20robust%2C%20simple%20genotyping-by-sequencing%20%28GBS%29%20approach%20for%20high%20diversity%20species&amp;author=RJ.%20Elshire&amp;author=JC.%20Glaubitz&amp;author=G.%20Sun&amp;author=JA.%20Poland&amp;author=K.%20Kawamoto&amp;author=ES.%20Buckler&amp;author=SE.%20Mitchell&amp;journal=PLoS%20One&amp;volume=6&amp;pages=E19379&amp;publication_year=2011&amp;doi=10.1371%2Fjournal.pone.0019379"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Fei S. Recent progresses on turfgrass molecular genetics and biotechnology. Acta Hortic. 2008;783:247–60.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Recent%20progresses%20on%20turfgrass%20molecular%20genetics%20and%20biotechnology&amp;author=S.%20Fei&amp;journal=Acta%20Hortic&amp;volume=783&amp;pages=247-260&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Fiedler JD, Lanzatella C, Okada M, Jenkins J, Schmutz J, Tobias CM. High-density single nucleotide polymorphism linkage map of lowland switchgrass using genotyping-by-sequencing. Plant Genome. 2015;8. doi:<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.3835/plantgenome2014.10.0065"><span class="RefSource">10.3835/plantgenome2014.10.0065</span></a></span>.</span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Garrison E, Marth G. Haplotype-based variant detection from short-read sequencing. Cornell University Library; 2012. <span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://arxiv.org/abs/1207.3907"><span class="RefSource">https://arxiv.org/abs/1207.3907</span></a></span>.</span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Glaubitz JC, Casstevens TM, Lu F, Harriman J, Elshire RJ, Sun Q, Buckler ES. TASSEL-GBS: a high capacity genotyping by sequencing analysis pipeline. PLoS One. 2014;9(2):e90346.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0090346"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24587335"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3938676"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=TASSEL-GBS%3A%20a%20high%20capacity%20genotyping%20by%20sequencing%20analysis%20pipeline&amp;author=JC.%20Glaubitz&amp;author=TM.%20Casstevens&amp;author=F.%20Lu&amp;author=J.%20Harriman&amp;author=RJ.%20Elshire&amp;author=Q.%20Sun&amp;author=ES.%20Buckler&amp;journal=PLoS%20One&amp;volume=9&amp;issue=2&amp;pages=e90346&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hanna WW, Elsner JE. Registration of ‘TifEagle’ bermudagrass. Crop Sci. 1999;39:1258.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2135/cropsci1999.0011183X003900040060x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Registration%20of%20%E2%80%98TifEagle%E2%80%99%20bermudagrass&amp;author=WW.%20Hanna&amp;author=JE.%20Elsner&amp;journal=Crop%20Sci&amp;volume=39&amp;pages=1258&amp;publication_year=1999"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Harris-Shultz KR, Schwartz BM, Brady JA. Identification of simple sequence repeat markers that differentiate bermudagrass cultivars derived from ‘Tifgreen’. J Amer Soc Hort Sci. 2011;136:211–8.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Identification%20of%20simple%20sequence%20repeat%20markers%20that%20differentiate%20bermudagrass%20cultivars%20derived%20from%20%E2%80%98Tifgreen%E2%80%99&amp;author=KR.%20Harris-Shultz&amp;author=BM.%20Schwartz&amp;author=JA.%20Brady&amp;journal=J%20Amer%20Soc%20Hort%20Sci&amp;volume=136&amp;pages=211-218&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Harris-Shultz KR, Schwartz BM, Hanna WW, Brady JA. Development, linkage mapping, and use of microsatellites in bermudagrass. J Amer Soc Hort Sci. 2010;135:511–20.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Development%2C%20linkage%20mapping%2C%20and%20use%20of%20microsatellites%20in%20bermudagrass&amp;author=KR.%20Harris-Shultz&amp;author=BM.%20Schwartz&amp;author=WW.%20Hanna&amp;author=JA.%20Brady&amp;journal=J%20Amer%20Soc%20Hort%20Sci&amp;volume=135&amp;pages=511-520&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hein MA. Registration of varieties and strains of bermudagrass, III. (<em class="EmphasisTypeItalic">Cynodon dactylon</em> (L.) Pers.). Agron J. 1961;53:276.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2134/agronj1961.00021962005300040021x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Registration%20of%20varieties%20and%20strains%20of%20bermudagrass%2C%20III.%20%28Cynodon%20dactylon%20%28L.%29%20Pers.%29&amp;author=MA.%20Hein&amp;journal=Agron%20J&amp;volume=53&amp;pages=276&amp;publication_year=1961"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hitchcock AS. Manual of the grasses of the United States. 2nd ed. New York: Dover Publications; 1971.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Manual%20of%20the%20grasses%20of%20the%20United%20States&amp;author=AS.%20Hitchcock&amp;publication_year=1971"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kaerwer HE, Kaerwer B (2001) Bermudagrass ‘P-18’. United States plant patent. 12084P3 date issued: 11 September.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Bermudagrass%20%E2%80%98P-18%E2%80%99.%20United%20States%20plant%20patent.%2012084P3%20date%20issued%3A%2011%20September&amp;author=HE.%20Kaerwer&amp;author=B.%20Kaerwer&amp;publication_year=2001"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR24" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kamps TL, Williams NR, Ortega VM, Chamusco KC, Harris-Shultz KR, Scully BT, Chase CD. DNA polymorphisms at bermudagrass microsatellite loci and their use in genotype fingerprinting. Crop Sci. 2011;51:1122–31.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2135/cropsci2010.08.0478"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=DNA%20polymorphisms%20at%20bermudagrass%20microsatellite%20loci%20and%20their%20use%20in%20genotype%20fingerprinting&amp;author=TL.%20Kamps&amp;author=NR.%20Williams&amp;author=VM.%20Ortega&amp;author=KC.%20Chamusco&amp;author=KR.%20Harris-Shultz&amp;author=BT.%20Scully&amp;author=CD.%20Chase&amp;journal=Crop%20Sci&amp;volume=51&amp;pages=1122-1131&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR25" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kang SY, Lee GJ, Lim KB, Lee HJ, Park IS, Chung SJ, Kim JB, Kim DS, Rhee HK. Genetic diversity among Korean bermudagrass (<em class="EmphasisTypeItalic">Cynodon</em> spp.) ecotypes characterized by morphological, cytological, and molecular approaches. Mol Cell. 2008;25:163–71.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20among%20Korean%20bermudagrass%20%28Cynodon%20spp.%29%20ecotypes%20characterized%20by%20morphological%2C%20cytological%2C%20and%20molecular%20approaches&amp;author=SY.%20Kang&amp;author=GJ.%20Lee&amp;author=KB.%20Lim&amp;author=HJ.%20Lee&amp;author=IS.%20Park&amp;author=SJ.%20Chung&amp;author=JB.%20Kim&amp;author=DS.%20Kim&amp;author=HK.%20Rhee&amp;journal=Mol%20Cell&amp;volume=25&amp;pages=163-171&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR26" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kenworthy KE, Taliaferro CM, Carver BF, Martin DL, Anderson JA. Genetic variation in <em class="EmphasisTypeItalic">Cynodon transvaalensis</em> Burtt-Davy. Crop Sci. 2006;46:2376–81.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2135/cropsci2006.02.0075"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20variation%20in%20Cynodon%20transvaalensis%20Burtt-Davy&amp;author=KE.%20Kenworthy&amp;author=CM.%20Taliaferro&amp;author=BF.%20Carver&amp;author=DL.%20Martin&amp;author=JA.%20Anderson&amp;journal=Crop%20Sci&amp;volume=46&amp;pages=2376-2381&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR27" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Langmead B, Salzberg SL. Fast gapped-read alignment with bowtie 2. Nat Methods. 2012;9:357–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nmeth.1923"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22388286"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3322381"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Fast%20gapped-read%20alignment%20with%20bowtie%202&amp;author=B.%20Langmead&amp;author=SL.%20Salzberg&amp;journal=Nat%20Methods&amp;volume=9&amp;pages=357-359&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR28" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, Homer N, Marth G, Abecasis G, Durbin R, et al. The sequence alignment/map format and SAMtools. Bioinformatics. 2009;25:2078–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btp352"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19505943"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2723002"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20sequence%20alignment%2Fmap%20format%20and%20SAMtools&amp;author=H.%20Li&amp;author=B.%20Handsaker&amp;author=A.%20Wysoker&amp;author=T.%20Fennell&amp;author=J.%20Ruan&amp;author=N.%20Homer&amp;author=G.%20Marth&amp;author=G.%20Abecasis&amp;author=R.%20Durbin&amp;journal=Bioinformatics&amp;volume=25&amp;pages=2078-2079&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR29" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lyman GT, Throssell CS, Johnson ME, Stacey GA, Brown CD. Golf course profile describes turfgrass, landscape, and environmental stewardship features. Applied Turfgrass Science. 2007; doi:<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.1094/ATS-2007-1107-01-RS"><span class="RefSource">10.1094/ATS-2007-1107-01-RS</span></a></span>.</span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR30" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">MacQueen JB. Some methods for classification and analysis of multivariate observations. Proceedings of the fifth symposium on math, statistics, and probability. Berkeley: University of California Press; 1967. p. 281–97.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Some%20methods%20for%20classification%20and%20analysis%20of%20multivariate%20observations.%20Proceedings%20of%20the%20fifth%20symposium%20on%20math%2C%20statistics%2C%20and%20probability&amp;author=JB.%20MacQueen&amp;publication_year=1967"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR31" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Magni S, Gaetani M, Caturegli L, Leto C, Tuttolomodo T, La Bella S, Virga G, Ntoulas N, Volterrani M. Phenotypic traits and establishment speed of 44 turf bermudagrass accessions. Acta Agric Scand Sect B Soil Plant Sci. 2014;64:722–33. <span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.1080/09064710.2014.955524"><span class="RefSource">http://dx.doi.org/10.1080/09064710.2014.955524</span></a></span>
                        </span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Phenotypic%20traits%20and%20establishment%20speed%20of%2044%20turf%20bermudagrass%20accessions&amp;author=S.%20Magni&amp;author=M.%20Gaetani&amp;author=L.%20Caturegli&amp;author=C.%20Leto&amp;author=T.%20Tuttolomodo&amp;author=S.%20Bella&amp;author=G.%20Virga&amp;author=N.%20Ntoulas&amp;author=M.%20Volterrani&amp;journal=Acta%20Agric%20Scand%20Sect%20B%20Soil%20Plant%20Sci&amp;volume=64&amp;pages=722-733&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR32" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Poland JA, Brown PJ, Sorrells ME, Jannink JL. Development of high-density genetic maps for barley and wheat using a novel two enzyme genotyping-by sequencing approach. PLoS One. 2012;7:E32253. doi:<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.1371/journal.pone.0032253"><span class="RefSource">10.1371/journal.pone.0032253</span></a></span>.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0032253"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22389690"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3289635"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Development%20of%20high-density%20genetic%20maps%20for%20barley%20and%20wheat%20using%20a%20novel%20two%20enzyme%20genotyping-by%20sequencing%20approach&amp;author=JA.%20Poland&amp;author=PJ.%20Brown&amp;author=ME.%20Sorrells&amp;author=JL.%20Jannink&amp;journal=PLoS%20One&amp;volume=7&amp;pages=E32253&amp;publication_year=2012&amp;doi=10.1371%2Fjournal.pone.0032253"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR33" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Poland JA, Rife TW. Genotyping-by-sequencing for plant breeding and genetics. Plant Genome. 2012;5:92–105. doi:<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.3853/plantgenome2012.05.0005"><span class="RefSource">10.3853/plantgenome2012.05.0005</span></a></span>.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.3835/plantgenome2012.05.0005"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genotyping-by-sequencing%20for%20plant%20breeding%20and%20genetics&amp;author=JA.%20Poland&amp;author=TW.%20Rife&amp;journal=Plant%20Genome&amp;volume=5&amp;pages=92-105&amp;publication_year=2012&amp;doi=10.3853%2Fplantgenome2012.05.0005"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR34" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Price HJ, Dillon SL, Hodnett G, Rooney WL, Ross L, Johnston JS. Genome evolution in the genus <em class="EmphasisTypeItalic">Sorghum</em> (Poaceae). Ann Bot. 2005;95:219–27. doi:<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.1093/aob/mci015"><span class="RefSource">10.1093/aob/mci015</span></a></span>.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/aob/mci015"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15596469"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4246720"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genome%20evolution%20in%20the%20genus%20Sorghum%20%28Poaceae%29&amp;author=HJ.%20Price&amp;author=SL.%20Dillon&amp;author=G.%20Hodnett&amp;author=WL.%20Rooney&amp;author=L.%20Ross&amp;author=JS.%20Johnston&amp;journal=Ann%20Bot&amp;volume=95&amp;pages=219-227&amp;publication_year=2005&amp;doi=10.1093%2Faob%2Fmci015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR35" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MAR, Bender D, Maller J, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Amer J Human Genet. 2007;81:559–75.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1086/519795"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=PLINK%3A%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&amp;author=S.%20Purcell&amp;author=B.%20Neale&amp;author=K.%20Todd-Brown&amp;author=L.%20Thomas&amp;author=MAR.%20Ferreira&amp;author=D.%20Bender&amp;author=J.%20Maller&amp;journal=Amer%20J%20Human%20Genet&amp;volume=81&amp;pages=559-575&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR36" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Reasor EH, Brosnan JT, Trigiano RN, Elsner JE, Henry GM, Schwartz BM. The genetic and phenotypic variability of interspecific hybrid bermudagrasses (<em class="EmphasisTypeItalic">Cynodon dactylon</em> (L.) Pers. x <em class="EmphasisTypeItalic">C. transvaalensis</em> Burtt-Davy) used on golf course putting greens. Planta. 2016;244:761–73. doi:<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.1007/s00425-016-2571-8"><span class="RefSource">10.1007/s00425-016-2571-8</span></a></span>.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00425-016-2573-8"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27448290"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5018024"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20genetic%20and%20phenotypic%20variability%20of%20interspecific%20hybrid%20bermudagrasses%20%28Cynodon%20dactylon%20%28L.%29%20Pers.%20x%20C.%20transvaalensis%20Burtt-Davy%29%20used%20on%20golf%20course%20putting%20greens&amp;author=EH.%20Reasor&amp;author=JT.%20Brosnan&amp;author=RN.%20Trigiano&amp;author=JE.%20Elsner&amp;author=GM.%20Henry&amp;author=BM.%20Schwartz&amp;journal=Planta&amp;volume=244&amp;pages=761-773&amp;publication_year=2016&amp;doi=10.1007%2Fs00425-016-2571-8"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR37" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Roche MB, Loch DS. Morphological and development comparisons of seven greens quality hybrid bermudagrass [<em class="EmphasisTypeItalic">Cynodon dactylon</em> (L.) Pers. X <em class="EmphasisTypeItalic">C. transvaalensis</em> Burtt-Davy] cultivars. Intl Turfgrass Soc Res J. 2005;10:627–34.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Morphological%20and%20development%20comparisons%20of%20seven%20greens%20quality%20hybrid%20bermudagrass%20%5BCynodon%20dactylon%20%28L.%29%20Pers.%20X%20C.%20transvaalensis%20Burtt-Davy%5D%20cultivars&amp;author=MB.%20Roche&amp;author=DS.%20Loch&amp;journal=Intl%20Turfgrass%20Soc%20Res%20J&amp;volume=10&amp;pages=627-634&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR38" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Romani M, Piano E, Carroni AM, Pecetti L. Evaluation of native bermudagrass (<em class="EmphasisTypeItalic">Cynodon dactylon</em>) germplasm from Italy for the selection of adapted turfgrass cultivars. Acta Hort. 2004;661:381–6.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.17660/ActaHortic.2004.661.52"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Evaluation%20of%20native%20bermudagrass%20%28Cynodon%20dactylon%29%20germplasm%20from%20Italy%20for%20the%20selection%20of%20adapted%20turfgrass%20cultivars&amp;author=M.%20Romani&amp;author=E.%20Piano&amp;author=AM.%20Carroni&amp;author=L.%20Pecetti&amp;journal=Acta%20Hort&amp;volume=661&amp;pages=381-386&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR39" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sahu PP, Pandey G, Sharma N, Puranik S, Muthamilarason M, Prasad M. Epigentic mechanisms of plant stress responses and adaptation. Plant Cell Rep. 2013;32:1151–9. doi:<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.1007/s00299-013-1462-x"><span class="RefSource">10.1007/s00299-013-1462-x</span></a></span>.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00299-013-1462-x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23719757"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Epigentic%20mechanisms%20of%20plant%20stress%20responses%20and%20adaptation&amp;author=PP.%20Sahu&amp;author=G.%20Pandey&amp;author=N.%20Sharma&amp;author=S.%20Puranik&amp;author=M.%20Muthamilarason&amp;author=M.%20Prasad&amp;journal=Plant%20Cell%20Rep&amp;volume=32&amp;pages=1151-1159&amp;publication_year=2013&amp;doi=10.1007%2Fs00299-013-1462-x"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR40" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wang Z, Wu Y, Martin DL, Gao H, Samuels T, Tan C. Identification of vegetatively propagated turf bermudagrass cultivars using simple sequence repeat markers. Crop Sci. 2010;50:2103–11.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2135/cropsci2010.02.0116"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Identification%20of%20vegetatively%20propagated%20turf%20bermudagrass%20cultivars%20using%20simple%20sequence%20repeat%20markers&amp;author=Z.%20Wang&amp;author=Y.%20Wu&amp;author=DL.%20Martin&amp;author=H.%20Gao&amp;author=T.%20Samuels&amp;author=C.%20Tan&amp;journal=Crop%20Sci&amp;volume=50&amp;pages=2103-2111&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR41" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhang LH, Ozias-Akins P, Kochert G, Kresovich S, Dean R, Hanna WW. Differentiation of bermudagrass (<em class="EmphasisTypeItalic">Cynodon</em> spp.) genotypes by AFLP analyses. Theor Appl Genet. 1999;98:895–902.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s001220051148"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Differentiation%20of%20bermudagrass%20%28Cynodon%20spp.%29%20genotypes%20by%20AFLP%20analyses&amp;author=LH.%20Zhang&amp;author=P.%20Ozias-Akins&amp;author=G.%20Kochert&amp;author=S.%20Kresovich&amp;author=R.%20Dean&amp;author=WW.%20Hanna&amp;journal=Theor%20Appl%20Genet&amp;volume=98&amp;pages=895-902&amp;publication_year=1999"><span><span>Google Scholar</span></span></a></span></span></cite></li></ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#TheAuthors" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="TheAuthors">Copyright</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="TheAuthors"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">©&nbsp;The Author(s)&nbsp;2017</div></div></section></div>
                        


                    </article>
                </main>

                <div class="c-content-layout__side u-text-sm">
                    <aside>
                        <div class="p-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="//hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-017-0043-3" data-track="click" data-track-category="PDF Download"
            data-track-action="Click download article PDF"
            data-track-label="10.1186/s41065-017-0043-3">Download PDF</a>
        </p>
    
                            
                            
    <button type="button"
       class="c-button c-button--secondary c-button--full-width c-button--expandable"
       data-toggle="collapse"
       data-target="export-citations"
       data-test="export-citations-button"
       data-sticky-update="true"
       aria-controls="export-citations"
       aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/s41065-017-0043-3"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0043-3?format=refman&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/s41065-017-0043-3"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0043-3?format=refman&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/s41065-017-0043-3"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0043-3?format=refman&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/s41065-017-0043-3"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0043-3?format=endnote&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/s41065-017-0043-3"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0043-3?format=endnote&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/s41065-017-0043-3"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0043-3?format=endnote&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/s41065-017-0043-3"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0043-3?format=bibtex&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/s41065-017-0043-3"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0043-3?format=bibtex&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/s41065-017-0043-3"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0043-3?format=bibtex&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
    </div>

                        </div>
                        
    
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
                                <div id="article-access-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/access-count/10.1186/s41065-017-0043-3 #accessCount-container"></div>
                                <div id="citation-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/citation-count/10.1186/s41065-017-0043-3 #count-container"></div>
                                <div id="altmetric-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/altmetric/10.1186/s41065-017-0043-3 #altmetric-container"></div>
                            </ul>
                        </div>
                        
    <div id="sociaMediaShare" class="p-separator u-js-shown">
        <h3 class="p-section-title">Share this article</h3>
        <ul id="shareButtons" class="c-social-links">
            <li class="c-social-links__item">
                <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-017-0043-3">
                    <span class="u-visually-hidden">Share on Twitter</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-twitter"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-017-0043-3">
                    <span class="u-visually-hidden">Share on Facebook</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-facebook"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click"
                        data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-017-0043-3">
                    <span class="u-visually-hidden">Share on LinkedIn</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-linkedin"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-017-0043-3">
                    <span class="u-visually-hidden">Share on Weibo</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-weibo"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnGooglePlus" class="js-btnShareOnGplus" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Google+" data-track-label="10.1186/s41065-017-0043-3">
                    <span class="u-visually-hidden">Share on Google Plus</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-gplus"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-017-0043-3">
                    <span class="u-visually-hidden">Share on Reddit</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-reddit"></use>
                    </svg>
                </button>
            </li>
        </ul>
    </div>
 
                        
    
        <div class="p-separator u-js-shown">
            <h3 class="p-section-title">See updates</h3>
            <button class="c-crossmark-button" data-target="crossmark" data-test="crossmark" data-track="click" data-track-category="Article Page" data-track-action="Click CrossMark" data-track-label="10.1186/s41065-017-0043-3">
                <img id="crossmark-icon" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" alt="Crossmark logo" />
            </button>
        </div>
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
                                <li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/s41065-017-0043-3" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=738&doi=10.1186%2Fs41065-017-0043-3&volume=155&issue=1&title=Genotypic+and+phenotypic+evaluation+of+off-type+grasses+in+hybrid+Bermudagrass+%5B+Cynodon+dactylon+%28L.%29+Pers.+x+C.+transvaalensis+Burtt-Davy%5D+putting+greens+using+genotyping-by-sequencing+and+morphological+characterization&author_name=Eric+H.+Reasor&start_page=1&end_page=12">Order reprint</a>

                                </li>
                            </ul>
                        </div>
                    </aside>
                    
    <div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">Hereditas</span>
                        </h4>
                        <p class="c-journal-footer__issn">ISSN: 1601-5223</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
                                
                                    <li class="c-journal-footer__contact-item">Submission enquiries: <a href="https://www.editorialmanager.com/here/" target="_blank">Access here and click Contact Us</a></li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a></li>
                                
                            </ul>
                        </div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class='tracker' style='display:none' src='/track/article/10.1186/s41065-017-0043-3' alt=""/>

         
    <footer>
        <div class="ctx-publisher-footer">
    <div class="c-navbar">
        <div class="u-container">
            <div class="c-navbar__logo u-margin-right-xl">
                <a href="https://www.biomedcentral.com">
                    <img src=/static/images/bmc/logos/logo-bmc-aj-594e03e20d.svg alt="BMC logo" data-test="navbar-logo-footer">

                </a>
            </div>
            <div class="c-navbar__content">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


            </div>
        </div>
    </div>
    <div class="c-publisher-footer" data-test="publisher-footer">
        <div class="u-container">
            
            <div class="c-flex c-flex--wrap c-flex--space-between" role="navigation" data-test="publisher-footer-menu">
                <div class="c-flex" aria-label="Publisher footer links">
                    
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/login">Receive BMC newsletters</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/account">Manage article alerts</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/policies">Policies</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/accessibility">Accessibility</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/press-centre">Press center</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/contact-us">Contact us</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/jobs">Careers</a>
                                    </li>
                                
                            </ul>
                        
                    
                </div>
                <div class="u-margin-bottom-lg">
                    <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                    <ul class="c-flex u-reset-list" data-test="footer-social-links">
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://twitter.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Twitter link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Twitter page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-twitter-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://www.facebook.com/BioMedCentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Facebook link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Facebook page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-facebook-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://plus.google.com/106226337984152901734/"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Google Plus link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Google Plus page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-gplus-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="http://www.weibo.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Weibo link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Weibo page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-weibo-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </div>
            <p class="u-reset-margin">
                By using this website, you agree to our
                <a class="u-text-inherit" href="//www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
                <a class="u-text-inherit" href="//www.biomedcentral.com/privacy-statement">Privacy
                    statement</a> and
                <a class="u-text-inherit" href="//www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
                
                    <a class="optanon-toggle-display u-text-inherit" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
                
            </p>
        </div>
    </div>
</div>


        <div class="c-sn-footer c-sn-footer--neutral">
    <div class="u-container">
        <img src=/static/images/bmc/logos/logo-springernature-8bd4e8d82f.svg class="c-sn-footer__logo" alt="Springer Nature Logo" itemprop="logo" >
        <p data-test="copyright">&#169; 2018 BioMed Central Ltd unless otherwise stated. Part of
            <a href="http://www.springernature.com"
               data-track="click"
               data-track-action="Click Springer Nature link"
               data-track-category="Corporate footer"
               itemscope itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>

    </div>

    
    <script type="text/javascript" src="//cdn.pbgrd.com/bmc.js" async="async"></script>
    <noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>
    <!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker -->

<script id="springerlink-event-tracker" src="https://event-tracker.springernature.com/dist/eventTracker.js"></script>
<script type="text/javascript">
    (function(){
        var dataLayer = window.dataLayer[0];
        var doi = dataLayer.content.article.doi;
        var imprint = dataLayer.content.contentInfo.imprint;
        var bpids = dataLayer.session.authentication.authenticationID;
        var publisher;
        var pageType = dataLayer.page.category.pageType;
        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }
        var eventObject = {'content_type' : pageType,'doi' : doi};

        if (imprint === 'BioMed Central') { publisher = 'BMC' }
        else if (imprint === 'SpringerOpen') { publisher = 'SpringerOpen' }
        else { throw new Error('Event Tracker Error: The publisher has an unexpected value') }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }
        new EventTracker({'platform':publisher}).sendEvent('display', eventObject);
    })();
</script>


    </body>
</html>

