<!DOCTYPE html>
<html lang="en" class="no-js">
    <head>
        <title>Genetic structure, divergence and admixture of Han Chinese, Japanese and Korean populations | Hereditas | Full Text</title>

        

    <meta name="citation_abstract" content="Han Chinese, Japanese and Korean, the three major ethnic groups of East Asia, share many similarities in appearance, language and culture etc., but their genetic relationships, divergence times and subsequent genetic exchanges have not been well studied. We conducted a genome-wide study and evaluated the population structure of 182 Han Chinese, 90 Japanese and 100 Korean individuals, together with the data of 630 individuals representing 8 populations wordwide. Our analyses revealed that Han Chinese, Japanese and Korean populations have distinct genetic makeup and can be well distinguished based on either the genome wide data or a panel of ancestry informative markers (AIMs). Their genetic structure corresponds well to their geographical distributions, indicating geographical isolation played a critical role in driving population differentiation in East Asia. The most recent common ancestor of the three populations was dated back to 3000 ~ 3600 years ago. Our analyses also revealed substantial admixture within the three populations which occurred subsequent to initial splits, and distinct gene introgression from surrounding populations, of which northern ancestral component is dominant. These estimations and findings facilitate to understanding population history and mechanism of human genetic diversity in East Asia, and have implications for both evolutionary and medical studies."/>

    <meta name="description" content="Han Chinese, Japanese and Korean, the three major ethnic groups of East Asia, share many similarities in appearance, language and culture etc., but their genetic relationships, divergence times and subsequent genetic exchanges have not been well studied. We conducted a genome-wide study and evaluated the population structure of 182 Han Chinese, 90 Japanese and 100 Korean individuals, together with the data of 630 individuals representing 8 populations wordwide. Our analyses revealed that Han Chinese, Japanese and Korean populations have distinct genetic makeup and can be well distinguished based on either the genome wide data or a panel of ancestry informative markers (AIMs). Their genetic structure corresponds well to their geographical distributions, indicating geographical isolation played a critical role in driving population differentiation in East Asia. The most recent common ancestor of the three populations was dated back to 3000 ~ 3600 years ago. Our analyses also revealed substantial admixture within the three populations which occurred subsequent to initial splits, and distinct gene introgression from surrounding populations, of which northern ancestral component is dominant. These estimations and findings facilitate to understanding population history and mechanism of human genetic diversity in East Asia, and have implications for both evolutionary and medical studies."/>

    <meta name="journal_id" content="41065"/>

    <meta name="dc.title" content="Genetic structure, divergence and admixture of Han Chinese, Japanese and Korean populations"/>

    <meta name="dc.source" content="Hereditas 2018 155:1"/>

    <meta name="dc.format" content="text/html"/>

    <meta name="dc.publisher" content="BioMed Central"/>

    <meta name="dc.date" content="2018-04-06"/>

    <meta name="dc.type" content="OriginalPaper"/>

    <meta name="dc.language" content="En"/>

    <meta name="dc.copyright" content="2018 The Author(s)"/>

    <meta name="dc.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="dc.description" content="Han Chinese, Japanese and Korean, the three major ethnic groups of East Asia, share many similarities in&#160;appearance, language and culture&#160;etc., but their genetic relationships, divergence times and subsequent genetic exchanges have not been well studied. We conducted a genome-wide study and evaluated the population structure of 182 Han Chinese, 90 Japanese and 100 Korean individuals, together with the data of 630 individuals representing 8 populations&#160;wordwide. Our analyses revealed that Han Chinese, Japanese and Korean populations have distinct genetic makeup and can be well distinguished based on either the genome wide data or a panel of ancestry informative markers (AIMs). Their genetic structure corresponds well to their geographical distributions, indicating geographical isolation played a critical role in driving population differentiation&#160;in East Asia. The most recent common ancestor of the three populations was dated back to 3000 ~&#8201;3600&#160;years ago. Our analyses also revealed substantial admixture within the three populations which occurred subsequent to initial splits, and distinct gene introgression from surrounding populations, of which northern ancestral component is dominant. These estimations and findings facilitate to understanding population history and mechanism of human genetic diversity in East Asia, and have implications for both evolutionary and medical studies."/>

    <meta name="prism.issn" content="1601-5223"/>

    <meta name="prism.publicationName" content="Hereditas"/>

    <meta name="prism.publicationDate" content="2018-04-06"/>

    <meta name="prism.volume" content="155"/>

    <meta name="prism.number" content="1"/>

    <meta name="prism.section" content="OriginalPaper"/>

    <meta name="prism.startingPage" content="19"/>

    <meta name="prism.copyright" content="2018 The Author(s)"/>

    <meta name="prism.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="prism.url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0057-5"/>

    <meta name="prism.doi" content="doi:10.1186/s41065-018-0057-5"/>

    <meta name="citation_pdf_url" content="https://hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-018-0057-5"/>

    <meta name="citation_fulltext_html_url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0057-5"/>

    <meta name="citation_journal_title" content="Hereditas"/>

    <meta name="citation_publisher" content="BioMed Central"/>

    <meta name="citation_issn" content="1601-5223"/>

    <meta name="citation_title" content="Genetic structure, divergence and admixture of Han Chinese, Japanese and Korean populations"/>

    <meta name="citation_volume" content="155"/>

    <meta name="citation_issue" content="1"/>

    <meta name="citation_publication_date" content="2018/12"/>

    <meta name="citation_online_date" content="2018/04/06"/>

    <meta name="citation_firstpage" content="19"/>

    <meta name="citation_article_type" content="Research"/>

    <meta name="citation_fulltext_world_readable" content=""/>

    <meta name="dc.identifier" content="doi:10.1186/s41065-018-0057-5"/>

    <meta name="DOI" content="10.1186/s41065-018-0057-5"/>

    <meta name="citation_doi" content="10.1186/s41065-018-0057-5"/>

    <meta name="dc.creator" content="Yuchen Wang"/>

    <meta name="dc.creator" content="Dongsheng Lu"/>

    <meta name="dc.creator" content="Yeun-Jun Chung"/>

    <meta name="dc.creator" content="Shuhua Xu"/>

    <meta name="dc.subject" content="Animal Genetics and Genomics"/>

    <meta name="dc.subject" content="Evolutionary Biology"/>

    <meta name="citation_reference" content="Fischer SR (2004). A History of Writing."/>

    <meta name="citation_reference" content="citation_journal_title=Eur J Hum Genet; citation_title=Identification of regions of positive selection using shared genomic segment analysis; citation_author=Z Cai, NJ Camp, L Cannon-Albright, A Thomas; citation_volume=19; citation_issue=6; citation_publication_date=2011; citation_pages=667-671; citation_doi=10.1038/ejhg.2010.257; citation_id=CR2"/>

    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=The international HapMap project; citation_author=; citation_volume=426; citation_issue=6968; citation_publication_date=2003; citation_pages=789-796; citation_doi=10.1038/nature02168; citation_id=CR3"/>

    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=A map of human genome variation from population-scale sequencing; citation_author=C Genomes Project, GR Abecasis, D Altshuler, A Auton, LD Brooks, RM Durbin; citation_volume=467; citation_issue=7319; citation_publication_date=2010; citation_pages=1061-1073; citation_doi=10.1038/nature09534; citation_id=CR4"/>

    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Mapping human genetic diversity in Asia; citation_author=C HP-AS, MA Abdulla, I Ahmed, A Assawamakin, J Bhak, SK Brahmachari; citation_volume=326; citation_issue=5959; citation_publication_date=2009; citation_pages=1541-1545; citation_doi=10.1126/science.1177074; citation_id=CR5"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS Genet; citation_title=Genomic analysis of natural selection and phenotypic variation in high-altitude mongolians; citation_author=J Xing, T Wuren, TS Simonson, WS Watkins, DJ Witherspoon, W Wu; citation_volume=9; citation_issue=7; citation_publication_date=2013; citation_pages=e1003634; citation_doi=10.1371/journal.pgen.1003634; citation_id=CR6"/>

    <meta name="citation_reference" content="citation_journal_title=Am J Hum Genet; citation_title=Genomic dissection of population substructure of Han Chinese and its implication in association studies; citation_author=SH Xu, XY Yin, SL Li, WF Jin, HY Lou, L Yang; citation_volume=85; citation_issue=6; citation_publication_date=2009; citation_pages=762-774; citation_doi=10.1016/j.ajhg.2009.10.015; citation_id=CR7"/>

    <meta name="citation_reference" content="citation_title=A panel of ancestry informative markers to estimate and correct potential effects of population stratification in Han Chinese; citation_publication_date=2013; citation_id=CR8; citation_author=P Qin; citation_author=Z Li; citation_author=W Jin; citation_author=D Lu; citation_author=H Lou; citation_author=J Shen; citation_publisher=European journal of human genetics"/>

    <meta name="citation_reference" content="citation_journal_title=Genome Res; citation_title=Human population dispersal &quot;out of Africa&quot; estimated from linkage disequilibrium and allele frequencies of SNPs; citation_author=BP McEvoy, JE Powell, ME Goddard, PM Visscher; citation_volume=21; citation_issue=6; citation_publication_date=2011; citation_pages=821-829; citation_doi=10.1101/gr.119636.110; citation_id=CR9"/>

    <meta name="citation_reference" content="citation_journal_title=Genetics; citation_title=Inference of population structure using multilocus genotype data; citation_author=JK Pritchard, M Stephens, P Donnelly; citation_volume=155; citation_issue=2; citation_publication_date=2000; citation_pages=945-959; citation_id=CR10"/>

    <meta name="citation_reference" content="citation_journal_title=Genome Res; citation_title=Fast model-based estimation of ancestry in unrelated individuals; citation_author=DH Alexander, J Novembre, K Lange; citation_volume=19; citation_issue=9; citation_publication_date=2009; citation_pages=1655-1664; citation_doi=10.1101/gr.094052.109; citation_id=CR11"/>

    <meta name="citation_reference" content="citation_journal_title=Genetics; citation_title=Ancient admixture in human history; citation_author=N Patterson, P Moorjani, Y Luo, S Mallick, N Rohland, Y Zhan; citation_volume=192; citation_issue=3; citation_publication_date=2012; citation_pages=1065-1093; citation_doi=10.1534/genetics.112.145037; citation_id=CR12"/>

    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Reconstructing Indian population history; citation_author=D Reich, K Thangaraj, N Patterson, AL Price, L Singh; citation_volume=461; citation_issue=7263; citation_publication_date=2009; citation_pages=489-494; citation_doi=10.1038/nature08365; citation_id=CR13"/>

    <meta name="citation_reference" content="citation_journal_title=Springer Semin Immunopathol; citation_title=Emerging roles and new functions of CD46; citation_author=MK Liszewski, C Kemper, JD Price, JP Atkinson; citation_volume=27; citation_issue=3; citation_publication_date=2005; citation_pages=345-358; citation_doi=10.1007/s00281-005-0002-3; citation_id=CR14"/>

    <meta name="citation_reference" content="citation_journal_title=Trends Immunol; citation_title=CD46: expanding beyond complement regulation; citation_author=RC Riley-Vargas, DB Gill, C Kemper, MK Liszewski, JP Atkinson; citation_volume=25; citation_issue=9; citation_publication_date=2004; citation_pages=496-503; citation_doi=10.1016/j.it.2004.07.004; citation_id=CR15"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Genet; citation_title=Integrated genotype calling and association analysis of SNPs, common copy number polymorphisms and rare CNVs; citation_author=JM Korn, FG Kuruvilla, SA McCarroll, A Wysoker, J Nemesh, S Cawley; citation_volume=40; citation_issue=10; citation_publication_date=2008; citation_pages=1253-1260; citation_doi=10.1038/ng.237; citation_id=CR16"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS Genet; citation_title=Population structure and eigenanalysis; citation_author=N Patterson, AL Price, D Reich; citation_volume=2; citation_issue=12; citation_publication_date=2006; citation_pages=e190; citation_doi=10.1371/journal.pgen.0020190; citation_id=CR17"/>

    <meta name="citation_reference" content="citation_journal_title=Annu Rev Genet; citation_title=Estimating F-statistics; citation_author=BS Weir, WG Hill; citation_volume=36; citation_publication_date=2002; citation_pages=721-750; citation_doi=10.1146/annurev.genet.36.050802.093940; citation_id=CR18"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=The neighbor-joining method: a new method for reconstructing phylogenetic trees; citation_author=N Saitou, M Nei; citation_volume=4; citation_issue=4; citation_publication_date=1987; citation_pages=406-425; citation_id=CR19"/>

    <meta name="citation_reference" content="citation_journal_title=Brief Bioinform; citation_title=MEGA3: integrated software for molecular evolutionary genetics analysis and sequence alignment; citation_author=S Kumar, K Tamura, M Nei; citation_volume=5; citation_issue=2; citation_publication_date=2004; citation_pages=150-163; citation_doi=10.1093/bib/5.2.150; citation_id=CR20"/>

    <meta name="citation_reference" content="citation_journal_title=Am J Hum Genet; citation_title=Maximum-likelihood estimation of evolutionary trees from continuous characters; citation_author=J Felsenstein; citation_volume=25; citation_issue=5; citation_publication_date=1973; citation_pages=471-492; citation_id=CR21"/>

    <meta name="citation_reference" content="citation_journal_title=Cladistics; citation_title=PHYLIP--phylogeny inference package (version 3.2); citation_author=J Felsenstein; citation_volume=5; citation_publication_date=1989; citation_pages=164-166; citation_id=CR22"/>

    <meta name="citation_reference" content="citation_journal_title=Genetics; citation_title=Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies; citation_author=D Falush, M Stephens, JK Pritchard; citation_volume=164; citation_issue=4; citation_publication_date=2003; citation_pages=1567-1587; citation_id=CR23"/>

    <meta name="citation_reference" content="citation_journal_title=Am J Hum Genet; citation_title=Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering; citation_author=SR Browning, BL Browning; citation_volume=81; citation_issue=5; citation_publication_date=2007; citation_pages=1084-1097; citation_doi=10.1086/521987; citation_id=CR24"/>

    <meta name="citation_reference" content="citation_journal_title=Stat Comput; citation_title=Delete-m jackknife for unequal m; citation_author=F Busing, E Meijer, R Van Der Leeden; citation_volume=9; citation_issue=1; citation_publication_date=1999; citation_pages=3-8; citation_doi=10.1023/A:1008800423698; citation_id=CR25"/>

    <meta name="citation_reference" content="Lipson M, Loh PR, Levin A, Reich D, Patterson N, Berger B. Efficient moment-based inference of admixture parameters and sources of gene flow. Mol Biol Evol. 2013;"/>

    <meta name="citation_author" content="Yuchen Wang"/>

    <meta name="citation_author_institution" content="Chinese Academy of Sciences (CAS) Key Laboratory of Computational Biology, Max Planck Independent Research Group on Population Genomics, CAS-MPG Partner Institute for Computational Biology (PICB), Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, Shanghai, China"/>

    <meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing, China"/>

    <meta name="citation_author" content="Dongsheng Lu"/>

    <meta name="citation_author_institution" content="Chinese Academy of Sciences (CAS) Key Laboratory of Computational Biology, Max Planck Independent Research Group on Population Genomics, CAS-MPG Partner Institute for Computational Biology (PICB), Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, Shanghai, China"/>

    <meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing, China"/>

    <meta name="citation_author" content="Yeun-Jun Chung"/>

    <meta name="citation_author_institution" content="Integrated Research Center for Genome Polymorphism, Department of Microbiology, The Catholic University Medical College, Seoul, South Korea"/>

    <meta name="citation_author" content="Shuhua Xu"/>

    <meta name="citation_author_institution" content="Chinese Academy of Sciences (CAS) Key Laboratory of Computational Biology, Max Planck Independent Research Group on Population Genomics, CAS-MPG Partner Institute for Computational Biology (PICB), Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, Shanghai, China"/>

    <meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing, China"/>

    <meta name="citation_author_institution" content="School of Life Science and Technology ShanghaiTech University, Shanghai, China"/>

    <meta name="citation_author_institution" content="Center for Excellence in Animal Evolution and Genetics, Chinese Academy of Sciences, Kunming, China"/>

    <meta name="citation_author_institution" content="Collaborative Innovation Center of Genetics and Development, Shanghai, China"/>



        <meta charset="UTF-8"/>
        <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
        <meta name="viewport" content="width=device-width, initial-scale=1"/>

        <!-- For IE 9 and below -->
<!--[if IE]><link rel="shortcut icon" href=/static/images/favicons//favicon.ico><![endif]-->
<!-- Touch Icons - iOS and Android 2.1+ -->
<link rel="apple-touch-icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>
<!-- Firefox, Chrome, Safari, IE 11+ and Opera -->
<link rel="icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>


        <script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script>
        <script>(function(d){if (sessionStorage.fontsLoaded){d.style.setProperty('--font-family-sans', 'Europa');d.style.setProperty('--font-family-serif', 'Noto Serif')}})(document.documentElement);</script>

        <link rel="stylesheet" media="screen" href=/static/styles/basic/basic-a31ca85bc2.css>
<link rel="stylesheet" media="print" href=/static/styles/bmc/print-3a6240f456.css>

<link rel="stylesheet" class="js-ctm" href=/static/styles/bmc/main-84414d3d47.css
      media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">

        

        <script src="/static/js/app-bundle-8d61e6ef37.js" defer></script>
        
        <script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script>
        
    <script type="text/javascript">
        config = {
            env: 'live',
            site: 'hereditasjournal.biomedcentral.com',
            siteWithPath: 'hereditasjournal.biomedcentral.com' + window.location.pathname,
            twitterHashtag: '',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/s41065-018-0057-5',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central'
        };
    </script>

        
    <script type="text/javascript">
        dataLayer = [{"content":{"article":{"doi":"10.1186/s41065-018-0057-5","articleType":"Research","peerReviewType":"Closed","supplement":null,"keywords":"Han Chinese;Japanese;Korean;Genetic ancestry;Population structure;Population divergence;Admixture;SNP"},"contentInfo":{"imprint":"BioMed Central","title":"Genetic structure, divergence and admixture of Han Chinese, Japanese and Korean populations","publishedAt":1522972800000,"publishedAtDate":"2018-04-06","author":["Yuchen Wang","Dongsheng Lu","Yeun-Jun Chung","Shuhua Xu"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":"CC BY + CC0","openAccess":true},"environment":"live"},"journal":{"siteKey":"hereditasjournal.biomedcentral.com","volume":"155","issue":"1","title":"Hereditas","type":"AJs Life","journalID":41065,"gaCode":"UA-62066850-11","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Research","publishingSegment":"Genetics & Heredity (Life)"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}}}]

    </script>

        <script src="https://cdn.cookielaw.org/consent/7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script>
        
    
        
    

        
    
        
            
                <!-- Google Tag Manager -->
                <script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script>
                <!-- End Google Tag Manager -->
            
        
    


        
    
    <link rel="canonical" href="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0057-5"/>
    

        
        
        <meta property="og:url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0057-5"/>
        <meta property="og:type" content="article"/>
        <meta property="og:site_name" content="Hereditas"/>
        <meta property="og:title" content="Genetic structure, divergence and admixture of Han Chinese, Japanese and Korean populations"/>
        <meta property="og:description" content="Han Chinese, Japanese and Korean, the three major ethnic groups of East Asia, share many similarities in&amp;nbsp;appearance, language and culture&amp;nbsp;etc., but their genetic relationships, divergence times and subsequent genetic exchanges have not been well studied. We conducted a genome-wide study and evaluated the population structure of 182 Han Chinese, 90 Japanese and 100 Korean individuals, together with the data of 630 individuals representing 8 populations&amp;nbsp;wordwide. Our analyses revealed that Han Chinese, Japanese and Korean populations have distinct genetic makeup and can be well distinguished based on either the genome wide data or a panel of ancestry informative markers (AIMs). Their genetic structure corresponds well to their geographical distributions, indicating geographical isolation played a critical role in driving population differentiation&amp;nbsp;in East Asia. The most recent common ancestor of the three populations was dated back to 3000 ~ 3600&amp;nbsp;years ago. Our analyses also revealed substantial admixture within the three populations which occurred subsequent to initial splits, and distinct gene introgression from surrounding populations, of which northern ancestral component is dominant. These estimations and findings facilitate to understanding population history and mechanism of human genetic diversity in East Asia, and have implications for both evolutionary and medical studies."/>
        <meta property="og:image" content="https://images.springer.com/sgw/journals/medium/41065.jpg"/>
    
    </head>

    <body class="journal journal-fulltext"
          
          >
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK"
                            height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><feColorMatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"/></filter></defs><symbol id="icon-arrow-down" viewBox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"/></symbol><symbol id="icon-arrow-left-bullet" viewBox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"/></symbol><symbol id="icon-arrow-right-bullet" viewBox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"/></symbol><symbol id="icon-arrow-right" viewBox="0 0 32 64"><path d="M0 0v64l32-32z"/></symbol><symbol id="icon-close" viewBox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"/></symbol><symbol id="icon-community" viewBox="0 0 60.643 31.01"><mask maskUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><path fill="#FFF" d="M0 31.01h60.643V0H0z" filter="url(#a)"/></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"/></symbol><symbol id="icon-download" viewBox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"/></symbol><symbol id="icon-ext-link" viewBox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"/><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"/><path fill="#676767" d="M9.7 0L16 6.2V0z"/></symbol><symbol id="icon-info-bordered" viewBox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"/><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"/></symbol><symbol id="icon-opr" viewBox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"/><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"/></symbol><symbol id="icon-quotation" viewBox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"/></symbol><symbol id="icon-remove" viewBox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"/><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"/></symbol><symbol id="icon-ribbon" viewBox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"/></symbol><symbol id="icon-rss" viewBox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"/><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"/><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"/></g></symbol><symbol id="icon-search" viewBox="0 0 16 16"><path d="M6.276 12.257C2.815 12.257 0 9.508 0 6.128 0 2.749 2.815 0 6.276 0c3.462 0 6.277 2.749 6.277 6.128.002 3.38-2.815 6.129-6.277 6.129zm0-10.772c-2.622 0-4.755 2.083-4.755 4.643 0 2.562 2.133 4.644 4.755 4.644 2.623 0 4.756-2.082 4.756-4.644 0-2.56-2.134-4.643-4.756-4.643z"/><path d="M16 13.751L13.641 16l-4.875-4.82 2.36-2.248L16 13.751z"/></symbol><symbol id="icon-speech-bubble" viewBox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"/></symbol><symbol id="icon-error" viewBox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"/></symbol><symbol id="icon-info" viewBox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"/></symbol><symbol id="icon-success" viewBox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"/></symbol><symbol id="icon-warning" viewBox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"/></symbol><symbol id="icon-facebook-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"/></symbol><symbol id="icon-facebook" viewBox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"/><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"/></symbol><symbol id="icon-gplus-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M480.5 273.117c.123.076.26.185.416.334.146.152.289.343.432.562.135.211.258.459.359.746.086.286.129.619.129 1.004-.012.7-.164 1.262-.465 1.683a5.374 5.374 0 0 1-.461.562c-.186.176-.375.35-.58.527a2.65 2.65 0 0 0-.322.41.908.908 0 0 0-.176.553c0 .205.059.375.18.508.102.129.199.238.299.336l.662.545a6.48 6.48 0 0 1 1.086 1.119c.293.415.445.955.459 1.625 0 .949-.42 1.789-1.256 2.521-.869.76-2.123 1.15-3.762 1.177-1.373-.015-2.4-.308-3.074-.879-.682-.534-1.023-1.172-1.023-1.917 0-.359.111-.766.336-1.211.215-.443.602-.837 1.172-1.174a6.126 6.126 0 0 1 1.998-.725 15.415 15.415 0 0 1 1.723-.173 3.996 3.996 0 0 1-.377-.596 1.347 1.347 0 0 1-.193-.74c0-.174.023-.317.074-.433a8.83 8.83 0 0 1 .121-.338 5.973 5.973 0 0 1-.633.035c-1.043-.012-1.838-.339-2.381-.979-.574-.603-.861-1.3-.861-2.097 0-.961.404-1.832 1.217-2.616.559-.46 1.135-.757 1.736-.898a8.494 8.494 0 0 1 1.67-.182h3.916l-1.209.704h-1.207v.005l-.005.002zm.754 11.276c0-.498-.162-.932-.484-1.302-.346-.348-.881-.778-1.609-1.29a4.045 4.045 0 0 0-.439-.02c-.102-.016-.354 0-.768.039a8.141 8.141 0 0 0-1.244.275c-.1.036-.24.096-.42.17a2.713 2.713 0 0 0-.553.355 1.985 1.985 0 0 0-.459.592c-.145.25-.215.551-.215.901 0 .687.311 1.253.932 1.698.596.444 1.404.68 2.432.688.928-.014 1.629-.217 2.117-.612.476-.385.71-.886.71-1.494zm-2.729-5.743a1.843 1.843 0 0 0 1.289-.556c.164-.249.271-.506.318-.767a7.32 7.32 0 0 0 .041-.659c0-.772-.195-1.552-.594-2.336a2.611 2.611 0 0 0-.73-.921 1.894 1.894 0 0 0-1.064-.352 1.723 1.723 0 0 0-1.324.64c-.299.437-.438.92-.428 1.457 0 .709.209 1.444.623 2.211.201.356.459.659.775.912.323.248.686.371 1.094.371zm9.698.077h-2.055v-2.056h-.996v2.056h-2.053v.998h2.053v2.052h.996v-2.052h2.055z"/></symbol><symbol id="icon-gplus" viewBox="0 0 24 24"><circle fill="#D84A36" cx="12" cy="12" r="12"/><g fill="#FFF"><path d="M12.012 12.111c-.304-.33-.07-.842.155-1.078.226-.241.639-.56.829-.74.193-.184.66-.605.674-1.868 0 0 .112-1.139-1.127-2.06h.977l1.038-.593h-3.507s-2.083-.067-3.085 1.207c-1 1.276-.966 2.871.114 3.77 1.08.898 2.371.648 2.371.648s-.197.422-.163.775c.036.353.421.849.504.965 0 0-2.386-.011-3.551 1.033-1.165 1.053-1.29 3.097 1.021 3.803 2.312.707 5.303-.055 5.906-2.277 0 0 .262-.908-.189-1.683-.459-.775-1.663-1.571-1.967-1.902zm-2.555-1.83c-.744-1-1.308-2.953 0-3.783 0 0 .955-.579 1.801.266.846.846 1.082 2.277.928 3.003-.15.727-.563.897-.739 1.008-.185.111-1.249.504-1.99-.494zm3.668 5.481c-.035.393-.083 1.254-1.491 1.664-1.409.408-3.826-.264-3.757-1.914 0 0 .006-.698.449-1.089.445-.396 1.094-.794 3.029-.819 0 0 .643.473 1.072.809.427.332.733.952.698 1.349z"/><path d="M16.323 11.055V9.803h-.752v1.252h-1.255v.748h1.255v1.256h.752v-1.256h1.248v-.748z"/></g></symbol><symbol id="icon-linkedin" viewBox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"/><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"/><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"/></g></symbol><symbol id="icon-reddit" viewBox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"/><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"/><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"/><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"/><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"/><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"/><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"/><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"/></symbol><symbol id="icon-twitter-bordered" viewBox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"/><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/></symbol><symbol id="icon-twitter" viewBox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"/><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"/></symbol><symbol id="icon-weibo-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"/><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"/><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"/></symbol><symbol id="icon-weibo" viewBox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"/><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"/><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"/><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"/><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"/><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"/></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner c-banner--info">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner" >
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1"
                 data-gpt-unitpath="/270604982/bmc/hereditasjournal/articles"
                 data-gpt-sizes="728x90,970x90"
                 data-gpt-targeting="pos=LB1;doi=10.1186/s41065-018-0057-5;kwrd=Han Chinese,Japanese,Korean,Genetic ancestry,Population structure,Population divergence,Admixture,SNP;pmc=L32030,L21001;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-018-0057-5&amp;kwrd=Han Chinese,Japanese,Korean,Genetic ancestry,Population structure,Population divergence,Admixture,SNP&amp;pmc=L32030,L21001&amp;">
                        <img data-test="gpt-advert-fallback-img"
                             src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-018-0057-5&amp;kwrd=Han Chinese,Japanese,Korean,Genetic ancestry,Population structure,Population divergence,Admixture,SNP&amp;pmc=L32030,L21001&amp;"
                             alt="Advertisement"
                             width="728"
                             height="90">
                    </a>
                </noscript>
            </div>
        </div>
    </div>

        
         
    <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>


<header class="c-popup-search ctx-publisher-header" id="top" data-test="publisher-header">
    <nav class="c-navbar">
        <div class="u-container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization" />
                <div class="c-logo">
    <a href="https://www.biomedcentral.com" itemprop="url">
        <img src=/static/images/bmc/logos/logo-bmc-white-aj-e3a70ae848.svg alt="BMC" itemprop="logo">
        <div class="c-logo__strapline">
            <img src=/static/images/bmc/logos/logo-bmc-white-strapline-sn-e7b1d6e67f.svg alt="Part of Springer Nature">
        </div>
    </a>
</div>
            </div>
            <button
                type="button"
                class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg"
                id="publisher-header-menu-button"
                data-toggle="collapse"
                data-test="publisher-header-menu-toggle"
                data-toggle-linked="publisher-header"
                data-target="publisher-header-search publisher-header-menu"
                aria-controls="publisher-header-menu publisher-header-search"
                aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


                <ul class="c-navbar__nav">
                    <li class="c-navbar__item">
                        <a data-replace="true"
                           data-placeholder="/placeholder/v1/username"
                           class="c-navbar__link"
                           href="https://www.biomedcentral.com/login"
                           data-test="login-link"
                           data-track="click"
                           data-track-category="Account"
                           data-track-action="Click Login Link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                    <li class="c-navbar__item">
                        <button
                            type="button"
                            class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg"
                            data-toggle="collapse"
                            data-test="header-search-button"
                            data-target="publisher-header-search publisher-header-menu publisher-header-menu-button"
                            aria-controls="publisher-header-search publisher-header-menu"
                            aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true">
                                    <use xlink:href="#icon-search"></use>
                                </svg>
                            </span>
                        </button>
                    </li>
                </ul>
            </div>
        </div>
    </nav>
    <div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field c-form-field--inline" method="GET" action="//www.biomedcentral.com/search" data-track="submit"
        data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label data-track-label="" data-test="global-search">
        <label for="publisherSearch" class="u-visually-hidden">Search all BMC articles</label>
        <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text" value="" placeholder="Search all BMC articles"/>
        <button class="c-button c-button--secondary" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true">
        <use xlink:href="#icon-search"></use>
    </svg>
</button>


        <input type="hidden" name="searchType" value="publisherSearch"/>
    </form>
</div>

            </div>
        </div>
    </div>
</header>


        
            <header class="c-journal-header c-journal-header--hereditas ctx-journal-header">
                <div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="/">
    
        
    
    <span class="c-journal-title__text ">Hereditas</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="u-container">
                        
                            <button type="button"
                                    class="c-navbar__link c-dropdown u-hidden-at-lg"
                                    data-toggle="collapse"
                                    data-test="journal-header-menu"
                                    data-toggle-linked="journal-header"
                                    data-target="journal-menu"
                                    aria-controls="journal-menu"
                                    aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav">
                                    <ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/'>Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/about'>About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href='/articles'>Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/submission-guidelines'>Submission Guidelines</a>
                                            </li>
                                        
                                    </ul>
                                </nav>
                            </div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--ocean-light-blue">
                    
                </div>
            </header>
            
        

        <div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-content-layout c-content-layout--fulltext" data-component="sticky-container">
                <div class="c-content-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3><ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc=""><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results">Results</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec13" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Discussion">Discussion</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec14" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Conclusion">Conclusion</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec15" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Methods">Methods</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li></ol></nav>
                </div>

                <main class="c-content-layout__main">
                    <article itemscope itemtype="http://schema.org/ScholarlyArticle">
                        
                        
    <div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
            <li class="c-article-identifiers__item" data-test="article-category">Research</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
    </div>

                        <div class="FulltextWrapper"><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">Genetic structure, divergence and admixture of Han Chinese, Japanese and Korean populations</h1></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true"><ul class="u-listReset"><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Yuchen&nbsp;Wang</span><sup><a href="#Aff1" class="AffiliationID">1</a>, <a href="#Aff2" class="AffiliationID">2</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Dongsheng&nbsp;Lu</span><sup><a href="#Aff1" class="AffiliationID">1</a>, <a href="#Aff2" class="AffiliationID">2</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Yeun-Jun&nbsp;Chung</span><sup><a href="#Aff3" class="AffiliationID">3</a></sup> and </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Shuhua&nbsp;Xu</span><sup><a href="#Aff1" class="AffiliationID">1</a>, <a href="#Aff2" class="AffiliationID">2</a>, <a href="#Aff4" class="AffiliationID">4</a>, <a href="#Aff5" class="AffiliationID">5</a>, <a href="#Aff6" class="AffiliationID">6</a></sup><a href="mailto:xushua@picb.ac.cn" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a></li></ul></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation"><div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">Hereditas</span></span><span class="ArticleCitation_Year">2018</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">155</strong>:19</span></span></div><p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/s41065-018-0057-5">https://doi.org/10.1186/s41065-018-0057-5</a></p><p class="Copyright">© &nbsp;The Author(s)&nbsp;2018</p></div></div><ul class="ArticleHistory u-reset-list"><li class="History HistoryReceived"><strong>Received: </strong><span>19&nbsp;January&nbsp;2018</span></li><li class="History HistoryAccepted"><strong>Accepted: </strong><span>23&nbsp;March&nbsp;2018</span></li><li class="History HistoryOnlineDate"><strong>Published: </strong><span itemprop="datePublished">6&nbsp;April&nbsp;2018</span></li></ul></div>
                        
                        
                        
    

                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2><div class="c-section__content" id="Abstract"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p id="Par1" class="Para">Han Chinese, Japanese and Korean, the three major ethnic groups of East Asia, share many similarities in&nbsp;appearance, language and culture&nbsp;etc., but their genetic relationships, divergence times and subsequent genetic exchanges have not been well studied.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Results</h3>
                <p id="Par2" class="Para">We conducted a genome-wide study and evaluated the population structure of 182 Han Chinese, 90 Japanese and 100 Korean individuals, together with the data of 630 individuals representing 8 populations&nbsp;wordwide. Our analyses revealed that Han Chinese, Japanese and Korean populations have distinct genetic makeup and can be well distinguished based on either the genome wide data or a panel of ancestry informative markers (AIMs). Their genetic structure corresponds well to their geographical distributions, indicating geographical isolation played a critical role in driving population differentiation&nbsp;in East Asia. The most recent common ancestor of the three populations was dated back to 3000 ~ 3600&nbsp;years ago. Our analyses also revealed substantial admixture within the three populations which occurred subsequent to initial splits, and distinct gene introgression from surrounding populations, of which northern ancestral component is dominant.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusions</h3>
                <p id="Par3" class="Para">These estimations and findings facilitate to understanding population history and mechanism of human genetic diversity in East Asia, and have implications for both evolutionary and medical studies.</p>
              </div></div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords"><li class="c-keywords__item" itemprop="about">Han Chinese</li><li class="c-keywords__item" itemprop="about">Japanese</li><li class="c-keywords__item" itemprop="about">Korean</li><li class="c-keywords__item" itemprop="about">Genetic ancestry</li><li class="c-keywords__item" itemprop="about">Population structure</li><li class="c-keywords__item" itemprop="about">Population divergence</li><li class="c-keywords__item" itemprop="about">Admixture</li><li class="c-keywords__item" itemprop="about">SNP</li></ul></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par29" class="Para">East Asia is one of the world’s most populated places, consisting of about 38% of the Asian population or about 22% of the world-wide population. East Asian peoples, especially the three major ethnicities, Han Chinese, Japanese and Korean, share many similarities in characteristics, for example, yellow skin, black eyes and black hair, short and flat noses, which make them hard to be distinguished by appearance. Moreover, East Asian people use similar languages and words, for example, Chinese characters are shared in Japanese, and also existed in Korean until their recent abolition in the 1940s [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par30" class="Para">While many studies have reported the overall picture of genetic structure of global populations, finer scale details of population structure and relevant issues in East Asia have not yet been well addressed. In fact, in many previous studies, samples of Han Chinese and Japanese populations were generally treated as a single group [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>], and notably, Korean samples are frequently absent from many important international collaborative projects due to their assumed similarity to Han Chinese and Japanese samples, such as the International HapMap Project [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>] and the 1000 Genomes Project [<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>]. A recent study [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>] provided a landscape of autosomal variation and an overall picture of the genetic relationship of Asian populations. However, fine scale genetic structure and relationship among Han Chinese, Japanese and Korean populations have not been revealed or well-studied, partially due to low density of the data.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par31" class="Para">Here, we newly genotyped 100 Korean samples, and conducted a joint population genetic analysis with 182 Han Chinese individuals and 90 Japanese individuals. We attempted to address some fundamental questions related to the population histories of the three ethnic groups. First, we asked what the genetic make-up of the three populations is and how well they are differentiated. Second, we attempted to dissect population history of the three ethnic groups and infer their genetic origins based on state-of-art methods developed recently. Finally, we asked whether there were subsequent gene-flow occurred since population splits. We believe these efforts can advance our understanding of human genetic diversity and migration history in East Asia, and provide insightful information for future medical studies in the three East Asian populations.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Results" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Results">Results</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Results" data-test="collapse-section"><section id="Sec3" class="Section2 RenderAsSection2">
                <h3 class="Heading">Population structure and genetic relationship</h3>
                <section id="Sec4" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Genetic difference measured by F<sub xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml">ST</sub>
                           </h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par32" class="Para">To assess the genetic relationship among East Asian groups, we examined genome-wide single nucleotide polymorphism (SNP) data in 1032 individuals representing 12 populations. We first calculated pairwise F<sub>ST</sub> based on genome-wide SNPs (Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>: Figure S1, Additional file <span class="InternalRef"><a href="#MOESM2">2</a></span>: Table S2). The three smallest F<sub>ST</sub> values are between north Han Chinese (CHB) and south Han Chinese (CHS) (F<sub>ST[CHB-CHS]</sub> = 0.0014), between Chinese Dai in Xishuangbanna (CDX) and Kinh in Ho Chi Minh City of Vietnam (KHV) (F<sub>ST[CDX-KHV]</sub> = 0.0024), and between CHB and Korean (KOR) (F<sub>ST[CHB-KOR]</sub> = 0.0026). Genetic difference between KOR and Japanese (JPT) (F<sub>ST[JPT-KOR]</sub> = 0.0033) is larger than that between KOR and CHB, but smaller than that between two Mongolian populations (Buryat Mongolian (BMON) and DU Mongolian in Qinghai (QHM) [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>], F<sub>ST[BMON-QHM]</sub> = 0.0053). These results are largely consistent with the geographic distribution of populations. Generally, pairwise F<sub>ST</sub> between Han Chinese, Japanese and Korean (0.0026~ 0.0090) are greater than that within Han Chinese (0.0014), but are much smaller than that between European (CEU) and any East Asian groups, for example, the smallest F<sub>ST</sub> between CEU and East Asian populations (F<sub>ST[CEU-BMON]</sub> = 0.0838; F<sub>ST[CEU-KHV]</sub> = 0.1059) are higher by an order of magnitude. These results suggested a comparatively closer relationship between Han Chinese, Japanese and Korean. More detailed information about the populations can be found in Additional file <span class="InternalRef"><a href="#MOESM3">3</a></span>: Table S1.</p>
                </section>
                <section id="Sec5" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Phylogenetic trees of populations and individuals</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par33" class="Para">A maximum likelihood (ML) tree reconstructed based on pairwise allele frequency difference provides a better visualization of the genetic relationship of populations (Fig.&nbsp;<span class="InternalRef"><a href="#Fig2">2a</a></span>). All the East Asian populations share a clade, and Mongolians are much closer to European than any&nbsp;other East Asian groups. Among the eight non-Mongolian East Asian populations, the Tibetan population&nbsp;(TIB) shows apparent differentiation from the other seven populations which are comprised of four typical mainland populations (CDX, CHB, CHS and KHV) and three island or peninsula populations (JPRK, JPT and KOR).</p>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par34" class="Para">An individual level neighbor-joining (NJ) tree of 1002 world-wide individuals based on genotyping differentiation was also constructed (Additional file <span class="InternalRef"><a href="#MOESM4">4</a></span>: Figure S2A). African and European individuals cluster together respectively, and East Asians individuals also have their own distinct cluster. In East Asians individuals, Mongolian, Ryukyuan and Tibetan have relatively distinct cluster, while Han Chinese, Japanese and Korean showed a relatively mixed phylogeny, suggest their much closer relationship, although substructures are also apparent. Han Chinese samples also show mixture with Southern populations (CDX and KHV).</p>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par35" class="Para">Using only 6 populations (two Han Chinese populations, Japanese, Korean and two Mongolian populations) to reconstruct an individual tree, we found the phylogeny of the populations became clearer (Additional file <span class="InternalRef"><a href="#MOESM4">4</a></span>: Figure S2B). Japanese individuals have their own cluster and Korean individuals are almost distinct from Han Chinese. North and South Han Chinese mixed together, but still have some substructure.</p>
                </section>
                <section id="Sec6" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Principal component analysis</h4>
                  <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par36" class="Para">We further performed principal components analysis (PCA) to examine the population structure on individual level (Figs.&nbsp;<span class="InternalRef"><a href="#Fig1">1b</a></span>, <span class="InternalRef"><a href="#Fig2">2b</a></span>, and Additional file <span class="InternalRef"><a href="#MOESM5">5</a></span>: Figure S3). We first analyzed East Asian populations with populations worldwide based on 9063 genome-wide SNPs with inter-marker distance &gt; 500&nbsp;kb (see <span class="InternalRef"><a href="#Sec15">Methods</a></span>) (Additional file <span class="InternalRef"><a href="#MOESM5">5</a></span>: Figure S3A). The first principal component (PC1) and the second principal component (PC2) explain 9.2 and 4.4% of the total variance (or 58.0% and 28.0% of the top ten PCs), respectively. East Asian groups cluster together closely and are located far from European or African in the PC plot, consistent with their differentiation from other continental populations. We observed some individuals, most of which are Mongolian individuals (BMON or&nbsp;QHM), extend towards CEU cluster, suggesting recent gene flow might have occurred between European and Mongolian populations. Analysis of a dataset with higher density markers (44,549 SNPs, after excluding JPRK samples due to lower marker density, Additional file <span class="InternalRef"><a href="#MOESM5">5</a></span>: Figure S3B) showed similar pattern except that individuals were more tightly clustered (PC1 and PC2 explained 10.2% and 4.7% of the variance respectively).<figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO1">
                        <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0057-5/MediaObjects/41065_2018_57_Fig1_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0057-5/MediaObjects/41065_2018_57_Fig1_HTML.gif" alt="Fig. 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0057-5/MediaObjects/41065_2018_57_Fig1_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0057-5/MediaObjects/41065_2018_57_Fig1_HTML.gif" alt="Fig. 1"></img></noscript></a>
                      </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 1</span>
                          <p xmlns="" class="SimplePara">Asian samples location and population relatedness. (<strong class="EmphasisTypeBold">a</strong>) Geographic location of the sampled populations (generated by R 2.15.2 and Microsoft PowerPoint 2010); (<strong class="EmphasisTypeBold">b</strong>) Principal component analysis (PCA) of all the 10 Asian sample populations</p>
                        </div></figcaption></figure><figure xmlns="" class="Figure" id="Fig2"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO2">
                        <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0057-5/MediaObjects/41065_2018_57_Fig2_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0057-5/MediaObjects/41065_2018_57_Fig2_HTML.gif" alt="Fig. 2" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0057-5/MediaObjects/41065_2018_57_Fig2_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0057-5/MediaObjects/41065_2018_57_Fig2_HTML.gif" alt="Fig. 2"></img></noscript></a>
                      </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 2</span>
                          <p xmlns="" class="SimplePara">Population level phylogenetic Tree and Principal component analysis (PCA). (A) The maximum likelihood tree was constructed based on pair-wise F<sub xmlns="http://www.w3.org/1999/xhtml">ST</sub> matrix. And the marked number are bootstrap value; (B) The top two PCs of individuals representing six East Asian populations, mapped to their corresponding geographic locations (generated by R 2.15.2 and Microsoft Excel 2010)</p>
                        </div></figcaption></figure></div>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par37" class="Para">Although East Asian individuals cluster very tightly in the context of analysis of worldwide populations, PCA of ten East Asian populations showed they have substantial substructure (Fig. <span class="InternalRef"><a href="#Fig1">1b</a></span>). For example, Han Chinese (CHB and CHS) and southern populations (CDX and KHV) were separated from other populations by PC1. According to coordinate of PC2 which explained 0.7% of the total variance, Mongolian and Tibetan are closely located in one side, while the island populations (Japanese and Ryukyuan) other side. Except Mongolian and Tibetan, other populations showed much closer relationship in the PC plot (Fig. <span class="InternalRef"><a href="#Fig1">1b</a></span>, Additional file <span class="InternalRef"><a href="#MOESM5">5</a></span>: Figure S3C). Some Tibetan and DU Mongolian (QHM) individuals distribute toward CHB cluster, suggesting gene flow between these northern populations. These patterns are even more obvious when JPRK samples were removed from the analysis (Additional file <span class="InternalRef"><a href="#MOESM5">5</a></span>: Figure S3C). High density dataset showed more pronounced substructure: On a finer scale, two Mongolian populations also showed substructure, and QHM individuals are sitting between BMON and TIB clusters which is consistent with their history [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>]. In all the analyses, Dai (CDX) and Vietnamese (KHV) always cluster closely and can be separated from Han Chinese, Japanese and Korean (Fig. <span class="InternalRef"><a href="#Fig1">1b</a></span>, Additional file <span class="InternalRef"><a href="#MOESM5">5</a></span>: Figure S3C).</p>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par38" class="Para">To examine genetic relationship of the typical East Asian populations, we did a separate PCA with some southern populations excluded (Fig. <span class="InternalRef"><a href="#Fig2">2b</a></span>). PC1 separates Mongolian from the other groups and explain 0.9% of the variance (or 21.9% of the top 10 PCs). PC2 separates populations living in&nbsp;the islands and peninsula from the mainland populations. Japanese and Korean each has each independent cluster, while two Han Chinese populations and two Mongolian both have some overlaps, and&nbsp;some Mongolian individuals are located toward CHB. Interestingly, the distribution of these clusters is generally consistent with the geographical distribution of these populations&nbsp;(Fig. <span class="InternalRef"><a href="#Fig2">2b</a></span>). We next performed PCA of only Han Chinese, Japanese and Korean individuals (Additional file <span class="InternalRef"><a href="#MOESM5">5</a></span>: Figure S3D), PC1 explained 0.8% of the variance (or 20.2% of the top 10 PCs) and separated Han Chinese from Japanese and Korean. PC2 separated CHB and KOR from CHS and JPT. Japanese and Korean each has its own cluster and both&nbsp;are located apart from Han Chinese while CHB and CHS cannot be completely separated at this scale. These results suggested that although similar in appearance, Han Chinese, Japanese and Korean are different in terms of genetic make-up, and the difference among the three groups are much larger than that between northern and southern Han Chinese.</p>
                </section>
                <section id="Sec7" class="Section3 RenderAsSection3">
                  <h4 class="Heading">K-mean clustering analysis</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par39" class="Para">We classified Han Chinese, Japanese and Korean individuals by K-means clustering with first 10 PCs (Additional file <span class="InternalRef"><a href="#MOESM6">6</a></span>: Figure S4). Assuming three clusters (K = 3), Han Chinese and Korean are not well distinguished, while all the Japanese individuals cluster perfectly. There are two clusters in Han Chinese and Korean, the north cluster (NC, in orange) and the south cluster (SC, in green). All the Korean individuals belong to NC while CHB and CHS have SC&nbsp;on average&nbsp;of&nbsp; 7.5% and 67.4% of SC, respectively. Assuming four clusters (K = 4), Han Chinese can be distinguished well from Korean. The NC at K = 3&nbsp;separated into Korean cluster (KC, in blue) and north Han Chinese cluster (NHC, in orange) [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>]. Around 64.0% of CHB individuals and 9.0% of KOR individuals are assigned to NHC, while 3.4 and 32.6% of CHB individuals to KC and SC (in green), respectively.</p>
                </section>
                <section id="Sec8" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Screen ancestry informative markers</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par40" class="Para">We further explored whether the three populations could be well distinguished with a small number of ancestry informative markers (AIMs, see <span class="InternalRef"><a href="#Sec15">Methods</a></span>). Individuals from any two populations can be distinguished with a sufficient number of AIMs, and we use Matthews correlation coefficient (MCC, see <span class="InternalRef"><a href="#Sec15">Methods</a></span>) to measure the clustering ability of AIMs (Additional file <span class="InternalRef"><a href="#MOESM7">7</a></span>: Figure S5). As a result, we could use only 89 SNPs (CHB/KOR), 46 SNPs (CHB/JPT), 44 SNPs (CHS/KOR), 26 SNPs (CHS/JPT) and 73 SNPs (JPT/KOR) respectively to perfectly distinguish each population pairs. These AIMs can facilitate discerning and controlling of population structure among major East Asian populations in future association studies [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>].</p>
                </section>
              </section><section id="Sec9" class="Section2 RenderAsSection2">
                <h3 class="Heading">Origins, divergence time and migration history of Han Chinese, Japanese and Korean</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par41" class="Para">To reveal the population history of the three East Asian ethnic groups, we conducted a series of population genetic analysis and estimation. We first estimated the effective population size (N<sub>e</sub>) [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>] of East Asian and worldwide&nbsp;populations from 5 to 250 thousand years ago (KYA), assuming 25&nbsp;years per generation [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>] (Fig.&nbsp;<span class="InternalRef"><a href="#Fig3">3</a></span>, Additional file <span class="InternalRef"><a href="#MOESM8">8</a></span>: Figure S6). Our analysis showed that African population (YRI) had the greatest N<sub>e</sub> before 500 generations ago (GA), and N<sub>e</sub> of European (CEU) was also greater than those of most East Asian populations (except for CDX and KHV) before 1000 GA. However, East Asian populations all showed strong recent growth (~ 1% per generation during the recent 500 generations), so that N<sub>e</sub> of most East Asian populations exceeded that of YRI during the recent ~ 12.5 KYA. In East Asia, CDX and KHV have the greatest N<sub>e</sub>, followed by CHB. Tibetan (TIB) and Mongolian (BMON and QHM) have greater N<sub>e</sub> than CHS and KOR before 500 GA, but grew slower than the later ones in recent 7500&nbsp;years. Because of low marker density of the data, the estimation of N<sub>e</sub> of JPRK might not be accurate, but apparently N<sub>e</sub> of JPRK was the smallest. Korean roughly has the same N<sub>e</sub> with Japanese and two Mongolian populations in history, but showed greater increasing rate in recent 12,500&nbsp;years.<figure xmlns="" class="Figure" id="Fig3"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO3">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0057-5/MediaObjects/41065_2018_57_Fig3_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0057-5/MediaObjects/41065_2018_57_Fig3_HTML.gif" alt="Fig. 3" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0057-5/MediaObjects/41065_2018_57_Fig3_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0057-5/MediaObjects/41065_2018_57_Fig3_HTML.gif" alt="Fig. 3"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 3</span>
                        <p xmlns="" class="SimplePara">Effective population size varied along time. Effective population size (N<sub xmlns="http://www.w3.org/1999/xhtml">e</sub>) was calculated from LD observations in 2500 distance classes (from 0.001&nbsp;cM to 2.5&nbsp;cM with the window size set as 0.001&nbsp;cM) for each population. Results from 2000 generations ago (GA) to 200 GA are shown (generated by Microsoft Office 365)</p>
                      </div></figcaption></figure></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par42" class="Para">We further estimated divergence time (T<sub>F</sub>) of each population pair (Tables&nbsp;<span class="InternalRef"><a href="#Tab1">1</a></span> and <span class="InternalRef"><a href="#Tab2">2</a></span>) based on F<sub>ST</sub> (Additional file <span class="InternalRef"><a href="#MOESM2">2</a></span>: Table S2) and N<sub>e</sub> following the formula T<sub>F</sub> = 2N<sub>e</sub>F<sub>ST</sub> (1) [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>]. Most non-African populations diverged from African ~ 82 KYA, consistent with the time of “Out of Africa” as suggested by mtDNA studies [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>]. European and East Asian populations diverged later than that between African and&nbsp;non-African populations. Among East Asian populations, CDX diverged from CEU ~ 56 KYA, while the other East Asian populations separated from European about 35~ 50 KYA.<figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 1</span>
                        <p xmlns="" class="SimplePara">Pairwise divergence time (T<sub xmlns="http://www.w3.org/1999/xhtml">F</sub>)</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col char="."></col><col char="."></col><col char="."></col><col char="."></col><col char="."></col><col char="."></col><col char="."></col><col char="."></col><col char="."></col><col char="."></col><col char="."></col><col char="."></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">T<sub xmlns="http://www.w3.org/1999/xhtml">F</sub>(GA)</p></th><th char="."><p xmlns="" class="SimplePara">BMON</p></th><th char="."><p xmlns="" class="SimplePara">CDX</p></th><th char="."><p xmlns="" class="SimplePara">CEU</p></th><th char="."><p xmlns="" class="SimplePara">CHB</p></th><th char="."><p xmlns="" class="SimplePara">CHS</p></th><th char="."><p xmlns="" class="SimplePara">JPRK</p></th><th char="."><p xmlns="" class="SimplePara">JPT</p></th><th char="."><p xmlns="" class="SimplePara">KHV</p></th><th char="."><p xmlns="" class="SimplePara">KOR</p></th><th char="."><p xmlns="" class="SimplePara">QHM</p></th><th char="."><p xmlns="" class="SimplePara">TIB</p></th><th char="."><p xmlns="" class="SimplePara">YRI</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">BMON</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">603</p></td><td char="."><p xmlns="" class="SimplePara">1399</p></td><td char="."><p xmlns="" class="SimplePara">276</p></td><td char="."><p xmlns="" class="SimplePara">315</p></td><td char="."><p xmlns="" class="SimplePara">321</p></td><td char="."><p xmlns="" class="SimplePara">265</p></td><td char="."><p xmlns="" class="SimplePara">478</p></td><td char="."><p xmlns="" class="SimplePara">238</p></td><td char="."><p xmlns="" class="SimplePara">88</p></td><td char="."><p xmlns="" class="SimplePara">284</p></td><td char="."><p xmlns="" class="SimplePara">3796</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">CDX</p></td><td char="."><p xmlns="" class="SimplePara">603</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">2256</p></td><td char="."><p xmlns="" class="SimplePara">218</p></td><td char="."><p xmlns="" class="SimplePara">118</p></td><td char="."><p xmlns="" class="SimplePara">482</p></td><td char="."><p xmlns="" class="SimplePara">374</p></td><td char="."><p xmlns="" class="SimplePara">60</p></td><td char="."><p xmlns="" class="SimplePara">320</p></td><td char="."><p xmlns="" class="SimplePara">1527</p></td><td char="."><p xmlns="" class="SimplePara">509</p></td><td char="."><p xmlns="" class="SimplePara">4518</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">CEU</p></td><td char="."><p xmlns="" class="SimplePara">1399</p></td><td char="."><p xmlns="" class="SimplePara">2256</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">1839</p></td><td char="."><p xmlns="" class="SimplePara">1690</p></td><td char="."><p xmlns="" class="SimplePara">1369</p></td><td char="."><p xmlns="" class="SimplePara">1676</p></td><td char="."><p xmlns="" class="SimplePara">1985</p></td><td char="."><p xmlns="" class="SimplePara">1782</p></td><td char="."><p xmlns="" class="SimplePara">1896</p></td><td char="."><p xmlns="" class="SimplePara">1589</p></td><td char="."><p xmlns="" class="SimplePara">3290</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">CHB</p></td><td char="."><p xmlns="" class="SimplePara">276</p></td><td char="."><p xmlns="" class="SimplePara">218</p></td><td char="."><p xmlns="" class="SimplePara">1839</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">24</p></td><td char="."><p xmlns="" class="SimplePara">238</p></td><td char="."><p xmlns="" class="SimplePara">122</p></td><td char="."><p xmlns="" class="SimplePara">149</p></td><td char="."><p xmlns="" class="SimplePara">47</p></td><td char="."><p xmlns="" class="SimplePara">1126</p></td><td char="."><p xmlns="" class="SimplePara">199</p></td><td char="."><p xmlns="" class="SimplePara">3948</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">CHS</p></td><td char="."><p xmlns="" class="SimplePara">315</p></td><td char="."><p xmlns="" class="SimplePara">118</p></td><td char="."><p xmlns="" class="SimplePara">1690</p></td><td char="."><p xmlns="" class="SimplePara">24</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">238</p></td><td char="."><p xmlns="" class="SimplePara">144</p></td><td char="."><p xmlns="" class="SimplePara">73</p></td><td char="."><p xmlns="" class="SimplePara">81</p></td><td char="."><p xmlns="" class="SimplePara">1059</p></td><td char="."><p xmlns="" class="SimplePara">236</p></td><td char="."><p xmlns="" class="SimplePara">3682</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">JPRK</p></td><td char="."><p xmlns="" class="SimplePara">321</p></td><td char="."><p xmlns="" class="SimplePara">482</p></td><td char="."><p xmlns="" class="SimplePara">1369</p></td><td char="."><p xmlns="" class="SimplePara">238</p></td><td char="."><p xmlns="" class="SimplePara">238</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">79</p></td><td char="."><p xmlns="" class="SimplePara">379</p></td><td char="."><p xmlns="" class="SimplePara">179</p></td><td char="."><p xmlns="" class="SimplePara">894</p></td><td char="."><p xmlns="" class="SimplePara">329</p></td><td char="."><p xmlns="" class="SimplePara">3156</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">JPT</p></td><td char="."><p xmlns="" class="SimplePara">265</p></td><td char="."><p xmlns="" class="SimplePara">374</p></td><td char="."><p xmlns="" class="SimplePara">1676</p></td><td char="."><p xmlns="" class="SimplePara">122</p></td><td char="."><p xmlns="" class="SimplePara">144</p></td><td char="."><p xmlns="" class="SimplePara">79</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">285</p></td><td char="."><p xmlns="" class="SimplePara">54</p></td><td char="."><p xmlns="" class="SimplePara">1030</p></td><td char="."><p xmlns="" class="SimplePara">279</p></td><td char="."><p xmlns="" class="SimplePara">3685</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">KHV</p></td><td char="."><p xmlns="" class="SimplePara">478</p></td><td char="."><p xmlns="" class="SimplePara">60</p></td><td char="."><p xmlns="" class="SimplePara">1985</p></td><td char="."><p xmlns="" class="SimplePara">149</p></td><td char="."><p xmlns="" class="SimplePara">73</p></td><td char="."><p xmlns="" class="SimplePara">379</p></td><td char="."><p xmlns="" class="SimplePara">285</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">235</p></td><td char="."><p xmlns="" class="SimplePara">1347</p></td><td char="."><p xmlns="" class="SimplePara">409</p></td><td char="."><p xmlns="" class="SimplePara">4172</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">KOR</p></td><td char="."><p xmlns="" class="SimplePara">238</p></td><td char="."><p xmlns="" class="SimplePara">320</p></td><td char="."><p xmlns="" class="SimplePara">1782</p></td><td char="."><p xmlns="" class="SimplePara">47</p></td><td char="."><p xmlns="" class="SimplePara">81</p></td><td char="."><p xmlns="" class="SimplePara">179</p></td><td char="."><p xmlns="" class="SimplePara">54</p></td><td char="."><p xmlns="" class="SimplePara">235</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">1050</p></td><td char="."><p xmlns="" class="SimplePara">225</p></td><td char="."><p xmlns="" class="SimplePara">3826</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">QHM</p></td><td char="."><p xmlns="" class="SimplePara">88</p></td><td char="."><p xmlns="" class="SimplePara">1527</p></td><td char="."><p xmlns="" class="SimplePara">1896</p></td><td char="."><p xmlns="" class="SimplePara">1126</p></td><td char="."><p xmlns="" class="SimplePara">1059</p></td><td char="."><p xmlns="" class="SimplePara">894</p></td><td char="."><p xmlns="" class="SimplePara">1030</p></td><td char="."><p xmlns="" class="SimplePara">1347</p></td><td char="."><p xmlns="" class="SimplePara">1050</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">998</p></td><td char="."><p xmlns="" class="SimplePara">3988</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">TIB</p></td><td char="."><p xmlns="" class="SimplePara">284</p></td><td char="."><p xmlns="" class="SimplePara">509</p></td><td char="."><p xmlns="" class="SimplePara">1589</p></td><td char="."><p xmlns="" class="SimplePara">199</p></td><td char="."><p xmlns="" class="SimplePara">236</p></td><td char="."><p xmlns="" class="SimplePara">329</p></td><td char="."><p xmlns="" class="SimplePara">279</p></td><td char="."><p xmlns="" class="SimplePara">409</p></td><td char="."><p xmlns="" class="SimplePara">225</p></td><td char="."><p xmlns="" class="SimplePara">998</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">3592</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">YRI</p></td><td char="."><p xmlns="" class="SimplePara">3796</p></td><td char="."><p xmlns="" class="SimplePara">4518</p></td><td char="."><p xmlns="" class="SimplePara">3290</p></td><td char="."><p xmlns="" class="SimplePara">3948</p></td><td char="."><p xmlns="" class="SimplePara">3682</p></td><td char="."><p xmlns="" class="SimplePara">3156</p></td><td char="."><p xmlns="" class="SimplePara">3685</p></td><td char="."><p xmlns="" class="SimplePara">4172</p></td><td char="."><p xmlns="" class="SimplePara">3826</p></td><td char="."><p xmlns="" class="SimplePara">3988</p></td><td char="."><p xmlns="" class="SimplePara">3592</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">Note: Pairwise divergence time (T<sub xmlns="http://www.w3.org/1999/xhtml">F</sub>), in units of generations ago (GA), estimated by 2N<sub xmlns="http://www.w3.org/1999/xhtml">e</sub>F<sub xmlns="http://www.w3.org/1999/xhtml">ST</sub></p>
                    </div></figure><figure xmlns="" id="Tab2" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 2</span>
                        <p xmlns="" class="SimplePara">Results of F<sub xmlns="http://www.w3.org/1999/xhtml">4</sub> test</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col></col><col></col><col char="."></col><col char="."></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">popO</p></th><th><p xmlns="" class="SimplePara">popA</p></th><th><p xmlns="" class="SimplePara">popB</p></th><th><p xmlns="" class="SimplePara">popC</p></th><th char="."><p xmlns="" class="SimplePara">F<sub xmlns="http://www.w3.org/1999/xhtml">4</sub>(A,O;C,B)</p></th><th char="."><p xmlns="" class="SimplePara">F<sub xmlns="http://www.w3.org/1999/xhtml">4</sub>(B,O;C,A)</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">YRI</p></td><td><p xmlns="" class="SimplePara">TIB</p></td><td><p xmlns="" class="SimplePara">CDX</p></td><td><p xmlns="" class="SimplePara">CHB</p></td><td char="."><p xmlns="" class="SimplePara">0.01255</p></td><td char="."><p xmlns="" class="SimplePara">0.01977</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">YRI</p></td><td><p xmlns="" class="SimplePara">TIB</p></td><td><p xmlns="" class="SimplePara">CDX</p></td><td><p xmlns="" class="SimplePara">CHS</p></td><td char="."><p xmlns="" class="SimplePara">0.01019</p></td><td char="."><p xmlns="" class="SimplePara">0.02478</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">YRI</p></td><td><p xmlns="" class="SimplePara">TIB</p></td><td><p xmlns="" class="SimplePara">CDX</p></td><td><p xmlns="" class="SimplePara">KOR</p></td><td char="."><p xmlns="" class="SimplePara">0.01333</p></td><td char="."><p xmlns="" class="SimplePara">0.01780</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">CEU</p></td><td><p xmlns="" class="SimplePara">TIB</p></td><td><p xmlns="" class="SimplePara">CDX</p></td><td><p xmlns="" class="SimplePara">CHB</p></td><td char="."><p xmlns="" class="SimplePara">0.01063</p></td><td char="."><p xmlns="" class="SimplePara">0.02007</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">CEU</p></td><td><p xmlns="" class="SimplePara">TIB</p></td><td><p xmlns="" class="SimplePara">CDX</p></td><td><p xmlns="" class="SimplePara">CHS</p></td><td char="."><p xmlns="" class="SimplePara">0.00957</p></td><td char="."><p xmlns="" class="SimplePara">0.02605</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">CEU</p></td><td><p xmlns="" class="SimplePara">TIB</p></td><td><p xmlns="" class="SimplePara">CDX</p></td><td><p xmlns="" class="SimplePara">KOR</p></td><td char="."><p xmlns="" class="SimplePara">0.01238</p></td><td char="."><p xmlns="" class="SimplePara">0.01917</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">YRI</p></td><td><p xmlns="" class="SimplePara">CDX</p></td><td><p xmlns="" class="SimplePara">JPRK</p></td><td><p xmlns="" class="SimplePara">CHB</p></td><td char="."><p xmlns="" class="SimplePara">0.01317</p></td><td char="."><p xmlns="" class="SimplePara">0.00911</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">YRI</p></td><td><p xmlns="" class="SimplePara">CDX</p></td><td><p xmlns="" class="SimplePara">JPRK</p></td><td><p xmlns="" class="SimplePara">CHS</p></td><td char="."><p xmlns="" class="SimplePara">0.01818</p></td><td char="."><p xmlns="" class="SimplePara">0.00843</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">YRI</p></td><td><p xmlns="" class="SimplePara">CDX</p></td><td><p xmlns="" class="SimplePara">JPRK</p></td><td><p xmlns="" class="SimplePara">JPT</p></td><td char="."><p xmlns="" class="SimplePara">0.00723</p></td><td char="."><p xmlns="" class="SimplePara">0.02230</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">YRI</p></td><td><p xmlns="" class="SimplePara">CDX</p></td><td><p xmlns="" class="SimplePara">JPRK</p></td><td><p xmlns="" class="SimplePara">KOR</p></td><td char="."><p xmlns="" class="SimplePara">0.01119</p></td><td char="."><p xmlns="" class="SimplePara">0.01547</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">CEU</p></td><td><p xmlns="" class="SimplePara">CDX</p></td><td><p xmlns="" class="SimplePara">JPRK</p></td><td><p xmlns="" class="SimplePara">CHB</p></td><td char="."><p xmlns="" class="SimplePara">0.01239</p></td><td char="."><p xmlns="" class="SimplePara">0.00734</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">CEU</p></td><td><p xmlns="" class="SimplePara">CDX</p></td><td><p xmlns="" class="SimplePara">JPRK</p></td><td><p xmlns="" class="SimplePara">CHS</p></td><td char="."><p xmlns="" class="SimplePara">0.01838</p></td><td char="."><p xmlns="" class="SimplePara">0.00788</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">CEU</p></td><td><p xmlns="" class="SimplePara">CDX</p></td><td><p xmlns="" class="SimplePara">JPRK</p></td><td><p xmlns="" class="SimplePara">JPT</p></td><td char="."><p xmlns="" class="SimplePara">0.00728</p></td><td char="."><p xmlns="" class="SimplePara">0.02052</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">CEU</p></td><td><p xmlns="" class="SimplePara">CDX</p></td><td><p xmlns="" class="SimplePara">JPRK</p></td><td><p xmlns="" class="SimplePara">KOR</p></td><td char="."><p xmlns="" class="SimplePara">0.01150</p></td><td char="."><p xmlns="" class="SimplePara">0.01442</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">YRI</p></td><td><p xmlns="" class="SimplePara">TIB</p></td><td><p xmlns="" class="SimplePara">JPRK</p></td><td><p xmlns="" class="SimplePara">JPT</p></td><td char="."><p xmlns="" class="SimplePara">0.00766</p></td><td char="."><p xmlns="" class="SimplePara">0.02798</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">YRI</p></td><td><p xmlns="" class="SimplePara">TIB</p></td><td><p xmlns="" class="SimplePara">JPRK</p></td><td><p xmlns="" class="SimplePara">KOR</p></td><td char="."><p xmlns="" class="SimplePara">0.01241</p></td><td char="."><p xmlns="" class="SimplePara">0.02116</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">CEU</p></td><td><p xmlns="" class="SimplePara">TIB</p></td><td><p xmlns="" class="SimplePara">JPRK</p></td><td><p xmlns="" class="SimplePara">JPT</p></td><td char="."><p xmlns="" class="SimplePara">0.00769</p></td><td char="."><p xmlns="" class="SimplePara">0.02849</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">CEU</p></td><td><p xmlns="" class="SimplePara">TIB</p></td><td><p xmlns="" class="SimplePara">JPRK</p></td><td><p xmlns="" class="SimplePara">KOR</p></td><td char="."><p xmlns="" class="SimplePara">0.01267</p></td><td char="."><p xmlns="" class="SimplePara">0.02239</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">Note: population A (popA) and population B (popB) are two donate populations, population C (popC) is a target population who received gene flow, population O (popO) is an out group. Additional file <span xmlns="http://www.w3.org/1999/xhtml" class="InternalRef"><a href="#MOESM14">14</a></span>: Figure S10C shows the model of F<sub xmlns="http://www.w3.org/1999/xhtml">4</sub> test. The value of F<sub xmlns="http://www.w3.org/1999/xhtml">4</sub>(A,O;C,B) stands for the size of the common part of the two pathway A to O and C to B (labeled as l) on the tree weighted by the gene flow proportion α (labeled as αl), and similarly F4(B,O;C,A) equals (1-α)m. And the calculation of F<sub xmlns="http://www.w3.org/1999/xhtml">4</sub> value noted in METHODS</p>
                    </div></figure></div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par43" class="Para">We found that the present-day Han Chinese and Japanese have the most recent common ancestor that can be dated back about 3.0~ 3.6 KYA (corresponding to the Shang Dynasty in Chinese history). Korean and northern Han Chinese had frequent communications in ancient time, and the divergence time between the two populations was estimated as ~ 1.2 KYA (corresponding to the later period of Three Kingdoms of Korea, or the Tang Dynasty in China). And Japanese and Korean separated ~ 1.4 KYA,&nbsp;a little earlier than that of Han Chinese and Korean (corresponding to Asuka period in Japan, or in the middle of Three Kingdoms period of Korea).</p>
              </section><section id="Sec10" class="Section2 RenderAsSection2">
                <h3 class="Heading">Gene flow among east Asian populations</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par44" class="Para">Considering their frequent communication in history, we are curious about how the initial common origin and recent gene flow contributed to the genetic make-up of Han Chinese, Japanese and Korean populations.</p>
                <section id="Sec11" class="Section3 RenderAsSection3">
                  <h4 class="Heading">STRUCTURE and ADMIXTURE analysis</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par45" class="Para">To examine fine-scale population structure and assess the genetic make-up of East Asian groups, we applied a model-based method, <em xmlns="" class="EmphasisTypeItalic">STRUCTURE</em> [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>], to analyze the genome-wide data with that of worldwide populations. East Asian populations showed distinct genetic component inferred from STRUCTURE which is totally different from those of European or African (Additional file <span class="InternalRef"><a href="#MOESM9">9</a></span>: Figure S7). There is no more than 0.5% of African’s contribution in East Asian’s genome can be detected, while in Mongolian there is considerable contribution from European ancestry (12.5~ 16.9%) (Additional file <span class="InternalRef"><a href="#MOESM9">9</a></span>: Figure S7, K = 3). We identified a component which we named northern East Asian component (NEAC) existing widely in the East Asian populations (Additional file <span class="InternalRef"><a href="#MOESM9">9</a></span>: Figure S7, in red). On the contrary, southern East Asian component (SEAC) exists mainly in typical mainland populations (CDX, CHB, CHS and KHV) (Additional file <span class="InternalRef"><a href="#MOESM9">9</a></span>: Figure S7, in green), and Ryukyuan component (RC) exists mainly in island and peninsula populations (JPRK, JPT and KOR) (Additional file <span class="InternalRef"><a href="#MOESM9">9</a></span>: Figure S7, K = 5~ 7, in yellow). QHM has some Tibetan component (TC) (~ 8.98%) while BMON has very limited TC (~ 0.5%) (Additional file <span class="InternalRef"><a href="#MOESM9">9</a></span>: Figure S7, K = 7, in purple). We could also observe considerable TC contribution in CHB and JPRK (5.2 and 2.6%, respectively) (Additional file <span class="InternalRef"><a href="#MOESM9">9</a></span>: Figure S7, K = 7, in purple).</p>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par46" class="Para">To further assess finer scale population structure and genetic composition of East Asian populations, we run independent STRUCTURE analyses in the ten East Asian populations without non-Asian population samples (Additional file <span class="InternalRef"><a href="#MOESM10">10</a></span>: Figure S8). At K = 3, we observed a clear north/south cline of genetic components. SEAC (in green) constituted the majority (92.7%) of the CDX genome; Mongolian and Tibetan showed distinct composition but both were influenced by NEAC significantly (5.7%~ 15.0%) (Additional file <span class="InternalRef"><a href="#MOESM10">10</a></span>: Figure S8, K = 4, in red). At K = 5, Mongolian had different composition from Tibetan but showed considerable contribution from the Tibetan component (TC) (3.1~ 14.5%), especially in QHM, who migrated to Qinghai-Tibetan Plateau about 500&nbsp;years ago and were reported to have adapted to local environment [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>]. TC was also observed widely in other populations such as Han Chinese and Korean (2.3~ 12.6%) (Additional file <span class="InternalRef"><a href="#MOESM10">10</a></span>: Figure S8, K = 5, in purple). At K = 6, RC was separated into two different components, one (in pink) mainly exists in island populations (JPT and JPRK) and has very little (1.4~ 1.7%) contribution to other populations, and the other (in yellow) exists both in island populations (JPRK and JPT, 39.1~ 66.7%) and northern populations (BMON, CHB and KOR, 4.3~ 41.7%) (Additional file <span class="InternalRef"><a href="#MOESM10">10</a></span>: Figure S8, K = 6).</p>
                  <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par47" class="Para">We confirmed the ancestry inference results with another commonly used method, <em xmlns="" class="EmphasisTypeItalic">ADMIXTURE</em> [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>] (Fig.&nbsp;<span class="InternalRef"><a href="#Fig4">4</a></span>). We ran ADMIXTURE from K = 2 to K = 12 and chose results at K = 5 as the best inference according to <em xmlns="" class="EmphasisTypeItalic">ADMIXTURE</em>’s cross-validation procedure (the best K has the lowest cross-validation error) (Additional file <span class="InternalRef"><a href="#MOESM11">11</a></span>: Figure S9). Similar to the STRUCTURE results, East Asian individuals demonstrated three typical ancestry components which were entirely different from African or European. No strong influence of African ancestry on East Asian populations was detected (3.1% in TIB and no more than 2.4% in any other East Asian populations). However, we observed considerable presence of European ancestry in Mongolian population (15.5~ 16.1%) and some other East Asian populations (up to 6.4%). As shown in Fig. <span class="InternalRef"><a href="#Fig4">4</a></span>, the three dominant ancestral components in East Asian populations are NEAC (in red), SEAC (in green) and RC (in yellow). NEAC is also dominant in Tibetan and Mongolian and uniformly distributes in Han Chinese (27.3 to 39.4%), Japanese (24.5%) and Korean (39.9%). SEAC and RC are frequent in CDX/KHV and JPRK respectively, and both exist in other East Asian populations, while their proportions are generally negatively correlated across populations.<figure xmlns="" class="Figure" id="Fig4"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO4">
                        <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0057-5/MediaObjects/41065_2018_57_Fig4_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0057-5/MediaObjects/41065_2018_57_Fig4_HTML.gif" alt="Fig. 4" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0057-5/MediaObjects/41065_2018_57_Fig4_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0057-5/MediaObjects/41065_2018_57_Fig4_HTML.gif" alt="Fig. 4"></img></noscript></a>
                      </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 4</span>
                          <p xmlns="" class="SimplePara">Admixture analysis of East Asian populations together with world-wide non-Asian populations. Result of K = 5 was shown and each vertical bar represents genome makeup of an sample individual (calculated by ADMIXTURE 1.23 [<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>] and R 2.15.2)</p>
                        </div></figcaption></figure></div>
                </section>
                <section id="Sec12" class="Section3 RenderAsSection3">
                  <h4 class="Heading">F test and D test</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par48" class="Para">We used the F<sub>3</sub> test [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>, <span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>] to detect potential admixture populations and a significant negative F<sub>3</sub> value suggests existence of gene flow and possible admixture pattern. We tried each populations combination (F<sub>3</sub> (population C; population A, population B)) and listed those with negative F<sub>3</sub> values&nbsp;(Additional file <span class="InternalRef"><a href="#MOESM12">12</a></span>: Table S3). We found that to some degree Han Chinese, Japanese, Korean populations each received gene flow from the other two. According to the F<sub>3</sub> results, Han Chinese may have common origin with CDX and KHV, and receive gene flow from northern groups (NEAC) and from islands groups (RC). Japanese has significant JPRK contribution and can be treated as an admixture of Ryukyuan and other East Asian populations (Additional file <span class="InternalRef"><a href="#MOESM12">12</a></span>: Table S3). The Korean population also has NEAC, SEAC and RC, but F<sub>3</sub> results strongly suggest that it received gene flow directly through Han Chinese and Japanese (Additional file <span class="InternalRef"><a href="#MOESM12">12</a></span>: Table S3). Our results showed that Mongolian populations (BMON and QHM) were admixed by European and some East Asian populations (for example, Han Chinese or Japanese, or their common ancestry, Additional file <span class="InternalRef"><a href="#MOESM12">12</a></span>: Table S3), consistent with their known history.</p>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par49" class="Para">We further performed F<sub>4</sub> test [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>, <span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>] to quantitatively estimate the gene flow from surrounding groups to Han Chinese, Japanese and Korean populations, respectively. F<sub>4</sub>(A, B; C, D) denotes the common part of two drift route A to B and C to D (Additional file <span class="InternalRef"><a href="#MOESM13">13</a></span>: Figure S11C). If A and B are two reference populations, C is a target population and O is the outgroup, then F<sub>4</sub>(O, A; B, C) and F<sub>4</sub>(O, B; A, C) have linear relation, which is equal to the gene flow ratio multiplied by common drift route (αl and (1-α)m, Additional file <span class="InternalRef"><a href="#MOESM13">13</a></span>: Figure S11C). Therefore, we can use linear regression method to estimate the gene flow (Additional file <span class="InternalRef"><a href="#MOESM14">14</a></span>: Figure S10). As a result, NEAC contributed 52.2% to CHB, 44.0% to CHS, 19.7% to JPT and 37.5% to KOR; RC contributed 14.3% to CHB, 12.4% to CHS, 52.6% to JPT and 33.8% to KOR; SEAC contributed 33.5% to CHB, 43.6% to CHS, 27.7% to JPT and 28.7% to KOR.</p>
                  <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par50" class="Para">Finally, we applied the D test (4 population test) [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>] to qualitatively estimate the gene flow Between Han Chinese, Japanese and Korean populations (Table <span class="InternalRef"><a href="#Tab3">3</a></span>). In D test, a significant nonzero D values suggest the existence of gene flow (Additional file <span class="InternalRef"><a href="#MOESM13">13</a></span>: Figure S11, more information can be found in Material and Methods part). Significant negative values were observed when D test were applied for comparing gene flow between Han Chinese and Japanese/Korean, indicating greater gene flow between Han Chinese and Korean than that between Han Chinese and Japanese. We further applied D test for detecting gene flow between Japanese/Korean and south/north Han Chinese, the D values were not significantly different from 0, indicating the gene flow between north Han Chinese and Japanese/Korean are almost equal to that between south Han Chinese and Japanese/Korean. These results remain consistent when different outgroups (CEU, YRI) were used in the analysis (Table <span class="InternalRef"><a href="#Tab3">3</a></span>).<figure xmlns="" id="Tab3" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 3</span>
                          <p xmlns="" class="SimplePara">Results of D test</p>
                        </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col></col><col></col><col char="."></col><col char="."></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">popW</p></th><th><p xmlns="" class="SimplePara">popA</p></th><th><p xmlns="" class="SimplePara">popY</p></th><th><p xmlns="" class="SimplePara">popZ</p></th><th char="."><p xmlns="" class="SimplePara">D value</p></th><th char="."><p xmlns="" class="SimplePara">Z score</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">CHB</p></td><td><p xmlns="" class="SimplePara">YRI</p></td><td><p xmlns="" class="SimplePara">JPT</p></td><td><p xmlns="" class="SimplePara">KOR</p></td><td char="."><p xmlns="" class="SimplePara">−0.0055</p></td><td char="."><p xmlns="" class="SimplePara">−5.061</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">CHS</p></td><td><p xmlns="" class="SimplePara">YRI</p></td><td><p xmlns="" class="SimplePara">JPT</p></td><td><p xmlns="" class="SimplePara">KOR</p></td><td char="."><p xmlns="" class="SimplePara">−0.0054</p></td><td char="."><p xmlns="" class="SimplePara">−4.865</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">JPT</p></td><td><p xmlns="" class="SimplePara">YRI</p></td><td><p xmlns="" class="SimplePara">CHB</p></td><td><p xmlns="" class="SimplePara">CHS</p></td><td char="."><p xmlns="" class="SimplePara">0.0014</p></td><td char="."><p xmlns="" class="SimplePara">1.515</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">KOR</p></td><td><p xmlns="" class="SimplePara">YRI</p></td><td><p xmlns="" class="SimplePara">CHB</p></td><td><p xmlns="" class="SimplePara">CHS</p></td><td char="."><p xmlns="" class="SimplePara">0.0016</p></td><td char="."><p xmlns="" class="SimplePara">1.649</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">CHB</p></td><td><p xmlns="" class="SimplePara">CEU</p></td><td><p xmlns="" class="SimplePara">JPT</p></td><td><p xmlns="" class="SimplePara">KOR</p></td><td char="."><p xmlns="" class="SimplePara">−0.0062</p></td><td char="."><p xmlns="" class="SimplePara">−6.748</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">CHS</p></td><td><p xmlns="" class="SimplePara">CEU</p></td><td><p xmlns="" class="SimplePara">JPT</p></td><td><p xmlns="" class="SimplePara">KOR</p></td><td char="."><p xmlns="" class="SimplePara">−0.0060</p></td><td char="."><p xmlns="" class="SimplePara">−6.596</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">JPT</p></td><td><p xmlns="" class="SimplePara">CEU</p></td><td><p xmlns="" class="SimplePara">CHB</p></td><td><p xmlns="" class="SimplePara">CHS</p></td><td char="."><p xmlns="" class="SimplePara">−0.0005</p></td><td char="."><p xmlns="" class="SimplePara">−0.595</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">KOR</p></td><td><p xmlns="" class="SimplePara">CEU</p></td><td><p xmlns="" class="SimplePara">CHB</p></td><td><p xmlns="" class="SimplePara">CHS</p></td><td char="."><p xmlns="" class="SimplePara">−0.0004</p></td><td char="."><p xmlns="" class="SimplePara">−0.446</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                        <p xmlns="" class="SimplePara">Note: Population A (popA) is outgroup, population W (popW) is a reference population, population Y (popY) and population Z (popZ) are two potential target populations. If D value is significantly positive, it suggests that the gene flow between population W and population Y was greater than that between population W and population Z. On the contrary, if D value is significantly negative, it means the gene flow between population W and population Y was smaller than that between population W and population Z. If D value is not significantly positive or negative, it suggests that the gene flow between population W and population Y are equal to that between population W and population Z, or there is no gene flow between population W and population Y or between population W and population Z</p>
                      </div></figure></div>
                </section>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec13" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Discussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Discussion">Discussion</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Discussion" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par51" class="Para">In this project, we conducted a genome-wide study of three East Asian populations based on genome-wide high-density SNP data. Our results showed that the three&nbsp;East Asian populations, Han Chinese, Japanese and Korean, although they resemble each other in appearance, have distinguishing genetic make-up and are differentiated apparently on genomic level. In this respect, we suggest the three East Asian ethnic groups should be treated as independent populations in the future studies rather than as a single group, particularly in evolutionary studies or medical studies where population structure matters. In addition to global differentiation, the AIMs we screened also indicated considerable differentiation of local genomic regions among the three populations. For example, several highly differentiated SNPs (AIMs) are enriched in the <em xmlns="" class="EmphasisTypeItalic">CD46</em> gene (on chromosome 1q32), which is a type I membrane protein and is a regulatory part of the complement system [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>, <span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>]. These highly differentiated genomic regions could be due to regional adaptation to some pathogens, although further studies are necessary to validate these signals and confirm this conclusion. These results further suggest the importance to consider population difference when medical studies are conducted in the three East Asian ethnic groups.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par52" class="Para">It is obvious that the genetic difference among the three East Asian groups initially resulted from population divergence due to pre-historical or historical migrations. Subsequently, different geographical locations where the three populations are located, mainland of China, Korean Peninsular and Japanese archipelago, respectively, apparently facilitated population differentiation due to physical isolation and independent genetic drift. Our estimations of population divergence time among the three groups, 1.2~ 3.6 KYA, are largely consistent with known history of the three populations and those related. However, considering that recent admixture could have reduced genetic difference between populations, it is likely the divergence time was underestimated.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par53" class="Para">We detected substantial gene flow among the three populations and also from the surrounding populations. For example, based on our analysis with the F<sub>3</sub> test, Korean received gene flow from Han Chinese and Japanese, and gene flow also happened between Han Chinese and Japanese (Additional file <span class="InternalRef"><a href="#MOESM12">12</a></span>: Table S3). These gene flows are expected to have reduced the genetic differentiation between the three ethnic groups. On the other hand, we also detected considerable gene flow from surrounding populations to the three populations studied. For instance, an ancestral population represented by Ryukyuan have contributed greater to Japanese than to Han Chinese, while southern ethnic group like Dai have contributed more to continent populations than to island and peninsula populations. Contrary to the gene flow among the three populations, these gene flows from surrounding populations are expected to have increased genetic difference among the three populations if they occurred independently and from different source populations. According to our results, the major source of gene flow to the three ethnic groups were substantially different, for example, the major source of gene flow to Han Chinese was from southern ethnic groups, the major source of gene flow to Japanese was from southern islands, and the major source of gene flow to Korean were from both mainland and islands. Therefore, those gene flows might have significantly contributed to further genetic differentiation of the three populations.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par54" class="Para">The three populations have similar but not identical demographical history; they all experience a strong population expansion in the last 20,000&nbsp;years. However, according to different geographic distribution, their effective population size and population&nbsp;expansion are different. North Han Chinese has greater N<sub>e</sub> than south Han Chinese in early time. Japanese and Korean have similar N<sub>e</sub> in most of the history, while Korean population expanded much faster than Japanese in recent thousands of years, suggested a higher potential for population expansion in peninsula than in islands. Our estimation showed that Han Chinese have much greater expansion speed than Japanese and Korean, suggested continental environment has even higher potential for population expansion than peninsula or islands. However, the estimated N<sub>e</sub> increase in East Asian populations could be, at least partially, explained by recent population admixture which is expected to increase population genetic diversity thus increase N<sub>e</sub>.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par55" class="Para">Finally, as we briefly mentioned above, natural selection might have substantially contributed to the genetic diversity of the populations. We identified several genomic regions with significant differentiation between Han Chinese, Japanese and Korean. For example, most highly differentiated SNPs enriched in the genes that related to neurological system process and cell-cell signal transmission. There are also significant selective signals in detected in EGF-like domain and Collagen-like domain. The physiological significance and evolutionary importance of these selective signals needed more study.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec14" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Conclusion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Conclusion">Conclusion</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Conclusion" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par56" class="Para">In summary, the genetic structures of the present-day Han Chinese, Japanese and Korean people were shaped jointly by initial population divergence, geographical isolation, subsequent gene flow and possibly regional natural selection. Our analysis of the genome-wide data of these populations and their surrounding neighbors should advance our understanding of population history and mechanism of human genetic diversity in East Asia.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec15" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Methods" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Methods">Methods</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Methods" data-test="collapse-section"><section id="Sec16" class="Section2 RenderAsSection2">
                <h3 class="Heading">Population and samples</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par57" class="Para">Peripheral blood samples of 100 Koreans unrelated individuals were collected from South Korea. Each individual was the offspring of a non-consanguineous marriage of members of the same nationality within three generations. Informed consents were acquired from the participants. All procedures followed were in accordance with the ethical standards of the Responsible Committee on Human Experimentation (approved by the Biomedical Research Ethics Committee of the Shanghai Institutes for Biological Sciences, No. ER-SIBS-261408) and the Helsinki Declaration of 1975, as revised in 2000. In addition, genome-wide single nucleotide polymorphisms (SNP) data of 182 unrelated Han Chinese, and 90 Japanese individuals were collected from the HapMap Project (International HapMap, 2003) and the 1000 Genomes Project [<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>], and comparisons were made with 663 individuals (630 individuals through Quality Control) representing 8 populations worldwide from the HapMap Project, the 1000 Genomes Project and the PanAsia SNP Project [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>] (Fig. <span class="InternalRef"><a href="#Fig1">1a</a></span>, Additional file <span class="InternalRef"><a href="#MOESM3">3</a></span>: Table S1).</p>
              </section><section id="Sec17" class="Section2 RenderAsSection2">
                <h3 class="Heading">Genotyping and data quality control</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par58" class="Para">A set of 934,968 loci were genotyped in 100 Korean samples with Affymetrix Genome-wide human SNP Array 6.0. *.CEL files containing raw intensity data were analyzed with Birdsuite version 1.5.3 [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>]. Only autosomal SNPs with missing rate less than 0.05 were employed for the downstream analyses. For the purposes of this study, only SNPs with Reference Sequence (RS) numbers and vendor-specified strands were used in combining data. For different purposes of analysis, several combined data sets were generated. The first two data sets (dataset 1 and dataset 2) contains 13,576 SNPs for 1002 individuals from all the twelve populations and 733,113 SNPs for 439 individuals from six East Asia populations (BMON, CHB, CHD, JPT, KOR and QHM). These datasets were used to estimate ASD. And dataset 1 was also prepared for principal component analysis [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>], STRUCTURE [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>] and Admixture [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>] analyses, and estimation of gene flow, so that SNPs in high linkage disequilibrium (LD) have been removed from this data set. The third dataset (dataset 3) contains 44,549 SNPs for 596 individuals representing nine East Asian populations (except JPRK samples), which was used for population structure analyses of East Asian populations only. In the fourth dataset (dataset 4), we kept all the SNPs genotyped in all the samples respectively, and used for estimating F<sub>ST</sub> and screening AIMs [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>]. The last dataset (dataset 5) was based on the dataset 4 but phased particularly for calculating LD.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par59" class="Para">To control for potential batch effect during our data integration, we corrected allele frequency of BMON and QHM [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>] by comparing genotypes of the identical CHB and JPT individuals between the data set of [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>] and our own data. Because CHB and JPT were also included in the same data set together with BMON and QHM in [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>]. To avoid any possible systematic bias introduced during data integration, we did SNP re-calling for HapMap individuals from raw Affymetrix intensity data (.CEL files).</p>
              </section><section id="Sec18" class="Section2 RenderAsSection2">
                <h3 class="Heading">Population structure analysis</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par60" class="Para">Genetic difference between populations was measured by F<sub>ST</sub> which was calculated following an unbiased estimate [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>]. Principal component analysis (PCA) was performed at individual level using EIGENSOFT [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>]. The map was drawn by R (version 2.15.2) based on the information provided by <span class="ExternalRef"><a href="http://www.mapsofworld.com/"><span class="RefSource">http://www.mapsofworld.com/</span></a></span>. We used an allele sharing distance (ASD) as a measure of genetic distance between individuals, and a 1002 × 1002 and a 439 × 439 inter-individual genetic distance matrixes were generated according to genotypes of 13,576 and 733,113 autosomal SNPs respectively (dataset 1&amp;2). The tree of individuals was reconstructed based on ASD distance and using Neighbor-Joining algorithm [<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>] with the Molecular Evolutionary Genetics Analysis software package (MEGA version 4.0) [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>]. Trees of populations as well as components were reconstructed using maximum likelihood method [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>] with CONTML program in PHYLIP package (version 3.695) [<span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>]. K-mean clustering analysis was done with R software. Ancestry of each person was inferred using a Bayesian cluster analysis as implemented in the STRUCTURE program [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>, <span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>]. We ran STRUCTURE from <em xmlns="" class="EmphasisTypeItalic">K</em> = 2 to <em xmlns="" class="EmphasisTypeItalic">K</em> = 7 and repeated 5 times for each single <em xmlns="" class="EmphasisTypeItalic">K</em>. All STRUCTURE runs used 15,000 iterations after a burn-in of length 15,000, with the admixture model and assuming that allele frequencies were correlated [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>]. Admixture [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>] was also employed for the same purpose.</p>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par61" class="Para">We screened Ancestry Informative Markers (AIMs) [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>] to distinguish Han Chinese, Japanese and Korean. These AIMs were selected by the following three criteria: 1) allele frequency highly differentiated between each two populations (measured by F<sub>ST</sub>); 2) inter-marker distance larger than 500 Kb; 3) Matthews correlation coefficient (MCC) of a panel of AIMs based on PC1 not less than 1.0. MCC is defined as the follows:<div id="Equ1" class="Equation EquationMathjax"><div class="EquationContent">$$ \mathrm{MCC}=\frac{TP\times TN- FP\times FN}{\sqrt{\left( TP+ FP\right)\left( TP+ FN\right)\left( TN+ FP\right)\left( TN+ FN\right)}} $$</div> <div class="EquationNumber">(5)</div></div>where TP is the number of true positives, TN is the number of true negatives, FP is the number of false positives and FN is the number of false negatives [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>]).</div>
              </section><section id="Sec19" class="Section2 RenderAsSection2">
                <h3 class="Heading">Estimation of effective population size and divergence time</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par62" class="Para">We first phased the genotype data with BEAGLE [<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>]. In order to improve the accuracy of the haplotype inference (phasing), we combined those datasets from the same source and did further estimation independently.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par63" class="Para">Then we calculated pairwise linkage disequilibrium (r<sup>2</sup>) of each SNPs pair with genetic distance less than 0.25&nbsp;cM in each population. And effective population size (N<sub>e</sub>) of <em xmlns="" class="EmphasisTypeItalic">t</em> generations ago (<em xmlns="" class="EmphasisTypeItalic">t</em> = 1/2<em xmlns="" class="EmphasisTypeItalic">c</em> (2) [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>]) can be estimated for each population in each distance category as N<sub>e</sub> = 1/(4<em xmlns="" class="EmphasisTypeItalic">c</em>)*[(1/r<sub>LD</sub><sup>2</sup>)-2] (3) [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>], where <em xmlns="" class="EmphasisTypeItalic">c</em> is the genetic distance. All r<sub>LD</sub><sup>2</sup> are adjust as (r<sub>LD</sub><sup>2</sup>–1/n), where n is sample size (number of chromosomes) [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>]. Divergence time (T<sub>F</sub>) can be estimated by 2N<sub>e</sub>F<sub>ST</sub> (4) [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>]. Here, F<sub>ST</sub> values were still calculated following an unbiased estimate. N<sub>e</sub> was calculated as the average of the harmonic means of the relevant recombination distance categories. We used distances categories 0.01&nbsp;cM to 0.25&nbsp;cM (corresponding to 200 to 5000 generations ago) to estimate inter-population N<sub>e</sub> [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>].</p>
              </section><section id="Sec20" class="Section2 RenderAsSection2">
                <h3 class="Heading">Gene flow estimation</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par64" class="Para">We used D test and F test to detect and estimate gene flow [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>]. All the variations and Z scores are calculated by delete-m jackknife [<span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>].</p>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par65" class="Para">In D test, D value indicates the difference of the two allele patterns ABBA and BABA (Additional file <span class="InternalRef"><a href="#MOESM13">13</a></span>: Figure S11A). A significant positive or negative D value (|Z| &gt; 2.58, <em xmlns="" class="EmphasisTypeItalic">p</em> &lt; 0.01) suggests the existence of gene flow either from W to Y or from W to Z. X is out group. In population genetics, D is calculated by<div id="Equ2" class="Equation EquationMathjax"><div class="EquationContent">$$ \widehat{N}{um}_i/\widehat{D}{en}_i $$</div> <div class="EquationNumber">(6)</div></div></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par66" class="Para">where<div id="Equ3" class="Equation EquationMathjax"><div class="EquationContent">$$ \widehat{N}{um}_i=\left(w-x\right)\left(y-z\right) $$</div> <div class="EquationNumber">(7)</div></div><div id="Equ4" class="Equation EquationMathjax"><div class="EquationContent">$$ \widehat{D}{en}_i=\left(w+x-2 wx\right)\left(y+z-2 yz\right) $$</div> <div class="EquationNumber">(8)</div></div></div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par67" class="Para">w, y, x and z are sample allele frequency of population W, Y, X, Z.</p>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par68" class="Para">In F test, F<sub>2</sub> value is calculated by<div id="Equ5" class="Equation EquationMathjax"><div class="EquationContent">$$ {F}_2\left(A,B\right)=E\left[{\left({a}^{\hbox{'}}-{b}^{\hbox{'}}\right)}^2\right] $$</div> <div class="EquationNumber">(9)</div></div></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par69" class="Para">(a and b are sample allele frequency of population A and B) and can be looked as the drift on edge A to B in a phylogeny tree. And<div id="Equ6" class="Equation EquationMathjax"><div class="EquationContent">$$ {F}_3\left(C:A,B\right)=E\left[\left({c}^{\hbox{'}}-{a}^{\hbox{'}}\right)\left({c}^{\hbox{'}}-{b}^{\hbox{'}}\right)\right] $$</div> <div class="EquationNumber">(10)</div></div>denotes the drift on the common path of edge C to A and C to B (D<sub>x</sub>, Additional file <span class="InternalRef"><a href="#MOESM13">13</a></span>: Figure S11B). If there is no admixture, F<sub>3</sub> value should be positive. A significant negative F<sub>3</sub> value suggests the existence of gene flow [<span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>, <span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>, <span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>]. Similarly,<div id="Equ7" class="Equation EquationMathjax"><div class="EquationContent">$$ {F}_4\left(A,B:C,D\right)=E\left[\left({a}^{\hbox{'}}-{b}^{\hbox{'}}\right)\left({c}^{\hbox{'}}-{d}^{\hbox{'}}\right)\right] $$</div> <div class="EquationNumber">(11)</div></div></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par70" class="Para">If there was gene flow in history, the drift from one population to another would go through more than one path in the phylogeny tree; the ratio of drift along each path is decided by the ratio of gene flow. If there were gene flow from population A to population C (with a ratio α) and from population B to population C (with a ratio 1-α), then<div id="Equ8" class="Equation EquationMathjax"><div class="EquationContent">$$ {F}_4\left(A,O;C,B\right)=\alpha l $$</div> <div class="EquationNumber">(12)</div></div><div id="Equ9" class="Equation EquationMathjax"><div class="EquationContent">$$ {F}_4\left(B,O;C,A\right)=\left(1-\alpha \right)m $$</div> <div class="EquationNumber">(13)</div></div></div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par71" class="Para">Where <em xmlns="" class="EmphasisTypeItalic">l</em> and <em xmlns="" class="EmphasisTypeItalic">m</em> are unknown, but if there is a series of population C that are admixed with ancestry from both population A and population B (A’ and B′) with different ratio, we can use a linear regression to eliminate these parameters, so that we can calculation each gene flow ratio α [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>].</p>
              </section></div></section>
          <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AbbreviationGroup Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Abbreviations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Abbreviations">Abbreviations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Abbreviations"><div class="DefinitionList">
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>AIMs:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par4" class="Para">Ancestry informative markers</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>ASD:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par5" class="Para">Allele sharing distance</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>BMON:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par6" class="Para">Buryat Mongolian</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>CDX:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par7" class="Para">Chinese Dai in Xishuangbanna</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>CEU:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par8" class="Para">Northern Europeans from Utah</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>CHB:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par9" class="Para">North Han Chinese</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>CHS:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par10" class="Para">South Han Chinese</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>IBD:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par11" class="Para">Isolation by distance</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>JPRK:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par12" class="Para">Ryukyuan</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>JPT:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par13" class="Para">Japanese in Tokyo</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>KHV:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par14" class="Para">Kinh in Ho Chi Minh City, Vietnam</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>KOR:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par15" class="Para">South Korean</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>LD:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par16" class="Para">Linkage disequilibrium</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>MCC:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par17" class="Para">Matthews correlation coefficient</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>ML tree:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par18" class="Para">Maximum likelihood tree</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>NEAC:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par19" class="Para">Northern East Asian component</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>NJ tree:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par20" class="Para">Neighbor-joining tree</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>PCA:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par21" class="Para">Principal components analysis</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>QHM:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par22" class="Para">Deedu (DU) Mongolians in Qinghai-Tibetan Plateau</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>RC:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par23" class="Para">Ryukyuan component</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>RS:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par24" class="Para">Reference Sequence</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>SEAC:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par25" class="Para">Southern East Asian component</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>TC:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par26" class="Para">Tibetan component</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>TIB:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par27" class="Para">Tibetan</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>YRI:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par28" class="Para">Yoruba in Ibadan, Nigeria</p>
                  </dd></dl>
              </div></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments"><h3 xmlns="" class="Heading">Acknowledgements</h3><p xmlns="" class="SimplePara">The authors thank Drs. Charleston Chiang, Lian Deng, Pengfei Qin, Minxian Wang, Jing Li, and Xiong Yang for their helpful discussion and comments.</p><div id="FPar1" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Funding</h3>
                <p id="Par72" class="Para">These studies were supported by the National Key Research and Development Program (2016YFC0906403), the National Natural Science Foundation of China (NSFC) grant (91731303, 31771388, and 31711530221), the National Science Fund for Distinguished Young Scholars (31525014), the Strategic Priority Research Program (XDB13040100) and Key Research Program of Frontier Sciences (QYZDJ-SSW-SYS009) of the Chinese Academy of Sciences (CAS), and the Program of Shanghai Academic Research Leader (16XD1404700). S.X. is Max-Planck Independent Research Group Leader and member of CAS Youth Innovation Promotion Association. S.X. also gratefully acknowledges the support of the National Program for Top-notch Young Innovative Talents of the “<em xmlns="" class="EmphasisTypeItalic">Wanren Jihua</em>” Project. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p>
              </div><div id="FPar2" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Availability of data and materials</h3>
                <p id="Par73" class="Para">The following additional data are available with the online version of this paper. Supplementary Tables S1-S3 and Figures S1-S11 are available online.</p>
              </div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="FormalPara FormalParaRenderingStyle1"><h3 xmlns="" class="Heading">Authors’ contributions</h3><p xmlns="" class="SimplePara">SX conceived and designed the study. YJ contributed partly to sample collection and genotype data. SX contributed reagents and materials. YW collected data and performed the analyses, with contribution from DL. SX and YW interpreted the data and wrote the paper. All authors have read and approved the manuscript. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Ethics">
              <div id="FPar3" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Ethics approval and consent to participate</h3>
                <p id="Par74" class="Para">All samples were collected with informed consent and approved by the Biomedical Research Ethics Committee of Shanghai Institutes for Biological Sciences. Prior to genotyping and analysis, all samples were stripped of personal identifiers (if any existed). All procedures were in accordance with the ethical standards of the Responsible Committee on Human Experimentation (approved by the Biomedical Research Ethics Committee of Shanghai Institutes for Biological Sciences, No. ER-SIBS-261408) and the Helsinki Declaration of 1975, as revised in 2000.</p>
              </div>
              <div id="FPar4" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Consent for publication</h3>
                <p id="Par75" class="Para">Not applicable.</p>
              </div>
              <div id="FPar5" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Competing interests</h3>
                <p id="Par76" class="Para">All authors declare that they have no competing interests.</p>
              </div>
              <div id="FPar6" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Publisher’s Note</h3>
                <p id="Par77" class="Para">Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
              </div>
            </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="License">
                <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Open Access</strong>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/4.0/"><span class="RefSource">http://creativecommons.org/licenses/by/4.0/</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/publicdomain/zero/1.0/"><span class="RefSource">http://creativecommons.org/publicdomain/zero/1.0/</span></a></span>) applies to the data made available in this article, unless otherwise stated.</p>
              </div><aside class="Appendix" id="App1">
              
                <h3 class="Heading">Additional files</h3><div id="widgetContainer" data-test="figshare-container"></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par78" class="Para">
                  <div class="MediaObject" id="MOESM1">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0057-5/MediaObjects/41065_2018_57_MOESM1_ESM.pdf"><span class="caption-number">Additional file 1:</span></a>
                          <span xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Figure S1.</strong> A heat map of pair-wise F<sub xmlns="http://www.w3.org/1999/xhtml">ST</sub>. A warmer (red) color means a larger F<sub xmlns="http://www.w3.org/1999/xhtml">ST</sub> value (generated by R 2.15.2). (PDF 6 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM2">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0057-5/MediaObjects/41065_2018_57_MOESM2_ESM.docx"><span class="caption-number">Additional file 2:</span></a>
                          <span xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Table S2.</strong> Global F<sub xmlns="http://www.w3.org/1999/xhtml">ST</sub> values between East Asian populations and world-wide groups. (DOCX 16 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM3">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0057-5/MediaObjects/41065_2018_57_MOESM3_ESM.docx"><span class="caption-number">Additional file 3:</span></a>
                          <span xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Table S1.</strong> Genotype data used in this research. (DOCX 21 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM4">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0057-5/MediaObjects/41065_2018_57_MOESM4_ESM.pdf"><span class="caption-number">Additional file 4:</span></a>
                          <span xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Figure S2.</strong> Individual level Neighbor-Joining Tree. (A) The NJ tree was constructed according pair-wise genotyping difference; (B) Individual NJ Tree of Han Chinese, Japanese, Korean and Mongolian individuals (generated by MEGA version 4.0). (PDF 6971 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM5">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0057-5/MediaObjects/41065_2018_57_MOESM5_ESM.pdf"><span class="caption-number">Additional file 5:</span></a>
                          <span xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Figure S3.</strong> Principle component analysis (PCA). (A~B) PCA results of East Asian groups with CEU and YRI. (C) PCA result of groups within East Asia populations, excluding JPRK individuals for a higher marker density. (D) PCA result of four East Asian populations, including Han Chinese, Japanese and Koreans samples. (PDF 2458 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM6">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0057-5/MediaObjects/41065_2018_57_MOESM6_ESM.pdf"><span class="caption-number">Additional file 6:</span></a>
                          <span xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Figure S4.</strong>&nbsp;K-means Cluster of PCA results. The results of Figure S3D were used to conduct the K-means cluster. Individuals that cluster together according to the top 10 PCs would be painted by the same color. Results of K = 3 and K = 4 are shown (generated by R 2.15.2). (PDF 116 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM7">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0057-5/MediaObjects/41065_2018_57_MOESM7_ESM.pdf"><span class="caption-number">Additional file 7:</span></a>
                          <span xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Figure S5.</strong> Matthews correlation coefficient (MCC) values increase when more AIMs are used to distinguish individuals. X axis is number of SNPs and Y axis is MCC values. (A) CHB-CHS; (B) CHB-JPT; (C) CHB-KOR; (D) CHS-JPT; (E) CHS-KOR; (F) JPT-KOR (generated by Microsoft Excel 2010). (PDF 99 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM8">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0057-5/MediaObjects/41065_2018_57_MOESM8_ESM.pdf"><span class="caption-number">Additional file 8:</span></a>
                          <span xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Figure S6.</strong> Effective population size varied along time. Results from 10,000 generations ago (GA) to 200 GA are shown (generated by Microsoft Office 2010). (PDF 112 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM9">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0057-5/MediaObjects/41065_2018_57_MOESM9_ESM.pdf"><span class="caption-number">Additional file 9:</span></a>
                          <span xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Figure S7.</strong> STRUCTURE analysis of East Asian samples with worldwide populations. Results from K = 3 to K = 7 are shown. Each vertical bar represents an individual and each color stands for a genetic component (generated by R 2.15.2). (PDF 141&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM10">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0057-5/MediaObjects/41065_2018_57_MOESM10_ESM.pdf"><span class="caption-number">Additional file 10:</span></a>
                          <span xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Figure S8.</strong> STRUCTURE Analysis of ten East Asian populations. Results from K = 2 to K = 6 are shown. Each vertical bar represents an individual and each color stands for a genetic component (generated by R 2.15.2). (PDF 152&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM11">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0057-5/MediaObjects/41065_2018_57_MOESM11_ESM.pdf"><span class="caption-number">Additional file 11:</span></a>
                          <span xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Figure S9.</strong> Cross-validation (CV) plot for the Admixture analysis. Result of K = 3 to K = 7 are shown. (PDF 158 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM12">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0057-5/MediaObjects/41065_2018_57_MOESM12_ESM.docx"><span class="caption-number">Additional file 12:</span></a>
                          <span xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Table S3.</strong> Part of F<sub xmlns="http://www.w3.org/1999/xhtml">3</sub> test results. (DOCX 18 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM13">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0057-5/MediaObjects/41065_2018_57_MOESM13_ESM.pdf"><span class="caption-number">Additional file 13:</span></a>
                          <span xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Figure S11.</strong> Models used in gene flow study. (A) Model of D test. X is an out group, we put CEU or YRI in X; W is the contributor while Y and Z are the receivers. By comparing the two allelic patterns BABA and ABBA, we can infer the relationship of scale of gene flow between W to Y and W to Z. (PDF 30 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM14">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0057-5/MediaObjects/41065_2018_57_MOESM14_ESM.pdf"><span class="caption-number">Additional file 14:</span></a>
                          <span xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Figure S10.</strong> Results of linear regression in F<sub xmlns="http://www.w3.org/1999/xhtml">4</sub> test. For each group (Outgroup O, donate population A, donate population B, target population X), X axis is F<sub xmlns="http://www.w3.org/1999/xhtml">4</sub>(B, O; X, A), Y axis is F<sub xmlns="http://www.w3.org/1999/xhtml">4</sub>(A, O; X, B). (A) Outgroup is YRI or CEU, donate populations are TIB and CDX, target population is Han Chinese or KOR; (B) Outgroup is YRI or CEU, donate populations are CDX and JPRK, target population is Han Chinese or JPT or KOR; (C) Outgroup is YRI or CEU, donate populations are TIB and JPRK, target population is JPT or KOR (generated by Microsoft Excel 2010). (PDF 226 kb)</span>
                        </div></div>
                  </div>
                </div>
              
            </aside></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authors’ Affiliations</a></h2><div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1"><span class="AffiliationNumber">(1)</span><div class="AffiliationText">Chinese Academy of Sciences (CAS) Key Laboratory of Computational Biology, Max Planck Independent Research Group on Population Genomics, CAS-MPG Partner Institute for Computational Biology (PICB), Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, Shanghai, 200031, China</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff2"><span class="AffiliationNumber">(2)</span><div class="AffiliationText">University of Chinese Academy of Sciences, Beijing, 100049, China</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff3"><span class="AffiliationNumber">(3)</span><div class="AffiliationText">Integrated Research Center for Genome Polymorphism, Department of Microbiology, The Catholic University Medical College, Seoul, Socho-gu, 137-701, South Korea</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff4"><span class="AffiliationNumber">(4)</span><div class="AffiliationText">School of Life Science and Technology ShanghaiTech University, Shanghai, 201210, China</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff5"><span class="AffiliationNumber">(5)</span><div class="AffiliationText">Center for Excellence in Animal Evolution and Genetics, Chinese Academy of Sciences, Kunming, 650223, China</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff6"><span class="AffiliationNumber">(6)</span><div class="AffiliationText">Collaborative Innovation Center of Genetics and Development, Shanghai, 200438, China</div></div></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper"><li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Fischer SR (2004). A History of Writing.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Fischer%20SR%20%282004%29.%20A%20History%20of%20Writing."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Cai Z, Camp NJ, Cannon-Albright L, Thomas A. Identification of regions of positive selection using shared genomic segment analysis. Eur J Hum Genet. 2011;19(6):667–71.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/ejhg.2010.257"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21304558"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3110045"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Identification%20of%20regions%20of%20positive%20selection%20using%20shared%20genomic%20segment%20analysis&amp;author=Z.%20Cai&amp;author=NJ.%20Camp&amp;author=L.%20Cannon-Albright&amp;author=A.%20Thomas&amp;journal=Eur%20J%20Hum%20Genet&amp;volume=19&amp;issue=6&amp;pages=667-671&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">International HapMap C. The international HapMap project. Nature. 2003;426(6968):789–96.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nature02168"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20international%20HapMap%20project&amp;journal=Nature&amp;volume=426&amp;issue=6968&amp;pages=789-796&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Genomes Project C, Abecasis GR, Altshuler D, Auton A, Brooks LD, Durbin RM, et al. A map of human genome variation from population-scale sequencing. Nature. 2010;467(7319):1061–73.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nature09534"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20map%20of%20human%20genome%20variation%20from%20population-scale%20sequencing&amp;author=C.%20Genomes%20Project&amp;author=GR.%20Abecasis&amp;author=D.%20Altshuler&amp;author=A.%20Auton&amp;author=LD.%20Brooks&amp;author=RM.%20Durbin&amp;journal=Nature&amp;volume=467&amp;issue=7319&amp;pages=1061-1073&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">HP-AS C, Abdulla MA, Ahmed I, Assawamakin A, Bhak J, Brahmachari SK, et al. Mapping human genetic diversity in Asia. Science. 2009;326(5959):1541–5.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1126/science.1177074"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Mapping%20human%20genetic%20diversity%20in%20Asia&amp;author=C.%20HP-AS&amp;author=MA.%20Abdulla&amp;author=I.%20Ahmed&amp;author=A.%20Assawamakin&amp;author=J.%20Bhak&amp;author=SK.%20Brahmachari&amp;journal=Science&amp;volume=326&amp;issue=5959&amp;pages=1541-1545&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Xing J, Wuren T, Simonson TS, Watkins WS, Witherspoon DJ, Wu W, et al. Genomic analysis of natural selection and phenotypic variation in high-altitude mongolians. PLoS Genet. 2013;9(7):e1003634.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pgen.1003634"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23874230"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3715426"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genomic%20analysis%20of%20natural%20selection%20and%20phenotypic%20variation%20in%20high-altitude%20mongolians&amp;author=J.%20Xing&amp;author=T.%20Wuren&amp;author=TS.%20Simonson&amp;author=WS.%20Watkins&amp;author=DJ.%20Witherspoon&amp;author=W.%20Wu&amp;journal=PLoS%20Genet&amp;volume=9&amp;issue=7&amp;pages=e1003634&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Xu SH, Yin XY, Li SL, Jin WF, Lou HY, Yang L, et al. Genomic dissection of population substructure of Han Chinese and its implication in association studies. Am J Hum Genet. 2009;85(6):762–74.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.ajhg.2009.10.015"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19944404"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2790582"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genomic%20dissection%20of%20population%20substructure%20of%20Han%20Chinese%20and%20its%20implication%20in%20association%20studies&amp;author=SH.%20Xu&amp;author=XY.%20Yin&amp;author=SL.%20Li&amp;author=WF.%20Jin&amp;author=HY.%20Lou&amp;author=L.%20Yang&amp;journal=Am%20J%20Hum%20Genet&amp;volume=85&amp;issue=6&amp;pages=762-774&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Qin P, Li Z, Jin W, Lu D, Lou H, Shen J, et al. A panel of ancestry informative markers to estimate and correct potential effects of population stratification in Han Chinese. EJHG: European journal of human genetics; 2013.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20panel%20of%20ancestry%20informative%20markers%20to%20estimate%20and%20correct%20potential%20effects%20of%20population%20stratification%20in%20Han%20Chinese&amp;author=P.%20Qin&amp;author=Z.%20Li&amp;author=W.%20Jin&amp;author=D.%20Lu&amp;author=H.%20Lou&amp;author=J.%20Shen&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">McEvoy BP, Powell JE, Goddard ME, Visscher PM. Human population dispersal "out of Africa" estimated from linkage disequilibrium and allele frequencies of SNPs. Genome Res. 2011;21(6):821–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1101/gr.119636.110"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21518737"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3106315"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Human%20population%20dispersal%20%22out%20of%20Africa%22%20estimated%20from%20linkage%20disequilibrium%20and%20allele%20frequencies%20of%20SNPs&amp;author=BP.%20McEvoy&amp;author=JE.%20Powell&amp;author=ME.%20Goddard&amp;author=PM.%20Visscher&amp;journal=Genome%20Res&amp;volume=21&amp;issue=6&amp;pages=821-829&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Pritchard JK, Stephens M, Donnelly P. Inference of population structure using multilocus genotype data. Genetics. 2000;155(2):945–59.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10835412"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1461096"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data&amp;author=JK.%20Pritchard&amp;author=M.%20Stephens&amp;author=P.%20Donnelly&amp;journal=Genetics&amp;volume=155&amp;issue=2&amp;pages=945-959&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Alexander DH, Novembre J, Lange K. Fast model-based estimation of ancestry in unrelated individuals. Genome Res. 2009;19(9):1655–64.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1101/gr.094052.109"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19648217"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2752134"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Fast%20model-based%20estimation%20of%20ancestry%20in%20unrelated%20individuals&amp;author=DH.%20Alexander&amp;author=J.%20Novembre&amp;author=K.%20Lange&amp;journal=Genome%20Res&amp;volume=19&amp;issue=9&amp;pages=1655-1664&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Patterson N, Moorjani P, Luo Y, Mallick S, Rohland N, Zhan Y, et al. Ancient admixture in human history. Genetics. 2012;192(3):1065–93.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1534/genetics.112.145037"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22960212"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522152"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Ancient%20admixture%20in%20human%20history&amp;author=N.%20Patterson&amp;author=P.%20Moorjani&amp;author=Y.%20Luo&amp;author=S.%20Mallick&amp;author=N.%20Rohland&amp;author=Y.%20Zhan&amp;journal=Genetics&amp;volume=192&amp;issue=3&amp;pages=1065-1093&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Reich D, Thangaraj K, Patterson N, Price AL, Singh L. Reconstructing Indian population history. Nature. 2009;461(7263):489–94.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nature08365"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19779445"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2842210"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Reconstructing%20Indian%20population%20history&amp;author=D.%20Reich&amp;author=K.%20Thangaraj&amp;author=N.%20Patterson&amp;author=AL.%20Price&amp;author=L.%20Singh&amp;journal=Nature&amp;volume=461&amp;issue=7263&amp;pages=489-494&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Liszewski MK, Kemper C, Price JD, Atkinson JP. Emerging roles and new functions of CD46. Springer Semin Immunopathol. 2005;27(3):345–58.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00281-005-0002-3"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16200405"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Emerging%20roles%20and%20new%20functions%20of%20CD46&amp;author=MK.%20Liszewski&amp;author=C.%20Kemper&amp;author=JD.%20Price&amp;author=JP.%20Atkinson&amp;journal=Springer%20Semin%20Immunopathol&amp;volume=27&amp;issue=3&amp;pages=345-358&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Riley-Vargas RC, Gill DB, Kemper C, Liszewski MK, Atkinson JP. CD46: expanding beyond complement regulation. Trends Immunol. 2004;25(9):496–503.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.it.2004.07.004"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15324743"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=CD46%3A%20expanding%20beyond%20complement%20regulation&amp;author=RC.%20Riley-Vargas&amp;author=DB.%20Gill&amp;author=C.%20Kemper&amp;author=MK.%20Liszewski&amp;author=JP.%20Atkinson&amp;journal=Trends%20Immunol&amp;volume=25&amp;issue=9&amp;pages=496-503&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Korn JM, Kuruvilla FG, McCarroll SA, Wysoker A, Nemesh J, Cawley S, et al. Integrated genotype calling and association analysis of SNPs, common copy number polymorphisms and rare CNVs. Nat Genet. 2008;40(10):1253–60.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/ng.237"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18776909"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2756534"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Integrated%20genotype%20calling%20and%20association%20analysis%20of%20SNPs%2C%20common%20copy%20number%20polymorphisms%20and%20rare%20CNVs&amp;author=JM.%20Korn&amp;author=FG.%20Kuruvilla&amp;author=SA.%20McCarroll&amp;author=A.%20Wysoker&amp;author=J.%20Nemesh&amp;author=S.%20Cawley&amp;journal=Nat%20Genet&amp;volume=40&amp;issue=10&amp;pages=1253-1260&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Patterson N, Price AL, Reich D. Population structure and eigenanalysis. PLoS Genet. 2006;2(12):e190.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pgen.0020190"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17194218"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1713260"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Population%20structure%20and%20eigenanalysis&amp;author=N.%20Patterson&amp;author=AL.%20Price&amp;author=D.%20Reich&amp;journal=PLoS%20Genet&amp;volume=2&amp;issue=12&amp;pages=e190&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Weir BS, Hill WG. Estimating F-statistics. Annu Rev Genet. 2002;36:721–50.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1146/annurev.genet.36.050802.093940"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12359738"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Estimating%20F-statistics&amp;author=BS.%20Weir&amp;author=WG.%20Hill&amp;journal=Annu%20Rev%20Genet&amp;volume=36&amp;pages=721-750&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Saitou N, Nei M. The neighbor-joining method: a new method for reconstructing phylogenetic trees. Mol Biol Evol. 1987;4(4):406–25.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=3447015"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20neighbor-joining%20method%3A%20a%20new%20method%20for%20reconstructing%20phylogenetic%20trees&amp;author=N.%20Saitou&amp;author=M.%20Nei&amp;journal=Mol%20Biol%20Evol&amp;volume=4&amp;issue=4&amp;pages=406-425&amp;publication_year=1987"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kumar S, Tamura K, Nei M. MEGA3: integrated software for molecular evolutionary genetics analysis and sequence alignment. Brief Bioinform. 2004;5(2):150–63.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bib/5.2.150"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15260895"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=MEGA3%3A%20integrated%20software%20for%20molecular%20evolutionary%20genetics%20analysis%20and%20sequence%20alignment&amp;author=S.%20Kumar&amp;author=K.%20Tamura&amp;author=M.%20Nei&amp;journal=Brief%20Bioinform&amp;volume=5&amp;issue=2&amp;pages=150-163&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Felsenstein J. Maximum-likelihood estimation of evolutionary trees from continuous characters. Am J Hum Genet. 1973;25(5):471–92.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=4741844"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1762641"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Maximum-likelihood%20estimation%20of%20evolutionary%20trees%20from%20continuous%20characters&amp;author=J.%20Felsenstein&amp;journal=Am%20J%20Hum%20Genet&amp;volume=25&amp;issue=5&amp;pages=471-492&amp;publication_year=1973"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Felsenstein J. PHYLIP--phylogeny inference package (version 3.2). Cladistics. 1989;5:164–6.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=PHYLIP--phylogeny%20inference%20package%20%28version%203.2%29&amp;author=J.%20Felsenstein&amp;journal=Cladistics&amp;volume=5&amp;pages=164-166&amp;publication_year=1989"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Falush D, Stephens M, Pritchard JK. Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies. Genetics. 2003;164(4):1567–87.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12930761"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1462648"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data%3A%20linked%20loci%20and%20correlated%20allele%20frequencies&amp;author=D.%20Falush&amp;author=M.%20Stephens&amp;author=JK.%20Pritchard&amp;journal=Genetics&amp;volume=164&amp;issue=4&amp;pages=1567-1587&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR24" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Browning SR, Browning BL. Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering. Am J Hum Genet. 2007;81(5):1084–97.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1086/521987"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17924348"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2265661"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Rapid%20and%20accurate%20haplotype%20phasing%20and%20missing-data%20inference%20for%20whole-genome%20association%20studies%20by%20use%20of%20localized%20haplotype%20clustering&amp;author=SR.%20Browning&amp;author=BL.%20Browning&amp;journal=Am%20J%20Hum%20Genet&amp;volume=81&amp;issue=5&amp;pages=1084-1097&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR25" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Busing F, Meijer E, Van Der Leeden R. Delete-m jackknife for unequal m. Stat Comput. 1999;9(1):3–8.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1023/A%3A1008800423698"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Delete-m%20jackknife%20for%20unequal%20m&amp;author=F.%20Busing&amp;author=E.%20Meijer&amp;author=R.%20Van%20Der%20Leeden&amp;journal=Stat%20Comput&amp;volume=9&amp;issue=1&amp;pages=3-8&amp;publication_year=1999"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR26" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lipson M, Loh PR, Levin A, Reich D, Patterson N, Berger B. Efficient moment-based inference of admixture parameters and sources of gene flow. Mol Biol Evol. 2013;<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Lipson%20M%2C%20Loh%20PR%2C%20Levin%20A%2C%20Reich%20D%2C%20Patterson%20N%2C%20Berger%20B.%20Efficient%20moment-based%20inference%20of%20admixture%20parameters%20and%20sources%20of%20gene%20flow.%20Mol%20Biol%20Evol.%202013%3B"><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li></ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#TheAuthors" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="TheAuthors">Copyright</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="TheAuthors"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">©&nbsp;The Author(s)&nbsp;2018</div></div></section></div>
                        


                    </article>
                </main>

                <div class="c-content-layout__side u-text-sm">
                    <aside>
                        <div class="p-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="//hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-018-0057-5" data-track="click" data-track-category="PDF Download"
            data-track-action="Click download article PDF"
            data-track-label="10.1186/s41065-018-0057-5">Download PDF</a>
        </p>
    
                            
                            
    <button type="button"
       class="c-button c-button--secondary c-button--full-width c-button--expandable"
       data-toggle="collapse"
       data-target="export-citations"
       data-test="export-citations-button"
       data-sticky-update="true"
       aria-controls="export-citations"
       aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/s41065-018-0057-5"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0057-5?format=refman&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/s41065-018-0057-5"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0057-5?format=refman&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/s41065-018-0057-5"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0057-5?format=refman&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/s41065-018-0057-5"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0057-5?format=endnote&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/s41065-018-0057-5"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0057-5?format=endnote&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/s41065-018-0057-5"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0057-5?format=endnote&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/s41065-018-0057-5"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0057-5?format=bibtex&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/s41065-018-0057-5"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0057-5?format=bibtex&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/s41065-018-0057-5"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0057-5?format=bibtex&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
    </div>

                        </div>
                        
    
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
                                <div id="article-access-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/access-count/10.1186/s41065-018-0057-5 #accessCount-container"></div>
                                <div id="citation-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/citation-count/10.1186/s41065-018-0057-5 #count-container"></div>
                                <div id="altmetric-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/altmetric/10.1186/s41065-018-0057-5 #altmetric-container"></div>
                            </ul>
                        </div>
                        
    <div id="sociaMediaShare" class="p-separator u-js-shown">
        <h3 class="p-section-title">Share this article</h3>
        <ul id="shareButtons" class="c-social-links">
            <li class="c-social-links__item">
                <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-018-0057-5">
                    <span class="u-visually-hidden">Share on Twitter</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-twitter"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-018-0057-5">
                    <span class="u-visually-hidden">Share on Facebook</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-facebook"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click"
                        data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-018-0057-5">
                    <span class="u-visually-hidden">Share on LinkedIn</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-linkedin"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-018-0057-5">
                    <span class="u-visually-hidden">Share on Weibo</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-weibo"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnGooglePlus" class="js-btnShareOnGplus" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Google+" data-track-label="10.1186/s41065-018-0057-5">
                    <span class="u-visually-hidden">Share on Google Plus</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-gplus"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-018-0057-5">
                    <span class="u-visually-hidden">Share on Reddit</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-reddit"></use>
                    </svg>
                </button>
            </li>
        </ul>
    </div>
 
                        
    
        <div class="p-separator u-js-shown">
            <h3 class="p-section-title">See updates</h3>
            <button class="c-crossmark-button" data-target="crossmark" data-test="crossmark" data-track="click" data-track-category="Article Page" data-track-action="Click CrossMark" data-track-label="10.1186/s41065-018-0057-5">
                <img id="crossmark-icon" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" alt="Crossmark logo" />
            </button>
        </div>
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
                                <li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/s41065-018-0057-5" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=738&doi=10.1186%2Fs41065-018-0057-5&volume=155&issue=1&title=Genetic+structure%2C+divergence+and+admixture+of+Han+Chinese%2C+Japanese+and+Korean+populations&author_name=Yuchen+Wang&start_page=1&end_page=12">Order reprint</a>

                                </li>
                            </ul>
                        </div>
                    </aside>
                    
    <div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">Hereditas</span>
                        </h4>
                        <p class="c-journal-footer__issn">ISSN: 1601-5223</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
                                
                                    <li class="c-journal-footer__contact-item">Submission enquiries: <a href="https://www.editorialmanager.com/here/" target="_blank">Access here and click Contact Us</a></li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a></li>
                                
                            </ul>
                        </div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class='tracker' style='display:none' src='/track/article/10.1186/s41065-018-0057-5' alt=""/>

         
    <footer>
        <div class="ctx-publisher-footer">
    <div class="c-navbar">
        <div class="u-container">
            <div class="c-navbar__logo u-margin-right-xl">
                <a href="https://www.biomedcentral.com">
                    <img src=/static/images/bmc/logos/logo-bmc-aj-594e03e20d.svg alt="BMC logo" data-test="navbar-logo-footer">

                </a>
            </div>
            <div class="c-navbar__content">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


            </div>
        </div>
    </div>
    <div class="c-publisher-footer" data-test="publisher-footer">
        <div class="u-container">
            
            <div class="c-flex c-flex--wrap c-flex--space-between" role="navigation" data-test="publisher-footer-menu">
                <div class="c-flex" aria-label="Publisher footer links">
                    
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/login">Receive BMC newsletters</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/account">Manage article alerts</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/policies">Policies</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/accessibility">Accessibility</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/press-centre">Press center</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/contact-us">Contact us</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/jobs">Careers</a>
                                    </li>
                                
                            </ul>
                        
                    
                </div>
                <div class="u-margin-bottom-lg">
                    <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                    <ul class="c-flex u-reset-list" data-test="footer-social-links">
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://twitter.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Twitter link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Twitter page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-twitter-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://www.facebook.com/BioMedCentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Facebook link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Facebook page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-facebook-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://plus.google.com/106226337984152901734/"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Google Plus link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Google Plus page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-gplus-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="http://www.weibo.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Weibo link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Weibo page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-weibo-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </div>
            <p class="u-reset-margin">
                By using this website, you agree to our
                <a class="u-text-inherit" href="//www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
                <a class="u-text-inherit" href="//www.biomedcentral.com/privacy-statement">Privacy
                    statement</a> and
                <a class="u-text-inherit" href="//www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
                
                    <a class="optanon-toggle-display u-text-inherit" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
                
            </p>
        </div>
    </div>
</div>


        <div class="c-sn-footer c-sn-footer--neutral">
    <div class="u-container">
        <img src=/static/images/bmc/logos/logo-springernature-8bd4e8d82f.svg class="c-sn-footer__logo" alt="Springer Nature Logo" itemprop="logo" >
        <p data-test="copyright">&#169; 2018 BioMed Central Ltd unless otherwise stated. Part of
            <a href="http://www.springernature.com"
               data-track="click"
               data-track-action="Click Springer Nature link"
               data-track-category="Corporate footer"
               itemscope itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>

    </div>

    
    <script type="text/javascript" src="//cdn.pbgrd.com/bmc.js" async="async"></script>
    <noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>
    <!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker -->

<script id="springerlink-event-tracker" src="https://event-tracker.springernature.com/dist/eventTracker.js"></script>
<script type="text/javascript">
    (function(){
        var dataLayer = window.dataLayer[0];
        var doi = dataLayer.content.article.doi;
        var imprint = dataLayer.content.contentInfo.imprint;
        var bpids = dataLayer.session.authentication.authenticationID;
        var publisher;
        var pageType = dataLayer.page.category.pageType;
        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }
        var eventObject = {'content_type' : pageType,'doi' : doi};

        if (imprint === 'BioMed Central') { publisher = 'BMC' }
        else if (imprint === 'SpringerOpen') { publisher = 'SpringerOpen' }
        else { throw new Error('Event Tracker Error: The publisher has an unexpected value') }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }
        new EventTracker({'platform':publisher}).sendEvent('display', eventObject);
    })();
</script>


    </body>
</html>

