<!DOCTYPE html>
<html lang="en" class="no-js">
    <head>
        <title>Long-read sequencing identified a causal structural variant in an exome-negative case and enabled preimplantation genetic diagnosis | Hereditas | Full Text</title>

        

    <meta name="citation_abstract" content="For a proportion of individuals judged clinically to have a recessive Mendelian disease, only one heterozygous pathogenic variant can be found from clinical whole exome sequencing (WES), posing a challenge to genetic diagnosis and genetic counseling. One possible reason is the limited ability to detect disease causal structural variants (SVs) from short reads sequencing technologies. Long reads sequencing can produce longer reads (typically 1000 bp or longer), therefore offering greatly improved ability to detect SVs that may be missed by short-read sequencing. Here we describe a case study, where WES identified only one heterozygous pathogenic variant for an individual suspected to have glycogen storage disease type Ia (GSD-Ia), which is an autosomal recessive disease caused by bi-allelic mutations in the G6PC gene. Through Nanopore long-read whole-genome sequencing, we identified a 7.1 kb deletion covering two exons on the other allele, suggesting that complex structural variants (SVs) may explain a fraction of cases when the second pathogenic allele is missing from WES on recessive diseases. Both breakpoints of the deletion are within Alu elements, and we designed Sanger sequencing and quantitative PCR assays based on the breakpoints for preimplantation genetic diagnosis (PGD) for the family planning on another child. Four embryos were obtained after in vitro fertilization (IVF), and an embryo without deletion in G6PC was transplanted after PGD and was confirmed by prenatal diagnosis, postnatal diagnosis, and subsequent lack of disease symptoms after birth. In summary, we present one of the first examples of using long-read sequencing to identify causal yet complex SVs in exome-negative patients, which subsequently enabled successful personalized PGD."/>

    <meta name="description" content="For a proportion of individuals judged clinically to have a recessive Mendelian disease, only one heterozygous pathogenic variant can be found from clinical whole exome sequencing (WES), posing a challenge to genetic diagnosis and genetic counseling. One possible reason is the limited ability to detect disease causal structural variants (SVs) from short reads sequencing technologies. Long reads sequencing can produce longer reads (typically 1000 bp or longer), therefore offering greatly improved ability to detect SVs that may be missed by short-read sequencing. Here we describe a case study, where WES identified only one heterozygous pathogenic variant for an individual suspected to have glycogen storage disease type Ia (GSD-Ia), which is an autosomal recessive disease caused by bi-allelic mutations in the G6PC gene. Through Nanopore long-read whole-genome sequencing, we identified a 7.1 kb deletion covering two exons on the other allele, suggesting that complex structural variants (SVs) may explain a fraction of cases when the second pathogenic allele is missing from WES on recessive diseases. Both breakpoints of the deletion are within Alu elements, and we designed Sanger sequencing and quantitative PCR assays based on the breakpoints for preimplantation genetic diagnosis (PGD) for the family planning on another child. Four embryos were obtained after in vitro fertilization (IVF), and an embryo without deletion in G6PC was transplanted after PGD and was confirmed by prenatal diagnosis, postnatal diagnosis, and subsequent lack of disease symptoms after birth. In summary, we present one of the first examples of using long-read sequencing to identify causal yet complex SVs in exome-negative patients, which subsequently enabled successful personalized PGD."/>

    <meta name="journal_id" content="41065"/>

    <meta name="dc.title" content="Long-read sequencing identified a causal structural variant in an exome-negative case and enabled preimplantation genetic diagnosis"/>

    <meta name="dc.source" content="Hereditas 2018 155:1"/>

    <meta name="dc.format" content="text/html"/>

    <meta name="dc.publisher" content="BioMed Central"/>

    <meta name="dc.date" content="2018-09-28"/>

    <meta name="dc.type" content="OriginalPaper"/>

    <meta name="dc.language" content="En"/>

    <meta name="dc.copyright" content="2018 The Author(s)"/>

    <meta name="dc.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="dc.description" content="For a proportion of individuals judged clinically to have a recessive Mendelian disease, only one heterozygous pathogenic variant can be found from clinical whole exome sequencing (WES), posing a challenge to genetic diagnosis and genetic counseling. One possible reason is the limited ability to detect disease causal structural variants (SVs) from short reads sequencing technologies. Long reads sequencing can produce longer reads (typically 1000&#160;bp or longer), therefore offering greatly improved ability to detect SVs that may be missed by short-read sequencing. Here we describe a case study, where WES identified only one heterozygous pathogenic variant for an individual suspected to have glycogen storage disease type Ia (GSD-Ia), which is an autosomal recessive disease caused by bi-allelic mutations in the G6PC gene. Through Nanopore long-read whole-genome sequencing, we identified a 7.1&#160;kb deletion covering two exons on the other allele, suggesting that complex structural variants (SVs) may explain a fraction of cases when the second pathogenic allele is missing from WES on recessive diseases. Both breakpoints of the deletion are within Alu elements, and we designed Sanger sequencing and quantitative PCR assays based on the breakpoints for preimplantation genetic diagnosis (PGD) for the family planning on another child. Four embryos were obtained after in vitro fertilization (IVF), and an embryo without deletion in G6PC was transplanted after PGD and was confirmed by prenatal diagnosis, postnatal diagnosis, and subsequent lack of disease symptoms after birth. In summary, we present one of the first examples of using long-read sequencing to identify causal yet complex SVs in exome-negative patients, which subsequently enabled successful personalized PGD."/>

    <meta name="prism.issn" content="1601-5223"/>

    <meta name="prism.publicationName" content="Hereditas"/>

    <meta name="prism.publicationDate" content="2018-09-28"/>

    <meta name="prism.volume" content="155"/>

    <meta name="prism.number" content="1"/>

    <meta name="prism.section" content="OriginalPaper"/>

    <meta name="prism.startingPage" content="32"/>

    <meta name="prism.copyright" content="2018 The Author(s)"/>

    <meta name="prism.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="prism.url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0069-1"/>

    <meta name="prism.doi" content="doi:10.1186/s41065-018-0069-1"/>

    <meta name="citation_pdf_url" content="https://hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-018-0069-1"/>

    <meta name="citation_fulltext_html_url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0069-1"/>

    <meta name="citation_journal_title" content="Hereditas"/>

    <meta name="citation_publisher" content="BioMed Central"/>

    <meta name="citation_issn" content="1601-5223"/>

    <meta name="citation_title" content="Long-read sequencing identified a causal structural variant in an exome-negative case and enabled preimplantation genetic diagnosis"/>

    <meta name="citation_volume" content="155"/>

    <meta name="citation_issue" content="1"/>

    <meta name="citation_publication_date" content="2018/12"/>

    <meta name="citation_online_date" content="2018/09/28"/>

    <meta name="citation_firstpage" content="32"/>

    <meta name="citation_article_type" content="Research"/>

    <meta name="citation_fulltext_world_readable" content=""/>

    <meta name="dc.identifier" content="doi:10.1186/s41065-018-0069-1"/>

    <meta name="DOI" content="10.1186/s41065-018-0069-1"/>

    <meta name="citation_doi" content="10.1186/s41065-018-0069-1"/>

    <meta name="dc.creator" content="Hefan Miao"/>

    <meta name="dc.creator" content="Jiapeng Zhou"/>

    <meta name="dc.creator" content="Qi Yang"/>

    <meta name="dc.creator" content="Fan Liang"/>

    <meta name="dc.creator" content="Depeng Wang"/>

    <meta name="dc.creator" content="Na Ma"/>

    <meta name="dc.creator" content="Bodi Gao"/>

    <meta name="dc.creator" content="Juan Du"/>

    <meta name="dc.creator" content="Ge Lin"/>

    <meta name="dc.creator" content="Kai Wang"/>

    <meta name="dc.creator" content="Qianjun Zhang"/>

    <meta name="dc.subject" content="Animal Genetics and Genomics"/>

    <meta name="dc.subject" content="Evolutionary Biology"/>

    <meta name="citation_reference" content="citation_journal_title=Genet Med; citation_title=Molecular diagnostic experience of whole-exome sequencing in adult patients; citation_author=JE Posey, JA Rosenfeld, RA James, M Bainbridge, Z Niu, X Wang, S Dhar, W Wiszniewski, ZH Akdemir, T Gambin; citation_volume=18; citation_issue=7; citation_publication_date=2016; citation_pages=678-685; citation_doi=10.1038/gim.2015.142; citation_id=CR1"/>

    <meta name="citation_reference" content="citation_journal_title=N Engl J Med; citation_title=Clinical whole-exome sequencing for the diagnosis of mendelian disorders; citation_author=Y Yang, DM Muzny, JG Reid, MN Bainbridge, A Willis, PA Ward, A Braxton, J Beuten, F Xia, Z Niu; citation_volume=369; citation_issue=16; citation_publication_date=2013; citation_pages=1502-1511; citation_doi=10.1056/NEJMoa1306555; citation_id=CR2"/>

    <meta name="citation_reference" content="citation_journal_title=Genet Med; citation_title=Clinical application of whole-exome sequencing across clinical indications; citation_author=K Retterer, J Juusola, MT Cho, P Vitazka, F Millan, F Gibellini, A Vertino-Bell, N Smaoui, J Neidich, KG Monaghan; citation_volume=18; citation_issue=7; citation_publication_date=2016; citation_pages=696-704; citation_doi=10.1038/gim.2015.148; citation_id=CR3"/>

    <meta name="citation_reference" content="citation_journal_title=Eur J Hum Genet; citation_title=Clinical exome sequencing: results from 2819 samples reflecting 1000 families; citation_author=D Trujillano, AM Bertoli-Avella, K Kumar Kandaswamy, ME Weiss, J Koster, A Marais, O Paknia, R Schroder, JM Garcia-Aznar, M Werber; citation_volume=25; citation_issue=2; citation_publication_date=2017; citation_pages=176-182; citation_doi=10.1038/ejhg.2016.146; citation_id=CR4"/>

    <meta name="citation_reference" content="citation_journal_title=Jama; citation_title=Clinical exome sequencing for genetic identification of rare Mendelian disorders; citation_author=H Lee, JL Deignan, N Dorrani, SP Strom, S Kantarci, F Quintero-Rivera, K Das, T Toy, B Harry, M Yourshaw; citation_volume=312; citation_issue=18; citation_publication_date=2014; citation_pages=1880-1887; citation_doi=10.1001/jama.2014.14604; citation_id=CR5"/>

    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Genome sequencing identifies major causes of severe intellectual disability; citation_author=Christian Gilissen, Jayne Y. Hehir-Kwa, Djie Tjwan Thung, Maartje van de Vorst, Bregje W. M. van Bon, Marjolein H. Willemsen, Michael Kwint, Irene M. Janssen, Alexander Hoischen, Annette Schenck, Richard Leach, Robert Klein, Rick Tearle, Tan Bo, Rolph Pfundt, Helger G. Yntema, Bert B. A. de Vries, Tjitske Kleefstra, Han G. Brunner, Lisenka E. L. M. Vissers, Joris A. Veltman; citation_volume=511; citation_issue=7509; citation_publication_date=2014; citation_pages=344-347; citation_doi=10.1038/nature13394; citation_id=CR6"/>

    <meta name="citation_reference" content="citation_journal_title=Lancet Respir Med; citation_title=Whole-genome sequencing for identification of Mendelian disorders in critically ill infants: a retrospective analysis of diagnostic and clinical findings; citation_author=LK Willig, JE Petrikin, LD Smith, CJ Saunders, I Thiffault, NA Miller, SE Soden, JA Cakici, SM Herd, G Twist; citation_volume=3; citation_issue=5; citation_publication_date=2015; citation_pages=377-387; citation_doi=10.1016/S2213-2600(15)00139-3; citation_id=CR7"/>

    <meta name="citation_reference" content="citation_title=Are whole-exome and whole-genome sequencing approaches cost-effective?; citation_publication_date=2018; citation_id=CR8; citation_author=K Schwarze; citation_author=J Buchanan; citation_author=JC Taylor; citation_author=S Wordsworth; citation_publisher=A systematic review of the literature"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Rev Genet; citation_title=Tandem repeats mediating genetic plasticity in health and disease; citation_author=AJ Hannan; citation_volume=19; citation_issue=5; citation_publication_date=2018; citation_pages=286-298; citation_doi=10.1038/nrg.2017.115; citation_id=CR9"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Rev Genet; citation_title=Towards precision medicine; citation_author=EA Ashley; citation_volume=17; citation_issue=9; citation_publication_date=2016; citation_pages=507-522; citation_doi=10.1038/nrg.2016.86; citation_id=CR10"/>

    <meta name="citation_reference" content="citation_journal_title=Genome Res; citation_title=PennCNV: an integrated hidden Markov model designed for high-resolution copy number variation detection in whole-genome SNP genotyping data; citation_author=K Wang, M Li, D Hadley, R Liu, J Glessner, SFA Grant, H Hakonarson, M Bucan; citation_volume=17; citation_publication_date=2007; citation_pages=1665-1674; citation_doi=10.1101/gr.6861907; citation_id=CR11"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=QuantiSNP: an objective Bayes hidden-Markov model to detect and accurately map copy number variation using SNP genotyping data; citation_author=S Colella, C Yau, JM Taylor, G Mirza, H Butler, P Clouston, AS Bassett, A Seller, CC Holmes, J Ragoussis; citation_volume=35; citation_issue=6; citation_publication_date=2007; citation_pages=2013-2025; citation_doi=10.1093/nar/gkm076; citation_id=CR12"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=A faster circular binary segmentation algorithm for the analysis of array CGH data; citation_author=ES Venkatraman, AB Olshen; citation_volume=23; citation_issue=6; citation_publication_date=2007; citation_pages=657-663; citation_doi=10.1093/bioinformatics/btl646; citation_id=CR13"/>

    <meta name="citation_reference" content="citation_journal_title=Biostatistics; citation_title=Circular binary segmentation for the analysis of array-based DNA copy number data; citation_author=AB Olshen, ES Venkatraman, R Lucito, M Wigler; citation_volume=5; citation_issue=4; citation_publication_date=2004; citation_pages=557-572; citation_doi=10.1093/biostatistics/kxh008; citation_id=CR14"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Rev Genet; citation_title=Genome structural variation discovery and genotyping; citation_author=C Alkan, BP Coe, EE Eichler; citation_volume=12; citation_issue=5; citation_publication_date=2011; citation_pages=363-376; citation_doi=10.1038/nrg2958; citation_id=CR15"/>

    <meta name="citation_reference" content="Merker JD, Wenger AM, Sneddon T, Grove M, Zappala Z, Fresard L, Waggott D, Utiramerur S, Hou Y, Smith KS, et al. Long-read genome sequencing identifies causal structural variation in a Mendelian disease. Genet Med. 2017."/>

    <meta name="citation_reference" content="citation_journal_title=Nat Genet; citation_title=Mutations causing medullary cystic kidney disease type 1 lie in a large VNTR in MUC1 missed by massively parallel sequencing; citation_author=A Kirby, A Gnirke, DB Jaffe, V Baresova, N Pochet, B Blumenstiel, C Ye, D Aird, C Stevens, JT Robinson; citation_volume=45; citation_issue=3; citation_publication_date=2013; citation_pages=299-303; citation_doi=10.1038/ng.2543; citation_id=CR17"/>

    <meta name="citation_reference" content="Sedlazeck FJ, Rescheneder P, Smolka M, Fang H, Nattestad M, von Haeseler A, Schatz MC: Accurate detection of complex structural variations using single-molecule sequencing. Nat Methods 2018, 15(6):461&#8211;468."/>

    <meta name="citation_reference" content="Sanchis-Juan A, Stephens J, French CE, Gleadall N, M&#233;gy K, Penkett C, Stirrups K, Delon I, Dewhurst E, Dolling H, et al. Complex structural variants resolved by short-read and long-read whole genome sequencing in Mendelian disorders. BioRxiv. 2018. 
                    https://doi.org/10.1101/281683
                    
                  ."/>

    <meta name="citation_reference" content="citation_journal_title=Genome Med; citation_title=Interrogating the &quot;unsequenceable&quot; genomic trinucleotide repeat disorders by long-read sequencing; citation_author=Q Liu, P Zhang, D Wang, W Gu, K Wang; citation_volume=9; citation_issue=1; citation_publication_date=2017; citation_pages=65; citation_doi=10.1186/s13073-017-0456-7; citation_id=CR20"/>

    <meta name="citation_reference" content="citation_journal_title=Genome Res; citation_title=Sequencing the unsequenceable: expanded CGG-repeat alleles of the fragile X gene; citation_author=EW Loomis, JS Eid, P Peluso, J Yin, L Hickey, D Rank, S McCalmon, RJ Hagerman, F Tassone, PJ Hagerman; citation_volume=23; citation_issue=1; citation_publication_date=2013; citation_pages=121-128; citation_doi=10.1101/gr.141705.112; citation_id=CR21"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Commun; citation_title=Long-read sequencing and de novo assembly of a Chinese genome; citation_author=L Shi, Y Guo, C Dong, J Huddleston, H Yang, X Han, A Fu, Q Li, N Li, S Gong; citation_volume=7; citation_publication_date=2016; citation_pages=12065; citation_doi=10.1038/ncomms12065; citation_id=CR22"/>

    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=De novo assembly and phasing of a Korean human genome; citation_author=JS Seo, A Rhie, J Kim, S Lee, MH Sohn, CU Kim, A Hastie, H Cao, JY Yun, J Kim; citation_volume=538; citation_issue=7624; citation_publication_date=2016; citation_pages=243-247; citation_doi=10.1038/nature20098; citation_id=CR23"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Biotechnol; citation_title=Nanopore sequencing and assembly of a human genome with ultra-long reads; citation_author=M Jain, S Koren, KH Miga, J Quick, AC Rand, TA Sasani, JR Tyson, AD Beggs, AT Dilthey, IT Fiddes; citation_volume=36; citation_issue=4; citation_publication_date=2018; citation_pages=338-345; citation_doi=10.1038/nbt.4060; citation_id=CR24"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Methods; citation_title=Assembly and diploid architecture of an individual human genome via single-molecule technologies; citation_author=M Pendleton, R Sebra, AW Pang, A Ummat, O Franzen, T Rausch, AM Stutz, W Stedman, T Anantharaman, A Hastie; citation_volume=12; citation_issue=8; citation_publication_date=2015; citation_pages=780-786; citation_doi=10.1038/nmeth.3454; citation_id=CR25"/>

    <meta name="citation_reference" content="citation_journal_title=Hum Mutat; citation_title=Mutations in the glucose-6-phosphatase-alpha (G6PC) gene that cause type Ia glycogen storage disease; citation_author=JY Chou, BC Mansfield; citation_volume=29; citation_issue=7; citation_publication_date=2008; citation_pages=921-930; citation_doi=10.1002/humu.20772; citation_id=CR26"/>

    <meta name="citation_reference" content="citation_journal_title=Eur J Pediatr; citation_title=Three novel mutations of the G6PC gene identified in Chinese patients with glycogen storage disease type Ia; citation_author=BX Zheng, Q Lin, M Li, Y Jin; citation_volume=174; citation_issue=1; citation_publication_date=2015; citation_pages=59-63; citation_doi=10.1007/s00431-014-2354-y; citation_id=CR27"/>

    <meta name="citation_reference" content="citation_journal_title=Genetics in medicine : official journal of the American College of Medical Genetics; citation_title=Diagnosis and management of glycogen storage disease type I: a practice guideline of the American College of Medical Genetics and Genomics; citation_author=PS Kishnani, SL Austin, JE Abdenur, P Arn, DS Bali, A Boney, WK Chung, AI Dagli, D Dale, D Koeberl; citation_volume=16; citation_issue=11; citation_publication_date=2014; citation_doi=10.1038/gim.2014.128; citation_id=CR28"/>

    <meta name="citation_reference" content="citation_journal_title=Eur J Pediatr; citation_title=Glycogen storage disease type I: diagnosis and phenotype/genotype correlation; citation_author=D Matern, HH Seydewitz, D Bali, C Lang, YT Chen; citation_volume=161; citation_issue=Suppl 1; citation_publication_date=2002; citation_pages=S10-S19; citation_doi=10.1007/BF02679989; citation_id=CR29"/>

    <meta name="citation_reference" content="citation_journal_title=J Inherit Metab Dis; citation_title=Type I glycogen storage diseases: disorders of the glucose-6-phosphatase/glucose-6-phosphate transporter complexes; citation_author=JY Chou, HS Jun, BC Mansfield; citation_volume=38; citation_issue=3; citation_publication_date=2015; citation_pages=511-519; citation_doi=10.1007/s10545-014-9772-x; citation_id=CR30"/>

    <meta name="citation_reference" content="Sedlazeck FJ, Rescheneder P, Smolka M, Fang H, Nattestad M, von Haeseler A, Schatz MC. Accurate detection of complex structural variations using single molecule sequencing. Nat Methods. 2018;15(6):461&#8211;468."/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Genome project data processing S: the sequence alignment/map format and SAMtools; citation_author=H Li, B Handsaker, A Wysoker, T Fennell, J Ruan, N Homer, G Marth, G Abecasis, R Durbin; citation_volume=25; citation_issue=16; citation_publication_date=2009; citation_pages=2078-2079; citation_doi=10.1093/bioinformatics/btp352; citation_id=CR32"/>

    <meta name="citation_reference" content="Maria, Nattestad C-SC, Schatz MC. Ribbon: Visualizing complex genome alignments and structural variation. bioRxiv. 2016. 
                    https://doi.org/10.1101/082123
                    
                  ."/>

    <meta name="citation_reference" content="citation_journal_title=Brief Bioinform; citation_title=Integrative genomics viewer (IGV): high-performance genomics data visualization and exploration; citation_author=H Thorvaldsdottir, JT Robinson, JP Mesirov; citation_volume=14; citation_issue=2; citation_publication_date=2013; citation_pages=178-192; citation_doi=10.1093/bib/bbs017; citation_id=CR34"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data; citation_author=K Wang, M Li, H Hakonarson; citation_volume=38; citation_issue=16; citation_publication_date=2010; citation_pages=e164; citation_doi=10.1093/nar/gkq603; citation_id=CR35"/>

    <meta name="citation_reference" content="citation_journal_title=Genet Med; citation_title=Standards and guidelines for the interpretation of sequence variants: a joint consensus recommendation of the American College of Medical Genetics and Genomics and the Association for Molecular Pathology; citation_author=S Richards, N Aziz, S Bale, D Bick, S Das, J Gastier-Foster, WW Grody, M Hegde, E Lyon, E Spector; citation_volume=17; citation_issue=5; citation_publication_date=2015; citation_pages=405-424; citation_doi=10.1038/gim.2015.30; citation_id=CR36"/>

    <meta name="citation_reference" content="citation_journal_title=Am J Hum Genet; citation_title=InterVar: clinical interpretation of genetic variants by the 2015 ACMG-AMP guidelines; citation_author=Q Li, K Wang; citation_volume=100; citation_issue=2; citation_publication_date=2017; citation_pages=267-280; citation_doi=10.1016/j.ajhg.2017.01.004; citation_id=CR37"/>

    <meta name="citation_reference" content="Alpha Scientists in Reproductive Medicine and ESHRE Special Interest Group of Embryology. The Istanbul consensus workshop on embryo assessment: proceedings of an expert meeting. Hum Reprod. 2011;26(6):1270&#8211;83."/>

    <meta name="citation_reference" content="citation_journal_title=Current opinion in virology; citation_title=Alu elements: an intrinsic source of human genome instability; citation_author=C Ade, AM Roy-Engel, PL Deininger; citation_volume=3; citation_issue=6; citation_publication_date=2013; citation_pages=639-645; citation_doi=10.1016/j.coviro.2013.09.002; citation_id=CR39"/>

    <meta name="citation_reference" content="citation_journal_title=Am J Hum Genet; citation_title=Human genomic deletions mediated by recombination between Alu elements; citation_author=SK Sen, K Han, J Wang, J Lee, H Wang, PA Callinan, M Dyer, R Cordaux, P Liang, MA Batzer; citation_volume=79; citation_issue=1; citation_publication_date=2006; citation_pages=41-53; citation_doi=10.1086/504600; citation_id=CR40"/>

    <meta name="citation_reference" content="citation_journal_title=Genomics Inform; citation_title=Structural variation of Alu element and human disease; citation_author=S Kim, CS Cho, K Han, J Lee; citation_volume=14; citation_issue=3; citation_publication_date=2016; citation_pages=70-77; citation_doi=10.5808/GI.2016.14.3.70; citation_id=CR41"/>

    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Mutations in the glucose-6-phosphatase gene that cause glycogen storage disease type 1a; citation_author=KJ Lei, LL Shelly, CJ Pan, JB Sidbury, JY Chou; citation_volume=262; citation_issue=5133; citation_publication_date=1993; citation_pages=580-583; citation_doi=10.1126/science.8211187; citation_id=CR42"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Rev Endocrinol; citation_title=Glycogen storage disease type I and G6Pase-beta deficiency: etiology and therapy; citation_author=JY Chou, HS Jun, BC Mansfield; citation_volume=6; citation_issue=12; citation_publication_date=2010; citation_pages=676-688; citation_doi=10.1038/nrendo.2010.189; citation_id=CR43"/>

    <meta name="citation_reference" content="citation_journal_title=Gene; citation_title=A novel homozygous no-stop mutation in G6PC gene from a Chinese patient with glycogen storage disease type Ia; citation_author=LL Gu, XH Li, Y Han, DH Zhang, QM Gong, XX Zhang; citation_volume=536; citation_issue=2; citation_publication_date=2014; citation_pages=362-365; citation_doi=10.1016/j.gene.2013.11.059; citation_id=CR44"/>

    <meta name="citation_reference" content="citation_journal_title=Sci Rep; citation_title=Novel metrics to measure coverage in whole exome sequencing datasets reveal local and global non-uniformity; citation_author=Q Wang, CS Shashikant, M Jensen, NS Altman, S Girirajan; citation_volume=7; citation_issue=1; citation_publication_date=2017; citation_pages=885; citation_doi=10.1038/s41598-017-01005-x; citation_id=CR45"/>

    <meta name="citation_author" content="Hefan Miao"/>

    <meta name="citation_author_institution" content="Institute of Reproductive and Stem Cell Engineering, Central South University, Changsha, China"/>

    <meta name="citation_author_institution" content="Reproductive and Genetic Hospital of CITIC-Xiangya, Changsha, China"/>

    <meta name="citation_author" content="Jiapeng Zhou"/>

    <meta name="citation_author_institution" content="GrandOmics Biosciences, Beijing, China"/>

    <meta name="citation_author" content="Qi Yang"/>

    <meta name="citation_author_institution" content="GrandOmics Biosciences, Beijing, China"/>

    <meta name="citation_author" content="Fan Liang"/>

    <meta name="citation_author_institution" content="GrandOmics Biosciences, Beijing, China"/>

    <meta name="citation_author" content="Depeng Wang"/>

    <meta name="citation_author_institution" content="GrandOmics Biosciences, Beijing, China"/>

    <meta name="citation_author" content="Na Ma"/>

    <meta name="citation_author_institution" content="Institute of Reproductive and Stem Cell Engineering, Central South University, Changsha, China"/>

    <meta name="citation_author_institution" content="Reproductive and Genetic Hospital of CITIC-Xiangya, Changsha, China"/>

    <meta name="citation_author" content="Bodi Gao"/>

    <meta name="citation_author_institution" content="Institute of Reproductive and Stem Cell Engineering, Central South University, Changsha, China"/>

    <meta name="citation_author_institution" content="Reproductive and Genetic Hospital of CITIC-Xiangya, Changsha, China"/>

    <meta name="citation_author" content="Juan Du"/>

    <meta name="citation_author_institution" content="Institute of Reproductive and Stem Cell Engineering, Central South University, Changsha, China"/>

    <meta name="citation_author_institution" content="Reproductive and Genetic Hospital of CITIC-Xiangya, Changsha, China"/>

    <meta name="citation_author" content="Ge Lin"/>

    <meta name="citation_author_institution" content="Institute of Reproductive and Stem Cell Engineering, Central South University, Changsha, China"/>

    <meta name="citation_author_institution" content="Reproductive and Genetic Hospital of CITIC-Xiangya, Changsha, China"/>

    <meta name="citation_author" content="Kai Wang"/>

    <meta name="citation_author_institution" content="Raymond G. Perelman Center for Cellular and Molecular Therapeutics, Children&#8217;s Hospital of Philadelphia, Philadelphia, USA"/>

    <meta name="citation_author" content="Qianjun Zhang"/>

    <meta name="citation_author_institution" content="Institute of Reproductive and Stem Cell Engineering, Central South University, Changsha, China"/>

    <meta name="citation_author_institution" content="Reproductive and Genetic Hospital of CITIC-Xiangya, Changsha, China"/>



        <meta charset="UTF-8"/>
        <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
        <meta name="viewport" content="width=device-width, initial-scale=1"/>

        <!-- For IE 9 and below -->
<!--[if IE]><link rel="shortcut icon" href=/static/images/favicons//favicon.ico><![endif]-->
<!-- Touch Icons - iOS and Android 2.1+ -->
<link rel="apple-touch-icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>
<!-- Firefox, Chrome, Safari, IE 11+ and Opera -->
<link rel="icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>


        <script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script>
        <script>(function(d){if (sessionStorage.fontsLoaded){d.style.setProperty('--font-family-sans', 'Europa');d.style.setProperty('--font-family-serif', 'Noto Serif')}})(document.documentElement);</script>

        <link rel="stylesheet" media="screen" href=/static/styles/basic/basic-a31ca85bc2.css>
<link rel="stylesheet" media="print" href=/static/styles/bmc/print-3a6240f456.css>

<link rel="stylesheet" class="js-ctm" href=/static/styles/bmc/main-84414d3d47.css
      media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">

        

        <script src="/static/js/app-bundle-8d61e6ef37.js" defer></script>
        
        <script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script>
        
    <script type="text/javascript">
        config = {
            env: 'live',
            site: 'hereditasjournal.biomedcentral.com',
            siteWithPath: 'hereditasjournal.biomedcentral.com' + window.location.pathname,
            twitterHashtag: '',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/s41065-018-0069-1',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central'
        };
    </script>

        
    <script type="text/javascript">
        dataLayer = [{"content":{"article":{"doi":"10.1186/s41065-018-0069-1","articleType":"Research","peerReviewType":"Closed","supplement":null,"keywords":"Whole-exome sequencing;WES;Structural variants;Long-read sequencing;G6PC;PGD;GSD-Ia"},"contentInfo":{"imprint":"BioMed Central","title":"Long-read sequencing identified a causal structural variant in an exome-negative case and enabled preimplantation genetic diagnosis","publishedAt":1538092800000,"publishedAtDate":"2018-09-28","author":["Hefan Miao","Jiapeng Zhou","Qi Yang","Fan Liang","Depeng Wang","Na Ma","Bodi Gao","Juan Du","Ge Lin","Kai Wang","Qianjun Zhang"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":"CC BY + CC0","openAccess":true},"environment":"live"},"journal":{"siteKey":"hereditasjournal.biomedcentral.com","volume":"155","issue":"1","title":"Hereditas","type":"AJs Life","journalID":41065,"gaCode":"UA-62066850-11","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Research","publishingSegment":"Genetics & Heredity (Life)"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}}}]

    </script>

        <script src="https://cdn.cookielaw.org/consent/7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script>
        
    
        
    

        
    
        
            
                <!-- Google Tag Manager -->
                <script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script>
                <!-- End Google Tag Manager -->
            
        
    


        
    
    <link rel="canonical" href="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0069-1"/>
    

        
        
        <meta property="og:url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0069-1"/>
        <meta property="og:type" content="article"/>
        <meta property="og:site_name" content="Hereditas"/>
        <meta property="og:title" content="Long-read sequencing identified a causal structural variant in an exome-negative case and enabled preimplantation genetic diagnosis"/>
        <meta property="og:description" content="For a proportion of individuals judged clinically to have a recessive Mendelian disease, only one heterozygous pathogenic variant can be found from clinical whole exome sequencing (WES), posing a challenge to genetic diagnosis and genetic counseling. One possible reason is the limited ability to detect disease causal structural variants (SVs) from short reads sequencing technologies. Long reads sequencing can produce longer reads (typically 1000&amp;nbsp;bp or longer), therefore offering greatly improved ability to detect SVs that may be missed by short-read sequencing. Here we describe a case study, where WES identified only one heterozygous pathogenic variant for an individual suspected to have glycogen storage disease type Ia (GSD-Ia), which is an autosomal recessive disease caused by bi-allelic mutations in the G6PC gene. Through Nanopore long-read whole-genome sequencing, we identified a 7.1&amp;nbsp;kb deletion covering two exons on the other allele, suggesting that complex structural variants (SVs) may explain a fraction of cases when the second pathogenic allele is missing from WES on recessive diseases. Both breakpoints of the deletion are within Alu elements, and we designed Sanger sequencing and quantitative PCR assays based on the breakpoints for preimplantation genetic diagnosis (PGD) for the family planning on another child. Four embryos were obtained after in vitro fertilization (IVF), and an embryo without deletion in G6PC was transplanted after PGD and was confirmed by prenatal diagnosis, postnatal diagnosis, and subsequent lack of disease symptoms after birth. In summary, we present one of the first examples of using long-read sequencing to identify causal yet complex SVs in exome-negative patients, which subsequently enabled successful personalized PGD."/>
        <meta property="og:image" content="https://images.springer.com/sgw/journals/medium/41065.jpg"/>
    
    </head>

    <body class="journal journal-fulltext"
          
          >
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK"
                            height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><feColorMatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"/></filter></defs><symbol id="icon-arrow-down" viewBox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"/></symbol><symbol id="icon-arrow-left-bullet" viewBox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"/></symbol><symbol id="icon-arrow-right-bullet" viewBox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"/></symbol><symbol id="icon-arrow-right" viewBox="0 0 32 64"><path d="M0 0v64l32-32z"/></symbol><symbol id="icon-close" viewBox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"/></symbol><symbol id="icon-community" viewBox="0 0 60.643 31.01"><mask maskUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><path fill="#FFF" d="M0 31.01h60.643V0H0z" filter="url(#a)"/></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"/></symbol><symbol id="icon-download" viewBox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"/></symbol><symbol id="icon-ext-link" viewBox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"/><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"/><path fill="#676767" d="M9.7 0L16 6.2V0z"/></symbol><symbol id="icon-info-bordered" viewBox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"/><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"/></symbol><symbol id="icon-opr" viewBox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"/><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"/></symbol><symbol id="icon-quotation" viewBox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"/></symbol><symbol id="icon-remove" viewBox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"/><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"/></symbol><symbol id="icon-ribbon" viewBox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"/></symbol><symbol id="icon-rss" viewBox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"/><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"/><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"/></g></symbol><symbol id="icon-search" viewBox="0 0 16 16"><path d="M6.276 12.257C2.815 12.257 0 9.508 0 6.128 0 2.749 2.815 0 6.276 0c3.462 0 6.277 2.749 6.277 6.128.002 3.38-2.815 6.129-6.277 6.129zm0-10.772c-2.622 0-4.755 2.083-4.755 4.643 0 2.562 2.133 4.644 4.755 4.644 2.623 0 4.756-2.082 4.756-4.644 0-2.56-2.134-4.643-4.756-4.643z"/><path d="M16 13.751L13.641 16l-4.875-4.82 2.36-2.248L16 13.751z"/></symbol><symbol id="icon-speech-bubble" viewBox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"/></symbol><symbol id="icon-error" viewBox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"/></symbol><symbol id="icon-info" viewBox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"/></symbol><symbol id="icon-success" viewBox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"/></symbol><symbol id="icon-warning" viewBox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"/></symbol><symbol id="icon-facebook-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"/></symbol><symbol id="icon-facebook" viewBox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"/><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"/></symbol><symbol id="icon-gplus-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M480.5 273.117c.123.076.26.185.416.334.146.152.289.343.432.562.135.211.258.459.359.746.086.286.129.619.129 1.004-.012.7-.164 1.262-.465 1.683a5.374 5.374 0 0 1-.461.562c-.186.176-.375.35-.58.527a2.65 2.65 0 0 0-.322.41.908.908 0 0 0-.176.553c0 .205.059.375.18.508.102.129.199.238.299.336l.662.545a6.48 6.48 0 0 1 1.086 1.119c.293.415.445.955.459 1.625 0 .949-.42 1.789-1.256 2.521-.869.76-2.123 1.15-3.762 1.177-1.373-.015-2.4-.308-3.074-.879-.682-.534-1.023-1.172-1.023-1.917 0-.359.111-.766.336-1.211.215-.443.602-.837 1.172-1.174a6.126 6.126 0 0 1 1.998-.725 15.415 15.415 0 0 1 1.723-.173 3.996 3.996 0 0 1-.377-.596 1.347 1.347 0 0 1-.193-.74c0-.174.023-.317.074-.433a8.83 8.83 0 0 1 .121-.338 5.973 5.973 0 0 1-.633.035c-1.043-.012-1.838-.339-2.381-.979-.574-.603-.861-1.3-.861-2.097 0-.961.404-1.832 1.217-2.616.559-.46 1.135-.757 1.736-.898a8.494 8.494 0 0 1 1.67-.182h3.916l-1.209.704h-1.207v.005l-.005.002zm.754 11.276c0-.498-.162-.932-.484-1.302-.346-.348-.881-.778-1.609-1.29a4.045 4.045 0 0 0-.439-.02c-.102-.016-.354 0-.768.039a8.141 8.141 0 0 0-1.244.275c-.1.036-.24.096-.42.17a2.713 2.713 0 0 0-.553.355 1.985 1.985 0 0 0-.459.592c-.145.25-.215.551-.215.901 0 .687.311 1.253.932 1.698.596.444 1.404.68 2.432.688.928-.014 1.629-.217 2.117-.612.476-.385.71-.886.71-1.494zm-2.729-5.743a1.843 1.843 0 0 0 1.289-.556c.164-.249.271-.506.318-.767a7.32 7.32 0 0 0 .041-.659c0-.772-.195-1.552-.594-2.336a2.611 2.611 0 0 0-.73-.921 1.894 1.894 0 0 0-1.064-.352 1.723 1.723 0 0 0-1.324.64c-.299.437-.438.92-.428 1.457 0 .709.209 1.444.623 2.211.201.356.459.659.775.912.323.248.686.371 1.094.371zm9.698.077h-2.055v-2.056h-.996v2.056h-2.053v.998h2.053v2.052h.996v-2.052h2.055z"/></symbol><symbol id="icon-gplus" viewBox="0 0 24 24"><circle fill="#D84A36" cx="12" cy="12" r="12"/><g fill="#FFF"><path d="M12.012 12.111c-.304-.33-.07-.842.155-1.078.226-.241.639-.56.829-.74.193-.184.66-.605.674-1.868 0 0 .112-1.139-1.127-2.06h.977l1.038-.593h-3.507s-2.083-.067-3.085 1.207c-1 1.276-.966 2.871.114 3.77 1.08.898 2.371.648 2.371.648s-.197.422-.163.775c.036.353.421.849.504.965 0 0-2.386-.011-3.551 1.033-1.165 1.053-1.29 3.097 1.021 3.803 2.312.707 5.303-.055 5.906-2.277 0 0 .262-.908-.189-1.683-.459-.775-1.663-1.571-1.967-1.902zm-2.555-1.83c-.744-1-1.308-2.953 0-3.783 0 0 .955-.579 1.801.266.846.846 1.082 2.277.928 3.003-.15.727-.563.897-.739 1.008-.185.111-1.249.504-1.99-.494zm3.668 5.481c-.035.393-.083 1.254-1.491 1.664-1.409.408-3.826-.264-3.757-1.914 0 0 .006-.698.449-1.089.445-.396 1.094-.794 3.029-.819 0 0 .643.473 1.072.809.427.332.733.952.698 1.349z"/><path d="M16.323 11.055V9.803h-.752v1.252h-1.255v.748h1.255v1.256h.752v-1.256h1.248v-.748z"/></g></symbol><symbol id="icon-linkedin" viewBox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"/><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"/><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"/></g></symbol><symbol id="icon-reddit" viewBox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"/><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"/><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"/><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"/><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"/><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"/><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"/><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"/></symbol><symbol id="icon-twitter-bordered" viewBox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"/><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/></symbol><symbol id="icon-twitter" viewBox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"/><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"/></symbol><symbol id="icon-weibo-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"/><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"/><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"/></symbol><symbol id="icon-weibo" viewBox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"/><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"/><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"/><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"/><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"/><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"/></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner c-banner--info">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner" >
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1"
                 data-gpt-unitpath="/270604982/bmc/hereditasjournal/articles"
                 data-gpt-sizes="728x90,970x90"
                 data-gpt-targeting="pos=LB1;doi=10.1186/s41065-018-0069-1;kwrd=Whole-exome sequencing,WES,Structural variants,Long-read sequencing,G6PC,PGD,GSD-Ia;pmc=L32030,L21001;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-018-0069-1&amp;kwrd=Whole-exome sequencing,WES,Structural variants,Long-read sequencing,G6PC,PGD,GSD-Ia&amp;pmc=L32030,L21001&amp;">
                        <img data-test="gpt-advert-fallback-img"
                             src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-018-0069-1&amp;kwrd=Whole-exome sequencing,WES,Structural variants,Long-read sequencing,G6PC,PGD,GSD-Ia&amp;pmc=L32030,L21001&amp;"
                             alt="Advertisement"
                             width="728"
                             height="90">
                    </a>
                </noscript>
            </div>
        </div>
    </div>

        
         
    <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>


<header class="c-popup-search ctx-publisher-header" id="top" data-test="publisher-header">
    <nav class="c-navbar">
        <div class="u-container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization" />
                <div class="c-logo">
    <a href="https://www.biomedcentral.com" itemprop="url">
        <img src=/static/images/bmc/logos/logo-bmc-white-aj-e3a70ae848.svg alt="BMC" itemprop="logo">
        <div class="c-logo__strapline">
            <img src=/static/images/bmc/logos/logo-bmc-white-strapline-sn-e7b1d6e67f.svg alt="Part of Springer Nature">
        </div>
    </a>
</div>
            </div>
            <button
                type="button"
                class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg"
                id="publisher-header-menu-button"
                data-toggle="collapse"
                data-test="publisher-header-menu-toggle"
                data-toggle-linked="publisher-header"
                data-target="publisher-header-search publisher-header-menu"
                aria-controls="publisher-header-menu publisher-header-search"
                aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


                <ul class="c-navbar__nav">
                    <li class="c-navbar__item">
                        <a data-replace="true"
                           data-placeholder="/placeholder/v1/username"
                           class="c-navbar__link"
                           href="https://www.biomedcentral.com/login"
                           data-test="login-link"
                           data-track="click"
                           data-track-category="Account"
                           data-track-action="Click Login Link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                    <li class="c-navbar__item">
                        <button
                            type="button"
                            class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg"
                            data-toggle="collapse"
                            data-test="header-search-button"
                            data-target="publisher-header-search publisher-header-menu publisher-header-menu-button"
                            aria-controls="publisher-header-search publisher-header-menu"
                            aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true">
                                    <use xlink:href="#icon-search"></use>
                                </svg>
                            </span>
                        </button>
                    </li>
                </ul>
            </div>
        </div>
    </nav>
    <div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field c-form-field--inline" method="GET" action="//www.biomedcentral.com/search" data-track="submit"
        data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label data-track-label="" data-test="global-search">
        <label for="publisherSearch" class="u-visually-hidden">Search all BMC articles</label>
        <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text" value="" placeholder="Search all BMC articles"/>
        <button class="c-button c-button--secondary" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true">
        <use xlink:href="#icon-search"></use>
    </svg>
</button>


        <input type="hidden" name="searchType" value="publisherSearch"/>
    </form>
</div>

            </div>
        </div>
    </div>
</header>


        
            <header class="c-journal-header c-journal-header--hereditas ctx-journal-header">
                <div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="/">
    
        
    
    <span class="c-journal-title__text ">Hereditas</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="u-container">
                        
                            <button type="button"
                                    class="c-navbar__link c-dropdown u-hidden-at-lg"
                                    data-toggle="collapse"
                                    data-test="journal-header-menu"
                                    data-toggle-linked="journal-header"
                                    data-target="journal-menu"
                                    aria-controls="journal-menu"
                                    aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav">
                                    <ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/'>Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/about'>About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href='/articles'>Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/submission-guidelines'>Submission Guidelines</a>
                                            </li>
                                        
                                    </ul>
                                </nav>
                            </div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--ocean-light-blue">
                    
                </div>
            </header>
            
        

        <div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-content-layout c-content-layout--fulltext" data-component="sticky-container">
                <div class="c-content-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3><ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc=""><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Methods">Methods</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec8" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results">Results</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec13" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Discussion">Discussion</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec14" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Conclusion">Conclusion</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li></ol></nav>
                </div>

                <main class="c-content-layout__main">
                    <article itemscope itemtype="http://schema.org/ScholarlyArticle">
                        
                        
    <div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
            <li class="c-article-identifiers__item" data-test="article-category">Research</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
    </div>

                        <div class="FulltextWrapper"><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">Long-read sequencing identified a causal structural variant in an exome-negative case and enabled preimplantation genetic diagnosis</h1></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true"><ul class="u-listReset"><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Hefan&nbsp;Miao</span><span class="EqualContributionSymbol">†</span><sup><a href="#Aff1" class="AffiliationID">1</a>, <a href="#Aff2" class="AffiliationID">2</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Jiapeng&nbsp;Zhou</span><span class="EqualContributionSymbol">†</span><sup><a href="#Aff3" class="AffiliationID">3</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Qi&nbsp;Yang</span><sup><a href="#Aff3" class="AffiliationID">3</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Fan&nbsp;Liang</span><sup><a href="#Aff3" class="AffiliationID">3</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Depeng&nbsp;Wang</span><sup><a href="#Aff3" class="AffiliationID">3</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Na&nbsp;Ma</span><sup><a href="#Aff1" class="AffiliationID">1</a>, <a href="#Aff2" class="AffiliationID">2</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Bodi&nbsp;Gao</span><sup><a href="#Aff1" class="AffiliationID">1</a>, <a href="#Aff2" class="AffiliationID">2</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Juan&nbsp;Du</span><sup><a href="#Aff1" class="AffiliationID">1</a>, <a href="#Aff2" class="AffiliationID">2</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Ge&nbsp;Lin</span><sup><a href="#Aff1" class="AffiliationID">1</a>, <a href="#Aff2" class="AffiliationID">2</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Kai&nbsp;Wang</span><sup><a href="#Aff4" class="AffiliationID">4</a></sup><a href="mailto:wangk@email.chop.edu" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a> and </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Qianjun&nbsp;Zhang</span><sup><a href="#Aff1" class="AffiliationID">1</a>, <a href="#Aff2" class="AffiliationID">2</a></sup><a href="mailto:zhangqianjun@csu.edu.cn" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a></li></ul><div class="AffiliationLegend"><div class="AffiliationLegendLine"><span class="EqualContributionSymbol">†</span>Contributed equally</div></div></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation"><div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">Hereditas</span></span><span class="ArticleCitation_Year">2018</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">155</strong>:32</span></span></div><p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/s41065-018-0069-1">https://doi.org/10.1186/s41065-018-0069-1</a></p><p class="Copyright">© &nbsp;The Author(s)&nbsp;2018</p></div></div><ul class="ArticleHistory u-reset-list"><li class="History HistoryReceived"><strong>Received: </strong><span>28&nbsp;June&nbsp;2018</span></li><li class="History HistoryAccepted"><strong>Accepted: </strong><span>13&nbsp;September&nbsp;2018</span></li><li class="History HistoryOnlineDate"><strong>Published: </strong><span itemprop="datePublished">28&nbsp;September&nbsp;2018</span></li></ul></div>
                        
                        
                        
    

                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2><div class="c-section__content" id="Abstract"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p id="Par1" class="Para">For a proportion of individuals judged clinically to have a recessive Mendelian disease, only one heterozygous pathogenic variant can be found from clinical whole exome sequencing (WES), posing a challenge to genetic diagnosis and genetic counseling. One possible reason is the limited ability to detect disease causal structural variants (SVs) from short reads sequencing technologies. Long reads sequencing can produce longer reads (typically 1000&nbsp;bp or longer), therefore offering greatly improved ability to detect SVs that may be missed by short-read sequencing.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Results</h3>
                <p id="Par2" class="Para">Here we describe a case study, where WES identified only one heterozygous pathogenic variant for an individual suspected to have glycogen storage disease type Ia (GSD-Ia), which is an autosomal recessive disease caused by bi-allelic mutations in the <em xmlns="" class="EmphasisTypeItalic">G6PC</em> gene. Through Nanopore long-read whole-genome sequencing, we identified a 7.1&nbsp;kb deletion covering two exons on the other allele, suggesting that complex structural variants (SVs) may explain a fraction of cases when the second pathogenic allele is missing from WES on recessive diseases. Both breakpoints of the deletion are within Alu elements, and we designed Sanger sequencing and quantitative PCR assays based on the breakpoints for preimplantation genetic diagnosis (PGD) for the family planning on another child. Four embryos were obtained after in vitro fertilization (IVF), and an embryo without deletion in <em xmlns="" class="EmphasisTypeItalic">G6PC</em> was transplanted after PGD and was confirmed by prenatal diagnosis, postnatal diagnosis, and subsequent lack of disease symptoms after birth.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusions</h3>
                <p id="Par3" class="Para">In summary, we present one of the first examples of using long-read sequencing to identify causal yet complex SVs in exome-negative patients, which subsequently enabled successful personalized PGD.</p>
              </div></div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords"><li class="c-keywords__item" itemprop="about">Whole-exome sequencing</li><li class="c-keywords__item" itemprop="about">WES</li><li class="c-keywords__item" itemprop="about">Structural variants</li><li class="c-keywords__item" itemprop="about">Long-read sequencing</li><li class="c-keywords__item" itemprop="about">G6PC</li><li class="c-keywords__item" itemprop="about">PGD</li><li class="c-keywords__item" itemprop="about">GSD-Ia</li></ul></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par13" class="Para">Whole exome sequencing (WES) is now widely used in genetic testing on patients who are suspected or have been clinically demonstrated to have genetic disorders. However, a large proportion (~ 60–70%) of patients judged clinically to have a Mendelian disease receive negative results on WES with current Illumina short-read sequencing technology [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>–<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>]. Compared to WES, the use of whole-genome sequencing (WGS) does not appear to significantly improve the diagnostic yield [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>, <span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>] or present much economic advantage [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>]. Therefore, WES/WGS-negative cases pose a significant challenge to the clinical diagnosis of genetic diseases. Several reasons may explain the lack of positive findings, such as the inefficiency of template DNA capture, the biases in sequencing coverage, the failure to call causal variants from data, the inability to catalog all functional variants (especially non-coding variants), the incorrect clinical interpretation of genetic variants, the possibility of complicated oligogenic disease in certain patients and the possibility of disease causal mechanisms due to somatic or epigenetic origin. Among these reasons, the limited ability to interrogate repeat elements such as tandem repeats [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>] and structural variants (SVs) [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>] may play important roles. Clinical microarrays such as SNP arrays or array-CGH can detect relatively large deletions and duplications [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>–<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>], but have difficulty to reliably identify small (&lt; 10&nbsp;kb) exonic deletions or complex SVs in clinical settings [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>]. Furthermore, several published studies demonstrated examples where disease causal SVs were missed by short-read WES/WGS [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>], or that certain classes of disease causal repeats failed to be identified by WES/WGS [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>]. In particular, conventional short-reads sequencing approaches have been reported to lack sensitivity, exhibit very high false positive rate, and misinterpret complex or nested SVs [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par14" class="Para">Long-read sequencing technologies, such as the 10X Genomics linked-read sequencing, the Oxford Nanopore Technologies (ONT) and PacBio single molecule real-time (SMRT) sequencing, offer complementary strengths to traditional WES/WGS based on short-read sequencing. Long-read sequencing can produce read length (typically 1000&nbsp;bp or longer) that is far higher than the 100-150&nbsp;bp produced by short-read sequencing, therefore allowing for the resolution of breakpoints of complex SVs [<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>] or the detection of long tandem repeats [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>, <span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>]. In particular, recent de novo human genome assemblies via long-read sequencing have revealed tens of thousands of SVs per genome, several times more than previously observed via WGS, suggesting an underestimation of the extent and complexity of SVs [<span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>–<span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par15" class="Para">In the present study, we applied a long-read whole-genome sequencing to yield the genetic diagnosis of glycogen storage disease type Ia (GSD-Ia) in a patient whose causal variants were unsolved by Sanger sequencing and WES. Glycogen storage disease type I (GSD-I) is a group of autosomal recessive metabolic diseases caused by defects in the glucose-6-phosphatase (G6Pase) complex, with an overall incidence of approximately 1:20,000–40,000 cases per live birth [<span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>, <span class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>]. The most common form, GSD-Ia, represents more than 80% of GSD-I cases [<span class="CitationRef"><a href="#CR28" data-toggle="citation">28</a></span>]. Mutations in the <em xmlns="" class="EmphasisTypeItalic">G6PC</em> gene have been found to be the cause of this disease. The <em xmlns="" class="EmphasisTypeItalic">G6PC</em> comprises of five exons on chromosome 17q21, and encodes a 35 kD monomeric protein, the G6Pase catalytic subunit, which plays a role in the endoplasmic reticulum [<span class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>, <span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>]. Through Sanger sequencing and WES, we were able to identify one deleterious mutation in the proband, yet we suspected the presence of a complex SV due to the observation of Mendelian inconsistency in the family. As a result of long-read sequencing, we made a positive diagnosis of GSD-Ia on the patient and accurately identified the breakpoints of a causal SV in the other allele of the G6PC gene, which further guided genetic counseling in the family and enabled a successful preimplantation genetic diagnosis (PGD) for in vitro fertilization (IVF) on the family.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Methods" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Methods">Methods</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Methods" data-test="collapse-section"><section id="Sec3" class="Section2 RenderAsSection2">
                <h3 class="Heading">Patient characteristics</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par16" class="Para">The study was approved by the CITIC-Xiangya Reproductive and Genetics Hospital, Central South University. The collection and use of tissues followed procedures that are in accordance with ethical standards as formulated in the Helsinki Declaration, and informed consent was obtained from the study participants. The proband was a 12&nbsp;year old boy with hepatosplenomegaly and growth retardation, and was diagnosed with possible GSD-Ia in Xiangya Hospital of Central South University in 2017&nbsp;(Table <span class="InternalRef"><a href="#Tab1">1</a></span>). The parents of the proband came to Xiangya Hospital to seek help to have another child via IVF, but genetic testing by Sanger sequencing and whole-exome sequencing (WES) failed to identify a definitive genetic cause of the disease in the family. Indeed, a homozygous mutation of c.326G &gt; A (p.C109Y) in exon 2 of the G6PC gene was identified via WES and was interpreted to be likely pathogenic. However, Sanger sequencing showed that the mother is a carrier, yet the father does not carry the mutation, therefore complicating genetic counseling on the family and subsequent design of PGD on the proposed IVF.<figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 1</span>
                        <p xmlns="" class="SimplePara">Biochemical indicators of the proband suggest a probable diagnosis of GSD-Ia</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col char="–"></colgroup><colgroup><col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Parameter</p></th><th><p xmlns="" class="SimplePara">Tested value</p></th><th char="–"><p xmlns="" class="SimplePara">Reference range</p></th><th><p xmlns="" class="SimplePara">Result</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">Glucose (mmol/L)</p></td><td><p xmlns="" class="SimplePara">0.17↓</p></td><td char="–"><p xmlns="" class="SimplePara">3.6–6.1</p></td><td><p xmlns="" class="SimplePara">Abnormal</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">CO2 (mmol/L)</p></td><td><p xmlns="" class="SimplePara">7.1↓</p></td><td char="–"><p xmlns="" class="SimplePara">19–33</p></td><td><p xmlns="" class="SimplePara">Abnormal</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Sodium (mmol/L)</p></td><td><p xmlns="" class="SimplePara">132.7</p></td><td char="–"><p xmlns="" class="SimplePara">135–153</p></td><td>&nbsp;</td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Chlorinum (mmol/L)</p></td><td><p xmlns="" class="SimplePara">89.2↓</p></td><td char="–"><p xmlns="" class="SimplePara">96–108</p></td><td><p xmlns="" class="SimplePara">Abnormal</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Calcium (mmol/L)</p></td><td><p xmlns="" class="SimplePara">2.65</p></td><td char="–"><p xmlns="" class="SimplePara">2–2.6</p></td><td>&nbsp;</td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Total bile acid (μmol/L)</p></td><td><p xmlns="" class="SimplePara">16.7↑</p></td><td char="–"><p xmlns="" class="SimplePara">0–12</p></td><td><p xmlns="" class="SimplePara">Abnormal</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">ALT (IU/L)</p></td><td><p xmlns="" class="SimplePara">257.4↑</p></td><td char="–"><p xmlns="" class="SimplePara">7–56</p></td><td><p xmlns="" class="SimplePara">Abnormal</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">AST (IU/L)</p></td><td><p xmlns="" class="SimplePara">357.6↑</p></td><td char="–"><p xmlns="" class="SimplePara">0–40</p></td><td><p xmlns="" class="SimplePara">Abnormal</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Triglyceride (mmol/L)</p></td><td><p xmlns="" class="SimplePara">5.74↑</p></td><td char="–"><p xmlns="" class="SimplePara">0.52–1.56</p></td><td><p xmlns="" class="SimplePara">Abnormal</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">HDL (mmol/L)</p></td><td><p xmlns="" class="SimplePara">2.01↑</p></td><td char="–"><p xmlns="" class="SimplePara">0.88–1.76</p></td><td><p xmlns="" class="SimplePara">Abnormal</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">Abbreviations: <em class="EmphasisTypeItalic">ALT</em> alanine aminotransferase, <em class="EmphasisTypeItalic">AST</em> aspartate aminotransferase, <em class="EmphasisTypeItalic">HDL</em> high-density lipoproteins</p>
                    </div></figure></div>
              </section><section id="Sec4" class="Section2 RenderAsSection2">
                <h3 class="Heading">Long-read sequencing by Oxford Nanopore technology</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par17" class="Para">Due to the presence of Mendelian inconsistency on the c.326G &gt; A mutation, and that sperm sequencing on the father was inconclusive, we suspected that a SV may have encompassed the exon but with breakpoints in non-exonic regions, evading detection by whole-exome sequencing. Due to the complex genomic architecture around this gene, we decided to sequence the patient by low-coverage long-read sequencing on the Oxford Nanopore sequencing platform. Genomic DNA was extracted and large insert-size libraries were created according to the manufacturer recommended protocols (Oxford Nanopore, UK). Five μg genomic DNA was sheared to ~ 5-25&nbsp;kb fragments using Megaruptor (Diagenode, B06010002), size selected (10-30&nbsp;kb) with a BluePippin (Sage Science, MA) to ensure removal of small DNA fragments. Subsequently, genomic libraries were prepared using the Ligation sequencing 1D kit SQK-LSK108 (Oxford Nanopore, UK). End-repair and dA-tailing of DNA fragments according to protocol recommendations was performed using the Ultra II End Prep module (NEB, E7546L). At last, the purified dA tailed sample, blunt/TA ligase master mix (#M0367, NEB), tethered 1D adapter mix using SQK-LSK108 were incubated and purified. Libraries were sequenced on R9.4 flowcells using GridION X5. Four GridION flowcells generated 2,251,269 base-called reads containing 35,595,548,336 bases with an average read length of 16,579&nbsp;bp. We used NGMLR [<span class="CitationRef"><a href="#CR31" data-toggle="citation">31</a></span>] to align the long reads to the human reference genome (GRCh37). Structural variations (SVs) were called by Sniffles [<span class="CitationRef"><a href="#CR31" data-toggle="citation">31</a></span>] and single nucleotide variants (SNVs) were called by SAMtools [<span class="CitationRef"><a href="#CR32" data-toggle="citation">32</a></span>] for comparison to WES results. Ribbon [<span class="CitationRef"><a href="#CR33" data-toggle="citation">33</a></span>] and IGV [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>] were used to visualize the alignment results and manually validate possible SV calls.</p>
              </section><section id="Sec5" class="Section2 RenderAsSection2">
                <h3 class="Heading">Sanger sequencing and quantitative real-time PCR</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par18" class="Para">Sanger sequencing and quantitative real-time PCR (RT-PCR) were conducted to validate the long-read sequencing result in the family. Genomic DNA was extracted from peripheral leukocytes using QIAamp® DNA Blood Mini Kit (Qiagen, Germany). The exons of the G6PC gene as well as the potential breakpoint junction were amplified using PCR primers (Table&nbsp;<span class="InternalRef"><a href="#Tab2">2</a></span>). Each PCR reaction was performed in a total volume of 40&nbsp;μl containing 20&nbsp;μl of GoTaq® 2 × Green Master Mix, 0.8&nbsp;μl of 10&nbsp;mM/l mixture of forward and reverse primers, 1.5&nbsp;μl of 80&nbsp;ng/μl genomic DNA template, and 17.6&nbsp;μl of nuclease-free water. The PCR was conducted under the following cycling conditions: 95&nbsp;°C for 5&nbsp;min, 35&nbsp;cycles of denaturation at 95&nbsp;°C for 30&nbsp;s, annealing at 58&nbsp;°C for 30&nbsp;s, and elongation at 72&nbsp;°C for 30&nbsp;s followed by a final elongation of 5 mins, and PCR products were sequenced using Sanger sequencing. The exons of the G6PC gene were also quantitatively tested using quantitative RT-PCR (Table <span class="InternalRef"><a href="#Tab2">2</a></span>). Data were analyzed using the <span class="stix">△</span><span class="stix">△</span>CT method.<figure xmlns="" id="Tab2" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 2</span>
                        <p xmlns="" class="SimplePara">Primers for Sanger sequencing. No.1–10: Exon primers; No.11–20: Quantitative real-time primers; No.21–26: Breakpoint primers</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">No.</p></th><th><p xmlns="" class="SimplePara">Primer</p></th><th><p xmlns="" class="SimplePara">Position</p></th><th><p xmlns="" class="SimplePara">Sequence (5′-3′)</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">G6PC-1F</p></td><td rowspan="2"><p xmlns="" class="SimplePara">Exon1</p></td><td><p xmlns="" class="SimplePara">CACCACCAAGCCTGGAATAAC</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">2</p></td><td><p xmlns="" class="SimplePara">G6PC-1R</p></td><td><p xmlns="" class="SimplePara">CAGACATTGCGAGAGCGAATG</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">3</p></td><td><p xmlns="" class="SimplePara">G6PC-2F</p></td><td rowspan="2"><p xmlns="" class="SimplePara">Exon2</p></td><td><p xmlns="" class="SimplePara">GCATTCATTCAGTAACCC</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">4</p></td><td><p xmlns="" class="SimplePara">G6PC-2R</p></td><td><p xmlns="" class="SimplePara">AGACAGAAGCTGAGTGGA</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">5</p></td><td><p xmlns="" class="SimplePara">G6PC-3F</p></td><td rowspan="2"><p xmlns="" class="SimplePara">Exon3</p></td><td><p xmlns="" class="SimplePara">CACCTTTACTCCATTCTCTTTC</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">6</p></td><td><p xmlns="" class="SimplePara">G6PC-3R</p></td><td><p xmlns="" class="SimplePara">GTGCCACAACTCTTAATCAGCG</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">7</p></td><td><p xmlns="" class="SimplePara">G6PC-4F</p></td><td rowspan="2"><p xmlns="" class="SimplePara">Exon4</p></td><td><p xmlns="" class="SimplePara">CACTGAGAGCACCTAAGTTTGC</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">8</p></td><td><p xmlns="" class="SimplePara">G6PC-4R</p></td><td><p xmlns="" class="SimplePara">CTGATTACACACAGGATGTGG</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">9</p></td><td><p xmlns="" class="SimplePara">G6PC-5F</p></td><td rowspan="2"><p xmlns="" class="SimplePara">Exon5</p></td><td><p xmlns="" class="SimplePara">CATGTCACCCACTCCTCCAAAC</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">10</p></td><td><p xmlns="" class="SimplePara">G6PC-5R</p></td><td><p xmlns="" class="SimplePara">GTCACTTGCTCCAAATACCAGTG</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">11</p></td><td><p xmlns="" class="SimplePara">G6PC-1ForF</p></td><td rowspan="2"><p xmlns="" class="SimplePara">5′-Flanking introns</p></td><td><p xmlns="" class="SimplePara">TTTCACAGTCCTCCGTGACC</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">12</p></td><td><p xmlns="" class="SimplePara">G6PC-1ForR</p></td><td><p xmlns="" class="SimplePara">AGGGCTTCTATATCTTGAGCTTTC</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">13</p></td><td><p xmlns="" class="SimplePara">G6PC-1QF</p></td><td rowspan="2"><p xmlns="" class="SimplePara">Exon1</p></td><td><p xmlns="" class="SimplePara">TCCAGTCAACACATTACCTCCA</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">14</p></td><td><p xmlns="" class="SimplePara">G6PC-1QR</p></td><td><p xmlns="" class="SimplePara">TAAAGACGAGGTTGAGCCAGTC</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">G6PC-2inF</p></td><td rowspan="2"><p xmlns="" class="SimplePara">Intron2–3</p></td><td><p xmlns="" class="SimplePara">AAGTTGGGACAAGGGAATCAGA</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">16</p></td><td><p xmlns="" class="SimplePara">G6PC-2inR</p></td><td><p xmlns="" class="SimplePara">CATTCTTAATTCCTCTACCCTGAGA</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">17</p></td><td><p xmlns="" class="SimplePara">G6PC-4QF</p></td><td rowspan="2"><p xmlns="" class="SimplePara">Exon4</p></td><td><p xmlns="" class="SimplePara">GCTGAAGGATCTGCACCTGT</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">18</p></td><td><p xmlns="" class="SimplePara">G6PC-4QR</p></td><td><p xmlns="" class="SimplePara">AGGGAGTCAGATCAGCCCAT</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">19</p></td><td><p xmlns="" class="SimplePara">G6PC-5QF</p></td><td rowspan="2"><p xmlns="" class="SimplePara">Exon5</p></td><td><p xmlns="" class="SimplePara">CAGCTTCGCCATCGGATTTT</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">20</p></td><td><p xmlns="" class="SimplePara">G6PC-5QR</p></td><td><p xmlns="" class="SimplePara">ACAATAGAGCTGAGGCGGAA</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">21</p></td><td><p xmlns="" class="SimplePara">G6PC-D1F</p></td><td>&nbsp;</td><td><p xmlns="" class="SimplePara">GTGGGGAAAATGCCTGAGGA</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">22</p></td><td><p xmlns="" class="SimplePara">G6PC-D2F</p></td><td>&nbsp;</td><td><p xmlns="" class="SimplePara">TTTTCACCCTTGGGAGCCTG</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">23</p></td><td><p xmlns="" class="SimplePara">G6PC-D3F</p></td><td>&nbsp;</td><td><p xmlns="" class="SimplePara">GGTCACCCTGTCCCACTAGA</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">24</p></td><td><p xmlns="" class="SimplePara">G6PC-D4F</p></td><td>&nbsp;</td><td><p xmlns="" class="SimplePara">CTCACCTGTTTTCCCACGGA</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">25</p></td><td><p xmlns="" class="SimplePara">G6PC-D5F</p></td><td>&nbsp;</td><td><p xmlns="" class="SimplePara">GGGAGGAGACTCCAGGTCAT</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">26</p></td><td><p xmlns="" class="SimplePara">G6PC-comR</p></td><td><p xmlns="" class="SimplePara">Intron2–3</p></td><td><p xmlns="" class="SimplePara">CTTTCCAGTCTGTGCCTCCAT</p></td></tr></tbody></table></div></figure></div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par19" class="Para">We further designed an easy assay to detect the deletion reliably by PCR, and performed nucleic acid gel electrophoresis and used the ß-Globin gene as an internal control to validate this assay. The amplification primers for the ß-globin gene are: Primer ß-F: 5’-TGAGTCTATGGGACGCTTGA-3′ and Primer ß-R: 5’-ATCCAGCCTTATCCCAACC-3′. The primers designed to amplify the sequence near the breakpoint were G6PC-DEL-F: 5’-GAGTTAGAAGGAGATGGCGGG-3′ and G6PC-DEL-R: 5’-GGCCTATCCTACATATTAATAGTT-3′ which generates a target fragment of 418&nbsp;bp.</p>
              </section><section id="Sec6" class="Section2 RenderAsSection2">
                <h3 class="Heading">Assessment of pathogenicity of the mutations</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par20" class="Para">We analyzed the WES data through the variant filtering pipelines implemented in the ANNOVAR software [<span class="CitationRef"><a href="#CR35" data-toggle="citation">35</a></span>], by focusing on coding variants and by removing common variants observed in the gnomAD database. We identified a novel and homozygous missense mutation (c.326G &gt; A; p.C109Y) in the G6PC gene. The novel missense mutation has not been recorded in the HGMD version 2017.4 (<span class="ExternalRef"><a href="http://www.hgmd.cf.ac.uk/ac/index.php"><span class="RefSource">http://www.hgmd.cf.ac.uk/ac/index.php</span></a></span>) or in the dbSNP (<span class="ExternalRef"><a href="https://www.ncbi.nlm.nih.gov/snp/"><span class="RefSource">https://www.ncbi.nlm.nih.gov/snp/</span></a></span>) database. According to the ACMG-AMP 2015 Standards and Guidelines [<span class="CitationRef"><a href="#CR36" data-toggle="citation">36</a></span>], and facilitated by the InterVar software tool [<span class="CitationRef"><a href="#CR37" data-toggle="citation">37</a></span>], we analyzed the pathogenicity of the novel mutation and determined that it is a likely pathogenic mutation responsible for the disease manifestation. No other candidate genes were found in our WES analysis that may explain the observed phenotypes of the proband.</p>
              </section><section id="Sec7" class="Section2 RenderAsSection2">
                <h3 class="Heading">In vitro fertilization (IVF) and pre-implementation genetic diagnosis (PGD)</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par21" class="Para">The parents of the proband had genetic counseling at the Reproductive &amp; Genetic Hospital of CITIC-Xiangya, and proceeded with in vitro fertilization (IVF). A total of four oocytes were retrieved; all were in metaphase II, and all of them were inseminated (day 0) by intracytoplasmic sperm injection. The embryos were cultured to blastocyst stage, and 2–3 zona pellucida cells were used to detect mutations in the G6PC gene, and the embryos were also scored according to the Istanbul consensus [<span class="CitationRef"><a href="#CR38" data-toggle="citation">38</a></span>] (Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>: Table S1). PGD was performed using assays designed for detecting both the missense mutation and the exonic deletion in G6PC. The best embryo scored by the Istanbul consensus [<span class="CitationRef"><a href="#CR38" data-toggle="citation">38</a></span>] was selected for implementation after PGD.</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec8" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Results" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Results">Results</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Results" data-test="collapse-section"><section id="Sec9" class="Section2 RenderAsSection2">
                <h3 class="Heading">Clinical examination</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par22" class="Para">We were presented with a 12&nbsp;year-old boy with hepatosplenomegaly and growth retardation at the Xiangya Hospital of Central South University, Hunan, China in 2017 (Fig.&nbsp;<span class="InternalRef"><a href="#Fig1">1a</a></span>). The clinical features include a rounded doll’s face, fatty cheeks and protuberant abdomen (Fig. <span class="InternalRef"><a href="#Fig1">1b</a></span>). Based on examination on the proband’s skeletal development by X-rays, his hip and wrist showed osteoporosis (Fig. <span class="InternalRef"><a href="#Fig1">1c</a></span>, <span class="InternalRef"><a href="#Fig1">d</a></span>). In the upper liver, right intercostal midline sixth intercostal, liver rib length is 51&nbsp;mm, thickness is 37&nbsp;mm. Maximum oblique diameter of right liver is 159&nbsp;mm, suggesting severe liver enlargement (Fig. <span class="InternalRef"><a href="#Fig1">1d</a></span>). Spleen is swollen to a thickness of 34&nbsp;mm.<figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO1">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0069-1/MediaObjects/41065_2018_69_Fig1_HTML.png"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0069-1/MediaObjects/41065_2018_69_Fig1_HTML.png" alt="Fig. 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0069-1/MediaObjects/41065_2018_69_Fig1_HTML.png"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0069-1/MediaObjects/41065_2018_69_Fig1_HTML.png" alt="Fig. 1"></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 1</span>
                        <p xmlns="" class="SimplePara">Clinical characteristics of the proband. (<strong class="EmphasisTypeBold">a</strong>) Pedigree of the family. III:3 represents the proband, whose older brother (III:2) has decreased. (<strong class="EmphasisTypeBold">b</strong>) The clinical features include a rounded doll’s face, fatty cheeks and protuberant abdomen. (<strong class="EmphasisTypeBold">c</strong>) X-ray films of the whole body of the patient. White arrows mark areas with obvious osteoporosis. (<strong class="EmphasisTypeBold">d</strong>) Focused view of X-ray film on the hand of the proband, where the wrist marked by white arrows has obvious osteoporosis. (<strong class="EmphasisTypeBold">e</strong>) Image of type-B ultrasonic on the proband shows severe liver enlargement. Blue color: The blood flow away from the detector of ultrasound B-mode scanner; Red color: The blood flow to the detector of ultrasound B-mode scanner</p>
                      </div></figcaption></figure></div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par23" class="Para">Additional biochemical assays were performed on the proband (Table <span class="InternalRef"><a href="#Tab1">1</a></span>). His fasting blood glucose value was 0.17&nbsp;mmol/l, which had reached a dangerously low value. Levels of cholesterol, triglycerides and chlorinum were abnormal. Investigation of liver function showed elevated aminotransferases (AST, ALT) and other biochemical abnormalities. Serum copper and ceruloplasmin were at normal levels. Altogether, these clinical and biochemical examinations indicated that the boy is likely to be affected with GSD-Ia.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par24" class="Para">The parents were non-consanguineous, and neither has any symptom of GSD-Ia (Fig. <span class="InternalRef"><a href="#Fig1">1a</a></span>). They came to Xiangya Hospital to seek help to obtain a genetic diagnosis and plan to have another child by in vitro fertilization (IVF). Given that GSD-Ia is a recessive disease, we hypothesized that the proband carries bi-allelic <em xmlns="" class="EmphasisTypeItalic">G6PC</em> mutations inherited from the father and mother, respectively. Yielding a confirmed genetic diagnosis and determining the exact disease causal variants are necessary to perform preimplantation genetic diagnosis (PGD) from in vitro fertilization (IVF).</p>
              </section><section id="Sec10" class="Section2 RenderAsSection2">
                <h3 class="Heading">Whole-exome and sanger sequencing identified one pathogenic variant</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par25" class="Para">To confirm the clinical diagnosis, we conducted clinical exome sequencing on the proband. A novel missense mutation of the <em xmlns="" class="EmphasisTypeItalic">G6PC</em> gene (c.326G &gt; A) was identified by WES (Fig.&nbsp;<span class="InternalRef"><a href="#Fig2">2a</a></span>), which appears to be homozygous and affects a highly conserved position in the protein sequence (Fig. <span class="InternalRef"><a href="#Fig2">2b</a></span>). Bioinformatics analysis by InterVar [<span class="CitationRef"><a href="#CR37" data-toggle="citation">37</a></span>] and manual examination of the ACMG-AMP 2015 guidelines [<span class="CitationRef"><a href="#CR36" data-toggle="citation">36</a></span>] determined the mutation to be likely pathogenic. We further validated the mutation by Sanger sequencing on all family members (Fig. <span class="InternalRef"><a href="#Fig2">2c</a></span>). However, the mutation was not detected in the father and was present in a heterozygous state in the mother (Fig. <span class="InternalRef"><a href="#Fig2">2c</a></span>). In order to examine whether germline mosaicism is present in the father, the DNA of the father’s sperm was sequenced, but the results were largely inconclusive as a small peak for A allele and an even smaller peak for C allele is present at the c.326 position (Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>: Figure S1).<figure xmlns="" class="Figure" id="Fig2"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO2">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0069-1/MediaObjects/41065_2018_69_Fig2_HTML.png"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0069-1/MediaObjects/41065_2018_69_Fig2_HTML.png" alt="Fig. 2" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0069-1/MediaObjects/41065_2018_69_Fig2_HTML.png"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0069-1/MediaObjects/41065_2018_69_Fig2_HTML.png" alt="Fig. 2"></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 2</span>
                        <p xmlns="" class="SimplePara">Identification of a c.326G &gt; A missense mutation in the <em class="EmphasisTypeItalic">G6PC</em> gene. (<strong class="EmphasisTypeBold">a</strong>) Whole-exome sequencing identified a homozygous c.326G &gt; A missense variant in exon 2 of the <em class="EmphasisTypeItalic">G6PC</em> gene. (<strong class="EmphasisTypeBold">b</strong>) The amino acid 109 (marked by red color) affected by c.326G &gt; A is highly conserved across different species. (<strong class="EmphasisTypeBold">c</strong>) Sanger sequencing on the pedigree showed that the father does not carry the c.326G &gt; A missense variant and that the mother carries a heterozygous c.326G &gt; A missense variant</p>
                      </div></figcaption></figure></div>
              </section><section id="Sec11" class="Section2 RenderAsSection2">
                <h3 class="Heading">Long-read sequencing identified a structural variant in the other allele</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par26" class="Para">To evaluate whether a structural variant is present in the proband that masks the c.326G &gt; A mutation as homozygous, we carried out long-read whole-genome sequencing on the proband using the Oxford Nanopore technology. We generated 2,251,269 base-called reads containing 35,595,548,336 bases (~12X whole-genome coverage) with an average read length of 16,579&nbsp;bp. Using the long-read sequencing data, a novel deletion on chr17 g.41049904_41057049del7146 (GRCh37) was detected in one allele of <em xmlns="" class="EmphasisTypeItalic">G6PC</em> (Fig.&nbsp;<span class="InternalRef"><a href="#Fig3">3a</a></span>), and the known heterozygous point mutation c.326G &gt; A was detected in the other allele. This deletion was supported by four reads, though with slightly discordant breakpoints due to possible alignment errors. The deletion completely covers the first two exons of the <em xmlns="" class="EmphasisTypeItalic">G6PC</em> gene, thus resulting in loss of function.<figure xmlns="" class="Figure" id="Fig3"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO3">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0069-1/MediaObjects/41065_2018_69_Fig3_HTML.png"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0069-1/MediaObjects/41065_2018_69_Fig3_HTML.png" alt="Fig. 3" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0069-1/MediaObjects/41065_2018_69_Fig3_HTML.png"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0069-1/MediaObjects/41065_2018_69_Fig3_HTML.png" alt="Fig. 3"></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 3</span>
                        <p xmlns="" class="SimplePara">Long-read sequencing identified a deletion in the <em class="EmphasisTypeItalic">G6PC</em> gene. (<strong class="EmphasisTypeBold">a</strong>) IGV screen shot of reads at the <em class="EmphasisTypeItalic">G6PC</em> locus. Four reads carry a deletion (chr17 g.41049904_41057049del7125 that starts from the first intron of the <em class="EmphasisTypeItalic">LINC00671</em> gene to intron 2 of the <em class="EmphasisTypeItalic">G6PC</em> gene. (<strong class="EmphasisTypeBold">b</strong>) Quantitative PCR validation of the deletion in the trio. Relative quantitation (RQ) of copy number was analyzed by the ΔΔCT method, and error bars represent standard deviation. The deletion includes exon 1F (5′-Flanking introns), exon 1 and exon 2, and the patient and his father are mutation carriers while his mother is normal. (<strong class="EmphasisTypeBold">c</strong>) Sanger validation of the deletion breakpoints. The first sequence shows the mutated genomic segment, while the second and third sequences show expected genomic segments if deletion is not present. The red arrow refers to the breakpoint, and a 7125&nbsp;bp sequence is deleted based on the human reference genome (GRCh37). (<strong class="EmphasisTypeBold">d</strong>) Depiction of the protein domains that were targeted by the non-synonymous mutation and the 7.1&nbsp;kb deletion. (<strong class="EmphasisTypeBold">e</strong>) Illustration of the genomic contexts of the two breakpoints, which are both located in known Alu elements. (<strong class="EmphasisTypeBold">f</strong>) Gel electrophoresis of the PCR product designed to detect the deletion. The lane marked with M represent GeneRuler 50&nbsp;bp DNA Ladder (Thermo Scientific™), and all lanes (except “-“lane) include an ~ 800&nbsp;bp internal control (β-Globin gene). A 418&nbsp;bp fragment can be amplified from the father and the proband</p>
                      </div></figcaption></figure></div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par27" class="Para">By quantitative RT-PCR, we estimated the copy number of each exon of the <em xmlns="" class="EmphasisTypeItalic">G6PC</em> gene from the proband and his parents (Fig. <span class="InternalRef"><a href="#Fig3">3b</a></span>). The copy numbers of deletion exon 1F (5′-Flanking region), exon 1 and exon 2 were only about half of the value of the control group, which indicated that the patient and his father both carry a heterozygous deletion.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par28" class="Para">To further clarify the location of the breakpoint of this large deletion, we designed six breakpoint primers (Table <span class="InternalRef"><a href="#Tab1">1</a></span>) to amplify and sequence the suspected breakpoint region. The result confirmed our prediction and found the breakpoints precisely at chr17:41049879 and chr17:41057003 (Fig. <span class="InternalRef"><a href="#Fig3">3c</a></span>). These breakpoints were only 20-50&nbsp;bp different from the predictions from the long-read sequencing data. This large deletion (chr17 g.41049879_41057003del7125) was thus 7125&nbsp;bp in length and contains the 5′ regulatory sequence as well as exon 1, intron 1, exon 2 and partial intron 2 of the <em xmlns="" class="EmphasisTypeItalic">G6PC</em> gene (Fig. <span class="InternalRef"><a href="#Fig3">3d</a></span>). Motif analysis was performed and found that the 5′ breakpoint was located in AluJr and the 3′ breakpoint in AluSx (Fig. <span class="InternalRef"><a href="#Fig3">3e</a></span>). The Alu family members have a sequence similarity of over 87% and cover 11% of the human genome [<span class="CitationRef"><a href="#CR39" data-toggle="citation">39</a></span>], and Alu-Alu recombination usually produces a fragment deletion via Alu recombination-mediated deletion (ARMD) [<span class="CitationRef"><a href="#CR40" data-toggle="citation">40</a></span>, <span class="CitationRef"><a href="#CR41" data-toggle="citation">41</a></span>]. Note that WES missed this deletion and judged the heterozygous missense mutation as homozygous, since we were unable to observe any obvious coverage differences between the several exons in the gene from WES (Fig. <span class="InternalRef"><a href="#Fig2">2a</a></span>). Therefore, the patient inherited the missense mutation (c.326G &gt; A) from his mother and inherited the deletion mutation (chr17 g.41049879_41057003del7125) from his father. We further designed a PCR-based assay to easily differentiate samples with and without deletion using a set of primers (G6PC-Del-F/G6PC-Del-R), which generates a 418&nbsp;bp target fragments in individuals carrying the deletion (Fig. <span class="InternalRef"><a href="#Fig3">3f</a></span>).</p>
              </section><section id="Sec12" class="Section2 RenderAsSection2">
                <h3 class="Heading">Preimplantation genetic diagnosis and implantation of the embryo</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par29" class="Para">In order to help the family to plan for another child, a reproductive intervention was carried out by in vitro fertilization with preimplantation genetic diagnosis (PGD). To avoid the allelic drop-out (ADO), microsatellite markers D17S760 (location 5′ of G6PC, − 0.9&nbsp;M), DS17S793 (location 5′ of G6PC, − 0.7&nbsp;M) and DS17S951 (location 3′ of G6PC, 0.8&nbsp;M) in linkage with the breakpoint junction and the point mutations were tested. Four embryos developed to blastocysts, and were collected and biopsied. The deletion mutation (chr17 g.41049879_41057003del7125) was not detected in four embryos, and we ruled out the possibility of allelic drop-out via linkage analysis by microsatellite markers. However, all the four embryos were identified to be carriers of the missense mutation (c.326G &gt; A). Data on the STR sites D17S760, DS17S793 and DS17S951 also indicated that the embryo inherited the maternal risk chromosome (Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>:&nbsp;Table S2). Furthermore, three alleles of D17S760 were found in embryo No. 2, suggesting that it may be partial trisomy in chromosome 17. Considering the state of the four embryos comprehensively, embryo No. 1 was implanted (Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>: Table S1).</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par30" class="Para">After the implantation of the embryo, the patient’s mother succeeded in pregnancy and came to our hospital for a review at the 19+ week of pregnancy. We obtained amniotic fluid cells of the fetus to extract DNA. The genetic testing confirmed that the fetus is a carrier of the missense mutation (c.326G &gt; A), and does not inherit the deleterious deletion. The newborn was revisited in our hospital in December 2017. The baby had a fasting blood glucose level of 5.5&nbsp;mmol/L and her B-ultrasonogram showed that the liver and kidneys were normal (Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>: Figure S2), confirming that she was not affected with GSD-Ia.</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec13" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Discussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Discussion">Discussion</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Discussion" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par31" class="Para">In the current study, we performed genetic diagnosis on an affected subject with suspected GSD-Ia through clinical whole-exome sequencing and long-read whole-genome sequencing. The proband was misidentified as a homozygote for the c.326G &gt; A mutation by WES, but later confirmed to be a compound heterozygous carrier of the c.326G &gt; A mutation and a 7.1&nbsp;kb deletion spanning this point mutation. The missense mutation and the deletion were inherited from the mother and father, respectively. Therefore, through combined exome sequencing and long-read whole-genome sequencing, we yielded a definitive genetic diagnosis on the proband, and used this information to design assays to enable successful personalized preimplantation genetic diagnosis following IVF.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par32" class="Para">After we identified the two causal mutations, we also retrospectively examined the exome sequencing data to understand why the deletion was not found previously. All exons in the <em xmlns="" class="EmphasisTypeItalic">G6PC</em> gene were covered well in the exome data: the mean depth for exon 1, exon 2, exon 3, exon 4 and exon 5 were 123, 74, 119, 190 and 259, respectively. However, due to the large variability of coverage between exons, and due to the presence of two intronic breakpoints, we did not determine that a deletion covering exon 1 and exon 2 was present from the short-read sequencing data. If we had performed qPCR assays on each exon after WES, we could have identified the deletion that covers exon 1 and 2; nevertheless, we still cannot find the exact breakpoint from WES or qPCR data, yet knowing the exact breakpoint will be important for the purpose of PGD.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par33" class="Para">The human <em xmlns="" class="EmphasisTypeItalic">G6PC</em> gene is a single-copy gene that contains five exons and spans 12.5&nbsp;kb of DNA on chromosome 17q21 [<span class="CitationRef"><a href="#CR42" data-toggle="citation">42</a></span>]. The <em xmlns="" class="EmphasisTypeItalic">G6PC</em> gene encodes G6Pase which is a 357 amino acid protein anchored to the endoplasmic reticulum (ER) membrane with nine transmembrane domains [<span class="CitationRef"><a href="#CR43" data-toggle="citation">43</a></span>]. The amino-terminus of the protein lies in the ER lumen with the enzymatic active site and the carboxyl-terminus in the cellular cytoplasm [<span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>, <span class="CitationRef"><a href="#CR44" data-toggle="citation">44</a></span>]. Based on the predicted structure on UniProt (<span class="ExternalRef"><a href="http://www.uniprot.org/uniprot/P35575"><span class="RefSource">http://www.uniprot.org/uniprot/P35575</span></a></span>), the novel point mutation (c.326G &gt; A; p.C109Y) detected in this study is located in the lumen of the ER and the deletion fragment (chr17:g.41049879_41057003del7125) contains at least five transmembrane domains. Since the deletion spans the starting codon of the protein, the allele carrying the deletion should not to be transcribed or translated.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par34" class="Para">To date, differential diagnosis of GSD generally relies on the molecular analysis and has replaced the traditional liver biopsy [<span class="CitationRef"><a href="#CR28" data-toggle="citation">28</a></span>, <span class="CitationRef"><a href="#CR44" data-toggle="citation">44</a></span>]. Detection and analysis of suspicious disease-causing mutations have become a powerful tool for differential diagnosis of GSD and can guide the implementation of PGD and the personalized treatment even further. WES has gradually become a powerful means by which clinicians and scientists can detect the underlying cause of various genetic diseases. However, a major shortcoming of WES is uneven coverage of sequence reads over the capture regions, contributing to many low coverage regions, which hinders accurate variant calling [<span class="CitationRef"><a href="#CR45" data-toggle="citation">45</a></span>]. This challenge did not affect our study per se, since we were able to identify the c.326G &gt; A mutation from WES accurately, but the uneven coverage from exome sequencing prevented us from finding the deletion covering two exons. Nevertheless, given that this mutation is a very rare mutation (not documented in public databases) and that there is no known consanguinity in the family, it is unlikely that the proband inherits both alleles from parents. Initially we suspected that the father carries germline mosaicism, yet sperm sequencing on the father did not fully resolve the question (Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>: Figure S1). A low coverage long-read whole-genome sequencing resolved this issue. A large deletion (initially designated as chr17:g.41049904_41057049del7146 based on alignment) was detected in the proband and his father. Then we conducted the Sanger sequencing and the quantitative RT-PCR to validate this result and refine the exact position of the breakpoint junction (chr17:g.41049879_41057003del7125). The slight inconsistence of the deletion breakpoints between initial long-read sequencing and subsequent Sanger sequencing were likely due to the higher error rate of long-read sequencing and the imperfect alignment of the reads. Long-read sequencing can identify complex SVs effectively, thus compensated for the shortcomings of WES and avoided a misdiagnosis and potential failure of PGD.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec14" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Conclusion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Conclusion">Conclusion</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Conclusion" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par35" class="Para">In summary, we present one of the first examples of using long-read sequencing to identify causal yet complex structural variants (SVs) in exome-negative patients, which subsequently enabled successful personalized preimplantation genetic diagnosis. Our study suggests that long-read sequencing offers a means to discover overlooked genetic variation in patients undiagnosed or misdiagnosed by short-read sequencing, and may potentially improve diagnostic yields in clinical settings, especially when only one pathogenic mutation is found in an affected individual suspected to carry a recessive disease.</p></div></section>
          <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section ArticleNotes" id="ArticleNotes"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Notes" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Notes">Notes</a></h2><div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Notes"><aside xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ArticleNote ArticleNoteAuthorContribution"><p xmlns="" class="SimplePara">Hefan Miao and Jiapeng Zhou contributed equally to this work.</p></aside></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AbbreviationGroup Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Abbreviations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Abbreviations">Abbreviations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Abbreviations"><div class="DefinitionList">
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>ACMG:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par4" class="Para">The American College of Medical Genetics and Genomics</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>G6PC:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par5" class="Para">glucose-6-phosphatase</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>GSD:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par6" class="Para">Glycogen storage disease</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>GSD-Ia:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par7" class="Para">Glycogen storage disease type Ia</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>HGMD:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par8" class="Para">Human gene mutation database</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>IVF:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par9" class="Para">In-vitro fertilization</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>PCR:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par10" class="Para">Polymerase chain reaction</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>PGD:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par11" class="Para">Preimplantation genetic diagnosis</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>WES:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par12" class="Para">Whole exome sequencing</p>
                  </dd></dl>
              </div></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments"><h3 xmlns="" class="Heading">Acknowledgments</h3><p xmlns="" class="SimplePara">We thank the patient and his family members for participating in this study. We thank the PGD group members and the IVF group members at the Reproductive &amp; Genetic Hospital of CITIC-Xiangya, who worked together to complete the project.</p><div id="FPar1" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Funding</h3>
                <p id="Par36" class="Para">This work was supported by the National Natural Science Foundation of China (81771645), the Reproductive &amp; Genetic Hospital of CITIC-Xiangya and Central South University.</p>
              </div><div id="FPar2" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Availability of data and materials</h3>
                <p id="Par37" class="Para">The following additional data are available with the online version of this paper. Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>: Tables S1 and Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>: Figures S1-S2 are available online. All raw long-read sequencing data, short-read sequencing data, Sanger sequencing data and real-time PCR data in the genomic region of interest are available through appropriate institutional MTA. The detailed instructions on the software tools used in the analysis are also available to facilitate reproduce the results presented in the current study.</p>
              </div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="FormalPara FormalParaRenderingStyle1"><h3 xmlns="" class="Heading">Authors’ contributions</h3><p xmlns="" class="SimplePara">HM and JZ lead the experimental design, facilitated data analysis and wrote the manuscript. QY, FL generated Nanopore sequencing data and performed data analysis. NM, BG, JD, GL performed clinical assessment of the patients, generated short-read sequencing data, performed preimplantation genetic diagnosis and advised on data interpretation. KW and QZ conceived and guided the execution of the study, and revised the manuscript. All authors read and approved the final manuscript.</p></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Ethics">
              <div id="FPar3" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Ethics approval and consent to participate</h3>
                <p id="Par38" class="Para">The research including human subjects, human material, human data, has been performed in accordance with the Declaration of Helsinki and was approved by the ethics committee of Reproductive and Genetic Hospital of CITIC-Xiangya.</p>
              </div>
              <div id="FPar4" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Consent for publication</h3>
                <p id="Par39" class="Para">The research contains individual person’s data (including detailed clinical phenotypes and images) and we have obtained consent from parents of the children.</p>
              </div>
              <div id="FPar5" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Competing interests</h3>
                <p id="Par40" class="Para">J.Z., Q.Y., F.L., D.W. are employees and K.W. is consultant of Grandomics Biosciences. All the other authors have no competing interests.</p>
              </div>
              <div id="FPar6" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Publisher’s Note</h3>
                <p id="Par41" class="Para">Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
              </div>
            </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="License">
                <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Open Access</strong>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/4.0/"><span class="RefSource">http://creativecommons.org/licenses/by/4.0/</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/publicdomain/zero/1.0/"><span class="RefSource">http://creativecommons.org/publicdomain/zero/1.0/</span></a></span>) applies to the data made available in this article, unless otherwise stated.</p>
              </div><aside class="Appendix" id="App1">
              
                <h3 class="Heading">Additional files</h3><div id="widgetContainer" data-test="figshare-container"></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par42" class="Para">
                  <div class="MediaObject" id="MOESM1">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0069-1/MediaObjects/41065_2018_69_MOESM1_ESM.docx"><span class="caption-number">Additional file 1:</span></a>
                          <span xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Table S1</strong>. Characteristics of four embryos after in vitro fertilization. The four embryos were scored according to the Istanbul consensus. Embryo No.1 was transplanted. <strong class="EmphasisTypeBold">Figure S1.</strong> Sanger sequencing on the blood samples (left panel) and sperm samples (right panel) of the father, to determine whether germline mosaicism on c.326G &gt; A is present. However, the results were inconclusive, and indeed a small peak of A allele and an even smaller peak for C allele is present at the c.326 position. <strong class="EmphasisTypeBold">Figure S2.</strong> Image of type-B ultrasonic of liver on the newborn baby. B-ultrasonogram showed that the liver and kidneys were normal. SP: Spleen; LK: Left kidney. (DOCX 372 kb)</span>
                        </div></div>
                  </div>
                </div>
              
            </aside></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authors’ Affiliations</a></h2><div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1"><span class="AffiliationNumber">(1)</span><div class="AffiliationText">Institute of Reproductive and Stem Cell Engineering, Central South University, Changsha, 410078, Hunan, China</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff2"><span class="AffiliationNumber">(2)</span><div class="AffiliationText">Reproductive and Genetic Hospital of CITIC-Xiangya, Changsha, 410078, Hunan, China</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff3"><span class="AffiliationNumber">(3)</span><div class="AffiliationText">GrandOmics Biosciences, Beijing, 102206, China</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff4"><span class="AffiliationNumber">(4)</span><div class="AffiliationText">Raymond G. Perelman Center for Cellular and Molecular Therapeutics, Children’s Hospital of Philadelphia, Philadelphia, PA&nbsp;19104, USA</div></div></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper"><li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Posey JE, Rosenfeld JA, James RA, Bainbridge M, Niu Z, Wang X, Dhar S, Wiszniewski W, Akdemir ZH, Gambin T, et al. Molecular diagnostic experience of whole-exome sequencing in adult patients. Genet Med. 2016;18(7):678–85.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/gim.2015.142"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Molecular%20diagnostic%20experience%20of%20whole-exome%20sequencing%20in%20adult%20patients&amp;author=JE.%20Posey&amp;author=JA.%20Rosenfeld&amp;author=RA.%20James&amp;author=M.%20Bainbridge&amp;author=Z.%20Niu&amp;author=X.%20Wang&amp;author=S.%20Dhar&amp;author=W.%20Wiszniewski&amp;author=ZH.%20Akdemir&amp;author=T.%20Gambin&amp;journal=Genet%20Med&amp;volume=18&amp;issue=7&amp;pages=678-685&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yang Y, Muzny DM, Reid JG, Bainbridge MN, Willis A, Ward PA, Braxton A, Beuten J, Xia F, Niu Z, et al. Clinical whole-exome sequencing for the diagnosis of mendelian disorders. N Engl J Med. 2013;369(16):1502–11.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1056/NEJMoa1306555"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Clinical%20whole-exome%20sequencing%20for%20the%20diagnosis%20of%20mendelian%20disorders&amp;author=Y.%20Yang&amp;author=DM.%20Muzny&amp;author=JG.%20Reid&amp;author=MN.%20Bainbridge&amp;author=A.%20Willis&amp;author=PA.%20Ward&amp;author=A.%20Braxton&amp;author=J.%20Beuten&amp;author=F.%20Xia&amp;author=Z.%20Niu&amp;journal=N%20Engl%20J%20Med&amp;volume=369&amp;issue=16&amp;pages=1502-1511&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Retterer K, Juusola J, Cho MT, Vitazka P, Millan F, Gibellini F, Vertino-Bell A, Smaoui N, Neidich J, Monaghan KG, et al. Clinical application of whole-exome sequencing across clinical indications. Genet Med. 2016;18(7):696–704.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/gim.2015.148"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Clinical%20application%20of%20whole-exome%20sequencing%20across%20clinical%20indications&amp;author=K.%20Retterer&amp;author=J.%20Juusola&amp;author=MT.%20Cho&amp;author=P.%20Vitazka&amp;author=F.%20Millan&amp;author=F.%20Gibellini&amp;author=A.%20Vertino-Bell&amp;author=N.%20Smaoui&amp;author=J.%20Neidich&amp;author=KG.%20Monaghan&amp;journal=Genet%20Med&amp;volume=18&amp;issue=7&amp;pages=696-704&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Trujillano D, Bertoli-Avella AM, Kumar Kandaswamy K, Weiss ME, Koster J, Marais A, Paknia O, Schroder R, Garcia-Aznar JM, Werber M, et al. Clinical exome sequencing: results from 2819 samples reflecting 1000 families. Eur J Hum Genet. 2017;25(2):176–82.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/ejhg.2016.146"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Clinical%20exome%20sequencing%3A%20results%20from%202819%20samples%20reflecting%201000%20families&amp;author=D.%20Trujillano&amp;author=AM.%20Bertoli-Avella&amp;author=K.%20Kumar%20Kandaswamy&amp;author=ME.%20Weiss&amp;author=J.%20Koster&amp;author=A.%20Marais&amp;author=O.%20Paknia&amp;author=R.%20Schroder&amp;author=JM.%20Garcia-Aznar&amp;author=M.%20Werber&amp;journal=Eur%20J%20Hum%20Genet&amp;volume=25&amp;issue=2&amp;pages=176-182&amp;publication_year=2017"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lee H, Deignan JL, Dorrani N, Strom SP, Kantarci S, Quintero-Rivera F, Das K, Toy T, Harry B, Yourshaw M, et al. Clinical exome sequencing for genetic identification of rare Mendelian disorders. Jama. 2014;312(18):1880–7.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1001/jama.2014.14604"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Clinical%20exome%20sequencing%20for%20genetic%20identification%20of%20rare%20Mendelian%20disorders&amp;author=H.%20Lee&amp;author=JL.%20Deignan&amp;author=N.%20Dorrani&amp;author=SP.%20Strom&amp;author=S.%20Kantarci&amp;author=F.%20Quintero-Rivera&amp;author=K.%20Das&amp;author=T.%20Toy&amp;author=B.%20Harry&amp;author=M.%20Yourshaw&amp;journal=Jama&amp;volume=312&amp;issue=18&amp;pages=1880-1887&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gilissen C, Hehir-Kwa JY, Thung DT, van de Vorst M, van Bon BW, Willemsen MH, Kwint M, Janssen IM, Hoischen A, Schenck A et al: Genome sequencing identifies major causes of severe intellectual disability. Nature 2014, 511(7509):344–347.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nature13394"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genome%20sequencing%20identifies%20major%20causes%20of%20severe%20intellectual%20disability&amp;author=Christian.%20Gilissen&amp;author=Jayne%20Y..%20Hehir-Kwa&amp;author=Djie%20Tjwan.%20Thung&amp;author=Maartje.%20van%20de%20Vorst&amp;author=Bregje%20W.%20M..%20van%20Bon&amp;author=Marjolein%20H..%20Willemsen&amp;author=Michael.%20Kwint&amp;author=Irene%20M..%20Janssen&amp;author=Alexander.%20Hoischen&amp;author=Annette.%20Schenck&amp;author=Richard.%20Leach&amp;author=Robert.%20Klein&amp;author=Rick.%20Tearle&amp;author=Tan.%20Bo&amp;author=Rolph.%20Pfundt&amp;author=Helger%20G..%20Yntema&amp;author=Bert%20B.%20A..%20de%20Vries&amp;author=Tjitske.%20Kleefstra&amp;author=Han%20G..%20Brunner&amp;author=Lisenka%20E.%20L.%20M..%20Vissers&amp;author=Joris%20A..%20Veltman&amp;journal=Nature&amp;volume=511&amp;issue=7509&amp;pages=344-347&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Willig LK, Petrikin JE, Smith LD, Saunders CJ, Thiffault I, Miller NA, Soden SE, Cakici JA, Herd SM, Twist G, et al. Whole-genome sequencing for identification of Mendelian disorders in critically ill infants: a retrospective analysis of diagnostic and clinical findings. Lancet Respir Med. 2015;3(5):377–87.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/S2213-2600(15)00139-3"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Whole-genome%20sequencing%20for%20identification%20of%20Mendelian%20disorders%20in%20critically%20ill%20infants%3A%20a%20retrospective%20analysis%20of%20diagnostic%20and%20clinical%20findings&amp;author=LK.%20Willig&amp;author=JE.%20Petrikin&amp;author=LD.%20Smith&amp;author=CJ.%20Saunders&amp;author=I.%20Thiffault&amp;author=NA.%20Miller&amp;author=SE.%20Soden&amp;author=JA.%20Cakici&amp;author=SM.%20Herd&amp;author=G.%20Twist&amp;journal=Lancet%20Respir%20Med&amp;volume=3&amp;issue=5&amp;pages=377-387&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Schwarze K, Buchanan J, Taylor JC, Wordsworth S. Are whole-exome and whole-genome sequencing approaches cost-effective? Genet Med: A systematic review of the literature; 2018.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Are%20whole-exome%20and%20whole-genome%20sequencing%20approaches%20cost-effective%3F&amp;author=K.%20Schwarze&amp;author=J.%20Buchanan&amp;author=JC.%20Taylor&amp;author=S.%20Wordsworth&amp;publication_year=2018"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hannan AJ. Tandem repeats mediating genetic plasticity in health and disease. Nat Rev Genet. 2018;19(5):286–98.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nrg.2017.115"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Tandem%20repeats%20mediating%20genetic%20plasticity%20in%20health%20and%20disease&amp;author=AJ.%20Hannan&amp;journal=Nat%20Rev%20Genet&amp;volume=19&amp;issue=5&amp;pages=286-298&amp;publication_year=2018"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ashley EA. Towards precision medicine. Nat Rev Genet. 2016;17(9):507–22.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nrg.2016.86"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Towards%20precision%20medicine&amp;author=EA.%20Ashley&amp;journal=Nat%20Rev%20Genet&amp;volume=17&amp;issue=9&amp;pages=507-522&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wang K, Li M, Hadley D, Liu R, Glessner J, Grant SFA, Hakonarson H, Bucan M. PennCNV: an integrated hidden Markov model designed for high-resolution copy number variation detection in whole-genome SNP genotyping data. Genome Res. 2007;17:1665–74.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1101/gr.6861907"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=PennCNV%3A%20an%20integrated%20hidden%20Markov%20model%20designed%20for%20high-resolution%20copy%20number%20variation%20detection%20in%20whole-genome%20SNP%20genotyping%20data&amp;author=K.%20Wang&amp;author=M.%20Li&amp;author=D.%20Hadley&amp;author=R.%20Liu&amp;author=J.%20Glessner&amp;author=SFA.%20Grant&amp;author=H.%20Hakonarson&amp;author=M.%20Bucan&amp;journal=Genome%20Res&amp;volume=17&amp;pages=1665-1674&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Colella S, Yau C, Taylor JM, Mirza G, Butler H, Clouston P, Bassett AS, Seller A, Holmes CC, Ragoussis J. QuantiSNP: an objective Bayes hidden-Markov model to detect and accurately map copy number variation using SNP genotyping data. Nucleic Acids Res. 2007;35(6):2013–25.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gkm076"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=QuantiSNP%3A%20an%20objective%20Bayes%20hidden-Markov%20model%20to%20detect%20and%20accurately%20map%20copy%20number%20variation%20using%20SNP%20genotyping%20data&amp;author=S.%20Colella&amp;author=C.%20Yau&amp;author=JM.%20Taylor&amp;author=G.%20Mirza&amp;author=H.%20Butler&amp;author=P.%20Clouston&amp;author=AS.%20Bassett&amp;author=A.%20Seller&amp;author=CC.%20Holmes&amp;author=J.%20Ragoussis&amp;journal=Nucleic%20Acids%20Res&amp;volume=35&amp;issue=6&amp;pages=2013-2025&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Venkatraman ES, Olshen AB. A faster circular binary segmentation algorithm for the analysis of array CGH data. Bioinformatics. 2007;23(6):657–63.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btl646"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20faster%20circular%20binary%20segmentation%20algorithm%20for%20the%20analysis%20of%20array%20CGH%20data&amp;author=ES.%20Venkatraman&amp;author=AB.%20Olshen&amp;journal=Bioinformatics&amp;volume=23&amp;issue=6&amp;pages=657-663&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Olshen AB, Venkatraman ES, Lucito R, Wigler M. Circular binary segmentation for the analysis of array-based DNA copy number data. Biostatistics. 2004;5(4):557–72.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/biostatistics/kxh008"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Circular%20binary%20segmentation%20for%20the%20analysis%20of%20array-based%20DNA%20copy%20number%20data&amp;author=AB.%20Olshen&amp;author=ES.%20Venkatraman&amp;author=R.%20Lucito&amp;author=M.%20Wigler&amp;journal=Biostatistics&amp;volume=5&amp;issue=4&amp;pages=557-572&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Alkan C, Coe BP, Eichler EE. Genome structural variation discovery and genotyping. Nat Rev Genet. 2011;12(5):363–76.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nrg2958"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genome%20structural%20variation%20discovery%20and%20genotyping&amp;author=C.%20Alkan&amp;author=BP.%20Coe&amp;author=EE.%20Eichler&amp;journal=Nat%20Rev%20Genet&amp;volume=12&amp;issue=5&amp;pages=363-376&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Merker JD, Wenger AM, Sneddon T, Grove M, Zappala Z, Fresard L, Waggott D, Utiramerur S, Hou Y, Smith KS, et al. Long-read genome sequencing identifies causal structural variation in a Mendelian disease. Genet Med. 2017.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Merker%20JD%2C%20Wenger%20AM%2C%20Sneddon%20T%2C%20Grove%20M%2C%20Zappala%20Z%2C%20Fresard%20L%2C%20Waggott%20D%2C%20Utiramerur%20S%2C%20Hou%20Y%2C%20Smith%20KS%2C%20et%20al.%20Long-read%20genome%20sequencing%20identifies%20causal%20structural%20variation%20in%20a%20Mendelian%20disease.%20Genet%20Med.%202017."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kirby A, Gnirke A, Jaffe DB, Baresova V, Pochet N, Blumenstiel B, Ye C, Aird D, Stevens C, Robinson JT, et al. Mutations causing medullary cystic kidney disease type 1 lie in a large VNTR in MUC1 missed by massively parallel sequencing. Nat Genet. 2013;45(3):299–303.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/ng.2543"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Mutations%20causing%20medullary%20cystic%20kidney%20disease%20type%201%20lie%20in%20a%20large%20VNTR%20in%20MUC1%20missed%20by%20massively%20parallel%20sequencing&amp;author=A.%20Kirby&amp;author=A.%20Gnirke&amp;author=DB.%20Jaffe&amp;author=V.%20Baresova&amp;author=N.%20Pochet&amp;author=B.%20Blumenstiel&amp;author=C.%20Ye&amp;author=D.%20Aird&amp;author=C.%20Stevens&amp;author=JT.%20Robinson&amp;journal=Nat%20Genet&amp;volume=45&amp;issue=3&amp;pages=299-303&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sedlazeck FJ, Rescheneder P, Smolka M, Fang H, Nattestad M, von Haeseler A, Schatz MC: Accurate detection of complex structural variations using single-molecule sequencing. Nat Methods 2018, 15(6):461–468.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Sedlazeck%20FJ%2C%20Rescheneder%20P%2C%20Smolka%20M%2C%20Fang%20H%2C%20Nattestad%20M%2C%20von%20Haeseler%20A%2C%20Schatz%20MC%3A%20Accurate%20detection%20of%20complex%20structural%20variations%20using%20single-molecule%20sequencing.%20Nat%20Methods%202018%2C%2015%286%29%3A461%E2%80%93468."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sanchis-Juan A, Stephens J, French CE, Gleadall N, Mégy K, Penkett C, Stirrups K, Delon I, Dewhurst E, Dolling H, et al. Complex structural variants resolved by short-read and long-read whole genome sequencing in Mendelian disorders. BioRxiv. 2018. <span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.1101/281683"><span class="RefSource">https://doi.org/10.1101/281683</span></a></span>.</span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Liu Q, Zhang P, Wang D, Gu W, Wang K. Interrogating the "unsequenceable" genomic trinucleotide repeat disorders by long-read sequencing. Genome Med. 2017;9(1):65.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/s13073-017-0456-7"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Interrogating%20the%20%22unsequenceable%22%20genomic%20trinucleotide%20repeat%20disorders%20by%20long-read%20sequencing&amp;author=Q.%20Liu&amp;author=P.%20Zhang&amp;author=D.%20Wang&amp;author=W.%20Gu&amp;author=K.%20Wang&amp;journal=Genome%20Med&amp;volume=9&amp;issue=1&amp;pages=65&amp;publication_year=2017"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Loomis EW, Eid JS, Peluso P, Yin J, Hickey L, Rank D, McCalmon S, Hagerman RJ, Tassone F, Hagerman PJ. Sequencing the unsequenceable: expanded CGG-repeat alleles of the fragile X gene. Genome Res. 2013;23(1):121–8.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1101/gr.141705.112"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Sequencing%20the%20unsequenceable%3A%20expanded%20CGG-repeat%20alleles%20of%20the%20fragile%20X%20gene&amp;author=EW.%20Loomis&amp;author=JS.%20Eid&amp;author=P.%20Peluso&amp;author=J.%20Yin&amp;author=L.%20Hickey&amp;author=D.%20Rank&amp;author=S.%20McCalmon&amp;author=RJ.%20Hagerman&amp;author=F.%20Tassone&amp;author=PJ.%20Hagerman&amp;journal=Genome%20Res&amp;volume=23&amp;issue=1&amp;pages=121-128&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Shi L, Guo Y, Dong C, Huddleston J, Yang H, Han X, Fu A, Li Q, Li N, Gong S, et al. Long-read sequencing and de novo assembly of a Chinese genome. Nat Commun. 2016;7:12065.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/ncomms12065"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Long-read%20sequencing%20and%20de%20novo%20assembly%20of%20a%20Chinese%20genome&amp;author=L.%20Shi&amp;author=Y.%20Guo&amp;author=C.%20Dong&amp;author=J.%20Huddleston&amp;author=H.%20Yang&amp;author=X.%20Han&amp;author=A.%20Fu&amp;author=Q.%20Li&amp;author=N.%20Li&amp;author=S.%20Gong&amp;journal=Nat%20Commun&amp;volume=7&amp;pages=12065&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Seo JS, Rhie A, Kim J, Lee S, Sohn MH, Kim CU, Hastie A, Cao H, Yun JY, Kim J, et al. De novo assembly and phasing of a Korean human genome. Nature. 2016;538(7624):243–7.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nature20098"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20and%20phasing%20of%20a%20Korean%20human%20genome&amp;author=JS.%20Seo&amp;author=A.%20Rhie&amp;author=J.%20Kim&amp;author=S.%20Lee&amp;author=MH.%20Sohn&amp;author=CU.%20Kim&amp;author=A.%20Hastie&amp;author=H.%20Cao&amp;author=JY.%20Yun&amp;author=J.%20Kim&amp;journal=Nature&amp;volume=538&amp;issue=7624&amp;pages=243-247&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR24" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Jain M, Koren S, Miga KH, Quick J, Rand AC, Sasani TA, Tyson JR, Beggs AD, Dilthey AT, Fiddes IT, et al. Nanopore sequencing and assembly of a human genome with ultra-long reads. Nat Biotechnol. 2018;36(4):338–45.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nbt.4060"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Nanopore%20sequencing%20and%20assembly%20of%20a%20human%20genome%20with%20ultra-long%20reads&amp;author=M.%20Jain&amp;author=S.%20Koren&amp;author=KH.%20Miga&amp;author=J.%20Quick&amp;author=AC.%20Rand&amp;author=TA.%20Sasani&amp;author=JR.%20Tyson&amp;author=AD.%20Beggs&amp;author=AT.%20Dilthey&amp;author=IT.%20Fiddes&amp;journal=Nat%20Biotechnol&amp;volume=36&amp;issue=4&amp;pages=338-345&amp;publication_year=2018"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR25" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Pendleton M, Sebra R, Pang AW, Ummat A, Franzen O, Rausch T, Stutz AM, Stedman W, Anantharaman T, Hastie A, et al. Assembly and diploid architecture of an individual human genome via single-molecule technologies. Nat Methods. 2015;12(8):780–6.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nmeth.3454"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Assembly%20and%20diploid%20architecture%20of%20an%20individual%20human%20genome%20via%20single-molecule%20technologies&amp;author=M.%20Pendleton&amp;author=R.%20Sebra&amp;author=AW.%20Pang&amp;author=A.%20Ummat&amp;author=O.%20Franzen&amp;author=T.%20Rausch&amp;author=AM.%20Stutz&amp;author=W.%20Stedman&amp;author=T.%20Anantharaman&amp;author=A.%20Hastie&amp;journal=Nat%20Methods&amp;volume=12&amp;issue=8&amp;pages=780-786&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR26" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Chou JY, Mansfield BC. Mutations in the glucose-6-phosphatase-alpha (G6PC) gene that cause type Ia glycogen storage disease. Hum Mutat. 2008;29(7):921–30.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1002/humu.20772"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Mutations%20in%20the%20glucose-6-phosphatase-alpha%20%28G6PC%29%20gene%20that%20cause%20type%20Ia%20glycogen%20storage%20disease&amp;author=JY.%20Chou&amp;author=BC.%20Mansfield&amp;journal=Hum%20Mutat&amp;volume=29&amp;issue=7&amp;pages=921-930&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR27" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zheng BX, Lin Q, Li M, Jin Y. Three novel mutations of the G6PC gene identified in Chinese patients with glycogen storage disease type Ia. Eur J Pediatr. 2015;174(1):59–63.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00431-014-2354-y"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Three%20novel%20mutations%20of%20the%20G6PC%20gene%20identified%20in%20Chinese%20patients%20with%20glycogen%20storage%20disease%20type%20Ia&amp;author=BX.%20Zheng&amp;author=Q.%20Lin&amp;author=M.%20Li&amp;author=Y.%20Jin&amp;journal=Eur%20J%20Pediatr&amp;volume=174&amp;issue=1&amp;pages=59-63&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR28" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kishnani PS, Austin SL, Abdenur JE, Arn P, Bali DS, Boney A, Chung WK, Dagli AI, Dale D, Koeberl D, et al. Diagnosis and management of glycogen storage disease type I: a practice guideline of the American College of Medical Genetics and Genomics. Genetics in medicine : official journal of the American College of Medical Genetics. 2014;16(11):e1.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/gim.2014.128"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Diagnosis%20and%20management%20of%20glycogen%20storage%20disease%20type%20I%3A%20a%20practice%20guideline%20of%20the%20American%20College%20of%20Medical%20Genetics%20and%20Genomics&amp;author=PS.%20Kishnani&amp;author=SL.%20Austin&amp;author=JE.%20Abdenur&amp;author=P.%20Arn&amp;author=DS.%20Bali&amp;author=A.%20Boney&amp;author=WK.%20Chung&amp;author=AI.%20Dagli&amp;author=D.%20Dale&amp;author=D.%20Koeberl&amp;journal=Genetics%20in%20medicine%20%3A%20official%20journal%20of%20the%20American%20College%20of%20Medical%20Genetics&amp;volume=16&amp;issue=11&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR29" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Matern D, Seydewitz HH, Bali D, Lang C, Chen YT. Glycogen storage disease type I: diagnosis and phenotype/genotype correlation. Eur J Pediatr. 2002;161(Suppl 1):S10–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/BF02679989"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Glycogen%20storage%20disease%20type%20I%3A%20diagnosis%20and%20phenotype%2Fgenotype%20correlation&amp;author=D.%20Matern&amp;author=HH.%20Seydewitz&amp;author=D.%20Bali&amp;author=C.%20Lang&amp;author=YT.%20Chen&amp;journal=Eur%20J%20Pediatr&amp;volume=161&amp;issue=Suppl%201&amp;pages=S10-S19&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR30" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Chou JY, Jun HS, Mansfield BC. Type I glycogen storage diseases: disorders of the glucose-6-phosphatase/glucose-6-phosphate transporter complexes. J Inherit Metab Dis. 2015;38(3):511–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s10545-014-9772-x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Type%20I%20glycogen%20storage%20diseases%3A%20disorders%20of%20the%20glucose-6-phosphatase%2Fglucose-6-phosphate%20transporter%20complexes&amp;author=JY.%20Chou&amp;author=HS.%20Jun&amp;author=BC.%20Mansfield&amp;journal=J%20Inherit%20Metab%20Dis&amp;volume=38&amp;issue=3&amp;pages=511-519&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR31" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sedlazeck FJ, Rescheneder P, Smolka M, Fang H, Nattestad M, von Haeseler A, Schatz MC. Accurate detection of complex structural variations using single molecule sequencing. Nat Methods. 2018;15(6):461–468.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Sedlazeck%20FJ%2C%20Rescheneder%20P%2C%20Smolka%20M%2C%20Fang%20H%2C%20Nattestad%20M%2C%20von%20Haeseler%20A%2C%20Schatz%20MC.%20Accurate%20detection%20of%20complex%20structural%20variations%20using%20single%20molecule%20sequencing.%20Nat%20Methods.%202018%3B15%286%29%3A461%E2%80%93468."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR32" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, Homer N, Marth G, Abecasis G, Durbin R. Genome project data processing S: the sequence alignment/map format and SAMtools. Bioinformatics. 2009;25(16):2078–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btp352"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genome%20project%20data%20processing%20S%3A%20the%20sequence%20alignment%2Fmap%20format%20and%20SAMtools&amp;author=H.%20Li&amp;author=B.%20Handsaker&amp;author=A.%20Wysoker&amp;author=T.%20Fennell&amp;author=J.%20Ruan&amp;author=N.%20Homer&amp;author=G.%20Marth&amp;author=G.%20Abecasis&amp;author=R.%20Durbin&amp;journal=Bioinformatics&amp;volume=25&amp;issue=16&amp;pages=2078-2079&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR33" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Maria, Nattestad C-SC, Schatz MC. Ribbon: Visualizing complex genome alignments and structural variation. bioRxiv. 2016. <span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.1101/082123"><span class="RefSource">https://doi.org/10.1101/082123</span></a></span>.</span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR34" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Thorvaldsdottir H, Robinson JT, Mesirov JP. Integrative genomics viewer (IGV): high-performance genomics data visualization and exploration. Brief Bioinform. 2013;14(2):178–92.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bib/bbs017"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Integrative%20genomics%20viewer%20%28IGV%29%3A%20high-performance%20genomics%20data%20visualization%20and%20exploration&amp;author=H.%20Thorvaldsdottir&amp;author=JT.%20Robinson&amp;author=JP.%20Mesirov&amp;journal=Brief%20Bioinform&amp;volume=14&amp;issue=2&amp;pages=178-192&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR35" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wang K, Li M, Hakonarson H. ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data. Nucleic Acids Res. 2010;38(16):e164.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gkq603"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=ANNOVAR%3A%20functional%20annotation%20of%20genetic%20variants%20from%20high-throughput%20sequencing%20data&amp;author=K.%20Wang&amp;author=M.%20Li&amp;author=H.%20Hakonarson&amp;journal=Nucleic%20Acids%20Res&amp;volume=38&amp;issue=16&amp;pages=e164&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR36" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Richards S, Aziz N, Bale S, Bick D, Das S, Gastier-Foster J, Grody WW, Hegde M, Lyon E, Spector E, et al. Standards and guidelines for the interpretation of sequence variants: a joint consensus recommendation of the American College of Medical Genetics and Genomics and the Association for Molecular Pathology. Genet Med. 2015;17(5):405–24.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/gim.2015.30"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Standards%20and%20guidelines%20for%20the%20interpretation%20of%20sequence%20variants%3A%20a%20joint%20consensus%20recommendation%20of%20the%20American%20College%20of%20Medical%20Genetics%20and%20Genomics%20and%20the%20Association%20for%20Molecular%20Pathology&amp;author=S.%20Richards&amp;author=N.%20Aziz&amp;author=S.%20Bale&amp;author=D.%20Bick&amp;author=S.%20Das&amp;author=J.%20Gastier-Foster&amp;author=WW.%20Grody&amp;author=M.%20Hegde&amp;author=E.%20Lyon&amp;author=E.%20Spector&amp;journal=Genet%20Med&amp;volume=17&amp;issue=5&amp;pages=405-424&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR37" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li Q, Wang K. InterVar: clinical interpretation of genetic variants by the 2015 ACMG-AMP guidelines. Am J Hum Genet. 2017;100(2):267–80.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.ajhg.2017.01.004"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=InterVar%3A%20clinical%20interpretation%20of%20genetic%20variants%20by%20the%202015%20ACMG-AMP%20guidelines&amp;author=Q.%20Li&amp;author=K.%20Wang&amp;journal=Am%20J%20Hum%20Genet&amp;volume=100&amp;issue=2&amp;pages=267-280&amp;publication_year=2017"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR38" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Alpha Scientists in Reproductive Medicine and ESHRE Special Interest Group of Embryology. The Istanbul consensus workshop on embryo assessment: proceedings of an expert meeting. Hum Reprod. 2011;26(6):1270–83.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Alpha%20Scientists%20in%20Reproductive%20Medicine%20and%20ESHRE%20Special%20Interest%20Group%20of%20Embryology.%20The%20Istanbul%20consensus%20workshop%20on%20embryo%20assessment%3A%20proceedings%20of%20an%20expert%20meeting.%20Hum%20Reprod.%202011%3B26%286%29%3A1270%E2%80%9383."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR39" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ade C, Roy-Engel AM, Deininger PL. Alu elements: an intrinsic source of human genome instability. Current opinion in virology. 2013;3(6):639–45.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.coviro.2013.09.002"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Alu%20elements%3A%20an%20intrinsic%20source%20of%20human%20genome%20instability&amp;author=C.%20Ade&amp;author=AM.%20Roy-Engel&amp;author=PL.%20Deininger&amp;journal=Current%20opinion%20in%20virology&amp;volume=3&amp;issue=6&amp;pages=639-645&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR40" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sen SK, Han K, Wang J, Lee J, Wang H, Callinan PA, Dyer M, Cordaux R, Liang P, Batzer MA. Human genomic deletions mediated by recombination between Alu elements. Am J Hum Genet. 2006;79(1):41–53.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1086/504600"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Human%20genomic%20deletions%20mediated%20by%20recombination%20between%20Alu%20elements&amp;author=SK.%20Sen&amp;author=K.%20Han&amp;author=J.%20Wang&amp;author=J.%20Lee&amp;author=H.%20Wang&amp;author=PA.%20Callinan&amp;author=M.%20Dyer&amp;author=R.%20Cordaux&amp;author=P.%20Liang&amp;author=MA.%20Batzer&amp;journal=Am%20J%20Hum%20Genet&amp;volume=79&amp;issue=1&amp;pages=41-53&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR41" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kim S, Cho CS, Han K, Lee J. Structural variation of Alu element and human disease. Genomics Inform. 2016;14(3):70–7.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.5808/GI.2016.14.3.70"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Structural%20variation%20of%20Alu%20element%20and%20human%20disease&amp;author=S.%20Kim&amp;author=CS.%20Cho&amp;author=K.%20Han&amp;author=J.%20Lee&amp;journal=Genomics%20Inform&amp;volume=14&amp;issue=3&amp;pages=70-77&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR42" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lei KJ, Shelly LL, Pan CJ, Sidbury JB, Chou JY. Mutations in the glucose-6-phosphatase gene that cause glycogen storage disease type 1a. Science. 1993;262(5133):580–3.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1126/science.8211187"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Mutations%20in%20the%20glucose-6-phosphatase%20gene%20that%20cause%20glycogen%20storage%20disease%20type%201a&amp;author=KJ.%20Lei&amp;author=LL.%20Shelly&amp;author=CJ.%20Pan&amp;author=JB.%20Sidbury&amp;author=JY.%20Chou&amp;journal=Science&amp;volume=262&amp;issue=5133&amp;pages=580-583&amp;publication_year=1993"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR43" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Chou JY, Jun HS, Mansfield BC. Glycogen storage disease type I and G6Pase-beta deficiency: etiology and therapy. Nat Rev Endocrinol. 2010;6(12):676–88.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nrendo.2010.189"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Glycogen%20storage%20disease%20type%20I%20and%20G6Pase-beta%20deficiency%3A%20etiology%20and%20therapy&amp;author=JY.%20Chou&amp;author=HS.%20Jun&amp;author=BC.%20Mansfield&amp;journal=Nat%20Rev%20Endocrinol&amp;volume=6&amp;issue=12&amp;pages=676-688&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR44" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gu LL, Li XH, Han Y, Zhang DH, Gong QM, Zhang XX. A novel homozygous no-stop mutation in G6PC gene from a Chinese patient with glycogen storage disease type Ia. Gene. 2014;536(2):362–5.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.gene.2013.11.059"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20novel%20homozygous%20no-stop%20mutation%20in%20G6PC%20gene%20from%20a%20Chinese%20patient%20with%20glycogen%20storage%20disease%20type%20Ia&amp;author=LL.%20Gu&amp;author=XH.%20Li&amp;author=Y.%20Han&amp;author=DH.%20Zhang&amp;author=QM.%20Gong&amp;author=XX.%20Zhang&amp;journal=Gene&amp;volume=536&amp;issue=2&amp;pages=362-365&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR45" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wang Q, Shashikant CS, Jensen M, Altman NS, Girirajan S. Novel metrics to measure coverage in whole exome sequencing datasets reveal local and global non-uniformity. Sci Rep. 2017;7(1):885.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/s41598-017-01005-x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Novel%20metrics%20to%20measure%20coverage%20in%20whole%20exome%20sequencing%20datasets%20reveal%20local%20and%20global%20non-uniformity&amp;author=Q.%20Wang&amp;author=CS.%20Shashikant&amp;author=M.%20Jensen&amp;author=NS.%20Altman&amp;author=S.%20Girirajan&amp;journal=Sci%20Rep&amp;volume=7&amp;issue=1&amp;pages=885&amp;publication_year=2017"><span><span>Google Scholar</span></span></a></span></span></cite></li></ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#TheAuthors" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="TheAuthors">Copyright</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="TheAuthors"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">©&nbsp;The Author(s)&nbsp;2018</div></div></section></div>
                        


                    </article>
                </main>

                <div class="c-content-layout__side u-text-sm">
                    <aside>
                        <div class="p-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="//hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-018-0069-1" data-track="click" data-track-category="PDF Download"
            data-track-action="Click download article PDF"
            data-track-label="10.1186/s41065-018-0069-1">Download PDF</a>
        </p>
    
                            
                            
    <button type="button"
       class="c-button c-button--secondary c-button--full-width c-button--expandable"
       data-toggle="collapse"
       data-target="export-citations"
       data-test="export-citations-button"
       data-sticky-update="true"
       aria-controls="export-citations"
       aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/s41065-018-0069-1"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0069-1?format=refman&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/s41065-018-0069-1"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0069-1?format=refman&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/s41065-018-0069-1"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0069-1?format=refman&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/s41065-018-0069-1"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0069-1?format=endnote&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/s41065-018-0069-1"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0069-1?format=endnote&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/s41065-018-0069-1"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0069-1?format=endnote&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/s41065-018-0069-1"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0069-1?format=bibtex&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/s41065-018-0069-1"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0069-1?format=bibtex&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/s41065-018-0069-1"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0069-1?format=bibtex&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
    </div>

                        </div>
                        
    
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
                                <div id="article-access-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/access-count/10.1186/s41065-018-0069-1 #accessCount-container"></div>
                                <div id="citation-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/citation-count/10.1186/s41065-018-0069-1 #count-container"></div>
                                <div id="altmetric-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/altmetric/10.1186/s41065-018-0069-1 #altmetric-container"></div>
                            </ul>
                        </div>
                        
    <div id="sociaMediaShare" class="p-separator u-js-shown">
        <h3 class="p-section-title">Share this article</h3>
        <ul id="shareButtons" class="c-social-links">
            <li class="c-social-links__item">
                <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-018-0069-1">
                    <span class="u-visually-hidden">Share on Twitter</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-twitter"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-018-0069-1">
                    <span class="u-visually-hidden">Share on Facebook</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-facebook"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click"
                        data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-018-0069-1">
                    <span class="u-visually-hidden">Share on LinkedIn</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-linkedin"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-018-0069-1">
                    <span class="u-visually-hidden">Share on Weibo</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-weibo"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnGooglePlus" class="js-btnShareOnGplus" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Google+" data-track-label="10.1186/s41065-018-0069-1">
                    <span class="u-visually-hidden">Share on Google Plus</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-gplus"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-018-0069-1">
                    <span class="u-visually-hidden">Share on Reddit</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-reddit"></use>
                    </svg>
                </button>
            </li>
        </ul>
    </div>
 
                        
    
        <div class="p-separator u-js-shown">
            <h3 class="p-section-title">See updates</h3>
            <button class="c-crossmark-button" data-target="crossmark" data-test="crossmark" data-track="click" data-track-category="Article Page" data-track-action="Click CrossMark" data-track-label="10.1186/s41065-018-0069-1">
                <img id="crossmark-icon" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" alt="Crossmark logo" />
            </button>
        </div>
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
                                <li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/s41065-018-0069-1" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=738&doi=10.1186%2Fs41065-018-0069-1&volume=155&issue=1&title=Long-read+sequencing+identified+a+causal+structural+variant+in+an+exome-negative+case+and+enabled+preimplantation+genetic+diagnosis&author_name=Hefan+Miao&start_page=1&end_page=9">Order reprint</a>

                                </li>
                            </ul>
                        </div>
                    </aside>
                    
    <div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">Hereditas</span>
                        </h4>
                        <p class="c-journal-footer__issn">ISSN: 1601-5223</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
                                
                                    <li class="c-journal-footer__contact-item">Submission enquiries: <a href="https://www.editorialmanager.com/here/" target="_blank">Access here and click Contact Us</a></li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a></li>
                                
                            </ul>
                        </div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class='tracker' style='display:none' src='/track/article/10.1186/s41065-018-0069-1' alt=""/>

         
    <footer>
        <div class="ctx-publisher-footer">
    <div class="c-navbar">
        <div class="u-container">
            <div class="c-navbar__logo u-margin-right-xl">
                <a href="https://www.biomedcentral.com">
                    <img src=/static/images/bmc/logos/logo-bmc-aj-594e03e20d.svg alt="BMC logo" data-test="navbar-logo-footer">

                </a>
            </div>
            <div class="c-navbar__content">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


            </div>
        </div>
    </div>
    <div class="c-publisher-footer" data-test="publisher-footer">
        <div class="u-container">
            
            <div class="c-flex c-flex--wrap c-flex--space-between" role="navigation" data-test="publisher-footer-menu">
                <div class="c-flex" aria-label="Publisher footer links">
                    
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/login">Receive BMC newsletters</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/account">Manage article alerts</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/policies">Policies</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/accessibility">Accessibility</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/press-centre">Press center</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/contact-us">Contact us</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/jobs">Careers</a>
                                    </li>
                                
                            </ul>
                        
                    
                </div>
                <div class="u-margin-bottom-lg">
                    <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                    <ul class="c-flex u-reset-list" data-test="footer-social-links">
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://twitter.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Twitter link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Twitter page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-twitter-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://www.facebook.com/BioMedCentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Facebook link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Facebook page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-facebook-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://plus.google.com/106226337984152901734/"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Google Plus link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Google Plus page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-gplus-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="http://www.weibo.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Weibo link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Weibo page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-weibo-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </div>
            <p class="u-reset-margin">
                By using this website, you agree to our
                <a class="u-text-inherit" href="//www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
                <a class="u-text-inherit" href="//www.biomedcentral.com/privacy-statement">Privacy
                    statement</a> and
                <a class="u-text-inherit" href="//www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
                
                    <a class="optanon-toggle-display u-text-inherit" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
                
            </p>
        </div>
    </div>
</div>


        <div class="c-sn-footer c-sn-footer--neutral">
    <div class="u-container">
        <img src=/static/images/bmc/logos/logo-springernature-8bd4e8d82f.svg class="c-sn-footer__logo" alt="Springer Nature Logo" itemprop="logo" >
        <p data-test="copyright">&#169; 2018 BioMed Central Ltd unless otherwise stated. Part of
            <a href="http://www.springernature.com"
               data-track="click"
               data-track-action="Click Springer Nature link"
               data-track-category="Corporate footer"
               itemscope itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>

    </div>

    
    <script type="text/javascript" src="//cdn.pbgrd.com/bmc.js" async="async"></script>
    <noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>
    <!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker -->

<script id="springerlink-event-tracker" src="https://event-tracker.springernature.com/dist/eventTracker.js"></script>
<script type="text/javascript">
    (function(){
        var dataLayer = window.dataLayer[0];
        var doi = dataLayer.content.article.doi;
        var imprint = dataLayer.content.contentInfo.imprint;
        var bpids = dataLayer.session.authentication.authenticationID;
        var publisher;
        var pageType = dataLayer.page.category.pageType;
        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }
        var eventObject = {'content_type' : pageType,'doi' : doi};

        if (imprint === 'BioMed Central') { publisher = 'BMC' }
        else if (imprint === 'SpringerOpen') { publisher = 'SpringerOpen' }
        else { throw new Error('Event Tracker Error: The publisher has an unexpected value') }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }
        new EventTracker({'platform':publisher}).sendEvent('display', eventObject);
    })();
</script>


    </body>
</html>

