<!DOCTYPE html>
<html lang="en" class="no-js">
    <head>
        <title>Estimation of genetic diversity in Gute sheep: pedigree and microsatellite analyses of an ancient Swedish breed | Hereditas | Full Text</title>

        

    <meta name="citation_abstract" content="Breeds with small population size are in danger of an increased inbreeding rate and loss of genetic diversity, which puts them at risk for extinction. In Sweden there are a number of local breeds, native breeds which have adapted to specific areas in Sweden, for which efforts are being made to keep them pure and healthy over time. One example of such a breed is the Swedish Gute sheep. The objective of this study was to estimate inbreeding and genetic diversity of Swedish Gute sheep. Three datasets were analysed: pedigree information of the whole population, pedigree information for 100 animals of the population, and microsatellite genotypes for 94 of the 100 animals. The average inbreeding coefficient for lambs born during a six year time period (2007–2012) did not increase during that time period. The inbreeding calculated from the entire pedigree (0.038) and for a sample of the population (0.018) was very low. Sheep were more heterozygous at the microsatellite markers than expected (average multilocus heterozygosity and Ritland inbreeding estimates 1.01845 and -0.03931) and five of seven microsatellite markers were not in Hardy Weinberg equilibrium due to heterozygosity excess. The total effective population size estimated from the pedigree information was 155.4 and the average harmonic mean effective population size estimated from microsatellites was 88.3. Pedigree and microsatellite genotype estimations of inbreeding were consistent with a breeding program with the purpose of reducing inbreeding. Our results showed that current breeding programs of the Swedish Gute sheep are consistent with efforts of keeping this breed viable and these breeding programs are an example for other small local breeds in conserving breeds for the future."/>

    <meta name="description" content="Breeds with small population size are in danger of an increased inbreeding rate and loss of genetic diversity, which puts them at risk for extinction. In Sweden there are a number of local breeds, native breeds which have adapted to specific areas in Sweden, for which efforts are being made to keep them pure and healthy over time. One example of such a breed is the Swedish Gute sheep. The objective of this study was to estimate inbreeding and genetic diversity of Swedish Gute sheep. Three datasets were analysed: pedigree information of the whole population, pedigree information for 100 animals of the population, and microsatellite genotypes for 94 of the 100 animals. The average inbreeding coefficient for lambs born during a six year time period (2007–2012) did not increase during that time period. The inbreeding calculated from the entire pedigree (0.038) and for a sample of the population (0.018) was very low. Sheep were more heterozygous at the microsatellite markers than expected (average multilocus heterozygosity and Ritland inbreeding estimates 1.01845 and -0.03931) and five of seven microsatellite markers were not in Hardy Weinberg equilibrium due to heterozygosity excess. The total effective population size estimated from the pedigree information was 155.4 and the average harmonic mean effective population size estimated from microsatellites was 88.3. Pedigree and microsatellite genotype estimations of inbreeding were consistent with a breeding program with the purpose of reducing inbreeding. Our results showed that current breeding programs of the Swedish Gute sheep are consistent with efforts of keeping this breed viable and these breeding programs are an example for other small local breeds in conserving breeds for the future."/>

    <meta name="journal_id" content="41065"/>

    <meta name="dc.title" content="Estimation of genetic diversity in Gute sheep: pedigree and microsatellite analyses of an ancient Swedish breed"/>

    <meta name="dc.source" content="Hereditas 2017 154:1"/>

    <meta name="dc.format" content="text/html"/>

    <meta name="dc.publisher" content="BioMed Central"/>

    <meta name="dc.date" content="2017-01-30"/>

    <meta name="dc.type" content="OriginalPaper"/>

    <meta name="dc.language" content="En"/>

    <meta name="dc.copyright" content="2017 The Author(s)"/>

    <meta name="dc.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="dc.description" content="Breeds with small population size are in danger of an increased inbreeding rate and loss of genetic diversity, which puts them at risk for extinction. In Sweden there are a number of local breeds, native breeds which have adapted to specific areas in Sweden, for which efforts are being made to keep them pure and healthy over time. One example of such a breed is the Swedish Gute sheep. The objective of this study was to estimate inbreeding and genetic diversity of Swedish Gute sheep. Three datasets were analysed: pedigree information of the whole population, pedigree information for 100 animals of the population, and microsatellite genotypes for 94 of the 100 animals. The average inbreeding coefficient for lambs born during a six year time period (2007&#8211;2012) did not increase during that time period. The inbreeding calculated from the entire pedigree (0.038) and for a sample of the population (0.018) was very low. Sheep were more heterozygous at the microsatellite markers than expected (average multilocus heterozygosity and Ritland inbreeding estimates 1.01845 and -0.03931) and five of seven microsatellite markers were not in Hardy Weinberg equilibrium due to heterozygosity excess. The total effective population size estimated from the pedigree information was 155.4 and the average harmonic mean effective population size estimated from microsatellites was 88.3. Pedigree and microsatellite genotype estimations of inbreeding were consistent with a breeding program with the purpose of reducing inbreeding. Our results showed that current breeding programs of the Swedish Gute sheep are consistent with efforts of keeping this breed viable and these breeding programs are an example for other small local breeds in conserving breeds for the future."/>

    <meta name="prism.issn" content="1601-5223"/>

    <meta name="prism.publicationName" content="Hereditas"/>

    <meta name="prism.publicationDate" content="2017-01-30"/>

    <meta name="prism.volume" content="154"/>

    <meta name="prism.number" content="1"/>

    <meta name="prism.section" content="OriginalPaper"/>

    <meta name="prism.startingPage" content="4"/>

    <meta name="prism.copyright" content="2017 The Author(s)"/>

    <meta name="prism.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="prism.url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0026-4"/>

    <meta name="prism.doi" content="doi:10.1186/s41065-017-0026-4"/>

    <meta name="citation_pdf_url" content="https://hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-017-0026-4"/>

    <meta name="citation_fulltext_html_url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0026-4"/>

    <meta name="citation_journal_title" content="Hereditas"/>

    <meta name="citation_publisher" content="BioMed Central"/>

    <meta name="citation_issn" content="1601-5223"/>

    <meta name="citation_title" content="Estimation of genetic diversity in Gute sheep: pedigree and microsatellite analyses of an ancient Swedish breed"/>

    <meta name="citation_volume" content="154"/>

    <meta name="citation_issue" content="1"/>

    <meta name="citation_publication_date" content="2017/12"/>

    <meta name="citation_online_date" content="2017/01/30"/>

    <meta name="citation_firstpage" content="4"/>

    <meta name="citation_article_type" content="Research"/>

    <meta name="citation_fulltext_world_readable" content=""/>

    <meta name="dc.identifier" content="doi:10.1186/s41065-017-0026-4"/>

    <meta name="DOI" content="10.1186/s41065-017-0026-4"/>

    <meta name="citation_doi" content="10.1186/s41065-017-0026-4"/>

    <meta name="dc.creator" content="Christina M. Rochus"/>

    <meta name="dc.creator" content="Anna M. Johansson"/>

    <meta name="dc.subject" content="Animal Genetics and Genomics"/>

    <meta name="dc.subject" content="Evolutionary Biology"/>

    <meta name="citation_reference" content="citation_journal_title=Animal; citation_title=North European short-tailed breeds of sheep: a review; citation_author=&#211;R D&#253;rmundsson, R Ni&#380;nikowski; citation_volume=4; citation_publication_date=2010; citation_pages=1275-1282; citation_doi=10.1017/S175173110999156X; citation_id=CR1"/>

    <meta name="citation_reference" content="Domestic Animal Diversity Information System (DAD-IS), Food and Agriculture Organization of the United Nations. 2016. 
                    http://dad.fao.org/
                    
                  . Accessed 29 Aug 2016."/>

    <meta name="citation_reference" content="citation_journal_title=Mol Ecol; citation_title=Native breeds demonstrate high contributions to the molecular variation in northern European sheep; citation_author=M Tapio, I Tapio, Z Grislis, LE Holm, S Jeppsson, J Kantanen; citation_volume=14; citation_publication_date=2005; citation_pages=3951-3963; citation_doi=10.1111/j.1365-294X.2005.02727.x; citation_id=CR3"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Genet; citation_title=Microsatellite-based genetic diversity and population structure of domestic sheep in northern Eurasia; citation_author=M Tapio, M Ozerov, I Tapio, M Toro, N Marzanov, M &#262;inkulov; citation_volume=11; citation_publication_date=2010; citation_pages=76; citation_doi=10.1186/1471-2156-11-76; citation_id=CR4"/>

    <meta name="citation_reference" content="citation_journal_title=Ann Genet Sel Anim; citation_title=A possible genetic interpretation of the colour variation in the fleece of the Gotland and Goth sheep; citation_author=S Adalsteinsson, JJ Lauvergne, JG Boyazoglu, ML Ryder; citation_volume=10; citation_publication_date=1978; citation_pages=329-342; citation_id=CR5"/>

    <meta name="citation_reference" content="citation_journal_title=Anim Genet; citation_title=The use of endogenous retroviruses as markers to describe the genetic relationships among local Swedish sheep breeds; citation_author=R Mukiibi, CM Rochus, G Andersson, AM Johansson; citation_volume=46; citation_publication_date=2015; citation_pages=220-223; citation_doi=10.1111/age.12264; citation_id=CR6"/>

    <meta name="citation_reference" content="citation_title=Pedig: A Fortran package for pedigree analysis suited for large populations; citation_publication_date=2002; citation_id=CR7; citation_author=D Boichard; citation_publisher=7th World Cong Genet Appl Livest Prod"/>

    <meta name="citation_reference" content="citation_journal_title=Genet Sel Evol; citation_title=Computing inbreeding coefficients in large populations; citation_author=THE Meuwissen, Z Luo; citation_volume=24; citation_publication_date=1992; citation_pages=305-313; citation_doi=10.1186/1297-9686-24-4-305; citation_id=CR8"/>

    <meta name="citation_reference" content="citation_title=EVA: Realized and predicted optimal genetic contributions; citation_publication_date=2006; citation_id=CR9; citation_author=P Berg; citation_author=J Nielsen; citation_author=MK S&#248;rensen; citation_publisher=8th World Cong Genet Appl Livest Prod"/>

    <meta name="citation_reference" content="citation_title=Introduction to quantitative genetics; citation_publication_date=1996; citation_id=CR10; citation_author=DS Falconer; citation_author=TFC Mackay; citation_publisher=Longman Scientific &amp; Technical"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Ecol Resour; citation_title=Genepop&#8217;007: a complete re-implementation of the genepop software for Windows and Linux; citation_author=F Rousset; citation_volume=8; citation_publication_date=2008; citation_pages=103-106; citation_doi=10.1111/j.1471-8286.2007.01931.x; citation_id=CR11"/>

    <meta name="citation_reference" content="citation_journal_title=Genet Res; citation_title=Estimators for pairwise relatedness and individual inbreeding coefficients; citation_author=K Ritland; citation_volume=67; citation_publication_date=1996; citation_pages=175-185; citation_doi=10.1017/S0016672300033620; citation_id=CR12"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Ecol Resour; citation_title=Coancestry: a program for simulating, estimating and analysing relatedness and inbreeding coefficients; citation_author=J Wang; citation_volume=11; citation_publication_date=2011; citation_pages=141-145; citation_doi=10.1111/j.1755-0998.2010.02885.x; citation_id=CR13"/>

    <meta name="citation_reference" content="citation_journal_title=Evolution; citation_title=Parasite-mediated selection against inbred Soay sheep in a free-living, island population; citation_author=DW Coltman, JG Pilkington, JA Smith, JM Pemberton; citation_volume=53; citation_publication_date=1999; citation_pages=1259-1267; citation_doi=10.2307/2640828; citation_id=CR14"/>

    <meta name="citation_reference" content="citation_journal_title=Evol App; citation_title=Detecting past changes of effective population size; citation_author=N Nikolic, C Chevalet; citation_volume=7; citation_publication_date=2014; citation_pages=663-681; citation_doi=10.1111/eva.12170; citation_id=CR15"/>

    <meta name="citation_reference" content="citation_journal_title=Genome Res; citation_title=Mutations in sheep microsatellites; citation_author=AM Crawford, RP Cuthbertson; citation_volume=6; citation_publication_date=1996; citation_pages=876-879; citation_doi=10.1101/gr.6.9.876; citation_id=CR16"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=adegenet: a R package for the multivariate analysis of genetic markers; citation_author=T Jombart; citation_volume=24; citation_publication_date=2008; citation_pages=1403-1405; citation_doi=10.1093/bioinformatics/btn129; citation_id=CR17"/>

    <meta name="citation_reference" content="citation_journal_title=Heredity; citation_title=Genetic structure of European sheep breeds; citation_author=LJ Lawson Handley, K Byrne, F Santucci, S Townsend, M Taylor, MW Bruford, GM Hewitt; citation_volume=99; citation_publication_date=2007; citation_pages=620-631; citation_doi=10.1038/sj.hdy.6801039; citation_id=CR18"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Ecol; citation_title=Comparison of microsatellite and blood protein diversity in sheep: inconsistencies in fragmented breeds; citation_author=M Tapio, I Miceikien&#233;, J Vikki, J Kantanen; citation_volume=12; citation_publication_date=2003; citation_pages=2045-2056; citation_doi=10.1046/j.1365-294X.2003.01893.x; citation_id=CR19"/>

    <meta name="citation_reference" content="citation_journal_title=Anim Genet; citation_title=Genetic diversity and subdivision of 57 European and Middle-Eastern sheep breeds; citation_author=C Peter, M Bruford, T Perez, S Dalamitra, G Hewitt, G Erhardt; citation_volume=38; citation_publication_date=2007; citation_pages=37-44; citation_doi=10.1111/j.1365-2052.2007.01561.x; citation_id=CR20"/>

    <meta name="citation_reference" content="citation_journal_title=Heredity; citation_title=Selection on MHC-linked microsatellite loci in sheep populations; citation_author=F Santucci, KM Ibrahim, A Bruzzone, GM Hewit; citation_volume=99; citation_publication_date=2007; citation_pages=340-348; citation_doi=10.1038/sj.hdy.6801006; citation_id=CR21"/>

    <meta name="citation_reference" content="citation_journal_title=Genetics; citation_title=Efficiency of the use of pedigree and molecular marker information in conservation programs; citation_author=J Fern&#225;ndez, B Villanueva, R Pong-Wong, MA Toro; citation_volume=170; citation_publication_date=2005; citation_pages=1313-1321; citation_doi=10.1534/genetics.104.037325; citation_id=CR22"/>

    <meta name="citation_reference" content="citation_journal_title=Heredity; citation_title=Understanding the relationship between the inbreeding coefficient and multilocus heterozygosity: theoretical expectations and empirical data; citation_author=J Slate, P David, KG Dodds, BA Veenvliet, BC Glass, TE Broad, JC McEwan; citation_volume=93; citation_publication_date=2004; citation_pages=255-265; citation_doi=10.1038/sj.hdy.6800485; citation_id=CR23"/>

    <meta name="citation_reference" content="Bensch S, Andr&#233;n H, Hansson B, Pedersen HC, Sand H, Sejberg D, et al. Selection for heterozygosity gives hope to a wild population of inbred wolves. PLoS One. 2016; doi: 
                    10.1371/journal.pone.000007
                    
                  ."/>

    <meta name="citation_reference" content="25. Li MH, Strand&#233;n I, Tiirikka T, Sev&#243;n-Aimonen ML, Kantanen J. A comparison of approaches to estimate the inbreeding coefficient and pairwise relatedness using genomic and pedigree data in a sheep population. PLoS One. 2011; doi:
                    10.1371/journal.pone.0026256
                    
                  ."/>

    <meta name="citation_reference" content="Wooliams JA, Gwaze GP, Meuwissen THE, Planchenault D, Renard JP, Thibier M, Wagner H, editors. Secondary guidelines for development of national farm animal genetic resources management plans: Management of small populations at risk. Food and Agriculture Organization of the United Nations. 1998."/>

    <meta name="citation_reference" content="citation_title=Long-term genetic contributions. Predictions of rates of inbreeding and genetic gain in selected populations; citation_publication_date=2000; citation_id=CR27; citation_author=P Bijma; citation_publisher=Wageningen University"/>

    <meta name="citation_reference" content="citation_journal_title=Animal Conserv; citation_title=How large must populations be to retain evolutionary potential?; citation_author=IR Franklin, R Frankham; citation_volume=1; citation_publication_date=1998; citation_pages=69-70; citation_doi=10.1111/j.1469-1795.1998.tb00228.x; citation_id=CR28"/>

    <meta name="citation_author" content="Christina M. Rochus"/>

    <meta name="citation_author_institution" content="Department of Animal Breeding and Genetics, Faculty of Veterinary Medicine and Animal Science, Swedish University of Agricultural Sciences, Uppsala, Sweden"/>

    <meta name="citation_author_institution" content="UFR G&#233;n&#233;tique, &#201;levage et Reproduction; Sciences de la Vie et Sant&#233;, AgroParisTech, Universit&#233; Paris-Saclay, Paris, France"/>

    <meta name="citation_author_institution" content="G&#233;n&#233;tique Physiologie Syst&#232;mes d&#8217;Elevage (GenPhySE), Universit&#233; de Toulouse, INRA, INPT, ENVT, Castanet-Tolosan, France"/>

    <meta name="citation_author" content="Anna M. Johansson"/>

    <meta name="citation_author_institution" content="Department of Animal Breeding and Genetics, Faculty of Veterinary Medicine and Animal Science, Swedish University of Agricultural Sciences, Uppsala, Sweden"/>



        <meta charset="UTF-8"/>
        <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
        <meta name="viewport" content="width=device-width, initial-scale=1"/>

        <!-- For IE 9 and below -->
<!--[if IE]><link rel="shortcut icon" href=/static/images/favicons//favicon.ico><![endif]-->
<!-- Touch Icons - iOS and Android 2.1+ -->
<link rel="apple-touch-icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>
<!-- Firefox, Chrome, Safari, IE 11+ and Opera -->
<link rel="icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>


        <script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script>
        <script>(function(d){if (sessionStorage.fontsLoaded){d.style.setProperty('--font-family-sans', 'Europa');d.style.setProperty('--font-family-serif', 'Noto Serif')}})(document.documentElement);</script>

        <link rel="stylesheet" media="screen" href=/static/styles/basic/basic-a31ca85bc2.css>
<link rel="stylesheet" media="print" href=/static/styles/bmc/print-3a6240f456.css>

<link rel="stylesheet" class="js-ctm" href=/static/styles/bmc/main-84414d3d47.css
      media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">

        

        <script src="/static/js/app-bundle-8d61e6ef37.js" defer></script>
        
        <script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script>
        
    <script type="text/javascript">
        config = {
            env: 'live',
            site: 'hereditasjournal.biomedcentral.com',
            siteWithPath: 'hereditasjournal.biomedcentral.com' + window.location.pathname,
            twitterHashtag: '',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/s41065-017-0026-4',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central'
        };
    </script>

        
    <script type="text/javascript">
        dataLayer = [{"content":{"article":{"doi":"10.1186/s41065-017-0026-4","articleType":"Research","peerReviewType":"Closed","supplement":null,"keywords":"Microsatellite Marker;Effective Population Size;Coat Colour;Pedigree Information;Sheep Breed"},"contentInfo":{"imprint":"BioMed Central","title":"Estimation of genetic diversity in Gute sheep: pedigree and microsatellite analyses of an ancient Swedish breed","publishedAt":1485734400000,"publishedAtDate":"2017-01-30","author":["Christina M. Rochus","Anna M. Johansson"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":"CC BY + CC0","openAccess":true},"environment":"live"},"journal":{"siteKey":"hereditasjournal.biomedcentral.com","volume":"154","issue":"1","title":"Hereditas","type":"AJs Life","journalID":41065,"gaCode":"UA-62066850-11","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Research","publishingSegment":"Genetics & Heredity (Life)"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}}}]

    </script>

        <script src="https://cdn.cookielaw.org/consent/7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script>
        
    
        
    

        
    
        
            
                <!-- Google Tag Manager -->
                <script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script>
                <!-- End Google Tag Manager -->
            
        
    


        
    
    <link rel="canonical" href="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0026-4"/>
    

        
        
        <meta property="og:url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0026-4"/>
        <meta property="og:type" content="article"/>
        <meta property="og:site_name" content="Hereditas"/>
        <meta property="og:title" content="Estimation of genetic diversity in Gute sheep: pedigree and microsatellite analyses of an ancient Swedish breed"/>
        <meta property="og:description" content="Breeds with small population size are in danger of an increased inbreeding rate and loss of genetic diversity, which puts them at risk for extinction. In Sweden there are a number of local breeds, native breeds which have adapted to specific areas in Sweden, for which efforts are being made to keep them pure and healthy over time. One example of such a breed is the Swedish Gute sheep. The objective of this study was to estimate inbreeding and genetic diversity of Swedish Gute sheep. Three datasets were analysed: pedigree information of the whole population, pedigree information for 100 animals of the population, and microsatellite genotypes for 94 of the 100 animals. The average inbreeding coefficient for lambs born during a six year time period (2007–2012) did not increase during that time period. The inbreeding calculated from the entire pedigree (0.038) and for a sample of the population (0.018) was very low. Sheep were more heterozygous at the microsatellite markers than expected (average multilocus heterozygosity and Ritland inbreeding estimates 1.01845 and -0.03931) and five of seven microsatellite markers were not in Hardy Weinberg equilibrium due to heterozygosity excess. The total effective population size estimated from the pedigree information was 155.4 and the average harmonic mean effective population size estimated from microsatellites was 88.3. Pedigree and microsatellite genotype estimations of inbreeding were consistent with a breeding program with the purpose of reducing inbreeding. Our results showed that current breeding programs of the Swedish Gute sheep are consistent with efforts of keeping this breed viable and these breeding programs are an example for other small local breeds in conserving breeds for the future."/>
        <meta property="og:image" content="https://images.springer.com/sgw/journals/medium/41065.jpg"/>
    
    </head>

    <body class="journal journal-fulltext"
          
          >
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK"
                            height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><feColorMatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"/></filter></defs><symbol id="icon-arrow-down" viewBox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"/></symbol><symbol id="icon-arrow-left-bullet" viewBox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"/></symbol><symbol id="icon-arrow-right-bullet" viewBox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"/></symbol><symbol id="icon-arrow-right" viewBox="0 0 32 64"><path d="M0 0v64l32-32z"/></symbol><symbol id="icon-close" viewBox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"/></symbol><symbol id="icon-community" viewBox="0 0 60.643 31.01"><mask maskUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><path fill="#FFF" d="M0 31.01h60.643V0H0z" filter="url(#a)"/></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"/></symbol><symbol id="icon-download" viewBox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"/></symbol><symbol id="icon-ext-link" viewBox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"/><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"/><path fill="#676767" d="M9.7 0L16 6.2V0z"/></symbol><symbol id="icon-info-bordered" viewBox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"/><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"/></symbol><symbol id="icon-opr" viewBox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"/><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"/></symbol><symbol id="icon-quotation" viewBox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"/></symbol><symbol id="icon-remove" viewBox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"/><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"/></symbol><symbol id="icon-ribbon" viewBox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"/></symbol><symbol id="icon-rss" viewBox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"/><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"/><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"/></g></symbol><symbol id="icon-search" viewBox="0 0 16 16"><path d="M6.276 12.257C2.815 12.257 0 9.508 0 6.128 0 2.749 2.815 0 6.276 0c3.462 0 6.277 2.749 6.277 6.128.002 3.38-2.815 6.129-6.277 6.129zm0-10.772c-2.622 0-4.755 2.083-4.755 4.643 0 2.562 2.133 4.644 4.755 4.644 2.623 0 4.756-2.082 4.756-4.644 0-2.56-2.134-4.643-4.756-4.643z"/><path d="M16 13.751L13.641 16l-4.875-4.82 2.36-2.248L16 13.751z"/></symbol><symbol id="icon-speech-bubble" viewBox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"/></symbol><symbol id="icon-error" viewBox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"/></symbol><symbol id="icon-info" viewBox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"/></symbol><symbol id="icon-success" viewBox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"/></symbol><symbol id="icon-warning" viewBox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"/></symbol><symbol id="icon-facebook-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"/></symbol><symbol id="icon-facebook" viewBox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"/><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"/></symbol><symbol id="icon-gplus-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M480.5 273.117c.123.076.26.185.416.334.146.152.289.343.432.562.135.211.258.459.359.746.086.286.129.619.129 1.004-.012.7-.164 1.262-.465 1.683a5.374 5.374 0 0 1-.461.562c-.186.176-.375.35-.58.527a2.65 2.65 0 0 0-.322.41.908.908 0 0 0-.176.553c0 .205.059.375.18.508.102.129.199.238.299.336l.662.545a6.48 6.48 0 0 1 1.086 1.119c.293.415.445.955.459 1.625 0 .949-.42 1.789-1.256 2.521-.869.76-2.123 1.15-3.762 1.177-1.373-.015-2.4-.308-3.074-.879-.682-.534-1.023-1.172-1.023-1.917 0-.359.111-.766.336-1.211.215-.443.602-.837 1.172-1.174a6.126 6.126 0 0 1 1.998-.725 15.415 15.415 0 0 1 1.723-.173 3.996 3.996 0 0 1-.377-.596 1.347 1.347 0 0 1-.193-.74c0-.174.023-.317.074-.433a8.83 8.83 0 0 1 .121-.338 5.973 5.973 0 0 1-.633.035c-1.043-.012-1.838-.339-2.381-.979-.574-.603-.861-1.3-.861-2.097 0-.961.404-1.832 1.217-2.616.559-.46 1.135-.757 1.736-.898a8.494 8.494 0 0 1 1.67-.182h3.916l-1.209.704h-1.207v.005l-.005.002zm.754 11.276c0-.498-.162-.932-.484-1.302-.346-.348-.881-.778-1.609-1.29a4.045 4.045 0 0 0-.439-.02c-.102-.016-.354 0-.768.039a8.141 8.141 0 0 0-1.244.275c-.1.036-.24.096-.42.17a2.713 2.713 0 0 0-.553.355 1.985 1.985 0 0 0-.459.592c-.145.25-.215.551-.215.901 0 .687.311 1.253.932 1.698.596.444 1.404.68 2.432.688.928-.014 1.629-.217 2.117-.612.476-.385.71-.886.71-1.494zm-2.729-5.743a1.843 1.843 0 0 0 1.289-.556c.164-.249.271-.506.318-.767a7.32 7.32 0 0 0 .041-.659c0-.772-.195-1.552-.594-2.336a2.611 2.611 0 0 0-.73-.921 1.894 1.894 0 0 0-1.064-.352 1.723 1.723 0 0 0-1.324.64c-.299.437-.438.92-.428 1.457 0 .709.209 1.444.623 2.211.201.356.459.659.775.912.323.248.686.371 1.094.371zm9.698.077h-2.055v-2.056h-.996v2.056h-2.053v.998h2.053v2.052h.996v-2.052h2.055z"/></symbol><symbol id="icon-gplus" viewBox="0 0 24 24"><circle fill="#D84A36" cx="12" cy="12" r="12"/><g fill="#FFF"><path d="M12.012 12.111c-.304-.33-.07-.842.155-1.078.226-.241.639-.56.829-.74.193-.184.66-.605.674-1.868 0 0 .112-1.139-1.127-2.06h.977l1.038-.593h-3.507s-2.083-.067-3.085 1.207c-1 1.276-.966 2.871.114 3.77 1.08.898 2.371.648 2.371.648s-.197.422-.163.775c.036.353.421.849.504.965 0 0-2.386-.011-3.551 1.033-1.165 1.053-1.29 3.097 1.021 3.803 2.312.707 5.303-.055 5.906-2.277 0 0 .262-.908-.189-1.683-.459-.775-1.663-1.571-1.967-1.902zm-2.555-1.83c-.744-1-1.308-2.953 0-3.783 0 0 .955-.579 1.801.266.846.846 1.082 2.277.928 3.003-.15.727-.563.897-.739 1.008-.185.111-1.249.504-1.99-.494zm3.668 5.481c-.035.393-.083 1.254-1.491 1.664-1.409.408-3.826-.264-3.757-1.914 0 0 .006-.698.449-1.089.445-.396 1.094-.794 3.029-.819 0 0 .643.473 1.072.809.427.332.733.952.698 1.349z"/><path d="M16.323 11.055V9.803h-.752v1.252h-1.255v.748h1.255v1.256h.752v-1.256h1.248v-.748z"/></g></symbol><symbol id="icon-linkedin" viewBox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"/><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"/><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"/></g></symbol><symbol id="icon-reddit" viewBox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"/><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"/><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"/><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"/><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"/><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"/><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"/><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"/></symbol><symbol id="icon-twitter-bordered" viewBox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"/><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/></symbol><symbol id="icon-twitter" viewBox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"/><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"/></symbol><symbol id="icon-weibo-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"/><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"/><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"/></symbol><symbol id="icon-weibo" viewBox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"/><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"/><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"/><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"/><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"/><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"/></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner c-banner--info">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner" >
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1"
                 data-gpt-unitpath="/270604982/bmc/hereditasjournal/articles"
                 data-gpt-sizes="728x90,970x90"
                 data-gpt-targeting="pos=LB1;doi=10.1186/s41065-017-0026-4;kwrd=Microsatellite Marker,Effective Population Size,Coat Colour,Pedigree Information,Sheep Breed;pmc=L32030,L21001;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-017-0026-4&amp;kwrd=Microsatellite Marker,Effective Population Size,Coat Colour,Pedigree Information,Sheep Breed&amp;pmc=L32030,L21001&amp;">
                        <img data-test="gpt-advert-fallback-img"
                             src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-017-0026-4&amp;kwrd=Microsatellite Marker,Effective Population Size,Coat Colour,Pedigree Information,Sheep Breed&amp;pmc=L32030,L21001&amp;"
                             alt="Advertisement"
                             width="728"
                             height="90">
                    </a>
                </noscript>
            </div>
        </div>
    </div>

        
         
    <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>


<header class="c-popup-search ctx-publisher-header" id="top" data-test="publisher-header">
    <nav class="c-navbar">
        <div class="u-container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization" />
                <div class="c-logo">
    <a href="https://www.biomedcentral.com" itemprop="url">
        <img src=/static/images/bmc/logos/logo-bmc-white-aj-e3a70ae848.svg alt="BMC" itemprop="logo">
        <div class="c-logo__strapline">
            <img src=/static/images/bmc/logos/logo-bmc-white-strapline-sn-e7b1d6e67f.svg alt="Part of Springer Nature">
        </div>
    </a>
</div>
            </div>
            <button
                type="button"
                class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg"
                id="publisher-header-menu-button"
                data-toggle="collapse"
                data-test="publisher-header-menu-toggle"
                data-toggle-linked="publisher-header"
                data-target="publisher-header-search publisher-header-menu"
                aria-controls="publisher-header-menu publisher-header-search"
                aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


                <ul class="c-navbar__nav">
                    <li class="c-navbar__item">
                        <a data-replace="true"
                           data-placeholder="/placeholder/v1/username"
                           class="c-navbar__link"
                           href="https://www.biomedcentral.com/login"
                           data-test="login-link"
                           data-track="click"
                           data-track-category="Account"
                           data-track-action="Click Login Link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                    <li class="c-navbar__item">
                        <button
                            type="button"
                            class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg"
                            data-toggle="collapse"
                            data-test="header-search-button"
                            data-target="publisher-header-search publisher-header-menu publisher-header-menu-button"
                            aria-controls="publisher-header-search publisher-header-menu"
                            aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true">
                                    <use xlink:href="#icon-search"></use>
                                </svg>
                            </span>
                        </button>
                    </li>
                </ul>
            </div>
        </div>
    </nav>
    <div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field c-form-field--inline" method="GET" action="//www.biomedcentral.com/search" data-track="submit"
        data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label data-track-label="" data-test="global-search">
        <label for="publisherSearch" class="u-visually-hidden">Search all BMC articles</label>
        <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text" value="" placeholder="Search all BMC articles"/>
        <button class="c-button c-button--secondary" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true">
        <use xlink:href="#icon-search"></use>
    </svg>
</button>


        <input type="hidden" name="searchType" value="publisherSearch"/>
    </form>
</div>

            </div>
        </div>
    </div>
</header>


        
            <header class="c-journal-header c-journal-header--hereditas ctx-journal-header">
                <div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="/">
    
        
    
    <span class="c-journal-title__text ">Hereditas</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="u-container">
                        
                            <button type="button"
                                    class="c-navbar__link c-dropdown u-hidden-at-lg"
                                    data-toggle="collapse"
                                    data-test="journal-header-menu"
                                    data-toggle-linked="journal-header"
                                    data-target="journal-menu"
                                    aria-controls="journal-menu"
                                    aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav">
                                    <ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/'>Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/about'>About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href='/articles'>Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/submission-guidelines'>Submission Guidelines</a>
                                            </li>
                                        
                                    </ul>
                                </nav>
                            </div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--ocean-light-blue">
                    
                </div>
            </header>
            
        

        <div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-content-layout c-content-layout--fulltext" data-component="sticky-container">
                <div class="c-content-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3><ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc=""><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Methods">Methods</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec5" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results">Results</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec6" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Discussion">Discussion</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec7" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Conclusions">Conclusions</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li></ol></nav>
                </div>

                <main class="c-content-layout__main">
                    <article itemscope itemtype="http://schema.org/ScholarlyArticle">
                        
                        
    <div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
            <li class="c-article-identifiers__item" data-test="article-category">Research</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
    </div>

                        <div class="FulltextWrapper"><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">Estimation of genetic diversity in Gute sheep: pedigree and microsatellite analyses of an ancient Swedish breed</h1></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true"><ul class="u-listReset"><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Christina&nbsp;M.&nbsp;Rochus</span><sup><a href="#Aff1" class="AffiliationID">1</a>, <a href="#Aff2" class="AffiliationID">2</a>, <a href="#Aff3" class="AffiliationID">3</a></sup> and </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Anna&nbsp;M.&nbsp;Johansson</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup><a href="mailto:Anna.Johansson@slu.se" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a></li></ul></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation"><div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">Hereditas</span></span><span class="ArticleCitation_Year">2017</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">154</strong>:4</span></span></div><p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/s41065-017-0026-4">https://doi.org/10.1186/s41065-017-0026-4</a></p><p class="Copyright">© &nbsp;The Author(s)&nbsp;2017</p></div></div><ul class="ArticleHistory u-reset-list"><li class="History HistoryReceived"><strong>Received: </strong><span>7&nbsp;September&nbsp;2016</span></li><li class="History HistoryAccepted"><strong>Accepted: </strong><span>20&nbsp;January&nbsp;2017</span></li><li class="History HistoryOnlineDate"><strong>Published: </strong><span itemprop="datePublished">30&nbsp;January&nbsp;2017</span></li></ul></div>
                        
                        
                        
    

                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2><div class="c-section__content" id="Abstract"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p id="Par1" class="Para">Breeds with small population size are in danger of an increased inbreeding rate and loss of genetic diversity, which puts them at risk for extinction. In Sweden there are a number of local breeds, native breeds which have adapted to specific areas in Sweden, for which efforts are being made to keep them pure and healthy over time. One example of such a breed is the Swedish Gute sheep. The objective of this study was to estimate inbreeding and genetic diversity of Swedish Gute sheep.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Results</h3>
                <p id="Par2" class="Para">Three datasets were analysed: pedigree information of the whole population, pedigree information for 100 animals of the population, and microsatellite genotypes for 94 of the 100 animals. The average inbreeding coefficient for lambs born during a six year time period (2007–2012) did not increase during that time period. The inbreeding calculated from the entire pedigree (0.038) and for a sample of the population (0.018) was very low. Sheep were more heterozygous at the microsatellite markers than expected (average multilocus heterozygosity and Ritland inbreeding estimates 1.01845 and -0.03931) and five of seven microsatellite markers were not in Hardy Weinberg equilibrium due to heterozygosity excess. The total effective population size estimated from the pedigree information was 155.4 and the average harmonic mean effective population size estimated from microsatellites was 88.3. Pedigree and microsatellite genotype estimations of inbreeding were consistent with a breeding program with the purpose of reducing inbreeding.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusion</h3>
                <p id="Par3" class="Para">Our results showed that current breeding programs of the Swedish Gute sheep are consistent with efforts of keeping this breed viable and these breeding programs are an example for other small local breeds in conserving breeds for the future.</p>
              </div></div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords"><li class="c-keywords__item" itemprop="about">Microsatellite Marker</li><li class="c-keywords__item" itemprop="about">Effective Population Size</li><li class="c-keywords__item" itemprop="about">Coat Colour</li><li class="c-keywords__item" itemprop="about">Pedigree Information</li><li class="c-keywords__item" itemprop="about">Sheep Breed</li></ul></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par4" class="Para">Gute sheep are an ancient breed from the Swedish island and province of Gotland, belonging to the North European short-tailed sheep group. Both Gute sheep and in general North European short-tailed sheep are characterized in part by their hardiness as well as coat colour and pattern variation [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>]. Many of the North European short-tailed sheep breeds are decreasing in population size [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>, <span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>] and have low heterogeneity [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>]. However these breeds are unique and their conservation is relevant because they add to overall species diversity [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>, <span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>]. The Gute sheep population experienced a severe bottleneck when polled sheep started to become more popular on Gotland and the horned sheep became rare. Beginning around 1920 the remaining horned sheep on Gotland were gathered by a few individuals interested in preserving the horned sheep. These sheep are the origins of the existing Gute sheep population today (personal communication with Gute sheep breed organizations in Sweden). Nevertheless, Gute sheep, unlike some other North European short-tailed sheep breeds, are not endangered at the moment, although the population is believed to be decreasing in size [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>]. Thanks to conservation efforts, there were an estimated 5200 animals in Sweden in 2012 [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>]. The first association working with the conservation of the Gute sheep, Föreningen Gutefåret, was formed in 1977, and an additional association, GutefårAkademin, was formed in 2007. Gute sheep breeding is focused on preserving the breed and not on improving production traits.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par5" class="Para">Gute sheep appear in the scientific literature, having been studied for both coat colour and population structure and genetic diversity. Classical genetic studies from the 1970s took advantage of the coat colour variation in Gute sheep, also then referred to as Goth sheep in English, to study coat colour inheritance in sheep [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>]. More recently, there have been population genetic studies which have included Gute sheep [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>, <span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>], but these studies only had a small number of individuals; 20 and 12 respectively. There have been no studies of Gute sheep genetic diversity using pedigree information.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par6" class="Para">The objectives of this study were to estimate inbreeding and genetic diversity of Swedish Gute sheep using pedigree data and microsatellite genotypes to identify; if the population is at potential risk due to high or increasing inbreeding and decreased genetic diversity; and if strategies need to be implemented to conserve the Swedish Gute sheep in the future.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Methods" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Methods">Methods</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Methods" data-test="collapse-section"><section id="Sec3" class="Section2 RenderAsSection2">
                <h3 class="Heading">Data</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par7" class="Para">Pedigree information for all registered Gute sheep in Sweden born from 1960 to 2014 was provided by Elitlamm, the recording program for sheep in Sweden. For the early records only a small proportion of the population were registered in ElitLamm, but in recent years a very large proportion of the Gute sheep population in Sweden has been registered in ElitLamm because animals must be registered in order to be entitled to financial support for the keeping of endangered breeds. Pedigree information included animal identification numbers, year of birth and parent identification numbers. The animal identification numbers were anonymous.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par8" class="Para">Blood samples and phenotypic records (tail length and information about horns) were collected from different parts of Sweden by a trained technician in collaboration with the association “Föreningen Gutefåret”, one of the two Gute sheep breed organizations in Sweden. Ethical permission to collect blood samples from sheep and use for genetic studies was approved before the study (Dnr C102/13). Samples were stored in the -80&nbsp;°C freezer of the SLU biobank. DNA was extracted from blood samples for 100 Gute sheep from a total of 13 flocks using the DNeasy mini kit for the QIAsymphony robot (Qiagen®, Hilden, Germany). Sheep were genotyped for a total of eight genetic markers; AME, a SNP marker in the amelogenin gene for sex determination; and seven microsatellite markers (INRA005, INRA023, INRA063, INRA172, MAF214, MAF65 and McM527). These markers are part of the ISAG kit for parentage testing. Genotyping was done at the certified lab at the Department of Animal Breeding and Genetics, SLU. The pedigree information for the 100 animals included flock identification, date of birth and official animal identification numbers for each animal, and their ancestors (which did not correspond with the anonymous animal identification numbers of the pedigree of the whole population).</p>
              </section><section id="Sec4" class="Section2 RenderAsSection2">
                <h3 class="Heading">Analyses</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par9" class="Para">Pedigree information of the whole population was used to estimate population parameters. Generation intervals, effective number of founders, effective number of ancestors, effective number of founder genomes and marginal contributions of ancestors were estimated with the software Pedig [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>]. Generation interval for sires of sires, dams of sires, sires of dams and dams of dams were calculated separately. The average was calculated from these four values and used as the overall generation interval. The coefficient of inbreeding by birth year [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>], pedigree completeness indices, number of complete generation equivalents and average coancestry within birth cohorts were estimated with software, EVA [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>]. Effective population size was calculated as one divided by double the annual rate of inbreeding multiplied by the generation interval [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>].</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par10" class="Para">The pedigree information for each of the 100 sheep with blood samples available were combined into one pedigree. This pedigree was used for calculating average coefficient of inbreeding and pedigree completeness with the software EVA [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>] for comparison with results for these parameters from the whole pedigree.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par11" class="Para">Diversity within populations (1-Q<sub>INTRA</sub>) and between populations (1-Q<sub>INTER</sub>), inbreeding in individuals compared with the population inbreeding (F<sub>IS</sub>), and Hardy Weinberg tests for heterozygosity deficiency and excess in microsatellite markers were estimated with the program Genepop v.4.3 [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>]. Inbreeding and relatedness among individuals [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>] were estimated using the microsatellite genotypes using software Coancestry v.1.0.1.5 [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>]. Multilocus heterozygosity was calculated as the proportion of typed loci at which an individual was heterozygous divided by the population heterozygosity at those same loci [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>]. Past changes in effective population size (average harmonic mean effective population size) were estimated using microsatellite genotypes in the software VarEff v.1.2 [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>] with an assumed mutation rate of 0.00013 [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>] and generation interval 3&nbsp;years. Population structure was estimated with the microsatellite genotypes using principal components analysis (PCA) using R package adegenet [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>].</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec5" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Results" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Results">Results</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Results" data-test="collapse-section"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par12" class="Para">A total of 70 474 Gute sheep births were recorded in our dataset since 1960, with 30 616 recorded births during the last six years (2007 to 2012) (Fig.&nbsp;<span class="InternalRef"><a href="#Fig1">1</a></span>). Animals born from 2007 to 2012 had a pedigree completion index for three generations greater than 0.8. The average generation interval for this cohort was 3.6&nbsp;years with generation intervals for sires being longer than for dams (Table&nbsp;<span class="InternalRef"><a href="#Tab1">1</a></span>), the total effective population size was 155.4 (Table&nbsp;<span class="InternalRef"><a href="#Tab1">1</a></span>) and the average inbreeding and average coancestry were low (0.038 and 0.007) (Fig.&nbsp;<span class="InternalRef"><a href="#Fig2">2a and b</a></span>). The distributions of inbreeding coefficients of animals born in 2007 and 2012 are in Fig.&nbsp;<span class="InternalRef"><a href="#Fig2">2c</a></span>. Measures of diversity and changes in population structure including effective number of founders, effective number of ancestors and effective number of founder genomes are reported in Table&nbsp;<span class="InternalRef"><a href="#Tab1">1</a></span>. From 1994 to 2003, the average inbreeding coefficient increased every year (from 0.0003 in 1994 to 0.037 in 2003), however, from 2003 to 2012 the average inbreeding coefficient remained relatively constant.<figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO1">
                    <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0026-4/MediaObjects/41065_2017_26_Fig1_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0026-4/MediaObjects/41065_2017_26_Fig1_HTML.gif" alt="Fig. 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0026-4/MediaObjects/41065_2017_26_Fig1_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0026-4/MediaObjects/41065_2017_26_Fig1_HTML.gif" alt="Fig. 1"></img></noscript></a>
                  </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 1</span>
                      <p xmlns="" class="SimplePara">Total number of Gute sheep recorded in Sweden by birth year</p>
                    </div></figcaption></figure>
                        <figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 1</span>
                      <p xmlns="" class="SimplePara">Generation interval (L) and effective population size (N<sub xmlns="http://www.w3.org/1999/xhtml">e</sub>) for Gute sheep in Sweden between 2007 and 2012 calculated using pedigree</p>
                    </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col></col></colgroup><thead><tr class="header"><th>&nbsp;</th><th><p xmlns="" class="SimplePara">Males</p></th><th><p xmlns="" class="SimplePara">Females</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">Generation interval (L) in years</p></td><td><p xmlns="" class="SimplePara">3.7</p></td><td><p xmlns="" class="SimplePara">3.5</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Effective population size (N<sub xmlns="http://www.w3.org/1999/xhtml">e</sub>)</p></td><td><p xmlns="" class="SimplePara">152.8</p></td><td><p xmlns="" class="SimplePara">157.7</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Number of founders</p></td><td><p xmlns="" class="SimplePara">575</p></td><td><p xmlns="" class="SimplePara">715</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Effective number of founders</p></td><td><p xmlns="" class="SimplePara">204.6</p></td><td><p xmlns="" class="SimplePara">225.7</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Effective number of founder genomes</p></td><td><p xmlns="" class="SimplePara">45.5</p></td><td><p xmlns="" class="SimplePara">52.8</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Effective number of ancestors</p></td><td><p xmlns="" class="SimplePara">109.6</p></td><td><p xmlns="" class="SimplePara">122.1</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                    <p xmlns="" class="SimplePara">Number of founders, effective number of founders, effective number of founder genomes and effective number of ancestors calculated using pedigree in all recorded Swedish Gute sheep</p>
                  </div></figure>
                        <figure xmlns="" class="Figure" id="Fig2"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO2">
                    <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0026-4/MediaObjects/41065_2017_26_Fig2_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0026-4/MediaObjects/41065_2017_26_Fig2_HTML.gif" alt="Fig. 2" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0026-4/MediaObjects/41065_2017_26_Fig2_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0026-4/MediaObjects/41065_2017_26_Fig2_HTML.gif" alt="Fig. 2"></img></noscript></a>
                  </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 2</span>
                      <p xmlns="" class="SimplePara">
                                    <strong class="EmphasisTypeBold">a</strong> Average inbreeding coefficient, <strong class="EmphasisTypeBold">b</strong> average coancestry for Gute sheep by birth year from 2007-2012 and <strong class="EmphasisTypeBold">c</strong> distribution of inbreeding coefficients of animals born in 2007 (solid grey line) and 2012 (dashed black line)</p>
                    </div></figcaption></figure>
                     </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par13" class="Para">A total of 94 animals (19 males and 75 females) from 13 flocks were successfully genotyped (Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>: Table S1). All of the 94 genotyped animals had horns (which is typical for this breed). Average tail length was 14.2&nbsp;cm (standard deviation: 1.45) with tail length ranging from 11&nbsp;cm to 18&nbsp;cm. The average inbreeding coefficient estimated from the pedigree was 0.018. The number of alleles per microsatellite marker ranged from three to seven. Five of seven of the microsatellite markers were not in Hardy Weinberg Equilibrium due to excess heterozygosity (Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>: Table S2). On average, flocks had excess heterozygosity (average F<sub>IS</sub> = -0.1102) and high diversity within and among flocks (1-Q<sub>INTRA</sub> = 0.7102 and 1-Q<sub>INTER</sub> = 0.6437) (Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>: Table S3). This was also verified by the analysis of individual animals, which showed excess heterozygosity and high diversity within and among individuals (F<sub>IS</sub> = -0.0561, 1-Q<sub>INTRA</sub> = 0.7190 and 1-Q<sub>INTER</sub> = 0.6808). Average Ritland inbreeding calculated for the 94 animals was low (F = 0.018). Multilocus heterozygosity and inbreeding calculated from microsatellites was highly correlated (0.90). The heterozygosity calculated using both methods was, on average, higher than expected (1.01845 and -0.03931). Inbreeding calculated from the pedigree had low correlations with both multilocus heterozygosity and inbreeding calculated from microsatellite markers (0.36 and 0.29). The average harmonic mean effective population size estimated from microsatellites has decreased by 4.2 (92.5 to 88.3) in the past five generations (~15&nbsp;years). Using principal component analysis (PCA) to detect population structure revealed that individuals from all thirteen flocks generally cluster together in one group (Fig.&nbsp;<span class="InternalRef"><a href="#Fig3">3</a></span>). There were three individuals who were separate from the main cluster however these individuals were missing the same microsatellite genotypes and their position in the plot in Fig.&nbsp;<span class="InternalRef"><a href="#Fig3">3</a></span> is likely a result of this.<figure xmlns="" class="Figure" id="Fig3"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO3">
                    <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0026-4/MediaObjects/41065_2017_26_Fig3_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0026-4/MediaObjects/41065_2017_26_Fig3_HTML.gif" alt="Fig. 3" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0026-4/MediaObjects/41065_2017_26_Fig3_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0026-4/MediaObjects/41065_2017_26_Fig3_HTML.gif" alt="Fig. 3"></img></noscript></a>
                  </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 3</span>
                      <p xmlns="" class="SimplePara">Population structure of Gute sheep genotyped with seven microsatellites using principal component analysis. Principal components 1 and 2 for individuals were plotted (shapes indicate which flock they were from)</p>
                    </div></figcaption></figure>
                     </div></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec6" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Discussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Discussion">Discussion</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Discussion" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par14" class="Para">We present the first analyses of genetic diversity of the Gute sheep population in Sweden. Pedigree and microsatellite genotype estimations of inbreeding were consistent with a breeding program aiming at reducing inbreeding.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par15" class="Para">Previous studies have suggested that Gute sheep is a unique breed with low heterogeneity [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>, <span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>], which we could not verify in our study. Tapio et al. estimated inbreeding in 32 north European sheep breeds, including Gute sheep, using microsatellite markers [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>]. All breeds in their study had greater F<sub>IS</sub> estimates than our estimate for Swedish Gute sheep and many of the F<sub>IS</sub> estimates indicated inbred populations (including for their Gute sheep population) [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>]. The heterogeneity in our study was higher than expected in the population, both seen in the multilocus heterozygosity (1.01845) and the fact that five out of seven microsatellite markers were not in Hardy Weinberg equilibrium because of excess heterozygosity. In addition, inbreeding calculated from the entire pedigree, from the pedigree of the 94 individuals and from their microsatellite markers was low. The following could have contributed to differences in results: our diversity and inbreeding estimates are from animals born after animals studied by Tapio et al. [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>]. Furthermore, we used pedigree data from the entire registered Swedish population and genotyped 94 animals from 13 different flocks with microsatellite genotypes while only 20 animals had been genotyped in the previous study [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>]. Finally, we only used seven microsatellite markers compared to the 25 used by Tapio et al. [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>] and this is a weakness of our study.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par16" class="Para">Inbreeding estimated based on data from the entire pedigree was more than twice the estimated inbreeding from the sample population (0.038 versus 0.018) which could be due to the amount of pedigree information available for each animal. The entire pedigree is more complete and has more depth: the average depth of the entire population pedigree in generations was 6.06 compared in contrast with 5.41 for the sample population. The founders of a pedigree are assumed to be unrelated and when information from more generations is available, there could be more relationships accounted for. Nevertheless, both estimates of inbreeding were low and our hypothesis is, that this could be because neither of the datasets could account for bottlenecks occurring earlier in time (ie. Before pedigree information was collected). However, the estimation of inbreeding from microsatellite genotypes of the sample population, should be able to account for historical inbreeding, but the estimation of inbreeding from microsatellite data was also low (Ritland inbreeding = -0.03931). We are therefore confident, that our estimation of a low inbreeding reflects reality.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par17" class="Para">This was also verified by the further analysis, which indeed indicated the existence of bottlenecks in the population. The difference between the effective number of founders and the effective number of ancestors indicated that bottlenecks have occurred in this population and the difference of half the effective population size and the effective number of founder genomes indicated that drift has been accelerating since the founder population.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par18" class="Para">Previous studies found heterozygote deficiency in European sheep breeds (including North European short-tailed breeds), which was thought to be due to subdivision among flocks creating the Wahlund effect and due to non-random mating [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>, <span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>–<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>]. In contrast we found that there was excess heterozygosity in Swedish Gute sheep. Excess heterozygosity in microsatellites in a population can be due to the structure of the breeding program, where unrelated animals are bred together as typically found in a population within a conservation program. For the sample population genotyped, 95% of animals had parents with different flock ID numbers indicating that they were born in different flocks, consistent with the breeding of animals that are unrelated. The use of pedigree information in a conservation breeding program has been shown to be a very powerful tool for maintaining genetic diversity and low inbreeding [<span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>] and our study showed similar results.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par19" class="Para">Heterozygosity and inbreeding calculated from microsatellite markers were highly correlated with each other but were not with inbreeding calculated from the pedigree. Regardless, we can conclude from these results that there was low inbreeding. Multilocus heterozygosity is not a good indicator of inbreeding calculated from a pedigree because it is not highly correlated unless there is a large amount of variation in inbreeding in a population [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>] and heterozygosity can be lost at a different rate than inbreeding is gained in a population [<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>]. Additionally, correlation between these estimates can be lower because of amount and quality of information in the recorded pedigree. In a study of Finnsheep, estimates of inbreeding were correlated, however, estimates were not as correlated in one subpopulation of Finnsheep (grey individuals) and the authors believed this was in part because of their less complete pedigree [<span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par20" class="Para">The effective population size estimated from microsatellites was 88.3, which was lower than the effective population size calculated from the whole pedigree (155.4). However both of these estimates fall well within or even above the recommended effective population size for small populations, which should be at least between 50 and 100 [<span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>, <span class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>]. On the other hand it is still important to maintain the conservation breeding program and monitor progress because in order to maintain genetic variation in the long term, an effective population size of at least 500 is needed [<span class="CitationRef"><a href="#CR28" data-toggle="citation">28</a></span>]. Nevertheless, the comparison of our results with that of earlier studies suggests that the current conservation strategy in the Swedish Gute sheep population is successful and should be maintained.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec7" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Conclusions" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Conclusions">Conclusions</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Conclusions" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par21" class="Para">The Swedish Gute sheep population can be an example of conservation for other small local breeds.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par22" class="Para">Despite evidence of historical bottlenecks, inbreeding estimates from the pedigree of all registered Gute sheep in Sweden and from the pedigree and microsatellite genotypes of a sample of the population indicated that the level of inbreeding was low and that inbreeding did not increase over the last six years of data analysed. While the Swedish Gute sheep population has an effective population size that is good for small populations, the current conservation program and monitoring of the average inbreeding should continue.</p></div></section>
          <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments"><h3 xmlns="" class="Heading">Acknowledgements</h3><p xmlns="" class="SimplePara">The project was performed in collaboration with the Föreningen Gutefåret association. The late professor Anna Näsholm was involved in the initial planning of the project.</p><div id="FPar1" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Funding</h3>
                <p id="Par23" class="Para">The collection of blood samples and genotyping and part of the working time for analyses was financed by a grant from the Swedish Board of Agriculture to the Föreningen Gutefåret association. CMR benefited from a joint grant from the European Commission and the Swedish University of Agricultural Sciences, within the framework of the Erasmus-Mundus joint doctorate “European Graduate School in Animal Breeding and Genetics”.</p>
              </div><div id="FPar2" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Availability of data and materials</h3>
                <p id="Par24" class="Para">The microsatellite genotypes (Additional file <span class="InternalRef"><a href="#MOESM2">2</a></span>) and the horn and tail phenotypes (Additional file <span class="InternalRef"><a href="#MOESM3">3</a></span>) of the 94 genotyped Gute sheep are available as additional supporting files. Please contact authors for access to anonymized pedigree data.</p>
              </div><div id="FPar3" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Authors’ contributions</h3>
                <p id="Par25" class="Para">CMR analysed the data and wrote the manuscript. AMJ conceived the project and assisted in manuscript writing. Both authors read and approved the final manuscript.</p>
              </div><div id="FPar4" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Competing interests</h3>
                <p id="Par26" class="Para">The authors declare that they have no competing interests.</p>
              </div><div id="FPar5" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Consent for publication</h3>
                <p id="Par27" class="Para">Not applicable.</p>
              </div><div id="FPar6" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Ethics approval</h3>
                <p id="Par28" class="Para">Ethical permission to collect blood samples from sheep and use for genetic studies was approved before the study (Dnr C102/13).</p>
              </div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="License">
                <p xmlns="" class="SimplePara">
                           <strong class="EmphasisTypeBold">Open Access</strong>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/4.0/"><span class="RefSource">http://creativecommons.org/licenses/by/4.0/</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/publicdomain/zero/1.0/"><span class="RefSource">http://creativecommons.org/publicdomain/zero/1.0/</span></a></span>) applies to the data made available in this article, unless otherwise stated.</p>
              </div><aside class="Appendix" id="App1">
              
                <h3 class="Heading">Additional files</h3><div id="widgetContainer" data-test="figshare-container"></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par29" class="Para">
                  <div class="MediaObject" id="MOESM1">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0026-4/MediaObjects/41065_2017_26_MOESM1_ESM.docx"><span class="caption-number">Additional file 1:</span></a>
                          <span xmlns="" class="SimplePara">Contains the following three tables. <strong class="EmphasisTypeBold">Table S1.</strong> 94 Gute sheep microsatellite genotyped by flock and sex. <strong class="EmphasisTypeBold">Table S2.</strong> Microsatellite markers, number of alleles and Hardy Weinberg equilibrium. <strong class="EmphasisTypeBold">Table S3.</strong> Gene diversity and inbreeding by flock over all loci with at least two individuals typed. (DOCX 21&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM2">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0026-4/MediaObjects/41065_2017_26_MOESM2_ESM.txt"><span class="caption-number">Additional file 2:</span></a>
                          <span xmlns="" class="SimplePara">Contains the microsatellite genotypes for the 94 gute sheep. (TXT 22&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM3">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0026-4/MediaObjects/41065_2017_26_MOESM3_ESM.txt"><span class="caption-number">Additional file 3:</span></a>
                          <span xmlns="" class="SimplePara">Contains the year of birth and horn and tail phenotypes for the 94 gute sheep with microsatellite genotypes. (TXT 2&nbsp;kb)</span>
                        </div></div>
                  </div>
                </div>
              
            </aside></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authors’ Affiliations</a></h2><div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1"><span class="AffiliationNumber">(1)</span><div class="AffiliationText">Department of Animal Breeding and Genetics, Faculty of Veterinary Medicine and Animal Science, Swedish University of Agricultural Sciences, Box 7023, Uppsala, SE-75007, Sweden</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff2"><span class="AffiliationNumber">(2)</span><div class="AffiliationText">UFR Génétique, Élevage et Reproduction; Sciences de la Vie et Santé, AgroParisTech, Université Paris-Saclay, Paris, France</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff3"><span class="AffiliationNumber">(3)</span><div class="AffiliationText">Génétique Physiologie Systèmes d’Elevage (GenPhySE), Université de Toulouse, INRA, INPT, ENVT, Castanet-Tolosan, France</div></div></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper"><li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Dýrmundsson ÓR, Niżnikowski R. North European short-tailed breeds of sheep: a review. Animal. 2010;4:1275–82.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1017/S175173110999156X"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22444647"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=North%20European%20short-tailed%20breeds%20of%20sheep%3A%20a%20review&amp;author=%C3%93R.%20D%C3%BDrmundsson&amp;author=R.%20Ni%C5%BCnikowski&amp;journal=Animal&amp;volume=4&amp;pages=1275-1282&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Domestic Animal Diversity Information System (DAD-IS), Food and Agriculture Organization of the United Nations. 2016. <span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://dad.fao.org/"><span class="RefSource">http://dad.fao.org/</span></a></span>. Accessed 29 Aug 2016.</span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Tapio M, Tapio I, Grislis Z, Holm LE, Jeppsson S, Kantanen J, et al. Native breeds demonstrate high contributions to the molecular variation in northern European sheep. Mol Ecol. 2005;14:3951–63.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1365-294X.2005.02727.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16262851"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Native%20breeds%20demonstrate%20high%20contributions%20to%20the%20molecular%20variation%20in%20northern%20European%20sheep&amp;author=M.%20Tapio&amp;author=I.%20Tapio&amp;author=Z.%20Grislis&amp;author=LE.%20Holm&amp;author=S.%20Jeppsson&amp;author=J.%20Kantanen&amp;journal=Mol%20Ecol&amp;volume=14&amp;pages=3951-3963&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Tapio M, Ozerov M, Tapio I, Toro M, Marzanov N, Ćinkulov M, et al. Microsatellite-based genetic diversity and population structure of domestic sheep in northern Eurasia. BMC Genet. 2010;11:76.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2156-11-76"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20698974"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2931448"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Microsatellite-based%20genetic%20diversity%20and%20population%20structure%20of%20domestic%20sheep%20in%20northern%20Eurasia&amp;author=M.%20Tapio&amp;author=M.%20Ozerov&amp;author=I.%20Tapio&amp;author=M.%20Toro&amp;author=N.%20Marzanov&amp;author=M.%20%C4%86inkulov&amp;journal=BMC%20Genet&amp;volume=11&amp;pages=76&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Adalsteinsson S, Lauvergne JJ, Boyazoglu JG, Ryder ML. A possible genetic interpretation of the colour variation in the fleece of the Gotland and Goth sheep. Ann Genet Sel Anim. 1978;10:329–42.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22896105"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2718911"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20possible%20genetic%20interpretation%20of%20the%20colour%20variation%20in%20the%20fleece%20of%20the%20Gotland%20and%20Goth%20sheep&amp;author=S.%20Adalsteinsson&amp;author=JJ.%20Lauvergne&amp;author=JG.%20Boyazoglu&amp;author=ML.%20Ryder&amp;journal=Ann%20Genet%20Sel%20Anim&amp;volume=10&amp;pages=329-342&amp;publication_year=1978"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Mukiibi R, Rochus CM, Andersson G, Johansson AM. The use of endogenous retroviruses as markers to describe the genetic relationships among local Swedish sheep breeds. Anim Genet. 2015;46:220–3.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/age.12264"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25644015"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20use%20of%20endogenous%20retroviruses%20as%20markers%20to%20describe%20the%20genetic%20relationships%20among%20local%20Swedish%20sheep%20breeds&amp;author=R.%20Mukiibi&amp;author=CM.%20Rochus&amp;author=G.%20Andersson&amp;author=AM.%20Johansson&amp;journal=Anim%20Genet&amp;volume=46&amp;pages=220-223&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Boichard D. Pedig: A Fortran package for pedigree analysis suited for large populations. Montpellier: 7th World Cong Genet Appl Livest Prod; 2002. p. 28–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Pedig%3A%20A%20Fortran%20package%20for%20pedigree%20analysis%20suited%20for%20large%20populations&amp;author=D.%20Boichard&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Meuwissen THE, Luo Z. Computing inbreeding coefficients in large populations. Genet Sel Evol. 1992;24:305–13.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1297-9686-24-4-305"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2711146"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Computing%20inbreeding%20coefficients%20in%20large%20populations&amp;author=THE.%20Meuwissen&amp;author=Z.%20Luo&amp;journal=Genet%20Sel%20Evol&amp;volume=24&amp;pages=305-313&amp;publication_year=1992"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Berg P, Nielsen J, Sørensen MK. EVA: Realized and predicted optimal genetic contributions. Belo Horizonte: 8th World Cong Genet Appl Livest Prod; 2006. p. 246p.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=EVA%3A%20Realized%20and%20predicted%20optimal%20genetic%20contributions&amp;author=P.%20Berg&amp;author=J.%20Nielsen&amp;author=MK.%20S%C3%B8rensen&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Falconer DS, Mackay TFC. Introduction to quantitative genetics. 4th ed. Burt Mill, Harlow: Longman Scientific &amp; Technical; 1996.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Introduction%20to%20quantitative%20genetics&amp;author=DS.%20Falconer&amp;author=TFC.%20Mackay&amp;publication_year=1996"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Rousset F. Genepop’007: a complete re-implementation of the genepop software for Windows and Linux. Mol Ecol Resour. 2008;8:103–6.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1471-8286.2007.01931.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21585727"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genepop%E2%80%99007%3A%20a%20complete%20re-implementation%20of%20the%20genepop%20software%20for%20Windows%20and%20Linux&amp;author=F.%20Rousset&amp;journal=Mol%20Ecol%20Resour&amp;volume=8&amp;pages=103-106&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ritland K. Estimators for pairwise relatedness and individual inbreeding coefficients. Genet Res. 1996;67:175–85.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1017/S0016672300033620"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Estimators%20for%20pairwise%20relatedness%20and%20individual%20inbreeding%20coefficients&amp;author=K.%20Ritland&amp;journal=Genet%20Res&amp;volume=67&amp;pages=175-185&amp;publication_year=1996"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wang J. Coancestry: a program for simulating, estimating and analysing relatedness and inbreeding coefficients. Mol Ecol Resour. 2011;11:141–5.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1755-0998.2010.02885.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21429111"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Coancestry%3A%20a%20program%20for%20simulating%2C%20estimating%20and%20analysing%20relatedness%20and%20inbreeding%20coefficients&amp;author=J.%20Wang&amp;journal=Mol%20Ecol%20Resour&amp;volume=11&amp;pages=141-145&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Coltman DW, Pilkington JG, Smith JA, Pemberton JM. Parasite-mediated selection against inbred Soay sheep in a free-living, island population. Evolution. 1999;53:1259–67.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2307/2640828"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Parasite-mediated%20selection%20against%20inbred%20Soay%20sheep%20in%20a%20free-living%2C%20island%20population&amp;author=DW.%20Coltman&amp;author=JG.%20Pilkington&amp;author=JA.%20Smith&amp;author=JM.%20Pemberton&amp;journal=Evolution&amp;volume=53&amp;pages=1259-1267&amp;publication_year=1999"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Nikolic N, Chevalet C. Detecting past changes of effective population size. Evol App. 2014;7:663–81.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/eva.12170"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Detecting%20past%20changes%20of%20effective%20population%20size&amp;author=N.%20Nikolic&amp;author=C.%20Chevalet&amp;journal=Evol%20App&amp;volume=7&amp;pages=663-681&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Crawford AM, Cuthbertson RP. Mutations in sheep microsatellites. Genome Res. 1996;6:876–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1101/gr.6.9.876"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8889555"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Mutations%20in%20sheep%20microsatellites&amp;author=AM.%20Crawford&amp;author=RP.%20Cuthbertson&amp;journal=Genome%20Res&amp;volume=6&amp;pages=876-879&amp;publication_year=1996"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Jombart T. adegenet: a R package for the multivariate analysis of genetic markers. Bioinformatics. 2008;24:1403–5.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btn129"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18397895"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=adegenet%3A%20a%20R%20package%20for%20the%20multivariate%20analysis%20of%20genetic%20markers&amp;author=T.%20Jombart&amp;journal=Bioinformatics&amp;volume=24&amp;pages=1403-1405&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lawson Handley LJ, Byrne K, Santucci F, Townsend S, Taylor M, Bruford MW, Hewitt GM. Genetic structure of European sheep breeds. Heredity. 2007;99:620–31.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/sj.hdy.6801039"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17700634"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20structure%20of%20European%20sheep%20breeds&amp;author=LJ.%20Lawson%20Handley&amp;author=K.%20Byrne&amp;author=F.%20Santucci&amp;author=S.%20Townsend&amp;author=M.%20Taylor&amp;author=MW.%20Bruford&amp;author=GM.%20Hewitt&amp;journal=Heredity&amp;volume=99&amp;pages=620-631&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Tapio M, Miceikiené I, Vikki J, Kantanen J. Comparison of microsatellite and blood protein diversity in sheep: inconsistencies in fragmented breeds. Mol Ecol. 2003;12:2045–56.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1046/j.1365-294X.2003.01893.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12859628"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparison%20of%20microsatellite%20and%20blood%20protein%20diversity%20in%20sheep%3A%20inconsistencies%20in%20fragmented%20breeds&amp;author=M.%20Tapio&amp;author=I.%20Miceikien%C3%A9&amp;author=J.%20Vikki&amp;author=J.%20Kantanen&amp;journal=Mol%20Ecol&amp;volume=12&amp;pages=2045-2056&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Peter C, Bruford M, Perez T, Dalamitra S, Hewitt G, Erhardt G, ECONOGENE Consortium. Genetic diversity and subdivision of 57 European and Middle-Eastern sheep breeds. Anim Genet. 2007;38:37–44.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1365-2052.2007.01561.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17257186"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20and%20subdivision%20of%2057%20European%20and%20Middle-Eastern%20sheep%20breeds&amp;author=C.%20Peter&amp;author=M.%20Bruford&amp;author=T.%20Perez&amp;author=S.%20Dalamitra&amp;author=G.%20Hewitt&amp;author=G.%20Erhardt&amp;journal=Anim%20Genet&amp;volume=38&amp;pages=37-44&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Santucci F, Ibrahim KM, Bruzzone A, Hewit GM. Selection on MHC-linked microsatellite loci in sheep populations. Heredity. 2007;99:340–8.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/sj.hdy.6801006"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17519962"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Selection%20on%20MHC-linked%20microsatellite%20loci%20in%20sheep%20populations&amp;author=F.%20Santucci&amp;author=KM.%20Ibrahim&amp;author=A.%20Bruzzone&amp;author=GM.%20Hewit&amp;journal=Heredity&amp;volume=99&amp;pages=340-348&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Fernández J, Villanueva B, Pong-Wong R, Toro MA. Efficiency of the use of pedigree and molecular marker information in conservation programs. Genetics. 2005;170:1313–21.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1534/genetics.104.037325"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15879510"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1451183"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Efficiency%20of%20the%20use%20of%20pedigree%20and%20molecular%20marker%20information%20in%20conservation%20programs&amp;author=J.%20Fern%C3%A1ndez&amp;author=B.%20Villanueva&amp;author=R.%20Pong-Wong&amp;author=MA.%20Toro&amp;journal=Genetics&amp;volume=170&amp;pages=1313-1321&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Slate J, David P, Dodds KG, Veenvliet BA, Glass BC, Broad TE, McEwan JC. Understanding the relationship between the inbreeding coefficient and multilocus heterozygosity: theoretical expectations and empirical data. Heredity. 2004;93:255–65.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/sj.hdy.6800485"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15254488"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Understanding%20the%20relationship%20between%20the%20inbreeding%20coefficient%20and%20multilocus%20heterozygosity%3A%20theoretical%20expectations%20and%20empirical%20data&amp;author=J.%20Slate&amp;author=P.%20David&amp;author=KG.%20Dodds&amp;author=BA.%20Veenvliet&amp;author=BC.%20Glass&amp;author=TE.%20Broad&amp;author=JC.%20McEwan&amp;journal=Heredity&amp;volume=93&amp;pages=255-265&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR24" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Bensch S, Andrén H, Hansson B, Pedersen HC, Sand H, Sejberg D, et al. Selection for heterozygosity gives hope to a wild population of inbred wolves. PLoS One. 2016; doi: <span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.1371/journal.pone.000007"><span class="RefSource">10.1371/journal.pone.000007</span></a></span>.</span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR25" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">25. Li MH, Strandén I, Tiirikka T, Sevón-Aimonen ML, Kantanen J. A comparison of approaches to estimate the inbreeding coefficient and pairwise relatedness using genomic and pedigree data in a sheep population. PLoS One. 2011; doi:<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.1371/journal.pone.0026256"><span class="RefSource">10.1371/journal.pone.0026256</span></a></span>.</span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR26" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wooliams JA, Gwaze GP, Meuwissen THE, Planchenault D, Renard JP, Thibier M, Wagner H, editors. Secondary guidelines for development of national farm animal genetic resources management plans: Management of small populations at risk. Food and Agriculture Organization of the United Nations. 1998.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Wooliams%20JA%2C%20Gwaze%20GP%2C%20Meuwissen%20THE%2C%20Planchenault%20D%2C%20Renard%20JP%2C%20Thibier%20M%2C%20Wagner%20H%2C%20editors.%20Secondary%20guidelines%20for%20development%20of%20national%20farm%20animal%20genetic%20resources%20management%20plans%3A%20Management%20of%20small%20populations%20at%20risk.%20Food%20and%20Agriculture%20Organization%20of%20the%20United%20Nations.%201998."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR27" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Bijma P. Long-term genetic contributions. Predictions of rates of inbreeding and genetic gain in selected populations, PhD Thesis. the Netherlands: Wageningen University; 2000.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Long-term%20genetic%20contributions.%20Predictions%20of%20rates%20of%20inbreeding%20and%20genetic%20gain%20in%20selected%20populations&amp;author=P.%20Bijma&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR28" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Franklin IR, Frankham R. How large must populations be to retain evolutionary potential? Animal Conserv. 1998;1:69–70.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1469-1795.1998.tb00228.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=How%20large%20must%20populations%20be%20to%20retain%20evolutionary%20potential%3F&amp;author=IR.%20Franklin&amp;author=R.%20Frankham&amp;journal=Animal%20Conserv&amp;volume=1&amp;pages=69-70&amp;publication_year=1998"><span><span>Google Scholar</span></span></a></span></span></cite></li></ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#TheAuthors" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="TheAuthors">Copyright</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="TheAuthors"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">©&nbsp;The Author(s)&nbsp;2017</div></div></section></div>
                        


                    </article>
                </main>

                <div class="c-content-layout__side u-text-sm">
                    <aside>
                        <div class="p-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="//hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-017-0026-4" data-track="click" data-track-category="PDF Download"
            data-track-action="Click download article PDF"
            data-track-label="10.1186/s41065-017-0026-4">Download PDF</a>
        </p>
    
                            
                            
    <button type="button"
       class="c-button c-button--secondary c-button--full-width c-button--expandable"
       data-toggle="collapse"
       data-target="export-citations"
       data-test="export-citations-button"
       data-sticky-update="true"
       aria-controls="export-citations"
       aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/s41065-017-0026-4"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0026-4?format=refman&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/s41065-017-0026-4"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0026-4?format=refman&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/s41065-017-0026-4"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0026-4?format=refman&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/s41065-017-0026-4"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0026-4?format=endnote&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/s41065-017-0026-4"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0026-4?format=endnote&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/s41065-017-0026-4"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0026-4?format=endnote&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/s41065-017-0026-4"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0026-4?format=bibtex&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/s41065-017-0026-4"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0026-4?format=bibtex&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/s41065-017-0026-4"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0026-4?format=bibtex&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
    </div>

                        </div>
                        
    
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
                                <div id="article-access-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/access-count/10.1186/s41065-017-0026-4 #accessCount-container"></div>
                                <div id="citation-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/citation-count/10.1186/s41065-017-0026-4 #count-container"></div>
                                <div id="altmetric-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/altmetric/10.1186/s41065-017-0026-4 #altmetric-container"></div>
                            </ul>
                        </div>
                        
    <div id="sociaMediaShare" class="p-separator u-js-shown">
        <h3 class="p-section-title">Share this article</h3>
        <ul id="shareButtons" class="c-social-links">
            <li class="c-social-links__item">
                <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-017-0026-4">
                    <span class="u-visually-hidden">Share on Twitter</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-twitter"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-017-0026-4">
                    <span class="u-visually-hidden">Share on Facebook</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-facebook"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click"
                        data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-017-0026-4">
                    <span class="u-visually-hidden">Share on LinkedIn</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-linkedin"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-017-0026-4">
                    <span class="u-visually-hidden">Share on Weibo</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-weibo"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnGooglePlus" class="js-btnShareOnGplus" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Google+" data-track-label="10.1186/s41065-017-0026-4">
                    <span class="u-visually-hidden">Share on Google Plus</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-gplus"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-017-0026-4">
                    <span class="u-visually-hidden">Share on Reddit</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-reddit"></use>
                    </svg>
                </button>
            </li>
        </ul>
    </div>
 
                        
    
        <div class="p-separator u-js-shown">
            <h3 class="p-section-title">See updates</h3>
            <button class="c-crossmark-button" data-target="crossmark" data-test="crossmark" data-track="click" data-track-category="Article Page" data-track-action="Click CrossMark" data-track-label="10.1186/s41065-017-0026-4">
                <img id="crossmark-icon" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" alt="Crossmark logo" />
            </button>
        </div>
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
                                <li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/s41065-017-0026-4" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=738&doi=10.1186%2Fs41065-017-0026-4&volume=154&issue=1&title=Estimation+of+genetic+diversity+in+Gute+sheep%3A+pedigree+and+microsatellite+analyses+of+an+ancient+Swedish+breed&author_name=Christina+M.+Rochus&start_page=1&end_page=7">Order reprint</a>

                                </li>
                            </ul>
                        </div>
                    </aside>
                    
    <div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">Hereditas</span>
                        </h4>
                        <p class="c-journal-footer__issn">ISSN: 1601-5223</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
                                
                                    <li class="c-journal-footer__contact-item">Submission enquiries: <a href="https://www.editorialmanager.com/here/" target="_blank">Access here and click Contact Us</a></li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a></li>
                                
                            </ul>
                        </div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class='tracker' style='display:none' src='/track/article/10.1186/s41065-017-0026-4' alt=""/>

         
    <footer>
        <div class="ctx-publisher-footer">
    <div class="c-navbar">
        <div class="u-container">
            <div class="c-navbar__logo u-margin-right-xl">
                <a href="https://www.biomedcentral.com">
                    <img src=/static/images/bmc/logos/logo-bmc-aj-594e03e20d.svg alt="BMC logo" data-test="navbar-logo-footer">

                </a>
            </div>
            <div class="c-navbar__content">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


            </div>
        </div>
    </div>
    <div class="c-publisher-footer" data-test="publisher-footer">
        <div class="u-container">
            
            <div class="c-flex c-flex--wrap c-flex--space-between" role="navigation" data-test="publisher-footer-menu">
                <div class="c-flex" aria-label="Publisher footer links">
                    
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/login">Receive BMC newsletters</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/account">Manage article alerts</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/policies">Policies</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/accessibility">Accessibility</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/press-centre">Press center</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/contact-us">Contact us</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/jobs">Careers</a>
                                    </li>
                                
                            </ul>
                        
                    
                </div>
                <div class="u-margin-bottom-lg">
                    <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                    <ul class="c-flex u-reset-list" data-test="footer-social-links">
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://twitter.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Twitter link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Twitter page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-twitter-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://www.facebook.com/BioMedCentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Facebook link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Facebook page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-facebook-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://plus.google.com/106226337984152901734/"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Google Plus link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Google Plus page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-gplus-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="http://www.weibo.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Weibo link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Weibo page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-weibo-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </div>
            <p class="u-reset-margin">
                By using this website, you agree to our
                <a class="u-text-inherit" href="//www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
                <a class="u-text-inherit" href="//www.biomedcentral.com/privacy-statement">Privacy
                    statement</a> and
                <a class="u-text-inherit" href="//www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
                
                    <a class="optanon-toggle-display u-text-inherit" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
                
            </p>
        </div>
    </div>
</div>


        <div class="c-sn-footer c-sn-footer--neutral">
    <div class="u-container">
        <img src=/static/images/bmc/logos/logo-springernature-8bd4e8d82f.svg class="c-sn-footer__logo" alt="Springer Nature Logo" itemprop="logo" >
        <p data-test="copyright">&#169; 2018 BioMed Central Ltd unless otherwise stated. Part of
            <a href="http://www.springernature.com"
               data-track="click"
               data-track-action="Click Springer Nature link"
               data-track-category="Corporate footer"
               itemscope itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>

    </div>

    
    <script type="text/javascript" src="//cdn.pbgrd.com/bmc.js" async="async"></script>
    <noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>
    <!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker -->

<script id="springerlink-event-tracker" src="https://event-tracker.springernature.com/dist/eventTracker.js"></script>
<script type="text/javascript">
    (function(){
        var dataLayer = window.dataLayer[0];
        var doi = dataLayer.content.article.doi;
        var imprint = dataLayer.content.contentInfo.imprint;
        var bpids = dataLayer.session.authentication.authenticationID;
        var publisher;
        var pageType = dataLayer.page.category.pageType;
        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }
        var eventObject = {'content_type' : pageType,'doi' : doi};

        if (imprint === 'BioMed Central') { publisher = 'BMC' }
        else if (imprint === 'SpringerOpen') { publisher = 'SpringerOpen' }
        else { throw new Error('Event Tracker Error: The publisher has an unexpected value') }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }
        new EventTracker({'platform':publisher}).sendEvent('display', eventObject);
    })();
</script>


    </body>
</html>

