<!DOCTYPE html>
<html lang="en" class="no-js">
    <head>
        <title>Genetic diversity and structure of core collection of winter mushroom (Flammulina velutipes) developed by genomic SSR markers | Hereditas | Full Text</title>

        

    <meta name="citation_abstract" content="A core collection is a subset of an entire collection that represents as much of the genetic diversity of the entire collection as possible. The establishment of a core collection for crops is practical for efficient management and use of germplasm. However, the establishment of a core collection of mushrooms is still in its infancy, and no established core collection of the economically important species Flammulina velutipes has been reported. We established the first core collection of F. velutipes, containing 32 strains based on 81 genetically different F. veltuipes strains. The allele retention proportion of the core collection for the entire collection was 100%. Moreover, the genetic diversity parameters (the effective number of alleles, Nei’s expected heterozygosity, the number of observed heterozygosity, and Shannon’s information index) of the core collection showed no significant differences from the entire collection (p > 0.01). Thus, the core collection is representative of the genetic diversity of the entire collection. Genetic structure analyses of the core collection revealed that the 32 strains could be clustered into 6 groups, among which groups 1 to 3 were cultivars and groups 4 to 6 were wild strains. The wild strains from different locations harbor their own specific alleles, and were clustered stringently in accordance with their geographic origins. Genetic diversity analyses of the core collection revealed that the wild strains possessed greater genetic diversity than the cultivars. We established the first core collection of F. velutipes in China, which is an important platform for efficient breeding of this mushroom in the future. In addition, the wild strains in the core collection possess favorable agronomic characters and produce unique bioactive compounds, adding value to the platform. More attention should be paid to wild strains in further strain breeding."/>

    <meta name="description" content="A core collection is a subset of an entire collection that represents as much of the genetic diversity of the entire collection as possible. The establishment of a core collection for crops is practical for efficient management and use of germplasm. However, the establishment of a core collection of mushrooms is still in its infancy, and no established core collection of the economically important species Flammulina velutipes has been reported. We established the first core collection of F. velutipes, containing 32 strains based on 81 genetically different F. veltuipes strains. The allele retention proportion of the core collection for the entire collection was 100%. Moreover, the genetic diversity parameters (the effective number of alleles, Nei’s expected heterozygosity, the number of observed heterozygosity, and Shannon’s information index) of the core collection showed no significant differences from the entire collection (p > 0.01). Thus, the core collection is representative of the genetic diversity of the entire collection. Genetic structure analyses of the core collection revealed that the 32 strains could be clustered into 6 groups, among which groups 1 to 3 were cultivars and groups 4 to 6 were wild strains. The wild strains from different locations harbor their own specific alleles, and were clustered stringently in accordance with their geographic origins. Genetic diversity analyses of the core collection revealed that the wild strains possessed greater genetic diversity than the cultivars. We established the first core collection of F. velutipes in China, which is an important platform for efficient breeding of this mushroom in the future. In addition, the wild strains in the core collection possess favorable agronomic characters and produce unique bioactive compounds, adding value to the platform. More attention should be paid to wild strains in further strain breeding."/>

    <meta name="journal_id" content="41065"/>

    <meta name="dc.title" content="Genetic diversity and structure of core collection of winter mushroom ( Flammulina velutipes ) developed by genomic SSR markers"/>

    <meta name="dc.source" content="Hereditas 2017 155:1"/>

    <meta name="dc.format" content="text/html"/>

    <meta name="dc.publisher" content="BioMed Central"/>

    <meta name="dc.date" content="2017-07-03"/>

    <meta name="dc.type" content="OriginalPaper"/>

    <meta name="dc.language" content="En"/>

    <meta name="dc.copyright" content="2017 The Author(s)"/>

    <meta name="dc.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="dc.description" content="A core collection is a subset of an entire collection that represents as much of the genetic diversity of the entire collection as possible. The establishment of a core collection for crops is practical for efficient management and use of germplasm. However, the establishment of a core collection of mushrooms is still in its infancy, and no established core collection of the economically important species Flammulina velutipes has been reported. We established the first core collection of F. velutipes, containing 32 strains based on 81 genetically different F. veltuipes strains. The allele retention proportion of the core collection for the entire collection was 100%. Moreover, the genetic diversity parameters (the effective number of alleles, Nei&#8217;s expected heterozygosity, the number of observed heterozygosity, and Shannon&#8217;s information index) of the core collection showed no significant differences from the entire collection (p&#160;&gt;&#160;0.01). Thus, the core collection is representative of the genetic diversity of the entire collection. Genetic structure analyses of the core collection revealed that the 32 strains could be clustered into 6 groups, among which groups 1 to 3 were cultivars and groups 4 to 6 were wild strains. The wild strains from different locations harbor their own specific alleles, and were clustered stringently in accordance with their geographic origins. Genetic diversity analyses of the core collection revealed that the wild strains possessed greater genetic diversity than the cultivars. We established the first core collection of F. velutipes in China, which is an important platform for efficient breeding of this mushroom in the future. In addition, the wild strains in the core collection possess favorable agronomic characters and produce unique bioactive compounds, adding value to the platform. More attention should be paid to wild strains in further strain breeding."/>

    <meta name="prism.issn" content="1601-5223"/>

    <meta name="prism.publicationName" content="Hereditas"/>

    <meta name="prism.publicationDate" content="2017-07-03"/>

    <meta name="prism.volume" content="155"/>

    <meta name="prism.number" content="1"/>

    <meta name="prism.section" content="OriginalPaper"/>

    <meta name="prism.startingPage" content="3"/>

    <meta name="prism.copyright" content="2017 The Author(s)"/>

    <meta name="prism.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="prism.url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0038-0"/>

    <meta name="prism.doi" content="doi:10.1186/s41065-017-0038-0"/>

    <meta name="citation_pdf_url" content="https://hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-017-0038-0"/>

    <meta name="citation_fulltext_html_url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0038-0"/>

    <meta name="citation_journal_title" content="Hereditas"/>

    <meta name="citation_publisher" content="BioMed Central"/>

    <meta name="citation_issn" content="1601-5223"/>

    <meta name="citation_title" content="Genetic diversity and structure of core collection of winter mushroom ( Flammulina velutipes ) developed by genomic SSR markers"/>

    <meta name="citation_volume" content="155"/>

    <meta name="citation_issue" content="1"/>

    <meta name="citation_publication_date" content="2018/12"/>

    <meta name="citation_online_date" content="2017/07/03"/>

    <meta name="citation_firstpage" content="3"/>

    <meta name="citation_article_type" content="Research"/>

    <meta name="citation_fulltext_world_readable" content=""/>

    <meta name="dc.identifier" content="doi:10.1186/s41065-017-0038-0"/>

    <meta name="DOI" content="10.1186/s41065-017-0038-0"/>

    <meta name="citation_doi" content="10.1186/s41065-017-0038-0"/>

    <meta name="dc.creator" content="Xiao Bin Liu"/>

    <meta name="dc.creator" content="Jing Li"/>

    <meta name="dc.creator" content="Zhu L. Yang"/>

    <meta name="dc.subject" content="Animal Genetics and Genomics"/>

    <meta name="dc.subject" content="Evolutionary Biology"/>

    <meta name="citation_reference" content="citation_title=Current plant genetic resources-a critical appraisal. In genetics: new frontiers; citation_publication_date=1984; citation_id=CR1; citation_author=OH Frankel; citation_author=B AHD; citation_publisher=Oxford &amp; IBH Publishing Co"/>

    <meta name="citation_reference" content="citation_journal_title=Genome; citation_title=Core collections: a practical approach to genetic resources management; citation_author=AHD Brown; citation_volume=31; citation_publication_date=1989; citation_pages=818-824; citation_doi=10.1139/g89-144; citation_id=CR2"/>

    <meta name="citation_reference" content="citation_journal_title=Genet Resour Crop Evol; citation_title=Establishment of a core collection for maize germplasm preserved in Chinese National Genebank using geographic distribution and characterization data; citation_author=Y Li, Y Shi, Y Cao, T Wang; citation_volume=51; citation_publication_date=2005; citation_pages=845-852; citation_doi=10.1007/s10722-005-8313-8; citation_id=CR3"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Breeding; citation_title=Genetic diversity and core collection evaluations in common wheat germplasm from the northwestern spring wheat region in China; citation_author=CY Hao, XY Zhang, LF Wang, XW Dong, XW Shang, JZ Jia; citation_volume=17; citation_publication_date=2006; citation_pages=69-77; citation_doi=10.1007/s11032-005-2453-6; citation_id=CR4"/>

    <meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=A worldwide bread wheat core collection arrayed in a 384-well plate; citation_author=F Balfourier, V Roussel, P Strelchenko, F Exbrayat-Vinson, P Sourdille, G Boutet; citation_volume=114; citation_publication_date=2007; citation_pages=1265-1275; citation_doi=10.1007/s00122-007-0517-1; citation_id=CR5"/>

    <meta name="citation_reference" content="citation_journal_title=Field Crops Res; citation_title=Establishing a soybean germplasm core collection; citation_author=MF Oliveira, RL Nelson, IO Geraldi, CD Cruz, JFF Toledo; citation_volume=119; citation_publication_date=2010; citation_pages=277-289; citation_doi=10.1016/j.fcr.2010.07.021; citation_id=CR6"/>

    <meta name="citation_reference" content="citation_journal_title=Mol breeding; citation_title=Development of an allele-mining set in rice using a heuristic algorithm and SSR genotype data with least redundancy for the post-genomic era; citation_author=W Zhao, GT Cho, KH Ma, JW Chung, JG Gwag, YJ Park; citation_volume=26; citation_publication_date=2010; citation_pages=639-651; citation_doi=10.1007/s11032-010-9400-x; citation_id=CR7"/>

    <meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=Selection of a core collection from the US germplasm collection of peanut; citation_author=CC Holbrook, WF Anderson, RN Pittman; citation_volume=33; citation_publication_date=1993; citation_pages=859-861; citation_doi=10.2135/cropsci1993.0011183X003300040044x; citation_id=CR8"/>

    <meta name="citation_reference" content="citation_journal_title=Hereditas; citation_title=Sampling strategies for composing a core collection of cultivated barley (Hordeum vulgare s. Lat.) collected in China; citation_author=TJ Hintum, R Bothmer, DL Visser; citation_volume=122; citation_publication_date=1995; citation_pages=7-17; citation_doi=10.1111/j.1601-5223.1995.00007.x; citation_id=CR9"/>

    <meta name="citation_reference" content="Ortiz R, Ruiz-Tapia EN, Mujica-Sanchez A. Sampling strategy for a core collection of Peruvian quinoa germplasm. Theor Appl Genet. 1998;96:475&#8211;83."/>

    <meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=Methods of constructing core collections by stepwise clustering with three sampling strategies based on the genotypic values of crops; citation_author=J Hu, J Zhu, HM Xu; citation_volume=101; citation_publication_date=2000; citation_pages=264-268; citation_doi=10.1007/s001220051478; citation_id=CR11"/>

    <meta name="citation_reference" content="citation_journal_title=World J Microbiol Biotechnol; citation_title=Using SSR markers to evaluate the genetic diversity of Lentinula edodes&#8217; natural germplasm in China; citation_author=Y Xiao, W Liu, Y Dai, C Fu, Y Bian; citation_volume=26; citation_publication_date=2010; citation_pages=527-536; citation_doi=10.1007/s11274-009-0202-4; citation_id=CR12"/>

    <meta name="citation_reference" content="citation_journal_title=Mycol Prog; citation_title=Development of SSR markers for typing cultivars in the mushroom Auricularia auricula-judae
                           ; citation_author=RY Zhang, DD Hu, JG Gu, QX Hu, XM Zuo, HX Wang; citation_volume=11; citation_publication_date=2012; citation_pages=587-592; citation_doi=10.1007/s11557-011-0798-2; citation_id=CR13"/>

    <meta name="citation_reference" content="Zane L, Bargelloni L, Patarnello T. Strategies for microsatellite isolation: a review. Mol Ecol. 2002;11:1&#8211;16."/>

    <meta name="citation_reference" content="citation_title=Molecular Ecology; citation_inbook_title=Molecular markers in ecology; citation_publication_date=2005; citation_pages=43-57; citation_id=CR15; citation_author=JR Freeland; citation_publisher=Chichester"/>

    <meta name="citation_reference" content="citation_journal_title=Ecol Lett; citation_title=Microsatellites for ecologists: a practical guide to using and evaluating microsatellite markers; citation_author=KA Selkoe, RJ Toonen; citation_volume=9; citation_publication_date=2006; citation_pages=615-629; citation_doi=10.1111/j.1461-0248.2006.00889.x; citation_id=CR16"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=PowerCore: a program applying the advanced M strategy with a heuristic search for establishing core sets; citation_author=KW Kim, HK Chung, GT Cho, KH Ma, D Chandrabalan, JG Gwag; citation_volume=23; citation_publication_date=2007; citation_pages=2155-2162; citation_doi=10.1093/bioinformatics/btm313; citation_id=CR17"/>

    <meta name="citation_reference" content="citation_journal_title=Genes Genom; citation_title=Analysis of genetic diversity among indigenous landraces from sesame (Sesamum indicum L.) core collection in China as revealed by SRAP and SSR markers; citation_author=Y Zhang, X Zhang, W Hua, L Wang, C Zhuo; citation_volume=32; citation_publication_date=2010; citation_pages=207-215; citation_doi=10.1007/s13258-009-0888-6; citation_id=CR18"/>

    <meta name="citation_reference" content="citation_journal_title=Tree Genet Genomes; citation_title=Developing a core collection of olive (Olea europaea L.) based on molecular markers (DArTs, SSRs, SNPs) and agronomic traits; citation_author=A Belaj, MDC Dominguez-Garc&#237;a, SG Atienza, NM Urd&#237;roz, RD Rosa, Z Satovic; citation_volume=8; citation_publication_date=2012; citation_pages=365-378; citation_doi=10.1007/s11295-011-0447-6; citation_id=CR19"/>

    <meta name="citation_reference" content="citation_journal_title=Breed Sci; citation_title=Evaluation of soybean germplasm conserved in NIAS genebank and development of mini core collections; citation_author=A Kaga, T Shimizu, S Watanabe, Y Tsubokura, Y Katayose, K Harada; citation_volume=61; citation_publication_date=2012; citation_pages=566-592; citation_doi=10.1270/jsbbs.61.566; citation_id=CR20"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Breed; citation_title=Efficiency of PowerCore in core set development using amplified fragment length polymorphic markers in mungbean; citation_author=KT Moe, JG Gwag, YJ Park; citation_volume=131; citation_publication_date=2012; citation_pages=110-117; citation_doi=10.1111/j.1439-0523.2011.01896.x; citation_id=CR21"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Breed; citation_title=Identification of a diverse mini-core panel of Indian rice germplasm based on genotyping using microsatellite markers; citation_author=KK Tiwari, A Singh, S Pattnaik, M Sandhu, S Kaur, S Jain; citation_volume=134; citation_publication_date=2015; citation_pages=164-171; citation_doi=10.1111/pbr.12252; citation_id=CR22"/>

    <meta name="citation_reference" content="citation_journal_title=Acta Hortic Sin; citation_title=A method for establishing core collection of Pleurotus ostreatus cultivated in China based on SSR markers; citation_author=H Li, Q Chen, CY Huang, BG Xie, A Zhang; citation_volume=39; citation_publication_date=2012; citation_pages=2023-2032; citation_id=CR23"/>

    <meta name="citation_reference" content="citation_journal_title=Mycosystema; citation_title=Development of EST-SSR markers in Pleurotus ostreatus and construction of primary core collection; citation_author=XM Zhang, Q Yao, ZY Gong, XX Gao, J Li, X Liu; citation_volume=32; citation_publication_date=2013; citation_pages=64-80; citation_id=CR24"/>

    <meta name="citation_reference" content="citation_journal_title=J Basic Microbiol; citation_title=Evaluating genetic diversity and constructing core collections of Chinese Lentinula edodes cultivars using ISSR and SRAP markers; citation_author=J Liu, Z-R Wang, C Li, Y-B Bian, Y Xiao; citation_volume=55; citation_publication_date=2015; citation_pages=749-760; citation_doi=10.1002/jobm.201400774; citation_id=CR25"/>

    <meta name="citation_reference" content="citation_journal_title=Biosci Biotechnol Biochem; citation_title=Use of whole crop sorghums as a raw material in consolidated bioprocessing bioethanol production using Flammulina velutipes
                           ; citation_author=R Mizuno, H Ichinose, M Honda, K Takabatake, I Sotome, T Takai; citation_volume=73; citation_publication_date=2009; citation_pages=1671-1673; citation_doi=10.1271/bbb.90099; citation_id=CR26"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=Comparative genomics of the mating-type loci of the mushroom Flammulina velutipes reveals widespread synteny and recent inversions; citation_author=AF Peer, SY Park, PG Shin, KY Jang, YB Yoo, YJ Park; citation_volume=6; citation_publication_date=2011; citation_pages=e22249; citation_doi=10.1371/journal.pone.0022249; citation_id=CR27"/>

    <meta name="citation_reference" content="citation_journal_title=Carbohydr Polym; citation_title=Bioactivity of the crude polysaccharides from fermented soybean curd residue by Flammulina velutipes
                           ; citation_author=M Shi, Y Yang, D Guan, Y Zhang, Z Zhang; citation_volume=89; citation_publication_date=2012; citation_pages=1268-1276; citation_doi=10.1016/j.carbpol.2012.04.047; citation_id=CR28"/>

    <meta name="citation_reference" content="citation_journal_title=Edible Fungi China; citation_title=Quality comparison and analysis on white Flammulina velutipes grown with bottle lines in China; citation_author=X Li, Y Li; citation_volume=33; citation_publication_date=2014; citation_pages=20-24; citation_id=CR29"/>

    <meta name="citation_reference" content="citation_journal_title=Gene; citation_title=Genetic diversity and breeding history of winter mushroom (Flammulina velutipes) in China uncovered by genomic SSR markers; citation_author=XB Liu, B Feng, J Li, C Yan, ZL Yang; citation_volume=591; citation_publication_date=2016; citation_pages=227-235; citation_doi=10.1016/j.gene.2016.07.009; citation_id=CR30"/>

    <meta name="citation_reference" content="citation_journal_title=Phytochem Bull; citation_title=A rapid DNA isolation procedure for small quantities of fresh leaf tissue; citation_author=JJ Doyle, JL Doyle; citation_volume=19; citation_publication_date=1987; citation_pages=11-15; citation_id=CR31"/>

    <meta name="citation_reference" content="Yeh FC, Yang RC, Boyle T. POPGENE, version 1.31. Microsoft window-based freeware for population genetic analysis. Alberta: University of Alberta; 1999. p. 1&#8211;28."/>

    <meta name="citation_reference" content="citation_journal_title=J Classif; citation_title=Phylogenetic models and reticulations; citation_author=FJ Rohlf; citation_volume=17; citation_publication_date=2000; citation_pages=185-189; citation_doi=10.1007/s003570000017; citation_id=CR33"/>

    <meta name="citation_reference" content="Evanno G, Regnaut S. Goudet J. Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study. Mol Ecol. 2005;14:2611&#8211;20."/>

    <meta name="citation_reference" content="citation_journal_title=Conserv Genet Resour; citation_title=vonHoldt BM. STRUCTURE HARVESTER: a website and program for visualizing STRUCTURE output and implementing the Evanno method; citation_author=DA Earl; citation_volume=4; citation_publication_date=2012; citation_pages=359-361; citation_doi=10.1007/s12686-011-9548-7; citation_id=CR35"/>

    <meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=Trends in genetic diversity among European maize cultivars and their parental components during the past 50 years; citation_author=JC Reif, S Hamrit, M Heckenberger, W Schipprack, HP Maurer, M Bohn; citation_volume=111; citation_publication_date=2005; citation_pages=838-845; citation_doi=10.1007/s00122-005-0004-5; citation_id=CR36"/>

    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=The molecular genetics of crop domestication; citation_author=JF Doebley, BS Gau, BD Smith; citation_volume=127; citation_publication_date=2006; citation_pages=1309-1321; citation_doi=10.1016/j.cell.2006.12.006; citation_id=CR37"/>

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci U S A; citation_title=Impacts of genetic bottlenecks on soybean genome diversity; citation_author=DL Hyten, Q Song, Y Zhu, IY Choi, RL Nelson, JM Costa; citation_volume=103; citation_publication_date=2006; citation_pages=16666-16671; citation_doi=10.1073/pnas.0604379103; citation_id=CR38"/>

    <meta name="citation_reference" content="citation_journal_title=Genet Soc Am; citation_title=Analysis of inbreeding depression in Agaricus bisporus
                           ; citation_author=J Xu; citation_volume=141; citation_publication_date=1995; citation_pages=137-145; citation_id=CR39"/>

    <meta name="citation_reference" content="citation_journal_title=J Org Chem; citation_title=Bioactive Sesquiterpenes from the edible mushroom Flammulina velutipes and their biosynthetic pathway confirmed by genome analysis and chemical evidence; citation_author=Q Tao, K Ma, Y Yang, K Wang, B Chen, Y Huang; citation_volume=81; citation_publication_date=2016; citation_pages=9867-9877; citation_doi=10.1021/acs.joc.6b01971; citation_id=CR40"/>

    <meta name="citation_author" content="Xiao Bin Liu"/>

    <meta name="citation_author_institution" content="Key Laboratory for Plant Diversity and Biogeography of East Asia, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, China"/>

    <meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing, China"/>

    <meta name="citation_author" content="Jing Li"/>

    <meta name="citation_author_institution" content="Key Laboratory for Plant Diversity and Biogeography of East Asia, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, China"/>

    <meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing, China"/>

    <meta name="citation_author_institution" content="State Key Laboratory of Conservation and Utilization for Bioresources in Yunnan, Yunnan University, Kunming, China"/>

    <meta name="citation_author" content="Zhu L. Yang"/>

    <meta name="citation_author_institution" content="Key Laboratory for Plant Diversity and Biogeography of East Asia, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, China"/>



        <meta charset="UTF-8"/>
        <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
        <meta name="viewport" content="width=device-width, initial-scale=1"/>

        <!-- For IE 9 and below -->
<!--[if IE]><link rel="shortcut icon" href=/static/images/favicons//favicon.ico><![endif]-->
<!-- Touch Icons - iOS and Android 2.1+ -->
<link rel="apple-touch-icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>
<!-- Firefox, Chrome, Safari, IE 11+ and Opera -->
<link rel="icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>


        <script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script>
        <script>(function(d){if (sessionStorage.fontsLoaded){d.style.setProperty('--font-family-sans', 'Europa');d.style.setProperty('--font-family-serif', 'Noto Serif')}})(document.documentElement);</script>

        <link rel="stylesheet" media="screen" href=/static/styles/basic/basic-a31ca85bc2.css>
<link rel="stylesheet" media="print" href=/static/styles/bmc/print-3a6240f456.css>

<link rel="stylesheet" class="js-ctm" href=/static/styles/bmc/main-84414d3d47.css
      media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">

        

        <script src="/static/js/app-bundle-8d61e6ef37.js" defer></script>
        
        <script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script>
        
    <script type="text/javascript">
        config = {
            env: 'live',
            site: 'hereditasjournal.biomedcentral.com',
            siteWithPath: 'hereditasjournal.biomedcentral.com' + window.location.pathname,
            twitterHashtag: '',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/s41065-017-0038-0',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central'
        };
    </script>

        
    <script type="text/javascript">
        dataLayer = [{"content":{"article":{"doi":"10.1186/s41065-017-0038-0","articleType":"Research","peerReviewType":"Closed","supplement":null,"keywords":"Core set;Molecular breeding;Molecular marker;Microsatellites"},"contentInfo":{"imprint":"BioMed Central","title":"Genetic diversity and structure of core collection of winter mushroom ( Flammulina velutipes ) developed by genomic SSR markers","publishedAt":1499040000000,"publishedAtDate":"2017-07-03","author":["Xiao Bin Liu","Jing Li","Zhu L. Yang"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":"CC BY + CC0","openAccess":true},"environment":"live"},"journal":{"siteKey":"hereditasjournal.biomedcentral.com","volume":"155","issue":"1","title":"Hereditas","type":"AJs Life","journalID":41065,"gaCode":"UA-62066850-11","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Research","publishingSegment":"Genetics & Heredity (Life)"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}}}]

    </script>

        <script src="https://cdn.cookielaw.org/consent/7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script>
        
    
        
    

        
    
        
            
                <!-- Google Tag Manager -->
                <script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script>
                <!-- End Google Tag Manager -->
            
        
    


        
    
    <link rel="canonical" href="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0038-0"/>
    

        
        
        <meta property="og:url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0038-0"/>
        <meta property="og:type" content="article"/>
        <meta property="og:site_name" content="Hereditas"/>
        <meta property="og:title" content="Genetic diversity and structure of core collection of winter mushroom ( Flammulina velutipes ) developed by genomic SSR markers"/>
        <meta property="og:description" content="A core collection is a subset of an entire collection that represents as much of the genetic diversity of the entire collection as possible. The establishment of a core collection for crops is practical for efficient management and use of germplasm. However, the establishment of a core collection of mushrooms is still in its infancy, and no established core collection of the economically important species Flammulina velutipes has been reported. We established the first core collection of F. velutipes, containing 32 strains based on 81 genetically different F. veltuipes strains. The allele retention proportion of the core collection for the entire collection was 100%. Moreover, the genetic diversity parameters (the effective number of alleles, Nei’s expected heterozygosity, the number of observed heterozygosity, and Shannon’s information index) of the core collection showed no significant differences from the entire collection (p&amp;nbsp;&amp;gt;&amp;nbsp;0.01). Thus, the core collection is representative of the genetic diversity of the entire collection. Genetic structure analyses of the core collection revealed that the 32 strains could be clustered into 6 groups, among which groups 1 to 3 were cultivars and groups 4 to 6 were wild strains. The wild strains from different locations harbor their own specific alleles, and were clustered stringently in accordance with their geographic origins. Genetic diversity analyses of the core collection revealed that the wild strains possessed greater genetic diversity than the cultivars. We established the first core collection of F. velutipes in China, which is an important platform for efficient breeding of this mushroom in the future. In addition, the wild strains in the core collection possess favorable agronomic characters and produce unique bioactive compounds, adding value to the platform. More attention should be paid to wild strains in further strain breeding."/>
        <meta property="og:image" content="https://images.springer.com/sgw/journals/medium/41065.jpg"/>
    
    </head>

    <body class="journal journal-fulltext"
          
          >
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK"
                            height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><feColorMatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"/></filter></defs><symbol id="icon-arrow-down" viewBox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"/></symbol><symbol id="icon-arrow-left-bullet" viewBox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"/></symbol><symbol id="icon-arrow-right-bullet" viewBox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"/></symbol><symbol id="icon-arrow-right" viewBox="0 0 32 64"><path d="M0 0v64l32-32z"/></symbol><symbol id="icon-close" viewBox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"/></symbol><symbol id="icon-community" viewBox="0 0 60.643 31.01"><mask maskUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><path fill="#FFF" d="M0 31.01h60.643V0H0z" filter="url(#a)"/></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"/></symbol><symbol id="icon-download" viewBox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"/></symbol><symbol id="icon-ext-link" viewBox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"/><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"/><path fill="#676767" d="M9.7 0L16 6.2V0z"/></symbol><symbol id="icon-info-bordered" viewBox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"/><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"/></symbol><symbol id="icon-opr" viewBox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"/><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"/></symbol><symbol id="icon-quotation" viewBox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"/></symbol><symbol id="icon-remove" viewBox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"/><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"/></symbol><symbol id="icon-ribbon" viewBox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"/></symbol><symbol id="icon-rss" viewBox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"/><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"/><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"/></g></symbol><symbol id="icon-search" viewBox="0 0 16 16"><path d="M6.276 12.257C2.815 12.257 0 9.508 0 6.128 0 2.749 2.815 0 6.276 0c3.462 0 6.277 2.749 6.277 6.128.002 3.38-2.815 6.129-6.277 6.129zm0-10.772c-2.622 0-4.755 2.083-4.755 4.643 0 2.562 2.133 4.644 4.755 4.644 2.623 0 4.756-2.082 4.756-4.644 0-2.56-2.134-4.643-4.756-4.643z"/><path d="M16 13.751L13.641 16l-4.875-4.82 2.36-2.248L16 13.751z"/></symbol><symbol id="icon-speech-bubble" viewBox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"/></symbol><symbol id="icon-error" viewBox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"/></symbol><symbol id="icon-info" viewBox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"/></symbol><symbol id="icon-success" viewBox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"/></symbol><symbol id="icon-warning" viewBox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"/></symbol><symbol id="icon-facebook-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"/></symbol><symbol id="icon-facebook" viewBox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"/><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"/></symbol><symbol id="icon-gplus-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M480.5 273.117c.123.076.26.185.416.334.146.152.289.343.432.562.135.211.258.459.359.746.086.286.129.619.129 1.004-.012.7-.164 1.262-.465 1.683a5.374 5.374 0 0 1-.461.562c-.186.176-.375.35-.58.527a2.65 2.65 0 0 0-.322.41.908.908 0 0 0-.176.553c0 .205.059.375.18.508.102.129.199.238.299.336l.662.545a6.48 6.48 0 0 1 1.086 1.119c.293.415.445.955.459 1.625 0 .949-.42 1.789-1.256 2.521-.869.76-2.123 1.15-3.762 1.177-1.373-.015-2.4-.308-3.074-.879-.682-.534-1.023-1.172-1.023-1.917 0-.359.111-.766.336-1.211.215-.443.602-.837 1.172-1.174a6.126 6.126 0 0 1 1.998-.725 15.415 15.415 0 0 1 1.723-.173 3.996 3.996 0 0 1-.377-.596 1.347 1.347 0 0 1-.193-.74c0-.174.023-.317.074-.433a8.83 8.83 0 0 1 .121-.338 5.973 5.973 0 0 1-.633.035c-1.043-.012-1.838-.339-2.381-.979-.574-.603-.861-1.3-.861-2.097 0-.961.404-1.832 1.217-2.616.559-.46 1.135-.757 1.736-.898a8.494 8.494 0 0 1 1.67-.182h3.916l-1.209.704h-1.207v.005l-.005.002zm.754 11.276c0-.498-.162-.932-.484-1.302-.346-.348-.881-.778-1.609-1.29a4.045 4.045 0 0 0-.439-.02c-.102-.016-.354 0-.768.039a8.141 8.141 0 0 0-1.244.275c-.1.036-.24.096-.42.17a2.713 2.713 0 0 0-.553.355 1.985 1.985 0 0 0-.459.592c-.145.25-.215.551-.215.901 0 .687.311 1.253.932 1.698.596.444 1.404.68 2.432.688.928-.014 1.629-.217 2.117-.612.476-.385.71-.886.71-1.494zm-2.729-5.743a1.843 1.843 0 0 0 1.289-.556c.164-.249.271-.506.318-.767a7.32 7.32 0 0 0 .041-.659c0-.772-.195-1.552-.594-2.336a2.611 2.611 0 0 0-.73-.921 1.894 1.894 0 0 0-1.064-.352 1.723 1.723 0 0 0-1.324.64c-.299.437-.438.92-.428 1.457 0 .709.209 1.444.623 2.211.201.356.459.659.775.912.323.248.686.371 1.094.371zm9.698.077h-2.055v-2.056h-.996v2.056h-2.053v.998h2.053v2.052h.996v-2.052h2.055z"/></symbol><symbol id="icon-gplus" viewBox="0 0 24 24"><circle fill="#D84A36" cx="12" cy="12" r="12"/><g fill="#FFF"><path d="M12.012 12.111c-.304-.33-.07-.842.155-1.078.226-.241.639-.56.829-.74.193-.184.66-.605.674-1.868 0 0 .112-1.139-1.127-2.06h.977l1.038-.593h-3.507s-2.083-.067-3.085 1.207c-1 1.276-.966 2.871.114 3.77 1.08.898 2.371.648 2.371.648s-.197.422-.163.775c.036.353.421.849.504.965 0 0-2.386-.011-3.551 1.033-1.165 1.053-1.29 3.097 1.021 3.803 2.312.707 5.303-.055 5.906-2.277 0 0 .262-.908-.189-1.683-.459-.775-1.663-1.571-1.967-1.902zm-2.555-1.83c-.744-1-1.308-2.953 0-3.783 0 0 .955-.579 1.801.266.846.846 1.082 2.277.928 3.003-.15.727-.563.897-.739 1.008-.185.111-1.249.504-1.99-.494zm3.668 5.481c-.035.393-.083 1.254-1.491 1.664-1.409.408-3.826-.264-3.757-1.914 0 0 .006-.698.449-1.089.445-.396 1.094-.794 3.029-.819 0 0 .643.473 1.072.809.427.332.733.952.698 1.349z"/><path d="M16.323 11.055V9.803h-.752v1.252h-1.255v.748h1.255v1.256h.752v-1.256h1.248v-.748z"/></g></symbol><symbol id="icon-linkedin" viewBox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"/><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"/><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"/></g></symbol><symbol id="icon-reddit" viewBox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"/><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"/><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"/><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"/><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"/><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"/><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"/><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"/></symbol><symbol id="icon-twitter-bordered" viewBox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"/><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/></symbol><symbol id="icon-twitter" viewBox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"/><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"/></symbol><symbol id="icon-weibo-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"/><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"/><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"/></symbol><symbol id="icon-weibo" viewBox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"/><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"/><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"/><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"/><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"/><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"/></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner c-banner--info">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner" >
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1"
                 data-gpt-unitpath="/270604982/bmc/hereditasjournal/articles"
                 data-gpt-sizes="728x90,970x90"
                 data-gpt-targeting="pos=LB1;doi=10.1186/s41065-017-0038-0;kwrd=Core set,Molecular breeding,Molecular marker,Microsatellites;pmc=L32030,L21001;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-017-0038-0&amp;kwrd=Core set,Molecular breeding,Molecular marker,Microsatellites&amp;pmc=L32030,L21001&amp;">
                        <img data-test="gpt-advert-fallback-img"
                             src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-017-0038-0&amp;kwrd=Core set,Molecular breeding,Molecular marker,Microsatellites&amp;pmc=L32030,L21001&amp;"
                             alt="Advertisement"
                             width="728"
                             height="90">
                    </a>
                </noscript>
            </div>
        </div>
    </div>

        
         
    <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>


<header class="c-popup-search ctx-publisher-header" id="top" data-test="publisher-header">
    <nav class="c-navbar">
        <div class="u-container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization" />
                <div class="c-logo">
    <a href="https://www.biomedcentral.com" itemprop="url">
        <img src=/static/images/bmc/logos/logo-bmc-white-aj-e3a70ae848.svg alt="BMC" itemprop="logo">
        <div class="c-logo__strapline">
            <img src=/static/images/bmc/logos/logo-bmc-white-strapline-sn-e7b1d6e67f.svg alt="Part of Springer Nature">
        </div>
    </a>
</div>
            </div>
            <button
                type="button"
                class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg"
                id="publisher-header-menu-button"
                data-toggle="collapse"
                data-test="publisher-header-menu-toggle"
                data-toggle-linked="publisher-header"
                data-target="publisher-header-search publisher-header-menu"
                aria-controls="publisher-header-menu publisher-header-search"
                aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


                <ul class="c-navbar__nav">
                    <li class="c-navbar__item">
                        <a data-replace="true"
                           data-placeholder="/placeholder/v1/username"
                           class="c-navbar__link"
                           href="https://www.biomedcentral.com/login"
                           data-test="login-link"
                           data-track="click"
                           data-track-category="Account"
                           data-track-action="Click Login Link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                    <li class="c-navbar__item">
                        <button
                            type="button"
                            class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg"
                            data-toggle="collapse"
                            data-test="header-search-button"
                            data-target="publisher-header-search publisher-header-menu publisher-header-menu-button"
                            aria-controls="publisher-header-search publisher-header-menu"
                            aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true">
                                    <use xlink:href="#icon-search"></use>
                                </svg>
                            </span>
                        </button>
                    </li>
                </ul>
            </div>
        </div>
    </nav>
    <div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field c-form-field--inline" method="GET" action="//www.biomedcentral.com/search" data-track="submit"
        data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label data-track-label="" data-test="global-search">
        <label for="publisherSearch" class="u-visually-hidden">Search all BMC articles</label>
        <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text" value="" placeholder="Search all BMC articles"/>
        <button class="c-button c-button--secondary" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true">
        <use xlink:href="#icon-search"></use>
    </svg>
</button>


        <input type="hidden" name="searchType" value="publisherSearch"/>
    </form>
</div>

            </div>
        </div>
    </div>
</header>


        
            <header class="c-journal-header c-journal-header--hereditas ctx-journal-header">
                <div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="/">
    
        
    
    <span class="c-journal-title__text ">Hereditas</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="u-container">
                        
                            <button type="button"
                                    class="c-navbar__link c-dropdown u-hidden-at-lg"
                                    data-toggle="collapse"
                                    data-test="journal-header-menu"
                                    data-toggle-linked="journal-header"
                                    data-target="journal-menu"
                                    aria-controls="journal-menu"
                                    aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav">
                                    <ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/'>Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/about'>About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href='/articles'>Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/submission-guidelines'>Submission Guidelines</a>
                                            </li>
                                        
                                    </ul>
                                </nav>
                            </div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--ocean-light-blue">
                    
                </div>
            </header>
            
        

        <div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-content-layout c-content-layout--fulltext" data-component="sticky-container">
                <div class="c-content-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3><ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc=""><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Methods">Methods</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec7" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results">Results</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec11" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Discussion">Discussion</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec14" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Conclusions">Conclusions</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li></ol></nav>
                </div>

                <main class="c-content-layout__main">
                    <article itemscope itemtype="http://schema.org/ScholarlyArticle">
                        
                        
    <div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
            <li class="c-article-identifiers__item" data-test="article-category">Research</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
    </div>

                        <div class="FulltextWrapper"><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">Genetic diversity and structure of core collection of winter mushroom (<em xmlns="" class="EmphasisTypeItalic">Flammulina velutipes</em>) developed by genomic SSR markers</h1></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true"><ul class="u-listReset"><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Xiao&nbsp;Bin&nbsp;Liu</span><sup><a href="#Aff1" class="AffiliationID">1</a>, <a href="#Aff2" class="AffiliationID">2</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Jing&nbsp;Li</span><sup><a href="#Aff1" class="AffiliationID">1</a>, <a href="#Aff2" class="AffiliationID">2</a>, <a href="#Aff3" class="AffiliationID">3</a></sup> and </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Zhu&nbsp;L.&nbsp;Yang</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup><a href="mailto:fungi@mail.kib.ac.cn" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a></li></ul></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation"><div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">Hereditas</span></span><span class="ArticleCitation_Year">2017</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">155</strong>:3</span></span></div><p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/s41065-017-0038-0">https://doi.org/10.1186/s41065-017-0038-0</a></p><p class="Copyright">© &nbsp;The Author(s)&nbsp;2017</p></div></div><ul class="ArticleHistory u-reset-list"><li class="History HistoryReceived"><strong>Received: </strong><span>21&nbsp;February&nbsp;2017</span></li><li class="History HistoryAccepted"><strong>Accepted: </strong><span>22&nbsp;June&nbsp;2017</span></li><li class="History HistoryOnlineDate"><strong>Published: </strong><span itemprop="datePublished">3&nbsp;July&nbsp;2017</span></li></ul></div>
                        
                        
                        
    

                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2><div class="c-section__content" id="Abstract"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p id="Par1" class="Para">A core collection is a subset of an entire collection that represents as much of the genetic diversity of the entire collection as possible. The establishment of a core collection for crops is practical for efficient management and use of germplasm. However, the establishment of a core collection of mushrooms is still in its infancy, and no established core collection of the economically important species <em xmlns="" class="EmphasisTypeItalic">Flammulina velutipes</em> has been reported.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Results</h3>
                <p id="Par2" class="Para">We established the first core collection of <em xmlns="" class="EmphasisTypeItalic">F. velutipes</em>, containing 32 strains based on 81 genetically different <em xmlns="" class="EmphasisTypeItalic">F. veltuipes</em> strains. The allele retention proportion of the core collection for the entire collection was 100%. Moreover, the genetic diversity parameters (the effective number of alleles, Nei’s expected heterozygosity, the number of observed heterozygosity, and Shannon’s information index) of the core collection showed no significant differences from the entire collection (<em xmlns="" class="EmphasisTypeItalic">p</em>&nbsp;&gt;&nbsp;0.01). Thus, the core collection is representative of the genetic diversity of the entire collection. Genetic structure analyses of the core collection revealed that the 32 strains could be clustered into 6 groups, among which groups 1 to 3 were cultivars and groups 4 to 6 were wild strains. The wild strains from different locations harbor their own specific alleles, and were clustered stringently in accordance with their geographic origins. Genetic diversity analyses of the core collection revealed that the wild strains possessed greater genetic diversity than the cultivars.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusion</h3>
                <p id="Par3" class="Para">We established the first core collection of <em xmlns="" class="EmphasisTypeItalic">F. velutipes</em> in China, which is an important platform for efficient breeding of this mushroom in the future. In addition, the wild strains in the core collection possess favorable agronomic characters and produce unique bioactive compounds, adding value to the platform. More attention should be paid to wild strains in further strain breeding.</p>
              </div></div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords"><li class="c-keywords__item" itemprop="about">Core set</li><li class="c-keywords__item" itemprop="about">Molecular breeding</li><li class="c-keywords__item" itemprop="about">Molecular marker</li><li class="c-keywords__item" itemprop="about">Microsatellites</li></ul></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par5" class="Para">A core collection is a subset of accessions that presents the maximum possible genetic diversity contained in an entire collection with minimum redundancy [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>, <span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>]. The establishment of a core collection for crops is practical for efficient management of germplasm. Core collections of most major food crops, such as <em xmlns="" class="EmphasisTypeItalic">Oryza sativa</em>, <em xmlns="" class="EmphasisTypeItalic">Zea mays</em>, <em xmlns="" class="EmphasisTypeItalic">Glycine max</em> and <em xmlns="" class="EmphasisTypeItalic">Triticum aestivum</em>, have already been established [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>–<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par6" class="Para">A core collection is traditionally constructed based on morphological and agronomic characters using different strategies, such as the constant allocation (C) strategy, the logarithm (L) strategy, the proportional allocation (P) strategy, and the random sampling (R) strategy [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>, <span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>–<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>]. However, most morphological and agronomic characters are quantitative traits that can be easily affected by environmental variation [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>–<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>]. Therefore, phenotypic data cannot directly reflect the genetic diversity of germplasm resources [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par7" class="Para">Conversely, molecular markers can directly reflect a germplasm’s genetic diversity at the DNA sequence level. Compared with other molecular markers, simple sequence repeats (SSRs) are randomly repeated DNA sequences, generally 1 to 6 base pairs in length per unit. SSRs can spread extensively throughout a genome. They are typically co-dominant, highly polymorphic, reproducible and easy to score [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>–<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>]. Based on molecular marker data, Kim et al. [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>] developed software named PowerCore by applying the advanced maximization (M) strategy with heuristic searching to establish a core collection (allele mining collection); it allows all alleles to be captured in a minimum number of accessions [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>]. It has been successfully used with many economically important crops, such as <em xmlns="" class="EmphasisTypeItalic">Oryza sativa</em>, <em xmlns="" class="EmphasisTypeItalic">Glycine max</em>, <em xmlns="" class="EmphasisTypeItalic">Olea europaea</em>, <em xmlns="" class="EmphasisTypeItalic">Vigna radiata</em>, and <em xmlns="" class="EmphasisTypeItalic">Sesamum indicum</em> and has been proven to be most suitable for establishing a core collection based on molecular data [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>–<span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par8" class="Para">However, the development of core collections of edible mushrooms is still at an early stage, and core collections have been established only in <em xmlns="" class="EmphasisTypeItalic">Pleurotus ostreatus</em> and <em xmlns="" class="EmphasisTypeItalic">Lentinula edodes</em> [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>–<span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>]. <em xmlns="" class="EmphasisTypeItalic">Flammulina velutipes</em> is cultivated on a large scale in East Asia [<span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>–<span class="CitationRef"><a href="#CR28" data-toggle="citation">28</a></span>]. China is currently the largest producer of <em xmlns="" class="EmphasisTypeItalic">F. velutipes</em>, with an annual production of 2.4 million tons [<span class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>]. In our previous study, we obtained 124 strains (110 cultivars from the spawn market of China and 14 wild strains from Yunnan, Sichuan, and Hunan provinces), and excluded cultivars labeled with confusing names, then screened out 81 strains that are genetically different [<span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>]. In order to efficiently manage and utilize of these genetically different strains, a smaller representative core collection without redundant strains is urgently needed.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par9" class="Para">In this study, we aimed to (i) establish the core collection of <em xmlns="" class="EmphasisTypeItalic">F. velutipes</em>; (ii) evaluate the genetic diversity of the core collection and the entire collection; and (iii) analyze the core collection’s genetic structure.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Methods" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Methods">Methods</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Methods" data-test="collapse-section"><section id="Sec3" class="Section2 RenderAsSection2">
                <h3 class="Heading">Strain materials and DNA extraction</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par10" class="Para">We used 81 strains of <em xmlns="" class="EmphasisTypeItalic">F. velutipes</em> in this study, including 67 cultivars and 14 wild strains (Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>: Table S1). Genomic DNA was extracted for each strain with the CTAB-based method [<span class="CitationRef"><a href="#CR31" data-toggle="citation">31</a></span>]. In each case, fresh mycelium was harvested from potato dextrose agar medium after inoculation for 10&nbsp;days at 23&nbsp;°C. The DNA concentration and purity were measured with a NanoDrop2000 spectrophotometer. The DNA solution of each sample was diluted to 100&nbsp;ng/μl.</p>
              </section><section id="Sec4" class="Section2 RenderAsSection2">
                <h3 class="Heading">SSR genotyping</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par11" class="Para">The 25 polymorphic SSR markers used in this study were developed by our previous study [<span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>]. The forward primer of each SSR was labeled with fluorescent dye (FAM) at the 5′ end (TSINGKE, Kunming). Polymerase chain reactions (PCR) were carried out in a total volume of 25&nbsp;μl, containing 1&nbsp;μl template DNA, 1&nbsp;μl bovine serum albumin, 2.5&nbsp;μl reaction buffer, 0.5&nbsp;μl deoxynucleoside triphosphate, 1&nbsp;μl for each primer, 0.3&nbsp;μl Taq DNA polymerase, and 17.7&nbsp;μl ddH<sub>2</sub>O. PCR was conducted on an ABI 2720 Thermal Cycler (Applied Biosystems, Foster City, CA) or an Eppendorf Master Cycler (Netheler-Hinz, Hamburg, Germany) under the following parameters: 94&nbsp;°C for 4&nbsp;min, then 35&nbsp;cycles of 94&nbsp;°C for 30&nbsp;s, 55&nbsp;°C for 30&nbsp;s, and 72&nbsp;°C for 30&nbsp;s, followed by a final extension step of 72&nbsp;°C for 8&nbsp;min. The PCR products were run in an ABI 3730 Genetic Analyzer using GeneScan 500 Rox as a size standard (Applied Biosystems); after a denaturation step at 98&nbsp;°C for 5&nbsp;min and shock chilling on ice, alleles of each locus were scored in base pairs with the GeneMapper v3.2 software package (Applied Biosystems), the size of the PCR products for each SSR was recorded in an Excel spreadsheet.</p>
              </section><section id="Sec5" class="Section2 RenderAsSection2">
                <h3 class="Heading">Development of a core collection for <em class="EmphasisTypeItalic">F. velutipes</em>
                        </h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par12" class="Para">The core collection was established based on genotyping data using PowerCore software [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>]. The heuristic algorithm that finds the optimum path from the initial to the final stages for sample selection was used (<span class="ExternalRef"><a href="http://genebank.rda"><span class="RefSource">http://genebank.rda</span></a></span>).</p>
              </section><section id="Sec6" class="Section2 RenderAsSection2">
                <h3 class="Heading">Data analysis</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par13" class="Para">The genetic diversity parameters (the effective number of alleles Ae, Nei’s expected heterozygosity H, the number of observed heterozygosity Ho, and Shannon’s information index I) and allele frequency of the core collection and of the entire collection were established with PopGene v1.31 [<span class="CitationRef"><a href="#CR32" data-toggle="citation">32</a></span>]. A dendrogram of the genetic relationships among the core collection strains was constructed based on the simple matching (SM) coefficient by applying the unweighted pair group method with arithmetic mean (UPGMA) using the NTSYSpc v2.10e [<span class="CitationRef"><a href="#CR33" data-toggle="citation">33</a></span>]. The genetic structure of the core collection was analyzed with STRUCTURE v2.3.4 based on an admixture model. Models were tested for K-values ranging from 2 to 10, with 10 independent runs per K value. For each run, the initial burn-in period was set to 100,000 with 100,000 MCMC iterations. To determine the most probable value of K, the deltaK method was used and implemented in Structure Harvester [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>, <span class="CitationRef"><a href="#CR35" data-toggle="citation">35</a></span>].</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec7" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Results" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Results">Results</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Results" data-test="collapse-section"><section id="Sec8" class="Section2 RenderAsSection2">
                <h3 class="Heading">Core collection construction</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par14" class="Para">A total of 153 alleles were amplified by 25 SSRs in the 81 strains [<span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>]. In this study, based on PowerCore calculation, the 153 alleles could be represented using a minimum of 32 strains, including 19 cultivars and 13 wild strains (Table <span class="InternalRef"><a href="#Tab1">1</a></span>). This finding suggests that the 32 strains could be a core collection of the 81 strains. The core collection sampling proportion is about 39.5%: for cultivars about 27.9%, and for wild strains about 92.9%.<figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 1</span>
                        <p xmlns="" class="SimplePara">Strains in the core collection selected by PowerCore</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Strain</p></th><th><p xmlns="" class="SimplePara">Cultivar/wild</p></th><th><p xmlns="" class="SimplePara">Pileus color</p></th><th><p xmlns="" class="SimplePara">origin</p></th><th><p xmlns="" class="SimplePara">Strain</p></th><th><p xmlns="" class="SimplePara">Cultivar/wild</p></th><th><p xmlns="" class="SimplePara">Pileus color</p></th><th><p xmlns="" class="SimplePara">origin</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">F37</p></td><td><p xmlns="" class="SimplePara">Cultivar</p></td><td><p xmlns="" class="SimplePara">Yellow</p></td><td><p xmlns="" class="SimplePara">Shandong</p></td><td><p xmlns="" class="SimplePara">F80</p></td><td><p xmlns="" class="SimplePara">Cultivar</p></td><td><p xmlns="" class="SimplePara">Yellow</p></td><td><p xmlns="" class="SimplePara">Henan</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">F114</p></td><td><p xmlns="" class="SimplePara">Cultivar</p></td><td><p xmlns="" class="SimplePara">Yellow</p></td><td><p xmlns="" class="SimplePara">Shandong</p></td><td><p xmlns="" class="SimplePara">F87</p></td><td><p xmlns="" class="SimplePara">Cultivar</p></td><td><p xmlns="" class="SimplePara">Yellow</p></td><td><p xmlns="" class="SimplePara">Hebei</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">F1</p></td><td><p xmlns="" class="SimplePara">Cultivar</p></td><td><p xmlns="" class="SimplePara">Yellow</p></td><td><p xmlns="" class="SimplePara">Beijing</p></td><td><p xmlns="" class="SimplePara">F79</p></td><td><p xmlns="" class="SimplePara">Cultivar</p></td><td><p xmlns="" class="SimplePara">Yellow</p></td><td><p xmlns="" class="SimplePara">Henan</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">F6</p></td><td><p xmlns="" class="SimplePara">Cultivar</p></td><td><p xmlns="" class="SimplePara">Yellow</p></td><td><p xmlns="" class="SimplePara">Beijing</p></td><td><p xmlns="" class="SimplePara">F146</p></td><td><p xmlns="" class="SimplePara">Wild</p></td><td><p xmlns="" class="SimplePara">Yellow</p></td><td><p xmlns="" class="SimplePara">Hunan</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">F89</p></td><td><p xmlns="" class="SimplePara">Cultivar</p></td><td><p xmlns="" class="SimplePara">Yellow</p></td><td><p xmlns="" class="SimplePara">Hebei</p></td><td><p xmlns="" class="SimplePara">F147</p></td><td><p xmlns="" class="SimplePara">Wild</p></td><td><p xmlns="" class="SimplePara">Yellow</p></td><td><p xmlns="" class="SimplePara">Hunan</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">F115</p></td><td><p xmlns="" class="SimplePara">Cultivar</p></td><td><p xmlns="" class="SimplePara">Yellow</p></td><td><p xmlns="" class="SimplePara">Shandong</p></td><td><p xmlns="" class="SimplePara">F148</p></td><td><p xmlns="" class="SimplePara">Wild</p></td><td><p xmlns="" class="SimplePara">Yellow</p></td><td><p xmlns="" class="SimplePara">Hunan</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">F58</p></td><td><p xmlns="" class="SimplePara">Cultivar</p></td><td><p xmlns="" class="SimplePara">Yellow</p></td><td><p xmlns="" class="SimplePara">Heilongjiang</p></td><td><p xmlns="" class="SimplePara">F77</p></td><td><p xmlns="" class="SimplePara">Wild</p></td><td><p xmlns="" class="SimplePara">Yellow</p></td><td><p xmlns="" class="SimplePara">Jilin</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">F112</p></td><td><p xmlns="" class="SimplePara">Cultivar</p></td><td><p xmlns="" class="SimplePara">Yellow</p></td><td><p xmlns="" class="SimplePara">Shandong</p></td><td><p xmlns="" class="SimplePara">F92</p></td><td><p xmlns="" class="SimplePara">Wild</p></td><td><p xmlns="" class="SimplePara">Yellow</p></td><td><p xmlns="" class="SimplePara">Sichuan</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">F117</p></td><td><p xmlns="" class="SimplePara">Cultivar</p></td><td><p xmlns="" class="SimplePara">White</p></td><td><p xmlns="" class="SimplePara">Liaoning</p></td><td><p xmlns="" class="SimplePara">F93</p></td><td><p xmlns="" class="SimplePara">Wild</p></td><td><p xmlns="" class="SimplePara">Yellow</p></td><td><p xmlns="" class="SimplePara">Sichuan</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">F106</p></td><td><p xmlns="" class="SimplePara">Cultivar</p></td><td><p xmlns="" class="SimplePara">Yellow</p></td><td><p xmlns="" class="SimplePara">Fujian</p></td><td><p xmlns="" class="SimplePara">F94</p></td><td><p xmlns="" class="SimplePara">Wild</p></td><td><p xmlns="" class="SimplePara">Yellow</p></td><td><p xmlns="" class="SimplePara">Sichuan</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">F133</p></td><td><p xmlns="" class="SimplePara">Cultivar</p></td><td><p xmlns="" class="SimplePara">White</p></td><td><p xmlns="" class="SimplePara">Hunan</p></td><td><p xmlns="" class="SimplePara">F91</p></td><td><p xmlns="" class="SimplePara">Wild</p></td><td><p xmlns="" class="SimplePara">Yellow</p></td><td><p xmlns="" class="SimplePara">Yunnan</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">F19</p></td><td><p xmlns="" class="SimplePara">Cultivar</p></td><td><p xmlns="" class="SimplePara">White</p></td><td><p xmlns="" class="SimplePara">Sichuan</p></td><td><p xmlns="" class="SimplePara">F149</p></td><td><p xmlns="" class="SimplePara">Wild</p></td><td><p xmlns="" class="SimplePara">Yellow</p></td><td><p xmlns="" class="SimplePara">Yunnan</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">F78</p></td><td><p xmlns="" class="SimplePara">Cultivar</p></td><td><p xmlns="" class="SimplePara">White</p></td><td><p xmlns="" class="SimplePara">Jilin</p></td><td><p xmlns="" class="SimplePara">F99</p></td><td><p xmlns="" class="SimplePara">Wild</p></td><td><p xmlns="" class="SimplePara">Yellow</p></td><td><p xmlns="" class="SimplePara">Yunnan</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">F26</p></td><td><p xmlns="" class="SimplePara">Cultivar</p></td><td><p xmlns="" class="SimplePara">White</p></td><td><p xmlns="" class="SimplePara">Shijiazhuang</p></td><td><p xmlns="" class="SimplePara">F101</p></td><td><p xmlns="" class="SimplePara">Wild</p></td><td><p xmlns="" class="SimplePara">Yellow</p></td><td><p xmlns="" class="SimplePara">Yunnan</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">F116</p></td><td><p xmlns="" class="SimplePara">Cultivar</p></td><td><p xmlns="" class="SimplePara">White</p></td><td><p xmlns="" class="SimplePara">Liaoning</p></td><td><p xmlns="" class="SimplePara">F98</p></td><td><p xmlns="" class="SimplePara">Wild</p></td><td><p xmlns="" class="SimplePara">Yellow</p></td><td><p xmlns="" class="SimplePara">Yunnan</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">F151</p></td><td><p xmlns="" class="SimplePara">Cultivar</p></td><td><p xmlns="" class="SimplePara">White</p></td><td><p xmlns="" class="SimplePara">Kunming</p></td><td><p xmlns="" class="SimplePara">F103</p></td><td><p xmlns="" class="SimplePara">Wild</p></td><td><p xmlns="" class="SimplePara">Yellow</p></td><td><p xmlns="" class="SimplePara">Yunnan</p></td></tr></tbody></table></div></figure>
                        </div>
              </section><section id="Sec9" class="Section2 RenderAsSection2">
                <h3 class="Heading">Genetic diversity of the core collection</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par15" class="Para">Statistics to describe the genetic diversity of the core collection and the entire collection for 25 SSR markers are summarized in Table <span class="InternalRef"><a href="#Tab2">2</a></span>. The <em xmlns="" class="EmphasisTypeItalic">t</em>-tests of the mean genetic diversity parameters (Ae, H, Ho, I) of the core collection and the entire collection were non-significant (<em xmlns="" class="EmphasisTypeItalic">p</em>&nbsp;&gt;&nbsp;0.01) (Table <span class="InternalRef"><a href="#Tab2">2</a></span>), which reveals that the genetic diversity of the core collection has no significant difference from that of the entire collection. In addition, based on an UPGMA dendrogram of the core collection and the entire collection (Fig. <span class="InternalRef"><a href="#Fig1">1</a></span>), the strains in different groups of the entire collection were uniformly selected for the core collection. Thus, the core collection could represent the genetic diversity of the entire collection. However, the distributions of allele frequency differ between the entire collection and the core collection (Additional file <span class="InternalRef"><a href="#MOESM2">2</a></span>: Table S2).<figure xmlns="" id="Tab2" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 2</span>
                        <p xmlns="" class="SimplePara">Genetic diversity parameters in entire collection and core collection</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col char="."></col><col char="."></col><col char="."></col><col char="."></col><col char="."></col><col char="."></col><col char="."></col><col char="."></col></colgroup><thead><tr class="header"><th rowspan="2"><p xmlns="" class="SimplePara">Locus name</p></th><th colspan="2" char="."><p xmlns="" class="SimplePara">Ae</p></th><th colspan="2" char="."><p xmlns="" class="SimplePara">H</p></th><th colspan="2" char="."><p xmlns="" class="SimplePara">Ho</p></th><th colspan="2" char="."><p xmlns="" class="SimplePara">I</p></th></tr><tr class="header"><th char="."><p xmlns="" class="SimplePara">Entire</p></th><th char="."><p xmlns="" class="SimplePara">Core</p></th><th char="."><p xmlns="" class="SimplePara">Entire</p></th><th char="."><p xmlns="" class="SimplePara">Core</p></th><th char="."><p xmlns="" class="SimplePara">Entire</p></th><th char="."><p xmlns="" class="SimplePara">Core</p></th><th char="."><p xmlns="" class="SimplePara">Entire</p></th><th char="."><p xmlns="" class="SimplePara">Core</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR1</p></td><td char="."><p xmlns="" class="SimplePara">1.833</p></td><td char="."><p xmlns="" class="SimplePara">2.779</p></td><td char="."><p xmlns="" class="SimplePara">0.455</p></td><td char="."><p xmlns="" class="SimplePara">0.64</p></td><td char="."><p xmlns="" class="SimplePara">0.37</p></td><td char="."><p xmlns="" class="SimplePara">0.5</p></td><td char="."><p xmlns="" class="SimplePara">0.926</p></td><td char="."><p xmlns="" class="SimplePara">1.29</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR2</p></td><td char="."><p xmlns="" class="SimplePara">2.846</p></td><td char="."><p xmlns="" class="SimplePara">3.85</p></td><td char="."><p xmlns="" class="SimplePara">0.649</p></td><td char="."><p xmlns="" class="SimplePara">0.74</p></td><td char="."><p xmlns="" class="SimplePara">0.617</p></td><td char="."><p xmlns="" class="SimplePara">0.719</p></td><td char="."><p xmlns="" class="SimplePara">1.286</p></td><td char="."><p xmlns="" class="SimplePara">1.589</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR4</p></td><td char="."><p xmlns="" class="SimplePara">1.051</p></td><td char="."><p xmlns="" class="SimplePara">1.136</p></td><td char="."><p xmlns="" class="SimplePara">0.049</p></td><td char="."><p xmlns="" class="SimplePara">0.12</p></td><td char="."><p xmlns="" class="SimplePara">0.037</p></td><td char="."><p xmlns="" class="SimplePara">0.094</p></td><td char="."><p xmlns="" class="SimplePara">0.141</p></td><td char="."><p xmlns="" class="SimplePara">0.299</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR5</p></td><td char="."><p xmlns="" class="SimplePara">1.025</p></td><td char="."><p xmlns="" class="SimplePara">1.065</p></td><td char="."><p xmlns="" class="SimplePara">0.025</p></td><td char="."><p xmlns="" class="SimplePara">0.061</p></td><td char="."><p xmlns="" class="SimplePara">0.025</p></td><td char="."><p xmlns="" class="SimplePara">0.063</p></td><td char="."><p xmlns="" class="SimplePara">0.075</p></td><td char="."><p xmlns="" class="SimplePara">0.161</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR7</p></td><td char="."><p xmlns="" class="SimplePara">2.949</p></td><td char="."><p xmlns="" class="SimplePara">3.717</p></td><td char="."><p xmlns="" class="SimplePara">0.661</p></td><td char="."><p xmlns="" class="SimplePara">0.731</p></td><td char="."><p xmlns="" class="SimplePara">0.716</p></td><td char="."><p xmlns="" class="SimplePara">0.625</p></td><td char="."><p xmlns="" class="SimplePara">1.241</p></td><td char="."><p xmlns="" class="SimplePara">1.424</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR15</p></td><td char="."><p xmlns="" class="SimplePara">1.964</p></td><td char="."><p xmlns="" class="SimplePara">1.882</p></td><td char="."><p xmlns="" class="SimplePara">0.491</p></td><td char="."><p xmlns="" class="SimplePara">0.469</p></td><td char="."><p xmlns="" class="SimplePara">0.346</p></td><td char="."><p xmlns="" class="SimplePara">0.25</p></td><td char="."><p xmlns="" class="SimplePara">0.684</p></td><td char="."><p xmlns="" class="SimplePara">0.662</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR19</p></td><td char="."><p xmlns="" class="SimplePara">1.839</p></td><td char="."><p xmlns="" class="SimplePara">2.004</p></td><td char="."><p xmlns="" class="SimplePara">0.456</p></td><td char="."><p xmlns="" class="SimplePara">0.501</p></td><td char="."><p xmlns="" class="SimplePara">0.58</p></td><td char="."><p xmlns="" class="SimplePara">0.594</p></td><td char="."><p xmlns="" class="SimplePara">0.728</p></td><td char="."><p xmlns="" class="SimplePara">0.854</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR21</p></td><td char="."><p xmlns="" class="SimplePara">3.905</p></td><td char="."><p xmlns="" class="SimplePara">5.02</p></td><td char="."><p xmlns="" class="SimplePara">0.744</p></td><td char="."><p xmlns="" class="SimplePara">0.801</p></td><td char="."><p xmlns="" class="SimplePara">0.457</p></td><td char="."><p xmlns="" class="SimplePara">0.438</p></td><td char="."><p xmlns="" class="SimplePara">1.576</p></td><td char="."><p xmlns="" class="SimplePara">1.757</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR22</p></td><td char="."><p xmlns="" class="SimplePara">3.619</p></td><td char="."><p xmlns="" class="SimplePara">5.171</p></td><td char="."><p xmlns="" class="SimplePara">0.724</p></td><td char="."><p xmlns="" class="SimplePara">0.807</p></td><td char="."><p xmlns="" class="SimplePara">0.444</p></td><td char="."><p xmlns="" class="SimplePara">0.313</p></td><td char="."><p xmlns="" class="SimplePara">1.59</p></td><td char="."><p xmlns="" class="SimplePara">1.949</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR23</p></td><td char="."><p xmlns="" class="SimplePara">1.552</p></td><td char="."><p xmlns="" class="SimplePara">1.468</p></td><td char="."><p xmlns="" class="SimplePara">0.356</p></td><td char="."><p xmlns="" class="SimplePara">0.319</p></td><td char="."><p xmlns="" class="SimplePara">0.37</p></td><td char="."><p xmlns="" class="SimplePara">0.25</p></td><td char="."><p xmlns="" class="SimplePara">0.624</p></td><td char="."><p xmlns="" class="SimplePara">0.618</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR24</p></td><td char="."><p xmlns="" class="SimplePara">1.613</p></td><td char="."><p xmlns="" class="SimplePara">1.779</p></td><td char="."><p xmlns="" class="SimplePara">0.38</p></td><td char="."><p xmlns="" class="SimplePara">0.438</p></td><td char="."><p xmlns="" class="SimplePara">0.173</p></td><td char="."><p xmlns="" class="SimplePara">0.219</p></td><td char="."><p xmlns="" class="SimplePara">0.78</p></td><td char="."><p xmlns="" class="SimplePara">0.946</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR25</p></td><td char="."><p xmlns="" class="SimplePara">1.212</p></td><td char="."><p xmlns="" class="SimplePara">1.556</p></td><td char="."><p xmlns="" class="SimplePara">0.175</p></td><td char="."><p xmlns="" class="SimplePara">0.357</p></td><td char="."><p xmlns="" class="SimplePara">0.074</p></td><td char="."><p xmlns="" class="SimplePara">0.125</p></td><td char="."><p xmlns="" class="SimplePara">0.475</p></td><td char="."><p xmlns="" class="SimplePara">0.874</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR26</p></td><td char="."><p xmlns="" class="SimplePara">1.148</p></td><td char="."><p xmlns="" class="SimplePara">1.253</p></td><td char="."><p xmlns="" class="SimplePara">0.129</p></td><td char="."><p xmlns="" class="SimplePara">0.202</p></td><td char="."><p xmlns="" class="SimplePara">0.136</p></td><td char="."><p xmlns="" class="SimplePara">0.219</p></td><td char="."><p xmlns="" class="SimplePara">0.3</p></td><td char="."><p xmlns="" class="SimplePara">0.45</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR32</p></td><td char="."><p xmlns="" class="SimplePara">3.187</p></td><td char="."><p xmlns="" class="SimplePara">5.988</p></td><td char="."><p xmlns="" class="SimplePara">0.686</p></td><td char="."><p xmlns="" class="SimplePara">0.833</p></td><td char="."><p xmlns="" class="SimplePara">0.259</p></td><td char="."><p xmlns="" class="SimplePara">0.219</p></td><td char="."><p xmlns="" class="SimplePara">1.669</p></td><td char="."><p xmlns="" class="SimplePara">2.163</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR45</p></td><td char="."><p xmlns="" class="SimplePara">2.385</p></td><td char="."><p xmlns="" class="SimplePara">3.127</p></td><td char="."><p xmlns="" class="SimplePara">0.581</p></td><td char="."><p xmlns="" class="SimplePara">0.68</p></td><td char="."><p xmlns="" class="SimplePara">0.482</p></td><td char="."><p xmlns="" class="SimplePara">0.313</p></td><td char="."><p xmlns="" class="SimplePara">1.12</p></td><td char="."><p xmlns="" class="SimplePara">1.388</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR87</p></td><td char="."><p xmlns="" class="SimplePara">2.617</p></td><td char="."><p xmlns="" class="SimplePara">3.894</p></td><td char="."><p xmlns="" class="SimplePara">0.618</p></td><td char="."><p xmlns="" class="SimplePara">0.743</p></td><td char="."><p xmlns="" class="SimplePara">0.457</p></td><td char="."><p xmlns="" class="SimplePara">0.406</p></td><td char="."><p xmlns="" class="SimplePara">1.196</p></td><td char="."><p xmlns="" class="SimplePara">1.559</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR65</p></td><td char="."><p xmlns="" class="SimplePara">1.276</p></td><td char="."><p xmlns="" class="SimplePara">1.767</p></td><td char="."><p xmlns="" class="SimplePara">0.216</p></td><td char="."><p xmlns="" class="SimplePara">0.434</p></td><td char="."><p xmlns="" class="SimplePara">0.148</p></td><td char="."><p xmlns="" class="SimplePara">0.313</p></td><td char="."><p xmlns="" class="SimplePara">0.536</p></td><td char="."><p xmlns="" class="SimplePara">0.95</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR124</p></td><td char="."><p xmlns="" class="SimplePara">2.383</p></td><td char="."><p xmlns="" class="SimplePara">2.922</p></td><td char="."><p xmlns="" class="SimplePara">0.580</p></td><td char="."><p xmlns="" class="SimplePara">0.658</p></td><td char="."><p xmlns="" class="SimplePara">0.482</p></td><td char="."><p xmlns="" class="SimplePara">0.563</p></td><td char="."><p xmlns="" class="SimplePara">1.039</p></td><td char="."><p xmlns="" class="SimplePara">1.289</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR133</p></td><td char="."><p xmlns="" class="SimplePara">2.176</p></td><td char="."><p xmlns="" class="SimplePara">4.047</p></td><td char="."><p xmlns="" class="SimplePara">0.541</p></td><td char="."><p xmlns="" class="SimplePara">0.753</p></td><td char="."><p xmlns="" class="SimplePara">0.222</p></td><td char="."><p xmlns="" class="SimplePara">0.313</p></td><td char="."><p xmlns="" class="SimplePara">1.254</p></td><td char="."><p xmlns="" class="SimplePara">1.755</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR95</p></td><td char="."><p xmlns="" class="SimplePara">1.895</p></td><td char="."><p xmlns="" class="SimplePara">1.979</p></td><td char="."><p xmlns="" class="SimplePara">0.472</p></td><td char="."><p xmlns="" class="SimplePara">0.495</p></td><td char="."><p xmlns="" class="SimplePara">0.605</p></td><td char="."><p xmlns="" class="SimplePara">0.656</p></td><td char="."><p xmlns="" class="SimplePara">0.725</p></td><td char="."><p xmlns="" class="SimplePara">0.742</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR107</p></td><td char="."><p xmlns="" class="SimplePara">1.553</p></td><td char="."><p xmlns="" class="SimplePara">2.186</p></td><td char="."><p xmlns="" class="SimplePara">0.356</p></td><td char="."><p xmlns="" class="SimplePara">0.543</p></td><td char="."><p xmlns="" class="SimplePara">0.272</p></td><td char="."><p xmlns="" class="SimplePara">0.344</p></td><td char="."><p xmlns="" class="SimplePara">0.826</p></td><td char="."><p xmlns="" class="SimplePara">1.2</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR119</p></td><td char="."><p xmlns="" class="SimplePara">1.225</p></td><td char="."><p xmlns="" class="SimplePara">1.535</p></td><td char="."><p xmlns="" class="SimplePara">0.184</p></td><td char="."><p xmlns="" class="SimplePara">0.349</p></td><td char="."><p xmlns="" class="SimplePara">0.086</p></td><td char="."><p xmlns="" class="SimplePara">0.125</p></td><td char="."><p xmlns="" class="SimplePara">0.425</p></td><td char="."><p xmlns="" class="SimplePara">0.726</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR128</p></td><td char="."><p xmlns="" class="SimplePara">1.705</p></td><td char="."><p xmlns="" class="SimplePara">1.636</p></td><td char="."><p xmlns="" class="SimplePara">0.413</p></td><td char="."><p xmlns="" class="SimplePara">0.389</p></td><td char="."><p xmlns="" class="SimplePara">0.469</p></td><td char="."><p xmlns="" class="SimplePara">0.313</p></td><td char="."><p xmlns="" class="SimplePara">0.647</p></td><td char="."><p xmlns="" class="SimplePara">0.657</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR132</p></td><td char="."><p xmlns="" class="SimplePara">3.024</p></td><td char="."><p xmlns="" class="SimplePara">4</p></td><td char="."><p xmlns="" class="SimplePara">0.669</p></td><td char="."><p xmlns="" class="SimplePara">0.75</p></td><td char="."><p xmlns="" class="SimplePara">0.667</p></td><td char="."><p xmlns="" class="SimplePara">0.594</p></td><td char="."><p xmlns="" class="SimplePara">1.293</p></td><td char="."><p xmlns="" class="SimplePara">1.521</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR136</p></td><td char="."><p xmlns="" class="SimplePara">1.119</p></td><td char="."><p xmlns="" class="SimplePara">1.334</p></td><td char="."><p xmlns="" class="SimplePara">0.106</p></td><td char="."><p xmlns="" class="SimplePara">0.251</p></td><td char="."><p xmlns="" class="SimplePara">0.012</p></td><td char="."><p xmlns="" class="SimplePara">0.031</p></td><td char="."><p xmlns="" class="SimplePara">0.25</p></td><td char="."><p xmlns="" class="SimplePara">0.496</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">mean</p></td><td char="."><p xmlns="" class="SimplePara">2.044</p></td><td char="."><p xmlns="" class="SimplePara">2.684</p></td><td char="."><p xmlns="" class="SimplePara">0.429</p></td><td char="."><p xmlns="" class="SimplePara">0.522</p></td><td char="."><p xmlns="" class="SimplePara">0.34</p></td><td char="."><p xmlns="" class="SimplePara">0.344</p></td><td char="."><p xmlns="" class="SimplePara">0.856</p></td><td char="."><p xmlns="" class="SimplePara">1.093</p></td></tr></tbody></table></div></figure>
                           <figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO1">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0038-0/MediaObjects/41065_2017_38_Fig1_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0038-0/MediaObjects/41065_2017_38_Fig1_HTML.gif" alt="Fig. 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0038-0/MediaObjects/41065_2017_38_Fig1_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0038-0/MediaObjects/41065_2017_38_Fig1_HTML.gif" alt="Fig. 1"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 1</span>
                        <p xmlns="" class="SimplePara">UPGMA dendrogram of core collection (32 accessions) and entire collection (81 accessions) of <em class="EmphasisTypeItalic">F. velutipes</em> constructed based on genetic similarity coefficients of SSR data in the core collection and entire collection, respectively. Strains connected with dashed lines show placement of core collection strains in entire collection</p>
                      </div></figcaption></figure>
                        </div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par16" class="Para">We further investigated the genetic diversity parameters (Ae, H, Ho, I) of the cultivars and wild strains in core collection, which demonstrated that the wild strains possess greater genetic diversity than the cultivars (Table <span class="InternalRef"><a href="#Tab3">3</a></span>).<figure xmlns="" id="Tab3" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 3</span>
                        <p xmlns="" class="SimplePara">Genetic diversity parameters in cultivars and wild strains of core collection</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col char="."></col><col char="."></col><col char="."></col><col char="."></col><col char="."></col><col char="."></col><col char="."></col><col char="."></col></colgroup><thead><tr class="header"><th rowspan="2"><p xmlns="" class="SimplePara">Locus name</p></th><th colspan="2" char="."><p xmlns="" class="SimplePara">Ae</p></th><th colspan="2" char="."><p xmlns="" class="SimplePara">H</p></th><th colspan="2" char="."><p xmlns="" class="SimplePara">Ho</p></th><th colspan="2" char="."><p xmlns="" class="SimplePara">I</p></th></tr><tr class="header"><th char="."><p xmlns="" class="SimplePara">Cultivar</p></th><th char="."><p xmlns="" class="SimplePara">Wild</p></th><th char="."><p xmlns="" class="SimplePara">Cultivar</p></th><th char="."><p xmlns="" class="SimplePara">Wild</p></th><th char="."><p xmlns="" class="SimplePara">Cultivar</p></th><th char="."><p xmlns="" class="SimplePara">Wild</p></th><th char="."><p xmlns="" class="SimplePara">Cultivar</p></th><th char="."><p xmlns="" class="SimplePara">Wild</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR1</p></td><td char="."><p xmlns="" class="SimplePara">1.925</p></td><td char="."><p xmlns="" class="SimplePara">4.225</p></td><td char="."><p xmlns="" class="SimplePara">0.481</p></td><td char="."><p xmlns="" class="SimplePara">0.763</p></td><td char="."><p xmlns="" class="SimplePara">0.421</p></td><td char="."><p xmlns="" class="SimplePara">0.615</p></td><td char="."><p xmlns="" class="SimplePara">0.884</p></td><td char="."><p xmlns="" class="SimplePara">1.554</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR2</p></td><td char="."><p xmlns="" class="SimplePara">2.704</p></td><td char="."><p xmlns="" class="SimplePara">4.173</p></td><td char="."><p xmlns="" class="SimplePara">0.63</p></td><td char="."><p xmlns="" class="SimplePara">0.76</p></td><td char="."><p xmlns="" class="SimplePara">0.684</p></td><td char="."><p xmlns="" class="SimplePara">0.769</p></td><td char="."><p xmlns="" class="SimplePara">1.156</p></td><td char="."><p xmlns="" class="SimplePara">1.724</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR4</p></td><td char="."><p xmlns="" class="SimplePara">1.113</p></td><td char="."><p xmlns="" class="SimplePara">1.17</p></td><td char="."><p xmlns="" class="SimplePara">0.101</p></td><td char="."><p xmlns="" class="SimplePara">0.145</p></td><td char="."><p xmlns="" class="SimplePara">0.053</p></td><td char="."><p xmlns="" class="SimplePara">0.154</p></td><td char="."><p xmlns="" class="SimplePara">0.243</p></td><td char="."><p xmlns="" class="SimplePara">0.325</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR5</p></td><td char="."><p xmlns="" class="SimplePara">1.054</p></td><td char="."><p xmlns="" class="SimplePara">1.08</p></td><td char="."><p xmlns="" class="SimplePara">0.051</p></td><td char="."><p xmlns="" class="SimplePara">0.074</p></td><td char="."><p xmlns="" class="SimplePara">0.053</p></td><td char="."><p xmlns="" class="SimplePara">0.077</p></td><td char="."><p xmlns="" class="SimplePara">0.122</p></td><td char="."><p xmlns="" class="SimplePara">0.163</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR7</p></td><td char="."><p xmlns="" class="SimplePara">2.854</p></td><td char="."><p xmlns="" class="SimplePara">4.173</p></td><td char="."><p xmlns="" class="SimplePara">0.65</p></td><td char="."><p xmlns="" class="SimplePara">0.76</p></td><td char="."><p xmlns="" class="SimplePara">0.684</p></td><td char="."><p xmlns="" class="SimplePara">0.539</p></td><td char="."><p xmlns="" class="SimplePara">1.169</p></td><td char="."><p xmlns="" class="SimplePara">1.512</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR15</p></td><td char="."><p xmlns="" class="SimplePara">1.915</p></td><td char="."><p xmlns="" class="SimplePara">1.08</p></td><td char="."><p xmlns="" class="SimplePara">0.478</p></td><td char="."><p xmlns="" class="SimplePara">0.074</p></td><td char="."><p xmlns="" class="SimplePara">0.368</p></td><td char="."><p xmlns="" class="SimplePara">0.077</p></td><td char="."><p xmlns="" class="SimplePara">0.671</p></td><td char="."><p xmlns="" class="SimplePara">0.163</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR19</p></td><td char="."><p xmlns="" class="SimplePara">1.761</p></td><td char="."><p xmlns="" class="SimplePara">2.414</p></td><td char="."><p xmlns="" class="SimplePara">0.432</p></td><td char="."><p xmlns="" class="SimplePara">0.586</p></td><td char="."><p xmlns="" class="SimplePara">0.632</p></td><td char="."><p xmlns="" class="SimplePara">0.539</p></td><td char="."><p xmlns="" class="SimplePara">0.624</p></td><td char="."><p xmlns="" class="SimplePara">1.077</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR21</p></td><td char="."><p xmlns="" class="SimplePara">4.349</p></td><td char="."><p xmlns="" class="SimplePara">4.568</p></td><td char="."><p xmlns="" class="SimplePara">0.77</p></td><td char="."><p xmlns="" class="SimplePara">0.781</p></td><td char="."><p xmlns="" class="SimplePara">0.474</p></td><td char="."><p xmlns="" class="SimplePara">0.385</p></td><td char="."><p xmlns="" class="SimplePara">1.68</p></td><td char="."><p xmlns="" class="SimplePara">1.668</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR22</p></td><td char="."><p xmlns="" class="SimplePara">3.39</p></td><td char="."><p xmlns="" class="SimplePara">5.541</p></td><td char="."><p xmlns="" class="SimplePara">0.705</p></td><td char="."><p xmlns="" class="SimplePara">0.82</p></td><td char="."><p xmlns="" class="SimplePara">0.316</p></td><td char="."><p xmlns="" class="SimplePara">0.308</p></td><td char="."><p xmlns="" class="SimplePara">1.367</p></td><td char="."><p xmlns="" class="SimplePara">1.938</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR23</p></td><td char="."><p xmlns="" class="SimplePara">1.447</p></td><td char="."><p xmlns="" class="SimplePara">1.49</p></td><td char="."><p xmlns="" class="SimplePara">0.309</p></td><td char="."><p xmlns="" class="SimplePara">0.328</p></td><td char="."><p xmlns="" class="SimplePara">0.368</p></td><td char="."><p xmlns="" class="SimplePara">0.077</p></td><td char="."><p xmlns="" class="SimplePara">0.553</p></td><td char="."><p xmlns="" class="SimplePara">0.619</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR24</p></td><td char="."><p xmlns="" class="SimplePara">1.296</p></td><td char="."><p xmlns="" class="SimplePara">2.561</p></td><td char="."><p xmlns="" class="SimplePara">0.229</p></td><td char="."><p xmlns="" class="SimplePara">0.61</p></td><td char="."><p xmlns="" class="SimplePara">0.053</p></td><td char="."><p xmlns="" class="SimplePara">0.462</p></td><td char="."><p xmlns="" class="SimplePara">0.389</p></td><td char="."><p xmlns="" class="SimplePara">1.197</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR25</p></td><td char="."><p xmlns="" class="SimplePara">1.113</p></td><td char="."><p xmlns="" class="SimplePara">2.661</p></td><td char="."><p xmlns="" class="SimplePara">0.101</p></td><td char="."><p xmlns="" class="SimplePara">0.624</p></td><td char="."><p xmlns="" class="SimplePara">0.105</p></td><td char="."><p xmlns="" class="SimplePara">0.154</p></td><td char="."><p xmlns="" class="SimplePara">0.243</p></td><td char="."><p xmlns="" class="SimplePara">1.323</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR26</p></td><td char="."><p xmlns="" class="SimplePara">1.238</p></td><td char="."><p xmlns="" class="SimplePara">1.266</p></td><td char="."><p xmlns="" class="SimplePara">0.193</p></td><td char="."><p xmlns="" class="SimplePara">0.21</p></td><td char="."><p xmlns="" class="SimplePara">0.211</p></td><td char="."><p xmlns="" class="SimplePara">0.231</p></td><td char="."><p xmlns="" class="SimplePara">0.396</p></td><td char="."><p xmlns="" class="SimplePara">0.431</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR32</p></td><td char="."><p xmlns="" class="SimplePara">2.935</p></td><td char="."><p xmlns="" class="SimplePara">9.389</p></td><td char="."><p xmlns="" class="SimplePara">0.659</p></td><td char="."><p xmlns="" class="SimplePara">0.894</p></td><td char="."><p xmlns="" class="SimplePara">0.211</p></td><td char="."><p xmlns="" class="SimplePara">0.231</p></td><td char="."><p xmlns="" class="SimplePara">1.313</p></td><td char="."><p xmlns="" class="SimplePara">2.322</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR45</p></td><td char="."><p xmlns="" class="SimplePara">2.208</p></td><td char="."><p xmlns="" class="SimplePara">4.447</p></td><td char="."><p xmlns="" class="SimplePara">0.547</p></td><td char="."><p xmlns="" class="SimplePara">0.775</p></td><td char="."><p xmlns="" class="SimplePara">0.263</p></td><td char="."><p xmlns="" class="SimplePara">0.385</p></td><td char="."><p xmlns="" class="SimplePara">0.922</p></td><td char="."><p xmlns="" class="SimplePara">1.612</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR87</p></td><td char="."><p xmlns="" class="SimplePara">2.057</p></td><td char="."><p xmlns="" class="SimplePara">4.225</p></td><td char="."><p xmlns="" class="SimplePara">0.514</p></td><td char="."><p xmlns="" class="SimplePara">0.763</p></td><td char="."><p xmlns="" class="SimplePara">0.368</p></td><td char="."><p xmlns="" class="SimplePara">0.462</p></td><td char="."><p xmlns="" class="SimplePara">0.826</p></td><td char="."><p xmlns="" class="SimplePara">1.693</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR65</p></td><td char="."><p xmlns="" class="SimplePara">1.054</p></td><td char="."><p xmlns="" class="SimplePara">3.25</p></td><td char="."><p xmlns="" class="SimplePara">0.051</p></td><td char="."><p xmlns="" class="SimplePara">0.692</p></td><td char="."><p xmlns="" class="SimplePara">0.053</p></td><td char="."><p xmlns="" class="SimplePara">0.692</p></td><td char="."><p xmlns="" class="SimplePara">0.122</p></td><td char="."><p xmlns="" class="SimplePara">1.434</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR124</p></td><td char="."><p xmlns="" class="SimplePara">2.13</p></td><td char="."><p xmlns="" class="SimplePara">2.397</p></td><td char="."><p xmlns="" class="SimplePara">0.531</p></td><td char="."><p xmlns="" class="SimplePara">0.583</p></td><td char="."><p xmlns="" class="SimplePara">0.526</p></td><td char="."><p xmlns="" class="SimplePara">0.615</p></td><td char="."><p xmlns="" class="SimplePara">0.919</p></td><td char="."><p xmlns="" class="SimplePara">1.232</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR133</p></td><td char="."><p xmlns="" class="SimplePara">2.087</p></td><td char="."><p xmlns="" class="SimplePara">7.042</p></td><td char="."><p xmlns="" class="SimplePara">0.521</p></td><td char="."><p xmlns="" class="SimplePara">0.858</p></td><td char="."><p xmlns="" class="SimplePara">0.158</p></td><td char="."><p xmlns="" class="SimplePara">0.539</p></td><td char="."><p xmlns="" class="SimplePara">0.988</p></td><td char="."><p xmlns="" class="SimplePara">2.018</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR95</p></td><td char="."><p xmlns="" class="SimplePara">2.034</p></td><td char="."><p xmlns="" class="SimplePara">1.9</p></td><td char="."><p xmlns="" class="SimplePara">0.508</p></td><td char="."><p xmlns="" class="SimplePara">0.473</p></td><td char="."><p xmlns="" class="SimplePara">0.684</p></td><td char="."><p xmlns="" class="SimplePara">0.615</p></td><td char="."><p xmlns="" class="SimplePara">0.779</p></td><td char="."><p xmlns="" class="SimplePara">0.667</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR107</p></td><td char="."><p xmlns="" class="SimplePara">1.576</p></td><td char="."><p xmlns="" class="SimplePara">3.189</p></td><td char="."><p xmlns="" class="SimplePara">0.366</p></td><td char="."><p xmlns="" class="SimplePara">0.686</p></td><td char="."><p xmlns="" class="SimplePara">0.421</p></td><td char="."><p xmlns="" class="SimplePara">0.231</p></td><td char="."><p xmlns="" class="SimplePara">0.829</p></td><td char="."><p xmlns="" class="SimplePara">1.334</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR119</p></td><td char="."><p xmlns="" class="SimplePara">1.232</p></td><td char="."><p xmlns="" class="SimplePara">1.988</p></td><td char="."><p xmlns="" class="SimplePara">0.188</p></td><td char="."><p xmlns="" class="SimplePara">0.497</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">0.308</p></td><td char="."><p xmlns="" class="SimplePara">0.337</p></td><td char="."><p xmlns="" class="SimplePara">0.882</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR128</p></td><td char="."><p xmlns="" class="SimplePara">1.498</p></td><td char="."><p xmlns="" class="SimplePara">1.857</p></td><td char="."><p xmlns="" class="SimplePara">0.332</p></td><td char="."><p xmlns="" class="SimplePara">0.462</p></td><td char="."><p xmlns="" class="SimplePara">0.316</p></td><td char="."><p xmlns="" class="SimplePara">0.308</p></td><td char="."><p xmlns="" class="SimplePara">0.515</p></td><td char="."><p xmlns="" class="SimplePara">0.79</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR132</p></td><td char="."><p xmlns="" class="SimplePara">3.181</p></td><td char="."><p xmlns="" class="SimplePara">3.634</p></td><td char="."><p xmlns="" class="SimplePara">0.686</p></td><td char="."><p xmlns="" class="SimplePara">0.725</p></td><td char="."><p xmlns="" class="SimplePara">0.684</p></td><td char="."><p xmlns="" class="SimplePara">0.462</p></td><td char="."><p xmlns="" class="SimplePara">1.324</p></td><td char="."><p xmlns="" class="SimplePara">1.391</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SSR136</p></td><td char="."><p xmlns="" class="SimplePara">1.111</p></td><td char="."><p xmlns="" class="SimplePara">1.699</p></td><td char="."><p xmlns="" class="SimplePara">0.1</p></td><td char="."><p xmlns="" class="SimplePara">0.411</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">0.077</p></td><td char="."><p xmlns="" class="SimplePara">0.206</p></td><td char="."><p xmlns="" class="SimplePara">0.693</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">mean</p></td><td char="."><p xmlns="" class="SimplePara">1.97</p></td><td char="."><p xmlns="" class="SimplePara">3.257**</p></td><td char="."><p xmlns="" class="SimplePara">0.405</p></td><td char="."><p xmlns="" class="SimplePara">0.574*</p></td><td char="."><p xmlns="" class="SimplePara">0.324</p></td><td char="."><p xmlns="" class="SimplePara">0.372*</p></td><td char="."><p xmlns="" class="SimplePara">0.743</p></td><td char="."><p xmlns="" class="SimplePara">1.19**</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">*indicates significant difference in genetic diversity parameter at 0.05 level between the cultivars and wild strains</p>
                      <p xmlns="" class="SimplePara">**indicates significant difference in genetic diversity parameter at 0.01 level between the cultivars and wild strains</p>
                    </div></figure>
                        </div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par17" class="Para">Among the 153 alleles in the core collection, 72 (47%) were specific for each group and could differentiate the six groups from each other (Fig. <span class="InternalRef"><a href="#Fig2">2</a></span>). Group 6 had the highest number of specific alleles (30), followed by group 4 (13), group 5 (11), group 1 (8), group 2 (7), and group 3 (3) (Fig. <span class="InternalRef"><a href="#Fig2">2</a></span>). Nearly 53% of the alleles (81 of 153) were common to all the groups and can thus be categorized as conserved alleles (Fig. <span class="InternalRef"><a href="#Fig2">2</a></span>).<figure xmlns="" class="Figure" id="Fig2"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO2">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0038-0/MediaObjects/41065_2017_38_Fig2_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0038-0/MediaObjects/41065_2017_38_Fig2_HTML.gif" alt="Fig. 2" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0038-0/MediaObjects/41065_2017_38_Fig2_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0038-0/MediaObjects/41065_2017_38_Fig2_HTML.gif" alt="Fig. 2"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 2</span>
                        <p xmlns="" class="SimplePara">Venn diagram demonstrates common and specific alleles distributed in 6 groups</p>
                      </div></figcaption></figure>
                        </div>
              </section><section id="Sec10" class="Section2 RenderAsSection2">
                <h3 class="Heading">Genetic structure of the core collection</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par18" class="Para">The admixture model-based clustering method was used in the STRUCTURE program to infer the genetic structure of the core collection. The optimum number of K was analyzed using delta K (ΔK). A strong peak of ΔK is six, which indicated that there were six groups in the core collection (Fig. <span class="InternalRef"><a href="#Fig3">3</a></span>). The cultivars were assigned to groups 1 to 3, and the wild strains were assigned to groups 4 to 6 (Fig. <span class="InternalRef"><a href="#Fig4">4</a></span>).<figure xmlns="" class="Figure" id="Fig3"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO3">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0038-0/MediaObjects/41065_2017_38_Fig3_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0038-0/MediaObjects/41065_2017_38_Fig3_HTML.gif" alt="Fig. 3" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0038-0/MediaObjects/41065_2017_38_Fig3_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0038-0/MediaObjects/41065_2017_38_Fig3_HTML.gif" alt="Fig. 3"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 3</span>
                        <p xmlns="" class="SimplePara">Estimation of number of populations for K ranging from 2 to 10 by ΔK values</p>
                      </div></figcaption></figure>
                           <figure xmlns="" class="Figure" id="Fig4"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO4">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0038-0/MediaObjects/41065_2017_38_Fig4_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0038-0/MediaObjects/41065_2017_38_Fig4_HTML.gif" alt="Fig. 4" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0038-0/MediaObjects/41065_2017_38_Fig4_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0038-0/MediaObjects/41065_2017_38_Fig4_HTML.gif" alt="Fig. 4"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 4</span>
                        <p xmlns="" class="SimplePara">Estimated genetic structure for K&nbsp;=&nbsp;6. Each group is labeled beneath the figure. Cultivars and wild strains are labeled above the figure. Each bar represents a single individual</p>
                      </div></figcaption></figure>
                        </div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par19" class="Para">A similar result was also shown in the dendrogram constructed with the UPGMA method (Fig. <span class="InternalRef"><a href="#Fig1">1</a></span>). For the cultivars, white strains were assigned to groups 2 and 3, and yellow strains were distributed throughout groups 1 to 3. The wild strains were clustered in groups 4 to 6, and each group was stringently in accordance with its geographic origins. The strains in group 4 were collected from Hunan Province, with the exception of strain F77, which was purchased from Spawn Company in Jilin Province in northeastern China. This strain shares similar alleles with the strains collected from Hunan Province, indicating that it may have been isolated from Hunan Province. The strains in groups 5 and 6 were collected from Sichuan and Yunnan Provinces, respectively.</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec11" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Discussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Discussion">Discussion</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Discussion" data-test="collapse-section"><section id="Sec12" class="Section2 RenderAsSection2">
                <h3 class="Heading">Representation of the core collection</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par20" class="Para">The successful formation of a core collection depends on maximum allelic representation efficiency and elimination of redundancy from the entire collection [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>]. In this study, we successfully developed a core collection of <em xmlns="" class="EmphasisTypeItalic">F. velutipes</em> with 100% allelic representation under 39.5% of the sampling proportion (28.4% for cultivars and 92.9% for wild strains) based on 25 SSR markers. The genetic diversity parameters of the core collection could represent of the entire collection. And the strains selected in the core collection can represent the different allele components of each group in the entire collection (Fig. <span class="InternalRef"><a href="#Fig1">1</a></span>). Our results proved that the advanced M strategy is powerful in capturing 100% allelic diversity in a core collection [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>, <span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>]. The differences of the allele frequency between the entire collection and the core collection may be due to the redundant alleles including some homozygous loci that were excluded during the core collection construction.</p>
              </section><section id="Sec13" class="Section2 RenderAsSection2">
                <h3 class="Heading">Further strain improvement of <em class="EmphasisTypeItalic">F. velutipes</em> based on the core collection</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par21" class="Para">Most crops inevitably undergo a drastic loss of genetic diversity during cultivation, and <em xmlns="" class="EmphasisTypeItalic">F. velutipes</em> is no exception [<span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>, <span class="CitationRef"><a href="#CR36" data-toggle="citation">36</a></span>–<span class="CitationRef"><a href="#CR38" data-toggle="citation">38</a></span>]. Lower genetic diversity among cultivars may lead to inbreeding depression [<span class="CitationRef"><a href="#CR39" data-toggle="citation">39</a></span>]. Thus, the core collection established in this study could effectively protect the cultivars’ heterogeneous germplasm resources and help avoid inbreeding depression for further strain improvement. Furthermore, the specific alleles harbored in different groups of cultivars may indicate different agronomic characters in each group. For example, in our previous experiment, the mycelium growth rate at 23&nbsp;°C of a yellow strain F1 (6.38&nbsp;±&nbsp;0.1&nbsp;mm/d) in group 1 was significantly greater than that of the industrialized white strain F3 (6.35&nbsp;±&nbsp;0.07&nbsp;mm/d) (<em xmlns="" class="EmphasisTypeItalic">p</em>&nbsp;&lt;&nbsp;0.01) (unpublished data). Therefore, F1 could be used to crossbreed with the industrialized white strains to facilitate mycelium growth and shorten the production time.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par22" class="Para">The wild strains harbor higher genetic diversity than the cultivars in the core collection. Meanwhile, several economically important agronomic traits, such as tolerance to high temperature and rich contents of sesquiterpenes, can also be found in wild strains [<span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>, <span class="CitationRef"><a href="#CR40" data-toggle="citation">40</a></span>]. In the cultivation of <em xmlns="" class="EmphasisTypeItalic">F. velutipes</em>, temperature is usually an important limiting factor. The cultivars’ fruiting temperature needed for stringent control is less than 15&nbsp;°C, which will result in high energy costs [<span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>]. However, we have gathered several wild strains of <em xmlns="" class="EmphasisTypeItalic">F. velutipes</em> from subtropical regions of China in summer, despite most of the wild strains of this species mainly forming fruiting bodies in winter. Thus, those wild strains are ideal samples to domesticate for tolerance to high temperatures. In fact, we did find a wild strain (F98), collected from Longling, Yunnan, that can grow more vigorously than cultivars at a higher temperature (18&nbsp;°C) [<span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>]. Further analyses on chemical components showed that this strain contains 15 new sesquiterpenes with various skeletons, some of which showed moderate antidiabetes and antitumor bioactivity [<span class="CitationRef"><a href="#CR40" data-toggle="citation">40</a></span>]. Thus, it is quite essential to keep as many wild strains as possible in constructing a core collection for <em xmlns="" class="EmphasisTypeItalic">F. velutipes</em>.</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec14" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Conclusions" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Conclusions">Conclusions</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Conclusions" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par23" class="Para">In conclusion, we have established the first core collection of <em xmlns="" class="EmphasisTypeItalic">F. velutipes</em> in China, which is an important platform for efficient breeding of this mushroom in the future. The core collection is representative of the entire collection. In addition, the wild strains in the core collection possess favorable agronomic characters and produce unique bioactive compounds, adding value to the platform. More attention should be paid to wild strains in further strain breeding.</p></div></section>
          <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments"><h3 xmlns="" class="Heading">Acknowledgements</h3><p xmlns="" class="SimplePara">The authors are grateful to Dr. Min Zhang (Liaoning Academy of Agricultural Sciences), Dr. Yong-Chang Zhao (Yunnan Academy of Agricultural Sciences), Dr. Jin-Xia Zhang (Chinese Academy of Agricultural Sciences), Dr. Bao-Gui Xie (Fujian Agriculture and Forestry University), Dr. Ping Zhang (Hunan Normal University) and Mr. Pan-Meng Wang (Kunming Institute of Botany, Chinese Academy of Sciences) for providing strains for this study. Prof. Dr. Won-Sik Kong (Mushroom Research Division, National Institute of Horticultural and Herbal Science, Rural Development Administration, Eumsung, Republic of Korea) is acknowledged for generously sharing the genome sequencing data of the strain KACC42780 for us. We also thank Dr. Bang Feng and Ms. Yang-Yang Cui (Kunming Institute of Botany, Chinese Academy of Sciences) for improving the manuscript. The anonymous reviewers are also gratefully acknowledged for their comments and suggestions. Dr. Heather Hallen-Adams (Food Science and Technology, University of Nebraska-Lincoln) is acknowledged for having polished the English.</p><div id="FPar1" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Funding</h3>
                <p id="Par24" class="Para">This study was supported by the National Basic Research Program of China (973 Program, No. 2014CB138305).</p>
              </div><div id="FPar2" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Availability of data and materials</h3>
                <p id="Par25" class="Para">The datasets used and analyzed during the current study available from the corresponding author on reasonable request.</p>
              </div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="FormalPara FormalParaRenderingStyle1"><h3 xmlns="" class="Heading">Authors’ contributions</h3><p xmlns="" class="SimplePara">ZLY and XBL conceived and designed the experiments. XBL and JL collected the strains. XBL generated the DNA sequences and analyzed the data. All authors wrote the paper.</p></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Ethics">
              <div id="FPar3" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Ethics approval and consent to participate</h3>
                <p id="Par26" class="Para">“Not applicable”.</p>
              </div>
              <div id="FPar4" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Consent for publication</h3>
                <p id="Par27" class="Para">“Not applicable”.</p>
              </div>
              <div id="FPar5" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Competing interests</h3>
                <p id="Par28" class="Para">The authors declare that they have no competing interests.</p>
              </div>
              <div id="FPar6" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Publisher’s Note</h3>
                <p id="Par29" class="Para">Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
              </div>
            </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="License">
                <p xmlns="" class="SimplePara">
                           <strong class="EmphasisTypeBold">Open Access</strong>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/4.0/"><span class="RefSource">http://creativecommons.org/licenses/by/4.0/</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/publicdomain/zero/1.0/"><span class="RefSource">http://creativecommons.org/publicdomain/zero/1.0/</span></a></span>) applies to the data made available in this article, unless otherwise stated.</p>
              </div><aside class="Appendix" id="App1">
              
                <h3 class="Heading">Additional files</h3><div id="widgetContainer" data-test="figshare-container"></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par30" class="Para">
                  <div class="MediaObject" id="MOESM1">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0038-0/MediaObjects/41065_2017_38_MOESM1_ESM.doc"><span class="caption-number">Additional file 1:</span></a>
                          <span xmlns="" class="SimplePara">Strains used in this study. (DOC 82&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM2">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0038-0/MediaObjects/41065_2017_38_MOESM2_ESM.xls"><span class="caption-number">Additional file 2:</span></a>
                          <span xmlns="" class="SimplePara">Allele frequence distributions between entire collection and core collection. (XLS 46&nbsp;kb)</span>
                        </div></div>
                  </div>
                </div>
              
            </aside></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authors’ Affiliations</a></h2><div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1"><span class="AffiliationNumber">(1)</span><div class="AffiliationText">Key Laboratory for Plant Diversity and Biogeography of East Asia, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, Yunnan, 650201, China</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff2"><span class="AffiliationNumber">(2)</span><div class="AffiliationText">University of Chinese Academy of Sciences, Beijing, 100049, China</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff3"><span class="AffiliationNumber">(3)</span><div class="AffiliationText">State Key Laboratory of Conservation and Utilization for Bioresources in Yunnan, Yunnan University, Kunming, Yunnan, 650091, China</div></div></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper"><li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Frankel OH, AHD B. Current plant genetic resources-a critical appraisal. In genetics: new frontiers, vol. 4. New Delhi: Oxford &amp; IBH Publishing Co; 1984. p. 1–11.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Current%20plant%20genetic%20resources-a%20critical%20appraisal.%20In%20genetics%3A%20new%20frontiers&amp;author=OH.%20Frankel&amp;author=B.%20AHD&amp;publication_year=1984"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Brown AHD. Core collections: a practical approach to genetic resources management. Genome. 1989;31:818–24.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1139/g89-144"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Core%20collections%3A%20a%20practical%20approach%20to%20genetic%20resources%20management&amp;author=AHD.%20Brown&amp;journal=Genome&amp;volume=31&amp;pages=818-824&amp;publication_year=1989"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li Y, Shi Y, Cao Y, Wang T. Establishment of a core collection for maize germplasm preserved in Chinese National Genebank using geographic distribution and characterization data. Genet Resour Crop Evol. 2005;51:845–52.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s10722-005-8313-8"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Establishment%20of%20a%20core%20collection%20for%20maize%20germplasm%20preserved%20in%20Chinese%20National%20Genebank%20using%20geographic%20distribution%20and%20characterization%20data&amp;author=Y.%20Li&amp;author=Y.%20Shi&amp;author=Y.%20Cao&amp;author=T.%20Wang&amp;journal=Genet%20Resour%20Crop%20Evol&amp;volume=51&amp;pages=845-852&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hao CY, Zhang XY, Wang LF, Dong XW, Shang XW, Jia JZ. Genetic diversity and core collection evaluations in common wheat germplasm from the northwestern spring wheat region in China. Mol Breeding. 2006;17:69–77.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11032-005-2453-6"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20and%20core%20collection%20evaluations%20in%20common%20wheat%20germplasm%20from%20the%20northwestern%20spring%20wheat%20region%20in%20China&amp;author=CY.%20Hao&amp;author=XY.%20Zhang&amp;author=LF.%20Wang&amp;author=XW.%20Dong&amp;author=XW.%20Shang&amp;author=JZ.%20Jia&amp;journal=Mol%20Breeding&amp;volume=17&amp;pages=69-77&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Balfourier F, Roussel V, Strelchenko P, Exbrayat-Vinson F, Sourdille P, Boutet G, et al. A worldwide bread wheat core collection arrayed in a 384-well plate. Theor Appl Genet. 2007;114:1265–75.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00122-007-0517-1"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17318494"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20worldwide%20bread%20wheat%20core%20collection%20arrayed%20in%20a%20384-well%20plate&amp;author=F.%20Balfourier&amp;author=V.%20Roussel&amp;author=P.%20Strelchenko&amp;author=F.%20Exbrayat-Vinson&amp;author=P.%20Sourdille&amp;author=G.%20Boutet&amp;journal=Theor%20Appl%20Genet&amp;volume=114&amp;pages=1265-1275&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Oliveira MF, Nelson RL, Geraldi IO, Cruz CD, de Toledo JFF. Establishing a soybean germplasm core collection. Field Crops Res. 2010;119:277–89.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.fcr.2010.07.021"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Establishing%20a%20soybean%20germplasm%20core%20collection&amp;author=MF.%20Oliveira&amp;author=RL.%20Nelson&amp;author=IO.%20Geraldi&amp;author=CD.%20Cruz&amp;author=JFF.%20Toledo&amp;journal=Field%20Crops%20Res&amp;volume=119&amp;pages=277-289&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhao W, Cho GT, Ma KH, Chung JW, Gwag JG, Park YJ. Development of an allele-mining set in rice using a heuristic algorithm and SSR genotype data with least redundancy for the post-genomic era. Mol breeding. 2010;26:639–51.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11032-010-9400-x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Development%20of%20an%20allele-mining%20set%20in%20rice%20using%20a%20heuristic%20algorithm%20and%20SSR%20genotype%20data%20with%20least%20redundancy%20for%20the%20post-genomic%20era&amp;author=W.%20Zhao&amp;author=GT.%20Cho&amp;author=KH.%20Ma&amp;author=JW.%20Chung&amp;author=JG.%20Gwag&amp;author=YJ.%20Park&amp;journal=Mol%20breeding&amp;volume=26&amp;pages=639-651&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Holbrook CC, Anderson WF, Pittman RN. Selection of a core collection from the US germplasm collection of peanut. Crop Sci. 1993;33:859–61.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2135/cropsci1993.0011183X003300040044x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Selection%20of%20a%20core%20collection%20from%20the%20US%20germplasm%20collection%20of%20peanut&amp;author=CC.%20Holbrook&amp;author=WF.%20Anderson&amp;author=RN.%20Pittman&amp;journal=Crop%20Sci&amp;volume=33&amp;pages=859-861&amp;publication_year=1993"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hintum TJ, Bothmer R, Visser DL. Sampling strategies for composing a core collection of cultivated barley (<em class="EmphasisTypeItalic">Hordeum vulgare</em> s. Lat.) collected in China. Hereditas. 1995;122:7–17.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1601-5223.1995.00007.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Sampling%20strategies%20for%20composing%20a%20core%20collection%20of%20cultivated%20barley%20%28Hordeum%20vulgare%20s.%20Lat.%29%20collected%20in%20China&amp;author=TJ.%20Hintum&amp;author=R.%20Bothmer&amp;author=DL.%20Visser&amp;journal=Hereditas&amp;volume=122&amp;pages=7-17&amp;publication_year=1995"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ortiz R, Ruiz-Tapia EN, Mujica-Sanchez A. Sampling strategy for a core collection of Peruvian quinoa germplasm. Theor Appl Genet. 1998;96:475–83.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Ortiz%20R%2C%20Ruiz-Tapia%20EN%2C%20Mujica-Sanchez%20A.%20Sampling%20strategy%20for%20a%20core%20collection%20of%20Peruvian%20quinoa%20germplasm.%20Theor%20Appl%20Genet.%201998%3B96%3A475%E2%80%9383."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hu J, Zhu J, Xu HM. Methods of constructing core collections by stepwise clustering with three sampling strategies based on the genotypic values of crops. Theor Appl Genet. 2000;101:264–8.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s001220051478"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Methods%20of%20constructing%20core%20collections%20by%20stepwise%20clustering%20with%20three%20sampling%20strategies%20based%20on%20the%20genotypic%20values%20of%20crops&amp;author=J.%20Hu&amp;author=J.%20Zhu&amp;author=HM.%20Xu&amp;journal=Theor%20Appl%20Genet&amp;volume=101&amp;pages=264-268&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Xiao Y, Liu W, Dai Y, Fu C, Bian Y. Using SSR markers to evaluate the genetic diversity of <em class="EmphasisTypeItalic">Lentinula edodes</em>’ natural germplasm in China. World J Microbiol Biotechnol. 2010;26:527–36.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11274-009-0202-4"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Using%20SSR%20markers%20to%20evaluate%20the%20genetic%20diversity%20of%20Lentinula%20edodes%E2%80%99%20natural%20germplasm%20in%20China&amp;author=Y.%20Xiao&amp;author=W.%20Liu&amp;author=Y.%20Dai&amp;author=C.%20Fu&amp;author=Y.%20Bian&amp;journal=World%20J%20Microbiol%20Biotechnol&amp;volume=26&amp;pages=527-536&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhang RY, Hu DD, Gu JG, Hu QX, Zuo XM, Wang HX. Development of SSR markers for typing cultivars in the mushroom <em class="EmphasisTypeItalic">Auricularia auricula-judae</em>. Mycol Prog. 2012;11:587–92.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11557-011-0798-2"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Development%20of%20SSR%20markers%20for%20typing%20cultivars%20in%20the%20mushroom%20Auricularia%20auricula-judae%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=RY.%20Zhang&amp;author=DD.%20Hu&amp;author=JG.%20Gu&amp;author=QX.%20Hu&amp;author=XM.%20Zuo&amp;author=HX.%20Wang&amp;journal=Mycol%20Prog&amp;volume=11&amp;pages=587-592&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zane L, Bargelloni L, Patarnello T. Strategies for microsatellite isolation: a review. Mol Ecol. 2002;11:1–16.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Zane%20L%2C%20Bargelloni%20L%2C%20Patarnello%20T.%20Strategies%20for%20microsatellite%20isolation%3A%20a%20review.%20Mol%20Ecol.%202002%3B11%3A1%E2%80%9316."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Freeland JR. Molecular Ecology. In: Molecular markers in ecology. Thomson Press: Chichester; 2005. p. 43–57.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Molecular%20Ecology&amp;author=JR.%20Freeland&amp;pages=43-57&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Selkoe KA, Toonen RJ. Microsatellites for ecologists: a practical guide to using and evaluating microsatellite markers. Ecol Lett. 2006;9:615–29.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1461-0248.2006.00889.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16643306"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Microsatellites%20for%20ecologists%3A%20a%20practical%20guide%20to%20using%20and%20evaluating%20microsatellite%20markers&amp;author=KA.%20Selkoe&amp;author=RJ.%20Toonen&amp;journal=Ecol%20Lett&amp;volume=9&amp;pages=615-629&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kim KW, Chung HK, Cho GT, Ma KH, Chandrabalan D, Gwag JG, et al. PowerCore: a program applying the advanced M strategy with a heuristic search for establishing core sets. Bioinformatics. 2007;23:2155–62.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btm313"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17586551"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=PowerCore%3A%20a%20program%20applying%20the%20advanced%20M%20strategy%20with%20a%20heuristic%20search%20for%20establishing%20core%20sets&amp;author=KW.%20Kim&amp;author=HK.%20Chung&amp;author=GT.%20Cho&amp;author=KH.%20Ma&amp;author=D.%20Chandrabalan&amp;author=JG.%20Gwag&amp;journal=Bioinformatics&amp;volume=23&amp;pages=2155-2162&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhang Y, Zhang X, Hua W, Wang L, Zhuo C. Analysis of genetic diversity among indigenous landraces from sesame (<em class="EmphasisTypeItalic">Sesamum indicum</em> L.) core collection in China as revealed by SRAP and SSR markers. Genes Genom. 2010;32:207–15.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s13258-009-0888-6"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Analysis%20of%20genetic%20diversity%20among%20indigenous%20landraces%20from%20sesame%20%28Sesamum%20indicum%20L.%29%20core%20collection%20in%20China%20as%20revealed%20by%20SRAP%20and%20SSR%20markers&amp;author=Y.%20Zhang&amp;author=X.%20Zhang&amp;author=W.%20Hua&amp;author=L.%20Wang&amp;author=C.%20Zhuo&amp;journal=Genes%20Genom&amp;volume=32&amp;pages=207-215&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Belaj A, del Dominguez-García MDC, Atienza SG, Urdíroz NM, Rosa RD, Satovic Z, et al. Developing a core collection of olive (<em class="EmphasisTypeItalic">Olea europaea</em> L.) based on molecular markers (DArTs, SSRs, SNPs) and agronomic traits. Tree Genet Genomes. 2012;8:365–78.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11295-011-0447-6"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Developing%20a%20core%20collection%20of%20olive%20%28Olea%20europaea%20L.%29%20based%20on%20molecular%20markers%20%28DArTs%2C%20SSRs%2C%20SNPs%29%20and%20agronomic%20traits&amp;author=A.%20Belaj&amp;author=MDC.%20Dominguez-Garc%C3%ADa&amp;author=SG.%20Atienza&amp;author=NM.%20Urd%C3%ADroz&amp;author=RD.%20Rosa&amp;author=Z.%20Satovic&amp;journal=Tree%20Genet%20Genomes&amp;volume=8&amp;pages=365-378&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kaga A, Shimizu T, Watanabe S, Tsubokura Y, Katayose Y, Harada K, et al. Evaluation of soybean germplasm conserved in NIAS genebank and development of mini core collections. Breed Sci. 2012;61:566–92.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1270/jsbbs.61.566"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23136496"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3406788"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Evaluation%20of%20soybean%20germplasm%20conserved%20in%20NIAS%20genebank%20and%20development%20of%20mini%20core%20collections&amp;author=A.%20Kaga&amp;author=T.%20Shimizu&amp;author=S.%20Watanabe&amp;author=Y.%20Tsubokura&amp;author=Y.%20Katayose&amp;author=K.%20Harada&amp;journal=Breed%20Sci&amp;volume=61&amp;pages=566-592&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Moe KT, Gwag JG, Park YJ. Efficiency of PowerCore in core set development using amplified fragment length polymorphic markers in mungbean. Plant Breed. 2012;131:110–7.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1439-0523.2011.01896.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Efficiency%20of%20PowerCore%20in%20core%20set%20development%20using%20amplified%20fragment%20length%20polymorphic%20markers%20in%20mungbean&amp;author=KT.%20Moe&amp;author=JG.%20Gwag&amp;author=YJ.%20Park&amp;journal=Plant%20Breed&amp;volume=131&amp;pages=110-117&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Tiwari KK, Singh A, Pattnaik S, Sandhu M, Kaur S, Jain S, et al. Identification of a diverse mini-core panel of Indian rice germplasm based on genotyping using microsatellite markers. Plant Breed. 2015;134:164–71.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/pbr.12252"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Identification%20of%20a%20diverse%20mini-core%20panel%20of%20Indian%20rice%20germplasm%20based%20on%20genotyping%20using%20microsatellite%20markers&amp;author=KK.%20Tiwari&amp;author=A.%20Singh&amp;author=S.%20Pattnaik&amp;author=M.%20Sandhu&amp;author=S.%20Kaur&amp;author=S.%20Jain&amp;journal=Plant%20Breed&amp;volume=134&amp;pages=164-171&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li H, Chen Q, Huang CY, Xie BG, Zhang A. A method for establishing core collection of <em class="EmphasisTypeItalic">Pleurotus ostreatus</em> cultivated in China based on SSR markers. Acta Hortic Sin. 2012;39:2023–32.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20method%20for%20establishing%20core%20collection%20of%20Pleurotus%20ostreatus%20cultivated%20in%20China%20based%20on%20SSR%20markers&amp;author=H.%20Li&amp;author=Q.%20Chen&amp;author=CY.%20Huang&amp;author=BG.%20Xie&amp;author=A.%20Zhang&amp;journal=Acta%20Hortic%20Sin&amp;volume=39&amp;pages=2023-2032&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR24" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhang XM, Yao Q, Gong ZY, Gao XX, Li J, Liu X, et al. Development of EST-SSR markers in <em class="EmphasisTypeItalic">Pleurotus ostreatus</em> and construction of primary core collection. Mycosystema. 2013;32:64–80.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Development%20of%20EST-SSR%20markers%20in%20Pleurotus%20ostreatus%20and%20construction%20of%20primary%20core%20collection&amp;author=XM.%20Zhang&amp;author=Q.%20Yao&amp;author=ZY.%20Gong&amp;author=XX.%20Gao&amp;author=J.%20Li&amp;author=X.%20Liu&amp;journal=Mycosystema&amp;volume=32&amp;pages=64-80&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR25" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Liu J, Wang Z-R, Li C, Bian Y-B, Xiao Y. Evaluating genetic diversity and constructing core collections of Chinese <em class="EmphasisTypeItalic">Lentinula edodes</em> cultivars using ISSR and SRAP markers. J Basic Microbiol. 2015;55:749–60.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1002/jobm.201400774"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25589225"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Evaluating%20genetic%20diversity%20and%20constructing%20core%20collections%20of%20Chinese%20Lentinula%20edodes%20cultivars%20using%20ISSR%20and%20SRAP%20markers&amp;author=J.%20Liu&amp;author=Z-R.%20Wang&amp;author=C.%20Li&amp;author=Y-B.%20Bian&amp;author=Y.%20Xiao&amp;journal=J%20Basic%20Microbiol&amp;volume=55&amp;pages=749-760&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR26" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Mizuno R, Ichinose H, Honda M, Takabatake K, Sotome I, Takai T. Use of whole crop sorghums as a raw material in consolidated bioprocessing bioethanol production using <em class="EmphasisTypeItalic">Flammulina velutipes</em>. Biosci Biotechnol Biochem. 2009;73:1671–3.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1271/bbb.90099"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19584528"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Use%20of%20whole%20crop%20sorghums%20as%20a%20raw%20material%20in%20consolidated%20bioprocessing%20bioethanol%20production%20using%20Flammulina%20velutipes%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=R.%20Mizuno&amp;author=H.%20Ichinose&amp;author=M.%20Honda&amp;author=K.%20Takabatake&amp;author=I.%20Sotome&amp;author=T.%20Takai&amp;journal=Biosci%20Biotechnol%20Biochem&amp;volume=73&amp;pages=1671-1673&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR27" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Van Peer AF, Park SY, Shin PG, Jang KY, Yoo YB, Park YJ, et al. Comparative genomics of the mating-type loci of the mushroom <em class="EmphasisTypeItalic">Flammulina velutipes</em> reveals widespread synteny and recent inversions. PLoS One. 2011;6:e22249.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0022249"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21799803"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3140503"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparative%20genomics%20of%20the%20mating-type%20loci%20of%20the%20mushroom%20Flammulina%20velutipes%20reveals%20widespread%20synteny%20and%20recent%20inversions&amp;author=AF.%20Peer&amp;author=SY.%20Park&amp;author=PG.%20Shin&amp;author=KY.%20Jang&amp;author=YB.%20Yoo&amp;author=YJ.%20Park&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e22249&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR28" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Shi M, Yang Y, Guan D, Zhang Y, Zhang Z. Bioactivity of the crude polysaccharides from fermented soybean curd residue by <em class="EmphasisTypeItalic">Flammulina velutipes</em>. Carbohydr Polym. 2012;89:1268–76.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.carbpol.2012.04.047"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24750941"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Bioactivity%20of%20the%20crude%20polysaccharides%20from%20fermented%20soybean%20curd%20residue%20by%20Flammulina%20velutipes%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=M.%20Shi&amp;author=Y.%20Yang&amp;author=D.%20Guan&amp;author=Y.%20Zhang&amp;author=Z.%20Zhang&amp;journal=Carbohydr%20Polym&amp;volume=89&amp;pages=1268-1276&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR29" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li X, Li Y. Quality comparison and analysis on white <em class="EmphasisTypeItalic">Flammulina velutipes</em> grown with bottle lines in China. Edible Fungi China. 2014;33:20–4.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Quality%20comparison%20and%20analysis%20on%20white%20Flammulina%20velutipes%20grown%20with%20bottle%20lines%20in%20China&amp;author=X.%20Li&amp;author=Y.%20Li&amp;journal=Edible%20Fungi%20China&amp;volume=33&amp;pages=20-24&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR30" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Liu XB, Feng B, Li J, Yan C, Yang ZL. Genetic diversity and breeding history of winter mushroom (<em class="EmphasisTypeItalic">Flammulina velutipes</em>) in China uncovered by genomic SSR markers. Gene. 2016;591:227–35.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.gene.2016.07.009"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27393649"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20and%20breeding%20history%20of%20winter%20mushroom%20%28Flammulina%20velutipes%29%20in%20China%20uncovered%20by%20genomic%20SSR%20markers&amp;author=XB.%20Liu&amp;author=B.%20Feng&amp;author=J.%20Li&amp;author=C.%20Yan&amp;author=ZL.%20Yang&amp;journal=Gene&amp;volume=591&amp;pages=227-235&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR31" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Doyle JJ, Doyle JL. A rapid DNA isolation procedure for small quantities of fresh leaf tissue. Phytochem Bull. 1987;19:11–5.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20rapid%20DNA%20isolation%20procedure%20for%20small%20quantities%20of%20fresh%20leaf%20tissue&amp;author=JJ.%20Doyle&amp;author=JL.%20Doyle&amp;journal=Phytochem%20Bull&amp;volume=19&amp;pages=11-15&amp;publication_year=1987"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR32" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yeh FC, Yang RC, Boyle T. POPGENE, version 1.31. Microsoft window-based freeware for population genetic analysis. Alberta: University of Alberta; 1999. p. 1–28.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Yeh%20FC%2C%20Yang%20RC%2C%20Boyle%20T.%20POPGENE%2C%20version%201.31.%20Microsoft%20window-based%20freeware%20for%20population%20genetic%20analysis.%20Alberta%3A%20University%20of%20Alberta%3B%201999.%20p.%201%E2%80%9328."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR33" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Rohlf FJ. Phylogenetic models and reticulations. J Classif. 2000;17:185–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s003570000017"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Phylogenetic%20models%20and%20reticulations&amp;author=FJ.%20Rohlf&amp;journal=J%20Classif&amp;volume=17&amp;pages=185-189&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR34" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Evanno G, Regnaut S. Goudet J. Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study. Mol Ecol. 2005;14:2611–20.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Evanno%20G%2C%20Regnaut%20S.%20Goudet%20J.%20Detecting%20the%20number%20of%20clusters%20of%20individuals%20using%20the%20software%20STRUCTURE%3A%20a%20simulation%20study.%20Mol%20Ecol.%202005%3B14%3A2611%E2%80%9320."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR35" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Earl DA. vonHoldt BM. STRUCTURE HARVESTER: a website and program for visualizing STRUCTURE output and implementing the Evanno method. Conserv Genet Resour. 2012;4:359–61.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s12686-011-9548-7"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=vonHoldt%20BM.%20STRUCTURE%20HARVESTER%3A%20a%20website%20and%20program%20for%20visualizing%20STRUCTURE%20output%20and%20implementing%20the%20Evanno%20method&amp;author=DA.%20Earl&amp;journal=Conserv%20Genet%20Resour&amp;volume=4&amp;pages=359-361&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR36" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Reif JC, Hamrit S, Heckenberger M, Schipprack W, Maurer HP, Bohn M, et al. Trends in genetic diversity among European maize cultivars and their parental components during the past 50 years. Theor Appl Genet. 2005;111:838–45.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00122-005-0004-5"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16034585"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Trends%20in%20genetic%20diversity%20among%20European%20maize%20cultivars%20and%20their%20parental%20components%20during%20the%20past%2050%20years&amp;author=JC.%20Reif&amp;author=S.%20Hamrit&amp;author=M.%20Heckenberger&amp;author=W.%20Schipprack&amp;author=HP.%20Maurer&amp;author=M.%20Bohn&amp;journal=Theor%20Appl%20Genet&amp;volume=111&amp;pages=838-845&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR37" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Doebley JF, Gau BS, Smith BD. The molecular genetics of crop domestication. Cell. 2006;127:1309–21.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.cell.2006.12.006"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17190597"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20molecular%20genetics%20of%20crop%20domestication&amp;author=JF.%20Doebley&amp;author=BS.%20Gau&amp;author=BD.%20Smith&amp;journal=Cell&amp;volume=127&amp;pages=1309-1321&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR38" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hyten DL, Song Q, Zhu Y, Choi IY, Nelson RL, Costa JM, et al. Impacts of genetic bottlenecks on soybean genome diversity. Proc Natl Acad Sci U S A. 2006;103:16666–71.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.0604379103"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17068128"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1624862"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Impacts%20of%20genetic%20bottlenecks%20on%20soybean%20genome%20diversity&amp;author=DL.%20Hyten&amp;author=Q.%20Song&amp;author=Y.%20Zhu&amp;author=IY.%20Choi&amp;author=RL.%20Nelson&amp;author=JM.%20Costa&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;volume=103&amp;pages=16666-16671&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR39" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Xu J. Analysis of inbreeding depression in <em class="EmphasisTypeItalic">Agaricus bisporus</em>. Genet Soc Am. 1995;141:137–45.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Analysis%20of%20inbreeding%20depression%20in%20Agaricus%20bisporus%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=J.%20Xu&amp;journal=Genet%20Soc%20Am&amp;volume=141&amp;pages=137-145&amp;publication_year=1995"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR40" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Tao Q, Ma K, Yang Y, Wang K, Chen B, Huang Y, et al. Bioactive Sesquiterpenes from the edible mushroom <em class="EmphasisTypeItalic">Flammulina velutipes</em> and their biosynthetic pathway confirmed by genome analysis and chemical evidence. J Org Chem. 2016;81:9867–77.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1021/acs.joc.6b01971"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27684789"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Bioactive%20Sesquiterpenes%20from%20the%20edible%20mushroom%20Flammulina%20velutipes%20and%20their%20biosynthetic%20pathway%20confirmed%20by%20genome%20analysis%20and%20chemical%20evidence&amp;author=Q.%20Tao&amp;author=K.%20Ma&amp;author=Y.%20Yang&amp;author=K.%20Wang&amp;author=B.%20Chen&amp;author=Y.%20Huang&amp;journal=J%20Org%20Chem&amp;volume=81&amp;pages=9867-9877&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li></ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#TheAuthors" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="TheAuthors">Copyright</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="TheAuthors"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">©&nbsp;The Author(s)&nbsp;2017</div></div></section></div>
                        


                    </article>
                </main>

                <div class="c-content-layout__side u-text-sm">
                    <aside>
                        <div class="p-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="//hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-017-0038-0" data-track="click" data-track-category="PDF Download"
            data-track-action="Click download article PDF"
            data-track-label="10.1186/s41065-017-0038-0">Download PDF</a>
        </p>
    
                            
                            
    <button type="button"
       class="c-button c-button--secondary c-button--full-width c-button--expandable"
       data-toggle="collapse"
       data-target="export-citations"
       data-test="export-citations-button"
       data-sticky-update="true"
       aria-controls="export-citations"
       aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/s41065-017-0038-0"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0038-0?format=refman&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/s41065-017-0038-0"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0038-0?format=refman&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/s41065-017-0038-0"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0038-0?format=refman&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/s41065-017-0038-0"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0038-0?format=endnote&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/s41065-017-0038-0"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0038-0?format=endnote&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/s41065-017-0038-0"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0038-0?format=endnote&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/s41065-017-0038-0"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0038-0?format=bibtex&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/s41065-017-0038-0"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0038-0?format=bibtex&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/s41065-017-0038-0"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0038-0?format=bibtex&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
    </div>

                        </div>
                        
    
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
                                <div id="article-access-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/access-count/10.1186/s41065-017-0038-0 #accessCount-container"></div>
                                <div id="citation-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/citation-count/10.1186/s41065-017-0038-0 #count-container"></div>
                                <div id="altmetric-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/altmetric/10.1186/s41065-017-0038-0 #altmetric-container"></div>
                            </ul>
                        </div>
                        
    <div id="sociaMediaShare" class="p-separator u-js-shown">
        <h3 class="p-section-title">Share this article</h3>
        <ul id="shareButtons" class="c-social-links">
            <li class="c-social-links__item">
                <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-017-0038-0">
                    <span class="u-visually-hidden">Share on Twitter</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-twitter"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-017-0038-0">
                    <span class="u-visually-hidden">Share on Facebook</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-facebook"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click"
                        data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-017-0038-0">
                    <span class="u-visually-hidden">Share on LinkedIn</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-linkedin"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-017-0038-0">
                    <span class="u-visually-hidden">Share on Weibo</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-weibo"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnGooglePlus" class="js-btnShareOnGplus" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Google+" data-track-label="10.1186/s41065-017-0038-0">
                    <span class="u-visually-hidden">Share on Google Plus</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-gplus"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-017-0038-0">
                    <span class="u-visually-hidden">Share on Reddit</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-reddit"></use>
                    </svg>
                </button>
            </li>
        </ul>
    </div>
 
                        
    
        <div class="p-separator u-js-shown">
            <h3 class="p-section-title">See updates</h3>
            <button class="c-crossmark-button" data-target="crossmark" data-test="crossmark" data-track="click" data-track-category="Article Page" data-track-action="Click CrossMark" data-track-label="10.1186/s41065-017-0038-0">
                <img id="crossmark-icon" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" alt="Crossmark logo" />
            </button>
        </div>
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
                                <li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/s41065-017-0038-0" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=738&doi=10.1186%2Fs41065-017-0038-0&volume=155&issue=1&title=Genetic+diversity+and+structure+of+core+collection+of+winter+mushroom+%28+Flammulina+velutipes+%29+developed+by+genomic+SSR+markers&author_name=Xiao+Bin+Liu&start_page=1&end_page=8">Order reprint</a>

                                </li>
                            </ul>
                        </div>
                    </aside>
                    
    <div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">Hereditas</span>
                        </h4>
                        <p class="c-journal-footer__issn">ISSN: 1601-5223</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
                                
                                    <li class="c-journal-footer__contact-item">Submission enquiries: <a href="https://www.editorialmanager.com/here/" target="_blank">Access here and click Contact Us</a></li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a></li>
                                
                            </ul>
                        </div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class='tracker' style='display:none' src='/track/article/10.1186/s41065-017-0038-0' alt=""/>

         
    <footer>
        <div class="ctx-publisher-footer">
    <div class="c-navbar">
        <div class="u-container">
            <div class="c-navbar__logo u-margin-right-xl">
                <a href="https://www.biomedcentral.com">
                    <img src=/static/images/bmc/logos/logo-bmc-aj-594e03e20d.svg alt="BMC logo" data-test="navbar-logo-footer">

                </a>
            </div>
            <div class="c-navbar__content">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


            </div>
        </div>
    </div>
    <div class="c-publisher-footer" data-test="publisher-footer">
        <div class="u-container">
            
            <div class="c-flex c-flex--wrap c-flex--space-between" role="navigation" data-test="publisher-footer-menu">
                <div class="c-flex" aria-label="Publisher footer links">
                    
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/login">Receive BMC newsletters</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/account">Manage article alerts</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/policies">Policies</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/accessibility">Accessibility</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/press-centre">Press center</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/contact-us">Contact us</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/jobs">Careers</a>
                                    </li>
                                
                            </ul>
                        
                    
                </div>
                <div class="u-margin-bottom-lg">
                    <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                    <ul class="c-flex u-reset-list" data-test="footer-social-links">
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://twitter.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Twitter link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Twitter page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-twitter-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://www.facebook.com/BioMedCentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Facebook link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Facebook page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-facebook-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://plus.google.com/106226337984152901734/"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Google Plus link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Google Plus page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-gplus-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="http://www.weibo.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Weibo link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Weibo page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-weibo-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </div>
            <p class="u-reset-margin">
                By using this website, you agree to our
                <a class="u-text-inherit" href="//www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
                <a class="u-text-inherit" href="//www.biomedcentral.com/privacy-statement">Privacy
                    statement</a> and
                <a class="u-text-inherit" href="//www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
                
                    <a class="optanon-toggle-display u-text-inherit" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
                
            </p>
        </div>
    </div>
</div>


        <div class="c-sn-footer c-sn-footer--neutral">
    <div class="u-container">
        <img src=/static/images/bmc/logos/logo-springernature-8bd4e8d82f.svg class="c-sn-footer__logo" alt="Springer Nature Logo" itemprop="logo" >
        <p data-test="copyright">&#169; 2018 BioMed Central Ltd unless otherwise stated. Part of
            <a href="http://www.springernature.com"
               data-track="click"
               data-track-action="Click Springer Nature link"
               data-track-category="Corporate footer"
               itemscope itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>

    </div>

    
    <script type="text/javascript" src="//cdn.pbgrd.com/bmc.js" async="async"></script>
    <noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>
    <!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker -->

<script id="springerlink-event-tracker" src="https://event-tracker.springernature.com/dist/eventTracker.js"></script>
<script type="text/javascript">
    (function(){
        var dataLayer = window.dataLayer[0];
        var doi = dataLayer.content.article.doi;
        var imprint = dataLayer.content.contentInfo.imprint;
        var bpids = dataLayer.session.authentication.authenticationID;
        var publisher;
        var pageType = dataLayer.page.category.pageType;
        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }
        var eventObject = {'content_type' : pageType,'doi' : doi};

        if (imprint === 'BioMed Central') { publisher = 'BMC' }
        else if (imprint === 'SpringerOpen') { publisher = 'SpringerOpen' }
        else { throw new Error('Event Tracker Error: The publisher has an unexpected value') }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }
        new EventTracker({'platform':publisher}).sendEvent('display', eventObject);
    })();
</script>


    </body>
</html>

