<!DOCTYPE html>
<html lang="en" class="no-js">
    <head>
        <title>Analysis of differentially expressed genes and adaptive mechanisms of Prunus triloba Lindl. under alkaline stress | Hereditas | Full Text</title>

        

    <meta name="citation_abstract" content="
                           Prunus triloba Lindl. is a naturally salt-alkaline-tolerant plant with several unique characteristics, and it can be used as the rootstock of Chinese plum (Prunus salicina Lindl.) in saline-alkaline soils. To comprehensively investigate the alkaline acclimation mechanisms in P. triloba, a series of analyses were conducted under alkaline stress, including analyses of the kinetics of molecular and physiological changes, and leaf microstructure. To understand the kinetics of molecular changes under short-term alkaline stress, we used Illumina HiSeq 2500 platform to identify alkaline stress-related differentially expressed genes (DEGs) in P. triloba. Approximately 53.0 million high-quality clean reads were generated from 59.6 million raw reads, and a total of 124,786 unigenes were obtained after de novo assembly of P. triloba transcriptome data. After alkaline stress treatment, a total of 8948 unigenes were identified as DEGs. Based on these DEGs, a Gene Ontology (GO) enrichment analysis was conducted, suggesting that 28 genes may play an important role in the early alkaline stress response. In addition, analysis of DEGs with the Kyoto Encyclopedia of Genes and Genomes (KEGG) revealed that pathways were significant at different treatment time points. A significant positive correlation was found between the quantitative real-time PCR (qRT-PCR) results and the RNA-Seq data for seven alkaline-related genes, confirming the reliability of the RNA-Seq results. Based on physiological analysis of P. triloba in response to long-term alkaline stress, we found that the internal microstructures of the leaves of P. triloba changed to adapt to long-term alkaline stress. Various physiological indexes indicated that the degree of membrane injury increased with increasing duration of alkaline stress, affecting photosynthesis in P. triloba seedlings. This represents the first investigation into the physiology and transcriptome of P. triloba in response to alkaline stress. The results of this study can enrich the genomic resources available for P. triloba, as well as deepening our understanding of molecular and physiological alkaline tolerance mechanisms in P. triloba. This will also provide new insights into our understanding of alkaline acclimation mechanisms in Chinese plum (Prunus salicina) trees."/>

    <meta name="description" content="
                           Prunus triloba Lindl. is a naturally salt-alkaline-tolerant plant with several unique characteristics, and it can be used as the rootstock of Chinese plum (Prunus salicina Lindl.) in saline-alkaline soils. To comprehensively investigate the alkaline acclimation mechanisms in P. triloba, a series of analyses were conducted under alkaline stress, including analyses of the kinetics of molecular and physiological changes, and leaf microstructure. To understand the kinetics of molecular changes under short-term alkaline stress, we used Illumina HiSeq 2500 platform to identify alkaline stress-related differentially expressed genes (DEGs) in P. triloba. Approximately 53.0 million high-quality clean reads were generated from 59.6 million raw reads, and a total of 124,786 unigenes were obtained after de novo assembly of P. triloba transcriptome data. After alkaline stress treatment, a total of 8948 unigenes were identified as DEGs. Based on these DEGs, a Gene Ontology (GO) enrichment analysis was conducted, suggesting that 28 genes may play an important role in the early alkaline stress response. In addition, analysis of DEGs with the Kyoto Encyclopedia of Genes and Genomes (KEGG) revealed that pathways were significant at different treatment time points. A significant positive correlation was found between the quantitative real-time PCR (qRT-PCR) results and the RNA-Seq data for seven alkaline-related genes, confirming the reliability of the RNA-Seq results. Based on physiological analysis of P. triloba in response to long-term alkaline stress, we found that the internal microstructures of the leaves of P. triloba changed to adapt to long-term alkaline stress. Various physiological indexes indicated that the degree of membrane injury increased with increasing duration of alkaline stress, affecting photosynthesis in P. triloba seedlings. This represents the first investigation into the physiology and transcriptome of P. triloba in response to alkaline stress. The results of this study can enrich the genomic resources available for P. triloba, as well as deepening our understanding of molecular and physiological alkaline tolerance mechanisms in P. triloba. This will also provide new insights into our understanding of alkaline acclimation mechanisms in Chinese plum (Prunus salicina) trees."/>

    <meta name="journal_id" content="41065"/>

    <meta name="dc.title" content="Analysis of differentially expressed genes and adaptive mechanisms of Prunus triloba Lindl. under alkaline stress"/>

    <meta name="dc.source" content="Hereditas 2017 154:1"/>

    <meta name="dc.format" content="text/html"/>

    <meta name="dc.publisher" content="BioMed Central"/>

    <meta name="dc.date" content="2017-05-04"/>

    <meta name="dc.type" content="OriginalPaper"/>

    <meta name="dc.language" content="En"/>

    <meta name="dc.copyright" content="2017 The Author(s)"/>

    <meta name="dc.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="dc.description" content="
                           Prunus triloba Lindl. is a naturally salt-alkaline-tolerant plant with several unique characteristics, and it can be used as the rootstock of Chinese plum (Prunus salicina Lindl.) in saline-alkaline soils. To comprehensively investigate the alkaline acclimation mechanisms in P. triloba, a series of analyses were conducted under alkaline stress, including analyses of the kinetics of molecular and physiological changes, and leaf microstructure. To understand the kinetics of molecular changes under short-term alkaline stress, we used Illumina HiSeq 2500 platform to identify alkaline stress-related differentially expressed genes (DEGs) in P. triloba. Approximately 53.0 million high-quality clean reads were generated from 59.6 million raw reads, and a total of 124,786 unigenes were obtained after de novo assembly of P. triloba transcriptome data. After alkaline stress treatment, a total of 8948 unigenes were identified as DEGs. Based on these DEGs, a Gene Ontology (GO) enrichment analysis was conducted, suggesting that 28 genes may play an important role in the early alkaline stress response. In addition, analysis of DEGs with the Kyoto Encyclopedia of Genes and Genomes (KEGG) revealed that pathways were significant at different treatment time points. A significant positive correlation was found between the quantitative real-time PCR (qRT-PCR) results and the RNA-Seq data for seven alkaline-related genes, confirming the reliability of the RNA-Seq results. Based on physiological analysis of P. triloba in response to long-term alkaline stress, we found that the internal microstructures of the leaves of P. triloba changed to adapt to long-term alkaline stress. Various physiological indexes indicated that the degree of membrane injury increased with increasing duration of alkaline stress, affecting photosynthesis in P. triloba seedlings. This represents the first investigation into the physiology and transcriptome of P. triloba in response to alkaline stress. The results of this study can enrich the genomic resources available for P. triloba, as well as deepening our understanding of molecular and physiological alkaline tolerance mechanisms in P. triloba. This will also provide new insights into our understanding of alkaline acclimation mechanisms in Chinese plum (Prunus salicina) trees."/>

    <meta name="prism.issn" content="1601-5223"/>

    <meta name="prism.publicationName" content="Hereditas"/>

    <meta name="prism.publicationDate" content="2017-05-04"/>

    <meta name="prism.volume" content="154"/>

    <meta name="prism.number" content="1"/>

    <meta name="prism.section" content="OriginalPaper"/>

    <meta name="prism.startingPage" content="10"/>

    <meta name="prism.copyright" content="2017 The Author(s)"/>

    <meta name="prism.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="prism.url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0031-7"/>

    <meta name="prism.doi" content="doi:10.1186/s41065-017-0031-7"/>

    <meta name="citation_pdf_url" content="https://hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-017-0031-7"/>

    <meta name="citation_fulltext_html_url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0031-7"/>

    <meta name="citation_journal_title" content="Hereditas"/>

    <meta name="citation_publisher" content="BioMed Central"/>

    <meta name="citation_issn" content="1601-5223"/>

    <meta name="citation_title" content="Analysis of differentially expressed genes and adaptive mechanisms of Prunus triloba Lindl. under alkaline stress"/>

    <meta name="citation_volume" content="154"/>

    <meta name="citation_issue" content="1"/>

    <meta name="citation_publication_date" content="2017/12"/>

    <meta name="citation_online_date" content="2017/05/04"/>

    <meta name="citation_firstpage" content="10"/>

    <meta name="citation_article_type" content="Research"/>

    <meta name="citation_fulltext_world_readable" content=""/>

    <meta name="dc.identifier" content="doi:10.1186/s41065-017-0031-7"/>

    <meta name="DOI" content="10.1186/s41065-017-0031-7"/>

    <meta name="citation_doi" content="10.1186/s41065-017-0031-7"/>

    <meta name="dc.creator" content="Jia Liu"/>

    <meta name="dc.creator" content="Yongqing Wang"/>

    <meta name="dc.creator" content="Qingtian Li"/>

    <meta name="dc.subject" content="Animal Genetics and Genomics"/>

    <meta name="dc.subject" content="Evolutionary Biology"/>

    <meta name="citation_reference" content="citation_journal_title=Chin Hortic Abstr; citation_title=Study on the selection of superior rootstocks for plum tress; citation_author=JY Cai, Z Sun, WQ Li, ZL Huang; citation_volume=8; citation_publication_date=2013; citation_pages=38-40; citation_id=CR1"/>

    <meta name="citation_reference" content="citation_journal_title=Ann Rev Plant Physiol; citation_title=Iron chlorosis; citation_author=JC Brown; citation_volume=7; citation_publication_date=2003; citation_pages=171-90; citation_doi=10.1146/annurev.pp.07.060156.001131; citation_id=CR2"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Soil; citation_title=Iron availability in plant tissues-iron chlorosis on calcareous soils; citation_author=K Mengel; citation_volume=165; citation_publication_date=1994; citation_pages=275-83; citation_doi=10.1007/BF00008070; citation_id=CR3"/>

    <meta name="citation_reference" content="citation_journal_title=Rural Ecol Environ; citation_title=Land salinization in China and the prevention countermeasures; citation_author=ZR Yu, JW Wang; citation_volume=13; citation_publication_date=1997; citation_pages=1-5; citation_id=CR4"/>

    <meta name="citation_reference" content="citation_title=Comparative structure study on four pecies of Rosaceae plants living in different ecosystem in Master&#8217;s Thesis; citation_publication_date=2005; citation_id=CR5; citation_author=H Gao; citation_publisher=Northeast Normal University"/>

    <meta name="citation_reference" content="citation_journal_title=J Beijing For Univ; citation_title=&#8220;Dual classification&#8221;: a new classification system for Chinese flower cultivars; citation_author=JY Chen; citation_volume=20; citation_publication_date=1998; citation_pages=1-6; citation_id=CR6"/>

    <meta name="citation_reference" content="citation_journal_title=North Hortic; citation_title=Bibliometrics analysis of research progress in Prunus triloba nearly thirty years in China; citation_author=LL Zhao; citation_volume=20; citation_publication_date=2014; citation_pages=202-4; citation_id=CR7"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Plant Breed; citation_title=Genetic diversity of wild populations of Prunus triloba revealed by AFLP; citation_author=QY Zhang, QX Zhang; citation_volume=10; citation_publication_date=2012; citation_pages=452-6; citation_id=CR8"/>

    <meta name="citation_reference" content="citation_journal_title=J Xinjiang Agr Univ; citation_title=Effects of low temperature stress on some cold resistance indexes of four kinds of root stock Prunus
                           ; citation_author=XH Su, W Qin, LQ Liu, F Chen, L Zhou; citation_volume=35; citation_publication_date=2012; citation_pages=112-5; citation_id=CR9"/>

    <meta name="citation_reference" content="citation_journal_title=Chin Hortic Abstr; citation_title=Research of drought resistance of flowering plum seedings; citation_author=HJ Li, HD Zhao, SM Cui, BY Ding; citation_volume=8; citation_publication_date=2011; citation_pages=1-4; citation_id=CR10"/>

    <meta name="citation_reference" content="citation_journal_title=Gene; citation_title=Gene expression profiling of soybean leaves and roots under salt, saline&#8211;alkali and drought stress by high-throughput Illumina sequencing; citation_author=XD Fan, JQ Wang, N Yang, YY Dong, L Liu, FW Wang; citation_volume=512; citation_publication_date=2013; citation_pages=392-402; citation_doi=10.1016/j.gene.2012.09.100; citation_id=CR11"/>

    <meta name="citation_reference" content="citation_title=Agricultural salinity assessment and management; citation_publication_date=2011; citation_id=CR12; citation_author=WW Wallender; citation_author=KK Tanji; citation_publisher=American Society of Civil Engineers (ASCE) Publications"/>

    <meta name="citation_reference" content="citation_journal_title=Acta Bot Sin; citation_title=Difference between salt (NaCl) and alkaline (Na2CO3) stresses on Puccinellia tenuiflora (Griseb.) Scribn. et Merr. plants; citation_author=DC Shi, LJ Yin; citation_volume=35; citation_publication_date=1993; citation_pages=144-9; citation_id=CR13"/>

    <meta name="citation_reference" content="citation_journal_title=Environ Exp Bot; citation_title=Effect of various salt&#8211;alkaline mixed stress conditions on sunflower seedlings and analysis of their stress factors; citation_author=DC Shi, YM Sheng; citation_volume=54; citation_publication_date=2005; citation_pages=8-21; citation_doi=10.1016/j.envexpbot.2004.05.003; citation_id=CR14"/>

    <meta name="citation_reference" content="citation_journal_title=Annu Rev Plant Biol; citation_title=Mechanisms of salinity tolerance; citation_author=R Munns, M Tester; citation_volume=59; citation_publication_date=2008; citation_pages=651-81; citation_doi=10.1146/annurev.arplant.59.032607.092911; citation_id=CR15"/>

    <meta name="citation_reference" content="citation_journal_title=Gene; citation_title=Deciphering the regulatory mechanisms of abiotic stress tolerance in plants by genomic approaches; citation_author=N Sreenivasulu, SK Sopory, PBK Kishor; citation_volume=388; citation_publication_date=2007; citation_pages=1-13; citation_doi=10.1016/j.gene.2006.10.009; citation_id=CR16"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Plant Biol; citation_title=
                              De novo sequencing, assembly, and analysis of the Taxodium &#8216;Zhongshansa&#8217; roots and shoots transcriptome in response to short-term waterlogging; citation_author=B Qi, Y Yang, Y Yin, M Xu, H Li; citation_volume=14; citation_publication_date=2014; citation_pages=1-12; citation_doi=10.1186/1471-2229-14-1; citation_id=CR17"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Genomics; citation_title=
                              De novo assembly of red clover transcriptome based on RNA-Seq data provides insight into drought response, gene discovery and marker identification; citation_author=SA Yates, MT Swain, MJ Hegarty, I Chernukin, M Lowe, GG Allison; citation_volume=15; citation_publication_date=2014; citation_pages=453; citation_doi=10.1186/1471-2164-15-453; citation_id=CR18"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Sci; citation_title=Microarray and suppression subtractive hybridization analyses of gene expression in Puccinellia tenuiflora after exposure to NaHCO3
                           ; citation_author=YC Wang, CP Yang, GF Liu, GD Zhang, QY Ban; citation_volume=173; citation_publication_date=2007; citation_pages=309-20; citation_doi=10.1016/j.plantsci.2007.06.011; citation_id=CR19"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data; citation_author=AM Bolger, M Lohse, B Usadel; citation_volume=30; citation_publication_date=2014; citation_pages=2114-20; citation_doi=10.1093/bioinformatics/btu170; citation_id=CR20"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Biotechnol; citation_title=Trinity: reconstructing a full-length transcriptome without a genome from RNA-Seq data; citation_author=MG Grabherr, BJ Haas, M Yassour, JZ Levin, DA Thompson, I Amit; citation_volume=29; citation_publication_date=2011; citation_pages=644-52; citation_doi=10.1038/nbt.1883; citation_id=CR21"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=TIGR Gene Indices clustering tools (TGICL): a software system for fast clustering of large EST datasets; citation_author=G Pertea, X Huang, F Liang, V Antonescu, R Sultana, S Karamycheva; citation_volume=19; citation_publication_date=2003; citation_pages=651-2; citation_doi=10.1093/bioinformatics/btg034; citation_id=CR22"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Protoc; citation_title=
                              De novo transcript sequence reconstruction from RNA-Seq: reference generation and analysis with Trinity; citation_author=BJ Haas, A Papanicolaou, M Yassour, M Grabherr, PD Blood, J Bowden; citation_volume=8; citation_publication_date=2013; citation_pages=1494-512; citation_doi=10.1038/nprot.2013.084; citation_id=CR23"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Methods; citation_title=Fast gapped-read alignment with Bowtie 2; citation_author=B Langmead, SL Salzberg; citation_volume=9; citation_publication_date=2012; citation_pages=357-9; citation_doi=10.1038/nmeth.1923; citation_id=CR24"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics; citation_title=RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome; citation_author=B Li, CN Dewey; citation_volume=12; citation_publication_date=2011; citation_pages=93-9; citation_doi=10.1186/1471-2105-12-93; citation_id=CR25"/>

    <meta name="citation_reference" content="citation_journal_title=Genome Biol; citation_title=Differential expression analysis for sequence count data; citation_author=S Anders, W Huber; citation_volume=11; citation_publication_date=2010; citation_pages=1-12; citation_doi=10.1186/gb-2010-11-10-r106; citation_id=CR26"/>

    <meta name="citation_reference" content="citation_title=GOseq: gene ontology testing for RNA-seq datasets; citation_publication_date=2012; citation_id=CR27; citation_author=MD Young; citation_author=MJ Wakeeld; citation_author=GK Smyth; citation_author=A Oshlack"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=KOBAS 2.0: a web server for annotation and identification of enriched pathways and diseases; citation_author=C Xie, XZ Mao, JJ Huang, Y Ding, JM Wu, S Dong; citation_volume=39; citation_publication_date=2011; citation_pages=W316-W22; citation_doi=10.1093/nar/gkr483; citation_id=CR28"/>

    <meta name="citation_reference" content="citation_title=Research technology of plant physiology. Master&#8217;s Thesis; citation_publication_date=2006; citation_id=CR29; citation_author=Q Sun; citation_author=JJ Hu; citation_publisher=Northwest Agriculture and Forestry University Press"/>

    <meta name="citation_reference" content="citation_title=Plant physiology laboratory manual; citation_publication_date=1985; citation_id=CR30; citation_author=YL Xue; citation_author=ZA Xia; citation_publisher=Shanghai Science and Technique Publishing House"/>

    <meta name="citation_reference" content="citation_title=Plant physiology experiment; citation_publication_date=1990; citation_id=CR31; citation_author=GL Zhu; citation_author=HW Zhong; citation_author=AQ Zhang; citation_publisher=Peking University Press"/>

    <meta name="citation_reference" content="citation_title=Principles and techniques of plant physiological and biochemical experiments; citation_publication_date=2006; citation_id=CR32; citation_author=HS Li; citation_publisher=Higher Education Press"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Plant Biol; citation_title=Global transcriptome profiling of wild soybean (Glycine soja) roots under NaHCO3 treatment; citation_author=Y Ge, Y Li, YM Zhu, X Bai, DK Lv, DJ Guo; citation_volume=10; citation_publication_date=2010; citation_pages=1-14; citation_doi=10.1186/1471-2229-10-153; citation_id=CR33"/>

    <meta name="citation_reference" content="citation_journal_title=Plant J; citation_title=Monitoring the expression profiles of 7000 Arabidopsis genes under drought, cold and high-salinity stresses using a full-length cDNA microarray; citation_author=M Seki, M Narusaka, J Ishida, T Nanjo, M Fujita, Y Oono; citation_volume=31; citation_publication_date=2002; citation_pages=279-92; citation_doi=10.1046/j.1365-313X.2002.01359.x; citation_id=CR34"/>

    <meta name="citation_reference" content="citation_journal_title=Acta Agron Sin; citation_title=Construction of SSH library and its analyses of cotton drought associated genes under drought stress; citation_author=DL Wang, WW Ye, JJ Wang, LY Song, WL Fan, YP Cui; citation_volume=36; citation_publication_date=2010; citation_pages=2035-44; citation_doi=10.1016/S1875-2780(09)60087-0; citation_id=CR35"/>

    <meta name="citation_reference" content="citation_journal_title=Agr Eng; citation_title=Progress on &#945;-linolenic acid; citation_author=J Yang, R Chang; citation_volume=1; citation_publication_date=2011; citation_pages=72-6; citation_id=CR36"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Plant; citation_title=Metabolic and phenotypic responses of greenhouse-grown maize hybrids to experimentally controlled drought stress; citation_author=S Witt, L Galicia, J Lisec, J Cairns, A Tiessen, JL Araus; citation_volume=5; citation_publication_date=2012; citation_pages=401-17; citation_doi=10.1093/mp/ssr102; citation_id=CR37"/>

    <meta name="citation_reference" content="citation_journal_title=Front Plant Sci; citation_title=Leaf metabolite profile of the Brazilian resurrection plant Barbacenia purpurea Hook. (Velloziaceae) shows two time-dependent responses during desiccation and recovering; citation_author=VF Suguiyama, EA Silva, ST Meirelles, DC Centeno, MR Braga; citation_volume=5; citation_publication_date=2014; citation_pages=96; citation_doi=10.3389/fpls.2014.00096; citation_id=CR38"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Soil; citation_title=Metabolic response of maize (Zea mays L.) plants to combined drought and salt stress; citation_author=CX Sun, XX Gao, JQ Fu, JH Zhou, XF Wu; citation_volume=388; citation_publication_date=2015; citation_pages=99-117; citation_doi=10.1007/s11104-014-2309-0; citation_id=CR39"/>

    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=A defined range of guard cell calcium oscillation parameters encodes stomatal movements; citation_author=GJ Allen, SP Chu, CL Harrington, K Schumacher, T Hoffmann, YY Tang; citation_volume=411; citation_publication_date=2001; citation_pages=1053-7; citation_doi=10.1038/35082575; citation_id=CR40"/>

    <meta name="citation_reference" content="citation_journal_title=J Agr Sci Technol; citation_title=Plant ABC transporters and their roles in the transmembrane transport of secondary metabolites; citation_author=HB Jin, DH Liu, KJ Zuo, ZQ Miao, YH Chen, XF Sun; citation_volume=9; citation_publication_date=2007; citation_pages=32-7; citation_id=CR41"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Physiol; citation_title=SAGE analysis of transcriptome responses in Arabidopsis roots exposed to 2,4,6-trinitrotoluene; citation_author=DR Ekman, WW Lorenz, AE Przybyla, NL Wolfe, JF Dean; citation_volume=133; citation_publication_date=2003; citation_pages=1397-1406; citation_doi=10.1104/pp.103.028019; citation_id=CR42"/>

    <meta name="citation_reference" content="citation_journal_title=Sci Hortic-Amsterdam; citation_title=Deciphering the possible mechanism of exogenous NO alleviating alkali stress on cucumber leaves by transcriptomic analysis; citation_author=ZX Gao, TC Wehner, H Chen, Y Lin, XF Wang, M Wei; citation_volume=150; citation_publication_date=2013; citation_pages=377-86; citation_doi=10.1016/j.scienta.2012.11.033; citation_id=CR43"/>

    <meta name="citation_reference" content="Huang LK, Yan HD, Zhao XX, Zhang XQ, Wang J, Frazier T, et al. Identifying differentially expressed genes under heat stress and developing molecular markers in orchardgrass (Dactylis glomerata L.) through transcriptome analysis. Mol Ecol Resour. 2015;15:1497&#8211;509."/>

    <meta name="citation_reference" content="citation_journal_title=Oecologia; citation_title=Leaf structure and anatomy as related to leaf mass per area variation in seedlings of a wide range of woody plant species and types; citation_author=P Castr-Od&#237;ez, JP Puyravaud, JHC Cornelissen; citation_volume=124; citation_publication_date=2000; citation_pages=476-86; citation_doi=10.1007/PL00008873; citation_id=CR45"/>

    <meta name="citation_reference" content="citation_journal_title=Ecol Environ; citation_title=Analyse of leaves&#8217; structure character and ecological adaptation of ten plants in the west alkaline district of Songnen plain; citation_author=LH Chi, FB Song; citation_volume=15; citation_publication_date=2006; citation_pages=1269-73; citation_id=CR46"/>

    <meta name="citation_reference" content="citation_journal_title=J Neimenggu Agri Univ; citation_title=Nutrition apparatus observation of 9 kinds of sparsity be in sever danger plant of Alashan area; citation_author=L Yan, H Li, X He, XR Wang; citation_volume=21; citation_publication_date=2001; citation_pages=65-71; citation_id=CR47"/>

    <meta name="citation_reference" content="citation_journal_title=Agri Res in the Arid Areas; citation_title=Research on physiological characteristics and the assessment of tolerance of different varieties of sweet sorghum seedlings to saline-sodic stress; citation_author=LY Dai, LJ Zhang, R Yan-Ye, HU Kai; citation_volume=13; citation_publication_date=2012; citation_pages=193-204; citation_id=CR48"/>

    <meta name="citation_reference" content="citation_journal_title=Acta Pratac Sin; citation_title=Different physiological responses of Aneurolepidium chinense to NaCl and Na2CO3
                           ; citation_author=H Yan, W Zhao, SJ Yin, DC Shi, DW Zhou; citation_volume=15; citation_publication_date=2006; citation_pages=49-55; citation_id=CR49"/>

    <meta name="citation_reference" content="citation_journal_title=Acta Agron Sin; citation_title=Physiological response of xiaobingmai (Triticum aestivum-Agropyron intermedium) to salt-stress and alkali-stress; citation_author=CW Yang, CY Li, HJ Yin, M Ju, DC Shi; citation_volume=33; citation_publication_date=2007; citation_pages=1255-61; citation_id=CR50"/>

    <meta name="citation_reference" content="citation_journal_title=Photosynthetica; citation_title=Response of xanthophyll cycle and chloroplastic antioxidant enzymes to chilling stress in tomato over-expressing glycerol-3-phosphate acyltransferase gene; citation_author=N Sui, M Li, XY Liu, N Wang, W Fang, QW Meng; citation_volume=45; citation_publication_date=2007; citation_pages=447-54; citation_doi=10.1007/s11099-007-0074-5; citation_id=CR51"/>

    <meta name="citation_reference" content="citation_journal_title=Subtropical Agri Res; citation_title=Advances in study of salt-stress tolerance and its physiological and biochemical indices in plants; citation_author=XJ Xu, BC Shen, JM Qi; citation_volume=3; citation_publication_date=2007; citation_pages=275-80; citation_id=CR52"/>

    <meta name="citation_reference" content="citation_journal_title=FEBS Lett; citation_title=Evidence of differential pH regulation of the Arabidopsis vacuolar Ca2+/H+ antiporters CAX1 and CAX2; citation_author=JK Pittman, T Shigaki, KD Hirschi; citation_volume=579; citation_publication_date=2005; citation_pages=2648-56; citation_doi=10.1016/j.febslet.2005.03.085; citation_id=CR53"/>

    <meta name="citation_reference" content="citation_journal_title=Sci Silvae Sin; citation_title=Physiological responses of populus alba var. pyramidalis leaves with different maturity to alkali stress; citation_author=X Zhang, XL Xiao, PB Li, YF Sun, WY Guo, YN Yao; citation_volume=51; citation_publication_date=2015; citation_pages=9-16; citation_id=CR54"/>

    <meta name="citation_reference" content="citation_journal_title=Photosynth Res; citation_title=Photooxidative reactions in chloroplast thylakoids. Evidence for a Fenton-type reaction promoted by superoxide or ascorbate; citation_author=BL Upham, LS Jahnke; citation_volume=8; citation_publication_date=1986; citation_pages=235-47; citation_doi=10.1007/BF00037131; citation_id=CR55"/>

    <meta name="citation_reference" content="citation_journal_title=Acta Phytoecologica Sin; citation_title=Effect of calcium on alleviation of decreased photosynthetic ability in salt-stressed maize leaves; citation_author=NH Zhang, HY Gao, Q Zou; citation_volume=29; citation_publication_date=2005; citation_pages=324-30; citation_id=CR56"/>

    <meta name="citation_reference" content="citation_journal_title=Acta Bot Boreali-Occidentalia Sin; citation_title=A study on rules of spatiao-temporal change and water use efficiency of corn; citation_author=F Li, Z Huang, L Shan; citation_volume=20; citation_publication_date=1999; citation_pages=1010-5; citation_id=CR57"/>

    <meta name="citation_author" content="Jia Liu"/>

    <meta name="citation_author_institution" content="College of Horticulture, Sichuan Agricultural University, Chengdu, People&#8217;s Republic of China"/>

    <meta name="citation_author_institution" content="Horticulture Research Institute, Sichuan Academy of Agricultural Sciences, Chengdu, People&#8217;s Republic of China"/>

    <meta name="citation_author_institution" content="Southwestern Key laboratory of Horticultural Crops Biology and Germplasm Enhancement, Ministry of Agriculture, Chengdu, People&#8217;s Republic of China"/>

    <meta name="citation_author" content="Yongqing Wang"/>

    <meta name="citation_author_institution" content="College of Horticulture, Sichuan Agricultural University, Chengdu, People&#8217;s Republic of China"/>

    <meta name="citation_author" content="Qingtian Li"/>

    <meta name="citation_author_institution" content="State Key Lab of Plant Genomics, Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Beijing, People&#8217;s Republic of China"/>



        <meta charset="UTF-8"/>
        <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
        <meta name="viewport" content="width=device-width, initial-scale=1"/>

        <!-- For IE 9 and below -->
<!--[if IE]><link rel="shortcut icon" href=/static/images/favicons//favicon.ico><![endif]-->
<!-- Touch Icons - iOS and Android 2.1+ -->
<link rel="apple-touch-icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>
<!-- Firefox, Chrome, Safari, IE 11+ and Opera -->
<link rel="icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>


        <script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script>
        <script>(function(d){if (sessionStorage.fontsLoaded){d.style.setProperty('--font-family-sans', 'Europa');d.style.setProperty('--font-family-serif', 'Noto Serif')}})(document.documentElement);</script>

        <link rel="stylesheet" media="screen" href=/static/styles/basic/basic-a31ca85bc2.css>
<link rel="stylesheet" media="print" href=/static/styles/bmc/print-3a6240f456.css>

<link rel="stylesheet" class="js-ctm" href=/static/styles/bmc/main-84414d3d47.css
      media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">

        

        <script src="/static/js/app-bundle-8d61e6ef37.js" defer></script>
        
        <script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script>
        
    <script type="text/javascript">
        config = {
            env: 'live',
            site: 'hereditasjournal.biomedcentral.com',
            siteWithPath: 'hereditasjournal.biomedcentral.com' + window.location.pathname,
            twitterHashtag: '',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/s41065-017-0031-7',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central'
        };
    </script>

        
    <script type="text/javascript">
        dataLayer = [{"content":{"article":{"doi":"10.1186/s41065-017-0031-7","articleType":"Research","peerReviewType":"Closed","supplement":null,"keywords":"Alkaline stress;Differentially expressed genes;Prunus triloba Lindl;Physiological analysis;RNA sequencing"},"contentInfo":{"imprint":"BioMed Central","title":"Analysis of differentially expressed genes and adaptive mechanisms of Prunus triloba Lindl. under alkaline stress","publishedAt":1493856000000,"publishedAtDate":"2017-05-04","author":["Jia Liu","Yongqing Wang","Qingtian Li"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":"CC BY + CC0","openAccess":true},"environment":"live"},"journal":{"siteKey":"hereditasjournal.biomedcentral.com","volume":"154","issue":"1","title":"Hereditas","type":"AJs Life","journalID":41065,"gaCode":"UA-62066850-11","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Research","publishingSegment":"Genetics & Heredity (Life)"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}}}]

    </script>

        <script src="https://cdn.cookielaw.org/consent/7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script>
        
    
        
    

        
    
        
            
                <!-- Google Tag Manager -->
                <script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script>
                <!-- End Google Tag Manager -->
            
        
    


        
    
    <link rel="canonical" href="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0031-7"/>
    

        
        
        <meta property="og:url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0031-7"/>
        <meta property="og:type" content="article"/>
        <meta property="og:site_name" content="Hereditas"/>
        <meta property="og:title" content="Analysis of differentially expressed genes and adaptive mechanisms of Prunus triloba Lindl. under alkaline stress"/>
        <meta property="og:description" content="Prunus triloba Lindl. is a naturally salt-alkaline-tolerant plant with several unique characteristics, and it can be used as the rootstock of Chinese plum (Prunus salicina Lindl.) in saline-alkaline soils. To comprehensively investigate the alkaline acclimation mechanisms in P. triloba, a series of analyses were conducted under alkaline stress, including analyses of the kinetics of molecular and physiological changes, and leaf microstructure. To understand the kinetics of molecular changes under short-term alkaline stress, we used Illumina HiSeq 2500 platform to identify alkaline stress-related differentially expressed genes (DEGs) in P. triloba. Approximately 53.0 million high-quality clean reads were generated from 59.6 million raw reads, and a total of 124,786 unigenes were obtained after de novo assembly of P. triloba transcriptome data. After alkaline stress treatment, a total of 8948 unigenes were identified as DEGs. Based on these DEGs, a Gene Ontology (GO) enrichment analysis was conducted, suggesting that 28 genes may play an important role in the early alkaline stress response. In addition, analysis of DEGs with the Kyoto Encyclopedia of Genes and Genomes (KEGG) revealed that pathways were significant at different treatment time points. A significant positive correlation was found between the quantitative real-time PCR (qRT-PCR) results and the RNA-Seq data for seven alkaline-related genes, confirming the reliability of the RNA-Seq results. Based on physiological analysis of P. triloba in response to long-term alkaline stress, we found that the internal microstructures of the leaves of P. triloba changed to adapt to long-term alkaline stress. Various physiological indexes indicated that the degree of membrane injury increased with increasing duration of alkaline stress, affecting photosynthesis in P. triloba seedlings. This represents the first investigation into the physiology and transcriptome of P. triloba in response to alkaline stress. The results of this study can enrich the genomic resources available for P. triloba, as well as deepening our understanding of molecular and physiological alkaline tolerance mechanisms in P. triloba. This will also provide new insights into our understanding of alkaline acclimation mechanisms in Chinese plum (Prunus salicina) trees."/>
        <meta property="og:image" content="https://images.springer.com/sgw/journals/medium/41065.jpg"/>
    
    </head>

    <body class="journal journal-fulltext"
          
          >
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK"
                            height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><feColorMatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"/></filter></defs><symbol id="icon-arrow-down" viewBox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"/></symbol><symbol id="icon-arrow-left-bullet" viewBox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"/></symbol><symbol id="icon-arrow-right-bullet" viewBox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"/></symbol><symbol id="icon-arrow-right" viewBox="0 0 32 64"><path d="M0 0v64l32-32z"/></symbol><symbol id="icon-close" viewBox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"/></symbol><symbol id="icon-community" viewBox="0 0 60.643 31.01"><mask maskUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><path fill="#FFF" d="M0 31.01h60.643V0H0z" filter="url(#a)"/></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"/></symbol><symbol id="icon-download" viewBox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"/></symbol><symbol id="icon-ext-link" viewBox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"/><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"/><path fill="#676767" d="M9.7 0L16 6.2V0z"/></symbol><symbol id="icon-info-bordered" viewBox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"/><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"/></symbol><symbol id="icon-opr" viewBox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"/><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"/></symbol><symbol id="icon-quotation" viewBox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"/></symbol><symbol id="icon-remove" viewBox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"/><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"/></symbol><symbol id="icon-ribbon" viewBox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"/></symbol><symbol id="icon-rss" viewBox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"/><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"/><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"/></g></symbol><symbol id="icon-search" viewBox="0 0 16 16"><path d="M6.276 12.257C2.815 12.257 0 9.508 0 6.128 0 2.749 2.815 0 6.276 0c3.462 0 6.277 2.749 6.277 6.128.002 3.38-2.815 6.129-6.277 6.129zm0-10.772c-2.622 0-4.755 2.083-4.755 4.643 0 2.562 2.133 4.644 4.755 4.644 2.623 0 4.756-2.082 4.756-4.644 0-2.56-2.134-4.643-4.756-4.643z"/><path d="M16 13.751L13.641 16l-4.875-4.82 2.36-2.248L16 13.751z"/></symbol><symbol id="icon-speech-bubble" viewBox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"/></symbol><symbol id="icon-error" viewBox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"/></symbol><symbol id="icon-info" viewBox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"/></symbol><symbol id="icon-success" viewBox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"/></symbol><symbol id="icon-warning" viewBox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"/></symbol><symbol id="icon-facebook-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"/></symbol><symbol id="icon-facebook" viewBox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"/><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"/></symbol><symbol id="icon-gplus-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M480.5 273.117c.123.076.26.185.416.334.146.152.289.343.432.562.135.211.258.459.359.746.086.286.129.619.129 1.004-.012.7-.164 1.262-.465 1.683a5.374 5.374 0 0 1-.461.562c-.186.176-.375.35-.58.527a2.65 2.65 0 0 0-.322.41.908.908 0 0 0-.176.553c0 .205.059.375.18.508.102.129.199.238.299.336l.662.545a6.48 6.48 0 0 1 1.086 1.119c.293.415.445.955.459 1.625 0 .949-.42 1.789-1.256 2.521-.869.76-2.123 1.15-3.762 1.177-1.373-.015-2.4-.308-3.074-.879-.682-.534-1.023-1.172-1.023-1.917 0-.359.111-.766.336-1.211.215-.443.602-.837 1.172-1.174a6.126 6.126 0 0 1 1.998-.725 15.415 15.415 0 0 1 1.723-.173 3.996 3.996 0 0 1-.377-.596 1.347 1.347 0 0 1-.193-.74c0-.174.023-.317.074-.433a8.83 8.83 0 0 1 .121-.338 5.973 5.973 0 0 1-.633.035c-1.043-.012-1.838-.339-2.381-.979-.574-.603-.861-1.3-.861-2.097 0-.961.404-1.832 1.217-2.616.559-.46 1.135-.757 1.736-.898a8.494 8.494 0 0 1 1.67-.182h3.916l-1.209.704h-1.207v.005l-.005.002zm.754 11.276c0-.498-.162-.932-.484-1.302-.346-.348-.881-.778-1.609-1.29a4.045 4.045 0 0 0-.439-.02c-.102-.016-.354 0-.768.039a8.141 8.141 0 0 0-1.244.275c-.1.036-.24.096-.42.17a2.713 2.713 0 0 0-.553.355 1.985 1.985 0 0 0-.459.592c-.145.25-.215.551-.215.901 0 .687.311 1.253.932 1.698.596.444 1.404.68 2.432.688.928-.014 1.629-.217 2.117-.612.476-.385.71-.886.71-1.494zm-2.729-5.743a1.843 1.843 0 0 0 1.289-.556c.164-.249.271-.506.318-.767a7.32 7.32 0 0 0 .041-.659c0-.772-.195-1.552-.594-2.336a2.611 2.611 0 0 0-.73-.921 1.894 1.894 0 0 0-1.064-.352 1.723 1.723 0 0 0-1.324.64c-.299.437-.438.92-.428 1.457 0 .709.209 1.444.623 2.211.201.356.459.659.775.912.323.248.686.371 1.094.371zm9.698.077h-2.055v-2.056h-.996v2.056h-2.053v.998h2.053v2.052h.996v-2.052h2.055z"/></symbol><symbol id="icon-gplus" viewBox="0 0 24 24"><circle fill="#D84A36" cx="12" cy="12" r="12"/><g fill="#FFF"><path d="M12.012 12.111c-.304-.33-.07-.842.155-1.078.226-.241.639-.56.829-.74.193-.184.66-.605.674-1.868 0 0 .112-1.139-1.127-2.06h.977l1.038-.593h-3.507s-2.083-.067-3.085 1.207c-1 1.276-.966 2.871.114 3.77 1.08.898 2.371.648 2.371.648s-.197.422-.163.775c.036.353.421.849.504.965 0 0-2.386-.011-3.551 1.033-1.165 1.053-1.29 3.097 1.021 3.803 2.312.707 5.303-.055 5.906-2.277 0 0 .262-.908-.189-1.683-.459-.775-1.663-1.571-1.967-1.902zm-2.555-1.83c-.744-1-1.308-2.953 0-3.783 0 0 .955-.579 1.801.266.846.846 1.082 2.277.928 3.003-.15.727-.563.897-.739 1.008-.185.111-1.249.504-1.99-.494zm3.668 5.481c-.035.393-.083 1.254-1.491 1.664-1.409.408-3.826-.264-3.757-1.914 0 0 .006-.698.449-1.089.445-.396 1.094-.794 3.029-.819 0 0 .643.473 1.072.809.427.332.733.952.698 1.349z"/><path d="M16.323 11.055V9.803h-.752v1.252h-1.255v.748h1.255v1.256h.752v-1.256h1.248v-.748z"/></g></symbol><symbol id="icon-linkedin" viewBox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"/><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"/><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"/></g></symbol><symbol id="icon-reddit" viewBox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"/><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"/><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"/><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"/><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"/><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"/><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"/><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"/></symbol><symbol id="icon-twitter-bordered" viewBox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"/><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/></symbol><symbol id="icon-twitter" viewBox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"/><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"/></symbol><symbol id="icon-weibo-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"/><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"/><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"/></symbol><symbol id="icon-weibo" viewBox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"/><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"/><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"/><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"/><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"/><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"/></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner c-banner--info">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner" >
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1"
                 data-gpt-unitpath="/270604982/bmc/hereditasjournal/articles"
                 data-gpt-sizes="728x90,970x90"
                 data-gpt-targeting="pos=LB1;doi=10.1186/s41065-017-0031-7;kwrd=Alkaline stress,Differentially expressed genes,Prunus triloba Lindl,Physiological analysis,RNA sequencing;pmc=L32030,L21001;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-017-0031-7&amp;kwrd=Alkaline stress,Differentially expressed genes,Prunus triloba Lindl,Physiological analysis,RNA sequencing&amp;pmc=L32030,L21001&amp;">
                        <img data-test="gpt-advert-fallback-img"
                             src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-017-0031-7&amp;kwrd=Alkaline stress,Differentially expressed genes,Prunus triloba Lindl,Physiological analysis,RNA sequencing&amp;pmc=L32030,L21001&amp;"
                             alt="Advertisement"
                             width="728"
                             height="90">
                    </a>
                </noscript>
            </div>
        </div>
    </div>

        
         
    <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>


<header class="c-popup-search ctx-publisher-header" id="top" data-test="publisher-header">
    <nav class="c-navbar">
        <div class="u-container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization" />
                <div class="c-logo">
    <a href="https://www.biomedcentral.com" itemprop="url">
        <img src=/static/images/bmc/logos/logo-bmc-white-aj-e3a70ae848.svg alt="BMC" itemprop="logo">
        <div class="c-logo__strapline">
            <img src=/static/images/bmc/logos/logo-bmc-white-strapline-sn-e7b1d6e67f.svg alt="Part of Springer Nature">
        </div>
    </a>
</div>
            </div>
            <button
                type="button"
                class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg"
                id="publisher-header-menu-button"
                data-toggle="collapse"
                data-test="publisher-header-menu-toggle"
                data-toggle-linked="publisher-header"
                data-target="publisher-header-search publisher-header-menu"
                aria-controls="publisher-header-menu publisher-header-search"
                aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


                <ul class="c-navbar__nav">
                    <li class="c-navbar__item">
                        <a data-replace="true"
                           data-placeholder="/placeholder/v1/username"
                           class="c-navbar__link"
                           href="https://www.biomedcentral.com/login"
                           data-test="login-link"
                           data-track="click"
                           data-track-category="Account"
                           data-track-action="Click Login Link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                    <li class="c-navbar__item">
                        <button
                            type="button"
                            class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg"
                            data-toggle="collapse"
                            data-test="header-search-button"
                            data-target="publisher-header-search publisher-header-menu publisher-header-menu-button"
                            aria-controls="publisher-header-search publisher-header-menu"
                            aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true">
                                    <use xlink:href="#icon-search"></use>
                                </svg>
                            </span>
                        </button>
                    </li>
                </ul>
            </div>
        </div>
    </nav>
    <div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field c-form-field--inline" method="GET" action="//www.biomedcentral.com/search" data-track="submit"
        data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label data-track-label="" data-test="global-search">
        <label for="publisherSearch" class="u-visually-hidden">Search all BMC articles</label>
        <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text" value="" placeholder="Search all BMC articles"/>
        <button class="c-button c-button--secondary" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true">
        <use xlink:href="#icon-search"></use>
    </svg>
</button>


        <input type="hidden" name="searchType" value="publisherSearch"/>
    </form>
</div>

            </div>
        </div>
    </div>
</header>


        
            <header class="c-journal-header c-journal-header--hereditas ctx-journal-header">
                <div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="/">
    
        
    
    <span class="c-journal-title__text ">Hereditas</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="u-container">
                        
                            <button type="button"
                                    class="c-navbar__link c-dropdown u-hidden-at-lg"
                                    data-toggle="collapse"
                                    data-test="journal-header-menu"
                                    data-toggle-linked="journal-header"
                                    data-target="journal-menu"
                                    aria-controls="journal-menu"
                                    aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav">
                                    <ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/'>Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/about'>About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href='/articles'>Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/submission-guidelines'>Submission Guidelines</a>
                                            </li>
                                        
                                    </ul>
                                </nav>
                            </div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--ocean-light-blue">
                    
                </div>
            </header>
            
        

        <div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-content-layout c-content-layout--fulltext" data-component="sticky-container">
                <div class="c-content-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3><ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc=""><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Methods">Methods</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec13" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results and discussion">Results and discussion</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec23" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Conclusion">Conclusion</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li></ol></nav>
                </div>

                <main class="c-content-layout__main">
                    <article itemscope itemtype="http://schema.org/ScholarlyArticle">
                        
                        
    <div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
            <li class="c-article-identifiers__item" data-test="article-category">Research</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
    </div>

                        <div class="FulltextWrapper"><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">Analysis of differentially expressed genes and adaptive mechanisms of <em xmlns="" class="EmphasisTypeItalic">Prunus triloba</em> Lindl. under alkaline stress</h1></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true"><ul class="u-listReset"><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Jia&nbsp;Liu</span><sup><a href="#Aff1" class="AffiliationID">1</a>, <a href="#Aff2" class="AffiliationID">2</a>, <a href="#Aff3" class="AffiliationID">3</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Yongqing&nbsp;Wang</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup><a href="mailto:yqw14@sicau.edu.cn" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a> and </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Qingtian&nbsp;Li</span><sup><a href="#Aff4" class="AffiliationID">4</a></sup></li></ul></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation"><div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">Hereditas</span></span><span class="ArticleCitation_Year">2017</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">154</strong>:10</span></span></div><p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/s41065-017-0031-7">https://doi.org/10.1186/s41065-017-0031-7</a></p><p class="Copyright">© &nbsp;The Author(s)&nbsp;2017</p></div></div><ul class="ArticleHistory u-reset-list"><li class="History HistoryReceived"><strong>Received: </strong><span>30&nbsp;January&nbsp;2017</span></li><li class="History HistoryAccepted"><strong>Accepted: </strong><span>13&nbsp;April&nbsp;2017</span></li><li class="History HistoryOnlineDate"><strong>Published: </strong><span itemprop="datePublished">4&nbsp;May&nbsp;2017</span></li></ul></div>
                        
                        
                        
    

                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2><div class="c-section__content" id="Abstract"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p id="Par1" class="Para">
                           <em xmlns="" class="EmphasisTypeItalic">Prunus triloba</em> Lindl. is a naturally salt-alkaline-tolerant plant with several unique characteristics, and it can be used as the rootstock of Chinese plum (<em xmlns="" class="EmphasisTypeItalic">Prunus salicina</em> Lindl.) in saline-alkaline soils. To comprehensively investigate the alkaline acclimation mechanisms in <em xmlns="" class="EmphasisTypeItalic">P. triloba</em>, a series of analyses were conducted under alkaline stress, including analyses of the kinetics of molecular and physiological changes, and leaf microstructure.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Results</h3>
                <p id="Par2" class="Para">To understand the kinetics of molecular changes under short-term alkaline stress, we used Illumina HiSeq 2500 platform to identify alkaline stress-related differentially expressed genes (DEGs) in <em xmlns="" class="EmphasisTypeItalic">P. triloba</em>. Approximately 53.0 million high-quality clean reads were generated from 59.6 million raw reads, and a total of 124,786 unigenes were obtained after <em xmlns="" class="EmphasisTypeItalic">de novo</em> assembly of <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> transcriptome data. After alkaline stress treatment, a total of 8948 unigenes were identified as DEGs. Based on these DEGs, a Gene Ontology (GO) enrichment analysis was conducted, suggesting that 28 genes may play an important role in the early alkaline stress response. In addition, analysis of DEGs with the Kyoto Encyclopedia of Genes and Genomes (KEGG) revealed that pathways were significant at different treatment time points. A significant positive correlation was found between the quantitative real-time PCR (qRT-PCR) results and the RNA-Seq data for seven alkaline-related genes, confirming the reliability of the RNA-Seq results. Based on physiological analysis of <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> in response to long-term alkaline stress, we found that the internal microstructures of the leaves of <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> changed to adapt to long-term alkaline stress. Various physiological indexes indicated that the degree of membrane injury increased with increasing duration of alkaline stress, affecting photosynthesis in <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> seedlings.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusions</h3>
                <p id="Par3" class="Para">This represents the first investigation into the physiology and transcriptome of <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> in response to alkaline stress. The results of this study can enrich the genomic resources available for <em xmlns="" class="EmphasisTypeItalic">P. triloba</em>, as well as deepening our understanding of molecular and physiological alkaline tolerance mechanisms in <em xmlns="" class="EmphasisTypeItalic">P. triloba</em>. This will also provide new insights into our understanding of alkaline acclimation mechanisms in Chinese plum (<em xmlns="" class="EmphasisTypeItalic">Prunus salicina</em>) trees.</p>
              </div></div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords"><li class="c-keywords__item" itemprop="about">Alkaline stress</li><li class="c-keywords__item" itemprop="about">Differentially expressed genes</li><li class="c-keywords__item" itemprop="about">
                        <em xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="EmphasisTypeItalic">Prunus triloba</em> Lindl</li><li class="c-keywords__item" itemprop="about">Physiological analysis</li><li class="c-keywords__item" itemprop="about">RNA sequencing</li></ul></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 SectionTypeIntroduction c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par26" class="Para">Chinese plum (<em xmlns="" class="EmphasisTypeItalic">Prunus salicina</em> Lindl.) is a small deciduous tree that belongs to the genus <em xmlns="" class="EmphasisTypeItalic">Prunus</em> in the Rosaceae family. The wild peach (<em xmlns="" class="EmphasisTypeItalic">Prunus persica</em> Lindl.) is widely used as its rootstock in the area south of the Yangtze River in the provinces of Yunnan, Guizhou, and Sichuan; in the arid area of the northwest district; and in Henan, Hebei, Shandong, and other areas in China. While the wild peach possesses the characteristics of high adaptability, developed roots, fast growth, good grafting affinity, and rapid germination speed [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>], it also possesses some less positive characteristics. The major drawback of the wild peach is its poor saline-alkaline resistance. When the soil pH reaches 7.5–9.0, iron chlorosis generally occurs in plant tissues. This not only causes plants to oxidize available Fe<sup>2+</sup>, leading to the precipitation Fe(OH)<sub>3</sub>, but also influences other iron dissolution pathways and reduces the stability of chelated iron, leading to iron malabsorption [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>, <span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par27" class="Para">The central hill area in the Sichuan Basin is the main planting region for plum trees in China, and it widely consists of calcareous purple soils. The pH values of these soils range from 7.69 to 8.47, and iron chlorosis is common among fruit trees in this area. Iron chlorosis occurs easily in regions with alkaline soil; when fruit trees are under alkaline stress for an extended period of time, they grow poorly, fail to grow flowers or fruits, and may even die, having a detrimental effect on the fruit tree industry [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>]. Recently, several methods have been explored for improving saline-alkaline soils [<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>, <span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>]. Among these, the selection of an appropriate saline-alkaline-tolerant plant suited to the type of saline-alkaline soil is the main method used to cultivate plants on saline-alkaline land [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>]. Therefore, it is important to study the effect of alkaline damage of various plants and to select alkaline-resistant varieties of fruit trees or improve the alkaline resistance of fruit trees. However, the alkaline tolerance of fruit trees is actually dependent on the rootstock alkaline tolerance. Because the rootstock variety directly affects the fruiting time, yield, and lifetime of fruit trees, so the screening of alkaline-tolerant rootstocks is an effective way to improve the alkaline resistance of fruit trees.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par28" class="Para">
                        <em xmlns="" class="EmphasisTypeItalic">Prunus triloba</em> Lindl., also known as flowering plum or flowering almond, is a deciduous and flowering shrub or small tree species of the genus <em xmlns="" class="EmphasisTypeItalic">Prunus</em> (family Rosaceae) native to northeastern, northwestern, and northern China [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>, <span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>]. <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> is a popular ornamental plant in China, especially known as an important early spring flowering ornamental in the landscape of northern China. There are many varieties of <em xmlns="" class="EmphasisTypeItalic">P. triloba</em>, and it has been cultivated for more than 300&nbsp;years [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>, <span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>]. It is an octaploid with a karyotype of 2n = 8x = 64, and it possesses the characteristics of good adaptability, tolerance to poorly drained soils and to low level management, tolerance to cold stress and to drought stress, and high disease resistance [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>, <span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>, <span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>]. In addition, <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> is a naturally salt-alkaline-tolerant plant, and it grows well in saline-alkaline soils with a pH of 8.8 (0.3% salt content). <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> has good grafting affinity with Chinese plum, bears fruit early, results in high yield, suggesting that it can be used as the rootstock of <em xmlns="" class="EmphasisTypeItalic">P. salicina</em> in the middle hill region in the Sichuan Basin [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par29" class="Para">A range of abiotic and biotic stresses can severely restrict plant growth and reduce crop productivity, of which soil salinity, alkalinity, or saline-alkalinity are the most common abiotic stresses encountered by plants [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>]. These types of soils are widely distributed across more than 100 countries [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>]. The salinization of soil is usually caused by neutral salt (NaCl, Na<sub>2</sub>SO<sub>4</sub>, etc.), while the alkalinization of soil is usually caused by alkaline salt (NaHCO<sub>3</sub>, Na<sub>2</sub>CO<sub>3</sub>, etc.) and soil saline-alkalinity is caused by the presence of both neutral and alkaline salts [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>]. In contrast to saline soil, which mainly affects plants through osmotic stress and ion toxicity, alkaline soil damages plants through high pH [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>]. Previous research showed that the negative effect of soil alkalinization on plants was greater than that of soil salinization [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>, <span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>]. While there have been a number of reports on the mechanisms of plant tolerance to salinity [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>], much less attention has been paid to the plant response to alkaline stress. Therefore, analysis of the underlying mechanisms of soil alkalinity on plant growth and development are urgently needed, especially for commercial crops.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par30" class="Para">Plant stress resistance is not the result of a single mechanism of action but rather is a complex stress response borne of various aspects of cell, physiological, and molecular biology. The physiological and biochemical mechanisms of plant abiotic stress tolerance have widely been studied by researchers, and the underlying molecular mechanisms have always been the focus of plant biology research area. The molecular mechanisms of plant abiotic stress tolerance involve multiple genes and complex genetic regulatory networks [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>]. Only by understanding the functions of these stress-inducible genes can we decipher the related physiological, biochemical, and molecular mechanisms of stress tolerance [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>]. Of late, emerging high-throughput techniques have become a significant means of achieving this goal. RNA sequencing (RNA-Seq) technology is frequently used to study plant stress tolerance because it is an economical, sensitive, and efficient tool for the identification and analysis of key stress-responsive genes. In addition, it can be used for non-model species [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>–<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par31" class="Para">Therefore, in order to comprehensively investigate alkaline acclimation mechanisms in <em xmlns="" class="EmphasisTypeItalic">P. triloba</em>, a series of analyses were performed under alkaline stress, including assessment of the kinetics of molecular and physiological changes, and leaf microstructure. The Illumina HiSeq 2500 platform was used to examine the transcriptomes of leaf samples from alkaline-treated or control <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> seedlings. We then <em xmlns="" class="EmphasisTypeItalic">de novo</em> assembled a transcriptome library of <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> to identify differentially expressed genes (DEGs) from plants under alkaline stress. To our knowledge, this is the first physiological and transcriptomic analysis of <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> in response to alkaline stress. Our results will enrich genomic resources for <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> and deepen our understanding of molecular and physiological alkaline tolerance mechanisms in <em xmlns="" class="EmphasisTypeItalic">P. triloba</em>. Our results also will provide new insight into understanding alkaline acclimation mechanisms in Chinese plum (<em xmlns="" class="EmphasisTypeItalic">P. salicina</em>) trees.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 SectionTypeMaterialsAndMethods c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Methods" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Methods">Methods</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Methods" data-test="collapse-section"><section id="Sec3" class="Section2 RenderAsSection2">
                <h3 class="Heading">Plant materials and alkaline stress treatment conditions</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par32" class="Para">The seeds of <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> were preserved in low temperature sand in November 2014. On the middle ten days of the second month of the second year, the seeds of <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> were sown in the earthenware pots, about 25&nbsp;cm in diameter, and with holes in the bottom. Each pot was filled with equal amount of soils, which were fully mixed by peat soil and turfy soil in the ratio of 1 to 1. When the height of seedlings reached to about 15&nbsp;cm, 120 plantlets with similar size and growth, housing outdoor with a transparent plastic film at the top to keep out the rain, were selected to subject to alkaline stress treatments as the test materials, and other 30 plantlets were selected as control group. The plantlets of alkaline stress treatments were watered using alkaline liquor, which was established by mixing NaHCO<sub>3</sub> and Na<sub>2</sub>CO<sub>3</sub> at 9:1 molar ratio, and the concentration and pH value of the solution were 50&nbsp;mmol/L and 9.11 ± 0.104, respectively. The alkaline solution was firstly used to water the test materials at 16:00 sunny day. Thereafter, the alkaline solution was poured every 3&nbsp;days, and every time until the pH of effluent from the bottom of pot reached more than 9.0. The control group was simultaneously with the same amount of water irrigation. The alkaline treatment lasted for 12&nbsp;days.</p>
              </section><section id="Sec4" class="Section2 RenderAsSection2">
                <h3 class="Heading">
                           <em class="EmphasisTypeItalic">De novo</em> transcriptome analysis of <em class="EmphasisTypeItalic">P. triloba</em> in response to short-term alkaline stress</h3>
                <section id="Sec5" class="Section3 RenderAsSection3">
                  <h4 class="Heading">RNA isolation, cDNA library construction and sequencing</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par33" class="Para">We separately sampled the leaf blades at 0, 1, 3, 6, and 12&nbsp;h after the alkaline stress. The harvested leaf samples were immediately frozen in liquid nitrogen for RNA extraction. This process was performed in two biological replicates at the each time point, of which each replicate consisted of 10 pooled leaves. The achieved ten <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> samples were isolated using the Universal Plant RNA Extraction Kit (Aidlab, Beijing, China) in accordance with the manufacturer’s protocol. RNA purity, concentration, and quality were examined with a Nanodrop 2000 UV-Vis spectrophotometer (Thermo Fisher Scientific Inc., USA), Qubit® 2.0 Fluorometer (Life Technologies, USA) and Agilent 2100 Eukaryote Total RNA Nano Kit (Agilent, Santa Clara, CA, USA), respectively.</p>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par34" class="Para">Total 10 RNA samples of <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> were then delivered to GENEWIZ, Inc. (Suzhou, China) for cDNA library construction and sequencing. In briefly, cDNA library was built for each of 10 RNA samples using the NEBNext Ultra RNA Library Prep Kit for Illumina (New England Biolabs, Ipswich, MA, USA). In order to ensure the quality of the library, a Beckman AMPure XP Bead Kit (Beckman Coulter, Danvers, MA, USA) and Agilent 2100 High Sensitivity DNA Kit (Agilent) were then used to purify and to detect the quality of the library, respectively, and the effective concentration of the library was accurately quantified with a ABI 7500 real time PCR system. Subsequently, cBOT automatic clustering of these qualified cDNA libraries were conducted using TruSeq PE Cluster Kit V3 (Illumina). Finally, High-throughput sequencing was sequenced on Illumina HiSeq 2500 platform by synthesis.</p>
                </section>
                <section id="Sec6" class="Section3 RenderAsSection3">
                  <h4 class="Heading">De novo transcriptome assembly</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par35" class="Para">The obtained sequencing raw data in FASTq format were initially processed by Trimmomatic (v0.30) [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>] to get high-quality clean data, which trimmed reads containing adaptors, filtered off low quality reads (bases with quality scores&lt;20 at the 3′ or 5′ end) and reads with poly-N, and removed sequences shorter than 20&nbsp;bp. The clean reads from each library were evaluated with the Q20 (the percentage of bases with a Phred value &gt;20), Q30 (the percentage of bases with a Phred value &gt;30), GC-content, and N-content, and were checked using fastqc (<span class="ExternalRef"><a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/"><span class="RefSource">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</span></a></span>). Then, <em xmlns="" class="EmphasisTypeItalic">de novo</em> assembly of the clean reads was achieved using the Trinity assembler (version r2013-02-25) [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>]. The software TGICL [<span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>] was further used to splice sequences and redundancy analysis in order to produce longer, more complete non-redundant sequences which were called unigenes. The software TransDecoder [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>] was introduced to predict the open reading frames (ORFs) of transcripts and to extract longest open-ended ORFs. Finally, the clean reads were mapped to the unigenes using Bowtie2 (v2.1.0) [<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>] with the default parameters.</p>
                </section>
                <section id="Sec7" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Quantification of gene expression levels and differential expression analysis</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par36" class="Para">The gene expression level of the assembled unigenes of each sample was separately estimated by software RSEM (V 1.2.4) [<span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>] using the FPKM (fragments per kilobase per million reads) method. Differential gene expression analysis of all samples after different periods of alkaline stress was performed using the R package DESeq (V 1.14.0) based on the negative binomial distribution [<span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>] to identify differentially expressed genes (DEGs) in <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> treated with alkaline stress. The <em xmlns="" class="EmphasisTypeItalic">P</em>-value corresponded to the differential gene expression test. A False discovery rate (FDR) method was used to determine the threshold of the <em xmlns="" class="EmphasisTypeItalic">P</em>-value. Genes with a FDR threshold ≤ 0.05 and |log2 (fold change)| &gt; 1 found by DESeq were designated as differentially expressed. Afterwards, GO functional categories were assigned to differentially expressed genes implementing by the R package GOseq based on the Walleius non-central hyper-geometric distribution [<span class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>]. DEGs also underwent KEGG pathway enrichment analysis in order to systematically understand the biological pathways involved in the DEGs using software KOBAS [<span class="CitationRef"><a href="#CR28" data-toggle="citation">28</a></span>].</p>
                </section>
                <section id="Sec8" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Gene validation of qRT-PCR analysis</h4>
                  <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par37" class="Para">To validate the DEGs detected by RNA-Seq, seven unigenes, which were determined to be significantly related to alkaline stress, were subjected to real-time quantitative reverse transcription-PCR (qRT-PCR) analysis. The RNA samples were extracted from the leaf tissues of <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> materials at 0&nbsp;h, 3&nbsp;h, 6&nbsp;h, and 12&nbsp;h after the alkaline stress using the same method described above. A TUREscript 1st Stand cDNA SYNTHESIS Kit (Aidlab Biotechnologies Co., Ltd, Beijing, China) was used to synthesize the cDNAs as described by the manufacturer using an Oligo (dT) Primer with 500&nbsp;ng total RNA. Specific primers designed for the seven selected genes were listed (Table&nbsp;<span class="InternalRef"><a href="#Tab1">1</a></span>). Real-time RT-PCR of the seven genes in technical and biological triplicates was performed using 2×SYBR® Green Supermix (DBI® Bioscience, Germany) in an optical 96-well plate with a qTOWER2.2 Real-Time PCR System (Anlytik Jena AG, Germany) in final reaction volumes of 25&nbsp;μL, each containing 5&nbsp;μL 1× 2×SYBR® Green Supermix (DBI® Bioscience, Germany), 0.5&nbsp;μL 200 nM of the reverse primer and sense primer, 1&nbsp;μL cDNA template, and 3&nbsp;μL ddH<sub>2</sub>O. The thermal cycling conditions were as follows: 95&nbsp;°C 3&nbsp;min; 39&nbsp;cycles of 95&nbsp;°C for 10&nbsp;s, 60&nbsp;°C for 30&nbsp;s, and then increasing the temperature to 95&nbsp;°C (+1&nbsp;°C/cycle, holding time 4&nbsp;s). The internal reference gene was the actin gene. The relative expression level of each gene was determined using threshold cycles, by the 2<sup>-<span class="stix">△</span><span class="stix">△</span></sup>Ct method.<figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 1</span>
                          <p xmlns="" class="SimplePara">Primer information for qRT-PCR</p>
                        </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Gene Ontology</p></th><th><p xmlns="" class="SimplePara">ID</p></th><th><p xmlns="" class="SimplePara">Upstream / downstream primer (5′-3′)</p></th></tr></thead><tbody><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">AP-2 complex subunit alpha</p></td><td rowspan="2"><p xmlns="" class="SimplePara">c78498_g1_i1</p></td><td><p xmlns="" class="SimplePara">CACTGGTATAGGCTGAAGAA</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">TTCGGTAACAATATGGTCAATG</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">ALDH18A1, P5CS</p></td><td rowspan="2"><p xmlns="" class="SimplePara">c80260_g1_i1</p></td><td><p xmlns="" class="SimplePara">TGGTATCTTAGTGGACTC</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">AACTTATTGGACTTGTGAA</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">mitogen-activated protein kinase 1/3</p></td><td rowspan="2"><p xmlns="" class="SimplePara">c81662_g1_i1</p></td><td><p xmlns="" class="SimplePara">GTGTTGCTGTTGGTGTTG</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">CTTCTTCCTCCTCCTCTCA</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">aroDE, DHQ-SDH</p></td><td rowspan="2"><p xmlns="" class="SimplePara">c84090_g1_i1</p></td><td><p xmlns="" class="SimplePara">GTAGTGTCAGTAGGTATGGAGAG</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">GAGGTTAGGTGCGTGGAT</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">AP2-like factor, ANT lineage</p></td><td rowspan="2"><p xmlns="" class="SimplePara">c85502_g2_i1</p></td><td><p xmlns="" class="SimplePara">GCAGCATCATAAGGAATC</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">GGCAGCAACAATATCAAT</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">trehalose 6-phosphate synthase/phosphatase</p></td><td rowspan="2"><p xmlns="" class="SimplePara">c85846_g1_i3</p></td><td><p xmlns="" class="SimplePara">GCCAGCATATTGTTGAAG</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">CCATCATTGACCATTCTTG</p></td></tr><tr class="noclass"><td rowspan="4"><p xmlns="" class="SimplePara">mitogen-activated protein kinase kinase 1</p></td><td rowspan="2"><p xmlns="" class="SimplePara">c88652_g1_i2</p></td><td><p xmlns="" class="SimplePara">CATCTATCTGTGGAAGGA</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">GCATCTGAATAGGAATCTG</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">ACTIN</p></td><td><p xmlns="" class="SimplePara">GCCGAGTATGATGAATCTG</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">GAGTCTGCCTATCCTTGA</p></td></tr></tbody></table></div></figure>
                           </div>
                </section>
              </section><section id="Sec9" class="Section2 RenderAsSection2">
                <h3 class="Heading">Physiological analysis of <em class="EmphasisTypeItalic">P. triloba</em> in response to long-term alkaline stress</h3>
                <section id="Sec10" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Effects of long-term alkaline stress on the leaf microstructure of <em class="EmphasisTypeItalic">P. triloba</em>
                           </h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par38" class="Para">The upper leaf blades of the test materials on 0 d and 12 d after the alkaline stress were sampled to immediately use for observing the blade structure through the microscope. The center area (5&nbsp;mm × 5&nbsp;mm) of both sides of leaves avoided the leaf vein were sampled by one side blade. The harvested slices were placed in distilled water in petri dish, and then their upper and lower epidermis were removed with a tweezer. The obtained transparent slices without upper and lower epidermis were selected to place in a slide center, and some distilled water were added to these slices, and then were covered with a coverslip. After above steps, the morphology structure of the transparent slice was examined by microscope. The thickness of the cuticle, the palisade tissue, the spongy tissue, the epidermal cells, and the stomata were as the observation indexes. Each index was observated 10 visual fields in the corresponding parts of slice, and its average number was counted, meanwhile, was photographed.</p>
                </section>
                <section id="Sec11" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Effects of long-term alkaline stress on <em class="EmphasisTypeItalic">P. triloba</em> seedling physiological indices</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par39" class="Para">We separately sampled the leaf blades of <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> at 16:00&nbsp;pm on day 0, 3 d, 6 d, 9 d and 12 d after the alkaline stress. The harvested leaf samples were used for the determination of the physiological activity indexes, and each index was sampled at random for three replicates. Ten physiological activity indexes were measured, including relative water content (RWC), relative electrical conductivity (REC), chlorophyll (Chl) content, soluble sugar content (SSC), soluble protein content (SPC), proline (Pro) content, malondialdehyde (MDA) content, the activity of peroxidase (POD), catalase (CAT), and superoxide dismutase (SOD). The RWC, the content of MDA, the activity of POD, and SOD were determined according to Sun and Hu [<span class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>]; the calculation of the REC was made according to Xue and Xia [<span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>]; the chlorophyll (Chl) content was determined according to Zhu et al [<span class="CitationRef"><a href="#CR31" data-toggle="citation">31</a></span>]; the SSC, SPC, Pro content, and the activity of CAT were measured according to Li [<span class="CitationRef"><a href="#CR32" data-toggle="citation">32</a></span>].</p>
                </section>
                <section id="Sec12" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Effects of long-term alkaline stress on <em class="EmphasisTypeItalic">P. triloba</em> seedling photosynthetic parameters</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par40" class="Para">The healthy and representative leaf blades of the test materials under alkaline stress and control materials on the 12th day were also used for the determination of diurnal dynamics of the photosynthetic characteristics. The photosynthetic parameters, including the net photosynthetic rate (Pn), stomatal conductance (Gs), the internal CO<sub>2</sub> concentration (Ci), and transpiration rate (Tr), were measured every 2&nbsp;h using a Li-6400XT portable photosynthesis system from 7:00&nbsp;am to 19:00&nbsp;pm. The water use efficiency (WUE) was calculated by the following formula: WUE = Pn/Tr. Each parameter was measured for nine replicates under a 3 (leaf) × 3 (plantlet) arrangement.</p>
                </section>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec13" class="Section1 RenderAsSection1 SectionTypeResults c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Resultsanddiscussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Resultsanddiscussion">Results and discussion</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Resultsanddiscussion" data-test="collapse-section"><section id="Sec14" class="Section2 RenderAsSection2">
                <h3 class="Heading">
                           <em class="EmphasisTypeItalic">De novo</em> transcriptome analysis of <em class="EmphasisTypeItalic">P. triloba</em> in response to short-term alkaline stress</h3>
                <section id="Sec15" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Sequence analysis and assembly</h4>
                  <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par41" class="Para">In this study, we used the Illumina HiSeq 2500 platform to obtain a total of 595,894,216 raw <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> reads during different stages of alkaline stress. These raw data were deposited in the NCBI SRA database under accession number SRP098782. After quality control trimming and filtering, there were 529,847,752 high-quality reads (~62.10 Gb clean data) remaining. The Q20, Q30, and GC content of the clean data were over 99.16, 97.69, and 45.16%, respectively, for each sample (Table&nbsp;<span class="InternalRef"><a href="#Tab2">2</a></span>). These results showed that our obtained clean reads were of a sufficiently high quality for subsequent analysis. Using the <em xmlns="" class="EmphasisTypeItalic">de novo</em> assembly software Trinity and TGICL, 124,786 unigenes were obtained for all samples, with an average size of 668.65&nbsp;bp and an N50 length of 1111&nbsp;bp (Fig.&nbsp;<span class="InternalRef"><a href="#Fig1">1</a></span>). Among the assembled unigenes, 80,060 (64.16%) were 200–500&nbsp;bp in length while 22,426 (17.97%) were 500–1000&nbsp;bp in length (Fig.&nbsp;<span class="InternalRef"><a href="#Fig1">1</a></span>). To our knowledge, this is the first large-scale <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> genomic resource to be made available. The obtained transcript sequences therefore enrich the genomic resources available for <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> and can be used not only for gene discovery but also for identification of simple sequence repeat (SSR) or single nucleotide polymorphism (SNP) markers.<figure xmlns="" id="Tab2" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 2</span>
                          <p xmlns="" class="SimplePara">Summary of sequences analysis</p>
                        </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col char="."></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Sample</p></th><th><p xmlns="" class="SimplePara">Raw reads</p></th><th><p xmlns="" class="SimplePara">Clean reads</p></th><th char="."><p xmlns="" class="SimplePara">Clean bases</p></th><th><p xmlns="" class="SimplePara">Q20 (%)</p></th><th><p xmlns="" class="SimplePara">Q30 (%)</p></th><th><p xmlns="" class="SimplePara">GC (%)</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">A1-0</p></td><td><p xmlns="" class="SimplePara">33,244,262</p></td><td><p xmlns="" class="SimplePara">30,036,644</p></td><td char="."><p xmlns="" class="SimplePara">3.51&nbsp;G</p></td><td><p xmlns="" class="SimplePara">99.43</p></td><td><p xmlns="" class="SimplePara">98.43</p></td><td><p xmlns="" class="SimplePara">46.00</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">A2-0</p></td><td><p xmlns="" class="SimplePara">67,111,312</p></td><td><p xmlns="" class="SimplePara">59,313,064</p></td><td char="."><p xmlns="" class="SimplePara">6.97&nbsp;G</p></td><td><p xmlns="" class="SimplePara">99.25</p></td><td><p xmlns="" class="SimplePara">97.94</p></td><td><p xmlns="" class="SimplePara">46.38</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">A1-1</p></td><td><p xmlns="" class="SimplePara">75,018,602</p></td><td><p xmlns="" class="SimplePara">66,678,644</p></td><td char="."><p xmlns="" class="SimplePara">7.80&nbsp;G</p></td><td><p xmlns="" class="SimplePara">99.28</p></td><td><p xmlns="" class="SimplePara">98.03</p></td><td><p xmlns="" class="SimplePara">45.41</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">A2-1</p></td><td><p xmlns="" class="SimplePara">50,426,168</p></td><td><p xmlns="" class="SimplePara">45,822,060</p></td><td char="."><p xmlns="" class="SimplePara">5.38&nbsp;G</p></td><td><p xmlns="" class="SimplePara">99.44</p></td><td><p xmlns="" class="SimplePara">98.43</p></td><td><p xmlns="" class="SimplePara">45.49</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">A1-3</p></td><td><p xmlns="" class="SimplePara">42,772,046</p></td><td><p xmlns="" class="SimplePara">38,114,654</p></td><td char="."><p xmlns="" class="SimplePara">4.45&nbsp;G</p></td><td><p xmlns="" class="SimplePara">99.27</p></td><td><p xmlns="" class="SimplePara">97.99</p></td><td><p xmlns="" class="SimplePara">46.08</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">A2-3</p></td><td><p xmlns="" class="SimplePara">65,192,034</p></td><td><p xmlns="" class="SimplePara">57,586,408</p></td><td char="."><p xmlns="" class="SimplePara">6.76&nbsp;G</p></td><td><p xmlns="" class="SimplePara">99.25</p></td><td><p xmlns="" class="SimplePara">97.93</p></td><td><p xmlns="" class="SimplePara">45.66</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">A1-6</p></td><td><p xmlns="" class="SimplePara">46,008,224</p></td><td><p xmlns="" class="SimplePara">41,153,398</p></td><td char="."><p xmlns="" class="SimplePara">4.81&nbsp;G</p></td><td><p xmlns="" class="SimplePara">99.27</p></td><td><p xmlns="" class="SimplePara">97.98</p></td><td><p xmlns="" class="SimplePara">45.46</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">A2-6</p></td><td><p xmlns="" class="SimplePara">68,608,076</p></td><td><p xmlns="" class="SimplePara">60,487,348</p></td><td char="."><p xmlns="" class="SimplePara">7.09&nbsp;G</p></td><td><p xmlns="" class="SimplePara">99.25</p></td><td><p xmlns="" class="SimplePara">97.95</p></td><td><p xmlns="" class="SimplePara">45.43</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">A1-12</p></td><td><p xmlns="" class="SimplePara">75,873,652</p></td><td><p xmlns="" class="SimplePara">67,335,634</p></td><td char="."><p xmlns="" class="SimplePara">7.90&nbsp;G</p></td><td><p xmlns="" class="SimplePara">99.16</p></td><td><p xmlns="" class="SimplePara">97.69</p></td><td><p xmlns="" class="SimplePara">45.21</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">A2-12</p></td><td><p xmlns="" class="SimplePara">71,639,840</p></td><td><p xmlns="" class="SimplePara">63,319,898</p></td><td char="."><p xmlns="" class="SimplePara">7.43&nbsp;G</p></td><td><p xmlns="" class="SimplePara">99.24</p></td><td><p xmlns="" class="SimplePara">97.91</p></td><td><p xmlns="" class="SimplePara">45.16</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Total</p></td><td><p xmlns="" class="SimplePara">595,894,216</p></td><td><p xmlns="" class="SimplePara">529,847,752</p></td><td char="."><p xmlns="" class="SimplePara">62.10&nbsp;G</p></td><td><p xmlns="" class="SimplePara">–</p></td><td><p xmlns="" class="SimplePara">–</p></td><td><p xmlns="" class="SimplePara">–</p></td></tr></tbody></table></div></figure>
                              <figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO1">
                        <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig1_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig1_HTML.gif" alt="Fig. 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig1_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig1_HTML.gif" alt="Fig. 1"></noscript></a>
                      </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 1</span>
                          <p xmlns="" class="SimplePara">Pie chart for length distribution of the assembled unigenes</p>
                        </div></figcaption></figure>
                           </div>
                </section>
                <section id="Sec16" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Quantitative analysis of gene expression and identification of differentially expressed genes</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par42" class="Para">In this study, the FPKM method was used to estimate the gene expression levels of assembled unigenes from each sample. Within the 10 samples, 20.27–32.49% of unigenes exhibited FPKM values between 0 and 1, while only 1.24–1.57% exhibited FPKM values over 60 (Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>). Gene expression levels in these samples were compared by assessing the distribution diagram of the FPKMs of all genes. As shown in the Additional file <span class="InternalRef"><a href="#MOESM2">2</a></span>, the density of genes among these 10 samples was highest at the log FPKM values about range within −0.30 to −0.60, and the maximum gene density was highest for sample A1-12.</p>
                  <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par43" class="Para">Genes with an FDR threshold ≤ 0.05 and |log2 (fold change)| &gt; 1 according to DESeq were designated as differentially expressed. In this study, among 124,786 unigenes, 8948 (7.17%) were identified as DEGs in leaves under alkaline stress. Among these, 0, 186, 3211 and 8347 DEGs were discovered through transcriptome comparison of the samples 1&nbsp;h vs. 0&nbsp;h, 3&nbsp;h vs. 0&nbsp;h, 6&nbsp;h vs. 0&nbsp;h, and 12&nbsp;h vs. 0&nbsp;h, respectively (Fig.&nbsp;<span class="InternalRef"><a href="#Fig2">2</a></span>). The number of DEGs found in the 12&nbsp;h vs. 0&nbsp;h comparison (8347) was therefore the highest. This result is inconsistent with the patterns found in <em xmlns="" class="EmphasisTypeItalic">Puccinellia tenuiflor</em> [<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>] and wild soybean (<em xmlns="" class="EmphasisTypeItalic">Glycine soja</em>) [<span class="CitationRef"><a href="#CR33" data-toggle="citation">33</a></span>] roots, where the number of DEGs at 6&nbsp;h was higher than that at 12 and 24&nbsp;h. This difference might be attributed to the fact that the cellular response to extracellular signals in the leaves occurs later than that in the roots. It is noteworthy that no DEGs were detected until after 3&nbsp;h of stress treatment in this study. This result was similar to that in wild soybean (<em xmlns="" class="EmphasisTypeItalic">G. soja</em>) [<span class="CitationRef"><a href="#CR33" data-toggle="citation">33</a></span>] roots, which also took 3&nbsp;h to respond to alkaline stress at the transcription level. Of the 8948 DEGs, 181 were up-regulated in 3&nbsp;h vs. 0&nbsp;h; 1999 were up-regulated in 6&nbsp;h vs. 0&nbsp;h; and 4895 were up-regulated in 12&nbsp;h vs. 0&nbsp;h. In contrast, 5, 1212, and 3452 DEGs were down-regulated in 3&nbsp;h vs. 0&nbsp;h, 6&nbsp;h vs. 0&nbsp;h, and 12&nbsp;h vs. 0&nbsp;h, respectively. The number of up-regulated genes was greater than that of down-regulated in 3&nbsp;h vs. 0&nbsp;h, 6&nbsp;h vs. 0&nbsp;h, and 12&nbsp;h vs. 0&nbsp;h, which was consistent with the <em xmlns="" class="EmphasisTypeItalic">Arabidopsis</em> transcriptome profile under various abiotic stresses [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>].<figure xmlns="" class="Figure" id="Fig2"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO2">
                        <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig2_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig2_HTML.gif" alt="Fig. 2" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig2_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig2_HTML.gif" alt="Fig. 2"></noscript></a>
                      </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 2</span>
                          <p xmlns="" class="SimplePara">Number of DEGs at different time points after under alkaline stress compared with 0&nbsp;h. Total number of up- (red bars) and down-regulated (blue bars) genes in leaves compared with the sample without stress (0&nbsp;h)</p>
                        </div></figcaption></figure>
                           </div>
                </section>
                <section id="Sec17" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Functional categorization of DEGs</h4>
                  <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par44" class="Para">To further characterize the temporal expression changes under alkaline stress, GO enrichment analysis (<em xmlns="" class="EmphasisTypeItalic">P</em>-value ≤ 0.01) of DEGs found in the 3&nbsp;h vs. 0&nbsp;h, 6&nbsp;h vs. 0&nbsp;h, and 12&nbsp;h vs. 0&nbsp;h was conducted with the whole transcriptome as the background. Analysis revealed enrichment of GO terms related to cellular components, molecular functions, and biological processes. Of the 186, 3211, and 8347 DEGs found in the 3&nbsp;h vs. 0&nbsp;h, 6&nbsp;h vs. 0&nbsp;h, and 12&nbsp;h vs. 0&nbsp;h comparison, respectively, 0, 24, and 72 DEGs were annotated in a GO cellular component category; 9, 146, and 307 DEGs were annotated in a GO molecular function ontology; and 11, 86, and 184 DEGs were annotated in a GO biological process category. For cellular component (Fig.&nbsp;<span class="InternalRef"><a href="#Fig3">3</a></span>), there was no gene annotated to the ontology term found in the 3&nbsp;h vs. 0&nbsp;h comparisons; whereas cell [16 (66.67%); 53 (73.61%)], cell part [16 (66.67%); 53 (73.61%)], intracellular part [13 (54.17%); 47 (65.28%)], and intracellular [13 (54.17%); 47 (65.28%)] were the most chasses for both the 6&nbsp;h vs. 0&nbsp;h, and 12&nbsp;h vs. 0&nbsp;h comparisons. In the molecular function group, as shown in Fig.&nbsp;<span class="InternalRef"><a href="#Fig4">4</a></span>, nucleic acid binding transcription factor activity (4, 44.44%) and sequence-specific DNA binding transcription factor activity (4, 44.44%) were the two most highly represented categories in the 3&nbsp;h vs. 0&nbsp;h comparisons; for the 6&nbsp;h vs. 0&nbsp;h and 12&nbsp;h vs. 0&nbsp;h comparisons, the top categories were binding [70 (47.95%); 152 (65.28%)], catalytic activity [51 (34.93%); 106 (34.53%)], heterocyclic compound binding [38 (26.03%); 79 (27.73%)], and organic cyclic compound binding [38 (26.03%); 79 (27.73%)]. Under biological process, genes from the 3&nbsp;h vs. 0&nbsp;h comparison were most commonly annotated as single-organism metabolic process (5, 45.45%), oxylipin metabolic process (2, 18.18%), and oxylipin biosynthetic process (2, 18.18%). Similarly, metabolic process category [49 (56.98%); 101 (54.89%)], cellular process [28 (32.56%); 78 (42.39%)], organic substance metabolic process [29 (33.72%); 70 (38.04%)], and primary metabolic process [27 (31.40%); 63 (34.24%)] were the mostly highly enriched in 6&nbsp;h vs. 0&nbsp;h and 12&nbsp;h vs. 0&nbsp;h comparisons (Fig.&nbsp;<span class="InternalRef"><a href="#Fig5">5</a></span>).<figure xmlns="" class="Figure" id="Fig3"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO3">
                        <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig3_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig3_HTML.gif" alt="Fig. 3" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig3_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig3_HTML.gif" alt="Fig. 3"></noscript></a>
                      </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 3</span>
                          <p xmlns="" class="SimplePara">Comparative distribution of GO enrichment of terms related to cellular components</p>
                        </div></figcaption></figure>
                              <figure xmlns="" class="Figure" id="Fig4"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO4">
                        <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig4_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig4_HTML.gif" alt="Fig. 4" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig4_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig4_HTML.gif" alt="Fig. 4"></noscript></a>
                      </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 4</span>
                          <p xmlns="" class="SimplePara">Comparative distribution of GO enrichment of terms related to molecular function</p>
                        </div></figcaption></figure>
                              <figure xmlns="" class="Figure" id="Fig5"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO5">
                        <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig5_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig5_HTML.gif" alt="Fig. 5" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig5_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig5_HTML.gif" alt="Fig. 5"></noscript></a>
                      </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 5</span>
                          <p xmlns="" class="SimplePara">Comparative distribution of GO enrichment of terms related to biological processes</p>
                        </div></figcaption></figure>
                           </div>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par45" class="Para">The results of GO enrichment analysis showed that few genes were annotated with GO ontology terms in the 3&nbsp;h vs. 0&nbsp;h comparison. These GO ontology terms included only nucleic acid binding transcription factor activity and sequence-specific DNA binding transcription factor activity in the molecular function group, and single-organism metabolic process, oxylipin metabolic process, and oxylipin biosynthetic process in the biological process group. These GO ontology terms showed the earliest response to alkaline stress and may generate a cascade responses following their induction at 3&nbsp;h of alkaline stress treatment. Therefore, these genes may play an important role in early alkaline resistance mechanisms. With increasing duration of alkaline stress treatment, more DEGs were annotated with more GO terms, of which cell and cell part in the cellular component group, binding and catalytic activity in the molecular function group, and metabolic process category and cellular process in the biological process group were the most represented categories, suggesting that the internal tissues and organs of <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> had undergone multiple processes of anabolism to cope with and adapt to alkaline stress. It is noteworthy that 28 genes (c57635_g1_i1, c64583_g1_i1, c64900_g1_i1, c71351_g1_i1, c72539_g1_i1, c73498_g1_i4, c73839_g1_i2, c73869_g1_i2, c74641_g1_i1, c75112_g1_i1, c76093_g2_i1, c76646_g1_i1, c78345_g1_i1, c78766_g1_i1, c79591_g1_i1, c79591_g1_i2, c79591_g1_i3, c81104_g2_i1, c82384_g1_i1, c82500_g1_i1, c83082_g2_i1, c83291_g1_i3, c83618_g1_i4, c83799_g2_i1, c86513_g2_i1, c87340_g1_i1, c87510_g1_i2, c87918_g1_i2) as being involved in response to stimulus, indicating they may play a role in the alkaline stress response. KEGG annotation of these 28 genes revealed that c71351_g1_i1 is a RNA polymerase-associated protein, and c73869_g1_i2 is a HSP20 family protein. In the COG analysis of these stress-related genes, only five genes were involved in posttranslational modification, protein turnover, chaperones (c73869_g1_i2, c74641_g1_i1, c78766_g1_i1), replication, recombination and repair (c82384_g1_i1), and nucleotide transport and metabolism (c83082_g2_i1). Therefore, there appears to be a link between the putative functions and annotations of these 28 genes and the alkaline stress response in <em xmlns="" class="EmphasisTypeItalic">P. triloba</em>.</p>
                  <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par46" class="Para">KEGG pathway enrichment analysis for DEGs was also conducted. There were several differences between the results of the KEGG analyses of the 3&nbsp;h vs. 0&nbsp;h genes, 6&nbsp;h vs. 0&nbsp;h genes, and 12&nbsp;h vs. 0&nbsp;h comparisons (Fig.&nbsp;<span class="InternalRef"><a href="#Fig6">6</a></span>). A total of 22, 403, and 1006 DEGs for 3&nbsp;h vs. 0&nbsp;h, 6&nbsp;h vs. 0&nbsp;h, and 12&nbsp;h vs. 0&nbsp;h comparison, respectively, were assigned to 42, 41, and 47 biological pathways. For 3&nbsp;h vs. 0&nbsp;h (Fig.&nbsp;<span class="InternalRef"><a href="#Fig6">6</a></span>), the largest categories were carbon metabolism (3, 13.64%), carbon fixation pathways in prokaryotes (3, 13.64%), terpenoid backbone biosynthesis (3, 13.64%), pyruvate metabolism (3, 13.64%), and alpha-linolenic acid metabolism (3, 13.64%). For 6&nbsp;h vs. 0&nbsp;h, as shown in Fig.&nbsp;<span class="InternalRef"><a href="#Fig6">6</a></span>, the top five most enriched pathway were carbon metabolism (37, 9.18%), starch and sucrose metabolism (23, 5.71%), plant-pathogen interaction (21, 5.21%), plant hormone signal transduction (20, 4.96%), and photosynthesis (20, 4.96%). In 47 assigned biological pathways of 12&nbsp;h vs. 0&nbsp;h comparison (Fig.&nbsp;<span class="InternalRef"><a href="#Fig6">6</a></span>), carbon metabolism pathway was also the most abundantly represented (77, 7.65%), followed by biosynthesis of amino acids (65, 6.46%), starch and sucrose metabolism (53, 5.27%), plant hormone signal transduction (46, 4.57%), and plant-pathogen interaction (41, 4.08%).<figure xmlns="" class="Figure" id="Fig6"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO6">
                        <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig6_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig6_HTML.gif" alt="Fig. 6" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig6_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig6_HTML.gif" alt="Fig. 6"></noscript></a>
                      </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 6</span>
                          <p xmlns="" class="SimplePara">KEGG enrichment analysis of the DEGs</p>
                        </div></figcaption></figure>
                           </div>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par47" class="Para">The results of KEGG pathways enrichment showed that metabolic pathway varied with different treatment durations. However, under alkaline stress treatment for 3&nbsp;h, 6&nbsp;h or 12&nbsp;h, the following metabolic pathways were consistently involved: carbon metabolism; carbon fixation in photosynthetic organisms; phenylpropanoid biosynthesis; glycine, serine and threonine metabolism; methane metabolism; pyruvate metabolism; glyoxylate and dicarboxylate metabolism; terpenoid backbone biosynthesis; alpha-Linolenic acid metabolism; tryptophan metabolism; limonene and pinene degradation. Among these, the carbon metabolism pathway included the most DEGs, suggesting that <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> requires a large amount of energy in response to alkaline stress and that alkaline stress may primarily influence gene transcription by altering the expression of carbon metabolism-related genes in <em xmlns="" class="EmphasisTypeItalic">P. triloba</em>. The pyruvate metabolism pathway involves many metabolic activities, such as glycolysis, the tricarboxylic acid cycle, and fatty acid metabolism. When the plant is exposed to a harsh external environment, the pyruvate metabolism pathway can first effectively adjust and balance the metabolic processes in vivo, then activate protective mechanism, participate in scavenging reactive oxygen species (ROS), and protect the cell membrane system, thus allowing the plant to adapt to an adverse external environment [<span class="CitationRef"><a href="#CR35" data-toggle="citation">35</a></span>]. The glyoxylate and dicarboxylate metabolism pathway can balance metabolic disorders and transport energy in plants to enhance stress resistance [<span class="CitationRef"><a href="#CR35" data-toggle="citation">35</a></span>]. The alpha-linolenic acid metabolism pathway produces a series of metabolites, the most important of which are DHA (docosahexaenoic acid) and EPA (eicosapentaenoic acid), as well as other polyunsaturated fatty acids. DHA and EPA increase lipid membrane fluidity and permeability and increase cells viability [<span class="CitationRef"><a href="#CR36" data-toggle="citation">36</a></span>]. The tryptophan metabolism pathway produces secondary metabolites when the plant is subjected to abiotic stress, enhancing stress tolerance according to previous abiotic stress studies [<span class="CitationRef"><a href="#CR37" data-toggle="citation">37</a></span>–<span class="CitationRef"><a href="#CR39" data-toggle="citation">39</a></span>]. Therefore, the enrichment of DEGs in these metabolic pathways indicates that genes involved in early mechanisms of protection, defense, and stress tolerance were differentially expressed under alkaline stress treatment of 3&nbsp;h. With increasing duration of alkaline stress, the number of related response genes increased, indicating that alkaline resistance in <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> is a complex train under the control of multiple genes. Some of the metabolic pathways only included DEGs after alkaline stress for a certain length of time. Under alkaline stress treatment for 3&nbsp;h, one gene (c58505_g1_i1) was found to be involved in the calcium signaling pathway, and one gene (c89089_g2_i1) was involved in the ABC transporter pathway. The calcium signaling pathway is an important control pathway of physiological and biochemical reactions in vivo, and it can indirectly affect plant physiological activities (such as stomatal closure, the development of pollen tubes, root elongation, etc.), as confirmed by previous research [<span class="CitationRef"><a href="#CR40" data-toggle="citation">40</a></span>]. The calcium signaling pathway was specifically identified after 3&nbsp;h of alkaline stress treatment, when it may help <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> better cope with alkaline stress by changing intracellular calcium ion concentrations, leading to stomatal closure to avoid ion toxicity. ABC (ATP-binding cassette) transporters constitute the largest protein family with the greatest variety of functions, which may involve the transmembrane transport of plant secondary metabolites, protecting plants from environmental stresses [<span class="CitationRef"><a href="#CR41" data-toggle="citation">41</a></span>]. The genes involved in these early specific pathways may play important roles in early alkaline resistance mechanisms and may generate a cascade of responses to subsequent alkaline stress. Under alkaline stress treatment for 6 and 12&nbsp;h, more DEGs were involved in special metabolic pathways, including starch and sucrose metabolism, plant hormone signal transduction, glycolysis/gluconeogenesis, photosynthesis, carotenoid biosynthesis, and photosynthesis-antenna proteins. The above results showed that these genes that involved in metabolic pathways expressed with various patterns as the alkaline stress time increasing, and there was a rapid response of multiple molecular mechanisms allowing for the perception, transduction, and response to alkaline stress in <em xmlns="" class="EmphasisTypeItalic">P. triloba</em>.</p>
                </section>
                <section id="Sec18" class="Section3 RenderAsSection3">
                  <h4 class="Heading">qRT-PCR validation of alkaline-related genes</h4>
                  <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par48" class="Para">The reliability of the expression patterns of alkaline-responsive DEGs detected by RNA-Seq were confirmed by qRT-PCR analysis with gene-specific primers. A subset of seven genes (c78498_g1_i1, c80260_g1_i1, c81662_g1_i1, c84090_g1_i1, c85502_g2_i1, c85846_g1_i3, and c88652_g1_i2) was selected, and the actin gene was used as an internal reference gene for data normalization. RNA-Seq analysis revealed that these seven genes were all up-regulated (Fig.&nbsp;<span class="InternalRef"><a href="#Fig7">7</a></span>), and most of expression patterns were consistent with those determined by qRT-PCR. Some differences in the expression patterns and degree of up-regulation of the alkaline-responsive DEGs were observed between RNA-Seq and qRT-PCR, which might be due to the different algorithms and principles used by the RNA-Seq and qRT-PCR methods [<span class="CitationRef"><a href="#CR42" data-toggle="citation">42</a></span>, <span class="CitationRef"><a href="#CR43" data-toggle="citation">43</a></span>]. It is thought that it is normal to find a few differences between these two methods, and our results are of a similar nature to those reported in Yates [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>] and Huang (2015) [<span class="CitationRef"><a href="#CR44" data-toggle="citation">44</a></span>]. They found that some genes detected by RNA-Seq also did not show similar expression patterns when compared to qRT-PCR. Moreover, all seven genes were significantly and differentially expressed at all three durations of alkaline stress (<em xmlns="" class="EmphasisTypeItalic">P</em>&lt;0.05), suggesting that they were involved in regulatory networks that were active during these alkaline stress stages. Taken together, similar patterns were obtained from the qRT-PCR and RNA-Seq experiments, confirming the reliability and accuracy of our sequencing methods. And for important alkaline-responsive DEGs, we will use the Northern Blot to re-validate them in the future research.<figure xmlns="" class="Figure" id="Fig7"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO7">
                        <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig7_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig7_HTML.gif" alt="Fig. 7" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig7_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig7_HTML.gif" alt="Fig. 7"></noscript></a>
                      </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 7</span>
                          <p xmlns="" class="SimplePara">Expression patterns of seven genes selected from alkalinity-responsive DEGs of <em class="EmphasisTypeItalic">Prunus triloba</em> by RNA-Seq and qRT-PCR. <strong class="EmphasisTypeBold">a</strong> Log2 fold change of expression was detected by RNA-Seq. <strong class="EmphasisTypeBold">b</strong> The relative gene expression was calculated using the 2<sup xmlns="http://www.w3.org/1999/xhtml">-<span class="stix">△</span><span class="stix">△</span>Ct</sup> method by qRT-PCR. Mean values within the same gene with different letters are significantly at <em class="EmphasisTypeItalic">P</em>&lt;0.01</p>
                        </div></figcaption></figure>
                           </div>
                </section>
              </section><section id="Sec19" class="Section2 RenderAsSection2">
                <h3 class="Heading">Physiological analysis of <em class="EmphasisTypeItalic">P. triloba</em> in response to long-term alkaline stress</h3>
                <section id="Sec20" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Effects of long-term alkaline stress on the leaf microstructure of <em class="EmphasisTypeItalic">P. triloba</em>
                           </h4>
                  <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par49" class="Para">The leaf is the main organ for plant photosynthesis and transpiration, and its structural characteristics best reflect the adaptation of plants to the environment [<span class="CitationRef"><a href="#CR45" data-toggle="citation">45</a></span>]. As shown in Fig.&nbsp;<span class="InternalRef"><a href="#Fig8">8</a></span>, the internal microstructures of the leaves of <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> changed under alkaline stress. Compared with the control group, the main changes in the <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> leaf structure that occurred under alkaline stress can summarized as follows: 1) the stomatal density of the lower epidermis greatly increased, the stomatal volume became smaller, and the stomata almost completely closed, in contrast to the control group whose stomata were semi-closed (Fig.&nbsp;<span class="InternalRef"><a href="#Fig8">8a</a></span>–<span class="InternalRef"><a href="#Fig8">b</a></span>); 2) the upper and lower epidermal cells became slightly longer and narrower, and their arrangement became more irregular (Fig.&nbsp;<span class="InternalRef"><a href="#Fig8">8c</a></span>–<span class="InternalRef"><a href="#Fig8">d</a></span>); 3) the thickness of the upper and lower epidermal cuticle increased (Fig.&nbsp;<span class="InternalRef"><a href="#Fig8">8e</a></span>–<span class="InternalRef"><a href="#Fig8">f</a></span>); and 4) The leaf thickness increased, the mesophyll structure was more compact, the leaf chamber space was significantly reduced, the palisade tissue cells became slender, the spongy tissue cells became smaller, and the thickness ratio of palisade tissue to spongy tissue increased (Fig.&nbsp;<span class="InternalRef"><a href="#Fig8">8g</a></span>–<span class="InternalRef"><a href="#Fig8">h</a></span>).<figure xmlns="" class="Figure" id="Fig8"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO8">
                        <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig8_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig8_HTML.gif" alt="Fig. 8" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig8_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig8_HTML.gif" alt="Fig. 8"></noscript></a>
                      </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 8</span>
                          <p xmlns="" class="SimplePara">Effects of control and alkaline stress treatments on leaf microstructure of <em class="EmphasisTypeItalic">Prunus triloba</em>. <strong class="EmphasisTypeBold">a</strong>, <strong class="EmphasisTypeBold">c</strong>, <strong class="EmphasisTypeBold">e</strong>, <strong class="EmphasisTypeBold">g</strong> represent the leaf stomata, upper epidermis cell, cuticle, and vertical section in the control group, repectively. <strong class="EmphasisTypeBold">b</strong>, <strong class="EmphasisTypeBold">d</strong>, <strong class="EmphasisTypeBold">f</strong>, <strong class="EmphasisTypeBold">h</strong> represent the leaf stomata, upper epidermis cell, cuticle, and vertical section in the alkaline stress group, repectively</p>
                        </div></figcaption></figure>
                           </div>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par50" class="Para">Alkaline stress can directly affect the water metabolism of plants. It is accepted that the larger area of the epidermal cells have a better water storage, which has an important significance in enhancing the regulation of water. The epidermal cuticle can prevent excessive water transpiration inside the plant and it has an important role in the mechanical support of epidermal cells; therefore, when the plant water supply is insufficient, it keeps plant tissues from withering rapidly [<span class="CitationRef"><a href="#CR46" data-toggle="citation">46</a></span>]. The highly developed palisade tissue not only helps mesophyll cells avoid scorching from strong sunlight but also effectively uses diffracted light for photosynthesis. Thicker, smaller, and more densely arranged palisade tissue cells result in a higher energy utilization efficiency in plants [<span class="CitationRef"><a href="#CR46" data-toggle="citation">46</a></span>, <span class="CitationRef"><a href="#CR47" data-toggle="citation">47</a></span>]. Therefore, in this study, the leaf structural changes observed in <em xmlns="" class="EmphasisTypeItalic">P. triloba</em>, including enlargement of the leaf epidermal cell area, thickening of the epidermal cuticle and palisade tissue, and transition to a denser arrangement of palisade tissue, reflect the plant’s responses to long-term alkaline stress.</p>
                </section>
                <section id="Sec21" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Effects of long-term alkaline stress on <em class="EmphasisTypeItalic">P. triloba</em> seedling physiological indices</h4>
                  <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par51" class="Para">It is widely accepted that under alkaline stress, plants not only bear the same osmotic stress and ion toxicity as those under salt stress, but they also must resist damage due to high pH. In this study, <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> seedlings were continuously treated with 50&nbsp;mmol/L alkaline solution for 0, 3, 6, 9, or 12&nbsp;days. Leaf samples collected under alkaline stress of different durations were used to measure various physiological indices in order to explore the physiological adaptation mechanisms of early <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> seedlings under alkaline stress. The RWC is the most direct reflection of the degree of injury to seedlings under alkaline stress. The RWC of leaves declined gradually from 82.11 to 51.24% with increasing duration of alkaline stress, indicating that the cell was becoming more and more dehydrated and damage was increasing as alkaline stress continued (Fig.&nbsp;<span class="InternalRef"><a href="#Fig9">9a</a></span>).<figure xmlns="" class="Figure" id="Fig9"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO9">
                        <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig9_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig9_HTML.gif" alt="Fig. 9" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig9_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig9_HTML.gif" alt="Fig. 9"></noscript></a>
                      </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 9</span>
                          <p xmlns="" class="SimplePara">Effects of alkaline stress on seedling physiological indices of <em class="EmphasisTypeItalic">Prunus triloba</em>. The x-axis represents the time (d) under alkaline stress, the y-axis are the various number of physiological indices, of which (<strong class="EmphasisTypeBold">a</strong>) The relative water content (RWC, %); <strong class="EmphasisTypeBold">b</strong> Soluble protein content (SPC, mg/g FW); <strong class="EmphasisTypeBold">c</strong> Soluble sugar content (SSC, mg/g); <strong class="EmphasisTypeBold">d</strong> Procontnet content (Pro, μg/g); <strong class="EmphasisTypeBold">e</strong> The activity of peroxidase (POD, μ/g FW. min); <strong class="EmphasisTypeBold">f</strong> The activity of catalase (CAT, μ/g FW); <strong class="EmphasisTypeBold">g</strong> The activity of superoxide dismutase (SOD, μ/g FW); <strong class="EmphasisTypeBold">h</strong> Relative electric conductivity (REC, %); <strong class="EmphasisTypeBold">i</strong> Malonaldehyde content (MDA, μm/g FW); <strong class="EmphasisTypeBold">j</strong> Total chlorophyll content (Chl, mg/g FW)</p>
                        </div></figcaption></figure>
                           </div>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par52" class="Para">Osmotic stress in plants is due to loss of water or difficulty in absorbing water, so the osmotic regulation of plant cells is a basic adaptation that improves stress tolerance [<span class="CitationRef"><a href="#CR48" data-toggle="citation">48</a></span>]. Osmotic protective substances produced in the cytoplasm are induced by saline-alkaline stress and consist mainly of low-molecular-weight cell-compatible organic substances (i.e. soluble sugars, soluble proteins, and proline); these maintain the proper osmotic potential of the cell, protecting it from dehydration and stabilizing the structure and function of biological macromolecules in cells [<span class="CitationRef"><a href="#CR49" data-toggle="citation">49</a></span>]. In this study, the SSC, SPC, and Pro content increased with increasing stress duration (Fig.&nbsp;<span class="InternalRef"><a href="#Fig9">9b</a></span>–<span class="InternalRef"><a href="#Fig9">d</a></span>).</p>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par53" class="Para">After osmotic stress and ion toxicity, further damage in plants is caused by saline-alkaline stress due to excessive accumulation of ROS, resulting in lipid peroxidation or even cell death [<span class="CitationRef"><a href="#CR50" data-toggle="citation">50</a></span>]. Our results suggested that 50&nbsp;mmol/L alkaline solution induced the accumulation of ROS and the oxidative damage of membrane lipids in <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> seedlings and that the extent of the damage increased with an increasing duration of alkaline stress. The antioxidant systems of plants evolved to play an important role in defending against damage caused by ROS [<span class="CitationRef"><a href="#CR51" data-toggle="citation">51</a></span>]. The protective enzymes in the enzyme-promoting defense system, including SOD, POD, and CAT, are particularly important. As shown in Fig.&nbsp;<span class="InternalRef"><a href="#Fig9">9e</a></span>–<span class="InternalRef"><a href="#Fig9">g</a></span>, the activities of SOD, POD, and CAT in the leaves of <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> seedlings all increased with an increasing duration of alkaline stress. The biggest increase in activity was exhibited by POD, followed by SOD and then CAT, suggesting that POD played the primary role in resisting oxidative damage induced by alkaline stress.</p>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par54" class="Para">The protoplasmic membrane is sensitive to the stress reaction, and an increase in cell membrane permeability is closely related to an adverse external environment [<span class="CitationRef"><a href="#CR52" data-toggle="citation">52</a></span>]. The REC is an indicator of cell membrane permeability, and it can reflect the degree of cell membrane damage [<span class="CitationRef"><a href="#CR53" data-toggle="citation">53</a></span>]. Moreover, MDA is produced during cell membrane lipid peroxidation. The MDA content can therefore reflect the degree of oxidative damage in an organism [<span class="CitationRef"><a href="#CR54" data-toggle="citation">54</a></span>]. Our results showed that as the duration of alkaline stress increased, leaf REC and MDA content both increased gradually (Fig.&nbsp;<span class="InternalRef"><a href="#Fig9">9h</a></span>–<span class="InternalRef"><a href="#Fig9">i</a></span>), indicating that plant damage was becoming more severe. MDA not only has a cytotoxic effect by damaging cell membrane structure, but it can also degrade chlorophyll, thereby reducing plant photosynthesis [<span class="CitationRef"><a href="#CR55" data-toggle="citation">55</a></span>]. The Chl content in this study decreased gradually, from 5.492 to 2.304&nbsp;mg/g, as the duration of alkaline stress increased (Fig.&nbsp;<span class="InternalRef"><a href="#Fig9">9j</a></span>). This may be due to the increasing MDA content, which may have inhibited the synthesis of chlorophyll.</p>
                </section>
                <section id="Sec22" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Effects of long-term alkaline stress on <em class="EmphasisTypeItalic">P. triloba</em> seedling photosynthetic parameters</h4>
                  <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par55" class="Para">The chlorophyll content is an important parameter that reflects the photosynthetic function of leaves. As mentioned above, the Chl content declined gradually with increasing stress duration. In addition, other photosynthetic parameters declined after 12&nbsp;days of alkaline stress, including Pn, Gs, Ci, and Tr. Among these parameters, Gs is closely related to the photosynthetic rate of plants. In most cases, a decline in Gs leads to a decrease in CO<sub>2</sub> supply, causing a reduction in the photosynthetic rate [<span class="CitationRef"><a href="#CR56" data-toggle="citation">56</a></span>]. Our results showed that the Pn rate and Gs of leaves of experimental and control seedlings presented similar bimodal changes as the duration of stress increased (Fig.&nbsp;<span class="InternalRef"><a href="#Fig10">10a</a></span>–<span class="InternalRef"><a href="#Fig10">b</a></span>), demonstrating that the change in the Pn rate was largely determined by stomatal elements.<figure xmlns="" class="Figure" id="Fig10"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO10">
                        <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig10_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig10_HTML.gif" alt="Fig. 10" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig10_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_Fig10_HTML.gif" alt="Fig. 10"></noscript></a>
                      </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 10</span>
                          <p xmlns="" class="SimplePara">Effects of alkaline stress on seedling photosynthetic characteristics of <em class="EmphasisTypeItalic">Prunus triloba</em>. The x-axis represents the measured time (h), the y-axis are the photosynthetic parameters content of the control and treatment materials, of which (<strong class="EmphasisTypeBold">a</strong>) Net photosynthetic rate (Pn, μmol· m<sup xmlns="http://www.w3.org/1999/xhtml">−2</sup>· s<sup xmlns="http://www.w3.org/1999/xhtml">−1</sup>); <strong class="EmphasisTypeBold">b</strong> Stomatal conductance (Gs, mmol· m<sup xmlns="http://www.w3.org/1999/xhtml">−2</sup>· s<sup xmlns="http://www.w3.org/1999/xhtml">−1</sup>); <strong class="EmphasisTypeBold">c</strong> Intercellular CO<sub xmlns="http://www.w3.org/1999/xhtml">2</sub> concentration (Ci, μL· L<sup xmlns="http://www.w3.org/1999/xhtml">−1</sup>); <strong class="EmphasisTypeBold">d</strong> Transpiration rate (Tr, mmol· m<sup xmlns="http://www.w3.org/1999/xhtml">−2</sup>· s<sup xmlns="http://www.w3.org/1999/xhtml">−1</sup>); <strong class="EmphasisTypeBold">e</strong> Water use efficiency (WUE, Pn/Tr). The red smooth curve in the picture represents the control material, whereas the blue smooth curve represents the treatment material</p>
                        </div></figcaption></figure>
                           </div>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par56" class="Para">As shown in Fig.&nbsp;<span class="InternalRef"><a href="#Fig10">10d</a></span>, the leaf Tr of the alkaline-stressed and control seedlings first increased and then decreased, peaking at 11:00 and 13:00, respectively, and exhibiting similar but also different trends compared with that of Pn rate and Gs. The change in Ci under alkaline stress was similar to that in the control group, which is inconsistent with the corresponding trend of the Pn rate (Fig.&nbsp;<span class="InternalRef"><a href="#Fig10">10c</a></span>). At some time points, the Ci and Pn rate showed similar trends (both increasing or both decreasing), while at other time points, the Ci and Pn rate exhibited opposite trends, suggesting that the decrease in the Pn rate was not only due to a stomatal limitation but was also restricted by non-stomatal factors. WUE is a comprehensive physiological and ecological index that reflects crop energy conversion efficiency in crop production and assesses crop growth suitability. The WUE at the single leaf level is often expressed as the ratio of the Pn rate to the Tr [<span class="CitationRef"><a href="#CR57" data-toggle="citation">57</a></span>]. The trend in the WUE of the leaves of <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> seedlings was consistent with that of the Pn rate both in the control and alkaline stress treatments (Fig.&nbsp;<span class="InternalRef"><a href="#Fig10">10e</a></span>). However, the alkaline stress treatment increased the WUE in <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> leaves, a phenomenon that needed to be explained by further research.</p>
                </section>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec23" class="Section1 RenderAsSection1 SectionTypeConclusion c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Conclusion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Conclusion">Conclusion</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Conclusion" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par57" class="Para">In this study, we conducted a comprehensive investigation of alkaline acclimation mechanisms in <em xmlns="" class="EmphasisTypeItalic">P. triloba.</em> First, <em xmlns="" class="EmphasisTypeItalic">de novo</em> transcriptome analysis of <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> was conducted in response to short-term alkaline stress using the Illumina sequencing platform. We identified a large number of DEGs that may be used for molecular breeding of <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> in the future. GO annotation analysis revealed 28 DEGs that may play an important role in the early alkaline stress response, and KEGG pathway analysis revealed a variety of pathways that may be important at different time points. In addition, the expression patterns of seven alkaline-related genes were validated with qRT-PCR, confirming the reliability of the RNA-Seq results. Second, a physiological analysis of <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> in response to long-term alkaline stress was also conducted. Changes in the internal microstructure of the leaves, physiological indices, and photosynthetic parameters of <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> under alkaline stress were observed, allowing <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> to adapt to long-term alkaline stress. Our study provides information on the short-term molecular mechanisms and long-term physiological mechanisms of alkaline tolerance in <em xmlns="" class="EmphasisTypeItalic">P. triloba</em> that can be used to study this plant as well as other fruit trees in the future.</p></div></section>
          <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AbbreviationGroup Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Abbreviations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Abbreviations">Abbreviations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Abbreviations"><div class="DefinitionList">
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>CAT:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par4" class="Para">Catalase</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>Chl:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par5" class="Para">Chlorophyll content</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>Ci:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par6" class="Para">Internal CO<sub>2</sub> concentration</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>DEGs:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par7" class="Para">Differentially expressed genes</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>FDR:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par8" class="Para">False discovery rate</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>GO:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par9" class="Para">Gene ontology</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>Gs:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par10" class="Para">Stomatal conductance</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>KEGG:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par11" class="Para">Kyoto encyclopedia of genes and genomes</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>MDA:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par12" class="Para">Malondialdehyde content</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>Pn:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par13" class="Para">Net photosynthetic rate</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>POD:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par14" class="Para">Peroxidase</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>Pro:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par15" class="Para">Proline content</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>qRT-PCR:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par16" class="Para">Quantitative real-time reverse-transcriptase PCR</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>REC:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par17" class="Para">Relative electrical conductivity</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>RWC:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par18" class="Para">Relative water content</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>SNP:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par19" class="Para">Single nucleotide polymorphism</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>SOD:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par20" class="Para">Superoxide dismutase</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>SPC:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par21" class="Para">Soluble protein content</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>SSC:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par22" class="Para">Soluble sugar content</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>SSR:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par23" class="Para">Simple sequence repeat</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>Tr:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par24" class="Para">Transpiration rate</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>WUE:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par25" class="Para">Water use efficiency</p>
                  </dd></dl>
              </div></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments"><h3 xmlns="" class="Heading">Acknowledgments</h3><p xmlns="" class="SimplePara">We thank all the participants for their contributions to this study. We also wish to express our warm thanks to danfeng (Chengdu) Technology Co., Ltd. Their ideas and help gave a valuable added dimension to our research.</p><div id="FPar1" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Funding</h3>
                <p id="Par58" class="Para">This research was financial supported by Chengdu Agricultural Technology Research and Development Project (2015-NY02-00026-NC) and Sichuan Fiscal Innovation Capacity Improvement Project Special Funds (2015QNJJ-010). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p>
              </div><div id="FPar2" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Availability of data and materials</h3>
                <p id="Par59" class="Para">The raw data that support the findings of this study are deposited in the NCBI SRA database under accession number SRP098782.</p>
              </div><div id="FPar3" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Authors’ contributions</h3>
                <p id="Par60" class="Para">Y-QW and JL conceived and designed the experiments; JL and Q-TL performed the experiments and analyzed the data; JL wrote the paper. All authors discussed the results and commented on the manuscript. All authors read and approved the final manuscript.</p>
              </div><div id="FPar4" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Competing interests</h3>
                <p id="Par61" class="Para">The authors declare that they have no competing interests.</p>
              </div><div id="FPar5" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Consent for publication</h3>
                <p id="Par62" class="Para">Not applicable</p>
              </div><div id="FPar6" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Ethics approval and consent to participate</h3>
                <p id="Par63" class="Para">This study was approved by the College of Horticulture, Sichuan Agricultural University; Horticulture Research Institute, Sichuan Academy of Agricultural Sciences; and Southwestern Key laboratory of Horticultural Crops Biology and Germplasm Enhancement, Ministry of Agriculture. No specific permissions were required for the collection of <em xmlns="" class="EmphasisTypeItalic">Prunus triloba</em> samples at locations in southwest China because the research was funded by Chengdu Agricultural Technology Research and Development Project (2015-NY02-00026-NC) and Sichuan Fiscal Innovation Capacity Improvement Project Special Funds (2015QNJJ-010) of the People’s Republic of China. In addition, no specific permission was required for the field study, which was conducted on the experimental farm at Sichuan Agricultural University. <em xmlns="" class="EmphasisTypeItalic">Prunus triloba</em> is not an endangered or protected species.</p>
              </div><div id="FPar7" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Publisher’s Note</h3>
                <p id="Par64" class="Para">Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
              </div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="License">
                <p xmlns="" class="SimplePara">
                           <strong class="EmphasisTypeBold">Open Access</strong>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/4.0/"><span class="RefSource">http://creativecommons.org/licenses/by/4.0/</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/publicdomain/zero/1.0/"><span class="RefSource">http://creativecommons.org/publicdomain/zero/1.0/</span></a></span>) applies to the data made available in this article, unless otherwise stated.</p>
              </div><aside class="Appendix" id="App1">
              
                <h3 class="Heading">Additional files</h3><div id="widgetContainer" data-test="figshare-container"></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par65" class="Para">
                  <div class="MediaObject" id="MOESM1">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_MOESM1_ESM.doc"><span class="caption-number">Additional file 1:</span></a>
                          <span xmlns="" class="SimplePara">Gene number of each sample in different interval of expression level. (DOC 37&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM2">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0031-7/MediaObjects/41065_2017_31_MOESM2_ESM.jpg"><span class="caption-number">Additional file 2:</span></a>
                          <span xmlns="" class="SimplePara">The comparison distribution diagram of FPKM of all genes in the 10 samples. (JPG 91&nbsp;kb)</span>
                        </div></div>
                  </div>
                </div>
              
            </aside></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authors’ Affiliations</a></h2><div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1"><span class="AffiliationNumber">(1)</span><div class="AffiliationText">College of Horticulture, Sichuan Agricultural University, Chengdu, Sichuan, 611130, People’s Republic of China</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff2"><span class="AffiliationNumber">(2)</span><div class="AffiliationText">Horticulture Research Institute, Sichuan Academy of Agricultural Sciences, Chengdu, Sichuan, 610066, People’s Republic of China</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff3"><span class="AffiliationNumber">(3)</span><div class="AffiliationText">Southwestern Key laboratory of Horticultural Crops Biology and Germplasm Enhancement, Ministry of Agriculture, Chengdu, Sichuan, 610066, People’s Republic of China</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff4"><span class="AffiliationNumber">(4)</span><div class="AffiliationText">State Key Lab of Plant Genomics, Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Beijing, 100101, People’s Republic of China</div></div></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper"><li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Cai JY, Sun Z, Li WQ, Huang ZL. Study on the selection of superior rootstocks for plum tress. Chin Hortic Abstr. 2013;8:38–40.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Study%20on%20the%20selection%20of%20superior%20rootstocks%20for%20plum%20tress&amp;author=JY.%20Cai&amp;author=Z.%20Sun&amp;author=WQ.%20Li&amp;author=ZL.%20Huang&amp;journal=Chin%20Hortic%20Abstr&amp;volume=8&amp;pages=38-40&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Brown JC. Iron chlorosis. Ann Rev Plant Physiol. 2003;7:171–90.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1146/annurev.pp.07.060156.001131"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Iron%20chlorosis&amp;author=JC.%20Brown&amp;journal=Ann%20Rev%20Plant%20Physiol&amp;volume=7&amp;pages=171-90&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Mengel K. Iron availability in plant tissues-iron chlorosis on calcareous soils. Plant Soil. 1994;165:275–83.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/BF00008070"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Iron%20availability%20in%20plant%20tissues-iron%20chlorosis%20on%20calcareous%20soils&amp;author=K.%20Mengel&amp;journal=Plant%20Soil&amp;volume=165&amp;pages=275-83&amp;publication_year=1994"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yu ZR, Wang JW. Land salinization in China and the prevention countermeasures. Rural Ecol Environ. 1997;13:1–5.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Land%20salinization%20in%20China%20and%20the%20prevention%20countermeasures&amp;author=ZR.%20Yu&amp;author=JW.%20Wang&amp;journal=Rural%20Ecol%20Environ&amp;volume=13&amp;pages=1-5&amp;publication_year=1997"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gao H. Comparative structure study on four pecies of Rosaceae plants living in different ecosystem in Master’s Thesis. Changchun, China: Northeast Normal University; 2005 (In Chinese).</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparative%20structure%20study%20on%20four%20pecies%20of%20Rosaceae%20plants%20living%20in%20different%20ecosystem%20in%20Master%E2%80%99s%20Thesis&amp;author=H.%20Gao&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Chen JY. “Dual classification”: a new classification system for Chinese flower cultivars. J Beijing For Univ. 1998;20:1–6.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=%E2%80%9CDual%20classification%E2%80%9D%3A%20a%20new%20classification%20system%20for%20Chinese%20flower%20cultivars&amp;author=JY.%20Chen&amp;journal=J%20Beijing%20For%20Univ&amp;volume=20&amp;pages=1-6&amp;publication_year=1998"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhao LL. Bibliometrics analysis of research progress in <em class="EmphasisTypeItalic">Prunus triloba</em> nearly thirty years in China. North Hortic. 2014;20:202–4.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Bibliometrics%20analysis%20of%20research%20progress%20in%20Prunus%20triloba%20nearly%20thirty%20years%20in%20China&amp;author=LL.%20Zhao&amp;journal=North%20Hortic&amp;volume=20&amp;pages=202-4&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhang QY, Zhang QX. Genetic diversity of wild populations of <em class="EmphasisTypeItalic">Prunus triloba</em> revealed by AFLP. Mol Plant Breed. 2012;10:452–6.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20of%20wild%20populations%20of%20Prunus%20triloba%20revealed%20by%20AFLP&amp;author=QY.%20Zhang&amp;author=QX.%20Zhang&amp;journal=Mol%20Plant%20Breed&amp;volume=10&amp;pages=452-6&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Su XH, Qin W, Liu LQ, Chen F, Zhou L. Effects of low temperature stress on some cold resistance indexes of four kinds of root stock <em class="EmphasisTypeItalic">Prunus</em>. J Xinjiang Agr Univ. 2012;35:112–5.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Effects%20of%20low%20temperature%20stress%20on%20some%20cold%20resistance%20indexes%20of%20four%20kinds%20of%20root%20stock%20Prunus%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=XH.%20Su&amp;author=W.%20Qin&amp;author=LQ.%20Liu&amp;author=F.%20Chen&amp;author=L.%20Zhou&amp;journal=J%20Xinjiang%20Agr%20Univ&amp;volume=35&amp;pages=112-5&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li HJ, Zhao HD, Cui SM, Ding BY. Research of drought resistance of flowering plum seedings. Chin Hortic Abstr. 2011;8:1–4.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Research%20of%20drought%20resistance%20of%20flowering%20plum%20seedings&amp;author=HJ.%20Li&amp;author=HD.%20Zhao&amp;author=SM.%20Cui&amp;author=BY.%20Ding&amp;journal=Chin%20Hortic%20Abstr&amp;volume=8&amp;pages=1-4&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Fan XD, Wang JQ, Yang N, Dong YY, Liu L, Wang FW, et al. Gene expression profiling of soybean leaves and roots under salt, saline–alkali and drought stress by high-throughput Illumina sequencing. Gene. 2013;512:392–402.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.gene.2012.09.100"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23063936"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Gene%20expression%20profiling%20of%20soybean%20leaves%20and%20roots%20under%20salt%2C%20saline%E2%80%93alkali%20and%20drought%20stress%20by%20high-throughput%20Illumina%20sequencing&amp;author=XD.%20Fan&amp;author=JQ.%20Wang&amp;author=N.%20Yang&amp;author=YY.%20Dong&amp;author=L.%20Liu&amp;author=FW.%20Wang&amp;journal=Gene&amp;volume=512&amp;pages=392-402&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wallender WW, Tanji KK. Agricultural salinity assessment and management. New York: American Society of Civil Engineers (ASCE) Publications; 2011.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1061/9780784411698"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Agricultural%20salinity%20assessment%20and%20management&amp;author=WW.%20Wallender&amp;author=KK.%20Tanji&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Shi DC, Yin LJ. Difference between salt (NaCl) and alkaline (Na<sub xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml">2</sub>CO<sub xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml">3</sub>) stresses on <em class="EmphasisTypeItalic">Puccinellia tenuiflora</em> (Griseb.) Scribn. et Merr. plants. Acta Bot Sin. 1993;35:144–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Difference%20between%20salt%20%28NaCl%29%20and%20alkaline%20%28Na2CO3%29%20stresses%20on%20Puccinellia%20tenuiflora%20%28Griseb.%29%20Scribn.%20et%20Merr.%20plants&amp;author=DC.%20Shi&amp;author=LJ.%20Yin&amp;journal=Acta%20Bot%20Sin&amp;volume=35&amp;pages=144-9&amp;publication_year=1993"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Shi DC, Sheng YM. Effect of various salt–alkaline mixed stress conditions on sunflower seedlings and analysis of their stress factors. Environ Exp Bot. 2005;54:8–21.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.envexpbot.2004.05.003"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Effect%20of%20various%20salt%E2%80%93alkaline%20mixed%20stress%20conditions%20on%20sunflower%20seedlings%20and%20analysis%20of%20their%20stress%20factors&amp;author=DC.%20Shi&amp;author=YM.%20Sheng&amp;journal=Environ%20Exp%20Bot&amp;volume=54&amp;pages=8-21&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Munns R, Tester M. Mechanisms of salinity tolerance. Annu Rev Plant Biol. 2008;59:651–81.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1146/annurev.arplant.59.032607.092911"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18444910"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Mechanisms%20of%20salinity%20tolerance&amp;author=R.%20Munns&amp;author=M.%20Tester&amp;journal=Annu%20Rev%20Plant%20Biol&amp;volume=59&amp;pages=651-81&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sreenivasulu N, Sopory SK, Kishor PBK. Deciphering the regulatory mechanisms of abiotic stress tolerance in plants by genomic approaches. Gene. 2007;388:1–13.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.gene.2006.10.009"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17134853"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Deciphering%20the%20regulatory%20mechanisms%20of%20abiotic%20stress%20tolerance%20in%20plants%20by%20genomic%20approaches&amp;author=N.%20Sreenivasulu&amp;author=SK.%20Sopory&amp;author=PBK.%20Kishor&amp;journal=Gene&amp;volume=388&amp;pages=1-13&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Qi B, Yang Y, Yin Y, Xu M, Li H. <em class="EmphasisTypeItalic">De novo</em> sequencing, assembly, and analysis of the <em class="EmphasisTypeItalic">Taxodium</em> ‘Zhongshansa’ roots and shoots transcriptome in response to short-term waterlogging. BMC Plant Biol. 2014;14:1–12.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2229-14-1"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20De%20novo%20sequencing%2C%20assembly%2C%20and%20analysis%20of%20the%20Taxodium%20%E2%80%98Zhongshansa%E2%80%99%20roots%20and%20shoots%20transcriptome%20in%20response%20to%20short-term%20waterlogging&amp;author=B.%20Qi&amp;author=Y.%20Yang&amp;author=Y.%20Yin&amp;author=M.%20Xu&amp;author=H.%20Li&amp;journal=BMC%20Plant%20Biol&amp;volume=14&amp;pages=1-12&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yates SA, Swain MT, Hegarty MJ, Chernukin I, Lowe M, Allison GG, et al. <em class="EmphasisTypeItalic">De novo</em> assembly of red clover transcriptome based on RNA-Seq data provides insight into drought response, gene discovery and marker identification. BMC Genomics. 2014;15:453.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2164-15-453"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24912738"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4144119"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20De%20novo%20assembly%20of%20red%20clover%20transcriptome%20based%20on%20RNA-Seq%20data%20provides%20insight%20into%20drought%20response%2C%20gene%20discovery%20and%20marker%20identification&amp;author=SA.%20Yates&amp;author=MT.%20Swain&amp;author=MJ.%20Hegarty&amp;author=I.%20Chernukin&amp;author=M.%20Lowe&amp;author=GG.%20Allison&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=453&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wang YC, Yang CP, Liu GF, Zhang GD, Ban QY. Microarray and suppression subtractive hybridization analyses of gene expression in <em class="EmphasisTypeItalic">Puccinellia tenuiflora</em> after exposure to NaHCO<sub xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml">3</sub>. Plant Sci. 2007;173:309–20.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.plantsci.2007.06.011"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Microarray%20and%20suppression%20subtractive%20hybridization%20analyses%20of%20gene%20expression%20in%20Puccinellia%20tenuiflora%20after%20exposure%20to%20NaHCO3%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=YC.%20Wang&amp;author=CP.%20Yang&amp;author=GF.%20Liu&amp;author=GD.%20Zhang&amp;author=QY.%20Ban&amp;journal=Plant%20Sci&amp;volume=173&amp;pages=309-20&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Bolger AM, Lohse M, Usadel B. Trimmomatic: a flexible trimmer for Illumina sequence data. Bioinformatics. 2014;30:2114–20.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btu170"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24695404"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4103590"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Trimmomatic%3A%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data&amp;author=AM.%20Bolger&amp;author=M.%20Lohse&amp;author=B.%20Usadel&amp;journal=Bioinformatics&amp;volume=30&amp;pages=2114-20&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Grabherr MG, Haas BJ, Yassour M, Levin JZ, Thompson DA, Amit I, et al. Trinity: reconstructing a full-length transcriptome without a genome from RNA-Seq data. Nat Biotechnol. 2011;29:644–52.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nbt.1883"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21572440"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3571712"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Trinity%3A%20reconstructing%20a%20full-length%20transcriptome%20without%20a%20genome%20from%20RNA-Seq%20data&amp;author=MG.%20Grabherr&amp;author=BJ.%20Haas&amp;author=M.%20Yassour&amp;author=JZ.%20Levin&amp;author=DA.%20Thompson&amp;author=I.%20Amit&amp;journal=Nat%20Biotechnol&amp;volume=29&amp;pages=644-52&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Pertea G, Huang X, Liang F, Antonescu V, Sultana R, Karamycheva S, et al. TIGR Gene Indices clustering tools (TGICL): a software system for fast clustering of large EST datasets. Bioinformatics. 2003;19:651–2.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btg034"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12651724"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=TIGR%20Gene%20Indices%20clustering%20tools%20%28TGICL%29%3A%20a%20software%20system%20for%20fast%20clustering%20of%20large%20EST%20datasets&amp;author=G.%20Pertea&amp;author=X.%20Huang&amp;author=F.%20Liang&amp;author=V.%20Antonescu&amp;author=R.%20Sultana&amp;author=S.%20Karamycheva&amp;journal=Bioinformatics&amp;volume=19&amp;pages=651-2&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Haas BJ, Papanicolaou A, Yassour M, Grabherr M, Blood PD, Bowden J, et al. <em class="EmphasisTypeItalic">De novo</em> transcript sequence reconstruction from RNA-Seq: reference generation and analysis with Trinity. Nat Protoc. 2013;8:1494–512.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nprot.2013.084"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23845962"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20De%20novo%20transcript%20sequence%20reconstruction%20from%20RNA-Seq%3A%20reference%20generation%20and%20analysis%20with%20Trinity&amp;author=BJ.%20Haas&amp;author=A.%20Papanicolaou&amp;author=M.%20Yassour&amp;author=M.%20Grabherr&amp;author=PD.%20Blood&amp;author=J.%20Bowden&amp;journal=Nat%20Protoc&amp;volume=8&amp;pages=1494-512&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR24" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Langmead B, Salzberg SL. Fast gapped-read alignment with Bowtie 2. Nat Methods. 2012;9:357–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nmeth.1923"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22388286"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3322381"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Fast%20gapped-read%20alignment%20with%20Bowtie%202&amp;author=B.%20Langmead&amp;author=SL.%20Salzberg&amp;journal=Nat%20Methods&amp;volume=9&amp;pages=357-9&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR25" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li B, Dewey CN. RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome. BMC Bioinformatics. 2011;12:93–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2105-12-93"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=RSEM%3A%20accurate%20transcript%20quantification%20from%20RNA-Seq%20data%20with%20or%20without%20a%20reference%20genome&amp;author=B.%20Li&amp;author=CN.%20Dewey&amp;journal=BMC%20Bioinformatics&amp;volume=12&amp;pages=93-9&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR26" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Anders S, Huber W. Differential expression analysis for sequence count data. Genome Biol. 2010;11:1–12.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/gb-2010-11-10-r106"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Differential%20expression%20analysis%20for%20sequence%20count%20data&amp;author=S.%20Anders&amp;author=W.%20Huber&amp;journal=Genome%20Biol&amp;volume=11&amp;pages=1-12&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR27" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Young MD, Wakeeld MJ, Smyth GK, Oshlack A. GOseq: gene ontology testing for RNA-seq datasets. 2012.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=GOseq%3A%20gene%20ontology%20testing%20for%20RNA-seq%20datasets&amp;author=MD.%20Young&amp;author=MJ.%20Wakeeld&amp;author=GK.%20Smyth&amp;author=A.%20Oshlack&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR28" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Xie C, Mao XZ, Huang JJ, Ding Y, Wu JM, Dong S, et al. KOBAS 2.0: a web server for annotation and identification of enriched pathways and diseases. Nucleic Acids Res. 2011;39:W316–W22.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gkr483"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21715386"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3125809"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=KOBAS%202.0%3A%20a%20web%20server%20for%20annotation%20and%20identification%20of%20enriched%20pathways%20and%20diseases&amp;author=C.%20Xie&amp;author=XZ.%20Mao&amp;author=JJ.%20Huang&amp;author=Y.%20Ding&amp;author=JM.%20Wu&amp;author=S.%20Dong&amp;journal=Nucleic%20Acids%20Res&amp;volume=39&amp;pages=W316-W22&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR29" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sun Q, Hu JJ. Research technology of plant physiology. Master’s Thesis. Yangling, China: Northwest Agriculture and Forestry University Press; 2006 (In Chinese).</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Research%20technology%20of%20plant%20physiology.%20Master%E2%80%99s%20Thesis&amp;author=Q.%20Sun&amp;author=JJ.%20Hu&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR30" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Xue YL, Xia ZA. Plant physiology laboratory manual. Shanghai: Shanghai Science and Technique Publishing House; 1985.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Plant%20physiology%20laboratory%20manual&amp;author=YL.%20Xue&amp;author=ZA.%20Xia&amp;publication_year=1985"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR31" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhu GL, Zhong HW, Zhang AQ. Plant physiology experiment. Beijing: Peking University Press; 1990.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Plant%20physiology%20experiment&amp;author=GL.%20Zhu&amp;author=HW.%20Zhong&amp;author=AQ.%20Zhang&amp;publication_year=1990"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR32" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li HS. Principles and techniques of plant physiological and biochemical experiments. Beijing: Higher Education Press; 2006.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Principles%20and%20techniques%20of%20plant%20physiological%20and%20biochemical%20experiments&amp;author=HS.%20Li&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR33" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ge Y, Li Y, Zhu YM, Bai X, Lv DK, Guo DJ, et al. Global transcriptome profiling of wild soybean (<em class="EmphasisTypeItalic">Glycine soja</em>) roots under NaHCO<sub xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml">3</sub> treatment. BMC Plant Biol. 2010;10:1–14.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2229-10-153"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Global%20transcriptome%20profiling%20of%20wild%20soybean%20%28Glycine%20soja%29%20roots%20under%20NaHCO3%20treatment&amp;author=Y.%20Ge&amp;author=Y.%20Li&amp;author=YM.%20Zhu&amp;author=X.%20Bai&amp;author=DK.%20Lv&amp;author=DJ.%20Guo&amp;journal=BMC%20Plant%20Biol&amp;volume=10&amp;pages=1-14&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR34" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Seki M, Narusaka M, Ishida J, Nanjo T, Fujita M, Oono Y, et al. Monitoring the expression profiles of 7000 Arabidopsis genes under drought, cold and high-salinity stresses using a full-length cDNA microarray. Plant J. 2002;31:279–92.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1046/j.1365-313X.2002.01359.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12164808"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Monitoring%20the%20expression%20profiles%20of%207000%20Arabidopsis%20genes%20under%20drought%2C%20cold%20and%20high-salinity%20stresses%20using%20a%20full-length%20cDNA%20microarray&amp;author=M.%20Seki&amp;author=M.%20Narusaka&amp;author=J.%20Ishida&amp;author=T.%20Nanjo&amp;author=M.%20Fujita&amp;author=Y.%20Oono&amp;journal=Plant%20J&amp;volume=31&amp;pages=279-92&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR35" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wang DL, Ye WW, Wang JJ, Song LY, Fan WL, Cui YP. Construction of SSH library and its analyses of cotton drought associated genes under drought stress. Acta Agron Sin. 2010;36:2035–44.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/S1875-2780(09)60087-0"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Construction%20of%20SSH%20library%20and%20its%20analyses%20of%20cotton%20drought%20associated%20genes%20under%20drought%20stress&amp;author=DL.%20Wang&amp;author=WW.%20Ye&amp;author=JJ.%20Wang&amp;author=LY.%20Song&amp;author=WL.%20Fan&amp;author=YP.%20Cui&amp;journal=Acta%20Agron%20Sin&amp;volume=36&amp;pages=2035-44&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR36" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yang J, Chang R. Progress on α-linolenic acid. Agr Eng. 2011;1:72–6.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Progress%20on%20%CE%B1-linolenic%20acid&amp;author=J.%20Yang&amp;author=R.%20Chang&amp;journal=Agr%20Eng&amp;volume=1&amp;pages=72-6&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR37" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Witt S, Galicia L, Lisec J, Cairns J, Tiessen A, Araus JL, et al. Metabolic and phenotypic responses of greenhouse-grown maize hybrids to experimentally controlled drought stress. Mol Plant. 2012;5:401–17.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/mp/ssr102"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22180467"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Metabolic%20and%20phenotypic%20responses%20of%20greenhouse-grown%20maize%20hybrids%20to%20experimentally%20controlled%20drought%20stress&amp;author=S.%20Witt&amp;author=L.%20Galicia&amp;author=J.%20Lisec&amp;author=J.%20Cairns&amp;author=A.%20Tiessen&amp;author=JL.%20Araus&amp;journal=Mol%20Plant&amp;volume=5&amp;pages=401-17&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR38" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Suguiyama VF, Silva EA, Meirelles ST, Centeno DC, Braga MR. Leaf metabolite profile of the Brazilian resurrection plant <em class="EmphasisTypeItalic">Barbacenia purpurea</em> Hook. (Velloziaceae) shows two time-dependent responses during desiccation and recovering. Front Plant Sci. 2014;5:96.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.3389/fpls.2014.00096"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24672534"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3953666"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Leaf%20metabolite%20profile%20of%20the%20Brazilian%20resurrection%20plant%20Barbacenia%20purpurea%20Hook.%20%28Velloziaceae%29%20shows%20two%20time-dependent%20responses%20during%20desiccation%20and%20recovering&amp;author=VF.%20Suguiyama&amp;author=EA.%20Silva&amp;author=ST.%20Meirelles&amp;author=DC.%20Centeno&amp;author=MR.%20Braga&amp;journal=Front%20Plant%20Sci&amp;volume=5&amp;pages=96&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR39" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sun CX, Gao XX, Fu JQ, Zhou JH, Wu XF. Metabolic response of maize (<em class="EmphasisTypeItalic">Zea mays</em> L.) plants to combined drought and salt stress. Plant Soil. 2015;388:99–117.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11104-014-2309-0"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Metabolic%20response%20of%20maize%20%28Zea%20mays%20L.%29%20plants%20to%20combined%20drought%20and%20salt%20stress&amp;author=CX.%20Sun&amp;author=XX.%20Gao&amp;author=JQ.%20Fu&amp;author=JH.%20Zhou&amp;author=XF.%20Wu&amp;journal=Plant%20Soil&amp;volume=388&amp;pages=99-117&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR40" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Allen GJ, Chu SP, Harrington CL, Schumacher K, Hoffmann T, Tang YY, et al. A defined range of guard cell calcium oscillation parameters encodes stomatal movements. Nature. 2001;411:1053–7.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/35082575"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11429606"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20defined%20range%20of%20guard%20cell%20calcium%20oscillation%20parameters%20encodes%20stomatal%20movements&amp;author=GJ.%20Allen&amp;author=SP.%20Chu&amp;author=CL.%20Harrington&amp;author=K.%20Schumacher&amp;author=T.%20Hoffmann&amp;author=YY.%20Tang&amp;journal=Nature&amp;volume=411&amp;pages=1053-7&amp;publication_year=2001"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR41" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Jin HB, Liu DH, Zuo KJ, Miao ZQ, Chen YH, Sun XF, et al. Plant ABC transporters and their roles in the transmembrane transport of secondary metabolites. J Agr Sci Technol. 2007;9:32–7.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Plant%20ABC%20transporters%20and%20their%20roles%20in%20the%20transmembrane%20transport%20of%20secondary%20metabolites&amp;author=HB.%20Jin&amp;author=DH.%20Liu&amp;author=KJ.%20Zuo&amp;author=ZQ.%20Miao&amp;author=YH.%20Chen&amp;author=XF.%20Sun&amp;journal=J%20Agr%20Sci%20Technol&amp;volume=9&amp;pages=32-7&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR42" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ekman DR, Lorenz WW, Przybyla AE, Wolfe NL, Dean JF. SAGE analysis of transcriptome responses in Arabidopsis roots exposed to 2,4,6-trinitrotoluene. Plant Physiol. 2003;133:1397–406.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1104/pp.103.028019"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14551330"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC281634"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=SAGE%20analysis%20of%20transcriptome%20responses%20in%20Arabidopsis%20roots%20exposed%20to%202%2C4%2C6-trinitrotoluene&amp;author=DR.%20Ekman&amp;author=WW.%20Lorenz&amp;author=AE.%20Przybyla&amp;author=NL.%20Wolfe&amp;author=JF.%20Dean&amp;journal=Plant%20Physiol&amp;volume=133&amp;pages=1397-1406&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR43" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gao ZX, Wehner TC, Chen H, Lin Y, Wang XF, Wei M, et al. Deciphering the possible mechanism of exogenous NO alleviating alkali stress on cucumber leaves by transcriptomic analysis. Sci Hortic-Amsterdam. 2013;150:377–86.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.scienta.2012.11.033"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Deciphering%20the%20possible%20mechanism%20of%20exogenous%20NO%20alleviating%20alkali%20stress%20on%20cucumber%20leaves%20by%20transcriptomic%20analysis&amp;author=ZX.%20Gao&amp;author=TC.%20Wehner&amp;author=H.%20Chen&amp;author=Y.%20Lin&amp;author=XF.%20Wang&amp;author=M.%20Wei&amp;journal=Sci%20Hortic-Amsterdam&amp;volume=150&amp;pages=377-86&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR44" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Huang LK, Yan HD, Zhao XX, Zhang XQ, Wang J, Frazier T, et al. Identifying differentially expressed genes under heat stress and developing molecular markers in orchardgrass (Dactylis glomerata L.) through transcriptome analysis. Mol Ecol Resour. 2015;15:1497–509.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Huang%20LK%2C%20Yan%20HD%2C%20Zhao%20XX%2C%20Zhang%20XQ%2C%20Wang%20J%2C%20Frazier%20T%2C%20et%20al.%20Identifying%20differentially%20expressed%20genes%20under%20heat%20stress%20and%20developing%20molecular%20markers%20in%20orchardgrass%20%28Dactylis%20glomerata%20L.%29%20through%20transcriptome%20analysis.%20Mol%20Ecol%20Resour.%202015%3B15%3A1497%E2%80%93509."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR45" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Castr-Odíez P, Puyravaud JP, Cornelissen JHC. Leaf structure and anatomy as related to leaf mass per area variation in seedlings of a wide range of woody plant species and types. Oecologia. 2000;124:476–86.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/PL00008873"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Leaf%20structure%20and%20anatomy%20as%20related%20to%20leaf%20mass%20per%20area%20variation%20in%20seedlings%20of%20a%20wide%20range%20of%20woody%20plant%20species%20and%20types&amp;author=P.%20Castr-Od%C3%ADez&amp;author=JP.%20Puyravaud&amp;author=JHC.%20Cornelissen&amp;journal=Oecologia&amp;volume=124&amp;pages=476-86&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR46" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Chi LH, Song FB. Analyse of leaves’ structure character and ecological adaptation of ten plants in the west alkaline district of Songnen plain. Ecol Environ. 2006;15:1269–73.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Analyse%20of%20leaves%E2%80%99%20structure%20character%20and%20ecological%20adaptation%20of%20ten%20plants%20in%20the%20west%20alkaline%20district%20of%20Songnen%20plain&amp;author=LH.%20Chi&amp;author=FB.%20Song&amp;journal=Ecol%20Environ&amp;volume=15&amp;pages=1269-73&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR47" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yan L, Li H, He X, Wang XR. Nutrition apparatus observation of 9 kinds of sparsity be in sever danger plant of Alashan area. J Neimenggu Agri Univ. 2001;21:65–71.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Nutrition%20apparatus%20observation%20of%209%20kinds%20of%20sparsity%20be%20in%20sever%20danger%20plant%20of%20Alashan%20area&amp;author=L.%20Yan&amp;author=H.%20Li&amp;author=X.%20He&amp;author=XR.%20Wang&amp;journal=J%20Neimenggu%20Agri%20Univ&amp;volume=21&amp;pages=65-71&amp;publication_year=2001"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR48" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Dai LY, Zhang LJ, Yan-Ye R, Kai HU. Research on physiological characteristics and the assessment of tolerance of different varieties of sweet sorghum seedlings to saline-sodic stress. Agri Res in the Arid Areas. 2012;13:193–204.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Research%20on%20physiological%20characteristics%20and%20the%20assessment%20of%20tolerance%20of%20different%20varieties%20of%20sweet%20sorghum%20seedlings%20to%20saline-sodic%20stress&amp;author=LY.%20Dai&amp;author=LJ.%20Zhang&amp;author=R.%20Yan-Ye&amp;author=HU.%20Kai&amp;journal=Agri%20Res%20in%20the%20Arid%20Areas&amp;volume=13&amp;pages=193-204&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR49" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yan H, Zhao W, Yin SJ, Shi DC, Zhou DW. Different physiological responses of Aneurolepidium chinense to NaCl and Na<sub xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml">2</sub>CO<sub xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml">3</sub>. Acta Pratac Sin. 2006;15:49–55.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Different%20physiological%20responses%20of%20Aneurolepidium%20chinense%20to%20NaCl%20and%20Na2CO3%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=H.%20Yan&amp;author=W.%20Zhao&amp;author=SJ.%20Yin&amp;author=DC.%20Shi&amp;author=DW.%20Zhou&amp;journal=Acta%20Pratac%20Sin&amp;volume=15&amp;pages=49-55&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR50" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yang CW, Li CY, Yin HJ, Ju M, Shi DC. Physiological response of xiaobingmai (<em class="EmphasisTypeItalic">Triticum aestivum</em>-<em class="EmphasisTypeItalic">Agropyron intermedium</em>) to salt-stress and alkali-stress. Acta Agron Sin. 2007;33:1255–61.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Physiological%20response%20of%20xiaobingmai%20%28Triticum%20aestivum-Agropyron%20intermedium%29%20to%20salt-stress%20and%20alkali-stress&amp;author=CW.%20Yang&amp;author=CY.%20Li&amp;author=HJ.%20Yin&amp;author=M.%20Ju&amp;author=DC.%20Shi&amp;journal=Acta%20Agron%20Sin&amp;volume=33&amp;pages=1255-61&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR51" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sui N, Li M, Liu XY, Wang N, Fang W, Meng QW. Response of xanthophyll cycle and chloroplastic antioxidant enzymes to chilling stress in tomato over-expressing glycerol-3-phosphate acyltransferase gene. Photosynthetica. 2007;45:447–54.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11099-007-0074-5"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Response%20of%20xanthophyll%20cycle%20and%20chloroplastic%20antioxidant%20enzymes%20to%20chilling%20stress%20in%20tomato%20over-expressing%20glycerol-3-phosphate%20acyltransferase%20gene&amp;author=N.%20Sui&amp;author=M.%20Li&amp;author=XY.%20Liu&amp;author=N.%20Wang&amp;author=W.%20Fang&amp;author=QW.%20Meng&amp;journal=Photosynthetica&amp;volume=45&amp;pages=447-54&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR52" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Xu XJ, Shen BC, Qi JM. Advances in study of salt-stress tolerance and its physiological and biochemical indices in plants. Subtropical Agri Res. 2007;3:275–80.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Advances%20in%20study%20of%20salt-stress%20tolerance%20and%20its%20physiological%20and%20biochemical%20indices%20in%20plants&amp;author=XJ.%20Xu&amp;author=BC.%20Shen&amp;author=JM.%20Qi&amp;journal=Subtropical%20Agri%20Res&amp;volume=3&amp;pages=275-80&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR53" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Pittman JK, Shigaki T, Hirschi KD. Evidence of differential pH regulation of the Arabidopsis vacuolar Ca<sup xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml">2+</sup>/H<sup xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml">+</sup> antiporters CAX1 and CAX2. FEBS Lett. 2005;579:2648–56.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.febslet.2005.03.085"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15862304"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Evidence%20of%20differential%20pH%20regulation%20of%20the%20Arabidopsis%20vacuolar%20Ca2%2B%2FH%2B%20antiporters%20CAX1%20and%20CAX2&amp;author=JK.%20Pittman&amp;author=T.%20Shigaki&amp;author=KD.%20Hirschi&amp;journal=FEBS%20Lett&amp;volume=579&amp;pages=2648-56&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR54" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhang X, Xiao XL, Li PB, Sun YF, Guo WY, Yao YN. Physiological responses of <em class="EmphasisTypeItalic">populus alba</em> var. <em class="EmphasisTypeItalic">pyramidalis</em> leaves with different maturity to alkali stress. Sci Silvae Sin. 2015;51:9–16.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Physiological%20responses%20of%20populus%20alba%20var.%20pyramidalis%20leaves%20with%20different%20maturity%20to%20alkali%20stress&amp;author=X.%20Zhang&amp;author=XL.%20Xiao&amp;author=PB.%20Li&amp;author=YF.%20Sun&amp;author=WY.%20Guo&amp;author=YN.%20Yao&amp;journal=Sci%20Silvae%20Sin&amp;volume=51&amp;pages=9-16&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR55" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Upham BL, Jahnke LS. Photooxidative reactions in chloroplast thylakoids. Evidence for a Fenton-type reaction promoted by superoxide or ascorbate. Photosynth Res. 1986;8:235–47.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/BF00037131"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24443261"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Photooxidative%20reactions%20in%20chloroplast%20thylakoids.%20Evidence%20for%20a%20Fenton-type%20reaction%20promoted%20by%20superoxide%20or%20ascorbate&amp;author=BL.%20Upham&amp;author=LS.%20Jahnke&amp;journal=Photosynth%20Res&amp;volume=8&amp;pages=235-47&amp;publication_year=1986"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR56" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhang NH, Gao HY, Zou Q. Effect of calcium on alleviation of decreased photosynthetic ability in salt-stressed maize leaves. Acta Phytoecologica Sin. 2005;29:324–30.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Effect%20of%20calcium%20on%20alleviation%20of%20decreased%20photosynthetic%20ability%20in%20salt-stressed%20maize%20leaves&amp;author=NH.%20Zhang&amp;author=HY.%20Gao&amp;author=Q.%20Zou&amp;journal=Acta%20Phytoecologica%20Sin&amp;volume=29&amp;pages=324-30&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR57" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li F, Huang Z, Shan L. A study on rules of spatiao-temporal change and water use efficiency of corn. Acta Bot Boreali-Occidentalia Sin. 1999;20:1010–5.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20study%20on%20rules%20of%20spatiao-temporal%20change%20and%20water%20use%20efficiency%20of%20corn&amp;author=F.%20Li&amp;author=Z.%20Huang&amp;author=L.%20Shan&amp;journal=Acta%20Bot%20Boreali-Occidentalia%20Sin&amp;volume=20&amp;pages=1010-5&amp;publication_year=1999"><span><span>Google Scholar</span></span></a></span></span></cite></li></ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#TheAuthors" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="TheAuthors">Copyright</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="TheAuthors"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">©&nbsp;The Author(s)&nbsp;2017</div></div></section></div>
                        


                    </article>
                </main>

                <div class="c-content-layout__side u-text-sm">
                    <aside>
                        <div class="p-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="//hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-017-0031-7" data-track="click" data-track-category="PDF Download"
            data-track-action="Click download article PDF"
            data-track-label="10.1186/s41065-017-0031-7">Download PDF</a>
        </p>
    
                            
                            
    <button type="button"
       class="c-button c-button--secondary c-button--full-width c-button--expandable"
       data-toggle="collapse"
       data-target="export-citations"
       data-test="export-citations-button"
       data-sticky-update="true"
       aria-controls="export-citations"
       aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/s41065-017-0031-7"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0031-7?format=refman&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/s41065-017-0031-7"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0031-7?format=refman&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/s41065-017-0031-7"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0031-7?format=refman&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/s41065-017-0031-7"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0031-7?format=endnote&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/s41065-017-0031-7"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0031-7?format=endnote&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/s41065-017-0031-7"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0031-7?format=endnote&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/s41065-017-0031-7"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0031-7?format=bibtex&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/s41065-017-0031-7"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0031-7?format=bibtex&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/s41065-017-0031-7"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0031-7?format=bibtex&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
    </div>

                        </div>
                        
    
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
                                <div id="article-access-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/access-count/10.1186/s41065-017-0031-7 #accessCount-container"></div>
                                <div id="citation-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/citation-count/10.1186/s41065-017-0031-7 #count-container"></div>
                                <div id="altmetric-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/altmetric/10.1186/s41065-017-0031-7 #altmetric-container"></div>
                            </ul>
                        </div>
                        
    <div id="sociaMediaShare" class="p-separator u-js-shown">
        <h3 class="p-section-title">Share this article</h3>
        <ul id="shareButtons" class="c-social-links">
            <li class="c-social-links__item">
                <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-017-0031-7">
                    <span class="u-visually-hidden">Share on Twitter</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-twitter"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-017-0031-7">
                    <span class="u-visually-hidden">Share on Facebook</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-facebook"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click"
                        data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-017-0031-7">
                    <span class="u-visually-hidden">Share on LinkedIn</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-linkedin"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-017-0031-7">
                    <span class="u-visually-hidden">Share on Weibo</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-weibo"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnGooglePlus" class="js-btnShareOnGplus" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Google+" data-track-label="10.1186/s41065-017-0031-7">
                    <span class="u-visually-hidden">Share on Google Plus</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-gplus"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-017-0031-7">
                    <span class="u-visually-hidden">Share on Reddit</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-reddit"></use>
                    </svg>
                </button>
            </li>
        </ul>
    </div>
 
                        
    
        <div class="p-separator u-js-shown">
            <h3 class="p-section-title">See updates</h3>
            <button class="c-crossmark-button" data-target="crossmark" data-test="crossmark" data-track="click" data-track-category="Article Page" data-track-action="Click CrossMark" data-track-label="10.1186/s41065-017-0031-7">
                <img id="crossmark-icon" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" alt="Crossmark logo" />
            </button>
        </div>
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
                                <li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/s41065-017-0031-7" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=738&doi=10.1186%2Fs41065-017-0031-7&volume=154&issue=1&title=Analysis+of+differentially+expressed+genes+and+adaptive+mechanisms+of+Prunus+triloba+Lindl.+under+alkaline+stress&author_name=Jia+Liu&start_page=1&end_page=16">Order reprint</a>

                                </li>
                            </ul>
                        </div>
                    </aside>
                    
    <div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">Hereditas</span>
                        </h4>
                        <p class="c-journal-footer__issn">ISSN: 1601-5223</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
                                
                                    <li class="c-journal-footer__contact-item">Submission enquiries: <a href="https://www.editorialmanager.com/here/" target="_blank">Access here and click Contact Us</a></li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a></li>
                                
                            </ul>
                        </div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class='tracker' style='display:none' src='/track/article/10.1186/s41065-017-0031-7' alt=""/>

         
    <footer>
        <div class="ctx-publisher-footer">
    <div class="c-navbar">
        <div class="u-container">
            <div class="c-navbar__logo u-margin-right-xl">
                <a href="https://www.biomedcentral.com">
                    <img src=/static/images/bmc/logos/logo-bmc-aj-594e03e20d.svg alt="BMC logo" data-test="navbar-logo-footer">

                </a>
            </div>
            <div class="c-navbar__content">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


            </div>
        </div>
    </div>
    <div class="c-publisher-footer" data-test="publisher-footer">
        <div class="u-container">
            
            <div class="c-flex c-flex--wrap c-flex--space-between" role="navigation" data-test="publisher-footer-menu">
                <div class="c-flex" aria-label="Publisher footer links">
                    
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/login">Receive BMC newsletters</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/account">Manage article alerts</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/policies">Policies</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/accessibility">Accessibility</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/press-centre">Press center</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/contact-us">Contact us</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/jobs">Careers</a>
                                    </li>
                                
                            </ul>
                        
                    
                </div>
                <div class="u-margin-bottom-lg">
                    <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                    <ul class="c-flex u-reset-list" data-test="footer-social-links">
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://twitter.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Twitter link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Twitter page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-twitter-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://www.facebook.com/BioMedCentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Facebook link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Facebook page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-facebook-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://plus.google.com/106226337984152901734/"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Google Plus link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Google Plus page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-gplus-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="http://www.weibo.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Weibo link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Weibo page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-weibo-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </div>
            <p class="u-reset-margin">
                By using this website, you agree to our
                <a class="u-text-inherit" href="//www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
                <a class="u-text-inherit" href="//www.biomedcentral.com/privacy-statement">Privacy
                    statement</a> and
                <a class="u-text-inherit" href="//www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
                
                    <a class="optanon-toggle-display u-text-inherit" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
                
            </p>
        </div>
    </div>
</div>


        <div class="c-sn-footer c-sn-footer--neutral">
    <div class="u-container">
        <img src=/static/images/bmc/logos/logo-springernature-8bd4e8d82f.svg class="c-sn-footer__logo" alt="Springer Nature Logo" itemprop="logo" >
        <p data-test="copyright">&#169; 2018 BioMed Central Ltd unless otherwise stated. Part of
            <a href="http://www.springernature.com"
               data-track="click"
               data-track-action="Click Springer Nature link"
               data-track-category="Corporate footer"
               itemscope itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>

    </div>

    
    <script type="text/javascript" src="//cdn.pbgrd.com/bmc.js" async="async"></script>
    <noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>
    <!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker -->

<script id="springerlink-event-tracker" src="https://event-tracker.springernature.com/dist/eventTracker.js"></script>
<script type="text/javascript">
    (function(){
        var dataLayer = window.dataLayer[0];
        var doi = dataLayer.content.article.doi;
        var imprint = dataLayer.content.contentInfo.imprint;
        var bpids = dataLayer.session.authentication.authenticationID;
        var publisher;
        var pageType = dataLayer.page.category.pageType;
        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }
        var eventObject = {'content_type' : pageType,'doi' : doi};

        if (imprint === 'BioMed Central') { publisher = 'BMC' }
        else if (imprint === 'SpringerOpen') { publisher = 'SpringerOpen' }
        else { throw new Error('Event Tracker Error: The publisher has an unexpected value') }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }
        new EventTracker({'platform':publisher}).sendEvent('display', eventObject);
    })();
</script>


    </body>
</html>

