<!DOCTYPE html>
<html lang="en" class="no-js">
    <head>
        <title>PCR-based assays for validation of single nucleotide polymorphism markers in rice and mungbean | Hereditas | Full Text</title>

        

    <meta name="citation_abstract" content="Single nucleotide polymorphism (SNP) markers are the method of choice for genetic analyses including diversity and quantitative trait loci (QTL) studies. Marker validation is essential for QTL studies, but the cost and workload are considerable when large numbers of markers need to be verified. Marker systems with low development costs would be most suitable for this task. We have tested allele specific polymerase chain reaction (PCR), tetra markers and a genotyping tool based on the single strand specific nuclease CEL-I to verify randomly selected SNP markers identified previously either with a SNP array or by genotyping by sequencing in rice and mungbean, respectively. The genotyping capacity of allele-specific PCR and tetra markers was affected by the sequence context surrounding the SNP; SNPs located in repeated sequences and in GC-rich stretches could not be correctly identified. In contrast, CEL-I digestion of mixed fragments produced from test and reference DNA reliably pinpointed the correct genotypes, yet scoring of the genotypes became complicated when multiple SNPs were present in the PCR fragments. A cost analysis showed that as long the sample number remains small, CEL-I genotyping is more cost-effective than tetra markers. CEL-I genotyping performed better in terms of genotyping accuracy and costs than tetra markers. The method is highly useful for validating SNPs in small to medium size germplasm panels."/>

    <meta name="description" content="Single nucleotide polymorphism (SNP) markers are the method of choice for genetic analyses including diversity and quantitative trait loci (QTL) studies. Marker validation is essential for QTL studies, but the cost and workload are considerable when large numbers of markers need to be verified. Marker systems with low development costs would be most suitable for this task. We have tested allele specific polymerase chain reaction (PCR), tetra markers and a genotyping tool based on the single strand specific nuclease CEL-I to verify randomly selected SNP markers identified previously either with a SNP array or by genotyping by sequencing in rice and mungbean, respectively. The genotyping capacity of allele-specific PCR and tetra markers was affected by the sequence context surrounding the SNP; SNPs located in repeated sequences and in GC-rich stretches could not be correctly identified. In contrast, CEL-I digestion of mixed fragments produced from test and reference DNA reliably pinpointed the correct genotypes, yet scoring of the genotypes became complicated when multiple SNPs were present in the PCR fragments. A cost analysis showed that as long the sample number remains small, CEL-I genotyping is more cost-effective than tetra markers. CEL-I genotyping performed better in terms of genotyping accuracy and costs than tetra markers. The method is highly useful for validating SNPs in small to medium size germplasm panels."/>

    <meta name="journal_id" content="41065"/>

    <meta name="dc.title" content="PCR-based assays for validation of single nucleotide polymorphism markers in rice and mungbean"/>

    <meta name="dc.source" content="Hereditas 2017 154:1"/>

    <meta name="dc.format" content="text/html"/>

    <meta name="dc.publisher" content="BioMed Central"/>

    <meta name="dc.date" content="2017-01-26"/>

    <meta name="dc.type" content="OriginalPaper"/>

    <meta name="dc.language" content="En"/>

    <meta name="dc.copyright" content="2017 The Author(s)"/>

    <meta name="dc.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="dc.description" content="Single nucleotide polymorphism (SNP) markers are the method of choice for genetic analyses including diversity and quantitative trait loci (QTL) studies. Marker validation is essential for QTL studies, but the cost and workload are considerable when large numbers of markers need to be verified. Marker systems with low development costs would be most suitable for this task. We have tested allele specific polymerase chain reaction (PCR), tetra markers and a genotyping tool based on the single strand specific nuclease CEL-I to verify randomly selected SNP markers identified previously either with a SNP array or by genotyping by sequencing in rice and mungbean, respectively. The genotyping capacity of allele-specific PCR and tetra markers was affected by the sequence context surrounding the SNP; SNPs located in repeated sequences and in GC-rich stretches could not be correctly identified. In contrast, CEL-I digestion of mixed fragments produced from test and reference DNA reliably pinpointed the correct genotypes, yet scoring of the genotypes became complicated when multiple SNPs were present in the PCR fragments. A cost analysis showed that as long the sample number remains small, CEL-I genotyping is more cost-effective than tetra markers. CEL-I genotyping performed better in terms of genotyping accuracy and costs than tetra markers. The method is highly useful for validating SNPs in small to medium size germplasm panels."/>

    <meta name="prism.issn" content="1601-5223"/>

    <meta name="prism.publicationName" content="Hereditas"/>

    <meta name="prism.publicationDate" content="2017-01-26"/>

    <meta name="prism.volume" content="154"/>

    <meta name="prism.number" content="1"/>

    <meta name="prism.section" content="OriginalPaper"/>

    <meta name="prism.startingPage" content="3"/>

    <meta name="prism.copyright" content="2017 The Author(s)"/>

    <meta name="prism.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="prism.url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-016-0024-y"/>

    <meta name="prism.doi" content="doi:10.1186/s41065-016-0024-y"/>

    <meta name="citation_pdf_url" content="https://hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-016-0024-y"/>

    <meta name="citation_fulltext_html_url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-016-0024-y"/>

    <meta name="citation_journal_title" content="Hereditas"/>

    <meta name="citation_publisher" content="BioMed Central"/>

    <meta name="citation_issn" content="1601-5223"/>

    <meta name="citation_title" content="PCR-based assays for validation of single nucleotide polymorphism markers in rice and mungbean"/>

    <meta name="citation_volume" content="154"/>

    <meta name="citation_issue" content="1"/>

    <meta name="citation_publication_date" content="2017/12"/>

    <meta name="citation_online_date" content="2017/01/26"/>

    <meta name="citation_firstpage" content="3"/>

    <meta name="citation_article_type" content="Research"/>

    <meta name="citation_fulltext_world_readable" content=""/>

    <meta name="dc.identifier" content="doi:10.1186/s41065-016-0024-y"/>

    <meta name="DOI" content="10.1186/s41065-016-0024-y"/>

    <meta name="citation_doi" content="10.1186/s41065-016-0024-y"/>

    <meta name="dc.creator" content="Thu Giang Thi Bui"/>

    <meta name="dc.creator" content="Nguyen Thi Lan Hoa"/>

    <meta name="dc.creator" content="Jo-yi Yen"/>

    <meta name="dc.creator" content="Roland Schafleitner"/>

    <meta name="dc.subject" content="Animal Genetics and Genomics"/>

    <meta name="dc.subject" content="Evolutionary Biology"/>

    <meta name="citation_reference" content="citation_journal_title=J Bioscience; citation_title=Large SNP arrays for genotyping in crop plants; citation_author=MW Ganal, A Polley, EM Graner, J Plieske, R Wieseke, H Luerssen, G Durstewitz; citation_volume=37; citation_publication_date=2012; citation_pages=821-8; citation_doi=10.1007/s12038-012-9225-3; citation_id=CR1"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Rev Genet; citation_title=Harnessing the power of RADseq for ecological and evolutionary genomics; citation_author=KR Andrews, JM Good, MR Miller, G Luikart, PA Hohenlohe; citation_volume=17; citation_publication_date=2016; citation_pages=81-92; citation_doi=10.1038/nrg.2015.28; citation_id=CR2"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Breeding; citation_title=Single nucleotide polymorphism genotyping using Kompetitive Allele Specific PCR (KASP): overview of the technology and its application in crop improvement; citation_author=K Semagn, R Babu, S Hearne, M Olsen; citation_volume=33; citation_publication_date=2014; citation_pages=1-14; citation_doi=10.1007/s11032-013-9917-x; citation_id=CR3"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=SNP2CAPS: a SNP and INDEL analysis tool for CAPS marker development; citation_author=T Thiel, R Kota, I Grosse, N Stein, A Graner; citation_volume=32; citation_publication_date=2004; citation_pages=e5; citation_doi=10.1093/nar/gnh006; citation_id=CR4"/>

    <meta name="citation_reference" content="citation_journal_title=Plant J; citation_title=dCAPS, a simple technique for the genetic analysis of single nucleotide polymorphisms: experimental applications in Arabidopsis thaliana genetics; citation_author=MM Neff, JD Neff, J Chory, AE Pepper; citation_volume=14; citation_publication_date=1998; citation_pages=387-92; citation_doi=10.1046/j.1365-313X.1998.00124.x; citation_id=CR5"/>

    <meta name="citation_reference" content="Gaudet M, Fara A-G, Beritognolo I, Sabatti M. Allele-specific PCR in SNP genotyping. In: Komar AA, editor. Single Nucleotide Polymorphisms: Methods and Protocols. New York: Humana Press; 2009. p. 415&#8211;24."/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=Allele specific amplification by tetra-primer PCR; citation_author=S Ye, S Humphries, F Green; citation_volume=20; citation_publication_date=1992; citation_pages=1152; citation_doi=10.1093/nar/20.5.1152; citation_id=CR7"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=An efficient procedure for genotyping single nucleotide polymorphisms; citation_author=S Ye, S Dhillon, X Ke, AR Collins, IN Day; citation_volume=29; citation_publication_date=2001; citation_pages=e88; citation_doi=10.1093/nar/29.17.e88; citation_id=CR8"/>

    <meta name="citation_reference" content="citation_journal_title=Genome; citation_title=Genotyping single nucleotide polymorphisms in barley by tetra-primer ARMS-PCR; citation_author=E Chiapparino, D Lee, P Donini; citation_volume=47; citation_publication_date=2004; citation_pages=414-20; citation_doi=10.1139/g03-130; citation_id=CR9"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Biotechnol; citation_title=Guidelines for the tetra-primer ARMS&#8211;PCR technique development; citation_author=RFV Medrano, CA Oliveira; citation_volume=56; citation_publication_date=2014; citation_pages=599-608; citation_id=CR10"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Breeding; citation_title=Rapid method for detecting SNPs on agarose gels and its application in candidate gene mapping; citation_author=C Raghavan, MEB Naredo, H Wang, G Atienza, B Liu, F Qiu, KL McNally, H Leung; citation_volume=19; citation_publication_date=2007; citation_pages=87-101; citation_doi=10.1007/s11032-006-9046-x; citation_id=CR11"/>

    <meta name="citation_reference" content="citation_journal_title=Crop Sci; citation_title=CEL-I nuclease digestion for SNP discovery and marker development in common bean (L.); citation_author=CH Galeano, M Gomez, LM Rodriguez, MW Blair; citation_volume=49; citation_publication_date=2009; citation_pages=381-94; citation_doi=10.2135/cropsci2008.07.0413; citation_id=CR12"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=Mutation detection using a novel plant endonuclease; citation_author=CA Oleykowski, CRB Mullins, AK Godwin, AT Yeung; citation_volume=26; citation_publication_date=1998; citation_pages=4597-602; citation_doi=10.1093/nar/26.20.4597; citation_id=CR13"/>

    <meta name="citation_reference" content="citation_journal_title=Biochemistry; citation_title=Purification, cloning, and characterization of the CEL-I nuclease; citation_author=B Yang, X Wen, NS Kodali, CA Oleykowski, CG Miller, J Kulinski, D Besack, JA Yeung, D Kowalski, AT Yeung; citation_volume=39; citation_publication_date=2000; citation_pages=3533-41; citation_doi=10.1021/bi992376z; citation_id=CR14"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=Mismatch cleavage by single-strand specific nucleases; citation_author=BJ Till, C Burtner, L Comai, S Henikoff; citation_volume=32; citation_publication_date=2004; citation_pages=2632-41; citation_doi=10.1093/nar/gkh599; citation_id=CR15"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=Whole genome sequencing of elite rice cultivars as a comprehensive information resource for marker assisted selection; citation_author=J Duitama, A Silva, Y Sanabria, DF Cruz, C Quintero, C Ballen, M Lorieux, B Scheffler, A Farmer, E Torres, J Oard; citation_volume=10; citation_publication_date=2015; citation_pages=e0124617; citation_doi=10.1371/journal.pone.0124617; citation_id=CR16"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Plant Biol; citation_title=dentification of single nucleotide polymorphic markers associated with resistance to bruchids (Callosobruchus spp.) in wild mungbean (Vigna radiata var. sublobata) and cultivated V. radiata through genotyping by sequencing and quantitative trait locus analysis; citation_author=R Schafleitner, SM Huang, SH Chu, JY Yen, CY Lin, MR Yan, B Krishnan, MS Liu, HF Lo, CY Chen, LFO Chen, DC Wu, TGT Bui, S Ramasamy, C-W Tung, N Ramakrishnan; citation_volume=16; citation_publication_date=2016; citation_pages=1; citation_doi=10.1186/s12870-016-0847-8; citation_id=CR17"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Breed; citation_title=High-throughput single nucleotide polymorphism genotyping for breeding applications in rice using the BeadXpress platform; citation_author=MJ Thomson, K Zhao, M Wright, KL McNally, J Rey, C-W Tung, A Reynolds, B Scheffler, G Eizenga, A McClung; citation_volume=29; citation_publication_date=2012; citation_pages=875-86; citation_doi=10.1007/s11032-011-9663-x; citation_id=CR18"/>

    <meta name="citation_reference" content="citation_journal_title=Phytochem Bull; citation_title=A rapid DNA isolation procedure for small quantities of fresh leaf tissue; citation_author=JJ Doyle; citation_volume=19; citation_publication_date=1987; citation_pages=11-5; citation_id=CR19"/>

    <meta name="citation_reference" content="citation_journal_title=Nature Comm; citation_title=Genome sequence of mungbean and insights into evolution within Vigna species; citation_author=YJ Kang, SK Kim, MY Kim, P Lestari, KH Kim, B-K Ha, TH Jun, WJ Hwang, T Lee, J Lee, S Shim, MY Yoon, YE Jang, KS Han, P Taeprayoon, N Yoon, P Somta, P Tanya, KS Kim, J-G Gwag, J-K Moon, J-H Lee, B Park, A Bombarely, JJ Doyle, SA Jackson, R Schafleitner, P Srinives, RK Varshney, S-H Lee; citation_volume=5; citation_publication_date=2014; citation_pages=5443; citation_doi=10.1038/ncomms6443; citation_id=CR20"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Plant Biol; citation_title=SNP-based analysis of genetic diversity reveals important alleles associated with seed size in rice; citation_author=W Tang, T Wu, J Ye, J Sun, Y Jiang, J Yu, J Tang, G Chen, C Wang, J Wan; citation_volume=16; citation_publication_date=2016; citation_pages=1; citation_doi=10.1186/s12870-015-0700-5; citation_id=CR21"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species; citation_author=RJ Elshire, JC Glaubitz, Q Sun, JA Poland, K Kawamoto, ES Buckler, SE Mitchell; citation_volume=6; citation_publication_date=2011; citation_pages=e19379; citation_doi=10.1371/journal.pone.0019379; citation_id=CR22"/>

    <meta name="citation_reference" content="citation_journal_title=Funct Integr Genomics; citation_title=Association mapping and genetic dissection of nitrogen use efficiency-related traits in rice (Oryza sativa L.); citation_author=Z Liu, C Zhu, Y Jiang, Y Tian, J Yu, H An, W Tang, J Sun, J Tang, G Chen, H Zhai; citation_volume=16; citation_publication_date=2016; citation_pages=323-33; citation_doi=10.1007/s10142-016-0486-z; citation_id=CR23"/>

    <meta name="citation_author" content="Thu Giang Thi Bui"/>

    <meta name="citation_author_institution" content="Plant Resources Center, Vietnam Academy of Agricultural Sciences, Ha Noi, Vietnam"/>

    <meta name="citation_author" content="Nguyen Thi Lan Hoa"/>

    <meta name="citation_author_institution" content="Plant Resources Center, Vietnam Academy of Agricultural Sciences, Ha Noi, Vietnam"/>

    <meta name="citation_author" content="Jo-yi Yen"/>

    <meta name="citation_author_institution" content="World Vegetable Center, Tainan, Taiwan"/>

    <meta name="citation_author" content="Roland Schafleitner"/>

    <meta name="citation_author_institution" content="World Vegetable Center, Tainan, Taiwan"/>



        <meta charset="UTF-8"/>
        <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
        <meta name="viewport" content="width=device-width, initial-scale=1"/>

        <!-- For IE 9 and below -->
<!--[if IE]><link rel="shortcut icon" href=/static/images/favicons//favicon.ico><![endif]-->
<!-- Touch Icons - iOS and Android 2.1+ -->
<link rel="apple-touch-icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>
<!-- Firefox, Chrome, Safari, IE 11+ and Opera -->
<link rel="icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>


        <script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script>
        <script>(function(d){if (sessionStorage.fontsLoaded){d.style.setProperty('--font-family-sans', 'Europa');d.style.setProperty('--font-family-serif', 'Noto Serif')}})(document.documentElement);</script>

        <link rel="stylesheet" media="screen" href=/static/styles/basic/basic-a31ca85bc2.css>
<link rel="stylesheet" media="print" href=/static/styles/bmc/print-3a6240f456.css>

<link rel="stylesheet" class="js-ctm" href=/static/styles/bmc/main-84414d3d47.css
      media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">

        

        <script src="/static/js/app-bundle-8d61e6ef37.js" defer></script>
        
        <script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script>
        
    <script type="text/javascript">
        config = {
            env: 'live',
            site: 'hereditasjournal.biomedcentral.com',
            siteWithPath: 'hereditasjournal.biomedcentral.com' + window.location.pathname,
            twitterHashtag: '',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/s41065-016-0024-y',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central'
        };
    </script>

        
    <script type="text/javascript">
        dataLayer = [{"content":{"article":{"doi":"10.1186/s41065-016-0024-y","articleType":"Research","peerReviewType":"Closed","supplement":null,"keywords":"Single nucleotide polymorphism;Genotyping;PCR-based markers;CEL-I"},"contentInfo":{"imprint":"BioMed Central","title":"PCR-based assays for validation of single nucleotide polymorphism markers in rice and mungbean","publishedAt":1485388800000,"publishedAtDate":"2017-01-26","author":["Thu Giang Thi Bui","Nguyen Thi Lan Hoa","Jo-yi Yen","Roland Schafleitner"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":"CC BY + CC0","openAccess":true},"environment":"live"},"journal":{"siteKey":"hereditasjournal.biomedcentral.com","volume":"154","issue":"1","title":"Hereditas","type":"AJs Life","journalID":41065,"gaCode":"UA-62066850-11","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Research","publishingSegment":"Genetics & Heredity (Life)"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}}}]

    </script>

        <script src="https://cdn.cookielaw.org/consent/7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script>
        
    
        
    

        
    
        
            
                <!-- Google Tag Manager -->
                <script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script>
                <!-- End Google Tag Manager -->
            
        
    


        
    
    <link rel="canonical" href="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-016-0024-y"/>
    

        
        
        <meta property="og:url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-016-0024-y"/>
        <meta property="og:type" content="article"/>
        <meta property="og:site_name" content="Hereditas"/>
        <meta property="og:title" content="PCR-based assays for validation of single nucleotide polymorphism markers in rice and mungbean"/>
        <meta property="og:description" content="Single nucleotide polymorphism (SNP) markers are the method of choice for genetic analyses including diversity and quantitative trait loci (QTL) studies. Marker validation is essential for QTL studies, but the cost and workload are considerable when large numbers of markers need to be verified. Marker systems with low development costs would be most suitable for this task. We have tested allele specific polymerase chain reaction (PCR), tetra markers and a genotyping tool based on the single strand specific nuclease CEL-I to verify randomly selected SNP markers identified previously either with a SNP array or by genotyping by sequencing in rice and mungbean, respectively. The genotyping capacity of allele-specific PCR and tetra markers was affected by the sequence context surrounding the SNP; SNPs located in repeated sequences and in GC-rich stretches could not be correctly identified. In contrast, CEL-I digestion of mixed fragments produced from test and reference DNA reliably pinpointed the correct genotypes, yet scoring of the genotypes became complicated when multiple SNPs were present in the PCR fragments. A cost analysis showed that as long the sample number remains small, CEL-I genotyping is more cost-effective than tetra markers. CEL-I genotyping performed better in terms of genotyping accuracy and costs than tetra markers. The method is highly useful for validating SNPs in small to medium size germplasm panels."/>
        <meta property="og:image" content="https://images.springer.com/sgw/journals/medium/41065.jpg"/>
    
    </head>

    <body class="journal journal-fulltext"
          
          >
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK"
                            height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><feColorMatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"/></filter></defs><symbol id="icon-arrow-down" viewBox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"/></symbol><symbol id="icon-arrow-left-bullet" viewBox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"/></symbol><symbol id="icon-arrow-right-bullet" viewBox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"/></symbol><symbol id="icon-arrow-right" viewBox="0 0 32 64"><path d="M0 0v64l32-32z"/></symbol><symbol id="icon-close" viewBox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"/></symbol><symbol id="icon-community" viewBox="0 0 60.643 31.01"><mask maskUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><path fill="#FFF" d="M0 31.01h60.643V0H0z" filter="url(#a)"/></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"/></symbol><symbol id="icon-download" viewBox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"/></symbol><symbol id="icon-ext-link" viewBox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"/><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"/><path fill="#676767" d="M9.7 0L16 6.2V0z"/></symbol><symbol id="icon-info-bordered" viewBox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"/><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"/></symbol><symbol id="icon-opr" viewBox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"/><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"/></symbol><symbol id="icon-quotation" viewBox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"/></symbol><symbol id="icon-remove" viewBox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"/><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"/></symbol><symbol id="icon-ribbon" viewBox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"/></symbol><symbol id="icon-rss" viewBox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"/><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"/><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"/></g></symbol><symbol id="icon-search" viewBox="0 0 16 16"><path d="M6.276 12.257C2.815 12.257 0 9.508 0 6.128 0 2.749 2.815 0 6.276 0c3.462 0 6.277 2.749 6.277 6.128.002 3.38-2.815 6.129-6.277 6.129zm0-10.772c-2.622 0-4.755 2.083-4.755 4.643 0 2.562 2.133 4.644 4.755 4.644 2.623 0 4.756-2.082 4.756-4.644 0-2.56-2.134-4.643-4.756-4.643z"/><path d="M16 13.751L13.641 16l-4.875-4.82 2.36-2.248L16 13.751z"/></symbol><symbol id="icon-speech-bubble" viewBox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"/></symbol><symbol id="icon-error" viewBox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"/></symbol><symbol id="icon-info" viewBox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"/></symbol><symbol id="icon-success" viewBox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"/></symbol><symbol id="icon-warning" viewBox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"/></symbol><symbol id="icon-facebook-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"/></symbol><symbol id="icon-facebook" viewBox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"/><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"/></symbol><symbol id="icon-gplus-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M480.5 273.117c.123.076.26.185.416.334.146.152.289.343.432.562.135.211.258.459.359.746.086.286.129.619.129 1.004-.012.7-.164 1.262-.465 1.683a5.374 5.374 0 0 1-.461.562c-.186.176-.375.35-.58.527a2.65 2.65 0 0 0-.322.41.908.908 0 0 0-.176.553c0 .205.059.375.18.508.102.129.199.238.299.336l.662.545a6.48 6.48 0 0 1 1.086 1.119c.293.415.445.955.459 1.625 0 .949-.42 1.789-1.256 2.521-.869.76-2.123 1.15-3.762 1.177-1.373-.015-2.4-.308-3.074-.879-.682-.534-1.023-1.172-1.023-1.917 0-.359.111-.766.336-1.211.215-.443.602-.837 1.172-1.174a6.126 6.126 0 0 1 1.998-.725 15.415 15.415 0 0 1 1.723-.173 3.996 3.996 0 0 1-.377-.596 1.347 1.347 0 0 1-.193-.74c0-.174.023-.317.074-.433a8.83 8.83 0 0 1 .121-.338 5.973 5.973 0 0 1-.633.035c-1.043-.012-1.838-.339-2.381-.979-.574-.603-.861-1.3-.861-2.097 0-.961.404-1.832 1.217-2.616.559-.46 1.135-.757 1.736-.898a8.494 8.494 0 0 1 1.67-.182h3.916l-1.209.704h-1.207v.005l-.005.002zm.754 11.276c0-.498-.162-.932-.484-1.302-.346-.348-.881-.778-1.609-1.29a4.045 4.045 0 0 0-.439-.02c-.102-.016-.354 0-.768.039a8.141 8.141 0 0 0-1.244.275c-.1.036-.24.096-.42.17a2.713 2.713 0 0 0-.553.355 1.985 1.985 0 0 0-.459.592c-.145.25-.215.551-.215.901 0 .687.311 1.253.932 1.698.596.444 1.404.68 2.432.688.928-.014 1.629-.217 2.117-.612.476-.385.71-.886.71-1.494zm-2.729-5.743a1.843 1.843 0 0 0 1.289-.556c.164-.249.271-.506.318-.767a7.32 7.32 0 0 0 .041-.659c0-.772-.195-1.552-.594-2.336a2.611 2.611 0 0 0-.73-.921 1.894 1.894 0 0 0-1.064-.352 1.723 1.723 0 0 0-1.324.64c-.299.437-.438.92-.428 1.457 0 .709.209 1.444.623 2.211.201.356.459.659.775.912.323.248.686.371 1.094.371zm9.698.077h-2.055v-2.056h-.996v2.056h-2.053v.998h2.053v2.052h.996v-2.052h2.055z"/></symbol><symbol id="icon-gplus" viewBox="0 0 24 24"><circle fill="#D84A36" cx="12" cy="12" r="12"/><g fill="#FFF"><path d="M12.012 12.111c-.304-.33-.07-.842.155-1.078.226-.241.639-.56.829-.74.193-.184.66-.605.674-1.868 0 0 .112-1.139-1.127-2.06h.977l1.038-.593h-3.507s-2.083-.067-3.085 1.207c-1 1.276-.966 2.871.114 3.77 1.08.898 2.371.648 2.371.648s-.197.422-.163.775c.036.353.421.849.504.965 0 0-2.386-.011-3.551 1.033-1.165 1.053-1.29 3.097 1.021 3.803 2.312.707 5.303-.055 5.906-2.277 0 0 .262-.908-.189-1.683-.459-.775-1.663-1.571-1.967-1.902zm-2.555-1.83c-.744-1-1.308-2.953 0-3.783 0 0 .955-.579 1.801.266.846.846 1.082 2.277.928 3.003-.15.727-.563.897-.739 1.008-.185.111-1.249.504-1.99-.494zm3.668 5.481c-.035.393-.083 1.254-1.491 1.664-1.409.408-3.826-.264-3.757-1.914 0 0 .006-.698.449-1.089.445-.396 1.094-.794 3.029-.819 0 0 .643.473 1.072.809.427.332.733.952.698 1.349z"/><path d="M16.323 11.055V9.803h-.752v1.252h-1.255v.748h1.255v1.256h.752v-1.256h1.248v-.748z"/></g></symbol><symbol id="icon-linkedin" viewBox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"/><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"/><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"/></g></symbol><symbol id="icon-reddit" viewBox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"/><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"/><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"/><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"/><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"/><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"/><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"/><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"/></symbol><symbol id="icon-twitter-bordered" viewBox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"/><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/></symbol><symbol id="icon-twitter" viewBox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"/><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"/></symbol><symbol id="icon-weibo-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"/><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"/><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"/></symbol><symbol id="icon-weibo" viewBox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"/><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"/><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"/><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"/><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"/><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"/></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner c-banner--info">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner" >
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1"
                 data-gpt-unitpath="/270604982/bmc/hereditasjournal/articles"
                 data-gpt-sizes="728x90,970x90"
                 data-gpt-targeting="pos=LB1;doi=10.1186/s41065-016-0024-y;kwrd=Single nucleotide polymorphism,Genotyping,PCR-based markers,CEL-I;pmc=L32030,L21001;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-016-0024-y&amp;kwrd=Single nucleotide polymorphism,Genotyping,PCR-based markers,CEL-I&amp;pmc=L32030,L21001&amp;">
                        <img data-test="gpt-advert-fallback-img"
                             src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-016-0024-y&amp;kwrd=Single nucleotide polymorphism,Genotyping,PCR-based markers,CEL-I&amp;pmc=L32030,L21001&amp;"
                             alt="Advertisement"
                             width="728"
                             height="90">
                    </a>
                </noscript>
            </div>
        </div>
    </div>

        
         
    <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>


<header class="c-popup-search ctx-publisher-header" id="top" data-test="publisher-header">
    <nav class="c-navbar">
        <div class="u-container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization" />
                <div class="c-logo">
    <a href="https://www.biomedcentral.com" itemprop="url">
        <img src=/static/images/bmc/logos/logo-bmc-white-aj-e3a70ae848.svg alt="BMC" itemprop="logo">
        <div class="c-logo__strapline">
            <img src=/static/images/bmc/logos/logo-bmc-white-strapline-sn-e7b1d6e67f.svg alt="Part of Springer Nature">
        </div>
    </a>
</div>
            </div>
            <button
                type="button"
                class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg"
                id="publisher-header-menu-button"
                data-toggle="collapse"
                data-test="publisher-header-menu-toggle"
                data-toggle-linked="publisher-header"
                data-target="publisher-header-search publisher-header-menu"
                aria-controls="publisher-header-menu publisher-header-search"
                aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


                <ul class="c-navbar__nav">
                    <li class="c-navbar__item">
                        <a data-replace="true"
                           data-placeholder="/placeholder/v1/username"
                           class="c-navbar__link"
                           href="https://www.biomedcentral.com/login"
                           data-test="login-link"
                           data-track="click"
                           data-track-category="Account"
                           data-track-action="Click Login Link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                    <li class="c-navbar__item">
                        <button
                            type="button"
                            class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg"
                            data-toggle="collapse"
                            data-test="header-search-button"
                            data-target="publisher-header-search publisher-header-menu publisher-header-menu-button"
                            aria-controls="publisher-header-search publisher-header-menu"
                            aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true">
                                    <use xlink:href="#icon-search"></use>
                                </svg>
                            </span>
                        </button>
                    </li>
                </ul>
            </div>
        </div>
    </nav>
    <div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field c-form-field--inline" method="GET" action="//www.biomedcentral.com/search" data-track="submit"
        data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label data-track-label="" data-test="global-search">
        <label for="publisherSearch" class="u-visually-hidden">Search all BMC articles</label>
        <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text" value="" placeholder="Search all BMC articles"/>
        <button class="c-button c-button--secondary" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true">
        <use xlink:href="#icon-search"></use>
    </svg>
</button>


        <input type="hidden" name="searchType" value="publisherSearch"/>
    </form>
</div>

            </div>
        </div>
    </div>
</header>


        
            <header class="c-journal-header c-journal-header--hereditas ctx-journal-header">
                <div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="/">
    
        
    
    <span class="c-journal-title__text ">Hereditas</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="u-container">
                        
                            <button type="button"
                                    class="c-navbar__link c-dropdown u-hidden-at-lg"
                                    data-toggle="collapse"
                                    data-test="journal-header-menu"
                                    data-toggle-linked="journal-header"
                                    data-target="journal-menu"
                                    aria-controls="journal-menu"
                                    aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav">
                                    <ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/'>Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/about'>About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href='/articles'>Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/submission-guidelines'>Submission Guidelines</a>
                                            </li>
                                        
                                    </ul>
                                </nav>
                            </div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--ocean-light-blue">
                    
                </div>
            </header>
            
        

        <div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-content-layout c-content-layout--fulltext" data-component="sticky-container">
                <div class="c-content-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3><ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc=""><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Methods">Methods</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec7" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results">Results</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec11" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Discussion">Discussion</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec12" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Conclusions">Conclusions</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li></ol></nav>
                </div>

                <main class="c-content-layout__main">
                    <article itemscope itemtype="http://schema.org/ScholarlyArticle">
                        
                        
    <div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
            <li class="c-article-identifiers__item" data-test="article-category">Research</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
    </div>

                        <div class="FulltextWrapper"><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">PCR-based assays for validation of single nucleotide polymorphism markers in rice and mungbean</h1></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true"><ul class="u-listReset"><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Thu&nbsp;Giang&nbsp;Thi&nbsp;Bui</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Nguyen&nbsp;Thi&nbsp;Lan&nbsp;Hoa</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Jo-yi&nbsp;Yen</span><sup><a href="#Aff2" class="AffiliationID">2</a></sup> and </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Roland&nbsp;Schafleitner</span><sup><a href="#Aff2" class="AffiliationID">2</a></sup><a href="mailto:roland.schafleitner@worldveg.org" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a></li></ul></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation"><div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">Hereditas</span></span><span class="ArticleCitation_Year">2017</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">154</strong>:3</span></span></div><p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/s41065-016-0024-y">https://doi.org/10.1186/s41065-016-0024-y</a></p><p class="Copyright">© &nbsp;The Author(s)&nbsp;2017</p></div></div><ul class="ArticleHistory u-reset-list"><li class="History HistoryReceived"><strong>Received: </strong><span>31&nbsp;August&nbsp;2016</span></li><li class="History HistoryAccepted"><strong>Accepted: </strong><span>20&nbsp;December&nbsp;2016</span></li><li class="History HistoryOnlineDate"><strong>Published: </strong><span itemprop="datePublished">26&nbsp;January&nbsp;2017</span></li></ul></div>
                        
                        
                        
    

                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2><div class="c-section__content" id="Abstract"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p id="Par1" class="Para">Single nucleotide polymorphism (SNP) markers are the method of choice for genetic analyses including diversity and quantitative trait loci (QTL) studies. Marker validation is essential for QTL studies, but the cost and workload are considerable when large numbers of markers need to be verified. Marker systems with low development costs would be most suitable for this task.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Results</h3>
                <p id="Par2" class="Para">We have tested allele specific polymerase chain reaction (PCR), tetra markers and a genotyping tool based on the single strand specific nuclease CEL-I to verify randomly selected SNP markers identified previously either with a SNP array or by genotyping by sequencing in rice and mungbean, respectively. The genotyping capacity of allele-specific PCR and tetra markers was affected by the sequence context surrounding the SNP; SNPs located in repeated sequences and in GC-rich stretches could not be correctly identified. In contrast, CEL-I digestion of mixed fragments produced from test and reference DNA reliably pinpointed the correct genotypes, yet scoring of the genotypes became complicated when multiple SNPs were present in the PCR fragments. A cost analysis showed that as long the sample number remains small, CEL-I genotyping is more cost-effective than tetra markers.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusions</h3>
                <p id="Par3" class="Para">CEL-I genotyping performed better in terms of genotyping accuracy and costs than tetra markers. The method is highly useful for validating SNPs in small to medium size germplasm panels.</p>
              </div></div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords"><li class="c-keywords__item" itemprop="about">Single nucleotide polymorphism</li><li class="c-keywords__item" itemprop="about">Genotyping</li><li class="c-keywords__item" itemprop="about">PCR-based markers</li><li class="c-keywords__item" itemprop="about">CEL-I</li></ul></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par16" class="Para">Single nucleotide polymorphisms (SNPs) have become the most widely used marker system for plant and animal genetic analyses. SNP arrays are available for a number of plant and animal species [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>] and tools combining SNP detection and genotyping such as genotyping by sequencing (GBS) made the use of these markers in genetic studies feasible and affordable for virtually any organism, including non-model species (recently reviewed by [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>]). Quantitative trait loci (QTL) analyses may yield a considerable number of markers that require verification of their correctness and diagnostic capacity to predict a phenotype. Depending on the number of markers to be tested, SNP marker validation can be even more costly and laborious than the genotyping experiment itself.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par17" class="Para">Several assay types are available for verifying SNP markers generated by GBS, SNP chips or similar technologies. Cost-effective commercial SNP assays are available [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>], but the cost saving promised by these assays are only attained in routine genotyping, when large sample numbers are analyzed with the same assay. For validation, the number of genotypes used for testing a candidate marker is typically small, and only a few of these markers will be chosen for routine genotyping, while most markers will be discarded. In this situation, the high development costs for commercial assays are not compensated for by their low running costs, making these markers an expensive option for validation experiments. What is more, commercial SNP genotyping tools such as KASP or Taqman assays require specialized laboratory equipment, which might not be available in every genotyping lab, especially not in developing countries. Consequently, for marker validation, simple SNP genotyping tools with low development costs are preferred.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par18" class="Para">Conversion of SNP markers to cleaved amplified polymorphic sequence (CAPS) markers is often used as a PCR-based method to genotype SNP markers [<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>], but similar to commercial kits, the method might not be practical for SNP validation. First, only a fraction of SNP markers can be converted to CAPS markers, and second, the restriction enzymes for CAPS assays cause similarly high development costs like commercial assays. dCAPS markers overcome the need for the SNP to fall within a restriction enzyme recognition site and eliminate the limitation that only a fraction of SNPs can be converted to dCAPS [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>], but the marker development costs remain high, as a restriction enzyme is required to distinguish the SNP alleles. PCR with allele-specific primers of different length has been proposed as a simple and cost-effective tool for PCR-based SNP genotyping [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>]. This method is very cheap and easy to perform, but may be constrained by the sequence context around the SNP, which restricts the options for primer design. Similarly, tetra-primer amplification has been described as an efficient low cost method for genotyping SNP markers [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>–<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>]. This method uses two locus-specific outer primers that asymmetrically flank the SNP under investigation, and two allele-specific inner primers, which produce a larger fragment for one allele, and a smaller fragment for the second allele. The bands of different size produced by one inner and outer primer pair can be easily detected on polyacrylamide or agarose gels [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>]. To increase the specificity of the genotyping assay, an additional mismatch base near the 3′ end of the allele-specific primers can be added [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>, <span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>]. The method may not work well for SNPs in cytosine and guanine-rich DNA regions, and restrictions for choosing the inner (allele-specific) primers may limit the assay performance and require laborious adjustments and assay optimization [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par19" class="Para">SNP genotyping using the single strand specific nuclease CEL-I has been proposed previously [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>, <span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>]. The mismatch-specific nuclease CEL-I is extracted at low cost by ammonium sulfate precipitation from common celery [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>, <span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>]. It detects mismatches in DNA double strands with high sensitivity and is commonly used to identify point mutations in methods known as TILLING and Eco-TILLING [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>]. CEL-I was used to genotype SNPs present in mixtures of relatively large PCR fragments (2,000&nbsp;bp) derived from two individuals after denaturation and re-naturation of the fragments [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>]. Previously reported protocols require stopping the enzyme reaction by adding EDTA, or even removing excess salt and concentrating the samples through precipitation, making the method relatively laborious [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par20" class="Para">Rice is the most important field crop in Asia and mungbean is increasingly used as a rotation crop in rice production systems. Marker-assisted selection is routinely performed in rice breeding and becomes also popular for mungbean [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>, <span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>]. Therefore, the present case study aimed to evaluate the performance and costs of low cost PCR-based SNP assays on these two crops. We have tested allele-specific PCR, tetra PCR and CEL-I digestion for their capacity to correctly validate the presence of SNPs previously identified in a rice germplasm panel genotyped by a GoldenGate assay, and in a bi-parental mungbean population, where SNPs were identified by GBS. Special consideration was given to streamline the assays to keep costs and labor requirements as low as possible. An ideal assay should have no or only minimal need for optimization, enabling designing and running a large number of assays for validation at minimal effort and cost. Therefore, in this study, no optimization and primer redesign was performed before assessing the performance and costs of the marker assays.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Methods" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Methods">Methods</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Methods" data-test="collapse-section"><section id="Sec3" class="Section2 RenderAsSection2">
                <h3 class="Heading">Plant material and SNP markers</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par21" class="Para">In a previous work, a set of 26 local Vietnamese <em xmlns="" class="EmphasisTypeItalic">Oryza sativa</em> spp. <em xmlns="" class="EmphasisTypeItalic">japonica</em> and spp. <em xmlns="" class="EmphasisTypeItalic">indica</em> rice cultivars held in the National Crop Genebank of the Plant Resources Center of the Vietnam Academy of Agricultural Sciences, Hanoi, Vietnam, were submitted to genetic diversity analysis at the Plant Breeding, Genetics and Biotechnology Division of the International Rice Research Institute, Los Baños, Philippines, using a 288 SNPs sub-set of the 384-SNP GoldenGate chip on a Fluidigm EP1 system according to [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>]. From the genotyping results, five SNPs were chosen at random for validation with allele-specific primers, tetra markers and CEL-I genotyping (Table&nbsp;<span class="InternalRef"><a href="#Tab1">1</a></span>). In total 10 out of the 26 lines (R1, R4, R5, R7, R10, R15, R17, R18, R20 and R25) were chosen for SNP validation by sequencing, as these lines, according to the GoldenGate assay, displayed different combinations of the 5 SNPs in homozygote or heterozygote state.<figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 1</span>
                        <p xmlns="" class="SimplePara">Outer and inner primer for tetra markers for rice. The outer primers were also used for producing PCR fragments for CEL-I genotyping. The fragment length refers to the fragments produced with the outer primers</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col></col><col></col><col></col><col></col><col></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Locus name</p></th><th><p xmlns="" class="SimplePara">Primer name</p></th><th><p xmlns="" class="SimplePara">Frag. length</p></th><th><p xmlns="" class="SimplePara">Outer Fwd</p></th><th><p xmlns="" class="SimplePara">Inner Fdw</p></th><th><p xmlns="" class="SimplePara">Inner Rev</p></th><th><p xmlns="" class="SimplePara">Outer Rev</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">4:22435296</p></td><td><p xmlns="" class="SimplePara">Tetra_1</p></td><td><p xmlns="" class="SimplePara">310</p></td><td><p xmlns="" class="SimplePara">GGAACATGTCGCATTCACAGT</p></td><td><p xmlns="" class="SimplePara">GCAGGTAAAACAGGTTAGG</p></td><td><p xmlns="" class="SimplePara">AATTGATGCCTGACTGGCTGACT</p></td><td><p xmlns="" class="SimplePara">CGACAGGGAGAAATCCTGAA</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">5:44806</p></td><td><p xmlns="" class="SimplePara">Tetra_2</p></td><td><p xmlns="" class="SimplePara">302</p></td><td><p xmlns="" class="SimplePara">ACACATCGATCCAGCTTACCA</p></td><td><p xmlns="" class="SimplePara">TAAGCTTCAGGCTCTAGCTTC</p></td><td><p xmlns="" class="SimplePara">GTGAGAAAGTAAAGGAC</p></td><td><p xmlns="" class="SimplePara">GCCTTGCAGGCTCCTATTTT</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">6:14504992</p></td><td><p xmlns="" class="SimplePara">Tetra_3</p></td><td><p xmlns="" class="SimplePara">288</p></td><td><p xmlns="" class="SimplePara">CGCCAAGAAAATGTGCAGTA</p></td><td><p xmlns="" class="SimplePara">TTCTCCTTGTGCCAGATAG</p></td><td><p xmlns="" class="SimplePara">CACCACGGAGGTTTCTAAT</p></td><td><p xmlns="" class="SimplePara">CCTTGCCCTGTTATTCCTGA</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">7:3650191</p></td><td><p xmlns="" class="SimplePara">Tetra_4</p></td><td><p xmlns="" class="SimplePara">222</p></td><td><p xmlns="" class="SimplePara">AGTGCCTGCATGATTGAACA</p></td><td><p xmlns="" class="SimplePara">CTGAAGTGAAAACAGAGC</p></td><td><p xmlns="" class="SimplePara">GTTTATTTCAAACGGGCG</p></td><td><p xmlns="" class="SimplePara">CAAGTTTGTAGGTGCTGACCA</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">9:6982338</p></td><td><p xmlns="" class="SimplePara">Tetra_5</p></td><td><p xmlns="" class="SimplePara">201</p></td><td><p xmlns="" class="SimplePara">GCGGAATTACACTGTTTTTGG</p></td><td><p xmlns="" class="SimplePara">TGTACTTGTATTTGCCGCTC</p></td><td><p xmlns="" class="SimplePara">CCCTAAGATGTCTTAAAAATA</p></td><td><p xmlns="" class="SimplePara">TTCTTGGGGTTGGCTACAAG</p></td></tr></tbody></table></div></figure>
                        </div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par22" class="Para">DNA of mungbean (<em xmlns="" class="EmphasisTypeItalic">Vigna radiata</em>) lines V2802, NM92, NM94 and of the wild mungbean line TC1966 (<em xmlns="" class="EmphasisTypeItalic">Vigna radiata</em> var. <em xmlns="" class="EmphasisTypeItalic">sublobata</em>)<em xmlns="" class="EmphasisTypeItalic">,</em> from F<sub>7</sub> progenies of V2802 x NM94 and F<sub>12</sub> progenies of TC1966 x NM92 was received from the World Vegetable Center mungbean breeding program. Putative SNP markers were identified in populations V2802 x NM94 and TC1966 x NM92 by genotyping by sequencing [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>]. Eight putative SNPs detected in this effort were selected at random for testing tetra markers and CEL-I genotyping (Table&nbsp;<span class="InternalRef"><a href="#Tab2">2</a></span>). The SNPs that were verified by sequencing, tetra markers and CEL-I on the mapping parents, were validated in 139&nbsp;F<sub>7</sub> progenies of V2802 x NM94 and 61&nbsp;F<sub>12</sub> offspring of TC1966 x NM92.<figure xmlns="" id="Tab2" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 2</span>
                        <p xmlns="" class="SimplePara">Outer and inner primer for tetra markers for mungbean. The outer primer were also used for producing PCR fragments for CEL-I genotyping. The fragment length refers to the fragments produced with the outer primers</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col></col><col></col><col></col><col></col><col></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Locus name</p></th><th><p xmlns="" class="SimplePara">Primer name</p></th><th><p xmlns="" class="SimplePara">Frag. length</p></th><th><p xmlns="" class="SimplePara">Outer Fwd</p></th><th><p xmlns="" class="SimplePara">Inner Fdw</p></th><th><p xmlns="" class="SimplePara">Inner Rev</p></th><th><p xmlns="" class="SimplePara">Outer Rev</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">1:26,370,595</p></td><td><p xmlns="" class="SimplePara">Tetra_6</p></td><td><p xmlns="" class="SimplePara">208</p></td><td><p xmlns="" class="SimplePara">CCGAAGATGTGTGATTCATG</p></td><td><p xmlns="" class="SimplePara">GACGATACTTGTCCAGATAT</p></td><td><p xmlns="" class="SimplePara">GAAGGGATTTTGTTAGGAGT</p></td><td><p xmlns="" class="SimplePara">AGCACTCAAGATGAAAGTGATC</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">2:23,741,639</p></td><td><p xmlns="" class="SimplePara">Tetra_7</p></td><td><p xmlns="" class="SimplePara">260</p></td><td><p xmlns="" class="SimplePara">ACTATCTGACCGAAAGGAA</p></td><td><p xmlns="" class="SimplePara">TTGGTACCAAATTCTGCACT</p></td><td><p xmlns="" class="SimplePara">ATCTTACGGTGAAGGACATT</p></td><td><p xmlns="" class="SimplePara">ATCTGCTGACAGGAGAATTCA</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">3:11,561,441</p></td><td><p xmlns="" class="SimplePara">Tetra_8</p></td><td><p xmlns="" class="SimplePara">236</p></td><td><p xmlns="" class="SimplePara">CCCCTAAAGCTGGAATATA</p></td><td><p xmlns="" class="SimplePara">TCTTCAAAGAGCTCCTTTTG</p></td><td><p xmlns="" class="SimplePara">GATTGTCACTTTGTCAAAG</p></td><td><p xmlns="" class="SimplePara">TGATCATCAAGCTACACTTC</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">3:10,830,938</p></td><td><p xmlns="" class="SimplePara">Tetra_9</p></td><td><p xmlns="" class="SimplePara">275</p></td><td><p xmlns="" class="SimplePara">AGCGAGGACAAGGAAGAATT</p></td><td><p xmlns="" class="SimplePara">TCTGCGGTGTTCTTAGAGT</p></td><td><p xmlns="" class="SimplePara">TTCCTCCAAATGCTGCAGAC</p></td><td><p xmlns="" class="SimplePara">GGCAAATTATTAGGATCCTGC</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">5:9,090,455</p></td><td><p xmlns="" class="SimplePara">Tetra_10</p></td><td><p xmlns="" class="SimplePara">250</p></td><td><p xmlns="" class="SimplePara">TCGCTTAACCAAGAATCGC</p></td><td><p xmlns="" class="SimplePara">CCAATGCGAATCAGATG</p></td><td><p xmlns="" class="SimplePara">AAAACGCACATCTGGTTCGTGT</p></td><td><p xmlns="" class="SimplePara">CGACATGGTGGCAATTTGAT</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">7:13,713,780</p></td><td><p xmlns="" class="SimplePara">Tetra_11</p></td><td><p xmlns="" class="SimplePara">253</p></td><td><p xmlns="" class="SimplePara">TAGCTGGTCCGTGTACTTTA</p></td><td><p xmlns="" class="SimplePara">TTTCCATTGTGGGTCGTGGAGT</p></td><td><p xmlns="" class="SimplePara">GCAGCAACATGTGCAACAT</p></td><td><p xmlns="" class="SimplePara">GGAACTATGCTTTGGGACTT</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">10:3,159,416</p></td><td><p xmlns="" class="SimplePara">Tetra_12</p></td><td><p xmlns="" class="SimplePara">269</p></td><td><p xmlns="" class="SimplePara">ATACTGGAGGGTTGTTTCTA</p></td><td><p xmlns="" class="SimplePara">TAAGCGTGCGCAGCCATAAACAAG</p></td><td><p xmlns="" class="SimplePara">GGGTTTTTTCGGAAATTCAAAG</p></td><td><p xmlns="" class="SimplePara">CGGTCTCAGAATCATAGTCTTG</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">12:9,262,432</p></td><td><p xmlns="" class="SimplePara">Tetra_13</p></td><td><p xmlns="" class="SimplePara">215</p></td><td><p xmlns="" class="SimplePara">AACCCCTTTATATAGGGTCTG</p></td><td><p xmlns="" class="SimplePara">TCATCTTCTCCGGGCAGCTCCTTC</p></td><td><p xmlns="" class="SimplePara">GAGGTGAAGGCACGTGACCA</p></td><td><p xmlns="" class="SimplePara">TCATTCGAGGAAAGGAGAGA</p></td></tr></tbody></table></div></figure>
                        </div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par23" class="Para">DNA extraction for sequencing, analysis by allele-specific primers, tetra primers and CEL-I was done from young leaves using the CTAB protocol described by [<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>]. PCR-fragments for sequencing were produced from DNA of the rice and mungbean genotypes using the outer primers listed in Tables&nbsp;<span class="InternalRef"><a href="#Tab1">1</a></span> and <span class="InternalRef"><a href="#Tab2">2</a></span>. The PCR fragments were submitted to Sanger sequencing at Genomics Ltd, Taiwan and the forward and reverse sequence reads were assembled and analyzed in DNAStar version 7.1.</p>
              </section><section id="Sec4" class="Section2 RenderAsSection2">
                <h3 class="Heading">Allele-specific PCR</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par24" class="Para">The allele-specific PCR technique was applied as described by [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>]. This method uses three primers: a common forward primer and two allele-specific reverse primers, where the 3′ prime end is specific for a SNP allele. In addition, each of the allele-specific primers contained a mismatch base near the 3′ end to destabilize the hybridization with the target sequence and increase the specifity of the allele-specific primer. One allele-specific primer contained a five base random extension at the 5′ site, and the primer for the second allele contained a 15 base extension, where the five 5′ bases corresponded to the extension of the first primer. The different length of the primers should result in size differences of the amplification products derived from the different alleles visible after gel electrophoresis of the PCR fragments. Primers were designed after retrieving the SNP sequence context in the <em xmlns="" class="EmphasisTypeItalic">O. sativa</em> reference at <span class="ExternalRef"><a href="http://rice.plantbiology.msu.edu/cgi-bin/gbrowse/rice/"><span class="RefSource">http://rice.plantbiology.msu.edu/cgi-bin/gbrowse/rice/</span></a></span> using Primer3 <span class="ExternalRef"><a href="http://bioinfo.ut.ee/primer3-0.4.0/"><span class="RefSource">http://bioinfo.ut.ee/primer3-0.4.0/</span></a></span>. The primer sequences and the amplicon sizes are shown in Table&nbsp;<span class="InternalRef"><a href="#Tab3">3</a></span>. PCR was performed on 20&nbsp;ng genomic DNA in 15&nbsp;μl reactions containing 0.2&nbsp;μM of each primer, 200&nbsp;μM of deoxyribonucleotides, 50&nbsp;mM KCl, 10&nbsp;mM Tris HCl (pH&nbsp;8.3), 1.5&nbsp;mM MgCl<sub>2</sub> and 0.5 units of Taq DNA polymerase on a DNA Gradient PCR machine (BIORAD) with an amplification profile of initial denaturation at 95&nbsp;°C for 10&nbsp;min, followed by 35&nbsp;cycles with 95&nbsp;°C for 30&nbsp;s, annealing at an assay-specific temperature from 48&nbsp;°C to 65&nbsp;°C for 45&nbsp;s, elongation at 72&nbsp;°C for 45&nbsp;s and terminal elongation at 72&nbsp;°C for 5&nbsp;min. PCR products (2&nbsp;μL/sample) were analyzed on 6% polyacrylamide gel and visualized after ethidium bromide staining under UV light.<figure xmlns="" id="Tab3" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 3</span>
                        <p xmlns="" class="SimplePara">Allele-specific primers for rice</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col></col><col></col><col></col><col></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Locus name</p></th><th><p xmlns="" class="SimplePara">Primer name</p></th><th><p xmlns="" class="SimplePara">Fragment length</p></th><th><p xmlns="" class="SimplePara">Common primer</p></th><th><p xmlns="" class="SimplePara">Allele-specific primer 1</p></th><th><p xmlns="" class="SimplePara">Allele-specific primer 2</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">4:22435296</p></td><td><p xmlns="" class="SimplePara">AS1</p></td><td><p xmlns="" class="SimplePara">219</p></td><td><p xmlns="" class="SimplePara">CGACAGGGAGAAATCCTGAA</p></td><td><p xmlns="" class="SimplePara">5’-GACTTGGCAGGTAAAACAGGTTA<span class="EmphasisTypeUnderline">CG</span>-3'</p></td><td><p xmlns="" class="SimplePara">5’-CACTACAAAGGCTGTGGCAGGTAAAACAGGAT<span class="EmphasisTypeUnderline">A</span>G<span class="EmphasisTypeUnderline">A</span>-3'</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">5:44806</p></td><td><p xmlns="" class="SimplePara">AS2</p></td><td><p xmlns="" class="SimplePara">189</p></td><td><p xmlns="" class="SimplePara">GCCTTGCAGGCTCCTATTTT</p></td><td><p xmlns="" class="SimplePara">5’-AGTCAAAGCTTCAGGCTCTAGCT<span class="EmphasisTypeUnderline">GC</span>-3’</p></td><td><p xmlns="" class="SimplePara">5’-CGACTAACTGTGACTAAGCTTCAGGCTCTAGC<span class="EmphasisTypeUnderline">A</span>T<span class="EmphasisTypeUnderline">A</span>-3’</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">6:14504992</p></td><td><p xmlns="" class="SimplePara">AS3</p></td><td><p xmlns="" class="SimplePara">209</p></td><td><p xmlns="" class="SimplePara">CCTTGCCCTGTTATTCCTGA</p></td><td><p xmlns="" class="SimplePara">5’-CCAGTTTTCTCCTTGTGCCAGAT<span class="EmphasisTypeUnderline">TG</span>-3’</p></td><td><p xmlns="" class="SimplePara">5’-GATGTCTACTCCAGTTTTCTCCTTGTGCCAGA<span class="EmphasisTypeUnderline">G</span>A<span class="EmphasisTypeUnderline">A</span>-3’</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">7:3650191</p></td><td><p xmlns="" class="SimplePara">AS4</p></td><td><p xmlns="" class="SimplePara">293</p></td><td><p xmlns="" class="SimplePara">CAAGTTTGTAGGTGCTGACCA</p></td><td><p xmlns="" class="SimplePara">5’-GATCCTAACTGAAGTGAAAACAG<span class="EmphasisTypeUnderline">CA</span>3’</p></td><td><p xmlns="" class="SimplePara">5’-TGCGTCATAGGATCCTAACTGAAGTGAAAACA<span class="EmphasisTypeUnderline">C</span>A<span class="EmphasisTypeUnderline">C</span>-3’</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">9:6982338</p></td><td><p xmlns="" class="SimplePara">AS5</p></td><td><p xmlns="" class="SimplePara">162</p></td><td><p xmlns="" class="SimplePara">TTCTTGGGGTTGGCTACAAG</p></td><td><p xmlns="" class="SimplePara">5’-ACTGCTGTACTTGTATTTGCCGC<span class="EmphasisTypeUnderline">AC</span>-3’</p></td><td><p xmlns="" class="SimplePara">5’-CATTGTATTTTTACGTGTACTTGTATTTGCCG<span class="EmphasisTypeUnderline">A</span>T<span class="EmphasisTypeUnderline">T</span>-3</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">The allele-specific base, as well as the additionally introduced variant base near the 3’end are underlined. The primer extension sequences are printed in small letters. The fragment length excludes the extensions of the allele-specific primers</p>
                    </div></figure>
                        </div>
              </section><section id="Sec5" class="Section2 RenderAsSection2">
                <h3 class="Heading">Tetra marker</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par25" class="Para">Outer and inner primers for tetra markers were designed in Primer3 (<span class="ExternalRef"><a href="http://bioinfo.ut.ee/primer3-0.4.0/"><span class="RefSource">http://bioinfo.ut.ee/primer3-0.4.0/</span></a></span>) using the rice and mungbean reference sequences (<span class="ExternalRef"><a href="http://rice.plantbiology.msu.edu/cgi-bin/gbrowse/rice/"><span class="RefSource">http://rice.plantbiology.msu.edu/cgi-bin/gbrowse/rice/</span></a></span> and <span class="ExternalRef"><a href="http://plantgenomics.snu.ac.kr/mediawiki-1.21.3/index.php/Main_Page"><span class="RefSource">http://plantgenomics.snu.ac.kr/mediawiki-1.21.3/index.php/Main_Page</span></a></span>). The additional mismatch base usually introduced for tetra ARMS PCR [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>] was not applied in our experiment. The outer and inner primers of the tetra markers are listed in Tables&nbsp;<span class="InternalRef"><a href="#Tab1">1</a></span> and <span class="InternalRef"><a href="#Tab2">2</a></span>. The PCR reactions were set up as described for allele-specific primers. The amplification profile was 94&nbsp;°C for 5&nbsp;min, followed by 30&nbsp;cycles of 94&nbsp;°C for 30&nbsp;s, 58&nbsp;°C for 45&nbsp;s, 72&nbsp;°C for 45&nbsp;s, and final extension for 7&nbsp;min at 72&nbsp;°C. PCR products (3&nbsp;μl) were size-fractionated on 6% non-denaturing polyacrylamide gels in 0.5 × TBE buffer. After electrophoresis, the gels were stained with 5&nbsp;μg/mL<sup>−1</sup> ethidium bromide and the bands were visualized under ultraviolet light. Tetra markers were tested on five SNPs of rice landraces and on eight SNPs of mungbean mapping parents.</p>
              </section><section id="Sec6" class="Section2 RenderAsSection2">
                <h3 class="Heading">CEL-I SNP genotyping method</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par26" class="Para">500&nbsp;g celery was purchased at a local farmer’s market. Leaves were removed and the stems were blended in a juicer. CEL-I was enriched through (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> precipitation as described by [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>]. Desalted CEL-I extract was stored in aliquots at −80&nbsp;°C. Test digestions using 0.5 to 2.5&nbsp;μl CEL-I extract showed that 1&nbsp;μl were optimal for genotyping. The CEL-I working solution for each reaction contained 1&nbsp;μL CEL-I enzyme, 1.5&nbsp;μL CEL-I buffer and 7.5&nbsp;μL distilled deionized water. 10× CEL-I buffer contained 1&nbsp;M MgSO<sub>4</sub>, 1&nbsp;M HEPES pH&nbsp;7.5, 2&nbsp;M KCl, 10% Triton X-100, and 20&nbsp;mg/ml BSA. PCR fragments produced with the outer primers of the tetra markers described above were used for CEL-I genotyping. PCR products derived from different plant lines were mixed in a ratio of a 1:1 ratio (v/v). For genotyping progenies of mapping parents, two assays per sample were prepared. In one assay the DNA of the progeny was mixed with DNA of mapping parent one, and in the second assay it was mixed with DNA of mapping parent two. For genotyping mapping parents, also two assays per locus were performed. In one assay the DNA of one mapping parent was mixed with the DNA of the other parent, and in the second assay the DNA of each mapping parent was genotyped separately. When the SNP base was identical with the base present in the DNA mixed with the sample, no cut by CEL-I was obtained, while in the presence of one or more polymorphic bases, CEL-I could introduce a cut into the fragment. This way, the reference, variant or heterozygote genotype at SNP base located in the tested PCR fragments could be determined. For the assay, the DNA double strands were denaturated by incubation at 99&nbsp;°C for 10&nbsp;min, reannealing the stands by cooling down to 70&nbsp;°C, and subsequently reducing the incubation temperature from 70&nbsp;°C to 49&nbsp;°C in 70&nbsp;cycles with −0.3&nbsp;°C intervals per 20&nbsp;s. 10&nbsp;μl ice-cold CEL-I working solution was added to the sample on ice, mixed and incubated at different temperatures. For PCR fragments of less than 300&nbsp;bp, the usually suggested CEL-I treatment incubation temperature of 45&nbsp;°C was too high and yielded strong background and weak bands, while a temperature of 36&nbsp;°C seemed to be suitable to yield bands after specifically cutting mismatch bases (Fig.&nbsp;<span class="InternalRef"><a href="#Fig1">1</a></span>). For the CEL-I genotyping in the current study, the samples were incubated at 34&nbsp;°C for 10&nbsp;min and then cooled down to −20&nbsp;°C. Electrophoresis and DNA fragment visualization was performed as described for tetra markers. A simplified version of CEL-I digestion was performed by performing PCR on DNA mixtures, rather than mixing fragments post-PCR.<figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO1">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-016-0024-y/MediaObjects/41065_2016_24_Fig1_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-016-0024-y/MediaObjects/41065_2016_24_Fig1_HTML.gif" alt="Fig. 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0024-y/MediaObjects/41065_2016_24_Fig1_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0024-y/MediaObjects/41065_2016_24_Fig1_HTML.gif" alt="Fig. 1"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 1</span>
                        <p xmlns="" class="SimplePara">Optimization of the incubation temperature for CEL-I treatment. The SNP tested was tetra_12. Incubation at 36 ° was found to be optimal to specifically cut at the mismatch site in position 171&nbsp;bp of the 269&nbsp;bp fragment, producing bands with 170 and 99&nbsp;bp. The additional smaller band below around 90&nbsp;bp is probably due to the sequence context around the SNP (please refer to the discussion)</p>
                      </div></figcaption></figure>
                        </div>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec7" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Results" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Results">Results</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Results" data-test="collapse-section"><section id="Sec8" class="Section2 RenderAsSection2">
                <h3 class="Heading">SNP validation in rice by sequencing and testing allele-specific PCR, tetra markers and CEL-I genotyping</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par27" class="Para">In total, five SNPs detected through the 384-SNP GoldenGate (<em xmlns="" class="EmphasisTypeItalic">Indica</em> x <em xmlns="" class="EmphasisTypeItalic">Indica</em>) array were selected for verification. Before testing PCR-based genotyping on these SNPs, the SNP data obtained from the GoldenGate array were verified by sequencing on 11 rice landraces (Table&nbsp;<span class="InternalRef"><a href="#Tab4">4</a></span>).<figure xmlns="" id="Tab4" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 4</span>
                        <p xmlns="" class="SimplePara">SNPs among rice landraces selected from Golden Gate genotyping data</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col></colgroup><thead><tr class="header"><th colspan="4"><p xmlns="" class="SimplePara">Golden Gate data</p></th><th colspan="11"><p xmlns="" class="SimplePara">Sequencing data</p></th></tr><tr class="header"><th><p xmlns="" class="SimplePara">Locus name</p></th><th><p xmlns="" class="SimplePara">Primer name</p></th><th><p xmlns="" class="SimplePara">Ref</p></th><th><p xmlns="" class="SimplePara">Var</p></th><th><p xmlns="" class="SimplePara">R1</p></th><th><p xmlns="" class="SimplePara">R4</p></th><th><p xmlns="" class="SimplePara">R5</p></th><th><p xmlns="" class="SimplePara">R7</p></th><th><p xmlns="" class="SimplePara">R10</p></th><th><p xmlns="" class="SimplePara">R15</p></th><th><p xmlns="" class="SimplePara">R17</p></th><th><p xmlns="" class="SimplePara">R18</p></th><th><p xmlns="" class="SimplePara">R20</p></th><th><p xmlns="" class="SimplePara">R21</p></th><th><p xmlns="" class="SimplePara">R25</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">4:22435296</p></td><td><p xmlns="" class="SimplePara">tetra_1</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">5:44806</p></td><td><p xmlns="" class="SimplePara">tetra_2</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">S</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">C</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">6:14504992</p></td><td><p xmlns="" class="SimplePara">tetra_3</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">R</p></td><td><p xmlns="" class="SimplePara">A</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">7:3650191</p></td><td><p xmlns="" class="SimplePara">tetra_4</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">9:6982338</p></td><td><p xmlns="" class="SimplePara">tetra_5</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">C</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">Ref.: reference sequence, Var.: variant. R1 – R25: rice landrace accession number</p>
                    </div></figure>
                        </div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par28" class="Para">Allele-specific PCR, in addition to the allele-specific bands, yielded several secondary bands for tetra_1, but still genotypes were predicted correctly except for one case (R14). The allele-specific markers for tetra _3 predicted heterozygote genotypes for 21 out of 26 genotypes (Fig.&nbsp;<span class="InternalRef"><a href="#Fig2">2</a></span>) and revealed the correct genotype only in one sample (R25). The other allele-specific primers did not produce bands that distinguished the SNP alleles (data not shown). This result showed that allele-specific primers as designed for this experiment are not useful for genotyping SNPs in rice samples. Allele-specific primers were not tested on mungbean.<figure xmlns="" class="Figure" id="Fig2"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO2">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-016-0024-y/MediaObjects/41065_2016_24_Fig2_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-016-0024-y/MediaObjects/41065_2016_24_Fig2_HTML.gif" alt="Fig. 2" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0024-y/MediaObjects/41065_2016_24_Fig2_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0024-y/MediaObjects/41065_2016_24_Fig2_HTML.gif" alt="Fig. 2"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 2</span>
                        <p xmlns="" class="SimplePara">Genotyping of rice using allele-specific markers. <strong class="EmphasisTypeBold">a</strong>) Tetra_1, <strong class="EmphasisTypeBold">b</strong>) Tetra_3. The SNP bases are indicated at the bottom of the gel images. The top row shows the allele-specific PCR results, and the SNP array genotyping or sequencing results are shown in the bottom row</p>
                      </div></figcaption></figure>
                        </div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par29" class="Para">Testing tetra markers on these SNPs showed that markers tetra_1, 2 and 3 had the tendency to show heterozygote genotypes, although the accessions were homozygote at the tested loci. The A variant for the G/A SNP in tetra_1 was correctly recognized by tetra markers, while most G variants were erroneously genotyped as G/A heterozygotes (Fig.&nbsp;<span class="InternalRef"><a href="#Fig3">3</a></span>). The G/A SNP of tetra_1 was located between G residues, which might have affected the specific initiation of amplification by primer for the G-allele. The SNP in tetra_2 was located in a repeat sequence (AGCTTG), and the C-specific primer initiated polymerase chain reaction at the target site and at the second AGCTTG motif located 13&nbsp;bp upstream the target sequence in all samples with the C allele present. In addition, the inner primer for the G allele was not specific and amplified a fragment in all samples suggesting erroneous heterozygote genotypes. Tetra marker tetra_3 gave in many cases erroneous heterozygote genotypes, as the inner reverse primer specific for the A allele yielded products, albeit at low amounts, when only the G allele was present. Only tetra markers tetra_4 gave the correct genotypes and detected the C-variant in R1. Tetra_5 scored seven homozygous samples. In contrast, CEL-I genotyping reliably detected all heterozygote and homozygote SNPs correctly (Fig.&nbsp;<span class="InternalRef"><a href="#Fig4">4</a></span>). The G/A SNP between R4 and R15, as well as the heterozygote G/C in R7 and the G/C SNP between R18 and R1 were correctly detected. The same with tetra_3, the heterozygote base in R21 and the SNP among lines R10 and R25 were detected. Also the SNPs in tetra 4 between R1 and R25 and R1 and R20 were correctly detected. Tetra-5 did not contain any SNP, therefore no CEL-I cut was observed in this fragment.<figure xmlns="" class="Figure" id="Fig3"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO3">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-016-0024-y/MediaObjects/41065_2016_24_Fig3_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-016-0024-y/MediaObjects/41065_2016_24_Fig3_HTML.gif" alt="Fig. 3" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0024-y/MediaObjects/41065_2016_24_Fig3_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0024-y/MediaObjects/41065_2016_24_Fig3_HTML.gif" alt="Fig. 3"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 3</span>
                        <p xmlns="" class="SimplePara">Genotyping selected rice accessions by tetra markers: <strong class="EmphasisTypeBold">a</strong>) tetra_1, <strong class="EmphasisTypeBold">b</strong>) tetra_2 and <strong class="EmphasisTypeBold">c</strong>) tetra_3, <strong class="EmphasisTypeBold">d</strong>) tetra_4, <strong class="EmphasisTypeBold">e</strong>) terta_5. The SNP bases indicated by the allele specific PCR detected by SNP array genotyping or sequencing are indicated on the gel pictures</p>
                      </div></figcaption></figure>
                           <figure xmlns="" class="Figure" id="Fig4"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO4">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-016-0024-y/MediaObjects/41065_2016_24_Fig4_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-016-0024-y/MediaObjects/41065_2016_24_Fig4_HTML.gif" alt="Fig. 4" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0024-y/MediaObjects/41065_2016_24_Fig4_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0024-y/MediaObjects/41065_2016_24_Fig4_HTML.gif" alt="Fig. 4"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 4</span>
                        <p xmlns="" class="SimplePara">CEL-I genotyping of rice accessions with tetra_1 (<strong class="EmphasisTypeBold">a</strong>), tetra_2 (<strong class="EmphasisTypeBold">b</strong>), tetra_3 (<strong class="EmphasisTypeBold">c</strong>),tetra_4 (<strong class="EmphasisTypeBold">d</strong>), and tetra_5 (<strong class="EmphasisTypeBold">e</strong>). The fragments produced by CEL-I by cutting at heterozygous loci (b for R7, c for R21) and at SNP loci are indicated with arrows and listed in Table&nbsp;<span xmlns="http://www.w3.org/1999/xhtml" class="InternalRef"><a href="#Tab4">4</a></span>
                                    </p>
                      </div></figcaption></figure>
                        </div>
              </section><section id="Sec9" class="Section2 RenderAsSection2">
                <h3 class="Heading">SNP validation in mungbean by sequencing and testing tetra markers and CEL-I genotyping</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par30" class="Para">In total 8 putative SNPs obtained from GBS (Table&nbsp;<span class="InternalRef"><a href="#Tab5">5</a></span>) were chosen for validation through sequencing, and testing of tetra-markers and CEL-I analysis. Sequencing confirmed the SNPs for tetra_6, 7, 9, 12 and 13, but not for tetra_8, 10 and 11 (Table&nbsp;<span class="InternalRef"><a href="#Tab5">5</a></span>). In addition to the SNPs previously found by genotyping by sequencing, Sanger sequencing of the PCR fragments produced with the outer tetra primers resulted in additional SNPs for tetra 6, 7, 9, 10 and 13. The outer primers for tetra_11 did not yield any product from TC1966 and therefore could not be sequenced. Tetra-11 was monomorphic between NM94 and V2802 and tetra_8 was monomorphic between all mapping parents. The amplicon of tetra_10 contained several heterozygous sites in all mapping parents, but sequencing did not corroborate the presence of the G/A polymorphism predicted by GBS. This sequencing exercise showed that 3 out of the 8 selected SNPs predicted by GBS were erroneous, probably due to wrong mapping of GBS tags to the reference sequence, and due to insufficient sequencing depth.<figure xmlns="" id="Tab5" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 5</span>
                        <p xmlns="" class="SimplePara">Mungbean SNPs detected by genotyping by sequencing among the mapping parents NM92, NM94, TC1966 and V2802 and targeted for genotyping by tetra markers and CEL-I</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col></colgroup><thead><tr class="header"><th colspan="7"><p xmlns="" class="SimplePara">GBS data</p></th><th colspan="4"><p xmlns="" class="SimplePara">Sequencing data</p></th></tr><tr class="header"><th><p xmlns="" class="SimplePara">Locus name</p></th><th><p xmlns="" class="SimplePara">Primer name</p></th><th><p xmlns="" class="SimplePara">SNP Position in PCR fragment</p></th><th><p xmlns="" class="SimplePara">NM92</p></th><th><p xmlns="" class="SimplePara">TC1966</p></th><th><p xmlns="" class="SimplePara">NM94</p></th><th><p xmlns="" class="SimplePara">V2802</p></th><th><p xmlns="" class="SimplePara">NM92</p></th><th><p xmlns="" class="SimplePara">TC1966</p></th><th><p xmlns="" class="SimplePara">NM94</p></th><th><p xmlns="" class="SimplePara">V2802</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">1:26,370,595</p></td><td><p xmlns="" class="SimplePara">tetra_6</p></td><td><p xmlns="" class="SimplePara">144</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">2:23,741,639</p></td><td><p xmlns="" class="SimplePara">tetra_7</p></td><td><p xmlns="" class="SimplePara">181</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">A</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">3:11,561,441</p></td><td><p xmlns="" class="SimplePara">tetra_8</p></td><td><p xmlns="" class="SimplePara">-</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">C</strong>
                            </p></td><td><p xmlns="" class="SimplePara">c</p></td><td><p xmlns="" class="SimplePara">c</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">3:10,830,938</p></td><td><p xmlns="" class="SimplePara">tetra_9</p></td><td><p xmlns="" class="SimplePara">200</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">5:9,090,455</p></td><td><p xmlns="" class="SimplePara">tetra_10</p></td><td><p xmlns="" class="SimplePara">-</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G,</p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">G</strong>
                            </p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">G</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">7:13,713,780</p></td><td><p xmlns="" class="SimplePara">tetra_11</p></td><td><p xmlns="" class="SimplePara">-</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">A</strong>
                            </p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">10:3,159,416</p></td><td><p xmlns="" class="SimplePara">tetra_12</p></td><td><p xmlns="" class="SimplePara">171</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">G</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">12:9,262,432</p></td><td><p xmlns="" class="SimplePara">tetra_13</p></td><td><p xmlns="" class="SimplePara">140</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">C</p></td><td><p xmlns="" class="SimplePara">C</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">The numbers refer to the position of the SNP in the PCR fragment obtained from the mapping parents using the outer tetra primers. SNPs that could not be validated by sequencing are labeled in bold</p>
                    </div></figure>
                        </div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par31" class="Para">All SNPs, including those that could not be confirmed by Sanger sequencing were submitted to testing by tetra marker and CEL-I genotyping. Tetra markers applied without an additional mismatch base genotyped four out of the five sequence-corroborated SNPs correctly. Only for tetra_13 did the tetra marker fail to reveal the correct genotype (Fig.&nbsp;<span class="InternalRef"><a href="#Fig5">5</a></span>). Genotyping of F7 populations TC1966 x NM92 and/or V2802 x NM94 with tetra_6, 7, 9 and 12 showed the segregation of the SNP genotype in the populations (Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>) and suggested that the tetra markers were appropriate for genotyping populations.<figure xmlns="" class="Figure" id="Fig5"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO5">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-016-0024-y/MediaObjects/41065_2016_24_Fig5_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-016-0024-y/MediaObjects/41065_2016_24_Fig5_HTML.gif" alt="Fig. 5" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0024-y/MediaObjects/41065_2016_24_Fig5_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0024-y/MediaObjects/41065_2016_24_Fig5_HTML.gif" alt="Fig. 5"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 5</span>
                        <p xmlns="" class="SimplePara">Test of tetra markers tetra-6, 7, 8, 9, 10 11, 12 and 13 on 4 mungbean mapping parents TV1966, NM92, V2802 and NM94. The bands indicating the SNP base are labeled with arrows</p>
                      </div></figcaption></figure>
                        </div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par32" class="Para">The outer primers of the tetra markers were used to amplify fragments from genomic DNA of the mapping parents containing the putative SNP sites. PCR amplification products from TC1966 were pooled with those from NM92, and products from NM94 were pooled with those from V2802. The pure samples and the mixtures were submitted to CEL-I treatment. In un-pooled PCR fragments obtained from the four mapping parents, CEL-I cutting was observed at the heterozygous bases in fragments of tetra_6, 7 and 8 in NM92 and tetra_10 in all mapping parents (Fig.&nbsp;<span class="InternalRef"><a href="#Fig6">6</a></span>, Table&nbsp;<span class="InternalRef"><a href="#Tab6">6</a></span>). High background was obtained with tetra_9 from all mapping parents. This fragment contained microsatellite-like motifs, which could have led to destabilization of the DNA double strand of the short fragments and could have transiently created cutting sites for CEL-I. Further reduction of the reaction temperature to avoid this effect was not tried. CEL-I treatment of mixtures of SNP-containing PCR fragments of the mapping parents resulted in the expected DNA fragments for all loci (Fig.&nbsp;<span class="InternalRef"><a href="#Fig6">6</a></span>, Tables&nbsp;<span class="InternalRef"><a href="#Tab5">5</a></span> and <span class="InternalRef"><a href="#Tab6">6</a></span>). In addition to the cuts at heterozygote sites, in mixtures of tetra_6 PCR fragments from NM92 and TC1966, an additional cut was produced at position 144 (Fig.&nbsp;<span class="InternalRef"><a href="#Fig6">6a</a></span>). In tetra_7, as expected from the sequencing data, the heterozygous site at position 181 of the PCR fragment from NM92 caused a CEL-I cut, resulting in a 181 and a 79&nbsp;bp band, while in mixtures of NM94/V2802 PCR fragments CEL-I genotyping correctly showed the T/A polymorphism in position 181 of the fragment (Fig.&nbsp;<span class="InternalRef"><a href="#Fig6">6b</a></span>, Tables&nbsp;<span class="InternalRef"><a href="#Tab5">5</a></span> and <span class="InternalRef"><a href="#Tab6">6</a></span>). The monomorphic locus tetra_8 remained un-cut by CEL-I, cuts were introduced at the heterozygous sites in NM92 (Fig.&nbsp;<span class="InternalRef"><a href="#Fig6">6c</a></span>, Table&nbsp;<span class="InternalRef"><a href="#Tab6">6</a></span>). Tetra_10 gave a complex pattern due to the presence of multiple SNPs in the PCR fragment (Table&nbsp;<span class="InternalRef"><a href="#Tab6">6</a></span>), while CEL-I restriction of tetra_11 fragments gave no cuts, as expected for these monomorphic fragments. Mixtures of fragments of tetra_12 from TC1966 and NM92, as well as from V2802 and NM94 gave the expected cuts at the SNP position 171 of the PCR fragment. Similarly mixtures of PCR fragments of tetra_13 from TC1966 and NM92 gave the expected pattern of cut bands at the SNP site.<figure xmlns="" class="Figure" id="Fig6"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO6">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-016-0024-y/MediaObjects/41065_2016_24_Fig6_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-016-0024-y/MediaObjects/41065_2016_24_Fig6_HTML.gif" alt="Fig. 6" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0024-y/MediaObjects/41065_2016_24_Fig6_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-016-0024-y/MediaObjects/41065_2016_24_Fig6_HTML.gif" alt="Fig. 6"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 6</span>
                        <p xmlns="" class="SimplePara">CEL-I test genotyping on pure and mixed DNA of 4 mungbean mapping parents for tetra_6 (<strong class="EmphasisTypeBold">a</strong>), tetra_7 (<strong class="EmphasisTypeBold">b</strong>), tetra_8 (<strong class="EmphasisTypeBold">c</strong>), tetra_9 (<strong class="EmphasisTypeBold">d</strong>), tetra_10 (<strong class="EmphasisTypeBold">e</strong>), tetra_11 (<strong class="EmphasisTypeBold">f</strong>), tetra_12 (<strong class="EmphasisTypeBold">g</strong>), and tetra_13 (<strong class="EmphasisTypeBold">h</strong>). CEL-I cuts at mismatch sites. Mismatches can appear at heterozygote sites in one genotype (as in NM92 with tetra_6 (<strong class="EmphasisTypeBold">a</strong>) in positions 60/65 and 94, or at SNP sites when PCR fragments from homozygote mapping parents are mixed, as in tetra_6 at position 144 of TC1966/NM92, or in tetra_12 (<strong class="EmphasisTypeBold">g</strong>). The SNPs among the mapping parents and the heterozygote loci causing the CEL-I fragments are labeled on the figures and listed in Tables&nbsp;<span xmlns="http://www.w3.org/1999/xhtml" class="InternalRef"><a href="#Tab5">5</a></span> and <span xmlns="http://www.w3.org/1999/xhtml" class="InternalRef"><a href="#Tab6">6</a></span>
                                    </p>
                      </div></figcaption></figure>
                           <figure xmlns="" id="Tab6" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 6</span>
                        <p xmlns="" class="SimplePara">Mungbean SNPs found by Sanger sequencing of PCR fragments produced with the outer tetra marker primers. These SNPs are located nearby the SNPs that were detected by genotyping by sequencing</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col></col><col></col><col></col><col></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Locus name</p></th><th><p xmlns="" class="SimplePara">Primer name</p></th><th><p xmlns="" class="SimplePara">NM92</p></th><th><p xmlns="" class="SimplePara">TC1966</p></th><th><p xmlns="" class="SimplePara">NM94</p></th><th><p xmlns="" class="SimplePara">V2802</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">1:26,370,595</p></td><td><p xmlns="" class="SimplePara">tetra_6</p></td><td><p xmlns="" class="SimplePara">60:K, 65:K, 94: V, 95:G</p></td><td><p xmlns="" class="SimplePara">64:T, 95:A</p></td><td><p xmlns="" class="SimplePara">none</p></td><td><p xmlns="" class="SimplePara">none</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">2:23,741,639</p></td><td><p xmlns="" class="SimplePara">tetra_7</p></td><td><p xmlns="" class="SimplePara">44:A, 127:T, 181: W</p></td><td><p xmlns="" class="SimplePara">44:T, 127: C</p></td><td><p xmlns="" class="SimplePara">127: C</p></td><td><p xmlns="" class="SimplePara">127: C,</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">3:11,561,441</p></td><td><p xmlns="" class="SimplePara">tetra_8</p></td><td><p xmlns="" class="SimplePara">56: M, 71:Y, 109:W</p></td><td><p xmlns="" class="SimplePara">none</p></td><td><p xmlns="" class="SimplePara">none</p></td><td><p xmlns="" class="SimplePara">none</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">3:10,830,938</p></td><td><p xmlns="" class="SimplePara">tetra_9</p></td><td><p xmlns="" class="SimplePara">107:A, 117:G</p></td><td><p xmlns="" class="SimplePara">107:G, 117:A</p></td><td><p xmlns="" class="SimplePara">86:A</p></td><td><p xmlns="" class="SimplePara">86:T</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">5:9,090,455</p></td><td><p xmlns="" class="SimplePara">tetra_10</p></td><td><p xmlns="" class="SimplePara">71:T, 77:T, 95:A, 118:G, 138:C, 213:Y,</p></td><td><p xmlns="" class="SimplePara">71:C, 77:Y, 95:S, 118:K, 138:Y, 213:C,</p></td><td><p xmlns="" class="SimplePara">71:Y, 77:Y, 95:A, 102:W, 149:S, 178:S, 213:Y, 225:C</p></td><td><p xmlns="" class="SimplePara">24:C, 71:Y, 77:Y, 168:W, 178:S</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">7:13,713,780</p></td><td><p xmlns="" class="SimplePara">tetra_11</p></td><td><p xmlns="" class="SimplePara">none</p></td><td><p xmlns="" class="SimplePara">none</p></td><td><p xmlns="" class="SimplePara">none</p></td><td><p xmlns="" class="SimplePara">none</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">10:3,159,416</p></td><td><p xmlns="" class="SimplePara">tetra_12</p></td><td><p xmlns="" class="SimplePara">none</p></td><td><p xmlns="" class="SimplePara">none</p></td><td><p xmlns="" class="SimplePara">none</p></td><td><p xmlns="" class="SimplePara">none</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">12:9,262,432</p></td><td><p xmlns="" class="SimplePara">tetra_13</p></td><td><p xmlns="" class="SimplePara">81:C, 93:T</p></td><td><p xmlns="" class="SimplePara">81:T, 93:C</p></td><td><p xmlns="" class="SimplePara">none</p></td><td><p xmlns="" class="SimplePara">none</p></td></tr></tbody></table></div></figure>
                        </div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par33" class="Para">CEL-I genotyping on populations was tested for tetra_7, 9 and 12 on F<sub>7</sub> families of V2802 x NM94 and for tetra_12 on F<sub>12</sub> families of populations TC1966 x NM92 (Additional file <span class="InternalRef"><a href="#MOESM2">2</a></span>). Instead of mixing the PCR fragments of the families with the fragments of the mapping parents, the PCR mastermix was spiked with either parent 1 or parent 2 DNA before amplification. For each sample, two amplification reactions were performed, one with parent 1 and another with parent 2-spiked DNA in order to detect heterozygous SNPs. Scoring of CEL-I genotyping was simple on amplicons with a single SNP like in tetra_12, but was also feasible when additional SNPs or heterozygous sites were present, like in tetra_9. By comparing the genotypes of the parent 1- and parent 2-spiked families, homozygote and heterozygote genotypes could be distinguished easily (Additional file <span class="InternalRef"><a href="#MOESM2">2</a></span>). For example, for tetra_9 samples 14 and 128 showed the cut bands in both, NM94 and V2802-spiked CEL_I reactions, indicating that the samples are heterozygote for the tetra_9 SNP (Additional file <span class="InternalRef"><a href="#MOESM2">2</a></span>).</p>
              </section><section id="Sec10" class="Section2 RenderAsSection2">
                <h3 class="Heading">Validation of SNP markers - Cost and working time considerations</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par34" class="Para">Using commercial SNP assays for marker validation is relatively expensive. For mapping experiments dealing with complex traits involving many loci and numerous markers associated with them, the validation costs for candidate markers may surpass the genotyping costs. Table&nbsp;<span class="InternalRef"><a href="#Tab7">7</a></span> shows a comparison of reagent costs and working time for different marker systems including CAPS (dCAPS), tetra markers, and CEL-I assays. Allele-specific markers were excluded from the calculation, as they did not perform sufficiently well in our experiments to be considered as a validation tool. DNA extraction costs were not considered, as these costs were equal for all listed assays. Many different restriction enzymes are required to cover SNPs in different sequence contexts, therefore developing new CAPS or dCAPS markers often requires purchasing new restriction enzymes. For the calculation in Table&nbsp;<span class="InternalRef"><a href="#Tab7">7</a></span> we assumed that for every third SNP a new kind of restriction enzyme has to be purchased. The cost of supplies to develop a marker assay are clearly highest for CAPS markers, followed by tetra markers and CEL-I genotyping. The time for designing and testing the markers is assumed to be higher for tetra and CAPS markers than for CEL-I genotyping. Mainly due to the restriction enzyme costs, CAPS development remains the most expensive system. For routine genotyping, CEL-I would be the most expensive and laborious option, as two assays must be performed for each locus to distinguish homozygote from heterozygote variants. Therefore, for marker validation on a small number of samples, where mostly marker development costs play a role, CEL-I genotyping seems to be cheapest; for genotyping a larger sample, tetra markers are more cost-effective.<figure xmlns="" id="Tab7" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 7</span>
                        <p xmlns="" class="SimplePara">Estimation of development and running costs for (d)CAPS, tetra marker and CEL-I-based genotyping (in US$)</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col></col><col></col></colgroup><thead><tr class="header"><th colspan="4"><p xmlns="" class="SimplePara">Development costs</p></th></tr><tr class="header"><th>&nbsp;</th><th><p xmlns="" class="SimplePara">CAPS (dCAPS)</p></th><th><p xmlns="" class="SimplePara">Tetra marker</p></th><th><p xmlns="" class="SimplePara">CEL-I</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">Primer</p></td><td><p xmlns="" class="SimplePara">7.5</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">7.5</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Restriction enzyme</p></td><td><p xmlns="" class="SimplePara">30</p></td><td>&nbsp;</td><td><p xmlns="" class="SimplePara">1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">PCR reagents</p></td><td><p xmlns="" class="SimplePara">0.48</p></td><td><p xmlns="" class="SimplePara">0.48</p></td><td><p xmlns="" class="SimplePara">0.96</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Gel electrophoresis and DNA band detection</p></td><td><p xmlns="" class="SimplePara">0.06</p></td><td><p xmlns="" class="SimplePara">0.06</p></td><td><p xmlns="" class="SimplePara">0.12</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">TOTAL</p></td><td><p xmlns="" class="SimplePara">38.04</p></td><td><p xmlns="" class="SimplePara">15.54</p></td><td><p xmlns="" class="SimplePara">9.58</p></td></tr><tr class="noclass"><td colspan="4"><p xmlns="" class="SimplePara">Work load</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara"> Approximate working time for development 1 assay (h)</p></td><td><p xmlns="" class="SimplePara">0.25</p></td><td><p xmlns="" class="SimplePara">0.25</p></td><td><p xmlns="" class="SimplePara">0.1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara"> Working time for testing</p></td><td><p xmlns="" class="SimplePara">0.12</p></td><td><p xmlns="" class="SimplePara">0.08</p></td><td><p xmlns="" class="SimplePara">0.12</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">TOTAL</p></td><td><p xmlns="" class="SimplePara">0.37</p></td><td><p xmlns="" class="SimplePara">0.33</p></td><td><p xmlns="" class="SimplePara">0.22</p></td></tr><tr class="noclass"><td colspan="4"><p xmlns="" class="SimplePara">Routine analysis</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara"> Hands-on time per assay (assuming 96 assays per run, in h)</p></td><td><p xmlns="" class="SimplePara">0.03</p></td><td><p xmlns="" class="SimplePara">0.025</p></td><td><p xmlns="" class="SimplePara">0.06</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara"> Reagent costs per assay</p></td><td><p xmlns="" class="SimplePara">0.13</p></td><td><p xmlns="" class="SimplePara">0.105</p></td><td><p xmlns="" class="SimplePara">0.19</p></td></tr></tbody></table></div></figure>
                        </div>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec11" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Discussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Discussion">Discussion</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Discussion" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par35" class="Para">Validation of molecular markers for breeding may require testing many candidate markers. Only a few of them will be adopted for marker-assisted selection, while most markers may be discarded. Consequently, marker validation may constitute a major cost factor. In order to keep validation costs low and to avoid the need for expensive instruments, SNPs are often converted to PCR-based markers. We have tested allele-specific PCR, tetra markers and a CEL-I based genotyping method for validating selected SNP markers derived from an array-based SNP genotyping study on rice and from a GBS experiment in mungbean. In parallel, the targeted SNPs were validated by sequencing. As expected, not all SNPs pinpointed by the GBS experiment were true SNPs. Two of the putative SNPs of mungbean were in fact monomorphic. In mungbean, chromosomal rearrangements or an indel in the GBS tag in comparison to the reference sequence [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>] led to erroneous mapping and resulted in false positive SNPs. In general array-based genotyping is considered more accurate than GBS, nevertheless GBS is widely used for genotyping rice [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>], mungbean [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>] and other crops. Shallow sequencing and erroneous mapping of sequence tags in GBS can yield wrong genotypes, but GBS is more cost effective than array-based genotyping and has the additional advantage that no previous information on the SNPs present in the samples is required [<span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par36" class="Para">Allele-specific PCR was unsuitable to genotype SNPs in the rice samples without assay optimization and therefore was abandoned. The accuracy of tetra marker genotyping was affected by the sequence context. Introduction of a sequence mismatch near the 3′ end of the primers to increase stability might have overcome this problem [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>] but was not tested, as the study addressed only the least work- intensive SNP genotyping. Optimization of the SNP assays beyond gradient PCR to determine the optimal annealing temperature for the allele-specific and tetra primers was avoided, as the study aimed to assess the performance of the assays without laborious modifications.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par37" class="Para">Previously published CEL-I protocols proved to be useful for SNP validation and genotyping [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>, <span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>], but could be simplified to make the assays cheaper and easier to use. Stopping the CEL-I reaction by adding EDTA, as described by published protocols, was found to be unnecessary; similarly, the previously suggested desalting of the reaction mixtures prior to loading the DNA on the gel was eliminated, without affecting the scoring of the bands. Lowering the reaction temperature of CEL-I below 36&nbsp;°C was essential to reduce the background. CEL-I genotyping for validation of SNP markers was the least costly method to develop, but the method required more resources per locus than tetra markers for genotyping large numbers of samples. Either PCR fragments containing the SNP have to be mixed after PCR, or DNA to be genotyped must be spiked with reference DNA. For each sample and locus, two CEL-I reactions have to be performed to distinguish homozygote from heterozygote genotypes, making the genotyping more expensive and more laborious. From our perspective, mixing DNA before PCR is the preferred CEL-I genotyping method, as spiking of the PCR master mix is less laborious and safer than handling the PCR fragments for combination after amplification. CEL-I genotyping also was more reliable than tetra markers, which may be affected by the sequence context of the SNP and require additional optimization of the assay, leading to longer development times and higher costs. However, CEL-I based genotyping was tedious on heterozygote samples, as the banding pattern became quite complex and difficult to score with increasing heterozygosity of the mapping parents.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par38" class="Para">It can be expected that more modern genotyping tools such as KASP markers [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>] are more accurate than the methods presented here, but require equipment such as a plate reader that might not be available to all users of molecular markers. By adopting an improved protocol for CEL-I genotyping we wanted to provide a simple and cheap method for genotyping for laboratories that lack specialized equipment.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec12" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Conclusions" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Conclusions">Conclusions</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Conclusions" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par39" class="Para">Allele-specific PCR was not suitable to genotype SNPs in rice. The genotyping accuracy of tetra markers was affected by low allele specifity of the inner primers, repeat sequences and nucleotide runs. In contrast, genotyping by using CEL-I digestion at low incubation temperatures performed well and was independent of the sequence context around the SNP. It was also more cost effective than tetra markers, as long as the number of tested samples remained small. Additional SNPs to the SNP selected for genotyping present in the PCR fragment can make scoring of CEL-I genotypes tedious. Therefore, PCR fragments for CEL-I genotyping should be kept small to minimize the presence of additional SNPs.</p></div></section>
          <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AbbreviationGroup Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Abbreviations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Abbreviations">Abbreviations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Abbreviations"><div class="DefinitionList">
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>ARMS PCR:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par4" class="Para">Amplification-refractory mutation system PCR</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>BSA:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par5" class="Para">Bovine serum albumin</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>CAPS:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par6" class="Para">Cleaved amplified polymorphic sequence</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>CTAB:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par7" class="Para">Cetyl trimethylammonium bromide</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>dCAPS:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par8" class="Para">Derived cleaved amplified polymorphic sequences</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>EDTA:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par9" class="Para">Ethylenediaminetetraacetic acid</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>GBS:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par10" class="Para">Genotyping by sequencing</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>HEPES:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par11" class="Para">4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>KASP:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par12" class="Para">Kompetitive Allele Specific PCR</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>PCR:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par13" class="Para">Polymerase chain reaction</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>QTL:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par14" class="Para">Quantitative trait locus</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>SNP:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par15" class="Para">Single nucleotide polymorphism</p>
                  </dd></dl>
              </div></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments"><h3 xmlns="" class="Heading">Acknowledgements</h3><p xmlns="" class="SimplePara">We thank the Federal Ministry for Economic Cooperation and Development, Germany, for co-funding the study. Support for World Vegetable Center research activities is provided by project donors and the following core donors: Republic of China (ROC), UK Department for International Development (DFID), United States Agency for International Development (USAID), Australian Centre for International Agricultural Research (ACIAR), Germany, Thailand, Philippines, Korea, and Japan.</p><div id="FPar1" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Funding</h3>
                <p id="Par40" class="Para">The study was funded by the Innovations Fund of the World Vegetable Center and by the Federal Ministry for Economic Cooperation and Development, Germany, Contract No.: 81180341.</p>
              </div><div id="FPar2" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Availability of data and supporting materials</h3>
                <p id="Par41" class="Para">All data produced by the study are disclosed in the manuscript and the additional files.</p>
              </div><div id="FPar3" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Authors’ contributions</h3>
                <p id="Par42" class="Para">TGTB participated with the design of the assays, performed the assays and contributed with drafting of the manuscript. NTLH designed the study on rice and designed the markers for this crop. YJY provided the mungbean material and contributed with the SNP assays for mungbean, SR planned the study, supervised the genotyping of mungbean and drafted the manuscript. All authors read and approved the final manuscript.</p>
              </div><div id="FPar4" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Competing interests</h3>
                <p id="Par43" class="Para">The authors declare that they have no competing interests.</p>
              </div><div id="FPar5" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Consent for publication</h3>
                <p id="Par44" class="Para">Not applicable.</p>
              </div><div id="FPar6" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Ethics approval and consent to participate</h3>
                <p id="Par45" class="Para">The plant material used in the study is deposited in the genebanks of the Plant Resources Center of the Vietnam Academy of Agricultural Sciences, Vietnam, or of the Workd Vegeteble Center, Taiwan and has been used in compliance with national regulations and international treaties.</p>
              </div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="License">
                <p xmlns="" class="SimplePara">
                           <strong class="EmphasisTypeBold">Open Access</strong>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/4.0/"><span class="RefSource">http://creativecommons.org/licenses/by/4.0/</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/publicdomain/zero/1.0/"><span class="RefSource">http://creativecommons.org/publicdomain/zero/1.0/</span></a></span>) applies to the data made available in this article, unless otherwise stated.</p>
              </div><aside class="Appendix" id="App1">
              
                <h3 class="Heading">Additional files</h3><div id="widgetContainer" data-test="figshare-container"></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par46" class="Para">
                  <div class="MediaObject" id="MOESM1">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-016-0024-y/MediaObjects/41065_2016_24_MOESM1_ESM.pdf"><span class="caption-number">Additional file 1:</span></a>
                          <span xmlns="" class="SimplePara">Genotyping segregating mungbean populations with tetra markers. (PDF 219&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM2">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-016-0024-y/MediaObjects/41065_2016_24_MOESM2_ESM.docx"><span class="caption-number">Additional file 2:</span></a>
                          <span xmlns="" class="SimplePara">CEL-I genotyping in segregating mungbean populations TC1966 x NM92 and V2802 x NM94. (DOCX 23250&nbsp;kb)</span>
                        </div></div>
                  </div>
                </div>
              
            </aside></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authors’ Affiliations</a></h2><div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1"><span class="AffiliationNumber">(1)</span><div class="AffiliationText">Plant Resources Center, Vietnam Academy of Agricultural Sciences, An Khanh, Hoai Duc, Ha Noi, Vietnam</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff2"><span class="AffiliationNumber">(2)</span><div class="AffiliationText">World Vegetable Center, 60 Yi Min Liao, Shanhua, Tainan, 74151, Taiwan</div></div></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper"><li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ganal MW, Polley A, Graner EM, Plieske J, Wieseke R, Luerssen H, Durstewitz G. Large SNP arrays for genotyping in crop plants. J Bioscience. 2012;37:821–8.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s12038-012-9225-3"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Large%20SNP%20arrays%20for%20genotyping%20in%20crop%20plants&amp;author=MW.%20Ganal&amp;author=A.%20Polley&amp;author=EM.%20Graner&amp;author=J.%20Plieske&amp;author=R.%20Wieseke&amp;author=H.%20Luerssen&amp;author=G.%20Durstewitz&amp;journal=J%20Bioscience&amp;volume=37&amp;pages=821-8&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Andrews KR, Good JM, Miller MR, Luikart G, Hohenlohe PA. Harnessing the power of RADseq for ecological and evolutionary genomics. Nat Rev Genet. 2016;17:81–92.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nrg.2015.28"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26729255"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4823021"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Harnessing%20the%20power%20of%20RADseq%20for%20ecological%20and%20evolutionary%20genomics&amp;author=KR.%20Andrews&amp;author=JM.%20Good&amp;author=MR.%20Miller&amp;author=G.%20Luikart&amp;author=PA.%20Hohenlohe&amp;journal=Nat%20Rev%20Genet&amp;volume=17&amp;pages=81-92&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Semagn K, Babu R, Hearne S, Olsen M. Single nucleotide polymorphism genotyping using Kompetitive Allele Specific PCR (KASP): overview of the technology and its application in crop improvement. Mol Breeding. 2014;33:1–14.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11032-013-9917-x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Single%20nucleotide%20polymorphism%20genotyping%20using%20Kompetitive%20Allele%20Specific%20PCR%20%28KASP%29%3A%20overview%20of%20the%20technology%20and%20its%20application%20in%20crop%20improvement&amp;author=K.%20Semagn&amp;author=R.%20Babu&amp;author=S.%20Hearne&amp;author=M.%20Olsen&amp;journal=Mol%20Breeding&amp;volume=33&amp;pages=1-14&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Thiel T, Kota R, Grosse I, Stein N, Graner A. SNP2CAPS: a SNP and INDEL analysis tool for CAPS marker development. Nucleic Acids Res. 2004;32:e5.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gnh006"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14704362"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC373308"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=SNP2CAPS%3A%20a%20SNP%20and%20INDEL%20analysis%20tool%20for%20CAPS%20marker%20development&amp;author=T.%20Thiel&amp;author=R.%20Kota&amp;author=I.%20Grosse&amp;author=N.%20Stein&amp;author=A.%20Graner&amp;journal=Nucleic%20Acids%20Res&amp;volume=32&amp;pages=e5&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Neff MM, Neff JD, Chory J, Pepper AE. dCAPS, a simple technique for the genetic analysis of single nucleotide polymorphisms: experimental applications in Arabidopsis thaliana genetics. Plant J. 1998;14:387–92.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1046/j.1365-313X.1998.00124.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9628033"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=dCAPS%2C%20a%20simple%20technique%20for%20the%20genetic%20analysis%20of%20single%20nucleotide%20polymorphisms%3A%20experimental%20applications%20in%20Arabidopsis%20thaliana%20genetics&amp;author=MM.%20Neff&amp;author=JD.%20Neff&amp;author=J.%20Chory&amp;author=AE.%20Pepper&amp;journal=Plant%20J&amp;volume=14&amp;pages=387-92&amp;publication_year=1998"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gaudet M, Fara A-G, Beritognolo I, Sabatti M. Allele-specific PCR in SNP genotyping. In: Komar AA, editor. Single Nucleotide Polymorphisms: Methods and Protocols. New York: Humana Press; 2009. p. 415–24.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Gaudet%20M%2C%20Fara%20A-G%2C%20Beritognolo%20I%2C%20Sabatti%20M.%20Allele-specific%20PCR%20in%20SNP%20genotyping.%20In%3A%20Komar%20AA%2C%20editor.%20Single%20Nucleotide%20Polymorphisms%3A%20Methods%20and%20Protocols.%20New%20York%3A%20Humana%20Press%3B%202009.%20p.%20415%E2%80%9324."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ye S, Humphries S, Green F. Allele specific amplification by tetra-primer PCR. Nucleic Acids Res. 1992;20:1152.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/20.5.1152"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1549488"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC312115"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Allele%20specific%20amplification%20by%20tetra-primer%20PCR&amp;author=S.%20Ye&amp;author=S.%20Humphries&amp;author=F.%20Green&amp;journal=Nucleic%20Acids%20Res&amp;volume=20&amp;pages=1152&amp;publication_year=1992"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ye S, Dhillon S, Ke X, Collins AR, Day IN. An efficient procedure for genotyping single nucleotide polymorphisms. Nucleic Acids Res. 2001;29:e88.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/29.17.e88"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11522844"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC55900"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=An%20efficient%20procedure%20for%20genotyping%20single%20nucleotide%20polymorphisms&amp;author=S.%20Ye&amp;author=S.%20Dhillon&amp;author=X.%20Ke&amp;author=AR.%20Collins&amp;author=IN.%20Day&amp;journal=Nucleic%20Acids%20Res&amp;volume=29&amp;pages=e88&amp;publication_year=2001"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Chiapparino E, Lee D, Donini P. Genotyping single nucleotide polymorphisms in barley by tetra-primer ARMS-PCR. Genome. 2004;47:414–20.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1139/g03-130"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15060595"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genotyping%20single%20nucleotide%20polymorphisms%20in%20barley%20by%20tetra-primer%20ARMS-PCR&amp;author=E.%20Chiapparino&amp;author=D.%20Lee&amp;author=P.%20Donini&amp;journal=Genome&amp;volume=47&amp;pages=414-20&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Medrano RFV, de Oliveira CA. Guidelines for the tetra-primer ARMS–PCR technique development. Mol Biotechnol. 2014;56:599–608.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24519268"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Guidelines%20for%20the%20tetra-primer%20ARMS%E2%80%93PCR%20technique%20development&amp;author=RFV.%20Medrano&amp;author=CA.%20Oliveira&amp;journal=Mol%20Biotechnol&amp;volume=56&amp;pages=599-608&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Raghavan C, Naredo MEB, Wang H, Atienza G, Liu B, Qiu F, McNally KL, Leung H. Rapid method for detecting SNPs on agarose gels and its application in candidate gene mapping. Mol Breeding. 2007;19:87–101.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11032-006-9046-x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Rapid%20method%20for%20detecting%20SNPs%20on%20agarose%20gels%20and%20its%20application%20in%20candidate%20gene%20mapping&amp;author=C.%20Raghavan&amp;author=MEB.%20Naredo&amp;author=H.%20Wang&amp;author=G.%20Atienza&amp;author=B.%20Liu&amp;author=F.%20Qiu&amp;author=KL.%20McNally&amp;author=H.%20Leung&amp;journal=Mol%20Breeding&amp;volume=19&amp;pages=87-101&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Galeano CH, Gomez M, Rodriguez LM, Blair MW. CEL-I nuclease digestion for SNP discovery and marker development in common bean (L.). Crop Sci. 2009;49:381–94.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2135/cropsci2008.07.0413"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=CEL-I%20nuclease%20digestion%20for%20SNP%20discovery%20and%20marker%20development%20in%20common%20bean%20%28L.%29&amp;author=CH.%20Galeano&amp;author=M.%20Gomez&amp;author=LM.%20Rodriguez&amp;author=MW.%20Blair&amp;journal=Crop%20Sci&amp;volume=49&amp;pages=381-94&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Oleykowski CA, Mullins CRB, Godwin AK, Yeung AT. Mutation detection using a novel plant endonuclease. Nucleic Acids Res. 1998;26:4597–602.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/26.20.4597"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9753726"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC147896"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Mutation%20detection%20using%20a%20novel%20plant%20endonuclease&amp;author=CA.%20Oleykowski&amp;author=CRB.%20Mullins&amp;author=AK.%20Godwin&amp;author=AT.%20Yeung&amp;journal=Nucleic%20Acids%20Res&amp;volume=26&amp;pages=4597-602&amp;publication_year=1998"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yang B, Wen X, Kodali NS, Oleykowski CA, Miller CG, Kulinski J, Besack D, Yeung JA, Kowalski D, Yeung AT. Purification, cloning, and characterization of the CEL-I nuclease. Biochemistry. 2000;39:3533–41.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1021/bi992376z"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10736152"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Purification%2C%20cloning%2C%20and%20characterization%20of%20the%20CEL-I%20nuclease&amp;author=B.%20Yang&amp;author=X.%20Wen&amp;author=NS.%20Kodali&amp;author=CA.%20Oleykowski&amp;author=CG.%20Miller&amp;author=J.%20Kulinski&amp;author=D.%20Besack&amp;author=JA.%20Yeung&amp;author=D.%20Kowalski&amp;author=AT.%20Yeung&amp;journal=Biochemistry&amp;volume=39&amp;pages=3533-41&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Till BJ, Burtner C, Comai L, Henikoff S. Mismatch cleavage by single-strand specific nucleases. Nucleic Acids Res. 2004;32:2632–41.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gkh599"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15141034"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC419476"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Mismatch%20cleavage%20by%20single-strand%20specific%20nucleases&amp;author=BJ.%20Till&amp;author=C.%20Burtner&amp;author=L.%20Comai&amp;author=S.%20Henikoff&amp;journal=Nucleic%20Acids%20Res&amp;volume=32&amp;pages=2632-41&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Duitama J, Silva A, Sanabria Y, Cruz DF, Quintero C, Ballen C, Lorieux M, Scheffler B, Farmer A, Torres E, Oard J. Whole genome sequencing of elite rice cultivars as a comprehensive information resource for marker assisted selection. PLoS One. 2015;10:e0124617.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0124617"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25923345"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4414565"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Whole%20genome%20sequencing%20of%20elite%20rice%20cultivars%20as%20a%20comprehensive%20information%20resource%20for%20marker%20assisted%20selection&amp;author=J.%20Duitama&amp;author=A.%20Silva&amp;author=Y.%20Sanabria&amp;author=DF.%20Cruz&amp;author=C.%20Quintero&amp;author=C.%20Ballen&amp;author=M.%20Lorieux&amp;author=B.%20Scheffler&amp;author=A.%20Farmer&amp;author=E.%20Torres&amp;author=J.%20Oard&amp;journal=PLoS%20One&amp;volume=10&amp;pages=e0124617&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Schafleitner R, Huang SM, Chu SH, Yen JY, Lin CY, Yan MR, Krishnan B, Liu MS, Lo HF, Chen CY, Chen LFO, Wu DC, Bui TGT, Ramasamy S, Tung C-W, Ramakrishnan N. dentification of single nucleotide polymorphic markers associated with resistance to bruchids (<em class="EmphasisTypeItalic">Callosobruchus</em> spp.) in wild mungbean (<em class="EmphasisTypeItalic">Vigna radiata</em> var. <em class="EmphasisTypeItalic">sublobata</em>) and cultivated <em class="EmphasisTypeItalic">V. radiata</em> through genotyping by sequencing and quantitative trait locus analysis. BMC Plant Biol. 2016;16:1.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/s12870-016-0847-8"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=dentification%20of%20single%20nucleotide%20polymorphic%20markers%20associated%20with%20resistance%20to%20bruchids%20%28Callosobruchus%20spp.%29%20in%20wild%20mungbean%20%28Vigna%20radiata%20var.%20sublobata%29%20and%20cultivated%20V.%20radiata%20through%20genotyping%20by%20sequencing%20and%20quantitative%20trait%20locus%20analysis&amp;author=R.%20Schafleitner&amp;author=SM.%20Huang&amp;author=SH.%20Chu&amp;author=JY.%20Yen&amp;author=CY.%20Lin&amp;author=MR.%20Yan&amp;author=B.%20Krishnan&amp;author=MS.%20Liu&amp;author=HF.%20Lo&amp;author=CY.%20Chen&amp;author=LFO.%20Chen&amp;author=DC.%20Wu&amp;author=TGT.%20Bui&amp;author=S.%20Ramasamy&amp;author=C-W.%20Tung&amp;author=N.%20Ramakrishnan&amp;journal=BMC%20Plant%20Biol&amp;volume=16&amp;pages=1&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Thomson MJ, Zhao K, Wright M, McNally KL, Rey J, Tung C-W, Reynolds A, Scheffler B, Eizenga G, McClung A. High-throughput single nucleotide polymorphism genotyping for breeding applications in rice using the BeadXpress platform. Mol Breed. 2012;29:875–86.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11032-011-9663-x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=High-throughput%20single%20nucleotide%20polymorphism%20genotyping%20for%20breeding%20applications%20in%20rice%20using%20the%20BeadXpress%20platform&amp;author=MJ.%20Thomson&amp;author=K.%20Zhao&amp;author=M.%20Wright&amp;author=KL.%20McNally&amp;author=J.%20Rey&amp;author=C-W.%20Tung&amp;author=A.%20Reynolds&amp;author=B.%20Scheffler&amp;author=G.%20Eizenga&amp;author=A.%20McClung&amp;journal=Mol%20Breed&amp;volume=29&amp;pages=875-86&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Doyle JJ. A rapid DNA isolation procedure for small quantities of fresh leaf tissue. Phytochem Bull. 1987;19:11–5.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20rapid%20DNA%20isolation%20procedure%20for%20small%20quantities%20of%20fresh%20leaf%20tissue&amp;author=JJ.%20Doyle&amp;journal=Phytochem%20Bull&amp;volume=19&amp;pages=11-5&amp;publication_year=1987"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kang YJ, Kim SK, Kim MY, Lestari P, Kim KH, Ha B-K, Jun TH, Hwang WJ, Lee T, Lee J, Shim S, Yoon MY, Jang YE, Han KS, Taeprayoon P, Yoon N, Somta P, Tanya P, Kim KS, Gwag J-G, Moon J-K, Lee J-H, Park B, Bombarely A, Doyle JJ, Jackson SA, Schafleitner R, Srinives P, Varshney RK, Lee S-H. Genome sequence of mungbean and insights into evolution within <em class="EmphasisTypeItalic">Vigna</em> species. Nature Comm. 2014;5:5443.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/ncomms6443"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20mungbean%20and%20insights%20into%20evolution%20within%20Vigna%20species&amp;author=YJ.%20Kang&amp;author=SK.%20Kim&amp;author=MY.%20Kim&amp;author=P.%20Lestari&amp;author=KH.%20Kim&amp;author=B-K.%20Ha&amp;author=TH.%20Jun&amp;author=WJ.%20Hwang&amp;author=T.%20Lee&amp;author=J.%20Lee&amp;author=S.%20Shim&amp;author=MY.%20Yoon&amp;author=YE.%20Jang&amp;author=KS.%20Han&amp;author=P.%20Taeprayoon&amp;author=N.%20Yoon&amp;author=P.%20Somta&amp;author=P.%20Tanya&amp;author=KS.%20Kim&amp;author=J-G.%20Gwag&amp;author=J-K.%20Moon&amp;author=J-H.%20Lee&amp;author=B.%20Park&amp;author=A.%20Bombarely&amp;author=JJ.%20Doyle&amp;author=SA.%20Jackson&amp;author=R.%20Schafleitner&amp;author=P.%20Srinives&amp;author=RK.%20Varshney&amp;author=S-H.%20Lee&amp;journal=Nature%20Comm&amp;volume=5&amp;pages=5443&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Tang W, Wu T, Ye J, Sun J, Jiang Y, Yu J, Tang J, Chen G, Wang C, Wan J. SNP-based analysis of genetic diversity reveals important alleles associated with seed size in rice. BMC Plant Biol. 2016;16:1.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/s12870-015-0700-5"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=SNP-based%20analysis%20of%20genetic%20diversity%20reveals%20important%20alleles%20associated%20with%20seed%20size%20in%20rice&amp;author=W.%20Tang&amp;author=T.%20Wu&amp;author=J.%20Ye&amp;author=J.%20Sun&amp;author=Y.%20Jiang&amp;author=J.%20Yu&amp;author=J.%20Tang&amp;author=G.%20Chen&amp;author=C.%20Wang&amp;author=J.%20Wan&amp;journal=BMC%20Plant%20Biol&amp;volume=16&amp;pages=1&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Elshire RJ, Glaubitz JC, Sun Q, Poland JA, Kawamoto K, Buckler ES, Mitchell SE. A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species. PLoS One. 2011;6:e19379.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0019379"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21573248"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3087801"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20robust%2C%20simple%20genotyping-by-sequencing%20%28GBS%29%20approach%20for%20high%20diversity%20species&amp;author=RJ.%20Elshire&amp;author=JC.%20Glaubitz&amp;author=Q.%20Sun&amp;author=JA.%20Poland&amp;author=K.%20Kawamoto&amp;author=ES.%20Buckler&amp;author=SE.%20Mitchell&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e19379&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Liu Z, Zhu C, Jiang Y, Tian Y, Yu J, An H, Tang W, Sun J, Tang J, Chen G, Zhai H. Association mapping and genetic dissection of nitrogen use efficiency-related traits in rice (<em class="EmphasisTypeItalic">Oryza sativa</em> L.). Funct Integr Genomics. 2016;16:323–33.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s10142-016-0486-z"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26922174"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Association%20mapping%20and%20genetic%20dissection%20of%20nitrogen%20use%20efficiency-related%20traits%20in%20rice%20%28Oryza%20sativa%20L.%29&amp;author=Z.%20Liu&amp;author=C.%20Zhu&amp;author=Y.%20Jiang&amp;author=Y.%20Tian&amp;author=J.%20Yu&amp;author=H.%20An&amp;author=W.%20Tang&amp;author=J.%20Sun&amp;author=J.%20Tang&amp;author=G.%20Chen&amp;author=H.%20Zhai&amp;journal=Funct%20Integr%20Genomics&amp;volume=16&amp;pages=323-33&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li></ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#TheAuthors" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="TheAuthors">Copyright</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="TheAuthors"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">©&nbsp;The Author(s)&nbsp;2017</div></div></section></div>
                        


                    </article>
                </main>

                <div class="c-content-layout__side u-text-sm">
                    <aside>
                        <div class="p-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="//hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-016-0024-y" data-track="click" data-track-category="PDF Download"
            data-track-action="Click download article PDF"
            data-track-label="10.1186/s41065-016-0024-y">Download PDF</a>
        </p>
    
                            
                            
    <button type="button"
       class="c-button c-button--secondary c-button--full-width c-button--expandable"
       data-toggle="collapse"
       data-target="export-citations"
       data-test="export-citations-button"
       data-sticky-update="true"
       aria-controls="export-citations"
       aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/s41065-016-0024-y"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0024-y?format=refman&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/s41065-016-0024-y"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0024-y?format=refman&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/s41065-016-0024-y"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0024-y?format=refman&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/s41065-016-0024-y"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0024-y?format=endnote&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/s41065-016-0024-y"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0024-y?format=endnote&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/s41065-016-0024-y"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0024-y?format=endnote&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/s41065-016-0024-y"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0024-y?format=bibtex&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/s41065-016-0024-y"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0024-y?format=bibtex&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/s41065-016-0024-y"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-016-0024-y?format=bibtex&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
    </div>

                        </div>
                        
    
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
                                <div id="article-access-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/access-count/10.1186/s41065-016-0024-y #accessCount-container"></div>
                                <div id="citation-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/citation-count/10.1186/s41065-016-0024-y #count-container"></div>
                                <div id="altmetric-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/altmetric/10.1186/s41065-016-0024-y #altmetric-container"></div>
                            </ul>
                        </div>
                        
    <div id="sociaMediaShare" class="p-separator u-js-shown">
        <h3 class="p-section-title">Share this article</h3>
        <ul id="shareButtons" class="c-social-links">
            <li class="c-social-links__item">
                <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-016-0024-y">
                    <span class="u-visually-hidden">Share on Twitter</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-twitter"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-016-0024-y">
                    <span class="u-visually-hidden">Share on Facebook</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-facebook"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click"
                        data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-016-0024-y">
                    <span class="u-visually-hidden">Share on LinkedIn</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-linkedin"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-016-0024-y">
                    <span class="u-visually-hidden">Share on Weibo</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-weibo"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnGooglePlus" class="js-btnShareOnGplus" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Google+" data-track-label="10.1186/s41065-016-0024-y">
                    <span class="u-visually-hidden">Share on Google Plus</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-gplus"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-016-0024-y">
                    <span class="u-visually-hidden">Share on Reddit</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-reddit"></use>
                    </svg>
                </button>
            </li>
        </ul>
    </div>
 
                        
    
        <div class="p-separator u-js-shown">
            <h3 class="p-section-title">See updates</h3>
            <button class="c-crossmark-button" data-target="crossmark" data-test="crossmark" data-track="click" data-track-category="Article Page" data-track-action="Click CrossMark" data-track-label="10.1186/s41065-016-0024-y">
                <img id="crossmark-icon" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" alt="Crossmark logo" />
            </button>
        </div>
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
                                <li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/s41065-016-0024-y" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=738&doi=10.1186%2Fs41065-016-0024-y&volume=154&issue=1&title=PCR-based+assays+for+validation+of+single+nucleotide+polymorphism+markers+in+rice+and+mungbean&author_name=Thu+Giang+Thi+Bui&start_page=1&end_page=13">Order reprint</a>

                                </li>
                            </ul>
                        </div>
                    </aside>
                    
    <div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">Hereditas</span>
                        </h4>
                        <p class="c-journal-footer__issn">ISSN: 1601-5223</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
                                
                                    <li class="c-journal-footer__contact-item">Submission enquiries: <a href="https://www.editorialmanager.com/here/" target="_blank">Access here and click Contact Us</a></li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a></li>
                                
                            </ul>
                        </div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class='tracker' style='display:none' src='/track/article/10.1186/s41065-016-0024-y' alt=""/>

         
    <footer>
        <div class="ctx-publisher-footer">
    <div class="c-navbar">
        <div class="u-container">
            <div class="c-navbar__logo u-margin-right-xl">
                <a href="https://www.biomedcentral.com">
                    <img src=/static/images/bmc/logos/logo-bmc-aj-594e03e20d.svg alt="BMC logo" data-test="navbar-logo-footer">

                </a>
            </div>
            <div class="c-navbar__content">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


            </div>
        </div>
    </div>
    <div class="c-publisher-footer" data-test="publisher-footer">
        <div class="u-container">
            
            <div class="c-flex c-flex--wrap c-flex--space-between" role="navigation" data-test="publisher-footer-menu">
                <div class="c-flex" aria-label="Publisher footer links">
                    
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/login">Receive BMC newsletters</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/account">Manage article alerts</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/policies">Policies</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/accessibility">Accessibility</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/press-centre">Press center</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/contact-us">Contact us</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/jobs">Careers</a>
                                    </li>
                                
                            </ul>
                        
                    
                </div>
                <div class="u-margin-bottom-lg">
                    <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                    <ul class="c-flex u-reset-list" data-test="footer-social-links">
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://twitter.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Twitter link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Twitter page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-twitter-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://www.facebook.com/BioMedCentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Facebook link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Facebook page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-facebook-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://plus.google.com/106226337984152901734/"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Google Plus link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Google Plus page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-gplus-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="http://www.weibo.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Weibo link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Weibo page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-weibo-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </div>
            <p class="u-reset-margin">
                By using this website, you agree to our
                <a class="u-text-inherit" href="//www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
                <a class="u-text-inherit" href="//www.biomedcentral.com/privacy-statement">Privacy
                    statement</a> and
                <a class="u-text-inherit" href="//www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
                
                    <a class="optanon-toggle-display u-text-inherit" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
                
            </p>
        </div>
    </div>
</div>


        <div class="c-sn-footer c-sn-footer--neutral">
    <div class="u-container">
        <img src=/static/images/bmc/logos/logo-springernature-8bd4e8d82f.svg class="c-sn-footer__logo" alt="Springer Nature Logo" itemprop="logo" >
        <p data-test="copyright">&#169; 2018 BioMed Central Ltd unless otherwise stated. Part of
            <a href="http://www.springernature.com"
               data-track="click"
               data-track-action="Click Springer Nature link"
               data-track-category="Corporate footer"
               itemscope itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>

    </div>

    
    <script type="text/javascript" src="//cdn.pbgrd.com/bmc.js" async="async"></script>
    <noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>
    <!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker -->

<script id="springerlink-event-tracker" src="https://event-tracker.springernature.com/dist/eventTracker.js"></script>
<script type="text/javascript">
    (function(){
        var dataLayer = window.dataLayer[0];
        var doi = dataLayer.content.article.doi;
        var imprint = dataLayer.content.contentInfo.imprint;
        var bpids = dataLayer.session.authentication.authenticationID;
        var publisher;
        var pageType = dataLayer.page.category.pageType;
        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }
        var eventObject = {'content_type' : pageType,'doi' : doi};

        if (imprint === 'BioMed Central') { publisher = 'BMC' }
        else if (imprint === 'SpringerOpen') { publisher = 'SpringerOpen' }
        else { throw new Error('Event Tracker Error: The publisher has an unexpected value') }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }
        new EventTracker({'platform':publisher}).sendEvent('display', eventObject);
    })();
</script>


    </body>
</html>

