<!DOCTYPE html>
<html lang="en" class="no-js">
    <head>
        <title>Characterizing the transcriptome and microsatellite markers for almond (Amygdalus communis L.) using the Illumina sequencing platform | Hereditas | Full Text</title>

        

    <meta name="citation_abstract" content="The almond tree (Prunus amygdalus Batsch) is an important nut tree grown in subtropical regions that produces nutrient-rich nuts. However, a paucity of genomic information and DNA markers has restricted the development of modern breeding technologies for almond trees. In this study, almonds were sequenced with Illumina paired-end sequencing technology to obtain transcriptome data and develop simple sequence repeats (SSR) markers. We generated approximately 64 million clean reads from the various tissues of mixed almonds, and a total of 42,135 unigenes with an average length of 988 bp were obtained in the present study. A total of 27,586 unigenes (57.7% of all unigenes generated) were annotated using several databases. A total of 112,812 unigenes were annotated with the Gene Ontology (GO) database and assigned to 82 functional sub-groups, and 29,075 unigenes were assigned to the KOG database and classified into 25 function classifications. There were 9470 unigenes assigned to 129 Kyoto Encyclopaedia of Genes and Genomes (KEGG) pathways from five categories in the KEGG pathway database. We further identified 8641 SSR markers from 48,012 unigenes. A total of 100 SSR markers were randomly selected to validate quality, and 82 markers could amplify the specific products of A. communis L., whereas 70 markers were successfully transferable to five species (A. ledebouriana, A. mongolica, A. pedunculata, A. tangutica, and A. triloba). Our study was the first to produce public transcriptome data from almonds. The development of SSR markers will promote genetics research and breeding programmes for almonds."/>

    <meta name="description" content="The almond tree (Prunus amygdalus Batsch) is an important nut tree grown in subtropical regions that produces nutrient-rich nuts. However, a paucity of genomic information and DNA markers has restricted the development of modern breeding technologies for almond trees. In this study, almonds were sequenced with Illumina paired-end sequencing technology to obtain transcriptome data and develop simple sequence repeats (SSR) markers. We generated approximately 64 million clean reads from the various tissues of mixed almonds, and a total of 42,135 unigenes with an average length of 988 bp were obtained in the present study. A total of 27,586 unigenes (57.7% of all unigenes generated) were annotated using several databases. A total of 112,812 unigenes were annotated with the Gene Ontology (GO) database and assigned to 82 functional sub-groups, and 29,075 unigenes were assigned to the KOG database and classified into 25 function classifications. There were 9470 unigenes assigned to 129 Kyoto Encyclopaedia of Genes and Genomes (KEGG) pathways from five categories in the KEGG pathway database. We further identified 8641 SSR markers from 48,012 unigenes. A total of 100 SSR markers were randomly selected to validate quality, and 82 markers could amplify the specific products of A. communis L., whereas 70 markers were successfully transferable to five species (A. ledebouriana, A. mongolica, A. pedunculata, A. tangutica, and A. triloba). Our study was the first to produce public transcriptome data from almonds. The development of SSR markers will promote genetics research and breeding programmes for almonds."/>

    <meta name="journal_id" content="41065"/>

    <meta name="dc.title" content="Characterizing the transcriptome and microsatellite markers for almond ( Amygdalus communis L.) using the Illumina sequencing platform"/>

    <meta name="dc.source" content="Hereditas 2017 155:1"/>

    <meta name="dc.format" content="text/html"/>

    <meta name="dc.publisher" content="BioMed Central"/>

    <meta name="dc.date" content="2017-10-19"/>

    <meta name="dc.type" content="OriginalPaper"/>

    <meta name="dc.language" content="En"/>

    <meta name="dc.copyright" content="2017 The Author(s)"/>

    <meta name="dc.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="dc.description" content="The almond tree (Prunus amygdalus Batsch) is an important nut tree grown in subtropical regions that produces nutrient-rich nuts. However, a paucity of genomic information and DNA markers has restricted the development of modern breeding technologies for almond trees. In this study, almonds were sequenced with Illumina paired-end sequencing technology to obtain transcriptome data and develop simple sequence repeats (SSR) markers. We generated approximately 64 million clean reads from the various tissues of mixed almonds, and a total of 42,135 unigenes with an average length of 988&#160;bp were obtained in the present study. A total of 27,586 unigenes (57.7% of all unigenes generated) were annotated using several databases. A total of 112,812 unigenes were annotated with the Gene Ontology (GO) database and assigned to 82 functional sub-groups, and 29,075 unigenes were assigned to the KOG database and classified into 25 function classifications. There were 9470 unigenes assigned to 129 Kyoto Encyclopaedia of Genes and Genomes (KEGG) pathways from five categories in the KEGG pathway database. We further identified 8641 SSR markers from 48,012 unigenes. A total of 100 SSR markers were randomly selected to validate quality, and 82 markers could amplify the specific products of A. communis L., whereas 70 markers were successfully transferable to five species (A. ledebouriana, A. mongolica, A. pedunculata, A. tangutica, and A. triloba). Our study was the first to produce public transcriptome data from almonds. The development of SSR markers will promote genetics research and breeding programmes for almonds."/>

    <meta name="prism.issn" content="1601-5223"/>

    <meta name="prism.publicationName" content="Hereditas"/>

    <meta name="prism.publicationDate" content="2017-10-19"/>

    <meta name="prism.volume" content="155"/>

    <meta name="prism.number" content="1"/>

    <meta name="prism.section" content="OriginalPaper"/>

    <meta name="prism.startingPage" content="14"/>

    <meta name="prism.copyright" content="2017 The Author(s)"/>

    <meta name="prism.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="prism.url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0049-x"/>

    <meta name="prism.doi" content="doi:10.1186/s41065-017-0049-x"/>

    <meta name="citation_pdf_url" content="https://hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-017-0049-x"/>

    <meta name="citation_fulltext_html_url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0049-x"/>

    <meta name="citation_journal_title" content="Hereditas"/>

    <meta name="citation_publisher" content="BioMed Central"/>

    <meta name="citation_issn" content="1601-5223"/>

    <meta name="citation_title" content="Characterizing the transcriptome and microsatellite markers for almond ( Amygdalus communis L.) using the Illumina sequencing platform"/>

    <meta name="citation_volume" content="155"/>

    <meta name="citation_issue" content="1"/>

    <meta name="citation_publication_date" content="2018/12"/>

    <meta name="citation_online_date" content="2017/10/19"/>

    <meta name="citation_firstpage" content="14"/>

    <meta name="citation_article_type" content="Research"/>

    <meta name="citation_fulltext_world_readable" content=""/>

    <meta name="dc.identifier" content="doi:10.1186/s41065-017-0049-x"/>

    <meta name="DOI" content="10.1186/s41065-017-0049-x"/>

    <meta name="citation_doi" content="10.1186/s41065-017-0049-x"/>

    <meta name="dc.creator" content="Linsen Zhang"/>

    <meta name="dc.creator" content="Xiaoni Yang"/>

    <meta name="dc.creator" content="Xiangning Qi"/>

    <meta name="dc.creator" content="Chunhui Guo"/>

    <meta name="dc.creator" content="Zhaobin Jing"/>

    <meta name="dc.subject" content="Animal Genetics and Genomics"/>

    <meta name="dc.subject" content="Evolutionary Biology"/>

    <meta name="citation_reference" content="citation_journal_title=Sci Rep; citation_title=Comparison of traditional and new generation DNA markers delares high genetic diversity and differentiated population structure of wild almond species; citation_author=K Sorkheh, MK Dehkordi, S Ercisli, A Hegedus, H Julia; citation_volume=7; citation_publication_date=2017; citation_pages=5966; citation_doi=10.1038/s41598-017-06084-4; citation_id=CR1"/>

    <meta name="citation_reference" content="citation_journal_title=Bioche Syst Ecol; citation_title=Seed traits, nutrient elements and assessment of genetic diversity for almond (Amygdalus spp.) endangered to China as revealed using SRAP markers; citation_author=ZB Jing, JM Cheng, CH Guo, XP Wang; citation_volume=49; citation_publication_date=2013; citation_pages=51-57; citation_doi=10.1016/j.bse.2013.03.015; citation_id=CR2"/>

    <meta name="citation_reference" content="citation_journal_title=Curr Bot; citation_title=Assessment of genetic diversity in late flowering almond varieties using ISSR molecular markers aimed to select genotypes tolerant to early spring frost in Yazd province; citation_author=MO Arman, RG Mohammad; citation_volume=2; citation_issue=1; citation_publication_date=2011; citation_pages=01-04; citation_id=CR3"/>

    <meta name="citation_reference" content="citation_journal_title=J Am Pomo Soc; citation_title=The utilization of related Prunus species for almond variety improvement; citation_author=T Gradziel, P Martinez-Gomez, F Dicenta, D Kester; citation_volume=55; citation_publication_date=2001; citation_pages=100-108; citation_id=CR4"/>

    <meta name="citation_reference" content="citation_journal_title=J Nutr; citation_title=Almond consumption reduces oxidative DNA damage and lipid peroxidation in male smokers; citation_author=N Li, X Xia, C Chen, J Blumberg, Y Song, W Zhang, X Zhang; citation_volume=137; citation_publication_date=2007; citation_pages=2717-2722; citation_id=CR5"/>

    <meta name="citation_reference" content="citation_journal_title=Biochem Sys Ecol; citation_title=Assessment of genetic diversity and population structure of Chinese wild almond, Amygdalus nana, using EST and genomic SSRs; citation_author=T Omirshat, YP Geng, LY Zeng, SS Dong, F Chen, J Chen, ZP Song; citation_volume=37; citation_publication_date=2009; citation_pages=46-153; citation_doi=10.1016/j.bse.2008.11.014; citation_id=CR6"/>

    <meta name="citation_reference" content="citation_journal_title=J Am Oil Chem Soc; citation_title=Development of pressurized extraction processes for oil recovery from wild almond (Amygdalus scoparia); citation_author=M Balvardi, J Mendiola, P Castro-Gomez, J Fontecha, K Rezaei, E Ibanez; citation_volume=92; citation_publication_date=2015; citation_pages=1503-1511; citation_doi=10.1007/s11746-015-2708-2; citation_id=CR7"/>

    <meta name="citation_reference" content="citation_journal_title=Food Hydrocoll; citation_title=Fractionation and some physicochemical properties of almond gum (Amygdalus communis L.) exudates; citation_author=A Rezaei, A Nasirpour, H Tavanai; citation_volume=60; citation_publication_date=2016; citation_pages=461-469; citation_doi=10.1016/j.foodhyd.2016.04.027; citation_id=CR8"/>

    <meta name="citation_reference" content="citation_journal_title=Genet Mol Res; citation_title=Development of simple sequence repeat markers in persimmon (Diospyros L.) and their potential use in related species; citation_author=Y Yang, ZB Jing, X Ruan, JM Cheng; citation_volume=14; citation_issue=1; citation_publication_date=2015; citation_pages=609-618; citation_doi=10.4238/2015.January.30.2; citation_id=CR9"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=De novo assembly, characterization of immature seed Transcriptome and development of Genic-SSR markers in black gram (Vigna Mungo L.) Hepper; citation_author=J Souframanien, KS Reddy; citation_volume=10; citation_issue=6; citation_publication_date=2015; citation_pages=e0128748; citation_doi=10.1371/journal.pone.0128748; citation_id=CR10"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Genet; citation_title=Mapping quantitative trait loci for kernel composition in almond; citation_author=CF Forcada, AF Marti, RS Company; citation_volume=13; citation_publication_date=2012; citation_pages=47; citation_doi=10.1186/1471-2156-13-47; citation_id=CR11"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Genet; citation_title=Construction of an almond linkage map in an Australian population nonpareil&#215;Lauranne; citation_author=I Tavassolian, G Rabiei, D Gregory, M Mnejja, MG Wirthensohn, PW Hunt, JP Gibson, CM Ford, M Sedgley, SB Wu; citation_volume=11; citation_publication_date=2010; citation_pages=551; citation_doi=10.1186/1471-2164-11-551; citation_id=CR12"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=Identification of genetic loci associated with quality traits in almond via association mapping; citation_author=CFI Forcada, N Oraguzie, S Reyes-Chin-Wo, MT Espiau, RSI Company, AFI Marti; citation_volume=10; citation_issue=6; citation_publication_date=2015; citation_pages=e0127656; citation_doi=10.1371/journal.pone.0127656; citation_id=CR13"/>

    <meta name="citation_reference" content="citation_journal_title=Sci Agric; citation_title=Development of an SRR-based identification key for Tunisian local almonds; citation_author=G Hassouna, K Elhem, BZ Tatiana, M Ahmed, G Yolanda; citation_volume=69; citation_issue=2; citation_publication_date=2012; citation_pages=108-113; citation_doi=10.1590/S0103-90162012000200004; citation_id=CR14"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Ecol Notes; citation_title=Development and transportability across Prunus species of 42 polymorphic almond microsatellites; citation_author=M Mnejja, J Garcia-Mas, W Howad, P Arus; citation_volume=5; citation_issue=3; citation_publication_date=2005; citation_pages=531-535; citation_doi=10.1111/j.1471-8286.2005.00977.x; citation_id=CR15"/>

    <meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=SSR allelic variation in almond (Prunus Dulcis mill.); citation_author=H Xie, Y Sui, FQ Chang, Y Xu, RC Ma; citation_volume=112; citation_issue=2; citation_publication_date=2006; citation_pages=366-372; citation_doi=10.1007/s00122-005-0138-5; citation_id=CR16"/>

    <meta name="citation_reference" content="citation_journal_title=Tree Genet Geno; citation_title=De novo transcriptome assembly and development of SSR markers of oaks Quercus austrocochinchinensis and Q. Kerrii (Fagaceae); citation_author=M An, M Deng, SS Zheng, YG Song; citation_volume=12; citation_issue=103; citation_publication_date=2016; citation_pages=1-9; citation_id=CR17"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=De novo assembly of Transcriptome sequencing in Caragana Korshinskii Kom. And characterization of EST-SSR markers; citation_author=Y Long, Y Wang, S Wu, J Wang, X Tian, X Pei; citation_volume=10; citation_issue=1; citation_publication_date=2015; citation_pages=e0115805; citation_doi=10.1371/journal.pone.0115805; citation_id=CR18"/>

    <meta name="citation_reference" content="citation_journal_title=J Genet; citation_title=Characterizing the transcriptome and molecular markers information for roach; citation_author=W Chi, XF Ma, JG Niu, M Zou; citation_volume=95; citation_publication_date=2016; citation_pages=45-51; citation_doi=10.1007/s12041-015-0597-7; citation_id=CR19"/>

    <meta name="citation_reference" content="citation_journal_title=Int J Mol Sci; citation_title=De novo characterization of flower bud Transcriptomes and the development of EST-SSR markers for the endangered tree Tapiscia Sinensis; citation_author=XJ Zhou, YY Wang, YN Xu, RS Yan, P Zhao, W Liu; citation_volume=16; citation_publication_date=2015; citation_pages=12855-12870; citation_doi=10.3390/ijms160612855; citation_id=CR20"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=De novo assembly and characterization of the Transcriptome and development of SSR markers in wax gourd (Benicasa hispida); citation_author=B Jiang, D Xie, W Liu, Q Peng, X He; citation_volume=8; citation_issue=8; citation_publication_date=2013; citation_pages=e71054; citation_doi=10.1371/journal.pone.0071054; citation_id=CR21"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Breeding; citation_title=The first Illumina-based de novo transcriptome sequencing and analysis of pumpkin (Cucurbita Moschata Duch.) and SSR marker development; citation_author=TQ Wu, SB Luo, R Wang, YJ Zhong, XM Xu, TE Lin, XM He; citation_volume=34; citation_publication_date=2014; citation_pages=1437-1447; citation_doi=10.1007/s11032-014-0128-x; citation_id=CR22"/>

    <meta name="citation_reference" content="citation_journal_title=Gene; citation_title=De novo transcriptome analysis of the Siberian apricot (Prunus Sibirica L.) and search for potential SSR markers by 454 pyrosequencing; citation_author=S Dong, Y Liu, J Niu, Y Ning, S Lin, Z Zhang; citation_volume=544; citation_publication_date=2014; citation_pages=220-227; citation_doi=10.1016/j.gene.2014.04.031; citation_id=CR23"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=De novo assembly and characterization of the fruit transcriptome of Chinese jujube (Ziziphus Jujuba mill.) using 454 pyrosequencing and the development of novel tri-nucleotide SSR markers; citation_author=Y Li, C Xu, X Lin, B Cui, R Wu, X Pang; citation_volume=9; citation_publication_date=2014; citation_pages=e106438; citation_doi=10.1371/journal.pone.0106438; citation_id=CR24"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=The first Illumian based de novo transcriptome sequencing and analysis of safflower flowers; citation_author=LL Huang, X Yang, P Sun, W Tong, SQ Hu; citation_volume=7; citation_publication_date=2012; citation_pages=e38653; citation_doi=10.1371/annotation/e4e8ca0c-f6e8-4b32-aae1-b5f8e0c7ebc3; citation_id=CR25"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Rep; citation_title=Marker assisted selection (MAS) for chickpea Fusarium oxysporum wilt resistant genotypes using PCR based molecular markers; citation_author=Z Ahmad, AS Mumtaz, A Ghafoor, A Ali, M Nisar; citation_volume=41; citation_publication_date=2014; citation_pages=6755-6762; citation_doi=10.1007/s11033-014-3561-3; citation_id=CR26"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Biotechnol; citation_title=Full-length transcriptome assembly from RNA-Seq data without a reference genome; citation_author=MG Grabherr, BJ Haas, M Yassour, JZ Levin, DA Thompson, I Amit, Z Adiconis; citation_volume=29; citation_publication_date=2011; citation_pages=644-652; citation_doi=10.1038/nbt.1883; citation_id=CR27"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Blast2GO, a universal tool for annotation, visualization and analysis in functional genomics research; citation_author=A Conesa, S Gotz, JM Garcia-Gomez, J Terol, M Talon, M Robles; citation_volume=21; citation_issue=18; citation_publication_date=2005; citation_pages=3674-3676; citation_doi=10.1093/bioinformatics/bti610; citation_id=CR28"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=WEGO, a web tool for plotting GO annotations; citation_author=J Ye, L Fang, H Zheng, Y Zhang, J Chen, Z Zhang, J Wang, S Li; citation_volume=34; citation_publication_date=2006; citation_pages=W293-W297; citation_doi=10.1093/nar/gkl031; citation_id=CR29"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=KEGG for linking genomes to life and the environment; citation_author=M Kanehisa, M Araki, S Goto, M Hattori, M Hirakawa, M Itoh, T Katayama; citation_volume=36; citation_publication_date=2008; citation_pages=D480-D484; citation_doi=10.1093/nar/gkm882; citation_id=CR30"/>

    <meta name="citation_author" content="Linsen Zhang"/>

    <meta name="citation_author_institution" content="College of Horticulture, Northwest A&amp;F University, Yangling, People&#8217;s Republic of China"/>

    <meta name="citation_author" content="Xiaoni Yang"/>

    <meta name="citation_author_institution" content="College of Horticulture, Northwest A&amp;F University, Yangling, People&#8217;s Republic of China"/>

    <meta name="citation_author" content="Xiangning Qi"/>

    <meta name="citation_author_institution" content="College of Horticulture, Northwest A&amp;F University, Yangling, People&#8217;s Republic of China"/>

    <meta name="citation_author" content="Chunhui Guo"/>

    <meta name="citation_author_institution" content="College of Horticulture, Northwest A&amp;F University, Yangling, People&#8217;s Republic of China"/>

    <meta name="citation_author" content="Zhaobin Jing"/>

    <meta name="citation_author_institution" content="College of Horticulture, Northwest A&amp;F University, Yangling, People&#8217;s Republic of China"/>



        <meta charset="UTF-8"/>
        <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
        <meta name="viewport" content="width=device-width, initial-scale=1"/>

        <!-- For IE 9 and below -->
<!--[if IE]><link rel="shortcut icon" href=/static/images/favicons//favicon.ico><![endif]-->
<!-- Touch Icons - iOS and Android 2.1+ -->
<link rel="apple-touch-icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>
<!-- Firefox, Chrome, Safari, IE 11+ and Opera -->
<link rel="icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>


        <script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script>
        <script>(function(d){if (sessionStorage.fontsLoaded){d.style.setProperty('--font-family-sans', 'Europa');d.style.setProperty('--font-family-serif', 'Noto Serif')}})(document.documentElement);</script>

        <link rel="stylesheet" media="screen" href=/static/styles/basic/basic-a31ca85bc2.css>
<link rel="stylesheet" media="print" href=/static/styles/bmc/print-3a6240f456.css>

<link rel="stylesheet" class="js-ctm" href=/static/styles/bmc/main-84414d3d47.css
      media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">

        

        <script src="/static/js/app-bundle-8d61e6ef37.js" defer></script>
        
        <script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script>
        
    <script type="text/javascript">
        config = {
            env: 'live',
            site: 'hereditasjournal.biomedcentral.com',
            siteWithPath: 'hereditasjournal.biomedcentral.com' + window.location.pathname,
            twitterHashtag: '',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/s41065-017-0049-x',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central'
        };
    </script>

        
    <script type="text/javascript">
        dataLayer = [{"content":{"article":{"doi":"10.1186/s41065-017-0049-x","articleType":"Research","peerReviewType":"Closed","supplement":null,"keywords":"Prunus amygdalus Batsch;Transcriptome;Functional classification;SSR markers"},"contentInfo":{"imprint":"BioMed Central","title":"Characterizing the transcriptome and microsatellite markers for almond ( Amygdalus communis L.) using the Illumina sequencing platform","publishedAt":1508371200000,"publishedAtDate":"2017-10-19","author":["Linsen Zhang","Xiaoni Yang","Xiangning Qi","Chunhui Guo","Zhaobin Jing"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":"CC BY + CC0","openAccess":true},"environment":"live"},"journal":{"siteKey":"hereditasjournal.biomedcentral.com","volume":"155","issue":"1","title":"Hereditas","type":"AJs Life","journalID":41065,"gaCode":"UA-62066850-11","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Research","publishingSegment":"Genetics & Heredity (Life)"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}}}]

    </script>

        <script src="https://cdn.cookielaw.org/consent/7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script>
        
    
        
    

        
    
        
            
                <!-- Google Tag Manager -->
                <script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script>
                <!-- End Google Tag Manager -->
            
        
    


        
    
    <link rel="canonical" href="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0049-x"/>
    

        
        
        <meta property="og:url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0049-x"/>
        <meta property="og:type" content="article"/>
        <meta property="og:site_name" content="Hereditas"/>
        <meta property="og:title" content="Characterizing the transcriptome and microsatellite markers for almond ( Amygdalus communis L.) using the Illumina sequencing platform"/>
        <meta property="og:description" content="The almond tree (Prunus amygdalus Batsch) is an important nut tree grown in subtropical regions that produces nutrient-rich nuts. However, a paucity of genomic information and DNA markers has restricted the development of modern breeding technologies for almond trees. In this study, almonds were sequenced with Illumina paired-end sequencing technology to obtain transcriptome data and develop simple sequence repeats (SSR) markers. We generated approximately 64 million clean reads from the various tissues of mixed almonds, and a total of 42,135 unigenes with an average length of 988&amp;nbsp;bp were obtained in the present study. A total of 27,586 unigenes (57.7% of all unigenes generated) were annotated using several databases. A total of 112,812 unigenes were annotated with the Gene Ontology (GO) database and assigned to 82 functional sub-groups, and 29,075 unigenes were assigned to the KOG database and classified into 25 function classifications. There were 9470 unigenes assigned to 129 Kyoto Encyclopaedia of Genes and Genomes (KEGG) pathways from five categories in the KEGG pathway database. We further identified 8641 SSR markers from 48,012 unigenes. A total of 100 SSR markers were randomly selected to validate quality, and 82 markers could amplify the specific products of A. communis L., whereas 70 markers were successfully transferable to five species (A. ledebouriana, A. mongolica, A. pedunculata, A. tangutica, and A. triloba). Our study was the first to produce public transcriptome data from almonds. The development of SSR markers will promote genetics research and breeding programmes for almonds."/>
        <meta property="og:image" content="https://images.springer.com/sgw/journals/medium/41065.jpg"/>
    
    </head>

    <body class="journal journal-fulltext"
          
          >
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK"
                            height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><feColorMatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"/></filter></defs><symbol id="icon-arrow-down" viewBox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"/></symbol><symbol id="icon-arrow-left-bullet" viewBox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"/></symbol><symbol id="icon-arrow-right-bullet" viewBox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"/></symbol><symbol id="icon-arrow-right" viewBox="0 0 32 64"><path d="M0 0v64l32-32z"/></symbol><symbol id="icon-close" viewBox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"/></symbol><symbol id="icon-community" viewBox="0 0 60.643 31.01"><mask maskUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><path fill="#FFF" d="M0 31.01h60.643V0H0z" filter="url(#a)"/></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"/></symbol><symbol id="icon-download" viewBox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"/></symbol><symbol id="icon-ext-link" viewBox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"/><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"/><path fill="#676767" d="M9.7 0L16 6.2V0z"/></symbol><symbol id="icon-info-bordered" viewBox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"/><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"/></symbol><symbol id="icon-opr" viewBox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"/><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"/></symbol><symbol id="icon-quotation" viewBox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"/></symbol><symbol id="icon-remove" viewBox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"/><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"/></symbol><symbol id="icon-ribbon" viewBox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"/></symbol><symbol id="icon-rss" viewBox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"/><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"/><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"/></g></symbol><symbol id="icon-search" viewBox="0 0 16 16"><path d="M6.276 12.257C2.815 12.257 0 9.508 0 6.128 0 2.749 2.815 0 6.276 0c3.462 0 6.277 2.749 6.277 6.128.002 3.38-2.815 6.129-6.277 6.129zm0-10.772c-2.622 0-4.755 2.083-4.755 4.643 0 2.562 2.133 4.644 4.755 4.644 2.623 0 4.756-2.082 4.756-4.644 0-2.56-2.134-4.643-4.756-4.643z"/><path d="M16 13.751L13.641 16l-4.875-4.82 2.36-2.248L16 13.751z"/></symbol><symbol id="icon-speech-bubble" viewBox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"/></symbol><symbol id="icon-error" viewBox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"/></symbol><symbol id="icon-info" viewBox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"/></symbol><symbol id="icon-success" viewBox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"/></symbol><symbol id="icon-warning" viewBox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"/></symbol><symbol id="icon-facebook-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"/></symbol><symbol id="icon-facebook" viewBox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"/><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"/></symbol><symbol id="icon-gplus-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M480.5 273.117c.123.076.26.185.416.334.146.152.289.343.432.562.135.211.258.459.359.746.086.286.129.619.129 1.004-.012.7-.164 1.262-.465 1.683a5.374 5.374 0 0 1-.461.562c-.186.176-.375.35-.58.527a2.65 2.65 0 0 0-.322.41.908.908 0 0 0-.176.553c0 .205.059.375.18.508.102.129.199.238.299.336l.662.545a6.48 6.48 0 0 1 1.086 1.119c.293.415.445.955.459 1.625 0 .949-.42 1.789-1.256 2.521-.869.76-2.123 1.15-3.762 1.177-1.373-.015-2.4-.308-3.074-.879-.682-.534-1.023-1.172-1.023-1.917 0-.359.111-.766.336-1.211.215-.443.602-.837 1.172-1.174a6.126 6.126 0 0 1 1.998-.725 15.415 15.415 0 0 1 1.723-.173 3.996 3.996 0 0 1-.377-.596 1.347 1.347 0 0 1-.193-.74c0-.174.023-.317.074-.433a8.83 8.83 0 0 1 .121-.338 5.973 5.973 0 0 1-.633.035c-1.043-.012-1.838-.339-2.381-.979-.574-.603-.861-1.3-.861-2.097 0-.961.404-1.832 1.217-2.616.559-.46 1.135-.757 1.736-.898a8.494 8.494 0 0 1 1.67-.182h3.916l-1.209.704h-1.207v.005l-.005.002zm.754 11.276c0-.498-.162-.932-.484-1.302-.346-.348-.881-.778-1.609-1.29a4.045 4.045 0 0 0-.439-.02c-.102-.016-.354 0-.768.039a8.141 8.141 0 0 0-1.244.275c-.1.036-.24.096-.42.17a2.713 2.713 0 0 0-.553.355 1.985 1.985 0 0 0-.459.592c-.145.25-.215.551-.215.901 0 .687.311 1.253.932 1.698.596.444 1.404.68 2.432.688.928-.014 1.629-.217 2.117-.612.476-.385.71-.886.71-1.494zm-2.729-5.743a1.843 1.843 0 0 0 1.289-.556c.164-.249.271-.506.318-.767a7.32 7.32 0 0 0 .041-.659c0-.772-.195-1.552-.594-2.336a2.611 2.611 0 0 0-.73-.921 1.894 1.894 0 0 0-1.064-.352 1.723 1.723 0 0 0-1.324.64c-.299.437-.438.92-.428 1.457 0 .709.209 1.444.623 2.211.201.356.459.659.775.912.323.248.686.371 1.094.371zm9.698.077h-2.055v-2.056h-.996v2.056h-2.053v.998h2.053v2.052h.996v-2.052h2.055z"/></symbol><symbol id="icon-gplus" viewBox="0 0 24 24"><circle fill="#D84A36" cx="12" cy="12" r="12"/><g fill="#FFF"><path d="M12.012 12.111c-.304-.33-.07-.842.155-1.078.226-.241.639-.56.829-.74.193-.184.66-.605.674-1.868 0 0 .112-1.139-1.127-2.06h.977l1.038-.593h-3.507s-2.083-.067-3.085 1.207c-1 1.276-.966 2.871.114 3.77 1.08.898 2.371.648 2.371.648s-.197.422-.163.775c.036.353.421.849.504.965 0 0-2.386-.011-3.551 1.033-1.165 1.053-1.29 3.097 1.021 3.803 2.312.707 5.303-.055 5.906-2.277 0 0 .262-.908-.189-1.683-.459-.775-1.663-1.571-1.967-1.902zm-2.555-1.83c-.744-1-1.308-2.953 0-3.783 0 0 .955-.579 1.801.266.846.846 1.082 2.277.928 3.003-.15.727-.563.897-.739 1.008-.185.111-1.249.504-1.99-.494zm3.668 5.481c-.035.393-.083 1.254-1.491 1.664-1.409.408-3.826-.264-3.757-1.914 0 0 .006-.698.449-1.089.445-.396 1.094-.794 3.029-.819 0 0 .643.473 1.072.809.427.332.733.952.698 1.349z"/><path d="M16.323 11.055V9.803h-.752v1.252h-1.255v.748h1.255v1.256h.752v-1.256h1.248v-.748z"/></g></symbol><symbol id="icon-linkedin" viewBox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"/><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"/><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"/></g></symbol><symbol id="icon-reddit" viewBox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"/><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"/><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"/><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"/><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"/><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"/><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"/><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"/></symbol><symbol id="icon-twitter-bordered" viewBox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"/><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/></symbol><symbol id="icon-twitter" viewBox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"/><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"/></symbol><symbol id="icon-weibo-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"/><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"/><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"/></symbol><symbol id="icon-weibo" viewBox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"/><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"/><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"/><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"/><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"/><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"/></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner c-banner--info">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner" >
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1"
                 data-gpt-unitpath="/270604982/bmc/hereditasjournal/articles"
                 data-gpt-sizes="728x90,970x90"
                 data-gpt-targeting="pos=LB1;doi=10.1186/s41065-017-0049-x;kwrd=Prunus amygdalus Batsch,Transcriptome,Functional classification,SSR markers;pmc=L32030,L21001;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-017-0049-x&amp;kwrd=Prunus amygdalus Batsch,Transcriptome,Functional classification,SSR markers&amp;pmc=L32030,L21001&amp;">
                        <img data-test="gpt-advert-fallback-img"
                             src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-017-0049-x&amp;kwrd=Prunus amygdalus Batsch,Transcriptome,Functional classification,SSR markers&amp;pmc=L32030,L21001&amp;"
                             alt="Advertisement"
                             width="728"
                             height="90">
                    </a>
                </noscript>
            </div>
        </div>
    </div>

        
         
    <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>


<header class="c-popup-search ctx-publisher-header" id="top" data-test="publisher-header">
    <nav class="c-navbar">
        <div class="u-container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization" />
                <div class="c-logo">
    <a href="https://www.biomedcentral.com" itemprop="url">
        <img src=/static/images/bmc/logos/logo-bmc-white-aj-e3a70ae848.svg alt="BMC" itemprop="logo">
        <div class="c-logo__strapline">
            <img src=/static/images/bmc/logos/logo-bmc-white-strapline-sn-e7b1d6e67f.svg alt="Part of Springer Nature">
        </div>
    </a>
</div>
            </div>
            <button
                type="button"
                class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg"
                id="publisher-header-menu-button"
                data-toggle="collapse"
                data-test="publisher-header-menu-toggle"
                data-toggle-linked="publisher-header"
                data-target="publisher-header-search publisher-header-menu"
                aria-controls="publisher-header-menu publisher-header-search"
                aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


                <ul class="c-navbar__nav">
                    <li class="c-navbar__item">
                        <a data-replace="true"
                           data-placeholder="/placeholder/v1/username"
                           class="c-navbar__link"
                           href="https://www.biomedcentral.com/login"
                           data-test="login-link"
                           data-track="click"
                           data-track-category="Account"
                           data-track-action="Click Login Link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                    <li class="c-navbar__item">
                        <button
                            type="button"
                            class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg"
                            data-toggle="collapse"
                            data-test="header-search-button"
                            data-target="publisher-header-search publisher-header-menu publisher-header-menu-button"
                            aria-controls="publisher-header-search publisher-header-menu"
                            aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true">
                                    <use xlink:href="#icon-search"></use>
                                </svg>
                            </span>
                        </button>
                    </li>
                </ul>
            </div>
        </div>
    </nav>
    <div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field c-form-field--inline" method="GET" action="//www.biomedcentral.com/search" data-track="submit"
        data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label data-track-label="" data-test="global-search">
        <label for="publisherSearch" class="u-visually-hidden">Search all BMC articles</label>
        <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text" value="" placeholder="Search all BMC articles"/>
        <button class="c-button c-button--secondary" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true">
        <use xlink:href="#icon-search"></use>
    </svg>
</button>


        <input type="hidden" name="searchType" value="publisherSearch"/>
    </form>
</div>

            </div>
        </div>
    </div>
</header>


        
            <header class="c-journal-header c-journal-header--hereditas ctx-journal-header">
                <div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="/">
    
        
    
    <span class="c-journal-title__text ">Hereditas</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="u-container">
                        
                            <button type="button"
                                    class="c-navbar__link c-dropdown u-hidden-at-lg"
                                    data-toggle="collapse"
                                    data-test="journal-header-menu"
                                    data-toggle-linked="journal-header"
                                    data-target="journal-menu"
                                    aria-controls="journal-menu"
                                    aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav">
                                    <ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/'>Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/about'>About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href='/articles'>Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/submission-guidelines'>Submission Guidelines</a>
                                            </li>
                                        
                                    </ul>
                                </nav>
                            </div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--ocean-light-blue">
                    
                </div>
            </header>
            
        

        <div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-content-layout c-content-layout--fulltext" data-component="sticky-container">
                <div class="c-content-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3><ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc=""><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results">Results</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec9" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Discussion">Discussion</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec16" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Conclusion">Conclusion</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec10" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Methods">Methods</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li></ol></nav>
                </div>

                <main class="c-content-layout__main">
                    <article itemscope itemtype="http://schema.org/ScholarlyArticle">
                        
                        
    <div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
            <li class="c-article-identifiers__item" data-test="article-category">Research</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
    </div>

                        <div class="FulltextWrapper"><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">Characterizing the transcriptome and microsatellite markers for almond (<em xmlns="" class="EmphasisTypeItalic">Amygdalus communis</em> L.) using the Illumina sequencing platform</h1></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true"><ul class="u-listReset"><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Linsen&nbsp;Zhang</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Xiaoni&nbsp;Yang</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Xiangning&nbsp;Qi</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Chunhui&nbsp;Guo</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup> and </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Zhaobin&nbsp;Jing</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup><a href="mailto:jingzhaobin@nwsuaf.edu.cn" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a></li></ul></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation"><div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">Hereditas</span></span><span class="ArticleCitation_Year">2017</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">155</strong>:14</span></span></div><p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/s41065-017-0049-x">https://doi.org/10.1186/s41065-017-0049-x</a></p><p class="Copyright">© &nbsp;The Author(s)&nbsp;2017</p></div></div><ul class="ArticleHistory u-reset-list"><li class="History HistoryReceived"><strong>Received: </strong><span>5&nbsp;December&nbsp;2016</span></li><li class="History HistoryAccepted"><strong>Accepted: </strong><span>9&nbsp;October&nbsp;2017</span></li><li class="History HistoryOnlineDate"><strong>Published: </strong><span itemprop="datePublished">19&nbsp;October&nbsp;2017</span></li></ul></div>
                        
                        
                        
    

                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2><div class="c-section__content" id="Abstract"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p id="Par1" class="Para">The almond tree (<em xmlns="" class="EmphasisTypeItalic">Prunus amygdalus</em> Batsch) is an important nut tree grown in subtropical regions that produces nutrient-rich nuts. However, a paucity of genomic information and DNA markers has restricted the development of modern breeding technologies for almond trees.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Results</h3>
                <p id="Par2" class="Para">In this study, almonds were sequenced with Illumina paired-end sequencing technology to obtain transcriptome data and develop simple sequence repeats (SSR) markers. We generated approximately 64 million clean reads from the various tissues of mixed almonds, and a total of 42,135 unigenes with an average length of 988&nbsp;bp were obtained in the present study. A total of 27,586 unigenes (57.7% of all unigenes generated) were annotated using several databases. A total of 112,812 unigenes were annotated with the Gene Ontology (GO) database and assigned to 82 functional sub-groups, and 29,075 unigenes were assigned to the KOG database and classified into 25 function classifications. There were 9470 unigenes assigned to 129 Kyoto Encyclopaedia of Genes and Genomes (KEGG) pathways from five categories in the KEGG pathway database. We further identified 8641 SSR markers from 48,012 unigenes. A total of 100 SSR markers were randomly selected to validate quality, and 82 markers could amplify the specific products of <em xmlns="" class="EmphasisTypeItalic">A. communis</em> L., whereas 70 markers were successfully transferable to five species (<em xmlns="" class="EmphasisTypeItalic">A. ledebouriana, A. mongolica, A. pedunculata, A. tangutica,</em> and <em xmlns="" class="EmphasisTypeItalic">A. triloba</em>).</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusions</h3>
                <p id="Par3" class="Para">Our study was the first to produce public transcriptome data from almonds. The development of SSR markers will promote genetics research and breeding programmes for almonds.</p>
              </div></div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords"><li class="c-keywords__item" itemprop="about">
                        <em xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="EmphasisTypeItalic">Prunus amygdalus</em> Batsch</li><li class="c-keywords__item" itemprop="about">Transcriptome</li><li class="c-keywords__item" itemprop="about">Functional classification</li><li class="c-keywords__item" itemprop="about">SSR markers</li></ul></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par9" class="Para">The almond (<em xmlns="" class="EmphasisTypeItalic">Prunus amygdalus</em> Batsch) is one of the world’s most important nut crops. The almond belongs to the genus <em xmlns="" class="EmphasisTypeItalic">Prunus</em>, subgenus <em xmlns="" class="EmphasisTypeItalic">Amygdalus</em> [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>]. Almonds are widely used in food production and have high nutritional value [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>]. Most <em xmlns="" class="EmphasisTypeItalic">Amygdalus</em> wild species are highly tolerant of cold, drought and salt [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>]. Almonds are a typical outbreeding species with gametophyte self-incompatibility [<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>]. They are also an antioxidant source [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>] and an important germplasm resource in breeding programmes [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>]. Because almonds have high nutritional value, past research has largely focused on physicochemical properties, seed nutrients and oil extraction [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>, <span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>, <span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>]. However, despite high economic and nutritional values, genome and genetic resources regarding almonds are scarce, which has restricted the development of modern breeding technologies for almonds.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par10" class="Para">Molecular markers are important tools used in evaluating genetic diversity among plant species and plant molecular breeding (marker-assisted breeding). In almond trees, expressed sequence tags (ESTs) have been developed, but currently, there are only 3926 ESTs and gene loci available for almond in the NCBI GenBank database. Therefore, the number of ESTs is not sufficient to address almond tree molecular breeding development. Simple sequence repeats (SSRs) are widely used in studies on genetic diversity and relationships of plants because they are highly polymorphic, co-dominant and reproducible [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>]. Moreover, SSRs markers could be used in the QTL mapping of important agronomical traits loci and marker-assisted selection in almond trees [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>–<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>]. Presently, however, only a few SSRs have been reported in almonds [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>–<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>]. SRAP and AFLP markers have been used to study genetic diversity and relationships in wild <em xmlns="" class="EmphasisTypeItalic">Amygdalus</em> species [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>]. A transcriptome is the complete collection of RNA that includes the full range of mRNA, tRNA, rRNA, and other noncoding RNA molecules expressed by one or a group of cells, organs or tissues in a particular environment or a specific developmental stage. The Illumina paired-end sequencing technique has been widely used for transcriptome analysis in plants [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>, <span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>–<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par11" class="Para">Transcriptome sequencing is helpful in developing SSR molecular markers, as it is reliable and efficient [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>]. However, transcriptomic and associated molecular markers in almonds have not yet been reported. In this study, Illumina sequencing technology was used to analyse the transcriptome of almonds and develop EST-SSR markers. To our knowledge, this is the first study to characterize the almond transcriptome. Our study will provide a foundation for almond molecular biology and molecular breeding of almonds.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Results" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Results">Results</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Results" data-test="collapse-section"><section id="Sec3" class="Section2 RenderAsSection2">
                <h3 class="Heading">Transcriptome generation and de novo assembly</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par12" class="Para">A total of 66,668,192 raw reads were generated from our Illumina HiSeq™ 2000 paired-end sequencing of almonds. The total length of the reads was approximately 10 Gigabase pairs (Gb); a total of 64,924,070 (97.38%) high-quality clean reads (1,440,780, 4.32%) as well as low-quality reads (1,702,602, 2.55%) were collected after removing the adapter. All high-quality reads were assembled using the Trinity program, and a total of 42,135 unigenes with an average length of 988&nbsp;bp and an N50 length of 1714&nbsp;bp were obtained. The unigenes ranged from 201&nbsp;bp to 15,555&nbsp;bp (Table&nbsp;<span class="InternalRef"><a href="#Tab1">1</a></span>). As shown in Fig.&nbsp;<span class="InternalRef"><a href="#Fig1">1</a></span>, 27,723 unigenes (65.80%) ranged from 201 to 1000&nbsp;bp, 8699 unigenes (20.65%) were longer than 1000&nbsp;bp and 5713 unigenes (13.56%) were longer than 2000&nbsp;bp. The coverage percentage of read-blasted unigenes was 55.74% (more than 10 reads), 19.17% (more than 100 reads), and 25.09% (more than 1000 reads) (Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>). All of the raw data were submitted to the NCBI database (accession number: PRJNA347906).<figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 1</span>
                        <p xmlns="" class="SimplePara">Summary of transcriptome data for persimmon</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Item</p></th><th><p xmlns="" class="SimplePara">Number</p></th></tr></thead><tbody><tr class="noclass"><td colspan="2"><p xmlns="" class="SimplePara">1. Raw sequences and Assembly statistics</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara"> Total reads</p></td><td><p xmlns="" class="SimplePara">66,668,192</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara"> Total nucleotides (bp)</p></td><td><p xmlns="" class="SimplePara">10,000,228,800</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara"> GC content percentage</p></td><td><p xmlns="" class="SimplePara">46.77%</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara"> Total unigenes (average length; N50; min-max length)</p></td><td><p xmlns="" class="SimplePara">42,135 (988; 1714; 201–15,555)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara"> Mean RPKM value of unigenes (min-mix RPKM value)</p></td><td><p xmlns="" class="SimplePara">14.8 (0–7109)</p></td></tr><tr class="noclass"><td colspan="2"><p xmlns="" class="SimplePara">2. Bioinformatics annotations of mango fruit unigenes</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara"> Gene annotation against Nr (%)</p></td><td><p xmlns="" class="SimplePara">29,315 (69.6)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara"> Gene annotation against Swiss-Prot (%)</p></td><td><p xmlns="" class="SimplePara">20,642 (49.0)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara"> Gene annotation against KEGG (%)</p></td><td><p xmlns="" class="SimplePara">11,055 (26.2)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara"> Gene annotation against KOG (%)</p></td><td><p xmlns="" class="SimplePara">16,794 (39.9)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara"> All annotation genes (%)</p></td><td><p xmlns="" class="SimplePara">29,464 (69.9)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara"> Without annotation genes (%)</p></td><td><p xmlns="" class="SimplePara">12,671 (30.1)</p></td></tr><tr class="noclass"><td colspan="2"><p xmlns="" class="SimplePara"> GO Ontology (%)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">  Biological process category</p></td><td><p xmlns="" class="SimplePara">51,202;</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">  Cellular component category</p></td><td><p xmlns="" class="SimplePara">24,483;</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">  Molecular function category</p></td><td><p xmlns="" class="SimplePara">37,133;</p></td></tr></tbody></table></div></figure>
                           <figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO1">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0049-x/MediaObjects/41065_2017_49_Fig1_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0049-x/MediaObjects/41065_2017_49_Fig1_HTML.gif" alt="Fig. 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0049-x/MediaObjects/41065_2017_49_Fig1_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0049-x/MediaObjects/41065_2017_49_Fig1_HTML.gif" alt="Fig. 1"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 1</span>
                        <p xmlns="" class="SimplePara">Length distributions of the unigenes. The x-axis indicates a different sequence size, and the y-axis indicates the unigene numbers of a specific sequence size</p>
                      </div></figcaption></figure>
                        </div>
              </section><section id="Sec4" class="Section2 RenderAsSection2">
                <h3 class="Heading">Functional annotation</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par13" class="Para">The function annotation of the unigenes was performed in the Nr, Swiss-Prot, Kyoto Encyclopaedia of Genes and Genomes (KEGG) and KOG databases by BLASTX (E-value &lt; 10<sup>−5</sup>). A total of 27,586 unigenes (57.7% of all unigenes) were annotated. A total of 29,315 unigenes (69.6% of all unigenes) of the largest match were annotated in the Nr database, followed by the Swiss-Prot (20,642, 49.0%), KOG (16,794, 39.9%) and KEGG (11,055, 26.2%) databases (Fig.&nbsp;<span class="InternalRef"><a href="#Fig2">2</a></span>). A total of 12,671 (42.5%) unigenes were not annotated in the four databases, indicating that these unigenes may be novel genes (Table <span class="InternalRef"><a href="#Tab1">1</a></span>). For the annotated unigenes in the Nr databases, the homologous sequences belonging to the species were analysed. The ten top-hit species were <em xmlns="" class="EmphasisTypeItalic">Prunus mume</em> (18,097, 61.73%), <em xmlns="" class="EmphasisTypeItalic">Malus domestica</em> (15,80, 5.40%), <em xmlns="" class="EmphasisTypeItalic">Theobroma cacao</em> (1238, 4.22%), <em xmlns="" class="EmphasisTypeItalic">Pyrus x bretschneideri</em> (1132, 3.86%), <em xmlns="" class="EmphasisTypeItalic">Prunus persica</em> (1061, 3.46%), <em xmlns="" class="EmphasisTypeItalic">Gossypium arboreum</em> (853, 2.90%), <em xmlns="" class="EmphasisTypeItalic">Brassica napus</em> (710, 2.42%), <em xmlns="" class="EmphasisTypeItalic">Medicago truncatula</em> (682, 2.33%), <em xmlns="" class="EmphasisTypeItalic">Fragaria vesca subsp. vesca</em> (665, 2.27%), <em xmlns="" class="EmphasisTypeItalic">Morus notabilis</em> (311, 1.06%), and others (3029, 10.34%).<figure xmlns="" class="Figure" id="Fig2"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO2">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0049-x/MediaObjects/41065_2017_49_Fig2_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0049-x/MediaObjects/41065_2017_49_Fig2_HTML.gif" alt="Fig. 2" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0049-x/MediaObjects/41065_2017_49_Fig2_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0049-x/MediaObjects/41065_2017_49_Fig2_HTML.gif" alt="Fig. 2"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 2</span>
                        <p xmlns="" class="SimplePara">Venn diagram of Nr, Swiss-Prot, KOG and KEGG databases. Venn diagram showing homology sequence numbers of the unigenes in the Nr, Swiss-Prot, GOG and KEGG databases</p>
                      </div></figcaption></figure>
                        </div>
              </section><section id="Sec5" class="Section2 RenderAsSection2">
                <h3 class="Heading">Functional classification by gene ontology (GO) and KOG</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par14" class="Para">To further evaluate the functions of the almond unigenes, we used GO assignments to classify the almond unigene functions. A total of 112,812 unigenes were assigned to 82 functional sub-groups. Of the three ontology categories, the largest was biological process (51,202 unigenes), followed by cellular component (37,133 unigenes) and molecular function (24,483 unigenes) (Fig.&nbsp;<span class="InternalRef"><a href="#Fig3">3</a></span>). For the biological process group, the most frequent process was metabolic process (11,647, 22.75%), followed by cellular process (10,967, 21.42%). Cell (8581, 23.11%) and cell part (8581, 23.11%) were the most highly represented groups in the cellular component category. For the molecular function category, binding (11,139, 45.50%) and catalytic activity (10,346, 27.86%) represented the greatest proportion. The GO classifications of the unigenes are listed in Additional file <span class="InternalRef"><a href="#MOESM2">2</a></span>.<figure xmlns="" class="Figure" id="Fig3"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO3">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0049-x/MediaObjects/41065_2017_49_Fig3_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0049-x/MediaObjects/41065_2017_49_Fig3_HTML.gif" alt="Fig. 3" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0049-x/MediaObjects/41065_2017_49_Fig3_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0049-x/MediaObjects/41065_2017_49_Fig3_HTML.gif" alt="Fig. 3"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 3</span>
                        <p xmlns="" class="SimplePara">GO function classification of the almond transcriptome. There are three main GO categories: biological process, cellular component and molecular function. The x-axis indicates the categories, and the y-axis indicates the number of the unigenes</p>
                      </div></figcaption></figure>
                        </div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par15" class="Para">KOG classifications were searched based on a BLAST search against the KOG database. A total of 29,075 unigenes were classified into 25 function classifications (Fig.&nbsp;<span class="InternalRef"><a href="#Fig4">4</a></span>). For the 25 KOG categories, the general function prediction was the largest group (5895, 20.28%); posttranslational modification, protein turnover, chaperones (3311, 11.39%), and signal transduction mechanisms (3152, 10.84%) had high percentages, and 15,012 unigenes were assigned to other functional categories (Additional file <span class="InternalRef"><a href="#MOESM3">3</a></span>).<figure xmlns="" class="Figure" id="Fig4"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO4">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0049-x/MediaObjects/41065_2017_49_Fig4_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0049-x/MediaObjects/41065_2017_49_Fig4_HTML.gif" alt="Fig. 4" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0049-x/MediaObjects/41065_2017_49_Fig4_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0049-x/MediaObjects/41065_2017_49_Fig4_HTML.gif" alt="Fig. 4"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 4</span>
                        <p xmlns="" class="SimplePara">KOG function classification of the almond transcriptome. A total of 25 categories were obtained. The x-axis indicates the categories, and the y-axis indicates the numbers of the unigenes</p>
                      </div></figcaption></figure>
                        </div>
              </section><section id="Sec6" class="Section2 RenderAsSection2">
                <h3 class="Heading">Functional classifications by KEGG</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par16" class="Para">The pathway annotations were used to analyse the biological functions of genes. In this study, 9470 unigenes were assigned to 129 KEGG pathways that belonged to five categories, namely, metabolic pathways (5648, 59.60%), genetic information processing (2636, 27.8%), cellular processes (512, 5.4%), environmental information processing (389, 4.1%) and organismal systems (285, 3.0%) (Additional file <span class="InternalRef"><a href="#MOESM4">4</a></span>). The majority of the unigene pathways were associated with ribosomes (443, 7.3%), carbon metabolism (320, 5.3%), protein processing in the endoplasmic reticulum (290, 4.8%), biosynthesis of amino acids (277, 4.9%), spliceosomes (274, 4.5%), plant hormone signal transduction (249, 4.1%) and endocytosis (245, 4.0%).</p>
              </section><section id="Sec7" class="Section2 RenderAsSection2">
                <h3 class="Heading">Development and characterisation of SSR markers</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par17" class="Para">In this study, the unigene sequences were used to develop new SSR markers with MISA software. A total of 8641 SSRs were identified from 48,012 unigenes. For the 8641 SSRs, di-nucleotide motifs were the most abundant form (5141, 59.5%), followed by tri-nucleotides (2416, 28.5%), tetra-nucleotides (606, 7.0%), hexa-nucleotides (277, 3.2%) and penta-nucleotides (201, 2.3%) (Table&nbsp;<span class="InternalRef"><a href="#Tab2">2</a></span>). In addition, the number of repeated units of the di-nucleotide motifs ranged from 6 to 15, and the tri-nucleotide, tetra-nucleotide, penta-nucleotide, and hexa-nucleotide motifs included 5 to 15, 4 to 10, 4 to 8, and 4 to 7, respectively. SSRs with six tandem repeats were the most frequent (1795, 20.8%), followed by five tandem repeats (1543, 17.9%), more than fifteen tandem repeats (1228, 14.2%), seven tandem repeats (1095, 12.7%), and others (Table&nbsp;<span class="InternalRef"><a href="#Tab3">3</a></span>). The most frequent motif types of SSRs were AG/CT (49.0%), followed by AAG/CTT (9.3%), AT/AT (5.9%), AC/GT (4.5%), AGG/CCT (3.6%) and others (Additional file <span class="InternalRef"><a href="#MOESM5">5</a></span>).<figure xmlns="" id="Tab2" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 2</span>
                        <p xmlns="" class="SimplePara">Summary of the EST-SSRr data for almond</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Item</p></th><th><p xmlns="" class="SimplePara">Number</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">Total number of sequences examined</p></td><td><p xmlns="" class="SimplePara">42,135</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Total size of examined sequences (bp)</p></td><td><p xmlns="" class="SimplePara">41,663,061</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Total number of identified SSRs</p></td><td><p xmlns="" class="SimplePara">8641</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Number of SSR containing sequences</p></td><td><p xmlns="" class="SimplePara">6780</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Number of sequences containing more than 1 SSR</p></td><td><p xmlns="" class="SimplePara">1433</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Number of SSRs present in compound formation</p></td><td><p xmlns="" class="SimplePara">752</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Di-nucleotide</p></td><td><p xmlns="" class="SimplePara">5141 (59.5%)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Tri-nucleotide</p></td><td><p xmlns="" class="SimplePara">2416 (28.5%)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Tetra-nucleotide</p></td><td><p xmlns="" class="SimplePara">606 (7.0%)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Penta-nucleotide</p></td><td><p xmlns="" class="SimplePara">201 (2.3%)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Hexa-nucleotide</p></td><td><p xmlns="" class="SimplePara">277 (3.2%)</p></td></tr></tbody></table></div></figure>
                           <figure xmlns="" id="Tab3" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 3</span>
                        <p xmlns="" class="SimplePara">Summary of the different repeat units of identified EST-SSRs</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Number of repeat unit</p></th><th><p xmlns="" class="SimplePara">Di-</p></th><th><p xmlns="" class="SimplePara">Tri-</p></th><th><p xmlns="" class="SimplePara">Tetra-</p></th><th><p xmlns="" class="SimplePara">Penta-</p></th><th><p xmlns="" class="SimplePara">Hexa</p></th><th><p xmlns="" class="SimplePara">Total</p></th><th><p xmlns="" class="SimplePara">Percentage (%)</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">4</p></td><td><p xmlns="" class="SimplePara">0</p></td><td><p xmlns="" class="SimplePara">0</p></td><td><p xmlns="" class="SimplePara">435</p></td><td><p xmlns="" class="SimplePara">151</p></td><td><p xmlns="" class="SimplePara">200</p></td><td><p xmlns="" class="SimplePara">786</p></td><td><p xmlns="" class="SimplePara">9.1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">5</p></td><td><p xmlns="" class="SimplePara">0</p></td><td><p xmlns="" class="SimplePara">1321</p></td><td><p xmlns="" class="SimplePara">121</p></td><td><p xmlns="" class="SimplePara">39</p></td><td><p xmlns="" class="SimplePara">62</p></td><td><p xmlns="" class="SimplePara">1543</p></td><td><p xmlns="" class="SimplePara">17.9</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">6</p></td><td><p xmlns="" class="SimplePara">1156</p></td><td><p xmlns="" class="SimplePara">592</p></td><td><p xmlns="" class="SimplePara">31</p></td><td><p xmlns="" class="SimplePara">7</p></td><td><p xmlns="" class="SimplePara">9</p></td><td><p xmlns="" class="SimplePara">1795</p></td><td><p xmlns="" class="SimplePara">20.8</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">7</p></td><td><p xmlns="" class="SimplePara">771</p></td><td><p xmlns="" class="SimplePara">302</p></td><td><p xmlns="" class="SimplePara">13</p></td><td><p xmlns="" class="SimplePara">3</p></td><td><p xmlns="" class="SimplePara">6</p></td><td><p xmlns="" class="SimplePara">1095</p></td><td><p xmlns="" class="SimplePara">12.7</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">8</p></td><td><p xmlns="" class="SimplePara">596</p></td><td><p xmlns="" class="SimplePara">67</p></td><td><p xmlns="" class="SimplePara">4</p></td><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">0</p></td><td><p xmlns="" class="SimplePara">668</p></td><td><p xmlns="" class="SimplePara">7.7</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">9</p></td><td><p xmlns="" class="SimplePara">566</p></td><td><p xmlns="" class="SimplePara">38</p></td><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">0</p></td><td><p xmlns="" class="SimplePara">0</p></td><td><p xmlns="" class="SimplePara">605</p></td><td><p xmlns="" class="SimplePara">7.0</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">10</p></td><td><p xmlns="" class="SimplePara">439</p></td><td><p xmlns="" class="SimplePara">58</p></td><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">0</p></td><td><p xmlns="" class="SimplePara">0</p></td><td><p xmlns="" class="SimplePara">498</p></td><td><p xmlns="" class="SimplePara">5.8</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">11</p></td><td><p xmlns="" class="SimplePara">237</p></td><td><p xmlns="" class="SimplePara">11</p></td><td><p xmlns="" class="SimplePara">0</p></td><td><p xmlns="" class="SimplePara">0</p></td><td><p xmlns="" class="SimplePara">0</p></td><td><p xmlns="" class="SimplePara">248</p></td><td><p xmlns="" class="SimplePara">2.9</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">12</p></td><td><p xmlns="" class="SimplePara">38</p></td><td><p xmlns="" class="SimplePara">9</p></td><td><p xmlns="" class="SimplePara">0</p></td><td><p xmlns="" class="SimplePara">0</p></td><td><p xmlns="" class="SimplePara">0</p></td><td><p xmlns="" class="SimplePara">47</p></td><td><p xmlns="" class="SimplePara">0.5</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">13</p></td><td><p xmlns="" class="SimplePara">5</p></td><td><p xmlns="" class="SimplePara">10</p></td><td><p xmlns="" class="SimplePara">0</p></td><td><p xmlns="" class="SimplePara">0</p></td><td><p xmlns="" class="SimplePara">0</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">0.2</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">14</p></td><td><p xmlns="" class="SimplePara">109</p></td><td><p xmlns="" class="SimplePara">4</p></td><td><p xmlns="" class="SimplePara">0</p></td><td><p xmlns="" class="SimplePara">0</p></td><td><p xmlns="" class="SimplePara">0</p></td><td><p xmlns="" class="SimplePara">113</p></td><td><p xmlns="" class="SimplePara">1.3</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">≥ 15</p></td><td><p xmlns="" class="SimplePara">1224</p></td><td><p xmlns="" class="SimplePara">4</p></td><td><p xmlns="" class="SimplePara">0</p></td><td><p xmlns="" class="SimplePara">0</p></td><td><p xmlns="" class="SimplePara">0</p></td><td><p xmlns="" class="SimplePara">1228</p></td><td><p xmlns="" class="SimplePara">14.2</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Total</p></td><td><p xmlns="" class="SimplePara">5141</p></td><td><p xmlns="" class="SimplePara">2416</p></td><td><p xmlns="" class="SimplePara">606</p></td><td><p xmlns="" class="SimplePara">201</p></td><td><p xmlns="" class="SimplePara">277</p></td><td><p xmlns="" class="SimplePara">8641</p></td><td><p xmlns="" class="SimplePara">100</p></td></tr></tbody></table></div></figure>
                        </div>
              </section><section id="Sec8" class="Section2 RenderAsSection2">
                <h3 class="Heading">Cross-species transferability of <em class="EmphasisTypeItalic">A. communis</em> SSR markers</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par18" class="Para">One hundred SSR sites were randomly selected to design SSR primers (Additional file <span class="InternalRef"><a href="#MOESM6">6</a></span>). Among these 100 primer pairs, 82 could amplify the specific products (these 82 markers are highlighted in red in Additional file <span class="InternalRef"><a href="#MOESM6">6</a></span>), while the remaining 18 did not generate PCR products. To validate the transferability of <em xmlns="" class="EmphasisTypeItalic">A. communis</em> SSR markers, five species (<em xmlns="" class="EmphasisTypeItalic">A. ledebouriana, A. mongolica, A. pedunculata, A. tangutica, and A. triloba</em>) (Additional file <span class="InternalRef"><a href="#MOESM7">7</a></span>) were assessed using the 82 SSR markers selected above. The results indicated that 70 SSR markers were transferable to these five species and that 12 SSR markers did not generate bands. The PCR amplification results of some primers are shown in Fig.&nbsp;<span class="InternalRef"><a href="#Fig5">5</a></span>. The UPMGA cluster analysis indicated that <em xmlns="" class="EmphasisTypeItalic">A. communis</em> and <em xmlns="" class="EmphasisTypeItalic">A. mongolica</em> are more closely related (Fig.&nbsp;<span class="InternalRef"><a href="#Fig6">6</a></span>).<figure xmlns="" class="Figure" id="Fig5"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO5">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0049-x/MediaObjects/41065_2017_49_Fig5_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0049-x/MediaObjects/41065_2017_49_Fig5_HTML.gif" alt="Fig. 5" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0049-x/MediaObjects/41065_2017_49_Fig5_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0049-x/MediaObjects/41065_2017_49_Fig5_HTML.gif" alt="Fig. 5"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 5</span>
                        <p xmlns="" class="SimplePara">Examples of polymorphic products amplified by different SSR primer pairs. 1–25 represent different primer pairs. M represents DL2000 DNA markers</p>
                      </div></figcaption></figure>
                           <figure xmlns="" class="Figure" id="Fig6"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO6">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0049-x/MediaObjects/41065_2017_49_Fig6_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0049-x/MediaObjects/41065_2017_49_Fig6_HTML.gif" alt="Fig. 6" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0049-x/MediaObjects/41065_2017_49_Fig6_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0049-x/MediaObjects/41065_2017_49_Fig6_HTML.gif" alt="Fig. 6"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 6</span>
                        <p xmlns="" class="SimplePara">Dendrogram obtained using UPGMA cluster analysis based on Nei’s genetic distances among the ten populations of almonds. The population abbreviations are the same as those in Additional file <span xmlns="http://www.w3.org/1999/xhtml" class="InternalRef"><a href="#MOESM7">7</a></span>
                                    </p>
                      </div></figcaption></figure>
                        </div>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec9" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Discussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Discussion">Discussion</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Discussion" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par19" class="Para">Almonds are one of the most important commercially cultivated crops in subtropical regions, specifically in southwest Asia, the Middle East, and the Mediterranean [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>] because almonds have a high nutrient value. Previous studies concerning almonds focused on physicochemical properties, seed nutrients and oil extraction [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>, <span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>, <span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>]. However, no studies have yet constructed a genetic linkage map, including QTL mapping of important agronomical traits and marker-assisted selection in almond trees. Until now, genome sequencing of some important fruits has been completed using NGS technologies. Transcriptome analysis based on NGS technologies, such as Illumina and 454 sequencing platforms, has provided an efficient tool for obtaining genomic data for some plants without a reference genome, such as wax gourds [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>] and pumpkins [<span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>]. Therefore, we sequenced the almond transcriptome to obtain genomic data and then developed many SSR markers. These transcriptome data will provide information for future studies on breeding and molecular biology.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par20" class="Para">In this study, a large number of transcriptomic unigenes (42135) was obtained using the Illumina HiSeq<sup>TM</sup> 2500 platform, and the average unigene length was 988&nbsp;bp. Consistent with recently published plant species, the average length of the unigenes was relatively long (835&nbsp;bp) compared to black gram (443&nbsp;bp) [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>], caragana (709&nbsp;bp) [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>], wax gourd (709&nbsp;bp) [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>], pumpkin (765&nbsp;bp) [<span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>], Siberian apricot (652&nbsp;bp) [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>], Chinese jujube (473.4&nbsp;bp) [<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>], and safflower (446&nbsp;bp) [<span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>]. These results indicate a higher quality of almond transcriptome sequencing and de novo assembly. In this study, approximately 27,586 unigenes (57.7% of all unigenes) were annotated by BLAST searches of the Nr, GO, Swiss-Prot, KEGG and KOG databases. Moreover, 12,671 (42.5%) of the unigenes did not annotate to any databases. Technical limitations, such as read length and sequencing depth, may account for these unannotated unigenes. For gene annotation, the sequences of unigenes were blasted against the Nr, Swiss-Prot, KEGG, GO and KOG databases. Approximately 27,586 unigenes (57.7% of all unigenes) were annotated in four protein databases, indicating that the transcriptomic data of almonds may have large transcript diversity. Additionally, approximately 12,671 (42.5%) unigenes were not annotated to the four databases, suggesting that some unigenes may be unique to almonds.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par21" class="Para">SSR markers have been important in some research, including the assessment of genetic diversity and genetic relationships, the construction of genetic maps, marker-assisted selection of important agronomic traits, and others [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>, <span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>]. Previous studies have also shown that SSR markers are highly polymorphic, codominant and easily reproducible [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>]. Due to the time-consuming and expensive nature of traditional methods for SSR marker development, few SSR markers have been reported for almonds, and no studies have reported the development of SSR markers in <em xmlns="" class="EmphasisTypeItalic">A. communis</em>, which has limited the application of SSR markers in almond trees. Transcriptome sequencing is an efficient technology for the development of SSR markers in plants, and the SSR markers for some plants have been reported using transcriptome sequencing [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>, <span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>–<span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>]. Our results produced a large number of transcriptome sequences that could be used to develop SSR markers in almonds. In total, 8641 SSR markers were identified from 48,012 unigenes. In this study, di-nucleotide motifs were the most abundant form, followed by tri-nucleotides, tetra-nucleotides, hexa-nucleotides and penta-nucleotides, which is similar to previous studies [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>–<span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>]. In addition, the most abundant di-nucleotides and tri-nucleotides were AG/CT and AAG/CTT, respectively, which was consistent with previous reports [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>–<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>]. To assess the quality of SSR markers, we randomly selected 100 pairs of primers and assessed them in five species. Eighty-two percent showed polymorphisms. This result was similar to results found in other plants. The UPMGA cluster analysis indicated that <em xmlns="" class="EmphasisTypeItalic">A. communis</em> and <em xmlns="" class="EmphasisTypeItalic">A. mongolica</em> were more closely related, which was consistent with Jing et al., who reported using SRAP markers [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>]. We believe that the new SSR markers will be used to study genetic diversity, genetic mapping, and, in particular, marker-assisted breeding for almonds.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec16" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Conclusion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Conclusion">Conclusion</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Conclusion" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par27" class="Para">This paper reports on the transcriptome characterizations of almond trees and provides a large number of SSR markers to elucidate the molecular biology of almond trees. To our knowledge, this is the first attempt to develop SSR markers for almonds using a transcriptome sequencing method, and these developed SSR markers will significantly contribute to genetic diversity studies, QTL mapping, and marker-assisted selection breeding for almonds. Notably, due to high transferability, these SSR markers may provide an efficient tool to accelerate molecular breeding in other <em xmlns="" class="EmphasisTypeItalic">Amygdalus</em> species.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec10" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Methods" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Methods">Methods</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Methods" data-test="collapse-section"><section id="Sec11" class="Section2 RenderAsSection2">
                <h3 class="Heading">Plant materials and RNA extraction</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par22" class="Para">Plants of <em xmlns="" class="EmphasisTypeItalic">A. communis</em> were grown at the experimental farm of Northwest A&amp;F University, Yangling, China. Tissues from leaves, flowers, stems and fruits were harvested from six individuals. The sampled tissues were immediately frozen in liquid nitrogen and stored at −80&nbsp;°C for later RNA extraction. The RNA samples were isolated using an E.Z.N.A.® Plant RNA Kit (Omega Bio-tek, Inc.) according to the manufacturer’s protocol. The quality and quantity of RNA were assessed using electrophoresis on 1% agarose gels and a NanoDrop 1000 spectrophotometer (Thermo Scientific, Wilmington, DE, USA), respectively. High-quality RNA was used for further analyses. Equal amounts of RNA from different samples were pooled for further studies.</p>
              </section><section id="Sec12" class="Section2 RenderAsSection2">
                <h3 class="Heading">cDNA library construction and transcriptome sequencing</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par23" class="Para">After the RNA was extracted, the cDNA library construction was performed using a TransCript® cDNA sample prep kit (TransGen Biotech, China). The ligation products were size-selected with agarose gel electrophoresis, PCR-amplified, and sequenced using Illumina HiSeqTM 2500 by Gene Denovo Biotechnology Co., Ltd. (Guangzhou, China).</p>
              </section><section id="Sec13" class="Section2 RenderAsSection2">
                <h3 class="Heading">Data filtering, de novo assembly and function annotation</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par24" class="Para">Reads obtained from the sequencing machines included raw reads containing adapters or low-quality bases that would affect the following assembly and analysis. Thus, to obtain high-quality clean reads, the clean reads were assembled using the Trinity assembly program [<span class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>]. The isoform was obtained using Trinity software. To eliminate redundant information, the longest isoform was taken as the gene to further analyse; this was defined as the unigene. The functional annotation of unigene sequences was performed by BLASTX search of the non-redundant (Nr) (<span class="ExternalRef"><a href="http://www.ncbi.nlm.nih.gov"><span class="RefSource">http://www.ncbi.nlm.nih.gov</span></a></span>) database, Swiss-Prot protein database, Kyoto Encyclopaedia of Genes and Genomes (KEGG) database (<span class="ExternalRef"><a href="http://www.genome.jp/kegg"><span class="RefSource">http://www.genome.jp/kegg</span></a></span>), Clusters of Orthologous Groups (KOG) database, and Gene Ontology (GO) database with an E-value &lt; 10–5. We used the Blast2GO program to analyse the GO annotations of the unigenes [<span class="CitationRef"><a href="#CR28" data-toggle="citation">28</a></span>]. The functional classifications were determined with WEGO software [<span class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>]. Pathway information of unigenes was collected from KEGG databases [<span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>].</p>
              </section><section id="Sec14" class="Section2 RenderAsSection2">
                <h3 class="Heading">SSR detection and primer design</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par25" class="Para">MISA software (<span class="ExternalRef"><a href="http://pgrc.ipk-gatersleben.de/misa/misa.html"><span class="RefSource">http://pgrc.ipk-gatersleben.de/misa/misa.html</span></a></span>) was used to identify microsatellites in the whole transcriptome. The parameters were as follows: definition (unit_size, min_repeats): 2–6 3–5 4–4 5–4 6–4; interruptions (max_difference_between_2_SSRs): 100. If the distance between two SSRs was shorter than 100&nbsp;bp, they were considered to be one SSR. Based on the MISA results, primer pairs of each SSR loci were designed using Primer premier 3.0 (PREMIER Biosoft International, Palo Alto, CA) in the flanking regions of SSRs.</p>
              </section><section id="Sec15" class="Section2 RenderAsSection2">
                <h3 class="Heading">Validation of SSR markers</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par26" class="Para">To validate the SSR markers, a total of 100 primer pairs were randomly selected and synthesized. Wild almond germplasm from six species (<em xmlns="" class="EmphasisTypeItalic">A. communis, A. ledebouriana, A. mongolica, A. pedunculata, A. tangutica, and A. triloba</em>) were used to validate the SSR markers. The total genomic DNA was extracted from fresh leaves using a plant DNA extraction kit (TIANGEN<sup>®</sup>, China). The DNA quality and concentration were tested with a NanoDrop ND 1000 spectrophotometer (Thermo Scientific, USA). PCR amplification reactions were performed in a 25&nbsp;μL volume, containing 40&nbsp;ng of DNA, 0.2&nbsp;mM dNTPs, 1.5 pM aliquots of forward and reverse primers, 2.5&nbsp;mM Mg<sup>2+</sup>, 1&nbsp;U Taq DNA polymerase (TaKaRa Biotechnology Dalian Co., Ltd., China), and 1× Taq Buffer (10&nbsp;mM Tris-HCl, pH&nbsp;8.3, 50&nbsp;mM KCl). PCR amplification was performed with the following conditions: initial denaturation at 94&nbsp;°C for 5&nbsp;min; 30&nbsp;cycles at 94&nbsp;°C for 30&nbsp;s, a primer-specific annealing temperature for 60&nbsp;s, and 72&nbsp;°C for 90&nbsp;s; and 72&nbsp;°C for 7&nbsp;min. PCR products were separated by electrophoresis on denaturing 6% polyacrylamide gels and visualized using silver staining. The molecular size of the amplified fragments was estimated using a 10-bp DNA ladder (TransGen Biotech, China).</p>
              </section></div></section>
          <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AbbreviationGroup Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Abbreviations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Abbreviations">Abbreviations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Abbreviations"><div class="DefinitionList">
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>GO:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par4" class="Para">Gene Ontology</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>KEGG:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par5" class="Para">Kyoto Encyclopaedia of Genes and Genomes</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>KOG:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par6" class="Para">Clusters of Orthologous Groups</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>Nr:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par7" class="Para">non-redundant</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>SSR:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par8" class="Para">simple sequence repeats marker</p>
                  </dd></dl>
              </div></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments"><h3 xmlns="" class="Heading">Acknowledgements</h3><p xmlns="" class="SimplePara">We would like to thank AJE (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://www.aje.com"><span class="RefSource">https://www.aje.com</span></a></span>/) for their expert language editing of this manuscript.</p><div id="FPar1" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Funding</h3>
                <p id="Par28" class="Para">This work was supported by the earmarked fund for China Agriculture Research System (CARS-28), China Postdoctoral Science Foundation (2015&nbsp;M582712), the Postdoctoral Science Foundation of Shaanxi Province (2016BSHYDZZ07), and the Science and Technology Research and Development Program of Shaanxi Province (2015KTZDNY02-03-01, 2015KJZDNY02-03-02, 2016KJXX-58).</p>
              </div><div id="FPar2" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Availability of data and materials</h3>
                <p id="Par29" class="Para">The data are available at <span class="ExternalRef"><a href="https://www.ncbi.nlm.nih.gov"><span class="RefSource">https://www.ncbi.nlm.nih.gov</span></a></span>/.</p>
              </div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="FormalPara FormalParaRenderingStyle1"><h3 xmlns="" class="Heading">Authors’ contributions</h3><p xmlns="" class="SimplePara">L-SZ and Z-BJ conceived the project and designed the experiments. X-NY, X-NQ, and C-HG performed the experiment. L-SZ, and Z-BJ wrote and revised the paper. All authors discussed the results and commented on the manuscript. All authors read and approved the final manuscript.</p></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Ethics">
              <div id="FPar3" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Ethics approval and consent to participate</h3>
                <p id="Par30" class="Para">Not applicable.</p>
              </div>
              <div id="FPar4" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Consent for publication</h3>
                <p id="Par31" class="Para">Not applicable.</p>
              </div>
              <div id="FPar5" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Competing interests</h3>
                <p id="Par32" class="Para">The authors declare that they have no competing interests.</p>
              </div>
              <div id="FPar6" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Publisher’s Note</h3>
                <p id="Par33" class="Para">Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
              </div>
            </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="License">
                <p xmlns="" class="SimplePara">
                           <strong class="EmphasisTypeBold">Open Access</strong>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/4.0/"><span class="RefSource">http://creativecommons.org/licenses/by/4.0/</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/publicdomain/zero/1.0/"><span class="RefSource">http://creativecommons.org/publicdomain/zero/1.0/</span></a></span>) applies to the data made available in this article, unless otherwise stated.</p>
              </div><aside class="Appendix" id="App1">
              
                <h3 class="Heading">Additional files</h3><div id="widgetContainer" data-test="figshare-container"></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par34" class="Para">
                  <div class="MediaObject" id="MOESM1">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0049-x/MediaObjects/41065_2017_49_MOESM1_ESM.png"><span class="caption-number">Additional file 1:</span></a>
                          <span xmlns="" class="SimplePara">The coverage percentage of reads blasted unigenes. (PNG 25 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM2">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0049-x/MediaObjects/41065_2017_49_MOESM2_ESM.xlsx"><span class="caption-number">Additional file 2:</span></a>
                          <span xmlns="" class="SimplePara">Frequency distribution of SSRs from different motif types. (XLSX 11 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM3">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0049-x/MediaObjects/41065_2017_49_MOESM3_ESM.xlsx"><span class="caption-number">Additional file 3:</span></a>
                          <span xmlns="" class="SimplePara">The gene ontology classification of assembled unigenes. (XLSX 10 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM4">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0049-x/MediaObjects/41065_2017_49_MOESM4_ESM.xlsx"><span class="caption-number">Additional file 4:</span></a>
                          <span xmlns="" class="SimplePara">The details of KOG classifications. (XLSX 14 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM5">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0049-x/MediaObjects/41065_2017_49_MOESM5_ESM.png"><span class="caption-number">Additional file 5:</span></a>
                          <span xmlns="" class="SimplePara">The 129 KEGG pathway annotations. (PNG 11 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM6">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0049-x/MediaObjects/41065_2017_49_MOESM6_ESM.xlsx"><span class="caption-number">Additional file 6:</span></a>
                          <span xmlns="" class="SimplePara">SSR sites developed in this study. (XLSX 1402 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM7">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0049-x/MediaObjects/41065_2017_49_MOESM7_ESM.docx"><span class="caption-number">Additional file 7:</span></a>
                          <span xmlns="" class="SimplePara">List of species in this study. (DOCX 16 kb)</span>
                        </div></div>
                  </div>
                </div>
              
            </aside></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authors’ Affiliations</a></h2><div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1"><span class="AffiliationNumber">(1)</span><div class="AffiliationText">College of Horticulture, Northwest A&amp;F University, Yangling, 712100, Shaanxi, People’s Republic of China</div></div></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper"><li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sorkheh K, Dehkordi MK, Ercisli S, Hegedus A, Julia H. Comparison of traditional and new generation DNA markers delares high genetic diversity and differentiated population structure of wild almond species. Sci Rep. 2017;7:5966.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/s41598-017-06084-4"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28729554"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5519738"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparison%20of%20traditional%20and%20new%20generation%20DNA%20markers%20delares%20high%20genetic%20diversity%20and%20differentiated%20population%20structure%20of%20wild%20almond%20species&amp;author=K.%20Sorkheh&amp;author=MK.%20Dehkordi&amp;author=S.%20Ercisli&amp;author=A.%20Hegedus&amp;author=H.%20Julia&amp;journal=Sci%20Rep&amp;volume=7&amp;pages=5966&amp;publication_year=2017"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Jing ZB, Cheng JM, Guo CH, Wang XP. Seed traits, nutrient elements and assessment of genetic diversity for almond (Amygdalus spp.) endangered to China as revealed using SRAP markers. Bioche Syst Ecol. 2013;49:51–7.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.bse.2013.03.015"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Seed%20traits%2C%20nutrient%20elements%20and%20assessment%20of%20genetic%20diversity%20for%20almond%20%28Amygdalus%20spp.%29%20endangered%20to%20China%20as%20revealed%20using%20SRAP%20markers&amp;author=ZB.%20Jing&amp;author=JM.%20Cheng&amp;author=CH.%20Guo&amp;author=XP.%20Wang&amp;journal=Bioche%20Syst%20Ecol&amp;volume=49&amp;pages=51-57&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Arman MO, Mohammad RG. Assessment of genetic diversity in late flowering almond varieties using ISSR molecular markers aimed to select genotypes tolerant to early spring frost in Yazd province. Curr Bot. 2011;2(1):01–4.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Assessment%20of%20genetic%20diversity%20in%20late%20flowering%20almond%20varieties%20using%20ISSR%20molecular%20markers%20aimed%20to%20select%20genotypes%20tolerant%20to%20early%20spring%20frost%20in%20Yazd%20province&amp;author=MO.%20Arman&amp;author=RG.%20Mohammad&amp;journal=Curr%20Bot&amp;volume=2&amp;issue=1&amp;pages=01-04&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gradziel T, Martinez-Gomez P, Dicenta F, Kester D. The utilization of related Prunus species for almond variety improvement. J Am Pomo Soc. 2001;55:100–8.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20utilization%20of%20related%20Prunus%20species%20for%20almond%20variety%20improvement&amp;author=T.%20Gradziel&amp;author=P.%20Martinez-Gomez&amp;author=F.%20Dicenta&amp;author=D.%20Kester&amp;journal=J%20Am%20Pomo%20Soc&amp;volume=55&amp;pages=100-108&amp;publication_year=2001"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li N, Xia X, Chen C, Blumberg J, Song Y, Zhang W, Zhang X, et al. Almond consumption reduces oxidative DNA damage and lipid peroxidation in male smokers. J Nutr. 2007;137:2717–22.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18029489"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Almond%20consumption%20reduces%20oxidative%20DNA%20damage%20and%20lipid%20peroxidation%20in%20male%20smokers&amp;author=N.%20Li&amp;author=X.%20Xia&amp;author=C.%20Chen&amp;author=J.%20Blumberg&amp;author=Y.%20Song&amp;author=W.%20Zhang&amp;author=X.%20Zhang&amp;journal=J%20Nutr&amp;volume=137&amp;pages=2717-2722&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Omirshat T, Geng YP, Zeng LY, Dong SS, Chen F, Chen J, Song ZP, et al. Assessment of genetic diversity and population structure of Chinese wild almond, Amygdalus nana, using EST and genomic SSRs. Biochem Sys Ecol. 2009;37:46–153.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.bse.2008.11.014"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Assessment%20of%20genetic%20diversity%20and%20population%20structure%20of%20Chinese%20wild%20almond%2C%20Amygdalus%20nana%2C%20using%20EST%20and%20genomic%20SSRs&amp;author=T.%20Omirshat&amp;author=YP.%20Geng&amp;author=LY.%20Zeng&amp;author=SS.%20Dong&amp;author=F.%20Chen&amp;author=J.%20Chen&amp;author=ZP.%20Song&amp;journal=Biochem%20Sys%20Ecol&amp;volume=37&amp;pages=46-153&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Balvardi M, Mendiola J, Castro-Gomez P, Fontecha J, Rezaei K, Ibanez E. Development of pressurized extraction processes for oil recovery from wild almond (Amygdalus scoparia). J Am Oil Chem Soc. 2015;92:1503–11.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11746-015-2708-2"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Development%20of%20pressurized%20extraction%20processes%20for%20oil%20recovery%20from%20wild%20almond%20%28Amygdalus%20scoparia%29&amp;author=M.%20Balvardi&amp;author=J.%20Mendiola&amp;author=P.%20Castro-Gomez&amp;author=J.%20Fontecha&amp;author=K.%20Rezaei&amp;author=E.%20Ibanez&amp;journal=J%20Am%20Oil%20Chem%20Soc&amp;volume=92&amp;pages=1503-1511&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Rezaei A, Nasirpour A, Tavanai H. Fractionation and some physicochemical properties of almond gum (Amygdalus communis L.) exudates. Food Hydrocoll. 2016;60:461–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.foodhyd.2016.04.027"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Fractionation%20and%20some%20physicochemical%20properties%20of%20almond%20gum%20%28Amygdalus%20communis%20L.%29%20exudates&amp;author=A.%20Rezaei&amp;author=A.%20Nasirpour&amp;author=H.%20Tavanai&amp;journal=Food%20Hydrocoll&amp;volume=60&amp;pages=461-469&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yang Y, Jing ZB, Ruan X, Cheng JM. Development of simple sequence repeat markers in persimmon (Diospyros L.) and their potential use in related species. Genet Mol Res. 2015;14(1):609–18.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.4238/2015.January.30.2"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25729996"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Development%20of%20simple%20sequence%20repeat%20markers%20in%20persimmon%20%28Diospyros%20L.%29%20and%20their%20potential%20use%20in%20related%20species&amp;author=Y.%20Yang&amp;author=ZB.%20Jing&amp;author=X.%20Ruan&amp;author=JM.%20Cheng&amp;journal=Genet%20Mol%20Res&amp;volume=14&amp;issue=1&amp;pages=609-618&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Souframanien J, Reddy KS. De novo assembly, characterization of immature seed Transcriptome and development of Genic-SSR markers in black gram (Vigna Mungo L.) Hepper. PLoS One. 2015;10(6):e0128748.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0128748"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26042595"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4456365"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%2C%20characterization%20of%20immature%20seed%20Transcriptome%20and%20development%20of%20Genic-SSR%20markers%20in%20black%20gram%20%28Vigna%20Mungo%20L.%29%20Hepper&amp;author=J.%20Souframanien&amp;author=KS.%20Reddy&amp;journal=PLoS%20One&amp;volume=10&amp;issue=6&amp;pages=e0128748&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Forcada CF, Marti AF, Company RS. Mapping quantitative trait loci for kernel composition in almond. BMC Genet. 2012;13:47.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2156-13-47"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Mapping%20quantitative%20trait%20loci%20for%20kernel%20composition%20in%20almond&amp;author=CF.%20Forcada&amp;author=AF.%20Marti&amp;author=RS.%20Company&amp;journal=BMC%20Genet&amp;volume=13&amp;pages=47&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Tavassolian I, Rabiei G, Gregory D, Mnejja M, Wirthensohn MG, Hunt PW, Gibson JP, Ford CM, Sedgley M, Wu SB. Construction of an almond linkage map in an Australian population nonpareil×Lauranne. BMC Genet. 2010;11:551.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2164-11-551"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Construction%20of%20an%20almond%20linkage%20map%20in%20an%20Australian%20population%20nonpareil%C3%97Lauranne&amp;author=I.%20Tavassolian&amp;author=G.%20Rabiei&amp;author=D.%20Gregory&amp;author=M.%20Mnejja&amp;author=MG.%20Wirthensohn&amp;author=PW.%20Hunt&amp;author=JP.%20Gibson&amp;author=CM.%20Ford&amp;author=M.%20Sedgley&amp;author=SB.%20Wu&amp;journal=BMC%20Genet&amp;volume=11&amp;pages=551&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Forcada CFI, Oraguzie N, Reyes-Chin-Wo S, Espiau MT, Company RSI, Marti AFI. Identification of genetic loci associated with quality traits in almond via association mapping. PLoS One. 2015;10(6):e0127656.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0127656"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Identification%20of%20genetic%20loci%20associated%20with%20quality%20traits%20in%20almond%20via%20association%20mapping&amp;author=CFI.%20Forcada&amp;author=N.%20Oraguzie&amp;author=S.%20Reyes-Chin-Wo&amp;author=MT.%20Espiau&amp;author=RSI.%20Company&amp;author=AFI.%20Marti&amp;journal=PLoS%20One&amp;volume=10&amp;issue=6&amp;pages=e0127656&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hassouna G, Elhem K, Tatiana BZ, Ahmed M, Yolanda G. Development of an SRR-based identification key for Tunisian local almonds. Sci Agric. 2012;69(2):108–13.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1590/S0103-90162012000200004"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Development%20of%20an%20SRR-based%20identification%20key%20for%20Tunisian%20local%20almonds&amp;author=G.%20Hassouna&amp;author=K.%20Elhem&amp;author=BZ.%20Tatiana&amp;author=M.%20Ahmed&amp;author=G.%20Yolanda&amp;journal=Sci%20Agric&amp;volume=69&amp;issue=2&amp;pages=108-113&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Mnejja M, Garcia-Mas J, Howad W, Arus P. Development and transportability across Prunus species of 42 polymorphic almond microsatellites. Mol Ecol Notes. 2005;5(3):531–5.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1471-8286.2005.00977.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Development%20and%20transportability%20across%20Prunus%20species%20of%2042%20polymorphic%20almond%20microsatellites&amp;author=M.%20Mnejja&amp;author=J.%20Garcia-Mas&amp;author=W.%20Howad&amp;author=P.%20Arus&amp;journal=Mol%20Ecol%20Notes&amp;volume=5&amp;issue=3&amp;pages=531-535&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Xie H, Sui Y, Chang FQ, Xu Y, Ma RC. SSR allelic variation in almond (Prunus Dulcis mill.). Theor Appl Genet. 2006;112(2):366–72.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00122-005-0138-5"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16307227"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=SSR%20allelic%20variation%20in%20almond%20%28Prunus%20Dulcis%20mill.%29&amp;author=H.%20Xie&amp;author=Y.%20Sui&amp;author=FQ.%20Chang&amp;author=Y.%20Xu&amp;author=RC.%20Ma&amp;journal=Theor%20Appl%20Genet&amp;volume=112&amp;issue=2&amp;pages=366-372&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">An M, Deng M, Zheng SS, Song YG. De novo transcriptome assembly and development of SSR markers of oaks Quercus austrocochinchinensis and Q. Kerrii (Fagaceae). Tree Genet Geno. 2016;12(103):1–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=De%20novo%20transcriptome%20assembly%20and%20development%20of%20SSR%20markers%20of%20oaks%20Quercus%20austrocochinchinensis%20and%20Q.%20Kerrii%20%28Fagaceae%29&amp;author=M.%20An&amp;author=M.%20Deng&amp;author=SS.%20Zheng&amp;author=YG.%20Song&amp;journal=Tree%20Genet%20Geno&amp;volume=12&amp;issue=103&amp;pages=1-9&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Long Y, Wang Y, Wu S, Wang J, Tian X, Pei X. De novo assembly of Transcriptome sequencing in Caragana Korshinskii Kom. And characterization of EST-SSR markers. PLoS One. 2015;10(1):e0115805.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0115805"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25629164"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4309406"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20of%20Transcriptome%20sequencing%20in%20Caragana%20Korshinskii%20Kom.%20And%20characterization%20of%20EST-SSR%20markers&amp;author=Y.%20Long&amp;author=Y.%20Wang&amp;author=S.%20Wu&amp;author=J.%20Wang&amp;author=X.%20Tian&amp;author=X.%20Pei&amp;journal=PLoS%20One&amp;volume=10&amp;issue=1&amp;pages=e0115805&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Chi W, Ma XF, Niu JG, Zou M. Characterizing the transcriptome and molecular markers information for roach. J Genet. 2016;95:45–51.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s12041-015-0597-7"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27019431"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Characterizing%20the%20transcriptome%20and%20molecular%20markers%20information%20for%20roach&amp;author=W.%20Chi&amp;author=XF.%20Ma&amp;author=JG.%20Niu&amp;author=M.%20Zou&amp;journal=J%20Genet&amp;volume=95&amp;pages=45-51&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhou XJ, Wang YY, Xu YN, Yan RS, Zhao P, Liu W. De novo characterization of flower bud Transcriptomes and the development of EST-SSR markers for the endangered tree Tapiscia Sinensis. Int J Mol Sci. 2015;16:12855–70.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.3390/ijms160612855"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26057749"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4490475"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=De%20novo%20characterization%20of%20flower%20bud%20Transcriptomes%20and%20the%20development%20of%20EST-SSR%20markers%20for%20the%20endangered%20tree%20Tapiscia%20Sinensis&amp;author=XJ.%20Zhou&amp;author=YY.%20Wang&amp;author=YN.%20Xu&amp;author=RS.%20Yan&amp;author=P.%20Zhao&amp;author=W.%20Liu&amp;journal=Int%20J%20Mol%20Sci&amp;volume=16&amp;pages=12855-12870&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Jiang B, Xie D, Liu W, Peng Q, He X. De novo assembly and characterization of the Transcriptome and development of SSR markers in wax gourd (Benicasa hispida). PLoS One. 2013;8(8):e71054.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0071054"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23951078"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3738631"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20and%20characterization%20of%20the%20Transcriptome%20and%20development%20of%20SSR%20markers%20in%20wax%20gourd%20%28Benicasa%20hispida%29&amp;author=B.%20Jiang&amp;author=D.%20Xie&amp;author=W.%20Liu&amp;author=Q.%20Peng&amp;author=X.%20He&amp;journal=PLoS%20One&amp;volume=8&amp;issue=8&amp;pages=e71054&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wu TQ, Luo SB, Wang R, Zhong YJ, Xu XM, Lin TE, He XM, et al. The first Illumina-based de novo transcriptome sequencing and analysis of pumpkin (Cucurbita Moschata Duch.) and SSR marker development. Mol Breeding. 2014;34:1437–47.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11032-014-0128-x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20first%20Illumina-based%20de%20novo%20transcriptome%20sequencing%20and%20analysis%20of%20pumpkin%20%28Cucurbita%20Moschata%20Duch.%29%20and%20SSR%20marker%20development&amp;author=TQ.%20Wu&amp;author=SB.%20Luo&amp;author=R.%20Wang&amp;author=YJ.%20Zhong&amp;author=XM.%20Xu&amp;author=TE.%20Lin&amp;author=XM.%20He&amp;journal=Mol%20Breeding&amp;volume=34&amp;pages=1437-1447&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Dong S, Liu Y, Niu J, Ning Y, Lin S, Zhang Z. De novo transcriptome analysis of the Siberian apricot (Prunus Sibirica L.) and search for potential SSR markers by 454 pyrosequencing. Gene. 2014;544:220–7.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.gene.2014.04.031"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24746601"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=De%20novo%20transcriptome%20analysis%20of%20the%20Siberian%20apricot%20%28Prunus%20Sibirica%20L.%29%20and%20search%20for%20potential%20SSR%20markers%20by%20454%20pyrosequencing&amp;author=S.%20Dong&amp;author=Y.%20Liu&amp;author=J.%20Niu&amp;author=Y.%20Ning&amp;author=S.%20Lin&amp;author=Z.%20Zhang&amp;journal=Gene&amp;volume=544&amp;pages=220-227&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR24" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li Y, Xu C, Lin X, Cui B, Wu R, Pang X. De novo assembly and characterization of the fruit transcriptome of Chinese jujube (Ziziphus Jujuba mill.) using 454 pyrosequencing and the development of novel tri-nucleotide SSR markers. PLoS One. 2014;9:e106438.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0106438"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25184704"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4153635"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20and%20characterization%20of%20the%20fruit%20transcriptome%20of%20Chinese%20jujube%20%28Ziziphus%20Jujuba%20mill.%29%20using%20454%20pyrosequencing%20and%20the%20development%20of%20novel%20tri-nucleotide%20SSR%20markers&amp;author=Y.%20Li&amp;author=C.%20Xu&amp;author=X.%20Lin&amp;author=B.%20Cui&amp;author=R.%20Wu&amp;author=X.%20Pang&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e106438&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR25" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Huang LL, Yang X, Sun P, Tong W, Hu SQ. The first Illumian based de novo transcriptome sequencing and analysis of safflower flowers. PLoS One. 2012;7:e38653.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/annotation/e4e8ca0c-f6e8-4b32-aae1-b5f8e0c7ebc3"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20first%20Illumian%20based%20de%20novo%20transcriptome%20sequencing%20and%20analysis%20of%20safflower%20flowers&amp;author=LL.%20Huang&amp;author=X.%20Yang&amp;author=P.%20Sun&amp;author=W.%20Tong&amp;author=SQ.%20Hu&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e38653&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR26" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ahmad Z, Mumtaz AS, Ghafoor A, Ali A, Nisar M. Marker assisted selection (MAS) for chickpea Fusarium oxysporum wilt resistant genotypes using PCR based molecular markers. Mol Biol Rep. 2014;41:6755–62.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11033-014-3561-3"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25017202"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4173118"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Marker%20assisted%20selection%20%28MAS%29%20for%20chickpea%20Fusarium%20oxysporum%20wilt%20resistant%20genotypes%20using%20PCR%20based%20molecular%20markers&amp;author=Z.%20Ahmad&amp;author=AS.%20Mumtaz&amp;author=A.%20Ghafoor&amp;author=A.%20Ali&amp;author=M.%20Nisar&amp;journal=Mol%20Biol%20Rep&amp;volume=41&amp;pages=6755-6762&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR27" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Grabherr MG, Haas BJ, Yassour M, Levin JZ, Thompson DA, Amit I, Adiconis Z, et al. Full-length transcriptome assembly from RNA-Seq data without a reference genome. Nat Biotechnol. 2011;29:644–52.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nbt.1883"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21572440"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3571712"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Full-length%20transcriptome%20assembly%20from%20RNA-Seq%20data%20without%20a%20reference%20genome&amp;author=MG.%20Grabherr&amp;author=BJ.%20Haas&amp;author=M.%20Yassour&amp;author=JZ.%20Levin&amp;author=DA.%20Thompson&amp;author=I.%20Amit&amp;author=Z.%20Adiconis&amp;journal=Nat%20Biotechnol&amp;volume=29&amp;pages=644-652&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR28" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Conesa A, Gotz S, Garcia-Gomez JM, Terol J, Talon M, Robles M. Blast2GO, a universal tool for annotation, visualization and analysis in functional genomics research. Bioinformatics. 2005;21(18):3674–6.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/bti610"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16081474"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Blast2GO%2C%20a%20universal%20tool%20for%20annotation%2C%20visualization%20and%20analysis%20in%20functional%20genomics%20research&amp;author=A.%20Conesa&amp;author=S.%20Gotz&amp;author=JM.%20Garcia-Gomez&amp;author=J.%20Terol&amp;author=M.%20Talon&amp;author=M.%20Robles&amp;journal=Bioinformatics&amp;volume=21&amp;issue=18&amp;pages=3674-3676&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR29" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ye J, Fang L, Zheng H, Zhang Y, Chen J, Zhang Z, Wang J, Li S, et al. WEGO, a web tool for plotting GO annotations. Nucleic Acids Res. 2006;34:W293–7.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gkl031"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16845012"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1538768"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=WEGO%2C%20a%20web%20tool%20for%20plotting%20GO%20annotations&amp;author=J.%20Ye&amp;author=L.%20Fang&amp;author=H.%20Zheng&amp;author=Y.%20Zhang&amp;author=J.%20Chen&amp;author=Z.%20Zhang&amp;author=J.%20Wang&amp;author=S.%20Li&amp;journal=Nucleic%20Acids%20Res&amp;volume=34&amp;pages=W293-W297&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR30" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kanehisa M, Araki M, Goto S, Hattori M, Hirakawa M, Itoh M, Katayama T, et al. KEGG for linking genomes to life and the environment. Nucleic Acids Res. 2008;36:D480–4.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gkm882"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18077471"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=KEGG%20for%20linking%20genomes%20to%20life%20and%20the%20environment&amp;author=M.%20Kanehisa&amp;author=M.%20Araki&amp;author=S.%20Goto&amp;author=M.%20Hattori&amp;author=M.%20Hirakawa&amp;author=M.%20Itoh&amp;author=T.%20Katayama&amp;journal=Nucleic%20Acids%20Res&amp;volume=36&amp;pages=D480-D484&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li></ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#TheAuthors" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="TheAuthors">Copyright</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="TheAuthors"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">©&nbsp;The Author(s)&nbsp;2017</div></div></section></div>
                        


                    </article>
                </main>

                <div class="c-content-layout__side u-text-sm">
                    <aside>
                        <div class="p-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="//hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-017-0049-x" data-track="click" data-track-category="PDF Download"
            data-track-action="Click download article PDF"
            data-track-label="10.1186/s41065-017-0049-x">Download PDF</a>
        </p>
    
                            
                            
    <button type="button"
       class="c-button c-button--secondary c-button--full-width c-button--expandable"
       data-toggle="collapse"
       data-target="export-citations"
       data-test="export-citations-button"
       data-sticky-update="true"
       aria-controls="export-citations"
       aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/s41065-017-0049-x"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0049-x?format=refman&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/s41065-017-0049-x"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0049-x?format=refman&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/s41065-017-0049-x"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0049-x?format=refman&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/s41065-017-0049-x"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0049-x?format=endnote&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/s41065-017-0049-x"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0049-x?format=endnote&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/s41065-017-0049-x"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0049-x?format=endnote&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/s41065-017-0049-x"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0049-x?format=bibtex&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/s41065-017-0049-x"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0049-x?format=bibtex&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/s41065-017-0049-x"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0049-x?format=bibtex&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
    </div>

                        </div>
                        
    
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
                                <div id="article-access-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/access-count/10.1186/s41065-017-0049-x #accessCount-container"></div>
                                <div id="citation-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/citation-count/10.1186/s41065-017-0049-x #count-container"></div>
                                <div id="altmetric-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/altmetric/10.1186/s41065-017-0049-x #altmetric-container"></div>
                            </ul>
                        </div>
                        
    <div id="sociaMediaShare" class="p-separator u-js-shown">
        <h3 class="p-section-title">Share this article</h3>
        <ul id="shareButtons" class="c-social-links">
            <li class="c-social-links__item">
                <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-017-0049-x">
                    <span class="u-visually-hidden">Share on Twitter</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-twitter"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-017-0049-x">
                    <span class="u-visually-hidden">Share on Facebook</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-facebook"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click"
                        data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-017-0049-x">
                    <span class="u-visually-hidden">Share on LinkedIn</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-linkedin"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-017-0049-x">
                    <span class="u-visually-hidden">Share on Weibo</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-weibo"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnGooglePlus" class="js-btnShareOnGplus" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Google+" data-track-label="10.1186/s41065-017-0049-x">
                    <span class="u-visually-hidden">Share on Google Plus</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-gplus"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-017-0049-x">
                    <span class="u-visually-hidden">Share on Reddit</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-reddit"></use>
                    </svg>
                </button>
            </li>
        </ul>
    </div>
 
                        
    
        <div class="p-separator u-js-shown">
            <h3 class="p-section-title">See updates</h3>
            <button class="c-crossmark-button" data-target="crossmark" data-test="crossmark" data-track="click" data-track-category="Article Page" data-track-action="Click CrossMark" data-track-label="10.1186/s41065-017-0049-x">
                <img id="crossmark-icon" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" alt="Crossmark logo" />
            </button>
        </div>
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
                                <li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/s41065-017-0049-x" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=738&doi=10.1186%2Fs41065-017-0049-x&volume=155&issue=1&title=Characterizing+the+transcriptome+and+microsatellite+markers+for+almond+%28+Amygdalus+communis+L.%29+using+the+Illumina+sequencing+platform&author_name=Linsen+Zhang&start_page=1&end_page=9">Order reprint</a>

                                </li>
                            </ul>
                        </div>
                    </aside>
                    
    <div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">Hereditas</span>
                        </h4>
                        <p class="c-journal-footer__issn">ISSN: 1601-5223</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
                                
                                    <li class="c-journal-footer__contact-item">Submission enquiries: <a href="https://www.editorialmanager.com/here/" target="_blank">Access here and click Contact Us</a></li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a></li>
                                
                            </ul>
                        </div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class='tracker' style='display:none' src='/track/article/10.1186/s41065-017-0049-x' alt=""/>

         
    <footer>
        <div class="ctx-publisher-footer">
    <div class="c-navbar">
        <div class="u-container">
            <div class="c-navbar__logo u-margin-right-xl">
                <a href="https://www.biomedcentral.com">
                    <img src=/static/images/bmc/logos/logo-bmc-aj-594e03e20d.svg alt="BMC logo" data-test="navbar-logo-footer">

                </a>
            </div>
            <div class="c-navbar__content">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


            </div>
        </div>
    </div>
    <div class="c-publisher-footer" data-test="publisher-footer">
        <div class="u-container">
            
            <div class="c-flex c-flex--wrap c-flex--space-between" role="navigation" data-test="publisher-footer-menu">
                <div class="c-flex" aria-label="Publisher footer links">
                    
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/login">Receive BMC newsletters</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/account">Manage article alerts</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/policies">Policies</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/accessibility">Accessibility</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/press-centre">Press center</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/contact-us">Contact us</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/jobs">Careers</a>
                                    </li>
                                
                            </ul>
                        
                    
                </div>
                <div class="u-margin-bottom-lg">
                    <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                    <ul class="c-flex u-reset-list" data-test="footer-social-links">
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://twitter.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Twitter link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Twitter page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-twitter-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://www.facebook.com/BioMedCentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Facebook link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Facebook page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-facebook-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://plus.google.com/106226337984152901734/"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Google Plus link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Google Plus page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-gplus-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="http://www.weibo.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Weibo link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Weibo page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-weibo-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </div>
            <p class="u-reset-margin">
                By using this website, you agree to our
                <a class="u-text-inherit" href="//www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
                <a class="u-text-inherit" href="//www.biomedcentral.com/privacy-statement">Privacy
                    statement</a> and
                <a class="u-text-inherit" href="//www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
                
                    <a class="optanon-toggle-display u-text-inherit" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
                
            </p>
        </div>
    </div>
</div>


        <div class="c-sn-footer c-sn-footer--neutral">
    <div class="u-container">
        <img src=/static/images/bmc/logos/logo-springernature-8bd4e8d82f.svg class="c-sn-footer__logo" alt="Springer Nature Logo" itemprop="logo" >
        <p data-test="copyright">&#169; 2018 BioMed Central Ltd unless otherwise stated. Part of
            <a href="http://www.springernature.com"
               data-track="click"
               data-track-action="Click Springer Nature link"
               data-track-category="Corporate footer"
               itemscope itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>

    </div>

    
    <script type="text/javascript" src="//cdn.pbgrd.com/bmc.js" async="async"></script>
    <noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>
    <!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker -->

<script id="springerlink-event-tracker" src="https://event-tracker.springernature.com/dist/eventTracker.js"></script>
<script type="text/javascript">
    (function(){
        var dataLayer = window.dataLayer[0];
        var doi = dataLayer.content.article.doi;
        var imprint = dataLayer.content.contentInfo.imprint;
        var bpids = dataLayer.session.authentication.authenticationID;
        var publisher;
        var pageType = dataLayer.page.category.pageType;
        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }
        var eventObject = {'content_type' : pageType,'doi' : doi};

        if (imprint === 'BioMed Central') { publisher = 'BMC' }
        else if (imprint === 'SpringerOpen') { publisher = 'SpringerOpen' }
        else { throw new Error('Event Tracker Error: The publisher has an unexpected value') }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }
        new EventTracker({'platform':publisher}).sendEvent('display', eventObject);
    })();
</script>


    </body>
</html>

