<!DOCTYPE html>
<html lang="en" class="no-js">
    <head>
        <title>A comparative genomics approach revealed evolutionary dynamics of microsatellite imperfection and conservation in genus Gossypium | Hereditas | Full Text</title>

        

    <meta name="citation_abstract" content="Ongoing molecular processes in a cell could target microsatellites, a kind of repetitive DNA, owing to length variations and motif imperfection. Mutational mechanisms underlying such kind of genetic variations have been extensively investigated in diverse organisms. However, obscure impact of ploidization, an evolutionary process of genome content duplication prevails mostly in plants, on non-coding DNA is poorly understood. Genome sequences of diversely originated plant species were examined for genome-wide motif imperfection pattern, and various analytical tools were employed to canvass characteristic relationships among repeat density, imperfection and length of microsatellites. Moreover, comparative genomics approach aided in exploration of microsatellites conservation footprints in Gossypium evolution. Based on our results, motif imperfection in repeat length was found intricately related to genomic abundance of imperfect microsatellites among 13 genomes. Microsatellite decay estimation depicted slower decay of long motif repeats which led to predominant abundance of 5-nt repeat motif in Gossypium species. Short motif repeats exhibited rapid decay through the evolution of Gossypium lineage ensuing drastic decrease of 2-nt repeats, of which, “AT” motif type dilapidated in cultivated tetraploids of cotton. The outcome could be a directive to explore comparative evolutionary footprints of simple non-coding genetic elements i.e., repeat elements, through the evolution of genus-specific characteristics in cotton genomes."/>

    <meta name="description" content="Ongoing molecular processes in a cell could target microsatellites, a kind of repetitive DNA, owing to length variations and motif imperfection. Mutational mechanisms underlying such kind of genetic variations have been extensively investigated in diverse organisms. However, obscure impact of ploidization, an evolutionary process of genome content duplication prevails mostly in plants, on non-coding DNA is poorly understood. Genome sequences of diversely originated plant species were examined for genome-wide motif imperfection pattern, and various analytical tools were employed to canvass characteristic relationships among repeat density, imperfection and length of microsatellites. Moreover, comparative genomics approach aided in exploration of microsatellites conservation footprints in Gossypium evolution. Based on our results, motif imperfection in repeat length was found intricately related to genomic abundance of imperfect microsatellites among 13 genomes. Microsatellite decay estimation depicted slower decay of long motif repeats which led to predominant abundance of 5-nt repeat motif in Gossypium species. Short motif repeats exhibited rapid decay through the evolution of Gossypium lineage ensuing drastic decrease of 2-nt repeats, of which, “AT” motif type dilapidated in cultivated tetraploids of cotton. The outcome could be a directive to explore comparative evolutionary footprints of simple non-coding genetic elements i.e., repeat elements, through the evolution of genus-specific characteristics in cotton genomes."/>

    <meta name="journal_id" content="41065"/>

    <meta name="dc.title" content="A comparative genomics approach revealed evolutionary dynamics of microsatellite imperfection and conservation in genus Gossypium"/>

    <meta name="dc.source" content="Hereditas 2017 154:1"/>

    <meta name="dc.format" content="text/html"/>

    <meta name="dc.publisher" content="BioMed Central"/>

    <meta name="dc.date" content="2017-05-18"/>

    <meta name="dc.type" content="OriginalPaper"/>

    <meta name="dc.language" content="En"/>

    <meta name="dc.copyright" content="2017 The Author(s)"/>

    <meta name="dc.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="dc.description" content="Ongoing molecular processes in a cell could target microsatellites, a kind of repetitive DNA, owing to length variations and motif imperfection. Mutational mechanisms underlying such kind of genetic variations have been extensively investigated in diverse organisms. However, obscure impact of ploidization, an evolutionary process of genome content duplication prevails mostly in plants, on non-coding DNA is poorly understood. Genome sequences of diversely originated plant species were examined for genome-wide motif imperfection pattern, and various analytical tools were employed to canvass characteristic relationships among repeat density, imperfection and length of microsatellites. Moreover, comparative genomics approach aided in exploration of microsatellites conservation footprints in Gossypium evolution. Based on our results, motif imperfection in repeat length was found intricately related to genomic abundance of imperfect microsatellites among 13 genomes. Microsatellite decay estimation depicted slower decay of long motif repeats which led to predominant abundance of 5-nt repeat motif in Gossypium species. Short motif repeats exhibited rapid decay through the evolution of Gossypium lineage ensuing drastic decrease of 2-nt repeats, of which, &#8220;AT&#8221; motif type dilapidated in cultivated tetraploids of cotton. The outcome could be a directive to explore comparative evolutionary footprints of simple non-coding genetic elements i.e., repeat elements, through the evolution of genus-specific characteristics in cotton genomes."/>

    <meta name="prism.issn" content="1601-5223"/>

    <meta name="prism.publicationName" content="Hereditas"/>

    <meta name="prism.publicationDate" content="2017-05-18"/>

    <meta name="prism.volume" content="154"/>

    <meta name="prism.number" content="1"/>

    <meta name="prism.section" content="OriginalPaper"/>

    <meta name="prism.startingPage" content="12"/>

    <meta name="prism.copyright" content="2017 The Author(s)"/>

    <meta name="prism.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="prism.url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0034-4"/>

    <meta name="prism.doi" content="doi:10.1186/s41065-017-0034-4"/>

    <meta name="citation_pdf_url" content="https://hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-017-0034-4"/>

    <meta name="citation_fulltext_html_url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0034-4"/>

    <meta name="citation_journal_title" content="Hereditas"/>

    <meta name="citation_publisher" content="BioMed Central"/>

    <meta name="citation_issn" content="1601-5223"/>

    <meta name="citation_title" content="A comparative genomics approach revealed evolutionary dynamics of microsatellite imperfection and conservation in genus Gossypium"/>

    <meta name="citation_volume" content="154"/>

    <meta name="citation_issue" content="1"/>

    <meta name="citation_publication_date" content="2017/12"/>

    <meta name="citation_online_date" content="2017/05/18"/>

    <meta name="citation_firstpage" content="12"/>

    <meta name="citation_article_type" content="Research"/>

    <meta name="citation_fulltext_world_readable" content=""/>

    <meta name="dc.identifier" content="doi:10.1186/s41065-017-0034-4"/>

    <meta name="DOI" content="10.1186/s41065-017-0034-4"/>

    <meta name="citation_doi" content="10.1186/s41065-017-0034-4"/>

    <meta name="dc.creator" content="Muhammad Mahmood Ahmed"/>

    <meta name="dc.creator" content="Chao Shen"/>

    <meta name="dc.creator" content="Anam Qadir Khan"/>

    <meta name="dc.creator" content="Muhammad Atif Wahid"/>

    <meta name="dc.creator" content="Muhammad Shaban"/>

    <meta name="dc.creator" content="Zhongxu Lin"/>

    <meta name="dc.subject" content="Animal Genetics and Genomics"/>

    <meta name="dc.subject" content="Evolutionary Biology"/>

    <meta name="citation_reference" content="Kruglyak S, Durrett RT, Schug MD, Aquadro CF. Equilibrium distributions of microsatellite repeat length resulting from a balance between slippage events and point mutations. Proc Natl Acad Sci U S A. 1998;95:10774&#8211;8."/>

    <meta name="citation_reference" content="citation_title=Microsatellites and other simple sequences: genomic context and mutational mechanisms; citation_inbook_title=Microsatellites: evolution and applications; citation_publication_date=1999; citation_pages=1-9; citation_id=CR2; citation_author=JM Hancock; citation_publisher=Oxford University Press"/>

    <meta name="citation_reference" content="citation_journal_title=Annu Rev Biochem; citation_title=Fidelity mechanisms in DNA replication; citation_author=H Echols, MF Goodman; citation_volume=60; citation_publication_date=1991; citation_pages=477-511; citation_doi=10.1146/annurev.bi.60.070191.002401; citation_id=CR3"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Educ; citation_title=DNA replication and causes of mutation; citation_author=LA Pray; citation_volume=1; citation_publication_date=2008; citation_pages=214; citation_id=CR4"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=Slippage synthesis of simple sequence DNA; citation_author=C Schl&#246;tterer, D Tautz; citation_volume=20; citation_publication_date=1992; citation_pages=211-215; citation_doi=10.1093/nar/20.2.211; citation_id=CR5"/>

    <meta name="citation_reference" content="Kimura M, Ohta T. Stepwise mutation model and distribution of allelic frequencies in a finite population. Proc Natl Acad Sci U S A. 1978;75:2868&#8211;72."/>

    <meta name="citation_reference" content="Di Rienzo A, Peterson AC, Garza JC, Valdes AM, Slatkin M, Freimer NB. Mutational processes of simple-sequence repeat loci in human populations. Proc Natl Acad Sci U S A. 1994;91:3166&#8211;70."/>

    <meta name="citation_reference" content="Nadir E, Margalit H, Gallily T, Ben-Sasson SA. Microsatellite spreading in the human genome: evolutionary mechanisms and structural implications. Proc Natl Acad Sci U S A. 1996;93:6470&#8211;5."/>

    <meta name="citation_reference" content="citation_journal_title=BMC Evol Biol; citation_title=Generation of microsatellite repeat families by RTE retrotransposons in lepidopteran genomes; citation_author=WT Tay, GT Behere, P Batterham, DG Heckel; citation_volume=10; citation_publication_date=2010; citation_pages=144; citation_doi=10.1186/1471-2148-10-144; citation_id=CR9"/>

    <meta name="citation_reference" content="citation_journal_title=Electrophoresis; citation_title=Imperfect units of an extended microsatellite structure involving single nucleotide changes; citation_author=W Wang, AH Bittles; citation_volume=22; citation_publication_date=2001; citation_pages=1095-1097; citation_doi=10.1002/1522-2683()22:6&lt;1095::AID-ELPS1095&gt;3.0.CO;2-A; citation_id=CR10"/>

    <meta name="citation_reference" content="citation_journal_title=Chromosoma; citation_title=Evolutionary dynamics of microsatellite DNA; citation_author=C Schlotterer; citation_volume=109; citation_publication_date=2000; citation_pages=365-371; citation_doi=10.1007/s004120000089; citation_id=CR11"/>

    <meta name="citation_reference" content="citation_journal_title=J Mol Evol; citation_title=Polymorphism, monomorphism, and sequences in conserved microsatellites in primate species; citation_author=A Blanquer-Maumont, B Crouau-Roy; citation_volume=41; citation_publication_date=1995; citation_pages=492-497; citation_doi=10.1007/BF00160321; citation_id=CR12"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Phylogenet Evol; citation_title=Ancient conservation of Trinucleotide microsatellite loci in Polistine wasps; citation_author=VO Ezenwa, JM Peters, Y Zhu, E Ar&#233;valo, MD Hastings, P Sepp&#228;; citation_volume=10; citation_publication_date=1998; citation_pages=168-177; citation_doi=10.1006/mpev.1998.0528; citation_id=CR13"/>

    <meta name="citation_reference" content="citation_journal_title=Genome; citation_title=Microsatellite landscape evolutionary dynamics across 450 million years of vertebrate genome evolution; citation_author=RH Adams, H Blackmon, J Reyes-Velasco, DR Schield, DC Card, AL Andrew; citation_volume=59; citation_publication_date=2016; citation_pages=295-310; citation_doi=10.1139/gen-2015-0124; citation_id=CR14"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Genet; citation_title=Comparative in silico analysis of SSRs in coding regions of high confidence predicted genes in Norway spruce (Picea abies) and loblolly pine (Pinus taeda); citation_author=SS Ranade, YC Lin, Y Van de Peer, MR Garcia-Gil; citation_volume=16; citation_publication_date=2015; citation_pages=149; citation_doi=10.1186/s12863-015-0304-y; citation_id=CR15"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Rep; citation_title=Polymorphism of CAG repeats in androgen receptor of carnivores; citation_author=Q Wang, X Zhang, X Wang, B Zeng, X Jia, R Hou; citation_volume=39; citation_publication_date=2012; citation_pages=2297-2303; citation_doi=10.1007/s11033-011-0979-8; citation_id=CR16"/>

    <meta name="citation_reference" content="CottonGen. 
                    https://www.cottongen.org
                    
                  /. 15 June 2016."/>

    <meta name="citation_reference" content="NCBI Genome Portal. 
                    https://www.ncbi.nlm.nih.gov/genome/?term
                    
                  . 15 June 2016."/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=SciRoKo: a new tool for whole genome microsatellite search and investigation; citation_author=R Kofler, C Schlotterer, T Lelley; citation_volume=23; citation_publication_date=2007; citation_pages=1683-1685; citation_doi=10.1093/bioinformatics/btm157; citation_id=CR19"/>

    <meta name="citation_reference" content="citation_journal_title=DNA Res; citation_title=Motif mismatches in microsatellites: insights from genome-wide investigation among 20 insect species; citation_author=SK Behura, DW Severson; citation_volume=22; citation_publication_date=2015; citation_pages=29-38; citation_doi=10.1093/dnares/dsu036; citation_id=CR20"/>

    <meta name="citation_reference" content="citation_journal_title=Aust NZ J Stat; citation_title=Generalized discriminant analysis based on distances; citation_author=MJ Anderson, J Robinson; citation_volume=45; citation_publication_date=2003; citation_pages=301-318; citation_doi=10.1111/1467-842X.00285; citation_id=CR21"/>

    <meta name="citation_reference" content="Canonical Analysis of Principal Coordinates (CAP). 
                    http://www.esapubs.org/archive/ecol/E084/011/suppl-1.htm
                    
                  . 10 May 2016."/>

    <meta name="citation_reference" content="The R project for statistical computing. 
                    https://www.r-project.org
                    
                  /. 10 May 2016."/>

    <meta name="citation_reference" content="CCP: Significance tests for Canonical Correlation Analysis (CCA). 
                    https://cran.r-project.org/web/packages/CCP/index.html
                    
                  . 25 May 2016."/>

    <meta name="citation_reference" content="citation_journal_title=Aust Ecol; citation_title=A new method for non-parametric multivariate analysis of variance; citation_author=MJ Anderson; citation_volume=26; citation_publication_date=2001; citation_pages=32-46; citation_id=CR25"/>

    <meta name="citation_reference" content="vegan: Community Ecology package. 
                    https://cran.r-project.org/web/packages/vegan/index.html
                    
                  . 25 May 2016."/>

    <meta name="citation_reference" content="citation_journal_title=J Mol Biol; citation_title=Basic local alignment search tool; citation_author=SF Altschul, W Gish, W Miller, EW Myers, DJ Lipman; citation_volume=215; citation_publication_date=1990; citation_pages=403-410; citation_doi=10.1016/S0022-2836(05)80360-2; citation_id=CR27"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Biotechnol; citation_title=Genome sequence of cultivated upland cotton (Gossypium hirsutum TM-1) provides insights into genome evolution; citation_author=FG Li, GY Fan, CR Lu, GH Xiao, CS Zou, RJ Kohel; citation_volume=33; citation_publication_date=2015; citation_pages=524-U242; citation_doi=10.1038/nbt.3208; citation_id=CR28"/>

    <meta name="citation_reference" content="Circos: Circular Visualization. 
                    http://circos.ca
                    
                  /. 20 July 2016."/>

    <meta name="citation_reference" content="citation_title=Taxonomy and evolution of the cotton genus; citation_inbook_title=Cotton, Agronomy; citation_publication_date=2015; citation_id=CR30; citation_author=JF Wendel; citation_author=CE Grover; citation_publisher=Monograph 24, ASA-CSSA-SSSA"/>

    <meta name="citation_reference" content="citation_journal_title=Infect Genet Evol; citation_title=Genome-wide scan for analysis of simple and imperfect microsatellites in diverse carlaviruses; citation_author=CM Alam, AK Singh, C Sharfuddin, S Ali; citation_volume=21; citation_publication_date=2014; citation_pages=287-294; citation_doi=10.1016/j.meegid.2013.11.018; citation_id=CR31"/>

    <meta name="citation_reference" content="citation_journal_title=J Hered; citation_title=Investigating the allelic evolution of an imperfect microsatellite locus in the Hawaiian mushroom Rhodocollybia laulaha; citation_author=MR Keirle, PG Avis, KA Feldheim, DE Hemmes, GM Mueller; citation_volume=102; citation_publication_date=2011; citation_pages=727-734; citation_doi=10.1093/jhered/esr099; citation_id=CR32"/>

    <meta name="citation_reference" content="citation_journal_title=FEBS Lett; citation_title=Divergent microsatellite evolution in the human and chimpanzee lineages; citation_author=Z Gaspari, C Ortutay, G Toth; citation_volume=581; citation_publication_date=2007; citation_pages=2523-2526; citation_doi=10.1016/j.febslet.2007.04.073; citation_id=CR33"/>

    <meta name="citation_reference" content="citation_journal_title=Genome Biol Evol; citation_title=Patterns of evolutionary conservation of microsatellites (SSRs) suggest a faster rate of genome evolution in hymenoptera than in Diptera; citation_author=E Stolle, JH Kidner, RF Moritz; citation_volume=5; citation_publication_date=2013; citation_pages=151-162; citation_doi=10.1093/gbe/evs133; citation_id=CR34"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Genet; citation_title=A direct characterization of human mutation based on microsatellites; citation_author=JX Sun, A Helgason, G Masson, SS Ebenesersdottir, H Li, S Mallick; citation_volume=44; citation_publication_date=2012; citation_pages=1161-1165; citation_doi=10.1038/ng.2398; citation_id=CR35"/>

    <meta name="citation_reference" content="Kapil A, Rai PK, Shanker A. ChloroSSRdb: a repository of perfect and imperfect chloroplastic simple sequence repeats (cpSSRs) of green plants. Database. 2014;2014."/>

    <meta name="citation_reference" content="citation_journal_title=Genet Mol Res; citation_title=Next-generation sequencing identification and characterization of microsatellite markers in aconitum austrokoreense Koidz., an endemic and endangered medicinal plant of Korea; citation_author=YE Yun, JN Yu, GH Nam, SA Ryu, S Kim, K Oh; citation_volume=14; citation_publication_date=2015; citation_pages=4812-4817; citation_doi=10.4238/2015.May.11.13; citation_id=CR37"/>

    <meta name="citation_reference" content="citation_journal_title=Int J Mol Sci; citation_title=Development of seven microsatellite markers using next generation sequencing for the conservation on the Korean population of Dorcus Hopei (E. Saunders, 1854) (Coleoptera, Lucanidae); citation_author=TH Kang, SH Han, SJ Park; citation_volume=16; citation_publication_date=2015; citation_pages=21330-21341; citation_doi=10.3390/ijms160921330; citation_id=CR38"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=Genome-wide analysis of microsatellite markers based on sequenced database in Chinese spring wheat (Triticum aestivum L.); citation_author=B Han, C Wang, Z Tang, Y Ren, Y Li, D Zhang; citation_volume=10; citation_publication_date=2015; citation_pages=e0141540; citation_doi=10.1371/journal.pone.0141540; citation_id=CR39"/>

    <meta name="citation_reference" content="citation_journal_title=Aust J Crop Sci; citation_title=Selection of core SSR markers for fingerprinting upland cotton cultivars and hybrids; citation_author=M Ahmed, H Guo, C Huang, X Zhang, Z Lin; citation_volume=7; citation_publication_date=2013; citation_pages=1912-1920; citation_id=CR40"/>

    <meta name="citation_reference" content="citation_journal_title=DNA Res; citation_title=Structure, evolution, and comparative genomics of tetraploid cotton based on a high-density genetic linkage map; citation_author=X Li, X Jin, H Wang, X Zhang, Z Lin; citation_volume=23; citation_publication_date=2016; citation_pages=283-293; citation_doi=10.1093/dnares/dsw016; citation_id=CR41"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Rep; citation_title=In-silico mining, type and frequency analysis of genic microsatellites of finger millet (Eleusine coracana (L.) Gaertn.): a comparative genomic analysis of NBS-LRR regions of finger millet with rice; citation_author=B Kalyana Babu, D Pandey, PK Agrawal, S Sood, A Kumar; citation_volume=41; citation_publication_date=2014; citation_pages=3081-3090; citation_doi=10.1007/s11033-014-3168-8; citation_id=CR42"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=Evolutionary dynamics of microsatellite distribution in plants: insight from the comparison of sequenced brassica, Arabidopsis and other angiosperm species; citation_author=J Shi, S Huang, D Fu, J Yu, X Wang, W Hua; citation_volume=8; citation_publication_date=2013; citation_pages=e59988; citation_doi=10.1371/journal.pone.0059988; citation_id=CR43"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=Development and characterization of genomic and expressed SSRs in citrus by genome-wide analysis; citation_author=SR Liu, WY Li, D Long, CG Hu, JZ Zhang; citation_volume=8; citation_publication_date=2013; citation_pages=e75149; citation_doi=10.1371/journal.pone.0075149; citation_id=CR44"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=Genome wide characterization of short tandem repeat markers in sweet orange (Citrus sinensis); citation_author=MK Biswas, Q Xu, C Mayer, X Deng; citation_volume=9; citation_publication_date=2014; citation_pages=e104182; citation_doi=10.1371/journal.pone.0104182; citation_id=CR45"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=Rapid divergence of microsatellite abundance among species of drosophila; citation_author=CL Ross, KA Dyer, T Erez, SJ Miller, J Jaenike, TA Markow; citation_volume=20; citation_publication_date=2003; citation_pages=1143-1157; citation_doi=10.1093/molbev/msg137; citation_id=CR46"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Genet; citation_title=The direction of microsatellite mutations is dependent upon allele length; citation_author=X Xu, M Peng, Z Fang; citation_volume=24; citation_publication_date=2000; citation_pages=396-399; citation_doi=10.1038/74238; citation_id=CR47"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=Size-dependent mutability and microsatellite constraints; citation_author=D Falush, Y Iwasa; citation_volume=16; citation_publication_date=1999; citation_pages=960-966; citation_doi=10.1093/oxfordjournals.molbev.a026185; citation_id=CR48"/>

    <meta name="citation_reference" content="citation_journal_title=Hum Mol Genet; citation_title=Sequence interruptions confer differential stability at microsatellite alleles in mismatch repair-deficient cells; citation_author=AL Bacon, SM Farrington, MG Dunlop; citation_volume=9; citation_publication_date=2000; citation_pages=2707-2713; citation_doi=10.1093/hmg/9.18.2707; citation_id=CR49"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=The relationship between microsatellite slippage mutation rate and the number of repeat units; citation_author=Y Lai, F Sun; citation_volume=20; citation_publication_date=2003; citation_pages=2123-2131; citation_doi=10.1093/molbev/msg228; citation_id=CR50"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Cell Biol; citation_title=Stabilizing effects of interruptions on Trinucleotide repeat expansions in Saccharomyces cerevisiae
                           ; citation_author=ML Rolfsmeier, RS Lahue; citation_volume=20; citation_publication_date=2000; citation_pages=173-180; citation_doi=10.1128/MCB.20.1.173-180.2000; citation_id=CR51"/>

    <meta name="citation_reference" content="citation_journal_title=Plant J; citation_title=Intimate association of microsatellite repeats with retrotransposons and other dispersed repetitive elements in barley; citation_author=L Ramsay, M Macaulay, L Cardle, M Morgante, S degli Ivanissevich, E Maestri; citation_volume=17; citation_publication_date=1999; citation_pages=415-425; citation_doi=10.1046/j.1365-313X.1999.00392.x; citation_id=CR52"/>

    <meta name="citation_reference" content="citation_journal_title=Mob Genet Elements; citation_title=Non-LTR retrotransposons and microsatellites: partners in genomic variation; citation_author=FC Grandi, W An; citation_volume=3; citation_publication_date=2013; citation_pages=e25674; citation_doi=10.4161/mge.25674; citation_id=CR53"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Genomics; citation_title=Development of chromosome-specific markers with high polymorphism for allotetraploid cotton based on genome-wide characterization of simple sequence repeats in diploid cottons (Gossypium arboreum L. and Gossypium raimondii Ulbrich; citation_author=C Lu, C Zou, Y Zhang, D Yu, H Cheng, P Jiang; citation_volume=16; citation_publication_date=2015; citation_pages=55; citation_doi=10.1186/s12864-015-1265-2; citation_id=CR54"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformation; citation_title=Distribution and characterization of simple sequence repeats in Gossypium raimondii genome; citation_author=C Zou, C Lu, Y Zhang, G Song; citation_volume=8; citation_publication_date=2012; citation_pages=801-806; citation_doi=10.6026/97320630008801; citation_id=CR55"/>

    <meta name="citation_reference" content="citation_journal_title=Sci Rep; citation_title=Genome-wide mining, characterization, and development of microsatellite markers in Gossypium species; citation_author=Q Wang, L Fang, J Chen, Y Hu, Z Si, S Wang; citation_volume=5; citation_publication_date=2015; citation_pages=10638; citation_doi=10.1038/srep10638; citation_id=CR56"/>

    <meta name="citation_reference" content="citation_journal_title=Am J Bot; citation_title=Rapid diversification of the cotton genus (Gossypium: Malvaceae) revealed by analysis of sixteen nuclear and chloroplast genes; citation_author=RC Cronn, RL Small, T Haselkorn, JF Wendel; citation_volume=89; citation_publication_date=2002; citation_pages=707-725; citation_doi=10.3732/ajb.89.4.707; citation_id=CR57"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Genomics; citation_title=Duplicate gene evolution, homoeologous recombination, and transcriptome characterization in allopolyploid cotton; citation_author=LE Flagel, JF Wendel, JA Udall; citation_volume=13; citation_publication_date=2012; citation_pages=1-13; citation_doi=10.1186/1471-2164-13-302; citation_id=CR58"/>

    <meta name="citation_author" content="Muhammad Mahmood Ahmed"/>

    <meta name="citation_author_institution" content="National Key Laboratory of Crop Genetic Improvement, College of Plant Science &amp; Technology, Huazhong Agricultural University, Wuhan, China"/>

    <meta name="citation_author" content="Chao Shen"/>

    <meta name="citation_author_institution" content="National Key Laboratory of Crop Genetic Improvement, College of Plant Science &amp; Technology, Huazhong Agricultural University, Wuhan, China"/>

    <meta name="citation_author" content="Anam Qadir Khan"/>

    <meta name="citation_author_institution" content="National Key Laboratory of Crop Genetic Improvement, College of Plant Science &amp; Technology, Huazhong Agricultural University, Wuhan, China"/>

    <meta name="citation_author" content="Muhammad Atif Wahid"/>

    <meta name="citation_author_institution" content="National Key Laboratory of Crop Genetic Improvement, College of Plant Science &amp; Technology, Huazhong Agricultural University, Wuhan, China"/>

    <meta name="citation_author" content="Muhammad Shaban"/>

    <meta name="citation_author_institution" content="National Key Laboratory of Crop Genetic Improvement, College of Plant Science &amp; Technology, Huazhong Agricultural University, Wuhan, China"/>

    <meta name="citation_author" content="Zhongxu Lin"/>

    <meta name="citation_author_institution" content="National Key Laboratory of Crop Genetic Improvement, College of Plant Science &amp; Technology, Huazhong Agricultural University, Wuhan, China"/>



        <meta charset="UTF-8"/>
        <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
        <meta name="viewport" content="width=device-width, initial-scale=1"/>

        <!-- For IE 9 and below -->
<!--[if IE]><link rel="shortcut icon" href=/static/images/favicons//favicon.ico><![endif]-->
<!-- Touch Icons - iOS and Android 2.1+ -->
<link rel="apple-touch-icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>
<!-- Firefox, Chrome, Safari, IE 11+ and Opera -->
<link rel="icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>


        <script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script>
        <script>(function(d){if (sessionStorage.fontsLoaded){d.style.setProperty('--font-family-sans', 'Europa');d.style.setProperty('--font-family-serif', 'Noto Serif')}})(document.documentElement);</script>

        <link rel="stylesheet" media="screen" href=/static/styles/basic/basic-a31ca85bc2.css>
<link rel="stylesheet" media="print" href=/static/styles/bmc/print-3a6240f456.css>

<link rel="stylesheet" class="js-ctm" href=/static/styles/bmc/main-84414d3d47.css
      media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">

        

        <script src="/static/js/app-bundle-8d61e6ef37.js" defer></script>
        
        <script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script>
        
    <script type="text/javascript">
        config = {
            env: 'live',
            site: 'hereditasjournal.biomedcentral.com',
            siteWithPath: 'hereditasjournal.biomedcentral.com' + window.location.pathname,
            twitterHashtag: '',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/s41065-017-0034-4',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central'
        };
    </script>

        
    <script type="text/javascript">
        dataLayer = [{"content":{"article":{"doi":"10.1186/s41065-017-0034-4","articleType":"Research","peerReviewType":"Closed","supplement":null,"keywords":"Gossypium;Microsatellites;Motif imperfection;Comparative genomics;Evolution"},"contentInfo":{"imprint":"BioMed Central","title":"A comparative genomics approach revealed evolutionary dynamics of microsatellite imperfection and conservation in genus Gossypium","publishedAt":1495065600000,"publishedAtDate":"2017-05-18","author":["Muhammad Mahmood Ahmed","Chao Shen","Anam Qadir Khan","Muhammad Atif Wahid","Muhammad Shaban","Zhongxu Lin"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":"CC BY + CC0","openAccess":true},"environment":"live"},"journal":{"siteKey":"hereditasjournal.biomedcentral.com","volume":"154","issue":"1","title":"Hereditas","type":"AJs Life","journalID":41065,"gaCode":"UA-62066850-11","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Research","publishingSegment":"Genetics & Heredity (Life)"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}}}]

    </script>

        <script src="https://cdn.cookielaw.org/consent/7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script>
        
    
        
    

        
    
        
            
                <!-- Google Tag Manager -->
                <script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script>
                <!-- End Google Tag Manager -->
            
        
    


        
    
    <link rel="canonical" href="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0034-4"/>
    

        
        
        <meta property="og:url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0034-4"/>
        <meta property="og:type" content="article"/>
        <meta property="og:site_name" content="Hereditas"/>
        <meta property="og:title" content="A comparative genomics approach revealed evolutionary dynamics of microsatellite imperfection and conservation in genus Gossypium"/>
        <meta property="og:description" content="Ongoing molecular processes in a cell could target microsatellites, a kind of repetitive DNA, owing to length variations and motif imperfection. Mutational mechanisms underlying such kind of genetic variations have been extensively investigated in diverse organisms. However, obscure impact of ploidization, an evolutionary process of genome content duplication prevails mostly in plants, on non-coding DNA is poorly understood. Genome sequences of diversely originated plant species were examined for genome-wide motif imperfection pattern, and various analytical tools were employed to canvass characteristic relationships among repeat density, imperfection and length of microsatellites. Moreover, comparative genomics approach aided in exploration of microsatellites conservation footprints in Gossypium evolution. Based on our results, motif imperfection in repeat length was found intricately related to genomic abundance of imperfect microsatellites among 13 genomes. Microsatellite decay estimation depicted slower decay of long motif repeats which led to predominant abundance of 5-nt repeat motif in Gossypium species. Short motif repeats exhibited rapid decay through the evolution of Gossypium lineage ensuing drastic decrease of 2-nt repeats, of which, “AT” motif type dilapidated in cultivated tetraploids of cotton. The outcome could be a directive to explore comparative evolutionary footprints of simple non-coding genetic elements i.e., repeat elements, through the evolution of genus-specific characteristics in cotton genomes."/>
        <meta property="og:image" content="https://images.springer.com/sgw/journals/medium/41065.jpg"/>
    
    </head>

    <body class="journal journal-fulltext"
          
          >
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK"
                            height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><feColorMatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"/></filter></defs><symbol id="icon-arrow-down" viewBox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"/></symbol><symbol id="icon-arrow-left-bullet" viewBox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"/></symbol><symbol id="icon-arrow-right-bullet" viewBox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"/></symbol><symbol id="icon-arrow-right" viewBox="0 0 32 64"><path d="M0 0v64l32-32z"/></symbol><symbol id="icon-close" viewBox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"/></symbol><symbol id="icon-community" viewBox="0 0 60.643 31.01"><mask maskUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><path fill="#FFF" d="M0 31.01h60.643V0H0z" filter="url(#a)"/></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"/></symbol><symbol id="icon-download" viewBox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"/></symbol><symbol id="icon-ext-link" viewBox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"/><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"/><path fill="#676767" d="M9.7 0L16 6.2V0z"/></symbol><symbol id="icon-info-bordered" viewBox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"/><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"/></symbol><symbol id="icon-opr" viewBox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"/><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"/></symbol><symbol id="icon-quotation" viewBox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"/></symbol><symbol id="icon-remove" viewBox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"/><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"/></symbol><symbol id="icon-ribbon" viewBox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"/></symbol><symbol id="icon-rss" viewBox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"/><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"/><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"/></g></symbol><symbol id="icon-search" viewBox="0 0 16 16"><path d="M6.276 12.257C2.815 12.257 0 9.508 0 6.128 0 2.749 2.815 0 6.276 0c3.462 0 6.277 2.749 6.277 6.128.002 3.38-2.815 6.129-6.277 6.129zm0-10.772c-2.622 0-4.755 2.083-4.755 4.643 0 2.562 2.133 4.644 4.755 4.644 2.623 0 4.756-2.082 4.756-4.644 0-2.56-2.134-4.643-4.756-4.643z"/><path d="M16 13.751L13.641 16l-4.875-4.82 2.36-2.248L16 13.751z"/></symbol><symbol id="icon-speech-bubble" viewBox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"/></symbol><symbol id="icon-error" viewBox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"/></symbol><symbol id="icon-info" viewBox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"/></symbol><symbol id="icon-success" viewBox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"/></symbol><symbol id="icon-warning" viewBox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"/></symbol><symbol id="icon-facebook-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"/></symbol><symbol id="icon-facebook" viewBox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"/><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"/></symbol><symbol id="icon-gplus-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M480.5 273.117c.123.076.26.185.416.334.146.152.289.343.432.562.135.211.258.459.359.746.086.286.129.619.129 1.004-.012.7-.164 1.262-.465 1.683a5.374 5.374 0 0 1-.461.562c-.186.176-.375.35-.58.527a2.65 2.65 0 0 0-.322.41.908.908 0 0 0-.176.553c0 .205.059.375.18.508.102.129.199.238.299.336l.662.545a6.48 6.48 0 0 1 1.086 1.119c.293.415.445.955.459 1.625 0 .949-.42 1.789-1.256 2.521-.869.76-2.123 1.15-3.762 1.177-1.373-.015-2.4-.308-3.074-.879-.682-.534-1.023-1.172-1.023-1.917 0-.359.111-.766.336-1.211.215-.443.602-.837 1.172-1.174a6.126 6.126 0 0 1 1.998-.725 15.415 15.415 0 0 1 1.723-.173 3.996 3.996 0 0 1-.377-.596 1.347 1.347 0 0 1-.193-.74c0-.174.023-.317.074-.433a8.83 8.83 0 0 1 .121-.338 5.973 5.973 0 0 1-.633.035c-1.043-.012-1.838-.339-2.381-.979-.574-.603-.861-1.3-.861-2.097 0-.961.404-1.832 1.217-2.616.559-.46 1.135-.757 1.736-.898a8.494 8.494 0 0 1 1.67-.182h3.916l-1.209.704h-1.207v.005l-.005.002zm.754 11.276c0-.498-.162-.932-.484-1.302-.346-.348-.881-.778-1.609-1.29a4.045 4.045 0 0 0-.439-.02c-.102-.016-.354 0-.768.039a8.141 8.141 0 0 0-1.244.275c-.1.036-.24.096-.42.17a2.713 2.713 0 0 0-.553.355 1.985 1.985 0 0 0-.459.592c-.145.25-.215.551-.215.901 0 .687.311 1.253.932 1.698.596.444 1.404.68 2.432.688.928-.014 1.629-.217 2.117-.612.476-.385.71-.886.71-1.494zm-2.729-5.743a1.843 1.843 0 0 0 1.289-.556c.164-.249.271-.506.318-.767a7.32 7.32 0 0 0 .041-.659c0-.772-.195-1.552-.594-2.336a2.611 2.611 0 0 0-.73-.921 1.894 1.894 0 0 0-1.064-.352 1.723 1.723 0 0 0-1.324.64c-.299.437-.438.92-.428 1.457 0 .709.209 1.444.623 2.211.201.356.459.659.775.912.323.248.686.371 1.094.371zm9.698.077h-2.055v-2.056h-.996v2.056h-2.053v.998h2.053v2.052h.996v-2.052h2.055z"/></symbol><symbol id="icon-gplus" viewBox="0 0 24 24"><circle fill="#D84A36" cx="12" cy="12" r="12"/><g fill="#FFF"><path d="M12.012 12.111c-.304-.33-.07-.842.155-1.078.226-.241.639-.56.829-.74.193-.184.66-.605.674-1.868 0 0 .112-1.139-1.127-2.06h.977l1.038-.593h-3.507s-2.083-.067-3.085 1.207c-1 1.276-.966 2.871.114 3.77 1.08.898 2.371.648 2.371.648s-.197.422-.163.775c.036.353.421.849.504.965 0 0-2.386-.011-3.551 1.033-1.165 1.053-1.29 3.097 1.021 3.803 2.312.707 5.303-.055 5.906-2.277 0 0 .262-.908-.189-1.683-.459-.775-1.663-1.571-1.967-1.902zm-2.555-1.83c-.744-1-1.308-2.953 0-3.783 0 0 .955-.579 1.801.266.846.846 1.082 2.277.928 3.003-.15.727-.563.897-.739 1.008-.185.111-1.249.504-1.99-.494zm3.668 5.481c-.035.393-.083 1.254-1.491 1.664-1.409.408-3.826-.264-3.757-1.914 0 0 .006-.698.449-1.089.445-.396 1.094-.794 3.029-.819 0 0 .643.473 1.072.809.427.332.733.952.698 1.349z"/><path d="M16.323 11.055V9.803h-.752v1.252h-1.255v.748h1.255v1.256h.752v-1.256h1.248v-.748z"/></g></symbol><symbol id="icon-linkedin" viewBox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"/><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"/><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"/></g></symbol><symbol id="icon-reddit" viewBox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"/><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"/><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"/><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"/><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"/><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"/><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"/><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"/></symbol><symbol id="icon-twitter-bordered" viewBox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"/><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/></symbol><symbol id="icon-twitter" viewBox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"/><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"/></symbol><symbol id="icon-weibo-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"/><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"/><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"/></symbol><symbol id="icon-weibo" viewBox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"/><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"/><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"/><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"/><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"/><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"/></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner c-banner--info">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner" >
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1"
                 data-gpt-unitpath="/270604982/bmc/hereditasjournal/articles"
                 data-gpt-sizes="728x90,970x90"
                 data-gpt-targeting="pos=LB1;doi=10.1186/s41065-017-0034-4;kwrd=Gossypium,Microsatellites,Motif imperfection,Comparative genomics,Evolution;pmc=L32030,L21001;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-017-0034-4&amp;kwrd=Gossypium,Microsatellites,Motif imperfection,Comparative genomics,Evolution&amp;pmc=L32030,L21001&amp;">
                        <img data-test="gpt-advert-fallback-img"
                             src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-017-0034-4&amp;kwrd=Gossypium,Microsatellites,Motif imperfection,Comparative genomics,Evolution&amp;pmc=L32030,L21001&amp;"
                             alt="Advertisement"
                             width="728"
                             height="90">
                    </a>
                </noscript>
            </div>
        </div>
    </div>

        
         
    <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>


<header class="c-popup-search ctx-publisher-header" id="top" data-test="publisher-header">
    <nav class="c-navbar">
        <div class="u-container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization" />
                <div class="c-logo">
    <a href="https://www.biomedcentral.com" itemprop="url">
        <img src=/static/images/bmc/logos/logo-bmc-white-aj-e3a70ae848.svg alt="BMC" itemprop="logo">
        <div class="c-logo__strapline">
            <img src=/static/images/bmc/logos/logo-bmc-white-strapline-sn-e7b1d6e67f.svg alt="Part of Springer Nature">
        </div>
    </a>
</div>
            </div>
            <button
                type="button"
                class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg"
                id="publisher-header-menu-button"
                data-toggle="collapse"
                data-test="publisher-header-menu-toggle"
                data-toggle-linked="publisher-header"
                data-target="publisher-header-search publisher-header-menu"
                aria-controls="publisher-header-menu publisher-header-search"
                aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


                <ul class="c-navbar__nav">
                    <li class="c-navbar__item">
                        <a data-replace="true"
                           data-placeholder="/placeholder/v1/username"
                           class="c-navbar__link"
                           href="https://www.biomedcentral.com/login"
                           data-test="login-link"
                           data-track="click"
                           data-track-category="Account"
                           data-track-action="Click Login Link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                    <li class="c-navbar__item">
                        <button
                            type="button"
                            class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg"
                            data-toggle="collapse"
                            data-test="header-search-button"
                            data-target="publisher-header-search publisher-header-menu publisher-header-menu-button"
                            aria-controls="publisher-header-search publisher-header-menu"
                            aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true">
                                    <use xlink:href="#icon-search"></use>
                                </svg>
                            </span>
                        </button>
                    </li>
                </ul>
            </div>
        </div>
    </nav>
    <div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field c-form-field--inline" method="GET" action="//www.biomedcentral.com/search" data-track="submit"
        data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label data-track-label="" data-test="global-search">
        <label for="publisherSearch" class="u-visually-hidden">Search all BMC articles</label>
        <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text" value="" placeholder="Search all BMC articles"/>
        <button class="c-button c-button--secondary" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true">
        <use xlink:href="#icon-search"></use>
    </svg>
</button>


        <input type="hidden" name="searchType" value="publisherSearch"/>
    </form>
</div>

            </div>
        </div>
    </div>
</header>


        
            <header class="c-journal-header c-journal-header--hereditas ctx-journal-header">
                <div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="/">
    
        
    
    <span class="c-journal-title__text ">Hereditas</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="u-container">
                        
                            <button type="button"
                                    class="c-navbar__link c-dropdown u-hidden-at-lg"
                                    data-toggle="collapse"
                                    data-test="journal-header-menu"
                                    data-toggle-linked="journal-header"
                                    data-target="journal-menu"
                                    aria-controls="journal-menu"
                                    aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav">
                                    <ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/'>Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/about'>About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href='/articles'>Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/submission-guidelines'>Submission Guidelines</a>
                                            </li>
                                        
                                    </ul>
                                </nav>
                            </div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--ocean-light-blue">
                    
                </div>
            </header>
            
        

        <div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-content-layout c-content-layout--fulltext" data-component="sticky-container">
                <div class="c-content-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3><ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc=""><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Methods">Methods</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec11" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results">Results</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec19" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Discussion">Discussion</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec20" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Conclusion">Conclusion</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li></ol></nav>
                </div>

                <main class="c-content-layout__main">
                    <article itemscope itemtype="http://schema.org/ScholarlyArticle">
                        
                        
    <div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
            <li class="c-article-identifiers__item" data-test="article-category">Research</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
    </div>

                        <div class="FulltextWrapper"><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">A comparative genomics approach revealed evolutionary dynamics of microsatellite imperfection and conservation in genus <em xmlns="" class="EmphasisTypeItalic">Gossypium</em>
                  </h1></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true"><ul class="u-listReset"><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Muhammad&nbsp;Mahmood&nbsp;Ahmed</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Chao&nbsp;Shen</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Anam&nbsp;Qadir&nbsp;Khan</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Muhammad&nbsp;Atif&nbsp;Wahid</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Muhammad&nbsp;Shaban</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup> and </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Zhongxu&nbsp;Lin</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup><a href="mailto:linzhongxu@mail.hzau.edu.cn" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a></li></ul></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation"><div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">Hereditas</span></span><span class="ArticleCitation_Year">2017</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">154</strong>:12</span></span></div><p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/s41065-017-0034-4">https://doi.org/10.1186/s41065-017-0034-4</a></p><p class="Copyright">© &nbsp;The Author(s)&nbsp;2017</p></div></div><ul class="ArticleHistory u-reset-list"><li class="History HistoryReceived"><strong>Received: </strong><span>14&nbsp;October&nbsp;2016</span></li><li class="History HistoryAccepted"><strong>Accepted: </strong><span>27&nbsp;April&nbsp;2017</span></li><li class="History HistoryOnlineDate"><strong>Published: </strong><span itemprop="datePublished">18&nbsp;May&nbsp;2017</span></li></ul></div>
                        
                        
                        
    

                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2><div class="c-section__content" id="Abstract"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p id="Par1" class="Para">Ongoing molecular processes in a cell could target microsatellites, a kind of repetitive DNA, owing to length variations and motif imperfection. Mutational mechanisms underlying such kind of genetic variations have been extensively investigated in diverse organisms. However, obscure impact of ploidization, an evolutionary process of genome content duplication prevails mostly in plants, on non-coding DNA is poorly understood.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Results</h3>
                <p id="Par2" class="Para">Genome sequences of diversely originated plant species were examined for genome-wide motif imperfection pattern, and various analytical tools were employed to canvass characteristic relationships among repeat density, imperfection and length of microsatellites. Moreover, comparative genomics approach aided in exploration of microsatellites conservation footprints in <em xmlns="" class="EmphasisTypeItalic">Gossypium</em> evolution. Based on our results, motif imperfection in repeat length was found intricately related to genomic abundance of imperfect microsatellites among 13 genomes. Microsatellite decay estimation depicted slower decay of long motif repeats which led to predominant abundance of 5-nt repeat motif in <em xmlns="" class="EmphasisTypeItalic">Gossypium</em> species. Short motif repeats exhibited rapid decay through the evolution of <em xmlns="" class="EmphasisTypeItalic">Gossypium</em> lineage ensuing drastic decrease of 2-nt repeats, of which, “AT” motif type dilapidated in cultivated tetraploids of cotton.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusion</h3>
                <p id="Par3" class="Para">The outcome could be a directive to explore comparative evolutionary footprints of simple non-coding genetic elements i.e., repeat elements, through the evolution of genus-specific characteristics in cotton genomes.</p>
              </div></div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords"><li class="c-keywords__item" itemprop="about">
                <em xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="EmphasisTypeItalic">Gossypium</em>
              </li><li class="c-keywords__item" itemprop="about">Microsatellites</li><li class="c-keywords__item" itemprop="about">Motif imperfection</li><li class="c-keywords__item" itemprop="about">Comparative genomics</li><li class="c-keywords__item" itemprop="about">Evolution</li></ul></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par16" class="Para">Microsatellites are DNA structural elements in which a short sequence pattern (motif) is repeated by various numbers and ubiquitously presented in genomes of eukaryotes and prokaryotes. Various mechanisms like replication slippage, unequal crossover, realignment after disassociation of replicating strands, mispairing and base substitution are causative to variations which ultimately lead to extensive polymorphism in microsatellites [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>, <span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>]. Replication slippage events are majorly responsible for extensive repeat length variations, and mispairing of replication strands is key determinant for base substitution in DNA sequence [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>]. Moreover, stable wobble mispairing and inefficiency of mismatch repair system could incorporate point mutations anywhere in genome sequence [<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>]. These processes could impinge on microsatellites and generate repeats with occasional mismatch, motif imperfection, in their repeat units.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par17" class="Para">During the replication process, insertion or deletion in repeat motif units due to reduced strand specificity or slippage errors could generate length variations. Various models comprehend and illustrate the mechanism of replication slippage event [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>]. A single unit repeat is inserted or deleted at a time according to stepwise mutation model [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>], while repeat unit variations were found associated to a distinct probability distribution under specific assumptions [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>]. Similarly, another model proposed generation of microsatellites as 3′ extension of retrotransposons in a way similar to polyadenylation of mRNA [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>]; while, idea of probable generation and insertion of microsatellites in 5′ region and inside the mobile elements further extended the model [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>]. On contrary, the mechanisms underlying mispairing and interruptions leading to single nucleotide changes are poorly understood [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par18" class="Para">Non-coding DNA elements, including microsatellites, also experience various evolutionary forces and show high mutation rates (10<sup>−2</sup> – 10<sup>−6</sup>) in response to selection forces as these usually go unnoticed [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>]. Thus, higher mutability and hyper variability of microsatellites could be key determinants of a dynamic equilibrium state in which rapid loss or conservation of non-coding DNA elements exist in species among or within a clade [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>]. Various studies reported microsatellite conservation over longer time [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>], and recently a study reported conservation up to 450 million years ago (MYA) in vertebrates [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>]. Similarly, evolutionary forces might regulate the mechanism which drives fate determination processes of non-coding or coding DNA elements. Thus, conservation and/or deterioration of DNA structural elements over a geographic time period determine impact of species-evolving processes like speciation, diversification, domestication and even duplication through polidization in plant genomes.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par19" class="Para">Non-coding elements constitute a major proportion of DNA in all forms of life and play a crucial role in modulating coding regions of DNA. Occurrence and applicability of microsatellites in coding sequence (CDS) have elucidated their functional importance [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>]. Since microsatellites are implied to regulate biological functions [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>], mutations in coding microsatellites could be informative and even point mutation might induce deleterious effects. However, our brief understandings about mechanisms underlying motif imperfection, either through base substitutions or point mutation, cause an eminent impediment in this regard.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par20" class="Para">In this report, genome-wide motif imperfection pattern were discerned in 13 plant genomes of diverse origins. The relationship between repeat length and degree of motif imperfection with its impact on the genomic abundance of imperfect microsatellites was determined, particularly for <em xmlns="" class="EmphasisTypeItalic">Gossypium</em> species. Moreover, evolutionary patterns of microsatellite conservation and/or loss among <em xmlns="" class="EmphasisTypeItalic">Gossypium</em> species were also established to ascertain structural consequences of whole genome duplication and allopolypolidization events through evolution of cultivated cotton tetraploids.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Methods" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Methods">Methods</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Methods" data-test="collapse-section"><section id="Sec3" class="Section2 RenderAsSection2">
                <h3 class="Heading">Genome assemblies of 13 plant species</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par21" class="Para">The genome sequences of thirteen plant species were investigated comprising ten dicots and three monocots. The dicot species belonged to family Brassicaceae (<em xmlns="" class="EmphasisTypeItalic">Arabidopsis thaliana</em> and <em xmlns="" class="EmphasisTypeItalic">Brassica rapa</em>), Fabaceae (<em xmlns="" class="EmphasisTypeItalic">Glycine max</em> and <em xmlns="" class="EmphasisTypeItalic">Phaseolus vulgaris</em>), Salicaceae (<em xmlns="" class="EmphasisTypeItalic">Populus trichocarpa</em>) and majorly to Malvaceae (<em xmlns="" class="EmphasisTypeItalic">Gossypium arboreum</em>, <em xmlns="" class="EmphasisTypeItalic">G. raimondii</em>, <em xmlns="" class="EmphasisTypeItalic">G. hirsutum</em>, <em xmlns="" class="EmphasisTypeItalic">G. barbadence</em> and <em xmlns="" class="EmphasisTypeItalic">Theobroma cacao</em>). The three monocots belonged to family Poaceae (<em xmlns="" class="EmphasisTypeItalic">Brachypodium distachyon</em>, <em xmlns="" class="EmphasisTypeItalic">Oryza sativa</em> and <em xmlns="" class="EmphasisTypeItalic">Zea mays</em>). In addition, coding sequences of annotated genes were also searched for imperfect repeats among four <em xmlns="" class="EmphasisTypeItalic">Gossypium</em> species. The genome assemblies and CDS sequences of <em xmlns="" class="EmphasisTypeItalic">Gossypium</em> species were retrieved from Cottongen [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>], while genome sequences of other nine plant species were obtained from NCBI Genome Portal [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>]. The name of each species was abbreviated to four letters where first capital letter denoted to <em xmlns="" class="EmphasisTypeItalic">genus</em> and trailing three letters as <em xmlns="" class="EmphasisTypeItalic">species</em> name.</p>
              </section><section id="Sec4" class="Section2 RenderAsSection2">
                <h3 class="Heading">Imperfect microsatellites identification</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par22" class="Para">The SciRoKo (v3.4) program utility [<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>] was used with default imperfect search parameters to identify imperfect microsatellites of varying motif length from 1 to 6 nucleotide (nt) that were designated as 1-nt, 2-nt, 3-nt, 4-nt, 5-nt and 6-nt among 13 plant genomes. Imperfect search criteria regarding repeat length and mismatch penalty were modified, as previously described [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>]. Maximum number of successive mismatch was set to ‘3’, and minimum repeat length was set to 15&nbsp;nt. Compound microsatellites were excluded where maximum distance for association between two repeats was less than 100&nbsp;nt. For motif standardization, each microsatellite motif underwent a two-step complete standardization procedure. In the first step, repeat motifs like “AG” and “GA” were all categorized as “AG” and termed as partially standardized motifs. Subsequently, in the second step, reverse complement sequence of motifs was determined and all microsatellites were assigned to completely standardized motifs. Eventually, motifs like “AG”, “GA”, “TC” and “CT” were all designated to a single group “AG”” for further analysis. Such complete motif standardization resulted in 2, 4, 10, 33, 102 and 350 group categories for 1-nt, 2-nt, 3-nt, 4-nt, 5-nt and 6-nt repeat motifs, respectively.</p>
              </section><section id="Sec5" class="Section2 RenderAsSection2">
                <h3 class="Heading">Motif imperfection and repeat length analyses</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par23" class="Para">The genomic abundance of imperfect repeats was ascertained with varying mismatch counts and repeat lengths. Initially, short length repeats were selected and motif imperfection was employed to discriminate between perfect (with no mismatch) and imperfect microsatellites (with mismatch &gt;0) among all genomes. It was noticed that loci with repeat length&nbsp;&lt;&nbsp;20&nbsp;nt did not carry mismatch, thus were excluded from further analysis. While, repeats &lt;25&nbsp;nt had only ‘1’ mismatch per locus and there were up to ‘2’ mismatch in repeat loci &lt;30&nbsp;nt. The two datasets were mined separately, one having genomic abundance of microsatellites with ‘0’ or ‘1’ mismatch (no or low degree imperfection). While, other comprised of exactly ‘1’ or ‘2’ mismatch (low or higher degree imperfection) for repeats of length 20 to 24&nbsp;nt (dataset I) and 25 to 29&nbsp;nt (dataset II), respectively.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par24" class="Para">Thereafter, three different analytical strategies were employed. Firstly, principal coordinates method [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>] was used to conduct generalized discriminant analysis on both datasets separately. The analysis generated canonical axes scores which were used to determine correlation among the variables, and statistical significance was estimated by permutation (<em xmlns="" class="EmphasisTypeItalic">n</em>&nbsp;=&nbsp;9999) method implemented in ‘CAP’ program [<span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>]. Moreover, the R package [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>] was also used to test the significance of canonical correlations by add-on package ‘CCP’ [<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>]. Secondly, longer microsatellites of length 35, 40, 45, 50, 55, 60, 65, 70, 75 and 80&nbsp;nt were targeted with different levels of imperfection (1, 2, 3 and 4 mismatch per loci). The relationships between motif imperfection and varied length repeats were estimated using permutational analysis of variance <em xmlns="" class="EmphasisTypeItalic">PERMANOVA</em> [<span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>] by calling “adonis” function under package vegan [<span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>], and statistical significance of <em xmlns="" class="EmphasisTypeItalic">F-statistic</em> was measured (0.05) after permutations (<em xmlns="" class="EmphasisTypeItalic">n</em>&nbsp;=&nbsp;999) in R environment. Thirdly, the genomic abundance of imperfect microsatellites was related to mismatch counts and repeat length through <em xmlns="" class="EmphasisTypeItalic">t-statistic</em> and significant difference were determined (0.05). A linear regression model was fitted to mismatch count and repeat length in each <em xmlns="" class="EmphasisTypeItalic">Gossypium</em> species.</p>
              </section><section id="Sec6" class="Section2 RenderAsSection2">
                <h3 class="Heading">Transposable elements (TEs) distribution of perfect vs imperfect microsatellites</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par25" class="Para">Perfect (no mismatch) and imperfect repeats (mismatch ≥1) were compared for relatedness to nearby intact TEs. Each chromosome was considered as independent segment, and TEs abundance was determined among the four cotton genomes in vicinity of microsatellites (500&nbsp;nt flanked region on both sides) for both sets of repeats. Then, a linear regression model was fitted to the repeat density and TEs abundance in each <em xmlns="" class="EmphasisTypeItalic">Gossypium</em> species.</p>
              </section><section id="Sec7" class="Section2 RenderAsSection2">
                <h3 class="Heading">Motif imperfection in coding region of <em class="EmphasisTypeItalic">Gossypium</em> species</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par26" class="Para">Annotated coding sequences were searched for imperfect repeats under default criteria. All publically available annotated single nucleotide polymorphism (SNP) markers sequences [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>] for cotton genomes were downloaded and employed for functional characterization of imperfect microsatellites in coding region.</p>
              </section><section id="Sec8" class="Section2 RenderAsSection2">
                <h3 class="Heading">Microsatellites conservation among <em class="EmphasisTypeItalic">Gossypium</em> species</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par27" class="Para">For conservation analysis, microsatellite libraries were developed separately for each species containing repeats flanked with 350&nbsp;nt on both sides. Because of the drawbacks pertained to sequencing homo-polymers, 1-nt repeats were excluded. Moreover, all compound and overlapping sequences were filtered out and Basic Local Alignment Search Tool (BLAST) was used to validate uniqueness of each locus by NCBI BLAST 2.2.31 [<span class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>] through self-BLASTing each library and discarding false positives. A custom perl script was used to hard mask repeat sequences for each locus, and genome to sub-genome (diploid vs sub-diploid in tetraploids) BLAST searches was conducted among libraries. The output was filtered for algorithms like alignment ≥50% of query cover, identity ≥70% and E_value 10<sup>−10</sup>. Since a whole genome duplication (WGD) event was reported through cotton tetraploids’ evolution [<span class="CitationRef"><a href="#CR28" data-toggle="citation">28</a></span>], duplication of each progenitor loci was allowed up to two physical positions for pair wise comparisons (A<sub>2</sub> vs A<sub>T</sub> &amp; D<sub>5</sub> vs D<sub>T</sub>) among homologous chromosomes. The results were validated in reciprocal BLAST searches. Thereafter, conserved microsatellites proportion was employed to develop an intuitive diagram for genome-wide microsatellite conservation by motif size using circos tool [<span class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>].</p>
              </section><section id="Sec9" class="Section2 RenderAsSection2">
                <h3 class="Heading">Estimate of microsatellite decay during paleopolyploidization</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par28" class="Para">The WGD event involved through paleopolyploidization of cultivated tetraploid species was estimated with varied divergence time [<span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>]. A representative median divergence time as ~6 MYA was used. Assuming steady loss of microsatellite loci over the period of divergence time, an exponential fitted decay function was employed to estimate “comparative exponential decay” of microsatellite in <em xmlns="" class="EmphasisTypeItalic">G. hirsutum</em> and <em xmlns="" class="EmphasisTypeItalic">G</em>. <em xmlns="" class="EmphasisTypeItalic">barbadence</em>. On contrary, the “relative decay” estimates of varying motif repeats in tetraploids were determined by comparing proportion of conserved repeats. Thereafter, “relative decay” and “comparative exponential decay” were fitted to an exponential decay function. All non-parametric test statistics were measured and tested for significance (0.05) in R environment.</p>
              </section><section id="Sec10" class="Section2 RenderAsSection2">
                <h3 class="Heading">Microsatellite relative abundance</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par29" class="Para">The repeat density patterns in cotton diploids exploited evolutionary footprints regarding impact of paleopolyploidization event on it in three different ways. Firstly, proportional abundance of microsatellites in diploid and tetraploid <em xmlns="" class="EmphasisTypeItalic">Gossypium</em> species was estimated by comparing their relative abundance in <em xmlns="" class="EmphasisTypeItalic">T. cacao</em>, a close relative belong to Malvaceae. Secondly, the relative abundance of microsatellites according to motif kind and size in four cotton genomes was determined individually. And lastly, comparative abundance of standardized 2-nt motif was analyzed in four cotton genomes. All non-parametric test statistics were measured and tested for significance (0.05) in R environment.</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec11" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Results" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Results">Results</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Results" data-test="collapse-section"><section id="Sec12" class="Section2 RenderAsSection2">
                <h3 class="Heading">Frequency and distribution of microsatellites among monocots and dicots</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par30" class="Para">Microsatellite distribution of varying motif length (1–6&nbsp;nt) was examined in ten dicots and three monocots. Generally, 2-nt and 3-nt repeats were found in higher percentage in dicots and monocots (Table <span class="InternalRef"><a href="#Tab1">1</a></span>), respectively. While, Atha exhibited almost equal proportion of three motifs (1–3&nbsp;nt). A plentiful abundance of 3-nt microsatellite was observed in Osat (33.67%) and the trend persisted among Bdis and Zmay. Among dicots, the proportion of 2-nt repeats prevailed and Brap featured the highest proportion of it (39.67%). However, frequency of 5-nt microsatellites predominated and featured specific to four <em xmlns="" class="EmphasisTypeItalic">Gossypium</em> genomes. Reckoning to the four cotton genomes, a drastic reduction in percentage of 2-nt repeats was observed in tetraploids (Ghir and Gbar) up to 11.95%. While, density of microsatellites with short motif repeats decreased, those with longer motifs (4–6&nbsp;nt) exhibited incessant abundance in tetraploids compared to their progenitors (Table <span class="InternalRef"><a href="#Tab1">1</a></span>).<figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 1</span>
                        <p xmlns="" class="SimplePara">Microsatellite distribution as percent motif abundance (%) among 13 plant genomes</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Species</p></th><th><p xmlns="" class="SimplePara">Abbreviation</p></th><th><p xmlns="" class="SimplePara">Mono- (1-nt)</p></th><th><p xmlns="" class="SimplePara">Di- 2-nt)</p></th><th><p xmlns="" class="SimplePara">Tri- (3-nt)</p></th><th><p xmlns="" class="SimplePara">Tetra- (4-nt)</p></th><th><p xmlns="" class="SimplePara">Penta- (5-nt)</p></th><th><p xmlns="" class="SimplePara">Hexa- (6-nt)</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">A. thaliana</em>
                            </p></td><td><p xmlns="" class="SimplePara">Atha</p></td><td><p xmlns="" class="SimplePara">24.05</p></td><td><p xmlns="" class="SimplePara">22.48</p></td><td><p xmlns="" class="SimplePara">24.48</p></td><td><p xmlns="" class="SimplePara">7.36</p></td><td><p xmlns="" class="SimplePara">15.29</p></td><td><p xmlns="" class="SimplePara">6.33</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">B. rapa</em>
                            </p></td><td><p xmlns="" class="SimplePara">Brap</p></td><td><p xmlns="" class="SimplePara">8.77</p></td><td><p xmlns="" class="SimplePara">39.68</p></td><td><p xmlns="" class="SimplePara">18.44</p></td><td><p xmlns="" class="SimplePara">10.12</p></td><td><p xmlns="" class="SimplePara">16.08</p></td><td><p xmlns="" class="SimplePara">6.91</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">G. arboreum</em>
                            </p></td><td><p xmlns="" class="SimplePara">Garb</p></td><td><p xmlns="" class="SimplePara">10.02</p></td><td><p xmlns="" class="SimplePara">22.02</p></td><td><p xmlns="" class="SimplePara">14.90</p></td><td><p xmlns="" class="SimplePara">14.12</p></td><td><p xmlns="" class="SimplePara">28.93</p></td><td><p xmlns="" class="SimplePara">10.01</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">G</em>. <em class="EmphasisTypeItalic">barbadence</em>
                                          </p></td><td><p xmlns="" class="SimplePara">Gbar</p></td><td><p xmlns="" class="SimplePara">6.43</p></td><td><p xmlns="" class="SimplePara">11.95</p></td><td><p xmlns="" class="SimplePara">17.12</p></td><td><p xmlns="" class="SimplePara">18.74</p></td><td><p xmlns="" class="SimplePara">33.05</p></td><td><p xmlns="" class="SimplePara">12.72</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">G. hirsutum</em>
                            </p></td><td><p xmlns="" class="SimplePara">Ghir</p></td><td><p xmlns="" class="SimplePara">3.72</p></td><td><p xmlns="" class="SimplePara">11.95</p></td><td><p xmlns="" class="SimplePara">17.58</p></td><td><p xmlns="" class="SimplePara">19.31</p></td><td><p xmlns="" class="SimplePara">34.37</p></td><td><p xmlns="" class="SimplePara">13.07</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">G</em>. <em class="EmphasisTypeItalic">raimondii</em>
                                          </p></td><td><p xmlns="" class="SimplePara">Grai</p></td><td><p xmlns="" class="SimplePara">6.89</p></td><td><p xmlns="" class="SimplePara">19.78</p></td><td><p xmlns="" class="SimplePara">14.78</p></td><td><p xmlns="" class="SimplePara">17.85</p></td><td><p xmlns="" class="SimplePara">29.88</p></td><td><p xmlns="" class="SimplePara">10.82</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">P. trichocarpa</em>
                            </p></td><td><p xmlns="" class="SimplePara">Ptri</p></td><td><p xmlns="" class="SimplePara">14.82</p></td><td><p xmlns="" class="SimplePara">27.05</p></td><td><p xmlns="" class="SimplePara">19.22</p></td><td><p xmlns="" class="SimplePara">12.53</p></td><td><p xmlns="" class="SimplePara">16.84</p></td><td><p xmlns="" class="SimplePara">8.54</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">G. max</em>
                            </p></td><td><p xmlns="" class="SimplePara">Gmax</p></td><td><p xmlns="" class="SimplePara">12.58</p></td><td><p xmlns="" class="SimplePara">32.54</p></td><td><p xmlns="" class="SimplePara">17.88</p></td><td><p xmlns="" class="SimplePara">11.62</p></td><td><p xmlns="" class="SimplePara">18.60</p></td><td><p xmlns="" class="SimplePara">7.79</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">P. vulgaris</em>
                            </p></td><td><p xmlns="" class="SimplePara">Pvul</p></td><td><p xmlns="" class="SimplePara">3.94</p></td><td><p xmlns="" class="SimplePara">30.60</p></td><td><p xmlns="" class="SimplePara">20.90</p></td><td><p xmlns="" class="SimplePara">14.00</p></td><td><p xmlns="" class="SimplePara">20.35</p></td><td><p xmlns="" class="SimplePara">10.20</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">T. cacao</em>
                            </p></td><td><p xmlns="" class="SimplePara">Tcac</p></td><td><p xmlns="" class="SimplePara">12.95</p></td><td><p xmlns="" class="SimplePara">32.60</p></td><td><p xmlns="" class="SimplePara">16.92</p></td><td><p xmlns="" class="SimplePara">12.54</p></td><td><p xmlns="" class="SimplePara">19.23</p></td><td><p xmlns="" class="SimplePara">5.76</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">O. sativa</em>
                            </p></td><td><p xmlns="" class="SimplePara">Osat</p></td><td><p xmlns="" class="SimplePara">5.02</p></td><td><p xmlns="" class="SimplePara">21.03</p></td><td><p xmlns="" class="SimplePara">33.67</p></td><td><p xmlns="" class="SimplePara">13.02</p></td><td><p xmlns="" class="SimplePara">17.12</p></td><td><p xmlns="" class="SimplePara">10.15</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">B. distachyon</em>
                            </p></td><td><p xmlns="" class="SimplePara">Bdis</p></td><td><p xmlns="" class="SimplePara">9.49</p></td><td><p xmlns="" class="SimplePara">11.99</p></td><td><p xmlns="" class="SimplePara">30.25</p></td><td><p xmlns="" class="SimplePara">18.10</p></td><td><p xmlns="" class="SimplePara">20.94</p></td><td><p xmlns="" class="SimplePara">11.51</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Z. mays</em>
                            </p></td><td><p xmlns="" class="SimplePara">Zmay</p></td><td><p xmlns="" class="SimplePara">6.16</p></td><td><p xmlns="" class="SimplePara">19.55</p></td><td><p xmlns="" class="SimplePara">28.39</p></td><td><p xmlns="" class="SimplePara">13.45</p></td><td><p xmlns="" class="SimplePara">20.66</p></td><td><p xmlns="" class="SimplePara">9.51</p></td></tr></tbody></table></div></figure>
                        </div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par31" class="Para">Our data depicted differential variations in repeat density (per Mb) and frequencies of imperfect microsatellites related to genome sizes among 13 plant genomes (Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>: Table S1). Density of imperfect microsatellites varied from 15.61 to 49.07%, and an eminent variation pattern was observed in monocots compared to dicots. Lower repeat density was noticed for species with genome sizes bigger than 1 Gb (10<sup>9</sup> bases) like Zmay, Ghir, Gbar and Garb and the trend persisted except for Bdis. All species depicted differential variations in imperfection percentage, and species belonging to same genus (<em xmlns="" class="EmphasisTypeItalic">Gossypium</em>) exhibited varied extent of imperfection. Moreover, differential variations in frequency of imperfect microsatellites was observed among chromosomes of 13 genomes (Additional file <span class="InternalRef"><a href="#MOESM2">2</a></span>: Table S2). The four <em xmlns="" class="EmphasisTypeItalic">Gossypium</em> species contained substantial variations among cognate homologous chromosomes.</p>
              </section><section id="Sec13" class="Section2 RenderAsSection2">
                <h3 class="Heading">Motif imperfection and repeat length relationship</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par32" class="Para">The average length of imperfect microsatellites increased over perfect ones among 13 plant species (Additional file <span class="InternalRef"><a href="#MOESM3">3</a></span>: Table S3). Based on significant canonical variations observed for both datasets (dataset I and II), the genomic abundance of microsatellites was found related to variations in repeat length and motif imperfection. The results showed a significant relationship between perfect and ‘1’ mismatch repeats where first squared canonical correlation to be 0.958 (<em xmlns="" class="EmphasisTypeItalic">p</em>&nbsp;=&nbsp;0.0002). Likewise, a significant canonical correlation was determined with first squared canonical variation to be 0.926 (<em xmlns="" class="EmphasisTypeItalic">p</em>&nbsp;=&nbsp;0.0008) for dataset II in which ‘1’ and ‘2’ mismatch repeats of varied lengths (25, 26, 27, 28, and 29&nbsp;nt) were included. In dataset I, ‘1’ mismatch repeats explained lesser canonical variation while ‘2’ mismatch repeats commuted in dataset II (Fig. <span class="InternalRef"><a href="#Fig1">1</a></span>). These findings affirmed that repeat length and motif imperfection were somehow modulated in subordinated ways.<figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO1">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0034-4/MediaObjects/41065_2017_34_Fig1_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0034-4/MediaObjects/41065_2017_34_Fig1_HTML.gif" alt="Fig. 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0034-4/MediaObjects/41065_2017_34_Fig1_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0034-4/MediaObjects/41065_2017_34_Fig1_HTML.gif" alt="Fig. 1"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 1</span>
                        <p xmlns="" class="SimplePara">Frequency variations of various mismatch repeats among 10 dicot [<em class="EmphasisTypeItalic">A. thaliana</em> (Atha), <em class="EmphasisTypeItalic">B. rapa</em> (Brap), <em class="EmphasisTypeItalic">G. arboreum</em> (Garb), <em class="EmphasisTypeItalic">G</em>. <em class="EmphasisTypeItalic">barbadence</em> (Gbar), <em class="EmphasisTypeItalic">G. hirsutum</em> (Ghir), <em class="EmphasisTypeItalic">P. trichocarpa</em> (Ptri), <em class="EmphasisTypeItalic">G. max</em> (Gmax), <em class="EmphasisTypeItalic">P. vulgaris</em> (Pvul) and <em class="EmphasisTypeItalic">T. cacao</em> (Tcac)] and 3 monocots [(<em class="EmphasisTypeItalic">O. sativa</em> (Osat), <em class="EmphasisTypeItalic">B. distachyon</em> (Bdis) and <em class="EmphasisTypeItalic">Z. mays</em> (Zmay)] shown as stacked bar graphs. Frequency variations of length repeats (20, 21, 22, 23 and 24&nbsp;nt) harboring (<strong class="EmphasisTypeBold">a</strong>) no mismatch or (<strong class="EmphasisTypeBold">b</strong>) exactly ‘1’ mismatch were presented and (<strong class="EmphasisTypeBold">c</strong>) canonical axes plotted as scatter plot. While frequency variations of length repeats (25, 26, 27, 28 and 29&nbsp;nt) which contained (<strong class="EmphasisTypeBold">d</strong>) exactly ‘1’ mismatch or (<strong class="EmphasisTypeBold">e</strong>) exactly ‘2’ mismatch presented and (<strong class="EmphasisTypeBold">f</strong>) canonical axes plotted as scatter plot</p>
                      </div></figcaption></figure>
                        </div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par33" class="Para">Degree of motif imperfection (mismatch count) could be another factor whose relationship with repeat length might determine genomic abundance of imperfect microsatellites. Outcome of <em xmlns="" class="EmphasisTypeItalic">PERMANOVA</em> analysis suggested a significant relationship among mismatch count, repeat length and frequency of imperfect microsatellites. The variation in genomic abundance of imperfect microsatellites was significantly related to repeat length (<em xmlns="" class="EmphasisTypeItalic">F</em>&nbsp;=&nbsp;8.050, <em xmlns="" class="EmphasisTypeItalic">p</em>&nbsp;=&nbsp;0.0029) and motif imperfection (<em xmlns="" class="EmphasisTypeItalic">F</em>&nbsp;=&nbsp;1.495, <em xmlns="" class="EmphasisTypeItalic">p</em>&nbsp;=&nbsp;0.0055). These findings speculated existence of an evolutionary mechanism which determined length and motif heterogeneity of microsatellites and the pattern prevailed among the studied set of genomes.</p>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par34" class="Para">Pair-wise <em xmlns="" class="EmphasisTypeItalic">Pearson</em> correlation estimates were calculated by controlling motif imperfection (Additional file <span class="InternalRef"><a href="#MOESM4">4</a></span>: Table S4a-b) for different repeat lengths (35, 40, 45, 50, 55, 60, 65, 70 and 75&nbsp;nt). The results established strong relationships among different length microsatellites carrying varying mismatch (Table <span class="InternalRef"><a href="#Tab2">2</a></span>). Genomic abundance of ‘1’ mismatch repeats was found more related to ‘2’ mismatch repeats, and the relationship maintained with ‘3’ mismatch repeats. The scenario became more stringent for pair-wise comparisons between ‘2’ mismatch and ‘3’ mismatch repeats. Thus, the genomic abundance of loci with lower degree of imperfection (1 mismatch) was related to varying repeat lengths. Moreover, repeats with low imperfection (1 mismatch) seemed to modulate genomic abundance of repeats with moderate imperfection (2 mismatch). Similarly, repeats with higher imperfection (3 mismatch) were more inflected by repeats which entertained ‘2’ mismatch. These findings suggested a significant role of motif imperfection in determining length of repeats.<figure xmlns="" id="Tab2" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 2</span>
                        <p xmlns="" class="SimplePara">Pair-wise correlation (<em class="EmphasisTypeItalic">Pearson</em>) estimates for imperfect microsatellites</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col char="."></col><col char="."></col><col char="."></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Repeat length (nt)</p></th><th char="."><p xmlns="" class="SimplePara">Mismatch 1 vs</p><p xmlns="" class="SimplePara">mismatch 2</p></th><th char="."><p xmlns="" class="SimplePara">Mismatch 1 vs</p><p xmlns="" class="SimplePara">mismatch 3</p></th><th char="."><p xmlns="" class="SimplePara">Mismatch 2 vs</p><p xmlns="" class="SimplePara">mismatch 3</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">30&nbsp;nt</p></td><td char="."><p xmlns="" class="SimplePara">0.983</p></td><td char="."><p xmlns="" class="SimplePara">0.921</p></td><td char="."><p xmlns="" class="SimplePara">0.952</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">35&nbsp;nt</p></td><td char="."><p xmlns="" class="SimplePara">0.919</p></td><td char="."><p xmlns="" class="SimplePara">0.919</p></td><td char="."><p xmlns="" class="SimplePara">0.971</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">40&nbsp;nt</p></td><td char="."><p xmlns="" class="SimplePara">0.911</p></td><td char="."><p xmlns="" class="SimplePara">0.807</p></td><td char="."><p xmlns="" class="SimplePara">0.894</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">45&nbsp;nt</p></td><td char="."><p xmlns="" class="SimplePara">
                              <span class="EmphasisTypeUnderline">0.493</span>
                            </p></td><td char="."><p xmlns="" class="SimplePara">0.640</p></td><td char="."><p xmlns="" class="SimplePara">0.763</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">50&nbsp;nt</p></td><td char="."><p xmlns="" class="SimplePara">0.767</p></td><td char="."><p xmlns="" class="SimplePara">0.545</p></td><td char="."><p xmlns="" class="SimplePara">0.821</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">55&nbsp;nt</p></td><td char="."><p xmlns="" class="SimplePara">0.695</p></td><td char="."><p xmlns="" class="SimplePara">0.699</p></td><td char="."><p xmlns="" class="SimplePara">0.822</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">60&nbsp;nt</p></td><td char="."><p xmlns="" class="SimplePara">0.558</p></td><td char="."><p xmlns="" class="SimplePara">0.605</p></td><td char="."><p xmlns="" class="SimplePara">0.643</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">65&nbsp;nt</p></td><td char="."><p xmlns="" class="SimplePara">0.868</p></td><td char="."><p xmlns="" class="SimplePara">0.742</p></td><td char="."><p xmlns="" class="SimplePara">0.774</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">70&nbsp;nt</p></td><td char="."><p xmlns="" class="SimplePara">0.776</p></td><td char="."><p xmlns="" class="SimplePara">0.629</p></td><td char="."><p xmlns="" class="SimplePara">
                              <span class="EmphasisTypeUnderline">0.423</span>
                            </p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">75&nbsp;nt</p></td><td char="."><p xmlns="" class="SimplePara">0.896</p></td><td char="."><p xmlns="" class="SimplePara">
                              <span class="EmphasisTypeUnderline">0.415</span>
                            </p></td><td char="."><p xmlns="" class="SimplePara">
                              <span class="EmphasisTypeUnderline">0.492</span>
                            </p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">80&nbsp;nt</p></td><td char="."><p xmlns="" class="SimplePara">0.742</p></td><td char="."><p xmlns="" class="SimplePara">0.852</p></td><td char="."><p xmlns="" class="SimplePara">0.867</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">The repeat length presented different lengths while frequencies of repeats harbored 1 mismatch (low imperfection), 2 mismatch (moderate imperfection) and 3 mismatch (high imperfection) were compared among 13 plant species. All estimates were significant while <span class="EmphasisTypeUnderline">underlined</span> were non-significant at 0.05</p>
                    </div></figure>
                        </div>
              </section><section id="Sec14" class="Section2 RenderAsSection2">
                <h3 class="Heading">Motif imperfection in <em class="EmphasisTypeItalic">Gossypium</em> genomes</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par35" class="Para">To draw deep insights into evolutionary footprints, subsets of mismatched repeats in four <em xmlns="" class="EmphasisTypeItalic">Gossypium</em> species were considered for following analyses. Linear regression between repeat length and motif imperfection was determined and a prominent relationship was observed in each case (Fig. <span class="InternalRef"><a href="#Fig2">2</a></span>), but with varied degree of variation accounted in the model (Additional file <span class="InternalRef"><a href="#MOESM5">5</a></span>: Table S5).<figure xmlns="" class="Figure" id="Fig2"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO2">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0034-4/MediaObjects/41065_2017_34_Fig2_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0034-4/MediaObjects/41065_2017_34_Fig2_HTML.gif" alt="Fig. 2" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0034-4/MediaObjects/41065_2017_34_Fig2_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0034-4/MediaObjects/41065_2017_34_Fig2_HTML.gif" alt="Fig. 2"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 2</span>
                        <p xmlns="" class="SimplePara">The mismatch (count) and repeat length relationship split among <em class="EmphasisTypeItalic">Gossypium</em> species. Average mismatch of each standardized motif was compared to their average repeat length (nt) in Garb (<em class="EmphasisTypeItalic">G. arboreum</em>), Grai (<em class="EmphasisTypeItalic">G</em>. <em class="EmphasisTypeItalic">raimondii</em>), Ghir (<em class="EmphasisTypeItalic">G. hirsutum</em>) and Gbar (<em class="EmphasisTypeItalic">G</em>. <em class="EmphasisTypeItalic">barbadence</em>)</p>
                      </div></figcaption></figure>
                        </div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par36" class="Para">The genomic abundance of intact TEs was determined in 500&nbsp;nt flanking region (both sides) of perfect and mismatched microsatellites (Additional file <span class="InternalRef"><a href="#MOESM6">6</a></span>: Table S6). The two sets of repeats were compared for relatedness between repeat frequency and intact TEs, and significant relationships were observed (Additional file <span class="InternalRef"><a href="#MOESM7">7</a></span>: Figure S1a-d). Generally, fewer TEs were observed in Grai and D<sub>T</sub> sub-genomes; while, higher abundance of TEs discerned for Garb and A<sub>T</sub> sub-genomes of tetraploids. Moreover, imperfect repeats were more colligated to genomic abundance of TEs than perfect ones except in Garb; where perfect repeats were more related than mismatched ones. This relatedness was further affirmed by <em xmlns="" class="EmphasisTypeItalic">Pearson</em> correlations and significance determined at 0.05 (Table <span class="InternalRef"><a href="#Tab3">3</a></span>). Correlation values of imperfect repeats were also expectedly higher than perfect ones, but not for Garb in which perfect repeats were most likely imbedded in TEs.<figure xmlns="" id="Tab3" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 3</span>
                        <p xmlns="" class="SimplePara">A comparative correlation estimates between perfect (no mismatch) and imperfect (mismatch ≥1) repeats along with their intact TEs in cotton genomes</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col char="."></col><col></col><col char="."></col><col></col></colgroup><thead><tr class="header"><th rowspan="2"><p xmlns="" class="SimplePara">Genome/</p><p xmlns="" class="SimplePara">Sub-genome</p></th><th colspan="2" char="."><p xmlns="" class="SimplePara">Perfect repeats</p></th><th colspan="2" char="."><p xmlns="" class="SimplePara">Imperfect repeats</p></th></tr><tr class="header"><th char="."><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Pearson</em> correlation</p></th><th><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">p</em> value</p></th><th char="."><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">Pearson</em> correlation</p></th><th><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">p</em> value</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">Garb</p></td><td char="."><p xmlns="" class="SimplePara">0.944</p></td><td><p xmlns="" class="SimplePara">1.13E-06</p></td><td char="."><p xmlns="" class="SimplePara">0.743</p></td><td><p xmlns="" class="SimplePara">3.54E-03</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Grai</p></td><td char="."><p xmlns="" class="SimplePara">0.631</p></td><td><p xmlns="" class="SimplePara">2.01E-02</p></td><td char="."><p xmlns="" class="SimplePara">0.835</p></td><td><p xmlns="" class="SimplePara">5.23E-04</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">GhirA<sub xmlns="http://www.w3.org/1999/xhtml">T</sub>
                                          </p></td><td char="."><p xmlns="" class="SimplePara">0.788</p></td><td><p xmlns="" class="SimplePara">1.37E-03</p></td><td char="."><p xmlns="" class="SimplePara">0.911</p></td><td><p xmlns="" class="SimplePara">1.47E-05</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">GhirD<sub xmlns="http://www.w3.org/1999/xhtml">T</sub>
                                          </p></td><td char="."><p xmlns="" class="SimplePara">0.615</p></td><td><p xmlns="" class="SimplePara">2.50E-02</p></td><td char="."><p xmlns="" class="SimplePara">0.939</p></td><td><p xmlns="" class="SimplePara">1.39E-06</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">GbarA<sub xmlns="http://www.w3.org/1999/xhtml">T</sub>
                                          </p></td><td char="."><p xmlns="" class="SimplePara">0.869</p></td><td><p xmlns="" class="SimplePara">1.11E-04</p></td><td char="."><p xmlns="" class="SimplePara">0.879</p></td><td><p xmlns="" class="SimplePara">7.49E-05</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">GbarD<sub xmlns="http://www.w3.org/1999/xhtml">T</sub>
                                          </p></td><td char="."><p xmlns="" class="SimplePara">0.648</p></td><td><p xmlns="" class="SimplePara">1.64E-02</p></td><td char="."><p xmlns="" class="SimplePara">0.846</p></td><td><p xmlns="" class="SimplePara">2.58E-04</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Ghir</p></td><td char="."><p xmlns="" class="SimplePara">0.833</p></td><td><p xmlns="" class="SimplePara">1.23E-07</p></td><td char="."><p xmlns="" class="SimplePara">0.921</p></td><td><p xmlns="" class="SimplePara">2.63E-11</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Gbar</p></td><td char="."><p xmlns="" class="SimplePara">0.876</p></td><td><p xmlns="" class="SimplePara">4.38E-09</p></td><td char="."><p xmlns="" class="SimplePara">0.941</p></td><td><p xmlns="" class="SimplePara">7.22E-13</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">Garb&nbsp;=&nbsp;<em class="EmphasisTypeItalic">G</em>. <em class="EmphasisTypeItalic">arboreum</em>, Grai&nbsp;=&nbsp;<em class="EmphasisTypeItalic">G</em>. <em class="EmphasisTypeItalic">raimondii</em>, GhirA<sub xmlns="http://www.w3.org/1999/xhtml">T =</sub> 
                                    <em class="EmphasisTypeItalic">G</em>. <em class="EmphasisTypeItalic">hirsutum</em> A sub-genome, GhirD<sub xmlns="http://www.w3.org/1999/xhtml">T</sub>&nbsp;=&nbsp;<em class="EmphasisTypeItalic">G</em>. <em class="EmphasisTypeItalic">hirsutum</em> D sub-genome, GbarA<sub xmlns="http://www.w3.org/1999/xhtml">T</sub>&nbsp;=&nbsp;<em class="EmphasisTypeItalic">G. barbadence</em> A sub-genome, GbarD<sub xmlns="http://www.w3.org/1999/xhtml">T</sub>&nbsp;=&nbsp;<em class="EmphasisTypeItalic">G</em>. <em class="EmphasisTypeItalic">barbadence</em> D sub-genome, Ghir&nbsp;=&nbsp;<em class="EmphasisTypeItalic">G</em>. <em class="EmphasisTypeItalic">hirsutum</em> and Gbar&nbsp;=&nbsp;<em class="EmphasisTypeItalic">G</em>. <em class="EmphasisTypeItalic">barbadence</em>
                                 </p>
                    </div></figure>
                        </div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par37" class="Para">Differential abundance of imperfect repeats along with intact TEs distribution was determined among individual chromosomes. Interestingly, the frequency of imperfect microsatellites and intact TEs for all species was found at par (Additional file <span class="InternalRef"><a href="#MOESM6">6</a></span>: Table S6). The higher abundance of intact TEs was observed in Garb than Grai, while vice versa was true for imperfect repeats. Chromosomes A05, A08, A10 and A11 exhibited higher density of both repeat elements, while a mixed pattern was observed for D03, D07 and D12 among tetraploids. Structural anomalies of chromosomes in two genetically distant progenitors, evolutionary processes, biased selection forces, mutations, deletions and translocations of larger DNA segments could be causative to such abrupt differentiations among tetraploids and diploids. Thus, comparative distributive pattern and its relatedness suggested more likely presence of intact TEs wherever a mismatch found in a microsatellite.</p>
              </section><section id="Sec15" class="Section2 RenderAsSection2">
                <h3 class="Heading">Motif interruptions in coding sequence</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par38" class="Para">Coding sequences of <em xmlns="" class="EmphasisTypeItalic">Gossypium</em> species were also searched for imperfection and results were summarized in Additional file <span class="InternalRef"><a href="#MOESM8">8</a></span>: Table S7. Generally, tri- and hexa- motif repeats prevailed, while only tri- motif repeats accounted for &gt;70% of microsatellites in CDS regions (Table <span class="InternalRef"><a href="#Tab4">4</a></span>). It was expected due to triplet and degenerate nature of codons. The motif imperfection in Grai repeats was elevated and slight increment in average repeat length was noticed. The tandem repeat density was reduced than genomic ones, while the higher proportion of genes in Ghir harbored repeats in exons (Additional file <span class="InternalRef"><a href="#MOESM8">8</a></span>: Table S7). Overall, the motif imperfection mechanisms targeted tri- and hexa- repeats in genomic and coding sequences respectively (Additional file <span class="InternalRef"><a href="#MOESM9">9</a></span>: Figure S2).<figure xmlns="" id="Tab4" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 4</span>
                        <p xmlns="" class="SimplePara">Summary and characteristics of microsatellites detected in CDS region</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col char="."></col><col char="."></col><col char="."></col><col char="."></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Category</p></th><th char="."><p xmlns="" class="SimplePara">Garb<sup xmlns="http://www.w3.org/1999/xhtml">a</sup>
                                          </p></th><th char="."><p xmlns="" class="SimplePara">Grai<sup xmlns="http://www.w3.org/1999/xhtml">b</sup>
                                          </p></th><th char="."><p xmlns="" class="SimplePara">Ghir<sup xmlns="http://www.w3.org/1999/xhtml">c</sup>
                                          </p></th><th char="."><p xmlns="" class="SimplePara">Gbar<sup xmlns="http://www.w3.org/1999/xhtml">d</sup>
                                          </p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">Overall SSR density (per Mb)</p></td><td char="."><p xmlns="" class="SimplePara">65.12</p></td><td char="."><p xmlns="" class="SimplePara">61.70</p></td><td char="."><p xmlns="" class="SimplePara">69.48</p></td><td char="."><p xmlns="" class="SimplePara">57.15</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">3-nt repeat density (per Mb)</p></td><td char="."><p xmlns="" class="SimplePara">44.07</p></td><td char="."><p xmlns="" class="SimplePara">41.43</p></td><td char="."><p xmlns="" class="SimplePara">47.76</p></td><td char="."><p xmlns="" class="SimplePara">38.22</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">6-nt repeat density (per Mb)</p></td><td char="."><p xmlns="" class="SimplePara">15.95</p></td><td char="."><p xmlns="" class="SimplePara">15.09</p></td><td char="."><p xmlns="" class="SimplePara">17.28</p></td><td char="."><p xmlns="" class="SimplePara">14.99</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Average mismatch (Imperfection)</p></td><td char="."><p xmlns="" class="SimplePara">0.25</p></td><td char="."><p xmlns="" class="SimplePara">0.30</p></td><td char="."><p xmlns="" class="SimplePara">0.25</p></td><td char="."><p xmlns="" class="SimplePara">0.22</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Average repeat length (nt)</p></td><td char="."><p xmlns="" class="SimplePara">21.15</p></td><td char="."><p xmlns="" class="SimplePara">22.05</p></td><td char="."><p xmlns="" class="SimplePara">21.57</p></td><td char="."><p xmlns="" class="SimplePara">21.19</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Genes with tandem repeat in CDS (n)</p></td><td char="."><p xmlns="" class="SimplePara">2471</p></td><td char="."><p xmlns="" class="SimplePara">2846</p></td><td char="."><p xmlns="" class="SimplePara">4964</p></td><td char="."><p xmlns="" class="SimplePara">2992</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Genes with mismatched motif (n)</p></td><td char="."><p xmlns="" class="SimplePara">441</p></td><td char="."><p xmlns="" class="SimplePara">475</p></td><td char="."><p xmlns="" class="SimplePara">881</p></td><td char="."><p xmlns="" class="SimplePara">698</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Compound microsatellites (n)</p></td><td char="."><p xmlns="" class="SimplePara">71</p></td><td char="."><p xmlns="" class="SimplePara">114</p></td><td char="."><p xmlns="" class="SimplePara">133</p></td><td char="."><p xmlns="" class="SimplePara">202</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">
                                    <sup xmlns="http://www.w3.org/1999/xhtml">a</sup>
                                    <em class="EmphasisTypeItalic">G</em>. <em class="EmphasisTypeItalic">arboreum</em> (Garb), <sup xmlns="http://www.w3.org/1999/xhtml">b</sup>
                                    <em class="EmphasisTypeItalic">G</em>. <em class="EmphasisTypeItalic">raimondii</em> (Grai), <sup xmlns="http://www.w3.org/1999/xhtml">c</sup>
                                    <em class="EmphasisTypeItalic">G</em>. <em class="EmphasisTypeItalic">hirsutum</em> (Ghir) and <sup xmlns="http://www.w3.org/1999/xhtml">d</sup>
                                    <em class="EmphasisTypeItalic">G</em>. <em class="EmphasisTypeItalic">barbadence</em> (Gbar)</p>
                    </div></figure>
                        </div>
              </section><section id="Sec16" class="Section2 RenderAsSection2">
                <h3 class="Heading">Long motif repeats well conserved in <em class="EmphasisTypeItalic">Gossypium</em> species</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par39" class="Para">Our data showed predominant abundance of large microsatellites in <em xmlns="" class="EmphasisTypeItalic">Gossypium</em> species compared to other dicots and monocots. Microsatellite conservation pattern elucidated the possible influence of paleopolyploidization event and traced out the evolutionary footprints in <em xmlns="" class="EmphasisTypeItalic">Gossypium</em> species. The microsatellites proportion of each A<sub>T</sub> and D<sub>T</sub> sub-genome in two cultivated species was ascertained which retained from the progenitors (Fig. <span class="InternalRef"><a href="#Fig3">3</a></span>). The shorter repeats were found less conserved, while repeats with long motifs remained intact as in progenitor species (Additional file <span class="InternalRef"><a href="#MOESM10">10</a></span>: Table S8). Among short repeats, the 2-nt microsatellites were found more deteriorated in tetraploids where Gbar retained the lowest proportion from two diploids. On the contrary, 3–6&nbsp;nt repeats were more conserved, while 5-nt repeats were highly conserved (56.86–78.26%) in both tetraploids. Overall, Ghir retained higher proportion repeats from Garb (66.58%); while, slightly lower proportion retained (61.39%) from another progenitor. Since microsatellites were more deteriorated through series of evolutionary events in Gbar, thus fewer loci retained intact (37.83–43.99%).<figure xmlns="" class="Figure" id="Fig3"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO3">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0034-4/MediaObjects/41065_2017_34_Fig3_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0034-4/MediaObjects/41065_2017_34_Fig3_HTML.gif" alt="Fig. 3" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0034-4/MediaObjects/41065_2017_34_Fig3_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0034-4/MediaObjects/41065_2017_34_Fig3_HTML.gif" alt="Fig. 3"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 3</span>
                        <p xmlns="" class="SimplePara">Intuitive diagram shows conserved microsatellite proportion based on motif length. Motif repeats like 2-nt (dinucleotide), 3-nt (trinucleotide), 4-nt (tetra- nucleotide), 5-nt (penta- nucleotide) and 6-nt (hexa- nucleotide) were compared for conservation among Garb (<em class="EmphasisTypeItalic">G. arboreum</em>), Grai (<em class="EmphasisTypeItalic">G</em>. <em class="EmphasisTypeItalic">raimondii</em>), Ghir (<em class="EmphasisTypeItalic">G. hirsutum</em>) and Gbar (<em class="EmphasisTypeItalic">G</em>. <em class="EmphasisTypeItalic">barbadence</em>)</p>
                      </div></figcaption></figure>
                        </div>
              </section><section id="Sec17" class="Section2 RenderAsSection2">
                <h3 class="Heading">Estimation of microsatellite decay</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par40" class="Para">The conserved microsatellite proportion and divergence time were fitted to an exponential decay function and microsatellites decay rate of varied length motifs was calculated among <em xmlns="" class="EmphasisTypeItalic">Gossypium</em> species. A static decay was observed in Ghir for loci from both progenitors except 2-nt repeats; of which D<sub>T</sub> originated repeats were lost significantly faster (<em xmlns="" class="EmphasisTypeItalic">Friedman statistic</em>&nbsp;=&nbsp;48.44, <em xmlns="" class="EmphasisTypeItalic">p</em>&nbsp;=&nbsp;0.00001) (Additional file <span class="InternalRef"><a href="#MOESM11">11</a></span>: Figure S3a). Whereas in Gbar, 2-nt repeats of D<sub>T</sub> origin were highly distorted and lost due to significantly faster decay (<em xmlns="" class="EmphasisTypeItalic">Friedman statistic</em>&nbsp;=&nbsp;55.083, <em xmlns="" class="EmphasisTypeItalic">p</em>&nbsp;=&nbsp;0.00001) (Additional file <span class="InternalRef"><a href="#MOESM11">11</a></span>: Figure S3b)<em xmlns="" class="EmphasisTypeItalic">.</em> In comparison, Ghir retained higher proportion of microsatellites from diploid species than Gbar. Similarly, large motif repeats (3-6 nt) were more evolutionary stable and conserved compared to 2-nt repeats in both A-genome and D-genome of <em xmlns="" class="EmphasisTypeItalic">Gossypium</em> lineage. Overall, repeats loci in Ghir depicted slower decay of repeats than Gbar (Fig. <span class="InternalRef"><a href="#Fig4">4a</a></span>), while a significant faster decay (<em xmlns="" class="EmphasisTypeItalic">Friedman statistic</em>&nbsp;=&nbsp;18.762, <em xmlns="" class="EmphasisTypeItalic">p</em>&nbsp;=&nbsp;0.0021) of Grai repeats was observed in tetraploids compared to Garb microsatellites. Considering both “relative decay” and “comparative exponential decay”, all repeat motif exhibited faster decay in Gbar than Ghir (Fig. <span class="InternalRef"><a href="#Fig4">4b</a></span>).<figure xmlns="" class="Figure" id="Fig4"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO4">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0034-4/MediaObjects/41065_2017_34_Fig4_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0034-4/MediaObjects/41065_2017_34_Fig4_HTML.gif" alt="Fig. 4" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0034-4/MediaObjects/41065_2017_34_Fig4_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0034-4/MediaObjects/41065_2017_34_Fig4_HTML.gif" alt="Fig. 4"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 4</span>
                        <p xmlns="" class="SimplePara">Comprison of microsatellite decay between two cotton tetraploids compared to repeats originated in diploids (Garb and Grai). <strong class="EmphasisTypeBold">a</strong> Relative loss of all SSRs among A<sub xmlns="http://www.w3.org/1999/xhtml">T</sub> and D<sub xmlns="http://www.w3.org/1999/xhtml">T</sub> subgenomes of tetraploids (Ghir and Gbar). The <em class="EmphasisTypeItalic">black bar within box</em> represens median and <em class="EmphasisTypeItalic">dotted horizontal line</em> represent <em class="EmphasisTypeItalic">y</em>&nbsp;=&nbsp;0, while significance was denoted as <em class="EmphasisTypeItalic">lowercase letter</em>. <strong class="EmphasisTypeBold">b</strong> Comparing relationship of microsatellite relative decay and comparative exponential decay between Ghir and Gbar</p>
                      </div></figcaption></figure>
                        </div>
              </section><section id="Sec18" class="Section2 RenderAsSection2">
                <h3 class="Heading">Biased distribution of dinucleotide repeats in <em class="EmphasisTypeItalic">Gossypium</em> species</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par41" class="Para">A WGD event followed by allopolypolidization could be causative to observed evolutionary pattern of microsatellite conservation in <em xmlns="" class="EmphasisTypeItalic">Gossypium</em> tetraploids. The relative abundance estimates of four <em xmlns="" class="EmphasisTypeItalic">Gossypium</em> species’ repeats relative to <em xmlns="" class="EmphasisTypeItalic">T. cacao</em> (as of their closest phylogenetic ancestor) further substantiated the microsatellites decay pattern in <em xmlns="" class="EmphasisTypeItalic">Gossypium</em> lineage (Additional file <span class="InternalRef"><a href="#MOESM12">12</a></span>: Figure S4a-d). The short (1–3&nbsp;nt) repeats were found less abundant in Garb (<em xmlns="" class="EmphasisTypeItalic">Kruskal-Wallis statistic</em>&nbsp;=&nbsp;24.15, <em xmlns="" class="EmphasisTypeItalic">p</em>&nbsp;=&nbsp;0.0002) and Grai (<em xmlns="" class="EmphasisTypeItalic">Kruskal-Wallis statistic</em>&nbsp;=&nbsp;21.68, <em xmlns="" class="EmphasisTypeItalic">p</em>&nbsp;=&nbsp;0.0006), while only 1-nt and 2-nt were drastically lost in Ghir (<em xmlns="" class="EmphasisTypeItalic">Kruskal-Wallis statistic</em>&nbsp;=&nbsp;17.23, <em xmlns="" class="EmphasisTypeItalic">p</em>&nbsp;=&nbsp;0.0041) and Gbar (<em xmlns="" class="EmphasisTypeItalic">Kruskal-Wallis statistic</em>&nbsp;=&nbsp;17.84, <em xmlns="" class="EmphasisTypeItalic">p</em>&nbsp;=&nbsp;0.0031). While comparing relative abundance among cotton genomes, the short repeats (1-nt and 2-nt) prevailed less (<em xmlns="" class="EmphasisTypeItalic">Kruskal-Wallis statistic</em>&nbsp;=&nbsp;15.70, <em xmlns="" class="EmphasisTypeItalic">p</em>&nbsp;=&nbsp;0.0077) than longer (4-6 nt) ones (Fig. <span class="InternalRef"><a href="#Fig5">5a</a></span>). As decay rates were not determined for 1-nt repeats, thereby 2-nt repeat motif was pointed as scarcely abundant. Moreover, all 2-nt standardized motifs were lost in comparative way, while “AT” motif repeats were more rapidly lost in Ghir and Gbar (Fig. <span class="InternalRef"><a href="#Fig5">5b</a></span>). The ploidization could be a major factor for drastic loss of “AT” standardized motif as despite of ploidy increase, tetraploids conserved lower proportion of “AT” motif repeats than diploids.<figure xmlns="" class="Figure" id="Fig5"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO5">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0034-4/MediaObjects/41065_2017_34_Fig5_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0034-4/MediaObjects/41065_2017_34_Fig5_HTML.gif" alt="Fig. 5" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0034-4/MediaObjects/41065_2017_34_Fig5_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0034-4/MediaObjects/41065_2017_34_Fig5_HTML.gif" alt="Fig. 5"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 5</span>
                        <p xmlns="" class="SimplePara">Relative abundances of <em class="EmphasisTypeItalic">Gossypium</em> microsatellite over <em class="EmphasisTypeItalic">T. cacao</em> SSRs. Statistical significance was determined using pair-wise <em class="EmphasisTypeItalic">t-test</em> comparisons and small letters ‘a’, ‘b’ and ‘c’ were assigned to distinct groups significant at 0.05. The <em class="EmphasisTypeItalic">black bar</em> within box represens median and <em class="EmphasisTypeItalic">dotted horizontal line</em> represent <em class="EmphasisTypeItalic">y</em>&nbsp;=&nbsp;0. <strong class="EmphasisTypeBold">a</strong> Relative abundance of different motif repeats in Garb, Grai, Ghir and Gbar compared to abundance in <em class="EmphasisTypeItalic">T. cacao</em>. <strong class="EmphasisTypeBold">b</strong> Relative abundance of standardized 2-nt repeat motifs in Garb, Grai, Ghir and Gbar over all 2-nt repeat motifs in <em class="EmphasisTypeItalic">T. cacao</em>
                                    </p>
                      </div></figcaption></figure>
                        </div>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec19" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Discussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Discussion">Discussion</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Discussion" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par42" class="Para">The study provided a comprehensive account of microsatellite imperfection in 13 plant genomes and explored motif imperfection and repeat length relationships in <em xmlns="" class="EmphasisTypeItalic">Gossypium</em> lineage. The microsatellite conservation pattern and evolutionary footprints of non-coding repetitive DNA elements were also discerned in closely related species of Malvaceae family.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par43" class="Para">Genomic abundance of imperfect microsatellites is well studied in simple organisms like viruses [<span class="CitationRef"><a href="#CR31" data-toggle="citation">31</a></span>], mushroom [<span class="CitationRef"><a href="#CR32" data-toggle="citation">32</a></span>], complex animal [<span class="CitationRef"><a href="#CR33" data-toggle="citation">33</a></span>], insect [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>], human [<span class="CitationRef"><a href="#CR35" data-toggle="citation">35</a></span>] and even in green plants [<span class="CitationRef"><a href="#CR36" data-toggle="citation">36</a></span>]. However, a clear understanding of imperfection mechanism, its role in genesis, preserving sequence variations in non-coding repeat elements and their potential impact on gene regulation are not well understood. As focus on evolution of microsatellites got reincarnated with the advent of high throughput sequencing technology [<span class="CitationRef"><a href="#CR37" data-toggle="citation">37</a></span>, <span class="CitationRef"><a href="#CR38" data-toggle="citation">38</a></span>], this study emphasized on role of motif imperfection in microsatellites stability. Plant species were emphasized as microsatellites are being extensively employed in revealing diversity, heterozygosity and exploiting phenotypic variations in plants [<span class="CitationRef"><a href="#CR39" data-toggle="citation">39</a></span>, <span class="CitationRef"><a href="#CR40" data-toggle="citation">40</a></span>]. Moreover, microsatellites have also been employed to unravel polyploidy in plants [<span class="CitationRef"><a href="#CR41" data-toggle="citation">41</a></span>]. Evolutionary dynamics of microsatellites were revealed in genus <em xmlns="" class="EmphasisTypeItalic">Gossypium</em>, a best model to investigate ploidy increase impact on microsatellite imperfection and conservation.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par44" class="Para">Various tools are available for imperfect repeats search, but SciRoKo [<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>] was preferred since it provides a utility to efficiently digest large genome datasets into 50&nbsp;Mb chunks. Moreover, assessing efficiency of various tools was beyond the scope of this study. Our results witnessed predomination of 3-nt and 2-nt repeats in monocots and dicots respectively. Many researchers have reported 3-nt repeats’ abundance as characteristics in monocots [<span class="CitationRef"><a href="#CR42" data-toggle="citation">42</a></span>, <span class="CitationRef"><a href="#CR43" data-toggle="citation">43</a></span>]. Similarly, predominate abundance of 2-nt repeats has been reported in dicots [<span class="CitationRef"><a href="#CR44" data-toggle="citation">44</a></span>, <span class="CitationRef"><a href="#CR45" data-toggle="citation">45</a></span>], but paramount abundance of other motifs is not out of the scope. The resultant fluctuations might be owed to variations in search parameters. Appropriate parameters were employed for imperfect microsatellites search in all studied plant genomes (Table <span class="InternalRef"><a href="#Tab1">1</a></span> &amp; Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>: Table S1). Likewise, the distribution and abundance of imperfect repeats varied among the studied species, and it also varied among homologous chromosomes of species (Additional file <span class="InternalRef"><a href="#MOESM2">2</a></span>: Table S2). Previous studies supported varied distribution of imperfect repeats among different species of genus <em xmlns="" class="EmphasisTypeItalic">Drosophila</em> [<span class="CitationRef"><a href="#CR46" data-toggle="citation">46</a></span>]. The trend was not affected by varying quality of 13 genome assemblies as a consistent pattern of motif imperfection was observed.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par45" class="Para">The genomic abundance of imperfect microsatellites was intricately related to degree of motif imperfection and repeat length variations. Previously, repeat unit variations were found linked to repeat length and mutations [<span class="CitationRef"><a href="#CR47" data-toggle="citation">47</a></span>, <span class="CitationRef"><a href="#CR48" data-toggle="citation">48</a></span>], while an equilibrium state was proposed to regulate abundance of mismatched repeats [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>]. A varying stability of mismatched repeats due to sequence interruptions has been experimentally tested [<span class="CitationRef"><a href="#CR49" data-toggle="citation">49</a></span>]. While, a study determined relationship between mutation and repeat units and explained why longer repeats likely to undergo contractions and short repeats experience expansion in case of slippage mutations [<span class="CitationRef"><a href="#CR50" data-toggle="citation">50</a></span>]. The longer repeats (&gt;40&nbsp;nt) with no mismatch or lower imperfection (mismatch 1 and/or 2) were more abundant than those with higher degree of imperfection (Additional file <span class="InternalRef"><a href="#MOESM4">4</a></span>: Table S4a-b). Moreover, a significant relationship between mismatch count and repeat length existed among <em xmlns="" class="EmphasisTypeItalic">Gossypium</em> species (Additional file <span class="InternalRef"><a href="#MOESM5">5</a></span>: Table S5). Generally, the longer repeats do not remain stable and immediately split to give shorter repeats as these undergo contraction. However, our results predicted longer repeats were more likely to harbor mismatch, but these repeats were less prone to contraction as motif interruptions could impart stabilizing effect to such repeats [<span class="CitationRef"><a href="#CR51" data-toggle="citation">51</a></span>]. Furthermore, a low to moderate positive correlation between microsatellites and TEs was recurrently reported [<span class="CitationRef"><a href="#CR52" data-toggle="citation">52</a></span>, <span class="CitationRef"><a href="#CR53" data-toggle="citation">53</a></span>] but not between imperfect microsatellites and TEs. Due to sequence interruptions, the longer repeats were found more related to TEs abundance except for Garb, where perfect repeats were more likely to be present in the vicinity of TEs (Table <span class="InternalRef"><a href="#Tab3">3</a></span>). Thus, it can be speculated that microsatellite genesis, prolonged stability, length variations and motif imperfection of mismatched repeats could be better comprehended when the replication slippage errors, polyadenylation of 5′ and 3′ regions of TEs are considered simultaneously.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par46" class="Para">Recent studies reported that 5-nt repeats were predominated in Garb and Grai [<span class="CitationRef"><a href="#CR54" data-toggle="citation">54</a></span>, <span class="CitationRef"><a href="#CR55" data-toggle="citation">55</a></span>]. However, another study reported 6-nt repeats prevailed in Ghir [<span class="CitationRef"><a href="#CR56" data-toggle="citation">56</a></span>]; while our results affirmed that 5-nt repeats were consistently frequent in four sequenced <em xmlns="" class="EmphasisTypeItalic">Gossypium</em> genomes. Although the different search parameters generated variations, our results were in accordance to all previous reports. Previously, phylogenetic and molecular studies have suggested that genus <em xmlns="" class="EmphasisTypeItalic">Gossypium</em> diverged from close relatives during <em xmlns="" class="EmphasisTypeItalic">Miocene</em> about 10–15 MYA [<span class="CitationRef"><a href="#CR57" data-toggle="citation">57</a></span>]. The two progenitor diploids; <em xmlns="" class="EmphasisTypeItalic">G. arboreum</em> and <em xmlns="" class="EmphasisTypeItalic">G</em>. <em xmlns="" class="EmphasisTypeItalic">raimondii</em>, underwent a series of rapid speciation and diversification. While allopolyploids accomplished domestication and diversified distribution in mid <em xmlns="" class="EmphasisTypeItalic">Pleistocene</em> about 1–2 MYA [<span class="CitationRef"><a href="#CR58" data-toggle="citation">58</a></span>]. Proportion of conserved microsatellites strongly suggested rapid loss of shorter repeats (especially 2-nt repeats) and relative abundance analysis evoked preservation of higher repeats in four sequenced cotton species. It was noted that 5-nt repeats were the second more abundant after 2-nt repeats in <em xmlns="" class="EmphasisTypeItalic">T. cacao</em>. Thus, it could be speculated that through the series of evolutionary landmarks in <em xmlns="" class="EmphasisTypeItalic">Gossypium</em> evolution; WGD, allopolypolidization and speciation, 2-nt repeats were drastically lost and 5-nt repeats prevailed until <em xmlns="" class="EmphasisTypeItalic">Gossypium</em> early diploids evolved. The 2-nt repeat density was higher in cotton diploids than tetraploids, since the loss of shorter repeats could be a gradual process (Table <span class="InternalRef"><a href="#Tab1">1</a></span>).</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec20" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Conclusion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Conclusion">Conclusion</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Conclusion" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par47" class="Para">This study presented a comprehensive view of motif imperfection pattern in 13 genome assemblies representing five taxonomic families in plant kingdom. It determined the pattern of microsatellite motif imperfection and decay in complex higher plants exhibiting polyploidy and laid foundation for further studies; as plenty of plant genome sequences available to question evolutionary theory about origin of life. The results explicated evolution and stability of motif-interrupted repeats, its relationship with repeat length and its impact on their genomic prevalence in monocot and dicots. Moreover, it exploited clues to evolutionary footprints for predominate abundance of 5-nt repeats, which appeared as a characteristic in genus <em xmlns="" class="EmphasisTypeItalic">Gossypium</em>. Likewise, 2-nt repeats experienced accelerated decay in <em xmlns="" class="EmphasisTypeItalic">Gossypium</em> tetraploids. When compared with a close relative <em xmlns="" class="EmphasisTypeItalic">T. cacao</em>, “AT” repeats started being less abundant in geographic time while <em xmlns="" class="EmphasisTypeItalic">Gossypium</em> lineage was undergoing paleopolyploidization processes through the evolution of today’s domesticated cotton. Characterization of “AT” motif decay hotspots and the impact of motif decay on modulation of gene regulatory elements is mandatory to fully comprehend the basic underlying mechanisms.</p></div></section>
          <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AbbreviationGroup Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Abbreviations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Abbreviations">Abbreviations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Abbreviations"><div class="DefinitionList">
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>BLAST:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par4" class="Para">Basic local alignment search tool</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>CDS:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par5" class="Para">Coding sequences</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>DNA:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par6" class="Para">Deoxyribonucleic acid</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>Gb:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par7" class="Para">giga (10<sup>9</sup>) bases</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>Mb:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par8" class="Para">mega (10<sup>6</sup>) bases</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>mRNA:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par9" class="Para">messenger RNA</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>MYA:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par10" class="Para">Million years ago</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>nt:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par11" class="Para">nucleotide</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>SNP:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par12" class="Para">Single nucleotide polymorphisms</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>SSR:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par13" class="Para">Simple sequence repeats</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>TEs:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par14" class="Para">Transposable elements</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>WGD:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par15" class="Para">Whole genome duplication</p>
                  </dd></dl>
              </div></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments"><h3 xmlns="" class="Heading">Acknowledgements</h3><p xmlns="" class="SimplePara">We acknowledge the National Natural Science Foundation of China (Program #31371674) for providing financial support for this work.</p><div id="FPar1" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Funding</h3>
                <p id="Par48" class="Para">This work was financially supported by the National Natural Science Foundation of China (Program #31371674).</p>
              </div><div id="FPar2" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Availability of data and materials</h3>
                <p id="Par49" class="Para">We have provided detailed information in material and methods section of our manuscript.</p>
              </div><div id="FPar3" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Authors’ contributions</h3>
                <p id="Par50" class="Para">ZXL and MMA both conceived the idea and designed the study. MMA, SC and AQK performed data retrieval, handling and compilation. MMA, MAW and MS performed all analytical and statistical analyses. MMA and AQK designed and edited figures while MMA wrote manuscript and ZXL reviewed it. All authors contributed in editing and improving the manuscript. All authors read and approved the final manuscript.</p>
              </div><div id="FPar4" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Competing interests</h3>
                <p id="Par51" class="Para">The authors declare they have no competing interests.</p>
              </div><div id="FPar5" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Consent for publication</h3>
                <p id="Par52" class="Para">Not applicable.</p>
              </div><div id="FPar6" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Ethics approval and consent to participate</h3>
                <p id="Par53" class="Para">Not applicable.</p>
              </div><div id="FPar7" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Publisher’s Note</h3>
                <p id="Par54" class="Para">Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
              </div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="License">
                <p xmlns="" class="SimplePara">
                           <strong class="EmphasisTypeBold">Open Access</strong>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/4.0/"><span class="RefSource">http://creativecommons.org/licenses/by/4.0/</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/publicdomain/zero/1.0/"><span class="RefSource">http://creativecommons.org/publicdomain/zero/1.0/</span></a></span>) applies to the data made available in this article, unless otherwise stated.</p>
              </div><aside class="Appendix" id="App1">
              
                <h3 class="Heading">Additional files</h3><div id="widgetContainer" data-test="figshare-container"></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par55" class="Para">
                  <div class="MediaObject" id="MOESM1">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0034-4/MediaObjects/41065_2017_34_MOESM1_ESM.xls"><span class="caption-number">Additional file 1: Table S1.</span></a>
                          <span xmlns="" class="SimplePara">Characteristics of imperfect microsatellites investigated among 13 plant genomes. (XLS 25&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM2">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0034-4/MediaObjects/41065_2017_34_MOESM2_ESM.xls"><span class="caption-number">Additional file 2: Table S2.</span></a>
                          <span xmlns="" class="SimplePara">Frequency of imperfect microsatellites distributed chromosomes and imperfection (%) in 13 plant genomes. (XLS 39&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM3">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0034-4/MediaObjects/41065_2017_34_MOESM3_ESM.xls"><span class="caption-number">Additional file 3: Table S3.</span></a>
                          <span xmlns="" class="SimplePara">Average length comparison between perfect and imperfect repeats. (XLS 33&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM4">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0034-4/MediaObjects/41065_2017_34_MOESM4_ESM.xls"><span class="caption-number">Additional file 4: Table S4.</span></a>
                          <span xmlns="" class="SimplePara">
                                          <strong class="EmphasisTypeBold">(a)</strong> Frequency of variable length repeats which harbored mismatch (1, 2, 3 and 4) in repeat motifs across 13 plant genomes; <strong class="EmphasisTypeBold">(b)</strong> Probability values for pair-wise length comparisons among imperfect repeats of 13 plant genomes. (XLS 33&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM5">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0034-4/MediaObjects/41065_2017_34_MOESM5_ESM.xls"><span class="caption-number">Additional file 5: Table S5.</span></a>
                          <span xmlns="" class="SimplePara">Regression analysis of motif Imperfection (mismatch count) in relation to repeat length among four <em class="EmphasisTypeItalic">Gossypium</em> species. (XLS 24&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM6">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0034-4/MediaObjects/41065_2017_34_MOESM6_ESM.xls"><span class="caption-number">Additional file 6: Table S6.</span></a>
                          <span xmlns="" class="SimplePara">Chromosomal distribution of perfect and imperfect microsatellites in relation with intact (TEs), embedded in 500&nbsp;nt flanked region on both sides, among four <em class="EmphasisTypeItalic">Gossypium</em> species. (XLS 28&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM7">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0034-4/MediaObjects/41065_2017_34_MOESM7_ESM.doc"><span class="caption-number">Additional file 7: Figure S1.</span></a>
                          <span xmlns="" class="SimplePara">Comparison between perfect (no mismatch) and imperfect repeats (mismatch ≥1) for correlation of microsatellites with intact TEs frequency in (a) <em class="EmphasisTypeItalic">G. arboreum</em> (Garb), (b) <em class="EmphasisTypeItalic">G. raimondii</em> (Grai), (c) <em class="EmphasisTypeItalic">G. hirsutum</em> (Ghir) and (d) <em class="EmphasisTypeItalic">G. barbadence</em> (Gbar). (DOC 913&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM8">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0034-4/MediaObjects/41065_2017_34_MOESM8_ESM.xls"><span class="caption-number">Additional file 8: Table S7.</span></a>
                          <span xmlns="" class="SimplePara">Motif distribution, density and imperfection of microsatellite repeats in coding sequences of four cotton genomes. (XLS 25&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM9">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0034-4/MediaObjects/41065_2017_34_MOESM9_ESM.doc"><span class="caption-number">Additional file 9: Figure S2.</span></a>
                          <span xmlns="" class="SimplePara">Comparing motif imperfection pattern between genomic and coding microsatellites of varying motif sizes (2-6 nt) in <em class="EmphasisTypeItalic">G. arboreum</em> (Garb), <em class="EmphasisTypeItalic">G. raimondii</em> (Grai), <em class="EmphasisTypeItalic">G. hirsutum</em> (Ghir) and <em class="EmphasisTypeItalic">G. barbadence</em> (Gbar). (DOC 51&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM10">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0034-4/MediaObjects/41065_2017_34_MOESM10_ESM.xls"><span class="caption-number">Additional file 10: Table S8.</span></a>
                          <span xmlns="" class="SimplePara">Genome to sub-genome comparison of microsatellite conservation analysis. (XLS 28&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM11">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0034-4/MediaObjects/41065_2017_34_MOESM11_ESM.doc"><span class="caption-number">Additional file 11: Figure S3.</span></a>
                          <span xmlns="" class="SimplePara">Relative loss of SSRs by motif length in (a) <em class="EmphasisTypeItalic">G. hirsutum</em> (Ghir) and (b) <em class="EmphasisTypeItalic">G</em>. <em class="EmphasisTypeItalic">barbadence</em> (Gbar). The loss of 2-6 nt SSRs compared to the loss of all SSRs (y&nbsp;=&nbsp;0, denoted by dotted line). Microsatellites of sub-genome A<sub xmlns="http://www.w3.org/1999/xhtml">T</sub> are shown in gray filling and D<sub xmlns="http://www.w3.org/1999/xhtml">T</sub> sub-genome shown in white. (DOC 78&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM12">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0034-4/MediaObjects/41065_2017_34_MOESM12_ESM.doc"><span class="caption-number">Additional file 12: Figure S4.</span></a>
                          <span xmlns="" class="SimplePara">Relative abundance of microsatellite for <em class="EmphasisTypeItalic">Gossypium</em> genomes, (a) <em class="EmphasisTypeItalic">G. arboreum</em> (Garb), (b) <em class="EmphasisTypeItalic">G. raimondii</em> (Grai), (c) <em class="EmphasisTypeItalic">G. hirsutum</em> (Ghir), and (d) <em class="EmphasisTypeItalic">G. barbadence</em> (Gbar), compared to distribution of <em class="EmphasisTypeItalic">T. cacao</em> SSRs density by motif length (y&nbsp;=&nbsp;0, denoted by dotted line). (DOC 153&nbsp;kb)</span>
                        </div></div>
                  </div>
                </div>
              
            </aside></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authors’ Affiliations</a></h2><div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1"><span class="AffiliationNumber">(1)</span><div class="AffiliationText">National Key Laboratory of Crop Genetic Improvement, College of Plant Science &amp; Technology, Huazhong Agricultural University, Wuhan, Hubei, 430070, China</div></div></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper"><li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kruglyak S, Durrett RT, Schug MD, Aquadro CF. Equilibrium distributions of microsatellite repeat length resulting from a balance between slippage events and point mutations. Proc Natl Acad Sci U S A. 1998;95:10774–8.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Kruglyak%20S%2C%20Durrett%20RT%2C%20Schug%20MD%2C%20Aquadro%20CF.%20Equilibrium%20distributions%20of%20microsatellite%20repeat%20length%20resulting%20from%20a%20balance%20between%20slippage%20events%20and%20point%20mutations.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%201998%3B95%3A10774%E2%80%938."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hancock JM. Microsatellites and other simple sequences: genomic context and mutational mechanisms. In: Goldstein DB, Schlötterer C, editors. Microsatellites: evolution and applications. Oxford: Oxford University Press; 1999. p. 1–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Microsatellites%20and%20other%20simple%20sequences%3A%20genomic%20context%20and%20mutational%20mechanisms&amp;author=JM.%20Hancock&amp;pages=1-9&amp;publication_year=1999"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Echols H, Goodman MF. Fidelity mechanisms in DNA replication. Annu Rev Biochem. 1991;60:477–511.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1146/annurev.bi.60.070191.002401"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1883202"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Fidelity%20mechanisms%20in%20DNA%20replication&amp;author=H.%20Echols&amp;author=MF.%20Goodman&amp;journal=Annu%20Rev%20Biochem&amp;volume=60&amp;pages=477-511&amp;publication_year=1991"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Pray LA. DNA replication and causes of mutation. Nat Educ. 2008;1:214.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=DNA%20replication%20and%20causes%20of%20mutation&amp;author=LA.%20Pray&amp;journal=Nat%20Educ&amp;volume=1&amp;pages=214&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Schlötterer C, Tautz D. Slippage synthesis of simple sequence DNA. Nucleic Acids Res. 1992;20:211–5.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/20.2.211"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1741246"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC310356"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Slippage%20synthesis%20of%20simple%20sequence%20DNA&amp;author=C.%20Schl%C3%B6tterer&amp;author=D.%20Tautz&amp;journal=Nucleic%20Acids%20Res&amp;volume=20&amp;pages=211-215&amp;publication_year=1992"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kimura M, Ohta T. Stepwise mutation model and distribution of allelic frequencies in a finite population. Proc Natl Acad Sci U S A. 1978;75:2868–72.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Kimura%20M%2C%20Ohta%20T.%20Stepwise%20mutation%20model%20and%20distribution%20of%20allelic%20frequencies%20in%20a%20finite%20population.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%201978%3B75%3A2868%E2%80%9372."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Di Rienzo A, Peterson AC, Garza JC, Valdes AM, Slatkin M, Freimer NB. Mutational processes of simple-sequence repeat loci in human populations. Proc Natl Acad Sci U S A. 1994;91:3166–70.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Di%20Rienzo%20A%2C%20Peterson%20AC%2C%20Garza%20JC%2C%20Valdes%20AM%2C%20Slatkin%20M%2C%20Freimer%20NB.%20Mutational%20processes%20of%20simple-sequence%20repeat%20loci%20in%20human%20populations.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%201994%3B91%3A3166%E2%80%9370."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Nadir E, Margalit H, Gallily T, Ben-Sasson SA. Microsatellite spreading in the human genome: evolutionary mechanisms and structural implications. Proc Natl Acad Sci U S A. 1996;93:6470–5.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Nadir%20E%2C%20Margalit%20H%2C%20Gallily%20T%2C%20Ben-Sasson%20SA.%20Microsatellite%20spreading%20in%20the%20human%20genome%3A%20evolutionary%20mechanisms%20and%20structural%20implications.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%201996%3B93%3A6470%E2%80%935."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Tay WT, Behere GT, Batterham P, Heckel DG. Generation of microsatellite repeat families by RTE retrotransposons in lepidopteran genomes. BMC Evol Biol. 2010;10:144.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2148-10-144"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20470440"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2887409"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Generation%20of%20microsatellite%20repeat%20families%20by%20RTE%20retrotransposons%20in%20lepidopteran%20genomes&amp;author=WT.%20Tay&amp;author=GT.%20Behere&amp;author=P.%20Batterham&amp;author=DG.%20Heckel&amp;journal=BMC%20Evol%20Biol&amp;volume=10&amp;pages=144&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wang W, Bittles AH. Imperfect units of an extended microsatellite structure involving single nucleotide changes. Electrophoresis. 2001;22:1095–7.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1002/1522-2683()22%3A6<1095%3A%3AAID-ELPS1095&gt;3.0.CO%3B2-A"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11358131"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Imperfect%20units%20of%20an%20extended%20microsatellite%20structure%20involving%20single%20nucleotide%20changes&amp;author=W.%20Wang&amp;author=AH.%20Bittles&amp;journal=Electrophoresis&amp;volume=22&amp;pages=1095-1097&amp;publication_year=2001"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Schlotterer C. Evolutionary dynamics of microsatellite DNA. Chromosoma. 2000;109:365–71.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s004120000089"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11072791"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Evolutionary%20dynamics%20of%20microsatellite%20DNA&amp;author=C.%20Schlotterer&amp;journal=Chromosoma&amp;volume=109&amp;pages=365-371&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Blanquer-Maumont A, Crouau-Roy B. Polymorphism, monomorphism, and sequences in conserved microsatellites in primate species. J Mol Evol. 1995;41:492–7.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/BF00160321"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=7563137"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Polymorphism%2C%20monomorphism%2C%20and%20sequences%20in%20conserved%20microsatellites%20in%20primate%20species&amp;author=A.%20Blanquer-Maumont&amp;author=B.%20Crouau-Roy&amp;journal=J%20Mol%20Evol&amp;volume=41&amp;pages=492-497&amp;publication_year=1995"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ezenwa VO, Peters JM, Zhu Y, Arévalo E, Hastings MD, Seppä P, et al. Ancient conservation of Trinucleotide microsatellite loci in Polistine wasps. Mol Phylogenet Evol. 1998;10:168–77.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1006/mpev.1998.0528"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9878228"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Ancient%20conservation%20of%20Trinucleotide%20microsatellite%20loci%20in%20Polistine%20wasps&amp;author=VO.%20Ezenwa&amp;author=JM.%20Peters&amp;author=Y.%20Zhu&amp;author=E.%20Ar%C3%A9valo&amp;author=MD.%20Hastings&amp;author=P.%20Sepp%C3%A4&amp;journal=Mol%20Phylogenet%20Evol&amp;volume=10&amp;pages=168-177&amp;publication_year=1998"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Adams RH, Blackmon H, Reyes-Velasco J, Schield DR, Card DC, Andrew AL, et al. Microsatellite landscape evolutionary dynamics across 450 million years of vertebrate genome evolution. Genome. 2016;59:295–310.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1139/gen-2015-0124"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27064176"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Microsatellite%20landscape%20evolutionary%20dynamics%20across%20450%20million%20years%20of%20vertebrate%20genome%20evolution&amp;author=RH.%20Adams&amp;author=H.%20Blackmon&amp;author=J.%20Reyes-Velasco&amp;author=DR.%20Schield&amp;author=DC.%20Card&amp;author=AL.%20Andrew&amp;journal=Genome&amp;volume=59&amp;pages=295-310&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ranade SS, Lin YC, Van de Peer Y, Garcia-Gil MR. Comparative in silico analysis of SSRs in coding regions of high confidence predicted genes in Norway spruce (<em class="EmphasisTypeItalic">Picea abies</em>) and loblolly pine (<em class="EmphasisTypeItalic">Pinus taeda</em>). BMC Genet. 2015;16:149.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/s12863-015-0304-y"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26706685"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4691297"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparative%20in%20silico%20analysis%20of%20SSRs%20in%20coding%20regions%20of%20high%20confidence%20predicted%20genes%20in%20Norway%20spruce%20%28Picea%20abies%29%20and%20loblolly%20pine%20%28Pinus%20taeda%29&amp;author=SS.%20Ranade&amp;author=YC.%20Lin&amp;author=Y.%20Van%20de%20Peer&amp;author=MR.%20Garcia-Gil&amp;journal=BMC%20Genet&amp;volume=16&amp;pages=149&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wang Q, Zhang X, Wang X, Zeng B, Jia X, Hou R, et al. Polymorphism of CAG repeats in androgen receptor of carnivores. Mol Biol Rep. 2012;39:2297–303.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11033-011-0979-8"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21643744"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Polymorphism%20of%20CAG%20repeats%20in%20androgen%20receptor%20of%20carnivores&amp;author=Q.%20Wang&amp;author=X.%20Zhang&amp;author=X.%20Wang&amp;author=B.%20Zeng&amp;author=X.%20Jia&amp;author=R.%20Hou&amp;journal=Mol%20Biol%20Rep&amp;volume=39&amp;pages=2297-2303&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">CottonGen. <span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://www.cottongen.org"><span class="RefSource">https://www.cottongen.org</span></a></span>/. 15 June 2016.</span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">NCBI Genome Portal. <span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://www.ncbi.nlm.nih.gov/genome/?term"><span class="RefSource">https://www.ncbi.nlm.nih.gov/genome/?term</span></a></span>. 15 June 2016.</span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kofler R, Schlotterer C, Lelley T. SciRoKo: a new tool for whole genome microsatellite search and investigation. Bioinformatics. 2007;23:1683–5.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btm157"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17463017"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=SciRoKo%3A%20a%20new%20tool%20for%20whole%20genome%20microsatellite%20search%20and%20investigation&amp;author=R.%20Kofler&amp;author=C.%20Schlotterer&amp;author=T.%20Lelley&amp;journal=Bioinformatics&amp;volume=23&amp;pages=1683-1685&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Behura SK, Severson DW. Motif mismatches in microsatellites: insights from genome-wide investigation among 20 insect species. DNA Res. 2015;22:29–38.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/dnares/dsu036"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25378245"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Motif%20mismatches%20in%20microsatellites%3A%20insights%20from%20genome-wide%20investigation%20among%2020%20insect%20species&amp;author=SK.%20Behura&amp;author=DW.%20Severson&amp;journal=DNA%20Res&amp;volume=22&amp;pages=29-38&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Anderson MJ, Robinson J. Generalized discriminant analysis based on distances. Aust NZ J Stat. 2003;45:301–18.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/1467-842X.00285"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Generalized%20discriminant%20analysis%20based%20on%20distances&amp;author=MJ.%20Anderson&amp;author=J.%20Robinson&amp;journal=Aust%20NZ%20J%20Stat&amp;volume=45&amp;pages=301-318&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Canonical Analysis of Principal Coordinates (CAP). <span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://www.esapubs.org/archive/ecol/E084/011/suppl-1.htm"><span class="RefSource">http://www.esapubs.org/archive/ecol/E084/011/suppl-1.htm</span></a></span>. 10 May 2016.</span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">The R project for statistical computing. <span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://www.r-project.org"><span class="RefSource">https://www.r-project.org</span></a></span>/. 10 May 2016.</span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR24" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">CCP: Significance tests for Canonical Correlation Analysis (CCA). <span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://cran.r-project.org/web/packages/CCP/index.html"><span class="RefSource">https://cran.r-project.org/web/packages/CCP/index.html</span></a></span>. 25 May 2016.</span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR25" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Anderson MJ. A new method for non-parametric multivariate analysis of variance. Aust Ecol. 2001;26:32–46.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20new%20method%20for%20non-parametric%20multivariate%20analysis%20of%20variance&amp;author=MJ.%20Anderson&amp;journal=Aust%20Ecol&amp;volume=26&amp;pages=32-46&amp;publication_year=2001"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR26" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">vegan: Community Ecology package. <span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://cran.r-project.org/web/packages/vegan/index.html"><span class="RefSource">https://cran.r-project.org/web/packages/vegan/index.html</span></a></span>. 25 May 2016.</span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR27" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ. Basic local alignment search tool. J Mol Biol. 1990;215:403–10.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/S0022-2836(05)80360-2"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=2231712"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Basic%20local%20alignment%20search%20tool&amp;author=SF.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=EW.%20Myers&amp;author=DJ.%20Lipman&amp;journal=J%20Mol%20Biol&amp;volume=215&amp;pages=403-410&amp;publication_year=1990"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR28" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li FG, Fan GY, Lu CR, Xiao GH, Zou CS, Kohel RJ, et al. Genome sequence of cultivated upland cotton (<em class="EmphasisTypeItalic">Gossypium hirsutum</em> TM-1) provides insights into genome evolution. Nat Biotechnol. 2015;33:524–U242.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nbt.3208"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25893780"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20cultivated%20upland%20cotton%20%28Gossypium%20hirsutum%20TM-1%29%20provides%20insights%20into%20genome%20evolution&amp;author=FG.%20Li&amp;author=GY.%20Fan&amp;author=CR.%20Lu&amp;author=GH.%20Xiao&amp;author=CS.%20Zou&amp;author=RJ.%20Kohel&amp;journal=Nat%20Biotechnol&amp;volume=33&amp;pages=524-U242&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR29" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Circos: Circular Visualization. <span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://circos.ca"><span class="RefSource">http://circos.ca</span></a></span>/. 20 July 2016.</span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR30" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wendel JF, Grover CE. Taxonomy and evolution of the cotton genus. In: Fang D, Percy R, editors. Cotton, Agronomy. Madison: Monograph 24, ASA-CSSA-SSSA; 2015.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Taxonomy%20and%20evolution%20of%20the%20cotton%20genus&amp;author=JF.%20Wendel&amp;author=CE.%20Grover&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR31" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Alam CM, Singh AK, Sharfuddin C, Ali S. Genome-wide scan for analysis of simple and imperfect microsatellites in diverse carlaviruses. Infect Genet Evol. 2014;21:287–94.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.meegid.2013.11.018"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24291012"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genome-wide%20scan%20for%20analysis%20of%20simple%20and%20imperfect%20microsatellites%20in%20diverse%20carlaviruses&amp;author=CM.%20Alam&amp;author=AK.%20Singh&amp;author=C.%20Sharfuddin&amp;author=S.%20Ali&amp;journal=Infect%20Genet%20Evol&amp;volume=21&amp;pages=287-294&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR32" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Keirle MR, Avis PG, Feldheim KA, Hemmes DE, Mueller GM. Investigating the allelic evolution of an imperfect microsatellite locus in the Hawaiian mushroom Rhodocollybia laulaha. J Hered. 2011;102:727–34.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/jhered/esr099"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21914665"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Investigating%20the%20allelic%20evolution%20of%20an%20imperfect%20microsatellite%20locus%20in%20the%20Hawaiian%20mushroom%20Rhodocollybia%20laulaha&amp;author=MR.%20Keirle&amp;author=PG.%20Avis&amp;author=KA.%20Feldheim&amp;author=DE.%20Hemmes&amp;author=GM.%20Mueller&amp;journal=J%20Hered&amp;volume=102&amp;pages=727-734&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR33" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gaspari Z, Ortutay C, Toth G. Divergent microsatellite evolution in the human and chimpanzee lineages. FEBS Lett. 2007;581:2523–6.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.febslet.2007.04.073"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17498704"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Divergent%20microsatellite%20evolution%20in%20the%20human%20and%20chimpanzee%20lineages&amp;author=Z.%20Gaspari&amp;author=C.%20Ortutay&amp;author=G.%20Toth&amp;journal=FEBS%20Lett&amp;volume=581&amp;pages=2523-2526&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR34" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Stolle E, Kidner JH, Moritz RF. Patterns of evolutionary conservation of microsatellites (SSRs) suggest a faster rate of genome evolution in hymenoptera than in Diptera. Genome Biol Evol. 2013;5:151–62.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/gbe/evs133"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23292136"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3595035"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Patterns%20of%20evolutionary%20conservation%20of%20microsatellites%20%28SSRs%29%20suggest%20a%20faster%20rate%20of%20genome%20evolution%20in%20hymenoptera%20than%20in%20Diptera&amp;author=E.%20Stolle&amp;author=JH.%20Kidner&amp;author=RF.%20Moritz&amp;journal=Genome%20Biol%20Evol&amp;volume=5&amp;pages=151-162&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR35" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sun JX, Helgason A, Masson G, Ebenesersdottir SS, Li H, Mallick S, et al. A direct characterization of human mutation based on microsatellites. Nat Genet. 2012;44:1161–5.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/ng.2398"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22922873"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3459271"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20direct%20characterization%20of%20human%20mutation%20based%20on%20microsatellites&amp;author=JX.%20Sun&amp;author=A.%20Helgason&amp;author=G.%20Masson&amp;author=SS.%20Ebenesersdottir&amp;author=H.%20Li&amp;author=S.%20Mallick&amp;journal=Nat%20Genet&amp;volume=44&amp;pages=1161-1165&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR36" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kapil A, Rai PK, Shanker A. ChloroSSRdb: a repository of perfect and imperfect chloroplastic simple sequence repeats (cpSSRs) of green plants. Database. 2014;2014.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Kapil%20A%2C%20Rai%20PK%2C%20Shanker%20A.%20ChloroSSRdb%3A%20a%20repository%20of%20perfect%20and%20imperfect%20chloroplastic%20simple%20sequence%20repeats%20%28cpSSRs%29%20of%20green%20plants.%20Database.%202014%3B2014."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR37" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yun YE, Yu JN, Nam GH, Ryu SA, Kim S, Oh K, et al. Next-generation sequencing identification and characterization of microsatellite markers in aconitum austrokoreense Koidz., an endemic and endangered medicinal plant of Korea. Genet Mol Res. 2015;14:4812–7.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.4238/2015.May.11.13"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25966255"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Next-generation%20sequencing%20identification%20and%20characterization%20of%20microsatellite%20markers%20in%20aconitum%20austrokoreense%20Koidz.%2C%20an%20endemic%20and%20endangered%20medicinal%20plant%20of%20Korea&amp;author=YE.%20Yun&amp;author=JN.%20Yu&amp;author=GH.%20Nam&amp;author=SA.%20Ryu&amp;author=S.%20Kim&amp;author=K.%20Oh&amp;journal=Genet%20Mol%20Res&amp;volume=14&amp;pages=4812-4817&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR38" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kang TH, Han SH, Park SJ. Development of seven microsatellite markers using next generation sequencing for the conservation on the Korean population of Dorcus Hopei (E. Saunders, 1854) (Coleoptera, Lucanidae). Int J Mol Sci. 2015;16:21330–41.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.3390/ijms160921330"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26370965"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4613255"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Development%20of%20seven%20microsatellite%20markers%20using%20next%20generation%20sequencing%20for%20the%20conservation%20on%20the%20Korean%20population%20of%20Dorcus%20Hopei%20%28E.%20Saunders%2C%201854%29%20%28Coleoptera%2C%20Lucanidae%29&amp;author=TH.%20Kang&amp;author=SH.%20Han&amp;author=SJ.%20Park&amp;journal=Int%20J%20Mol%20Sci&amp;volume=16&amp;pages=21330-21341&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR39" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Han B, Wang C, Tang Z, Ren Y, Li Y, Zhang D, et al. Genome-wide analysis of microsatellite markers based on sequenced database in Chinese spring wheat (<em class="EmphasisTypeItalic">Triticum aestivum</em> L.). PLoS One. 2015;10:e0141540.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0141540"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26536014"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4633229"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20microsatellite%20markers%20based%20on%20sequenced%20database%20in%20Chinese%20spring%20wheat%20%28Triticum%20aestivum%20L.%29&amp;author=B.%20Han&amp;author=C.%20Wang&amp;author=Z.%20Tang&amp;author=Y.%20Ren&amp;author=Y.%20Li&amp;author=D.%20Zhang&amp;journal=PLoS%20One&amp;volume=10&amp;pages=e0141540&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR40" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ahmed M, Guo H, Huang C, Zhang X, Lin Z. Selection of core SSR markers for fingerprinting upland cotton cultivars and hybrids. Aust J Crop Sci. 2013;7:1912–20.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Selection%20of%20core%20SSR%20markers%20for%20fingerprinting%20upland%20cotton%20cultivars%20and%20hybrids&amp;author=M.%20Ahmed&amp;author=H.%20Guo&amp;author=C.%20Huang&amp;author=X.%20Zhang&amp;author=Z.%20Lin&amp;journal=Aust%20J%20Crop%20Sci&amp;volume=7&amp;pages=1912-1920&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR41" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li X, Jin X, Wang H, Zhang X, Lin Z. Structure, evolution, and comparative genomics of tetraploid cotton based on a high-density genetic linkage map. DNA Res. 2016;23:283–93.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/dnares/dsw016"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27084896"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4909315"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Structure%2C%20evolution%2C%20and%20comparative%20genomics%20of%20tetraploid%20cotton%20based%20on%20a%20high-density%20genetic%20linkage%20map&amp;author=X.%20Li&amp;author=X.%20Jin&amp;author=H.%20Wang&amp;author=X.%20Zhang&amp;author=Z.%20Lin&amp;journal=DNA%20Res&amp;volume=23&amp;pages=283-293&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR42" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kalyana Babu B, Pandey D, Agrawal PK, Sood S, Kumar A. In-silico mining, type and frequency analysis of genic microsatellites of finger millet (<em class="EmphasisTypeItalic">Eleusine coracana</em> (L.) Gaertn.): a comparative genomic analysis of NBS-LRR regions of finger millet with rice. Mol Biol Rep. 2014;41:3081–90.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11033-014-3168-8"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24477586"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=In-silico%20mining%2C%20type%20and%20frequency%20analysis%20of%20genic%20microsatellites%20of%20finger%20millet%20%28Eleusine%20coracana%20%28L.%29%20Gaertn.%29%3A%20a%20comparative%20genomic%20analysis%20of%20NBS-LRR%20regions%20of%20finger%20millet%20with%20rice&amp;author=B.%20Kalyana%20Babu&amp;author=D.%20Pandey&amp;author=PK.%20Agrawal&amp;author=S.%20Sood&amp;author=A.%20Kumar&amp;journal=Mol%20Biol%20Rep&amp;volume=41&amp;pages=3081-3090&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR43" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Shi J, Huang S, Fu D, Yu J, Wang X, Hua W, et al. Evolutionary dynamics of microsatellite distribution in plants: insight from the comparison of sequenced brassica, Arabidopsis and other angiosperm species. PLoS One. 2013;8:e59988.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0059988"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23555856"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3610691"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Evolutionary%20dynamics%20of%20microsatellite%20distribution%20in%20plants%3A%20insight%20from%20the%20comparison%20of%20sequenced%20brassica%2C%20Arabidopsis%20and%20other%20angiosperm%20species&amp;author=J.%20Shi&amp;author=S.%20Huang&amp;author=D.%20Fu&amp;author=J.%20Yu&amp;author=X.%20Wang&amp;author=W.%20Hua&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e59988&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR44" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Liu SR, Li WY, Long D, Hu CG, Zhang JZ. Development and characterization of genomic and expressed SSRs in citrus by genome-wide analysis. PLoS One. 2013;8:e75149.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0075149"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24204572"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3810390"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Development%20and%20characterization%20of%20genomic%20and%20expressed%20SSRs%20in%20citrus%20by%20genome-wide%20analysis&amp;author=SR.%20Liu&amp;author=WY.%20Li&amp;author=D.%20Long&amp;author=CG.%20Hu&amp;author=JZ.%20Zhang&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e75149&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR45" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Biswas MK, Xu Q, Mayer C, Deng X. Genome wide characterization of short tandem repeat markers in sweet orange (<em class="EmphasisTypeItalic">Citrus sinensis</em>). PLoS One. 2014;9:e104182.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0104182"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25148383"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4141690"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genome%20wide%20characterization%20of%20short%20tandem%20repeat%20markers%20in%20sweet%20orange%20%28Citrus%20sinensis%29&amp;author=MK.%20Biswas&amp;author=Q.%20Xu&amp;author=C.%20Mayer&amp;author=X.%20Deng&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e104182&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR46" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ross CL, Dyer KA, Erez T, Miller SJ, Jaenike J, Markow TA. Rapid divergence of microsatellite abundance among species of drosophila. Mol Biol Evol. 2003;20:1143–57.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/molbev/msg137"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12777536"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Rapid%20divergence%20of%20microsatellite%20abundance%20among%20species%20of%20drosophila&amp;author=CL.%20Ross&amp;author=KA.%20Dyer&amp;author=T.%20Erez&amp;author=SJ.%20Miller&amp;author=J.%20Jaenike&amp;author=TA.%20Markow&amp;journal=Mol%20Biol%20Evol&amp;volume=20&amp;pages=1143-1157&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR47" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Xu X, Peng M, Fang Z. The direction of microsatellite mutations is dependent upon allele length. Nat Genet. 2000;24:396–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/74238"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10742105"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20direction%20of%20microsatellite%20mutations%20is%20dependent%20upon%20allele%20length&amp;author=X.%20Xu&amp;author=M.%20Peng&amp;author=Z.%20Fang&amp;journal=Nat%20Genet&amp;volume=24&amp;pages=396-399&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR48" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Falush D, Iwasa Y. Size-dependent mutability and microsatellite constraints. Mol Biol Evol. 1999;16:960–6.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/oxfordjournals.molbev.a026185"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Size-dependent%20mutability%20and%20microsatellite%20constraints&amp;author=D.%20Falush&amp;author=Y.%20Iwasa&amp;journal=Mol%20Biol%20Evol&amp;volume=16&amp;pages=960-966&amp;publication_year=1999"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR49" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Bacon AL, Farrington SM, Dunlop MG. Sequence interruptions confer differential stability at microsatellite alleles in mismatch repair-deficient cells. Hum Mol Genet. 2000;9:2707–13.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/hmg/9.18.2707"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11063729"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Sequence%20interruptions%20confer%20differential%20stability%20at%20microsatellite%20alleles%20in%20mismatch%20repair-deficient%20cells&amp;author=AL.%20Bacon&amp;author=SM.%20Farrington&amp;author=MG.%20Dunlop&amp;journal=Hum%20Mol%20Genet&amp;volume=9&amp;pages=2707-2713&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR50" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lai Y, Sun F. The relationship between microsatellite slippage mutation rate and the number of repeat units. Mol Biol Evol. 2003;20:2123–31.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/molbev/msg228"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12949124"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20relationship%20between%20microsatellite%20slippage%20mutation%20rate%20and%20the%20number%20of%20repeat%20units&amp;author=Y.%20Lai&amp;author=F.%20Sun&amp;journal=Mol%20Biol%20Evol&amp;volume=20&amp;pages=2123-2131&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR51" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Rolfsmeier ML, Lahue RS. Stabilizing effects of interruptions on Trinucleotide repeat expansions in <em class="EmphasisTypeItalic">Saccharomyces cerevisiae</em>. Mol Cell Biol. 2000;20:173–80.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1128/MCB.20.1.173-180.2000"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10594019"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC85072"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Stabilizing%20effects%20of%20interruptions%20on%20Trinucleotide%20repeat%20expansions%20in%20Saccharomyces%20cerevisiae%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=ML.%20Rolfsmeier&amp;author=RS.%20Lahue&amp;journal=Mol%20Cell%20Biol&amp;volume=20&amp;pages=173-180&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR52" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ramsay L, Macaulay M, Cardle L, Morgante M, degli Ivanissevich S, Maestri E, et al. Intimate association of microsatellite repeats with retrotransposons and other dispersed repetitive elements in barley. Plant J. 1999;17:415–25.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1046/j.1365-313X.1999.00392.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10205898"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Intimate%20association%20of%20microsatellite%20repeats%20with%20retrotransposons%20and%20other%20dispersed%20repetitive%20elements%20in%20barley&amp;author=L.%20Ramsay&amp;author=M.%20Macaulay&amp;author=L.%20Cardle&amp;author=M.%20Morgante&amp;author=S.%20degli%20Ivanissevich&amp;author=E.%20Maestri&amp;journal=Plant%20J&amp;volume=17&amp;pages=415-425&amp;publication_year=1999"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR53" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Grandi FC, An W. Non-LTR retrotransposons and microsatellites: partners in genomic variation. Mob Genet Elements. 2013;3:e25674.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.4161/mge.25674"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24195012"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3812793"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Non-LTR%20retrotransposons%20and%20microsatellites%3A%20partners%20in%20genomic%20variation&amp;author=FC.%20Grandi&amp;author=W.%20An&amp;journal=Mob%20Genet%20Elements&amp;volume=3&amp;pages=e25674&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR54" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lu C, Zou C, Zhang Y, Yu D, Cheng H, Jiang P, et al. Development of chromosome-specific markers with high polymorphism for allotetraploid cotton based on genome-wide characterization of simple sequence repeats in diploid cottons (<em class="EmphasisTypeItalic">Gossypium arboreum</em> L. and <em class="EmphasisTypeItalic">Gossypium raimondii</em> Ulbrich). BMC Genomics. 2015;16:55.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/s12864-015-1265-2"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25652321"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4325953"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Development%20of%20chromosome-specific%20markers%20with%20high%20polymorphism%20for%20allotetraploid%20cotton%20based%20on%20genome-wide%20characterization%20of%20simple%20sequence%20repeats%20in%20diploid%20cottons%20%28Gossypium%20arboreum%20L.%20and%20Gossypium%20raimondii%20Ulbrich&amp;author=C.%20Lu&amp;author=C.%20Zou&amp;author=Y.%20Zhang&amp;author=D.%20Yu&amp;author=H.%20Cheng&amp;author=P.%20Jiang&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=55&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR55" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zou C, Lu C, Zhang Y, Song G. Distribution and characterization of simple sequence repeats in <em class="EmphasisTypeItalic">Gossypium raimondii</em> genome. Bioinformation. 2012;8:801–6.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.6026/97320630008801"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23139588"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3488841"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Distribution%20and%20characterization%20of%20simple%20sequence%20repeats%20in%20Gossypium%20raimondii%20genome&amp;author=C.%20Zou&amp;author=C.%20Lu&amp;author=Y.%20Zhang&amp;author=G.%20Song&amp;journal=Bioinformation&amp;volume=8&amp;pages=801-806&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR56" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wang Q, Fang L, Chen J, Hu Y, Si Z, Wang S, et al. Genome-wide mining, characterization, and development of microsatellite markers in <em class="EmphasisTypeItalic">Gossypium</em> species. Sci Rep. 2015;5:10638.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/srep10638"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26030481"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4650602"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genome-wide%20mining%2C%20characterization%2C%20and%20development%20of%20microsatellite%20markers%20in%20Gossypium%20species&amp;author=Q.%20Wang&amp;author=L.%20Fang&amp;author=J.%20Chen&amp;author=Y.%20Hu&amp;author=Z.%20Si&amp;author=S.%20Wang&amp;journal=Sci%20Rep&amp;volume=5&amp;pages=10638&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR57" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Cronn RC, Small RL, Haselkorn T, Wendel JF. Rapid diversification of the cotton genus (<em class="EmphasisTypeItalic">Gossypium</em>: Malvaceae) revealed by analysis of sixteen nuclear and chloroplast genes. Am J Bot. 2002;89:707–25.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.3732/ajb.89.4.707"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21665671"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Rapid%20diversification%20of%20the%20cotton%20genus%20%28Gossypium%3A%20Malvaceae%29%20revealed%20by%20analysis%20of%20sixteen%20nuclear%20and%20chloroplast%20genes&amp;author=RC.%20Cronn&amp;author=RL.%20Small&amp;author=T.%20Haselkorn&amp;author=JF.%20Wendel&amp;journal=Am%20J%20Bot&amp;volume=89&amp;pages=707-725&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR58" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Flagel LE, Wendel JF, Udall JA. Duplicate gene evolution, homoeologous recombination, and transcriptome characterization in allopolyploid cotton. BMC Genomics. 2012;13:1–13.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2164-13-302"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Duplicate%20gene%20evolution%2C%20homoeologous%20recombination%2C%20and%20transcriptome%20characterization%20in%20allopolyploid%20cotton&amp;author=LE.%20Flagel&amp;author=JF.%20Wendel&amp;author=JA.%20Udall&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=1-13&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li></ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#TheAuthors" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="TheAuthors">Copyright</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="TheAuthors"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">©&nbsp;The Author(s)&nbsp;2017</div></div></section></div>
                        


                    </article>
                </main>

                <div class="c-content-layout__side u-text-sm">
                    <aside>
                        <div class="p-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="//hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-017-0034-4" data-track="click" data-track-category="PDF Download"
            data-track-action="Click download article PDF"
            data-track-label="10.1186/s41065-017-0034-4">Download PDF</a>
        </p>
    
                            
                            
    <button type="button"
       class="c-button c-button--secondary c-button--full-width c-button--expandable"
       data-toggle="collapse"
       data-target="export-citations"
       data-test="export-citations-button"
       data-sticky-update="true"
       aria-controls="export-citations"
       aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/s41065-017-0034-4"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0034-4?format=refman&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/s41065-017-0034-4"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0034-4?format=refman&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/s41065-017-0034-4"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0034-4?format=refman&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/s41065-017-0034-4"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0034-4?format=endnote&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/s41065-017-0034-4"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0034-4?format=endnote&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/s41065-017-0034-4"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0034-4?format=endnote&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/s41065-017-0034-4"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0034-4?format=bibtex&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/s41065-017-0034-4"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0034-4?format=bibtex&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/s41065-017-0034-4"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0034-4?format=bibtex&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
    </div>

                        </div>
                        
    
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
                                <div id="article-access-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/access-count/10.1186/s41065-017-0034-4 #accessCount-container"></div>
                                <div id="citation-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/citation-count/10.1186/s41065-017-0034-4 #count-container"></div>
                                <div id="altmetric-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/altmetric/10.1186/s41065-017-0034-4 #altmetric-container"></div>
                            </ul>
                        </div>
                        
    <div id="sociaMediaShare" class="p-separator u-js-shown">
        <h3 class="p-section-title">Share this article</h3>
        <ul id="shareButtons" class="c-social-links">
            <li class="c-social-links__item">
                <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-017-0034-4">
                    <span class="u-visually-hidden">Share on Twitter</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-twitter"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-017-0034-4">
                    <span class="u-visually-hidden">Share on Facebook</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-facebook"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click"
                        data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-017-0034-4">
                    <span class="u-visually-hidden">Share on LinkedIn</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-linkedin"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-017-0034-4">
                    <span class="u-visually-hidden">Share on Weibo</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-weibo"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnGooglePlus" class="js-btnShareOnGplus" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Google+" data-track-label="10.1186/s41065-017-0034-4">
                    <span class="u-visually-hidden">Share on Google Plus</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-gplus"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-017-0034-4">
                    <span class="u-visually-hidden">Share on Reddit</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-reddit"></use>
                    </svg>
                </button>
            </li>
        </ul>
    </div>
 
                        
    
        <div class="p-separator u-js-shown">
            <h3 class="p-section-title">See updates</h3>
            <button class="c-crossmark-button" data-target="crossmark" data-test="crossmark" data-track="click" data-track-category="Article Page" data-track-action="Click CrossMark" data-track-label="10.1186/s41065-017-0034-4">
                <img id="crossmark-icon" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" alt="Crossmark logo" />
            </button>
        </div>
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
                                <li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/s41065-017-0034-4" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=738&doi=10.1186%2Fs41065-017-0034-4&volume=154&issue=1&title=A+comparative+genomics+approach+revealed+evolutionary+dynamics+of+microsatellite+imperfection+and+conservation+in+genus+Gossypium&author_name=Muhammad+Mahmood+Ahmed&start_page=1&end_page=12">Order reprint</a>

                                </li>
                            </ul>
                        </div>
                    </aside>
                    
    <div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">Hereditas</span>
                        </h4>
                        <p class="c-journal-footer__issn">ISSN: 1601-5223</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
                                
                                    <li class="c-journal-footer__contact-item">Submission enquiries: <a href="https://www.editorialmanager.com/here/" target="_blank">Access here and click Contact Us</a></li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a></li>
                                
                            </ul>
                        </div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class='tracker' style='display:none' src='/track/article/10.1186/s41065-017-0034-4' alt=""/>

         
    <footer>
        <div class="ctx-publisher-footer">
    <div class="c-navbar">
        <div class="u-container">
            <div class="c-navbar__logo u-margin-right-xl">
                <a href="https://www.biomedcentral.com">
                    <img src=/static/images/bmc/logos/logo-bmc-aj-594e03e20d.svg alt="BMC logo" data-test="navbar-logo-footer">

                </a>
            </div>
            <div class="c-navbar__content">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


            </div>
        </div>
    </div>
    <div class="c-publisher-footer" data-test="publisher-footer">
        <div class="u-container">
            
            <div class="c-flex c-flex--wrap c-flex--space-between" role="navigation" data-test="publisher-footer-menu">
                <div class="c-flex" aria-label="Publisher footer links">
                    
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/login">Receive BMC newsletters</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/account">Manage article alerts</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/policies">Policies</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/accessibility">Accessibility</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/press-centre">Press center</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/contact-us">Contact us</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/jobs">Careers</a>
                                    </li>
                                
                            </ul>
                        
                    
                </div>
                <div class="u-margin-bottom-lg">
                    <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                    <ul class="c-flex u-reset-list" data-test="footer-social-links">
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://twitter.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Twitter link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Twitter page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-twitter-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://www.facebook.com/BioMedCentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Facebook link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Facebook page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-facebook-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://plus.google.com/106226337984152901734/"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Google Plus link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Google Plus page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-gplus-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="http://www.weibo.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Weibo link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Weibo page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-weibo-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </div>
            <p class="u-reset-margin">
                By using this website, you agree to our
                <a class="u-text-inherit" href="//www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
                <a class="u-text-inherit" href="//www.biomedcentral.com/privacy-statement">Privacy
                    statement</a> and
                <a class="u-text-inherit" href="//www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
                
                    <a class="optanon-toggle-display u-text-inherit" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
                
            </p>
        </div>
    </div>
</div>


        <div class="c-sn-footer c-sn-footer--neutral">
    <div class="u-container">
        <img src=/static/images/bmc/logos/logo-springernature-8bd4e8d82f.svg class="c-sn-footer__logo" alt="Springer Nature Logo" itemprop="logo" >
        <p data-test="copyright">&#169; 2018 BioMed Central Ltd unless otherwise stated. Part of
            <a href="http://www.springernature.com"
               data-track="click"
               data-track-action="Click Springer Nature link"
               data-track-category="Corporate footer"
               itemscope itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>

    </div>

    
    <script type="text/javascript" src="//cdn.pbgrd.com/bmc.js" async="async"></script>
    <noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>
    <!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker -->

<script id="springerlink-event-tracker" src="https://event-tracker.springernature.com/dist/eventTracker.js"></script>
<script type="text/javascript">
    (function(){
        var dataLayer = window.dataLayer[0];
        var doi = dataLayer.content.article.doi;
        var imprint = dataLayer.content.contentInfo.imprint;
        var bpids = dataLayer.session.authentication.authenticationID;
        var publisher;
        var pageType = dataLayer.page.category.pageType;
        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }
        var eventObject = {'content_type' : pageType,'doi' : doi};

        if (imprint === 'BioMed Central') { publisher = 'BMC' }
        else if (imprint === 'SpringerOpen') { publisher = 'SpringerOpen' }
        else { throw new Error('Event Tracker Error: The publisher has an unexpected value') }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }
        new EventTracker({'platform':publisher}).sendEvent('display', eventObject);
    })();
</script>


    </body>
</html>

