<!DOCTYPE html>
<html lang="en" class="no-js">
    <head>
        <title>Contrasting patterns of nucleotide polymorphism suggest different selective regimes within different parts of the PgiC1 gene in Festuca ovina L. | Hereditas | Full Text</title>

        

    <meta name="citation_abstract" content="Phosphoglucose isomerase (PGI, EC 5.3.1.9) is an essential metabolic enzyme in all eukaryotes. An earlier study of the PgiC1 gene, which encodes cytosolic PGI in the grass Festuca ovina L., revealed a marked difference in the levels of nucleotide polymorphism between the 5’ and 3’ portions of the gene. In the present study, we characterized the sequence polymorphism in F. ovina PgiC1 in more detail and examined possible explanations for the non-uniform pattern of nucleotide polymorphism across the gene. Our study confirms that the two portions of the PgiC1 gene show substantially different levels of DNA polymorphism and also suggests that the peptide encoded by the 3’ portion of PgiC1 is functionally and structurally more important than that encoded by the 5’ portion. Although there was some evidence of purifying selection (d
                           N/d
                           S test) on the 5’ portion of the gene, the signature of purifying selection was considerably stronger on the 3’ portion of the gene (d
                           N/d
                           S and McDonald–Kreitman tests). Several tests support the action of balancing selection within the 5’ portion of the gene. Wall’s B and Q tests were significant only for the 5’ portion of the gene. There were also marked peaks of nucleotide diversity, Tajima’s D and the d
                           N/d
                           S ratio at or around a PgiC1 codon site (within the 5’ portion of the gene) that a previous study had suggested was subject to positive diversifying selection. Our results suggest that the two portions of the gene have been subject to different selective regimes. Purifying selection appears to have been the main force contributing to the relatively low level of polymorphism within the 3’ portion of the sequence. In contrast, it is possible that balancing selection has contributed to the maintenance of the polymorphism within the 5’ portion of the gene."/>

    <meta name="description" content="Phosphoglucose isomerase (PGI, EC 5.3.1.9) is an essential metabolic enzyme in all eukaryotes. An earlier study of the PgiC1 gene, which encodes cytosolic PGI in the grass Festuca ovina L., revealed a marked difference in the levels of nucleotide polymorphism between the 5’ and 3’ portions of the gene. In the present study, we characterized the sequence polymorphism in F. ovina PgiC1 in more detail and examined possible explanations for the non-uniform pattern of nucleotide polymorphism across the gene. Our study confirms that the two portions of the PgiC1 gene show substantially different levels of DNA polymorphism and also suggests that the peptide encoded by the 3’ portion of PgiC1 is functionally and structurally more important than that encoded by the 5’ portion. Although there was some evidence of purifying selection (d
                           N/d
                           S test) on the 5’ portion of the gene, the signature of purifying selection was considerably stronger on the 3’ portion of the gene (d
                           N/d
                           S and McDonald–Kreitman tests). Several tests support the action of balancing selection within the 5’ portion of the gene. Wall’s B and Q tests were significant only for the 5’ portion of the gene. There were also marked peaks of nucleotide diversity, Tajima’s D and the d
                           N/d
                           S ratio at or around a PgiC1 codon site (within the 5’ portion of the gene) that a previous study had suggested was subject to positive diversifying selection. Our results suggest that the two portions of the gene have been subject to different selective regimes. Purifying selection appears to have been the main force contributing to the relatively low level of polymorphism within the 3’ portion of the sequence. In contrast, it is possible that balancing selection has contributed to the maintenance of the polymorphism within the 5’ portion of the gene."/>

    <meta name="journal_id" content="41065"/>

    <meta name="dc.title" content="Contrasting patterns of nucleotide polymorphism suggest different selective regimes within different parts of the PgiC1 gene in Festuca ovina L."/>

    <meta name="dc.source" content="Hereditas 2017 154:1"/>

    <meta name="dc.format" content="text/html"/>

    <meta name="dc.publisher" content="BioMed Central"/>

    <meta name="dc.date" content="2017-05-18"/>

    <meta name="dc.type" content="OriginalPaper"/>

    <meta name="dc.language" content="En"/>

    <meta name="dc.copyright" content="2017 The Author(s)"/>

    <meta name="dc.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="dc.description" content="Phosphoglucose isomerase (PGI, EC 5.3.1.9) is an essential metabolic enzyme in all eukaryotes. An earlier study of the PgiC1 gene, which encodes cytosolic PGI in the grass Festuca ovina L., revealed a marked difference in the levels of nucleotide polymorphism between the 5&#8217; and 3&#8217; portions of the gene. In the present study, we characterized the sequence polymorphism in F. ovina PgiC1 in more detail and examined possible explanations for the non-uniform pattern of nucleotide polymorphism across the gene. Our study confirms that the two portions of the PgiC1 gene show substantially different levels of DNA polymorphism and also suggests that the peptide encoded by the 3&#8217; portion of PgiC1 is functionally and structurally more important than that encoded by the 5&#8217; portion. Although there was some evidence of purifying selection (d
                           N/d
                           S test) on the 5&#8217; portion of the gene, the signature of purifying selection was considerably stronger on the 3&#8217; portion of the gene (d
                           N/d
                           S and McDonald&#8211;Kreitman tests). Several tests support the action of balancing selection within the 5&#8217; portion of the gene. Wall&#8217;s B and Q tests were significant only for the 5&#8217; portion of the gene. There were also marked peaks of nucleotide diversity, Tajima&#8217;s D and the d
                           N/d
                           S ratio at or around a PgiC1 codon site (within the 5&#8217; portion of the gene) that a previous study had suggested was subject to positive diversifying selection. Our results suggest that the two portions of the gene have been subject to different selective regimes. Purifying selection appears to have been the main force contributing to the relatively low level of polymorphism within the 3&#8217; portion of the sequence. In contrast, it is possible that balancing selection has contributed to the maintenance of the polymorphism within the 5&#8217; portion of the gene."/>

    <meta name="prism.issn" content="1601-5223"/>

    <meta name="prism.publicationName" content="Hereditas"/>

    <meta name="prism.publicationDate" content="2017-05-18"/>

    <meta name="prism.volume" content="154"/>

    <meta name="prism.number" content="1"/>

    <meta name="prism.section" content="OriginalPaper"/>

    <meta name="prism.startingPage" content="11"/>

    <meta name="prism.copyright" content="2017 The Author(s)"/>

    <meta name="prism.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="prism.url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0032-6"/>

    <meta name="prism.doi" content="doi:10.1186/s41065-017-0032-6"/>

    <meta name="citation_pdf_url" content="https://hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-017-0032-6"/>

    <meta name="citation_fulltext_html_url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0032-6"/>

    <meta name="citation_journal_title" content="Hereditas"/>

    <meta name="citation_publisher" content="BioMed Central"/>

    <meta name="citation_issn" content="1601-5223"/>

    <meta name="citation_title" content="Contrasting patterns of nucleotide polymorphism suggest different selective regimes within different parts of the PgiC1 gene in Festuca ovina L."/>

    <meta name="citation_volume" content="154"/>

    <meta name="citation_issue" content="1"/>

    <meta name="citation_publication_date" content="2017/12"/>

    <meta name="citation_online_date" content="2017/05/18"/>

    <meta name="citation_firstpage" content="11"/>

    <meta name="citation_article_type" content="Research"/>

    <meta name="citation_fulltext_world_readable" content=""/>

    <meta name="dc.identifier" content="doi:10.1186/s41065-017-0032-6"/>

    <meta name="DOI" content="10.1186/s41065-017-0032-6"/>

    <meta name="citation_doi" content="10.1186/s41065-017-0032-6"/>

    <meta name="dc.creator" content="Yuan Li"/>

    <meta name="dc.creator" content="Bengt Hansson"/>

    <meta name="dc.creator" content="Lena Ghatnekar"/>

    <meta name="dc.creator" content="Honor C. Prentice"/>

    <meta name="dc.subject" content="Animal Genetics and Genomics"/>

    <meta name="dc.subject" content="Evolutionary Biology"/>

    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=A map of rice genome variation reveals the origin of cultivated rice; citation_author=X Huang, N Kurata, X Wei, Z-X Wang, A Wang, Q Zhao; citation_volume=490; citation_publication_date=2012; citation_pages=497-501; citation_doi=10.1038/nature11532; citation_id=CR1"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Genet; citation_title=Comparative population genomics of maize domestication and improvement; citation_author=MB Hufford, X Xu, J Heerwaarden, T Pyh&#228;j&#228;rvi, J-M Chia, RA Cartwright; citation_volume=44; citation_publication_date=2012; citation_pages=808-11; citation_doi=10.1038/ng.2309; citation_id=CR2"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Genet; citation_title=Genome-wide genetic changes during modern breeding of maize; citation_author=Y Jiao, H Zhao, L Ren, W Song, B Zeng, J Guo; citation_volume=44; citation_publication_date=2012; citation_pages=812-5; citation_doi=10.1038/ng.2312; citation_id=CR3"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=DNA polymorphism at the FRIGIDA gene in Arabidopsis thaliana: extensive nonsynonymous variation is consistent with local selection for flowering time; citation_author=V Corre, F Roux, X Reboud; citation_volume=19; citation_publication_date=2002; citation_pages=1261-71; citation_doi=10.1093/oxfordjournals.molbev.a004187; citation_id=CR4"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=Molecular population genetics of redundant floral-regulatory genes in Arabidopsis thaliana
                           ; citation_author=RC Moore, SR Grant, MD Purugganan; citation_volume=22; citation_publication_date=2005; citation_pages=91-103; citation_doi=10.1093/molbev/msh261; citation_id=CR5"/>

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci U S A; citation_title=Signatures of selection among sex-determining alleles of the honey bee; citation_author=M Hasselmann, M Beye; citation_volume=101; citation_publication_date=2004; citation_pages=4888-93; citation_doi=10.1073/pnas.0307147101; citation_id=CR6"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Evol Biol; citation_title=Diversity and selective sweep in the OsAMT1; 1 genomic region of rice; citation_author=Z Ding, C Wang, S Chen, S Yu; citation_volume=11; citation_publication_date=2011; citation_pages=61; citation_doi=10.1186/1471-2148-11-61; citation_id=CR7"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Rev Genet; citation_title=Effective population size and patterns of molecular evolution and variation; citation_author=B Charlesworth; citation_volume=10; citation_publication_date=2009; citation_pages=195-205; citation_doi=10.1038/nrg2526; citation_id=CR8"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Rev Genet; citation_title=Genomic signatures of selection at linked sites: unifying the disparity among species; citation_author=AD Cutter, BA Payseur; citation_volume=14; citation_publication_date=2013; citation_pages=262-74; citation_doi=10.1038/nrg3425; citation_id=CR9"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=Selective constraints, amino acid composition, and the rate of protein evolution; citation_author=NJ Tourasse, W-H Li; citation_volume=17; citation_publication_date=2000; citation_pages=656-64; citation_doi=10.1093/oxfordjournals.molbev.a026344; citation_id=CR10"/>

    <meta name="citation_reference" content="citation_journal_title=Genetics; citation_title=A test of neutral molecular evolution based on nucleotide data; citation_author=RR Hudson, M Kreitman, M Aguad&#233;; citation_volume=116; citation_publication_date=1987; citation_pages=153-9; citation_id=CR11"/>

    <meta name="citation_reference" content="citation_title=The neutral theory of molecular evolution; citation_publication_date=1983; citation_id=CR12; citation_author=M Kimura; citation_publisher=Cambridge University Press"/>

    <meta name="citation_reference" content="citation_journal_title=Trends Parasitol; citation_title=Detecting signatures of balancing selection to identify targets of anti-parasite immunity; citation_author=GD Weedall, DJ Conway; citation_volume=26; citation_publication_date=2010; citation_pages=363-9; citation_doi=10.1016/j.pt.2010.04.002; citation_id=CR13"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS Genet; citation_title=Balancing selection and its effects on sequences in nearby genome regions; citation_author=D Charlesworth; citation_volume=2; citation_publication_date=2006; citation_pages=e64; citation_doi=10.1371/journal.pgen.0020064; citation_id=CR14"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Ecol; citation_title=Neutral and selective processes shape MHC gene diversity and expression in stocked brook charr populations (Salvelinus fontinalis); citation_author=FC Lamaze, SA Pavey, E Normandeau, G Roy, D Garant, L Bernatchez; citation_volume=23; citation_publication_date=2014; citation_pages=1730-48; citation_doi=10.1111/mec.12684; citation_id=CR15"/>

    <meta name="citation_reference" content="citation_journal_title=J Evol Biol; citation_title=MHC studies in nonmodel vertebrates: what have we learned about natural selection in 15 years?; citation_author=L Bernatchez, C Landry; citation_volume=16; citation_publication_date=2003; citation_pages=363-77; citation_doi=10.1046/j.1420-9101.2003.00531.x; citation_id=CR16"/>

    <meta name="citation_reference" content="citation_journal_title=Genetics; citation_title=Testing for effects of recombination rate on nucleotide diversity in natural populations of Arabidopsis lyrata
                           ; citation_author=SI Wright, JP Foxe, L DeRose-Wilson, A Kawabe, M Looseley, BS Gaut; citation_volume=174; citation_publication_date=2006; citation_pages=1421-30; citation_doi=10.1534/genetics.106.062588; citation_id=CR17"/>

    <meta name="citation_reference" content="Ghatnekar L. Genetic analysis of cytosolic PGI in Festuca ovina. PhD thesis. Lund University, 2003."/>

    <meta name="citation_reference" content="citation_title=The causes of molecular evolution; citation_publication_date=1991; citation_id=CR19; citation_author=JH Gillespie; citation_publisher=Oxford University Press"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Ecol; citation_title=Nature&#8217;s inordinate fondness for metabolic enzymes: why metabolic enzyme loci are so frequently targets of selection; citation_author=JH Marden; citation_volume=22; citation_publication_date=2013; citation_pages=5743-64; citation_doi=10.1111/mec.12534; citation_id=CR20"/>

    <meta name="citation_reference" content="citation_journal_title=J Mol Biol; citation_title=Crystallographic structure analysis of glucose 6-phosphate isomerase at 3.5 &#197; resolution; citation_author=PJ Shaw, H Muirhead; citation_volume=109; citation_publication_date=1977; citation_pages=475-85; citation_doi=10.1016/S0022-2836(77)80025-9; citation_id=CR21"/>

    <meta name="citation_reference" content="citation_journal_title=J Mol Evol; citation_title=Emergence of complex haplotypes from microevolutionary variation in sequence and structure of Colias phosphoglucose isomerase; citation_author=B Wang, WB Watt, C Aakre, N Hawthorne; citation_volume=68; citation_publication_date=2009; citation_pages=433-47; citation_doi=10.1007/s00239-009-9237-2; citation_id=CR22"/>

    <meta name="citation_reference" content="citation_journal_title=Biol J Linn Soc Lond; citation_title=The adaptive significance of electrophoretic mobility in phosphoglucose isomerase (PGI); citation_author=BJ Riddoch; citation_volume=50; citation_publication_date=1993; citation_pages=1-17; citation_doi=10.1111/j.1095-8312.1993.tb00915.x; citation_id=CR23"/>

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci U S A; citation_title=Functional and physiological consequences of genetic variation at phosphoglucose isomerase: Heat shock protein expression is related to enzyme genotype in a montane beetle; citation_author=EP Dahlhoff, NE Rank; citation_volume=97; citation_publication_date=2000; citation_pages=10056-61; citation_doi=10.1073/pnas.160277697; citation_id=CR24"/>

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci U S A; citation_title=Eggs, enzymes, and evolution: natural genetic variants change insect fecundity; citation_author=WB Watt; citation_volume=89; citation_publication_date=1992; citation_pages=10608-12; citation_doi=10.1073/pnas.89.22.10608; citation_id=CR25"/>

    <meta name="citation_reference" content="citation_journal_title=J Ecol; citation_title=Associations between allele frequencies in Festuca ovina and habitat variation in the alvar grasslands on the Baltic island of &#214;land; citation_author=HC Prentice, M L&#246;nn, LP Lefkovitch, H Runyeon; citation_volume=83; citation_publication_date=1995; citation_pages=391-402; citation_doi=10.2307/2261593; citation_id=CR26"/>

    <meta name="citation_reference" content="citation_title=Grasses of the British Isles; citation_publication_date=2009; citation_id=CR27; citation_author=T Cope; citation_author=A Gray; citation_publisher=Botanical Society of the British Isles"/>

    <meta name="citation_reference" content="citation_journal_title=J Ecol; citation_title=Changes in allozyme frequencies in Festuca ovina populations after a 9-year nutrient/water experiment; citation_author=HC Prentice, M L&#246;nn, H Lager, E Ros&#233;n, E Maarel; citation_volume=88; citation_publication_date=2000; citation_pages=331-47; citation_doi=10.1046/j.1365-2745.2000.00454.x; citation_id=CR28"/>

    <meta name="citation_reference" content="citation_journal_title=Proc R Soc Lond Ser B Biol Sci; citation_title=A horizontally transferred nuclear gene is associated with microhabitat variation in a natural plant population; citation_author=HC Prentice, Y Li, M L&#246;nn, A Tunlid, L Ghatnekar; citation_volume=282; citation_publication_date=2015; citation_pages=20152453; citation_doi=10.1098/rspb.2015.2453; citation_id=CR29"/>

    <meta name="citation_reference" content="citation_journal_title=Heredity (Edinb); citation_title=A polymorphic duplicated locus for cytosolic PGI segregating in sheep&#8217;s fescue (Festuca ovina L.); citation_author=L Ghatnekar; citation_volume=83; citation_publication_date=1999; citation_pages=451-9; citation_doi=10.1038/sj.hdy.6885750; citation_id=CR30"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Phylogenet Evol; citation_title=Origin and timing of the horizontal transfer of a PgiC gene from Poa to Festuca ovina
                           ; citation_author=P Vallenback, M Jaarola, L Ghatnekar, BO Bengtsson; citation_volume=46; citation_publication_date=2008; citation_pages=890-6; citation_doi=10.1016/j.ympev.2007.11.031; citation_id=CR31"/>

    <meta name="citation_reference" content="citation_journal_title=Plos One; citation_title=Structure of the natural transgene PgiC2 in the common grass Festuca ovina
                           ; citation_author=P Vallenback, L Ghatnekar, BO Bengtsson; citation_volume=5; citation_publication_date=2010; citation_pages=e13529; citation_doi=10.1371/journal.pone.0013529; citation_id=CR32"/>

    <meta name="citation_reference" content="citation_journal_title=Genetica; citation_title=Geographic and molecular variation in a natural plant transgene; citation_author=P Vallenback, BO Bengtsson, L Ghatnekar; citation_volume=138; citation_publication_date=2010; citation_pages=355-62; citation_doi=10.1007/s10709-009-9425-4; citation_id=CR33"/>

    <meta name="citation_reference" content="citation_journal_title=Plos One; citation_title=Evidence for positive selection within the PgiC1 locus in the grass Festuca ovina
                           ; citation_author=Y Li, B Canb&#228;ck, T Johansson, A Tunlid, HC Prentice; citation_volume=10; citation_publication_date=2015; citation_pages=e0125831; citation_doi=10.1371/journal.pone.0125831; citation_id=CR34"/>

    <meta name="citation_reference" content="citation_journal_title=Biochemistry; citation_title=Crystal structure of rabbit phosphoglucose isomerase complexed with 5-phospho-D-arabinonate identifies the role of Glu357 in catalysis; citation_author=CJ Jeffery, R Hardr&#233;, L Salmon; citation_volume=40; citation_publication_date=2001; citation_pages=1560-6; citation_doi=10.1021/bi0018483; citation_id=CR35"/>

    <meta name="citation_reference" content="citation_title=Molecular evolutionary genetics; citation_publication_date=1987; citation_id=CR36; citation_author=M Nei; citation_publisher=Columbia University Press"/>

    <meta name="citation_reference" content="citation_journal_title=Theor Popul Biol; citation_title=On the number of segregating sites in genetical models without recombination; citation_author=GA Watterson; citation_volume=7; citation_publication_date=1975; citation_pages=256-76; citation_doi=10.1016/0040-5809(75)90020-9; citation_id=CR37"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=DnaSP v5: a software for comprehensive analysis of DNA polymorphism data; citation_author=P Librado, J Rozas; citation_volume=25; citation_publication_date=2009; citation_pages=1451-2; citation_doi=10.1093/bioinformatics/btp187; citation_id=CR38"/>

    <meta name="citation_reference" content="citation_journal_title=Genetics; citation_title=Statistical properties of the number of recombination events in the history of a sample of DNA sequences; citation_author=RR Hudson, NL Kaplan; citation_volume=111; citation_publication_date=1985; citation_pages=147-64; citation_id=CR39"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Rev Genet; citation_title=Estimating recombination rates from population-genetic data; citation_author=MPH Stumpf, GAT McVean; citation_volume=4; citation_publication_date=2003; citation_pages=959-68; citation_doi=10.1038/nrg1227; citation_id=CR40"/>

    <meta name="citation_reference" content="citation_journal_title=Genetics; citation_title=Estimating diversifying selection and functional constraint in the presence of recombination; citation_author=DJ Wilson, G McVean; citation_volume=172; citation_publication_date=2006; citation_pages=1411-25; citation_doi=10.1534/genetics.105.044917; citation_id=CR41"/>

    <meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=Linkage disequilibrium in finite populations; citation_author=WG Hill, A Robertson; citation_volume=38; citation_publication_date=1968; citation_pages=226-31; citation_doi=10.1007/BF01245622; citation_id=CR42"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Haploview: analysis and visualization of LD and haplotype maps; citation_author=JC Barrett, B Fry, J Maller, MJ Daly; citation_volume=21; citation_publication_date=2005; citation_pages=263-5; citation_doi=10.1093/bioinformatics/bth457; citation_id=CR43"/>

    <meta name="citation_reference" content="citation_journal_title=Genetics; citation_title=Statistical method for testing the neutral mutation hypothesis by DNA polymorphism; citation_author=F Tajima; citation_volume=123; citation_publication_date=1989; citation_pages=585-95; citation_id=CR44"/>

    <meta name="citation_reference" content="citation_journal_title=Genetics; citation_title=Hitchhiking under positive Darwinian selection; citation_author=JC Fay, CI Wu; citation_volume=155; citation_publication_date=2000; citation_pages=1405-13; citation_id=CR45"/>

    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Adaptive protein evolution at the Adh locus in Drosophila
                           ; citation_author=JH McDonald, M Kreitman; citation_volume=351; citation_publication_date=1991; citation_pages=652-4; citation_doi=10.1038/351652a0; citation_id=CR46"/>

    <meta name="citation_reference" content="citation_journal_title=Genet Res; citation_title=Recombination and the power of statistical tests of neutrality; citation_author=JD Wall; citation_volume=74; citation_publication_date=1999; citation_pages=65-79; citation_doi=10.1017/S0016672399003870; citation_id=CR47"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=ConSurf 2010: calculating evolutionary conservation in sequence and structure of proteins and nucleic acids; citation_author=H Ashkenazy, E Erez, E Martz, T Pupko, N Ben-Tal; citation_volume=38; citation_publication_date=2010; citation_pages=W529-33; citation_doi=10.1093/nar/gkq399; citation_id=CR48"/>

    <meta name="citation_reference" content="Suzek BE, Wang Y, Huang H, McGarvey PB, Wu CH, the UniProt Consortium. UniRef clusters: a comprehensive and scalable alternative for improving sequence similarity searches. Bioinformatics. 2015;31:926&#8211;32."/>

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci U S A; citation_title=Sequence context-specific profiles for homology searching; citation_author=A Biegert, J S&#246;ding; citation_volume=106; citation_publication_date=2009; citation_pages=3770-5; citation_doi=10.1073/pnas.0810767106; citation_id=CR50"/>

    <meta name="citation_reference" content="Yamada KD, Tomii K, Katoh K. Application of the MAFFT sequence alignment program to large data&#8212;reexamination of the usefulness of chained guide trees. Bioinformatics. 2016;32:3246&#8211;51."/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform; citation_author=K Katoh, K Misawa, K Kuma, T Miyata; citation_volume=30; citation_publication_date=2002; citation_pages=3059-66; citation_doi=10.1093/nar/gkf436; citation_id=CR52"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Rate4Site: an algorithmic tool for the identification of functional regions in proteins by surface mapping of evolutionary determinants within their homologues; citation_author=T Pupko, RE Bell, I Mayrose, F Glaser, N Ben-Tal; citation_volume=18; citation_publication_date=2002; citation_pages=S71-7; citation_doi=10.1093/bioinformatics/18.suppl_1.S71; citation_id=CR53"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=Comparison of site-specific rate-inference methods for protein sequences: empirical Bayesian methods are superior; citation_author=I Mayrose, D Graur, N Ben-Tal, T Pupko; citation_volume=21; citation_publication_date=2004; citation_pages=1781-91; citation_doi=10.1093/molbev/msh194; citation_id=CR54"/>

    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=The structure of haplotype blocks in the human genome; citation_author=SB Gabriel, SF Schaffner, H Nguyen, JM Moore, J Roy, B Blumenstiel; citation_volume=296; citation_publication_date=2002; citation_pages=2225-9; citation_doi=10.1126/science.1069424; citation_id=CR55"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Ecol; citation_title=Recurrent gene deletions and the evolution of adaptive cyanogenesis polymorphisms in white clover (Trifolium repens L.); citation_author=KM Olsen, NJ Kooyers, LL Small; citation_volume=22; citation_publication_date=2013; citation_pages=724-38; citation_doi=10.1111/j.1365-294X.2012.05667.x; citation_id=CR56"/>

    <meta name="citation_reference" content="citation_journal_title=Annu Rev Genom Hum G; citation_title=The neutral theory of molecular evolution in the genomic era; citation_author=M Nei, Y Suzuki, M Nozawa; citation_volume=11; citation_publication_date=2010; citation_pages=265-89; citation_doi=10.1146/annurev-genom-082908-150129; citation_id=CR57"/>

    <meta name="citation_reference" content="citation_journal_title=Annu Rev Genom Hum G; citation_title=Methods to detect selection in populations with applications to the human; citation_author=M Kreitman; citation_volume=1; citation_publication_date=2000; citation_pages=539-59; citation_doi=10.1146/annurev.genom.1.1.539; citation_id=CR58"/>

    <meta name="citation_reference" content="citation_journal_title=Acta Crystallogr F Struct Biol Commun; citation_title=Structural studies of phosphoglucose isomerase from Mycobacterium tuberculosis H37Rv; citation_author=K Anand, D Mathur, A Anant, LC Garg; citation_volume=66; citation_publication_date=2010; citation_pages=490-7; citation_doi=10.1107/S1744309110011656; citation_id=CR59"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=Nucleotide polymorphism at a gene (Pgi) under balancing selection in a butterfly metapopulation; citation_author=CW Wheat, CR Hagg, JH Marden, I Hanski, MJ Frilander; citation_volume=27; citation_publication_date=2010; citation_pages=267-81; citation_doi=10.1093/molbev/msp227; citation_id=CR60"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=From DNA to fitness differences: sequences and structures of adaptive variants of Colias phosphoglucose isomerase (PGI); citation_author=CW Wheat, WB Watt, DD Pollock, PM Schulte; citation_volume=23; citation_publication_date=2006; citation_pages=499-512; citation_doi=10.1093/molbev/msj062; citation_id=CR61"/>

    <meta name="citation_reference" content="citation_journal_title=J Mol Evol; citation_title=Evolutionary Genomics of Colias Phosphoglucose Isomerase (PGI) Introns; citation_author=BQ Wang, JM DePasse, WB Watt; citation_volume=74; citation_publication_date=2012; citation_pages=96-111; citation_doi=10.1007/s00239-012-9492-5; citation_id=CR62"/>

    <meta name="citation_reference" content="citation_journal_title=Genetics; citation_title=DNA polymorphism at the cytosolic phosphoglucose isomerase (PgiC) locus of the wild plant Arabidopsis thaliana
                           ; citation_author=A Kawabe, K Yamane, NT Miyashita; citation_volume=156; citation_publication_date=2000; citation_pages=1339-47; citation_id=CR63"/>

    <meta name="citation_reference" content="citation_journal_title=Evolution; citation_title=Toward a selection theory of molecular evolution; citation_author=MW Hahn; citation_volume=62; citation_publication_date=2008; citation_pages=255-65; citation_doi=10.1111/j.1558-5646.2007.00308.x; citation_id=CR64"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Ecol Resour; citation_title=Analysing recombination in nucleotide sequences; citation_author=DP Martin, P Lemey, D Posada; citation_volume=11; citation_publication_date=2011; citation_pages=943-55; citation_doi=10.1111/j.1755-0998.2011.03026.x; citation_id=CR65"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Ecol; citation_title=The past, present and future of genomic scans for selection; citation_author=JD Jensen, M Foll, L Bernatchez; citation_volume=25; citation_publication_date=2016; citation_pages=1-4; citation_doi=10.1111/mec.13493; citation_id=CR66"/>

    <meta name="citation_reference" content="citation_journal_title=Genome Res; citation_title=How reliable are empirical genomic scans for selective sweeps?; citation_author=KM Teshima, G Coop, M Przeworski; citation_volume=16; citation_publication_date=2006; citation_pages=702-12; citation_doi=10.1101/gr.5105206; citation_id=CR67"/>

    <meta name="citation_reference" content="citation_journal_title=Heredity (Edinb); citation_title=Detecting loci under selection in a hierarchically structured population; citation_author=L Excoffier, T Hofer, M Foll; citation_volume=103; citation_publication_date=2009; citation_pages=285-98; citation_doi=10.1038/hdy.2009.74; citation_id=CR68"/>

    <meta name="citation_reference" content="citation_journal_title=Biochim Biophys Acta; citation_title=Effects of inherited mutations on catalytic activity and structural stability of human glucose-6-phosphate isomerase expressed in Escherichia coli
                           ; citation_author=HY Lin, YH Kao, ST Chen, M Meng; citation_volume=1794; citation_publication_date=2009; citation_pages=315-23; citation_doi=10.1016/j.bbapap.2008.11.004; citation_id=CR69"/>

    <meta name="citation_reference" content="citation_journal_title=Biochemistry; citation_title=A tale of two isomerases: compact versus extended active sites in ketosteroid isomerase and phosphoglucose isomerase; citation_author=S Somarowthu, HR Brodkin, JA D&#8217;Aquino, D Ringe, MJ Ondrechen, PJ Beuning; citation_volume=50; citation_publication_date=2011; citation_pages=9283-95; citation_doi=10.1021/bi201089v; citation_id=CR70"/>

    <meta name="citation_author" content="Yuan Li"/>

    <meta name="citation_author_institution" content="Department of Biology, Lund University, Lund, Sweden"/>

    <meta name="citation_author" content="Bengt Hansson"/>

    <meta name="citation_author_institution" content="Department of Biology, Lund University, Lund, Sweden"/>

    <meta name="citation_author" content="Lena Ghatnekar"/>

    <meta name="citation_author_institution" content="Department of Biology, Lund University, Lund, Sweden"/>

    <meta name="citation_author" content="Honor C. Prentice"/>

    <meta name="citation_author_institution" content="Department of Biology, Lund University, Lund, Sweden"/>



        <meta charset="UTF-8"/>
        <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
        <meta name="viewport" content="width=device-width, initial-scale=1"/>

        <!-- For IE 9 and below -->
<!--[if IE]><link rel="shortcut icon" href=/static/images/favicons//favicon.ico><![endif]-->
<!-- Touch Icons - iOS and Android 2.1+ -->
<link rel="apple-touch-icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>
<!-- Firefox, Chrome, Safari, IE 11+ and Opera -->
<link rel="icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>


        <script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script>
        <script>(function(d){if (sessionStorage.fontsLoaded){d.style.setProperty('--font-family-sans', 'Europa');d.style.setProperty('--font-family-serif', 'Noto Serif')}})(document.documentElement);</script>

        <link rel="stylesheet" media="screen" href=/static/styles/basic/basic-a31ca85bc2.css>
<link rel="stylesheet" media="print" href=/static/styles/bmc/print-3a6240f456.css>

<link rel="stylesheet" class="js-ctm" href=/static/styles/bmc/main-84414d3d47.css
      media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">

        

        <script src="/static/js/app-bundle-8d61e6ef37.js" defer></script>
        
        <script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script>
        
    <script type="text/javascript">
        config = {
            env: 'live',
            site: 'hereditasjournal.biomedcentral.com',
            siteWithPath: 'hereditasjournal.biomedcentral.com' + window.location.pathname,
            twitterHashtag: '',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/s41065-017-0032-6',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central'
        };
    </script>

        
    <script type="text/javascript">
        dataLayer = [{"content":{"article":{"doi":"10.1186/s41065-017-0032-6","articleType":"Research","peerReviewType":"Closed","supplement":null,"keywords":"Festuca ovina;Cytosolic phosphoglucose isomerase;PgiC1;Nucleotide polymorphism;Purifying selection"},"contentInfo":{"imprint":"BioMed Central","title":"Contrasting patterns of nucleotide polymorphism suggest different selective regimes within different parts of the PgiC1 gene in Festuca ovina L.","publishedAt":1495065600000,"publishedAtDate":"2017-05-18","author":["Yuan Li","Bengt Hansson","Lena Ghatnekar","Honor C. Prentice"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":"CC BY + CC0","openAccess":true},"environment":"live"},"journal":{"siteKey":"hereditasjournal.biomedcentral.com","volume":"154","issue":"1","title":"Hereditas","type":"AJs Life","journalID":41065,"gaCode":"UA-62066850-11","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Research","publishingSegment":"Genetics & Heredity (Life)"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}}}]

    </script>

        <script src="https://cdn.cookielaw.org/consent/7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script>
        
    
        
    

        
    
        
            
                <!-- Google Tag Manager -->
                <script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script>
                <!-- End Google Tag Manager -->
            
        
    


        
    
    <link rel="canonical" href="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0032-6"/>
    

        
        
        <meta property="og:url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0032-6"/>
        <meta property="og:type" content="article"/>
        <meta property="og:site_name" content="Hereditas"/>
        <meta property="og:title" content="Contrasting patterns of nucleotide polymorphism suggest different selective regimes within different parts of the PgiC1 gene in Festuca ovina L."/>
        <meta property="og:description" content="Phosphoglucose isomerase (PGI, EC 5.3.1.9) is an essential metabolic enzyme in all eukaryotes. An earlier study of the PgiC1 gene, which encodes cytosolic PGI in the grass Festuca ovina L., revealed a marked difference in the levels of nucleotide polymorphism between the 5’ and 3’ portions of the gene. In the present study, we characterized the sequence polymorphism in F. ovina PgiC1 in more detail and examined possible explanations for the non-uniform pattern of nucleotide polymorphism across the gene. Our study confirms that the two portions of the PgiC1 gene show substantially different levels of DNA polymorphism and also suggests that the peptide encoded by the 3’ portion of PgiC1 is functionally and structurally more important than that encoded by the 5’ portion. Although there was some evidence of purifying selection (d N/d S test) on the 5’ portion of the gene, the signature of purifying selection was considerably stronger on the 3’ portion of the gene (d N/d S and McDonald–Kreitman tests). Several tests support the action of balancing selection within the 5’ portion of the gene. Wall’s B and Q tests were significant only for the 5’ portion of the gene. There were also marked peaks of nucleotide diversity, Tajima’s D and the d N/d S ratio at or around a PgiC1 codon site (within the 5’ portion of the gene) that a previous study had suggested was subject to positive diversifying selection. Our results suggest that the two portions of the gene have been subject to different selective regimes. Purifying selection appears to have been the main force contributing to the relatively low level of polymorphism within the 3’ portion of the sequence. In contrast, it is possible that balancing selection has contributed to the maintenance of the polymorphism within the 5’ portion of the gene."/>
        <meta property="og:image" content="https://images.springer.com/sgw/journals/medium/41065.jpg"/>
    
    </head>

    <body class="journal journal-fulltext"
          
          >
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK"
                            height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><feColorMatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"/></filter></defs><symbol id="icon-arrow-down" viewBox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"/></symbol><symbol id="icon-arrow-left-bullet" viewBox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"/></symbol><symbol id="icon-arrow-right-bullet" viewBox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"/></symbol><symbol id="icon-arrow-right" viewBox="0 0 32 64"><path d="M0 0v64l32-32z"/></symbol><symbol id="icon-close" viewBox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"/></symbol><symbol id="icon-community" viewBox="0 0 60.643 31.01"><mask maskUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><path fill="#FFF" d="M0 31.01h60.643V0H0z" filter="url(#a)"/></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"/></symbol><symbol id="icon-download" viewBox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"/></symbol><symbol id="icon-ext-link" viewBox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"/><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"/><path fill="#676767" d="M9.7 0L16 6.2V0z"/></symbol><symbol id="icon-info-bordered" viewBox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"/><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"/></symbol><symbol id="icon-opr" viewBox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"/><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"/></symbol><symbol id="icon-quotation" viewBox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"/></symbol><symbol id="icon-remove" viewBox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"/><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"/></symbol><symbol id="icon-ribbon" viewBox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"/></symbol><symbol id="icon-rss" viewBox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"/><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"/><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"/></g></symbol><symbol id="icon-search" viewBox="0 0 16 16"><path d="M6.276 12.257C2.815 12.257 0 9.508 0 6.128 0 2.749 2.815 0 6.276 0c3.462 0 6.277 2.749 6.277 6.128.002 3.38-2.815 6.129-6.277 6.129zm0-10.772c-2.622 0-4.755 2.083-4.755 4.643 0 2.562 2.133 4.644 4.755 4.644 2.623 0 4.756-2.082 4.756-4.644 0-2.56-2.134-4.643-4.756-4.643z"/><path d="M16 13.751L13.641 16l-4.875-4.82 2.36-2.248L16 13.751z"/></symbol><symbol id="icon-speech-bubble" viewBox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"/></symbol><symbol id="icon-error" viewBox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"/></symbol><symbol id="icon-info" viewBox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"/></symbol><symbol id="icon-success" viewBox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"/></symbol><symbol id="icon-warning" viewBox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"/></symbol><symbol id="icon-facebook-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"/></symbol><symbol id="icon-facebook" viewBox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"/><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"/></symbol><symbol id="icon-gplus-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M480.5 273.117c.123.076.26.185.416.334.146.152.289.343.432.562.135.211.258.459.359.746.086.286.129.619.129 1.004-.012.7-.164 1.262-.465 1.683a5.374 5.374 0 0 1-.461.562c-.186.176-.375.35-.58.527a2.65 2.65 0 0 0-.322.41.908.908 0 0 0-.176.553c0 .205.059.375.18.508.102.129.199.238.299.336l.662.545a6.48 6.48 0 0 1 1.086 1.119c.293.415.445.955.459 1.625 0 .949-.42 1.789-1.256 2.521-.869.76-2.123 1.15-3.762 1.177-1.373-.015-2.4-.308-3.074-.879-.682-.534-1.023-1.172-1.023-1.917 0-.359.111-.766.336-1.211.215-.443.602-.837 1.172-1.174a6.126 6.126 0 0 1 1.998-.725 15.415 15.415 0 0 1 1.723-.173 3.996 3.996 0 0 1-.377-.596 1.347 1.347 0 0 1-.193-.74c0-.174.023-.317.074-.433a8.83 8.83 0 0 1 .121-.338 5.973 5.973 0 0 1-.633.035c-1.043-.012-1.838-.339-2.381-.979-.574-.603-.861-1.3-.861-2.097 0-.961.404-1.832 1.217-2.616.559-.46 1.135-.757 1.736-.898a8.494 8.494 0 0 1 1.67-.182h3.916l-1.209.704h-1.207v.005l-.005.002zm.754 11.276c0-.498-.162-.932-.484-1.302-.346-.348-.881-.778-1.609-1.29a4.045 4.045 0 0 0-.439-.02c-.102-.016-.354 0-.768.039a8.141 8.141 0 0 0-1.244.275c-.1.036-.24.096-.42.17a2.713 2.713 0 0 0-.553.355 1.985 1.985 0 0 0-.459.592c-.145.25-.215.551-.215.901 0 .687.311 1.253.932 1.698.596.444 1.404.68 2.432.688.928-.014 1.629-.217 2.117-.612.476-.385.71-.886.71-1.494zm-2.729-5.743a1.843 1.843 0 0 0 1.289-.556c.164-.249.271-.506.318-.767a7.32 7.32 0 0 0 .041-.659c0-.772-.195-1.552-.594-2.336a2.611 2.611 0 0 0-.73-.921 1.894 1.894 0 0 0-1.064-.352 1.723 1.723 0 0 0-1.324.64c-.299.437-.438.92-.428 1.457 0 .709.209 1.444.623 2.211.201.356.459.659.775.912.323.248.686.371 1.094.371zm9.698.077h-2.055v-2.056h-.996v2.056h-2.053v.998h2.053v2.052h.996v-2.052h2.055z"/></symbol><symbol id="icon-gplus" viewBox="0 0 24 24"><circle fill="#D84A36" cx="12" cy="12" r="12"/><g fill="#FFF"><path d="M12.012 12.111c-.304-.33-.07-.842.155-1.078.226-.241.639-.56.829-.74.193-.184.66-.605.674-1.868 0 0 .112-1.139-1.127-2.06h.977l1.038-.593h-3.507s-2.083-.067-3.085 1.207c-1 1.276-.966 2.871.114 3.77 1.08.898 2.371.648 2.371.648s-.197.422-.163.775c.036.353.421.849.504.965 0 0-2.386-.011-3.551 1.033-1.165 1.053-1.29 3.097 1.021 3.803 2.312.707 5.303-.055 5.906-2.277 0 0 .262-.908-.189-1.683-.459-.775-1.663-1.571-1.967-1.902zm-2.555-1.83c-.744-1-1.308-2.953 0-3.783 0 0 .955-.579 1.801.266.846.846 1.082 2.277.928 3.003-.15.727-.563.897-.739 1.008-.185.111-1.249.504-1.99-.494zm3.668 5.481c-.035.393-.083 1.254-1.491 1.664-1.409.408-3.826-.264-3.757-1.914 0 0 .006-.698.449-1.089.445-.396 1.094-.794 3.029-.819 0 0 .643.473 1.072.809.427.332.733.952.698 1.349z"/><path d="M16.323 11.055V9.803h-.752v1.252h-1.255v.748h1.255v1.256h.752v-1.256h1.248v-.748z"/></g></symbol><symbol id="icon-linkedin" viewBox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"/><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"/><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"/></g></symbol><symbol id="icon-reddit" viewBox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"/><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"/><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"/><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"/><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"/><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"/><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"/><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"/></symbol><symbol id="icon-twitter-bordered" viewBox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"/><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/></symbol><symbol id="icon-twitter" viewBox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"/><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"/></symbol><symbol id="icon-weibo-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"/><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"/><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"/></symbol><symbol id="icon-weibo" viewBox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"/><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"/><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"/><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"/><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"/><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"/></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner c-banner--info">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner" >
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1"
                 data-gpt-unitpath="/270604982/bmc/hereditasjournal/articles"
                 data-gpt-sizes="728x90,970x90"
                 data-gpt-targeting="pos=LB1;doi=10.1186/s41065-017-0032-6;kwrd=Festuca ovina,Cytosolic phosphoglucose isomerase,PgiC1,Nucleotide polymorphism,Purifying selection;pmc=L32030,L21001;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-017-0032-6&amp;kwrd=Festuca ovina,Cytosolic phosphoglucose isomerase,PgiC1,Nucleotide polymorphism,Purifying selection&amp;pmc=L32030,L21001&amp;">
                        <img data-test="gpt-advert-fallback-img"
                             src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-017-0032-6&amp;kwrd=Festuca ovina,Cytosolic phosphoglucose isomerase,PgiC1,Nucleotide polymorphism,Purifying selection&amp;pmc=L32030,L21001&amp;"
                             alt="Advertisement"
                             width="728"
                             height="90">
                    </a>
                </noscript>
            </div>
        </div>
    </div>

        
         
    <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>


<header class="c-popup-search ctx-publisher-header" id="top" data-test="publisher-header">
    <nav class="c-navbar">
        <div class="u-container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization" />
                <div class="c-logo">
    <a href="https://www.biomedcentral.com" itemprop="url">
        <img src=/static/images/bmc/logos/logo-bmc-white-aj-e3a70ae848.svg alt="BMC" itemprop="logo">
        <div class="c-logo__strapline">
            <img src=/static/images/bmc/logos/logo-bmc-white-strapline-sn-e7b1d6e67f.svg alt="Part of Springer Nature">
        </div>
    </a>
</div>
            </div>
            <button
                type="button"
                class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg"
                id="publisher-header-menu-button"
                data-toggle="collapse"
                data-test="publisher-header-menu-toggle"
                data-toggle-linked="publisher-header"
                data-target="publisher-header-search publisher-header-menu"
                aria-controls="publisher-header-menu publisher-header-search"
                aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


                <ul class="c-navbar__nav">
                    <li class="c-navbar__item">
                        <a data-replace="true"
                           data-placeholder="/placeholder/v1/username"
                           class="c-navbar__link"
                           href="https://www.biomedcentral.com/login"
                           data-test="login-link"
                           data-track="click"
                           data-track-category="Account"
                           data-track-action="Click Login Link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                    <li class="c-navbar__item">
                        <button
                            type="button"
                            class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg"
                            data-toggle="collapse"
                            data-test="header-search-button"
                            data-target="publisher-header-search publisher-header-menu publisher-header-menu-button"
                            aria-controls="publisher-header-search publisher-header-menu"
                            aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true">
                                    <use xlink:href="#icon-search"></use>
                                </svg>
                            </span>
                        </button>
                    </li>
                </ul>
            </div>
        </div>
    </nav>
    <div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field c-form-field--inline" method="GET" action="//www.biomedcentral.com/search" data-track="submit"
        data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label data-track-label="" data-test="global-search">
        <label for="publisherSearch" class="u-visually-hidden">Search all BMC articles</label>
        <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text" value="" placeholder="Search all BMC articles"/>
        <button class="c-button c-button--secondary" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true">
        <use xlink:href="#icon-search"></use>
    </svg>
</button>


        <input type="hidden" name="searchType" value="publisherSearch"/>
    </form>
</div>

            </div>
        </div>
    </div>
</header>


        
            <header class="c-journal-header c-journal-header--hereditas ctx-journal-header">
                <div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="/">
    
        
    
    <span class="c-journal-title__text ">Hereditas</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="u-container">
                        
                            <button type="button"
                                    class="c-navbar__link c-dropdown u-hidden-at-lg"
                                    data-toggle="collapse"
                                    data-test="journal-header-menu"
                                    data-toggle-linked="journal-header"
                                    data-target="journal-menu"
                                    aria-controls="journal-menu"
                                    aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav">
                                    <ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/'>Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/about'>About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href='/articles'>Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/submission-guidelines'>Submission Guidelines</a>
                                            </li>
                                        
                                    </ul>
                                </nav>
                            </div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--ocean-light-blue">
                    
                </div>
            </header>
            
        

        <div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-content-layout c-content-layout--fulltext" data-component="sticky-container">
                <div class="c-content-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3><ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc=""><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Methods">Methods</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec6" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results">Results</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec11" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Discussion">Discussion</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec14" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Conclusions">Conclusions</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li></ol></nav>
                </div>

                <main class="c-content-layout__main">
                    <article itemscope itemtype="http://schema.org/ScholarlyArticle">
                        
                        
    <div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
            <li class="c-article-identifiers__item" data-test="article-category">Research</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
    </div>

                        <div class="FulltextWrapper"><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">Contrasting patterns of nucleotide polymorphism suggest different selective regimes within different parts of the <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> gene in <em xmlns="" class="EmphasisTypeItalic">Festuca ovina</em> L.</h1></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true"><ul class="u-listReset"><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Yuan&nbsp;Li</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup><a href="mailto:Yuan.Li@biol.lu.se" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Bengt&nbsp;Hansson</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Lena&nbsp;Ghatnekar</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup> and </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Honor&nbsp;C.&nbsp;Prentice</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup></li></ul></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation"><div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">Hereditas</span></span><span class="ArticleCitation_Year">2017</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">154</strong>:11</span></span></div><p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/s41065-017-0032-6">https://doi.org/10.1186/s41065-017-0032-6</a></p><p class="Copyright">© &nbsp;The Author(s)&nbsp;2017</p></div></div><ul class="ArticleHistory u-reset-list"><li class="History HistoryReceived"><strong>Received: </strong><span>10&nbsp;December&nbsp;2016</span></li><li class="History HistoryAccepted"><strong>Accepted: </strong><span>13&nbsp;April&nbsp;2017</span></li><li class="History HistoryOnlineDate"><strong>Published: </strong><span itemprop="datePublished">18&nbsp;May&nbsp;2017</span></li></ul></div>
                        
                        
                        
    

                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2><div class="c-section__content" id="Abstract"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p id="Par1" class="Para">Phosphoglucose isomerase (PGI, EC 5.3.1.9) is an essential metabolic enzyme in all eukaryotes. An earlier study of the <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> gene, which encodes cytosolic PGI in the grass <em xmlns="" class="EmphasisTypeItalic">Festuca ovina</em> L., revealed a marked difference in the levels of nucleotide polymorphism between the 5’ and 3’ portions of the gene.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Methods</h3>
                <p id="Par2" class="Para">In the present study, we characterized the sequence polymorphism in <em xmlns="" class="EmphasisTypeItalic">F. ovina PgiC1</em> in more detail and examined possible explanations for the non-uniform pattern of nucleotide polymorphism across the gene.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Results</h3>
                <p id="Par3" class="Para">Our study confirms that the two portions of the <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> gene show substantially different levels of DNA polymorphism and also suggests that the peptide encoded by the 3’ portion of <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> is functionally and structurally more important than that encoded by the 5’ portion. Although there was some evidence of purifying selection (<em xmlns="" class="EmphasisTypeItalic">d</em>
                           <sub>N</sub>/<em xmlns="" class="EmphasisTypeItalic">d</em>
                           <sub>S</sub> test) on the 5’ portion of the gene, the signature of purifying selection was considerably stronger on the 3’ portion of the gene (<em xmlns="" class="EmphasisTypeItalic">d</em>
                           <sub>N</sub>/<em xmlns="" class="EmphasisTypeItalic">d</em>
                           <sub>S</sub> and McDonald–Kreitman tests). Several tests support the action of balancing selection within the 5’ portion of the gene. Wall’s <em xmlns="" class="EmphasisTypeItalic">B</em> and <em xmlns="" class="EmphasisTypeItalic">Q</em> tests were significant only for the 5’ portion of the gene. There were also marked peaks of nucleotide diversity, Tajima’s <em xmlns="" class="EmphasisTypeItalic">D</em> and the <em xmlns="" class="EmphasisTypeItalic">d</em>
                           <sub>N</sub>/<em xmlns="" class="EmphasisTypeItalic">d</em>
                           <sub>S</sub> ratio at or around a <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> codon site (within the 5’ portion of the gene) that a previous study had suggested was subject to positive diversifying selection.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec4">
                <h3 xmlns="" class="Heading">Conclusions</h3>
                <p id="Par4" class="Para">Our results suggest that the two portions of the gene have been subject to different selective regimes. Purifying selection appears to have been the main force contributing to the relatively low level of polymorphism within the 3’ portion of the sequence. In contrast, it is possible that balancing selection has contributed to the maintenance of the polymorphism within the 5’ portion of the gene.</p>
              </div></div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords"><li class="c-keywords__item" itemprop="about">
                <em xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="EmphasisTypeItalic">Festuca ovina</em>
              </li><li class="c-keywords__item" itemprop="about">Cytosolic phosphoglucose isomerase</li><li class="c-keywords__item" itemprop="about">
                <em xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="EmphasisTypeItalic">PgiC1</em>
              </li><li class="c-keywords__item" itemprop="about">Nucleotide polymorphism</li><li class="c-keywords__item" itemprop="about">Purifying selection</li></ul></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 SectionTypeIntroduction c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par12" class="Para">Levels of nucleotide polymorphism have been shown to vary greatly between different parts of the genome (e.g. [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>–<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>]), and there may also be variation in the levels of polymorphism within individual genes (e.g. [<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>–<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>]). A non-uniform pattern of nucleotide polymorphism within genes may arise if different types of selective pressure are operating on different regions of the gene (cf. [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>, <span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>]). Different regions of a gene may code for peptides that have different structural or functional significances, and the regions of a gene with more stringent structural and/or functional requirements are expected to be subject to stronger purifying selection [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>] and, therefore, tend to show lower levels of nucleotide polymorphism than regions that are subject to less stringent constraints [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>, <span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>]. Positive directional selection may reduce the levels of local nucleotide polymorphism within a gene [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>], while balancing selection may increase the levels of nucleotide polymorphism at, and in the vicinity of, the selected sites [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>, <span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>]. A classic example of a case where selection results in non-uniform levels of nucleotide polymorphism between different gene regions is that of the major histocompatibility complex (MHC) genes. These genes are crucial for the ability of a vertebrate host’s immune system to detect evolving pathogens, and it is frequently suggested that the maintenance of the high levels of non-synonymous polymorphism in the MHC gene regions encoding the antigen binding site is a reflection of pathogen-driven balancing selection [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>, <span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>]. In addition to selective processes, varying rates of recombination and mutation, as well as stochastic processes, may also contribute to non-uniform levels of nucleotide polymorphism between different regions of a gene (cf. [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>, <span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>]).</p><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par13" class="Para">The <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> gene, which encodes the cytosolic version of the metabolic enzyme phosphoglucose isomerase (PGI, EC 5.3.1.9), in the grass <em xmlns="" class="EmphasisTypeItalic">Festuca ovina</em> L., represents one of the few reported cases in which the levels of nucleotide polymorphism differ substantially between the 3’ and 5’ portions of a gene [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>]. PGI catalyses the second step of glycolysis [<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>], and is also known to have diverse moonlighting functions (see the references in [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>]). The functional PGI enzyme is formed by two monomers, with each monomer being composed of two main domains (the “small domain” and the “large domain”) [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>, <span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>] (Fig.&nbsp;<span class="InternalRef"><a href="#Fig1">1-a</a></span>). High levels of allozyme/isozyme variation have been frequently reported for PGI in many different species [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>]. Observed differences in enzyme activity between PGI variants in a number of species are consistent with observed associations between the PGI variation and environmental variables or life-history traits – suggesting that the loci coding for PGI may be under selection (e.g. [<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>, <span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>]).<figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO1">
                    <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0032-6/MediaObjects/41065_2017_32_Fig1_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0032-6/MediaObjects/41065_2017_32_Fig1_HTML.gif" alt="Fig. 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0032-6/MediaObjects/41065_2017_32_Fig1_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0032-6/MediaObjects/41065_2017_32_Fig1_HTML.gif" alt="Fig. 1"></img></noscript></a>
                  </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 1</span>
                      <p xmlns="" class="SimplePara">The structure of the <em class="EmphasisTypeItalic">PgiC1</em> gene and the 3-D protein structure of its gene product. <strong class="EmphasisTypeBold">a</strong> PGI dimer, coded for by <em class="EmphasisTypeItalic">PgiC1</em>, in <em class="EmphasisTypeItalic">F. ovina</em> and homology modelled in an earlier study [<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>]. One monomer is shown in <em class="EmphasisTypeItalic">yellow</em>. In the other monomer, the large domain is shown in <em class="EmphasisTypeItalic">green</em> and the small domain is shown in <em class="EmphasisTypeItalic">dark blue</em>. The three active site residues (equivalent to Lys516, Glu360, and His391 in <em class="EmphasisTypeItalic">F. ovina</em>) that are directly involved in the PGI isomerization reaction are shown in <em class="EmphasisTypeItalic">red</em>. The rest of the monomer is represented in <em class="EmphasisTypeItalic">light blue</em>. <strong class="EmphasisTypeBold">b</strong> The gene structure is summarized for the part of the <em class="EmphasisTypeItalic">PgiC1</em> gene corresponding to the 1 633&nbsp;bp sequence of the 29 Öland sequences characterized in the present study. <em class="EmphasisTypeItalic">Boxes</em> represent the exons and lines represent introns. The gene structure is scaled according to an earlier published <em class="EmphasisTypeItalic">PgiC1</em> gene sequence (GenBank accession numbers HQ616103). The 5’ and 3’ portions (see Fig.&nbsp;<span xmlns="http://www.w3.org/1999/xhtml" class="InternalRef"><a href="#Fig2">2</a></span>) of the <em class="EmphasisTypeItalic">PgiC1</em> gene that are compared in the present study include, respectively, exons 1–12 and exons 13–22</p>
                    </div></figcaption></figure>
                     </div><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par14" class="Para">
                        <em xmlns="" class="EmphasisTypeItalic">Festuca ovina</em> is a perennial, tussock-forming and outcrossing grass, with wind-dispersed pollen and seeds [<span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>]. The species has a broad ecological amplitude and is widespread in unfertilized grasslands in northern Europe (e.g. [<span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>, <span class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>]). The steppe-like “alvar” grasslands on the Baltic island of Öland (Sweden) are characterized by a fine-scale edaphic mosaic, with moist and dry, and high and low pH microhabitats. Earlier studies suggest that cytosolic PGI isozyme variation in <em xmlns="" class="EmphasisTypeItalic">F. ovina</em> may be involved in fine-scale microhabitat adaptation on Öland [<span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>, <span class="CitationRef"><a href="#CR28" data-toggle="citation">28</a></span>, <span class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>]. Analysis of replicated samples from different alvar sites shows that, despite the fact that <em xmlns="" class="EmphasisTypeItalic">F. ovina</em> is strongly outcrossing, the frequencies of different cytosolic PGI isozyme electromorphs are significantly associated with microhabitat variation in the alvar grasslands and that electromorph frequencies change in response to experimental habitat manipulation [<span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>, <span class="CitationRef"><a href="#CR28" data-toggle="citation">28</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par15" class="Para">In <em xmlns="" class="EmphasisTypeItalic">F. ovina</em>, cytosolic PGI is coded for by two loci, <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> and <em xmlns="" class="EmphasisTypeItalic">PgiC2</em> [<span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>]. The “native” <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> locus is present in all <em xmlns="" class="EmphasisTypeItalic">F. ovina</em> individuals, whereas <em xmlns="" class="EmphasisTypeItalic">PgiC2</em> is only present in some individuals and appears to have been horizontally acquired from a distantly related grass genus [<span class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>, <span class="CitationRef"><a href="#CR31" data-toggle="citation">31</a></span>–<span class="CitationRef"><a href="#CR33" data-toggle="citation">33</a></span>]. Earlier analyses of the <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> gene in <em xmlns="" class="EmphasisTypeItalic">F. ovina</em> suggest that two <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> amino acid codon sites may be affected by positive selection [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>], and SNP (single nucleotide polymorphism) alleles at these two codon sites show significant associations with microhabitat variables in the alvar grasslands (Y Li, B Hansson, M Lönn, HC Prentice, unpublished results).</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par16" class="Para">The uneven distribution of polymorphic nucleotide sites along the <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> gene was noted in an earlier study that included five <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> coding sequences from Skåne, S Sweden [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>]. The longest intron (intron 12, Fig.&nbsp;<span class="InternalRef"><a href="#Fig1">1</a></span>-b) was used as a demarcation point between the polymorphic 5’ portion of the gene and the, substantially less polymorphic, 3’ portion of the gene. The aim of the present study was to investigate the possible evolutionary mechanisms that may have contributed to the contrasting levels of nucleotide polymorphism in the two portions of the <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> gene in <em xmlns="" class="EmphasisTypeItalic">F. ovina</em>. We analysed the levels of <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> nucleotide polymorphism within a larger dataset (29 <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> cDNA sequences) from <em xmlns="" class="EmphasisTypeItalic">F. ovina</em> individuals collected from the alvar grasslands on Öland, and carried out a range of tests to assess the relative importance of different types of selection that may have contributed to the non-uniform pattern of nucleotide polymorphism within <em xmlns="" class="EmphasisTypeItalic">PgiC1</em>.</p><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par17" class="Para">The 3’ portion of <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> in <em xmlns="" class="EmphasisTypeItalic">F. ovina</em> encodes the structurally important large domain and three functionally essential active site residues (Figs.&nbsp;<span class="InternalRef"><a href="#Fig1">1-a</a></span> and <span class="InternalRef"><a href="#Fig2">2</a></span>). The extensive inter-monomer interaction between the large domains of the two monomers is necessary for the formation of a stable PGI dimer [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>] and the three active site residues (equivalent to Glu360, His391 and Lys516 in <em xmlns="" class="EmphasisTypeItalic">F. ovina</em>) participate directly in the isomerization reaction of PGI [<span class="CitationRef"><a href="#CR35" data-toggle="citation">35</a></span>]. If the 3’ portion of <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> codes for products that are subject to greater structural or functional constraints than the products of the 5’ portion of the gene, then a relatively stronger level of purifying selection (i.e. negative selection) may be expected to have contributed to the low level of nucleotide polymorphism within the 3’ portion of <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> in <em xmlns="" class="EmphasisTypeItalic">F. ovina</em>. The 5’ portion of <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> contains the amino acid codon sites 173 and 200 (Fig.&nbsp;<span class="InternalRef"><a href="#Fig2">2</a></span>). If these sites are under balancing selection (i.e. positive intraspecific diversifying selection), as suggested by [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>], then balancing selection targeting the two sites might be expected to contribute to the high level of nucleotide polymorphism within the 5’ portion of <em xmlns="" class="EmphasisTypeItalic">PgiC1</em>. The present study provides support for the prediction that there is a stronger purifying selection on the 3’ portion than on the 5’ portion of <em xmlns="" class="EmphasisTypeItalic">PgiC1</em>, and suggests that there is balancing selection on the 5’ portion of the gene.<figure xmlns="" class="Figure" id="Fig2"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO2">
                    <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0032-6/MediaObjects/41065_2017_32_Fig2_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0032-6/MediaObjects/41065_2017_32_Fig2_HTML.gif" alt="Fig. 2" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0032-6/MediaObjects/41065_2017_32_Fig2_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0032-6/MediaObjects/41065_2017_32_Fig2_HTML.gif" alt="Fig. 2"></img></noscript></a>
                  </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 2</span>
                      <p xmlns="" class="SimplePara">Sliding window analyses of nucleotide diversity (<em class="EmphasisTypeItalic">π</em>
                                    <sub xmlns="http://www.w3.org/1999/xhtml">T</sub>), Tajima’s <em class="EmphasisTypeItalic">D</em> and <em class="EmphasisTypeItalic">ω</em>. The ticks on the x axis represent the boundary of each analysed <em class="EmphasisTypeItalic">PgiC1</em> exon within the <em class="EmphasisTypeItalic">PgiC1</em> coding sequence. In <em class="EmphasisTypeItalic">F. ovina</em>, <em class="EmphasisTypeItalic">PgiC1</em> exons 5–12 encode the small domain of a PGI monomer while exons 13–21 encode the large domain. The two dots on the <em class="EmphasisTypeItalic">x axis</em> show the locations of the two <em class="EmphasisTypeItalic">PgiC1</em> codon sites (173 and 200) that were earlier identified as being candidates for positive diversifying selection [<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>]. The three stars on the <em class="EmphasisTypeItalic">x axis</em> represent the three active site residues (equivalent to Lys516, Glu360, and His391 in <em class="EmphasisTypeItalic">F. ovina</em>) that are directly involved in the PGI isomerization reaction [<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR35" data-toggle="citation">35</a></span>]. The <em class="EmphasisTypeItalic">grey dotted vertical line</em> shows the location of intron 12, which is used as the demarcation point for defining the 5’ and 3’ portions of <em class="EmphasisTypeItalic">PgiC1</em> sequence. The <em class="EmphasisTypeItalic">brown dotted vertical line</em> indicates codon site 200 which is under positive diversifying selection and located at or near to peaks of <em class="EmphasisTypeItalic">π</em>
                                    <sub xmlns="http://www.w3.org/1999/xhtml">T</sub>, Tajima’s <em class="EmphasisTypeItalic">D</em> and <em class="EmphasisTypeItalic">ω</em>
                                 </p>
                    </div></figcaption></figure>
                     </div></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 SectionTypeMaterialsAndMethods c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Methods" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Methods">Methods</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Methods" data-test="collapse-section"><section id="Sec3" class="Section2 RenderAsSection2">
                <h3 class="Heading">Plant material and sequences</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par18" class="Para">The present study examined variation within 29 <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> cDNA sequences (GenBank accession numbers KF487737-KF487765, [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>]) from Öland populations of <em xmlns="" class="EmphasisTypeItalic">F. ovina</em>. The sequences were derived from 15 individuals that were chosen to represent the five cytosolic PGI electromorphs (EMs 1, 2, 4, 5 and 6) that occur most frequently within populations of <em xmlns="" class="EmphasisTypeItalic">F. ovina</em> on Öland [<span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>, <span class="CitationRef"><a href="#CR28" data-toggle="citation">28</a></span>, <span class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>] – with a particular focus on the two most common electromorphs, EM 1 and EM 2 [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>]. The sequences were obtained by, first, synthesizing the total cDNA from the total RNA of each studied <em xmlns="" class="EmphasisTypeItalic">F. ovina</em> individual [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>]. The <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> cDNA was then PCR-amplified from the synthesized total cDNA, and the amplified <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> cDNA was cloned and sequenced [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>]. Two <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> cDNA alleles were acquired from each (diploid) individual, giving a total of 30 alleles from the 15 studied individuals [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>]. However, one of the alleles (GenBank accession number KF487766) contained an aberrant (113&nbsp;bp) insertion [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>] and was excluded from the analyses in the present study, unless specified.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par19" class="Para">Each of the 29 analysed <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> cDNA sequences covers 96% (1 633&nbsp;bp) of the full-length (1701&nbsp;bp, excluding stop codon) <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> coding sequence, and ranges from exon 1 to exon 22 (Fig.&nbsp;<span class="InternalRef"><a href="#Fig1">1</a></span>-b). For comparative purposes, we also downloaded the five Skåne <em xmlns="" class="EmphasisTypeItalic">F. ovina PgiC1</em> coding sequences (GenBank accession numbers DQ225731-DQ225735) which were examined in the earlier study that noted the difference in the levels of polymorphism between the 5’ and 3’ portions of <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>]. These five Skåne sequences represent the common cytosolic PGI isozyme electromorphs 1, 2 and 6, as well as the rare electromorph 8 [<span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>, <span class="CitationRef"><a href="#CR28" data-toggle="citation">28</a></span>]. Each of the five sequences covers 1 182&nbsp;bp, out of the full-length <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> coding sequence [<span class="CitationRef"><a href="#CR32" data-toggle="citation">32</a></span>], and ranges from exons 5 to 11 and from exons 13 to 21.</p>
              </section><section id="Sec4" class="Section2 RenderAsSection2">
                <h3 class="Heading">Analysis of sequence data</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par20" class="Para">The number of polymorphic sites (<em xmlns="" class="EmphasisTypeItalic">S</em>), nucleotide diversity (<em xmlns="" class="EmphasisTypeItalic">π</em>; [<span class="CitationRef"><a href="#CR36" data-toggle="citation">36</a></span>]), Watterson’s estimator of the population mutation rate (<em xmlns="" class="EmphasisTypeItalic">θ</em>
                           <sub>W</sub>; [<span class="CitationRef"><a href="#CR37" data-toggle="citation">37</a></span>]), and the number of haplotypes (<em xmlns="" class="EmphasisTypeItalic">N</em>
                           <sub>h</sub>) were calculated using DnaSP v. 5.10.01 [<span class="CitationRef"><a href="#CR38" data-toggle="citation">38</a></span>]. All the statistics were calculated, separately, for the two portions of the five Skåne <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> sequences (5’: exons 5-11; 3’: exons 13-21) and of the 29 Öland (5’: exons 1-12; 3’: exons 13-22) <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> sequences. Total nucleotide diversity (<em xmlns="" class="EmphasisTypeItalic">π</em>
                           <sub>T</sub>) was estimated separately for each of the 22 studied <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> exons for the 29 Öland <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> sequences. Sliding window analysis of <em xmlns="" class="EmphasisTypeItalic">π</em>
                           <sub>T</sub> was also carried out for the 29 Öland <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> sequences using DnaSP v. 5.10.01 (window length: 100&nbsp;bp; step size: 10&nbsp;bp). The remaining analyses only considered the 29 Öland <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> sequences, unless specified.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par21" class="Para">The level of recombination was estimated (as the minimum number of recombination events, <em xmlns="" class="EmphasisTypeItalic">R</em>
                           <sub>M</sub>, using the method of Hudson RR and Kaplan NL [<span class="CitationRef"><a href="#CR39" data-toggle="citation">39</a></span>] as implemented in DnaSP v. 5.10.01) for each of the two <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> gene portions in the 29 Öland sequences. The level of recombination was also estimated as the population recombination rate (<em xmlns="" class="EmphasisTypeItalic">ρ</em> = 4<em xmlns="" class="EmphasisTypeItalic">N</em>
                           <sub>e</sub>
                           <em xmlns="" class="EmphasisTypeItalic">r</em>, where <em xmlns="" class="EmphasisTypeItalic">N</em>e is the effective population size and <em xmlns="" class="EmphasisTypeItalic">r</em> is the per-generation per-site recombination rate [<span class="CitationRef"><a href="#CR40" data-toggle="citation">40</a></span>]), using the program omegaMap [<span class="CitationRef"><a href="#CR41" data-toggle="citation">41</a></span>]. We used the same procedure as in [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>] to run omegaMap, but used a sliding window of 10 codons to estimate <em xmlns="" class="EmphasisTypeItalic">ρ</em> in the present study. The level of linkage disequilibrium (LD) within <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> was estimated using <em xmlns="" class="EmphasisTypeItalic">r</em>
                           <sup>2</sup> statistics [<span class="CitationRef"><a href="#CR42" data-toggle="citation">42</a></span>], calculated between all pairs of polymorphic sites (excluding a single site that segregates with more than two nucleotides [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>]), using Haploview v. 4.2 [<span class="CitationRef"><a href="#CR43" data-toggle="citation">43</a></span>]. The genotypes of the 15 studied Öland individuals [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>], at each of the analyzed polymorphic sites, were used as input to the Haploview analyses. In order to generate a complete set of genotype data for the 15 individuals, as required for the Haploview analyses, we included the additional sequence (GenBank accession number KF487766) – acquired from the 15 <em xmlns="" class="EmphasisTypeItalic">F. ovina</em> individuals but containing an aberrant insertion that may have resulted from incomplete splicing of the <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> precursor mRNA [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>]. This insertion was removed for the Haploview analyses.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par22" class="Para">The <em xmlns="" class="EmphasisTypeItalic">d</em>N/<em xmlns="" class="EmphasisTypeItalic">d</em>S ratio (<em xmlns="" class="EmphasisTypeItalic">ω</em>) (<em xmlns="" class="EmphasisTypeItalic">dN</em> = non-synonymous substitution rate; <em xmlns="" class="EmphasisTypeItalic">dS</em> = synonymous substitution rate) was estimated, together with <em xmlns="" class="EmphasisTypeItalic">ρ</em> (using omegaMap), for each amino acid codon translated from the <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> sequence. The estimated <em xmlns="" class="EmphasisTypeItalic">ω</em> value was used to examine whether purifying (i.e. negative selection, <em xmlns="" class="EmphasisTypeItalic">ω</em> &lt; 1) or balancing (i.e. positive intraspecific diversifying) selection (<em xmlns="" class="EmphasisTypeItalic">ω</em> &gt; 1) may have contributed to the amounts and patterning of sequence variation within and between the two gene portions for the 29 studied <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> sequences (cf. [<span class="CitationRef"><a href="#CR41" data-toggle="citation">41</a></span>]). Sliding window analysis of <em xmlns="" class="EmphasisTypeItalic">ω</em> was also carried out (manually, on the basis of the results from OmegaMap) with a window length of 99 and a step size of 12.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par23" class="Para">Neutrality tests, including the Hudson-Kreitman-Aguadé (HKA) test [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>], Tajima’s <em xmlns="" class="EmphasisTypeItalic">D</em> test [<span class="CitationRef"><a href="#CR44" data-toggle="citation">44</a></span>], Fay and Wu’s <em xmlns="" class="EmphasisTypeItalic">H</em> test [<span class="CitationRef"><a href="#CR45" data-toggle="citation">45</a></span>], MacDonald and Kreitman’s (MK) test [<span class="CitationRef"><a href="#CR46" data-toggle="citation">46</a></span>] and Wall’s <em xmlns="" class="EmphasisTypeItalic">B</em> and <em xmlns="" class="EmphasisTypeItalic">Q</em> tests [<span class="CitationRef"><a href="#CR47" data-toggle="citation">47</a></span>] were also used to examine whether selection may have contributed to the amounts and patterning of sequence variation within each of the two <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> gene portions in the 29 sequences<em xmlns="" class="EmphasisTypeItalic">.</em> All the neutrality tests were carried out using DnaSP v. 5.10.01. The significances of the Tajima’s <em xmlns="" class="EmphasisTypeItalic">D</em>, Fay and Wu’s <em xmlns="" class="EmphasisTypeItalic">H</em>, and Wall’s <em xmlns="" class="EmphasisTypeItalic">B</em> and <em xmlns="" class="EmphasisTypeItalic">Q</em> were conservatively estimated (without allowing for recombination), using 10 000 coalescent simulations and on the basis of the observed number of segregating sites. A single <em xmlns="" class="EmphasisTypeItalic">PgiC</em> cDNA sequence from <em xmlns="" class="EmphasisTypeItalic">F. altissima</em> (GenBank accession number DQ225740), encoding cytosolic PGI, was used as the outgroup for the HKA test, Fay and Wu’s <em xmlns="" class="EmphasisTypeItalic">H</em> test and the MK test. We used the HKA test to compare the 5’- and 3’-portions of <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> in terms of intraspecific polymorphism (within <em xmlns="" class="EmphasisTypeItalic">F. ovina</em>) and interspecific divergence (between <em xmlns="" class="EmphasisTypeItalic">F. ovina</em> and <em xmlns="" class="EmphasisTypeItalic">F. altissima</em>)<em xmlns="" class="EmphasisTypeItalic">.</em> In the MK test, we compared the ratio of <em xmlns="" class="EmphasisTypeItalic">D</em>
                           <sub>N</sub>/<em xmlns="" class="EmphasisTypeItalic">D</em>
                           <sub>S</sub> (<em xmlns="" class="EmphasisTypeItalic">D</em>
                           <sub>N</sub> or <em xmlns="" class="EmphasisTypeItalic">D</em>
                           <sub>S</sub> = the number of fixed non-synonymous [for <em xmlns="" class="EmphasisTypeItalic">D</em>
                           <sub>N</sub>] or synonymous [for <em xmlns="" class="EmphasisTypeItalic">D</em>
                           <sub>S</sub>] substitutions per gene between <em xmlns="" class="EmphasisTypeItalic">F. ovina</em> and <em xmlns="" class="EmphasisTypeItalic">F. altissima</em>), with the ratio of <em xmlns="" class="EmphasisTypeItalic">P</em>
                           <sub>N</sub>/<em xmlns="" class="EmphasisTypeItalic">P</em>
                           <sub>S</sub> (<em xmlns="" class="EmphasisTypeItalic">P</em>
                           <sub>N</sub> or <em xmlns="" class="EmphasisTypeItalic">P</em>
                           <sub>S</sub> = the number of non-synonymous [for <em xmlns="" class="EmphasisTypeItalic">P</em>
                           <sub>N</sub>] or synonymous [for <em xmlns="" class="EmphasisTypeItalic">P</em>
                           <sub>S</sub>] polymorphic sites per gene within <em xmlns="" class="EmphasisTypeItalic">F. ovin</em>a). The degree of synonymous divergence (<em xmlns="" class="EmphasisTypeItalic">K</em>
                           <sub>S</sub> [JC]: Jukes-Cantor corrected number of synonymous substitutions per synonymous site) between the outgroup <em xmlns="" class="EmphasisTypeItalic">PgiC</em> sequence and the 29 <em xmlns="" class="EmphasisTypeItalic">F. ovina PgiC1</em> sequences was 0.240. Sliding window analyses were also carried out, respectively, for Tajima’s <em xmlns="" class="EmphasisTypeItalic">D</em>, <em xmlns="" class="EmphasisTypeItalic">ω</em>, Fay and Wu’s <em xmlns="" class="EmphasisTypeItalic">H</em>, as well as for <em xmlns="" class="EmphasisTypeItalic">K</em>a/<em xmlns="" class="EmphasisTypeItalic">K</em>s (the number of nonsynonymous substitutions per nonsynonymous site/the number of sysnonymous substitutions per synonymous site), with a window length of 100&nbsp;bp and a step size of 10&nbsp;bp using DnaSP v. 5.10.01.</p>
              </section><section id="Sec5" class="Section2 RenderAsSection2">
                <h3 class="Heading">Analysis of the evolutionary conservation of amino acid sites</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par24" class="Para">The degree of evolutionary conservation at each of the respective PGI amino acid sites corresponding to the <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> translated amino acid sites was estimated, on the basis of the phylogenetic relationships among a large set of homologous sequences, from a wide range of different species, using the online application ConSurf Server [<span class="CitationRef"><a href="#CR48" data-toggle="citation">48</a></span>]. The database UniRef90 [<span class="CitationRef"><a href="#CR49" data-toggle="citation">49</a></span>] was searched for sequences that were homologous with the <em xmlns="" class="EmphasisTypeItalic">F. ovina PgiC1</em> input sequence, using CSI-BLAST [<span class="CitationRef"><a href="#CR50" data-toggle="citation">50</a></span>] (cutoff E-value = 0.0001; number of interactions = 3; maximum homologs to collect = 150). Within CSI-BLAST, redundant sequences were filtered out by clustering blast hits with a sequence identity of 95% or more and only using one representative of each cluster in the analysis. BLAST hits that shared a sequence identity of less than 35% with the input sequence were ignored. A multiple-species alignment (Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>: File S1) of the acquired homologous sequences was constructed using MAFFT [<span class="CitationRef"><a href="#CR51" data-toggle="citation">51</a></span>, <span class="CitationRef"><a href="#CR52" data-toggle="citation">52</a></span>], and this alignment was then used to build a phylogenetic tree using the neighbour-joining algorithm as implemented in the Rate4Site program [<span class="CitationRef"><a href="#CR53" data-toggle="citation">53</a></span>]. The level of evolutionary conservation was then estimated, as a conservation score for each amino acid site using an empirical Bayesian algorithm [<span class="CitationRef"><a href="#CR54" data-toggle="citation">54</a></span>] implemented in the ConSurf Server. The lower the conservation score, the more evolutionarily conserved are the amino acid residues at that specific site. The translated amino acid sequence from a <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> sequence (GenBank accession number KF487738), representing the most common <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> sequence in <em xmlns="" class="EmphasisTypeItalic">F. ovina</em>, was used as the input to the ConSurf Server.</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec6" class="Section1 RenderAsSection1 SectionTypeResults c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Results" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Results">Results</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Results" data-test="collapse-section"><section id="Sec7" class="Section2 RenderAsSection2">
                <h3 class="Heading">Sequence variation</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par25" class="Para">The analyses of the five Skåne <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> coding sequences confirmed the earlier finding [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>] that the 5’ portion of the gene (<em xmlns="" class="EmphasisTypeItalic">L</em> [sequence length] = 570, <em xmlns="" class="EmphasisTypeItalic">S</em> = 30, <em xmlns="" class="EmphasisTypeItalic">N</em>
                           <sub>h</sub> = 5, <em xmlns="" class="EmphasisTypeItalic">π</em> = 0.025, <em xmlns="" class="EmphasisTypeItalic">θ</em>
                           <sub>W</sub> = 0.025) was considerably more polymorphic than the 3’ portion (<em xmlns="" class="EmphasisTypeItalic">L</em> = 612, <em xmlns="" class="EmphasisTypeItalic">S</em> = 3, <em xmlns="" class="EmphasisTypeItalic">N</em>
                           <sub>h</sub> = 4, <em xmlns="" class="EmphasisTypeItalic">π</em> = 0.002, <em xmlns="" class="EmphasisTypeItalic">θ</em>
                           <sub>W</sub> = 0.002). Also in agreement with the observed pattern of sequence polymorphism within the five Skåne <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> sequences, the analyses of the 29 Öland <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> cDNA sequences showed that the 5’ portion of the sequence (<em xmlns="" class="EmphasisTypeItalic">π</em>
                           <sub>T</sub> = 0.019, <em xmlns="" class="EmphasisTypeItalic">θ</em>
                           <sub>W</sub> = 0.020) was substantially more polymorphic than the 3’ portion of the sequence (<em xmlns="" class="EmphasisTypeItalic">π</em>
                           <sub>T</sub> = 0.004, <em xmlns="" class="EmphasisTypeItalic">θ</em>
                           <sub>W</sub> = 0.007) (Fig.&nbsp;<span class="InternalRef"><a href="#Fig2">2</a></span>, Table&nbsp;<span class="InternalRef"><a href="#Tab1">1</a></span>). There was a significant difference between the <em xmlns="" class="EmphasisTypeItalic">π</em>
                           <sub>T</sub> values for each of the 12 exons (exons 1–12) within the 5’ portion of the <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> gene in the 29 sequences, and those for each of the 10 exons (exons 13–22) within the 3’ portion of the 29 sequences (Wilcoxon rank sum test; W = 96, <em xmlns="" class="EmphasisTypeItalic">P</em> = 0.019, Additional file <span class="InternalRef"><a href="#MOESM2">2</a></span>: Table S1). The sliding window analyses of <em xmlns="" class="EmphasisTypeItalic">π</em>
                           <sub>T</sub> showed that the <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> codon site 200 (under positive diversifying selection [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>]) was within the highest peak of <em xmlns="" class="EmphasisTypeItalic">π</em>
                           <sub>T</sub> at exon 8 within the 5’ portion of the gene (Fig.&nbsp;<span class="InternalRef"><a href="#Fig2">2</a></span>).<figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 1</span>
                        <p xmlns="" class="SimplePara">Analyses of the 3’ and 5’ portions of 29 <em class="EmphasisTypeItalic">PgiC1</em> sequences from Öland <em class="EmphasisTypeItalic">F. ovina</em>
                                    </p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col></col><col></col><col></col><col></col><col></col><col char="."></col><col char="."></col><col></col><col></col><col></col><col></col><col char="."></col></colgroup><thead><tr class="header"><th>&nbsp;</th><th><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">L</em>
                            </p></th><th><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">S</em>
                            </p></th><th><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">N</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">h</sub>
                            </p></th><th><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">π</em> (×10<sup xmlns="http://www.w3.org/1999/xhtml">−3</sup>)</p></th><th><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">θ</em>
                                            <sub xmlns="http://www.w3.org/1999/xhtml">W</sub> (×10<sup xmlns="http://www.w3.org/1999/xhtml">−3</sup>)</p></th><th><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">R</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">M</sub>
                            </p></th><th char="."><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">ρ</em>
                            </p></th><th char="."><p xmlns="" class="SimplePara">Mean <em class="EmphasisTypeItalic">ω</em>
                                          </p></th><th><p xmlns="" class="SimplePara">Tajima’s <em class="EmphasisTypeItalic">D</em>
                                          </p></th><th><p xmlns="" class="SimplePara">Fay &amp; Wu’s <em class="EmphasisTypeItalic">H</em>
                                          </p></th><th><p xmlns="" class="SimplePara">Wall’s <em class="EmphasisTypeItalic">B</em>
                                          </p></th><th><p xmlns="" class="SimplePara">Wall’s <em class="EmphasisTypeItalic">Q</em>
                                          </p></th><th char="."><p xmlns="" class="SimplePara">Mean conservation score</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">5’ portion</p></td><td><p xmlns="" class="SimplePara">872</p></td><td><p xmlns="" class="SimplePara">68</p></td><td><p xmlns="" class="SimplePara">21</p></td><td><p xmlns="" class="SimplePara">19</p></td><td><p xmlns="" class="SimplePara">20</p></td><td><p xmlns="" class="SimplePara">20</p></td><td char="."><p xmlns="" class="SimplePara">0.383</p></td><td char="."><p xmlns="" class="SimplePara">0.280</p></td><td><p xmlns="" class="SimplePara">−0.195 n.s.</p></td><td><p xmlns="" class="SimplePara">3.355 n.s.</p></td><td><p xmlns="" class="SimplePara">0.015*</p></td><td><p xmlns="" class="SimplePara">0.030*</p></td><td char="."><p xmlns="" class="SimplePara">0.105</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">3’ portion</p></td><td><p xmlns="" class="SimplePara">761</p></td><td><p xmlns="" class="SimplePara">20</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">4</p></td><td><p xmlns="" class="SimplePara">7</p></td><td><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">0.026</p></td><td char="."><p xmlns="" class="SimplePara">0.128</p></td><td><p xmlns="" class="SimplePara">−1.363 n.s.</p></td><td><p xmlns="" class="SimplePara">−1.768 n.s.</p></td><td><p xmlns="" class="SimplePara">0.000 n.s.</p></td><td><p xmlns="" class="SimplePara">0.000 n.s.</p></td><td char="."><p xmlns="" class="SimplePara">−0.121</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">The table shows the lengths of the 5’ and 3’ portions of the sequence (<em class="EmphasisTypeItalic">L</em>), the number of segregating sites (<em class="EmphasisTypeItalic">S</em>), the number of haplotypes (<em class="EmphasisTypeItalic">N</em>
                                    <sub xmlns="http://www.w3.org/1999/xhtml">h</sub>), the total nucleotide diversity (<em class="EmphasisTypeItalic">π</em>), the total Watterson’s estimator of population mutation rate (per site) (<em class="EmphasisTypeItalic">θ</em>
                                    <sub xmlns="http://www.w3.org/1999/xhtml">W</sub>), the minimum number of recombination events (<em class="EmphasisTypeItalic">R</em>
                                    <sub xmlns="http://www.w3.org/1999/xhtml">M</sub>), the population recombination parameter (<em class="EmphasisTypeItalic">ρ</em>) (per site), the average <em class="EmphasisTypeItalic">ω</em> ratio, Tajima’s <em class="EmphasisTypeItalic">D</em>, Fay &amp; Wu’s <em class="EmphasisTypeItalic">H</em>, and Wall’s <em class="EmphasisTypeItalic">B</em> and <em class="EmphasisTypeItalic">Q</em>, as well as the mean conservation score (the smaller the value, the more conserved) estimated using the online ConSurf Server. Because parts of the coding sequence are not available for the outgroup sequence from <em class="EmphasisTypeItalic">F. altissima</em>, the 5’ and 3’ portions of <em class="EmphasisTypeItalic">F. ovina PgiC1</em> that were considered in the Fay and Wu’s <em class="EmphasisTypeItalic">H</em> test span, respectively, coding sequence nucleotide positions 259-828 &amp; 919-1530</p>
                      <p xmlns="" class="SimplePara">*0.01 &lt; <em class="EmphasisTypeItalic">P</em> &lt; 0.05; n.s. non-significant</p>
                    </div></figure>
                        </div>
              </section><section id="Sec8" class="Section2 RenderAsSection2">
                <h3 class="Heading">Recombination and linkage disequilibrium</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par26" class="Para">The analyses of the 29 <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> sequences from Öland revealed a high overall level of recombination (<em xmlns="" class="EmphasisTypeItalic">R</em>
                           <sub>M</sub> = 22; <em xmlns="" class="EmphasisTypeItalic">ρ</em> = 0.217, Additional file <span class="InternalRef"><a href="#MOESM3">3</a></span>: Table S2). The level of recombination in the highly polymorphic 5’ portion (<em xmlns="" class="EmphasisTypeItalic">R</em>
                           <sub>M</sub> = 20; <em xmlns="" class="EmphasisTypeItalic">ρ</em> = 0.383) was substantially higher than that for the less variable 3’ portion of the gene (<em xmlns="" class="EmphasisTypeItalic">R</em>
                           <sub>M</sub> = 1; <em xmlns="" class="EmphasisTypeItalic">ρ</em> = 0.026) (Table&nbsp;<span class="InternalRef"><a href="#Tab1">1</a></span> and Fig.&nbsp;<span class="InternalRef"><a href="#Fig3">3_a</a></span>). However, the matrix of <em xmlns="" class="EmphasisTypeItalic">r</em>
                           <sup>2</sup> values (Fig.&nbsp;<span class="InternalRef"><a href="#Fig3">3_b</a></span>) shows that there is a low level of LD throughout the entire <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> gene, with no “strong LD” blocks (cf. [<span class="CitationRef"><a href="#CR55" data-toggle="citation">55</a></span>]).<figure xmlns="" class="Figure" id="Fig3"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO3">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0032-6/MediaObjects/41065_2017_32_Fig3_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0032-6/MediaObjects/41065_2017_32_Fig3_HTML.gif" alt="Fig. 3" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0032-6/MediaObjects/41065_2017_32_Fig3_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0032-6/MediaObjects/41065_2017_32_Fig3_HTML.gif" alt="Fig. 3"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 3</span>
                        <p xmlns="" class="SimplePara">Levels of recombination and linkage disequilibrium (LD) within the <em class="EmphasisTypeItalic">F. ovina PgiC1</em> sequence. <strong class="EmphasisTypeBold">a</strong> Population recombination rate (<em class="EmphasisTypeItalic">ρ</em>) across <em class="EmphasisTypeItalic">PgiC1</em>. The <em class="EmphasisTypeItalic">heavy line</em> represents the mean value of <em class="EmphasisTypeItalic">ρ</em>, while the <em class="EmphasisTypeItalic">thin lines</em> represent the upper and lower 95% HPD (highest posterior density) interval bounds for the posterior distribution of <em class="EmphasisTypeItalic">ρ</em>. <strong class="EmphasisTypeBold">b</strong> LD matrix for <em class="EmphasisTypeItalic">PgiC1</em>. The level of LD is measured by the pairwise correlation coefficient <em class="EmphasisTypeItalic">r</em>
                                       <sup xmlns="http://www.w3.org/1999/xhtml">2</sup> values [<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR42" data-toggle="citation">42</a></span>] for all the polymorphic nucleotide sites (except for one that segregates into more than two nucleotides [<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>]). Shades of <em class="EmphasisTypeItalic">grey</em> indicate the <em class="EmphasisTypeItalic">r</em>
                                       <sup xmlns="http://www.w3.org/1999/xhtml">2</sup> values, ranging from <em class="EmphasisTypeItalic">r</em>
                                       <sup xmlns="http://www.w3.org/1999/xhtml">2</sup> = 0 (<em class="EmphasisTypeItalic">white</em>) to <em class="EmphasisTypeItalic">r</em>
                                       <sup xmlns="http://www.w3.org/1999/xhtml">2</sup> = 1 (<em class="EmphasisTypeItalic">black</em>). The proportional spacing of the polymorphic sites, which is scaled according to an earlier published <em class="EmphasisTypeItalic">PgiC1</em> gene sequence (GenBank accession number HQ616103), is indicated by <em class="EmphasisTypeItalic">black vertical lines</em> on a <em class="EmphasisTypeItalic">white horizontal bar</em> (shown above the LD matrix)</p>
                      </div></figcaption></figure>
                        </div>
              </section><section id="Sec9" class="Section2 RenderAsSection2">
                <h3 class="Heading">The <em class="EmphasisTypeItalic">ω</em> and neutrality tests</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par27" class="Para">The fact that the overall <em xmlns="" class="EmphasisTypeItalic">ω</em> (<em xmlns="" class="EmphasisTypeItalic">d</em>N/<em xmlns="" class="EmphasisTypeItalic">d</em>S ratio) value for the entire <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> sequence was substantially lower than 1 (mean <em xmlns="" class="EmphasisTypeItalic">ω</em> over all the studied amino acid codons = 0.209, Additional file <span class="InternalRef"><a href="#MOESM3">3</a></span>: Table S2) indicates that purifying selection has acted on the overall sequence. Purifying selection is also indicated for both the 5’ portion (291 amino acid codons; average <em xmlns="" class="EmphasisTypeItalic">ω</em> = 0.280) and the 3’ portion (253 amino acid codons; average <em xmlns="" class="EmphasisTypeItalic">ω</em> = 0.128) of the sequence. The fact that codons within the 3’ portion of the <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> sequence had a significantly lower <em xmlns="" class="EmphasisTypeItalic">ω</em> than those in the 5’ portion (Wilcoxon rank sum test; W = 42229, <em xmlns="" class="EmphasisTypeItalic">P</em> = 0.003) suggests that the 3’ portion may be under stronger purifying selection than the 5’ portion. The sliding window analyses of <em xmlns="" class="EmphasisTypeItalic">ω</em> showed a marked plateau of <em xmlns="" class="EmphasisTypeItalic">ω</em> values (between exons 8-9 within the 5’ portion) around the <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> codon site 200 (Fig.&nbsp;<span class="InternalRef"><a href="#Fig2">2</a></span>), that a previous study had suggested was subject to positive diversifying selection [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>]. There were 40 segregating sites out of the total of 570 nucleotide sites included in the analysis of the 5’ portion, whereas there were only 14 segregating sites out of 612 sites in the 3’ portion (Additional file <span class="InternalRef"><a href="#MOESM4">4</a></span>: Table S3). The levels of interspecific DNA divergence between the two gene portions were similar: 29.6 nucleotide differences among 570 sites for the 5’ portion, and 35.8 nucleotide differences among 612 sites for the 3’ portion (Additional file <span class="InternalRef"><a href="#MOESM4">4</a></span>: Table S3). However the HKA test didn’t reject the null neutral model (<em xmlns="" class="EmphasisTypeItalic">P</em> = 0.064, Additional file <span class="InternalRef"><a href="#MOESM4">4</a></span>: Table S3).</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par28" class="Para">Wall’s <em xmlns="" class="EmphasisTypeItalic">B</em> and <em xmlns="" class="EmphasisTypeItalic">Q</em> values were significant for the 5’ portion of the <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> sequence (Table&nbsp;<span class="InternalRef"><a href="#Tab1">1</a></span>), indicating an excess of LD between adjacent segregating sites, which may reflect balancing selection (cf. [<span class="CitationRef"><a href="#CR47" data-toggle="citation">47</a></span>, <span class="CitationRef"><a href="#CR56" data-toggle="citation">56</a></span>]). Wall’s <em xmlns="" class="EmphasisTypeItalic">B</em> and <em xmlns="" class="EmphasisTypeItalic">Q</em> values were non-significant for the 3’portion of the <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> sequence (Table&nbsp;<span class="InternalRef"><a href="#Tab1">1</a></span>). There was a negative, but non-significant (<em xmlns="" class="EmphasisTypeItalic">P</em> = 0.064, Table&nbsp;<span class="InternalRef"><a href="#Tab1">1</a></span>) Tajima’s <em xmlns="" class="EmphasisTypeItalic">D</em> for the 3’ portion of the <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> sequence, while the Tajima’s <em xmlns="" class="EmphasisTypeItalic">D</em> for the 5’ portion was near zero (<em xmlns="" class="EmphasisTypeItalic">D</em> = -0.195, Table&nbsp;<span class="InternalRef"><a href="#Tab1">1</a></span>). The sliding window analyses of the Tajima’s <em xmlns="" class="EmphasisTypeItalic">D</em> showed that the highest peak of <em xmlns="" class="EmphasisTypeItalic">D</em> was located at the <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> codon site 200 (exon 8) (Fig.&nbsp;<span class="InternalRef"><a href="#Fig2">2</a></span>) which was identified as a potential target of positive diversifying selection in an earlier study [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>]. The <em xmlns="" class="EmphasisTypeItalic">D</em> value at this peak (<em xmlns="" class="EmphasisTypeItalic">D</em> = 1.439) was near-significant (<em xmlns="" class="EmphasisTypeItalic">P</em> = 0.060; post hoc significance test without correction for multiple tests). The Fay &amp; Wu’s <em xmlns="" class="EmphasisTypeItalic">H</em> values were non-significant for both the 5’ (<em xmlns="" class="EmphasisTypeItalic">P</em> = 0.669) and the 3’ (<em xmlns="" class="EmphasisTypeItalic">P</em> = 0.148) portions of the <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> sequence (Table&nbsp;<span class="InternalRef"><a href="#Tab1">1</a></span>). The sliding window analysis of Fay and Wu’s <em xmlns="" class="EmphasisTypeItalic">H</em> showed that the deepest valley of <em xmlns="" class="EmphasisTypeItalic">H</em> was at exon 16 (within the 3’ portion), near which a striking valley of Tajima’s <em xmlns="" class="EmphasisTypeItalic">D</em> was also observed (Additional file <span class="InternalRef"><a href="#MOESM5">5</a></span>: Figure S1). Both <em xmlns="" class="EmphasisTypeItalic">D</em> and <em xmlns="" class="EmphasisTypeItalic">H</em> had negative values in these valleys (Additional file <span class="InternalRef"><a href="#MOESM5">5</a></span>: Figure S1), and according to the post hoc tests of significance (not subject to multiple test correction), the <em xmlns="" class="EmphasisTypeItalic">H</em> was significant (<em xmlns="" class="EmphasisTypeItalic">H</em> = -1.862, <em xmlns="" class="EmphasisTypeItalic">P</em> = 0.006), while <em xmlns="" class="EmphasisTypeItalic">D</em> was near-significant (<em xmlns="" class="EmphasisTypeItalic">D</em> = -1.509, <em xmlns="" class="EmphasisTypeItalic">P</em> = 0.095).</p>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par29" class="Para">The MK test rejected the neutral null model for the 3’ portion of the <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> sequence (Fisher’s exact test, <em xmlns="" class="EmphasisTypeItalic">P</em> = 0.018). The significant MK test reflects a lower <em xmlns="" class="EmphasisTypeItalic">D</em>
                           <sub>N</sub>/<em xmlns="" class="EmphasisTypeItalic">D</em>
                           <sub>S</sub> ratio (0.064) relative to the <em xmlns="" class="EmphasisTypeItalic">P</em>
                           <sub>N</sub>/<em xmlns="" class="EmphasisTypeItalic">P</em>
                           <sub>S</sub> ratio (0.556), which is consistent with purifying selection within this portion of the gene (Table&nbsp;<span class="InternalRef"><a href="#Tab2">2</a></span>). The MK test for the 5’ portion of the sequence was non-significant (<em xmlns="" class="EmphasisTypeItalic">D</em>
                           <sub>N</sub>/<em xmlns="" class="EmphasisTypeItalic">D</em>
                           <sub>S</sub> = 0.235; <em xmlns="" class="EmphasisTypeItalic">P</em>
                           <sub>N</sub>/<em xmlns="" class="EmphasisTypeItalic">P</em>
                           <sub>S</sub> = 0.323; Fisher’s exact test, <em xmlns="" class="EmphasisTypeItalic">P</em> = 0.755, Table&nbsp;<span class="InternalRef"><a href="#Tab2">2</a></span>).<figure xmlns="" id="Tab2" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 2</span>
                        <p xmlns="" class="SimplePara">MacDonald-Kreitman tests for the 3’ and 5’ portions of <em class="EmphasisTypeItalic">F. ovina PgiC1</em>
                                    </p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col></col><col></col><col></col><col></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Gene</p></th><th colspan="2"><p xmlns="" class="SimplePara">Synonymous</p></th><th colspan="2"><p xmlns="" class="SimplePara">Non-synonymous</p></th><th><p xmlns="" class="SimplePara">Fisher’s exact test</p></th></tr><tr class="header"><th><p xmlns="" class="SimplePara">Portion</p></th><th><p xmlns="" class="SimplePara">Fixed<sup xmlns="http://www.w3.org/1999/xhtml">a</sup> (<em class="EmphasisTypeItalic">D</em>
                                            <sub xmlns="http://www.w3.org/1999/xhtml">S</sub>)</p></th><th><p xmlns="" class="SimplePara">Poly<sup xmlns="http://www.w3.org/1999/xhtml">b</sup> (<em class="EmphasisTypeItalic">P</em>
                                            <sub xmlns="http://www.w3.org/1999/xhtml">S</sub>)</p></th><th><p xmlns="" class="SimplePara">Fixed<sup xmlns="http://www.w3.org/1999/xhtml">a</sup> (<em class="EmphasisTypeItalic">D</em>
                                            <sub xmlns="http://www.w3.org/1999/xhtml">N</sub>)</p></th><th><p xmlns="" class="SimplePara">Poly<sup xmlns="http://www.w3.org/1999/xhtml">b</sup> (<em class="EmphasisTypeItalic">P</em>
                                            <sub xmlns="http://www.w3.org/1999/xhtml">N</sub>)</p></th><th><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">P-</em>value</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">5’ portion</p></td><td><p xmlns="" class="SimplePara">17</p></td><td><p xmlns="" class="SimplePara">31</p></td><td><p xmlns="" class="SimplePara">4</p></td><td><p xmlns="" class="SimplePara">10</p></td><td><p xmlns="" class="SimplePara">0.755 n.s.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">3’ portion</p></td><td><p xmlns="" class="SimplePara">31</p></td><td><p xmlns="" class="SimplePara">9</p></td><td><p xmlns="" class="SimplePara">2</p></td><td><p xmlns="" class="SimplePara">5</p></td><td><p xmlns="" class="SimplePara">0.018*</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">Because parts of the coding sequence are not available for the outgroup sequence from <em class="EmphasisTypeItalic">F. altissima</em>, the 5’ and 3’ portions of <em class="EmphasisTypeItalic">F. ovina PgiC1</em> that were considered in the test span, respectively, coding sequence nucleotide positions 259-828 &amp; 919-1530</p>
                      <p xmlns="" class="SimplePara">
                                    <sup xmlns="http://www.w3.org/1999/xhtml">a</sup>The number of fixed substitutions (per gene) between <em class="EmphasisTypeItalic">F. ovina</em> and the outgroup (the number of nucleotide sites that are fixed for different nucleotides in <em class="EmphasisTypeItalic">F. ovina</em> and the outgroup)</p>
                      <p xmlns="" class="SimplePara">
                                    <sup xmlns="http://www.w3.org/1999/xhtml">b</sup>The number of nucleotide polymorphic sites (per gene) within <em class="EmphasisTypeItalic">F. ovina</em>
                                 </p>
                      <p xmlns="" class="SimplePara">*0.01 &lt; <em class="EmphasisTypeItalic">P</em> &lt; 0.05; n.s. non-significant</p>
                    </div></figure>
                        </div>
              </section><section id="Sec10" class="Section2 RenderAsSection2">
                <h3 class="Heading">Degree of amino acid site conservation</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par30" class="Para">The PGI amino acid sites showed a tendency to be less evolutionarily conserved within the region corresponding to the 5’ portion of the translated <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> amino acid sequence (291 amino acid sites; mean conservation score = 0.105) than those corresponding to the 3’ portion of the translated <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> amino acid sequence (253 amino acid sites; mean conservation score = -0.121) (Wilcoxon rank sum test; W = 39886, <em xmlns="" class="EmphasisTypeItalic">P</em> = 0.093) (Table&nbsp;<span class="InternalRef"><a href="#Tab1">1</a></span>, Additional file <span class="InternalRef"><a href="#MOESM6">6</a></span>: Table S4).</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec11" class="Section1 RenderAsSection1 SectionTypeDiscussion c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Discussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Discussion">Discussion</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Discussion" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par31" class="Para">Analyses of the 29 Öland <em xmlns="" class="EmphasisTypeItalic">F. ovina PgiC1</em> cDNA sequences in the present study, together with the analyses of the five Skåne <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> sequences, shows that the nucleotide polymorphism is not evenly distributed within the <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> gene (Fig.&nbsp;<span class="InternalRef"><a href="#Fig2">2</a></span>). The 5’ portion of the <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> sequence is substantially more polymorphic than the 3’ portion, and our analyses suggest that the difference in the level of polymorphism may have resulted from different selective regimes in the two portions of the gene.</p><section id="Sec12" class="Section2 RenderAsSection2">
                <h3 class="Heading">Which evolutionary mechanisms may have contributed to the relatively low level of nucleotide polymorphism within the 3’ portion of the <em class="EmphasisTypeItalic">PgiC1</em> sequence?</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par32" class="Para">The parts of a protein that are more important for the stability and/or function of an enzyme are likely to be subject to stronger purifying selection [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>] and, therefore, tend to exhibit a lower level of intraspecific polymorphism than the parts of the protein with less stringent functional and structural requirement [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>, <span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>]. In <em xmlns="" class="EmphasisTypeItalic">F. ovina</em>, the 3’ portion of the <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> sequence encodes a peptide that includes the structurally important large domain of the PGI monomer [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>] and the three most conserved, functionally essential, active site residues [<span class="CitationRef"><a href="#CR35" data-toggle="citation">35</a></span>]. The fact that the peptide translated from the 3’ portion of <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> contains important components of the 3-D structure of PGI suggests that this peptide may have a greater overall importance for the function of PGI than the peptide translated from the 5’ portion of <em xmlns="" class="EmphasisTypeItalic">PgiC1</em>. The suggested difference in the functional and structural significance of the translated peptides between the two portions of <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> is supported by the estimated amino acid conservation scores in the present study, which show that the PGI amino acid sites corresponding to the 3’ portion of the <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> sequence tend to be evolutionarily more conserved than those of the 5’ portion in a wide range of species (cf. [<span class="CitationRef"><a href="#CR48" data-toggle="citation">48</a></span>]).</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par33" class="Para">If the peptide translated from the 3’ portion of <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> is more important for the function of the PGI enzyme than the peptide from the 5’ portion, then the 3’ portion of the gene may be expected to be under stronger purifying selection than the 5’ portion (cf. [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>, <span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>]). In line with this expectation, the average <em xmlns="" class="EmphasisTypeItalic">ω</em> value for the 3’ portion of <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> was considerably lower than that for the 5’ portion. The average values for both portions were much lower than 1 (suggesting purifying selection [<span class="CitationRef"><a href="#CR41" data-toggle="citation">41</a></span>]). The fact that the value of Tajima’s <em xmlns="" class="EmphasisTypeItalic">D</em> was more strongly negative for the 3’ portion (<em xmlns="" class="EmphasisTypeItalic">D</em> = -1.363; <em xmlns="" class="EmphasisTypeItalic">P</em> = 0.064) than for the 5’ portion (<em xmlns="" class="EmphasisTypeItalic">D</em> = -0.195; <em xmlns="" class="EmphasisTypeItalic">P</em> = 0.484) of the <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> sequence, is also consistent with the 3’ portion being under stronger purifying selection than the 5’ portion of the sequence (cf. [<span class="CitationRef"><a href="#CR57" data-toggle="citation">57</a></span>]). The significant MK test result for the 3’ portion and the non-significance of the test for the 5’ portion of the <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> sequence may, again, suggest that the 3’ portion is under stronger purifying selection than the 5’ portion of the sequence. The significant MK test for the 3’ portion of <em xmlns="" class="EmphasisTypeItalic">PgiC1</em>, with a lower ratio of fixed non-synonymous/synonymous substitutions between species (<em xmlns="" class="EmphasisTypeItalic">D</em>
                           <sub>
                    <em xmlns="" class="EmphasisTypeItalic">N</em>
                  </sub>/<em xmlns="" class="EmphasisTypeItalic">D</em>
                           <sub>
                    <em xmlns="" class="EmphasisTypeItalic">S</em>
                  </sub> = 0.064) than the ratio of non-synonymous/synonymous polymorphism within species (<em xmlns="" class="EmphasisTypeItalic">P</em>
                           <sub>
                    <em xmlns="" class="EmphasisTypeItalic">N</em>
                  </sub>/<em xmlns="" class="EmphasisTypeItalic">P</em>
                           <sub>
                    <em xmlns="" class="EmphasisTypeItalic">S</em>
                  </sub> = 0.556), suggests purifying selection, where the within-species nonsynonymous polymorphism that is maintained in selection–mutation balance consists mainly of weakly deleterious mutations [<span class="CitationRef"><a href="#CR58" data-toggle="citation">58</a></span>].</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par34" class="Para">Because the PGI protein structural elements are closely similar in a wide range of organisms (e.g. [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>, <span class="CitationRef"><a href="#CR35" data-toggle="citation">35</a></span>, <span class="CitationRef"><a href="#CR59" data-toggle="citation">59</a></span>]), the functional and structural significance and the pattern of nucleotide polymorphism within the <em xmlns="" class="EmphasisTypeItalic">Pgi</em> gene might be expected to be similar between <em xmlns="" class="EmphasisTypeItalic">F. ovina</em> and other species. However, at present, only a few studies of <em xmlns="" class="EmphasisTypeItalic">Pgi</em> have investigated gene-wide patterns of nucleotide polymorphism. Two of these studies, on the butterflies <em xmlns="" class="EmphasisTypeItalic">Melitaea cinxia</em> [<span class="CitationRef"><a href="#CR60" data-toggle="citation">60</a></span>] and <em xmlns="" class="EmphasisTypeItalic">Colias eurytheme</em> [<span class="CitationRef"><a href="#CR61" data-toggle="citation">61</a></span>, <span class="CitationRef"><a href="#CR62" data-toggle="citation">62</a></span>], reveal a nearly uniformly high level of nucleotide polymorphism across the entire <em xmlns="" class="EmphasisTypeItalic">Pgi</em> gene. The pattern of nucleotide polymorphism in these two species was interpreted in terms of balancing selection (targeting a few amino acid sites located within the large domain) and moderate to high levels of LD within the entire <em xmlns="" class="EmphasisTypeItalic">Pgi</em> gene [<span class="CitationRef"><a href="#CR60" data-toggle="citation">60</a></span>, <span class="CitationRef"><a href="#CR61" data-toggle="citation">61</a></span>]. A high level of synonymous polymorphism was observed regionally around a nonsynonynous mutation within the 5’ portion of the <em xmlns="" class="EmphasisTypeItalic">PgiC</em> gene in <em xmlns="" class="EmphasisTypeItalic">Arabidopsis thaliana</em>, and interpreted in terms of balancing selection and an overall low level of recombination [<span class="CitationRef"><a href="#CR63" data-toggle="citation">63</a></span>].</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par35" class="Para">In contrast to the <em xmlns="" class="EmphasisTypeItalic">Pgi</em> genes of the three species mentioned above, the <em xmlns="" class="EmphasisTypeItalic">F. ovina PgiC1</em> gene showed a high level of recombination (<em xmlns="" class="EmphasisTypeItalic">R</em>
                           <sub>M</sub> = 22): <em xmlns="" class="EmphasisTypeItalic">M. cinxia</em> had a <em xmlns="" class="EmphasisTypeItalic">R</em>
                           <sub>M</sub> of 6 [<span class="CitationRef"><a href="#CR60" data-toggle="citation">60</a></span>], whereas <em xmlns="" class="EmphasisTypeItalic">C. eurytheme R</em>
                           <sub>M</sub> = 11 [<span class="CitationRef"><a href="#CR61" data-toggle="citation">61</a></span>] and <em xmlns="" class="EmphasisTypeItalic">A. thaliana</em> showed no clear evidence of recombination [<span class="CitationRef"><a href="#CR63" data-toggle="citation">63</a></span>]). The high estimated recombination value is consistent with the fact that <em xmlns="" class="EmphasisTypeItalic">F. ovina</em> is a highly outcrossing species [<span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>], and may indicate that different parts of the sequence could have evolved (at least to some extent) independently, resulting in a non-uniform pattern of nucleotide polymorphism across the sequence. Most of the identified recombination is within the 5’ portion of the <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> sequence, while the 3’ portion of the sequence shows limited recombination. However, the relatively low level of recombination detected for the 3’ portion of the sequence may be a consequence of purifying selection having removed variation at both the sites under selection and linked neutral sites (cf. [<span class="CitationRef"><a href="#CR64" data-toggle="citation">64</a></span>]) – thereby removing the molecular signatures of recombination and lowering the numbers of identified recombination events (cf. [<span class="CitationRef"><a href="#CR65" data-toggle="citation">65</a></span>]). The hypothesis that purifying selection may have removed the detectable signatures of recombination within the 3’ portion of <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> agrees with the <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> LD matrix, where a uniformly low level of LD is observed along the entire <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> cDNA sequence.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par36" class="Para">In addition to purifying selection, a selective sweep may also have contributed to the low level of nucleotide diversity within the 3’ portion of the gene: the near, but non-significant (<em xmlns="" class="EmphasisTypeItalic">P</em> = 0.064), result of the HKA test suggests that variation-reducing selective forces may be acting on the <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> 3’ portion and/or variation-increasing forces acting on the 5’ portion (cf. [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>]). The sliding window analyses revealed striking valleys of both Fay and Wu’s <em xmlns="" class="EmphasisTypeItalic">H</em> and Tajima’s <em xmlns="" class="EmphasisTypeItalic">D</em> at exon 16 (within the 3’ portion) (Additional file <span class="InternalRef"><a href="#MOESM5">5</a></span>: Figure S1). In these valleys, the Fay and Wu’s <em xmlns="" class="EmphasisTypeItalic">H</em> is significantly negative (<em xmlns="" class="EmphasisTypeItalic">H</em> = -1.862; <em xmlns="" class="EmphasisTypeItalic">P</em> = 0.006) and the Tajima’s <em xmlns="" class="EmphasisTypeItalic">D</em> is also near-significantly negative (<em xmlns="" class="EmphasisTypeItalic">D</em> = -1.509; <em xmlns="" class="EmphasisTypeItalic">P</em> = 0.095). These negative values reflect a high-frequency of derived SNPs (around the valleys), suggesting a selective sweep (cf. [<span class="CitationRef"><a href="#CR45" data-toggle="citation">45</a></span>]). A nearby shallow valley of total nucleotide diversity (Fig.&nbsp;<span class="InternalRef"><a href="#Fig2">2</a></span>) is also suggestive of a selective sweep [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>]. The valleys of <em xmlns="" class="EmphasisTypeItalic">D</em> and <em xmlns="" class="EmphasisTypeItalic">H</em> at exon 16 are close to active site residue His391 (Additional file <span class="InternalRef"><a href="#MOESM5">5</a></span>: Figure S1). Sequence patterns that are identified as the signatures of selection in neutrality tests (e.g. Tajima’s <em xmlns="" class="EmphasisTypeItalic">D</em> and Fay and Wu’s <em xmlns="" class="EmphasisTypeItalic">H</em>) may also result from factors such as population size changes or reflect population structure [<span class="CitationRef"><a href="#CR66" data-toggle="citation">66</a></span>–<span class="CitationRef"><a href="#CR68" data-toggle="citation">68</a></span>]. In the case of the highly outcrossing populations of <em xmlns="" class="EmphasisTypeItalic">F. ovina</em> on Öland, population structure is unlikely to be a confounding factor in the neutrality tests. However, because possible confounding effects resulting from changes in population size cannot be excluded, the selective sweep suggested by the <em xmlns="" class="EmphasisTypeItalic">H</em> and <em xmlns="" class="EmphasisTypeItalic">D</em> tests should be interpreted with caution.</p>
              </section><section id="Sec13" class="Section2 RenderAsSection2">
                <h3 class="Heading">Which evolutionary mechanisms may have contributed to the relatively high level of nucleotide polymorphism within the 5’ portion of the <em class="EmphasisTypeItalic">PgiC1</em> sequence?</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par37" class="Para">An earlier study [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>] identified two <em xmlns="" class="EmphasisTypeItalic">F. ovina PgiC1</em> codon sites (sites 173 and 200) as candidate targets of balancing selection (i.e. positive intraspecific diversifying selection) with a considerably stronger signal of selection for site 200 than for site 173 [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>]. The sliding window analyses in the present study support the results of the earlier study and reveal a marked plateau of <em xmlns="" class="EmphasisTypeItalic">ω</em> around the selected site 200 (but no peak associated with site 173) (Fig.&nbsp;<span class="InternalRef"><a href="#Fig2">2</a></span>). Protein structure modelling suggests that the translated amino acid polymorphism at these two <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> sites may affect either the interaction between the two monomers, or the domain-domain packing of the encoded PGI enzyme and, thus, influence the biochemical properties of the cytosolic PGI enzyme in <em xmlns="" class="EmphasisTypeItalic">F. ovina</em> [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>]. Biochemical studies in humans have shown that mutations at a few amino acid sites, which have similar 3-D structural locations to the two selected amino acid sites in <em xmlns="" class="EmphasisTypeItalic">F. ovina</em>, significantly affect the activity of the PGI enzyme [<span class="CitationRef"><a href="#CR69" data-toggle="citation">69</a></span>, <span class="CitationRef"><a href="#CR70" data-toggle="citation">70</a></span>].</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par38" class="Para">Within <em xmlns="" class="EmphasisTypeItalic">F. ovina PgiC1</em>, both codon sites 173 and 200, which were previously identified as candidates for balancing selection [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>], are located within the 5’ portion of the sequence (Fig.&nbsp;<span class="InternalRef"><a href="#Fig2">2</a></span>). The significant results from the Wall’s <em xmlns="" class="EmphasisTypeItalic">B</em> and <em xmlns="" class="EmphasisTypeItalic">Q</em> tests (Table&nbsp;<span class="InternalRef"><a href="#Tab1">1</a></span>) support the suggestion that there has been balancing selection on the 5’ portion of the <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> sequence in <em xmlns="" class="EmphasisTypeItalic">F. ovina</em> (cf. [<span class="CitationRef"><a href="#CR47" data-toggle="citation">47</a></span>]). In addition to the significant <em xmlns="" class="EmphasisTypeItalic">B</em> and <em xmlns="" class="EmphasisTypeItalic">Q</em> tests, signals of balancing selection were also detected at the putative selected site 200. The highest peaks of, respectively, positive Tajima’s <em xmlns="" class="EmphasisTypeItalic">D</em> and total nucleotide diversity were observed at or around codon site 200 in the sliding window analyses (Fig.&nbsp;<span class="InternalRef"><a href="#Fig2">2</a></span>), and these peaks are a typical signature of balancing selection (cf. [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>, <span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>]). No marked peak or plateau for polymorphism or for Tajima’s <em xmlns="" class="EmphasisTypeItalic">D</em> was observed for the second putative selected site (site 173) in the present study (Fig.&nbsp;<span class="InternalRef"><a href="#Fig2">2</a></span>), in agreement with the previous study [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>] which showed a weaker signal of balancing selection for site 173 than for site 200.</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec14" class="Section1 RenderAsSection1 SectionTypeConclusion c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Conclusions" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Conclusions">Conclusions</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Conclusions" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par39" class="Para">The <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> gene in <em xmlns="" class="EmphasisTypeItalic">F. ovina</em> represents one of the few reported cases in which the levels of nucleotide polymorphism differ substantially between the 5’ and 3’ portions of a gene. The present study suggests that the contrasting levels of nucleotide polymorphism between the two portions of <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> may have resulted from different selective regimes in the two gene portions. Relatively strong purifying selection appears to have reduced the level of polymorphism within the 3’ portion, whereas balancing selection may have contributed to the maintenance of the polymorphism in the 5’ portion of the sequence. A high overall level of recombination and a low level of LD within <em xmlns="" class="EmphasisTypeItalic">PgiC1</em> may have allowed partially independent selection and evolution within the two portions of the gene.</p></div></section>
          <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AbbreviationGroup Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Abbreviations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Abbreviations">Abbreviations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Abbreviations"><div class="DefinitionList">
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>HKA:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par5" class="Para">Hudson-Kreitman-Aguadé test</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>HPD:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par6" class="Para">Highest posterior density</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>LD:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par7" class="Para">Linkage disequilibrium</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>MHC:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par8" class="Para">The major histocompatibility complex</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>MK test:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par9" class="Para">MacDonald and Kreitman’s test</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>PGI:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par10" class="Para">Phosphoglucose isomerase</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>SNP:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par11" class="Para">Single nucleotide polymorphism</p>
                  </dd></dl>
              </div></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments"><h3 xmlns="" class="Heading">Acknowledgements</h3><p xmlns="" class="SimplePara">We would like to thank Torbjörn Säll and Anders Tunlid for comments on an earlier version of the manuscript.</p><div id="FPar1" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Funding</h3>
                <p id="Par40" class="Para">The study was supported by grant 621-2008-5617 (to H.C.P and Anders Tunlid) from the Swedish Research Council. Bengt Hansson also acknowledges support from the Swedish Research Council (grant 621-2014-5222). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p>
              </div><div id="FPar2" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Availability of data and materials</h3>
                <p id="Par41" class="Para">All data generated or analyzed during this study are either available in the GenBank database (GenBank accession numbers DQ225731-DQ225735 &amp; KF487737-KF487765; <span class="ExternalRef"><a href="https://www.ncbi.nlm.nih.gov/nucleotide"><span class="RefSource">https://www.ncbi.nlm.nih.gov/nucleotide</span></a></span>) or included in this published article and its additional files.</p>
              </div><div id="FPar3" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Authors’ contributions</h3>
                <p id="Par42" class="Para">YL conceived the study with input from BH, HCP and LG. YL was responsible for the data collection and data analyses. YL wrote the paper with input from BH and HCP. All authors read and approved the final manuscript.</p>
              </div><div id="FPar4" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Competing interests</h3>
                <p id="Par43" class="Para">The authors declare that they have no competing interests.</p>
              </div><div id="FPar5" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Consent for publication</h3>
                <p id="Par44" class="Para">Not applicable.</p>
              </div><div id="FPar6" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Ethics approval and consent to participate</h3>
                <p id="Par45" class="Para">Not applicable.</p>
              </div><div id="FPar7" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Publisher’s Note</h3>
                <p id="Par46" class="Para">Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
              </div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="License">
                <p xmlns="" class="SimplePara">
                           <strong class="EmphasisTypeBold">Open Access</strong>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/4.0/"><span class="RefSource">http://creativecommons.org/licenses/by/4.0/</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/publicdomain/zero/1.0/"><span class="RefSource">http://creativecommons.org/publicdomain/zero/1.0/</span></a></span>) applies to the data made available in this article, unless otherwise stated.</p>
              </div><aside class="Appendix" id="App1">
              
                <h3 class="Heading">Additional files</h3><div id="widgetContainer" data-test="figshare-container"></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par47" class="Para">
                  <div class="MediaObject" id="MOESM1">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0032-6/MediaObjects/41065_2017_32_MOESM1_ESM.txt"><span class="caption-number">Additional file 1:</span></a>
                          <span xmlns="" class="SimplePara">
                                          <strong class="EmphasisTypeBold">File S1.</strong> The aligned multiple-species homologous amino acid sequences acquired from the Consurf Server. (TXT 96&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM2">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0032-6/MediaObjects/41065_2017_32_MOESM2_ESM.docx"><span class="caption-number">Additional file 2: Table S1.</span></a>
                          <span xmlns="" class="SimplePara">The total nucleotide diversity (<em class="EmphasisTypeItalic">π</em>
                                          <sub xmlns="http://www.w3.org/1999/xhtml">T</sub>) for each studied <em class="EmphasisTypeItalic">PgiC1</em> exon. (DOCX 47&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM3">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0032-6/MediaObjects/41065_2017_32_MOESM3_ESM.xls"><span class="caption-number">Additional file 3: Table S2.</span></a>
                          <span xmlns="" class="SimplePara">Estimates of <em class="EmphasisTypeItalic">ω</em> and <em class="EmphasisTypeItalic">ρ</em> (made with omegaMap) for each analyzed <em class="EmphasisTypeItalic">PgiC1</em> codon. (XLS 119&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM4">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0032-6/MediaObjects/41065_2017_32_MOESM4_ESM.doc"><span class="caption-number">Additional file 4: Table S3.</span></a>
                          <span xmlns="" class="SimplePara">Comparison (Hudson-Kreitman-Aguadé test) between the 5’ and 3’ portions of the sequenced <em class="EmphasisTypeItalic">F. ovina PgiC1</em> in terms of level of polymorphism and level of divergence from the outgroup <em class="EmphasisTypeItalic">F. altissima</em>. (DOC 33&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM5">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0032-6/MediaObjects/41065_2017_32_MOESM5_ESM.tif"><span class="caption-number">Additional file 5: Figure S1.</span></a>
                          <span xmlns="" class="SimplePara">Results for the sliding window analyses of, respectively, Tajima’s <em class="EmphasisTypeItalic">D</em>, Fay and Wu’s <em class="EmphasisTypeItalic">H</em>, and <em class="EmphasisTypeItalic">K</em>a/<em class="EmphasisTypeItalic">K</em>s. The ticks on the x axis represent the boundary of each analysed <em class="EmphasisTypeItalic">PgiC1</em> exon within the <em class="EmphasisTypeItalic">PgiC1</em> coding sequence. In <em class="EmphasisTypeItalic">F. ovina</em>, <em class="EmphasisTypeItalic">PgiC1</em> exons 5–12 encode the small domain of a PGI monomer while exons 13–21 encode the large domain. The three stars on the x axis represent the three active site residues (equivalent to Lys516, Glu360, and His391 in <em class="EmphasisTypeItalic">F. ovina</em>) that are directly involved in the PGI isomerization reaction [<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR35" data-toggle="citation">35</a></span>]. The dotted vertical line highlights a position where both <em class="EmphasisTypeItalic">D</em> and <em class="EmphasisTypeItalic">H</em> have marked valleys. (TIF 1634&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM6">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0032-6/MediaObjects/41065_2017_32_MOESM6_ESM.xls"><span class="caption-number">Additional file 6: Table S4.</span></a>
                          <span xmlns="" class="SimplePara">The estimated (normalized) conservation scores for each PGI amino acid site. (XLS 49&nbsp;kb)</span>
                        </div></div>
                  </div>
                </div>
              
            </aside></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authors’ Affiliations</a></h2><div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1"><span class="AffiliationNumber">(1)</span><div class="AffiliationText">Department of Biology, Lund University, Lund, Sweden</div></div></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper"><li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Huang X, Kurata N, Wei X, Wang Z-X, Wang A, Zhao Q, et al. A map of rice genome variation reveals the origin of cultivated rice. Nature. 2012;490:497–501.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nature11532"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23034647"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20map%20of%20rice%20genome%20variation%20reveals%20the%20origin%20of%20cultivated%20rice&amp;author=X.%20Huang&amp;author=N.%20Kurata&amp;author=X.%20Wei&amp;author=Z-X.%20Wang&amp;author=A.%20Wang&amp;author=Q.%20Zhao&amp;journal=Nature&amp;volume=490&amp;pages=497-501&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hufford MB, Xu X, van Heerwaarden J, Pyhäjärvi T, Chia J-M, Cartwright RA, et al. Comparative population genomics of maize domestication and improvement. Nat Genet. 2012;44:808–11.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/ng.2309"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22660546"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparative%20population%20genomics%20of%20maize%20domestication%20and%20improvement&amp;author=MB.%20Hufford&amp;author=X.%20Xu&amp;author=J.%20Heerwaarden&amp;author=T.%20Pyh%C3%A4j%C3%A4rvi&amp;author=J-M.%20Chia&amp;author=RA.%20Cartwright&amp;journal=Nat%20Genet&amp;volume=44&amp;pages=808-11&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Jiao Y, Zhao H, Ren L, Song W, Zeng B, Guo J, et al. Genome-wide genetic changes during modern breeding of maize. Nat Genet. 2012;44:812–5.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/ng.2312"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22660547"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genome-wide%20genetic%20changes%20during%20modern%20breeding%20of%20maize&amp;author=Y.%20Jiao&amp;author=H.%20Zhao&amp;author=L.%20Ren&amp;author=W.%20Song&amp;author=B.%20Zeng&amp;author=J.%20Guo&amp;journal=Nat%20Genet&amp;volume=44&amp;pages=812-5&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Le Corre V, Roux F, Reboud X. DNA polymorphism at the <em class="EmphasisTypeItalic">FRIGIDA</em> gene in <em class="EmphasisTypeItalic">Arabidopsis thaliana</em>: extensive nonsynonymous variation is consistent with local selection for flowering time. Mol Biol Evol. 2002;19:1261–71.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/oxfordjournals.molbev.a004187"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12140238"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=DNA%20polymorphism%20at%20the%20FRIGIDA%20gene%20in%20Arabidopsis%20thaliana%3A%20extensive%20nonsynonymous%20variation%20is%20consistent%20with%20local%20selection%20for%20flowering%20time&amp;author=V.%20Corre&amp;author=F.%20Roux&amp;author=X.%20Reboud&amp;journal=Mol%20Biol%20Evol&amp;volume=19&amp;pages=1261-71&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Moore RC, Grant SR, Purugganan MD. Molecular population genetics of redundant floral-regulatory genes in <em class="EmphasisTypeItalic">Arabidopsis thaliana</em>. Mol Biol Evol. 2005;22:91–103.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/molbev/msh261"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15371526"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Molecular%20population%20genetics%20of%20redundant%20floral-regulatory%20genes%20in%20Arabidopsis%20thaliana%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=RC.%20Moore&amp;author=SR.%20Grant&amp;author=MD.%20Purugganan&amp;journal=Mol%20Biol%20Evol&amp;volume=22&amp;pages=91-103&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hasselmann M, Beye M. Signatures of selection among sex-determining alleles of the honey bee. Proc Natl Acad Sci U S A. 2004;101:4888–93.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.0307147101"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15051879"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC387344"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Signatures%20of%20selection%20among%20sex-determining%20alleles%20of%20the%20honey%20bee&amp;author=M.%20Hasselmann&amp;author=M.%20Beye&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;volume=101&amp;pages=4888-93&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ding Z, Wang C, Chen S, Yu S. Diversity and selective sweep in the OsAMT1; 1 genomic region of rice. BMC Evol Biol. 2011;11:61.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2148-11-61"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21385389"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3062601"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Diversity%20and%20selective%20sweep%20in%20the%20OsAMT1%3B%201%20genomic%20region%20of%20rice&amp;author=Z.%20Ding&amp;author=C.%20Wang&amp;author=S.%20Chen&amp;author=S.%20Yu&amp;journal=BMC%20Evol%20Biol&amp;volume=11&amp;pages=61&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Charlesworth B. Effective population size and patterns of molecular evolution and variation. Nat Rev Genet. 2009;10:195–205.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nrg2526"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19204717"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Effective%20population%20size%20and%20patterns%20of%20molecular%20evolution%20and%20variation&amp;author=B.%20Charlesworth&amp;journal=Nat%20Rev%20Genet&amp;volume=10&amp;pages=195-205&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Cutter AD, Payseur BA. Genomic signatures of selection at linked sites: unifying the disparity among species. Nat Rev Genet. 2013;14:262–74.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nrg3425"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23478346"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4066956"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genomic%20signatures%20of%20selection%20at%20linked%20sites%3A%20unifying%20the%20disparity%20among%20species&amp;author=AD.%20Cutter&amp;author=BA.%20Payseur&amp;journal=Nat%20Rev%20Genet&amp;volume=14&amp;pages=262-74&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Tourasse NJ, Li W-H. Selective constraints, amino acid composition, and the rate of protein evolution. Mol Biol Evol. 2000;17:656–64.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/oxfordjournals.molbev.a026344"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10742056"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Selective%20constraints%2C%20amino%20acid%20composition%2C%20and%20the%20rate%20of%20protein%20evolution&amp;author=NJ.%20Tourasse&amp;author=W-H.%20Li&amp;journal=Mol%20Biol%20Evol&amp;volume=17&amp;pages=656-64&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hudson RR, Kreitman M, Aguadé M. A test of neutral molecular evolution based on nucleotide data. Genetics. 1987;116:153–9.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=3110004"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1203113"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20test%20of%20neutral%20molecular%20evolution%20based%20on%20nucleotide%20data&amp;author=RR.%20Hudson&amp;author=M.%20Kreitman&amp;author=M.%20Aguad%C3%A9&amp;journal=Genetics&amp;volume=116&amp;pages=153-9&amp;publication_year=1987"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kimura M. The neutral theory of molecular evolution. Cambridge: Cambridge University Press; 1983.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1017/CBO9780511623486"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20neutral%20theory%20of%20molecular%20evolution&amp;author=M.%20Kimura&amp;publication_year=1983"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Weedall GD, Conway DJ. Detecting signatures of balancing selection to identify targets of anti-parasite immunity. Trends Parasitol. 2010;26:363–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.pt.2010.04.002"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20466591"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Detecting%20signatures%20of%20balancing%20selection%20to%20identify%20targets%20of%20anti-parasite%20immunity&amp;author=GD.%20Weedall&amp;author=DJ.%20Conway&amp;journal=Trends%20Parasitol&amp;volume=26&amp;pages=363-9&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Charlesworth D. Balancing selection and its effects on sequences in nearby genome regions. PLoS Genet. 2006;2:e64.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pgen.0020064"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16683038"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1449905"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Balancing%20selection%20and%20its%20effects%20on%20sequences%20in%20nearby%20genome%20regions&amp;author=D.%20Charlesworth&amp;journal=PLoS%20Genet&amp;volume=2&amp;pages=e64&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lamaze FC, Pavey SA, Normandeau E, Roy G, Garant D, Bernatchez L. Neutral and selective processes shape MHC gene diversity and expression in stocked brook charr populations (<em class="EmphasisTypeItalic">Salvelinus fontinalis</em>). Mol Ecol. 2014;23:1730–48.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/mec.12684"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24795997"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Neutral%20and%20selective%20processes%20shape%20MHC%20gene%20diversity%20and%20expression%20in%20stocked%20brook%20charr%20populations%20%28Salvelinus%20fontinalis%29&amp;author=FC.%20Lamaze&amp;author=SA.%20Pavey&amp;author=E.%20Normandeau&amp;author=G.%20Roy&amp;author=D.%20Garant&amp;author=L.%20Bernatchez&amp;journal=Mol%20Ecol&amp;volume=23&amp;pages=1730-48&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Bernatchez L, Landry C. MHC studies in nonmodel vertebrates: what have we learned about natural selection in 15 years? J Evol Biol. 2003;16:363–77.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1046/j.1420-9101.2003.00531.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14635837"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=MHC%20studies%20in%20nonmodel%20vertebrates%3A%20what%20have%20we%20learned%20about%20natural%20selection%20in%2015%20years%3F&amp;author=L.%20Bernatchez&amp;author=C.%20Landry&amp;journal=J%20Evol%20Biol&amp;volume=16&amp;pages=363-77&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wright SI, Foxe JP, DeRose-Wilson L, Kawabe A, Looseley M, Gaut BS, et al. Testing for effects of recombination rate on nucleotide diversity in natural populations of <em class="EmphasisTypeItalic">Arabidopsis lyrata</em>. Genetics. 2006;174:1421–30.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1534/genetics.106.062588"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16951057"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1667078"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Testing%20for%20effects%20of%20recombination%20rate%20on%20nucleotide%20diversity%20in%20natural%20populations%20of%20Arabidopsis%20lyrata%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=SI.%20Wright&amp;author=JP.%20Foxe&amp;author=L.%20DeRose-Wilson&amp;author=A.%20Kawabe&amp;author=M.%20Looseley&amp;author=BS.%20Gaut&amp;journal=Genetics&amp;volume=174&amp;pages=1421-30&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ghatnekar L. Genetic analysis of cytosolic PGI in Festuca ovina. PhD thesis. Lund University, 2003.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Ghatnekar%20L.%20Genetic%20analysis%20of%20cytosolic%20PGI%20in%20Festuca%20ovina.%20PhD%20thesis.%20Lund%20University%2C%202003."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gillespie JH. The causes of molecular evolution. New York: Oxford University Press; 1991.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20causes%20of%20molecular%20evolution&amp;author=JH.%20Gillespie&amp;publication_year=1991"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Marden JH. Nature’s inordinate fondness for metabolic enzymes: why metabolic enzyme loci are so frequently targets of selection. Mol Ecol. 2013;22:5743–64.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/mec.12534"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24106889"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Nature%E2%80%99s%20inordinate%20fondness%20for%20metabolic%20enzymes%3A%20why%20metabolic%20enzyme%20loci%20are%20so%20frequently%20targets%20of%20selection&amp;author=JH.%20Marden&amp;journal=Mol%20Ecol&amp;volume=22&amp;pages=5743-64&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Shaw PJ, Muirhead H. Crystallographic structure analysis of glucose 6-phosphate isomerase at 3.5 Å resolution. J Mol Biol. 1977;109:475–85.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/S0022-2836(77)80025-9"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=833853"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Crystallographic%20structure%20analysis%20of%20glucose%206-phosphate%20isomerase%20at%203.5%20%C3%85%20resolution&amp;author=PJ.%20Shaw&amp;author=H.%20Muirhead&amp;journal=J%20Mol%20Biol&amp;volume=109&amp;pages=475-85&amp;publication_year=1977"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wang B, Watt WB, Aakre C, Hawthorne N. Emergence of complex haplotypes from microevolutionary variation in sequence and structure of <em class="EmphasisTypeItalic">Colias</em> phosphoglucose isomerase. J Mol Evol. 2009;68:433–47.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00239-009-9237-2"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19424742"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Emergence%20of%20complex%20haplotypes%20from%20microevolutionary%20variation%20in%20sequence%20and%20structure%20of%20Colias%20phosphoglucose%20isomerase&amp;author=B.%20Wang&amp;author=WB.%20Watt&amp;author=C.%20Aakre&amp;author=N.%20Hawthorne&amp;journal=J%20Mol%20Evol&amp;volume=68&amp;pages=433-47&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Riddoch BJ. The adaptive significance of electrophoretic mobility in phosphoglucose isomerase (PGI). Biol J Linn Soc Lond. 1993;50:1–17.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1095-8312.1993.tb00915.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20adaptive%20significance%20of%20electrophoretic%20mobility%20in%20phosphoglucose%20isomerase%20%28PGI%29&amp;author=BJ.%20Riddoch&amp;journal=Biol%20J%20Linn%20Soc%20Lond&amp;volume=50&amp;pages=1-17&amp;publication_year=1993"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR24" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Dahlhoff EP, Rank NE. Functional and physiological consequences of genetic variation at phosphoglucose isomerase: Heat shock protein expression is related to enzyme genotype in a montane beetle. Proc Natl Acad Sci U S A. 2000;97:10056–61.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.160277697"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10944188"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC27685"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Functional%20and%20physiological%20consequences%20of%20genetic%20variation%20at%20phosphoglucose%20isomerase%3A%20Heat%20shock%20protein%20expression%20is%20related%20to%20enzyme%20genotype%20in%20a%20montane%20beetle&amp;author=EP.%20Dahlhoff&amp;author=NE.%20Rank&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;volume=97&amp;pages=10056-61&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR25" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Watt WB. Eggs, enzymes, and evolution: natural genetic variants change insect fecundity. Proc Natl Acad Sci U S A. 1992;89:10608–12.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.89.22.10608"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1438255"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC50390"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Eggs%2C%20enzymes%2C%20and%20evolution%3A%20natural%20genetic%20variants%20change%20insect%20fecundity&amp;author=WB.%20Watt&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;volume=89&amp;pages=10608-12&amp;publication_year=1992"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR26" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Prentice HC, Lönn M, Lefkovitch LP, Runyeon H. Associations between allele frequencies in <em class="EmphasisTypeItalic">Festuca ovina</em> and habitat variation in the alvar grasslands on the Baltic island of Öland. J Ecol. 1995;83:391–402.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2307/2261593"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Associations%20between%20allele%20frequencies%20in%20Festuca%20ovina%20and%20habitat%20variation%20in%20the%20alvar%20grasslands%20on%20the%20Baltic%20island%20of%20%C3%96land&amp;author=HC.%20Prentice&amp;author=M.%20L%C3%B6nn&amp;author=LP.%20Lefkovitch&amp;author=H.%20Runyeon&amp;journal=J%20Ecol&amp;volume=83&amp;pages=391-402&amp;publication_year=1995"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR27" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Cope T, Gray A. Grasses of the British Isles. London: Botanical Society of the British Isles; 2009.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Grasses%20of%20the%20British%20Isles&amp;author=T.%20Cope&amp;author=A.%20Gray&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR28" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Prentice HC, Lönn M, Lager H, Rosén E, van der Maarel E. Changes in allozyme frequencies in <em class="EmphasisTypeItalic">Festuca ovina</em> populations after a 9-year nutrient/water experiment. J Ecol. 2000;88:331–47.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1046/j.1365-2745.2000.00454.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Changes%20in%20allozyme%20frequencies%20in%20Festuca%20ovina%20populations%20after%20a%209-year%20nutrient%2Fwater%20experiment&amp;author=HC.%20Prentice&amp;author=M.%20L%C3%B6nn&amp;author=H.%20Lager&amp;author=E.%20Ros%C3%A9n&amp;author=E.%20Maarel&amp;journal=J%20Ecol&amp;volume=88&amp;pages=331-47&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR29" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Prentice HC, Li Y, Lönn M, Tunlid A, Ghatnekar L. A horizontally transferred nuclear gene is associated with microhabitat variation in a natural plant population. Proc R Soc Lond Ser B Biol Sci. 2015;282:20152453.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1098/rspb.2015.2453"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20horizontally%20transferred%20nuclear%20gene%20is%20associated%20with%20microhabitat%20variation%20in%20a%20natural%20plant%20population&amp;author=HC.%20Prentice&amp;author=Y.%20Li&amp;author=M.%20L%C3%B6nn&amp;author=A.%20Tunlid&amp;author=L.%20Ghatnekar&amp;journal=Proc%20R%20Soc%20Lond%20Ser%20B%20Biol%20Sci&amp;volume=282&amp;pages=20152453&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR30" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ghatnekar L. A polymorphic duplicated locus for cytosolic PGI segregating in sheep’s fescue (<em class="EmphasisTypeItalic">Festuca ovina</em> L.). Heredity (Edinb). 1999;83:451–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/sj.hdy.6885750"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20polymorphic%20duplicated%20locus%20for%20cytosolic%20PGI%20segregating%20in%20sheep%E2%80%99s%20fescue%20%28Festuca%20ovina%20L.%29&amp;author=L.%20Ghatnekar&amp;journal=Heredity%20%28Edinb%29&amp;volume=83&amp;pages=451-9&amp;publication_year=1999"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR31" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Vallenback P, Jaarola M, Ghatnekar L, Bengtsson BO. Origin and timing of the horizontal transfer of a <em class="EmphasisTypeItalic">PgiC</em> gene from <em class="EmphasisTypeItalic">Poa</em> to <em class="EmphasisTypeItalic">Festuca ovina</em>. Mol Phylogenet Evol. 2008;46:890–6.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.ympev.2007.11.031"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18226929"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Origin%20and%20timing%20of%20the%20horizontal%20transfer%20of%20a%20PgiC%20gene%20from%20Poa%20to%20Festuca%20ovina%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=P.%20Vallenback&amp;author=M.%20Jaarola&amp;author=L.%20Ghatnekar&amp;author=BO.%20Bengtsson&amp;journal=Mol%20Phylogenet%20Evol&amp;volume=46&amp;pages=890-6&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR32" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Vallenback P, Ghatnekar L, Bengtsson BO. Structure of the natural transgene <em class="EmphasisTypeItalic">PgiC2</em> in the common grass <em class="EmphasisTypeItalic">Festuca ovina</em>. Plos One. 2010;5:e13529.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0013529"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20976007"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2958129"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Structure%20of%20the%20natural%20transgene%20PgiC2%20in%20the%20common%20grass%20Festuca%20ovina%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=P.%20Vallenback&amp;author=L.%20Ghatnekar&amp;author=BO.%20Bengtsson&amp;journal=Plos%20One&amp;volume=5&amp;pages=e13529&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR33" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Vallenback P, Bengtsson BO, Ghatnekar L. Geographic and molecular variation in a natural plant transgene. Genetica. 2010;138:355–62.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s10709-009-9425-4"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20128113"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Geographic%20and%20molecular%20variation%20in%20a%20natural%20plant%20transgene&amp;author=P.%20Vallenback&amp;author=BO.%20Bengtsson&amp;author=L.%20Ghatnekar&amp;journal=Genetica&amp;volume=138&amp;pages=355-62&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR34" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li Y, Canbäck B, Johansson T, Tunlid A, Prentice HC. Evidence for positive selection within the <em class="EmphasisTypeItalic">PgiC1</em> locus in the grass <em class="EmphasisTypeItalic">Festuca ovina</em>. Plos One. 2015;10:e0125831.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0125831"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25946223"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4422690"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Evidence%20for%20positive%20selection%20within%20the%20PgiC1%20locus%20in%20the%20grass%20Festuca%20ovina%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=Y.%20Li&amp;author=B.%20Canb%C3%A4ck&amp;author=T.%20Johansson&amp;author=A.%20Tunlid&amp;author=HC.%20Prentice&amp;journal=Plos%20One&amp;volume=10&amp;pages=e0125831&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR35" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Jeffery CJ, Hardré R, Salmon L. Crystal structure of rabbit phosphoglucose isomerase complexed with 5-phospho-D-arabinonate identifies the role of Glu357 in catalysis. Biochemistry. 2001;40:1560–6.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1021/bi0018483"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11327814"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Crystal%20structure%20of%20rabbit%20phosphoglucose%20isomerase%20complexed%20with%205-phospho-D-arabinonate%20identifies%20the%20role%20of%20Glu357%20in%20catalysis&amp;author=CJ.%20Jeffery&amp;author=R.%20Hardr%C3%A9&amp;author=L.%20Salmon&amp;journal=Biochemistry&amp;volume=40&amp;pages=1560-6&amp;publication_year=2001"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR36" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Nei M. Molecular evolutionary genetics. New York: Columbia University Press; 1987.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Molecular%20evolutionary%20genetics&amp;author=M.%20Nei&amp;publication_year=1987"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR37" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Watterson GA. On the number of segregating sites in genetical models without recombination. Theor Popul Biol. 1975;7:256–76.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/0040-5809(75)90020-9"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1145509"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=On%20the%20number%20of%20segregating%20sites%20in%20genetical%20models%20without%20recombination&amp;author=GA.%20Watterson&amp;journal=Theor%20Popul%20Biol&amp;volume=7&amp;pages=256-76&amp;publication_year=1975"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR38" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Librado P, Rozas J. DnaSP v5: a software for comprehensive analysis of DNA polymorphism data. Bioinformatics. 2009;25:1451–2.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btp187"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19346325"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=DnaSP%20v5%3A%20a%20software%20for%20comprehensive%20analysis%20of%20DNA%20polymorphism%20data&amp;author=P.%20Librado&amp;author=J.%20Rozas&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1451-2&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR39" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hudson RR, Kaplan NL. Statistical properties of the number of recombination events in the history of a sample of DNA sequences. Genetics. 1985;111:147–64.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=4029609"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1202594"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Statistical%20properties%20of%20the%20number%20of%20recombination%20events%20in%20the%20history%20of%20a%20sample%20of%20DNA%20sequences&amp;author=RR.%20Hudson&amp;author=NL.%20Kaplan&amp;journal=Genetics&amp;volume=111&amp;pages=147-64&amp;publication_year=1985"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR40" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Stumpf MPH, McVean GAT. Estimating recombination rates from population-genetic data. Nat Rev Genet. 2003;4:959–68.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nrg1227"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14631356"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Estimating%20recombination%20rates%20from%20population-genetic%20data&amp;author=MPH.%20Stumpf&amp;author=GAT.%20McVean&amp;journal=Nat%20Rev%20Genet&amp;volume=4&amp;pages=959-68&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR41" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wilson DJ, McVean G. Estimating diversifying selection and functional constraint in the presence of recombination. Genetics. 2006;172:1411–25.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1534/genetics.105.044917"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16387887"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1456295"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Estimating%20diversifying%20selection%20and%20functional%20constraint%20in%20the%20presence%20of%20recombination&amp;author=DJ.%20Wilson&amp;author=G.%20McVean&amp;journal=Genetics&amp;volume=172&amp;pages=1411-25&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR42" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hill WG, Robertson A. Linkage disequilibrium in finite populations. Theor Appl Genet. 1968;38:226–31.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/BF01245622"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24442307"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Linkage%20disequilibrium%20in%20finite%20populations&amp;author=WG.%20Hill&amp;author=A.%20Robertson&amp;journal=Theor%20Appl%20Genet&amp;volume=38&amp;pages=226-31&amp;publication_year=1968"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR43" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics. 2005;21:263–5.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/bth457"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15297300"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Haploview%3A%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&amp;author=JC.%20Barrett&amp;author=B.%20Fry&amp;author=J.%20Maller&amp;author=MJ.%20Daly&amp;journal=Bioinformatics&amp;volume=21&amp;pages=263-5&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR44" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Tajima F. Statistical method for testing the neutral mutation hypothesis by DNA polymorphism. Genetics. 1989;123:585–95.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=2513255"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1203831"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Statistical%20method%20for%20testing%20the%20neutral%20mutation%20hypothesis%20by%20DNA%20polymorphism&amp;author=F.%20Tajima&amp;journal=Genetics&amp;volume=123&amp;pages=585-95&amp;publication_year=1989"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR45" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Fay JC, Wu CI. Hitchhiking under positive Darwinian selection. Genetics. 2000;155:1405–13.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10880498"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1461156"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Hitchhiking%20under%20positive%20Darwinian%20selection&amp;author=JC.%20Fay&amp;author=CI.%20Wu&amp;journal=Genetics&amp;volume=155&amp;pages=1405-13&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR46" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">McDonald JH, Kreitman M. Adaptive protein evolution at the <em class="EmphasisTypeItalic">Adh</em> locus in <em class="EmphasisTypeItalic">Drosophila</em>. Nature. 1991;351:652–4.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/351652a0"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1904993"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Adaptive%20protein%20evolution%20at%20the%20Adh%20locus%20in%20Drosophila%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=JH.%20McDonald&amp;author=M.%20Kreitman&amp;journal=Nature&amp;volume=351&amp;pages=652-4&amp;publication_year=1991"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR47" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wall JD. Recombination and the power of statistical tests of neutrality. Genet Res. 1999;74:65–79.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1017/S0016672399003870"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Recombination%20and%20the%20power%20of%20statistical%20tests%20of%20neutrality&amp;author=JD.%20Wall&amp;journal=Genet%20Res&amp;volume=74&amp;pages=65-79&amp;publication_year=1999"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR48" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ashkenazy H, Erez E, Martz E, Pupko T, Ben-Tal N. ConSurf 2010: calculating evolutionary conservation in sequence and structure of proteins and nucleic acids. Nucleic Acids Res. 2010;38:W529–33.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gkq399"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20478830"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2896094"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=ConSurf%202010%3A%20calculating%20evolutionary%20conservation%20in%20sequence%20and%20structure%20of%20proteins%20and%20nucleic%20acids&amp;author=H.%20Ashkenazy&amp;author=E.%20Erez&amp;author=E.%20Martz&amp;author=T.%20Pupko&amp;author=N.%20Ben-Tal&amp;journal=Nucleic%20Acids%20Res&amp;volume=38&amp;pages=W529-33&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR49" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Suzek BE, Wang Y, Huang H, McGarvey PB, Wu CH, the UniProt Consortium. UniRef clusters: a comprehensive and scalable alternative for improving sequence similarity searches. Bioinformatics. 2015;31:926–32.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Suzek%20BE%2C%20Wang%20Y%2C%20Huang%20H%2C%20McGarvey%20PB%2C%20Wu%20CH%2C%20the%20UniProt%20Consortium.%20UniRef%20clusters%3A%20a%20comprehensive%20and%20scalable%20alternative%20for%20improving%20sequence%20similarity%20searches.%20Bioinformatics.%202015%3B31%3A926%E2%80%9332."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR50" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Biegert A, Söding J. Sequence context-specific profiles for homology searching. Proc Natl Acad Sci U S A. 2009;106:3770–5.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.0810767106"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19234132"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2645910"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Sequence%20context-specific%20profiles%20for%20homology%20searching&amp;author=A.%20Biegert&amp;author=J.%20S%C3%B6ding&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;volume=106&amp;pages=3770-5&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR51" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yamada KD, Tomii K, Katoh K. Application of the MAFFT sequence alignment program to large data—reexamination of the usefulness of chained guide trees. Bioinformatics. 2016;32:3246–51.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Yamada%20KD%2C%20Tomii%20K%2C%20Katoh%20K.%20Application%20of%20the%20MAFFT%20sequence%20alignment%20program%20to%20large%20data%E2%80%94reexamination%20of%20the%20usefulness%20of%20chained%20guide%20trees.%20Bioinformatics.%202016%3B32%3A3246%E2%80%9351."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR52" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Katoh K, Misawa K, Kuma K, Miyata T. MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform. Nucleic Acids Res. 2002;30:3059–66.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gkf436"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12136088"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC135756"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=MAFFT%3A%20a%20novel%20method%20for%20rapid%20multiple%20sequence%20alignment%20based%20on%20fast%20Fourier%20transform&amp;author=K.%20Katoh&amp;author=K.%20Misawa&amp;author=K.%20Kuma&amp;author=T.%20Miyata&amp;journal=Nucleic%20Acids%20Res&amp;volume=30&amp;pages=3059-66&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR53" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Pupko T, Bell RE, Mayrose I, Glaser F, Ben-Tal N. Rate4Site: an algorithmic tool for the identification of functional regions in proteins by surface mapping of evolutionary determinants within their homologues. Bioinformatics. 2002;18:S71–7.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/18.suppl_1.S71"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12169533"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Rate4Site%3A%20an%20algorithmic%20tool%20for%20the%20identification%20of%20functional%20regions%20in%20proteins%20by%20surface%20mapping%20of%20evolutionary%20determinants%20within%20their%20homologues&amp;author=T.%20Pupko&amp;author=RE.%20Bell&amp;author=I.%20Mayrose&amp;author=F.%20Glaser&amp;author=N.%20Ben-Tal&amp;journal=Bioinformatics&amp;volume=18&amp;pages=S71-7&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR54" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Mayrose I, Graur D, Ben-Tal N, Pupko T. Comparison of site-specific rate-inference methods for protein sequences: empirical Bayesian methods are superior. Mol Biol Evol. 2004;21:1781–91.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/molbev/msh194"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15201400"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparison%20of%20site-specific%20rate-inference%20methods%20for%20protein%20sequences%3A%20empirical%20Bayesian%20methods%20are%20superior&amp;author=I.%20Mayrose&amp;author=D.%20Graur&amp;author=N.%20Ben-Tal&amp;author=T.%20Pupko&amp;journal=Mol%20Biol%20Evol&amp;volume=21&amp;pages=1781-91&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR55" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gabriel SB, Schaffner SF, Nguyen H, Moore JM, Roy J, Blumenstiel B, et al. The structure of haplotype blocks in the human genome. Science. 2002;296:2225–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1126/science.1069424"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12029063"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20structure%20of%20haplotype%20blocks%20in%20the%20human%20genome&amp;author=SB.%20Gabriel&amp;author=SF.%20Schaffner&amp;author=H.%20Nguyen&amp;author=JM.%20Moore&amp;author=J.%20Roy&amp;author=B.%20Blumenstiel&amp;journal=Science&amp;volume=296&amp;pages=2225-9&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR56" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Olsen KM, Kooyers NJ, Small LL. Recurrent gene deletions and the evolution of adaptive cyanogenesis polymorphisms in white clover (<em class="EmphasisTypeItalic">Trifolium repens</em> L.). Mol Ecol. 2013;22:724–38.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1365-294X.2012.05667.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22694056"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Recurrent%20gene%20deletions%20and%20the%20evolution%20of%20adaptive%20cyanogenesis%20polymorphisms%20in%20white%20clover%20%28Trifolium%20repens%20L.%29&amp;author=KM.%20Olsen&amp;author=NJ.%20Kooyers&amp;author=LL.%20Small&amp;journal=Mol%20Ecol&amp;volume=22&amp;pages=724-38&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR57" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Nei M, Suzuki Y, Nozawa M. The neutral theory of molecular evolution in the genomic era. Annu Rev Genom Hum G. 2010;11:265–89.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1146/annurev-genom-082908-150129"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20neutral%20theory%20of%20molecular%20evolution%20in%20the%20genomic%20era&amp;author=M.%20Nei&amp;author=Y.%20Suzuki&amp;author=M.%20Nozawa&amp;journal=Annu%20Rev%20Genom%20Hum%20G&amp;volume=11&amp;pages=265-89&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR58" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kreitman M. Methods to detect selection in populations with applications to the human. Annu Rev Genom Hum G. 2000;1:539–59.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1146/annurev.genom.1.1.539"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Methods%20to%20detect%20selection%20in%20populations%20with%20applications%20to%20the%20human&amp;author=M.%20Kreitman&amp;journal=Annu%20Rev%20Genom%20Hum%20G&amp;volume=1&amp;pages=539-59&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR59" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Anand K, Mathur D, Anant A, Garg LC. Structural studies of phosphoglucose isomerase from <em class="EmphasisTypeItalic">Mycobacterium tuberculosis</em> H37Rv. Acta Crystallogr F Struct Biol Commun. 2010;66:490–7.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1107/S1744309110011656"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Structural%20studies%20of%20phosphoglucose%20isomerase%20from%20Mycobacterium%20tuberculosis%20H37Rv&amp;author=K.%20Anand&amp;author=D.%20Mathur&amp;author=A.%20Anant&amp;author=LC.%20Garg&amp;journal=Acta%20Crystallogr%20F%20Struct%20Biol%20Commun&amp;volume=66&amp;pages=490-7&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR60" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wheat CW, Hagg CR, Marden JH, Hanski I, Frilander MJ. Nucleotide polymorphism at a gene (<em class="EmphasisTypeItalic">Pgi</em>) under balancing selection in a butterfly metapopulation. Mol Biol Evol. 2010;27:267–81.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/molbev/msp227"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19793833"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Nucleotide%20polymorphism%20at%20a%20gene%20%28Pgi%29%20under%20balancing%20selection%20in%20a%20butterfly%20metapopulation&amp;author=CW.%20Wheat&amp;author=CR.%20Hagg&amp;author=JH.%20Marden&amp;author=I.%20Hanski&amp;author=MJ.%20Frilander&amp;journal=Mol%20Biol%20Evol&amp;volume=27&amp;pages=267-81&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR61" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wheat CW, Watt WB, Pollock DD, Schulte PM. From DNA to fitness differences: sequences and structures of adaptive variants of <em class="EmphasisTypeItalic">Colias</em> phosphoglucose isomerase (PGI). Mol Biol Evol. 2006;23:499–512.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/molbev/msj062"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16292000"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=From%20DNA%20to%20fitness%20differences%3A%20sequences%20and%20structures%20of%20adaptive%20variants%20of%20Colias%20phosphoglucose%20isomerase%20%28PGI%29&amp;author=CW.%20Wheat&amp;author=WB.%20Watt&amp;author=DD.%20Pollock&amp;author=PM.%20Schulte&amp;journal=Mol%20Biol%20Evol&amp;volume=23&amp;pages=499-512&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR62" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wang BQ, DePasse JM, Watt WB. Evolutionary Genomics of <em class="EmphasisTypeItalic">Colias</em> Phosphoglucose Isomerase (PGI) Introns. J Mol Evol. 2012;74:96–111.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00239-012-9492-5"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22388354"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Evolutionary%20Genomics%20of%20Colias%20Phosphoglucose%20Isomerase%20%28PGI%29%20Introns&amp;author=BQ.%20Wang&amp;author=JM.%20DePasse&amp;author=WB.%20Watt&amp;journal=J%20Mol%20Evol&amp;volume=74&amp;pages=96-111&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR63" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kawabe A, Yamane K, Miyashita NT. DNA polymorphism at the cytosolic phosphoglucose isomerase (<em class="EmphasisTypeItalic">PgiC</em>) locus of the wild plant <em class="EmphasisTypeItalic">Arabidopsis thaliana</em>. Genetics. 2000;156:1339–47.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11063706"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1461314"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=DNA%20polymorphism%20at%20the%20cytosolic%20phosphoglucose%20isomerase%20%28PgiC%29%20locus%20of%20the%20wild%20plant%20Arabidopsis%20thaliana%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=A.%20Kawabe&amp;author=K.%20Yamane&amp;author=NT.%20Miyashita&amp;journal=Genetics&amp;volume=156&amp;pages=1339-47&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR64" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hahn MW. Toward a selection theory of molecular evolution. Evolution. 2008;62:255–65.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1558-5646.2007.00308.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18302709"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Toward%20a%20selection%20theory%20of%20molecular%20evolution&amp;author=MW.%20Hahn&amp;journal=Evolution&amp;volume=62&amp;pages=255-65&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR65" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Martin DP, Lemey P, Posada D. Analysing recombination in nucleotide sequences. Mol Ecol Resour. 2011;11:943–55.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1755-0998.2011.03026.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21592314"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Analysing%20recombination%20in%20nucleotide%20sequences&amp;author=DP.%20Martin&amp;author=P.%20Lemey&amp;author=D.%20Posada&amp;journal=Mol%20Ecol%20Resour&amp;volume=11&amp;pages=943-55&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR66" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Jensen JD, Foll M, Bernatchez L. The past, present and future of genomic scans for selection. Mol Ecol. 2016;25:1–4.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/mec.13493"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26745554"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20past%2C%20present%20and%20future%20of%20genomic%20scans%20for%20selection&amp;author=JD.%20Jensen&amp;author=M.%20Foll&amp;author=L.%20Bernatchez&amp;journal=Mol%20Ecol&amp;volume=25&amp;pages=1-4&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR67" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Teshima KM, Coop G, Przeworski M. How reliable are empirical genomic scans for selective sweeps? Genome Res. 2006;16:702–12.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1101/gr.5105206"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16687733"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1473181"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=How%20reliable%20are%20empirical%20genomic%20scans%20for%20selective%20sweeps%3F&amp;author=KM.%20Teshima&amp;author=G.%20Coop&amp;author=M.%20Przeworski&amp;journal=Genome%20Res&amp;volume=16&amp;pages=702-12&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR68" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Excoffier L, Hofer T, Foll M. Detecting loci under selection in a hierarchically structured population. Heredity (Edinb). 2009;103:285–98.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/hdy.2009.74"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Detecting%20loci%20under%20selection%20in%20a%20hierarchically%20structured%20population&amp;author=L.%20Excoffier&amp;author=T.%20Hofer&amp;author=M.%20Foll&amp;journal=Heredity%20%28Edinb%29&amp;volume=103&amp;pages=285-98&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR69" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lin HY, Kao YH, Chen ST, Meng M. Effects of inherited mutations on catalytic activity and structural stability of human glucose-6-phosphate isomerase expressed in <em class="EmphasisTypeItalic">Escherichia coli</em>. Biochim Biophys Acta. 2009;1794:315–23.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.bbapap.2008.11.004"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19064002"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Effects%20of%20inherited%20mutations%20on%20catalytic%20activity%20and%20structural%20stability%20of%20human%20glucose-6-phosphate%20isomerase%20expressed%20in%20Escherichia%20coli%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=HY.%20Lin&amp;author=YH.%20Kao&amp;author=ST.%20Chen&amp;author=M.%20Meng&amp;journal=Biochim%20Biophys%20Acta&amp;volume=1794&amp;pages=315-23&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR70" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Somarowthu S, Brodkin HR, D’Aquino JA, Ringe D, Ondrechen MJ, Beuning PJ. A tale of two isomerases: compact versus extended active sites in ketosteroid isomerase and phosphoglucose isomerase. Biochemistry. 2011;50:9283–95.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1021/bi201089v"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21970785"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20tale%20of%20two%20isomerases%3A%20compact%20versus%20extended%20active%20sites%20in%20ketosteroid%20isomerase%20and%20phosphoglucose%20isomerase&amp;author=S.%20Somarowthu&amp;author=HR.%20Brodkin&amp;author=JA.%20D%E2%80%99Aquino&amp;author=D.%20Ringe&amp;author=MJ.%20Ondrechen&amp;author=PJ.%20Beuning&amp;journal=Biochemistry&amp;volume=50&amp;pages=9283-95&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li></ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#TheAuthors" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="TheAuthors">Copyright</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="TheAuthors"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">©&nbsp;The Author(s)&nbsp;2017</div></div></section></div>
                        


                    </article>
                </main>

                <div class="c-content-layout__side u-text-sm">
                    <aside>
                        <div class="p-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="//hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-017-0032-6" data-track="click" data-track-category="PDF Download"
            data-track-action="Click download article PDF"
            data-track-label="10.1186/s41065-017-0032-6">Download PDF</a>
        </p>
    
                            
                            
    <button type="button"
       class="c-button c-button--secondary c-button--full-width c-button--expandable"
       data-toggle="collapse"
       data-target="export-citations"
       data-test="export-citations-button"
       data-sticky-update="true"
       aria-controls="export-citations"
       aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/s41065-017-0032-6"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0032-6?format=refman&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/s41065-017-0032-6"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0032-6?format=refman&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/s41065-017-0032-6"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0032-6?format=refman&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/s41065-017-0032-6"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0032-6?format=endnote&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/s41065-017-0032-6"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0032-6?format=endnote&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/s41065-017-0032-6"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0032-6?format=endnote&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/s41065-017-0032-6"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0032-6?format=bibtex&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/s41065-017-0032-6"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0032-6?format=bibtex&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/s41065-017-0032-6"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0032-6?format=bibtex&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
    </div>

                        </div>
                        
    
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
                                <div id="article-access-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/access-count/10.1186/s41065-017-0032-6 #accessCount-container"></div>
                                <div id="citation-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/citation-count/10.1186/s41065-017-0032-6 #count-container"></div>
                                <div id="altmetric-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/altmetric/10.1186/s41065-017-0032-6 #altmetric-container"></div>
                            </ul>
                        </div>
                        
    <div id="sociaMediaShare" class="p-separator u-js-shown">
        <h3 class="p-section-title">Share this article</h3>
        <ul id="shareButtons" class="c-social-links">
            <li class="c-social-links__item">
                <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-017-0032-6">
                    <span class="u-visually-hidden">Share on Twitter</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-twitter"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-017-0032-6">
                    <span class="u-visually-hidden">Share on Facebook</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-facebook"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click"
                        data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-017-0032-6">
                    <span class="u-visually-hidden">Share on LinkedIn</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-linkedin"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-017-0032-6">
                    <span class="u-visually-hidden">Share on Weibo</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-weibo"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnGooglePlus" class="js-btnShareOnGplus" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Google+" data-track-label="10.1186/s41065-017-0032-6">
                    <span class="u-visually-hidden">Share on Google Plus</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-gplus"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-017-0032-6">
                    <span class="u-visually-hidden">Share on Reddit</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-reddit"></use>
                    </svg>
                </button>
            </li>
        </ul>
    </div>
 
                        
    
        <div class="p-separator u-js-shown">
            <h3 class="p-section-title">See updates</h3>
            <button class="c-crossmark-button" data-target="crossmark" data-test="crossmark" data-track="click" data-track-category="Article Page" data-track-action="Click CrossMark" data-track-label="10.1186/s41065-017-0032-6">
                <img id="crossmark-icon" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" alt="Crossmark logo" />
            </button>
        </div>
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
                                <li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/s41065-017-0032-6" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=738&doi=10.1186%2Fs41065-017-0032-6&volume=154&issue=1&title=Contrasting+patterns+of+nucleotide+polymorphism+suggest+different+selective+regimes+within+different+parts+of+the+PgiC1+gene+in+Festuca+ovina+L.&author_name=Yuan+Li&start_page=1&end_page=10">Order reprint</a>

                                </li>
                            </ul>
                        </div>
                    </aside>
                    
    <div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">Hereditas</span>
                        </h4>
                        <p class="c-journal-footer__issn">ISSN: 1601-5223</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
                                
                                    <li class="c-journal-footer__contact-item">Submission enquiries: <a href="https://www.editorialmanager.com/here/" target="_blank">Access here and click Contact Us</a></li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a></li>
                                
                            </ul>
                        </div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class='tracker' style='display:none' src='/track/article/10.1186/s41065-017-0032-6' alt=""/>

         
    <footer>
        <div class="ctx-publisher-footer">
    <div class="c-navbar">
        <div class="u-container">
            <div class="c-navbar__logo u-margin-right-xl">
                <a href="https://www.biomedcentral.com">
                    <img src=/static/images/bmc/logos/logo-bmc-aj-594e03e20d.svg alt="BMC logo" data-test="navbar-logo-footer">

                </a>
            </div>
            <div class="c-navbar__content">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


            </div>
        </div>
    </div>
    <div class="c-publisher-footer" data-test="publisher-footer">
        <div class="u-container">
            
            <div class="c-flex c-flex--wrap c-flex--space-between" role="navigation" data-test="publisher-footer-menu">
                <div class="c-flex" aria-label="Publisher footer links">
                    
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/login">Receive BMC newsletters</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/account">Manage article alerts</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/policies">Policies</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/accessibility">Accessibility</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/press-centre">Press center</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/contact-us">Contact us</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/jobs">Careers</a>
                                    </li>
                                
                            </ul>
                        
                    
                </div>
                <div class="u-margin-bottom-lg">
                    <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                    <ul class="c-flex u-reset-list" data-test="footer-social-links">
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://twitter.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Twitter link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Twitter page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-twitter-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://www.facebook.com/BioMedCentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Facebook link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Facebook page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-facebook-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://plus.google.com/106226337984152901734/"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Google Plus link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Google Plus page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-gplus-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="http://www.weibo.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Weibo link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Weibo page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-weibo-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </div>
            <p class="u-reset-margin">
                By using this website, you agree to our
                <a class="u-text-inherit" href="//www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
                <a class="u-text-inherit" href="//www.biomedcentral.com/privacy-statement">Privacy
                    statement</a> and
                <a class="u-text-inherit" href="//www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
                
                    <a class="optanon-toggle-display u-text-inherit" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
                
            </p>
        </div>
    </div>
</div>


        <div class="c-sn-footer c-sn-footer--neutral">
    <div class="u-container">
        <img src=/static/images/bmc/logos/logo-springernature-8bd4e8d82f.svg class="c-sn-footer__logo" alt="Springer Nature Logo" itemprop="logo" >
        <p data-test="copyright">&#169; 2018 BioMed Central Ltd unless otherwise stated. Part of
            <a href="http://www.springernature.com"
               data-track="click"
               data-track-action="Click Springer Nature link"
               data-track-category="Corporate footer"
               itemscope itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>

    </div>

    
    <script type="text/javascript" src="//cdn.pbgrd.com/bmc.js" async="async"></script>
    <noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>
    <!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker -->

<script id="springerlink-event-tracker" src="https://event-tracker.springernature.com/dist/eventTracker.js"></script>
<script type="text/javascript">
    (function(){
        var dataLayer = window.dataLayer[0];
        var doi = dataLayer.content.article.doi;
        var imprint = dataLayer.content.contentInfo.imprint;
        var bpids = dataLayer.session.authentication.authenticationID;
        var publisher;
        var pageType = dataLayer.page.category.pageType;
        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }
        var eventObject = {'content_type' : pageType,'doi' : doi};

        if (imprint === 'BioMed Central') { publisher = 'BMC' }
        else if (imprint === 'SpringerOpen') { publisher = 'SpringerOpen' }
        else { throw new Error('Event Tracker Error: The publisher has an unexpected value') }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }
        new EventTracker({'platform':publisher}).sendEvent('display', eventObject);
    })();
</script>


    </body>
</html>

