<!DOCTYPE html>
<html lang="en" class="no-js">
    <head>
        <title>Mutations in the gene of the Gα subunit of the heterotrimeric G protein are the cause for the brachytic1 semi-dwarf phenotype in barley and applicable for practical breeding | Hereditas | Full Text</title>

        

    <meta name="citation_abstract" content="Short-culm mutants have been widely used in breeding programs to increase lodging resistance. In barley (Hordeum vulgare L.), several hundreds of short-culm mutants have been isolated over the years. The objective of the present study was to identify the Brachytic1 (Brh1) semi-dwarfing gene and to test its effect on yield and malting quality. Double-haploid lines generated through a cross between a brh1.a mutant and the European elite malting cultivar Quench, showed good malting quality but a decrease in yield. Especially the activities of the starch degrading enzymes β-amylase and free limit dextrinase were high. A syntenic approach comparing markers in barley to those in rice (Oryza sativa L.), sorghum (Sorghum bicolor Moench) and brachypodium (Brachypodium distachyon P. Beauv) helped us to identify Brh1 as an orthologue of rice D1 encoding the Gα subunit of a heterotrimeric G protein. We demonstrated that Brh1 is allelic to Ari-m. Sixteen different mutant alleles were described at the DNA level. Mutants in the Brh1 locus are deficient in the Gα subunit of a heterotrimeric G protein, which shows that heterotrimeric G proteins are important regulators of culm length in barley. Mutant alleles do not have any major negative effects on malting quality."/>

    <meta name="description" content="Short-culm mutants have been widely used in breeding programs to increase lodging resistance. In barley (Hordeum vulgare L.), several hundreds of short-culm mutants have been isolated over the years. The objective of the present study was to identify the Brachytic1 (Brh1) semi-dwarfing gene and to test its effect on yield and malting quality. Double-haploid lines generated through a cross between a brh1.a mutant and the European elite malting cultivar Quench, showed good malting quality but a decrease in yield. Especially the activities of the starch degrading enzymes β-amylase and free limit dextrinase were high. A syntenic approach comparing markers in barley to those in rice (Oryza sativa L.), sorghum (Sorghum bicolor Moench) and brachypodium (Brachypodium distachyon P. Beauv) helped us to identify Brh1 as an orthologue of rice D1 encoding the Gα subunit of a heterotrimeric G protein. We demonstrated that Brh1 is allelic to Ari-m. Sixteen different mutant alleles were described at the DNA level. Mutants in the Brh1 locus are deficient in the Gα subunit of a heterotrimeric G protein, which shows that heterotrimeric G proteins are important regulators of culm length in barley. Mutant alleles do not have any major negative effects on malting quality."/>

    <meta name="journal_id" content="41065"/>

    <meta name="dc.title" content="Mutations in the gene of the G&#945; subunit of the heterotrimeric G protein are the cause for the brachytic1 semi-dwarf phenotype in barley and applicable for practical breeding"/>

    <meta name="dc.source" content="Hereditas 2017 155:1"/>

    <meta name="dc.format" content="text/html"/>

    <meta name="dc.publisher" content="BioMed Central"/>

    <meta name="dc.date" content="2017-09-05"/>

    <meta name="dc.type" content="OriginalPaper"/>

    <meta name="dc.language" content="En"/>

    <meta name="dc.copyright" content="2017 The Author(s)"/>

    <meta name="dc.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="dc.description" content="Short-culm mutants have been widely used in breeding programs to increase lodging resistance. In barley (Hordeum vulgare L.), several hundreds of short-culm mutants have been isolated over the years. The objective of the present study was to identify the Brachytic1 (Brh1) semi-dwarfing gene and to test its effect on yield and malting quality. Double-haploid lines generated through a cross between a brh1.a mutant and the European elite malting cultivar Quench, showed good malting quality but a decrease in yield. Especially the activities of the starch degrading enzymes &#946;-amylase and free limit dextrinase were high. A syntenic approach comparing markers in barley to those in rice (Oryza sativa L.), sorghum (Sorghum bicolor Moench) and brachypodium (Brachypodium distachyon P. Beauv) helped us to identify Brh1 as an orthologue of rice D1 encoding the G&#945; subunit of a heterotrimeric G protein. We demonstrated that Brh1 is allelic to Ari-m. Sixteen different mutant alleles were described at the DNA level. Mutants in the Brh1 locus are deficient in the G&#945; subunit of a heterotrimeric G protein, which shows that heterotrimeric G proteins are important regulators of culm length in barley. Mutant alleles do not have any major negative effects on malting quality."/>

    <meta name="prism.issn" content="1601-5223"/>

    <meta name="prism.publicationName" content="Hereditas"/>

    <meta name="prism.publicationDate" content="2017-09-05"/>

    <meta name="prism.volume" content="155"/>

    <meta name="prism.number" content="1"/>

    <meta name="prism.section" content="OriginalPaper"/>

    <meta name="prism.startingPage" content="10"/>

    <meta name="prism.copyright" content="2017 The Author(s)"/>

    <meta name="prism.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="prism.url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0045-1"/>

    <meta name="prism.doi" content="doi:10.1186/s41065-017-0045-1"/>

    <meta name="citation_pdf_url" content="https://hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-017-0045-1"/>

    <meta name="citation_fulltext_html_url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0045-1"/>

    <meta name="citation_journal_title" content="Hereditas"/>

    <meta name="citation_publisher" content="BioMed Central"/>

    <meta name="citation_issn" content="1601-5223"/>

    <meta name="citation_title" content="Mutations in the gene of the G&#945; subunit of the heterotrimeric G protein are the cause for the brachytic1 semi-dwarf phenotype in barley and applicable for practical breeding"/>

    <meta name="citation_volume" content="155"/>

    <meta name="citation_issue" content="1"/>

    <meta name="citation_publication_date" content="2018/12"/>

    <meta name="citation_online_date" content="2017/09/05"/>

    <meta name="citation_firstpage" content="10"/>

    <meta name="citation_article_type" content="Research"/>

    <meta name="citation_fulltext_world_readable" content=""/>

    <meta name="dc.identifier" content="doi:10.1186/s41065-017-0045-1"/>

    <meta name="DOI" content="10.1186/s41065-017-0045-1"/>

    <meta name="citation_doi" content="10.1186/s41065-017-0045-1"/>

    <meta name="dc.creator" content="Ilka Braumann"/>

    <meta name="dc.creator" content="Christoph Dockter"/>

    <meta name="dc.creator" content="Sebastian Beier"/>

    <meta name="dc.creator" content="Axel Himmelbach"/>

    <meta name="dc.creator" content="Finn Lok"/>

    <meta name="dc.creator" content="Udda Lundqvist"/>

    <meta name="dc.creator" content="Birgitte Skadhauge"/>

    <meta name="dc.creator" content="Nils Stein"/>

    <meta name="dc.creator" content="Shakhira Zakhrabekova"/>

    <meta name="dc.creator" content="Ruonan Zhou"/>

    <meta name="dc.creator" content="Mats Hansson"/>

    <meta name="dc.subject" content="Animal Genetics and Genomics"/>

    <meta name="dc.subject" content="Evolutionary Biology"/>

    <meta name="citation_reference" content="citation_journal_title=Euphytica; citation_title=Global impact of mutation-derived varieties; citation_author=BS Ahloowalia, M Maluszynski, K Nichterlein; citation_volume=135; citation_publication_date=2004; citation_pages=187-204; citation_doi=10.1023/B:EUPH.0000014914.85465.4f; citation_id=CR1"/>

    <meta name="citation_reference" content="citation_journal_title=J Biomed Biotechnol; citation_title=Advances in BAC-based physical mapping and map integration strategies in plants; citation_author=R Ariyadasa, N Stein; citation_volume=2012; citation_publication_date=2012; citation_pages=184854; citation_doi=10.1155/2012/184854; citation_id=CR2"/>

    <meta name="citation_reference" content="citation_journal_title=Breed Sci; citation_title=Loss-of-function of a rice gibberellin biosynthetic gene, GA20 oxidase (GA20ox-2), led to the rice &quot;green revolution&quot;; citation_author=M Ashikari, A Sasaki, M Ueguchi-Tanaka; citation_volume=52; citation_publication_date=2002; citation_pages=143-150; citation_doi=10.1270/jsbbs.52.143; citation_id=CR3"/>

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci U S A; citation_title=Rice gibberellin-insensitive dwarf mutant gene Dwarf 1 encodes the alpha-subunit of GTP-binding protein; citation_author=M Ashikari, J Wu, M Yano, T Sasaki, A Yoshimura; citation_volume=96; citation_publication_date=1999; citation_pages=10284-10289; citation_doi=10.1073/pnas.96.18.10284; citation_id=CR4"/>

    <meta name="citation_reference" content="citation_journal_title=Barley Genet. Newslett.; citation_title=GA response in semidwarf barley; citation_author=A B&#246;rner; citation_volume=25; citation_publication_date=1996; citation_pages=24-27; citation_id=CR5"/>

    <meta name="citation_reference" content="citation_journal_title=Crop Breed Appl Biotech; citation_title=Effect of induced lodging on grain yield and quality of brewing barley; citation_author=E Caier&#227;o; citation_volume=6; citation_publication_date=2006; citation_pages=215-221; citation_doi=10.12702/1984-7033.v06n03a05; citation_id=CR6"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Physiol; citation_title=A semidwarf phenotype of barley uzu results from a nucleotide substitution in the gene encoding a putative brassinosteroid receptor; citation_author=M Chono, I Honda, H Zeniya; citation_volume=133; citation_publication_date=2003; citation_pages=1209-1219; citation_doi=10.1104/pp.103.026195; citation_id=CR7"/>

    <meta name="citation_reference" content="citation_journal_title=J Hered; citation_title=Characterization and molecular mapping of genes determining semidwarfism in barley; citation_author=LS Dahleen, LJ Vander Wal, JD Franckowiak; citation_volume=96; citation_publication_date=2005; citation_pages=654-662; citation_doi=10.1093/jhered/esi125; citation_id=CR8"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Physiol; citation_title=Induced variations in brassinosteroid genes define barley height and sturdiness, and expand the green revolution genetic toolkit; citation_author=C Dockter, D Gruszka, I Braumann; citation_volume=166; citation_publication_date=2014; citation_pages=1912-1927; citation_doi=10.1104/pp.114.250738; citation_id=CR9"/>

    <meta name="citation_reference" content="citation_journal_title=J Exp Bot; citation_title=Improving barley culm robustness for secured crop yield in a changing climate; citation_author=C Dockter, M Hansson; citation_volume=66; citation_publication_date=2015; citation_pages=3499-3509; citation_doi=10.1093/jxb/eru521; citation_id=CR10"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Physiol; citation_title=Genetic dissection of barley morphology and development; citation_author=A Druka, J Franckowiak, U Lundqvist; citation_volume=155; citation_publication_date=2011; citation_pages=617-627; citation_doi=10.1104/pp.110.166249; citation_id=CR11"/>

    <meta name="citation_reference" content="citation_journal_title=Euphytica; citation_title=Mutation genetics of salt tolerance in barley: an assessment of golden promise and other semi-dwarf mutants; citation_author=BP Foster; citation_volume=120; citation_publication_date=2001; citation_pages=317-328; citation_doi=10.1023/A:1017592618298; citation_id=CR12"/>

    <meta name="citation_reference" content="citation_journal_title=Z&#252;chter/Genet Breed Res; citation_title=Yield efficiency of the X-ray mutant Sval&#246;f&#39;s &#39;Pallas barley; citation_author=&#197; Gustafsson, G Ekman; citation_volume=37; citation_publication_date=1967; citation_pages=42-46; citation_id=CR13"/>

    <meta name="citation_reference" content="citation_journal_title=Biosci Biotechnol Biochem; citation_title=Uzu mutation in barley (Hordeum vulgare L.) reduces the leaf unrolling response to brassinolide; citation_author=I Honda, H Zeniya, K Yoneyama, M Chono, S Kaneko, Y Watanabe; citation_volume=67; citation_publication_date=2003; citation_pages=1194-1197; citation_doi=10.1271/bbb.67.1194; citation_id=CR14"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS Genet; citation_title=The U-box E3 ubiquitin ligase TUD1 functions with a heterotrimeric G alpha subunit to regulate Brassinosteroid-mediated growth in rice; citation_author=X Hu, Q Qian, T Xu; citation_volume=9; citation_publication_date=2013; citation_pages=e1003391; citation_doi=10.1371/journal.pgen.1003391; citation_id=CR15"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Genet; citation_title=Natural variation at the DEP1 locus enhances grain yield in rice; citation_author=X Huang, Q Qian, Z Liu; citation_volume=41; citation_publication_date=2009; citation_pages=494-497; citation_doi=10.1038/ng.352; citation_id=CR16"/>

    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=A physical, genetic and functional sequence assembly of the barley genome; citation_author=KF Mayer, R Waugh; citation_volume=491; citation_publication_date=2012; citation_pages=711-716; citation_id=CR17"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Genomics; citation_title=Marker development using SLAF-seq and whole-genome shotgun strategy to finemap the semi-dwarf gene ari-e in barley; citation_author=Q Jia, C Tan, J Wang; citation_volume=17; citation_publication_date=2016; citation_pages=911; citation_doi=10.1186/s12864-016-3247-4; citation_id=CR18"/>

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci U S A; citation_title=GTPase acceleration as the rate-limiting step in Arabidopsis G protein-coupled sugar signaling; citation_author=CA Johnston, JP Taylor, Y Gao; citation_volume=104; citation_publication_date=2007; citation_pages=17317-17322; citation_doi=10.1073/pnas.0704751104; citation_id=CR19"/>

    <meta name="citation_reference" content="citation_journal_title=Hereditas; citation_title=Induction of breviaristatum mutants in barley; citation_author=J Kucera, U Lundqvist, A Gustafsson; citation_volume=80; citation_publication_date=1975; citation_pages=263-278; citation_doi=10.1111/j.1601-5223.1975.tb01525.x; citation_id=CR20"/>

    <meta name="citation_reference" content="citation_journal_title=Barley Genet. Newslett.; citation_title=Fine mapping of a semi-dwarf gene brachytic1 in barley; citation_author=M Li, D Kudrna, A Kleinhofs; citation_volume=31; citation_publication_date=2001; citation_pages=14-17; citation_id=CR21"/>

    <meta name="citation_reference" content="Mascher M, Gundlach H, Himmelbach A, et al. A chromosome conformation capture ordered sequence of the barley genome: Nature In press; 2017."/>

    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=&#39;Green revolution&#39; genes encode mutant gibberellin response modulators; citation_author=J Peng, DE Richards, NM Hartley; citation_volume=400; citation_publication_date=1999; citation_pages=256-261; citation_doi=10.1038/22307; citation_id=CR23"/>

    <meta name="citation_reference" content="citation_journal_title=Genetics; citation_title=The nature of the interactions of genes affecting four quantitative characters in a cross between Hordeum deficiens and vulgare; citation_author=L Powers; citation_volume=21; citation_publication_date=1936; citation_pages=398-420; citation_id=CR24"/>

    <meta name="citation_reference" content="citation_journal_title=Agricult Rev; citation_title=Lodging in cereals; citation_author=S Rajkumara; citation_volume=29; citation_publication_date=2008; citation_pages=55-60; citation_id=CR25"/>

    <meta name="citation_reference" content="citation_journal_title=Breed Sci; citation_title=Spontaneous brassinolide-insensitive barley mutants &#8220;uzu.&#8221; adapted to East Asia; citation_author=D Sashio, K Tanno, M Chono, I Honda, H Kitano, K Tekeda; citation_volume=54; citation_publication_date=2004; citation_pages=409-416; citation_doi=10.1270/jsbbs.54.409; citation_id=CR26"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Genomics; citation_title=BAC library resources for map-based cloning and physical map construction in barley (Hordeum Vulgare L.); citation_author=D Schulte, R Ariyadasa, B Shi; citation_volume=12; citation_publication_date=2011; citation_pages=247; citation_doi=10.1186/1471-2164-12-247; citation_id=CR27"/>

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci U S A; citation_title=Semidwarf (sd-1), &#8220;green revolution&#8221; rice, contains a defective gibberellin 20-oxidase gene; citation_author=W Spielmeyer, MH Ellis, PM Chandler; citation_volume=99; citation_publication_date=2002; citation_pages=9043-9048; citation_doi=10.1073/pnas.132266399; citation_id=CR28"/>

    <meta name="citation_reference" content="citation_journal_title=Curr Opin Plant Biol; citation_title=G protein signaling in plants: minus times minus equals plus; citation_author=D Stateczny, J Oppenheimer, P Bommert; citation_volume=34; citation_publication_date=2016; citation_pages=127-135; citation_doi=10.1016/j.pbi.2016.11.001; citation_id=CR29"/>

    <meta name="citation_reference" content="citation_journal_title=J Agric Res; citation_title=Genetic and cytological studies on a brachytic mutant in barley; citation_author=SP Swenson; citation_volume=60; citation_publication_date=1940; citation_pages=687-713; citation_id=CR30"/>

    <meta name="citation_reference" content="citation_title=The origin and evolution of cultivated barley; citation_inbook_title=Advances in genetics; citation_publication_date=1955; citation_pages=227-266; citation_id=CR31; citation_author=R Takahashi; citation_publisher=Academic Press"/>

    <meta name="citation_reference" content="citation_journal_title=Annu Rev Plant Biol; citation_title=The plant heterotrimeric G-protein complex; citation_author=BR Temple, AM Jones; citation_volume=58; citation_publication_date=2007; citation_pages=249-266; citation_doi=10.1146/annurev.arplant.58.032806.103827; citation_id=CR32"/>

    <meta name="citation_reference" content="citation_journal_title=Barley Genet Newslett; citation_title=Allelic relationships of genes for short-awned mutants in barley; citation_author=T Tsuchiya; citation_volume=4; citation_publication_date=1974; citation_pages=80-81; citation_id=CR33"/>

    <meta name="citation_reference" content="citation_journal_title=Open Biol; citation_title=Heterotrimeric G protein signalling in the plant kingdom; citation_author=D Urano, JG Chen, JR Botella, AM Jones; citation_volume=3; citation_publication_date=2013; citation_pages=120186; citation_doi=10.1098/rsob.120186; citation_id=CR34"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=HvDep1 is a positive regulator of culm elongation and grain size in barley and impacts yield in an environment-dependent manner; citation_author=T Wendt, I Holme, C Dockter; citation_volume=11; citation_publication_date=2016; citation_pages=e0168924; citation_doi=10.1371/journal.pone.0168924; citation_id=CR35"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Plant Biol; citation_title=Characterization of the sdw1 semi-dwarf gene in barley; citation_author=Y Xu, Q Jia, G Zhou; citation_volume=17; citation_publication_date=2017; citation_pages=11; citation_doi=10.1186/s12870-016-0964-4; citation_id=CR36"/>

    <meta name="citation_reference" content="citation_journal_title=Canadian J Plant Sci; citation_title=Analysis of dwarfing genes in Zhepi 1 and Aizao 3: two dwarfing gene donors in barley breeding in China; citation_author=J Zhang, Z Li, C Zhang; citation_volume=87; citation_publication_date=2007; citation_pages=93-96; citation_doi=10.4141/P06-042; citation_id=CR37"/>

    <meta name="citation_reference" content="citation_journal_title=Euphytica; citation_title=Tracing sources of dwarfing genes in barley breeding in China; citation_author=J Zhang, W Zhang; citation_volume=131; citation_publication_date=2003; citation_pages=285-292; citation_doi=10.1023/A:1023955131523; citation_id=CR38"/>

    <meta name="citation_author" content="Ilka Braumann"/>

    <meta name="citation_author_institution" content="Carlsberg Research Laboratory, Copenhagen V, Denmark"/>

    <meta name="citation_author" content="Christoph Dockter"/>

    <meta name="citation_author_institution" content="Carlsberg Research Laboratory, Copenhagen V, Denmark"/>

    <meta name="citation_author" content="Sebastian Beier"/>

    <meta name="citation_author_institution" content="Leibniz Institute of Plant Genetics and Crop Plant Research (IPK), Stadt Seeland, Germany"/>

    <meta name="citation_author" content="Axel Himmelbach"/>

    <meta name="citation_author_institution" content="Leibniz Institute of Plant Genetics and Crop Plant Research (IPK), Stadt Seeland, Germany"/>

    <meta name="citation_author" content="Finn Lok"/>

    <meta name="citation_author_institution" content="Carlsberg Research Laboratory, Copenhagen V, Denmark"/>

    <meta name="citation_author" content="Udda Lundqvist"/>

    <meta name="citation_author_institution" content="Nordic Genetic Resource Center (NordGen), Alnarp, Sweden"/>

    <meta name="citation_author" content="Birgitte Skadhauge"/>

    <meta name="citation_author_institution" content="Carlsberg Research Laboratory, Copenhagen V, Denmark"/>

    <meta name="citation_author" content="Nils Stein"/>

    <meta name="citation_author_institution" content="Leibniz Institute of Plant Genetics and Crop Plant Research (IPK), Stadt Seeland, Germany"/>

    <meta name="citation_author" content="Shakhira Zakhrabekova"/>

    <meta name="citation_author_institution" content="Department of Biology, Lund University, Lund, Sweden"/>

    <meta name="citation_author" content="Ruonan Zhou"/>

    <meta name="citation_author_institution" content="Leibniz Institute of Plant Genetics and Crop Plant Research (IPK), Stadt Seeland, Germany"/>

    <meta name="citation_author" content="Mats Hansson"/>

    <meta name="citation_author_institution" content="Department of Biology, Lund University, Lund, Sweden"/>



        <meta charset="UTF-8"/>
        <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
        <meta name="viewport" content="width=device-width, initial-scale=1"/>

        <!-- For IE 9 and below -->
<!--[if IE]><link rel="shortcut icon" href=/static/images/favicons//favicon.ico><![endif]-->
<!-- Touch Icons - iOS and Android 2.1+ -->
<link rel="apple-touch-icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>
<!-- Firefox, Chrome, Safari, IE 11+ and Opera -->
<link rel="icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>


        <script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script>
        <script>(function(d){if (sessionStorage.fontsLoaded){d.style.setProperty('--font-family-sans', 'Europa');d.style.setProperty('--font-family-serif', 'Noto Serif')}})(document.documentElement);</script>

        <link rel="stylesheet" media="screen" href=/static/styles/basic/basic-a31ca85bc2.css>
<link rel="stylesheet" media="print" href=/static/styles/bmc/print-3a6240f456.css>

<link rel="stylesheet" class="js-ctm" href=/static/styles/bmc/main-84414d3d47.css
      media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">

        

        <script src="/static/js/app-bundle-8d61e6ef37.js" defer></script>
        
        <script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script>
        
    <script type="text/javascript">
        config = {
            env: 'live',
            site: 'hereditasjournal.biomedcentral.com',
            siteWithPath: 'hereditasjournal.biomedcentral.com' + window.location.pathname,
            twitterHashtag: '',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/s41065-017-0045-1',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central'
        };
    </script>

        
    <script type="text/javascript">
        dataLayer = [{"content":{"article":{"doi":"10.1186/s41065-017-0045-1","articleType":"Research","peerReviewType":"Closed","supplement":null,"keywords":"Ari-i;Ari-m;Brh1;Hordeum vulgare;Semi-dwarf"},"contentInfo":{"imprint":"BioMed Central","title":"Mutations in the gene of the Gα subunit of the heterotrimeric G protein are the cause for the brachytic1 semi-dwarf phenotype in barley and applicable for practical breeding","publishedAt":1504569600000,"publishedAtDate":"2017-09-05","author":["Ilka Braumann","Christoph Dockter","Sebastian Beier","Axel Himmelbach","Finn Lok","Udda Lundqvist","Birgitte Skadhauge","Nils Stein","Shakhira Zakhrabekova","Ruonan Zhou","Mats Hansson"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":"CC BY + CC0","openAccess":true},"environment":"live"},"journal":{"siteKey":"hereditasjournal.biomedcentral.com","volume":"155","issue":"1","title":"Hereditas","type":"AJs Life","journalID":41065,"gaCode":"UA-62066850-11","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Research","publishingSegment":"Genetics & Heredity (Life)"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}}}]

    </script>

        <script src="https://cdn.cookielaw.org/consent/7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script>
        
    
        
    

        
    
        
            
                <!-- Google Tag Manager -->
                <script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script>
                <!-- End Google Tag Manager -->
            
        
    


        
    
    <link rel="canonical" href="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0045-1"/>
    

        
        
        <meta property="og:url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0045-1"/>
        <meta property="og:type" content="article"/>
        <meta property="og:site_name" content="Hereditas"/>
        <meta property="og:title" content="Mutations in the gene of the Gα subunit of the heterotrimeric G protein are the cause for the brachytic1 semi-dwarf phenotype in barley and applicable for practical breeding"/>
        <meta property="og:description" content="Short-culm mutants have been widely used in breeding programs to increase lodging resistance. In barley (Hordeum vulgare L.), several hundreds of short-culm mutants have been isolated over the years. The objective of the present study was to identify the Brachytic1 (Brh1) semi-dwarfing gene and to test its effect on yield and malting quality. Double-haploid lines generated through a cross between a brh1.a mutant and the European elite malting cultivar Quench, showed good malting quality but a decrease in yield. Especially the activities of the starch degrading enzymes β-amylase and free limit dextrinase were high. A syntenic approach comparing markers in barley to those in rice (Oryza sativa L.), sorghum (Sorghum bicolor Moench) and brachypodium (Brachypodium distachyon P. Beauv) helped us to identify Brh1 as an orthologue of rice D1 encoding the Gα subunit of a heterotrimeric G protein. We demonstrated that Brh1 is allelic to Ari-m. Sixteen different mutant alleles were described at the DNA level. Mutants in the Brh1 locus are deficient in the Gα subunit of a heterotrimeric G protein, which shows that heterotrimeric G proteins are important regulators of culm length in barley. Mutant alleles do not have any major negative effects on malting quality."/>
        <meta property="og:image" content="https://images.springer.com/sgw/journals/medium/41065.jpg"/>
    
    </head>

    <body class="journal journal-fulltext"
          
          >
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK"
                            height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><feColorMatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"/></filter></defs><symbol id="icon-arrow-down" viewBox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"/></symbol><symbol id="icon-arrow-left-bullet" viewBox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"/></symbol><symbol id="icon-arrow-right-bullet" viewBox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"/></symbol><symbol id="icon-arrow-right" viewBox="0 0 32 64"><path d="M0 0v64l32-32z"/></symbol><symbol id="icon-close" viewBox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"/></symbol><symbol id="icon-community" viewBox="0 0 60.643 31.01"><mask maskUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><path fill="#FFF" d="M0 31.01h60.643V0H0z" filter="url(#a)"/></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"/></symbol><symbol id="icon-download" viewBox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"/></symbol><symbol id="icon-ext-link" viewBox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"/><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"/><path fill="#676767" d="M9.7 0L16 6.2V0z"/></symbol><symbol id="icon-info-bordered" viewBox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"/><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"/></symbol><symbol id="icon-opr" viewBox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"/><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"/></symbol><symbol id="icon-quotation" viewBox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"/></symbol><symbol id="icon-remove" viewBox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"/><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"/></symbol><symbol id="icon-ribbon" viewBox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"/></symbol><symbol id="icon-rss" viewBox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"/><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"/><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"/></g></symbol><symbol id="icon-search" viewBox="0 0 16 16"><path d="M6.276 12.257C2.815 12.257 0 9.508 0 6.128 0 2.749 2.815 0 6.276 0c3.462 0 6.277 2.749 6.277 6.128.002 3.38-2.815 6.129-6.277 6.129zm0-10.772c-2.622 0-4.755 2.083-4.755 4.643 0 2.562 2.133 4.644 4.755 4.644 2.623 0 4.756-2.082 4.756-4.644 0-2.56-2.134-4.643-4.756-4.643z"/><path d="M16 13.751L13.641 16l-4.875-4.82 2.36-2.248L16 13.751z"/></symbol><symbol id="icon-speech-bubble" viewBox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"/></symbol><symbol id="icon-error" viewBox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"/></symbol><symbol id="icon-info" viewBox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"/></symbol><symbol id="icon-success" viewBox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"/></symbol><symbol id="icon-warning" viewBox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"/></symbol><symbol id="icon-facebook-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"/></symbol><symbol id="icon-facebook" viewBox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"/><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"/></symbol><symbol id="icon-gplus-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M480.5 273.117c.123.076.26.185.416.334.146.152.289.343.432.562.135.211.258.459.359.746.086.286.129.619.129 1.004-.012.7-.164 1.262-.465 1.683a5.374 5.374 0 0 1-.461.562c-.186.176-.375.35-.58.527a2.65 2.65 0 0 0-.322.41.908.908 0 0 0-.176.553c0 .205.059.375.18.508.102.129.199.238.299.336l.662.545a6.48 6.48 0 0 1 1.086 1.119c.293.415.445.955.459 1.625 0 .949-.42 1.789-1.256 2.521-.869.76-2.123 1.15-3.762 1.177-1.373-.015-2.4-.308-3.074-.879-.682-.534-1.023-1.172-1.023-1.917 0-.359.111-.766.336-1.211.215-.443.602-.837 1.172-1.174a6.126 6.126 0 0 1 1.998-.725 15.415 15.415 0 0 1 1.723-.173 3.996 3.996 0 0 1-.377-.596 1.347 1.347 0 0 1-.193-.74c0-.174.023-.317.074-.433a8.83 8.83 0 0 1 .121-.338 5.973 5.973 0 0 1-.633.035c-1.043-.012-1.838-.339-2.381-.979-.574-.603-.861-1.3-.861-2.097 0-.961.404-1.832 1.217-2.616.559-.46 1.135-.757 1.736-.898a8.494 8.494 0 0 1 1.67-.182h3.916l-1.209.704h-1.207v.005l-.005.002zm.754 11.276c0-.498-.162-.932-.484-1.302-.346-.348-.881-.778-1.609-1.29a4.045 4.045 0 0 0-.439-.02c-.102-.016-.354 0-.768.039a8.141 8.141 0 0 0-1.244.275c-.1.036-.24.096-.42.17a2.713 2.713 0 0 0-.553.355 1.985 1.985 0 0 0-.459.592c-.145.25-.215.551-.215.901 0 .687.311 1.253.932 1.698.596.444 1.404.68 2.432.688.928-.014 1.629-.217 2.117-.612.476-.385.71-.886.71-1.494zm-2.729-5.743a1.843 1.843 0 0 0 1.289-.556c.164-.249.271-.506.318-.767a7.32 7.32 0 0 0 .041-.659c0-.772-.195-1.552-.594-2.336a2.611 2.611 0 0 0-.73-.921 1.894 1.894 0 0 0-1.064-.352 1.723 1.723 0 0 0-1.324.64c-.299.437-.438.92-.428 1.457 0 .709.209 1.444.623 2.211.201.356.459.659.775.912.323.248.686.371 1.094.371zm9.698.077h-2.055v-2.056h-.996v2.056h-2.053v.998h2.053v2.052h.996v-2.052h2.055z"/></symbol><symbol id="icon-gplus" viewBox="0 0 24 24"><circle fill="#D84A36" cx="12" cy="12" r="12"/><g fill="#FFF"><path d="M12.012 12.111c-.304-.33-.07-.842.155-1.078.226-.241.639-.56.829-.74.193-.184.66-.605.674-1.868 0 0 .112-1.139-1.127-2.06h.977l1.038-.593h-3.507s-2.083-.067-3.085 1.207c-1 1.276-.966 2.871.114 3.77 1.08.898 2.371.648 2.371.648s-.197.422-.163.775c.036.353.421.849.504.965 0 0-2.386-.011-3.551 1.033-1.165 1.053-1.29 3.097 1.021 3.803 2.312.707 5.303-.055 5.906-2.277 0 0 .262-.908-.189-1.683-.459-.775-1.663-1.571-1.967-1.902zm-2.555-1.83c-.744-1-1.308-2.953 0-3.783 0 0 .955-.579 1.801.266.846.846 1.082 2.277.928 3.003-.15.727-.563.897-.739 1.008-.185.111-1.249.504-1.99-.494zm3.668 5.481c-.035.393-.083 1.254-1.491 1.664-1.409.408-3.826-.264-3.757-1.914 0 0 .006-.698.449-1.089.445-.396 1.094-.794 3.029-.819 0 0 .643.473 1.072.809.427.332.733.952.698 1.349z"/><path d="M16.323 11.055V9.803h-.752v1.252h-1.255v.748h1.255v1.256h.752v-1.256h1.248v-.748z"/></g></symbol><symbol id="icon-linkedin" viewBox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"/><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"/><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"/></g></symbol><symbol id="icon-reddit" viewBox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"/><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"/><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"/><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"/><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"/><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"/><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"/><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"/></symbol><symbol id="icon-twitter-bordered" viewBox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"/><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/></symbol><symbol id="icon-twitter" viewBox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"/><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"/></symbol><symbol id="icon-weibo-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"/><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"/><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"/></symbol><symbol id="icon-weibo" viewBox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"/><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"/><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"/><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"/><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"/><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"/></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner c-banner--info">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner" >
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1"
                 data-gpt-unitpath="/270604982/bmc/hereditasjournal/articles"
                 data-gpt-sizes="728x90,970x90"
                 data-gpt-targeting="pos=LB1;doi=10.1186/s41065-017-0045-1;kwrd=Ari-i,Ari-m,Brh1,Hordeum vulgare,Semi-dwarf;pmc=L32030,L21001;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-017-0045-1&amp;kwrd=Ari-i,Ari-m,Brh1,Hordeum vulgare,Semi-dwarf&amp;pmc=L32030,L21001&amp;">
                        <img data-test="gpt-advert-fallback-img"
                             src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-017-0045-1&amp;kwrd=Ari-i,Ari-m,Brh1,Hordeum vulgare,Semi-dwarf&amp;pmc=L32030,L21001&amp;"
                             alt="Advertisement"
                             width="728"
                             height="90">
                    </a>
                </noscript>
            </div>
        </div>
    </div>

        
         
    <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>


<header class="c-popup-search ctx-publisher-header" id="top" data-test="publisher-header">
    <nav class="c-navbar">
        <div class="u-container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization" />
                <div class="c-logo">
    <a href="https://www.biomedcentral.com" itemprop="url">
        <img src=/static/images/bmc/logos/logo-bmc-white-aj-e3a70ae848.svg alt="BMC" itemprop="logo">
        <div class="c-logo__strapline">
            <img src=/static/images/bmc/logos/logo-bmc-white-strapline-sn-e7b1d6e67f.svg alt="Part of Springer Nature">
        </div>
    </a>
</div>
            </div>
            <button
                type="button"
                class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg"
                id="publisher-header-menu-button"
                data-toggle="collapse"
                data-test="publisher-header-menu-toggle"
                data-toggle-linked="publisher-header"
                data-target="publisher-header-search publisher-header-menu"
                aria-controls="publisher-header-menu publisher-header-search"
                aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


                <ul class="c-navbar__nav">
                    <li class="c-navbar__item">
                        <a data-replace="true"
                           data-placeholder="/placeholder/v1/username"
                           class="c-navbar__link"
                           href="https://www.biomedcentral.com/login"
                           data-test="login-link"
                           data-track="click"
                           data-track-category="Account"
                           data-track-action="Click Login Link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                    <li class="c-navbar__item">
                        <button
                            type="button"
                            class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg"
                            data-toggle="collapse"
                            data-test="header-search-button"
                            data-target="publisher-header-search publisher-header-menu publisher-header-menu-button"
                            aria-controls="publisher-header-search publisher-header-menu"
                            aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true">
                                    <use xlink:href="#icon-search"></use>
                                </svg>
                            </span>
                        </button>
                    </li>
                </ul>
            </div>
        </div>
    </nav>
    <div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field c-form-field--inline" method="GET" action="//www.biomedcentral.com/search" data-track="submit"
        data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label data-track-label="" data-test="global-search">
        <label for="publisherSearch" class="u-visually-hidden">Search all BMC articles</label>
        <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text" value="" placeholder="Search all BMC articles"/>
        <button class="c-button c-button--secondary" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true">
        <use xlink:href="#icon-search"></use>
    </svg>
</button>


        <input type="hidden" name="searchType" value="publisherSearch"/>
    </form>
</div>

            </div>
        </div>
    </div>
</header>


        
            <header class="c-journal-header c-journal-header--hereditas ctx-journal-header">
                <div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="/">
    
        
    
    <span class="c-journal-title__text ">Hereditas</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="u-container">
                        
                            <button type="button"
                                    class="c-navbar__link c-dropdown u-hidden-at-lg"
                                    data-toggle="collapse"
                                    data-test="journal-header-menu"
                                    data-toggle-linked="journal-header"
                                    data-target="journal-menu"
                                    aria-controls="journal-menu"
                                    aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav">
                                    <ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/'>Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/about'>About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href='/articles'>Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/submission-guidelines'>Submission Guidelines</a>
                                            </li>
                                        
                                    </ul>
                                </nav>
                            </div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--ocean-light-blue">
                    
                </div>
            </header>
            
        

        <div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-content-layout c-content-layout--fulltext" data-component="sticky-container">
                <div class="c-content-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3><ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc=""><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Methods">Methods</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec18" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results">Results</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec24" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Discussion">Discussion</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec25" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Conclusions">Conclusions</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li></ol></nav>
                </div>

                <main class="c-content-layout__main">
                    <article itemscope itemtype="http://schema.org/ScholarlyArticle">
                        
                        
    <div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
            <li class="c-article-identifiers__item" data-test="article-category">Research</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
    </div>

                        <div class="FulltextWrapper"><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">Mutations in the gene of the Gα subunit of the heterotrimeric G protein are the cause for the <em xmlns="" class="EmphasisTypeItalic">brachytic1</em> semi-dwarf phenotype in barley and applicable for practical breeding</h1></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true"><ul class="u-listReset"><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Ilka&nbsp;Braumann</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Christoph&nbsp;Dockter</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Sebastian&nbsp;Beier</span><sup><a href="#Aff2" class="AffiliationID">2</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Axel&nbsp;Himmelbach</span><sup><a href="#Aff2" class="AffiliationID">2</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Finn&nbsp;Lok</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Udda&nbsp;Lundqvist</span><sup><a href="#Aff3" class="AffiliationID">3</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Birgitte&nbsp;Skadhauge</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Nils&nbsp;Stein</span><sup><a href="#Aff2" class="AffiliationID">2</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Shakhira&nbsp;Zakhrabekova</span><sup><a href="#Aff4" class="AffiliationID">4</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Ruonan&nbsp;Zhou</span><sup><a href="#Aff2" class="AffiliationID">2</a></sup> and </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Mats&nbsp;Hansson</span><sup><a href="#Aff4" class="AffiliationID">4</a></sup><a href="mailto:mats.hansson@biol.lu.se" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a><a class="Orcid" title="View ORCID ID profile" href="http://orcid.org/0000-0002-0168-9968" target="_blank" rel="noopener"><span class="OrcidIcon" aria-hidden="true"></span><span class="u-srOnly">View ORCID ID profile</span></a></li></ul></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation"><div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">Hereditas</span></span><span class="ArticleCitation_Year">2017</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">155</strong>:10</span></span></div><p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/s41065-017-0045-1">https://doi.org/10.1186/s41065-017-0045-1</a></p><p class="Copyright">© &nbsp;The Author(s)&nbsp;2017</p></div></div><ul class="ArticleHistory u-reset-list"><li class="History HistoryReceived"><strong>Received: </strong><span>22&nbsp;April&nbsp;2017</span></li><li class="History HistoryAccepted"><strong>Accepted: </strong><span>21&nbsp;August&nbsp;2017</span></li><li class="History HistoryOnlineDate"><strong>Published: </strong><span itemprop="datePublished">5&nbsp;September&nbsp;2017</span></li></ul></div>
                        
                        
                        
    

                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2><div class="c-section__content" id="Abstract"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p id="Par1" class="Para">Short-culm mutants have been widely used in breeding programs to increase lodging resistance. In barley (<em xmlns="" class="EmphasisTypeItalic">Hordeum vulgare</em> L.), several hundreds of short-culm mutants have been isolated over the years. The objective of the present study was to identify the <em xmlns="" class="EmphasisTypeItalic">Brachytic1</em> (<em xmlns="" class="EmphasisTypeItalic">Brh1</em>) semi-dwarfing gene and to test its effect on yield and malting quality.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Results</h3>
                <p id="Par2" class="Para">Double-haploid lines generated through a cross between a <em xmlns="" class="EmphasisTypeItalic">brh1.a</em> mutant and the European elite malting cultivar Quench, showed good malting quality but a decrease in yield. Especially the activities of the starch degrading enzymes β-amylase and free limit dextrinase were high. A syntenic approach comparing markers in barley to those in rice (<em xmlns="" class="EmphasisTypeItalic">Oryza sativa</em> L.), sorghum (<em xmlns="" class="EmphasisTypeItalic">Sorghum bicolor</em> Moench) and brachypodium (<em xmlns="" class="EmphasisTypeItalic">Brachypodium distachyon</em> P. Beauv) helped us to identify <em xmlns="" class="EmphasisTypeItalic">Brh1</em> as an orthologue of rice <em xmlns="" class="EmphasisTypeItalic">D1</em> encoding the Gα subunit of a heterotrimeric G protein. We demonstrated that <em xmlns="" class="EmphasisTypeItalic">Brh1</em> is allelic to <em xmlns="" class="EmphasisTypeItalic">Ari-m</em>. Sixteen different mutant alleles were described at the DNA level.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusions</h3>
                <p id="Par3" class="Para">Mutants in the <em xmlns="" class="EmphasisTypeItalic">Brh1</em> locus are deficient in the Gα subunit of a heterotrimeric G protein, which shows that heterotrimeric G proteins are important regulators of culm length in barley. Mutant alleles do not have any major negative effects on malting quality.</p>
              </div></div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords"><li class="c-keywords__item" itemprop="about">
                <em xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="EmphasisTypeItalic">Ari-i</em>
              </li><li class="c-keywords__item" itemprop="about">
                <em xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="EmphasisTypeItalic">Ari-m</em>
              </li><li class="c-keywords__item" itemprop="about">
                <em xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="EmphasisTypeItalic">Brh1</em>
              </li><li class="c-keywords__item" itemprop="about">
                <em xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="EmphasisTypeItalic">Hordeum vulgare</em>
              </li><li class="c-keywords__item" itemprop="about">Semi-dwarf</li></ul></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par4" class="Para">Semi-dwarfism is regarded as a valuable trait in cereal breeding, as cultivars with a short stature show increased lodging resistance [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>, <span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>]. Due to the potential agronomic usefulness, many different types of semi-dwarf mutants have been identified in barley (<em xmlns="" class="EmphasisTypeItalic">Hordeum vulgare</em>) during the last 70&nbsp;years. These semi-dwarf mutants have obtained different names since they often have other pleiotropic characters which might have been in focus of the involved research group. The major semi-dwarf subgroups are <em xmlns="" class="EmphasisTypeItalic">Brachytic</em> (<em xmlns="" class="EmphasisTypeItalic">Brh</em>), <em xmlns="" class="EmphasisTypeItalic">Breviaristatum</em> (<em xmlns="" class="EmphasisTypeItalic">Ari</em>), <em xmlns="" class="EmphasisTypeItalic">Slender dwarf</em> (<em xmlns="" class="EmphasisTypeItalic">Sld</em>), <em xmlns="" class="EmphasisTypeItalic">Erectoides</em> (<em xmlns="" class="EmphasisTypeItalic">Ert</em>), <em xmlns="" class="EmphasisTypeItalic">Semidwarf</em> (<em xmlns="" class="EmphasisTypeItalic">Sdw</em>), <em xmlns="" class="EmphasisTypeItalic">Semi-brachytic</em> (<em xmlns="" class="EmphasisTypeItalic">Uzu</em>) and <em xmlns="" class="EmphasisTypeItalic">Dense spike</em> (<em xmlns="" class="EmphasisTypeItalic">Dsp</em>) [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>]. However, only a few barley semi-dwarfing genes have successfully been applied in breeding programs – The <em xmlns="" class="EmphasisTypeItalic">uzu1.a</em> allele was one of the first short-culm alleles to be used [<span class="CitationRef"><a href="#CR31" data-toggle="citation">31</a></span>] and has been introduced in almost all Japanese hull-less barley cultivars [<span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>]; two different alleles of the <em xmlns="" class="EmphasisTypeItalic">Semidwarf 1</em> gene, <em xmlns="" class="EmphasisTypeItalic">sdw1.c</em> (originally named <em xmlns="" class="EmphasisTypeItalic">denso</em>) and <em xmlns="" class="EmphasisTypeItalic">sdw1.d</em>, are most likely the dominating short-culm alleles in the today’s European elite cultivars [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>]; <em xmlns="" class="EmphasisTypeItalic">ert-k.32</em> was isolated after X-ray treatment in the cultivar Bonus and released as the cultivar Pallas [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>]; <em xmlns="" class="EmphasisTypeItalic">ari-e.GP</em> was isolated after γ-ray mutagenesis of cultivar Maythorpe in 1956 and released as Golden Promise [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>, <span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>]; <em xmlns="" class="EmphasisTypeItalic">sdw4.ba</em> was isolated after γ-ray treatment of the Chinese cultivar Zhenongguangmangerleng in the late 1960s and is now probably widespread in many semi-dwarf barley accessions in China through the cultivar released as Zhepi 1 [<span class="CitationRef"><a href="#CR37" data-toggle="citation">37</a></span>, <span class="CitationRef"><a href="#CR38" data-toggle="citation">38</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par5" class="Para">Plant hormones like gibberellic acid (GA) and brassinosteroids (BR) are the major regulators of plant height. It is therefore not surprising that successful “Green revolution” semi-dwarfing genes can be exemplified by <em xmlns="" class="EmphasisTypeItalic">Reduced height1</em> (<em xmlns="" class="EmphasisTypeItalic">Rht1</em>) in wheat encoding a transcriptional regulator with a DELLA domain involved in GA signaling [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>], <em xmlns="" class="EmphasisTypeItalic">Semi-dwarf1</em> (<em xmlns="" class="EmphasisTypeItalic">Sd1</em>) in rice [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>, <span class="CitationRef"><a href="#CR28" data-toggle="citation">28</a></span>] and <em xmlns="" class="EmphasisTypeItalic">Sdw1</em> in barley [<span class="CitationRef"><a href="#CR36" data-toggle="citation">36</a></span>] both deficient in a gibberellin 20-oxidase of the GA biosynthetic pathway, and <em xmlns="" class="EmphasisTypeItalic">Uzu1</em> in barley encoding the brassinosteroid receptor [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>, <span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>].</p><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par6" class="Para">An interesting mutant, not only in the context of plant breeding but also for identification of genetic factors determining plant height, is <em xmlns="" class="EmphasisTypeItalic">brachytic1</em> (<em xmlns="" class="EmphasisTypeItalic">brh1</em>), which was originally identified after a spontaneous mutation event in the barley cultivar Himalaya [<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>, <span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>]. Outgrown plants have an upright gross morphology with short leaves, culms, spikes, awns and kernels, and their seedling leaf is reduced in length (Fig. <span class="InternalRef"><a href="#Fig1">1</a></span>). The mutant phenotype is easy to classify at all stages of growth (<span class="ExternalRef"><a href="http://www.nordgen.org/bgs"><span class="RefSource">www.nordgen.org/bgs</span></a></span>). In total, ten recessive <em xmlns="" class="EmphasisTypeItalic">brh1</em> alleles have been isolated in a wide range of different genetic backgrounds. One allele was previously classified as <em xmlns="" class="EmphasisTypeItalic">ari-i</em> but shown to be allelic to <em xmlns="" class="EmphasisTypeItalic">brh1</em> through diallelic crosses [<span class="CitationRef"><a href="#CR33" data-toggle="citation">33</a></span>] (Table <span class="InternalRef"><a href="#Tab1">1</a></span>). <em xmlns="" class="EmphasisTypeItalic">Brh1</em> was mapped to chromosome 7HS and is inherited in a monofactorial recessive manner [<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>, <span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>]. Eight of the ten <em xmlns="" class="EmphasisTypeItalic">brh1</em> alleles are further available as near-isogenic lines through recurrent backcrosses to the two-row spring barley cultivar Bowman (Table <span class="InternalRef"><a href="#Tab1">1</a></span>) [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>]. Trait analysis of <em xmlns="" class="EmphasisTypeItalic">brh1</em> mutants in the genetic background of Bowman showed that several <em xmlns="" class="EmphasisTypeItalic">brh1</em> alleles reduce lodging without causing significant yield reduction [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>]. In addition, the globe-shaped form of <em xmlns="" class="EmphasisTypeItalic">brh1</em> mutant seeds could also be of potential value; such seed form is generally preferred by maltsters as it is associated with an even germination in the malt house. However, the overall effect of <em xmlns="" class="EmphasisTypeItalic">brh1</em> mutations on malting is currently unknown. Still, when elucidating if <em xmlns="" class="EmphasisTypeItalic">brh1</em> alleles could indeed serve as an alternative dwarfing gene in barley, it is of particular importance to exclude a negative impact of the mutation on malting quality.<figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO1">
                    <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0045-1/MediaObjects/41065_2017_45_Fig1_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0045-1/MediaObjects/41065_2017_45_Fig1_HTML.gif" alt="Fig. 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0045-1/MediaObjects/41065_2017_45_Fig1_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0045-1/MediaObjects/41065_2017_45_Fig1_HTML.gif" alt="Fig. 1"></img></noscript></a>
                  </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 1</span>
                      <p xmlns="" class="SimplePara">Phenotype of <em class="EmphasisTypeItalic">brh1</em> mutants in barley compared to the respective mother cultivar. (<strong class="EmphasisTypeBold">a</strong>) Awn length of cultivar Steptoe, (<strong class="EmphasisTypeBold">b</strong>) <em class="EmphasisTypeItalic">brh1.ae</em>, (<strong class="EmphasisTypeBold">c</strong>) cultivar Bowman, and (<strong class="EmphasisTypeBold">d</strong>) near-isogenic line BW074 (<em class="EmphasisTypeItalic">brh1.a</em>). <strong class="EmphasisTypeBold">e</strong>. The erectness of seedlings of cultivar Bowman (left) and BW074 (<em class="EmphasisTypeItalic">brh1.a</em>). <strong class="EmphasisTypeBold">f</strong>. Overall size of Bowman and BW074 (<em class="EmphasisTypeItalic">brh1.a</em>) after heading. <strong class="EmphasisTypeBold">g</strong>. Global-shaped kernels of <em class="EmphasisTypeItalic">brh1.a</em> (bottom) compared to cultivar Himalaya (top)</p>
                    </div></figcaption></figure>
                        <figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 1</span>
                      <p xmlns="" class="SimplePara">Known brh1 mutant alleles, their genetic background, their availability as Bowman near-isogenic lines, and introgression regions on chromosome 7H</p>
                    </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col></col><col></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Allele</p></th><th><p xmlns="" class="SimplePara">Genetic background</p></th><th><p xmlns="" class="SimplePara">Bowman near-isogenic line</p></th><th><p xmlns="" class="SimplePara">Introgression region on 7HS (cM)</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">ari-i.38</em>
                            <sup xmlns="http://www.w3.org/1999/xhtml">b</sup>
                          </p></td><td><p xmlns="" class="SimplePara">Bonus</p></td><td><p xmlns="" class="SimplePara">BW047<sup xmlns="http://www.w3.org/1999/xhtml">b</sup>
                                       </p></td><td><p xmlns="" class="SimplePara">0 – 13.19</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">brh1.a</em>
                            <sup xmlns="http://www.w3.org/1999/xhtml">a</sup>
                          </p></td><td><p xmlns="" class="SimplePara">Himalaya</p></td><td><p xmlns="" class="SimplePara">BW074<sup xmlns="http://www.w3.org/1999/xhtml">b</sup>
                                       </p></td><td><p xmlns="" class="SimplePara">0 – 13.19</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">brh1.aa</em>
                            <sup xmlns="http://www.w3.org/1999/xhtml">
                              <em xmlns="" class="EmphasisTypeItalic">d</em>
                            </sup>
                          </p></td><td><p xmlns="" class="SimplePara">Aapo</p></td><td><p xmlns="" class="SimplePara">BW075<sup xmlns="http://www.w3.org/1999/xhtml">b</sup>
                                       </p></td><td><p xmlns="" class="SimplePara">0 – 48.45</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">brh1.ae</em>
                            <sup xmlns="http://www.w3.org/1999/xhtml">
                              <em xmlns="" class="EmphasisTypeItalic">c</em>
                            </sup>
                          </p></td><td><p xmlns="" class="SimplePara">Steptoe</p></td><td><p xmlns="" class="SimplePara">BW076<sup xmlns="http://www.w3.org/1999/xhtml">b</sup>
                                       </p></td><td><p xmlns="" class="SimplePara">0 – 20.56</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">brh1.c</em>
                            <sup xmlns="http://www.w3.org/1999/xhtml">a</sup>
                          </p></td><td><p xmlns="" class="SimplePara">Moravian</p></td><td><p xmlns="" class="SimplePara">no line</p></td><td><p xmlns="" class="SimplePara">-</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">brh1.e</em>
                            <sup xmlns="http://www.w3.org/1999/xhtml">a</sup>
                          </p></td><td><p xmlns="" class="SimplePara">Aramir</p></td><td><p xmlns="" class="SimplePara">BW077<sup xmlns="http://www.w3.org/1999/xhtml">b</sup>
                                       </p></td><td><p xmlns="" class="SimplePara">0 – 13.19</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">brh1.f</em>
                            <sup xmlns="http://www.w3.org/1999/xhtml">a</sup>
                          </p></td><td><p xmlns="" class="SimplePara">Domen</p></td><td><p xmlns="" class="SimplePara">no line</p></td><td><p xmlns="" class="SimplePara">-</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">brh1.t</em>
                            <sup xmlns="http://www.w3.org/1999/xhtml">a</sup>
                          </p></td><td><p xmlns="" class="SimplePara">Akashinriki</p></td><td><p xmlns="" class="SimplePara">BW078<sup xmlns="http://www.w3.org/1999/xhtml">b</sup>
                                       </p></td><td><p xmlns="" class="SimplePara">8.77 – 26</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">brh1.x</em>
                            <sup xmlns="http://www.w3.org/1999/xhtml">a</sup>
                          </p></td><td><p xmlns="" class="SimplePara">Volla</p></td><td><p xmlns="" class="SimplePara">BW079<sup xmlns="http://www.w3.org/1999/xhtml">b</sup>
                                       </p></td><td><p xmlns="" class="SimplePara">0 – 8.77</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">brh1.z</em>
                            <sup xmlns="http://www.w3.org/1999/xhtml">d</sup>
                          </p></td><td><p xmlns="" class="SimplePara">Aapo</p></td><td><p xmlns="" class="SimplePara">BW080<sup xmlns="http://www.w3.org/1999/xhtml">b</sup>
                                       </p></td><td><p xmlns="" class="SimplePara">0 – 32.35</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                    <p xmlns="" class="SimplePara">The near-isogenic lines have been prepared previously as well as the determination of the introgressions [<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>]</p>
                    <p xmlns="" class="SimplePara">Source of seed material: <sup xmlns="http://www.w3.org/1999/xhtml">a</sup>NSGC, National Small Grains Collection, U.S. Department of Agriculture - Agricultural Research Service, Aberdeen, Idaho, USA; <sup xmlns="http://www.w3.org/1999/xhtml">b</sup>Nordic Genetic Resource Center, Alnarp, Sweden; <sup xmlns="http://www.w3.org/1999/xhtml">c</sup>Andris Kleinhofs, Washington State University, Pullman, WA, USA; <sup xmlns="http://www.w3.org/1999/xhtml">d</sup>not available</p>
                  </div></figure>
                     </div><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par7" class="Para">The aim of the present study was to identify the <em xmlns="" class="EmphasisTypeItalic">Brh1</em> semi-dwarfing gene and to test its effect on yield and malting quality. We show that <em xmlns="" class="EmphasisTypeItalic">Brh1</em> encodes a subunit of a heterotrimeric G protein and mutant alleles of <em xmlns="" class="EmphasisTypeItalic">Brh1</em> do not have any major negative effects on malting quality. Heterotrimeric G proteins transfer signals from receptors (G-protein-coupled receptors) to downstream enzymes (effectors) in signaling cascades [<span class="CitationRef"><a href="#CR32" data-toggle="citation">32</a></span>]. The heterotrimeric G protein complex consists of one Gα, one Gβ and one Gγ subunit. <em xmlns="" class="EmphasisTypeItalic">Brh1</em> encodes a Gα subunit. This is especially interesting in the view that it was recently shown that barley <em xmlns="" class="EmphasisTypeItalic">Ari-e</em> encodes a Gγ subunit [<span class="CitationRef"><a href="#CR35" data-toggle="citation">35</a></span>], being orthologous to <em xmlns="" class="EmphasisTypeItalic">DEP1</em> in rice [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>]. In addition, barley <em xmlns="" class="EmphasisTypeItalic">Brh2</em> encodes a U-box E3 ubiquitin ligase, orthologous to <em xmlns="" class="EmphasisTypeItalic">TUD1</em> in rice (Braumann et al. unpublished). TUD1 has been suggested to physically interact with the Gα subunit in rice [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>]. Thus, within a short period of time, heterotrimeric G proteins have been identified as a key regulator of plant architecture in barley.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Methods" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Methods">Methods</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Methods" data-test="collapse-section"><section id="Sec3" class="Section2 RenderAsSection2">
                <h3 class="Heading">Plant material</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par8" class="Para">All mutant lines were obtained from public non-profit organizations or individual contributors: <em xmlns="" class="EmphasisTypeItalic">brh1.a</em>, <em xmlns="" class="EmphasisTypeItalic">brh1.e</em>, <em xmlns="" class="EmphasisTypeItalic">brh1.t</em>, <em xmlns="" class="EmphasisTypeItalic">brh1.x</em>, <em xmlns="" class="EmphasisTypeItalic">brh1.c</em> and <em xmlns="" class="EmphasisTypeItalic">brh1.f</em> from the National Small Grains Collection (U.S. Department of Agriculture - Agricultural Research Service, Aberdeen, Idaho, USA); <em xmlns="" class="EmphasisTypeItalic">brh1.ae</em> from Andris Kleinhofs, (Washington State University, Pullman, WA, USA); <em xmlns="" class="EmphasisTypeItalic">ari-i</em>.38 and the eight Bowman near-isogenic lines BW047 (<em xmlns="" class="EmphasisTypeItalic">ari-i.38</em>), BW074 (<em xmlns="" class="EmphasisTypeItalic">brh1a</em>), BW075 (<em xmlns="" class="EmphasisTypeItalic">brh1.aa</em>), BW076 (<em xmlns="" class="EmphasisTypeItalic">brh1.ae</em>), BW077 (<em xmlns="" class="EmphasisTypeItalic">brh1.e</em>), BW078 (<em xmlns="" class="EmphasisTypeItalic">brh1.t</em>), BW079 (<em xmlns="" class="EmphasisTypeItalic">brh1.x</em>) and BW080 (<em xmlns="" class="EmphasisTypeItalic">brh1.z</em>) from the Nordic Genetic Resource Center (NordGen, Alnarp, Sweden).</p>
              </section><section id="Sec4" class="Section2 RenderAsSection2">
                <h3 class="Heading">Phenotypic description of mutant and wild type lines</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par9" class="Para">For phenotypic descriptions, plants were grown in the greenhouse under 16-h-light / 8-h-dark cycles. The culm length measured is the distance between the soil and the collar of the spike. Further, the distances from the collar of the spike to the tip of the spike and to the tip of the longest awn was measured. Awn length was normalized to the length of the spike by forming the ratio of [length from spike collar to tip of longest awn] / [length from spike collar to tip of spike]. Per line 12 plants were grown in large square formed pots. The significance of differences in culm length and awn length between mutant lines and respective wild types was tested by a two sided t-test for two independent populations with equal variance assumption in Microsoft Excel. The number of biological replicates was <em xmlns="" class="EmphasisTypeItalic">n</em>&nbsp;=&nbsp;3 (in the case of BW047) to <em xmlns="" class="EmphasisTypeItalic">n</em>&nbsp;=&nbsp;12 depending on the number of seeds that germinated per line.</p>
              </section><section id="Sec5" class="Section2 RenderAsSection2">
                <h3 class="Heading">Generation of double-haploid plants</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par10" class="Para">In order to transfer <em xmlns="" class="EmphasisTypeItalic">brh1.a</em> into a more modern genetic background we performed crosses between the Bowman near-isogenic line BW074 (<em xmlns="" class="EmphasisTypeItalic">brh1.a</em>) and the European elite malting cultivar Quench. F<sub>1</sub> seeds were subjected to embryo rescue 25&nbsp;days after fertilization. The outer parts of the husk of the developing grains were removed and the grains were sterilized in 4% sodium hypochlorite for 10&nbsp;min followed by rinsing in sterile water. The embryos were put onto BAP medium [0.44% (<em xmlns="" class="EmphasisTypeItalic">w</em>/<em xmlns="" class="EmphasisTypeItalic">v</em>) MS powder (Sigma M5524), 0.075% (<em xmlns="" class="EmphasisTypeItalic">w</em>/<em xmlns="" class="EmphasisTypeItalic">v</em>) glutamine, 6% maltose (<em xmlns="" class="EmphasisTypeItalic">w</em>/<em xmlns="" class="EmphasisTypeItalic">v</em>), 1&nbsp;μg/ml BAP (Sigma B9394) and 0.35% phytagel (<em xmlns="" class="EmphasisTypeItalic">w</em>/<em xmlns="" class="EmphasisTypeItalic">v</em>), pH&nbsp;5.8]. After 6&nbsp;weeks, the developing F<sub>1</sub> plants were transferred to soil. Anthers from undeveloped spikes were transferred into a 0.7&nbsp;M mannitol solution in petri dishes and incubated for 3&nbsp;days at 24&nbsp;°C. Then the anthers were transferred on BAP medium and incubated at 24&nbsp;°C for 8&nbsp;weeks in darkness. Developing calluses were transferred into containers containing MS sucrose medium (0.22% (<em xmlns="" class="EmphasisTypeItalic">w</em>/<em xmlns="" class="EmphasisTypeItalic">v</em>) MS, 1.5% (<em xmlns="" class="EmphasisTypeItalic">w</em>/<em xmlns="" class="EmphasisTypeItalic">v</em>) sucrose) and exposed to light for several days. Regenerating plants that turned green were planted into soil and left for seed formation in the greenhouse.</p>
              </section><section id="Sec6" class="Section2 RenderAsSection2">
                <h3 class="Heading">Field tests</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par11" class="Para">Field testing was done on the Danish island of Fyn during summers 2013 and 2014 using standard nitrogen fertilizer application for malting barley in plots of 7.5&nbsp;m<sup>2</sup> in size. 3600 grains were sown per experimental plot. All lines were tested in triplicates in randomized plots. The experimental plots were harvested using a combiner. The seeds were cleaned prior to weighing and the fraction above 2.2&nbsp;mm was kept. The significance of differences in grain yield per plot was tested by a two sided t-test for two independent populations with equal variance assumption in Microsoft Excel.</p>
              </section><section id="Sec7" class="Section2 RenderAsSection2">
                <h3 class="Heading">Germination test</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par12" class="Para">All barley samples used in the manual malting experiments were evaluated for the parameters Germination index, Germination energy and Water sensitivity. Data was based on a sample size of 100 barley grains for a 4&nbsp;ml or 8&nbsp;ml germination test according to Analytica-EBC Method 3.6.2 (<span class="ExternalRef"><a href="http://analytica-ebc.com/index.php?mod=contents&amp;scat=10"><span class="RefSource">http://analytica-ebc.com/index.php?mod=contents&amp;scat=10</span></a></span>).</p>
              </section><section id="Sec8" class="Section2 RenderAsSection2">
                <h3 class="Heading">Characterization of barley samples</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par13" class="Para">Thousand kernel weights were determined by automatic counting using a Data Count JR (Data Technologies, Israel). Size fractionation within four classes (X&nbsp;&gt;&nbsp;2.8&nbsp;mm), (2.8&nbsp;mm&nbsp;&lt;&nbsp;X&nbsp;&gt;&nbsp;2.5&nbsp;mm), (2.5&nbsp;mm&nbsp;&lt;&nbsp;X&nbsp;&gt;&nbsp;2.2&nbsp;mm) and (X&nbsp;&lt;&nbsp;2.2&nbsp;mm) was done using a Pfeuffer Sortimat K3 (Pfeuffer GmbH, Germany). Size fractionation data was calculated from a 100&nbsp;g sample. Protein, water and starch content of the barley samples were determined using a near infrared transmittance Foss 1241 NIT instrument (Foss, Denmark) and barley calibration Foss BY213271 provided by the manufacturer. 24&nbsp;h prior to micro malting, the water content of the 100&nbsp;g sample was re-determined using the same Foss 1241 NIT instrument but barley calibration Foss BY303300. All samples tested in micro malting experiments in this study were grown in rows in New Zealand from September 2012 to February 2013.</p>
              </section><section id="Sec9" class="Section2 RenderAsSection2">
                <h3 class="Heading">Manual micro malting</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par14" class="Para">All samples were malted using a manual micro malting system developed in house at the Carlsberg Research Laboratory. The micro malting was performed in a Termaks incubator KB8400 (Termaks AS, Norway) at 13.5&nbsp;°C during the steeping and germination phase. The barley was malted in individual sample cups each holding 100&nbsp;g of barley seeds. One batch of seeds was malted per line. The barley samples were steeped as a forced submission of all grains in a closed sample grid chamber for 3&nbsp;h. The steeping procedure was performed in three 24&nbsp;h intervals. Time recording started when grains were immersed in water first time and set to 0&nbsp;h. The second steeping period started after 24&nbsp;h and third after 48&nbsp;h. The actual water uptake of the individual barley samples was estimated after a centrifugal removal of water from the sample. Steeping degree was calculated in relation to dry matter of the sample using a Foss 1241 NIT instrument (Foss A/S, Denmark) for determination of dry matter compared to the total sample weight of the sample after centrifugation. Near infrared transmittance is a non-destructive seed analysis, whereby a given chemical compound is assigned to a specific wavelength between 900 – 1050&nbsp;nm. Based on a calibration of the instrument it is then possible predict the concentration of that given compound. Following the last steep the barley samples were maintained at a steeping degree at approximately 45% during the germination step. After the germination process the barley samples were kiln dried in the Termaks incubator for 21&nbsp;h. The temperature profile of the kiln process was a two-step ramping profile and an isothermal terminal step monitored by a Eurotherm 815 control unit (Eurotherm by Schneider Electric, USA). First ramping step started at set point 30&nbsp;°C and a linear ramping at 2&nbsp;°C / h to the breakpoint at 55&nbsp;°C. Second linear ramping was at 4&nbsp;°C / h and reaching a maximum at 82&nbsp;°C. This temperature was kept constant for 1.5&nbsp;h. The process was based on a recirculating air flow using 80% fresh air. The kiln samples were cured using a manual root removal system from Wissenschaftliche Station für Brauerei, Munich, Germany. Final malt samples were stored at 20&nbsp;°C prior to near infrared transmittance analysis and enzyme activity measurement.</p>
              </section><section id="Sec10" class="Section2 RenderAsSection2">
                <h3 class="Heading">Malt analysis</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par15" class="Para">Analysis of malt samples for water content, protein content, soluble protein and extract were determined with a Foss 1241 NIT (Foss A/S, Denmark) instrument using the Malt calibration Foss MA000010 provided by the manufacturer. Prior to enzyme activity analysis the malt samples were milled using a standard Foss Cyclotech mill (Foss A/S, Denmark) equipped with a tungsten carbide grinding ring (Foss 10,004,463), nickel plated impeller (Foss 1000 2666) and a 1&nbsp;mm outlet screen (Foss 10,001,989). All enzyme activity measurements of barley malt were made within 48&nbsp;h after milling of the dry sample.</p>
              </section><section id="Sec11" class="Section2 RenderAsSection2">
                <h3 class="Heading">α-amylase activity</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par16" class="Para">α-amylase activity of final malt was based on malt flour made according to the sample preparation description in the Malt Analysis section above. α-amylase activity assays were measured using a Ceralpha kit (K-CERA) from Megazyme (Ireland) using standard laboratory equipment. The amylase assays were made according to the manufacturer’s protocol (K-CERA 01/12). Measurements were made in technical duplicates. Calculation of amylase activity was based on the formula in the Megazyme protocol (K-CERA 01/12). The significance of differences in α-amylase activity was tested by a two sided t-test for two independent populations with equal variance assumption in Microsoft Excel comparing the four double haploid lines carrying the <em xmlns="" class="EmphasisTypeItalic">brh1.a</em> allele to the three lines in their progeny, BW074, Bowman and Quench.</p>
              </section><section id="Sec12" class="Section2 RenderAsSection2">
                <h3 class="Heading">β-amylase activity</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par17" class="Para">β-amylase activity of final malt was based on malt flour made according to the sample preparation description in the Malt Analysis section above. β-amylase activity assays were measured using Betamyl kit (K-BETA3) from Megazyme (Ireland) using standard laboratory equipment. The amylase assays were made according to the manufacturer’s protocol (K-BETA3 10/10). Measurements were made in technical duplicates. Calculation of β-amylase activity was based on the formula in the Megazyme protocol (K-BETA3 10/10). The significance of differences in β-amylase activity was tested by a two sided t-test for two independent populations with equal variance assumption in Microsoft Excel comparing the four double haploid lines carrying the <em xmlns="" class="EmphasisTypeItalic">brh1.a</em> allele to the three lines in their progeny, BW074, Bowman and Quench.</p>
              </section><section id="Sec13" class="Section2 RenderAsSection2">
                <h3 class="Heading">Limit dextrinase activity</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par18" class="Para">Limit dextrinase activity of final malt was based on malt flour made according to the sample preparation description in the Malt Analysis section above. Limit dextrinase activity assays were measured using a Limit Dextrizyme kit T-LDZ1000 from Megazyme (Ireland) using standard laboratory equipment. The limit dextrinase assays were made according to manufacturer’s protocol T-LDZ1000 07/9. Measurements were made in technical duplicates. Calculation of limit dextrinase activity was based on the formula in the Megazyme protocol T-LDZ1000 07/9. The significance of differences in Limit dextrinase activity was tested by a two sided t-test for two independent populations with equal variance assumption in Microsoft Excel comparing the four double haploid lines carrying the <em xmlns="" class="EmphasisTypeItalic">brh1.a</em> allele to the three lines in their progeny, BW074, Bowman and Quench.</p>
              </section><section id="Sec14" class="Section2 RenderAsSection2">
                <h3 class="Heading">Public databases</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par19" class="Para">Public sequence databases were accessed via the following web pages: “<span class="ExternalRef"><a href="http://www.gramene.org/"><span class="RefSource">http://www.gramene.org/</span></a></span>” for sorghum, “<span class="ExternalRef"><a href="http://rice.plantbiology.msu.edu/analyses_search_blast.shtml%E2%80%9D"><span class="RefSource">http://rice.plantbiology.msu.edu/analyses_search_blast.shtml</span></a></span>” for rice, and “<span class="ExternalRef"><a href="http://brachypodium.org/%E2%80%9D"><span class="RefSource">http://brachypodium.org/</span></a></span>” for brachypodium. The NCBI probe database was accessed via the link <span class="ExternalRef"><a href="http://www.ncbi.nlm.nih.gov/probe"><span class="RefSource">http://www.ncbi.nlm.nih.gov/probe</span></a></span>. The HarvEST database was accessed using the link <span class="ExternalRef"><a href="http://harvest.ucr.edu/"><span class="RefSource">http://harvest.ucr.edu/</span></a></span>.</p>
              </section><section id="Sec15" class="Section2 RenderAsSection2">
                <h3 class="Heading">Screening of a pooled BAC library of the barley genome</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par20" class="Para">Bacterial artificial chromosome (BAC) clones originating from six independent BAC libraries were previously made from the barley cultivar Morex [<span class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>] and pooled in multidimensional levels [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>]. Primers specific for the <em xmlns="" class="EmphasisTypeItalic">Brh1</em> target region (Table <span class="InternalRef"><a href="#Tab2">2</a></span>) were obtained from Eurofins MWG Operon, Ebersberg (Germany) and were used for a PCR screening of the pooled BAC libraries to identify BAC clones containing the region surrounding the <em xmlns="" class="EmphasisTypeItalic">Brh1</em> gene.<figure xmlns="" id="Tab2" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 2</span>
                        <p xmlns="" class="SimplePara">Oligonucleotides used as PCR primers to amplify and sequence the gene encoding the Gα-subunit of the heterotrimeric G protein and to amplify gene fragments located in the genomic region surrounding this gene in a BAC pool screening</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col></col><col></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Primer name</p></th><th><p xmlns="" class="SimplePara">Sequence (5′&nbsp;→&nbsp;3′)</p></th><th><p xmlns="" class="SimplePara">Template</p></th><th><p xmlns="" class="SimplePara">Length of amplification product [bp]</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">bw-Fr1-F1</p></td><td><p xmlns="" class="SimplePara">TGTCAAGATGATGGCTTCAG</p></td><td><p xmlns="" class="SimplePara">Genomic DNA</p></td><td char="("><p xmlns="" class="SimplePara">703 (with bw-Fr1-R1)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">bw_Fr1-Fseq</p></td><td><p xmlns="" class="SimplePara">AAAAAGACGATGATGAAGC</p></td><td><p xmlns="" class="SimplePara">Genomic DNA</p></td><td>&nbsp;</td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">bw-Fr1-R1</p></td><td><p xmlns="" class="SimplePara">GATCGACGAGAGCATGAGAC</p></td><td><p xmlns="" class="SimplePara">Genomic DNA</p></td><td>&nbsp;</td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">bw-Fr2-F1</p></td><td><p xmlns="" class="SimplePara">GGCGAGGTAGAAAGCAAAAG</p></td><td><p xmlns="" class="SimplePara">Genomic DNA</p></td><td char="("><p xmlns="" class="SimplePara">790 (with bw-Fr2-R1)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">bw-Fr2-R1</p></td><td><p xmlns="" class="SimplePara">TTTTGAGGTATCCATCCATCTC</p></td><td><p xmlns="" class="SimplePara">Genomic DNA</p></td><td>&nbsp;</td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">bw-Fr3-F1</p></td><td><p xmlns="" class="SimplePara">CGCACACAGTCAAAGGAAC</p></td><td><p xmlns="" class="SimplePara">Genomic DNA</p></td><td char="("><p xmlns="" class="SimplePara">896 (with bw-Fr3-R1)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">bw-Fr3-R1</p></td><td><p xmlns="" class="SimplePara">GACAGCATTGCACAAGGAG</p></td><td><p xmlns="" class="SimplePara">Genomic DNA</p></td><td>&nbsp;</td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">bw-Fr4-F1</p></td><td><p xmlns="" class="SimplePara">TGTCCCAGATCCTCAAACTG</p></td><td><p xmlns="" class="SimplePara">Genomic DNA</p></td><td char="("><p xmlns="" class="SimplePara">901 (with bw-Fr4-R1)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">bw-Fr4-R1</p></td><td><p xmlns="" class="SimplePara">TTGTCCCTTGTAAACTGTTGG</p></td><td><p xmlns="" class="SimplePara">Genomic DNA</p></td><td>&nbsp;</td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">bw-Fr5-F1</p></td><td><p xmlns="" class="SimplePara">TCCTCCTGCAAAATCTCTCC</p></td><td><p xmlns="" class="SimplePara">Genomic DNA</p></td><td char="("><p xmlns="" class="SimplePara">903 (with bw-Fr5-R1)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">bw-Fr5-R1</p></td><td><p xmlns="" class="SimplePara">TTAGACTCGGCATTTTGAGG</p></td><td><p xmlns="" class="SimplePara">Genomic DNA</p></td><td>&nbsp;</td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">bw-Fr6-F1</p></td><td><p xmlns="" class="SimplePara">ACCTGAATGGCTGGATCTTC</p></td><td><p xmlns="" class="SimplePara">Genomic DNA</p></td><td char="("><p xmlns="" class="SimplePara">857 (with bw-Fr6-R1)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">bw-Fr6-R1</p></td><td><p xmlns="" class="SimplePara">GCATAGTGGGGATTATTCAGG</p></td><td><p xmlns="" class="SimplePara">Genomic DNA</p></td><td>&nbsp;</td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">bw-Fr7-F1</p></td><td><p xmlns="" class="SimplePara">GGTCCAAACAGGTTCAGTTG</p></td><td><p xmlns="" class="SimplePara">Genomic DNA</p></td><td char="("><p xmlns="" class="SimplePara">824 (with bw-Fr7-R1)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">bw-Fr7-R1</p></td><td><p xmlns="" class="SimplePara">GATTCTGCACGAGACAAAGG</p></td><td><p xmlns="" class="SimplePara">Genomic DNA</p></td><td>&nbsp;</td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">bw-Fr8-F1</p></td><td><p xmlns="" class="SimplePara">GGAATCAGTCTTTCCAGATCC</p></td><td><p xmlns="" class="SimplePara">Genomic DNA</p></td><td char="("><p xmlns="" class="SimplePara">705 (with bw-Fr8-R2)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">bw-Fr8-R2</p></td><td><p xmlns="" class="SimplePara">CCAAAATACCCATACCAACC</p></td><td><p xmlns="" class="SimplePara">Genomic DNA</p></td><td>&nbsp;</td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">bw-Fr9-F1</p></td><td><p xmlns="" class="SimplePara">GAGCGAGCCAGAGATTTTG</p></td><td><p xmlns="" class="SimplePara">Genomic DNA</p></td><td char="("><p xmlns="" class="SimplePara">711 (with bw-Fr9-R1)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">bw-Fr9-R1</p></td><td><p xmlns="" class="SimplePara">TGTATATGACGGAGCAGCAAG</p></td><td><p xmlns="" class="SimplePara">Genomic DNA</p></td><td>&nbsp;</td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">bw_Fr9-Rseq</p></td><td><p xmlns="" class="SimplePara">GCTTGTTGGTTGTAGCTCAG</p></td><td><p xmlns="" class="SimplePara">Genomic DNA</p></td><td>&nbsp;</td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">IL130</p></td><td><p xmlns="" class="SimplePara">CAACAATATGGGCATTACAT</p></td><td><p xmlns="" class="SimplePara">BAC</p></td><td char="("><p xmlns="" class="SimplePara">350 (with IL131)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">IL131</p></td><td><p xmlns="" class="SimplePara">CGAGGAGTTCTACAAATCAT</p></td><td><p xmlns="" class="SimplePara">BAC</p></td><td>&nbsp;</td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">IL132</p></td><td><p xmlns="" class="SimplePara">CTTGCCCACTTACCAGGTGG</p></td><td><p xmlns="" class="SimplePara">BAC</p></td><td char="("><p xmlns="" class="SimplePara">300 (with IL133)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">IL133</p></td><td><p xmlns="" class="SimplePara">ACGGGATCTCCTTATGGAGC</p></td><td><p xmlns="" class="SimplePara">BAC</p></td><td>&nbsp;</td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">IL134</p></td><td><p xmlns="" class="SimplePara">ATGATCGACTGCTACATCC</p></td><td><p xmlns="" class="SimplePara">BAC</p></td><td char="("><p xmlns="" class="SimplePara">318 (with IL135)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">IL135</p></td><td><p xmlns="" class="SimplePara">GGTTCTGTCGTTGTACCTAG</p></td><td><p xmlns="" class="SimplePara">BAC</p></td><td>&nbsp;</td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">IL136</p></td><td><p xmlns="" class="SimplePara">CACCAAGCAGCAGATGATC</p></td><td><p xmlns="" class="SimplePara">BAC</p></td><td char="("><p xmlns="" class="SimplePara">152 (with IL137)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">IL137</p></td><td><p xmlns="" class="SimplePara">CCAGTTTGTTGGATGTTTCC</p></td><td><p xmlns="" class="SimplePara">BAC</p></td><td>&nbsp;</td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">IL138</p></td><td><p xmlns="" class="SimplePara">GAGGGCTTCTATTGAAAGTG</p></td><td><p xmlns="" class="SimplePara">BAC</p></td><td char="("><p xmlns="" class="SimplePara">510 (with IL139)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">IL139</p></td><td><p xmlns="" class="SimplePara">GGTACTGACCAACTGTTACG</p></td><td><p xmlns="" class="SimplePara">BAC</p></td><td>&nbsp;</td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">IL140</p></td><td><p xmlns="" class="SimplePara">AAAAGGCTGACTCAAAGACC</p></td><td><p xmlns="" class="SimplePara">BAC</p></td><td char="("><p xmlns="" class="SimplePara">318 (with IL141)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">IL141</p></td><td><p xmlns="" class="SimplePara">CGACAAGCTCCATAAGGTAC</p></td><td><p xmlns="" class="SimplePara">BAC</p></td><td>&nbsp;</td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">IL142</p></td><td><p xmlns="" class="SimplePara">TCTGACCCAGAAACTTGATC</p></td><td><p xmlns="" class="SimplePara">BAC</p></td><td char="("><p xmlns="" class="SimplePara">651 (with IL143)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">IL143</p></td><td><p xmlns="" class="SimplePara">GAAAGGGAGAGAGAAATCG</p></td><td><p xmlns="" class="SimplePara">BAC</p></td><td>&nbsp;</td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">IL144</p></td><td><p xmlns="" class="SimplePara">GGTCTGACCTCTTATTTCCC</p></td><td><p xmlns="" class="SimplePara">BAC</p></td><td char="("><p xmlns="" class="SimplePara">512 (with IL145)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">IL145</p></td><td><p xmlns="" class="SimplePara">AAAGCGAGGCATTCACTAG</p></td><td><p xmlns="" class="SimplePara">BAC</p></td><td>&nbsp;</td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">IL146</p></td><td><p xmlns="" class="SimplePara">GCTCAAAGTGGGAATACGAG</p></td><td><p xmlns="" class="SimplePara">BAC</p></td><td char="("><p xmlns="" class="SimplePara">475 (with IL147)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">IL147</p></td><td><p xmlns="" class="SimplePara">CTTGTGCTGCCAACTAGATG</p></td><td><p xmlns="" class="SimplePara">BAC</p></td><td>&nbsp;</td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">IL148</p></td><td><p xmlns="" class="SimplePara">AAGAGCATCCAGTGACAAG</p></td><td><p xmlns="" class="SimplePara">BAC</p></td><td char="("><p xmlns="" class="SimplePara">633 (with IL149)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">IL149</p></td><td><p xmlns="" class="SimplePara">GGATCTCCGCTTAGGTATAG</p></td><td><p xmlns="" class="SimplePara">BAC</p></td><td>&nbsp;</td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">IL150</p></td><td><p xmlns="" class="SimplePara">CGCTTTCCCTATTTTGTCAG</p></td><td><p xmlns="" class="SimplePara">BAC</p></td><td char="("><p xmlns="" class="SimplePara">576 (with IL151)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">IL151</p></td><td><p xmlns="" class="SimplePara">ACTTCACGGAGATTGATGC</p></td><td><p xmlns="" class="SimplePara">BAC</p></td><td>&nbsp;</td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">IL152</p></td><td><p xmlns="" class="SimplePara">AGCCAGCAATCCTTGTTAGC</p></td><td><p xmlns="" class="SimplePara">BAC</p></td><td char="("><p xmlns="" class="SimplePara">510 (with IL153)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">IL153</p></td><td><p xmlns="" class="SimplePara">ACCTTGAGCCAGGGAAAAG</p></td><td><p xmlns="" class="SimplePara">BAC</p></td><td>&nbsp;</td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">IL201</p></td><td><p xmlns="" class="SimplePara">CTCTGGCTCGCTCATCAACC</p></td><td><p xmlns="" class="SimplePara">RNA</p></td><td char="("><p xmlns="" class="SimplePara">1229 (with IL202)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">IL202</p></td><td><p xmlns="" class="SimplePara">AGCGTCTCATGCTCTCGTCG</p></td><td><p xmlns="" class="SimplePara">RNA</p></td><td>&nbsp;</td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">IL203</p></td><td><p xmlns="" class="SimplePara">GTAAACGAGGCAGAGGCAGC</p></td><td><p xmlns="" class="SimplePara">RNA</p></td><td char="("><p xmlns="" class="SimplePara">461 (with IL204)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">IL204</p></td><td><p xmlns="" class="SimplePara">CAGTCGGGAACTTGCAGCAC</p></td><td><p xmlns="" class="SimplePara">RNA</p></td><td>&nbsp;</td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">IL205</p></td><td><p xmlns="" class="SimplePara">CGGAACTCAAGGGCTACACG</p></td><td><p xmlns="" class="SimplePara">RNA</p></td><td char="("><p xmlns="" class="SimplePara">600 (with IL206)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">IL206</p></td><td><p xmlns="" class="SimplePara">CGAACAGCTCCTTGGTCTCC</p></td><td><p xmlns="" class="SimplePara">RNA</p></td><td>&nbsp;</td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">IL207</p></td><td><p xmlns="" class="SimplePara">AAATCTCTGGCTCGCTCATC</p></td><td><p xmlns="" class="SimplePara">RNA</p></td><td char="("><p xmlns="" class="SimplePara">1389 (with IL208)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">IL208</p></td><td><p xmlns="" class="SimplePara">AGCACAAAGGAGTCACAAGG</p></td><td><p xmlns="" class="SimplePara">RNA</p></td><td>&nbsp;</td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">IL209</p></td><td><p xmlns="" class="SimplePara">ACTAGGAGGAAGCATGAGAC</p></td><td><p xmlns="" class="SimplePara">RNA</p></td><td char="("><p xmlns="" class="SimplePara">1305 (with IL210)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">IL210</p></td><td><p xmlns="" class="SimplePara">CAATACAGAGTTGCGACACC</p></td><td><p xmlns="" class="SimplePara">RNA</p></td><td>&nbsp;</td></tr></tbody></table></div></figure>
                        </div>
              </section><section id="Sec16" class="Section2 RenderAsSection2">
                <h3 class="Heading">PCR amplification and sequencing of the Gα-subunit of the heterotrimeric G protein in barley</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par21" class="Para">Oligonucleotides used for PCR amplification and DNA sequencing of the Gα-subunit of the heterotrimeric G protein are listed in Table <span class="InternalRef"><a href="#Tab2">2</a></span>. PCR amplification from genomic DNA and sequencing of PCR products was done at LGC Genomics GmbH (Berlin, Germany) in the case of all <em xmlns="" class="EmphasisTypeItalic">brh1</em> original mutant lines, near-isogenic lines and the respective mother cultivars. In the case of <em xmlns="" class="EmphasisTypeItalic">ari-m</em> mutant lines and their respective mother cultivars, isolation of genomic DNA and PCR amplification was done using the REDExtract-N-Amp™ Plant PCR Kit (Sigma-Aldrich, St. Louis, USA), and sequencing of PCR products was done at StarSEQ (Mainz, Germany).</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par22" class="Para">RNA was isolated from seedlings leaves with the help of Trizol reagent (Thermo Fisher Scientific, Waltham, USA). For one-step RT-PCR of RNA the SuperscriptIII one-step RT-PCR kit (Thermo Fisher Scientific, Waltham, USA) with HIFI taq enzyme was used. Sequencing of PCR products was done at StarSEQ (Mainz, Germany).</p>
              </section><section id="Sec17" class="Section2 RenderAsSection2">
                <h3 class="Heading">Allelism crosses to confirm identity of the <em class="EmphasisTypeItalic">Ari-m</em> and the <em class="EmphasisTypeItalic">Brh1</em> locus in barley</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par23" class="Para">Allelism tests were performed through crosses between BW074 (<em xmlns="" class="EmphasisTypeItalic">brh1.a</em>) and BW051 (<em xmlns="" class="EmphasisTypeItalic">ari-m.28</em>). Plants were grown in a greenhouse at 18°C under 16-h-light / 8-h-dark cycles. Light intensity were set to a photon flux of 300 μmol m<sup>−2</sup>&nbsp;s<sup>−1</sup>. Plants were pollinated 3&nbsp;days after emasculation. F<sub>1</sub> progenies of this cross were genotyped to confirm successful crossings, and grown to maturity for phenotypic analysis to confirm allelism through visual inspection of plants.</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec18" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Results" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Results">Results</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Results" data-test="collapse-section"><section id="Sec19" class="Section2 RenderAsSection2">
                <h3 class="Heading">Identification of the <em class="EmphasisTypeItalic">Brh1</em> locus</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par24" class="Para">The <em xmlns="" class="EmphasisTypeItalic">Brh1</em> gene is located on barley chromosome 7HS, flanked by markers CDO545 (distal) and R3139 (centromeric) [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>]. The two markers are both located 0.8&nbsp;cM away from the <em xmlns="" class="EmphasisTypeItalic">Brh1</em> locus. A third marker, MWG2074, was shown to co-segregate with the <em xmlns="" class="EmphasisTypeItalic">brh1</em> phenotype in the considered mapping population [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>]. Due to the close proximity of the three markers to each other and <em xmlns="" class="EmphasisTypeItalic">Brh1</em>, we decided to sequence their neighboring genes in all <em xmlns="" class="EmphasisTypeItalic">brh1</em> mutants in order to identify the <em xmlns="" class="EmphasisTypeItalic">Brh1</em> gene. The work was initiated before barley genomic sequence information was publically available [<span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>]. Therefore, the marker sequences were downloaded from the NCBI probe database and used for blastn analysis of genomic databases. As a first step, sequences and genes homologous to the three barley markers of interest were identified in the genomes of rice (<em xmlns="" class="EmphasisTypeItalic">Oryza sativa</em>), sorghum (<em xmlns="" class="EmphasisTypeItalic">Sorghum bicolor</em>) and brachypodium (<em xmlns="" class="EmphasisTypeItalic">Brachypodium distachyon</em>) (Table <span class="InternalRef"><a href="#Tab3">3</a></span>). We postulated that a syntenic approach could help us to visualize barley genes in the region. However, the comparison showed that the gene content of the <em xmlns="" class="EmphasisTypeItalic">Brh1</em> target region is not fully conserved among the different grass genomes as not all markers flanking the <em xmlns="" class="EmphasisTypeItalic">Brh1</em> locus in barley could be identified in close proximity to each other in any of the rice, sorghum and brachypodium genomes (Fig. <span class="InternalRef"><a href="#Fig2">2</a></span>). The two markers CDO545 and MWG2074, which are flanking the <em xmlns="" class="EmphasisTypeItalic">Brh1</em> locus in barley, were found to be located close to each other on chromosome 1 in brachypodium. Further, six genes were present in the region of interest in all three organisms investigated (green and gray arrows in Fig. <span class="InternalRef"><a href="#Fig2">2b</a></span>). In case of five of these genes, a unigene could be identified in the barley EST database HarvEST (<span class="ExternalRef"><a href="http://harvest.ucr.edu"><span class="RefSource">http://harvest.ucr.edu</span></a></span>/). Primers were designed for these unigenes and a PCR-based screening of pooled BAC libraries was performed. We identified ten BAC clones, located in two finger-print contigs on chromosome 7H. The two finger-print contigs, 44,369 and 44,201, were both anchored to the genomic position at 12.74&nbsp;cM on chromosome 7H (17). The BAC clones identified in the PCR-screen were overlapping with three sequenced BAC clones (17), providing us with a partial sequence of the <em xmlns="" class="EmphasisTypeItalic">Brh1</em> target region in barley. The six genes predicted to be located in this region could all be identified on the sequenced BACs. The three sequenced BAC clones were searched for appropriate candidate genes for the <em xmlns="" class="EmphasisTypeItalic">Brh1</em> mutation. On BAC clone HVVMRX83KHA0060C10 we could identify the Morex locus MLOC_67224.3, a sequence corresponding to the NCBI accession AF267485. This gene encodes the Gα subunit of a heterotrimeric G protein in barley. Mutations in the Gα subunit are known to be the cause of the semi-dwarf phenotype in the <em xmlns="" class="EmphasisTypeItalic">dwarf1</em> (<em xmlns="" class="EmphasisTypeItalic">d1</em>) rice mutants [<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>], making the Gα subunit a promising candidate gene for the <em xmlns="" class="EmphasisTypeItalic">brh1</em> mutation in barley. Still, the location of the barley gene was a surprising finding. The <em xmlns="" class="EmphasisTypeItalic">D1</em> gene in rice (locus Os05g26890) is located on chromosome 5 and not in the region on chromosome 6, which was shown to be syntenic to the <em xmlns="" class="EmphasisTypeItalic">Brh1</em> region in barley [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>]. Further, blastp analyses identified Sb01g045320 in sorghum and Bradi2G60350 in brachypodium as the orthologous genes of barley <em xmlns="" class="EmphasisTypeItalic">Brh1</em>. Neither of the sorghum and brachypodium genes are in regions syntenic to barley.<figure xmlns="" id="Tab3" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 3</span>
                        <p xmlns="" class="SimplePara">Markers in the vicinity of <em class="EmphasisTypeItalic">Brh1</em> [<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>] and the gene models associated with the best blastn hit in the genomes of rice, sorghum and brachypodium</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col></col><col></col><col></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Marker</p></th><th><p xmlns="" class="SimplePara">NCBI accession no.</p></th><th><p xmlns="" class="SimplePara">Rice blast hit</p></th><th><p xmlns="" class="SimplePara">Sorghum blast hit</p></th><th><p xmlns="" class="SimplePara">Brachypodium blast hit</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">CDO545</p></td><td><p xmlns="" class="SimplePara">AA231869.1</p><p xmlns="" class="SimplePara">AA231713.1</p></td><td><p xmlns="" class="SimplePara">Os06g02550.2</p></td><td><p xmlns="" class="SimplePara">Sb10g001310</p></td><td><p xmlns="" class="SimplePara">Bradi1g50590.1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">MWG2074</p></td><td><p xmlns="" class="SimplePara">AJ234755.1</p><p xmlns="" class="SimplePara">AJ234754.1</p></td><td><p xmlns="" class="SimplePara">no hit on chromosome 6</p></td><td><p xmlns="" class="SimplePara">no hit on chromosome 10</p></td><td><p xmlns="" class="SimplePara">Bradi1g50650.1</p><p xmlns="" class="SimplePara">Bradi1g50660.1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R3139</p></td><td><p xmlns="" class="SimplePara">AU082160</p><p xmlns="" class="SimplePara">D25093</p></td><td><p xmlns="" class="SimplePara">Os06g03560</p></td><td><p xmlns="" class="SimplePara">Sb10g001530</p><p xmlns="" class="SimplePara">(with D25093)</p></td><td><p xmlns="" class="SimplePara">no conclusive hit</p></td></tr></tbody></table></div></figure>
                           <figure xmlns="" class="Figure" id="Fig2"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO2">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0045-1/MediaObjects/41065_2017_45_Fig2_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0045-1/MediaObjects/41065_2017_45_Fig2_HTML.gif" alt="Fig. 2" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0045-1/MediaObjects/41065_2017_45_Fig2_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0045-1/MediaObjects/41065_2017_45_Fig2_HTML.gif" alt="Fig. 2"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 2</span>
                        <p xmlns="" class="SimplePara">The <em class="EmphasisTypeItalic">Brh1</em> genomic region in barley and the syntenic regions in rice, sorghum and brachypodium. <strong class="EmphasisTypeBold">a</strong>. <em class="EmphasisTypeItalic">Brh1</em> has been mapped between markers CDO545 (green, NCBI accession numbers AA231869.1 and AA231713.1) and BDC129 (yellow), R3139 (blue, AU082160, D25093) and MWG2074 (red, AJ234754.1, AJ234755.1) [<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>]. MWG2074 was shown to have two loci, the B locus (middle, red) was co-segregating with <em class="EmphasisTypeItalic">Brh1</em>. Marker sequences were used to identify matching loci in rice, sorghum and brachypodium (color codes kept). No obvious hits to orthologous sequences could be found for marker MWG2074 in rice and sorghum, or for R3139 in brachypodium. <strong class="EmphasisTypeBold">b</strong>. Gene content of the <em class="EmphasisTypeItalic">Brh1</em> syntenic region in brachypodium as identified in Fig. <span xmlns="http://www.w3.org/1999/xhtml" class="InternalRef"><a href="#Fig2">2a</a></span>. The loci (Bradi1g50590 and Bradi1g50660) matching markers CDO545 and MWG2074 are highlighted green and red, respectively. The genes flanked by loci Bradi1g50590 and Bradi1g50660 are found in the same order in rice and sorghum. The orthologous barley genes were used to screen pooled BAC libraries</p>
                      </div></figcaption></figure>
                        </div>
              </section><section id="Sec20" class="Section2 RenderAsSection2">
                <h3 class="Heading">
                           <em class="EmphasisTypeItalic">Brh1</em> encodes the Gα subunit in barley</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par25" class="Para">Mutations in the gene encoding the Gα subunit in rice cause the <em xmlns="" class="EmphasisTypeItalic">d1</em> mutant phenotype, a semi-dwarf phenotype similar to the <em xmlns="" class="EmphasisTypeItalic">brh1</em> phenotype in barley. Rice <em xmlns="" class="EmphasisTypeItalic">d1</em> mutants are characterized by reduced plant height, shorter panicles, and shorter, globe-shaped seeds [<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>]. The barley <em xmlns="" class="EmphasisTypeItalic">brh1</em> mutation also causes height reduction, changes in the spike architecture and is characterized by a rounded seed shape (Fig. <span class="InternalRef"><a href="#Fig1">1</a></span>). The matching rice phenotype and the localization of the Gα gene in the <em xmlns="" class="EmphasisTypeItalic">brh1</em> target region made the gene encoding the Gα subunit an obvious candidate gene for the <em xmlns="" class="EmphasisTypeItalic">brh1</em> mutation. Therefore, we sequenced the gene in the <em xmlns="" class="EmphasisTypeItalic">brh1</em> near-isogenic lines and in the cultivar Bowman. In each of the eight near-isogenic lines, but not in Bowman, a mutation was found with strong postulated impact on the resulting protein (Fig. <span class="InternalRef"><a href="#Fig3">3</a></span>, Table <span class="InternalRef"><a href="#Tab4">4</a></span>). In BW076 (<em xmlns="" class="EmphasisTypeItalic">brh1.ae</em>) only two of the nine PCR fragments that were generated for sequencing of the <em xmlns="" class="EmphasisTypeItalic">Brh1</em> gene could be amplified from genomic DNA suggesting a partial deletion of the gene in this mutant line. Small deletions of 1 or 2&nbsp;bp caused frame shifts in BW047 (<em xmlns="" class="EmphasisTypeItalic">ari-i.38</em>), BW075 (<em xmlns="" class="EmphasisTypeItalic">brh1.aa</em>), BW077 (<em xmlns="" class="EmphasisTypeItalic">brh1.e</em>), BW078 (<em xmlns="" class="EmphasisTypeItalic">brh1.t</em>) and BW080 (<em xmlns="" class="EmphasisTypeItalic">brh1.z</em>). The frame shifts introduced premature stop codons in all lines except BW074 (<em xmlns="" class="EmphasisTypeItalic">brh1.a</em>) where a new stop codon was introduced in the 3′ untranslated region after a frame shift, thus extending exon 13. In BW079 (<em xmlns="" class="EmphasisTypeItalic">brh1.x</em>) a point mutation introduced a premature stop deleting the last 20 amino acids of the protein.<figure xmlns="" class="Figure" id="Fig3"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO3">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0045-1/MediaObjects/41065_2017_45_Fig3_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0045-1/MediaObjects/41065_2017_45_Fig3_HTML.gif" alt="Fig. 3" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0045-1/MediaObjects/41065_2017_45_Fig3_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0045-1/MediaObjects/41065_2017_45_Fig3_HTML.gif" alt="Fig. 3"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 3</span>
                        <p xmlns="" class="SimplePara">Distribution of mutations identified in the gene of the Gα subunit of the heterotrimeric G protein in <em class="EmphasisTypeItalic">brh1</em>, <em class="EmphasisTypeItalic">ari-i</em> and <em class="EmphasisTypeItalic">ari-m</em> barley mutants. The gene contains 13 exons and 12 introns over 3780&nbsp;bp. The deduced polypeptide is 383 amino-acid residues. More detailed information about the mutations is found in Table <span xmlns="http://www.w3.org/1999/xhtml" class="InternalRef"><a href="#Tab4">4</a></span>. Mutations <em class="EmphasisTypeItalic">brh1.ae</em> and <em class="EmphasisTypeItalic">ari-m.251</em> are large deletions of &gt;3&nbsp;kb not shown in the figure</p>
                      </div></figcaption></figure>
                           <figure xmlns="" id="Tab4" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 4</span>
                        <p xmlns="" class="SimplePara">Mutations identified in recessive <em class="EmphasisTypeItalic">brh1</em> alleles and their deduced effect at protein level</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col></col><col></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Allele</p></th><th><p xmlns="" class="SimplePara">Position of mutation<sup xmlns="http://www.w3.org/1999/xhtml">a</sup>
                                          </p></th><th><p xmlns="" class="SimplePara">Type of mutation</p></th><th><p xmlns="" class="SimplePara">Effect of mutation on protein</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">BW047 (<em class="EmphasisTypeItalic">ari-i.38</em>)</p></td><td><p xmlns="" class="SimplePara">2345</p></td><td><p xmlns="" class="SimplePara">a to t</p></td><td><p xmlns="" class="SimplePara">Mutation shifts splice site between intron 8 and exon 9 leading to frame shift. 203 native amino-acid residues followed by FSAAPLERAKEAERYTGCMM.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">BW074 (<em class="EmphasisTypeItalic">brh1.a</em>)</p></td><td><p xmlns="" class="SimplePara">3745-3746</p></td><td><p xmlns="" class="SimplePara">2&nbsp;bp deletion</p></td><td><p xmlns="" class="SimplePara">Frame shift eliminates wild type stop codon and extends exon 13. 372 native residues followed by 152 additional aa residues.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">BW075 (<em class="EmphasisTypeItalic">brh1.aa</em>)</p></td><td><p xmlns="" class="SimplePara">2205</p></td><td><p xmlns="" class="SimplePara">1&nbsp;bp deletion</p></td><td><p xmlns="" class="SimplePara">Frame shift in exon 8. 185 native residues followed by MCSMQEYGQMGL.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">BW076 (<em class="EmphasisTypeItalic">brh1.ae</em>)</p></td><td><p xmlns="" class="SimplePara">Deletion (minimal region: −723 to 2819</p></td><td><p xmlns="" class="SimplePara">Large deletion, no transcript detectable</p></td><td><p xmlns="" class="SimplePara">Probably no protein produced</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">brh1.c</em>
                            </p></td><td><p xmlns="" class="SimplePara">3745-3746</p></td><td><p xmlns="" class="SimplePara">2&nbsp;bp deletion</p></td><td><p xmlns="" class="SimplePara">Frame shift eliminates wild type stop codon and extends exon 13. 372 native residues followed by 152 additional  residues.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">BW077 (<em class="EmphasisTypeItalic">brh1.e</em>)</p></td><td><p xmlns="" class="SimplePara">813</p></td><td><p xmlns="" class="SimplePara">g to a</p></td><td><p xmlns="" class="SimplePara">Mutation eliminates splice site between intron 4 and exon 5 leading to frameshift. 88 native residues followed by VCYYWKGCVSYLFPHLLHSGDYSGNLHWIFLWECVYTSYY.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                                            <em class="EmphasisTypeItalic">brh1.</em>f</p></td><td><p xmlns="" class="SimplePara">1512-1515</p></td><td><p xmlns="" class="SimplePara">4&nbsp;bp deletion</p></td><td><p xmlns="" class="SimplePara">Frame shift and premature stop in exon 6. 129 native residues followed by SLIKNSYRM.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">BW078 (<em class="EmphasisTypeItalic">brh1.t</em>)</p></td><td><p xmlns="" class="SimplePara">2346</p></td><td><p xmlns="" class="SimplePara">g to a</p></td><td><p xmlns="" class="SimplePara">Mutation shifts splice site between intron 8 and exon 9. 203 native residues followed by FSATPLERAKEVERYTGCMM.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">BW079 (<em class="EmphasisTypeItalic">brh1.x</em>)</p></td><td><p xmlns="" class="SimplePara">3718</p></td><td><p xmlns="" class="SimplePara">a to t</p></td><td><p xmlns="" class="SimplePara">Premature stop in exon 13. Truncated protein of 363  residues.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">BW080 (<em class="EmphasisTypeItalic">brh1.z</em>)</p></td><td><p xmlns="" class="SimplePara">2205</p></td><td><p xmlns="" class="SimplePara">1&nbsp;bp deletion</p></td><td><p xmlns="" class="SimplePara">Frame shift in exon 8. 185 native residues followed by MCSMQEYGQMGL.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">ari-m.12</em>
                            </p></td><td><p xmlns="" class="SimplePara">3178</p></td><td><p xmlns="" class="SimplePara">g to a</p></td><td><p xmlns="" class="SimplePara">Premature stop in exon 10. Truncated protein of 270 native residues.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">BW051 (<em class="EmphasisTypeItalic">ari-m.28</em>)</p></td><td><p xmlns="" class="SimplePara">3178</p></td><td><p xmlns="" class="SimplePara">g to a</p></td><td><p xmlns="" class="SimplePara">Premature stop in exon 10. Truncated protein of 270 native residues.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">ari-m.141</em>
                            </p></td><td><p xmlns="" class="SimplePara">164</p></td><td><p xmlns="" class="SimplePara">g to a</p></td><td><p xmlns="" class="SimplePara">Mutation eliminates splice site between intron 1 and exon 2. 21 native residues followed by VSSFPYHLLDSFNSSCPVLSCPVLS.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">ari-m.177</em>
                            </p></td><td><p xmlns="" class="SimplePara">2345</p></td><td><p xmlns="" class="SimplePara">a to t</p></td><td><p xmlns="" class="SimplePara">Mutation shifts splice site between intron 8 and exon 9 leading to frame shift. 203 native residues followed by FSAAPLERAKEAERYTGCMM.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">ari-m.251</em>
                            </p></td><td>&nbsp;</td><td><p xmlns="" class="SimplePara">Large deletion</p></td><td><p xmlns="" class="SimplePara">No protein produced.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">ari-m.269</em>
                            </p></td><td><p xmlns="" class="SimplePara">3449</p></td><td><p xmlns="" class="SimplePara">g to a</p></td><td><p xmlns="" class="SimplePara">Premature stop in exon 12. Truncated protein of 306  residues.</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">The deduced polypeptide of <em class="EmphasisTypeItalic">Brh1</em> (AF267485) is 383 amino-acid residues</p>
                      <p xmlns="" class="SimplePara">
                                    <sup xmlns="http://www.w3.org/1999/xhtml">a</sup>First position is the “A” of the ATG start codon in the Bowman genomic sequence</p>
                    </div></figure>
                           <figure xmlns="" class="Figure" id="Fig4"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO4">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0045-1/MediaObjects/41065_2017_45_Fig4_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0045-1/MediaObjects/41065_2017_45_Fig4_HTML.gif" alt="Fig. 4" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0045-1/MediaObjects/41065_2017_45_Fig4_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0045-1/MediaObjects/41065_2017_45_Fig4_HTML.gif" alt="Fig. 4"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 4</span>
                        <p xmlns="" class="SimplePara">Phenotype of F<sub xmlns="http://www.w3.org/1999/xhtml">1</sub> plants generated from a cross between the Bowman near-isogenic lines BW051 (<em class="EmphasisTypeItalic">ari-m.28</em>) and BW074 (<em class="EmphasisTypeItalic">brh1.a</em>). <strong class="EmphasisTypeBold">a</strong>. Overall height of cultivar Bowman (left) compared to an F<sub xmlns="http://www.w3.org/1999/xhtml">1</sub> plant of a cross between BW051 x BW074. <strong class="EmphasisTypeBold">b</strong>. Overall height of BW051 x BW074 F<sub xmlns="http://www.w3.org/1999/xhtml">1</sub> plant (middle), BW051 (left) and BW074 (right), carrying the recessive <em class="EmphasisTypeItalic">ari-m.28</em> and <em class="EmphasisTypeItalic">brh1.a</em> alleles, respectively. <strong class="EmphasisTypeBold">c</strong>. Awn phenotypes of Bowman (top), BW051, F<sub xmlns="http://www.w3.org/1999/xhtml">1</sub> of BW051 x BW074 and BW074</p>
                      </div></figcaption></figure>
                           <figure xmlns="" class="Figure" id="Fig5"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO5">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0045-1/MediaObjects/41065_2017_45_Fig5_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0045-1/MediaObjects/41065_2017_45_Fig5_HTML.gif" alt="Fig. 5" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0045-1/MediaObjects/41065_2017_45_Fig5_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0045-1/MediaObjects/41065_2017_45_Fig5_HTML.gif" alt="Fig. 5"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 5</span>
                        <p xmlns="" class="SimplePara">Culm length (<strong class="EmphasisTypeBold">a</strong>) and awn/spike ratio (<strong class="EmphasisTypeBold">b</strong>) in <em class="EmphasisTypeItalic">brh1</em> mutants compared to their respective mother cultivars. The plants were grown in a green house. A <em class="EmphasisTypeItalic">p</em>-value of &lt;0.05 was considered as significant and indicated with an asterisk in the figure</p>
                      </div></figcaption></figure>
                           <figure xmlns="" class="Figure" id="Fig6"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO6">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0045-1/MediaObjects/41065_2017_45_Fig6_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0045-1/MediaObjects/41065_2017_45_Fig6_HTML.gif" alt="Fig. 6" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0045-1/MediaObjects/41065_2017_45_Fig6_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0045-1/MediaObjects/41065_2017_45_Fig6_HTML.gif" alt="Fig. 6"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 6</span>
                        <p xmlns="" class="SimplePara">Grain yield in field trials. Near-isogenic line BW074 (<em class="EmphasisTypeItalic">brh1.a</em>) and two double-haploid lines obtained from a cross of cultivar Quench x BW074 are compared to Quench and Bowman. The field trials were performed on the Danish island of Fyn in the summer 2013 (<strong class="EmphasisTypeBold">a</strong>) and 2014 (<strong class="EmphasisTypeBold">b</strong>). All represented lines were tested in triplicates. Significance was tested in relation to BW074. DH_A_1, DH_A_2, DH_B_1, and DH_B_2 designate double-haploid lines obtained from a cross of Quench x BW074. Lines A and B depict two independent double-haploid lines regenerated after tissue culture, 1 and 2 relates to separate propagations of the same line. A <em class="EmphasisTypeItalic">p</em>-value of &lt;0.05 was considered as significant</p>
                      </div></figcaption></figure>
                           <figure xmlns="" class="Figure" id="Fig7"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO7">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0045-1/MediaObjects/41065_2017_45_Fig7_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0045-1/MediaObjects/41065_2017_45_Fig7_HTML.gif" alt="Fig. 7" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0045-1/MediaObjects/41065_2017_45_Fig7_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0045-1/MediaObjects/41065_2017_45_Fig7_HTML.gif" alt="Fig. 7"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 7</span>
                        <p xmlns="" class="SimplePara">Measurement of α-amylase, β-amylase and free limit dextrinase activity in malt samples obtained from near-isogenic line BW074 (<em class="EmphasisTypeItalic">brh1.a</em>), two double-haploid lines obtained from a cross of cultivar Quench x BW074, and of cultivars Quench and Bowman. Two measurements per sample were performed. Standard deviations are indicated in the figure. The averages of the activities of the double-haploid lines were compared to the average of the activities of BW074, Bowman and Quench in a t-test, which revealed significant differences in enzyme activities for β-amylase and free limit dextrinase (<em class="EmphasisTypeItalic">p</em>&nbsp;&lt;&nbsp;0.05)</p>
                      </div></figcaption></figure>
                        </div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par26" class="Para">We also sequenced the gene encoding the Gα subunit from the original mutants and two additional lines, <em xmlns="" class="EmphasisTypeItalic">brh1.c</em> and <em xmlns="" class="EmphasisTypeItalic">brh1.f</em>, for which no near-isogenic lines have been generated (Fig. <span class="InternalRef"><a href="#Fig3">3</a></span>, Table <span class="InternalRef"><a href="#Tab4">4</a></span>). All mutations were verified and they were not present in the respective mother cultivar (Table <span class="InternalRef"><a href="#Tab1">1</a></span>). Mutation <em xmlns="" class="EmphasisTypeItalic">brh1.c</em> was based on a 2&nbsp;bp deletion and identical to <em xmlns="" class="EmphasisTypeItalic">brh1.a</em>. Mutation <em xmlns="" class="EmphasisTypeItalic">brh1.f</em> was based on a 4&nbsp;bp deletion in exon 6 leading to a premature stop. We noticed several SNPs between Bowman and the mother cultivars (Bonus, Himalaya, Steptoe, Moravian, Aramir, Domen, Akashinriki and Volla) with the exception of Aapo, which was identical to Bowman over the sequenced region (Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span> Table S1). We noted that the SNP pattern in mutant <em xmlns="" class="EmphasisTypeItalic">brh1.c</em> was not the same as its mother cultivar Moravian. Instead they were identical to those of <em xmlns="" class="EmphasisTypeItalic">brh1.a</em> and Himalaya. Still mature plants of the mutant lines <em xmlns="" class="EmphasisTypeItalic">brh1.c</em> and <em xmlns="" class="EmphasisTypeItalic">brh1.a</em> show different morphological characters – <em xmlns="" class="EmphasisTypeItalic">brh1.c</em> is two-rowed, while <em xmlns="" class="EmphasisTypeItalic">brh1.a</em> is six-rowed. We therefore suggest that the <em xmlns="" class="EmphasisTypeItalic">brh1</em> mutation was introduced into Moravian by crossing to a <em xmlns="" class="EmphasisTypeItalic">brh1.a</em> mutant line instead of an independent mutational event.</p>
              </section><section id="Sec21" class="Section2 RenderAsSection2">
                <h3 class="Heading">Identification of additional <em class="EmphasisTypeItalic">brh1</em> alleles</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par27" class="Para">When propagating various near-isogenic lines, we noticed that BW051 (<em xmlns="" class="EmphasisTypeItalic">ari-m.28</em>) has a <em xmlns="" class="EmphasisTypeItalic">brh1</em>-like phenotype with overall reduction in height and characteristic short awns. In addition, the introgression residing from the original mutant donor <em xmlns="" class="EmphasisTypeItalic">ari-m.28</em> in BW051 is reduced to a minor genetic element in the telomeric region of chromosome 7HS overlapping with the introgression regions of all <em xmlns="" class="EmphasisTypeItalic">brh1</em> Bowman near-isogenic lines (Table <span class="InternalRef"><a href="#Tab1">1</a></span>, Table <span class="InternalRef"><a href="#Tab5">5</a></span>). When sequencing the gene encoding the Gα subunit in BW051 we found a single nucleotide substitution in exon 10 leading to a premature stop codon (Fig. <span class="InternalRef"><a href="#Fig3">3</a></span>, Table <span class="InternalRef"><a href="#Tab4">4</a></span>). The same mutation was found in the original mutant line <em xmlns="" class="EmphasisTypeItalic">ari-m.28</em>, which was isolated in the cultivar Bonus [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>]. We also performed a cross between BW074 (<em xmlns="" class="EmphasisTypeItalic">brh1.a</em>) and BW051 (<em xmlns="" class="EmphasisTypeItalic">ari-m.28</em>). The F<sub>1</sub> plants displayed a semi-dwarf and short-awn mutant phenotype, which further proves that <em xmlns="" class="EmphasisTypeItalic">ari-m.28</em> and <em xmlns="" class="EmphasisTypeItalic">brh1.a</em> are allelic (Fig. <span class="InternalRef"><a href="#Fig4">4</a></span>).<figure xmlns="" id="Tab5" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 5</span>
                        <p xmlns="" class="SimplePara">Summary of <em class="EmphasisTypeItalic">breviaristatum-m</em> (<em class="EmphasisTypeItalic">ari-m</em>) alleles, their genetic backgrounds, their availability as Bowman near-isogenic lines, and introgression regions on chromosome 7H</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col></col><col></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Allele</p></th><th><p xmlns="" class="SimplePara">Genetic background</p></th><th><p xmlns="" class="SimplePara">Bowman near-isogenic line</p></th><th><p xmlns="" class="SimplePara">Introgression region on 7HS (cM)</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">ari-m.12</em>
                              <sup xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">a</em>
                              </sup>
                            </p></td><td><p xmlns="" class="SimplePara">Bonus</p></td><td><p xmlns="" class="SimplePara">no line available</p></td><td><p xmlns="" class="SimplePara">-</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">ari-m.28</em>
                              <sup xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">a</em>
                              </sup>
                            </p></td><td><p xmlns="" class="SimplePara">Bonus</p></td><td><p xmlns="" class="SimplePara">BW051<sup xmlns="http://www.w3.org/1999/xhtml">a</sup>
                                          </p></td><td><p xmlns="" class="SimplePara">4.74 – 9.55</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">ari-m.141</em>
                              <sup xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">a</em>
                              </sup>
                            </p></td><td><p xmlns="" class="SimplePara">Foma</p></td><td><p xmlns="" class="SimplePara">no line available</p></td><td><p xmlns="" class="SimplePara">-</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">ari-m.177</em>
                              <sup xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">a</em>
                              </sup>
                            </p></td><td><p xmlns="" class="SimplePara">Foma</p></td><td><p xmlns="" class="SimplePara">no line available</p></td><td><p xmlns="" class="SimplePara">-</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">ari-m.251</em>
                              <sup xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">a</em>
                              </sup>
                            </p></td><td><p xmlns="" class="SimplePara">Kristina</p></td><td><p xmlns="" class="SimplePara">no line available</p></td><td><p xmlns="" class="SimplePara">-</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">ari-m.269</em>
                              <sup xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">a</em>
                              </sup>
                            </p></td><td><p xmlns="" class="SimplePara">Kristina</p></td><td><p xmlns="" class="SimplePara">no line available</p></td><td><p xmlns="" class="SimplePara">-</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">The near-isogenic line has been prepared previously as well as the determination of the introgression region [<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>]</p>
                      <p xmlns="" class="SimplePara">
                                    <sup xmlns="http://www.w3.org/1999/xhtml">a</sup>Source of seed material: Nordic Genetic Resource Center, Alnarp, Sweden</p>
                    </div></figure>
                        </div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par28" class="Para">In addition to <em xmlns="" class="EmphasisTypeItalic">ari-m.28</em>, five other <em xmlns="" class="EmphasisTypeItalic">ari-m</em> alleles are available (Table <span class="InternalRef"><a href="#Tab5">5</a></span>) (<span class="ExternalRef"><a href="http://www.nordgen.org/bgs"><span class="RefSource">www.nordgen.org/bgs</span></a></span>). A mutation in the gene encoding the Gα subunit could be identified in each of them (Fig. <span class="InternalRef"><a href="#Fig3">3</a></span>, Table <span class="InternalRef"><a href="#Tab4">4</a></span>). The mutations found in <em xmlns="" class="EmphasisTypeItalic">ari-m.12</em> and <em xmlns="" class="EmphasisTypeItalic">ari-m.28</em> are identical nucleotide substitutions introducing stop codons in exon 10. Mutant line <em xmlns="" class="EmphasisTypeItalic">ari-m.177</em> was identical to <em xmlns="" class="EmphasisTypeItalic">ari-i.38</em> (BW047), affecting a splice site between intron 8 and exon 9. Line <em xmlns="" class="EmphasisTypeItalic">ari-m.141</em> also showed a mutation in a splice site – between intron 1 and exon 2. Mutation <em xmlns="" class="EmphasisTypeItalic">ari-m.269</em> is a nucleotide substitution introducing a stop codons in exon 12. Finally, <em xmlns="" class="EmphasisTypeItalic">ari-m.251</em> is a large deletion since no parts of the <em xmlns="" class="EmphasisTypeItalic">Brh1</em> locus could be amplified by PCR.</p>
              </section><section id="Sec22" class="Section2 RenderAsSection2">
                <h3 class="Heading">Transfer of the <em class="EmphasisTypeItalic">brh1</em> mutation to a modern elite malting barley background</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par29" class="Para">The available <em xmlns="" class="EmphasisTypeItalic">brh1</em> mutant alleles have been generated in several genetically different mother cultivars. In our trials, they all showed a pronounced reduction in culm length and awn length (Fig. <span class="InternalRef"><a href="#Fig5">5</a></span>). To the best of our knowledge, none of the described <em xmlns="" class="EmphasisTypeItalic">brh1</em> alleles have been tested in European breeding material. Therefore, we performed crosses between the Bowman near-isogenic line BW074 (<em xmlns="" class="EmphasisTypeItalic">brh1.a</em>) and the European elite malting cultivar Quench. Double-haploid plants were generated. Two independent double-haploid lines, designated DH_A and DH_B that displayed the phenotype characteristic for <em xmlns="" class="EmphasisTypeItalic">brh1</em> mutant lines, were selected. These lines were propagated and finally tested under field conditions in two subsequent years (Fig. <span class="InternalRef"><a href="#Fig6">6</a></span>). It was observed that cultivar Bowman produced significantly lower grain yield than cultivar Quench. This was expected, as Bowman is a North American cultivar while Quench was bred for agriculture in Europe. Mutant line BW074 had a yield comparable to that of Bowman. The double-haploid lines yielded higher than Bowman and BW074 but lower than Quench. A repetition of the field trial with a reduced number of lines in the following year gave similar results (Fig. <span class="InternalRef"><a href="#Fig6">6b</a></span>). We conclude that the yield of <em xmlns="" class="EmphasisTypeItalic">brh1</em> mutant lines can be significantly increased by backcrossing to a modern elite line.</p>
              </section><section id="Sec23" class="Section2 RenderAsSection2">
                <h3 class="Heading">The effect of the <em class="EmphasisTypeItalic">brh1</em> mutation on malting quality</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par30" class="Para">The <em xmlns="" class="EmphasisTypeItalic">brh1</em> double-haploid lines generated in this study were tested in laboratory scale micro malting experiments together with the near-isogenic line BW074 (<em xmlns="" class="EmphasisTypeItalic">brh1.a</em>) and the cultivars Quench and Bowman. We measured the activity of three key starch degrading enzymes α-amylase, β-amylase and free limit dextrinase (Fig. <span class="InternalRef"><a href="#Fig7">7</a></span>). The <em xmlns="" class="EmphasisTypeItalic">brh1</em> double-haploid lines showed an increase in activity, especially for β-amylase and free limit dextrinase activity. The reason for this observation remains elusive but can partly be caused by a changed volume-area ratio in the global shaped <em xmlns="" class="EmphasisTypeItalic">brh1</em> seeds. Still, the presented results do not indicate that the <em xmlns="" class="EmphasisTypeItalic">brh1</em> mutation has a negative effect on malting quality in barley.</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec24" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Discussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Discussion">Discussion</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Discussion" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par31" class="Para">While there are several genes encoding Gα, Gβ and Gγ subunits of heterotrimeric G proteins in mammals, there is only one Gα, one Gβ and three Gγ encoding genes in arabidopsis, and one Gα, one Gβ and four Gγ genes in rice [<span class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>]. Analysis of the recently sequenced barley genome [<span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>, <span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>]) reveals one Gα (gene identity in Mascher et al. [<span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>] HORVU7Hr1G008720.11), one Gβ (HORVU4Hr1G003110.4) and four Gγ genes (HORVU5Hr1G093530.3, HORVU5Hr1G093550.1, AK367089, HORVU5Hr1G061840.7). In mammalian systems, binding of a ligand to the G-protein-coupled receptor causes a rapid exchange of GDP to GTP bound by the Gα-subunit, which promotes a dissociation of activated Gα-GTP from the Gβγ heterodimer [<span class="CitationRef"><a href="#CR32" data-toggle="citation">32</a></span>]. Activated Gα-GTP and/or Gβγ interacts with effector proteins. A different system for G-protein signaling is found in plants. The arabidopsis Gα subunit spontaneously binds GTP in vitro [<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>] but its ability to stimulate effector molecules is inhibited by its association to a membrane bound Regulator-of-G-protein-signaling (RGS) protein [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>]. Upon binding of a ligand to the RGS protein, the Gα-GTP subunit is released and can interact with effector molecules.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par32" class="Para">In the present study we demonstrated that barley <em xmlns="" class="EmphasisTypeItalic">Brh1</em> is orthologous to rice <em xmlns="" class="EmphasisTypeItalic">D1</em> encoding a Gα-subunit of a heterotrimeric G protein [<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>]. In addition, we characterized nine <em xmlns="" class="EmphasisTypeItalic">brh1</em>, one <em xmlns="" class="EmphasisTypeItalic">ari-i</em> and six <em xmlns="" class="EmphasisTypeItalic">ari-m</em> mutants at the DNA level, and demonstrate that <em xmlns="" class="EmphasisTypeItalic">Ari-m</em> is allelic to <em xmlns="" class="EmphasisTypeItalic">Brh1</em>. It has previously been reported that <em xmlns="" class="EmphasisTypeItalic">Brh1</em> and <em xmlns="" class="EmphasisTypeItalic">Ari-i</em> are allelic [<span class="CitationRef"><a href="#CR33" data-toggle="citation">33</a></span>]. All sixteen mutations result in severe changes of the <em xmlns="" class="EmphasisTypeItalic">Brh1</em> gene and gene product – deletions, nonsense mutations or frame shifts leading to modified polypeptides. It was recently shown that the barley <em xmlns="" class="EmphasisTypeItalic">Ari-e</em> encodes a Gγ subunit of a heterotrimeric G protein [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>, <span class="CitationRef"><a href="#CR35" data-toggle="citation">35</a></span>], being orthologous to <em xmlns="" class="EmphasisTypeItalic">DEP1</em> in rice [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>]. Further, the gene product of <em xmlns="" class="EmphasisTypeItalic">TUD1</em> in rice, a U-box E3 ubiquitin ligase, has been suggested to directly interact with the rice Gα subunit and is therefore directly responsible for the turnover of heterotrimeric G proteins [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>]. We found recently that barley <em xmlns="" class="EmphasisTypeItalic">Brh2</em> is orthologous to <em xmlns="" class="EmphasisTypeItalic">TUD1</em> in rice (Braumann et al. unpublished). Barley <em xmlns="" class="EmphasisTypeItalic">brh1</em>, <em xmlns="" class="EmphasisTypeItalic">brh2</em> and <em xmlns="" class="EmphasisTypeItalic">ari-e</em> mutants have short-culm phenotypes, which is now understood in the light of their connection to the heterotrimeric G protein.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par33" class="Para">Although genes encoding subunits of the heterotrimeric G protein complex now start to be identified, it is still not clear in which signaling pathway the complex is participating. The response to both brassinosteroids and gibberellic acids have been investigated for <em xmlns="" class="EmphasisTypeItalic">brh1</em> mutants in previous studies. In contrast to <em xmlns="" class="EmphasisTypeItalic">uzu</em> mutants, <em xmlns="" class="EmphasisTypeItalic">brh1</em> mutants are brassinosteroid sensitive [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>], making it unlikely that <em xmlns="" class="EmphasisTypeItalic">brh1</em> is part of the brassinosteroid signaling pathway. It has further been shown that application of gibberellic acid leads to increased elongation of <em xmlns="" class="EmphasisTypeItalic">brh1</em> plants in a similar extend as for wild type plants. Still, both gibberellic acid treated and untreated <em xmlns="" class="EmphasisTypeItalic">brh1</em> plants are significantly shorter than wild type plants that had received the same treatment [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>]. These findings disprove the direct involvement of the <em xmlns="" class="EmphasisTypeItalic">Brh1</em> gene in either gibberellic acid signaling or biosynthesis.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par34" class="Para">Dahleen et al. [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>] did a detailed study of the agronomic performance of five recessive <em xmlns="" class="EmphasisTypeItalic">brh1</em> lines – BW047 (<em xmlns="" class="EmphasisTypeItalic">ari-i.38</em>), BW074 (<em xmlns="" class="EmphasisTypeItalic">brh1.a</em>), BW078 (<em xmlns="" class="EmphasisTypeItalic">brh1.t</em>), BW080 (<em xmlns="" class="EmphasisTypeItalic">brh1.z</em>) and BW075 (<em xmlns="" class="EmphasisTypeItalic">brh1.aa</em>). The parameters including plant height, lodging, yield, number of kernels per spike and kernel weight. All <em xmlns="" class="EmphasisTypeItalic">brh1</em> plants were shown to be significantly shorter and more lodging resistant than the cultivar Bowman. The mutants displayed a lower kernel weight but otherwise yields were like Bowman with the exception of BW075 (<em xmlns="" class="EmphasisTypeItalic">bri1.aa</em>), which yielded lower. Based on their results it was concluded that the <em xmlns="" class="EmphasisTypeItalic">brh1</em> mutations introduce agronomically valuable traits like shorter plant stature and lower lodging potential while negative effects of the mutation on agronomical key traits like yield seem to be neglectable [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>]. Our study indicated a lower yield due to the <em xmlns="" class="EmphasisTypeItalic">brh1</em> mutation but we assume that recurrent backcrossing to Quench would increase the yield like it did in the Bowman near-isogenic lines studied by Dahleen et al. [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>]. More important, our results showed no negative effect on malting qualities due to the <em xmlns="" class="EmphasisTypeItalic">brh1</em> mutation. We therefore suggest that <em xmlns="" class="EmphasisTypeItalic">Brh1</em> might be a competitive breeding alternative to other used semi-dwarfing genes like <em xmlns="" class="EmphasisTypeItalic">Ari-e</em>, <em xmlns="" class="EmphasisTypeItalic">Ert-k</em>, <em xmlns="" class="EmphasisTypeItalic">Sdw1</em>, <em xmlns="" class="EmphasisTypeItalic">Sdw4</em> and <em xmlns="" class="EmphasisTypeItalic">Uzu1</em>.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec25" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Conclusions" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Conclusions">Conclusions</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Conclusions" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par35" class="Para">Mutants in the <em xmlns="" class="EmphasisTypeItalic">Brh1</em> locus are deficient in the Gα subunit of a heterotrimeric G protein, which shows that heterotrimeric G proteins are important regulators of culm length in barley. It is still not clear in which signaling pathway the complex is participating. Sixteen mutant alleles were characterized at DNA level and display a repertoire of deletions and nucleotide substitutions. While our study suggests a lower yield due to mutations in <em xmlns="" class="EmphasisTypeItalic">Brh1</em>, we did not see any negative effects on malting quality. Thus, <em xmlns="" class="EmphasisTypeItalic">brh1</em> mutant alleles might be used in order to breed for robust and lodging resistant malting barley cultivars.</p></div></section>
          <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments"><h3 xmlns="" class="Heading">Acknowledgements</h3><p xmlns="" class="SimplePara">This study was supported by the Carlsberg Foundation, Grønt Udviklings- og Demonstrations Program (GUDP, Denmark, 34009-12-0522), Deutsche Forschungsgemeinschaft (Germany, DO1482/1-1), the Crafoord Foundation (20150748) and the CF Lundström Foundation.</p><div id="FPar1" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Funding</h3>
                <p id="Par36" class="Para">This study was supported by the Carlsberg Foundation, Grønt Udviklings- og Demonstrations Program (GUDP, Denmark, 34,009-12-0522), Deutsche Forschungsgemeinschaft (Germany, DO1482/1-1), the Crafoord Foundation (20150748) and the CF Lundström Foundation.</p>
              </div><div id="FPar2" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Availability of data and materials</h3>
                <p id="Par37" class="Para">Not applicable.</p>
              </div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="FormalPara FormalParaRenderingStyle1"><h3 xmlns="" class="Heading">Authors’ contributions</h3><p xmlns="" class="SimplePara">IB, CD, SB, AH, FL, UL, SZ and RZ performed experiments. IB, BS, NS and MH designed research. IB and MH wrote the paper. All authors read and approved the final manuscript.</p></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Ethics">
              <div id="FPar3" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Ethics approval and consent to participate</h3>
                <p id="Par38" class="Para">Not applicable.</p>
              </div>
              <div id="FPar4" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Consent for publication</h3>
                <p id="Par39" class="Para">Not applicable.</p>
              </div>
              <div id="FPar5" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Competing interests</h3>
                <p id="Par40" class="Para">The authors declare that they have no competing interests.</p>
              </div>
              <div id="FPar6" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Publisher’s Note</h3>
                <p id="Par41" class="Para">Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
              </div>
            </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="License">
                <p xmlns="" class="SimplePara">
                           <strong class="EmphasisTypeBold">Open Access</strong>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/4.0/"><span class="RefSource">http://creativecommons.org/licenses/by/4.0/</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/publicdomain/zero/1.0/"><span class="RefSource">http://creativecommons.org/publicdomain/zero/1.0/</span></a></span>) applies to the data made available in this article, unless otherwise stated.</p>
              </div><aside class="Appendix" id="App1">
              
                <h3 class="Heading">Additional files</h3><div id="widgetContainer" data-test="figshare-container"></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par42" class="Para">
                  <div class="MediaObject" id="MOESM1">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-017-0045-1/MediaObjects/41065_2017_45_MOESM1_ESM.xlsx"><span class="caption-number">Additional file 1: Table S1.</span></a>
                          <span xmlns="" class="SimplePara">SNP and mutation information for Brh1 mutant lines and original background cultivars. (XLSX 121&nbsp;kb)</span>
                        </div></div>
                  </div>
                </div>
              
            </aside></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authors’ Affiliations</a></h2><div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1"><span class="AffiliationNumber">(1)</span><div class="AffiliationText">Carlsberg Research Laboratory, J. C. Jacobsens Gade 4, DK-1799&nbsp;Copenhagen V, Denmark</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff2"><span class="AffiliationNumber">(2)</span><div class="AffiliationText">Leibniz Institute of Plant Genetics and Crop Plant Research (IPK), OT Gatersleben, DE-06466&nbsp;Stadt Seeland, Germany</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff3"><span class="AffiliationNumber">(3)</span><div class="AffiliationText">Nordic Genetic Resource Center (NordGen), Smedjevägen 3, SE-23053&nbsp;Alnarp, Sweden</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff4"><span class="AffiliationNumber">(4)</span><div class="AffiliationText">Department of Biology, Lund University, Sölvegatan 35, SE-22362&nbsp;Lund, Sweden</div></div></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper"><li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ahloowalia BS, Maluszynski M, Nichterlein K. Global impact of mutation-derived varieties. Euphytica. 2004;135:187–204.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1023/B%3AEUPH.0000014914.85465.4f"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Global%20impact%20of%20mutation-derived%20varieties&amp;author=BS.%20Ahloowalia&amp;author=M.%20Maluszynski&amp;author=K.%20Nichterlein&amp;journal=Euphytica&amp;volume=135&amp;pages=187-204&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ariyadasa R, Stein N. Advances in BAC-based physical mapping and map integration strategies in plants. J Biomed Biotechnol. 2012;2012:184854.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1155/2012/184854"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22500080"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3303678"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Advances%20in%20BAC-based%20physical%20mapping%20and%20map%20integration%20strategies%20in%20plants&amp;author=R.%20Ariyadasa&amp;author=N.%20Stein&amp;journal=J%20Biomed%20Biotechnol&amp;volume=2012&amp;pages=184854&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ashikari M, Sasaki A, Ueguchi-Tanaka M, et al. Loss-of-function of a rice gibberellin biosynthetic gene, GA20 oxidase (GA20ox-2), led to the rice "green revolution". Breed Sci. 2002;52:143–50.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1270/jsbbs.52.143"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Loss-of-function%20of%20a%20rice%20gibberellin%20biosynthetic%20gene%2C%20GA20%20oxidase%20%28GA20ox-2%29%2C%20led%20to%20the%20rice%20%22green%20revolution%22&amp;author=M.%20Ashikari&amp;author=A.%20Sasaki&amp;author=M.%20Ueguchi-Tanaka&amp;journal=Breed%20Sci&amp;volume=52&amp;pages=143-150&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ashikari M, Wu J, Yano M, Sasaki T, Yoshimura A. Rice gibberellin-insensitive dwarf mutant gene Dwarf 1 encodes the alpha-subunit of GTP-binding protein. Proc Natl Acad Sci U S A. 1999;96:10284–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.96.18.10284"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10468600"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC17880"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Rice%20gibberellin-insensitive%20dwarf%20mutant%20gene%20Dwarf%201%20encodes%20the%20alpha-subunit%20of%20GTP-binding%20protein&amp;author=M.%20Ashikari&amp;author=J.%20Wu&amp;author=M.%20Yano&amp;author=T.%20Sasaki&amp;author=A.%20Yoshimura&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;volume=96&amp;pages=10284-10289&amp;publication_year=1999"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Börner A. GA response in semidwarf barley. Barley Genet. Newslett. 1996;25:24–7.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=GA%20response%20in%20semidwarf%20barley&amp;author=A.%20B%C3%B6rner&amp;journal=Barley%20Genet.%20Newslett.&amp;volume=25&amp;pages=24-27&amp;publication_year=1996"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Caierão E. Effect of induced lodging on grain yield and quality of brewing barley. Crop Breed Appl Biotech. 2006;6:215–21.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.12702/1984-7033.v06n03a05"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Effect%20of%20induced%20lodging%20on%20grain%20yield%20and%20quality%20of%20brewing%20barley&amp;author=E.%20Caier%C3%A3o&amp;journal=Crop%20Breed%20Appl%20Biotech&amp;volume=6&amp;pages=215-221&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Chono M, Honda I, Zeniya H, et al. A semidwarf phenotype of barley uzu results from a nucleotide substitution in the gene encoding a putative brassinosteroid receptor. Plant Physiol. 2003;133:1209–19.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1104/pp.103.026195"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14551335"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC281616"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20semidwarf%20phenotype%20of%20barley%20uzu%20results%20from%20a%20nucleotide%20substitution%20in%20the%20gene%20encoding%20a%20putative%20brassinosteroid%20receptor&amp;author=M.%20Chono&amp;author=I.%20Honda&amp;author=H.%20Zeniya&amp;journal=Plant%20Physiol&amp;volume=133&amp;pages=1209-1219&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Dahleen LS, Vander Wal LJ, Franckowiak JD. Characterization and molecular mapping of genes determining semidwarfism in barley. J Hered. 2005;96:654–62.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/jhered/esi125"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16251510"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Characterization%20and%20molecular%20mapping%20of%20genes%20determining%20semidwarfism%20in%20barley&amp;author=LS.%20Dahleen&amp;author=LJ.%20Vander%20Wal&amp;author=JD.%20Franckowiak&amp;journal=J%20Hered&amp;volume=96&amp;pages=654-662&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Dockter C, Gruszka D, Braumann I, et al. Induced variations in brassinosteroid genes define barley height and sturdiness, and expand the green revolution genetic toolkit. Plant Physiol. 2014;166:1912–27.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1104/pp.114.250738"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25332507"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4256852"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Induced%20variations%20in%20brassinosteroid%20genes%20define%20barley%20height%20and%20sturdiness%2C%20and%20expand%20the%20green%20revolution%20genetic%20toolkit&amp;author=C.%20Dockter&amp;author=D.%20Gruszka&amp;author=I.%20Braumann&amp;journal=Plant%20Physiol&amp;volume=166&amp;pages=1912-1927&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Dockter C, Hansson M. Improving barley culm robustness for secured crop yield in a changing climate. J Exp Bot. 2015;66:3499–509.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/jxb/eru521"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25614659"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Improving%20barley%20culm%20robustness%20for%20secured%20crop%20yield%20in%20a%20changing%20climate&amp;author=C.%20Dockter&amp;author=M.%20Hansson&amp;journal=J%20Exp%20Bot&amp;volume=66&amp;pages=3499-3509&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Druka A, Franckowiak J, Lundqvist U, et al. Genetic dissection of barley morphology and development. Plant Physiol. 2011;155:617–27.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1104/pp.110.166249"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21088227"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20dissection%20of%20barley%20morphology%20and%20development&amp;author=A.%20Druka&amp;author=J.%20Franckowiak&amp;author=U.%20Lundqvist&amp;journal=Plant%20Physiol&amp;volume=155&amp;pages=617-627&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Foster BP. Mutation genetics of salt tolerance in barley: an assessment of golden promise and other semi-dwarf mutants. Euphytica. 2001;120:317–28.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1023/A%3A1017592618298"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Mutation%20genetics%20of%20salt%20tolerance%20in%20barley%3A%20an%20assessment%20of%20golden%20promise%20and%20other%20semi-dwarf%20mutants&amp;author=BP.%20Foster&amp;journal=Euphytica&amp;volume=120&amp;pages=317-328&amp;publication_year=2001"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gustafsson Å, Ekman G. Yield efficiency of the X-ray mutant Svalöf's 'Pallas barley. Züchter/Genet Breed Res. 1967;37:42–6.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Yield%20efficiency%20of%20the%20X-ray%20mutant%20Sval%C3%B6f%27s%20%27Pallas%20barley&amp;author=%C3%85.%20Gustafsson&amp;author=G.%20Ekman&amp;journal=Z%C3%BCchter%2FGenet%20Breed%20Res&amp;volume=37&amp;pages=42-46&amp;publication_year=1967"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Honda I, Zeniya H, Yoneyama K, Chono M, Kaneko S, Watanabe Y. Uzu mutation in barley (Hordeum vulgare L.) reduces the leaf unrolling response to brassinolide. Biosci Biotechnol Biochem. 2003;67:1194–7.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1271/bbb.67.1194"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12834311"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Uzu%20mutation%20in%20barley%20%28Hordeum%20vulgare%20L.%29%20reduces%20the%20leaf%20unrolling%20response%20to%20brassinolide&amp;author=I.%20Honda&amp;author=H.%20Zeniya&amp;author=K.%20Yoneyama&amp;author=M.%20Chono&amp;author=S.%20Kaneko&amp;author=Y.%20Watanabe&amp;journal=Biosci%20Biotechnol%20Biochem&amp;volume=67&amp;pages=1194-1197&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hu X, Qian Q, Xu T, et al. The U-box E3 ubiquitin ligase TUD1 functions with a heterotrimeric G alpha subunit to regulate Brassinosteroid-mediated growth in rice. PLoS Genet. 2013;9:e1003391.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pgen.1003391"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23526892"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3597501"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20U-box%20E3%20ubiquitin%20ligase%20TUD1%20functions%20with%20a%20heterotrimeric%20G%20alpha%20subunit%20to%20regulate%20Brassinosteroid-mediated%20growth%20in%20rice&amp;author=X.%20Hu&amp;author=Q.%20Qian&amp;author=T.%20Xu&amp;journal=PLoS%20Genet&amp;volume=9&amp;pages=e1003391&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Huang X, Qian Q, Liu Z, et al. Natural variation at the DEP1 locus enhances grain yield in rice. Nat Genet. 2009;41:494–7.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/ng.352"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19305410"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Natural%20variation%20at%20the%20DEP1%20locus%20enhances%20grain%20yield%20in%20rice&amp;author=X.%20Huang&amp;author=Q.%20Qian&amp;author=Z.%20Liu&amp;journal=Nat%20Genet&amp;volume=41&amp;pages=494-497&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">International Barley Genome Sequencing, C, Mayer KF, Waugh R, et al. A physical, genetic and functional sequence assembly of the barley genome. Nature. 2012;491:711–6.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20physical%2C%20genetic%20and%20functional%20sequence%20assembly%20of%20the%20barley%20genome&amp;author=KF.%20Mayer&amp;author=R.%20Waugh&amp;journal=Nature&amp;volume=491&amp;pages=711-716&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Jia Q, Tan C, Wang J, et al. Marker development using SLAF-seq and whole-genome shotgun strategy to finemap the semi-dwarf gene ari-e in barley. BMC Genomics. 2016;17:911.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/s12864-016-3247-4"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27835941"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5106812"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Marker%20development%20using%20SLAF-seq%20and%20whole-genome%20shotgun%20strategy%20to%20finemap%20the%20semi-dwarf%20gene%20ari-e%20in%20barley&amp;author=Q.%20Jia&amp;author=C.%20Tan&amp;author=J.%20Wang&amp;journal=BMC%20Genomics&amp;volume=17&amp;pages=911&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Johnston CA, Taylor JP, Gao Y, et al. GTPase acceleration as the rate-limiting step in Arabidopsis G protein-coupled sugar signaling. Proc Natl Acad Sci U S A. 2007;104:17317–22.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.0704751104"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17951432"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2077254"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=GTPase%20acceleration%20as%20the%20rate-limiting%20step%20in%20Arabidopsis%20G%20protein-coupled%20sugar%20signaling&amp;author=CA.%20Johnston&amp;author=JP.%20Taylor&amp;author=Y.%20Gao&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;volume=104&amp;pages=17317-17322&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kucera J, Lundqvist U, Gustafsson A. Induction of breviaristatum mutants in barley. Hereditas. 1975;80:263–78.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1601-5223.1975.tb01525.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1102498"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Induction%20of%20breviaristatum%20mutants%20in%20barley&amp;author=J.%20Kucera&amp;author=U.%20Lundqvist&amp;author=A.%20Gustafsson&amp;journal=Hereditas&amp;volume=80&amp;pages=263-278&amp;publication_year=1975"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li M, Kudrna D, Kleinhofs A. Fine mapping of a semi-dwarf gene brachytic1 in barley. Barley Genet. Newslett. 2001;31:14–7.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Fine%20mapping%20of%20a%20semi-dwarf%20gene%20brachytic1%20in%20barley&amp;author=M.%20Li&amp;author=D.%20Kudrna&amp;author=A.%20Kleinhofs&amp;journal=Barley%20Genet.%20Newslett.&amp;volume=31&amp;pages=14-17&amp;publication_year=2001"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Mascher M, Gundlach H, Himmelbach A, et al. A chromosome conformation capture ordered sequence of the barley genome: Nature In press; 2017.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Mascher%20M%2C%20Gundlach%20H%2C%20Himmelbach%20A%2C%20et%20al.%20A%20chromosome%20conformation%20capture%20ordered%20sequence%20of%20the%20barley%20genome%3A%20Nature%20In%20press%3B%202017."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Peng J, Richards DE, Hartley NM, et al. 'Green revolution' genes encode mutant gibberellin response modulators. Nature. 1999;400:256–61.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/22307"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10421366"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=%27Green%20revolution%27%20genes%20encode%20mutant%20gibberellin%20response%20modulators&amp;author=J.%20Peng&amp;author=DE.%20Richards&amp;author=NM.%20Hartley&amp;journal=Nature&amp;volume=400&amp;pages=256-261&amp;publication_year=1999"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR24" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Powers L. The nature of the interactions of genes affecting four quantitative characters in a cross between Hordeum deficiens and vulgare. Genetics. 1936;21:398–420.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17246802"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1208683"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20nature%20of%20the%20interactions%20of%20genes%20affecting%20four%20quantitative%20characters%20in%20a%20cross%20between%20Hordeum%20deficiens%20and%20vulgare&amp;author=L.%20Powers&amp;journal=Genetics&amp;volume=21&amp;pages=398-420&amp;publication_year=1936"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR25" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Rajkumara S. Lodging in cereals. Agricult Rev. 2008;29:55–60.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Lodging%20in%20cereals&amp;author=S.%20Rajkumara&amp;journal=Agricult%20Rev&amp;volume=29&amp;pages=55-60&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR26" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sashio D, Tanno K, Chono M, Honda I, Kitano H, Tekeda K. Spontaneous brassinolide-insensitive barley mutants “uzu.” adapted to East Asia. Breed Sci. 2004;54:409–16.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1270/jsbbs.54.409"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Spontaneous%20brassinolide-insensitive%20barley%20mutants%20%E2%80%9Cuzu.%E2%80%9D%20adapted%20to%20East%20Asia&amp;author=D.%20Sashio&amp;author=K.%20Tanno&amp;author=M.%20Chono&amp;author=I.%20Honda&amp;author=H.%20Kitano&amp;author=K.%20Tekeda&amp;journal=Breed%20Sci&amp;volume=54&amp;pages=409-416&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR27" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Schulte D, Ariyadasa R, Shi B, et al. BAC library resources for map-based cloning and physical map construction in barley (Hordeum Vulgare L.). BMC Genomics. 2011;12:247.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2164-12-247"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21595870"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3224359"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=BAC%20library%20resources%20for%20map-based%20cloning%20and%20physical%20map%20construction%20in%20barley%20%28Hordeum%20Vulgare%20L.%29&amp;author=D.%20Schulte&amp;author=R.%20Ariyadasa&amp;author=B.%20Shi&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=247&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR28" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Spielmeyer W, Ellis MH, Chandler PM. Semidwarf (sd-1), “green revolution” rice, contains a defective gibberellin 20-oxidase gene. Proc Natl Acad Sci U S A. 2002;99:9043–8.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.132266399"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12077303"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC124420"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Semidwarf%20%28sd-1%29%2C%20%E2%80%9Cgreen%20revolution%E2%80%9D%20rice%2C%20contains%20a%20defective%20gibberellin%2020-oxidase%20gene&amp;author=W.%20Spielmeyer&amp;author=MH.%20Ellis&amp;author=PM.%20Chandler&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;volume=99&amp;pages=9043-9048&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR29" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Stateczny D, Oppenheimer J, Bommert P. G protein signaling in plants: minus times minus equals plus. Curr Opin Plant Biol. 2016;34:127–35.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.pbi.2016.11.001"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27875794"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=G%20protein%20signaling%20in%20plants%3A%20minus%20times%20minus%20equals%20plus&amp;author=D.%20Stateczny&amp;author=J.%20Oppenheimer&amp;author=P.%20Bommert&amp;journal=Curr%20Opin%20Plant%20Biol&amp;volume=34&amp;pages=127-135&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR30" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Swenson SP. Genetic and cytological studies on a brachytic mutant in barley. J Agric Res. 1940;60:687–713.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20and%20cytological%20studies%20on%20a%20brachytic%20mutant%20in%20barley&amp;author=SP.%20Swenson&amp;journal=J%20Agric%20Res&amp;volume=60&amp;pages=687-713&amp;publication_year=1940"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR31" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Takahashi R. The origin and evolution of cultivated barley. In: Demerec M, editor. Advances in genetics. New York: Academic Press; 1955. p. 227–66.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20origin%20and%20evolution%20of%20cultivated%20barley&amp;author=R.%20Takahashi&amp;pages=227-266&amp;publication_year=1955"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR32" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Temple BR, Jones AM. The plant heterotrimeric G-protein complex. Annu Rev Plant Biol. 2007;58:249–66.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1146/annurev.arplant.58.032806.103827"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17201690"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20plant%20heterotrimeric%20G-protein%20complex&amp;author=BR.%20Temple&amp;author=AM.%20Jones&amp;journal=Annu%20Rev%20Plant%20Biol&amp;volume=58&amp;pages=249-266&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR33" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Tsuchiya T. Allelic relationships of genes for short-awned mutants in barley. Barley Genet Newslett. 1974;4:80–1.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Allelic%20relationships%20of%20genes%20for%20short-awned%20mutants%20in%20barley&amp;author=T.%20Tsuchiya&amp;journal=Barley%20Genet%20Newslett&amp;volume=4&amp;pages=80-81&amp;publication_year=1974"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR34" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Urano D, Chen JG, Botella JR, Jones AM. Heterotrimeric G protein signalling in the plant kingdom. Open Biol. 2013;3:120186.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1098/rsob.120186"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23536550"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3718340"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Heterotrimeric%20G%20protein%20signalling%20in%20the%20plant%20kingdom&amp;author=D.%20Urano&amp;author=JG.%20Chen&amp;author=JR.%20Botella&amp;author=AM.%20Jones&amp;journal=Open%20Biol&amp;volume=3&amp;pages=120186&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR35" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wendt T, Holme I, Dockter C, et al. HvDep1 is a positive regulator of culm elongation and grain size in barley and impacts yield in an environment-dependent manner. PLoS One. 2016;11:e0168924.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0168924"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28005988"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5179111"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=HvDep1%20is%20a%20positive%20regulator%20of%20culm%20elongation%20and%20grain%20size%20in%20barley%20and%20impacts%20yield%20in%20an%20environment-dependent%20manner&amp;author=T.%20Wendt&amp;author=I.%20Holme&amp;author=C.%20Dockter&amp;journal=PLoS%20One&amp;volume=11&amp;pages=e0168924&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR36" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Xu Y, Jia Q, Zhou G, et al. Characterization of the sdw1 semi-dwarf gene in barley. BMC Plant Biol. 2017;17:11.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/s12870-016-0964-4"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28086794"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5237212"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Characterization%20of%20the%20sdw1%20semi-dwarf%20gene%20in%20barley&amp;author=Y.%20Xu&amp;author=Q.%20Jia&amp;author=G.%20Zhou&amp;journal=BMC%20Plant%20Biol&amp;volume=17&amp;pages=11&amp;publication_year=2017"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR37" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhang J, Li Z, Zhang C. Analysis of dwarfing genes in Zhepi 1 and Aizao 3: two dwarfing gene donors in barley breeding in China. Canadian J Plant Sci. 2007;87:93–6.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.4141/P06-042"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Analysis%20of%20dwarfing%20genes%20in%20Zhepi%201%20and%20Aizao%203%3A%20two%20dwarfing%20gene%20donors%20in%20barley%20breeding%20in%20China&amp;author=J.%20Zhang&amp;author=Z.%20Li&amp;author=C.%20Zhang&amp;journal=Canadian%20J%20Plant%20Sci&amp;volume=87&amp;pages=93-96&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR38" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhang J, Zhang W. Tracing sources of dwarfing genes in barley breeding in China. Euphytica. 2003;131:285–92.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1023/A%3A1023955131523"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Tracing%20sources%20of%20dwarfing%20genes%20in%20barley%20breeding%20in%20China&amp;author=J.%20Zhang&amp;author=W.%20Zhang&amp;journal=Euphytica&amp;volume=131&amp;pages=285-292&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li></ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#TheAuthors" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="TheAuthors">Copyright</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="TheAuthors"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">©&nbsp;The Author(s)&nbsp;2017</div></div></section></div>
                        


                    </article>
                </main>

                <div class="c-content-layout__side u-text-sm">
                    <aside>
                        <div class="p-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="//hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-017-0045-1" data-track="click" data-track-category="PDF Download"
            data-track-action="Click download article PDF"
            data-track-label="10.1186/s41065-017-0045-1">Download PDF</a>
        </p>
    
                            
                            
    <button type="button"
       class="c-button c-button--secondary c-button--full-width c-button--expandable"
       data-toggle="collapse"
       data-target="export-citations"
       data-test="export-citations-button"
       data-sticky-update="true"
       aria-controls="export-citations"
       aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/s41065-017-0045-1"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0045-1?format=refman&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/s41065-017-0045-1"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0045-1?format=refman&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/s41065-017-0045-1"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0045-1?format=refman&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/s41065-017-0045-1"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0045-1?format=endnote&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/s41065-017-0045-1"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0045-1?format=endnote&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/s41065-017-0045-1"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0045-1?format=endnote&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/s41065-017-0045-1"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0045-1?format=bibtex&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/s41065-017-0045-1"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0045-1?format=bibtex&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/s41065-017-0045-1"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0045-1?format=bibtex&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
    </div>

                        </div>
                        
    
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
                                <div id="article-access-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/access-count/10.1186/s41065-017-0045-1 #accessCount-container"></div>
                                <div id="citation-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/citation-count/10.1186/s41065-017-0045-1 #count-container"></div>
                                <div id="altmetric-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/altmetric/10.1186/s41065-017-0045-1 #altmetric-container"></div>
                            </ul>
                        </div>
                        
    <div id="sociaMediaShare" class="p-separator u-js-shown">
        <h3 class="p-section-title">Share this article</h3>
        <ul id="shareButtons" class="c-social-links">
            <li class="c-social-links__item">
                <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-017-0045-1">
                    <span class="u-visually-hidden">Share on Twitter</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-twitter"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-017-0045-1">
                    <span class="u-visually-hidden">Share on Facebook</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-facebook"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click"
                        data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-017-0045-1">
                    <span class="u-visually-hidden">Share on LinkedIn</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-linkedin"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-017-0045-1">
                    <span class="u-visually-hidden">Share on Weibo</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-weibo"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnGooglePlus" class="js-btnShareOnGplus" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Google+" data-track-label="10.1186/s41065-017-0045-1">
                    <span class="u-visually-hidden">Share on Google Plus</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-gplus"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-017-0045-1">
                    <span class="u-visually-hidden">Share on Reddit</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-reddit"></use>
                    </svg>
                </button>
            </li>
        </ul>
    </div>
 
                        
    
        <div class="p-separator u-js-shown">
            <h3 class="p-section-title">See updates</h3>
            <button class="c-crossmark-button" data-target="crossmark" data-test="crossmark" data-track="click" data-track-category="Article Page" data-track-action="Click CrossMark" data-track-label="10.1186/s41065-017-0045-1">
                <img id="crossmark-icon" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" alt="Crossmark logo" />
            </button>
        </div>
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
                                <li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/s41065-017-0045-1" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=738&doi=10.1186%2Fs41065-017-0045-1&volume=155&issue=1&title=Mutations+in+the+gene+of+the+G%CE%B1+subunit+of+the+heterotrimeric+G+protein+are+the+cause+for+the+brachytic1+semi-dwarf+phenotype+in+barley+and+applicable+for+practical+breeding&author_name=Ilka+Braumann&start_page=1&end_page=13">Order reprint</a>

                                </li>
                            </ul>
                        </div>
                    </aside>
                    
    <div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">Hereditas</span>
                        </h4>
                        <p class="c-journal-footer__issn">ISSN: 1601-5223</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
                                
                                    <li class="c-journal-footer__contact-item">Submission enquiries: <a href="https://www.editorialmanager.com/here/" target="_blank">Access here and click Contact Us</a></li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a></li>
                                
                            </ul>
                        </div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class='tracker' style='display:none' src='/track/article/10.1186/s41065-017-0045-1' alt=""/>

         
    <footer>
        <div class="ctx-publisher-footer">
    <div class="c-navbar">
        <div class="u-container">
            <div class="c-navbar__logo u-margin-right-xl">
                <a href="https://www.biomedcentral.com">
                    <img src=/static/images/bmc/logos/logo-bmc-aj-594e03e20d.svg alt="BMC logo" data-test="navbar-logo-footer">

                </a>
            </div>
            <div class="c-navbar__content">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


            </div>
        </div>
    </div>
    <div class="c-publisher-footer" data-test="publisher-footer">
        <div class="u-container">
            
            <div class="c-flex c-flex--wrap c-flex--space-between" role="navigation" data-test="publisher-footer-menu">
                <div class="c-flex" aria-label="Publisher footer links">
                    
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/login">Receive BMC newsletters</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/account">Manage article alerts</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/policies">Policies</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/accessibility">Accessibility</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/press-centre">Press center</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/contact-us">Contact us</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/jobs">Careers</a>
                                    </li>
                                
                            </ul>
                        
                    
                </div>
                <div class="u-margin-bottom-lg">
                    <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                    <ul class="c-flex u-reset-list" data-test="footer-social-links">
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://twitter.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Twitter link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Twitter page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-twitter-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://www.facebook.com/BioMedCentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Facebook link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Facebook page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-facebook-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://plus.google.com/106226337984152901734/"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Google Plus link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Google Plus page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-gplus-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="http://www.weibo.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Weibo link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Weibo page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-weibo-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </div>
            <p class="u-reset-margin">
                By using this website, you agree to our
                <a class="u-text-inherit" href="//www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
                <a class="u-text-inherit" href="//www.biomedcentral.com/privacy-statement">Privacy
                    statement</a> and
                <a class="u-text-inherit" href="//www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
                
                    <a class="optanon-toggle-display u-text-inherit" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
                
            </p>
        </div>
    </div>
</div>


        <div class="c-sn-footer c-sn-footer--neutral">
    <div class="u-container">
        <img src=/static/images/bmc/logos/logo-springernature-8bd4e8d82f.svg class="c-sn-footer__logo" alt="Springer Nature Logo" itemprop="logo" >
        <p data-test="copyright">&#169; 2018 BioMed Central Ltd unless otherwise stated. Part of
            <a href="http://www.springernature.com"
               data-track="click"
               data-track-action="Click Springer Nature link"
               data-track-category="Corporate footer"
               itemscope itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>

    </div>

    
    <script type="text/javascript" src="//cdn.pbgrd.com/bmc.js" async="async"></script>
    <noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>
    <!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker -->

<script id="springerlink-event-tracker" src="https://event-tracker.springernature.com/dist/eventTracker.js"></script>
<script type="text/javascript">
    (function(){
        var dataLayer = window.dataLayer[0];
        var doi = dataLayer.content.article.doi;
        var imprint = dataLayer.content.contentInfo.imprint;
        var bpids = dataLayer.session.authentication.authenticationID;
        var publisher;
        var pageType = dataLayer.page.category.pageType;
        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }
        var eventObject = {'content_type' : pageType,'doi' : doi};

        if (imprint === 'BioMed Central') { publisher = 'BMC' }
        else if (imprint === 'SpringerOpen') { publisher = 'SpringerOpen' }
        else { throw new Error('Event Tracker Error: The publisher has an unexpected value') }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }
        new EventTracker({'platform':publisher}).sendEvent('display', eventObject);
    })();
</script>


    </body>
</html>

