<!DOCTYPE html>
<html lang="en" class="no-js">
    <head>
        <title>Genetic diversity and structure of tea plant in Qinba area in China by three types of molecular markers | Hereditas | Full Text</title>

        

    <meta name="citation_abstract" content="Qinba area has a long history of tea planting and is a northernmost region in China where Camellia sinensis L. is grown. In order to provide basic data for selection and optimization of molecular markers of tea plants. 118 markers, including 40 EST-SSR, 40 SRAP and 38 SCoT markers were used to evaluate the genetic diversity of 50 tea plant (Camellia sinensis.) samples collected from Qinb. tea germplasm, assess population structure. In this study, a total of 414 alleles were obtained using 38 pairs of SCoT primers, with an average of 10.89 alleles per primer. The percentage of polymorphic bands (PPB), polymorphism information content (PIC), resolving power (Rp), effective multiplex ratio (EMR), average band informativeness (Ibav), and marker index (MI) were 96.14%, 0.79, 6.71, 10.47, 0.58, and 6.07 respectively. 338 alleles were amplified via 40 pairs of SRAP (8.45 per primer), with PPB, PIC, Rp, EMR, Ibav, and MI values of 89.35%, 0.77, 5.11, 7.55, 0.61, and 4.61, respectively. Furthermore, 320 alleles have been detected using 40 EST-SSR primers (8.00 per primer), with PPB, PIC, Rp, EMR, Ibav, and MI values of 94.06%, 0.85, 4.48, 7.53, 0.56, and 4.22 respectively. These results indicated that SCoT markers had higher efficiency. Mantel test was used to analyze the genetic distance matrix generated by EST-SSRs, SRAPs and SCoTs. The results showed that the correlation between the genetic distance matrix based on EST-SSR and that based on SRAP was very small (r = 0.01), followed by SCoT and SRAP (r = 0.17), then by SCoT and EST-SSR (r = 0.19). The 50 tea samples were divided into two sub-populations using STRUCTURE, Neighbor-joining (NJ) method and principal component analyses (PCA). The results produced by STRUCTURE were completely consistent with the PCA analysis. Furthermore, there is no obvious relationship between the results produced using sub-populational and geographical data. Among the three types of markers, SCoT markers has many advantages in terms of NPB, PPB, Rp, EMR, and MI. Nevertheless, the values of PIC showed different trends, with the highest values generated with EST-SSR, followed by SCoT and SRAP. The average band informativeness showed similar trends. Correlation between genetic distances produced by three different molecular markers were very small, thus it is not recommended to use a single marker to evaluate genetic diversity and population structure. It is hence suggested that combining of different types of molecular markers should be used to evaluate the genetic diversity and population structure. It also seems crucial to screen out, for each type of molecular markers, core markers of Camellia sinensis. This study revealed that genes of exotic plant varieties have been constantly integrated into the gene pool of Qinba area tea. A low level of genetic diversity was observed; this is shown by an average coefficient of genetic similarity of 0.74."/>

    <meta name="description" content="Qinba area has a long history of tea planting and is a northernmost region in China where Camellia sinensis L. is grown. In order to provide basic data for selection and optimization of molecular markers of tea plants. 118 markers, including 40 EST-SSR, 40 SRAP and 38 SCoT markers were used to evaluate the genetic diversity of 50 tea plant (Camellia sinensis.) samples collected from Qinb. tea germplasm, assess population structure. In this study, a total of 414 alleles were obtained using 38 pairs of SCoT primers, with an average of 10.89 alleles per primer. The percentage of polymorphic bands (PPB), polymorphism information content (PIC), resolving power (Rp), effective multiplex ratio (EMR), average band informativeness (Ibav), and marker index (MI) were 96.14%, 0.79, 6.71, 10.47, 0.58, and 6.07 respectively. 338 alleles were amplified via 40 pairs of SRAP (8.45 per primer), with PPB, PIC, Rp, EMR, Ibav, and MI values of 89.35%, 0.77, 5.11, 7.55, 0.61, and 4.61, respectively. Furthermore, 320 alleles have been detected using 40 EST-SSR primers (8.00 per primer), with PPB, PIC, Rp, EMR, Ibav, and MI values of 94.06%, 0.85, 4.48, 7.53, 0.56, and 4.22 respectively. These results indicated that SCoT markers had higher efficiency. Mantel test was used to analyze the genetic distance matrix generated by EST-SSRs, SRAPs and SCoTs. The results showed that the correlation between the genetic distance matrix based on EST-SSR and that based on SRAP was very small (r = 0.01), followed by SCoT and SRAP (r = 0.17), then by SCoT and EST-SSR (r = 0.19). The 50 tea samples were divided into two sub-populations using STRUCTURE, Neighbor-joining (NJ) method and principal component analyses (PCA). The results produced by STRUCTURE were completely consistent with the PCA analysis. Furthermore, there is no obvious relationship between the results produced using sub-populational and geographical data. Among the three types of markers, SCoT markers has many advantages in terms of NPB, PPB, Rp, EMR, and MI. Nevertheless, the values of PIC showed different trends, with the highest values generated with EST-SSR, followed by SCoT and SRAP. The average band informativeness showed similar trends. Correlation between genetic distances produced by three different molecular markers were very small, thus it is not recommended to use a single marker to evaluate genetic diversity and population structure. It is hence suggested that combining of different types of molecular markers should be used to evaluate the genetic diversity and population structure. It also seems crucial to screen out, for each type of molecular markers, core markers of Camellia sinensis. This study revealed that genes of exotic plant varieties have been constantly integrated into the gene pool of Qinba area tea. A low level of genetic diversity was observed; this is shown by an average coefficient of genetic similarity of 0.74."/>

    <meta name="journal_id" content="41065"/>

    <meta name="dc.title" content="Genetic diversity and structure of tea plant in Qinba area in China by three types of molecular markers"/>

    <meta name="dc.source" content="Hereditas 2018 155:1"/>

    <meta name="dc.format" content="text/html"/>

    <meta name="dc.publisher" content="BioMed Central"/>

    <meta name="dc.date" content="2018-05-09"/>

    <meta name="dc.type" content="OriginalPaper"/>

    <meta name="dc.language" content="En"/>

    <meta name="dc.copyright" content="2018 The Author(s)"/>

    <meta name="dc.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="dc.description" content="Qinba area has a long history of tea planting and is a northernmost region in China where Camellia sinensis L. is grown. In order to provide basic data for selection and optimization of molecular markers of tea plants. 118 markers, including 40 EST-SSR, 40 SRAP and 38 SCoT markers were used to evaluate the genetic diversity of 50 tea plant (Camellia sinensis.) samples collected from Qinb. tea germplasm, assess population structure. In this study, a total of 414 alleles were obtained using 38 pairs of SCoT primers, with an average of 10.89 alleles per primer. The percentage of polymorphic bands (PPB), polymorphism information content (PIC), resolving power (Rp), effective multiplex ratio (EMR), average band informativeness (Ibav), and marker index (MI) were 96.14%, 0.79, 6.71, 10.47, 0.58, and 6.07 respectively. 338 alleles were amplified via 40 pairs of SRAP (8.45 per primer), with PPB, PIC, Rp, EMR, Ibav, and MI values of 89.35%, 0.77, 5.11, 7.55, 0.61, and 4.61, respectively. Furthermore, 320 alleles have been detected using 40 EST-SSR primers (8.00 per primer), with PPB, PIC, Rp, EMR, Ibav, and MI values of 94.06%, 0.85, 4.48, 7.53, 0.56, and 4.22 respectively. These results indicated that SCoT markers had higher efficiency. Mantel test was used to analyze the genetic distance matrix generated by EST-SSRs, SRAPs and SCoTs. The results showed that the correlation between the genetic distance matrix based on EST-SSR and that based on SRAP was very small (r&#8201;=&#8201;0.01), followed by SCoT and SRAP (r&#8201;=&#8201;0.17), then by SCoT and EST-SSR (r&#8201;=&#8201;0.19). The 50 tea samples were divided into two sub-populations using STRUCTURE, Neighbor-joining (NJ) method and principal component analyses (PCA). The results produced by STRUCTURE were completely consistent with the PCA analysis. Furthermore, there is no obvious relationship between the results produced using sub-populational and geographical data. Among the three types of markers, SCoT markers has many advantages in terms of NPB, PPB, Rp, EMR, and MI. Nevertheless, the values of PIC showed different trends, with the highest values generated with EST-SSR, followed by SCoT and SRAP. The average band informativeness showed similar trends. Correlation between genetic distances produced by three different molecular markers were very small, thus it is not recommended to use a single marker to evaluate genetic diversity and population structure. It is hence suggested that combining of different types of molecular markers should be used to evaluate the genetic diversity and population structure. It also seems crucial to screen out, for each type of molecular markers, core markers of Camellia sinensis. This study revealed that genes of exotic plant varieties have been constantly integrated into the gene pool of Qinba area tea. A low level of genetic diversity was observed; this is shown by an average coefficient of genetic similarity of 0.74."/>

    <meta name="prism.issn" content="1601-5223"/>

    <meta name="prism.publicationName" content="Hereditas"/>

    <meta name="prism.publicationDate" content="2018-05-09"/>

    <meta name="prism.volume" content="155"/>

    <meta name="prism.number" content="1"/>

    <meta name="prism.section" content="OriginalPaper"/>

    <meta name="prism.startingPage" content="22"/>

    <meta name="prism.copyright" content="2018 The Author(s)"/>

    <meta name="prism.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="prism.url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0058-4"/>

    <meta name="prism.doi" content="doi:10.1186/s41065-018-0058-4"/>

    <meta name="citation_pdf_url" content="https://hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-018-0058-4"/>

    <meta name="citation_fulltext_html_url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0058-4"/>

    <meta name="citation_journal_title" content="Hereditas"/>

    <meta name="citation_publisher" content="BioMed Central"/>

    <meta name="citation_issn" content="1601-5223"/>

    <meta name="citation_title" content="Genetic diversity and structure of tea plant in Qinba area in China by three types of molecular markers"/>

    <meta name="citation_volume" content="155"/>

    <meta name="citation_issue" content="1"/>

    <meta name="citation_publication_date" content="2018/12"/>

    <meta name="citation_online_date" content="2018/05/09"/>

    <meta name="citation_firstpage" content="22"/>

    <meta name="citation_article_type" content="Research"/>

    <meta name="citation_fulltext_world_readable" content=""/>

    <meta name="dc.identifier" content="doi:10.1186/s41065-018-0058-4"/>

    <meta name="DOI" content="10.1186/s41065-018-0058-4"/>

    <meta name="citation_doi" content="10.1186/s41065-018-0058-4"/>

    <meta name="dc.creator" content="Yu Zhang"/>

    <meta name="dc.creator" content="Xiaojuan Zhang"/>

    <meta name="dc.creator" content="Xi Chen"/>

    <meta name="dc.creator" content="Wang Sun"/>

    <meta name="dc.creator" content="Jiao Li"/>

    <meta name="dc.subject" content="Animal Genetics and Genomics"/>

    <meta name="dc.subject" content="Evolutionary Biology"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Plant Breed; citation_title=Comparative analysis of genetic diversity among tea cultivars from China, Japan and Kenya by ISSR and EST-SSR; citation_author=MZ Yao, L Chen, CL Ma; citation_volume=7; citation_issue=5; citation_publication_date=2009; citation_pages=897-903; citation_id=CR1"/>

    <meta name="citation_reference" content="citation_journal_title=J Tea Sci; citation_title=The association analysis of phenotypic traits with EST-SSR markers in tea plants; citation_author=MZ Yao, TT Qiao, CL Ma; citation_volume=30; citation_issue=1; citation_publication_date=2010; citation_pages=45-51; citation_id=CR2"/>

    <meta name="citation_reference" content="citation_journal_title=Chin J Trop Crop; citation_title=Analysis of genetic diversity of tea plants by using EST-SSR and ISSR markers; citation_author=B Liu, X Sun, Y Li; citation_volume=30; citation_issue=11; citation_publication_date=2009; citation_pages=1577-1583; citation_id=CR3"/>

    <meta name="citation_reference" content="citation_journal_title=Acta Agron Sin; citation_title=EST-SSR genetic diversity and population structure of tea landraces and developed cultivars (lines) in Zhejiang Province, China; citation_author=TT Qiao, CL Ma, YH Zhou; citation_volume=36; citation_issue=5; citation_publication_date=2010; citation_pages=744-753; citation_doi=10.3724/SP.J.1006.2010.00744; citation_id=CR4"/>

    <meta name="citation_reference" content="citation_journal_title=Hunan Agric Sci; citation_title=Genetic diversity and genetic structure of 16 tea cultivars based on SSR markers; citation_author=SJ Li, X Wang, JH Duan, LJ Dong, SG Zhang; citation_volume=12; citation_issue=23; citation_publication_date=2011; citation_pages=6-9; citation_id=CR5"/>

    <meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=Sequence-related amplified polymorphism (SRAP), a new marker system based on a simple PCR reaction: its application to mapping and gene tagging in Brassica; citation_author=G Li, CF Quiros; citation_volume=103; citation_issue=2; citation_publication_date=2001; citation_pages=455-461; citation_doi=10.1007/s001220100570; citation_id=CR6"/>

    <meta name="citation_reference" content="citation_journal_title=Chin J Appl Ecol; citation_title=Genetic diversity of Camellia sinensis germplasm in Guangdong Province based on morphological parameters and SRAP markers; citation_author=CW Shen, ZX Ning, JA Huang; citation_volume=20; citation_issue=7; citation_publication_date=2009; citation_pages=1551-1558; citation_id=CR7"/>

    <meta name="citation_reference" content="citation_journal_title=J Nucl Agric Sci; citation_title=Analysis of genetic diversity of camellia sinensis germplasm in Guangdong Province by srap and issr markers; citation_author=CW Shen, JA Huang, SH Zhao, ZX Ning, JX Li, CY Zhao, D Chen; citation_volume=24; citation_issue=5; citation_publication_date=2010; citation_pages=948-955; citation_id=CR8"/>

    <meta name="citation_reference" content="citation_journal_title=Guizhou Agric Sci; citation_title=Wu YS, Xu JY, Chen H, Wu Q. Genetic diversity and relationship of 30 tea plant germplasms in Sichuan revealed by SRAP marker; citation_author=CY Xi, Q Tang; citation_volume=41; citation_issue=2; citation_publication_date=2013; citation_pages=6-9; citation_id=CR9"/>

    <meta name="citation_reference" content="citation_journal_title=J Tea Sci; citation_title=Comparison of parents identification for tea variety based on SSR, SRAP and ISSR markers; citation_author=Z Liu, Y Zhao, PD Yang, Y Chen, J Ning, Y Yang; citation_volume=34; citation_issue=6; citation_publication_date=2014; citation_pages=617-624; citation_id=CR10"/>

    <meta name="citation_reference" content="citation_journal_title=Acta Bot Bor-Occid Sin; citation_title=Genetic diversity of capsicum frutescens in China as revealed by SRAP and SSR markers; citation_author=XJ Chen, KH Zhou, HX Zong, R Fang; citation_volume=32; citation_issue=11; citation_publication_date=2012; citation_pages=2201-2205; citation_id=CR11"/>

    <meta name="citation_reference" content="citation_journal_title=J Tea Sci; citation_title=SRAP marker analysis of genetic diversity and relationship in Wuyi rock tea Germplasm resources; citation_author=FG Xia, XW Zhong, F Wu; citation_volume=37; citation_issue=1; citation_publication_date=2017; citation_pages=78-85; citation_id=CR12"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Mol Biol Rep; citation_title=Start codon targeted (SCoT) polymorphism:a simple, novel DNA marker technique for generating gene-targeted markers in plants; citation_author=C Bertrand, J David, CY Mackill; citation_volume=27; citation_issue=1; citation_publication_date=2009; citation_pages=86-93; citation_doi=10.1007/s11105-008-0060-5; citation_id=CR13"/>

    <meta name="citation_reference" content="citation_journal_title=J Plant Genet Resour; citation_title=Application of SCoT molecular marker in construction of molecular genetic linkage map of saccharum spontaneum L; citation_author=T Luo, HX Yang, HF Cen, XH Liu, YJ Gao, WX Duan; citation_volume=14; citation_issue=4; citation_publication_date=2013; citation_pages=704-710; citation_id=CR14"/>

    <meta name="citation_reference" content="citation_journal_title=Acta Pratacult Sin; citation_title=Analysis of genetic diversity in a cocksfoot (Dactylis glomerata) variety using SCoT markers; citation_author=LF Jiang, XQ Zhang, LK Huang, X Ma, DF Yan, Q Hu; citation_volume=23; citation_issue=1; citation_publication_date=2014; citation_pages=229-238; citation_id=CR15"/>

    <meta name="citation_reference" content="Luo C. Study on SCoT marker and analysis on genes of stress-related and important flowering time in mango. Nanning Guangxi: Guangxi University; 2012."/>

    <meta name="citation_reference" content="citation_journal_title=Acta Agron Sin; citation_title=Application of SCoT molecular marker in genus arachis; citation_author=FQ Xiong, J Jiang, RC Zhong, ZQ Han, LQ He, Z Li, WJ Zhuang, RH Tang; citation_volume=36; citation_issue=12; citation_publication_date=2010; citation_pages=2055-2061; citation_doi=10.3724/SP.J.1006.2010.02055; citation_id=CR17"/>

    <meta name="citation_reference" content="Chen S. Study on genetic diversity and smut resistance evaluation of sugarcane parents. Guangzhou: South China Agricultural University; 2016."/>

    <meta name="citation_reference" content="citation_journal_title=Guihaia; citation_title=Genetic diversity of podocarpus by SCoT markers; citation_author=YL Wei, XH He, C Luo, H Chen; citation_volume=32; citation_issue=1; citation_publication_date=2012; citation_pages=90-93; citation_id=CR19"/>

    <meta name="citation_reference" content="citation_journal_title=Acta Agriculturae Boreali-Sinica; citation_title=Orthogonal optimization of SCoT-PCR system and primer screening of tree peony; citation_author=XG Hou, J Wang, T Jia, YQ Zhang, J Hou, JJ Li; citation_volume=26; citation_issue=5; citation_publication_date=2011; citation_pages=92-96; citation_id=CR20"/>

    <meta name="citation_reference" content="citation_journal_title=Cancer Res; citation_title=The detection of disease clustering and a generalized regression approach; citation_author=N Mantel; citation_volume=27; citation_issue=2; citation_publication_date=1967; citation_pages=209-220; citation_id=CR21"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Ecol; citation_title=Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study; citation_author=G Evanno, S Regnaut, J Goudet; citation_volume=14; citation_issue=8; citation_publication_date=2005; citation_pages=2611-2620; citation_doi=10.1111/j.1365-294X.2005.02553.x; citation_id=CR22"/>

    <meta name="citation_reference" content="citation_title=NTSYS-pc - numerical taxonomy and multivariate analysis System; citation_publication_date=1998; citation_id=CR23; citation_author=FJ Rohlf"/>

    <meta name="citation_reference" content="citation_title=DNA fingerprinting and plant variety protection; citation_inbook_title=Genome mapping in plants; citation_publication_date=1996; citation_pages=95-110; citation_id=CR24; citation_author=S Smith; citation_author=T Helentjaris; citation_publisher=Landes Company"/>

    <meta name="citation_reference" content="citation_journal_title=Am J Hum Genet; citation_title=Construction of a genetic linkage map in man using restriction fragment length polymorphisms; citation_author=D Botstein, RL White, M Skolnick; citation_volume=32; citation_issue=3; citation_publication_date=1980; citation_pages=314-331; citation_id=CR25"/>

    <meta name="citation_author" content="Yu Zhang"/>

    <meta name="citation_author_institution" content="School of Biological Science and Engineering, Shaanxi University of Technology, Hanzhong, People&#8217;s Republic of China"/>

    <meta name="citation_author" content="Xiaojuan Zhang"/>

    <meta name="citation_author_institution" content="School of Biological Science and Engineering, Shaanxi University of Technology, Hanzhong, People&#8217;s Republic of China"/>

    <meta name="citation_author" content="Xi Chen"/>

    <meta name="citation_author_institution" content="Qili Food and Drug Supervision and Management Institute, Hanzhong, People&#8217;s Republic of China"/>

    <meta name="citation_author" content="Wang Sun"/>

    <meta name="citation_author_institution" content="School of Biological Science and Engineering, Shaanxi University of Technology, Hanzhong, People&#8217;s Republic of China"/>

    <meta name="citation_author" content="Jiao Li"/>

    <meta name="citation_author_institution" content="Hanzhong Institute of Agricultural Sciences, Hanzhong, People&#8217;s Republic of China"/>



        <meta charset="UTF-8"/>
        <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
        <meta name="viewport" content="width=device-width, initial-scale=1"/>

        <!-- For IE 9 and below -->
<!--[if IE]><link rel="shortcut icon" href=/static/images/favicons//favicon.ico><![endif]-->
<!-- Touch Icons - iOS and Android 2.1+ -->
<link rel="apple-touch-icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>
<!-- Firefox, Chrome, Safari, IE 11+ and Opera -->
<link rel="icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>


        <script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script>
        <script>(function(d){if (sessionStorage.fontsLoaded){d.style.setProperty('--font-family-sans', 'Europa');d.style.setProperty('--font-family-serif', 'Noto Serif')}})(document.documentElement);</script>

        <link rel="stylesheet" media="screen" href=/static/styles/basic/basic-a31ca85bc2.css>
<link rel="stylesheet" media="print" href=/static/styles/bmc/print-3a6240f456.css>

<link rel="stylesheet" class="js-ctm" href=/static/styles/bmc/main-84414d3d47.css
      media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">

        

        <script src="/static/js/app-bundle-8d61e6ef37.js" defer></script>
        
        <script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script>
        
    <script type="text/javascript">
        config = {
            env: 'live',
            site: 'hereditasjournal.biomedcentral.com',
            siteWithPath: 'hereditasjournal.biomedcentral.com' + window.location.pathname,
            twitterHashtag: '',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/s41065-018-0058-4',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central'
        };
    </script>

        
    <script type="text/javascript">
        dataLayer = [{"content":{"article":{"doi":"10.1186/s41065-018-0058-4","articleType":"Research","peerReviewType":"Closed","supplement":null,"keywords":"Camellia sinensis;Marker efficiency;Correlation coefficient;Genetic diversity;Population structure"},"contentInfo":{"imprint":"BioMed Central","title":"Genetic diversity and structure of tea plant in Qinba area in China by three types of molecular markers","publishedAt":1525824000000,"publishedAtDate":"2018-05-09","author":["Yu Zhang","Xiaojuan Zhang","Xi Chen","Wang Sun","Jiao Li"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":"CC BY + CC0","openAccess":true},"environment":"live"},"journal":{"siteKey":"hereditasjournal.biomedcentral.com","volume":"155","issue":"1","title":"Hereditas","type":"AJs Life","journalID":41065,"gaCode":"UA-62066850-11","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Research","publishingSegment":"Genetics & Heredity (Life)"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}}}]

    </script>

        <script src="https://cdn.cookielaw.org/consent/7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script>
        
    
        
    

        
    
        
            
                <!-- Google Tag Manager -->
                <script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script>
                <!-- End Google Tag Manager -->
            
        
    


        
    
    <link rel="canonical" href="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0058-4"/>
    

        
        
        <meta property="og:url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0058-4"/>
        <meta property="og:type" content="article"/>
        <meta property="og:site_name" content="Hereditas"/>
        <meta property="og:title" content="Genetic diversity and structure of tea plant in Qinba area in China by three types of molecular markers"/>
        <meta property="og:description" content="Qinba area has a long history of tea planting and is a northernmost region in China where Camellia sinensis L. is grown. In order to provide basic data for selection and optimization of molecular markers of tea plants. 118 markers, including 40 EST-SSR, 40 SRAP and 38 SCoT markers were used to evaluate the genetic diversity of 50 tea plant (Camellia sinensis.) samples collected from Qinb. tea germplasm, assess population structure. In this study, a total of 414 alleles were obtained using 38 pairs of SCoT primers, with an average of 10.89 alleles per primer. The percentage of polymorphic bands (PPB), polymorphism information content (PIC), resolving power (Rp), effective multiplex ratio (EMR), average band informativeness (Ibav), and marker index (MI) were 96.14%, 0.79, 6.71, 10.47, 0.58, and 6.07 respectively. 338 alleles were amplified via 40 pairs of SRAP (8.45 per primer), with PPB, PIC, Rp, EMR, Ibav, and MI values of 89.35%, 0.77, 5.11, 7.55, 0.61, and 4.61, respectively. Furthermore, 320 alleles have been detected using 40 EST-SSR primers (8.00 per primer), with PPB, PIC, Rp, EMR, Ibav, and MI values of 94.06%, 0.85, 4.48, 7.53, 0.56, and 4.22 respectively. These results indicated that SCoT markers had higher efficiency. Mantel test was used to analyze the genetic distance matrix generated by EST-SSRs, SRAPs and SCoTs. The results showed that the correlation between the genetic distance matrix based on EST-SSR and that based on SRAP was very small (r &#x3D; 0.01), followed by SCoT and SRAP (r &#x3D; 0.17), then by SCoT and EST-SSR (r &#x3D; 0.19). The 50 tea samples were divided into two sub-populations using STRUCTURE, Neighbor-joining (NJ) method and principal component analyses (PCA). The results produced by STRUCTURE were completely consistent with the PCA analysis. Furthermore, there is no obvious relationship between the results produced using sub-populational and geographical data. Among the three types of markers, SCoT markers has many advantages in terms of NPB, PPB, Rp, EMR, and MI. Nevertheless, the values of PIC showed different trends, with the highest values generated with EST-SSR, followed by SCoT and SRAP. The average band informativeness showed similar trends. Correlation between genetic distances produced by three different molecular markers were very small, thus it is not recommended to use a single marker to evaluate genetic diversity and population structure. It is hence suggested that combining of different types of molecular markers should be used to evaluate the genetic diversity and population structure. It also seems crucial to screen out, for each type of molecular markers, core markers of Camellia sinensis. This study revealed that genes of exotic plant varieties have been constantly integrated into the gene pool of Qinba area tea. A low level of genetic diversity was observed; this is shown by an average coefficient of genetic similarity of 0.74."/>
        <meta property="og:image" content="https://images.springer.com/sgw/journals/medium/41065.jpg"/>
    
    </head>

    <body class="journal journal-fulltext"
          
          >
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK"
                            height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><feColorMatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"/></filter></defs><symbol id="icon-arrow-down" viewBox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"/></symbol><symbol id="icon-arrow-left-bullet" viewBox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"/></symbol><symbol id="icon-arrow-right-bullet" viewBox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"/></symbol><symbol id="icon-arrow-right" viewBox="0 0 32 64"><path d="M0 0v64l32-32z"/></symbol><symbol id="icon-close" viewBox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"/></symbol><symbol id="icon-community" viewBox="0 0 60.643 31.01"><mask maskUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><path fill="#FFF" d="M0 31.01h60.643V0H0z" filter="url(#a)"/></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"/></symbol><symbol id="icon-download" viewBox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"/></symbol><symbol id="icon-ext-link" viewBox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"/><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"/><path fill="#676767" d="M9.7 0L16 6.2V0z"/></symbol><symbol id="icon-info-bordered" viewBox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"/><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"/></symbol><symbol id="icon-opr" viewBox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"/><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"/></symbol><symbol id="icon-quotation" viewBox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"/></symbol><symbol id="icon-remove" viewBox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"/><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"/></symbol><symbol id="icon-ribbon" viewBox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"/></symbol><symbol id="icon-rss" viewBox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"/><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"/><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"/></g></symbol><symbol id="icon-search" viewBox="0 0 16 16"><path d="M6.276 12.257C2.815 12.257 0 9.508 0 6.128 0 2.749 2.815 0 6.276 0c3.462 0 6.277 2.749 6.277 6.128.002 3.38-2.815 6.129-6.277 6.129zm0-10.772c-2.622 0-4.755 2.083-4.755 4.643 0 2.562 2.133 4.644 4.755 4.644 2.623 0 4.756-2.082 4.756-4.644 0-2.56-2.134-4.643-4.756-4.643z"/><path d="M16 13.751L13.641 16l-4.875-4.82 2.36-2.248L16 13.751z"/></symbol><symbol id="icon-speech-bubble" viewBox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"/></symbol><symbol id="icon-error" viewBox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"/></symbol><symbol id="icon-info" viewBox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"/></symbol><symbol id="icon-success" viewBox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"/></symbol><symbol id="icon-warning" viewBox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"/></symbol><symbol id="icon-facebook-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"/></symbol><symbol id="icon-facebook" viewBox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"/><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"/></symbol><symbol id="icon-gplus-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M480.5 273.117c.123.076.26.185.416.334.146.152.289.343.432.562.135.211.258.459.359.746.086.286.129.619.129 1.004-.012.7-.164 1.262-.465 1.683a5.374 5.374 0 0 1-.461.562c-.186.176-.375.35-.58.527a2.65 2.65 0 0 0-.322.41.908.908 0 0 0-.176.553c0 .205.059.375.18.508.102.129.199.238.299.336l.662.545a6.48 6.48 0 0 1 1.086 1.119c.293.415.445.955.459 1.625 0 .949-.42 1.789-1.256 2.521-.869.76-2.123 1.15-3.762 1.177-1.373-.015-2.4-.308-3.074-.879-.682-.534-1.023-1.172-1.023-1.917 0-.359.111-.766.336-1.211.215-.443.602-.837 1.172-1.174a6.126 6.126 0 0 1 1.998-.725 15.415 15.415 0 0 1 1.723-.173 3.996 3.996 0 0 1-.377-.596 1.347 1.347 0 0 1-.193-.74c0-.174.023-.317.074-.433a8.83 8.83 0 0 1 .121-.338 5.973 5.973 0 0 1-.633.035c-1.043-.012-1.838-.339-2.381-.979-.574-.603-.861-1.3-.861-2.097 0-.961.404-1.832 1.217-2.616.559-.46 1.135-.757 1.736-.898a8.494 8.494 0 0 1 1.67-.182h3.916l-1.209.704h-1.207v.005l-.005.002zm.754 11.276c0-.498-.162-.932-.484-1.302-.346-.348-.881-.778-1.609-1.29a4.045 4.045 0 0 0-.439-.02c-.102-.016-.354 0-.768.039a8.141 8.141 0 0 0-1.244.275c-.1.036-.24.096-.42.17a2.713 2.713 0 0 0-.553.355 1.985 1.985 0 0 0-.459.592c-.145.25-.215.551-.215.901 0 .687.311 1.253.932 1.698.596.444 1.404.68 2.432.688.928-.014 1.629-.217 2.117-.612.476-.385.71-.886.71-1.494zm-2.729-5.743a1.843 1.843 0 0 0 1.289-.556c.164-.249.271-.506.318-.767a7.32 7.32 0 0 0 .041-.659c0-.772-.195-1.552-.594-2.336a2.611 2.611 0 0 0-.73-.921 1.894 1.894 0 0 0-1.064-.352 1.723 1.723 0 0 0-1.324.64c-.299.437-.438.92-.428 1.457 0 .709.209 1.444.623 2.211.201.356.459.659.775.912.323.248.686.371 1.094.371zm9.698.077h-2.055v-2.056h-.996v2.056h-2.053v.998h2.053v2.052h.996v-2.052h2.055z"/></symbol><symbol id="icon-gplus" viewBox="0 0 24 24"><circle fill="#D84A36" cx="12" cy="12" r="12"/><g fill="#FFF"><path d="M12.012 12.111c-.304-.33-.07-.842.155-1.078.226-.241.639-.56.829-.74.193-.184.66-.605.674-1.868 0 0 .112-1.139-1.127-2.06h.977l1.038-.593h-3.507s-2.083-.067-3.085 1.207c-1 1.276-.966 2.871.114 3.77 1.08.898 2.371.648 2.371.648s-.197.422-.163.775c.036.353.421.849.504.965 0 0-2.386-.011-3.551 1.033-1.165 1.053-1.29 3.097 1.021 3.803 2.312.707 5.303-.055 5.906-2.277 0 0 .262-.908-.189-1.683-.459-.775-1.663-1.571-1.967-1.902zm-2.555-1.83c-.744-1-1.308-2.953 0-3.783 0 0 .955-.579 1.801.266.846.846 1.082 2.277.928 3.003-.15.727-.563.897-.739 1.008-.185.111-1.249.504-1.99-.494zm3.668 5.481c-.035.393-.083 1.254-1.491 1.664-1.409.408-3.826-.264-3.757-1.914 0 0 .006-.698.449-1.089.445-.396 1.094-.794 3.029-.819 0 0 .643.473 1.072.809.427.332.733.952.698 1.349z"/><path d="M16.323 11.055V9.803h-.752v1.252h-1.255v.748h1.255v1.256h.752v-1.256h1.248v-.748z"/></g></symbol><symbol id="icon-linkedin" viewBox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"/><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"/><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"/></g></symbol><symbol id="icon-reddit" viewBox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"/><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"/><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"/><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"/><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"/><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"/><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"/><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"/></symbol><symbol id="icon-twitter-bordered" viewBox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"/><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/></symbol><symbol id="icon-twitter" viewBox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"/><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"/></symbol><symbol id="icon-weibo-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"/><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"/><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"/></symbol><symbol id="icon-weibo" viewBox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"/><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"/><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"/><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"/><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"/><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"/></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner c-banner--info">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner" >
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1"
                 data-gpt-unitpath="/270604982/bmc/hereditasjournal/articles"
                 data-gpt-sizes="728x90,970x90"
                 data-gpt-targeting="pos=LB1;doi=10.1186/s41065-018-0058-4;kwrd=Camellia sinensis,Marker efficiency,Correlation coefficient,Genetic diversity,Population structure;pmc=L32030,L21001;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-018-0058-4&amp;kwrd=Camellia sinensis,Marker efficiency,Correlation coefficient,Genetic diversity,Population structure&amp;pmc=L32030,L21001&amp;">
                        <img data-test="gpt-advert-fallback-img"
                             src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-018-0058-4&amp;kwrd=Camellia sinensis,Marker efficiency,Correlation coefficient,Genetic diversity,Population structure&amp;pmc=L32030,L21001&amp;"
                             alt="Advertisement"
                             width="728"
                             height="90">
                    </a>
                </noscript>
            </div>
        </div>
    </div>

        
         
    <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>


<header class="c-popup-search ctx-publisher-header" id="top" data-test="publisher-header">
    <nav class="c-navbar">
        <div class="u-container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization" />
                <div class="c-logo">
    <a href="https://www.biomedcentral.com" itemprop="url">
        <img src=/static/images/bmc/logos/logo-bmc-white-aj-e3a70ae848.svg alt="BMC" itemprop="logo">
        <div class="c-logo__strapline">
            <img src=/static/images/bmc/logos/logo-bmc-white-strapline-sn-e7b1d6e67f.svg alt="Part of Springer Nature">
        </div>
    </a>
</div>
            </div>
            <button
                type="button"
                class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg"
                id="publisher-header-menu-button"
                data-toggle="collapse"
                data-test="publisher-header-menu-toggle"
                data-toggle-linked="publisher-header"
                data-target="publisher-header-search publisher-header-menu"
                aria-controls="publisher-header-menu publisher-header-search"
                aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


                <ul class="c-navbar__nav">
                    <li class="c-navbar__item">
                        <a data-replace="true"
                           data-placeholder="/placeholder/v1/username"
                           class="c-navbar__link"
                           href="https://www.biomedcentral.com/login"
                           data-test="login-link"
                           data-track="click"
                           data-track-category="Account"
                           data-track-action="Click Login Link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                    <li class="c-navbar__item">
                        <button
                            type="button"
                            class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg"
                            data-toggle="collapse"
                            data-test="header-search-button"
                            data-target="publisher-header-search publisher-header-menu publisher-header-menu-button"
                            aria-controls="publisher-header-search publisher-header-menu"
                            aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true">
                                    <use xlink:href="#icon-search"></use>
                                </svg>
                            </span>
                        </button>
                    </li>
                </ul>
            </div>
        </div>
    </nav>
    <div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field c-form-field--inline" method="GET" action="//www.biomedcentral.com/search" data-track="submit"
        data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label data-track-label="" data-test="global-search">
        <label for="publisherSearch" class="u-visually-hidden">Search all BMC articles</label>
        <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text" value="" placeholder="Search all BMC articles"/>
        <button class="c-button c-button--secondary" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true">
        <use xlink:href="#icon-search"></use>
    </svg>
</button>


        <input type="hidden" name="searchType" value="publisherSearch"/>
    </form>
</div>

            </div>
        </div>
    </div>
</header>


        
            <header class="c-journal-header c-journal-header--hereditas ctx-journal-header">
                <div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="/">
    
        
    
    <span class="c-journal-title__text ">Hereditas</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="u-container">
                        
                            <button type="button"
                                    class="c-navbar__link c-dropdown u-hidden-at-lg"
                                    data-toggle="collapse"
                                    data-test="journal-header-menu"
                                    data-toggle-linked="journal-header"
                                    data-target="journal-menu"
                                    aria-controls="journal-menu"
                                    aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav">
                                    <ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/'>Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/about'>About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href='/articles'>Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/submission-guidelines'>Submission Guidelines</a>
                                            </li>
                                        
                                    </ul>
                                </nav>
                            </div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--ocean-light-blue">
                    
                </div>
            </header>
            
        

        <div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-content-layout c-content-layout--fulltext" data-component="sticky-container">
                <div class="c-content-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3><ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc=""><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results and discussion">Results and discussion</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec9" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Conclusions">Conclusions</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec10" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Methods">Methods</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li></ol></nav>
                </div>

                <main class="c-content-layout__main">
                    <article itemscope itemtype="http://schema.org/ScholarlyArticle">
                        
                        
    <div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
            <li class="c-article-identifiers__item" data-test="article-category">Research</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
    </div>

                        <div class="FulltextWrapper"><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">Genetic diversity and structure of tea plant in Qinba area in China by three types of molecular markers</h1></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true"><ul class="u-listReset"><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Yu&nbsp;Zhang</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup><a href="mailto:yuzhang20160315@outlook.com" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Xiaojuan&nbsp;Zhang</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Xi&nbsp;Chen</span><sup><a href="#Aff2" class="AffiliationID">2</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Wang&nbsp;Sun</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup> and </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Jiao&nbsp;Li</span><sup><a href="#Aff3" class="AffiliationID">3</a></sup></li></ul></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation"><div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">Hereditas</span></span><span class="ArticleCitation_Year">2018</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">155</strong>:22</span></span></div><p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/s41065-018-0058-4">https://doi.org/10.1186/s41065-018-0058-4</a></p><p class="Copyright">© &nbsp;The Author(s)&nbsp;2018</p></div></div><ul class="ArticleHistory u-reset-list"><li class="History HistoryReceived"><strong>Received: </strong><span>20&nbsp;December&nbsp;2017</span></li><li class="History HistoryAccepted"><strong>Accepted: </strong><span>26&nbsp;March&nbsp;2018</span></li><li class="History HistoryOnlineDate"><strong>Published: </strong><span itemprop="datePublished">9&nbsp;May&nbsp;2018</span></li></ul></div>
                        
                        
                        
    

                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2><div class="c-section__content" id="Abstract"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p id="Par1" class="Para">Qinba area has a long history of tea planting and is a northernmost region in China where <em xmlns="" class="EmphasisTypeItalic">Camellia sinensis</em> L. is grown. In order to provide basic data for selection and optimization of molecular markers of tea plants. 118 markers, including 40 EST-SSR, 40 SRAP and 38 SCoT markers were used to evaluate the genetic diversity of 50 tea plant (<em xmlns="" class="EmphasisTypeItalic">Camellia sinensis.</em>) samples collected from Qinb. tea germplasm, assess population structure.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Results</h3>
                <p id="Par2" class="Para">In this study, a total of 414 alleles were obtained using 38 pairs of SCoT primers, with an average of 10.89 alleles per primer. The percentage of polymorphic bands (PPB), polymorphism information content (PIC), resolving power (Rp), effective multiplex ratio (EMR), average band informativeness (Ib<sub>av</sub>), and marker index (MI) were 96.14%, 0.79, 6.71, 10.47, 0.58, and 6.07 respectively. 338 alleles were amplified via 40 pairs of SRAP (8.45 per primer), with PPB, PIC, Rp, EMR, Ib<sub>av,</sub> and MI values of 89.35%, 0.77, 5.11, 7.55, 0.61, and 4.61, respectively. Furthermore, 320 alleles have been detected using 40 EST-SSR primers (8.00 per primer), with PPB, PIC, Rp, EMR, Ib<sub>av</sub>, and MI values of 94.06%, 0.85, 4.48, 7.53, 0.56, and 4.22 respectively. These results indicated that SCoT markers had higher efficiency.</p>
                <p id="Par3" class="Para">Mantel test was used to analyze the genetic distance matrix generated by EST-SSRs, SRAPs and SCoTs. The results showed that the correlation between the genetic distance matrix based on EST-SSR and that based on SRAP was very small (<em xmlns="" class="EmphasisTypeItalic">r</em> = 0.01), followed by SCoT and SRAP (<em xmlns="" class="EmphasisTypeItalic">r</em> = 0.17), then by SCoT and EST-SSR (<em xmlns="" class="EmphasisTypeItalic">r</em> = 0.19).</p>
                <p id="Par4" class="Para">The 50 tea samples were divided into two sub-populations using STRUCTURE, Neighbor-joining (NJ) method and principal component analyses (PCA). The results produced by STRUCTURE were completely consistent with the PCA analysis. Furthermore, there is no obvious relationship between the results produced using sub-populational and geographical data.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusion</h3>
                <p id="Par5" class="Para">Among the three types of markers, SCoT markers has many advantages in terms of NPB, PPB, Rp, EMR, and MI. Nevertheless, the values of PIC showed different trends, with the highest values generated with EST-SSR, followed by SCoT and SRAP. The average band informativeness showed similar trends. Correlation between genetic distances produced by three different molecular markers were very small, thus it is not recommended to use a single marker to evaluate genetic diversity and population structure. It is hence suggested that combining of different types of molecular markers should be used to evaluate the genetic diversity and population structure. It also seems crucial to screen out, for each type of molecular markers, core markers of <em xmlns="" class="EmphasisTypeItalic">Camellia sinensis</em>. This study revealed that genes of exotic plant varieties have been constantly integrated into the gene pool of Qinba area tea. A low level of genetic diversity was observed; this is shown by an average coefficient of genetic similarity of 0.74.</p>
              </div></div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords"><li class="c-keywords__item" itemprop="about">
                <em xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="EmphasisTypeItalic">Camellia sinensis</em>
              </li><li class="c-keywords__item" itemprop="about">Marker efficiency</li><li class="c-keywords__item" itemprop="about">Correlation coefficient</li><li class="c-keywords__item" itemprop="about">Genetic diversity</li><li class="c-keywords__item" itemprop="about">Population structure</li></ul></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par19" class="Para">Evaluation of genetic diversity and population structure has significant implications for genetic improvement in plant breeding. It has been well established that the genetic basis of biological organisms is concealed within the genome sequence, and that base-pair substitution, insertion, deletion, and other alterations can lead to genetic diversity; the diversity of organisms are manifested through phenotypic, chromosomal and proteomic differences. DNA molecular markers, having stable performance, high polymorphism and other properties, are increasingly employed in taxonomical, genetic evolutionary, breeding, and cloning studies. The use of different molecular markers and different primers for a same marker may result in amplification of distinct regions of the genome. Theoretically, higher numbers of polymorphic markers used are associated with wider amplified regions that covers the entire genome and more accurate results.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par20" class="Para">EST-SSR (Expressed Sequence Tag-Simple Sequence Repeat) molecular markers have been widely used with many species and for many applications, such as genetic linkage mapping, comparative mapping, and evaluation of genetic diversity [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>–<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>]. SRAP (Sequence related amplified polymorphism) was first used on <em xmlns="" class="EmphasisTypeItalic">Brassica</em> in 2001 by Li G [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>]. The genetic diversity and population structure analysis of <em xmlns="" class="EmphasisTypeItalic">Camellia sinensis</em> by SRAP [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>–<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>] have already been reported. SCoT (Start codon targeted polymorphism) marker was designed according to the Kozak sequence pattern and was developed after the discovery of the conservativeness of the initiation codon ATG (+ 1, + 2, + 3) flanking sequences, in which the positions + 4, + 7, + 8, and + 9 are occupied by nucleotides G, A, C, and C, respectively. These seven nucleotides are generally conserved. At positions − 3, − 6, and − 9, G is the usual nucleotide. Primers can therefore be designed according to the conservativeness of the initiation sequence SCoT marker allows single primer amplification of the region between two genes. Bertrand et al. first applied this marker on <em xmlns="" class="EmphasisTypeItalic">Oryza sativa</em> [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>]. Lately, SCoT molecular marker has been used to access the genetic diversity of plant species such as <em xmlns="" class="EmphasisTypeItalic">Saccharum spontaneum L</em> [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>], <em xmlns="" class="EmphasisTypeItalic">Dactylis glomerata</em> [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>], <em xmlns="" class="EmphasisTypeItalic">Mangifera indica</em> [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>], <em xmlns="" class="EmphasisTypeItalic">Arachis hypogaea</em> [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>], <em xmlns="" class="EmphasisTypeItalic">Saccharum officinarum</em> [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>], <em xmlns="" class="EmphasisTypeItalic">Podocarpus macrophyllus</em> [<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>] and <em xmlns="" class="EmphasisTypeItalic">Paeonia suffruticosa</em> [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>]. Nevertheless, no similar study has been conducted on <em xmlns="" class="EmphasisTypeItalic">Camellia sinensis</em>. Tea plant is an allogamous species; theoretically, after prolonged spontaneous hybridization, the genetic background of tea plant should be increasingly complex.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par21" class="Para">China is one of the main sources of tea germplasms. Currently, there are 1,100,000&nbsp;ha of tea planting area, with different regions growing different types and different varieties of tea according to topographic, soil, and climatic characteristics. Xinan, Huanan, Jiangnan, and Jiangbei represent the four main districts of tea planting area in China. The Qinba area belongs to the Jiangbei district. In this research, 50 tea varieties, including those collected from different districts, common tea plant species, as well as local species in the Qinba area, were genotyped with EST-SSR, SRAP, and SCoT markers. Herein we constructed three types of molecular marker dataset which have important applications in diversity analysis, marker efficiency analysis, and correlation analysis that use these marker systems. Our study allowed the establishment of population structure, providing significant insights into the selection of molecular markers for tea plant breeding.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Resultsanddiscussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Resultsanddiscussion">Results and discussion</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Resultsanddiscussion" data-test="collapse-section"><section id="Sec3" class="Section2 RenderAsSection2">
                <h3 class="Heading">Marker efficiency analysis</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par22" class="Para">In this study, three types of molecular markers were used to differentiate tea plant accessions. A total of 1072 bands were produced using 118 primer pairs. 38 SCoT, 40 SRAP and 40 EST-SSR primers were selected for further studies according to the percentage of polymorphic bands (PPB), polymorphism information content (PIC) and the degree of clear band selected markers using six selected genotypes (Table&nbsp;<span class="InternalRef"><a href="#Tab1">1</a></span>). A total of 414, 338, and 320 bands were obtained using SCoT, SRAP and EST-SSR markers, respectively from the 50 test materials, which included 398, 302, and 301 polymorphic bands, with PPBs of 96.13%, 89.35%, and 94.06%. Comparisons of the three types of markers are shown in Table&nbsp;<span class="InternalRef"><a href="#Tab2">2</a></span>. SCoT markers have a higher marker efficiency and are excellent for the appraisal of polymorphic loci, except that its polymorphic information content is lower than that of EST-SSR.<figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 1</span>
                        <p xmlns="" class="SimplePara">Amplification results of EST-SSR, SRAP, and SCoT primers</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col char="."></col><col char="."></col><col char="."></col><col char="."></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Primer name</p></th><th><p xmlns="" class="SimplePara">Sequence(5′-3′)</p></th><th char="."><p xmlns="" class="SimplePara">Total number of bands(TNB)</p></th><th char="."><p xmlns="" class="SimplePara">The number of polymorphic bands(NPB)</p></th><th char="."><p xmlns="" class="SimplePara">Percentage of polymorphic bands(PPB)%</p></th><th char="."><p xmlns="" class="SimplePara">Polymorphism information content(PIC)</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT1</p></td><td><p xmlns="" class="SimplePara">CAACAATGGCTACCACCC</p></td><td char="."><p xmlns="" class="SimplePara">6</p></td><td char="."><p xmlns="" class="SimplePara">5</p></td><td char="."><p xmlns="" class="SimplePara">83.33</p></td><td char="."><p xmlns="" class="SimplePara">0.98</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT2</p></td><td><p xmlns="" class="SimplePara">CAACAATGGCTACCACCG</p></td><td char="."><p xmlns="" class="SimplePara">10</p></td><td char="."><p xmlns="" class="SimplePara">10</p></td><td char="."><p xmlns="" class="SimplePara">100</p></td><td char="."><p xmlns="" class="SimplePara">0.98</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT3</p></td><td><p xmlns="" class="SimplePara">CAACAATGGCTACCACCT</p></td><td char="."><p xmlns="" class="SimplePara">7</p></td><td char="."><p xmlns="" class="SimplePara">7</p></td><td char="."><p xmlns="" class="SimplePara">100</p></td><td char="."><p xmlns="" class="SimplePara">0.98</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT4</p></td><td><p xmlns="" class="SimplePara">CAACAATGGCTACCACGA</p></td><td char="."><p xmlns="" class="SimplePara">11</p></td><td char="."><p xmlns="" class="SimplePara">11</p></td><td char="."><p xmlns="" class="SimplePara">100</p></td><td char="."><p xmlns="" class="SimplePara">0.84</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT5</p></td><td><p xmlns="" class="SimplePara">CAACAATGGCTACCACGC</p></td><td char="."><p xmlns="" class="SimplePara">13</p></td><td char="."><p xmlns="" class="SimplePara">13</p></td><td char="."><p xmlns="" class="SimplePara">100</p></td><td char="."><p xmlns="" class="SimplePara">0.96</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT6</p></td><td><p xmlns="" class="SimplePara">CAACAATGGCTACCACGT</p></td><td char="."><p xmlns="" class="SimplePara">10</p></td><td char="."><p xmlns="" class="SimplePara">10</p></td><td char="."><p xmlns="" class="SimplePara">100</p></td><td char="."><p xmlns="" class="SimplePara">0.89</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT7</p></td><td><p xmlns="" class="SimplePara">CAACAATGGCTACCAGCA</p></td><td char="."><p xmlns="" class="SimplePara">13</p></td><td char="."><p xmlns="" class="SimplePara">12</p></td><td char="."><p xmlns="" class="SimplePara">92.31</p></td><td char="."><p xmlns="" class="SimplePara">0.88</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT8</p></td><td><p xmlns="" class="SimplePara">CAACAATGGCTACCAGCC</p></td><td char="."><p xmlns="" class="SimplePara">12</p></td><td char="."><p xmlns="" class="SimplePara">11</p></td><td char="."><p xmlns="" class="SimplePara">91.67</p></td><td char="."><p xmlns="" class="SimplePara">0.95</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT9</p></td><td><p xmlns="" class="SimplePara">ACGACATGGCGACCATCG</p></td><td char="."><p xmlns="" class="SimplePara">10</p></td><td char="."><p xmlns="" class="SimplePara">10</p></td><td char="."><p xmlns="" class="SimplePara">100</p></td><td char="."><p xmlns="" class="SimplePara">0.92</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT10</p></td><td><p xmlns="" class="SimplePara">ACGACATGGCGACCACGC</p></td><td char="."><p xmlns="" class="SimplePara">8</p></td><td char="."><p xmlns="" class="SimplePara">8</p></td><td char="."><p xmlns="" class="SimplePara">100</p></td><td char="."><p xmlns="" class="SimplePara">0.93</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT11</p></td><td><p xmlns="" class="SimplePara">ACCATGGCTACCACCGAC</p></td><td char="."><p xmlns="" class="SimplePara">9</p></td><td char="."><p xmlns="" class="SimplePara">9</p></td><td char="."><p xmlns="" class="SimplePara">100</p></td><td char="."><p xmlns="" class="SimplePara">0.87</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT12</p></td><td><p xmlns="" class="SimplePara">ACCATGGCTACCACCGCC</p></td><td char="."><p xmlns="" class="SimplePara">13</p></td><td char="."><p xmlns="" class="SimplePara">12</p></td><td char="."><p xmlns="" class="SimplePara">92.31</p></td><td char="."><p xmlns="" class="SimplePara">0.90</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT13</p></td><td><p xmlns="" class="SimplePara">ACCATGGCTACCACCGCG</p></td><td char="."><p xmlns="" class="SimplePara">12</p></td><td char="."><p xmlns="" class="SimplePara">10</p></td><td char="."><p xmlns="" class="SimplePara">83.33</p></td><td char="."><p xmlns="" class="SimplePara">0.87</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT14</p></td><td><p xmlns="" class="SimplePara">ACGACATGGCGACCCACA</p></td><td char="."><p xmlns="" class="SimplePara">13</p></td><td char="."><p xmlns="" class="SimplePara">12</p></td><td char="."><p xmlns="" class="SimplePara">92.31</p></td><td char="."><p xmlns="" class="SimplePara">0.91</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT15</p></td><td><p xmlns="" class="SimplePara">ACCATGGCTACCACCGGG</p></td><td char="."><p xmlns="" class="SimplePara">14</p></td><td char="."><p xmlns="" class="SimplePara">14</p></td><td char="."><p xmlns="" class="SimplePara">100</p></td><td char="."><p xmlns="" class="SimplePara">0.98</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT16</p></td><td><p xmlns="" class="SimplePara">CCATGGCTACCACCGCCA</p></td><td char="."><p xmlns="" class="SimplePara">10</p></td><td char="."><p xmlns="" class="SimplePara">9</p></td><td char="."><p xmlns="" class="SimplePara">90</p></td><td char="."><p xmlns="" class="SimplePara">0.91</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT17</p></td><td><p xmlns="" class="SimplePara">CCATGGCTACCACCGCAC</p></td><td char="."><p xmlns="" class="SimplePara">10</p></td><td char="."><p xmlns="" class="SimplePara">10</p></td><td char="."><p xmlns="" class="SimplePara">100</p></td><td char="."><p xmlns="" class="SimplePara">0.96</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT18</p></td><td><p xmlns="" class="SimplePara">CCATGGCTACCACCGCAG</p></td><td char="."><p xmlns="" class="SimplePara">6</p></td><td char="."><p xmlns="" class="SimplePara">6</p></td><td char="."><p xmlns="" class="SimplePara">100</p></td><td char="."><p xmlns="" class="SimplePara">0.94</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT19</p></td><td><p xmlns="" class="SimplePara">ACCATGGCTACCACCGCA</p></td><td char="."><p xmlns="" class="SimplePara">12</p></td><td char="."><p xmlns="" class="SimplePara">12</p></td><td char="."><p xmlns="" class="SimplePara">100</p></td><td char="."><p xmlns="" class="SimplePara">0.87</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT20</p></td><td><p xmlns="" class="SimplePara">CATGGCTACCACCGGCCC</p></td><td char="."><p xmlns="" class="SimplePara">12</p></td><td char="."><p xmlns="" class="SimplePara">12</p></td><td char="."><p xmlns="" class="SimplePara">100</p></td><td char="."><p xmlns="" class="SimplePara">0.95</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT21</p></td><td><p xmlns="" class="SimplePara">ACGACATGGCGACCAACT</p></td><td char="."><p xmlns="" class="SimplePara">9</p></td><td char="."><p xmlns="" class="SimplePara">9</p></td><td char="."><p xmlns="" class="SimplePara">100</p></td><td char="."><p xmlns="" class="SimplePara">0.93</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT22</p></td><td><p xmlns="" class="SimplePara">ACGACATGGCGACCAACC</p></td><td char="."><p xmlns="" class="SimplePara">11</p></td><td char="."><p xmlns="" class="SimplePara">11</p></td><td char="."><p xmlns="" class="SimplePara">100</p></td><td char="."><p xmlns="" class="SimplePara">0.95</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT23</p></td><td><p xmlns="" class="SimplePara">ACGACATGGCGACCATCC</p></td><td char="."><p xmlns="" class="SimplePara">6</p></td><td char="."><p xmlns="" class="SimplePara">6</p></td><td char="."><p xmlns="" class="SimplePara">100</p></td><td char="."><p xmlns="" class="SimplePara">0.77</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT24</p></td><td><p xmlns="" class="SimplePara">ACGACATGGCGACCACGG</p></td><td char="."><p xmlns="" class="SimplePara">10</p></td><td char="."><p xmlns="" class="SimplePara">10</p></td><td char="."><p xmlns="" class="SimplePara">100</p></td><td char="."><p xmlns="" class="SimplePara">0.95</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT25</p></td><td><p xmlns="" class="SimplePara">ACGACATGGCGACCACGT</p></td><td char="."><p xmlns="" class="SimplePara">13</p></td><td char="."><p xmlns="" class="SimplePara">12</p></td><td char="."><p xmlns="" class="SimplePara">92.31</p></td><td char="."><p xmlns="" class="SimplePara">0.89</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT26</p></td><td><p xmlns="" class="SimplePara">ACCATGGCTACCACCGCT</p></td><td char="."><p xmlns="" class="SimplePara">10</p></td><td char="."><p xmlns="" class="SimplePara">10</p></td><td char="."><p xmlns="" class="SimplePara">100</p></td><td char="."><p xmlns="" class="SimplePara">0.97</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT27</p></td><td><p xmlns="" class="SimplePara">ACCATGGCTACCACCGGA</p></td><td char="."><p xmlns="" class="SimplePara">9</p></td><td char="."><p xmlns="" class="SimplePara">9</p></td><td char="."><p xmlns="" class="SimplePara">100</p></td><td char="."><p xmlns="" class="SimplePara">0.90</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT28</p></td><td><p xmlns="" class="SimplePara">CCATGGCTACCACCGCCC</p></td><td char="."><p xmlns="" class="SimplePara">13</p></td><td char="."><p xmlns="" class="SimplePara">13</p></td><td char="."><p xmlns="" class="SimplePara">100</p></td><td char="."><p xmlns="" class="SimplePara">0.77</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT29</p></td><td><p xmlns="" class="SimplePara">CCATGGCTACCACCGCCG</p></td><td char="."><p xmlns="" class="SimplePara">13</p></td><td char="."><p xmlns="" class="SimplePara">11</p></td><td char="."><p xmlns="" class="SimplePara">100</p></td><td char="."><p xmlns="" class="SimplePara">0.89</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT30</p></td><td><p xmlns="" class="SimplePara">CCATGGCTACCACCGGCT</p></td><td char="."><p xmlns="" class="SimplePara">4</p></td><td char="."><p xmlns="" class="SimplePara">4</p></td><td char="."><p xmlns="" class="SimplePara">100</p></td><td char="."><p xmlns="" class="SimplePara">0.92</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT31</p></td><td><p xmlns="" class="SimplePara">CCATGGCTACCACCGCAA</p></td><td char="."><p xmlns="" class="SimplePara">10</p></td><td char="."><p xmlns="" class="SimplePara">10</p></td><td char="."><p xmlns="" class="SimplePara">100</p></td><td char="."><p xmlns="" class="SimplePara">0.89</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT32</p></td><td><p xmlns="" class="SimplePara">CCATGGCTACCACCGCAT</p></td><td char="."><p xmlns="" class="SimplePara">7</p></td><td char="."><p xmlns="" class="SimplePara">7</p></td><td char="."><p xmlns="" class="SimplePara">100</p></td><td char="."><p xmlns="" class="SimplePara">0.98</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT33</p></td><td><p xmlns="" class="SimplePara">CATGGCTACCACCGGCCA</p></td><td char="."><p xmlns="" class="SimplePara">17</p></td><td char="."><p xmlns="" class="SimplePara">17</p></td><td char="."><p xmlns="" class="SimplePara">100</p></td><td char="."><p xmlns="" class="SimplePara">0.87</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT34</p></td><td><p xmlns="" class="SimplePara">CCATGGCGACCACCGGCA</p></td><td char="."><p xmlns="" class="SimplePara">11</p></td><td char="."><p xmlns="" class="SimplePara">10</p></td><td char="."><p xmlns="" class="SimplePara">90.91</p></td><td char="."><p xmlns="" class="SimplePara">0.71</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT35</p></td><td><p xmlns="" class="SimplePara">CCATGGCGACCACCGGCG</p></td><td char="."><p xmlns="" class="SimplePara">17</p></td><td char="."><p xmlns="" class="SimplePara">15</p></td><td char="."><p xmlns="" class="SimplePara">88.24</p></td><td char="."><p xmlns="" class="SimplePara">0.81</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT36</p></td><td><p xmlns="" class="SimplePara">CCATGGCGACCACCGGCC</p></td><td char="."><p xmlns="" class="SimplePara">14</p></td><td char="."><p xmlns="" class="SimplePara">13</p></td><td char="."><p xmlns="" class="SimplePara">92.86</p></td><td char="."><p xmlns="" class="SimplePara">0.68</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT37</p></td><td><p xmlns="" class="SimplePara">CCATGGCGACCACCGCCG</p></td><td char="."><p xmlns="" class="SimplePara">16</p></td><td char="."><p xmlns="" class="SimplePara">16</p></td><td char="."><p xmlns="" class="SimplePara">100</p></td><td char="."><p xmlns="" class="SimplePara">0.74</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">SCoT38</p></td><td><p xmlns="" class="SimplePara">CCATGGTCACCACCGGCG</p></td><td char="."><p xmlns="" class="SimplePara">13</p></td><td char="."><p xmlns="" class="SimplePara">12</p></td><td char="."><p xmlns="" class="SimplePara">92.31</p></td><td char="."><p xmlns="" class="SimplePara">0.56</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP1</p></td><td><p xmlns="" class="SimplePara">Me1:TGAGTCCAAACCGGATA</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">8</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">8</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.79</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em6: GACTGCGTACGAATTCTA</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP2</p></td><td><p xmlns="" class="SimplePara">Me1:TGAGTCCAAACCGGATA</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">7</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">7</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.87</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em10:GACTGCGTACGAATTCCA</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP3</p></td><td><p xmlns="" class="SimplePara">Me1:TGAGTCCAAACCGGATA</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">8</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">8</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.82</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em13:GACTGCGTACGAATTTGA</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP4</p></td><td><p xmlns="" class="SimplePara">Me2:TGAGTCCAAACCGGCTT</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">9</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">8</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">88.89</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.98</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em5:GACTGCGTACGAATTCTG</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP5</p></td><td><p xmlns="" class="SimplePara">Me2:TGAGTCCAAACCGGCTT</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">8</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">7</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">87.5</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.94</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em9:GACTGCGTACGAATTCAT</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP6</p></td><td><p xmlns="" class="SimplePara">Me2:TGAGTCCAAACCGGCTT</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">9</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">8</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">88.89</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.95</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em11:GACTGCGTACGAATTCAC</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP7</p></td><td><p xmlns="" class="SimplePara">Me2:TGAGTCCAAACCGGCTT</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">10</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">8</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">80</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.85</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em14:GACTGCGTACGAATTCTT</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP8</p></td><td><p xmlns="" class="SimplePara">Me3:TGAGTCCAAACCGGCTG</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">8</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">8</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.86</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em1:GACTGCGTACGAATTATC</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP9</p></td><td><p xmlns="" class="SimplePara">Me3:TGAGTCCAAACCGGCTG</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">9</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">9</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.78</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em8:GACTGCGTACGAATTTCC</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP10</p></td><td><p xmlns="" class="SimplePara">Me4:TGAGTCCAAACCGGCCA</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">12</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">10</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">83.33</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.95</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em1:GACTGCGTACGAATTATC</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP11</p></td><td><p xmlns="" class="SimplePara">Me4:TGAGTCCAAACCGGCCA</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">11</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">10</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">90.9</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.98</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em2:GACTGCGTACGAATTTAT</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP12</p></td><td><p xmlns="" class="SimplePara">Me4:TGAGTCCAAACCGGCCA</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">7</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">7</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.99</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em3:GACTGCGTACGAATTTAG</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP13</p></td><td><p xmlns="" class="SimplePara">Me4:TGAGTCCAAACCGGCCA</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">8</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">8</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.96</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em7:GACTGCGTACGAATTTCT</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP14</p></td><td><p xmlns="" class="SimplePara">Me5:TGAGTCCAAACCGGGTA</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">10</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">10</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.96</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em2:GACTGCGTACGAATTTAT</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP15</p></td><td><p xmlns="" class="SimplePara">Me5:TGAGTCCAAACCGGGTA</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">10</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">9</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">90</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.93</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em3:GACTGCGTACGAATTTAG</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP16</p></td><td><p xmlns="" class="SimplePara">Me5:TGAGTCCAAACCGGGTA</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">9</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">9</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.98</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em5:GACTGCGTACGAATTCTG</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP17</p></td><td><p xmlns="" class="SimplePara">Me6:TGAGTCCAAACCGGTGA</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">11</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">10</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">90.9</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.92</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em2:GACTGCGTACGAATTTAT</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP18</p></td><td><p xmlns="" class="SimplePara">Me6:TGAGTCCAAACCGGTGA</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">12</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">9</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">81.82</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.85</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em3:GACTGCGTACGAATTTAG</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP19</p></td><td><p xmlns="" class="SimplePara">Me7:TGAGTCCAAACCGGTGT</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">13</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">11</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">84.62</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.95</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em4r:GACTGCGTACGAATTTGT</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP20</p></td><td><p xmlns="" class="SimplePara">Me7:TGAGTCCAAACCGGTGT</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">12</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">11</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">91.67</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.84</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em9:GACTGCGTACGAATTCAT</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP21</p></td><td><p xmlns="" class="SimplePara">Me7:TGAGTCCAAACCGGTGT</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">7</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">6</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">75</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.86</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em12:GACTGCGTACGAATTCAA</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP22</p></td><td><p xmlns="" class="SimplePara">Me7:TGAGTCCAAACCGGTGT</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">7</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">7</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.92</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em14:GACTGCGTACGAATTCTT</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP23</p></td><td><p xmlns="" class="SimplePara">Me8:TGAGTCCAAACCGGACC</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">5</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">5</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.96</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em2:GACTGCGTACGAATTTAT</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP24</p></td><td><p xmlns="" class="SimplePara">Me8:TGAGTCCAAACCGGACC</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">11</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">8</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">72.73</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.94</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em4:GACTGCGTACGAATTTGT</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP25</p></td><td><p xmlns="" class="SimplePara">Me8:TGAGTCCAAACCGGACC</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">10</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">8</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">80</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.95</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em8:GACTGCGTACGAATTTCC</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP26</p></td><td><p xmlns="" class="SimplePara">Me8:TGAGTCCAAACCGGACC</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">8</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">7</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">87.5</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.94</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em13:GACTGCGTACGAATTTGA</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP27</p></td><td><p xmlns="" class="SimplePara">Me9:CTTACTTAGACCGGAGT</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">7</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">6</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">85.71</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.97</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em2:GACTGCGTACGAATTTAT</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP28</p></td><td><p xmlns="" class="SimplePara">Me9:CTTACTTAGACCGGAGT</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">9</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">7</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">77.78</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.97</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em3:GACTGCGTACGAATTTAG</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP29</p></td><td><p xmlns="" class="SimplePara">Me9:CTTACTTAGACCGGAGT</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">5</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">5</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.98</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em5:GACTGCGTACGAATTCTG</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP30</p></td><td><p xmlns="" class="SimplePara">Me9:CTTACTTAGACCGGAGT</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">6</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">6</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.99</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em6:GACTGCGTACGAATTCTA</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP31</p></td><td><p xmlns="" class="SimplePara">Me9:CTTACTTAGACCGGAGT</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">7</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">6</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">85.71</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">1.00</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em9:GACTGCGTACGAATTCAT</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP32</p></td><td><p xmlns="" class="SimplePara">Me9:CTTACTTAGACCGGAGT</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">8</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">7</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">87.5</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.90</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em14:GACTGCGTACGAATTCTT</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP33</p></td><td><p xmlns="" class="SimplePara">Me10:TGAGTCCAAACCGGAAA</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">9</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">7</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">77.78</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.97</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em5:GACTGCGTACGAATTCTG</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP34</p></td><td><p xmlns="" class="SimplePara">Me10:TGAGTCCAAACCGGAAA</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">8</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">8</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.96</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em7:GACTGCGTACGAATTTCT</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP35</p></td><td><p xmlns="" class="SimplePara">Me10:TGAGTCCAAACCGGAAA</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">5</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">5</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.93</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em13:GACTGCGTACGAATTTGA</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP36</p></td><td><p xmlns="" class="SimplePara">Me10:TGAGTCCAAACCGGAAA</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">8</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">6</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">75</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.99</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em14:GACTGCGTACGAATTCTT</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP37</p></td><td><p xmlns="" class="SimplePara">Me11:GTACATAGAACCGGAGT</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">6</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">5</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">83.33</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.96</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em4:GACTGCGTACGAATTTGT</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP38</p></td><td><p xmlns="" class="SimplePara">Me11:GTACATAGAACCGGAGT</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">6</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">5</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">83.33</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">1.00</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em5:GACTGCGTACGAATTCTG</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP39</p></td><td><p xmlns="" class="SimplePara">Me11:GTACATAGAACCGGAGT</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">5</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">5</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.97</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em7:GACTGCGTACGAATTTCT</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">SRAP40</p></td><td><p xmlns="" class="SimplePara">Me11:GTACATAGAACCGGAGT</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">10</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">8</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">80</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.98</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Em14:GACTGCGTACGAATTCTT</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR2</p></td><td><p xmlns="" class="SimplePara">F:GTCAAGAAAGCTCAAGGC</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">11</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">10</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">90.91</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.96</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:GATGGGCTTGTCTTCGTC</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR4</p></td><td><p xmlns="" class="SimplePara">F:GTCAAGAAAGCTCAAGGC</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">7</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">7</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.96</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:TGTCTTGTGACCAAATTGAC</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR5</p></td><td><p xmlns="" class="SimplePara">F:GTCAAGAAAGCTCAAGGC</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">6</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">6</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.89</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:TGAAGTGGCGGCGGAAGA</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR7</p></td><td><p xmlns="" class="SimplePara">F:GTCAAGAAAGCTCAAGGC</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">5</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">5</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.97</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:GTCAAGTCAAAAACGCCG</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR9</p></td><td><p xmlns="" class="SimplePara">F:CCACCGTTGATTCTACTTT</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">12</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">11</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">91.67</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.97</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:AACAGAGCATACCCAGAAG</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR14</p></td><td><p xmlns="" class="SimplePara">F:CCACCGTTGATTCTACTTT</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">9</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">9</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.99</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:AAGACCCATACAAAAGATACT</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR15</p></td><td><p xmlns="" class="SimplePara">F:CCACCGTTGATTCTACTTT</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">8</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">7</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">87.5</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.92</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:GATGGGCTTGTCTTCGTC</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR17</p></td><td><p xmlns="" class="SimplePara">F:CCACCGTTGATTCTACTTT</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">9</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">9</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.89</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:GTCAAGTCAAAAACGCCG</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR19</p></td><td><p xmlns="" class="SimplePara">F:CCACCGTTGATTCTACTTT</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">6</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">6</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.98</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:CTGCGAACCCTCTTGACC</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR20</p></td><td><p xmlns="" class="SimplePara">F:ATCCACCGTATGATGCT</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">5</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">5</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.98</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:GATGGGCTTGTCTTCGTC</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR23</p></td><td><p xmlns="" class="SimplePara">F:GAATCAGTGAATAAAGCGTGTA</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">8</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">8</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.97</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:TGAAGTGGCGGCGGAAGA</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR24</p></td><td><p xmlns="" class="SimplePara">F:GAATCAGTGAATAAAGCGTGTA</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">7</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">7</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.99</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:TTGGTAGCCTCTTCTTTTG</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR26</p></td><td><p xmlns="" class="SimplePara">F:CTCCGATTACTTTCTTCC</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">7</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">6</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">85.71</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.98</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:GATGACGATGGAGTGGG</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR27</p></td><td><p xmlns="" class="SimplePara">F:CATAGTAGAGAAGACCACCA</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">8</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">7</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">87.5</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.99</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:GATGGGCTTGTCTTCGTC</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR30</p></td><td><p xmlns="" class="SimplePara">F:CATAGTAGAGAAGACCACCA</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">7</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">6</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">85.71</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.94</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:GATGACGATGGAGTGGG</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR33</p></td><td><p xmlns="" class="SimplePara">F:GAAAGTGCGAAACCAAAC</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">3</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">3</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.98</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:TGAAGTGGCGGCGGAAGA</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR35</p></td><td><p xmlns="" class="SimplePara">F:GAAAGTGCGAAACCAAAC</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">6</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">6</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.93</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:GTCAAGTCAAAAACGCCG</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR38</p></td><td><p xmlns="" class="SimplePara">F:GAAAGTGCGAAACCAAAC</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">9</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">9</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.94</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:CTGCGAACCCTCTTGACC</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR39</p></td><td><p xmlns="" class="SimplePara">F:CAAGCAATACATACACACA</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">16</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">14</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">87.5</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.86</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:AAGACCCATACAAAAGATACT</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR43</p></td><td><p xmlns="" class="SimplePara">F:CAAGCAATACATACACACA</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">5</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">5</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.98</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:AAAACAAGCCACCTCTA</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR47</p></td><td><p xmlns="" class="SimplePara">F:CTCTTGATTGGTGCCTTTA</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">11</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">10</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">90.91</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.92</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:AAGACCCATACAAAAGATACT</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR49</p></td><td><p xmlns="" class="SimplePara">F:CTCTTGATTGGTGCCTTTA</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">9</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">9</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.92</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:GATGGGCTTGTCTTCGTC</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR50</p></td><td><p xmlns="" class="SimplePara">F:CTCTTGATTGGTGCCTTTA</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">7</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">7</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.98</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:TGTCTTGTGACCAAATTGAC</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR52</p></td><td><p xmlns="" class="SimplePara">F:CTCTTGATTGGTGCCTTTA</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">12</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">11</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">91.67</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.82</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:TGAAGTGGCGGCGGAAGA</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR53</p></td><td><p xmlns="" class="SimplePara">F:CATTGCCTTGATGCTGA</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">9</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">9</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.90</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:AAGACCCATACAAAAGATACT</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR56</p></td><td><p xmlns="" class="SimplePara">F:CATTGCCTTGATGCTGA</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">9</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">9</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.91</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:GATGGGCTTGTCTTCGTC</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR58</p></td><td><p xmlns="" class="SimplePara">F:CATTGCCTTGATGCTGA</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">12</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">10</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">83.33</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.98</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:TGAAGTGGCGGCGGAAGA</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR59</p></td><td><p xmlns="" class="SimplePara">F:CATTGCCTTGATGCTGA</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">13</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">11</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">84.62</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.96</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:GTCAAGTCAAAAACGCCG</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">EST-SSR64</p></td><td><p xmlns="" class="SimplePara">F:CCACCGTTGATTCTACTTT R:TGTCTTGTGACCAAATTGAC</p></td><td char="."><p xmlns="" class="SimplePara">8</p></td><td char="."><p xmlns="" class="SimplePara">8</p></td><td char="."><p xmlns="" class="SimplePara">100</p></td><td char="."><p xmlns="" class="SimplePara">0.89</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR67</p></td><td><p xmlns="" class="SimplePara">F:CCACCGTTGATTCTACTTT</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">9</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">8</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">88.89</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.91</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:TGAAGTGGCGGCGGAAGA</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR69</p></td><td><p xmlns="" class="SimplePara">F:ATCCACCGTATGATGCT</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">8</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">7</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">87.5</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.76</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:AAGACCCATACAAAAGATACT</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR72</p></td><td><p xmlns="" class="SimplePara">F:ATCCACCGTATGATGCT</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">9</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">9</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.83</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:GTCAAGTCAAAAACGCCG</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR74</p></td><td><p xmlns="" class="SimplePara">F:ATCCACCGTATGATGCT</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">9</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">8</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">88.89</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.88</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:TTGGTAGCCTCTTCTTTTG</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR76</p></td><td><p xmlns="" class="SimplePara">F:GAATCAGTGAATAAAGCGTGTA</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">6</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">6</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.98</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:GATGGGCTTGTCTTCGTC</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR77</p></td><td><p xmlns="" class="SimplePara">F:GAATCAGTGAATAAAGCGTGTA</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">7</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">6</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">85.72</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.94</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:GCAGGTTAGCGGTGGTTA</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR85</p></td><td><p xmlns="" class="SimplePara">F:GAATCAGTGAATAAAGCGTGTA</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">6</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">6</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.94</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:GATGACGATGGAGTGGG</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR88</p></td><td><p xmlns="" class="SimplePara">F:GAAAGTGCGAAACCAAAC</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">8</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">7</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">87.5</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.97</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:AAGACCCATACAAAAGATACT</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR91</p></td><td><p xmlns="" class="SimplePara">F:GAAAGTGCGAAACCAAAC</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">5</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">5</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.95</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:TGTCTTGTGACCAAATTGAC</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR99</p></td><td><p xmlns="" class="SimplePara">F:CAAGCAATACATACACACA</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">4</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">4</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.94</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:TTGGTAGCCTCTTCTTTTG</p></td></tr><tr class="noclass"><td rowspan="2"><p xmlns="" class="SimplePara">EST-SSR40</p></td><td><p xmlns="" class="SimplePara">F:ATCCACCGTATGATGCT</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">5</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">5</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">100</p></td><td rowspan="2" char="."><p xmlns="" class="SimplePara">0.98</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">R:TGTCTTGTGACCAAATTGAC</p></td></tr></tbody></table></div></figure><figure xmlns="" id="Tab2" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 2</span>
                        <p xmlns="" class="SimplePara">Comparison of the efficiency of EST-SSR, SRAP, and SCoT primers</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col char="."></col><col char="."></col><col char="."></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Type of marker</p></th><th char="."><p xmlns="" class="SimplePara">SCoT</p></th><th char="."><p xmlns="" class="SimplePara">SRAP</p></th><th char="."><p xmlns="" class="SimplePara">EST-SST</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">Number of primers</p></td><td char="."><p xmlns="" class="SimplePara">38</p></td><td char="."><p xmlns="" class="SimplePara">40</p></td><td char="."><p xmlns="" class="SimplePara">40</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Total number of bands(TNB)</p></td><td char="."><p xmlns="" class="SimplePara">414</p></td><td char="."><p xmlns="" class="SimplePara">338</p></td><td char="."><p xmlns="" class="SimplePara">320</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Average number of loci per assay</p></td><td char="."><p xmlns="" class="SimplePara">10.89</p></td><td char="."><p xmlns="" class="SimplePara">8.45</p></td><td char="."><p xmlns="" class="SimplePara">8.00</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Number of polymorphic bands(NPB)</p></td><td char="."><p xmlns="" class="SimplePara">398</p></td><td char="."><p xmlns="" class="SimplePara">302</p></td><td char="."><p xmlns="" class="SimplePara">301</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Percentage of polymorphic bands (PPB)</p></td><td char="."><p xmlns="" class="SimplePara">96.14%</p></td><td char="."><p xmlns="" class="SimplePara">89.35%</p></td><td char="."><p xmlns="" class="SimplePara">94.06%</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Polymorphism information content (PIC)</p></td><td char="."><p xmlns="" class="SimplePara">0.79</p></td><td char="."><p xmlns="" class="SimplePara">0.77</p></td><td char="."><p xmlns="" class="SimplePara">0.85</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Resolving power (Rp)</p></td><td char="."><p xmlns="" class="SimplePara">6.71</p></td><td char="."><p xmlns="" class="SimplePara">5.11</p></td><td char="."><p xmlns="" class="SimplePara">4.48</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Effective multiplex ratio (EMR)</p></td><td char="."><p xmlns="" class="SimplePara">10.47</p></td><td char="."><p xmlns="" class="SimplePara">7.55</p></td><td char="."><p xmlns="" class="SimplePara">7.53</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Average band informativeness (Ib<sub xmlns="http://www.w3.org/1999/xhtml">av</sub>)</p></td><td char="."><p xmlns="" class="SimplePara">0.58</p></td><td char="."><p xmlns="" class="SimplePara">0.61</p></td><td char="."><p xmlns="" class="SimplePara">0.56</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Marker index (MI)</p></td><td char="."><p xmlns="" class="SimplePara">6.07</p></td><td char="."><p xmlns="" class="SimplePara">4.61</p></td><td char="."><p xmlns="" class="SimplePara">4.22</p></td></tr></tbody></table></div></figure></div>
              </section><section id="Sec4" class="Section2 RenderAsSection2">
                <h3 class="Heading">Correlation analysis among genetic distance matrices by three-types of marker dataset</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par23" class="Para">Mantel tests [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>] were used to measure the correlation between the genetic distance matrices generated by SCoT, SRAP and EST-SSR molecular markers. <em xmlns="" class="EmphasisTypeItalic">r</em> ≥ 0.9, 0.8 ≤ <em xmlns="" class="EmphasisTypeItalic">r</em> &lt; 0.9, 0.7 ≤ <em xmlns="" class="EmphasisTypeItalic">r</em> &lt; 0.8, and <em xmlns="" class="EmphasisTypeItalic">r</em> &lt; 0.7 represented significant correlation, moderate correlation, weak correlation, and no correlation, respectively. In the present study, the coefficients of correlation (r) between the genetic distance matrices of SCoT and EST-SSR markers, SCoT and SRAP markers, and SRAP and EST-SSR markers were 0.19, 0.17, and 0.01, respectively (Fig.&nbsp;<span class="InternalRef"><a href="#Fig1">1</a></span>). Different molecular markers and different primers of the same marker all yielded distinct amplification products, which reflected the polymorphism of the genomic regions; hence, utilization of different marker designing strategies will produce different results. Theoretically, the validity of the results should improve with increasing numbers of markers and increasing coverage of the genome. Therefore, we employed three types of molecular markers to generated 1072 bands and to perform genetic constitution analyses.<figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO1">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0058-4/MediaObjects/41065_2018_58_Fig1_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0058-4/MediaObjects/41065_2018_58_Fig1_HTML.gif" alt="Fig. 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0058-4/MediaObjects/41065_2018_58_Fig1_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0058-4/MediaObjects/41065_2018_58_Fig1_HTML.gif" alt="Fig. 1"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 1</span>
                        <p xmlns="" class="SimplePara">The correlation between the genetic distance matrices using Mantel tests</p>
                      </div></figcaption></figure></div>
              </section><section id="Sec5" class="Section2 RenderAsSection2">
                <h3 class="Heading">Genetic constitution analysis</h3>
                <section id="Sec6" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Analysis using STRUCTURE</h4>
                  <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par24" class="Para">One thousand seventy-two polymorphic bands with MAF (minor allele frequency) &lt; 5% were used to elucidate the population structure of the entire pool of tea germplasms. In this study, STRUCTURE&nbsp;2.3.4, which applies a Bayesian clustering algorithm, was used to simulate population genetic structure based on the assumption that the 1072 loci were independent. Using a membership probability threshold of 0.60, population K values from 1 to 10 were simulated with 20 iterations for each K using 10,000 burn-in periods followed by 10,000 Markov Chain Monte Carlo iterations in order to obtain an estimate of the most probable number of population. Delta K was plotted against K values; the best number of clusters was determined following the method proposed by Evanno et al. [<span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>] and obtained via the Structure Harvester platform (<span class="ExternalRef"><a href="http://taylor0.biology.ucla.edu/structureHarvester/"><span class="RefSource">http://taylor0.biology.ucla.edu/structureHarvester/</span></a></span>). Delta K reached a maximum value at K = 2, suggesting that the 50 tea germplasm were best divided into two subgroups (Fig.&nbsp;<span class="InternalRef"><a href="#Fig2">2</a></span>).<figure xmlns="" class="Figure" id="Fig2"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO2">
                        <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0058-4/MediaObjects/41065_2018_58_Fig2_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0058-4/MediaObjects/41065_2018_58_Fig2_HTML.gif" alt="Fig. 2" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0058-4/MediaObjects/41065_2018_58_Fig2_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0058-4/MediaObjects/41065_2018_58_Fig2_HTML.gif" alt="Fig. 2"></img></noscript></a>
                      </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 2</span>
                          <p xmlns="" class="SimplePara">STRUCTURE analysis of the number of population for K. The number of subpopulations(k) was identified based on maximum likelihood and k values. The most likely value of k identified by STRUCTURE was observed at k = 2. Note: Green bands: Group 1, Red bands: Group 2. The proportion of each color reflects the probability that each of the test materials (numbered from 1 to 50) belongs the corresponding group</p>
                        </div></figcaption></figure></div>
                </section>
                <section id="Sec7" class="Section3 RenderAsSection3">
                  <h4 class="Heading">UPGMA clustering</h4>
                  <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par25" class="Para">A dendrogram was constructed with cluster analysis using the unweighted pair-group method with arithmetic means (UPGMA), which demonstrated that the 50 genotypes could be clearly divided into 2 groups (Fig.&nbsp;<span class="InternalRef"><a href="#Fig3">3</a></span>). Group I included 27 varieties, and group II contained 23 varieties. The average similarity coefficient was 0.74. The two most closely related materials were 15 and 16, which have a sister line with a genetic similarity coefficient of 0.93.<figure xmlns="" class="Figure" id="Fig3"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO3">
                        <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0058-4/MediaObjects/41065_2018_58_Fig3_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0058-4/MediaObjects/41065_2018_58_Fig3_HTML.gif" alt="Fig. 3" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0058-4/MediaObjects/41065_2018_58_Fig3_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0058-4/MediaObjects/41065_2018_58_Fig3_HTML.gif" alt="Fig. 3"></img></noscript></a>
                      </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 3</span>
                          <p xmlns="" class="SimplePara">Cluster dendrogram of 50 tea genotypes constructed based on UPGMA by EST-SSR, SRAP and SCoT</p>
                        </div></figcaption></figure></div>
                </section>
                <section id="Sec8" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Principal components analysis</h4>
                  <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par26" class="Para">The top three principal components were used to analyze population structure. Principal component analysis was conducted under NTSYS-pc2.10e [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>]. The results showed that the three PCs had contribution rates of 15.97%, 8.50% and 6.17%. PCA separated the 50 genotypes into two major groups (Fig.&nbsp;<span class="InternalRef"><a href="#Fig4">4</a></span>) which were consistent with the STRUCTURE and UPGMA results. GroupI consisted of 18 genotypes (Fig. <span class="InternalRef"><a href="#Fig4">4</a></span>, left), with the other 32 genotypes belonging to group II (Fig. <span class="InternalRef"><a href="#Fig4">4</a></span>, right).<figure xmlns="" class="Figure" id="Fig4"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO4">
                        <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0058-4/MediaObjects/41065_2018_58_Fig4_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0058-4/MediaObjects/41065_2018_58_Fig4_HTML.gif" alt="Fig. 4" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0058-4/MediaObjects/41065_2018_58_Fig4_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0058-4/MediaObjects/41065_2018_58_Fig4_HTML.gif" alt="Fig. 4"></img></noscript></a>
                      </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 4</span>
                          <p xmlns="" class="SimplePara">PCA plots based on the first three components</p>
                        </div></figcaption></figure></div>
                  <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par27" class="Para">The analysis performed using STRUCTURE, UPGMA and PCA yielded similar results, clustering the 50 genotypes into 2 sub-populations. Of note, PCA results had good consistency with previous results from STRUCTURE. The results generated using UPGMA were slightly different from those using STRUCTURE and PCA (Table&nbsp;<span class="InternalRef"><a href="#Tab3">3</a></span>) and bold numbers in group 1 by UPGMA represent the differences between the results using STRUCTURE and PCA and the results using NJ.<figure xmlns="" id="Tab3" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 3</span>
                          <p xmlns="" class="SimplePara">Comparison of the clustering by STRUCTURE, PCA and UPGME</p>
                        </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col char="."></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Clustering method</p></th><th><p xmlns="" class="SimplePara">Code of group1</p></th><th char="."><p xmlns="" class="SimplePara">Code of group2</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">STRUCTURE</p></td><td><p xmlns="" class="SimplePara">1,3,4,5,6,8,9,10,11,12,13,14,22,40,42,43,48,49</p></td><td char="."><p xmlns="" class="SimplePara">2,7,15,16,17,18,19,20,21,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,41,44,45,46,47,50</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">PCA</p></td><td><p xmlns="" class="SimplePara">1,3,4,5,6,8,9,10,11,12,13,14,22,40,42,43,48,49</p></td><td char="."><p xmlns="" class="SimplePara">2,7,15,16,17,18,19,20,21,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,41,44,45,46,47,50</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">UPGMA</p></td><td><p xmlns="" class="SimplePara">1,3,4,5,6,<strong class="EmphasisTypeBold">7</strong>,8,9,10,11,12,13,14,22,<strong class="EmphasisTypeBold">25</strong>,<strong class="EmphasisTypeBold">27</strong>,<strong class="EmphasisTypeBold">32</strong>,<strong class="EmphasisTypeBold">35</strong>,<strong class="EmphasisTypeBold">37</strong>, <strong class="EmphasisTypeBold">39</strong>,40,<strong class="EmphasisTypeBold">41</strong>, 42,43, <strong class="EmphasisTypeBold">44</strong>,48,49</p></td><td char="."><p xmlns="" class="SimplePara">2,15,16,17,18,19,20,21,23,24,26,28,29,30,31,33,34,36,38,45,46,47,50</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                        <p xmlns="" class="SimplePara">Note: Bold numbers in group 1 by UPGMA method refer to the code that different from STRUCTURE and PC method</p>
                      </div></figure></div>
                </section>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec9" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Conclusions" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Conclusions">Conclusions</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Conclusions" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par28" class="Para">We firstly reported the use of SCoT markers to analysis genetic diversity of tea germplasms. The results showed that SCoT markers revealed high genetic diversity among tea resources. In the future, we planed to select core SCoT markers. Different kinds of molecular markers can reveal different and complementary information of the same genome. Thus, we highly recommend using more marker types for comprehensive evaluation of genetic diversity and structure.&nbsp;50 accessions were clustered into 2 sub-populations based on STRUCTURE, UPGMA and PCA; there was no obvious differences between imported and local germplasms. The genes of exotic varieties have been constantly integrated into the gene pool of Qinba tea through long-term (20–25&nbsp;years) tea breeding and production activities. The selection of varieties with economic characters was emphasized during the process of breeding, resulting in the loss of some tea resources and the decrease of genetic diversity; thus, it is necessary to introduce new tea tree resources in order to broaden the genetic diversity.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec10" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Methods" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Methods">Methods</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Methods" data-test="collapse-section"><section id="Sec11" class="Section2 RenderAsSection2">
                <h3 class="Heading">Plant materials</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par29" class="Para">A total of 50 tea plant genotypes, representing most tea germplasm of the Qinba area in China, were collected from the tea experimental farm of the Hanzhong Institute of Agricultural Sciences during the 2016 growing season (Table&nbsp;<span class="InternalRef"><a href="#Tab4">4</a></span>).<figure xmlns="" id="Tab4" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 4</span>
                        <p xmlns="" class="SimplePara">The 50 tea plant samples used for marker (EST-SSR, SRAP and SCoT) genotyping</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col></col><col char="."></col><col></col><col></col><col char="."></col><col></col><col></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Code</p></th><th><p xmlns="" class="SimplePara">Name</p></th><th><p xmlns="" class="SimplePara">Origin of tea district</p></th><th char="."><p xmlns="" class="SimplePara">Code</p></th><th><p xmlns="" class="SimplePara">Name</p></th><th><p xmlns="" class="SimplePara">Origin of tea district</p></th><th char="."><p xmlns="" class="SimplePara">Code</p></th><th><p xmlns="" class="SimplePara">Name</p></th><th><p xmlns="" class="SimplePara">Origin of tea district</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">BeiBa13–2</p></td><td><p xmlns="" class="SimplePara">Jiangbei</p></td><td char="."><p xmlns="" class="SimplePara">18</p></td><td><p xmlns="" class="SimplePara">FuDingDaBai</p></td><td><p xmlns="" class="SimplePara">Huanan</p></td><td char="."><p xmlns="" class="SimplePara">35</p></td><td><p xmlns="" class="SimplePara">ZhenBa14–5</p></td><td><p xmlns="" class="SimplePara">Jiangbei</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">2</p></td><td><p xmlns="" class="SimplePara">BeiBa13–7</p></td><td><p xmlns="" class="SimplePara">Jiangbei</p></td><td char="."><p xmlns="" class="SimplePara">19</p></td><td><p xmlns="" class="SimplePara">YunKang10</p></td><td><p xmlns="" class="SimplePara">Xinan</p></td><td char="."><p xmlns="" class="SimplePara">36</p></td><td><p xmlns="" class="SimplePara">Xi14–37</p></td><td><p xmlns="" class="SimplePara">Jiangbei</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">3</p></td><td><p xmlns="" class="SimplePara">Xi13–16</p></td><td><p xmlns="" class="SimplePara">Jiangbei</p></td><td char="."><p xmlns="" class="SimplePara">20</p></td><td><p xmlns="" class="SimplePara">FuXiang3</p></td><td><p xmlns="" class="SimplePara">Huanan</p></td><td char="."><p xmlns="" class="SimplePara">37</p></td><td><p xmlns="" class="SimplePara">Xi14–36</p></td><td><p xmlns="" class="SimplePara">Jiangbei</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">4</p></td><td><p xmlns="" class="SimplePara">Ning13–6</p></td><td><p xmlns="" class="SimplePara">Jiangbei</p></td><td char="."><p xmlns="" class="SimplePara">21</p></td><td><p xmlns="" class="SimplePara">XiangBoLu</p></td><td><p xmlns="" class="SimplePara">Jiangnan</p></td><td char="."><p xmlns="" class="SimplePara">38</p></td><td><p xmlns="" class="SimplePara">ZhenBa14–19</p></td><td><p xmlns="" class="SimplePara">Jiangbei</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">5</p></td><td><p xmlns="" class="SimplePara">BeiBa12–1</p></td><td><p xmlns="" class="SimplePara">Jiangbei</p></td><td char="."><p xmlns="" class="SimplePara">22</p></td><td><p xmlns="" class="SimplePara">FuXuan9</p></td><td><p xmlns="" class="SimplePara">Huanan</p></td><td char="."><p xmlns="" class="SimplePara">39</p></td><td><p xmlns="" class="SimplePara">DaJiaoBan1</p></td><td><p xmlns="" class="SimplePara">Jiangbei</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">6</p></td><td><p xmlns="" class="SimplePara">BeiBa12–3</p></td><td><p xmlns="" class="SimplePara">Jiangbei</p></td><td char="."><p xmlns="" class="SimplePara">23</p></td><td><p xmlns="" class="SimplePara">ZhuYeQi</p></td><td><p xmlns="" class="SimplePara">Jiangnan</p></td><td char="."><p xmlns="" class="SimplePara">40</p></td><td><p xmlns="" class="SimplePara">WuNiuZao</p></td><td><p xmlns="" class="SimplePara">Jiangnan</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">7</p></td><td><p xmlns="" class="SimplePara">BeiBa11–2</p></td><td><p xmlns="" class="SimplePara">Jiangbei</p></td><td char="."><p xmlns="" class="SimplePara">24</p></td><td><p xmlns="" class="SimplePara">LongJing</p></td><td><p xmlns="" class="SimplePara">Jiangnan</p></td><td char="."><p xmlns="" class="SimplePara">41</p></td><td><p xmlns="" class="SimplePara">CuiFeng</p></td><td><p xmlns="" class="SimplePara">Jiangnan</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">8</p></td><td><p xmlns="" class="SimplePara">YuSun2</p></td><td><p xmlns="" class="SimplePara">Jiangnan</p></td><td char="."><p xmlns="" class="SimplePara">25</p></td><td><p xmlns="" class="SimplePara">BeiBa14–32</p></td><td><p xmlns="" class="SimplePara">Jiangbei</p></td><td char="."><p xmlns="" class="SimplePara">42</p></td><td><p xmlns="" class="SimplePara">BeiBa12–4</p></td><td><p xmlns="" class="SimplePara">Jiangbei</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">9</p></td><td><p xmlns="" class="SimplePara">BeiBa11–6</p></td><td><p xmlns="" class="SimplePara">Jiangbei</p></td><td char="."><p xmlns="" class="SimplePara">26</p></td><td><p xmlns="" class="SimplePara">BeiBa14–37</p></td><td><p xmlns="" class="SimplePara">Jiangbei</p></td><td char="."><p xmlns="" class="SimplePara">43</p></td><td><p xmlns="" class="SimplePara">HuangJinCha1</p></td><td><p xmlns="" class="SimplePara">Jiangnan</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">10</p></td><td><p xmlns="" class="SimplePara">ZaoBaiJian</p></td><td><p xmlns="" class="SimplePara">Xinan</p></td><td char="."><p xmlns="" class="SimplePara">27</p></td><td><p xmlns="" class="SimplePara">Ning14–14-2</p></td><td><p xmlns="" class="SimplePara">Jiangbei</p></td><td char="."><p xmlns="" class="SimplePara">44</p></td><td><p xmlns="" class="SimplePara">BaiHaoZao</p></td><td><p xmlns="" class="SimplePara">Jiangnan</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">11</p></td><td><p xmlns="" class="SimplePara">JinGuanYin</p></td><td><p xmlns="" class="SimplePara">Huanan</p></td><td char="."><p xmlns="" class="SimplePara">28</p></td><td><p xmlns="" class="SimplePara">Ning14–51</p></td><td><p xmlns="" class="SimplePara">Jiangbei</p></td><td char="."><p xmlns="" class="SimplePara">45</p></td><td><p xmlns="" class="SimplePara">JinMuDan</p></td><td><p xmlns="" class="SimplePara">Huanan</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">12</p></td><td><p xmlns="" class="SimplePara">ShanCha1</p></td><td><p xmlns="" class="SimplePara">Jiangbei</p></td><td char="."><p xmlns="" class="SimplePara">29</p></td><td><p xmlns="" class="SimplePara">Xi14–10</p></td><td><p xmlns="" class="SimplePara">Jiangbei</p></td><td char="."><p xmlns="" class="SimplePara">46</p></td><td><p xmlns="" class="SimplePara">ChunBoLu</p></td><td><p xmlns="" class="SimplePara">Huanan</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">13</p></td><td><p xmlns="" class="SimplePara">PingYangTeZao</p></td><td><p xmlns="" class="SimplePara">Jiangnan</p></td><td char="."><p xmlns="" class="SimplePara">30</p></td><td><p xmlns="" class="SimplePara">ZhenBa14–39</p></td><td><p xmlns="" class="SimplePara">Jiangbei</p></td><td char="."><p xmlns="" class="SimplePara">47</p></td><td><p xmlns="" class="SimplePara">Echa1</p></td><td><p xmlns="" class="SimplePara">Jiangnan</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">14</p></td><td><p xmlns="" class="SimplePara">DanGui</p></td><td><p xmlns="" class="SimplePara">Huanan</p></td><td char="."><p xmlns="" class="SimplePara">31</p></td><td><p xmlns="" class="SimplePara">ZhenBa14–22</p></td><td><p xmlns="" class="SimplePara">Jiangbei</p></td><td char="."><p xmlns="" class="SimplePara">48</p></td><td><p xmlns="" class="SimplePara">ChunYu1</p></td><td><p xmlns="" class="SimplePara">Jiangnan</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">Ning13–3</p></td><td><p xmlns="" class="SimplePara">Jiangbei</p></td><td char="."><p xmlns="" class="SimplePara">32</p></td><td><p xmlns="" class="SimplePara">Ning14–50</p></td><td><p xmlns="" class="SimplePara">Jiangbei</p></td><td char="."><p xmlns="" class="SimplePara">49</p></td><td><p xmlns="" class="SimplePara">ShiFuCui</p></td><td><p xmlns="" class="SimplePara">Jiangbei</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">16</p></td><td><p xmlns="" class="SimplePara">Xi13–10-1</p></td><td><p xmlns="" class="SimplePara">Jiangbei</p></td><td char="."><p xmlns="" class="SimplePara">33</p></td><td><p xmlns="" class="SimplePara">Xi14–1</p></td><td><p xmlns="" class="SimplePara">Jiangbei</p></td><td char="."><p xmlns="" class="SimplePara">50</p></td><td><p xmlns="" class="SimplePara">BeiBa14–42</p></td><td><p xmlns="" class="SimplePara">Jiangbei</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">17</p></td><td><p xmlns="" class="SimplePara">Ning13–14</p></td><td><p xmlns="" class="SimplePara">Jiangbei</p></td><td char="."><p xmlns="" class="SimplePara">34</p></td><td><p xmlns="" class="SimplePara">Xi14–50</p></td><td><p xmlns="" class="SimplePara">Jiangbei</p></td><td char=".">&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr></tbody></table></div></figure></div>
              </section><section id="Sec12" class="Section2 RenderAsSection2">
                <h3 class="Heading">DNA extraction and marker genotyping</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par30" class="Para">Genomic DNA was extracted from fresh leaves of each individual using the modified CTAB technique and detected with 0.8% agarose gel electrophoresis. PCR was carried out as follows: 2 × Taq Master Mix (7.5&nbsp;μL), forward and reverse primers (1&nbsp;μL each, 2&nbsp;μL for SCoT primers), RNase-free water (3.5&nbsp;μL), and tea genomic DNA (2&nbsp;μL). In order to improve the effect of PCR amplification, changing annealing temperature was used in a PCR reaction system; the reactions were programed as follows: initial denaturation at 94.0&nbsp;°C for 5&nbsp;min, denaturation at 94.0&nbsp;°C for 1&nbsp;min, annealing at 60.0&nbsp;°C for 1&nbsp;min, and extension at 72.0&nbsp;°C for 1&nbsp;min, for a total of 10&nbsp;cycles; subsequently, a total of 35&nbsp;cycles of denaturation at 94.0&nbsp;°C for 30&nbsp;s, annealing at 35&nbsp;°C for 30&nbsp;s, and extension at 72.0&nbsp;°C for 1&nbsp;min were performed. The duration of extension was 10&nbsp;min; then storage at 4.0&nbsp;°C. The selected primers were synthesized by Shanghai Sangon Biological Engineering Technology and Service Company (Shanghai, China). Initially, six germplasms (LongJing, ShanCha1, ChunBoLu, BeiBa11–6, Ning13–6, ZaoBaiJian) were used to screen markers for high polymorphim. Then, 40 pairs of clear and highly polymorphic EST-SSR and SRAP markers, and 38 paris of SCoT marker primers were selected from 154 EST-SSR pairs, 154 SRAP pairs, 125 SCoT pairs. Electrophoresis was performed using 8% non-denaturing polyacrylamide gel under 160&nbsp;V voltage; the bands were visualized via silver staining.</p>
              </section><section id="Sec13" class="Section2 RenderAsSection2">
                <h3 class="Heading">Genetic variation and marker efficiency analysis</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par31" class="Para">Following electrophoresis, each amplification band corresponded to a primer hybridization locus and was considered as an effective molecular marker. Each polymorphic band detected by a same given primer represented an allelic mutation. In order to generate molecular data matrices, clear bands for each fragment were scored in every accession for each primer pair and recorded as 1 (presence of a fragment), 0 (absence of a fragment), and 9 (complete absence of band). Excel was used to compute the marker index (MI) of the three types of markers and the marker frequencies of the three types of markers were compared. MI values were obtained from the average band informativeness (Ib<sub>av</sub>) and the effectiveness multiplex ratio (EMR); EMR represents the number of polymorphic loci and Ib<sub>av</sub> is given by the following formula:<div id="Equa" class="Equation EquationMathjax"><div class="EquationContent">$$ {Ib}_{av}=\frac{1}{n}\sum \limits_{i=1}^n\left(1-\left(2\left|0.5-{P}_i\right|\right)\right), $$</div></div>where P<sub>i</sub> represents the proportion of the i<sup>th</sup> sample in the amplified locus and n represents the total number of amplified loci. Using the method reported by Smith et al. [<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>], the value of the polymorphism information content (PIC) was calculated with the formula:<div id="Equb" class="Equation EquationMathjax"><div class="EquationContent">$$ PIC=1-\sum \limits_{i=1}^n{P_i}^2-\sum \limits_{i=1}^{n-1}\sum \limits_{j=i+1}^n2{P_i}^2{P_j}^2, $$</div></div>where PIC represents the PIC value of the i<sup>th</sup> locus and P<sub>ij</sub> represents the frequency that allele j appears in the i<sup>th</sup> locus. The value of PIC varies from 0 to 1, with 0 indicating an absence of polymorphism at a given locus and 1 reflecting multiple alleles at a given locus. The level of polymorphism of each marker was assessed by the polymorphism information content (Botstein et al. [<span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>]), which measures the extent of genetic variation: PIC values smaller than 0.25 indicates low levels of polymorphism associated to a locus, PIC values between 0.25 and 0.5 imply moderate levels of polymorphism, while PIC values greater than 0.5 indicate high levels of polymorphism.</div>
              </section><section id="Sec14" class="Section2 RenderAsSection2">
                <h3 class="Heading">Correlation analysis among genetic distance matrices by three-types of marker dataset</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par32" class="Para">Mantel test was carried out with the batch file of the NTSYS-pc2.10e software.</p>
                <section id="Sec15" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Genetic constitution analysis</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par33" class="Para">STRUCTURE v2.3.4 was used to assess the population structure of the 50 tea genotypes with 1072 loci. The number of sub-population (K) was set from 1 to 10 based on admixture models and correlated band frequencies. Genetic similarity coefficients were computed using the SM functionality of the NTSYS-pc2.10e software, cluster analysis were conducted using the UPGMA method, and the principal component analysis using the batch file under the NTSYS-pc2.10e software.</p>
                </section>
              </section></div></section>
          <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AbbreviationGroup Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Abbreviations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Abbreviations">Abbreviations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Abbreviations"><div class="DefinitionList">
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>EMR:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par6" class="Para">Effective multiplex ratio</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>EST-SSR:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par7" class="Para">Expressed sequence tags-Simple sequence repeats</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>Ibav:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par8" class="Para">Average band informativeness</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>MAF:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par9" class="Para">Minor allele frequency</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>MI:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par10" class="Para">Marker index</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>NJ:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par11" class="Para">Neighbor-joining</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>PCA:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par12" class="Para">Principal component analyses</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>PIC:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par13" class="Para">Polymorphism information content</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>PPB:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par14" class="Para">Percentage of polymorphic bands</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>Rp:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par15" class="Para">Resolving power</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>SCoT:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par16" class="Para">Start codon targeted polymorphism</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>SRAP:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par17" class="Para">Sequence-related amplified polymorphism</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>UPGMA:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par18" class="Para">Unweighted pair group method with arithmetic mean</p>
                  </dd></dl>
              </div></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments"><h3 xmlns="" class="Heading">Funding</h3><p xmlns="" class="SimplePara">This study was supported by the earmarked fund for the Natural Science Basic Research Project of Shaanxi Province (2013JZ008), the Sci-technological Project of Shaanxi Province (2016KTCQ02–06) and the Qinling-Bashan Mountains Bioresources Comprehensive Development C. I. C (QBXT-17-5).</p></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="FormalPara FormalParaRenderingStyle1"><h3 xmlns="" class="Heading">Authors’ contributions</h3><p xmlns="" class="SimplePara">YZ analyzed the data, and wrote the manuscript. XZ edited and revised the manuscript. XC, WS and JL performed the experiments, and all authors approved the manuscript.</p></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Ethics">
              <div id="FPar1" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Ethics approval and consent to participate</h3>
                <p id="Par34" class="Para">Not applicable.</p>
              </div>
              <div id="FPar2" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Consent for publication</h3>
                <p id="Par35" class="Para">Not applicable.</p>
              </div>
              <div id="FPar3" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Competing interests</h3>
                <p id="Par36" class="Para">The authors declare that they have no competing interests.</p>
              </div>
              <div id="FPar4" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Publisher’s Note</h3>
                <p id="Par37" class="Para">Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
              </div>
            </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="License">
                <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Open Access</strong>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/4.0/"><span class="RefSource">http://creativecommons.org/licenses/by/4.0/</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/publicdomain/zero/1.0/"><span class="RefSource">http://creativecommons.org/publicdomain/zero/1.0/</span></a></span>) applies to the data made available in this article, unless otherwise stated.</p>
              </div></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authors’ Affiliations</a></h2><div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1"><span class="AffiliationNumber">(1)</span><div class="AffiliationText">School of Biological Science and Engineering, Shaanxi University of Technology, Hanzhong, Shaanxi Province, 723000, People’s Republic of China</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff2"><span class="AffiliationNumber">(2)</span><div class="AffiliationText">Qili Food and Drug Supervision and Management Institute, Hanzhong, Shaanxi Province, 723000, People’s Republic of China</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff3"><span class="AffiliationNumber">(3)</span><div class="AffiliationText">Hanzhong Institute of Agricultural Sciences, Hanzhong, Shaanxi Province, 723000, People’s Republic of China</div></div></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper"><li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yao MZ, Chen L, Ma CL, et al. Comparative analysis of genetic diversity among tea cultivars from China, Japan and Kenya by ISSR and EST-SSR. Mol Plant Breed. 2009;7(5):897–903.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20genetic%20diversity%20among%20tea%20cultivars%20from%20China%2C%20Japan%20and%20Kenya%20by%20ISSR%20and%20EST-SSR&amp;author=MZ.%20Yao&amp;author=L.%20Chen&amp;author=CL.%20Ma&amp;journal=Mol%20Plant%20Breed&amp;volume=7&amp;issue=5&amp;pages=897-903&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yao MZ, Qiao TT, Ma CL, et al. The association analysis of phenotypic traits with EST-SSR markers in tea plants. J Tea Sci. 2010;30(1):45–51.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20association%20analysis%20of%20phenotypic%20traits%20with%20EST-SSR%20markers%20in%20tea%20plants&amp;author=MZ.%20Yao&amp;author=TT.%20Qiao&amp;author=CL.%20Ma&amp;journal=J%20Tea%20Sci&amp;volume=30&amp;issue=1&amp;pages=45-51&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Liu B, Sun X, Li Y, et al. Analysis of genetic diversity of tea plants by using EST-SSR and ISSR markers. Chin J Trop Crop. 2009;30(11):1577–83.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Analysis%20of%20genetic%20diversity%20of%20tea%20plants%20by%20using%20EST-SSR%20and%20ISSR%20markers&amp;author=B.%20Liu&amp;author=X.%20Sun&amp;author=Y.%20Li&amp;journal=Chin%20J%20Trop%20Crop&amp;volume=30&amp;issue=11&amp;pages=1577-1583&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Qiao TT, Ma CL, Zhou YH, et al. EST-SSR genetic diversity and population structure of tea landraces and developed cultivars (lines) in Zhejiang Province, China. Acta Agron Sin. 2010;36(5):744–53.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.3724/SP.J.1006.2010.00744"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=EST-SSR%20genetic%20diversity%20and%20population%20structure%20of%20tea%20landraces%20and%20developed%20cultivars%20%28lines%29%20in%20Zhejiang%20Province%2C%20China&amp;author=TT.%20Qiao&amp;author=CL.%20Ma&amp;author=YH.%20Zhou&amp;journal=Acta%20Agron%20Sin&amp;volume=36&amp;issue=5&amp;pages=744-753&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li SJ, Wang X, Duan JH, Dong LJ, Zhang SG. Genetic diversity and genetic structure of 16 tea cultivars based on SSR markers. Hunan Agric Sci. 2011;12(23):6–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20and%20genetic%20structure%20of%2016%20tea%20cultivars%20based%20on%20SSR%20markers&amp;author=SJ.%20Li&amp;author=X.%20Wang&amp;author=JH.%20Duan&amp;author=LJ.%20Dong&amp;author=SG.%20Zhang&amp;journal=Hunan%20Agric%20Sci&amp;volume=12&amp;issue=23&amp;pages=6-9&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li G, Quiros CF. Sequence-related amplified polymorphism (SRAP), a new marker system based on a simple PCR reaction: its application to mapping and gene tagging in Brassica. Theor Appl Genet. 2001;103(2):455–61.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s001220100570"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Sequence-related%20amplified%20polymorphism%20%28SRAP%29%2C%20a%20new%20marker%20system%20based%20on%20a%20simple%20PCR%20reaction%3A%20its%20application%20to%20mapping%20and%20gene%20tagging%20in%20Brassica&amp;author=G.%20Li&amp;author=CF.%20Quiros&amp;journal=Theor%20Appl%20Genet&amp;volume=103&amp;issue=2&amp;pages=455-461&amp;publication_year=2001"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Shen CW, Ning ZX, Huang JA, et al. Genetic diversity of Camellia sinensis germplasm in Guangdong Province based on morphological parameters and SRAP markers. Chin J Appl Ecol. 2009;20(7):1551–8.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20of%20Camellia%20sinensis%20germplasm%20in%20Guangdong%20Province%20based%20on%20morphological%20parameters%20and%20SRAP%20markers&amp;author=CW.%20Shen&amp;author=ZX.%20Ning&amp;author=JA.%20Huang&amp;journal=Chin%20J%20Appl%20Ecol&amp;volume=20&amp;issue=7&amp;pages=1551-1558&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Shen CW, Huang JA, Zhao SH, Ning ZX, Li JX, Zhao CY, Chen D. Analysis of genetic diversity of camellia sinensis germplasm in Guangdong Province by srap and issr markers. J Nucl Agric Sci. 2010;24(5):948–55.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Analysis%20of%20genetic%20diversity%20of%20camellia%20sinensis%20germplasm%20in%20Guangdong%20Province%20by%20srap%20and%20issr%20markers&amp;author=CW.%20Shen&amp;author=JA.%20Huang&amp;author=SH.%20Zhao&amp;author=ZX.%20Ning&amp;author=JX.%20Li&amp;author=CY.%20Zhao&amp;author=D.%20Chen&amp;journal=J%20Nucl%20Agric%20Sci&amp;volume=24&amp;issue=5&amp;pages=948-955&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Xi CY, Tang Q. Wu YS, Xu JY, Chen H, Wu Q. Genetic diversity and relationship of 30 tea plant germplasms in Sichuan revealed by SRAP marker. Guizhou Agric Sci. 2013;41(2):6–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Wu%20YS%2C%20Xu%20JY%2C%20Chen%20H%2C%20Wu%20Q.%20Genetic%20diversity%20and%20relationship%20of%2030%20tea%20plant%20germplasms%20in%20Sichuan%20revealed%20by%20SRAP%20marker&amp;author=CY.%20Xi&amp;author=Q.%20Tang&amp;journal=Guizhou%20Agric%20Sci&amp;volume=41&amp;issue=2&amp;pages=6-9&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Liu Z, Zhao Y, Yang PD, Chen Y, Ning J, Yang Y. Comparison of parents identification for tea variety based on SSR, SRAP and ISSR markers. J Tea Sci. 2014;34(6):617–24.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparison%20of%20parents%20identification%20for%20tea%20variety%20based%20on%20SSR%2C%20SRAP%20and%20ISSR%20markers&amp;author=Z.%20Liu&amp;author=Y.%20Zhao&amp;author=PD.%20Yang&amp;author=Y.%20Chen&amp;author=J.%20Ning&amp;author=Y.%20Yang&amp;journal=J%20Tea%20Sci&amp;volume=34&amp;issue=6&amp;pages=617-624&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Chen XJ, Zhou KH, Zong HX, Fang R. Genetic diversity of capsicum frutescens in China as revealed by SRAP and SSR markers. Acta Bot Bor-Occid Sin. 2012;32(11):2201–5.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20of%20capsicum%20frutescens%20in%20China%20as%20revealed%20by%20SRAP%20and%20SSR%20markers&amp;author=XJ.%20Chen&amp;author=KH.%20Zhou&amp;author=HX.%20Zong&amp;author=R.%20Fang&amp;journal=Acta%20Bot%20Bor-Occid%20Sin&amp;volume=32&amp;issue=11&amp;pages=2201-2205&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Xia FG, Zhong XW, Wu F, et al. SRAP marker analysis of genetic diversity and relationship in Wuyi rock tea Germplasm resources. J Tea Sci. 2017;37(1):78–85.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=SRAP%20marker%20analysis%20of%20genetic%20diversity%20and%20relationship%20in%20Wuyi%20rock%20tea%20Germplasm%20resources&amp;author=FG.%20Xia&amp;author=XW.%20Zhong&amp;author=F.%20Wu&amp;journal=J%20Tea%20Sci&amp;volume=37&amp;issue=1&amp;pages=78-85&amp;publication_year=2017"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Bertrand C, David J, Mackill CY, et al. Start codon targeted (SCoT) polymorphism:a simple, novel DNA marker technique for generating gene-targeted markers in plants. Plant Mol Biol Rep. 2009;27(1):86–93.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11105-008-0060-5"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Start%20codon%20targeted%20%28SCoT%29%20polymorphism%3Aa%20simple%2C%20novel%20DNA%20marker%20technique%20for%20generating%20gene-targeted%20markers%20in%20plants&amp;author=C.%20Bertrand&amp;author=J.%20David&amp;author=CY.%20Mackill&amp;journal=Plant%20Mol%20Biol%20Rep&amp;volume=27&amp;issue=1&amp;pages=86-93&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Luo T, Yang HX, Cen HF, Liu XH, Gao YJ, Duan WX, et al. Application of SCoT molecular marker in construction of molecular genetic linkage map of saccharum spontaneum L. J Plant Genet Resour. 2013;14(4):704–10.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Application%20of%20SCoT%20molecular%20marker%20in%20construction%20of%20molecular%20genetic%20linkage%20map%20of%20saccharum%20spontaneum%20L&amp;author=T.%20Luo&amp;author=HX.%20Yang&amp;author=HF.%20Cen&amp;author=XH.%20Liu&amp;author=YJ.%20Gao&amp;author=WX.%20Duan&amp;journal=J%20Plant%20Genet%20Resour&amp;volume=14&amp;issue=4&amp;pages=704-710&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Jiang LF, Zhang XQ, Huang LK, Ma X, Yan DF, Hu Q, et al. Analysis of genetic diversity in a cocksfoot (Dactylis glomerata) variety using SCoT markers. Acta Pratacult Sin. 2014;23(1):229–38.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Analysis%20of%20genetic%20diversity%20in%20a%20cocksfoot%20%28Dactylis%20glomerata%29%20variety%20using%20SCoT%20markers&amp;author=LF.%20Jiang&amp;author=XQ.%20Zhang&amp;author=LK.%20Huang&amp;author=X.%20Ma&amp;author=DF.%20Yan&amp;author=Q.%20Hu&amp;journal=Acta%20Pratacult%20Sin&amp;volume=23&amp;issue=1&amp;pages=229-238&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Luo C. Study on SCoT marker and analysis on genes of stress-related and important flowering time in mango. Nanning Guangxi: Guangxi University; 2012.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Luo%20C.%20Study%20on%20SCoT%20marker%20and%20analysis%20on%20genes%20of%20stress-related%20and%20important%20flowering%20time%20in%20mango.%20Nanning%20Guangxi%3A%20Guangxi%20University%3B%202012."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Xiong FQ, Jiang J, Zhong RC, Han ZQ, He LQ, Li Z, Zhuang WJ, Tang RH. Application of SCoT molecular marker in genus arachis. Acta Agron Sin. 2010;36(12):2055–61.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.3724/SP.J.1006.2010.02055"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Application%20of%20SCoT%20molecular%20marker%20in%20genus%20arachis&amp;author=FQ.%20Xiong&amp;author=J.%20Jiang&amp;author=RC.%20Zhong&amp;author=ZQ.%20Han&amp;author=LQ.%20He&amp;author=Z.%20Li&amp;author=WJ.%20Zhuang&amp;author=RH.%20Tang&amp;journal=Acta%20Agron%20Sin&amp;volume=36&amp;issue=12&amp;pages=2055-2061&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Chen S. Study on genetic diversity and smut resistance evaluation of sugarcane parents. Guangzhou: South China Agricultural University; 2016.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Chen%20S.%20Study%20on%20genetic%20diversity%20and%20smut%20resistance%20evaluation%20of%20sugarcane%20parents.%20Guangzhou%3A%20South%20China%20Agricultural%20University%3B%202016."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wei YL, He XH, Luo C, Chen H. Genetic diversity of podocarpus by SCoT markers. Guihaia. 2012;32(1):90–3.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20of%20podocarpus%20by%20SCoT%20markers&amp;author=YL.%20Wei&amp;author=XH.%20He&amp;author=C.%20Luo&amp;author=H.%20Chen&amp;journal=Guihaia&amp;volume=32&amp;issue=1&amp;pages=90-93&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hou XG, Wang J, Jia T, Zhang YQ, Hou J, Li JJ. Orthogonal optimization of SCoT-PCR system and primer screening of tree peony. Acta Agriculturae Boreali-Sinica. 2011;26(5):92–6.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Orthogonal%20optimization%20of%20SCoT-PCR%20system%20and%20primer%20screening%20of%20tree%20peony&amp;author=XG.%20Hou&amp;author=J.%20Wang&amp;author=T.%20Jia&amp;author=YQ.%20Zhang&amp;author=J.%20Hou&amp;author=JJ.%20Li&amp;journal=Acta%20Agriculturae%20Boreali-Sinica&amp;volume=26&amp;issue=5&amp;pages=92-96&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Mantel N. The detection of disease clustering and a generalized regression approach. Cancer Res. 1967;27(2):209–20.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=6018555"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20detection%20of%20disease%20clustering%20and%20a%20generalized%20regression%20approach&amp;author=N.%20Mantel&amp;journal=Cancer%20Res&amp;volume=27&amp;issue=2&amp;pages=209-220&amp;publication_year=1967"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Evanno G, Regnaut S, Goudet J. Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study. Mol Ecol. 2005;14(8):2611–20.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1365-294X.2005.02553.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15969739"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Detecting%20the%20number%20of%20clusters%20of%20individuals%20using%20the%20software%20STRUCTURE%3A%20a%20simulation%20study&amp;author=G.%20Evanno&amp;author=S.%20Regnaut&amp;author=J.%20Goudet&amp;journal=Mol%20Ecol&amp;volume=14&amp;issue=8&amp;pages=2611-2620&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Rohlf FJ. NTSYS-pc - numerical taxonomy and multivariate analysis System; 1998. p. 2.1.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=NTSYS-pc%20-%20numerical%20taxonomy%20and%20multivariate%20analysis%20System&amp;author=FJ.%20Rohlf&amp;publication_year=1998"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR24" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Smith S, Helentjaris T. DNA fingerprinting and plant variety protection. In: Paterson AH, editor. Genome mapping in plants. Texas: Landes Company; 1996. p. 95–110.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=DNA%20fingerprinting%20and%20plant%20variety%20protection&amp;author=S.%20Smith&amp;author=T.%20Helentjaris&amp;pages=95-110&amp;publication_year=1996"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR25" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Botstein D, White RL, Skolnick M, et al. Construction of a genetic linkage map in man using restriction fragment length polymorphisms. Am J Hum Genet. 1980;32(3):314–31.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=6247908"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1686077"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20genetic%20linkage%20map%20in%20man%20using%20restriction%20fragment%20length%20polymorphisms&amp;author=D.%20Botstein&amp;author=RL.%20White&amp;author=M.%20Skolnick&amp;journal=Am%20J%20Hum%20Genet&amp;volume=32&amp;issue=3&amp;pages=314-331&amp;publication_year=1980"><span><span>Google Scholar</span></span></a></span></span></cite></li></ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#TheAuthors" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="TheAuthors">Copyright</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="TheAuthors"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">©&nbsp;The Author(s)&nbsp;2018</div></div></section></div>
                        


                    </article>
                </main>

                <div class="c-content-layout__side u-text-sm">
                    <aside>
                        <div class="p-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="//hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-018-0058-4" data-track="click" data-track-category="PDF Download"
            data-track-action="Click download article PDF"
            data-track-label="10.1186/s41065-018-0058-4">Download PDF</a>
        </p>
    
                            
                            
    <button type="button"
       class="c-button c-button--secondary c-button--full-width c-button--expandable"
       data-toggle="collapse"
       data-target="export-citations"
       data-test="export-citations-button"
       data-sticky-update="true"
       aria-controls="export-citations"
       aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/s41065-018-0058-4"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0058-4?format=refman&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/s41065-018-0058-4"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0058-4?format=refman&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/s41065-018-0058-4"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0058-4?format=refman&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/s41065-018-0058-4"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0058-4?format=endnote&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/s41065-018-0058-4"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0058-4?format=endnote&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/s41065-018-0058-4"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0058-4?format=endnote&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/s41065-018-0058-4"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0058-4?format=bibtex&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/s41065-018-0058-4"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0058-4?format=bibtex&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/s41065-018-0058-4"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0058-4?format=bibtex&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
    </div>

                        </div>
                        
    
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
                                <div id="article-access-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/access-count/10.1186/s41065-018-0058-4 #accessCount-container"></div>
                                <div id="citation-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/citation-count/10.1186/s41065-018-0058-4 #count-container"></div>
                                <div id="altmetric-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/altmetric/10.1186/s41065-018-0058-4 #altmetric-container"></div>
                            </ul>
                        </div>
                        
    <div id="sociaMediaShare" class="p-separator u-js-shown">
        <h3 class="p-section-title">Share this article</h3>
        <ul id="shareButtons" class="c-social-links">
            <li class="c-social-links__item">
                <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-018-0058-4">
                    <span class="u-visually-hidden">Share on Twitter</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-twitter"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-018-0058-4">
                    <span class="u-visually-hidden">Share on Facebook</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-facebook"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click"
                        data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-018-0058-4">
                    <span class="u-visually-hidden">Share on LinkedIn</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-linkedin"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-018-0058-4">
                    <span class="u-visually-hidden">Share on Weibo</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-weibo"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnGooglePlus" class="js-btnShareOnGplus" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Google+" data-track-label="10.1186/s41065-018-0058-4">
                    <span class="u-visually-hidden">Share on Google Plus</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-gplus"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-018-0058-4">
                    <span class="u-visually-hidden">Share on Reddit</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-reddit"></use>
                    </svg>
                </button>
            </li>
        </ul>
    </div>
 
                        
    
        <div class="p-separator u-js-shown">
            <h3 class="p-section-title">See updates</h3>
            <button class="c-crossmark-button" data-target="crossmark" data-test="crossmark" data-track="click" data-track-category="Article Page" data-track-action="Click CrossMark" data-track-label="10.1186/s41065-018-0058-4">
                <img id="crossmark-icon" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" alt="Crossmark logo" />
            </button>
        </div>
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
                                <li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/s41065-018-0058-4" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=738&doi=10.1186%2Fs41065-018-0058-4&volume=155&issue=1&title=Genetic+diversity+and+structure+of+tea+plant+in+Qinba+area+in+China+by+three+types+of+molecular+markers&author_name=Yu+Zhang&start_page=1&end_page=12">Order reprint</a>

                                </li>
                            </ul>
                        </div>
                    </aside>
                    
    <div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">Hereditas</span>
                        </h4>
                        <p class="c-journal-footer__issn">ISSN: 1601-5223</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
                                
                                    <li class="c-journal-footer__contact-item">Submission enquiries: <a href="https://www.editorialmanager.com/here/" target="_blank">Access here and click Contact Us</a></li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a></li>
                                
                            </ul>
                        </div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class='tracker' style='display:none' src='/track/article/10.1186/s41065-018-0058-4' alt=""/>

         
    <footer>
        <div class="ctx-publisher-footer">
    <div class="c-navbar">
        <div class="u-container">
            <div class="c-navbar__logo u-margin-right-xl">
                <a href="https://www.biomedcentral.com">
                    <img src=/static/images/bmc/logos/logo-bmc-aj-594e03e20d.svg alt="BMC logo" data-test="navbar-logo-footer">

                </a>
            </div>
            <div class="c-navbar__content">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


            </div>
        </div>
    </div>
    <div class="c-publisher-footer" data-test="publisher-footer">
        <div class="u-container">
            
            <div class="c-flex c-flex--wrap c-flex--space-between" role="navigation" data-test="publisher-footer-menu">
                <div class="c-flex" aria-label="Publisher footer links">
                    
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/login">Receive BMC newsletters</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/account">Manage article alerts</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/policies">Policies</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/accessibility">Accessibility</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/press-centre">Press center</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/contact-us">Contact us</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/jobs">Careers</a>
                                    </li>
                                
                            </ul>
                        
                    
                </div>
                <div class="u-margin-bottom-lg">
                    <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                    <ul class="c-flex u-reset-list" data-test="footer-social-links">
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://twitter.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Twitter link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Twitter page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-twitter-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://www.facebook.com/BioMedCentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Facebook link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Facebook page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-facebook-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://plus.google.com/106226337984152901734/"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Google Plus link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Google Plus page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-gplus-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="http://www.weibo.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Weibo link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Weibo page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-weibo-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </div>
            <p class="u-reset-margin">
                By using this website, you agree to our
                <a class="u-text-inherit" href="//www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
                <a class="u-text-inherit" href="//www.biomedcentral.com/privacy-statement">Privacy
                    statement</a> and
                <a class="u-text-inherit" href="//www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
                
                    <a class="optanon-toggle-display u-text-inherit" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
                
            </p>
        </div>
    </div>
</div>


        <div class="c-sn-footer c-sn-footer--neutral">
    <div class="u-container">
        <img src=/static/images/bmc/logos/logo-springernature-8bd4e8d82f.svg class="c-sn-footer__logo" alt="Springer Nature Logo" itemprop="logo" >
        <p data-test="copyright">&#169; 2018 BioMed Central Ltd unless otherwise stated. Part of
            <a href="http://www.springernature.com"
               data-track="click"
               data-track-action="Click Springer Nature link"
               data-track-category="Corporate footer"
               itemscope itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>

    </div>

    
    <script type="text/javascript" src="//cdn.pbgrd.com/bmc.js" async="async"></script>
    <noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>
    <!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker -->

<script id="springerlink-event-tracker" src="https://event-tracker.springernature.com/dist/eventTracker.js"></script>
<script type="text/javascript">
    (function(){
        var dataLayer = window.dataLayer[0];
        var doi = dataLayer.content.article.doi;
        var imprint = dataLayer.content.contentInfo.imprint;
        var bpids = dataLayer.session.authentication.authenticationID;
        var publisher;
        var pageType = dataLayer.page.category.pageType;
        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }
        var eventObject = {'content_type' : pageType,'doi' : doi};

        if (imprint === 'BioMed Central') { publisher = 'BMC' }
        else if (imprint === 'SpringerOpen') { publisher = 'SpringerOpen' }
        else { throw new Error('Event Tracker Error: The publisher has an unexpected value') }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }
        new EventTracker({'platform':publisher}).sendEvent('display', eventObject);
    })();
</script>


    </body>
</html>

