<!DOCTYPE html>
<html lang="en" class="no-js">
    <head>
        <title>Hybrid identification and genetic variation of Elymus sibiricus hybrid populations using EST-SSR markers | Hereditas | Full Text</title>

        

    <meta name="citation_abstract" content="
                           Elymus sibiricus is an important native grass in Qinghai-Tibetan Plateau. Seed shattering is a serious problem for E. sibiricus, especially at harvest time. Cross breeding is an effective way to create new varieties with beneficial characteristic or improved traits, and to broaden genetic base. In this study, we created five hybrid populations by crossing seven E. sibiricus genotypes that have seed shattering variation. Then, nine EST-SSR primers were used for hybrid identification based on DNA fingerprinting, and genetic diversity analysis of hybrid populations and their respective parents. A total of 15 hybrids were identified. An analysis of amplified polymorphic bands among genuine hybrids and their respective parents revealed mainly two types of markers: 1) hybrids shared bands exclusively amplified in both parents; 2)hybrids shared bands exclusively amplified in male parents. For each hybrid population, the total number of amplified bands ranged from 37 to 57, the percentage of polymorphism varied from 65.12% to 75.68%, with an average of 70.51%. Novel bands found in each hybrid population varied from 0 to 9 bands, suggesting an occurrence of rearrangements in the hybrid population. The structure analysis revealed that all hybrid populations and parents were assigned to eight groups. The principal coordinate analysis (PCoA) showed similar results. In general, this study proved EST-SSR markers are efficient for hybrid identification, and suggested more genetic variation could be captured in hybrid populations by crossing breeding."/>

    <meta name="description" content="
                           Elymus sibiricus is an important native grass in Qinghai-Tibetan Plateau. Seed shattering is a serious problem for E. sibiricus, especially at harvest time. Cross breeding is an effective way to create new varieties with beneficial characteristic or improved traits, and to broaden genetic base. In this study, we created five hybrid populations by crossing seven E. sibiricus genotypes that have seed shattering variation. Then, nine EST-SSR primers were used for hybrid identification based on DNA fingerprinting, and genetic diversity analysis of hybrid populations and their respective parents. A total of 15 hybrids were identified. An analysis of amplified polymorphic bands among genuine hybrids and their respective parents revealed mainly two types of markers: 1) hybrids shared bands exclusively amplified in both parents; 2)hybrids shared bands exclusively amplified in male parents. For each hybrid population, the total number of amplified bands ranged from 37 to 57, the percentage of polymorphism varied from 65.12% to 75.68%, with an average of 70.51%. Novel bands found in each hybrid population varied from 0 to 9 bands, suggesting an occurrence of rearrangements in the hybrid population. The structure analysis revealed that all hybrid populations and parents were assigned to eight groups. The principal coordinate analysis (PCoA) showed similar results. In general, this study proved EST-SSR markers are efficient for hybrid identification, and suggested more genetic variation could be captured in hybrid populations by crossing breeding."/>

    <meta name="journal_id" content="41065"/>

    <meta name="dc.title" content="Hybrid identification and genetic variation of Elymus sibiricus hybrid populations using EST-SSR markers"/>

    <meta name="dc.source" content="Hereditas 2017 154:1"/>

    <meta name="dc.format" content="text/html"/>

    <meta name="dc.publisher" content="BioMed Central"/>

    <meta name="dc.date" content="2017-12-12"/>

    <meta name="dc.type" content="OriginalPaper"/>

    <meta name="dc.language" content="En"/>

    <meta name="dc.copyright" content="2017 The Author(s)"/>

    <meta name="dc.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="dc.description" content="
                           Elymus sibiricus is an important native grass in Qinghai-Tibetan Plateau. Seed shattering is a serious problem for E. sibiricus, especially at harvest time. Cross breeding is an effective way to create new varieties with beneficial characteristic or improved traits, and to broaden genetic base. In this study, we created five hybrid populations by crossing seven E. sibiricus genotypes that have seed shattering variation. Then, nine EST-SSR primers were used for hybrid identification based on DNA fingerprinting, and genetic diversity analysis of hybrid populations and their respective parents. A total of 15 hybrids were identified. An analysis of amplified polymorphic bands among genuine hybrids and their respective parents revealed mainly two types of markers: 1) hybrids shared bands exclusively amplified in both parents; 2)hybrids shared bands exclusively amplified in male parents. For each hybrid population, the total number of amplified bands ranged from 37 to 57, the percentage of polymorphism varied from 65.12% to 75.68%, with an average of 70.51%. Novel bands found in each hybrid population varied from 0 to 9 bands, suggesting an occurrence of rearrangements in the hybrid population. The structure analysis revealed that all hybrid populations and parents were assigned to eight groups. The principal coordinate analysis (PCoA) showed similar results. In general, this study proved EST-SSR markers are efficient for hybrid identification, and suggested more genetic variation could be captured in hybrid populations by crossing breeding."/>

    <meta name="prism.issn" content="1601-5223"/>

    <meta name="prism.publicationName" content="Hereditas"/>

    <meta name="prism.publicationDate" content="2017-12-12"/>

    <meta name="prism.volume" content="154"/>

    <meta name="prism.number" content="1"/>

    <meta name="prism.section" content="OriginalPaper"/>

    <meta name="prism.startingPage" content="15"/>

    <meta name="prism.copyright" content="2017 The Author(s)"/>

    <meta name="prism.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="prism.url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0053-1"/>

    <meta name="prism.doi" content="doi:10.1186/s41065-017-0053-1"/>

    <meta name="citation_pdf_url" content="https://hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-017-0053-1"/>

    <meta name="citation_fulltext_html_url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0053-1"/>

    <meta name="citation_journal_title" content="Hereditas"/>

    <meta name="citation_publisher" content="BioMed Central"/>

    <meta name="citation_issn" content="1601-5223"/>

    <meta name="citation_title" content="Hybrid identification and genetic variation of Elymus sibiricus hybrid populations using EST-SSR markers"/>

    <meta name="citation_volume" content="154"/>

    <meta name="citation_issue" content="1"/>

    <meta name="citation_publication_date" content="2017/12"/>

    <meta name="citation_online_date" content="2017/12/12"/>

    <meta name="citation_firstpage" content="15"/>

    <meta name="citation_article_type" content="Research"/>

    <meta name="citation_fulltext_world_readable" content=""/>

    <meta name="dc.identifier" content="doi:10.1186/s41065-017-0053-1"/>

    <meta name="DOI" content="10.1186/s41065-017-0053-1"/>

    <meta name="citation_doi" content="10.1186/s41065-017-0053-1"/>

    <meta name="dc.creator" content="Xuhong Zhao"/>

    <meta name="dc.creator" content="Junchao Zhang"/>

    <meta name="dc.creator" content="Zongyu Zhang"/>

    <meta name="dc.creator" content="Yanrong Wang"/>

    <meta name="dc.creator" content="Wengang Xie"/>

    <meta name="dc.subject" content="Animal Genetics and Genomics"/>

    <meta name="dc.subject" content="Evolutionary Biology"/>

    <meta name="citation_reference" content="citation_journal_title=Conserv Genet Resour; citation_title=Development and characterization of 53 polymorphic genomic-SSR markers in Siberian wild rye (Elymus sibiricus L.); citation_author=YT Lei, YY Zhao, F Yu, Y Li, Q Dou; citation_volume=6; citation_publication_date=2014; citation_pages=861-864; citation_doi=10.1007/s12686-014-0225-5; citation_id=CR1"/>

    <meta name="citation_reference" content="citation_journal_title=Molecules; citation_title=Assessment of worldwide genetic diversity of Siberian wild rye (Elymus sibiricus L.) germplasm based on gliadin analysis; citation_author=X Ma, SY Chen, XQ Zhang, SQ Bai, CB Zhang; citation_volume=17; citation_publication_date=2012; citation_pages=4424-4434; citation_doi=10.3390/molecules17044424; citation_id=CR2"/>

    <meta name="citation_reference" content="citation_journal_title=Club; citation_title=Recognition of Elymus sibiricus L. from Alaska and the district of Mackenzie. Bull Torrey Bot; citation_author=WM Bowden, WJ Cody; citation_volume=88; citation_publication_date=1961; citation_pages=153-155; citation_doi=10.2307/2482709; citation_id=CR3"/>

    <meta name="citation_reference" content="citation_journal_title=Biochem Syst Ecol; citation_title=Assessment of genetic diversity of Siberian wild rye (Elymus sibiricus L.) germplasms with variation of seed shattering and implication for future genetic improvement; citation_author=WG Xie, XH Zhao, JQ Zhang, YR Wang, WX Liu; citation_volume=58; citation_publication_date=2015; citation_pages=211-218; citation_doi=10.1016/j.bse.2014.12.006; citation_id=CR4"/>

    <meta name="citation_reference" content="citation_journal_title=Chin Bull Bot; citation_title=Genetic diversity of Elymus sibiricus and its breeding in China; citation_author=JJ Yan, SQ Bai, X Ma, YM Gan, JB Zhang; citation_volume=24; citation_publication_date=2007; citation_pages=226-231; citation_id=CR5"/>

    <meta name="citation_reference" content="citation_journal_title=Molecules; citation_title=Assessing and broadening genetic diversity of Elymus sibiricus germplasm for the improvement of seed shattering; citation_author=ZY Zhang, JC Zhang, XH Zhao, WG Xie, YR Wang; citation_volume=21; citation_issue=7; citation_publication_date=2016; citation_pages=869; citation_doi=10.3390/molecules21070869; citation_id=CR6"/>

    <meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=Resynthesized lines from domesticated and wild Brassica taxa and their hybrids with B. napus L.: genetic diversity and hybrid yield; citation_author=T Jesske, B Olberg, A Schierholy, HC Becker; citation_volume=126; citation_publication_date=2013; citation_pages=1053-1065; citation_doi=10.1007/s00122-012-2036-y; citation_id=CR7"/>

    <meta name="citation_reference" content="citation_journal_title=Agr Sci China; citation_title=Relationship between hybrid performance and genetic diversity based on SSRs and ISSRs in Brassica napus L; citation_author=JX Shen, TD Fu, GS Yang; citation_volume=2; citation_publication_date=2003; citation_pages=1083-1090; citation_id=CR8"/>

    <meta name="citation_reference" content="citation_journal_title=Genet Mol Res; citation_title=Genetic diversity and structure of natural Dactylis glomerata L. populations revealed by morphological and microsatellite-based (SSR/ISSR) markers; citation_author=P Madesis, EM Abraham, A Kalivas, I Ganopoulos, A Tsaftaris; citation_volume=13; citation_publication_date=2014; citation_pages=4226-4240; citation_doi=10.4238/2014.June.9.8; citation_id=CR9"/>

    <meta name="citation_reference" content="citation_journal_title=J Plant Res; citation_title=Diagnostic set of microsatellite markers for hybrid purity testing and molecular identification of hybrids and parental lines in sorghum
                           ; citation_author=A Lalit, V Manjusha, L Suman; citation_volume=1; citation_publication_date=2014; citation_pages=1-4; citation_id=CR10"/>

    <meta name="citation_reference" content="citation_journal_title=Genet Mol Res; citation_title=Morphological and genetic characteristics of hybrid combinations of Dactylis glomerata
                           ; citation_author=YF Zhao, XQ Zhang, X Ma, WG Xie, LK Huang; citation_volume=13; citation_publication_date=2014; citation_pages=2491-2503; citation_doi=10.4238/2014.January.28.1; citation_id=CR11"/>

    <meta name="citation_reference" content="citation_journal_title=Am J Exp Agri; citation_title=Application of SSR markers for genetic purity analysis of parental inbred lines and some commercial hybrid maize (Zea mays L.); citation_author=IO Daniel, JA Adetumbi, O Oyelakin, SA Olakojo, MO Ajala, SO Onagbesan; citation_volume=2; citation_publication_date=2012; citation_pages=597-606; citation_id=CR12"/>

    <meta name="citation_reference" content="citation_journal_title=Euphytica; citation_title=Identification of informative SSR markers capable of distinguishing hybrid Rice parental lines and their utilization in seed purity assessment; citation_author=RM Sundaram, B Naveenkumar, SK Biradar, SM Balachandran, B Mishra, M IlyasAhmed; citation_volume=163; citation_publication_date=2008; citation_pages=215-224; citation_doi=10.1007/s10681-007-9630-0; citation_id=CR13"/>

    <meta name="citation_reference" content="citation_journal_title=Genet Mol Res; citation_title=Identification of sunflower (Helianthus annuus, Asteraceae) hybrids using simple-sequence repeat markers; citation_author=A Iqbal, HA Sadaqat, AS Khan, M Amjad; citation_volume=10; citation_publication_date=2010; citation_pages=102-106; citation_doi=10.4238/vol10-1gmr918; citation_id=CR14"/>

    <meta name="citation_reference" content="citation_journal_title=Hortscience; citation_title=Evaluation of genetic purity of F1 hybrid seeds in cabbage with RAPD, ISSR, SRAP, and SSR markers; citation_author=LW Liu, G Liu, YQ Gong, WH Dai, Y Wang, FM Yu; citation_volume=2; citation_publication_date=2007; citation_pages=724-727; citation_id=CR15"/>

    <meta name="citation_reference" content="citation_journal_title=Breeding Sci; citation_title=Considerable heterogeneity in commercial F1 varieties of bunching onion (Allium fistulosum) and the proposal of a breeding scheme for conferring genetic traceability using SSR markers; citation_author=H Tsukazaki, H Fukuoka, YS Song, K Yamashita, T Wako, A Kojima; citation_volume=56; citation_publication_date=2006; citation_pages=321-326; citation_doi=10.1270/jsbbs.56.321; citation_id=CR16"/>

    <meta name="citation_reference" content="citation_journal_title=Seed Sci Technol; citation_title=Genetic purity testing of loose-curd cauliflower hybrids using SSR markers and grow out test; citation_author=Z Zhao, H Gu, X Sheng, H Yu, J Wang, J Cao; citation_volume=40; citation_publication_date=2012; citation_pages=209-214; citation_doi=10.15258/sst.2012.40.2.06; citation_id=CR17"/>

    <meta name="citation_reference" content="citation_journal_title=Acta Pratac Sin.; citation_title=Identification and genetic variation analysis of Orchardgrass hybrids (Dactylis glomerata) by SSR molecular markers; citation_author=WG Xie, XQ Zhang, YX Cheng; citation_volume=19; citation_publication_date=2010; citation_pages=212-217; citation_id=CR18"/>

    <meta name="citation_reference" content="citation_journal_title=Pratac Sci; citation_title=Distinguishing Elymus nutans from Elymus sibiricus in Qinghai - Tibet plateau using a SSR marker; citation_author=YT Lei, QW Dou; citation_volume=29; citation_publication_date=2012; citation_pages=937-942; citation_id=CR19"/>

    <meta name="citation_reference" content="citation_journal_title=Bioch Syst Ecol; citation_title=Siberian wild rye ( Elymus sibiricus, l.): genetic diversity of germplasm determined using DNA fingerprinting and SCoT markers; citation_author=WG Xie, JC Zhang, XH Zhao, JQ Zhang, YR Wang; citation_volume=60; citation_publication_date=2015; citation_pages=186-192; citation_doi=10.1016/j.bse.2015.04.021; citation_id=CR20"/>

    <meta name="citation_reference" content="citation_journal_title=Guangdong Agri Sci; citation_title=Improved SDS method for general plant genomic DNA extraction; citation_author=Z Shan, HL Wu, CL Li, H Chen, Q Wu; citation_volume=8; citation_publication_date=2011; citation_pages=113-115; citation_id=CR21"/>

    <meta name="citation_reference" content="citation_journal_title=Genetics; citation_title=Inference of population structure from multilocus genotype data; citation_author=JK Pritchard, M Stephens, P Donnelly; citation_volume=155; citation_publication_date=2000; citation_pages=945-959; citation_id=CR22"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Ecol; citation_title=Detecting the number of clusters of individuals using the software structure: a simulation study; citation_author=G Evanno, S Regnaut, J Goudet; citation_volume=14; citation_publication_date=2005; citation_pages=2611-2620; citation_doi=10.1111/j.1365-294X.2005.02553.x; citation_id=CR23"/>

    <meta name="citation_reference" content="citation_title=NTSYS-pc: numerical taxonomy and multivariate analysis system; citation_publication_date=1992; citation_id=CR24; citation_author=FJ Rohlf; citation_publisher=Exeter Software"/>

    <meta name="citation_reference" content="citation_journal_title=J Appl Genet; citation_title=Potential of start Codon targeted (SCoT) markers for DNA fingerprinting of newly synthesized tritordeums and their respective parents; citation_author=SL Cabo, A Carvalho, LP Martins, A Mart&#237;n, JE Lima-Brito; citation_volume=55; citation_publication_date=2014; citation_pages=307-312; citation_doi=10.1007/s13353-014-0211-3; citation_id=CR25"/>

    <meta name="citation_reference" content="citation_journal_title=Genet Resour Crop Ev; citation_title=Constructing DNA fingerprinting of Hemarthria cultivars using EST-SSR and SCoT markers; citation_author=LK Huang, X Huang, HD Yan, GH Yin, XQ Zhang, Y Tian; citation_volume=61; citation_publication_date=2014; citation_pages=1047-1055; citation_doi=10.1007/s10722-014-0107-4; citation_id=CR26"/>

    <meta name="citation_reference" content="citation_journal_title=Trends Plant Sci; citation_title=Polymorphism revealed by simple sequence repeats; citation_author=W Powell, GC Machray, P Jim; citation_volume=7; citation_publication_date=1996; citation_pages=215-222; citation_doi=10.1016/S1360-1385(96)86898-0; citation_id=CR27"/>

    <meta name="citation_reference" content="citation_journal_title=Acta Pratac Sin; citation_title=Genetic diversity of wild Elymus sibiricus germplasm from the Qinghai-Tibetan plateau in China detected by SRAP markers; citation_author=JJ Yan, SQ Bai, XQ Zhang, MH You, CB Zhang, DX Li; citation_volume=19; citation_publication_date=2010; citation_pages=173-183; citation_id=CR28"/>

    <meta name="citation_reference" content="citation_journal_title=Molecules; citation_title=Potential of start Codon targeted (SCoT) markers to estimate genetic diversity and relationships among Chinese Elymus sibiricus accessions; citation_author=JC Zhang, WG Xie, YR Wang, XH Zhao; citation_volume=20; citation_publication_date=2015; citation_pages=5987-6001; citation_doi=10.3390/molecules20045987; citation_id=CR29"/>

    <meta name="citation_author" content="Xuhong Zhao"/>

    <meta name="citation_author_institution" content="The State Key Laboratory of Grassland Agro-ecosystems, Key Laboratory of Grassland Livestock Industry Innovation, Ministry of Agriculture, College of Pastoral Agriculture Science and Technology, Lanzhou University, Lanzhou, People&#8217;s Republic of China"/>

    <meta name="citation_author" content="Junchao Zhang"/>

    <meta name="citation_author_institution" content="The State Key Laboratory of Grassland Agro-ecosystems, Key Laboratory of Grassland Livestock Industry Innovation, Ministry of Agriculture, College of Pastoral Agriculture Science and Technology, Lanzhou University, Lanzhou, People&#8217;s Republic of China"/>

    <meta name="citation_author" content="Zongyu Zhang"/>

    <meta name="citation_author_institution" content="The State Key Laboratory of Grassland Agro-ecosystems, Key Laboratory of Grassland Livestock Industry Innovation, Ministry of Agriculture, College of Pastoral Agriculture Science and Technology, Lanzhou University, Lanzhou, People&#8217;s Republic of China"/>

    <meta name="citation_author" content="Yanrong Wang"/>

    <meta name="citation_author_institution" content="The State Key Laboratory of Grassland Agro-ecosystems, Key Laboratory of Grassland Livestock Industry Innovation, Ministry of Agriculture, College of Pastoral Agriculture Science and Technology, Lanzhou University, Lanzhou, People&#8217;s Republic of China"/>

    <meta name="citation_author" content="Wengang Xie"/>

    <meta name="citation_author_institution" content="The State Key Laboratory of Grassland Agro-ecosystems, Key Laboratory of Grassland Livestock Industry Innovation, Ministry of Agriculture, College of Pastoral Agriculture Science and Technology, Lanzhou University, Lanzhou, People&#8217;s Republic of China"/>



        <meta charset="UTF-8"/>
        <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
        <meta name="viewport" content="width=device-width, initial-scale=1"/>

        <!-- For IE 9 and below -->
<!--[if IE]><link rel="shortcut icon" href=/static/images/favicons//favicon.ico><![endif]-->
<!-- Touch Icons - iOS and Android 2.1+ -->
<link rel="apple-touch-icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>
<!-- Firefox, Chrome, Safari, IE 11+ and Opera -->
<link rel="icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>


        <script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script>
        <script>(function(d){if (sessionStorage.fontsLoaded){d.style.setProperty('--font-family-sans', 'Europa');d.style.setProperty('--font-family-serif', 'Noto Serif')}})(document.documentElement);</script>

        <link rel="stylesheet" media="screen" href=/static/styles/basic/basic-a31ca85bc2.css>
<link rel="stylesheet" media="print" href=/static/styles/bmc/print-3a6240f456.css>

<link rel="stylesheet" class="js-ctm" href=/static/styles/bmc/main-84414d3d47.css
      media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">

        

        <script src="/static/js/app-bundle-8d61e6ef37.js" defer></script>
        
        <script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script>
        
    <script type="text/javascript">
        config = {
            env: 'live',
            site: 'hereditasjournal.biomedcentral.com',
            siteWithPath: 'hereditasjournal.biomedcentral.com' + window.location.pathname,
            twitterHashtag: '',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/s41065-017-0053-1',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central'
        };
    </script>

        
    <script type="text/javascript">
        dataLayer = [{"content":{"article":{"doi":"10.1186/s41065-017-0053-1","articleType":"Research","peerReviewType":"Closed","supplement":null,"keywords":"Elymus sibiricus;EST-SSR marker;Hybrid identification;Genetic diversity"},"contentInfo":{"imprint":"BioMed Central","title":"Hybrid identification and genetic variation of Elymus sibiricus hybrid populations using EST-SSR markers","publishedAt":1513036800000,"publishedAtDate":"2017-12-12","author":["Xuhong Zhao","Junchao Zhang","Zongyu Zhang","Yanrong Wang","Wengang Xie"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":"CC BY + CC0","openAccess":true},"environment":"live"},"journal":{"siteKey":"hereditasjournal.biomedcentral.com","volume":"154","issue":"1","title":"Hereditas","type":"AJs Life","journalID":41065,"gaCode":"UA-62066850-11","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Research","publishingSegment":"Genetics & Heredity (Life)"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}}}]

    </script>

        <script src="https://cdn.cookielaw.org/consent/7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script>
        
    
        
    

        
    
        
            
                <!-- Google Tag Manager -->
                <script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script>
                <!-- End Google Tag Manager -->
            
        
    


        
    
    <link rel="canonical" href="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0053-1"/>
    

        
        
        <meta property="og:url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-017-0053-1"/>
        <meta property="og:type" content="article"/>
        <meta property="og:site_name" content="Hereditas"/>
        <meta property="og:title" content="Hybrid identification and genetic variation of Elymus sibiricus hybrid populations using EST-SSR markers"/>
        <meta property="og:description" content="Elymus sibiricus is an important native grass in Qinghai-Tibetan Plateau. Seed shattering is a serious problem for E. sibiricus, especially at harvest time. Cross breeding is an effective way to create new varieties with beneficial characteristic or improved traits, and to broaden genetic base. In this study, we created five hybrid populations by crossing seven E. sibiricus genotypes that have seed shattering variation. Then, nine EST-SSR primers were used for hybrid identification based on DNA fingerprinting, and genetic diversity analysis of hybrid populations and their respective parents. A total of 15 hybrids were identified. An analysis of amplified polymorphic bands among genuine hybrids and their respective parents revealed mainly two types of markers: 1) hybrids shared bands exclusively amplified in both parents; 2)hybrids shared bands exclusively amplified in male parents. For each hybrid population, the total number of amplified bands ranged from 37 to 57, the percentage of polymorphism varied from 65.12% to 75.68%, with an average of 70.51%. Novel bands found in each hybrid population varied from 0 to 9 bands, suggesting an occurrence of rearrangements in the hybrid population. The structure analysis revealed that all hybrid populations and parents were assigned to eight groups. The principal coordinate analysis (PCoA) showed similar results. In general, this study proved EST-SSR markers are efficient for hybrid identification, and suggested more genetic variation could be captured in hybrid populations by crossing breeding."/>
        <meta property="og:image" content="https://images.springer.com/sgw/journals/medium/41065.jpg"/>
    
    </head>

    <body class="journal journal-fulltext"
          
          >
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK"
                            height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><feColorMatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"/></filter></defs><symbol id="icon-arrow-down" viewBox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"/></symbol><symbol id="icon-arrow-left-bullet" viewBox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"/></symbol><symbol id="icon-arrow-right-bullet" viewBox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"/></symbol><symbol id="icon-arrow-right" viewBox="0 0 32 64"><path d="M0 0v64l32-32z"/></symbol><symbol id="icon-close" viewBox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"/></symbol><symbol id="icon-community" viewBox="0 0 60.643 31.01"><mask maskUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><path fill="#FFF" d="M0 31.01h60.643V0H0z" filter="url(#a)"/></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"/></symbol><symbol id="icon-download" viewBox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"/></symbol><symbol id="icon-ext-link" viewBox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"/><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"/><path fill="#676767" d="M9.7 0L16 6.2V0z"/></symbol><symbol id="icon-info-bordered" viewBox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"/><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"/></symbol><symbol id="icon-opr" viewBox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"/><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"/></symbol><symbol id="icon-quotation" viewBox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"/></symbol><symbol id="icon-remove" viewBox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"/><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"/></symbol><symbol id="icon-ribbon" viewBox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"/></symbol><symbol id="icon-rss" viewBox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"/><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"/><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"/></g></symbol><symbol id="icon-search" viewBox="0 0 16 16"><path d="M6.276 12.257C2.815 12.257 0 9.508 0 6.128 0 2.749 2.815 0 6.276 0c3.462 0 6.277 2.749 6.277 6.128.002 3.38-2.815 6.129-6.277 6.129zm0-10.772c-2.622 0-4.755 2.083-4.755 4.643 0 2.562 2.133 4.644 4.755 4.644 2.623 0 4.756-2.082 4.756-4.644 0-2.56-2.134-4.643-4.756-4.643z"/><path d="M16 13.751L13.641 16l-4.875-4.82 2.36-2.248L16 13.751z"/></symbol><symbol id="icon-speech-bubble" viewBox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"/></symbol><symbol id="icon-error" viewBox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"/></symbol><symbol id="icon-info" viewBox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"/></symbol><symbol id="icon-success" viewBox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"/></symbol><symbol id="icon-warning" viewBox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"/></symbol><symbol id="icon-facebook-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"/></symbol><symbol id="icon-facebook" viewBox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"/><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"/></symbol><symbol id="icon-gplus-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M480.5 273.117c.123.076.26.185.416.334.146.152.289.343.432.562.135.211.258.459.359.746.086.286.129.619.129 1.004-.012.7-.164 1.262-.465 1.683a5.374 5.374 0 0 1-.461.562c-.186.176-.375.35-.58.527a2.65 2.65 0 0 0-.322.41.908.908 0 0 0-.176.553c0 .205.059.375.18.508.102.129.199.238.299.336l.662.545a6.48 6.48 0 0 1 1.086 1.119c.293.415.445.955.459 1.625 0 .949-.42 1.789-1.256 2.521-.869.76-2.123 1.15-3.762 1.177-1.373-.015-2.4-.308-3.074-.879-.682-.534-1.023-1.172-1.023-1.917 0-.359.111-.766.336-1.211.215-.443.602-.837 1.172-1.174a6.126 6.126 0 0 1 1.998-.725 15.415 15.415 0 0 1 1.723-.173 3.996 3.996 0 0 1-.377-.596 1.347 1.347 0 0 1-.193-.74c0-.174.023-.317.074-.433a8.83 8.83 0 0 1 .121-.338 5.973 5.973 0 0 1-.633.035c-1.043-.012-1.838-.339-2.381-.979-.574-.603-.861-1.3-.861-2.097 0-.961.404-1.832 1.217-2.616.559-.46 1.135-.757 1.736-.898a8.494 8.494 0 0 1 1.67-.182h3.916l-1.209.704h-1.207v.005l-.005.002zm.754 11.276c0-.498-.162-.932-.484-1.302-.346-.348-.881-.778-1.609-1.29a4.045 4.045 0 0 0-.439-.02c-.102-.016-.354 0-.768.039a8.141 8.141 0 0 0-1.244.275c-.1.036-.24.096-.42.17a2.713 2.713 0 0 0-.553.355 1.985 1.985 0 0 0-.459.592c-.145.25-.215.551-.215.901 0 .687.311 1.253.932 1.698.596.444 1.404.68 2.432.688.928-.014 1.629-.217 2.117-.612.476-.385.71-.886.71-1.494zm-2.729-5.743a1.843 1.843 0 0 0 1.289-.556c.164-.249.271-.506.318-.767a7.32 7.32 0 0 0 .041-.659c0-.772-.195-1.552-.594-2.336a2.611 2.611 0 0 0-.73-.921 1.894 1.894 0 0 0-1.064-.352 1.723 1.723 0 0 0-1.324.64c-.299.437-.438.92-.428 1.457 0 .709.209 1.444.623 2.211.201.356.459.659.775.912.323.248.686.371 1.094.371zm9.698.077h-2.055v-2.056h-.996v2.056h-2.053v.998h2.053v2.052h.996v-2.052h2.055z"/></symbol><symbol id="icon-gplus" viewBox="0 0 24 24"><circle fill="#D84A36" cx="12" cy="12" r="12"/><g fill="#FFF"><path d="M12.012 12.111c-.304-.33-.07-.842.155-1.078.226-.241.639-.56.829-.74.193-.184.66-.605.674-1.868 0 0 .112-1.139-1.127-2.06h.977l1.038-.593h-3.507s-2.083-.067-3.085 1.207c-1 1.276-.966 2.871.114 3.77 1.08.898 2.371.648 2.371.648s-.197.422-.163.775c.036.353.421.849.504.965 0 0-2.386-.011-3.551 1.033-1.165 1.053-1.29 3.097 1.021 3.803 2.312.707 5.303-.055 5.906-2.277 0 0 .262-.908-.189-1.683-.459-.775-1.663-1.571-1.967-1.902zm-2.555-1.83c-.744-1-1.308-2.953 0-3.783 0 0 .955-.579 1.801.266.846.846 1.082 2.277.928 3.003-.15.727-.563.897-.739 1.008-.185.111-1.249.504-1.99-.494zm3.668 5.481c-.035.393-.083 1.254-1.491 1.664-1.409.408-3.826-.264-3.757-1.914 0 0 .006-.698.449-1.089.445-.396 1.094-.794 3.029-.819 0 0 .643.473 1.072.809.427.332.733.952.698 1.349z"/><path d="M16.323 11.055V9.803h-.752v1.252h-1.255v.748h1.255v1.256h.752v-1.256h1.248v-.748z"/></g></symbol><symbol id="icon-linkedin" viewBox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"/><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"/><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"/></g></symbol><symbol id="icon-reddit" viewBox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"/><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"/><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"/><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"/><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"/><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"/><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"/><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"/></symbol><symbol id="icon-twitter-bordered" viewBox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"/><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/></symbol><symbol id="icon-twitter" viewBox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"/><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"/></symbol><symbol id="icon-weibo-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"/><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"/><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"/></symbol><symbol id="icon-weibo" viewBox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"/><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"/><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"/><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"/><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"/><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"/></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner c-banner--info">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner" >
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1"
                 data-gpt-unitpath="/270604982/bmc/hereditasjournal/articles"
                 data-gpt-sizes="728x90,970x90"
                 data-gpt-targeting="pos=LB1;doi=10.1186/s41065-017-0053-1;kwrd=Elymus sibiricus,EST-SSR marker,Hybrid identification,Genetic diversity;pmc=L32030,L21001;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-017-0053-1&amp;kwrd=Elymus sibiricus,EST-SSR marker,Hybrid identification,Genetic diversity&amp;pmc=L32030,L21001&amp;">
                        <img data-test="gpt-advert-fallback-img"
                             src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-017-0053-1&amp;kwrd=Elymus sibiricus,EST-SSR marker,Hybrid identification,Genetic diversity&amp;pmc=L32030,L21001&amp;"
                             alt="Advertisement"
                             width="728"
                             height="90">
                    </a>
                </noscript>
            </div>
        </div>
    </div>

        
         
    <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>


<header class="c-popup-search ctx-publisher-header" id="top" data-test="publisher-header">
    <nav class="c-navbar">
        <div class="u-container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization" />
                <div class="c-logo">
    <a href="https://www.biomedcentral.com" itemprop="url">
        <img src=/static/images/bmc/logos/logo-bmc-white-aj-e3a70ae848.svg alt="BMC" itemprop="logo">
        <div class="c-logo__strapline">
            <img src=/static/images/bmc/logos/logo-bmc-white-strapline-sn-e7b1d6e67f.svg alt="Part of Springer Nature">
        </div>
    </a>
</div>
            </div>
            <button
                type="button"
                class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg"
                id="publisher-header-menu-button"
                data-toggle="collapse"
                data-test="publisher-header-menu-toggle"
                data-toggle-linked="publisher-header"
                data-target="publisher-header-search publisher-header-menu"
                aria-controls="publisher-header-menu publisher-header-search"
                aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


                <ul class="c-navbar__nav">
                    <li class="c-navbar__item">
                        <a data-replace="true"
                           data-placeholder="/placeholder/v1/username"
                           class="c-navbar__link"
                           href="https://www.biomedcentral.com/login"
                           data-test="login-link"
                           data-track="click"
                           data-track-category="Account"
                           data-track-action="Click Login Link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                    <li class="c-navbar__item">
                        <button
                            type="button"
                            class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg"
                            data-toggle="collapse"
                            data-test="header-search-button"
                            data-target="publisher-header-search publisher-header-menu publisher-header-menu-button"
                            aria-controls="publisher-header-search publisher-header-menu"
                            aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true">
                                    <use xlink:href="#icon-search"></use>
                                </svg>
                            </span>
                        </button>
                    </li>
                </ul>
            </div>
        </div>
    </nav>
    <div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field c-form-field--inline" method="GET" action="//www.biomedcentral.com/search" data-track="submit"
        data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label data-track-label="" data-test="global-search">
        <label for="publisherSearch" class="u-visually-hidden">Search all BMC articles</label>
        <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text" value="" placeholder="Search all BMC articles"/>
        <button class="c-button c-button--secondary" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true">
        <use xlink:href="#icon-search"></use>
    </svg>
</button>


        <input type="hidden" name="searchType" value="publisherSearch"/>
    </form>
</div>

            </div>
        </div>
    </div>
</header>


        
            <header class="c-journal-header c-journal-header--hereditas ctx-journal-header">
                <div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="/">
    
        
    
    <span class="c-journal-title__text ">Hereditas</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="u-container">
                        
                            <button type="button"
                                    class="c-navbar__link c-dropdown u-hidden-at-lg"
                                    data-toggle="collapse"
                                    data-test="journal-header-menu"
                                    data-toggle-linked="journal-header"
                                    data-target="journal-menu"
                                    aria-controls="journal-menu"
                                    aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav">
                                    <ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/'>Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/about'>About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href='/articles'>Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/submission-guidelines'>Submission Guidelines</a>
                                            </li>
                                        
                                    </ul>
                                </nav>
                            </div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--ocean-light-blue">
                    
                </div>
            </header>
            
        

        <div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-content-layout c-content-layout--fulltext" data-component="sticky-container">
                <div class="c-content-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3><ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc=""><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Methods">Methods</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec6" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results">Results</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec10" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Discussion">Discussion</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li></ol></nav>
                </div>

                <main class="c-content-layout__main">
                    <article itemscope itemtype="http://schema.org/ScholarlyArticle">
                        
                        
    <div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
            <li class="c-article-identifiers__item" data-test="article-category">Research</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
    </div>

                        <div class="FulltextWrapper"><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">Hybrid identification and genetic variation of <em xmlns="" class="EmphasisTypeItalic">Elymus sibiricus</em> hybrid populations using EST-SSR markers</h1></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true"><ul class="u-listReset"><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Xuhong&nbsp;Zhao</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Junchao&nbsp;Zhang</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Zongyu&nbsp;Zhang</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Yanrong&nbsp;Wang</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup> and </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Wengang&nbsp;Xie</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup><a href="mailto:xiewg@lzu.edu.cn" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a></li></ul></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation"><div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">Hereditas</span></span><span class="ArticleCitation_Year">2017</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">154</strong>:15</span></span></div><p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/s41065-017-0053-1">https://doi.org/10.1186/s41065-017-0053-1</a></p><p class="Copyright"> &nbsp;The Author(s)&nbsp;2017</p></div></div><ul class="ArticleHistory u-reset-list"><li class="History HistoryReceived"><strong>Received: </strong><span>10&nbsp;March&nbsp;2017</span></li><li class="History HistoryAccepted"><strong>Accepted: </strong><span>28&nbsp;November&nbsp;2017</span></li><li class="History HistoryOnlineDate"><strong>Published: </strong><span itemprop="datePublished">12&nbsp;December&nbsp;2017</span></li></ul></div>
                        
                        
                        
    

                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2><div class="c-section__content" id="Abstract"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p id="Par1" class="Para">
                           <em xmlns="" class="EmphasisTypeItalic">Elymus sibiricus</em> is an important native grass in Qinghai-Tibetan Plateau. Seed shattering is a serious problem for <em xmlns="" class="EmphasisTypeItalic">E. sibiricus</em>, especially at harvest time. Cross breeding is an effective way to create new varieties with beneficial characteristic or improved traits, and to broaden genetic base.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Results</h3>
                <p id="Par2" class="Para">In this study, we created five hybrid populations by crossing seven <em xmlns="" class="EmphasisTypeItalic">E. sibiricus</em> genotypes that have seed shattering variation. Then, nine EST-SSR primers were used for hybrid identification based on DNA fingerprinting, and genetic diversity analysis of hybrid populations and their respective parents. A total of 15 hybrids were identified. An analysis of amplified polymorphic bands among genuine hybrids and their respective parents revealed mainly two types of markers: 1) hybrids shared bands exclusively amplified in both parents; 2)hybrids shared bands exclusively amplified in male parents. For each hybrid population, the total number of amplified bands ranged from 37 to 57, the percentage of polymorphism varied from 65.12% to 75.68%, with an average of 70.51%. Novel bands found in each hybrid population varied from 0 to 9 bands, suggesting an occurrence of rearrangements in the hybrid population. The structure analysis revealed that all hybrid populations and parents were assigned to eight groups. The principal coordinate analysis (PCoA) showed similar results.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusions</h3>
                <p id="Par3" class="Para">In general, this study proved EST-SSR markers are efficient for hybrid identification, and suggested more genetic variation could be captured in hybrid populations by crossing breeding.</p>
              </div></div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords"><li class="c-keywords__item" itemprop="about">
                <em xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="EmphasisTypeItalic">Elymus sibiricus</em>
              </li><li class="c-keywords__item" itemprop="about">EST-SSR marker</li><li class="c-keywords__item" itemprop="about">Hybrid identification</li><li class="c-keywords__item" itemprop="about">Genetic diversity</li></ul></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par12" class="Para">
                        <em xmlns="" class="EmphasisTypeItalic">Elymus sibiricus</em> L., named also siberian wild rye, is a perennial, caespitose, gramineous <em xmlns="" class="EmphasisTypeItalic">Elymus</em> forage grass [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>], indigenous to Northern Asia [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>]. Its natural geographic distribution extends from Sweden to Japan and even to parts of Alaska and Canada [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>]. It is wildly utilized in cultivated pasture and natural grassland in north China, owing to its excellent tolerance to low temperature and drought, and good forage quality and palatability [<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par13" class="Para">In recent years, overgrazing and climate change resulted in grassland degeneration, it has created a need for revegetation/restoration of rangeland ecosystems in north China. As a native grass in north China, <em xmlns="" class="EmphasisTypeItalic">E.sibiricus</em> have the potential to be used in degenerated grassland due to its good adaptability to local environment. However, few <em xmlns="" class="EmphasisTypeItalic">E.sibiricus</em> cultivars are available for revegetation /restoration projects in these regions. Until now, progress in cultivar development and improvement in <em xmlns="" class="EmphasisTypeItalic">E.sibiricus</em> has seriously lagged behind crop plants even other forage grasses. In the last 20&nbsp;years, only 6 cultivars have been developed from wild materials [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>]. Meanwhile, this species has serious problem with seed shattering. In previous study, we have identified some low seed shattering genotypes from wild <em xmlns="" class="EmphasisTypeItalic">E.sibiricus</em> germplasm originated from northeastern Qinghai-Tibet Plateau [<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>]. These materials are valuable genetic resources for seed shattering improvement in future breeding program. Cross breeding is an effective way to create new varieties with beneficial characteristic or improved traits, and to broaden genetic base of <em xmlns="" class="EmphasisTypeItalic">E. sibiricus</em> [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>]. Generally, these resynthesized breeding materials are genetically diverse from inbred line/cultivar [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>], and might have higher heterosis than their parents [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>]. Traditionally, we identify hybrids and analyze genetic diversity through morphological traits such as plant height, inflorescence, leaf shape, etc. The process of identification is time-consuming, and the accuracy of morphological identification is also limited due to the influence of environment factors. Compared to conventional hybrid identification through morphological difference in the course of plant breeding, marker-assisted selection (MAS) is more efficient way as it is reproducible, reliable, and independent from environmental conditions, it therefore could be used to mirror directly genetic diversity [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par14" class="Para">Among the available molecular markers, SSRs (simple sequence repeats) or microsatellites, offer an important DNA marker system for hybrid purity testing because of their co-dominance, reproducibility, robustness, and multi-allelic nature [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>]. The characteristics of co-dominance give enormous advantages to SSR marker, which can be more clearly to present the band feature of parent plant and their offspring in plant breeding [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>]. SSR markers have been widely used to assess hybrid purity in maize [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>], rice [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>], sunflower [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>], cabbage [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>], bunching onion [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>], cauliflower [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>] and some forage grasses like orchardgrass [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>]. In addition, SSR markers have been successfully used to distinguish <em xmlns="" class="EmphasisTypeItalic">E. sibiricus</em> and <em xmlns="" class="EmphasisTypeItalic">E. nutans</em> originated from Qinghai-Tibet Plateau [<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>] and detect genetic diversity and variation of <em xmlns="" class="EmphasisTypeItalic">E. sibiricus</em> accessions worldwide [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>]. There are no previous reports of hybrid identification using EST-SSR markers. Genetic information on hybrid populations is limited in <em xmlns="" class="EmphasisTypeItalic">E.sibiricus</em>.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par15" class="Para">In this study, we used EST-SSR markers to identify hybrid based on specially amplified DNA fingerprinting and analyze genetic diversity of five hybrid populations and their respective parents in <em xmlns="" class="EmphasisTypeItalic">E. sibiricus</em>. This study will help lay a foundation for future <em xmlns="" class="EmphasisTypeItalic">E. sibiricus</em> breeding program.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Methods" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Methods">Methods</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Methods" data-test="collapse-section"><section id="Sec3" class="Section2 RenderAsSection2">
                <h3 class="Heading">Plant materials</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par16" class="Para">A total of seven <em xmlns="" class="EmphasisTypeItalic">E. sibiricus</em> accessions and their offsprings were used in this study (Tables&nbsp;<span class="InternalRef"><a href="#Tab1">1</a></span> and <span class="InternalRef"><a href="#Tab2">2</a></span>). These seven accessions had different geographical origins, they were genetically and morphological divergent. According to previous genetic diversity study these accessions were clustered to different groups based on SCoT markers [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>]. In addition, their selection was also based primarily on several contrasting agronomic traits: seed shattering, plant height, panicle length, etc. <em xmlns="" class="EmphasisTypeItalic">E. sibiricus</em> is not an endangered or protected species, thus, no permission was required for collecting these samples in China. Single genotype from each parental accession was used to generate the F<sub>1</sub> hybrids by hand pollination in June 2014. A total of 15&nbsp;F<sub>1</sub> individuals were derived from five pairs of cross (Table&nbsp;<span class="InternalRef"><a href="#Tab2">2</a></span>). F<sub>1</sub> seeds were harvested from the female parents in August 2014. The F<sub>1</sub> individuals were grown in a greenhouse at approximately 22&nbsp;C and a 16&nbsp;h photoperiod until they were 8&nbsp;weeks old. The F<sub>1</sub> individuals were then transplanted to the field in the campus experimental station at Lanzhou University, Yuzhong, Gansu, China (10334 E, 3534 N). No any permissions were required to carry out field experiment. A total of 15&nbsp;F<sub>1</sub> plants were tested by EST-SSR markers to confirm their hybrid status.<figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 1</span>
                        <p xmlns="" class="SimplePara">The origin and morphological traits of parents used in this study</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col></col><col></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Parents</p></th><th><p xmlns="" class="SimplePara">Status</p></th><th><p xmlns="" class="SimplePara">Origin</p></th><th><p xmlns="" class="SimplePara">Morphological characteristics</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">Y1005-1</p></td><td><p xmlns="" class="SimplePara">Wild</p></td><td><p xmlns="" class="SimplePara">Ruo ergai, Sichuan, China</p></td><td><p xmlns="" class="SimplePara">High seed shattering</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">LQ03-1</p></td><td><p xmlns="" class="SimplePara">Wild</p></td><td><p xmlns="" class="SimplePara">Luqu, Gansu, China</p></td><td><p xmlns="" class="SimplePara">Medium seed shattering</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">PI 348916</p></td><td><p xmlns="" class="SimplePara">Wild</p></td><td><p xmlns="" class="SimplePara">Alaska, United State</p></td><td><p xmlns="" class="SimplePara">Low seed shattering, Late flowering</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">XH09-4</p></td><td><p xmlns="" class="SimplePara">Wild</p></td><td><p xmlns="" class="SimplePara">Xiahe, Gansu, China</p></td><td><p xmlns="" class="SimplePara">High seed shattering</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">ZhN06-1</p></td><td><p xmlns="" class="SimplePara">Wild</p></td><td><p xmlns="" class="SimplePara">Zhuoni, Gansu, China</p></td><td><p xmlns="" class="SimplePara">Low seed shattering</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">ZhN06-11</p></td><td><p xmlns="" class="SimplePara">Wild</p></td><td><p xmlns="" class="SimplePara">Zhuoni, Gansu, China</p></td><td><p xmlns="" class="SimplePara">Low seed shattering, tall, long panicle</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Chuancao No. 2</p></td><td><p xmlns="" class="SimplePara">Cultivar</p></td><td><p xmlns="" class="SimplePara">Hongyuan, Sichuan, China</p></td><td><p xmlns="" class="SimplePara">Tall, early flowering, medium seed shattering</p></td></tr></tbody></table></div></figure>
                           <figure xmlns="" id="Tab2" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 2</span>
                        <p xmlns="" class="SimplePara">Parents and hybrid populations used in this study</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col></col><col></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Population</p></th><th><p xmlns="" class="SimplePara">Female parents</p></th><th><p xmlns="" class="SimplePara">Male parents</p></th><th><p xmlns="" class="SimplePara">F1 individuals</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">ZhN06-1</p></td><td><p xmlns="" class="SimplePara">Y1005-1</p></td><td><p xmlns="" class="SimplePara">1-H1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">2</p></td><td><p xmlns="" class="SimplePara">Y1005-1</p></td><td><p xmlns="" class="SimplePara">ZhN06-11</p></td><td><p xmlns="" class="SimplePara">2-H2, 2-H3, 2-H4, 2-H5</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">3</p></td><td><p xmlns="" class="SimplePara">ZhN06-11</p></td><td><p xmlns="" class="SimplePara">Y1005-1</p></td><td><p xmlns="" class="SimplePara">3-H1, 3-H2 3-H3, 3-H4, 3-H5</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">4</p></td><td><p xmlns="" class="SimplePara">XH09-4</p></td><td><p xmlns="" class="SimplePara">LQ03-1</p></td><td><p xmlns="" class="SimplePara">4-H1, 4-H2, 4-H3</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">5</p></td><td><p xmlns="" class="SimplePara">Chuancao No. 2</p></td><td><p xmlns="" class="SimplePara">PI348916</p></td><td><p xmlns="" class="SimplePara">5-H1, 5-H2</p></td></tr></tbody></table></div></figure>
                        </div>
              </section><section id="Sec4" class="Section2 RenderAsSection2">
                <h3 class="Heading">DNA extraction and PCR amplification</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par17" class="Para">Genomic DNA was extracted from parental plants and individual hybrid plant tissue using SDS (sodium dodecyl sulfate) method [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>] (Shan et al., 2011). DNA quantity and quality were verified using a Nanodrop spectrophotometer (NanoDrop Products, Wilmington, DE, USA) and agarose gel electrophoresis. Finally, the DNA concentration was adjusted with ddH<sub>2</sub>O to 25&nbsp;ng/L and stored at 20&nbsp;C prior to PCR amplification. A total of nine selected polymorphic EST-SSR primers were used for genotyping (Table&nbsp;<span class="InternalRef"><a href="#Tab3">3</a></span>). These primers had been used in a previous genetic diversity study in <em xmlns="" class="EmphasisTypeItalic">E. sibiricus</em> [<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>]. The optimal reaction system for <em xmlns="" class="EmphasisTypeItalic">E.sibiricus</em> was as follows: 2&nbsp;L 25&nbsp;ng/L DNA, 7.5&nbsp;L 2 Reaction Mix (Tiangen Beijing, China), 0.5&nbsp;L 10&nbsp;M forward primer, 0.5&nbsp;L 10&nbsp;M reverse primer, 0.2&nbsp;L (2.5&nbsp;U/L) Golden DNA Polymerase (Tiangen Beijing, China), and 4.3&nbsp;L of sterile ddH<sub>2</sub>O in a total of 15&nbsp;L reaction volume. PCR amplification was carried out as described by Xie et al. [<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>]. Then amplification fragments were separated on 6% denatured polyacrylamide gels. After electrophoresis, the gel was stained by AgNO<sub>3</sub> solution. Then gel was photographed by a Gel Doc (TM) XY System (Bio-Rad, Hercules, CA, USA).<figure xmlns="" id="Tab3" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 3</span>
                        <p xmlns="" class="SimplePara">The 9 EST-SSR primers used in this study</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Primer name</p></th><th><p xmlns="" class="SimplePara">Forward primers</p></th><th><p xmlns="" class="SimplePara">Reverse primers</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">Elw1420s081</p></td><td><p xmlns="" class="SimplePara">GGATAGACCCATGAGCTGACTGAT</p></td><td><p xmlns="" class="SimplePara">CTTTCTCCACAAGTTGAACACAACA</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Elw3545s194</p></td><td><p xmlns="" class="SimplePara">CAGCACTAGTATCCACCTCCACCT</p></td><td><p xmlns="" class="SimplePara">TGTTACAGCCTCTTCAGGCTCTTC</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Elw5627s404</p></td><td><p xmlns="" class="SimplePara">AGATGAAGCTGGTAACCGAGACAG</p></td><td><p xmlns="" class="SimplePara">ATTTCCTCTAATGGAAGCTCTGGC</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Ps1830</p></td><td><p xmlns="" class="SimplePara">GACTCGGCGAAAGGACTCTCT</p></td><td><p xmlns="" class="SimplePara">CTCGACGTCCTTCATGAGCTT</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Ps2283</p></td><td><p xmlns="" class="SimplePara">GCCACAACAAGAGAAGACCTTGC</p></td><td><p xmlns="" class="SimplePara">GACCTGCATGATGCTCTCGC</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Ps3577</p></td><td><p xmlns="" class="SimplePara">CATCTTGCATATAGCTCCTTCGCT</p></td><td><p xmlns="" class="SimplePara">CTCAAGAAACCCACAATCCAATTC</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Ps938</p></td><td><p xmlns="" class="SimplePara">TTGCTCCTATGGTTCCACGTAGTT</p></td><td><p xmlns="" class="SimplePara">AAAGTGAAATTCTGCCATCAGAGC</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Ltc0055</p></td><td><p xmlns="" class="SimplePara">AAGAAGAAGAGGCCGAGGAATAAA</p></td><td><p xmlns="" class="SimplePara">CGTGGATGTGCTGCAGGTAGTA</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Ltc0157</p></td><td><p xmlns="" class="SimplePara">GCAATGAACACTGAATCAATCGAG</p></td><td><p xmlns="" class="SimplePara">CGTGTGAGACTCATCGATGTTACC</p></td></tr></tbody></table></div></figure>
                        </div>
              </section><section id="Sec5" class="Section2 RenderAsSection2">
                <h3 class="Heading">Data analysis</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par18" class="Para">The amplified bands were scored as present (1) or absent (0), and only reproducible bands were considered. STRUCTURE v2.3.4 software was used to analyze the population structure of <em xmlns="" class="EmphasisTypeItalic">E. sibiricus</em> accessions and hybrid populations, with the admixture mode, burn-in period of 10,000 iterations and a run of 100,000 replications of Markov Chain Monte Carlo (MCMC) after burn in [<span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>]. For each run, 10 independent runs of STRUCTURE were performed with the number of clusters (K) varying from 1 to 8. Mean L (K) and delta K (K) were estimated using the method described by Evanno et al. [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>] To detect genetic relationship among different accessions, a principal coordinate analysis (PCoA) was constructed based on Jaccards genetic similarity matrix using DCENTER module in NTSYS (version 2.10), which is more informative regarding distances among major groups [<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>]. <em xmlns="" class="EmphasisTypeItalic">E. sibiricus</em> hybrid identification was carried out according to a method used in orchardgrass [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>]. Single primer pairs or primer combinations that were diagnostic for parental plants and hybrid plants were used.</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec6" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Results" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Results">Results</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Results" data-test="collapse-section"><section id="Sec7" class="Section2 RenderAsSection2">
                <h3 class="Heading">EST-SSR marker transferability</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par19" class="Para">EST-SSR markers previously developed from Snake River wheatgrass (<em xmlns="" class="EmphasisTypeItalic">Elymus wawawaiensis</em>), thick spike wheatgrass (<em xmlns="" class="EmphasisTypeItalic">Elymus lanceolatusd</em>), bluebunch wheatgrass (<em xmlns="" class="EmphasisTypeItalic">Pseudoroegeneria spicata</em>) and <em xmlns="" class="EmphasisTypeItalic">Leymus</em> species were used in this study. Finally, a total of 9 primers were selected and used for hybrid identification and genetic diversity analysis, of which 3 from <em xmlns="" class="EmphasisTypeItalic">Elymus</em> (Elw hereafter), 4 from <em xmlns="" class="EmphasisTypeItalic">Pseudoroegneria</em> (Ps hereafter) and 2 from <em xmlns="" class="EmphasisTypeItalic">Leymus</em> (Lt hereafter). The results proved that all 9 primers can be successfully amplified across 22 <em xmlns="" class="EmphasisTypeItalic">E.sibiricus</em> plants used in this study, with 100% the transferability rate.</p>
              </section><section id="Sec8" class="Section2 RenderAsSection2">
                <h3 class="Heading">Hybrid identification</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par20" class="Para">Single primers or primer combinations that were diagnostic for parental plants and hybrid plants were used. In the study, an analysis of amplified polymorphic bands among genuine hybrids and their respective parents mainly revealed two types of markers: 1) hybrids shared bands exclusively amplified in both parents (Fig.&nbsp;<span class="InternalRef"><a href="#Fig1">1a</a></span>); 2) hybrids shared bands exclusively amplified in male parents (Fig.&nbsp;<span class="InternalRef"><a href="#Fig1">1b</a></span>); Finally, according to this method, 15 hybrids were successfully identified using different primers and then used for the genetic diversity analysis (Table&nbsp;<span class="InternalRef"><a href="#Tab2">2</a></span>).<figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO1">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0053-1/MediaObjects/41065_2017_53_Fig1_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0053-1/MediaObjects/41065_2017_53_Fig1_HTML.gif" alt="Fig. 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0053-1/MediaObjects/41065_2017_53_Fig1_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0053-1/MediaObjects/41065_2017_53_Fig1_HTML.gif" alt="Fig. 1"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 1</span>
                        <p xmlns="" class="SimplePara">EST-SSR profiles produced with different primers in hybrid and their respective parents. <strong class="EmphasisTypeBold">a</strong>: hybrids shared bands exclusively amplified in both parents; <strong class="EmphasisTypeBold">b</strong> hybrids shared bands exclusively amplified in male parents</p>
                      </div></figcaption></figure>
                        </div>
              </section><section id="Sec9" class="Section2 RenderAsSection2">
                <h3 class="Heading">EST-SSR polymorphism and genetic relationship in hybrid populations and parents</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par21" class="Para">In this study, 9 primers were used for genetic diversity analysis among five hybrid populations and their parents (Table&nbsp;<span class="InternalRef"><a href="#Tab4">4</a></span>). The total number of amplified bands ranged from 37 (Pop 1) to 57(Pop 2, 3), the percentage of polymorphism varied from 65.12 (Pop 5) to 75.68% (Pop 1), with an average of 70.51%.<figure xmlns="" id="Tab4" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 4</span>
                        <p xmlns="" class="SimplePara">EST-SSR results achieved in hybrid populations and their parents</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col char="."></col><col char="."></col><col char="."></col><col char="."></col><col char="."></col><col char="."></col><col char="."></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Population</p></th><th><p xmlns="" class="SimplePara">Primer</p></th><th char="."><p xmlns="" class="SimplePara">T</p></th><th char="."><p xmlns="" class="SimplePara">M</p></th><th char="."><p xmlns="" class="SimplePara">TP</p></th><th char="."><p xmlns="" class="SimplePara">MF1</p></th><th char="."><p xmlns="" class="SimplePara">FF1</p></th><th char="."><p xmlns="" class="SimplePara">EF1</p></th><th char="."><p xmlns="" class="SimplePara">PPB (%)</p></th></tr></thead><tbody><tr class="noclass"><td rowspan="7"><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">Elw1420s081</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">100.00</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Elw3545s194</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">0.00</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Elw5627s404</p></td><td char="."><p xmlns="" class="SimplePara">14</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">14</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">6</p></td><td char="."><p xmlns="" class="SimplePara">100.00</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Ps3577</p></td><td char="."><p xmlns="" class="SimplePara">4</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">50.00</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Ps938</p></td><td char="."><p xmlns="" class="SimplePara">7</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">6</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">85.71</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Ltc0055</p></td><td char="."><p xmlns="" class="SimplePara">6</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">50.00</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Total</p></td><td char="."><p xmlns="" class="SimplePara">37</p></td><td char="."><p xmlns="" class="SimplePara">9</p></td><td char="."><p xmlns="" class="SimplePara">28</p></td><td char="."><p xmlns="" class="SimplePara">12</p></td><td char="."><p xmlns="" class="SimplePara">12</p></td><td char="."><p xmlns="" class="SimplePara">7</p></td><td char="."><p xmlns="" class="SimplePara">75.68</p></td></tr><tr class="noclass"><td rowspan="10"><p xmlns="" class="SimplePara">2</p></td><td><p xmlns="" class="SimplePara">Elw1420s081</p></td><td char="."><p xmlns="" class="SimplePara">6</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">5</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">4</p></td><td char="."><p xmlns="" class="SimplePara">83.33</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Elw3545s194</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">66.67</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Elw5627s404</p></td><td char="."><p xmlns="" class="SimplePara">14</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">12</p></td><td char="."><p xmlns="" class="SimplePara">4</p></td><td char="."><p xmlns="" class="SimplePara">5</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">85.71</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Ps1830</p></td><td char="."><p xmlns="" class="SimplePara">9</p></td><td char="."><p xmlns="" class="SimplePara">4</p></td><td char="."><p xmlns="" class="SimplePara">5</p></td><td char="."><p xmlns="" class="SimplePara">5</p></td><td char="."><p xmlns="" class="SimplePara">5</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">55.56</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Ps2283</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">50.00</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Ps3577</p></td><td char="."><p xmlns="" class="SimplePara">4</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">50.00</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Ps938</p></td><td char="."><p xmlns="" class="SimplePara">5</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">5</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">60.00</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Ltc0055</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">0.00</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Ltc0157</p></td><td char="."><p xmlns="" class="SimplePara">11</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">8</p></td><td char="."><p xmlns="" class="SimplePara">4</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">72.73</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Total</p></td><td char="."><p xmlns="" class="SimplePara">57</p></td><td char="."><p xmlns="" class="SimplePara">19</p></td><td char="."><p xmlns="" class="SimplePara">38</p></td><td char="."><p xmlns="" class="SimplePara">27</p></td><td char="."><p xmlns="" class="SimplePara">24</p></td><td char="."><p xmlns="" class="SimplePara">9</p></td><td char="."><p xmlns="" class="SimplePara">66.67</p></td></tr><tr class="noclass"><td rowspan="10"><p xmlns="" class="SimplePara">3</p></td><td><p xmlns="" class="SimplePara">Elw1420s081</p></td><td char="."><p xmlns="" class="SimplePara">5</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">4</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">80.00</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Elw3545s194</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">66.67</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Elw5627s404</p></td><td char="."><p xmlns="" class="SimplePara">14</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">13</p></td><td char="."><p xmlns="" class="SimplePara">5</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">92.86</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Ps1830</p></td><td char="."><p xmlns="" class="SimplePara">10</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">7</p></td><td char="."><p xmlns="" class="SimplePara">4</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">70.00</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Ps2283</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">50.00</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Ps3577</p></td><td char="."><p xmlns="" class="SimplePara">4</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">50.00</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Ps938</p></td><td char="."><p xmlns="" class="SimplePara">5</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">4</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">80.00</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Ltc0055</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">0.00</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Ltc0157</p></td><td char="."><p xmlns="" class="SimplePara">11</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">8</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">4</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">72.73</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Total</p></td><td char="."><p xmlns="" class="SimplePara">57</p></td><td char="."><p xmlns="" class="SimplePara">16</p></td><td char="."><p xmlns="" class="SimplePara">41</p></td><td char="."><p xmlns="" class="SimplePara">22</p></td><td char="."><p xmlns="" class="SimplePara">20</p></td><td char="."><p xmlns="" class="SimplePara">8</p></td><td char="."><p xmlns="" class="SimplePara">71.93</p></td></tr><tr class="noclass"><td rowspan="9"><p xmlns="" class="SimplePara">4</p></td><td><p xmlns="" class="SimplePara">Elw3545s194</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">0.00</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Elw5627s404</p></td><td char="."><p xmlns="" class="SimplePara">7</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">6</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">4</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">85.71</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Ps1830</p></td><td char="."><p xmlns="" class="SimplePara">7</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">5</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">6</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">71.43</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Ps2283</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">66.67</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Ps3577</p></td><td char="."><p xmlns="" class="SimplePara">8</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">7</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">87.50</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Ps938</p></td><td char="."><p xmlns="" class="SimplePara">6</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">6</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">100.00</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Ltc0055</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">0.00</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Ltc0157</p></td><td char="."><p xmlns="" class="SimplePara">5</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">4</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">80.00</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Total</p></td><td char="."><p xmlns="" class="SimplePara">41</p></td><td char="."><p xmlns="" class="SimplePara">11</p></td><td char="."><p xmlns="" class="SimplePara">30</p></td><td char="."><p xmlns="" class="SimplePara">12</p></td><td char="."><p xmlns="" class="SimplePara">23</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">73.17</p></td></tr><tr class="noclass"><td rowspan="10"><p xmlns="" class="SimplePara">5</p></td><td><p xmlns="" class="SimplePara">Elw1420s081</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">100.00</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Elw3545s194</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">0.00</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Elw5627s404</p></td><td char="."><p xmlns="" class="SimplePara">6</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">6</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">100.00</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Ps1830</p></td><td char="."><p xmlns="" class="SimplePara">8</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">6</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">75.00</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Ps2283</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">100.00</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Ps3577</p></td><td char="."><p xmlns="" class="SimplePara">4</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">50.00</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Ps938</p></td><td char="."><p xmlns="" class="SimplePara">4</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">2</p></td><td char="."><p xmlns="" class="SimplePara">75.00</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Ltc0055</p></td><td char="."><p xmlns="" class="SimplePara">4</p></td><td char="."><p xmlns="" class="SimplePara">4</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">4</p></td><td char="."><p xmlns="" class="SimplePara">4</p></td><td char="."><p xmlns="" class="SimplePara">0</p></td><td char="."><p xmlns="" class="SimplePara">0.00</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Ltc0157</p></td><td char="."><p xmlns="" class="SimplePara">8</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">5</p></td><td char="."><p xmlns="" class="SimplePara">3</p></td><td char="."><p xmlns="" class="SimplePara">5</p></td><td char="."><p xmlns="" class="SimplePara">1</p></td><td char="."><p xmlns="" class="SimplePara">62.50</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Total</p></td><td char="."><p xmlns="" class="SimplePara">43</p></td><td char="."><p xmlns="" class="SimplePara">15</p></td><td char="."><p xmlns="" class="SimplePara">28</p></td><td char="."><p xmlns="" class="SimplePara">16</p></td><td char="."><p xmlns="" class="SimplePara">24</p></td><td char="."><p xmlns="" class="SimplePara">7</p></td><td char="."><p xmlns="" class="SimplePara">65.12</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">T=Total number of amplified bands; M=Number of monomorphic bands; TP=Total number of polymorphic bands; MF1=Bands shared by male parents and hybrids; FF1=Bands shared by female parents and hybrids; EF1=Bands exclusively present in hybrids; PPB=Percentage of polymorphic bands</p>
                    </div></figure>
                        </div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par22" class="Para">Regarding the parental origin of the amplified bands inherited by hybrid populations, similar percentage of parental origin were found in three hybrid populations. The percentages in the first three populations didnt reach 50%. For example, in the population 1, 32.43% were inherited from ZhN06-1 and 32.43% were from Y1005-1. In the population 2, 47.37% were from ZhN06-11 and Y1005-1 had 42.11% origin. In the population 3, 38.59% were inherited from male parent and 35.09% from female parent. However, in the population 4 and 5, 56.09% and 55.81% amplified bands were inherited from male parents, respectively. Some types of polymorphism evidenced an occurrence of rearrangements in the hybrid populations that resulted from the gain of novel bands (not seen in parental genomes). Total number of bands that exclusively present in hybrid populations ranged from 0 (Pop4) to 9 (Pop2), with an average of 6.2.</p>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par23" class="Para">The population structure of hybrid populations and their parents was analyzed in this study. Based on maximum likelihood and delta K (K) values, the number of optimum groups was eight (Fig.&nbsp;<span class="InternalRef"><a href="#Fig2">2</a></span>). All accessions were assigned to eight groups. group1 included LQ03-1, 1-H1, 2-H2, 2-H3, 2-H5; group 2: PI348916, 2-H4, 3-H4, 3-H5; group 3: ZhN06-11, 3-H1, 3-H2; group 4: 4-H2, 4-H3; group 5: 3-H3; group 6: ZhN06-1, Chuancao No.2, 5-H1, 5-H2; group 7: XH09-4, 4-H1; group 8: Y1005-1. Every group was mixed genetic ingredient of other groups in different extent. Especially, LQ03-1, PI348916, ZhN06-11, 3-H3, 1-H1 and 2-H5 had more complicated genetic constitutes than the other accessions in the study. 1-H1, 2-H5 and 3-H3 were the F<sub>1</sub> individuals from population 1, 2 and 3, respectively. However, some accessions presented a purer genetic ingredient within their groups. 4-H1, 5-H1 and 5-H2, the F<sub>1</sub> plant of population 4 and 5, had same genetic constitutes to female XH09-4 and Chuancao No.2 respectively. The rest of individuals for pure genetic constitutes included 2-H2, 2-H3, 2-H4, 3-H1, 3-H2 and 3-H5, they came from population 2 and 3.<figure xmlns="" class="Figure" id="Fig2"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO2">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0053-1/MediaObjects/41065_2017_53_Fig2_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0053-1/MediaObjects/41065_2017_53_Fig2_HTML.gif" alt="Fig. 2" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0053-1/MediaObjects/41065_2017_53_Fig2_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0053-1/MediaObjects/41065_2017_53_Fig2_HTML.gif" alt="Fig. 2"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 2</span>
                        <p xmlns="" class="SimplePara">Eight groups of 22 <em class="EmphasisTypeItalic">E. sibiricus</em> accessions inferred from STRUCTURE analysis and the description of detected the optimum value of K by using graphical method. <strong class="EmphasisTypeBold">a</strong> Mean L (K) over 20 runs for each K value; <strong class="EmphasisTypeBold">b</strong> Maximum delta K (4&nbsp;K) values were used to determine the uppermost level of structure for K ranging from 2 to 10, here K is 8 and eight clusters; <strong class="EmphasisTypeBold">c</strong> The vertical coordinate of each group indicates the membership coefficients for each accession. 1-22 represented respectively: ZhN06-1, Y1005-1, 1-H1, Chuancao No.2, PI348916, 5-H1, 5-H2, ZhN06-11, 3-H1, 2-H2, 3-H2, 3-H3, 2-H3, XH09-4, LQ03-1, 4-H1, 4-H2, 4-H3, 2-H4, 3-H4, 3-H5, 2-H5</p>
                      </div></figcaption></figure>
                        </div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par24" class="Para">The principal coordinate analysis (PCoA) showed about 64.72% of the total variation was described by the first three PCo (Fig.&nbsp;<span class="InternalRef"><a href="#Fig3">3</a></span>). The results of PCoA analysis were similar to structure analysis.<figure xmlns="" class="Figure" id="Fig3"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO3">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-017-0053-1/MediaObjects/41065_2017_53_Fig3_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-017-0053-1/MediaObjects/41065_2017_53_Fig3_HTML.gif" alt="Fig. 3" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0053-1/MediaObjects/41065_2017_53_Fig3_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-017-0053-1/MediaObjects/41065_2017_53_Fig3_HTML.gif" alt="Fig. 3"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 3</span>
                        <p xmlns="" class="SimplePara">Principal coordinates analysis for EST-SSR markers using genetic similarity matrix for 22 <em class="EmphasisTypeItalic">E. sibiricus</em> accessions. Red dot and blue dot represented parents and hybrid populations, respectively</p>
                      </div></figcaption></figure>
                        </div>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec10" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Discussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Discussion">Discussion</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Discussion" data-test="collapse-section"><section id="Sec11" class="Section2 RenderAsSection2">
                <h3 class="Heading">EST-SSR markers for hybrid identification</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par25" class="Para">Traditional hybrid identification depends on morphological traits like plant height, leaf shape, flower color, growth habit, and so on. But environmental factors often affect the accuracy of identification. DNA fingerprinting techniques are efficient alternatives to morphological identification, and make plant identification easy and accurate. Molecular markers have been successfully applied to identification of crop and forage grass species [<span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>, <span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>]. EST-SSRs are codominant markers that can show evidently the heterozygosity of offspring populations by the bands from parents and hybrid plants [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>, <span class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>]. Our study is the first report of <em xmlns="" class="EmphasisTypeItalic">E. sibiricus</em> hybrids identification using EST-SSR markers. Based on our results single primers or primer combinations that were diagnostic for parental plants and hybrid plants could be used for hybrid identification. Fifteen hybrids were identified with two types of DNA fingerprinting. In general, this study demonstrated that EST-SSR markers can effectively identify <em xmlns="" class="EmphasisTypeItalic">E.sibiricus</em> hybrids through special DNA fingerprinting profiles. This study will lay a foundation for <em xmlns="" class="EmphasisTypeItalic">E. sibiricus</em> breeding in future.</p>
              </section><section id="Sec12" class="Section2 RenderAsSection2">
                <h3 class="Heading">EST-SSR marker for genetic diversity in <em class="EmphasisTypeItalic">E. sibiricus</em>
                        </h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par26" class="Para">EST-SSRs are highly polymorphic, abundant and are accessible to research laboratories via published primers sequences [<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>]. These published primers are especially important resources for species like <em xmlns="" class="EmphasisTypeItalic">E. sibiriucs</em> with few molecular markers available. In this study, a total of 9 EST-SSR markers from three different genuses were used for hybrid identification and genetic diversity analysis. The transferability rate of EST-SSRs was 100%. Results of this study were consistent with previous reports that EST-SSR markers have high transferability rate among species. Information on genetic diversity and hybrid population and their parents can improve our understanding of breeding materials.Based on our results the percentage of polymorphism varied from 65.12 (Pop 5) to 75.68% (Pop 1), with an average of 70.51%. The average percentage of polymorphism (PPB) of five hybrid populations were lower than previous reports of SRAP (PPB=86.5%) [<span class="CitationRef"><a href="#CR28" data-toggle="citation">28</a></span>], SSR (PPB=89.4%) [<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>], and SCoT (PPB=91.9%) [<span class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>]. The major reason for relatively low PPB could be small sample size of each population. In addition, hybrid populations derived from seven parent genotypes, and genetic base was relatively narrow. This can be supported by results of structure analysis. In general, these accessions and their offspring did not show major genetic structure, most of accessions were assigned into mixed groups, indicating relatively narrow genetic base.</p>
              </section><section id="Sec13" class="Section2 RenderAsSection2">
                <h3 class="Heading">Broadening genetic diversity for breeding</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par27" class="Para">In this study, some types of polymorphism have evidenced potentially an occurrence of chromosome variation or gene rearrangements in hybrid populations that resulted from the gain of novel bands. The results of the present study suggested that more genetic diversity and new variation could be captured by crossing breeding. Whether these novel bands were responsible to new genes associated to seed shattering or other important traits, it was still not clear. It is still difficult to address particular mechanisms understanding the chromosomal or genomic rearrangements in response to novel bands. In the future, molecular markers combined with sequence data might provide evidence to the inheritance of transcribed regions and gene functions.</p>
              </section></div></section>
          <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AbbreviationGroup Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Abbreviations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Abbreviations">Abbreviations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Abbreviations"><div class="DefinitionList">
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>EF1:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par4" class="Para">Bands exclusively present in hybrids</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>FF1:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par5" class="Para">Bands shared by female parents and hybrids</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>M:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par6" class="Para">Number of monomorphic bands</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>MF1:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par7" class="Para">Bands shared by male parents and hybrids</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>PCoA:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par8" class="Para">Principal coordinate analysis</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>PPB:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par9" class="Para">Percentage of polymorphic bands</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>T:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par10" class="Para">Total number of amplified bands</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>TP:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par11" class="Para">Total number of polymorphic bands</p>
                  </dd></dl>
              </div></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments"><h3 xmlns="" class="Heading">Acknowledgements</h3><p xmlns="" class="SimplePara">We thanks grants from the Program for Changjiang Scholars and Innovative Research Team in University (IRT-17R50), Chinese National Basic Research Program (973 Program) (No. 2014CB138704), the Open Project Program of State Key Laboratory of Grassland Agro-ecosystems, 111 program (B12002).</p><div id="FPar1" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Funding</h3>
                <p id="Par28" class="Para">This study was supported by grants from the Chinese National Basic Research Program (973 Program) (No. 2014CB138704), the Open Project Program of State Key Laboratory of Grassland Agro-ecosystems, 111 program (B12002).</p>
              </div><div id="FPar2" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Availability of data and materials</h3>
                <p id="Par29" class="Para">The data supporting the conclusions of this article are included within the article.</p>
              </div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="FormalPara FormalParaRenderingStyle1"><h3 xmlns="" class="Heading">Authors contributions</h3><p xmlns="" class="SimplePara">WX, YW conceived and designed the experiments. XZ performed the experiments. ZZ and JZ provided help with data analysis. XZ and WX wrote and revised the manuscript. All authors have read and approved the final manuscript.</p></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Ethics">
              <div id="FPar3" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Ethics approval and consent to participate</h3>
                <p id="Par30" class="Para">Not applicable</p>
              </div>
              <div id="FPar4" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Consent for publication</h3>
                <p id="Par31" class="Para">Not applicable</p>
              </div>
              <div id="FPar5" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Competing interests</h3>
                <p id="Par32" class="Para">The authors declare that they have no competing interests.</p>
              </div>
              <div id="FPar6" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Publishers Note</h3>
                <p id="Par33" class="Para">Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
              </div>
            </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="License">
                <p xmlns="" class="SimplePara">
                           <strong class="EmphasisTypeBold">Open Access</strong>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/4.0/"><span class="RefSource">http://creativecommons.org/licenses/by/4.0/</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/publicdomain/zero/1.0/"><span class="RefSource">http://creativecommons.org/publicdomain/zero/1.0/</span></a></span>) applies to the data made available in this article, unless otherwise stated.</p>
              </div></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authors Affiliations</a></h2><div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1"><span class="AffiliationNumber">(1)</span><div class="AffiliationText">The State Key Laboratory of Grassland Agro-ecosystems, Key Laboratory of Grassland Livestock Industry Innovation, Ministry of Agriculture, College of Pastoral Agriculture Science and Technology, Lanzhou University, Lanzhou, 730020, Peoples Republic of China</div></div></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper"><li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lei YT, Zhao YY, Yu F, Li Y, Dou Q. Development and characterization of 53 polymorphic genomic-SSR markers in Siberian wild rye (<em class="EmphasisTypeItalic">Elymus sibiricus</em> L.). Conserv Genet Resour. 2014;6:8614.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s12686-014-0225-5"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Development%20and%20characterization%20of%2053%20polymorphic%20genomic-SSR%20markers%20in%20Siberian%20wild%20rye%20%28Elymus%20sibiricus%20L.%29&amp;author=YT.%20Lei&amp;author=YY.%20Zhao&amp;author=F.%20Yu&amp;author=Y.%20Li&amp;author=Q.%20Dou&amp;journal=Conserv%20Genet%20Resour&amp;volume=6&amp;pages=861-864&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ma X, Chen SY, Zhang XQ, Bai SQ, Zhang CB. Assessment of worldwide genetic diversity of Siberian wild rye (<em class="EmphasisTypeItalic">Elymus sibiricus</em> L.) germplasm based on gliadin analysis. Molecules. 2012;17:442434.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.3390/molecules17044424"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22499189"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Assessment%20of%20worldwide%20genetic%20diversity%20of%20Siberian%20wild%20rye%20%28Elymus%20sibiricus%20L.%29%20germplasm%20based%20on%20gliadin%20analysis&amp;author=X.%20Ma&amp;author=SY.%20Chen&amp;author=XQ.%20Zhang&amp;author=SQ.%20Bai&amp;author=CB.%20Zhang&amp;journal=Molecules&amp;volume=17&amp;pages=4424-4434&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Bowden WM, Cody WJ. Recognition of <em class="EmphasisTypeItalic">Elymus sibiricus</em> L. from Alaska and the district of Mackenzie. Bull Torrey Bot. Club. 1961;88:1535.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2307/2482709"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Recognition%20of%20Elymus%20sibiricus%20L.%20from%20Alaska%20and%20the%20district%20of%20Mackenzie.%20Bull%20Torrey%20Bot&amp;author=WM.%20Bowden&amp;author=WJ.%20Cody&amp;journal=Club&amp;volume=88&amp;pages=153-155&amp;publication_year=1961"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Xie WG, Zhao XH, Zhang JQ, Wang YR, Liu WX. Assessment of genetic diversity of Siberian wild rye (<em class="EmphasisTypeItalic">Elymus sibiricus</em> L.) germplasms with variation of seed shattering and implication for future genetic improvement. Biochem Syst Ecol. 2015;58:2118.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.bse.2014.12.006"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Assessment%20of%20genetic%20diversity%20of%20Siberian%20wild%20rye%20%28Elymus%20sibiricus%20L.%29%20germplasms%20with%20variation%20of%20seed%20shattering%20and%20implication%20for%20future%20genetic%20improvement&amp;author=WG.%20Xie&amp;author=XH.%20Zhao&amp;author=JQ.%20Zhang&amp;author=YR.%20Wang&amp;author=WX.%20Liu&amp;journal=Biochem%20Syst%20Ecol&amp;volume=58&amp;pages=211-218&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yan JJ, Bai SQ, Ma X, Gan YM, Zhang JB. Genetic diversity of <em class="EmphasisTypeItalic">Elymus sibiricus</em> and its breeding in China. Chin Bull Bot. 2007;24:22631.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20of%20Elymus%20sibiricus%20and%20its%20breeding%20in%20China&amp;author=JJ.%20Yan&amp;author=SQ.%20Bai&amp;author=X.%20Ma&amp;author=YM.%20Gan&amp;author=JB.%20Zhang&amp;journal=Chin%20Bull%20Bot&amp;volume=24&amp;pages=226-231&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhang ZY, Zhang JC, Zhao XH, Xie WG, Wang YR. Assessing and broadening genetic diversity of <em class="EmphasisTypeItalic">Elymus sibiricus</em> germplasm for the improvement of seed shattering. Molecules. 2016;21(7):869.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.3390/molecules21070869"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Assessing%20and%20broadening%20genetic%20diversity%20of%20Elymus%20sibiricus%20germplasm%20for%20the%20improvement%20of%20seed%20shattering&amp;author=ZY.%20Zhang&amp;author=JC.%20Zhang&amp;author=XH.%20Zhao&amp;author=WG.%20Xie&amp;author=YR.%20Wang&amp;journal=Molecules&amp;volume=21&amp;issue=7&amp;pages=869&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Jesske T, Olberg B, Schierholy A, Becker HC. Resynthesized lines from domesticated and wild <em class="EmphasisTypeItalic">Brassica</em> taxa and their hybrids with <em class="EmphasisTypeItalic">B. napus</em> L.: genetic diversity and hybrid yield. Theor Appl Genet. 2013;126:105365.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00122-012-2036-y"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23328861"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3607727"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Resynthesized%20lines%20from%20domesticated%20and%20wild%20Brassica%20taxa%20and%20their%20hybrids%20with%20B.%20napus%20L.%3A%20genetic%20diversity%20and%20hybrid%20yield&amp;author=T.%20Jesske&amp;author=B.%20Olberg&amp;author=A.%20Schierholy&amp;author=HC.%20Becker&amp;journal=Theor%20Appl%20Genet&amp;volume=126&amp;pages=1053-1065&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Shen JX, Fu TD, Yang GS. Relationship between hybrid performance and genetic diversity based on SSRs and ISSRs in <em class="EmphasisTypeItalic">Brassica napus</em> L. Agr Sci China. 2003;2:108390.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Relationship%20between%20hybrid%20performance%20and%20genetic%20diversity%20based%20on%20SSRs%20and%20ISSRs%20in%20Brassica%20napus%20L&amp;author=JX.%20Shen&amp;author=TD.%20Fu&amp;author=GS.%20Yang&amp;journal=Agr%20Sci%20China&amp;volume=2&amp;pages=1083-1090&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Madesis P, Abraham EM, Kalivas A, Ganopoulos I, Tsaftaris A. Genetic diversity and structure of natural <em class="EmphasisTypeItalic">Dactylis glomerata</em> L. populations revealed by morphological and microsatellite-based (SSR/ISSR) markers. Genet Mol Res. 2014;13:422640.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.4238/2014.June.9.8"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25036166"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20and%20structure%20of%20natural%20Dactylis%20glomerata%20L.%20populations%20revealed%20by%20morphological%20and%20microsatellite-based%20%28SSR%2FISSR%29%20markers&amp;author=P.%20Madesis&amp;author=EM.%20Abraham&amp;author=A.%20Kalivas&amp;author=I.%20Ganopoulos&amp;author=A.%20Tsaftaris&amp;journal=Genet%20Mol%20Res&amp;volume=13&amp;pages=4226-4240&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lalit A, Manjusha V, Suman L. Diagnostic set of microsatellite markers for hybrid purity testing and molecular identification of hybrids and parental lines in <em class="EmphasisTypeItalic">sorghum</em>. J Plant Res. 2014;1:14.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Diagnostic%20set%20of%20microsatellite%20markers%20for%20hybrid%20purity%20testing%20and%20molecular%20identification%20of%20hybrids%20and%20parental%20lines%20in%20sorghum%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=A.%20Lalit&amp;author=V.%20Manjusha&amp;author=L.%20Suman&amp;journal=J%20Plant%20Res&amp;volume=1&amp;pages=1-4&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhao YF, Zhang XQ, Ma X, Xie WG, Huang LK. Morphological and genetic characteristics of hybrid combinations of <em class="EmphasisTypeItalic">Dactylis glomerata</em>. Genet Mol Res. 2014;13:2491503.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.4238/2014.January.28.1"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24615091"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Morphological%20and%20genetic%20characteristics%20of%20hybrid%20combinations%20of%20Dactylis%20glomerata%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=YF.%20Zhao&amp;author=XQ.%20Zhang&amp;author=X.%20Ma&amp;author=WG.%20Xie&amp;author=LK.%20Huang&amp;journal=Genet%20Mol%20Res&amp;volume=13&amp;pages=2491-2503&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Daniel IO, Adetumbi JA, Oyelakin O, Olakojo SA, Ajala MO, Onagbesan SO. Application of SSR markers for genetic purity analysis of parental inbred lines and some commercial hybrid maize (<em class="EmphasisTypeItalic">Zea mays</em> L.). Am J Exp Agri. 2012;2:597606.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Application%20of%20SSR%20markers%20for%20genetic%20purity%20analysis%20of%20parental%20inbred%20lines%20and%20some%20commercial%20hybrid%20maize%20%28Zea%20mays%20L.%29&amp;author=IO.%20Daniel&amp;author=JA.%20Adetumbi&amp;author=O.%20Oyelakin&amp;author=SA.%20Olakojo&amp;author=MO.%20Ajala&amp;author=SO.%20Onagbesan&amp;journal=Am%20J%20Exp%20Agri&amp;volume=2&amp;pages=597-606&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sundaram RM, Naveenkumar B, Biradar SK, Balachandran SM, Mishra B, IlyasAhmed M, et al. Identification of informative SSR markers capable of distinguishing hybrid Rice parental lines and their utilization in seed purity assessment. Euphytica. 2008;163:21524.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s10681-007-9630-0"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Identification%20of%20informative%20SSR%20markers%20capable%20of%20distinguishing%20hybrid%20Rice%20parental%20lines%20and%20their%20utilization%20in%20seed%20purity%20assessment&amp;author=RM.%20Sundaram&amp;author=B.%20Naveenkumar&amp;author=SK.%20Biradar&amp;author=SM.%20Balachandran&amp;author=B.%20Mishra&amp;author=M.%20IlyasAhmed&amp;journal=Euphytica&amp;volume=163&amp;pages=215-224&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Iqbal A, Sadaqat HA, Khan AS, Amjad M. Identification of sunflower <em class="EmphasisTypeItalic">(Helianthus annuus</em>, Asteraceae) hybrids using simple-sequence repeat markers. Genet Mol Res. 2010;10:1026.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.4238/vol10-1gmr918"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Identification%20of%20sunflower%20%28Helianthus%20annuus%2C%20Asteraceae%29%20hybrids%20using%20simple-sequence%20repeat%20markers&amp;author=A.%20Iqbal&amp;author=HA.%20Sadaqat&amp;author=AS.%20Khan&amp;author=M.%20Amjad&amp;journal=Genet%20Mol%20Res&amp;volume=10&amp;pages=102-106&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Liu LW, Liu G, Gong YQ, Dai WH, Wang Y, Yu FM, et al. Evaluation of genetic purity of F<sub xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml">1</sub> hybrid seeds in cabbage with RAPD, ISSR, SRAP, and SSR markers. Hortscience. 2007;2:7247.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Evaluation%20of%20genetic%20purity%20of%20F1%20hybrid%20seeds%20in%20cabbage%20with%20RAPD%2C%20ISSR%2C%20SRAP%2C%20and%20SSR%20markers&amp;author=LW.%20Liu&amp;author=G.%20Liu&amp;author=YQ.%20Gong&amp;author=WH.%20Dai&amp;author=Y.%20Wang&amp;author=FM.%20Yu&amp;journal=Hortscience&amp;volume=2&amp;pages=724-727&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Tsukazaki H, Fukuoka H, Song YS, Yamashita K, Wako T, Kojima A. Considerable heterogeneity in commercial F1 varieties of bunching onion (<em class="EmphasisTypeItalic">Allium fistulosum</em>) and the proposal of a breeding scheme for conferring genetic traceability using SSR markers. Breeding Sci. 2006;56:3216.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1270/jsbbs.56.321"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Considerable%20heterogeneity%20in%20commercial%20F1%20varieties%20of%20bunching%20onion%20%28Allium%20fistulosum%29%20and%20the%20proposal%20of%20a%20breeding%20scheme%20for%20conferring%20genetic%20traceability%20using%20SSR%20markers&amp;author=H.%20Tsukazaki&amp;author=H.%20Fukuoka&amp;author=YS.%20Song&amp;author=K.%20Yamashita&amp;author=T.%20Wako&amp;author=A.%20Kojima&amp;journal=Breeding%20Sci&amp;volume=56&amp;pages=321-326&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhao Z, Gu H, Sheng X, Yu H, Wang J, Cao J. Genetic purity testing of loose-curd cauliflower hybrids using SSR markers and grow out test. Seed Sci Technol. 2012;40:20914.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.15258/sst.2012.40.2.06"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20purity%20testing%20of%20loose-curd%20cauliflower%20hybrids%20using%20SSR%20markers%20and%20grow%20out%20test&amp;author=Z.%20Zhao&amp;author=H.%20Gu&amp;author=X.%20Sheng&amp;author=H.%20Yu&amp;author=J.%20Wang&amp;author=J.%20Cao&amp;journal=Seed%20Sci%20Technol&amp;volume=40&amp;pages=209-214&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Xie WG, Zhang XQ, Cheng YX. Identification and genetic variation analysis of Orchardgrass hybrids (<em class="EmphasisTypeItalic">Dactylis glomerata</em>) by SSR molecular markers. Acta Pratac Sin. 2010;19:2127.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Identification%20and%20genetic%20variation%20analysis%20of%20Orchardgrass%20hybrids%20%28Dactylis%20glomerata%29%20by%20SSR%20molecular%20markers&amp;author=WG.%20Xie&amp;author=XQ.%20Zhang&amp;author=YX.%20Cheng&amp;journal=Acta%20Pratac%20Sin.&amp;volume=19&amp;pages=212-217&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lei YT, Dou QW. Distinguishing <em class="EmphasisTypeItalic">Elymus nutans</em> from <em class="EmphasisTypeItalic">Elymus sibiricus</em> in Qinghai - Tibet plateau using a SSR marker. Pratac Sci. 2012;29:93742.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Distinguishing%20Elymus%20nutans%20from%20Elymus%20sibiricus%20in%20Qinghai%20-%20Tibet%20plateau%20using%20a%20SSR%20marker&amp;author=YT.%20Lei&amp;author=QW.%20Dou&amp;journal=Pratac%20Sci&amp;volume=29&amp;pages=937-942&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Xie WG, Zhang JC, Zhao XH, Zhang JQ, Wang YR. Siberian wild rye ( <em class="EmphasisTypeItalic">Elymus sibiricus</em>, l.): genetic diversity of germplasm determined using DNA fingerprinting and SCoT markers. Bioch Syst Ecol. 2015;60:18692.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.bse.2015.04.021"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Siberian%20wild%20rye%20%28%20Elymus%20sibiricus%2C%20l.%29%3A%20genetic%20diversity%20of%20germplasm%20determined%20using%20DNA%20fingerprinting%20and%20SCoT%20markers&amp;author=WG.%20Xie&amp;author=JC.%20Zhang&amp;author=XH.%20Zhao&amp;author=JQ.%20Zhang&amp;author=YR.%20Wang&amp;journal=Bioch%20Syst%20Ecol&amp;volume=60&amp;pages=186-192&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Shan Z, Wu HL, Li CL, Chen H, Wu Q. Improved SDS method for general plant genomic DNA extraction. Guangdong Agri Sci. 2011;8:1135.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Improved%20SDS%20method%20for%20general%20plant%20genomic%20DNA%20extraction&amp;author=Z.%20Shan&amp;author=HL.%20Wu&amp;author=CL.%20Li&amp;author=H.%20Chen&amp;author=Q.%20Wu&amp;journal=Guangdong%20Agri%20Sci&amp;volume=8&amp;pages=113-115&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Pritchard JK, Stephens M, Donnelly P. Inference of population structure from multilocus genotype data. Genetics. 2000;155:94559.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10835412"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1461096"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Inference%20of%20population%20structure%20from%20multilocus%20genotype%20data&amp;author=JK.%20Pritchard&amp;author=M.%20Stephens&amp;author=P.%20Donnelly&amp;journal=Genetics&amp;volume=155&amp;pages=945-959&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Evanno G, Regnaut S, Goudet J. Detecting the number of clusters of individuals using the software structure: a simulation study. Mol Ecol. 2005;14:261120.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1365-294X.2005.02553.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15969739"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Detecting%20the%20number%20of%20clusters%20of%20individuals%20using%20the%20software%20structure%3A%20a%20simulation%20study&amp;author=G.%20Evanno&amp;author=S.%20Regnaut&amp;author=J.%20Goudet&amp;journal=Mol%20Ecol&amp;volume=14&amp;pages=2611-2620&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR24" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Rohlf FJ. NTSYS-pc: numerical taxonomy and multivariate analysis system. New York, NY, USA: Exeter Software; 1992.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=NTSYS-pc%3A%20numerical%20taxonomy%20and%20multivariate%20analysis%20system&amp;author=FJ.%20Rohlf&amp;publication_year=1992"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR25" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Cabo SL, Carvalho A, Martins LP, Martn A, Lima-Brito JE. Potential of start Codon targeted (SCoT) markers for DNA fingerprinting of newly synthesized tritordeums and their respective parents. J Appl Genet. 2014;55:30712.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s13353-014-0211-3"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24733248"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Potential%20of%20start%20Codon%20targeted%20%28SCoT%29%20markers%20for%20DNA%20fingerprinting%20of%20newly%20synthesized%20tritordeums%20and%20their%20respective%20parents&amp;author=SL.%20Cabo&amp;author=A.%20Carvalho&amp;author=LP.%20Martins&amp;author=A.%20Mart%C3%ADn&amp;author=JE.%20Lima-Brito&amp;journal=J%20Appl%20Genet&amp;volume=55&amp;pages=307-312&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR26" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Huang LK, Huang X, Yan HD, Yin GH, Zhang XQ, Tian Y, et al. Constructing DNA fingerprinting of Hemarthria cultivars using EST-SSR and SCoT markers. Genet Resour Crop Ev. 2014;61:104755.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s10722-014-0107-4"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Constructing%20DNA%20fingerprinting%20of%20Hemarthria%20cultivars%20using%20EST-SSR%20and%20SCoT%20markers&amp;author=LK.%20Huang&amp;author=X.%20Huang&amp;author=HD.%20Yan&amp;author=GH.%20Yin&amp;author=XQ.%20Zhang&amp;author=Y.%20Tian&amp;journal=Genet%20Resour%20Crop%20Ev&amp;volume=61&amp;pages=1047-1055&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR27" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Powell W, Machray GC, Jim P. Polymorphism revealed by simple sequence repeats. Trends Plant Sci. 1996;7:21522.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/S1360-1385(96)86898-0"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Polymorphism%20revealed%20by%20simple%20sequence%20repeats&amp;author=W.%20Powell&amp;author=GC.%20Machray&amp;author=P.%20Jim&amp;journal=Trends%20Plant%20Sci&amp;volume=7&amp;pages=215-222&amp;publication_year=1996"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR28" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yan JJ, Bai SQ, Zhang XQ, You MH, Zhang CB, Li DX, et al. Genetic diversity of wild <em class="EmphasisTypeItalic">Elymus sibiricus</em> germplasm from the Qinghai-Tibetan plateau in China detected by SRAP markers. Acta Pratac Sin. 2010;19:17383.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20of%20wild%20Elymus%20sibiricus%20germplasm%20from%20the%20Qinghai-Tibetan%20plateau%20in%20China%20detected%20by%20SRAP%20markers&amp;author=JJ.%20Yan&amp;author=SQ.%20Bai&amp;author=XQ.%20Zhang&amp;author=MH.%20You&amp;author=CB.%20Zhang&amp;author=DX.%20Li&amp;journal=Acta%20Pratac%20Sin&amp;volume=19&amp;pages=173-183&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR29" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhang JC, Xie WG, Wang YR, Zhao XH. Potential of start Codon targeted (SCoT) markers to estimate genetic diversity and relationships among Chinese <em class="EmphasisTypeItalic">Elymus sibiricus</em> accessions. Molecules. 2015;20:59876001.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.3390/molecules20045987"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25853316"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Potential%20of%20start%20Codon%20targeted%20%28SCoT%29%20markers%20to%20estimate%20genetic%20diversity%20and%20relationships%20among%20Chinese%20Elymus%20sibiricus%20accessions&amp;author=JC.%20Zhang&amp;author=WG.%20Xie&amp;author=YR.%20Wang&amp;author=XH.%20Zhao&amp;journal=Molecules&amp;volume=20&amp;pages=5987-6001&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li></ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#TheAuthors" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="TheAuthors">Copyright</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="TheAuthors"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">&nbsp;The Author(s)&nbsp;2017</div></div></section></div>
                        


                    </article>
                </main>

                <div class="c-content-layout__side u-text-sm">
                    <aside>
                        <div class="p-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="//hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-017-0053-1" data-track="click" data-track-category="PDF Download"
            data-track-action="Click download article PDF"
            data-track-label="10.1186/s41065-017-0053-1">Download PDF</a>
        </p>
    
                            
                            
    <button type="button"
       class="c-button c-button--secondary c-button--full-width c-button--expandable"
       data-toggle="collapse"
       data-target="export-citations"
       data-test="export-citations-button"
       data-sticky-update="true"
       aria-controls="export-citations"
       aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/s41065-017-0053-1"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0053-1?format=refman&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/s41065-017-0053-1"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0053-1?format=refman&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/s41065-017-0053-1"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0053-1?format=refman&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/s41065-017-0053-1"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0053-1?format=endnote&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/s41065-017-0053-1"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0053-1?format=endnote&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/s41065-017-0053-1"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0053-1?format=endnote&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/s41065-017-0053-1"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0053-1?format=bibtex&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/s41065-017-0053-1"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0053-1?format=bibtex&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/s41065-017-0053-1"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-017-0053-1?format=bibtex&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
    </div>

                        </div>
                        
    
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
                                <div id="article-access-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/access-count/10.1186/s41065-017-0053-1 #accessCount-container"></div>
                                <div id="citation-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/citation-count/10.1186/s41065-017-0053-1 #count-container"></div>
                                <div id="altmetric-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/altmetric/10.1186/s41065-017-0053-1 #altmetric-container"></div>
                            </ul>
                        </div>
                        
    <div id="sociaMediaShare" class="p-separator u-js-shown">
        <h3 class="p-section-title">Share this article</h3>
        <ul id="shareButtons" class="c-social-links">
            <li class="c-social-links__item">
                <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-017-0053-1">
                    <span class="u-visually-hidden">Share on Twitter</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-twitter"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-017-0053-1">
                    <span class="u-visually-hidden">Share on Facebook</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-facebook"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click"
                        data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-017-0053-1">
                    <span class="u-visually-hidden">Share on LinkedIn</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-linkedin"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-017-0053-1">
                    <span class="u-visually-hidden">Share on Weibo</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-weibo"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnGooglePlus" class="js-btnShareOnGplus" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Google+" data-track-label="10.1186/s41065-017-0053-1">
                    <span class="u-visually-hidden">Share on Google Plus</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-gplus"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-017-0053-1">
                    <span class="u-visually-hidden">Share on Reddit</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-reddit"></use>
                    </svg>
                </button>
            </li>
        </ul>
    </div>
 
                        
    
        <div class="p-separator u-js-shown">
            <h3 class="p-section-title">See updates</h3>
            <button class="c-crossmark-button" data-target="crossmark" data-test="crossmark" data-track="click" data-track-category="Article Page" data-track-action="Click CrossMark" data-track-label="10.1186/s41065-017-0053-1">
                <img id="crossmark-icon" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" alt="Crossmark logo" />
            </button>
        </div>
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
                                <li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/s41065-017-0053-1" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=738&doi=10.1186%2Fs41065-017-0053-1&volume=154&issue=1&title=Hybrid+identification+and+genetic+variation+of+Elymus+sibiricus+hybrid+populations+using+EST-SSR+markers&author_name=Xuhong+Zhao&start_page=1&end_page=7">Order reprint</a>

                                </li>
                            </ul>
                        </div>
                    </aside>
                    
    <div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">Hereditas</span>
                        </h4>
                        <p class="c-journal-footer__issn">ISSN: 1601-5223</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
                                
                                    <li class="c-journal-footer__contact-item">Submission enquiries: <a href="https://www.editorialmanager.com/here/" target="_blank">Access here and click Contact Us</a></li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a></li>
                                
                            </ul>
                        </div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class='tracker' style='display:none' src='/track/article/10.1186/s41065-017-0053-1' alt=""/>

         
    <footer>
        <div class="ctx-publisher-footer">
    <div class="c-navbar">
        <div class="u-container">
            <div class="c-navbar__logo u-margin-right-xl">
                <a href="https://www.biomedcentral.com">
                    <img src=/static/images/bmc/logos/logo-bmc-aj-594e03e20d.svg alt="BMC logo" data-test="navbar-logo-footer">

                </a>
            </div>
            <div class="c-navbar__content">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


            </div>
        </div>
    </div>
    <div class="c-publisher-footer" data-test="publisher-footer">
        <div class="u-container">
            
            <div class="c-flex c-flex--wrap c-flex--space-between" role="navigation" data-test="publisher-footer-menu">
                <div class="c-flex" aria-label="Publisher footer links">
                    
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/login">Receive BMC newsletters</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/account">Manage article alerts</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/policies">Policies</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/accessibility">Accessibility</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/press-centre">Press center</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/contact-us">Contact us</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/jobs">Careers</a>
                                    </li>
                                
                            </ul>
                        
                    
                </div>
                <div class="u-margin-bottom-lg">
                    <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                    <ul class="c-flex u-reset-list" data-test="footer-social-links">
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://twitter.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Twitter link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Twitter page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-twitter-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://www.facebook.com/BioMedCentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Facebook link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Facebook page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-facebook-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://plus.google.com/106226337984152901734/"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Google Plus link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Google Plus page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-gplus-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="http://www.weibo.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Weibo link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Weibo page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-weibo-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </div>
            <p class="u-reset-margin">
                By using this website, you agree to our
                <a class="u-text-inherit" href="//www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
                <a class="u-text-inherit" href="//www.biomedcentral.com/privacy-statement">Privacy
                    statement</a> and
                <a class="u-text-inherit" href="//www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
                
                    <a class="optanon-toggle-display u-text-inherit" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
                
            </p>
        </div>
    </div>
</div>


        <div class="c-sn-footer c-sn-footer--neutral">
    <div class="u-container">
        <img src=/static/images/bmc/logos/logo-springernature-8bd4e8d82f.svg class="c-sn-footer__logo" alt="Springer Nature Logo" itemprop="logo" >
        <p data-test="copyright">&#169; 2018 BioMed Central Ltd unless otherwise stated. Part of
            <a href="http://www.springernature.com"
               data-track="click"
               data-track-action="Click Springer Nature link"
               data-track-category="Corporate footer"
               itemscope itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>

    </div>

    
    <script type="text/javascript" src="//cdn.pbgrd.com/bmc.js" async="async"></script>
    <noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>
    <!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker -->

<script id="springerlink-event-tracker" src="https://event-tracker.springernature.com/dist/eventTracker.js"></script>
<script type="text/javascript">
    (function(){
        var dataLayer = window.dataLayer[0];
        var doi = dataLayer.content.article.doi;
        var imprint = dataLayer.content.contentInfo.imprint;
        var bpids = dataLayer.session.authentication.authenticationID;
        var publisher;
        var pageType = dataLayer.page.category.pageType;
        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }
        var eventObject = {'content_type' : pageType,'doi' : doi};

        if (imprint === 'BioMed Central') { publisher = 'BMC' }
        else if (imprint === 'SpringerOpen') { publisher = 'SpringerOpen' }
        else { throw new Error('Event Tracker Error: The publisher has an unexpected value') }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }
        new EventTracker({'platform':publisher}).sendEvent('display', eventObject);
    })();
</script>


    </body>
</html>

