<!DOCTYPE html>
<html lang="en" class="no-js">
    <head>
        <title>Comparative transcriptome reveal the potential adaptive evolutionary genes in Andrias davidianus | Hereditas | Full Text</title>

        

    <meta name="citation_abstract" content="To search the evidence of molecular evolution mechanism for aquatic and cave habitat in Andrias davidianus, the evolution analysis was carried out among several species transcriptome data. The transcriptome data of Notophthalmus viridescens, Xenopus tropicalis, Cynops pyrrhogaster, Hynobius chinensis and A. davidianus were obtained from the Genbank and reassembled except Xenopus tropicalis. The BLAST search of transcriptome data obtained 1244 single-copy orthologous genes among five species. A phylogenetic tree showed A. davidianus to have the closest relationship to H. chinensis. Fourteen positively selected genes were detected in A. davidianus and N. vridescens group and fifteen in A. davidianus and H. chinensis group. Five genes were shared in the both groups which involved in the immune system, suggesting that A. davidianus adaptation to an aquatic and cave environment required rapid evolution of the immune system compared to N. viridescens and H. chinensis."/>

    <meta name="description" content="To search the evidence of molecular evolution mechanism for aquatic and cave habitat in Andrias davidianus, the evolution analysis was carried out among several species transcriptome data. The transcriptome data of Notophthalmus viridescens, Xenopus tropicalis, Cynops pyrrhogaster, Hynobius chinensis and A. davidianus were obtained from the Genbank and reassembled except Xenopus tropicalis. The BLAST search of transcriptome data obtained 1244 single-copy orthologous genes among five species. A phylogenetic tree showed A. davidianus to have the closest relationship to H. chinensis. Fourteen positively selected genes were detected in A. davidianus and N. vridescens group and fifteen in A. davidianus and H. chinensis group. Five genes were shared in the both groups which involved in the immune system, suggesting that A. davidianus adaptation to an aquatic and cave environment required rapid evolution of the immune system compared to N. viridescens and H. chinensis."/>

    <meta name="journal_id" content="41065"/>

    <meta name="dc.title" content="Comparative transcriptome reveal the potential adaptive evolutionary genes in Andrias davidianus"/>

    <meta name="dc.source" content="Hereditas 2018 155:1"/>

    <meta name="dc.format" content="text/html"/>

    <meta name="dc.publisher" content="BioMed Central"/>

    <meta name="dc.date" content="2018-02-20"/>

    <meta name="dc.type" content="Report"/>

    <meta name="dc.language" content="En"/>

    <meta name="dc.copyright" content="2018 The Author(s)"/>

    <meta name="dc.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="dc.description" content="To search the evidence of molecular evolution mechanism for aquatic and cave habitat in Andrias davidianus, the evolution analysis was carried out among several species transcriptome data. The transcriptome data of Notophthalmus viridescens, Xenopus tropicalis, Cynops pyrrhogaster, Hynobius chinensis and A. davidianus were obtained from the Genbank and reassembled except Xenopus tropicalis. The BLAST search of transcriptome data obtained 1244 single-copy orthologous genes among five species. A phylogenetic tree showed A. davidianus to have the closest relationship to H. chinensis. Fourteen positively selected genes were detected in A. davidianus and N. vridescens group and fifteen in A. davidianus and H. chinensis group. Five genes were shared in the both groups which involved in the immune system, suggesting that A. davidianus adaptation to an aquatic and cave environment required rapid evolution of the immune system compared to N. viridescens and H. chinensis."/>

    <meta name="prism.issn" content="1601-5223"/>

    <meta name="prism.publicationName" content="Hereditas"/>

    <meta name="prism.publicationDate" content="2018-02-20"/>

    <meta name="prism.volume" content="155"/>

    <meta name="prism.number" content="1"/>

    <meta name="prism.section" content="Report"/>

    <meta name="prism.startingPage" content="18"/>

    <meta name="prism.copyright" content="2018 The Author(s)"/>

    <meta name="prism.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="prism.url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0056-6"/>

    <meta name="prism.doi" content="doi:10.1186/s41065-018-0056-6"/>

    <meta name="citation_pdf_url" content="https://hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-018-0056-6"/>

    <meta name="citation_fulltext_html_url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0056-6"/>

    <meta name="citation_journal_title" content="Hereditas"/>

    <meta name="citation_publisher" content="BioMed Central"/>

    <meta name="citation_issn" content="1601-5223"/>

    <meta name="citation_title" content="Comparative transcriptome reveal the potential adaptive evolutionary genes in Andrias davidianus"/>

    <meta name="citation_volume" content="155"/>

    <meta name="citation_issue" content="1"/>

    <meta name="citation_publication_date" content="2018/12"/>

    <meta name="citation_online_date" content="2018/02/20"/>

    <meta name="citation_firstpage" content="18"/>

    <meta name="citation_article_type" content="Brief report"/>

    <meta name="citation_fulltext_world_readable" content=""/>

    <meta name="dc.identifier" content="doi:10.1186/s41065-018-0056-6"/>

    <meta name="DOI" content="10.1186/s41065-018-0056-6"/>

    <meta name="citation_doi" content="10.1186/s41065-018-0056-6"/>

    <meta name="dc.creator" content="Qiaomu Hu"/>

    <meta name="dc.creator" content="Quanhe Wang"/>

    <meta name="dc.creator" content="Yan Meng"/>

    <meta name="dc.creator" content="Haifeng Tian"/>

    <meta name="dc.creator" content="Hanbing Xiao"/>

    <meta name="dc.subject" content="Animal Genetics and Genomics"/>

    <meta name="dc.subject" content="Evolutionary Biology"/>

    <meta name="citation_reference" content="citation_journal_title=Copeia; citation_title=Amphibian cytogenetics and evolution; citation_author=LA Lowcock, DM Green, SK Sessions; citation_volume=1992; citation_issue=2; citation_publication_date=1991; citation_pages=588; citation_doi=10.2307/1446229; citation_id=CR1"/>

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci U S A; citation_title=Phylogeny, evolution, and biogeography of Asiatic salamanders (Hynobiidae); citation_author=P Zhang, YQ Chen, H Zhou, YF Liu, XL Wang, TJ Papenfuss; citation_volume=103; citation_issue=19; citation_publication_date=2006; citation_pages=7360-7365; citation_doi=10.1073/pnas.0602325103; citation_id=CR2"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Biotechnol J; citation_title=Large-scale transcriptome analysis in chickpea (Cicer arietinum L.), an orphan legume crop of the semi-arid tropics of Asia and Africa; citation_author=PJ Hiremath, A Farmer, SB Cannon, J Woodward, H Kudapa, R Tuteja; citation_volume=9; citation_issue=8; citation_publication_date=2011; citation_pages=922-931; citation_doi=10.1111/j.1467-7652.2011.00625.x; citation_id=CR3"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Genomics; citation_title=Sequencing and characterization of the guppy (Poecilia reticulata) transcriptome; citation_author=BA Fraser, CJ Weadick, I Janowitz, FH Rodd, KA Hughes; citation_volume=12; citation_issue=1; citation_publication_date=2011; citation_pages=202; citation_doi=10.1186/1471-2164-12-202; citation_id=CR4"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Genomics; citation_title=Comparative transcriptome analysis and marker development of two closely related primrose species (Primula poissonii and Primula wilsonii); citation_author=L Zhang, HF Yan, W Wu, H Yu, XJ Ge; citation_volume=14; citation_issue=1; citation_publication_date=2013; citation_pages=329; citation_doi=10.1186/1471-2164-14-329; citation_id=CR5"/>

    <meta name="citation_reference" content="citation_journal_title=Gene; citation_title=Transcriptome analysis reveals positive selection on the divergent between topmouth culter and zebrafish; citation_author=L Ren, XJ Tan, YF Xiong, K Xu, Y Zhou, H Zhong; citation_volume=552; citation_issue=2; citation_publication_date=2014; citation_pages=265-271; citation_doi=10.1016/j.gene.2014.09.053; citation_id=CR6"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Genomics; citation_title=Transcriptome characterisation of Pinus Tabuliformis and evolution of genes in the Pinus phylogeny; citation_author=SH Niu, ZX Li, HW Yuan, XY Chen, Y Li, W Li; citation_volume=14; citation_issue=1; citation_publication_date=2013; citation_pages=263; citation_doi=10.1186/1471-2164-14-263; citation_id=CR7"/>

    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Earliest known crown-group salamanders; citation_author=KQ Gao, NH Shubin; citation_volume=422; citation_issue=6930; citation_publication_date=2003; citation_pages=424-428; citation_doi=10.1038/nature01491; citation_id=CR8"/>

    <meta name="citation_reference" content="citation_journal_title=J Comput Biol; citation_title=A greedy algorithm for aligning DNA sequences; citation_author=Z Zhang, S Schwartz, L Wagner, W Miller; citation_volume=7; citation_issue=1-2; citation_publication_date=2000; citation_pages=203-214; citation_doi=10.1089/10665270050081478; citation_id=CR9"/>

    <meta name="citation_reference" content="citation_journal_title=Genome Res; citation_title=OrthoMCL: identification of ortholog groups for eukaryotic genomes; citation_author=L Li, SC Jr, DS Roos; citation_volume=13; citation_publication_date=2003; citation_pages=2178-2189; citation_doi=10.1101/gr.1224503; citation_id=CR10"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics; citation_title=VennDiagram: a package for the generation of highly-customizable Venn and Euler diagrams in R; citation_author=H Chen, PC Boutros; citation_volume=12; citation_issue=1; citation_publication_date=2011; citation_pages=35; citation_doi=10.1186/1471-2105-12-35; citation_id=CR11"/>

    <meta name="citation_reference" content="citation_journal_title=Bmc Bioinformatics; citation_title=A perl package and an alignment tool for phylogenetic networks; citation_author=V Gabriel, R Francesc, C Gabriel; citation_volume=9; citation_issue=1; citation_publication_date=2008; citation_pages=1-5; citation_doi=10.1186/1471-2105-9-1; citation_id=CR12"/>

    <meta name="citation_reference" content="citation_journal_title=Genetics; citation_title=Combining bioinformatics and phylogenetics to identify large sets of single-copy orthologous genes (COSII) for comparative, evolutionary and systematic studies: a test case in the euasterid plant clade; citation_author=F Wu, LA Mueller, D Crouzillat, V P&#233;tiard, SD Tanksley; citation_volume=174; citation_issue=3; citation_publication_date=2006; citation_pages=1407; citation_doi=10.1534/genetics.106.062455; citation_id=CR13"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput; citation_author=RC Edgar; citation_volume=32; citation_issue=5; citation_publication_date=2004; citation_pages=1792-1797; citation_doi=10.1093/nar/gkh340; citation_id=CR14"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=PAML 4: Phylogenetic analysis by maximum likelihood; citation_author=Z Yang; citation_volume=24; citation_issue=8; citation_publication_date=2007; citation_pages=1586-1591; citation_doi=10.1093/molbev/msm088; citation_id=CR15"/>

    <meta name="citation_reference" content="citation_journal_title=Trends Ecol Evol; citation_title=Statistical methods for detecting molecular adaptation; citation_author=Z Yang, JP Bielawski; citation_volume=15; citation_issue=12; citation_publication_date=2000; citation_pages=496; citation_doi=10.1016/S0169-5347(00)01994-7; citation_id=CR16"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Commun; citation_title=European sea bass genome and its variation provide insights into adaptation to euryhalinity and speciation; citation_author=M Tine, H Kuhl, PA Gagnaire, B Louro, E Desmarais, RS Martins; citation_volume=5; citation_publication_date=2014; citation_pages=5770; citation_doi=10.1038/ncomms6770; citation_id=CR17"/>

    <meta name="citation_reference" content="citation_journal_title=Gigascience; citation_title=The draft genome of blunt snout bream (Megalobrama amblycephala) reveals the development of intermuscular bone and adaptation to herbivorous diet; citation_author=H Liu, CL Chen, ZX Gao, JM Min, YM Gu, JB Jian; citation_volume=6; citation_publication_date=2017; citation_pages=1-13; citation_id=CR18"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Genet; citation_title=The genome and transcriptome of Japanese flounder provide insights into flatfish asymmetry; citation_author=CW Shao, BL Bao, ZY Xie, XY Chen, B Li, XD Jia; citation_volume=49; citation_issue=1; citation_publication_date=2017; citation_pages=119-124; citation_doi=10.1038/ng.3732; citation_id=CR19"/>

    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Evolution of genes and genomes on the drosophila phylogeny; citation_author=AG Clark, MB Eisen, DR Smith, CM Bergman, B Oliver, TA Markow; citation_volume=450; citation_issue=7167; citation_publication_date=2007; citation_pages=203-218; citation_doi=10.1038/nature06341; citation_id=CR20"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=Transcriptomic analysis of endangered Chinese salamander: identification of immune, sex and reproduction-related genes and genetic markers; citation_author=RB Che, YN Sun, RX Wang, TJ Xu; citation_volume=9; citation_publication_date=2014; citation_pages=e87940; citation_doi=10.1371/journal.pone.0087940; citation_id=CR21"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Genomics; citation_title=Transcriptomic analysis of the differentiating ovary of the protogynous ricefield eel Monopterus albus; citation_author=JF Cai, W Yang, D Chen, YZ Zhang, Z He, WM Zhang; citation_volume=18; citation_issue=1; citation_publication_date=2017; citation_pages=573; citation_doi=10.1186/s12864-017-3953-6; citation_id=CR22"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Phylogenet Evol; citation_title=A 454 sequencing approach for large scale phylogenomic analysis of the common emperor scorpion (Pandinus imperator); citation_author=F Roeding, J Borner, M Kube, S Klages, R Reinhardt, T Burmester; citation_volume=53; citation_issue=3; citation_publication_date=2009; citation_pages=826-834; citation_doi=10.1016/j.ympev.2009.08.014; citation_id=CR23"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=A phylogenomic approach to resolve the arthropod tree of life; citation_author=K Meusemann, BM von Reumont, S Simon, F Roeding, S Strauss, P Kuck; citation_volume=27; citation_issue=11; citation_publication_date=2010; citation_pages=2451-2464; citation_doi=10.1093/molbev/msq130; citation_id=CR24"/>

    <meta name="citation_reference" content="citation_journal_title=Genetics; citation_title=Evolutionary expressed sequence tag analysis of drosophila female reproductive tracts identifies genes subjected to positive selection; citation_author=WJ Swanson, A Wong, MF Wolfner, CF Aquadro; citation_volume=168; citation_issue=3; citation_publication_date=2004; citation_pages=1457-1465; citation_doi=10.1534/genetics.104.030478; citation_id=CR25"/>

    <meta name="citation_reference" content="citation_journal_title=Fish Shellfish Immunol; citation_title=Identification and characterization of a cystatin gene from Chinese mitten crab Eriocheir sinensis; citation_author=FM Li, XM Gai, LL Wang, LS Song, H Zhang, LM Qiu; citation_volume=29; citation_issue=3; citation_publication_date=2010; citation_pages=521-529; citation_doi=10.1016/j.fsi.2010.05.015; citation_id=CR26"/>

    <meta name="citation_reference" content="citation_journal_title=Eur J Cancer; citation_title=Aven blocks DNA damage-induced apoptosis by stabilising Bcl-xL; citation_author=O Kutuk, SG Temel, S Tolunay, H Basaga; citation_volume=46; citation_issue=13; citation_publication_date=2010; citation_pages=2494-2505; citation_doi=10.1016/j.ejca.2010.06.011; citation_id=CR27"/>

    <meta name="citation_reference" content="citation_journal_title=Eur J Surg Oncol; citation_title=Combined evaluation of centromere protein H and Ki-67 as prognostic biomarker for patients with gastric carcinoma; citation_author=WL He, YH Li, DJ Yang, W Song, XL Chen, FK Liu; citation_volume=39; citation_issue=2; citation_publication_date=2013; citation_pages=141-149; citation_doi=10.1016/j.ejso.2012.08.023; citation_id=CR28"/>

    <meta name="citation_reference" content="citation_journal_title=J Hepatol; citation_title=Characterization of the CD40L/Oncostatin M/Oncostatin M receptor axis as an antiviral and immunostimulatory system disrupted in chronic HCV infection; citation_author=E Larrea, I Echeverria, JI Riezu-Boj, R Aldabe, L Guembe, I Sola; citation_volume=60; citation_issue=3; citation_publication_date=2014; citation_pages=482-489; citation_doi=10.1016/j.jhep.2013.10.016; citation_id=CR29"/>

    <meta name="citation_author" content="Qiaomu Hu"/>

    <meta name="citation_author_institution" content="Yangtze River Fisheries Research Institute, Chinese Academy of Fishery Sciences, Wuhan, China"/>

    <meta name="citation_author" content="Quanhe Wang"/>

    <meta name="citation_author_institution" content="Yangtze River Fisheries Research Institute, Chinese Academy of Fishery Sciences, Wuhan, China"/>

    <meta name="citation_author_institution" content="College of Life Science, Yangtze University, Jingzhou, China"/>

    <meta name="citation_author" content="Yan Meng"/>

    <meta name="citation_author_institution" content="Yangtze River Fisheries Research Institute, Chinese Academy of Fishery Sciences, Wuhan, China"/>

    <meta name="citation_author" content="Haifeng Tian"/>

    <meta name="citation_author_institution" content="Yangtze River Fisheries Research Institute, Chinese Academy of Fishery Sciences, Wuhan, China"/>

    <meta name="citation_author" content="Hanbing Xiao"/>

    <meta name="citation_author_institution" content="Yangtze River Fisheries Research Institute, Chinese Academy of Fishery Sciences, Wuhan, China"/>



        <meta charset="UTF-8"/>
        <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
        <meta name="viewport" content="width=device-width, initial-scale=1"/>

        <!-- For IE 9 and below -->
<!--[if IE]><link rel="shortcut icon" href=/static/images/favicons//favicon.ico><![endif]-->
<!-- Touch Icons - iOS and Android 2.1+ -->
<link rel="apple-touch-icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>
<!-- Firefox, Chrome, Safari, IE 11+ and Opera -->
<link rel="icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>


        <script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script>
        <script>(function(d){if (sessionStorage.fontsLoaded){d.style.setProperty('--font-family-sans', 'Europa');d.style.setProperty('--font-family-serif', 'Noto Serif')}})(document.documentElement);</script>

        <link rel="stylesheet" media="screen" href=/static/styles/basic/basic-a31ca85bc2.css>
<link rel="stylesheet" media="print" href=/static/styles/bmc/print-3a6240f456.css>

<link rel="stylesheet" class="js-ctm" href=/static/styles/bmc/main-84414d3d47.css
      media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">

        

        <script src="/static/js/app-bundle-8d61e6ef37.js" defer></script>
        
        <script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script>
        
    <script type="text/javascript">
        config = {
            env: 'live',
            site: 'hereditasjournal.biomedcentral.com',
            siteWithPath: 'hereditasjournal.biomedcentral.com' + window.location.pathname,
            twitterHashtag: '',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/s41065-018-0056-6',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central'
        };
    </script>

        
    <script type="text/javascript">
        dataLayer = [{"content":{"article":{"doi":"10.1186/s41065-018-0056-6","articleType":"Brief report","peerReviewType":"Closed","supplement":null,"keywords":"Andrias davidianus;Comparative transcriptome;Evolution analysis;Adaptive genes"},"contentInfo":{"imprint":"BioMed Central","title":"Comparative transcriptome reveal the potential adaptive evolutionary genes in Andrias davidianus","publishedAt":1519084800000,"publishedAtDate":"2018-02-20","author":["Qiaomu Hu","Quanhe Wang","Yan Meng","Haifeng Tian","Hanbing Xiao"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":"CC BY + CC0","openAccess":true},"environment":"live"},"journal":{"siteKey":"hereditasjournal.biomedcentral.com","volume":"155","issue":"1","title":"Hereditas","type":"AJs Life","journalID":41065,"gaCode":"UA-62066850-11","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Brief report","publishingSegment":"Genetics & Heredity (Life)"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}}}]

    </script>

        <script src="https://cdn.cookielaw.org/consent/7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script>
        
    
        
    

        
    
        
            
                <!-- Google Tag Manager -->
                <script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script>
                <!-- End Google Tag Manager -->
            
        
    


        
    
    <link rel="canonical" href="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0056-6"/>
    

        
        
        <meta property="og:url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0056-6"/>
        <meta property="og:type" content="article"/>
        <meta property="og:site_name" content="Hereditas"/>
        <meta property="og:title" content="Comparative transcriptome reveal the potential adaptive evolutionary genes in Andrias davidianus"/>
        <meta property="og:description" content="To search the evidence of molecular evolution mechanism for aquatic and cave habitat in Andrias davidianus, the evolution analysis was carried out among several species transcriptome data. The transcriptome data of Notophthalmus viridescens, Xenopus tropicalis, Cynops pyrrhogaster, Hynobius chinensis and A. davidianus were obtained from the Genbank and reassembled except Xenopus tropicalis. The BLAST search of transcriptome data obtained 1244 single-copy orthologous genes among five species. A phylogenetic tree showed A. davidianus to have the closest relationship to H. chinensis. Fourteen positively selected genes were detected in A. davidianus and N. vridescens group and fifteen in A. davidianus and H. chinensis group. Five genes were shared in the both groups which involved in the immune system, suggesting that A. davidianus adaptation to an aquatic and cave environment required rapid evolution of the immune system compared to N. viridescens and H. chinensis."/>
        <meta property="og:image" content="https://images.springer.com/sgw/journals/medium/41065.jpg"/>
    
    </head>

    <body class="journal journal-fulltext"
          
          >
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK"
                            height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><feColorMatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"/></filter></defs><symbol id="icon-arrow-down" viewBox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"/></symbol><symbol id="icon-arrow-left-bullet" viewBox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"/></symbol><symbol id="icon-arrow-right-bullet" viewBox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"/></symbol><symbol id="icon-arrow-right" viewBox="0 0 32 64"><path d="M0 0v64l32-32z"/></symbol><symbol id="icon-close" viewBox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"/></symbol><symbol id="icon-community" viewBox="0 0 60.643 31.01"><mask maskUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><path fill="#FFF" d="M0 31.01h60.643V0H0z" filter="url(#a)"/></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"/></symbol><symbol id="icon-download" viewBox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"/></symbol><symbol id="icon-ext-link" viewBox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"/><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"/><path fill="#676767" d="M9.7 0L16 6.2V0z"/></symbol><symbol id="icon-info-bordered" viewBox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"/><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"/></symbol><symbol id="icon-opr" viewBox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"/><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"/></symbol><symbol id="icon-quotation" viewBox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"/></symbol><symbol id="icon-remove" viewBox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"/><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"/></symbol><symbol id="icon-ribbon" viewBox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"/></symbol><symbol id="icon-rss" viewBox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"/><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"/><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"/></g></symbol><symbol id="icon-search" viewBox="0 0 16 16"><path d="M6.276 12.257C2.815 12.257 0 9.508 0 6.128 0 2.749 2.815 0 6.276 0c3.462 0 6.277 2.749 6.277 6.128.002 3.38-2.815 6.129-6.277 6.129zm0-10.772c-2.622 0-4.755 2.083-4.755 4.643 0 2.562 2.133 4.644 4.755 4.644 2.623 0 4.756-2.082 4.756-4.644 0-2.56-2.134-4.643-4.756-4.643z"/><path d="M16 13.751L13.641 16l-4.875-4.82 2.36-2.248L16 13.751z"/></symbol><symbol id="icon-speech-bubble" viewBox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"/></symbol><symbol id="icon-error" viewBox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"/></symbol><symbol id="icon-info" viewBox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"/></symbol><symbol id="icon-success" viewBox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"/></symbol><symbol id="icon-warning" viewBox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"/></symbol><symbol id="icon-facebook-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"/></symbol><symbol id="icon-facebook" viewBox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"/><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"/></symbol><symbol id="icon-gplus-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M480.5 273.117c.123.076.26.185.416.334.146.152.289.343.432.562.135.211.258.459.359.746.086.286.129.619.129 1.004-.012.7-.164 1.262-.465 1.683a5.374 5.374 0 0 1-.461.562c-.186.176-.375.35-.58.527a2.65 2.65 0 0 0-.322.41.908.908 0 0 0-.176.553c0 .205.059.375.18.508.102.129.199.238.299.336l.662.545a6.48 6.48 0 0 1 1.086 1.119c.293.415.445.955.459 1.625 0 .949-.42 1.789-1.256 2.521-.869.76-2.123 1.15-3.762 1.177-1.373-.015-2.4-.308-3.074-.879-.682-.534-1.023-1.172-1.023-1.917 0-.359.111-.766.336-1.211.215-.443.602-.837 1.172-1.174a6.126 6.126 0 0 1 1.998-.725 15.415 15.415 0 0 1 1.723-.173 3.996 3.996 0 0 1-.377-.596 1.347 1.347 0 0 1-.193-.74c0-.174.023-.317.074-.433a8.83 8.83 0 0 1 .121-.338 5.973 5.973 0 0 1-.633.035c-1.043-.012-1.838-.339-2.381-.979-.574-.603-.861-1.3-.861-2.097 0-.961.404-1.832 1.217-2.616.559-.46 1.135-.757 1.736-.898a8.494 8.494 0 0 1 1.67-.182h3.916l-1.209.704h-1.207v.005l-.005.002zm.754 11.276c0-.498-.162-.932-.484-1.302-.346-.348-.881-.778-1.609-1.29a4.045 4.045 0 0 0-.439-.02c-.102-.016-.354 0-.768.039a8.141 8.141 0 0 0-1.244.275c-.1.036-.24.096-.42.17a2.713 2.713 0 0 0-.553.355 1.985 1.985 0 0 0-.459.592c-.145.25-.215.551-.215.901 0 .687.311 1.253.932 1.698.596.444 1.404.68 2.432.688.928-.014 1.629-.217 2.117-.612.476-.385.71-.886.71-1.494zm-2.729-5.743a1.843 1.843 0 0 0 1.289-.556c.164-.249.271-.506.318-.767a7.32 7.32 0 0 0 .041-.659c0-.772-.195-1.552-.594-2.336a2.611 2.611 0 0 0-.73-.921 1.894 1.894 0 0 0-1.064-.352 1.723 1.723 0 0 0-1.324.64c-.299.437-.438.92-.428 1.457 0 .709.209 1.444.623 2.211.201.356.459.659.775.912.323.248.686.371 1.094.371zm9.698.077h-2.055v-2.056h-.996v2.056h-2.053v.998h2.053v2.052h.996v-2.052h2.055z"/></symbol><symbol id="icon-gplus" viewBox="0 0 24 24"><circle fill="#D84A36" cx="12" cy="12" r="12"/><g fill="#FFF"><path d="M12.012 12.111c-.304-.33-.07-.842.155-1.078.226-.241.639-.56.829-.74.193-.184.66-.605.674-1.868 0 0 .112-1.139-1.127-2.06h.977l1.038-.593h-3.507s-2.083-.067-3.085 1.207c-1 1.276-.966 2.871.114 3.77 1.08.898 2.371.648 2.371.648s-.197.422-.163.775c.036.353.421.849.504.965 0 0-2.386-.011-3.551 1.033-1.165 1.053-1.29 3.097 1.021 3.803 2.312.707 5.303-.055 5.906-2.277 0 0 .262-.908-.189-1.683-.459-.775-1.663-1.571-1.967-1.902zm-2.555-1.83c-.744-1-1.308-2.953 0-3.783 0 0 .955-.579 1.801.266.846.846 1.082 2.277.928 3.003-.15.727-.563.897-.739 1.008-.185.111-1.249.504-1.99-.494zm3.668 5.481c-.035.393-.083 1.254-1.491 1.664-1.409.408-3.826-.264-3.757-1.914 0 0 .006-.698.449-1.089.445-.396 1.094-.794 3.029-.819 0 0 .643.473 1.072.809.427.332.733.952.698 1.349z"/><path d="M16.323 11.055V9.803h-.752v1.252h-1.255v.748h1.255v1.256h.752v-1.256h1.248v-.748z"/></g></symbol><symbol id="icon-linkedin" viewBox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"/><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"/><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"/></g></symbol><symbol id="icon-reddit" viewBox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"/><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"/><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"/><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"/><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"/><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"/><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"/><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"/></symbol><symbol id="icon-twitter-bordered" viewBox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"/><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/></symbol><symbol id="icon-twitter" viewBox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"/><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"/></symbol><symbol id="icon-weibo-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"/><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"/><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"/></symbol><symbol id="icon-weibo" viewBox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"/><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"/><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"/><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"/><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"/><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"/></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner c-banner--info">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner" >
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1"
                 data-gpt-unitpath="/270604982/bmc/hereditasjournal/articles"
                 data-gpt-sizes="728x90,970x90"
                 data-gpt-targeting="pos=LB1;doi=10.1186/s41065-018-0056-6;kwrd=Andrias davidianus,Comparative transcriptome,Evolution analysis,Adaptive genes;pmc=L32030,L21001;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-018-0056-6&amp;kwrd=Andrias davidianus,Comparative transcriptome,Evolution analysis,Adaptive genes&amp;pmc=L32030,L21001&amp;">
                        <img data-test="gpt-advert-fallback-img"
                             src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-018-0056-6&amp;kwrd=Andrias davidianus,Comparative transcriptome,Evolution analysis,Adaptive genes&amp;pmc=L32030,L21001&amp;"
                             alt="Advertisement"
                             width="728"
                             height="90">
                    </a>
                </noscript>
            </div>
        </div>
    </div>

        
         
    <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>


<header class="c-popup-search ctx-publisher-header" id="top" data-test="publisher-header">
    <nav class="c-navbar">
        <div class="u-container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization" />
                <div class="c-logo">
    <a href="https://www.biomedcentral.com" itemprop="url">
        <img src=/static/images/bmc/logos/logo-bmc-white-aj-e3a70ae848.svg alt="BMC" itemprop="logo">
        <div class="c-logo__strapline">
            <img src=/static/images/bmc/logos/logo-bmc-white-strapline-sn-e7b1d6e67f.svg alt="Part of Springer Nature">
        </div>
    </a>
</div>
            </div>
            <button
                type="button"
                class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg"
                id="publisher-header-menu-button"
                data-toggle="collapse"
                data-test="publisher-header-menu-toggle"
                data-toggle-linked="publisher-header"
                data-target="publisher-header-search publisher-header-menu"
                aria-controls="publisher-header-menu publisher-header-search"
                aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


                <ul class="c-navbar__nav">
                    <li class="c-navbar__item">
                        <a data-replace="true"
                           data-placeholder="/placeholder/v1/username"
                           class="c-navbar__link"
                           href="https://www.biomedcentral.com/login"
                           data-test="login-link"
                           data-track="click"
                           data-track-category="Account"
                           data-track-action="Click Login Link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                    <li class="c-navbar__item">
                        <button
                            type="button"
                            class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg"
                            data-toggle="collapse"
                            data-test="header-search-button"
                            data-target="publisher-header-search publisher-header-menu publisher-header-menu-button"
                            aria-controls="publisher-header-search publisher-header-menu"
                            aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true">
                                    <use xlink:href="#icon-search"></use>
                                </svg>
                            </span>
                        </button>
                    </li>
                </ul>
            </div>
        </div>
    </nav>
    <div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field c-form-field--inline" method="GET" action="//www.biomedcentral.com/search" data-track="submit"
        data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label data-track-label="" data-test="global-search">
        <label for="publisherSearch" class="u-visually-hidden">Search all BMC articles</label>
        <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text" value="" placeholder="Search all BMC articles"/>
        <button class="c-button c-button--secondary" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true">
        <use xlink:href="#icon-search"></use>
    </svg>
</button>


        <input type="hidden" name="searchType" value="publisherSearch"/>
    </form>
</div>

            </div>
        </div>
    </div>
</header>


        
            <header class="c-journal-header c-journal-header--hereditas ctx-journal-header">
                <div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="/">
    
        
    
    <span class="c-journal-title__text ">Hereditas</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="u-container">
                        
                            <button type="button"
                                    class="c-navbar__link c-dropdown u-hidden-at-lg"
                                    data-toggle="collapse"
                                    data-test="journal-header-menu"
                                    data-toggle-linked="journal-header"
                                    data-target="journal-menu"
                                    aria-controls="journal-menu"
                                    aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav">
                                    <ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/'>Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/about'>About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href='/articles'>Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/submission-guidelines'>Submission Guidelines</a>
                                            </li>
                                        
                                    </ul>
                                </nav>
                            </div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--ocean-light-blue">
                    
                </div>
            </header>
            
        

        <div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-content-layout c-content-layout--fulltext" data-component="sticky-container">
                <div class="c-content-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3><ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc=""><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Methods">Methods</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec6" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results">Results</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec9" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Discussion">Discussion</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li></ol></nav>
                </div>

                <main class="c-content-layout__main">
                    <article itemscope itemtype="http://schema.org/ScholarlyArticle">
                        
                        
    <div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
            <li class="c-article-identifiers__item" data-test="article-category">Brief report</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
    </div>

                        <div class="FulltextWrapper"><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">Comparative transcriptome reveal the potential adaptive evolutionary genes in <em xmlns="" class="EmphasisTypeItalic">Andrias davidianus</em></h1></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true"><ul class="u-listReset"><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Qiaomu&nbsp;Hu</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup><a href="mailto:hqmu0806@163.com" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Quanhe&nbsp;Wang</span><sup><a href="#Aff1" class="AffiliationID">1</a>, <a href="#Aff2" class="AffiliationID">2</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Yan&nbsp;Meng</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Haifeng&nbsp;Tian</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup> and </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Hanbing&nbsp;Xiao</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup><a href="mailto:xhb@yfi.ac.cn" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a></li></ul></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation"><div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">Hereditas</span></span><span class="ArticleCitation_Year">2018</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">155</strong>:18</span></span></div><p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/s41065-018-0056-6">https://doi.org/10.1186/s41065-018-0056-6</a></p><p class="Copyright">© &nbsp;The Author(s)&nbsp;2018</p></div></div><ul class="ArticleHistory u-reset-list"><li class="History HistoryReceived"><strong>Received: </strong><span>22&nbsp;October&nbsp;2017</span></li><li class="History HistoryAccepted"><strong>Accepted: </strong><span>29&nbsp;January&nbsp;2018</span></li><li class="History HistoryOnlineDate"><strong>Published: </strong><span itemprop="datePublished">20&nbsp;February&nbsp;2018</span></li></ul></div>
                        
                        
                        
    

                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2><div class="c-section__content" id="Abstract"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par1" class="Para">To search the evidence of molecular evolution mechanism for aquatic and cave habitat in <em xmlns="" class="EmphasisTypeItalic">Andrias davidianus</em>, the evolution analysis was carried out among several species transcriptome data. The transcriptome data of <em xmlns="" class="EmphasisTypeItalic">Notophthalmus viridescens</em>, <em xmlns="" class="EmphasisTypeItalic">Xenopus tropicalis, Cynops pyrrhogaster, Hynobius chinensis</em> and <em xmlns="" class="EmphasisTypeItalic">A. davidianus</em> were obtained from the Genbank and reassembled except <em xmlns="" class="EmphasisTypeItalic">Xenopus tropicalis</em>. The BLAST search of transcriptome data obtained 1244 single-copy orthologous genes among five species<em xmlns="" class="EmphasisTypeItalic">.</em> A phylogenetic tree showed <em xmlns="" class="EmphasisTypeItalic">A. davidianus</em> to have the closest relationship to <em xmlns="" class="EmphasisTypeItalic">H. chinensis</em>. Fourteen positively selected genes were detected in <em xmlns="" class="EmphasisTypeItalic">A. davidianus</em> and <em xmlns="" class="EmphasisTypeItalic">N. vridescens</em> group and fifteen in <em xmlns="" class="EmphasisTypeItalic">A. davidianus</em> and <em xmlns="" class="EmphasisTypeItalic">H. chinensis</em> group. Five genes were shared in the both groups which involved in the immune system, suggesting that <em xmlns="" class="EmphasisTypeItalic">A. davidianus</em> adaptation to an aquatic and cave environment required rapid evolution of the immune system compared to <em xmlns="" class="EmphasisTypeItalic">N. viridescens</em> and <em xmlns="" class="EmphasisTypeItalic">H. chinensis</em>.</p></div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords"><li class="c-keywords__item" itemprop="about">
                <em xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="EmphasisTypeItalic">Andrias davidianus</em>
              </li><li class="c-keywords__item" itemprop="about">Comparative transcriptome</li><li class="c-keywords__item" itemprop="about">Evolution analysis</li><li class="c-keywords__item" itemprop="about">Adaptive genes</li></ul></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par6" class="Para">Amphibians played an important role as a transitional group linking aquatic to terrestrial in the evolution of vertebrates [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>]. To elucidate evolutionary history, the genome and mitochondrial DNA are traditionally used to estimate divergence time [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>]. Transcriptome sequencing has become a viable alternative to provide rapid developing genomic resources in non-model organisms [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>, <span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>]. Comparative transcriptome analysis is used to estimate the non-synonymous substitution (Ka) and synonymous substitution (Ks) rates to calculate the evolutionary rate [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>, <span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>] and hence, to identify genes involved in environmental adaptation. Distribution of synonymous substitutions can be used to calculate the divergent time based on the coding sequence [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>, <span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par7" class="Para">The Chinese giant salamander <em xmlns="" class="EmphasisTypeItalic">Andrias davidianus</em> is a typical urodele, and an important species both as a biological resource and with respect to its value as a living fossil [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>]. The species was historically widespread in China, but environmental degradation and human killing have led to its severe decline in the wild. From 1980s, it is classified as endangered by the International Union for Conservation of Nature and Nature Resources. Because of its irreplaceable protection status and good taste, artificial propagation technology was studied and succeeded at the end of 1990s. Success of artificial propagation technology provided a value way to protect the wild resources. In wild, it is aquatic in all life stages and typically inhabits rocky crevices in banks of streams and lakes, as well as subterranean rivers. To identify genes possibly related to <em xmlns="" class="EmphasisTypeItalic">A. davidianus</em> adaptation to its aquatic life history and to a cave habitat, transcriptome data of other amphibian species were obtained from GenBank, and comparative transcriptome analysis was carried out to detect genes positively selected for in evolution.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Methods" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Methods">Methods</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Methods" data-test="collapse-section"><section id="Sec3" class="Section2 RenderAsSection2">
                <h3 class="Heading">RNA extraction and sequencing</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par8" class="Para">Total RNA was extracted from five ovaries and testes using Trizol reagent (Invitrogen, USA) according manufacturer’s instructions and treated with RNase-free DNase I (Takara, China) to remove the genomic DNA, respectively. After RNA quality and quantity test, RNA was broken into short fragment, and first-strand cDNA was synthesized, and then the sequencing adapter was added. The cDNA libraries were constructed and sequenced on the Illumina sequencing platform (Illumina HiSeq™ 2500). All raw reads, low quality sequences, and reads containing adaptor sequences were removed, and the clean reads were obtained.</p>
              </section><section id="Sec4" class="Section2 RenderAsSection2">
                <h3 class="Heading">Identification of orthologues genes and phylogenetic analysis</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par9" class="Para">Two gonad transcriptome data (SRR3308418 and SRR3308420) of <em xmlns="" class="EmphasisTypeItalic">A. davidianus</em> were provided by my lab. To expand data of <em xmlns="" class="EmphasisTypeItalic">A. davidianus,</em> transcriptome data of skin (SRX729810) and spleen (SRX729743) were obtained from NCBI database and reassembled with the gonad transcriptome data. Transcriptome data of <em xmlns="" class="EmphasisTypeItalic">N. viridescens</em> from heart, lens, brain, eye, liver, lung, spleen, kidney, testis, and ovary (ERR108189), <em xmlns="" class="EmphasisTypeItalic">C. pyrrhogaster</em> lens and neural retina (SRR1051839), <em xmlns="" class="EmphasisTypeItalic">H. chinensis</em> whole body (SRR1042328) and <em xmlns="" class="EmphasisTypeItalic">X. tropicalis</em> from genome sequencing (GCA_000004195) were also obtained. The unigenes were reassembled from the downloaded raw reads, except for <em xmlns="" class="EmphasisTypeItalic">X. tropicalis</em>. The numbers of unigenes for each species is given in Table&nbsp;<span class="InternalRef"><a href="#Tab1">1</a></span>. BLASTN software was used to align sequences, with the cutoff E-value set at 1e-7 [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>]. OrthoMCL software was applied to classify the gene family [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>]. Orthologous genes were obtained, and Venn diagrams were used to obtain the gene number [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>]. The orthologous genes were used to construct the phylogenetic tree by the NJ method with 1000 bootstrap replications.<figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 1</span>
                        <p xmlns="" class="SimplePara">Results of the assembly for each study species</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col char="."></col><col char="."></col><col char="."></col><col char="."></col><col char="."></col><col char="."></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Species</p></th><th char="."><p xmlns="" class="SimplePara">Transcriptome</p></th><th char="."><p xmlns="" class="SimplePara">N50</p></th><th char="."><p xmlns="" class="SimplePara">GC %</p></th><th char="."><p xmlns="" class="SimplePara">Max Length bp</p></th><th char="."><p xmlns="" class="SimplePara">Min Length bp</p></th><th char="."><p xmlns="" class="SimplePara">CDS</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">Xenopus tropicalis</em> (XT)</p></td><td char="."><p xmlns="" class="SimplePara">22,855</p></td><td char="."><p xmlns="" class="SimplePara">2418</p></td><td char="."><p xmlns="" class="SimplePara">45.21</p></td><td char="."><p xmlns="" class="SimplePara">94,440</p></td><td char="."><p xmlns="" class="SimplePara">114</p></td><td char="."><p xmlns="" class="SimplePara">22,718</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">Cynops pyrrhogaste</em> (CP)</p></td><td char="."><p xmlns="" class="SimplePara">122,913</p></td><td char="."><p xmlns="" class="SimplePara">1596</p></td><td char="."><p xmlns="" class="SimplePara">44.61</p></td><td char="."><p xmlns="" class="SimplePara">18,379</p></td><td char="."><p xmlns="" class="SimplePara">201</p></td><td char="."><p xmlns="" class="SimplePara">49,986</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">Notophthalmus viridescens</em> (NV)</p></td><td char="."><p xmlns="" class="SimplePara">31,998</p></td><td char="."><p xmlns="" class="SimplePara">392</p></td><td char="."><p xmlns="" class="SimplePara">44.48</p></td><td char="."><p xmlns="" class="SimplePara">9697</p></td><td char="."><p xmlns="" class="SimplePara">201</p></td><td char="."><p xmlns="" class="SimplePara">17,943</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">Hynobius chinensis</em> (HC)</p></td><td char="."><p xmlns="" class="SimplePara">103,800</p></td><td char="."><p xmlns="" class="SimplePara">426</p></td><td char="."><p xmlns="" class="SimplePara">47.54</p></td><td char="."><p xmlns="" class="SimplePara">15,293</p></td><td char="."><p xmlns="" class="SimplePara">201</p></td><td char="."><p xmlns="" class="SimplePara">51,362</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">Andrias davidianus</em> (AD)</p></td><td char="."><p xmlns="" class="SimplePara">85,868</p></td><td char="."><p xmlns="" class="SimplePara">1492</p></td><td char="."><p xmlns="" class="SimplePara">48.61</p></td><td char="."><p xmlns="" class="SimplePara">17,741</p></td><td char="."><p xmlns="" class="SimplePara">201</p></td><td char="."><p xmlns="" class="SimplePara">43,402</p></td></tr></tbody></table></div></figure></div>
              </section><section id="Sec5" class="Section2 RenderAsSection2">
                <h3 class="Heading">Estimate of substitution rates among species</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par10" class="Para">Form the orthologous gene, only one orthologous gene in other species was classed as single-copy orthologous by PERL package [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>, <span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>]. The single-copy orthologous genes were identified to calculate the synonymous substitution rates (Ks) and non-synonymous rate (Ka). The amino acid sequences were aligned by Muscle software [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>]. The aligned sequences were converted to corresponding nucleotide sequences. Synonymous substitution rates (Ks) and non-synonymous rates (Ka) were estimated between species pairs by sit model under Codeml program in PAML package [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>]. The best threshold was set at 0.5 based on the Ka/Ks value according to previous reports [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>, <span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>]. Value of two fold log-likelihood difference was used to perform a Chi-squared test and the difference of the parameter number was set as the degree in the Chi-squared. Positively selected sites were allowed when P was &lt; 0.05 and posterior probability was &gt; 0.95 [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>]. A Ka/Ks value &gt; 1 indicated strong positive selection, from 0.5 to 1 indicated weak positive selection, and a value &lt; 0.1 indicated negative selection.</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec6" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Results" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Results">Results</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Results" data-test="collapse-section"><section id="Sec7" class="Section2 RenderAsSection2">
                <h3 class="Heading">Orthologue identification and phylogenetic analysis</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par11" class="Para">To identify the phylogenetic relationship among the species, large-scale transcriptome characterizations were carried out for <em xmlns="" class="EmphasisTypeItalic">N. viridescens, X. tropicalis, C. pyrrhogaster, H. chinensis,</em> and <em xmlns="" class="EmphasisTypeItalic">A. davidianus</em>, and transcriptome data were downloaded and reassembled (Table <span class="InternalRef"><a href="#Tab1">1</a></span>). Comparative analysis yielded 4279 gene families and 34,246 putative orthologous genes (Fig.<span class="InternalRef"><a href="#Fig1">1</a></span>). To construct the phylogenetic tree with <em xmlns="" class="EmphasisTypeItalic">X. tropicalis</em> as out-group, 1244 single-copy orthologous genes were identified. The phylogenetic tree showed <em xmlns="" class="EmphasisTypeItalic">A. davidianus</em> to have the closest relationship to <em xmlns="" class="EmphasisTypeItalic">H. chinensis</em>, with <em xmlns="" class="EmphasisTypeItalic">N. viridescens</em> and <em xmlns="" class="EmphasisTypeItalic">C. pyrrhogaster</em> clustered on one separate branch (Additional&nbsp;file&nbsp;<span class="InternalRef"><a href="#MOESM1">1</a></span>: Figure S1).<figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO1">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0056-6/MediaObjects/41065_2018_56_Fig1_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0056-6/MediaObjects/41065_2018_56_Fig1_HTML.gif" alt="Fig. 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0056-6/MediaObjects/41065_2018_56_Fig1_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0056-6/MediaObjects/41065_2018_56_Fig1_HTML.gif" alt="Fig. 1"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 1</span>
                        <p xmlns="" class="SimplePara">Venn diagrams showing the unigenes for comparative transcriptomes. The superscript indicates the protein family and the subscript indicates the unigenes</p>
                      </div></figcaption></figure></div>
              </section><section id="Sec8" class="Section2 RenderAsSection2">
                <h3 class="Heading">Evolutionary profile of <em class="EmphasisTypeItalic">Andrias davidianus</em> genes</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par12" class="Para">We analyzed the evolutionary pattern of 1244 single-copy orthologous genes in <em xmlns="" class="EmphasisTypeItalic">A. davidianus, H. chinensis, and N. viridescens.</em> Synonymous (Ks) and non-synonymous (Ka) substitutions per site were observed (Fig.&nbsp;<span class="InternalRef"><a href="#Fig2">2</a></span>). A majority of sequence pairs showed a Ka /Ks&nbsp;&lt;&nbsp;0.5, implying that these genes involved negative selection. Fifteen rapidly evolving sequences were identified with Ka/Ks &gt; 0.5 between <em xmlns="" class="EmphasisTypeItalic">A. davidianus</em> and <em xmlns="" class="EmphasisTypeItalic">H. chinensis</em>, and 14 such sites were observed between <em xmlns="" class="EmphasisTypeItalic">A. davidianus</em> and <em xmlns="" class="EmphasisTypeItalic">N. viridescens</em> (Additional&nbsp;file&nbsp;<span class="InternalRef"><a href="#MOESM2">2</a></span>: Table S1).<figure xmlns="" class="Figure" id="Fig2"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO2">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0056-6/MediaObjects/41065_2018_56_Fig2_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0056-6/MediaObjects/41065_2018_56_Fig2_HTML.gif" alt="Fig. 2" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0056-6/MediaObjects/41065_2018_56_Fig2_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0056-6/MediaObjects/41065_2018_56_Fig2_HTML.gif" alt="Fig. 2"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 2</span>
                        <p xmlns="" class="SimplePara">Ka/Ks ratio of 1244 single-copy orthologous genes. <strong class="EmphasisTypeBold">a</strong>. Ka/Ks distribution in <em class="EmphasisTypeItalic">Andrias davidianus</em> and <em class="EmphasisTypeItalic">Hynobius chinensis</em>, <strong class="EmphasisTypeBold">b</strong>. Ka/Ks distribution in <em class="EmphasisTypeItalic">Andrias davidianus</em> and <em class="EmphasisTypeItalic">Notophthalmus viridescens</em>. The solid line shows the threshold of Ka/Ks&nbsp;=&nbsp;1, the dashed line marked the weak positive selection threshold of Ka/Ks&nbsp;=&nbsp;0.5, and the short dashed line represented threshold of Ka/Ks&nbsp;=&nbsp;0.1</p>
                      </div></figcaption></figure></div>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec9" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Discussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Discussion">Discussion</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Discussion" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par13" class="Para">Next-generation sequencing technology yielded a large number of sequences at the low cost and provides more sequences compared to traditional sequencing methods [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>, <span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>]. Due to the cost and the throughput, genome-wide detection of the adaptive evolution gene was performed in many species by next-generation sequencing [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>, <span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>]. Comparative phylogenetic analysis at the genome level improved the precision of evolutionary inference compared to single gene [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>]. However, because of the large genome of the <em xmlns="" class="EmphasisTypeItalic">A. davidianus,</em> evolutionary analysis by comparative genome was hard to carry out. Transcriptome sequencing was a valuable way to obtain large-scale sequences without reference genome [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>, <span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>]. Phylogenetic analysis of transcriptome sequence data exhibited high supported tree topologies in many species [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>, <span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par14" class="Para">To elucidate the phylogenetic evolution of <em xmlns="" class="EmphasisTypeItalic">A. davidianus</em>, comparative transcriptome analysis was conducted to construct the phylogenetic tree with <em xmlns="" class="EmphasisTypeItalic">X. tropicalis</em> as out-group. To search adaptive gene for aquatic and cave life, molecular evolution was analyzed among the related species. Synonymous substitution rates (Ks) and non-synonymous substitution rates (Ka) were calculated according to the phylogenetic tree by PAML software [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>, <span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>], with the optimal threshold for selecting the positively expressed sequence tag (EST) of 0.5 based on previous study [<span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>]. Several positively selected genes were detected. Similar results were found in topmouth culter <em xmlns="" class="EmphasisTypeItalic">Erythroculter ilishaeformis</em> and zebrafish <em xmlns="" class="EmphasisTypeItalic">Danio rerio</em>, in which 38 candidate genes exhibited signs of positive selection with dN/dS ratios &gt; 0.5 [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>]. Five genes related to the immune system [<span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>–<span class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>] [cystatin-like, oncostatin-M-specific receptor subunit beta isoform X1(OSMF), exonuclease, cell death regulator Aven, and centromere protein H] were detected in the <em xmlns="" class="EmphasisTypeItalic">A. davidianus</em>/<em xmlns="" class="EmphasisTypeItalic">H. chinensis</em> and <em xmlns="" class="EmphasisTypeItalic">A. davidianus</em>/<em xmlns="" class="EmphasisTypeItalic">N. viridescens</em> groups. <em xmlns="" class="EmphasisTypeItalic">Andrias davidianus</em> is aquatic and inhabiting subterranean rivers and caves while <em xmlns="" class="EmphasisTypeItalic">N. viridescens</em> and <em xmlns="" class="EmphasisTypeItalic">H. chinensis</em> are mainly terrestrial and only special stage in water.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par15" class="Para">Aquatic and cave dwelling organisms generally encounter more bacteria than do terrestrial animals. Thus, the <em xmlns="" class="EmphasisTypeItalic">A. davidianus</em> immune system should show more rapid mutations, as was confirmed in our investigation. Due to lack of full-length according to the transcriptome sequencing, many gene relevant to positive selection was omitted and Ka/Ks ratio was decreased from normal level [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>]. Further study will be carried out to identify the genes under positive selection.</p></div></section>
          <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AbbreviationGroup Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Abbreviations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Abbreviations">Abbreviations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Abbreviations"><div class="DefinitionList">
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>EST:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par2" class="Para">Expressed sequence tag</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>Ka:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par3" class="Para">Non-synonymous substitution rates</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>Ks:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par4" class="Para">Synonymous substitution rates</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>OSMF:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par5" class="Para">Ooncostatin-M-specific receptor subunit beta isoform X1</p>
                  </dd></dl>
              </div></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments"><h3 xmlns="" class="Heading">Acknowledgments</h3><p xmlns="" class="SimplePara">The authors would like to thank the Linen lab group for comments on data analysis.</p><div id="FPar1" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Funding</h3>
                <p id="Par16" class="Para">This work was supported by National Nature Science Foundation of China (31502155). This work was supported by National Nonprofit Institute Research Grant (2017JBF0205). Key Laboratory of Freshwater Aquatic Biotechnology and Breeding, Ministry of Agriculture (KF-2017-06).</p>
              </div><div id="FPar2" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Availability of data and materials</h3>
                <p id="Par17" class="Para">The variation data reported in the paper was deposited in the GenBank.</p>
              </div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="FormalPara FormalParaRenderingStyle1"><h3 xmlns="" class="Heading">Authors’ contributions</h3><p xmlns="" class="SimplePara">QH analyze the data and write the manuscript. QW collected the data. YM and HT participated in the data analyzed and revised the manuscript. The study was conceived by HX. All the authors participated during the discussion and approved of its final manuscript.</p></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Ethics">
              <div id="FPar3" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Ethics approval and consent to participate</h3>
                <p id="Par18" class="Para">Not applicable.</p>
              </div>
              <div id="FPar4" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Consent for publication</h3>
                <p id="Par19" class="Para">Not applicable.</p>
              </div>
              <div id="FPar5" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Competing interests</h3>
                <p id="Par20" class="Para">The authors declare that they have no competing interests.</p>
              </div>
              <div id="FPar6" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Publisher’s Note</h3>
                <p id="Par21" class="Para">Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
              </div>
            </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="License">
                <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Open Access</strong> This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/4.0/"><span class="RefSource">http://creativecommons.org/licenses/by/4.0/</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/publicdomain/zero/1.0/"><span class="RefSource">http://creativecommons.org/publicdomain/zero/1.0/</span></a></span>) applies to the data made available in this article, unless otherwise stated.</p>
              </div><aside class="Appendix" id="App1">
              
                <h3 class="Heading">Additional files</h3><div id="widgetContainer" data-test="figshare-container"></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par22" class="Para">
                  <div class="MediaObject" id="MOESM1">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0056-6/MediaObjects/41065_2018_56_MOESM1_ESM.tif"><span class="caption-number">Additional file 1:</span></a>
                          <span xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Figure S1.</strong> Phylogenetic tree of selected species based on 1244 single-copy orthologous genes. (TIFF 212&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM2">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0056-6/MediaObjects/41065_2018_56_MOESM2_ESM.docx"><span class="caption-number">Additional file 2:</span></a>
                          <span xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Table S1.</strong> Orthologs gene under positive selection among species. (DOCX 18 kb)</span>
                        </div></div>
                  </div>
                </div>
              
            </aside></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authors’ Affiliations</a></h2><div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1"><span class="AffiliationNumber">(1)</span><div class="AffiliationText">Yangtze River Fisheries Research Institute, Chinese Academy of Fishery Sciences, Wuhan, Hubei, 430223, China</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff2"><span class="AffiliationNumber">(2)</span><div class="AffiliationText">College of Life Science, Yangtze University, Jingzhou, 434025, China</div></div></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper"><li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lowcock LA, Green DM, Sessions SK. Amphibian cytogenetics and evolution. Copeia. 1991;1992(2):588.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2307/1446229"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Amphibian%20cytogenetics%20and%20evolution&amp;author=LA.%20Lowcock&amp;author=DM.%20Green&amp;author=SK.%20Sessions&amp;journal=Copeia&amp;volume=1992&amp;issue=2&amp;pages=588&amp;publication_year=1991"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhang P, Chen YQ, Zhou H, Liu YF, Wang XL, Papenfuss TJ, et al. Phylogeny, evolution, and biogeography of Asiatic salamanders (Hynobiidae). Proc Natl Acad Sci U S A. 2006;103(19):7360–5.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.0602325103"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16648252"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1464346"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Phylogeny%2C%20evolution%2C%20and%20biogeography%20of%20Asiatic%20salamanders%20%28Hynobiidae%29&amp;author=P.%20Zhang&amp;author=YQ.%20Chen&amp;author=H.%20Zhou&amp;author=YF.%20Liu&amp;author=XL.%20Wang&amp;author=TJ.%20Papenfuss&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;volume=103&amp;issue=19&amp;pages=7360-7365&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hiremath PJ, Farmer A, Cannon SB, Woodward J, Kudapa H, Tuteja R, et al. Large-scale transcriptome analysis in chickpea (Cicer arietinum L.), an orphan legume crop of the semi-arid tropics of Asia and Africa. Plant Biotechnol J. 2011;9(8):922–31.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1467-7652.2011.00625.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21615673"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3437486"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Large-scale%20transcriptome%20analysis%20in%20chickpea%20%28Cicer%20arietinum%20L.%29%2C%20an%20orphan%20legume%20crop%20of%20the%20semi-arid%20tropics%20of%20Asia%20and%20Africa&amp;author=PJ.%20Hiremath&amp;author=A.%20Farmer&amp;author=SB.%20Cannon&amp;author=J.%20Woodward&amp;author=H.%20Kudapa&amp;author=R.%20Tuteja&amp;journal=Plant%20Biotechnol%20J&amp;volume=9&amp;issue=8&amp;pages=922-931&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Fraser BA, Weadick CJ, Janowitz I, Rodd FH, Hughes KA. Sequencing and characterization of the guppy (Poecilia reticulata) transcriptome. BMC Genomics. 2011;12(1):202.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2164-12-202"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21507250"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3113783"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Sequencing%20and%20characterization%20of%20the%20guppy%20%28Poecilia%20reticulata%29%20transcriptome&amp;author=BA.%20Fraser&amp;author=CJ.%20Weadick&amp;author=I.%20Janowitz&amp;author=FH.%20Rodd&amp;author=KA.%20Hughes&amp;journal=BMC%20Genomics&amp;volume=12&amp;issue=1&amp;pages=202&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhang L, Yan HF, Wu W, Yu H, Ge XJ. Comparative transcriptome analysis and marker development of two closely related primrose species (Primula poissonii and Primula wilsonii). BMC Genomics. 2013;14(1):329.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2164-14-329"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23672467"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3658987"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparative%20transcriptome%20analysis%20and%20marker%20development%20of%20two%20closely%20related%20primrose%20species%20%28Primula%20poissonii%20and%20Primula%20wilsonii%29&amp;author=L.%20Zhang&amp;author=HF.%20Yan&amp;author=W.%20Wu&amp;author=H.%20Yu&amp;author=XJ.%20Ge&amp;journal=BMC%20Genomics&amp;volume=14&amp;issue=1&amp;pages=329&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ren L, Tan XJ, Xiong YF, Xu K, Zhou Y, Zhong H, et al. Transcriptome analysis reveals positive selection on the divergent between topmouth culter and zebrafish. Gene. 2014;552(2):265–71.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.gene.2014.09.053"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25267534"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Transcriptome%20analysis%20reveals%20positive%20selection%20on%20the%20divergent%20between%20topmouth%20culter%20and%20zebrafish&amp;author=L.%20Ren&amp;author=XJ.%20Tan&amp;author=YF.%20Xiong&amp;author=K.%20Xu&amp;author=Y.%20Zhou&amp;author=H.%20Zhong&amp;journal=Gene&amp;volume=552&amp;issue=2&amp;pages=265-271&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Niu SH, Li ZX, Yuan HW, Chen XY, Li Y, Li W. Transcriptome characterisation of Pinus Tabuliformis and evolution of genes in the Pinus phylogeny. BMC Genomics. 2013;14(1):263.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2164-14-263"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23597112"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3640921"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Transcriptome%20characterisation%20of%20Pinus%20Tabuliformis%20and%20evolution%20of%20genes%20in%20the%20Pinus%20phylogeny&amp;author=SH.%20Niu&amp;author=ZX.%20Li&amp;author=HW.%20Yuan&amp;author=XY.%20Chen&amp;author=Y.%20Li&amp;author=W.%20Li&amp;journal=BMC%20Genomics&amp;volume=14&amp;issue=1&amp;pages=263&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gao KQ, Shubin NH. Earliest known crown-group salamanders. Nature. 2003;422(6930):424–8.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nature01491"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12660782"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Earliest%20known%20crown-group%20salamanders&amp;author=KQ.%20Gao&amp;author=NH.%20Shubin&amp;journal=Nature&amp;volume=422&amp;issue=6930&amp;pages=424-428&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhang Z, Schwartz S, Wagner L, Miller W. A greedy algorithm for aligning DNA sequences. J Comput Biol. 2000;7(1-2):203–14.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1089/10665270050081478"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10890397"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20greedy%20algorithm%20for%20aligning%20DNA%20sequences&amp;author=Z.%20Zhang&amp;author=S.%20Schwartz&amp;author=L.%20Wagner&amp;author=W.%20Miller&amp;journal=J%20Comput%20Biol&amp;volume=7&amp;issue=1-2&amp;pages=203-214&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li L, Jr SC, Roos DS. OrthoMCL: identification of ortholog groups for eukaryotic genomes. Genome Res. 2003;13:2178–89.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1101/gr.1224503"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12952885"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC403725"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=OrthoMCL%3A%20identification%20of%20ortholog%20groups%20for%20eukaryotic%20genomes&amp;author=L.%20Li&amp;author=SC.%20Jr&amp;author=DS.%20Roos&amp;journal=Genome%20Res&amp;volume=13&amp;pages=2178-2189&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Chen H, Boutros PC. VennDiagram: a package for the generation of highly-customizable Venn and Euler diagrams in R. BMC Bioinformatics. 2011;12(1):35.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2105-12-35"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21269502"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3041657"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=VennDiagram%3A%20a%20package%20for%20the%20generation%20of%20highly-customizable%20Venn%20and%20Euler%20diagrams%20in%20R&amp;author=H.%20Chen&amp;author=PC.%20Boutros&amp;journal=BMC%20Bioinformatics&amp;volume=12&amp;issue=1&amp;pages=35&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gabriel V, Francesc R, Gabriel C. A perl package and an alignment tool for phylogenetic networks. Bmc Bioinformatics. 2008;9(1):1–5.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2105-9-1"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20perl%20package%20and%20an%20alignment%20tool%20for%20phylogenetic%20networks&amp;author=V.%20Gabriel&amp;author=R.%20Francesc&amp;author=C.%20Gabriel&amp;journal=Bmc%20Bioinformatics&amp;volume=9&amp;issue=1&amp;pages=1-5&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wu F, Mueller LA, Crouzillat D, Pétiard V, Tanksley SD. Combining bioinformatics and phylogenetics to identify large sets of single-copy orthologous genes (COSII) for comparative, evolutionary and systematic studies: a test case in the euasterid plant clade. Genetics. 2006;174(3):1407.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1534/genetics.106.062455"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16951058"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1667096"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Combining%20bioinformatics%20and%20phylogenetics%20to%20identify%20large%20sets%20of%20single-copy%20orthologous%20genes%20%28COSII%29%20for%20comparative%2C%20evolutionary%20and%20systematic%20studies%3A%20a%20test%20case%20in%20the%20euasterid%20plant%20clade&amp;author=F.%20Wu&amp;author=LA.%20Mueller&amp;author=D.%20Crouzillat&amp;author=V.%20P%C3%A9tiard&amp;author=SD.%20Tanksley&amp;journal=Genetics&amp;volume=174&amp;issue=3&amp;pages=1407&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Edgar RC. MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nucleic Acids Res. 2004;32(5):1792–7.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gkh340"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15034147"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC390337"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=MUSCLE%3A%20multiple%20sequence%20alignment%20with%20high%20accuracy%20and%20high%20throughput&amp;author=RC.%20Edgar&amp;journal=Nucleic%20Acids%20Res&amp;volume=32&amp;issue=5&amp;pages=1792-1797&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yang Z. PAML 4: Phylogenetic analysis by maximum likelihood. Mol Biol Evol. 2007;24(8):1586–91.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/molbev/msm088"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17483113"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=PAML%204%3A%20Phylogenetic%20analysis%20by%20maximum%20likelihood&amp;author=Z.%20Yang&amp;journal=Mol%20Biol%20Evol&amp;volume=24&amp;issue=8&amp;pages=1586-1591&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yang Z, Bielawski JP. Statistical methods for detecting molecular adaptation. Trends Ecol Evol. 2000;15(12):496.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/S0169-5347(00)01994-7"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11114436"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Statistical%20methods%20for%20detecting%20molecular%20adaptation&amp;author=Z.%20Yang&amp;author=JP.%20Bielawski&amp;journal=Trends%20Ecol%20Evol&amp;volume=15&amp;issue=12&amp;pages=496&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Tine M, Kuhl H, Gagnaire PA, Louro B, Desmarais E, Martins RS, et al. European sea bass genome and its variation provide insights into adaptation to euryhalinity and speciation. Nat Commun. 2014;5:5770.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/ncomms6770"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25534655"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4284805"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=European%20sea%20bass%20genome%20and%20its%20variation%20provide%20insights%20into%20adaptation%20to%20euryhalinity%20and%20speciation&amp;author=M.%20Tine&amp;author=H.%20Kuhl&amp;author=PA.%20Gagnaire&amp;author=B.%20Louro&amp;author=E.%20Desmarais&amp;author=RS.%20Martins&amp;journal=Nat%20Commun&amp;volume=5&amp;pages=5770&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Liu H, Chen CL, Gao ZX, Min JM, Gu YM, Jian JB, et al. The draft genome of blunt snout bream (Megalobrama amblycephala) reveals the development of intermuscular bone and adaptation to herbivorous diet. Gigascience. 2017;6:1–13.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20draft%20genome%20of%20blunt%20snout%20bream%20%28Megalobrama%20amblycephala%29%20reveals%20the%20development%20of%20intermuscular%20bone%20and%20adaptation%20to%20herbivorous%20diet&amp;author=H.%20Liu&amp;author=CL.%20Chen&amp;author=ZX.%20Gao&amp;author=JM.%20Min&amp;author=YM.%20Gu&amp;author=JB.%20Jian&amp;journal=Gigascience&amp;volume=6&amp;pages=1-13&amp;publication_year=2017"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Shao CW, Bao BL, Xie ZY, Chen XY, Li B, Jia XD, et al. The genome and transcriptome of Japanese flounder provide insights into flatfish asymmetry. Nat Genet. 2017;49(1):119–24.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/ng.3732"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27918537"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20genome%20and%20transcriptome%20of%20Japanese%20flounder%20provide%20insights%20into%20flatfish%20asymmetry&amp;author=CW.%20Shao&amp;author=BL.%20Bao&amp;author=ZY.%20Xie&amp;author=XY.%20Chen&amp;author=B.%20Li&amp;author=XD.%20Jia&amp;journal=Nat%20Genet&amp;volume=49&amp;issue=1&amp;pages=119-124&amp;publication_year=2017"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Clark AG, Eisen MB, Smith DR, Bergman CM, Oliver B, Markow TA, et al. Evolution of genes and genomes on the drosophila phylogeny. Nature. 2007;450(7167):203–18.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nature06341"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17994087"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Evolution%20of%20genes%20and%20genomes%20on%20the%20drosophila%20phylogeny&amp;author=AG.%20Clark&amp;author=MB.%20Eisen&amp;author=DR.%20Smith&amp;author=CM.%20Bergman&amp;author=B.%20Oliver&amp;author=TA.%20Markow&amp;journal=Nature&amp;volume=450&amp;issue=7167&amp;pages=203-218&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Che RB, Sun YN, Wang RX, Xu TJ. Transcriptomic analysis of endangered Chinese salamander: identification of immune, sex and reproduction-related genes and genetic markers. PLoS One. 2014;9:e87940.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0087940"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24498226"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3909259"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Transcriptomic%20analysis%20of%20endangered%20Chinese%20salamander%3A%20identification%20of%20immune%2C%20sex%20and%20reproduction-related%20genes%20and%20genetic%20markers&amp;author=RB.%20Che&amp;author=YN.%20Sun&amp;author=RX.%20Wang&amp;author=TJ.%20Xu&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e87940&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Cai JF, Yang W, Chen D, Zhang YZ, He Z, Zhang WM, et al. Transcriptomic analysis of the differentiating ovary of the protogynous ricefield eel Monopterus albus. BMC Genomics. 2017;18(1):573.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/s12864-017-3953-6"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28768496"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5541746"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Transcriptomic%20analysis%20of%20the%20differentiating%20ovary%20of%20the%20protogynous%20ricefield%20eel%20Monopterus%20albus&amp;author=JF.%20Cai&amp;author=W.%20Yang&amp;author=D.%20Chen&amp;author=YZ.%20Zhang&amp;author=Z.%20He&amp;author=WM.%20Zhang&amp;journal=BMC%20Genomics&amp;volume=18&amp;issue=1&amp;pages=573&amp;publication_year=2017"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Roeding F, Borner J, Kube M, Klages S, Reinhardt R, Burmester T. A 454 sequencing approach for large scale phylogenomic analysis of the common emperor scorpion (Pandinus imperator). Mol Phylogenet Evol. 2009;53(3):826–34.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.ympev.2009.08.014"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19695333"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20454%20sequencing%20approach%20for%20large%20scale%20phylogenomic%20analysis%20of%20the%20common%20emperor%20scorpion%20%28Pandinus%20imperator%29&amp;author=F.%20Roeding&amp;author=J.%20Borner&amp;author=M.%20Kube&amp;author=S.%20Klages&amp;author=R.%20Reinhardt&amp;author=T.%20Burmester&amp;journal=Mol%20Phylogenet%20Evol&amp;volume=53&amp;issue=3&amp;pages=826-834&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR24" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Meusemann K, von Reumont BM, Simon S, Roeding F, Strauss S, Kuck P, et al. A phylogenomic approach to resolve the arthropod tree of life. Mol Biol Evol. 2010;27(11):2451–64.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/molbev/msq130"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20534705"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20phylogenomic%20approach%20to%20resolve%20the%20arthropod%20tree%20of%20life&amp;author=K.%20Meusemann&amp;author=BM.%20von%20Reumont&amp;author=S.%20Simon&amp;author=F.%20Roeding&amp;author=S.%20Strauss&amp;author=P.%20Kuck&amp;journal=Mol%20Biol%20Evol&amp;volume=27&amp;issue=11&amp;pages=2451-2464&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR25" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Swanson WJ, Wong A, Wolfner MF, Aquadro CF. Evolutionary expressed sequence tag analysis of drosophila female reproductive tracts identifies genes subjected to positive selection. Genetics. 2004;168(3):1457–65.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1534/genetics.104.030478"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15579698"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1448773"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Evolutionary%20expressed%20sequence%20tag%20analysis%20of%20drosophila%20female%20reproductive%20tracts%20identifies%20genes%20subjected%20to%20positive%20selection&amp;author=WJ.%20Swanson&amp;author=A.%20Wong&amp;author=MF.%20Wolfner&amp;author=CF.%20Aquadro&amp;journal=Genetics&amp;volume=168&amp;issue=3&amp;pages=1457-1465&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR26" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li FM, Gai XM, Wang LL, Song LS, Zhang H, Qiu LM, et al. Identification and characterization of a cystatin gene from Chinese mitten crab Eriocheir sinensis. Fish Shellfish Immunol. 2010;29(3):521–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.fsi.2010.05.015"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20570735"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Identification%20and%20characterization%20of%20a%20cystatin%20gene%20from%20Chinese%20mitten%20crab%20Eriocheir%20sinensis&amp;author=FM.%20Li&amp;author=XM.%20Gai&amp;author=LL.%20Wang&amp;author=LS.%20Song&amp;author=H.%20Zhang&amp;author=LM.%20Qiu&amp;journal=Fish%20Shellfish%20Immunol&amp;volume=29&amp;issue=3&amp;pages=521-529&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR27" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kutuk O, Temel SG, Tolunay S, Basaga H. Aven blocks DNA damage-induced apoptosis by stabilising Bcl-xL. Eur J Cancer. 2010;46(13):2494–505.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.ejca.2010.06.011"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20619636"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Aven%20blocks%20DNA%20damage-induced%20apoptosis%20by%20stabilising%20Bcl-xL&amp;author=O.%20Kutuk&amp;author=SG.%20Temel&amp;author=S.%20Tolunay&amp;author=H.%20Basaga&amp;journal=Eur%20J%20Cancer&amp;volume=46&amp;issue=13&amp;pages=2494-2505&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR28" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">He WL, Li YH, Yang DJ, Song W, Chen XL, Liu FK, et al. Combined evaluation of centromere protein H and Ki-67 as prognostic biomarker for patients with gastric carcinoma. Eur J Surg Oncol. 2013;39(2):141–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.ejso.2012.08.023"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22999412"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Combined%20evaluation%20of%20centromere%20protein%20H%20and%20Ki-67%20as%20prognostic%20biomarker%20for%20patients%20with%20gastric%20carcinoma&amp;author=WL.%20He&amp;author=YH.%20Li&amp;author=DJ.%20Yang&amp;author=W.%20Song&amp;author=XL.%20Chen&amp;author=FK.%20Liu&amp;journal=Eur%20J%20Surg%20Oncol&amp;volume=39&amp;issue=2&amp;pages=141-149&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR29" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Larrea E, Echeverria I, Riezu-Boj JI, Aldabe R, Guembe L, Sola I, et al. Characterization of the CD40L/Oncostatin M/Oncostatin M receptor axis as an antiviral and immunostimulatory system disrupted in chronic HCV infection. J Hepatol. 2014;60(3):482–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.jhep.2013.10.016"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24418171"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Characterization%20of%20the%20CD40L%2FOncostatin%20M%2FOncostatin%20M%20receptor%20axis%20as%20an%20antiviral%20and%20immunostimulatory%20system%20disrupted%20in%20chronic%20HCV%20infection&amp;author=E.%20Larrea&amp;author=I.%20Echeverria&amp;author=JI.%20Riezu-Boj&amp;author=R.%20Aldabe&amp;author=L.%20Guembe&amp;author=I.%20Sola&amp;journal=J%20Hepatol&amp;volume=60&amp;issue=3&amp;pages=482-489&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li></ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#TheAuthors" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="TheAuthors">Copyright</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="TheAuthors"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">©&nbsp;The Author(s)&nbsp;2018</div></div></section></div>
                        


                    </article>
                </main>

                <div class="c-content-layout__side u-text-sm">
                    <aside>
                        <div class="p-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="//hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-018-0056-6" data-track="click" data-track-category="PDF Download"
            data-track-action="Click download article PDF"
            data-track-label="10.1186/s41065-018-0056-6">Download PDF</a>
        </p>
    
                            
                            
    <button type="button"
       class="c-button c-button--secondary c-button--full-width c-button--expandable"
       data-toggle="collapse"
       data-target="export-citations"
       data-test="export-citations-button"
       data-sticky-update="true"
       aria-controls="export-citations"
       aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/s41065-018-0056-6"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0056-6?format=refman&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/s41065-018-0056-6"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0056-6?format=refman&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/s41065-018-0056-6"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0056-6?format=refman&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/s41065-018-0056-6"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0056-6?format=endnote&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/s41065-018-0056-6"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0056-6?format=endnote&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/s41065-018-0056-6"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0056-6?format=endnote&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/s41065-018-0056-6"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0056-6?format=bibtex&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/s41065-018-0056-6"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0056-6?format=bibtex&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/s41065-018-0056-6"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0056-6?format=bibtex&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
    </div>

                        </div>
                        
    
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
                                <div id="article-access-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/access-count/10.1186/s41065-018-0056-6 #accessCount-container"></div>
                                <div id="citation-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/citation-count/10.1186/s41065-018-0056-6 #count-container"></div>
                                <div id="altmetric-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/altmetric/10.1186/s41065-018-0056-6 #altmetric-container"></div>
                            </ul>
                        </div>
                        
    <div id="sociaMediaShare" class="p-separator u-js-shown">
        <h3 class="p-section-title">Share this article</h3>
        <ul id="shareButtons" class="c-social-links">
            <li class="c-social-links__item">
                <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-018-0056-6">
                    <span class="u-visually-hidden">Share on Twitter</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-twitter"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-018-0056-6">
                    <span class="u-visually-hidden">Share on Facebook</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-facebook"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click"
                        data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-018-0056-6">
                    <span class="u-visually-hidden">Share on LinkedIn</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-linkedin"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-018-0056-6">
                    <span class="u-visually-hidden">Share on Weibo</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-weibo"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnGooglePlus" class="js-btnShareOnGplus" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Google+" data-track-label="10.1186/s41065-018-0056-6">
                    <span class="u-visually-hidden">Share on Google Plus</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-gplus"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-018-0056-6">
                    <span class="u-visually-hidden">Share on Reddit</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-reddit"></use>
                    </svg>
                </button>
            </li>
        </ul>
    </div>
 
                        
    
        <div class="p-separator u-js-shown">
            <h3 class="p-section-title">See updates</h3>
            <button class="c-crossmark-button" data-target="crossmark" data-test="crossmark" data-track="click" data-track-category="Article Page" data-track-action="Click CrossMark" data-track-label="10.1186/s41065-018-0056-6">
                <img id="crossmark-icon" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" alt="Crossmark logo" />
            </button>
        </div>
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
                                <li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/s41065-018-0056-6" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=738&doi=10.1186%2Fs41065-018-0056-6&volume=155&issue=1&title=Comparative+transcriptome+reveal+the+potential+adaptive+evolutionary+genes+in+Andrias+davidianus&author_name=Qiaomu+Hu&start_page=1&end_page=5">Order reprint</a>

                                </li>
                            </ul>
                        </div>
                    </aside>
                    
    <div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">Hereditas</span>
                        </h4>
                        <p class="c-journal-footer__issn">ISSN: 1601-5223</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
                                
                                    <li class="c-journal-footer__contact-item">Submission enquiries: <a href="https://www.editorialmanager.com/here/" target="_blank">Access here and click Contact Us</a></li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a></li>
                                
                            </ul>
                        </div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class='tracker' style='display:none' src='/track/article/10.1186/s41065-018-0056-6' alt=""/>

         
    <footer>
        <div class="ctx-publisher-footer">
    <div class="c-navbar">
        <div class="u-container">
            <div class="c-navbar__logo u-margin-right-xl">
                <a href="https://www.biomedcentral.com">
                    <img src=/static/images/bmc/logos/logo-bmc-aj-594e03e20d.svg alt="BMC logo" data-test="navbar-logo-footer">

                </a>
            </div>
            <div class="c-navbar__content">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


            </div>
        </div>
    </div>
    <div class="c-publisher-footer" data-test="publisher-footer">
        <div class="u-container">
            
            <div class="c-flex c-flex--wrap c-flex--space-between" role="navigation" data-test="publisher-footer-menu">
                <div class="c-flex" aria-label="Publisher footer links">
                    
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/login">Receive BMC newsletters</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/account">Manage article alerts</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/policies">Policies</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/accessibility">Accessibility</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/press-centre">Press center</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/contact-us">Contact us</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/jobs">Careers</a>
                                    </li>
                                
                            </ul>
                        
                    
                </div>
                <div class="u-margin-bottom-lg">
                    <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                    <ul class="c-flex u-reset-list" data-test="footer-social-links">
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://twitter.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Twitter link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Twitter page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-twitter-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://www.facebook.com/BioMedCentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Facebook link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Facebook page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-facebook-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://plus.google.com/106226337984152901734/"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Google Plus link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Google Plus page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-gplus-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="http://www.weibo.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Weibo link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Weibo page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-weibo-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </div>
            <p class="u-reset-margin">
                By using this website, you agree to our
                <a class="u-text-inherit" href="//www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
                <a class="u-text-inherit" href="//www.biomedcentral.com/privacy-statement">Privacy
                    statement</a> and
                <a class="u-text-inherit" href="//www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
                
                    <a class="optanon-toggle-display u-text-inherit" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
                
            </p>
        </div>
    </div>
</div>


        <div class="c-sn-footer c-sn-footer--neutral">
    <div class="u-container">
        <img src=/static/images/bmc/logos/logo-springernature-8bd4e8d82f.svg class="c-sn-footer__logo" alt="Springer Nature Logo" itemprop="logo" >
        <p data-test="copyright">&#169; 2018 BioMed Central Ltd unless otherwise stated. Part of
            <a href="http://www.springernature.com"
               data-track="click"
               data-track-action="Click Springer Nature link"
               data-track-category="Corporate footer"
               itemscope itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>

    </div>

    
    <script type="text/javascript" src="//cdn.pbgrd.com/bmc.js" async="async"></script>
    <noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>
    <!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker -->

<script id="springerlink-event-tracker" src="https://event-tracker.springernature.com/dist/eventTracker.js"></script>
<script type="text/javascript">
    (function(){
        var dataLayer = window.dataLayer[0];
        var doi = dataLayer.content.article.doi;
        var imprint = dataLayer.content.contentInfo.imprint;
        var bpids = dataLayer.session.authentication.authenticationID;
        var publisher;
        var pageType = dataLayer.page.category.pageType;
        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }
        var eventObject = {'content_type' : pageType,'doi' : doi};

        if (imprint === 'BioMed Central') { publisher = 'BMC' }
        else if (imprint === 'SpringerOpen') { publisher = 'SpringerOpen' }
        else { throw new Error('Event Tracker Error: The publisher has an unexpected value') }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }
        new EventTracker({'platform':publisher}).sendEvent('display', eventObject);
    })();
</script>


    </body>
</html>

