<!DOCTYPE html>
<html lang="en" class="no-js">
    <head>
        <title>Characterization of new microsatellite markers based on the transcriptome sequencing of Clematis finetiana | Hereditas | Full Text</title>

        

    <meta name="citation_abstract" content="Clematis is the biggest genus in the family Ranunculaceae with about 300 species. Clematis is also a globally important commercial group of flowers, especially in the United States and European countries. Their petals with different colors and shapes make the genus the “Queen of the Vines”. However, the genomic information and phylogeny of Clematis based on existing molecular studies are limited. In this paper, new microsatellites (SSR) markers were identified from the transcriptome data of C. finetiana obtained using the Illumina paired-end sequencing technology. Sequences on a total of 71,900 high-quality unigenes with the mean length of 865 bp were produced in this study. There were 6192unigenes annotated and classified into 49 functional sub-groups in three main ontology categories in GO (Gen Ontology) database,14,022 unigenes mapped to COGs (Clusters of Orthologous Groups) database and classified into 25 functional categories, and 21,494 unigenes obtained and divided into 128 pathways of KEGG (Kyoto Encyclopedia of Genes) Database. A total of 7532 SSRs were discovered from 6337 unigenes. We randomly tested 210 primer pairs, of which 52 primer pairs were able to generate specific products, and 19 possessed polymorphism in the 13 wild populations of six species from Clematis, which were used as a test material. The dataset of C. finetiana transcriptome and the identified new SSR markers will promote genetic research and breeding effort in Clematis."/>

    <meta name="description" content="Clematis is the biggest genus in the family Ranunculaceae with about 300 species. Clematis is also a globally important commercial group of flowers, especially in the United States and European countries. Their petals with different colors and shapes make the genus the “Queen of the Vines”. However, the genomic information and phylogeny of Clematis based on existing molecular studies are limited. In this paper, new microsatellites (SSR) markers were identified from the transcriptome data of C. finetiana obtained using the Illumina paired-end sequencing technology. Sequences on a total of 71,900 high-quality unigenes with the mean length of 865 bp were produced in this study. There were 6192unigenes annotated and classified into 49 functional sub-groups in three main ontology categories in GO (Gen Ontology) database,14,022 unigenes mapped to COGs (Clusters of Orthologous Groups) database and classified into 25 functional categories, and 21,494 unigenes obtained and divided into 128 pathways of KEGG (Kyoto Encyclopedia of Genes) Database. A total of 7532 SSRs were discovered from 6337 unigenes. We randomly tested 210 primer pairs, of which 52 primer pairs were able to generate specific products, and 19 possessed polymorphism in the 13 wild populations of six species from Clematis, which were used as a test material. The dataset of C. finetiana transcriptome and the identified new SSR markers will promote genetic research and breeding effort in Clematis."/>

    <meta name="journal_id" content="41065"/>

    <meta name="dc.title" content="Characterization of new microsatellite markers based on the transcriptome sequencing of Clematis finetiana"/>

    <meta name="dc.source" content="Hereditas 2018 155:1"/>

    <meta name="dc.format" content="text/html"/>

    <meta name="dc.publisher" content="BioMed Central"/>

    <meta name="dc.date" content="2018-05-15"/>

    <meta name="dc.type" content="OriginalPaper"/>

    <meta name="dc.language" content="En"/>

    <meta name="dc.copyright" content="2018 The Author(s)"/>

    <meta name="dc.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="dc.description" content="Clematis is the biggest genus in the family Ranunculaceae with about 300 species. Clematis is also a globally important commercial group of flowers, especially in the United States and European countries. Their petals with different colors and shapes make the genus the &#8220;Queen of the Vines&#8221;. However, the genomic information and phylogeny of Clematis based on existing molecular studies are limited. In this paper, new microsatellites (SSR) markers were identified from the transcriptome data of C. finetiana obtained using the Illumina paired-end sequencing technology. Sequences on a total of 71,900 high-quality unigenes with the mean length of 865&#160;bp were produced in this study. There were 6192unigenes annotated and classified into 49 functional sub-groups in three main ontology categories in GO (Gen Ontology) database,14,022 unigenes mapped to COGs (Clusters of Orthologous Groups) database and classified into 25 functional categories, and 21,494 unigenes obtained and divided into 128 pathways of KEGG (Kyoto Encyclopedia of Genes) Database. A total of 7532 SSRs were discovered from 6337 unigenes. We randomly tested 210 primer pairs, of which 52 primer pairs were able to generate specific products, and 19 possessed polymorphism in the 13 wild populations of six species from Clematis, which were used as a test material. The dataset of C. finetiana transcriptome and the identified new SSR markers will promote genetic research and breeding effort in Clematis."/>

    <meta name="prism.issn" content="1601-5223"/>

    <meta name="prism.publicationName" content="Hereditas"/>

    <meta name="prism.publicationDate" content="2018-05-15"/>

    <meta name="prism.volume" content="155"/>

    <meta name="prism.number" content="1"/>

    <meta name="prism.section" content="OriginalPaper"/>

    <meta name="prism.startingPage" content="23"/>

    <meta name="prism.copyright" content="2018 The Author(s)"/>

    <meta name="prism.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="prism.url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0060-x"/>

    <meta name="prism.doi" content="doi:10.1186/s41065-018-0060-x"/>

    <meta name="citation_pdf_url" content="https://hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-018-0060-x"/>

    <meta name="citation_fulltext_html_url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0060-x"/>

    <meta name="citation_journal_title" content="Hereditas"/>

    <meta name="citation_publisher" content="BioMed Central"/>

    <meta name="citation_issn" content="1601-5223"/>

    <meta name="citation_title" content="Characterization of new microsatellite markers based on the transcriptome sequencing of Clematis finetiana"/>

    <meta name="citation_volume" content="155"/>

    <meta name="citation_issue" content="1"/>

    <meta name="citation_publication_date" content="2018/12"/>

    <meta name="citation_online_date" content="2018/05/15"/>

    <meta name="citation_firstpage" content="23"/>

    <meta name="citation_article_type" content="Research"/>

    <meta name="citation_fulltext_world_readable" content=""/>

    <meta name="dc.identifier" content="doi:10.1186/s41065-018-0060-x"/>

    <meta name="DOI" content="10.1186/s41065-018-0060-x"/>

    <meta name="citation_doi" content="10.1186/s41065-018-0060-x"/>

    <meta name="dc.creator" content="Zhigao Liu"/>

    <meta name="dc.creator" content="Weili Shao"/>

    <meta name="dc.creator" content="Yamei Shen"/>

    <meta name="dc.creator" content="Mengcheng Ji"/>

    <meta name="dc.creator" content="Wenchao Chen"/>

    <meta name="dc.creator" content="Ying Ye"/>

    <meta name="dc.creator" content="Yongbao Shen"/>

    <meta name="dc.subject" content="Animal Genetics and Genomics"/>

    <meta name="dc.subject" content="Evolutionary Biology"/>

    <meta name="citation_reference" content="citation_title=Clematis; citation_inbook_title=Flora of China; citation_publication_date=2001; citation_pages=97-165; citation_id=CR1; citation_author=WT Wang; citation_author=B Bartholomew; citation_publisher=Science Press, St. Louis"/>

    <meta name="citation_reference" content="citation_journal_title=Chin Sci Bull; citation_title=Chemical and biological research of Clematis medicinal resources; citation_author=DC Hao, XJ Gu, PG Xiao, Y Peng; citation_volume=58; citation_issue=10; citation_publication_date=2013; citation_pages=1120-1129; citation_doi=10.1007/s11434-012-5628-7; citation_id=CR2"/>

    <meta name="citation_reference" content="citation_journal_title=Acta Phytotaxon Sin; citation_title=A new system of classification of the genus Clematis (Ranunculaceae); citation_author=WT Wang, LQ Li; citation_volume=43; citation_issue=5; citation_publication_date=2005; citation_pages=431-488; citation_id=CR3"/>

    <meta name="citation_reference" content="citation_journal_title=Braz J Bot; citation_title=Karyotype analysis on 11 species of the genus Clematis; citation_author=L Sheng, K Ji, L Yu; citation_volume=37; citation_issue=4; citation_publication_date=2014; citation_pages=601-608; citation_doi=10.1007/s40415-014-0099-5; citation_id=CR4"/>

    <meta name="citation_reference" content="citation_journal_title=J Asian Nat Prod Res; citation_title=Cytotoxic triterpene saponins from Clematis mandshurica; citation_author=YX He, L Li, K Zhang, ZR Liu; citation_volume=13; citation_issue=12; citation_publication_date=2011; citation_pages=1104-1109; citation_doi=10.1080/10286020.2011.618453; citation_id=CR5"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Genomics; citation_title=Comparative transcriptome analysis of geographically distinct virulent and attenuated Babesia bovis strains reveals similar gene expression changes through attenuation; citation_author=MJ Pedroni, KS Sondgeroth, GM Gallegolopez, I Echaide, AO Lau; citation_volume=4; citation_issue=1; citation_publication_date=2013; citation_pages=763; citation_doi=10.1186/1471-2164-14-763; citation_id=CR6"/>

    <meta name="citation_reference" content="citation_journal_title=Tree Genet Genome; citation_title=Transcriptome characterization and detection of gene expression differences in aspen (Populus tremuloides ); citation_author=HS Rai, KE Mock, BA Richardson, RC Cronn, KJ Hayden, JW Wright, BJ Knaus, PG Wolf; citation_volume=9; citation_issue=4; citation_publication_date=2013; citation_pages=1031-1041; citation_doi=10.1007/s11295-013-0615-y; citation_id=CR7"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS Genet; citation_title=Comparative evolutionary and developmental dynamics of the cotton ( Gossypium hirsutum ) fiber transcriptome; citation_author=MJ Yoo, JF Wendel; citation_volume=10; citation_issue=1; citation_publication_date=2014; citation_pages=e1004073; citation_doi=10.1371/journal.pgen.1004073; citation_id=CR8"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=Identification of transcriptome-derived microsatellite markers and their association with the growth performance of the mud crab (Scylla paramamosain); citation_author=H Ma, W Jiang, P Liu, N Feng, Q Ma, C Ma, S Li; citation_volume=9; citation_issue=2; citation_publication_date=2014; citation_pages=e89134; citation_doi=10.1371/journal.pone.0089134; citation_id=CR9"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Sci; citation_title=Interspecific transferability and comparative mapping of barley EST-SSR markers in wheat, rye and rice; citation_author=RK Varshney, R Sigmund, A B&#246;rner, V Korzun, N Stein, ME Sorrells, P Langridge, A Graner; citation_volume=168; citation_issue=1; citation_publication_date=2005; citation_pages=195-202; citation_doi=10.1016/j.plantsci.2004.08.001; citation_id=CR10"/>

    <meta name="citation_reference" content="citation_journal_title=J Am Soc Hortic Sci; citation_title=Intersimple sequence repeat fingerprinting and genetic variation in a collection of Clematis cultivars and commercial germplasm; citation_author=N Gardner, SC Hokanson; citation_volume=40; citation_issue=7; citation_publication_date=2005; citation_pages=1982-1987; citation_id=CR11"/>

    <meta name="citation_reference" content="citation_journal_title=Sci Hortic-Amsterdam; citation_title=Confirmation of Clematis hybrids using molecular markers; citation_author=Y Tao, LY Wang, MS Roh; citation_volume=125; citation_issue=2; citation_publication_date=2010; citation_pages=136-145; citation_doi=10.1016/j.scienta.2010.03.005; citation_id=CR12"/>

    <meta name="citation_reference" content="citation_journal_title=Syst Bot; citation_title=Phylogenetic analyses of Clematis (Ranunculaceae) based on sequences of nuclear ribosomal ITS and three plastid regions; citation_author=L Xie, J Wen, LQ Li; citation_volume=36; citation_issue=4; citation_publication_date=2011; citation_pages=907-921; citation_doi=10.1600/036364411X604921; citation_id=CR13"/>

    <meta name="citation_reference" content="citation_journal_title=J Inner Mongolia Medical College; citation_title=rDNA-ITS sequence analysis of the traditional mongolian medicinal plants in Clematis L; citation_author=LI Xiao, P Wang, L Zhao; citation_volume=34; citation_issue=4; citation_publication_date=2012; citation_pages=325-330; citation_id=CR14"/>

    <meta name="citation_reference" content="citation_journal_title=Sida Contributions to Botany; citation_title=The actin I intron&#8212;a phylogenetically informative DNA region in Clematis (Ranunculaceae); citation_author=JM Slomba, JR Garey, FB Essig; citation_volume=21; citation_issue=2; citation_publication_date=2004; citation_pages=879-886; citation_id=CR15"/>

    <meta name="citation_reference" content="citation_journal_title=Phytotaxa; citation_title=Taxonomic status of Clematis acerifolia var. elobata, based on molecular evidence; citation_author=SX Yan, HJ Liu, LL Lin, S Liao, YL Jin, LY Pei, L Xie; citation_volume=268; citation_issue=3; citation_publication_date=2016; citation_pages=209; citation_doi=10.11646/phytotaxa.268.3.5; citation_id=CR16"/>

    <meta name="citation_reference" content="citation_journal_title=DNA Cell Biol; citation_title=Transcriptome profiling of Clematis apiifolia: insights into heat-stress responses; citation_author=L Gao, Y Ma, P Wang, S Wang, R Yang, Q Wang, L Li; citation_volume=36; citation_issue=11; citation_publication_date=2017; citation_pages=938-946; citation_doi=10.1089/dna.2017.3850; citation_id=CR17"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Biotechnol; citation_title=Full-length transcriptome assembly from rna-seq data without a reference genome; citation_author=MG Grabherr, BJ Haas, M Yassour, JZ Levin, DA Thompson, I Amit, X Adiconis; citation_volume=29; citation_issue=7; citation_publication_date=2011; citation_pages=644; citation_doi=10.1038/nbt.1883; citation_id=CR18"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Plant Biol; citation_title=Enhanced metabolic process to indole alkaloids in Clematis terniflora DC after exposure to high level of UV-B irradiation followed by the dark; citation_author=CX Gao, BX Yang, DD Zhang, M Chen, JK Tian; citation_volume=16; citation_publication_date=2016; citation_pages=231; citation_doi=10.1186/s12870-016-0920-3; citation_id=CR19"/>

    <meta name="citation_reference" content="citation_journal_title=Am Lab; citation_title=Use of a pen-shaped capillary gel electrophoresis cartridge for cost-effective DNA fragment analysis; citation_author=V Amirkhanian, E Tsai; citation_volume=44; citation_issue=7; citation_publication_date=2012; citation_pages=16-17; citation_id=CR20"/>

    <meta name="citation_reference" content="citation_journal_title=Belg J Bot; citation_title=Population genetic analysis of co-dominant and dominant markers and quantitative traits; citation_author=FC Yeh, TJB Boyle; citation_volume=129; citation_publication_date=1997; citation_pages=157; citation_id=CR21"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Power marker: an integrated analysis environment for genetic marker analysis; citation_author=K Liu, SV Muse; citation_volume=21; citation_issue=9; citation_publication_date=2005; citation_pages=2128-2129; citation_doi=10.1093/bioinformatics/bti282; citation_id=CR22"/>

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci U S A; citation_title=Mathematical model for studying genetic variation in terms of restriction endonucleases; citation_author=M Nei, WH Li; citation_volume=76; citation_issue=10; citation_publication_date=1979; citation_pages=5269-5273; citation_doi=10.1073/pnas.76.10.5269; citation_id=CR23"/>

    <meta name="citation_reference" content="citation_journal_title=Perspect Plant Ecol; citation_title=Phylogeny and classification of Ranunculales: evidence from four molecular loci and morphological data; citation_author=W Wang, AM Lu, Y Ren, ME Endress, ZD Chen; citation_volume=11; citation_issue=2; citation_publication_date=2009; citation_pages=81-110; citation_doi=10.1016/j.ppees.2009.01.001; citation_id=CR24"/>

    <meta name="citation_reference" content="citation_journal_title=Acta Phytotaxon Sin.; citation_title=A revision of Clematis sect. Tubulosae (Ranunculaceae); citation_author=W Wang, L Xie; citation_volume=45; citation_issue=4; citation_publication_date=2007; citation_pages=425-457; citation_doi=10.1360/aps06114; citation_id=CR25"/>

    <meta name="citation_reference" content="citation_journal_title=Sci Rep; citation_title=Comparison of traditional and new generation DNA markers declares high genetic diversity and differentiated population structure of wild almond species; citation_author=K Sorkheh, MK Dehkordi, S Ercisli, A Hegedus, J Hal&#225;sz; citation_volume=7; citation_issue=1; citation_publication_date=2017; citation_pages=1-17; citation_doi=10.1038/s41598-017-06084-4; citation_id=CR26"/>

    <meta name="citation_reference" content="citation_journal_title=Chin Tradit Herbal Drugs; citation_title=ITS sequence analysis of eight medicinal plants in Clematis L; citation_author=M Jiang, YQ Zhou, RR Li; citation_volume=42; citation_issue=9; citation_publication_date=2011; citation_pages=1802-1806; citation_id=CR27"/>

    <meta name="citation_reference" content="citation_journal_title=Trees; citation_title=De novo transcriptome sequencing and identification of genes related to salt stress in Eucommia ulmoides Oliver; citation_author=L Wang, HY Du, TZ Li, TN Wuyun; citation_volume=32; citation_issue=1; citation_publication_date=2017; citation_pages=151-163; citation_doi=10.1007/s00468-017-1620-9; citation_id=CR28"/>

    <meta name="citation_reference" content="citation_journal_title=Plos One; citation_title=De novo assembly, characterization of immature seed transcriptome and development of genic-SSR markers in black gram [Vigna mungo (L.) Hepper]; citation_author=J Souframanien, KS Reddy; citation_volume=10; citation_issue=6; citation_publication_date=2015; citation_pages=e0128748; citation_doi=10.1371/journal.pone.0128748; citation_id=CR29"/>

    <meta name="citation_reference" content="citation_journal_title=Gene; citation_title=De novo transcriptome analysis of the Siberian apricot (Prunus sibirica L.) and search for potential SSR markers by 454 pyrosequencing; citation_author=S Dong, Y Liu, J Niu, Y Ning, S Lin, Z Zhang; citation_volume=544; citation_issue=2; citation_publication_date=2014; citation_pages=220; citation_doi=10.1016/j.gene.2014.04.031; citation_id=CR30"/>

    <meta name="citation_reference" content="citation_journal_title=Hereditas; citation_title=Characterizing the transcriptome and microsatellite markers for almond (Amygdalus communis L.) using the illumina sequencing platform; citation_author=LS Zhang, XN Yang, XN Qi, CH Guo, ZB Jing; citation_volume=155; citation_issue=1; citation_publication_date=2018; citation_pages=14; citation_doi=10.1186/s41065-017-0049-x; citation_id=CR31"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Breed; citation_title=The first illumina-based de novo transcriptome sequencing and analysis of pumpkin (Cucurbita moschata Duch.) and SSR marker development; citation_author=T Wu, S Luo, R Wang, Y Zhong, X Xu, Y Lin, X He; citation_volume=34; citation_issue=3; citation_publication_date=2014; citation_pages=1437-1447; citation_doi=10.1007/s11032-014-0128-x; citation_id=CR32"/>

    <meta name="citation_reference" content="citation_journal_title=Genes &amp; Genom; citation_title=Development and characterization of simple sequence repeat (SSR) markers based on a full-length cDNA library of napier grass (Pennisetum purpureum Schum); citation_author=J Wang, Z Chen, S Jin, Z Hu, Y Huang, Y Diao; citation_volume=3; citation_publication_date=2017; citation_pages=1-9; citation_id=CR33"/>

    <meta name="citation_reference" content="citation_journal_title=J Zhejiang Univ Sci B; citation_title=Development of genic SSR markers from transcriptome sequencing of pear buds; citation_author=X Yue, G Liu, Y Zong, Y Teng, D Cai; citation_volume=5; citation_issue=4; citation_publication_date=2014; citation_pages=303-312; citation_doi=10.1631/jzus.B1300240; citation_id=CR34"/>

    <meta name="citation_reference" content="citation_journal_title=Genet Mol Res; citation_title=Development of simple sequence repeat markers in persimmon (Diospyros L) and their potential use in related species; citation_author=Y Yang, ZB Jing, XF Ruan, JM Cheng; citation_volume=14; citation_issue=1; citation_publication_date=2015; citation_pages=609; citation_doi=10.4238/2015.January.30.2; citation_id=CR35"/>

    <meta name="citation_reference" content="citation_journal_title=Bot J Linn Soc; citation_title=Phylogenetic relationships of Clematis (Ranunculaceae) based on chloroplast and nuclear DNA sequences; citation_author=O Miikeda, K Kita, T Handa, T Yukawa; citation_volume=152; citation_issue=2; citation_publication_date=2006; citation_pages=153-168; citation_doi=10.1111/j.1095-8339.2006.00551.x; citation_id=CR36"/>

    <meta name="citation_reference" content="citation_journal_title=Plos One; citation_title=De novo assembly and characterization of the fruit transcriptome of Chinese jujube (Ziziphus jujuba Mill.) using 454 pyrosequencing and the development of novel tri-nucleotide SSR markers; citation_author=Y Li, C Xu, X Lin, B Cui, R Wu, X Pang; citation_volume=9; citation_issue=9; citation_publication_date=2014; citation_pages=e106438; citation_doi=10.1371/journal.pone.0106438; citation_id=CR37"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=The first illumina-based De novo transcriptome sequencing and analysis of safflower flowers; citation_author=LL Huang, X Yang, P Sun, T Wen, SQ Hu; citation_volume=7; citation_issue=6; citation_publication_date=2012; citation_pages=e38653; citation_doi=10.1371/journal.pone.0038653; citation_id=CR38"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=Development and validation of EST-SSR markers from the transcriptome of adzuki bean (Vigna angularis); citation_author=H Chen, L Liu, L Wang, S Wang, P Somta, X Cheng; citation_volume=10; citation_issue=7; citation_publication_date=2015; citation_pages=e0131939; citation_doi=10.1371/journal.pone.0131939; citation_id=CR39"/>

    <meta name="citation_reference" content="citation_journal_title=Bmc Genomics; citation_title=De novo assembly and characterization of bark transcriptome using illumina sequencing and development of EST-SSR markers in rubber tree (Hevea brasiliensis Muell. Arg.); citation_author=D Li, D Zhi, Q Bi, X Liu, Z Men; citation_volume=13; citation_issue=1; citation_publication_date=2012; citation_pages=192; citation_doi=10.1186/1471-2164-13-192; citation_id=CR40"/>

    <meta name="citation_author" content="Zhigao Liu"/>

    <meta name="citation_author_institution" content="College of Landscape Architecture, Nanjing Forestry University, Nanjing, People&#8217;s Republic of China"/>

    <meta name="citation_author_institution" content="College of Landscape Architecture, Zhejiang A &amp; F University, Zhejiang, People&#8217;s Republic of China"/>

    <meta name="citation_author" content="Weili Shao"/>

    <meta name="citation_author_institution" content="College of Landscape Architecture, Zhejiang A &amp; F University, Zhejiang, People&#8217;s Republic of China"/>

    <meta name="citation_author" content="Yamei Shen"/>

    <meta name="citation_author_institution" content="College of Landscape Architecture, Zhejiang A &amp; F University, Zhejiang, People&#8217;s Republic of China"/>

    <meta name="citation_author" content="Mengcheng Ji"/>

    <meta name="citation_author_institution" content="College of Landscape Architecture, Zhejiang A &amp; F University, Zhejiang, People&#8217;s Republic of China"/>

    <meta name="citation_author" content="Wenchao Chen"/>

    <meta name="citation_author_institution" content="College of Landscape Architecture, Zhejiang A &amp; F University, Zhejiang, People&#8217;s Republic of China"/>

    <meta name="citation_author" content="Ying Ye"/>

    <meta name="citation_author_institution" content="College of Landscape Architecture, Zhejiang A &amp; F University, Zhejiang, People&#8217;s Republic of China"/>

    <meta name="citation_author" content="Yongbao Shen"/>

    <meta name="citation_author_institution" content="College of Landscape Architecture, Nanjing Forestry University, Nanjing, People&#8217;s Republic of China"/>



        <meta charset="UTF-8"/>
        <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
        <meta name="viewport" content="width=device-width, initial-scale=1"/>

        <!-- For IE 9 and below -->
<!--[if IE]><link rel="shortcut icon" href=/static/images/favicons//favicon.ico><![endif]-->
<!-- Touch Icons - iOS and Android 2.1+ -->
<link rel="apple-touch-icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>
<!-- Firefox, Chrome, Safari, IE 11+ and Opera -->
<link rel="icon" href=/static/images/favicons/bmc/favicon-3f5c56eaaa.png>


        <script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script>
        <script>(function(d){if (sessionStorage.fontsLoaded){d.style.setProperty('--font-family-sans', 'Europa');d.style.setProperty('--font-family-serif', 'Noto Serif')}})(document.documentElement);</script>

        <link rel="stylesheet" media="screen" href=/static/styles/basic/basic-a31ca85bc2.css>
<link rel="stylesheet" media="print" href=/static/styles/bmc/print-3a6240f456.css>

<link rel="stylesheet" class="js-ctm" href=/static/styles/bmc/main-84414d3d47.css
      media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">

        

        <script src="/static/js/app-bundle-8d61e6ef37.js" defer></script>
        
        <script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script>
        
    <script type="text/javascript">
        config = {
            env: 'live',
            site: 'hereditasjournal.biomedcentral.com',
            siteWithPath: 'hereditasjournal.biomedcentral.com' + window.location.pathname,
            twitterHashtag: '',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/s41065-018-0060-x',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central'
        };
    </script>

        
    <script type="text/javascript">
        dataLayer = [{"content":{"article":{"doi":"10.1186/s41065-018-0060-x","articleType":"Research","peerReviewType":"Closed","supplement":null,"keywords":"Clematis finetiana;Marker development,Transcriptome sequencing;SSRs"},"contentInfo":{"imprint":"BioMed Central","title":"Characterization of new microsatellite markers based on the transcriptome sequencing of Clematis finetiana","publishedAt":1526342400000,"publishedAtDate":"2018-05-15","author":["Zhigao Liu","Weili Shao","Yamei Shen","Mengcheng Ji","Wenchao Chen","Ying Ye","Yongbao Shen"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":"CC BY + CC0","openAccess":true},"environment":"live"},"journal":{"siteKey":"hereditasjournal.biomedcentral.com","volume":"155","issue":"1","title":"Hereditas","type":"AJs Life","journalID":41065,"gaCode":"UA-62066850-11","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Research","publishingSegment":"Genetics & Heredity (Life)"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}}}]

    </script>

        <script src="https://cdn.cookielaw.org/consent/7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script>
        
    
        
    

        
    
        
            
                <!-- Google Tag Manager -->
                <script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script>
                <!-- End Google Tag Manager -->
            
        
    


        
    
    <link rel="canonical" href="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0060-x"/>
    

        
        
        <meta property="og:url" content="https://hereditasjournal.biomedcentral.com/articles/10.1186/s41065-018-0060-x"/>
        <meta property="og:type" content="article"/>
        <meta property="og:site_name" content="Hereditas"/>
        <meta property="og:title" content="Characterization of new microsatellite markers based on the transcriptome sequencing of Clematis finetiana"/>
        <meta property="og:description" content="Clematis is the biggest genus in the family Ranunculaceae with about 300 species. Clematis is also a globally important commercial group of flowers, especially in the United States and European countries. Their petals with different colors and shapes make the genus the “Queen of the Vines”. However, the genomic information and phylogeny of Clematis based on existing molecular studies are limited. In this paper, new microsatellites (SSR) markers were identified from the transcriptome data of C. finetiana obtained using the Illumina paired-end sequencing technology. Sequences on a total of 71,900 high-quality unigenes with the mean length of 865&amp;nbsp;bp were produced in this study. There were 6192unigenes annotated and classified into 49 functional sub-groups in three main ontology categories in GO (Gen Ontology) database,14,022 unigenes mapped to COGs (Clusters of Orthologous Groups) database and classified into 25 functional categories, and 21,494 unigenes obtained and divided into 128 pathways of KEGG (Kyoto Encyclopedia of Genes) Database. A total of 7532 SSRs were discovered from 6337 unigenes. We randomly tested 210 primer pairs, of which 52 primer pairs were able to generate specific products, and 19 possessed polymorphism in the 13 wild populations of six species from Clematis, which were used as a test material. The dataset of C. finetiana transcriptome and the identified new SSR markers will promote genetic research and breeding effort in Clematis."/>
        <meta property="og:image" content="https://images.springer.com/sgw/journals/medium/41065.jpg"/>
    
    </head>

    <body class="journal journal-fulltext"
          
          >
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK"
                            height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><feColorMatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"/></filter></defs><symbol id="icon-arrow-down" viewBox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"/></symbol><symbol id="icon-arrow-left-bullet" viewBox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"/></symbol><symbol id="icon-arrow-right-bullet" viewBox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"/></symbol><symbol id="icon-arrow-right" viewBox="0 0 32 64"><path d="M0 0v64l32-32z"/></symbol><symbol id="icon-close" viewBox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"/></symbol><symbol id="icon-community" viewBox="0 0 60.643 31.01"><mask maskUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><path fill="#FFF" d="M0 31.01h60.643V0H0z" filter="url(#a)"/></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"/></symbol><symbol id="icon-download" viewBox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"/></symbol><symbol id="icon-ext-link" viewBox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"/><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"/><path fill="#676767" d="M9.7 0L16 6.2V0z"/></symbol><symbol id="icon-info-bordered" viewBox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"/><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"/></symbol><symbol id="icon-opr" viewBox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"/><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"/></symbol><symbol id="icon-quotation" viewBox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"/></symbol><symbol id="icon-remove" viewBox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"/><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"/></symbol><symbol id="icon-ribbon" viewBox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"/></symbol><symbol id="icon-rss" viewBox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"/><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"/><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"/></g></symbol><symbol id="icon-search" viewBox="0 0 16 16"><path d="M6.276 12.257C2.815 12.257 0 9.508 0 6.128 0 2.749 2.815 0 6.276 0c3.462 0 6.277 2.749 6.277 6.128.002 3.38-2.815 6.129-6.277 6.129zm0-10.772c-2.622 0-4.755 2.083-4.755 4.643 0 2.562 2.133 4.644 4.755 4.644 2.623 0 4.756-2.082 4.756-4.644 0-2.56-2.134-4.643-4.756-4.643z"/><path d="M16 13.751L13.641 16l-4.875-4.82 2.36-2.248L16 13.751z"/></symbol><symbol id="icon-speech-bubble" viewBox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"/></symbol><symbol id="icon-error" viewBox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"/></symbol><symbol id="icon-info" viewBox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"/></symbol><symbol id="icon-success" viewBox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"/></symbol><symbol id="icon-warning" viewBox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"/></symbol><symbol id="icon-facebook-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"/></symbol><symbol id="icon-facebook" viewBox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"/><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"/></symbol><symbol id="icon-gplus-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M480.5 273.117c.123.076.26.185.416.334.146.152.289.343.432.562.135.211.258.459.359.746.086.286.129.619.129 1.004-.012.7-.164 1.262-.465 1.683a5.374 5.374 0 0 1-.461.562c-.186.176-.375.35-.58.527a2.65 2.65 0 0 0-.322.41.908.908 0 0 0-.176.553c0 .205.059.375.18.508.102.129.199.238.299.336l.662.545a6.48 6.48 0 0 1 1.086 1.119c.293.415.445.955.459 1.625 0 .949-.42 1.789-1.256 2.521-.869.76-2.123 1.15-3.762 1.177-1.373-.015-2.4-.308-3.074-.879-.682-.534-1.023-1.172-1.023-1.917 0-.359.111-.766.336-1.211.215-.443.602-.837 1.172-1.174a6.126 6.126 0 0 1 1.998-.725 15.415 15.415 0 0 1 1.723-.173 3.996 3.996 0 0 1-.377-.596 1.347 1.347 0 0 1-.193-.74c0-.174.023-.317.074-.433a8.83 8.83 0 0 1 .121-.338 5.973 5.973 0 0 1-.633.035c-1.043-.012-1.838-.339-2.381-.979-.574-.603-.861-1.3-.861-2.097 0-.961.404-1.832 1.217-2.616.559-.46 1.135-.757 1.736-.898a8.494 8.494 0 0 1 1.67-.182h3.916l-1.209.704h-1.207v.005l-.005.002zm.754 11.276c0-.498-.162-.932-.484-1.302-.346-.348-.881-.778-1.609-1.29a4.045 4.045 0 0 0-.439-.02c-.102-.016-.354 0-.768.039a8.141 8.141 0 0 0-1.244.275c-.1.036-.24.096-.42.17a2.713 2.713 0 0 0-.553.355 1.985 1.985 0 0 0-.459.592c-.145.25-.215.551-.215.901 0 .687.311 1.253.932 1.698.596.444 1.404.68 2.432.688.928-.014 1.629-.217 2.117-.612.476-.385.71-.886.71-1.494zm-2.729-5.743a1.843 1.843 0 0 0 1.289-.556c.164-.249.271-.506.318-.767a7.32 7.32 0 0 0 .041-.659c0-.772-.195-1.552-.594-2.336a2.611 2.611 0 0 0-.73-.921 1.894 1.894 0 0 0-1.064-.352 1.723 1.723 0 0 0-1.324.64c-.299.437-.438.92-.428 1.457 0 .709.209 1.444.623 2.211.201.356.459.659.775.912.323.248.686.371 1.094.371zm9.698.077h-2.055v-2.056h-.996v2.056h-2.053v.998h2.053v2.052h.996v-2.052h2.055z"/></symbol><symbol id="icon-gplus" viewBox="0 0 24 24"><circle fill="#D84A36" cx="12" cy="12" r="12"/><g fill="#FFF"><path d="M12.012 12.111c-.304-.33-.07-.842.155-1.078.226-.241.639-.56.829-.74.193-.184.66-.605.674-1.868 0 0 .112-1.139-1.127-2.06h.977l1.038-.593h-3.507s-2.083-.067-3.085 1.207c-1 1.276-.966 2.871.114 3.77 1.08.898 2.371.648 2.371.648s-.197.422-.163.775c.036.353.421.849.504.965 0 0-2.386-.011-3.551 1.033-1.165 1.053-1.29 3.097 1.021 3.803 2.312.707 5.303-.055 5.906-2.277 0 0 .262-.908-.189-1.683-.459-.775-1.663-1.571-1.967-1.902zm-2.555-1.83c-.744-1-1.308-2.953 0-3.783 0 0 .955-.579 1.801.266.846.846 1.082 2.277.928 3.003-.15.727-.563.897-.739 1.008-.185.111-1.249.504-1.99-.494zm3.668 5.481c-.035.393-.083 1.254-1.491 1.664-1.409.408-3.826-.264-3.757-1.914 0 0 .006-.698.449-1.089.445-.396 1.094-.794 3.029-.819 0 0 .643.473 1.072.809.427.332.733.952.698 1.349z"/><path d="M16.323 11.055V9.803h-.752v1.252h-1.255v.748h1.255v1.256h.752v-1.256h1.248v-.748z"/></g></symbol><symbol id="icon-linkedin" viewBox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"/><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"/><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"/></g></symbol><symbol id="icon-reddit" viewBox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"/><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"/><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"/><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"/><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"/><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"/><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"/><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"/></symbol><symbol id="icon-twitter-bordered" viewBox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"/><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/></symbol><symbol id="icon-twitter" viewBox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"/><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"/></symbol><symbol id="icon-weibo-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"/><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"/><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"/></symbol><symbol id="icon-weibo" viewBox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"/><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"/><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"/><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"/><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"/><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"/></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner c-banner--info">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner" >
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1"
                 data-gpt-unitpath="/270604982/bmc/hereditasjournal/articles"
                 data-gpt-sizes="728x90,970x90"
                 data-gpt-targeting="pos=LB1;doi=10.1186/s41065-018-0060-x;kwrd=Clematis finetiana,Marker development,Transcriptome sequencing,SSRs;pmc=L32030,L21001;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-018-0060-x&amp;kwrd=Clematis finetiana,Marker development,Transcriptome sequencing,SSRs&amp;pmc=L32030,L21001&amp;">
                        <img data-test="gpt-advert-fallback-img"
                             src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/hereditasjournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s41065-018-0060-x&amp;kwrd=Clematis finetiana,Marker development,Transcriptome sequencing,SSRs&amp;pmc=L32030,L21001&amp;"
                             alt="Advertisement"
                             width="728"
                             height="90">
                    </a>
                </noscript>
            </div>
        </div>
    </div>

        
         
    <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>


<header class="c-popup-search ctx-publisher-header" id="top" data-test="publisher-header">
    <nav class="c-navbar">
        <div class="u-container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization" />
                <div class="c-logo">
    <a href="https://www.biomedcentral.com" itemprop="url">
        <img src=/static/images/bmc/logos/logo-bmc-white-aj-e3a70ae848.svg alt="BMC" itemprop="logo">
        <div class="c-logo__strapline">
            <img src=/static/images/bmc/logos/logo-bmc-white-strapline-sn-e7b1d6e67f.svg alt="Part of Springer Nature">
        </div>
    </a>
</div>
            </div>
            <button
                type="button"
                class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg"
                id="publisher-header-menu-button"
                data-toggle="collapse"
                data-test="publisher-header-menu-toggle"
                data-toggle-linked="publisher-header"
                data-target="publisher-header-search publisher-header-menu"
                aria-controls="publisher-header-menu publisher-header-search"
                aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


                <ul class="c-navbar__nav">
                    <li class="c-navbar__item">
                        <a data-replace="true"
                           data-placeholder="/placeholder/v1/username"
                           class="c-navbar__link"
                           href="https://www.biomedcentral.com/login"
                           data-test="login-link"
                           data-track="click"
                           data-track-category="Account"
                           data-track-action="Click Login Link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                    <li class="c-navbar__item">
                        <button
                            type="button"
                            class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg"
                            data-toggle="collapse"
                            data-test="header-search-button"
                            data-target="publisher-header-search publisher-header-menu publisher-header-menu-button"
                            aria-controls="publisher-header-search publisher-header-menu"
                            aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true">
                                    <use xlink:href="#icon-search"></use>
                                </svg>
                            </span>
                        </button>
                    </li>
                </ul>
            </div>
        </div>
    </nav>
    <div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field c-form-field--inline" method="GET" action="//www.biomedcentral.com/search" data-track="submit"
        data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label data-track-label="" data-test="global-search">
        <label for="publisherSearch" class="u-visually-hidden">Search all BMC articles</label>
        <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text" value="" placeholder="Search all BMC articles"/>
        <button class="c-button c-button--secondary" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true">
        <use xlink:href="#icon-search"></use>
    </svg>
</button>


        <input type="hidden" name="searchType" value="publisherSearch"/>
    </form>
</div>

            </div>
        </div>
    </div>
</header>


        
            <header class="c-journal-header c-journal-header--hereditas ctx-journal-header">
                <div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="/">
    
        
    
    <span class="c-journal-title__text ">Hereditas</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="u-container">
                        
                            <button type="button"
                                    class="c-navbar__link c-dropdown u-hidden-at-lg"
                                    data-toggle="collapse"
                                    data-test="journal-header-menu"
                                    data-toggle-linked="journal-header"
                                    data-target="journal-menu"
                                    aria-controls="journal-menu"
                                    aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav">
                                    <ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/'>Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/about'>About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href='/articles'>Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/submission-guidelines'>Submission Guidelines</a>
                                            </li>
                                        
                                    </ul>
                                </nav>
                            </div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--ocean-light-blue">
                    
                </div>
            </header>
            
        

        <div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-content-layout c-content-layout--fulltext" data-component="sticky-container">
                <div class="c-content-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3><ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc=""><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Methods">Methods</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec11" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results">Results</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec16" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Discussion">Discussion</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec17" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Conclusion">Conclusion</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li></ol></nav>
                </div>

                <main class="c-content-layout__main">
                    <article itemscope itemtype="http://schema.org/ScholarlyArticle">
                        
                        
    <div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
            <li class="c-article-identifiers__item" data-test="article-category">Research</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
    </div>

                        <div class="FulltextWrapper"><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">Characterization of new microsatellite markers based on the transcriptome sequencing of <em xmlns="" class="EmphasisTypeItalic">Clematis finetiana</em></h1></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true"><ul class="u-listReset"><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Zhigao&nbsp;Liu</span><sup><a href="#Aff1" class="AffiliationID">1</a>, <a href="#Aff2" class="AffiliationID">2</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Weili&nbsp;Shao</span><sup><a href="#Aff2" class="AffiliationID">2</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Yamei&nbsp;Shen</span><sup><a href="#Aff2" class="AffiliationID">2</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Mengcheng&nbsp;Ji</span><sup><a href="#Aff2" class="AffiliationID">2</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Wenchao&nbsp;Chen</span><sup><a href="#Aff2" class="AffiliationID">2</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Ying&nbsp;Ye</span><sup><a href="#Aff2" class="AffiliationID">2</a></sup> and </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Yongbao&nbsp;Shen</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup><a href="mailto:1692362952@qq.com" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a></li></ul></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation"><div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">Hereditas</span></span><span class="ArticleCitation_Year">2018</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">155</strong>:23</span></span></div><p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/s41065-018-0060-x">https://doi.org/10.1186/s41065-018-0060-x</a></p><p class="Copyright">© &nbsp;The Author(s)&nbsp;2018</p></div></div><ul class="ArticleHistory u-reset-list"><li class="History HistoryReceived"><strong>Received: </strong><span>18&nbsp;December&nbsp;2017</span></li><li class="History HistoryAccepted"><strong>Accepted: </strong><span>2&nbsp;May&nbsp;2018</span></li><li class="History HistoryOnlineDate"><strong>Published: </strong><span itemprop="datePublished">15&nbsp;May&nbsp;2018</span></li></ul></div>
                        
                        
                        
    

                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2><div class="c-section__content" id="Abstract"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p id="Par1" class="Para"><em xmlns="" class="EmphasisTypeItalic">Clematis</em> is the biggest genus in the family Ranunculaceae with about 300 species. <em xmlns="" class="EmphasisTypeItalic">Clematis</em> is also a globally important commercial group of flowers, especially in the United States and European countries. Their petals with different colors and shapes make the genus the “Queen of the Vines”. However, the genomic information and phylogeny of <em xmlns="" class="EmphasisTypeItalic">Clematis</em> based on existing molecular studies are limited. In this paper, new microsatellites (SSR) markers were identified from the transcriptome data of <em xmlns="" class="EmphasisTypeItalic">C. finetiana</em> obtained using the Illumina paired-end sequencing technology.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Results</h3>
                <p id="Par2" class="Para">Sequences on a total of 71,900 high-quality unigenes with the mean length of 865&nbsp;bp were produced in this study. There were 6192unigenes annotated and classified into 49 functional sub-groups in three main ontology categories in GO (Gen Ontology) database,14,022 unigenes mapped to COGs (Clusters of Orthologous Groups) database and classified into 25 functional categories, and 21,494 unigenes obtained and divided into 128 pathways of KEGG (Kyoto Encyclopedia of Genes) Database. A total of 7532 SSRs were discovered from 6337 unigenes. We randomly tested 210 primer pairs, of which 52 primer pairs were able to generate specific products, and 19 possessed polymorphism in the 13 wild populations of six species from <em xmlns="" class="EmphasisTypeItalic">Clematis</em>, which were used as a test material.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusions</h3>
                <p id="Par3" class="Para">The dataset of <em xmlns="" class="EmphasisTypeItalic">C. finetiana</em> transcriptome and the identified new SSR markers will promote genetic research and breeding effort in <em xmlns="" class="EmphasisTypeItalic">Clematis</em>.</p>
              </div></div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords"><li class="c-keywords__item" itemprop="about">
                <em xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="EmphasisTypeItalic">Clematis finetiana</em>
              </li><li class="c-keywords__item" itemprop="about">Marker development,Transcriptome sequencing</li><li class="c-keywords__item" itemprop="about">SSRs</li></ul></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par9" class="Para">There are about 300 species [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>] in the genus <em xmlns="" class="EmphasisTypeItalic">Clematis</em> L., mainly distributed in the temperate zone of the northern hemisphere [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>].China has abundant germplasm resources of <em xmlns="" class="EmphasisTypeItalic">Clematis</em>, 147 species (49%) were distributed throughout the country, especially in the southwestern area [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>]. <em xmlns="" class="EmphasisTypeItalic">Clematis</em> is the largest genus in the family Ranunculaceae, consists of typically vigorous, woody, climbing vines, and is famous for its diverse flower shapes and colors [<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>]; hundreds of cultivars make it the “Queen of the Vines”. <em xmlns="" class="EmphasisTypeItalic">C. finetiana</em>is widely distributed in south China, in Zhejiang Fujian, Guangxi, Sichuan and Yunnan provinces. It is exploited and used as a medicinal plant because of triterpenoid saponins, flavonoids and many other compounds present in roots and leaves [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>]. <em xmlns="" class="EmphasisTypeItalic">C. finetiana</em> is an evergreen species, which enhances its use in balconies and for fences. It can keep growing vigorously in hot summer conditions, which makes it different from many other cultivars and species of <em xmlns="" class="EmphasisTypeItalic">Clematis</em>. <em xmlns="" class="EmphasisTypeItalic">C. finetiana</em> is loved by home gardening enthusiasts for its lovely white flowers and excellent heat-resistance properties. It is a good resource for heat tolerance breeding of ornamental <em xmlns="" class="EmphasisTypeItalic">Clematis</em>.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par10" class="Para">Microsatellite (SSR) markers are an important tool for the evaluation of genetic diversity and differentiation between species and populations [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>, <span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>]. This marker type has a generally good transferability between closely related species and it is a useful tool in genetic mapping as well [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>–<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>]. Previously, only few molecular marker studies and DNA sequencing-based investigations have been reported on <em xmlns="" class="EmphasisTypeItalic">Clematis</em>, including the use of ISSR primers [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>], randomly amplified polymorphic DNA [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>], ITS sequencing [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>, <span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>], and single nucleotide polymorphisms (SNP) analyses of chloroplast regions accD, rps16, rpl16, trnS-trnG,atpB-rbcL, trnV-atpE and matK [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>, <span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>]. Currently, there is a lack of SSR markers capable of effectively detecting polymorphisms in <em xmlns="" class="EmphasisTypeItalic">Clematis</em>.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par11" class="Para">Transcriptome sequencing has been widely used for characterizing transcriptional events in a specific tissue or during a given period. It is a very useful tool also for research on non-model species that lack sequenced genome information. The high-throughput character and low cost makes RNA-seq a good choice for genetic investigations. The data resulting from transcriptome sequencing is valuable also for molecular marker development, such as microsatellite (SSR) and single nucleotide polymorphism (SNP) markers. To the best of our knowledge, very few research reports are available about the application of RNA-seqin studies on <em xmlns="" class="EmphasisTypeItalic">Clematis</em> [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>] and no information of SSR markers in <em xmlns="" class="EmphasisTypeItalic">Clematis</em> has been published. To improve precision in genetic analyses on <em xmlns="" class="EmphasisTypeItalic">Clematis</em>, we developed SSR markers based on the transcriptome sequencing of <em xmlns="" class="EmphasisTypeItalic">C. finetiana</em> and utilized them to investigate inter- and intraspecific diversity and differentiation and genetic relationships among <em xmlns="" class="EmphasisTypeItalic">Clematis</em> samples.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Methods" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Methods">Methods</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Methods" data-test="collapse-section"><section id="Sec3" class="Section2 RenderAsSection2">
                <h3 class="Heading">Materials and methods</h3>
                <section id="Sec4" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Plant materials</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par12" class="Para">Young leaves, stems and roots of <em xmlns="" class="EmphasisTypeItalic">C. finetiana</em> were collected from three individuals. All samples were frozen in liquid nitrogen immediately and stored at − 80&nbsp;°C until RNA extraction. Young leaves of wild <em xmlns="" class="EmphasisTypeItalic">Clematis</em> germplasm of six species including112 individuals (Additional&nbsp;file&nbsp;<span class="InternalRef"><a href="#MOESM1">1</a></span>) were also harvested and dried with silica gel prior to DNA extraction.</p>
                </section>
                <section id="Sec5" class="Section3 RenderAsSection3">
                  <h4 class="Heading">RNA extraction, cDNA library contraction and RNA-seq sequencing</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par13" class="Para">Total RNA was isolated using the TRIzol reagent (Invitrogen, Carlsbad, CA, USA) and RNeasy® mini kit (Qiagen, Valencia, CA) according to the manufacturer’s protocol. The quality and concentration of RNA were assessed by electrophoresis on a 1.2% agarose gel and using Nanophotometer Pearl/P360 (Implen, Munich, Germany). Equal amounts of purified RNA from different tissues were pooled together for cDNA library construction and transcriptome sequencing. TransCript cDNA sample prep kit (TransGen Biotech, China) was used for cDNA library construction. Agilent 2100 Bioanaylzer (Agilent Technologies, Palo Alto, CA, USA) and1.2% agarose gel electrophoresis were used in qualification of the cDNA library. Then the library was sequenced using Illumina HiSeq™2000(Illumina).</p>
                </section>
                <section id="Sec6" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Data filtering, de novo assembly and unigene function annotation</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par14" class="Para">SeqPreq (<span class="ExternalRef"><a href="https://github.com/jstjohn/SeqPrep)"><span class="RefSource">https://github.com/jstjohn/SeqPrep</span></a></span>)and sickle (<span class="ExternalRef"><a href="https://github.com/najoshi/sickle"><span class="RefSource">https://github.com/najoshi/sickle</span></a></span>) were used to remove sequencing adapters and trim low-quality sequences. After that, Trinity [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>] software was used to assemble all clean high-quality reads. The expression level of transcripts was measured by RSEM, and the result was reported by units of TPM (transcripts per million).BLASTX was employed to annotate the function of unigene sequences using no redundant (Nr) protein database (NCBI), Swiss Prot database, KEGG (Kyoto Encyclopedia of Genes and Genomes), COG (Clusters of Orthologous Groups) and InterPro database with an E-value&lt;10<sup>− 5</sup>. Blast2GO software was used for gene ontology (GO) annotation.</p>
                </section>
                <section id="Sec7" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Real-time quantitative RT-PCR for verifying gene expression profiles</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par15" class="Para">The equivalent mixed RNA samples (young leaves, stems and roots) from three individuals of <em xmlns="" class="EmphasisTypeItalic">C. finetiana</em> which used for transcriptome sequencing were used as three biological replicates for the qRT-PCR experiment. And 12 genes with different expression levels were randomly selected for validating the expression results of RNA-seq sequencing. Primers (Additional file <span class="InternalRef"><a href="#MOESM2">2</a></span>) were designed by Primer Premier 5(<span class="ExternalRef"><a href="http://downloads.fyxm.net/download-now-Primer-Premier-Others-Home-&amp;-Education-101178.html"><span class="RefSource">http://downloads.fyxm.net/download-now-Primer-Premier-Others-Home-&amp;-Education-101178.html</span></a></span>) based on the selected unigene sequences. PCR reaction mixture was consisted of 7&nbsp;μl ddH<sub>2</sub>O, 1&nbsp;μl cDNA, 10&nbsp;μl 2 × SYBR® Select Master Mix (Applied Biosystems, VIC, Australia), and 1&nbsp;μl each primer. qRT-PCR was carried out on ABI ViiA™ 7 Real-Time PCR system (Applied Biosystems, CA, USA) followed the cycling conditions: 95&nbsp;°C for 2&nbsp;min, 50&nbsp;cycles at 95&nbsp;°C for 10&nbsp;s, 60&nbsp;°C for 10&nbsp;s and 72&nbsp;°C for 40&nbsp;s. <em xmlns="" class="EmphasisTypeItalic">GAPDH</em> [<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>] was used as an internal control for normalizing the expression level of the 12 genes used for qRT-PCR testing. The specific primers of <em xmlns="" class="EmphasisTypeItalic">GAPDH</em> were listed in (Additional&nbsp;file&nbsp;<span class="InternalRef"><a href="#MOESM2">2</a></span>). The relative expression level of the selected gene was calculated via the 2<sup>–ΔCt</sup>.</p>
                </section>
                <section id="Sec8" class="Section3 RenderAsSection3">
                  <h4 class="Heading">SSR detection and primer design</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par16" class="Para">SSR discovery was performed using the MISA software (<span class="ExternalRef"><a href="http://pgrc.ipk-gatersleben.de/misa"><span class="RefSource">http://pgrc.ipk-gatersleben.de/misa</span></a></span>) with the parameters (unit size-min repeats) as follows: 1–12, 2–6, 3–5, 4–5, 5–4, 6–4. Primer pairs of each detected SSR locus were designed by Primer3 (<span class="ExternalRef"><a href="http://bioinfo.ut.ee/primer3"><span class="RefSource">http://bioinfo.ut.ee/primer3</span></a></span>) with default parameters. A total of 210 primer pairs (Additional&nbsp;file&nbsp;<span class="InternalRef"><a href="#MOESM3">3</a></span>) were randomly chosen and synthesized for the validation of SSR markers.</p>
                </section>
                <section id="Sec9" class="Section3 RenderAsSection3">
                  <h4 class="Heading">DNA extraction, validation of SSR markers and genetic analysis</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par17" class="Para">Total genomic DNA was extracted from dry leaf tissue of 112 wild individuals from 13 populations of six <em xmlns="" class="EmphasisTypeItalic">Clematis</em> species (Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>) using the E.Z.N.A Plant DNA Mini Kit Spin Protocol (Omega Bio-tek, GA, USA) according to manufacturer’s instructions. The quality and concentration of DNA were determined by Nanophotometer Pearl/P360 (Implen, Munich, Germany).</p>
                </section>
                <section id="Sec10" class="Section3 RenderAsSection3">
                  <h4 class="Heading">PCR amplification and data analysis</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par18" class="Para">PCR reactions for SSR regions were carried out in 20-μl volumes by mixing the following components: 11&nbsp;μl of ddH<sub>2</sub>O, 2&nbsp;μl of 10× buffer, 0.4&nbsp;μl of 10&nbsp;mM dNTP, 0.6&nbsp;μl of Dynazyme II DNA polymerase (Thermo Fisher Scientific, 2&nbsp;U/μl), 2&nbsp;μl of genomic DNA (about 20&nbsp;ng) and 2&nbsp;μl (5&nbsp;pmol/μl) of both primers. The PCR reactions were carried out with an initial denaturation at 94&nbsp;°C for 45&nbsp;s, followed by 35&nbsp;cycles of 30&nbsp;s at 94&nbsp;°C, 30&nbsp;s at SSR-specific annealing temperature (Table S2) and 40&nbsp;s of elongation at 72&nbsp;°C, and with a final elongation at 72&nbsp;°C for 5&nbsp;min. After amplification, the PCR products were diluted at 1:2–1:5 (depending on the concentration) with Milli-Q water. DNA fragments were analyzed using a capillary electrophoresis system Qsep 100DNA Analyzer (BiOptic, Taiwan, China) [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>]. The observed numbers of alleles (Na), effective numbers of alleles (Ne), expected heterozygosities (He) and observed heterozygosities (Ho) were determined by Popgene1.32 [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>]. Polymorphism information contents (PIC) were estimated by PowerMarker V3.25 [<span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>] (Liu and Muse, 2005). Phylogenetic trees were constructed by PowerMarker V3.25 using the UPGMA method based on Nei’s (1979) genetic distances [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>].</p>
                </section>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec11" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Results" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Results">Results</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Results" data-test="collapse-section"><section id="Sec12" class="Section2 RenderAsSection2">
                <h3 class="Heading">RNA-seq sequencing and de novo assembly</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par19" class="Para">A total of 11.12 Gb clean bases were produced from the transcriptome sequencing of <em xmlns="" class="EmphasisTypeItalic">C. finetiana</em>. 111.18&nbsp;Mb (99.91%) clean reads with 97.56% Q20 and 93.66% Q30 were collected through data filtering. 71,900 unigenes were generated when all high-quality reads were assembled. The total length, mean length, N50 and GC content of these unigenes were 62,250, 865&nbsp;bp, 1469&nbsp;bp, and 42.35% (Table&nbsp;<span class="InternalRef"><a href="#Tab1">1</a></span>). The lengths of 21,676 (30.15%) unigenes were between 300 and 400&nbsp;bp, 43,200 (60.08%) unigenes were between 400 and 2000&nbsp;bp, 4760 (6.62%) unigenes were between 2000 and 3000, and 2264 (3.15%) unigenes were longer than 3000&nbsp;bp (Fig.&nbsp;<span class="InternalRef"><a href="#Fig1">1</a></span>).<figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 1</span>
                        <p xmlns="" class="SimplePara">Summary of transcriptome data for <em class="EmphasisTypeItalic">C. finetiana</em></p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Item</p></th><th><p xmlns="" class="SimplePara">Number</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">1. Raw sequences and assembly statistics</p></td><td>&nbsp;</td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Total amount of clean reads(Mb)</p></td><td><p xmlns="" class="SimplePara">111.18</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Total amount of clean bases(Gb)</p></td><td><p xmlns="" class="SimplePara">11.12</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">GC content percentage (%)</p></td><td><p xmlns="" class="SimplePara">42.35</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Clean reads proportion (%)</p></td><td><p xmlns="" class="SimplePara">99.91</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Total number of unigenes,</p></td><td><p xmlns="" class="SimplePara">71, 900</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Mean length of unigenes(bp), N50(bp), GC content(%)</p></td><td><p xmlns="" class="SimplePara">865, 1469, 42.35</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">2.Statistics of unigene annotation</p></td><td>&nbsp;</td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Gene annotation against Nr (%)</p></td><td><p xmlns="" class="SimplePara">36, 015 (50.09%)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Gene annotation against Swiss-Prot (%)</p></td><td><p xmlns="" class="SimplePara">23, 982 (33.35%)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Gene annotation against KEGG (%)</p></td><td><p xmlns="" class="SimplePara">21, 494 (29.89%)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Gene annotation against COG (%)</p></td><td><p xmlns="" class="SimplePara">14, 022 (19.50%)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Gene annotation against GO (%)</p></td><td><p xmlns="" class="SimplePara">6192 (8.61%)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Gene annotation against Interpro (%)</p></td><td><p xmlns="" class="SimplePara">27, 004 (37.56%)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">All annotated genes (%)</p></td><td><p xmlns="" class="SimplePara">38, 814(53.98%)</p></td></tr></tbody></table></div></figure><figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO1">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0060-x/MediaObjects/41065_2018_60_Fig1_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0060-x/MediaObjects/41065_2018_60_Fig1_HTML.gif" alt="Fig. 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0060-x/MediaObjects/41065_2018_60_Fig1_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0060-x/MediaObjects/41065_2018_60_Fig1_HTML.gif" alt="Fig. 1"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 1</span>
                        <p xmlns="" class="SimplePara">Length distribution of all unigenes obtained from the <em class="EmphasisTypeItalic">C. finetiana</em> transcriptome. The x-axis indicates a different sequence size, and the y-axis indicates the unigene numbers of a specific sequence size</p>
                      </div></figcaption></figure></div>
              </section><section id="Sec13" class="Section2 RenderAsSection2">
                <h3 class="Heading">Functional annotation</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par20" class="Para">38,814 (53.98%) out of 71,900 unigenes were annotated by at least one of the following databases: Nr, Swissprot, KEGG, COG, Interpro and GO. Nr database was the largest matched database with 36,015 unigenes (50.09% of all unigenes) annotated, followed by the Swissprot (23,982, 33.35%), KEGG (21,494, 29.89%), COG (1022, 19.50%), and Interpro (27,004, 37.56%) database (Table <span class="InternalRef"><a href="#Tab1">1</a></span>). The Nr annotation indicated that 58% of unigenes showed high homology with e-values under 1e-30, 41% of unigenes showed very high homology with e-values under 1e-60 (Fig.&nbsp;<span class="InternalRef"><a href="#Fig2">2a</a></span>). The top-hit species in similarity search against the Nr database included <em xmlns="" class="EmphasisTypeItalic">Nelumbonucifera</em> (14,304, 39.72%), <em xmlns="" class="EmphasisTypeItalic">Vitisvinifera</em> (4652, 12.92%), <em xmlns="" class="EmphasisTypeItalic">Theobroma cacao</em> (1196, 3.32%), <em xmlns="" class="EmphasisTypeItalic">Citrus sinensis</em> (929, 2.58%) and others (14,923, 41.47%) (Fig. <span class="InternalRef"><a href="#Fig2">2b</a></span>).<figure xmlns="" class="Figure" id="Fig2"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO2">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0060-x/MediaObjects/41065_2018_60_Fig2_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0060-x/MediaObjects/41065_2018_60_Fig2_HTML.gif" alt="Fig. 2" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0060-x/MediaObjects/41065_2018_60_Fig2_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0060-x/MediaObjects/41065_2018_60_Fig2_HTML.gif" alt="Fig. 2"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 2</span>
                        <p xmlns="" class="SimplePara">Characteristics of homology search in <em class="EmphasisTypeItalic">C. finetiana</em> unigenes. (<strong class="EmphasisTypeBold">a</strong>) E-value distribution of the BLASTx hits against the nr database. <strong class="EmphasisTypeBold">b</strong> Top-hit species in similarity search of unigenes</p>
                      </div></figcaption></figure></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par21" class="Para">The GO database was employed to predict the possible function of <em xmlns="" class="EmphasisTypeItalic">C. finetiana</em> unigenes. 6192 unigenes were annotated and classified into 49 functional sub-groups in three ontology categories, including biological processes, molecular functions and cellular components (Fig.&nbsp;<span class="InternalRef"><a href="#Fig3">3a</a></span>). Biological processes was the largest category with 12,526 unigenes, followed by cellular components (9289 unigenes) and molecular functions (7381 unigenes). The main groups within biological processes included metabolic processes (3495, 56.44%), cellular processes (2959,47.79%) and single-organism processes (2230, 36.01%). Cell (2101, 33.93%), cell parts (2101, 33.93%) and organelles (1496,24.16%) were the most frequent groups in the cellular components category. Catalytic activity (3596, 58.08%), binding (2840, 45.87%) and transporter activity (371, 5.99%) were the most frequent groups in the molecular functions category. According to the COG database, a total of 14,022unigenes were classified into 25 functional categories, which covered most life processes (Fig. <span class="InternalRef"><a href="#Fig3">3b</a></span>). General function was the biggest category with 4045 unigenes (6.85%), followed by the replication, recombination and repair category (2144, 3.63%) and the transcription category (2024, 3.43%).<figure xmlns="" class="Figure" id="Fig3"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO3">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0060-x/MediaObjects/41065_2018_60_Fig3_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0060-x/MediaObjects/41065_2018_60_Fig3_HTML.gif" alt="Fig. 3" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0060-x/MediaObjects/41065_2018_60_Fig3_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0060-x/MediaObjects/41065_2018_60_Fig3_HTML.gif" alt="Fig. 3"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 3</span>
                        <p xmlns="" class="SimplePara">Annotation of the <em class="EmphasisTypeItalic">C. finetiana</em> transcriptome. <strong class="EmphasisTypeBold">a</strong> GO classification of unigenes. The x-axis indicates the categories, and the y-axis indicates the number of the unigenes. <strong class="EmphasisTypeBold">b</strong> COG classification of unigenes. The x-axis indicates the categories, and the y-axis indicates the number of the unigenes</p>
                      </div></figcaption></figure></div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par22" class="Para">Pathway annotations were employed to further evaluate the biological functions of the unigenes. In all, 21,494 unigenes were obtained and divided into 128 KEGG pathways (Additional&nbsp;file&nbsp;<span class="InternalRef"><a href="#MOESM4">4</a></span>) which belonged to five level-1 categories, namely, cellular processes (1013, 4.71%), environmental information processing (1430, 6.65%), genetic information processing (6492, 30.20%), metabolism (13,536, 62.98) and organismal systems (1928, 8.67%). In level-2 categories, global map (5275, 24.54%), translation (2766, 12.87%), folding, sorting and degradation (1902, 8.85%), environmental adaptation (1873, 8.71%) and carbohydrate metabolism (1611, 7.50%) were the five largest categories.</p>
              </section><section id="Sec14" class="Section2 RenderAsSection2">
                <h3 class="Heading">RT-qPCR validation of gene expression profiles</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par23" class="Para">In order to verify the reliability of the transcriptome sequencing result, 12unigenes were randomly selected and the expression levels of them were evaluated via RT-qPCR (Fig. <span class="InternalRef"><a href="#Fig4">4</a></span>). Genes with similar expression levels in RNA-Seq (such as CL6568.Contig2, CL5267.Contig1 and CL1188.Contig1) showed a little undulating expression change in the results of RT-qPCR due to the different of sensitivity and algorithms. All tested genes presented similar change trends in RNA-Seq and qPCR experiment, indicating that the results of transcriptome sequencing were reliable.<figure xmlns="" class="Figure" id="Fig4"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO4">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0060-x/MediaObjects/41065_2018_60_Fig4_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0060-x/MediaObjects/41065_2018_60_Fig4_HTML.gif" alt="Fig. 4" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0060-x/MediaObjects/41065_2018_60_Fig4_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0060-x/MediaObjects/41065_2018_60_Fig4_HTML.gif" alt="Fig. 4"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 4</span>
                        <p xmlns="" class="SimplePara">Validation of expression levels of selected genes</p>
                      </div></figcaption></figure></div>
              </section><section id="Sec15" class="Section2 RenderAsSection2">
                <h3 class="Heading">Development and validation of SSR markers</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par24" class="Para">In this study, MISA perl script (<span class="ExternalRef"><a href="http://pgrc.ipk-gatersleben.de/misa"><span class="RefSource">http://pgrc.ipk-gatersleben.de/misa</span></a></span>) was used to detect microsatellites in unigene sequences. A total of 7532 SSRs were recognized from 6337 unigenes (Table&nbsp;<span class="InternalRef"><a href="#Tab2">2</a></span>). Di-nucleotides were the most abundant type (3114, 41.34%), followed by tri-nucleotides (2594, 34.44%) hexa-nucleotides (515, 6.84%), penta-nucleotides (126, 1.67%) and tetra-nucleotides (94, 1.25%). AG/CT (2410, 37.40%) was the most abundant motif type, followed by AAG/CTT (730, 11.33%), AC/GT (453, 7.03%), ACC/GGT (387, 6.01%), AAC/GTT (348, 5.04%), ATC/ATG (320, 4.97%) and others (Fig. <span class="InternalRef"><a href="#Fig5">5</a></span>). The number of repeat motifs of SSR loci (except mono-nucleotides) ranged from 4 to 28, SSRs with six motif repeats were the most common type (1715, 26.62%), followed by five (1537, 23.68%), seven (947, 14.70%) and eight repeats (659, 10.23%) types, respectively (Additional&nbsp;file&nbsp;<span class="InternalRef"><a href="#MOESM5">5</a></span>).<figure xmlns="" id="Tab2" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 2</span>
                        <p xmlns="" class="SimplePara">Summary of microsatellite data of <em class="EmphasisTypeItalic">C. finetiana</em> transcriptome</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Item</p></th><th><p xmlns="" class="SimplePara">Number</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">Total number of sequences examined</p></td><td><p xmlns="" class="SimplePara">71, 900</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Total size of examined sequences (bp)</p></td><td><p xmlns="" class="SimplePara">62, 250,256</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Total number of identified SSRs</p></td><td><p xmlns="" class="SimplePara">7532</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Number of SSR containing sequences</p></td><td><p xmlns="" class="SimplePara">6337</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Number of sequences containing more than 1 SSR</p></td><td><p xmlns="" class="SimplePara">961</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Number of SSRs present in compound formation<sup xmlns="http://www.w3.org/1999/xhtml">a</sup></p></td><td><p xmlns="" class="SimplePara">568</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara"><sup xmlns="http://www.w3.org/1999/xhtml">a</sup>The SSR locus containing at least 2 repeat motifs</p>
                    </div></figure><figure xmlns="" class="Figure" id="Fig5"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO5">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0060-x/MediaObjects/41065_2018_60_Fig5_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0060-x/MediaObjects/41065_2018_60_Fig5_HTML.gif" alt="Fig. 5" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0060-x/MediaObjects/41065_2018_60_Fig5_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0060-x/MediaObjects/41065_2018_60_Fig5_HTML.gif" alt="Fig. 5"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 5</span>
                        <p xmlns="" class="SimplePara">Frequency distribution of SSR repeats types. Motif types of di-nucleotides and tri-nucleotides are represented. The x-axis indicates the categories, and the y-axis indicates the number of the unigenes</p>
                      </div></figcaption></figure></div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par25" class="Para">In all, 210 SSR sites (Additional file <span class="InternalRef"><a href="#MOESM3">3</a></span>) were randomly selected for primer designing. Among them, 52 (24.76%) primer pairs generated expected specific products, 38 (18.10%) primer pairs amplified PCR products much smaller or larger than the expect size, while the other 120 (57.14%) primer pairs did not amplify at all. The other five species (<em xmlns="" class="EmphasisTypeItalic">C. brevicaudata, C. apiifolia</em>, <em xmlns="" class="EmphasisTypeItalic">C. uncinata</em>, <em xmlns="" class="EmphasisTypeItalic">C. lasiandra</em> and <em xmlns="" class="EmphasisTypeItalic">C. henryi</em>) were assessed to evaluate the transferability of the 52 SSR well working markers. The results showed that 19 loci were polymorphic, and a total of 135 alleles were discovered at those 19 SSR loci among the samples including 112 accessions from 13 populations of 6 <em xmlns="" class="EmphasisTypeItalic">Clematis</em> species. The allele sizes were around the predicted allele sizes (Additional&nbsp;file&nbsp;<span class="InternalRef"><a href="#MOESM6">6</a></span>; Additional&nbsp;file&nbsp;<span class="InternalRef"><a href="#MOESM7">7</a></span>). The allele numbers per locus (Na) rangedfrom 3 to 11 (average 7.11), the effective number of alleles (Ne) ranged from 1.80 to 9.65 (average 4.44). The ranges of expected homozygosity (Ho) and expected heterozygosity (He) were from 0.10 to 0.57(average 0.25) and 0.43 to 0.86 (average 0.75), respectively. The polymorphism information contents (PIC) per locus ranged from 0.41 to 0.88 (average 0.72), for the whole dataset (Additional&nbsp;file&nbsp;<span class="InternalRef"><a href="#MOESM8">8</a></span>).</p>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par26" class="Para">The UPGMA tree (Fig.&nbsp;<span class="InternalRef"><a href="#Fig6">6</a></span>) was constructed based on Nei’s genetic distances. Two main clusters were generated. Different populations from <em xmlns="" class="EmphasisTypeItalic">C. apiifolia, C.finetiana</em> and <em xmlns="" class="EmphasisTypeItalic">C. uncinata</em> were grouped together, respectively. <em xmlns="" class="EmphasisTypeItalic">C. lasiandra</em> and <em xmlns="" class="EmphasisTypeItalic">C. henryi</em> belonged to group II and the other 4 species were distributed into another bigger group I, in which <em xmlns="" class="EmphasisTypeItalic">C. brevicaudata</em> and <em xmlns="" class="EmphasisTypeItalic">C. apiifolia</em> were closely related, and <em xmlns="" class="EmphasisTypeItalic">C. uncinata</em> and <em xmlns="" class="EmphasisTypeItalic">C.finetiana</em> were grouped together.<figure xmlns="" class="Figure" id="Fig6"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO6">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs41065-018-0060-x/MediaObjects/41065_2018_60_Fig6_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs41065-018-0060-x/MediaObjects/41065_2018_60_Fig6_HTML.gif" alt="Fig. 6" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0060-x/MediaObjects/41065_2018_60_Fig6_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs41065-018-0060-x/MediaObjects/41065_2018_60_Fig6_HTML.gif" alt="Fig. 6"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 6</span>
                        <p xmlns="" class="SimplePara">Phylogenetic tree of 13 populations from 6 <em class="EmphasisTypeItalic">Clematis</em> species based on 19 SSR loci and utilizing Nei’s genetic distances. The population abbreviations are the same as those in Additional file <span xmlns="http://www.w3.org/1999/xhtml" class="InternalRef"><a href="#MOESM1">1</a></span></p>
                      </div></figcaption></figure></div>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec16" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Discussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Discussion">Discussion</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Discussion" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par27" class="Para"><em xmlns="" class="EmphasisTypeItalic">Clematis</em> cultivars are well known for their diverse flower types and colors; especially the large flowered hybrids make <em xmlns="" class="EmphasisTypeItalic">Clematis</em> one of the most popular flowers. However, only few molecular marker-based approaches have been used to assist breeding of ornamental <em xmlns="" class="EmphasisTypeItalic">Clematis</em>. Inter-simple sequence repeat markers (ISSR) have been used to fingerprint 32 vining cultivars and five no-vining species for assessing genetic relationships and cultivar identification [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>]. Random amplified polymorphic DNA (RAPD) has been used to confirm the identity of <em xmlns="" class="EmphasisTypeItalic">Clematis</em> hybrids [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>]. There are several earlier reports about the systematic classification and phylogeny research on <em xmlns="" class="EmphasisTypeItalic">Clematis</em>, which are mainly based on phenotypic characteristics, such as the shape and length of sepals, the indumentum of filaments, the structure of pollen and plant morphology [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>, <span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>, <span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>, <span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>]. A few reports have been published about the interspecific relationships among <em xmlns="" class="EmphasisTypeItalic">Clematis</em> based on molecular markers [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>, <span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>, <span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>, <span class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>], but such research is still limited due to the lack of genomic information of <em xmlns="" class="EmphasisTypeItalic">Clematis</em>. In this study, the transcriptome of <em xmlns="" class="EmphasisTypeItalic">C. finetiana</em> was sequenced in order to obtain genomic data and to develop SSR markers.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par28" class="Para">In the present study, a total of 71,900 unigenes was obtained via transcriptome sequencing with an average length of 865&nbsp;bp. The average length of sequenced <em xmlns="" class="EmphasisTypeItalic">C. finetiana</em> unigenes was longer compared to recently released transcriptome sequencing results on <em xmlns="" class="EmphasisTypeItalic">Eucommia ulmoides</em> (645&nbsp;bp) [<span class="CitationRef"><a href="#CR28" data-toggle="citation">28</a></span>], <em xmlns="" class="EmphasisTypeItalic">Vigna mungo</em> (443&nbsp;bp) [<span class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>] and <em xmlns="" class="EmphasisTypeItalic">Prunus sibirica</em> (652&nbsp;bp) [<span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>]. As a result of BLAST searches, a total of 38,814 unigenes (53.98%) were annotated by Nr, Swiss-Prot, COG, GO, Interpro and KEGG databases, while about 33,086 unigenes (46.02%) were not annotated by any of these six databases, thus indicating that some unigenes may be particular to <em xmlns="" class="EmphasisTypeItalic">Clematis.</em> Transcriptome sequencing is also useful tool for the development of SSR markers, and there are previous reports about the development of SSRs in plants through transcriptome sequencing [<span class="CitationRef"><a href="#CR31" data-toggle="citation">31</a></span>–<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par29" class="Para">Microsatelite markers are co-dominant, highly polymorphic and easily reproducible [<span class="CitationRef"><a href="#CR35" data-toggle="citation">35</a></span>]. They are important tools to examine genetic diversity, the assessment of genetic relationships and population genetic structure in plants [<span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>]. The application of SSR markers in the study of <em xmlns="" class="EmphasisTypeItalic">Clematis</em> has been limited due to the expenses and time-consuming work when developing markers by traditional methods. Some other molecular marker systems have been applied to studies on <em xmlns="" class="EmphasisTypeItalic">Clematis</em>. ITS sequences have been analyzed to provide molecular evidence for the identification of 14 medicinal <em xmlns="" class="EmphasisTypeItalic">Clematis</em> species [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>, <span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>]. The sequences of chloroplast DNA (atpB-rbcL spacer region, matK, trnK, trnL intron, andtrnL-trnF spacer region) and the nuclear actin I intron have been used for the analysis of phylogenetic relationships within the genus <em xmlns="" class="EmphasisTypeItalic">Clematis</em>. The result showed that the taxonomic status of several species was not consistent with previous interspecific classifications based on morphology [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>, <span class="CitationRef"><a href="#CR36" data-toggle="citation">36</a></span>]. However, none of them have utilized SSR markers.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par30" class="Para">In this study, a high quantity of high-quality transcriptome sequences was obtained, which could be used to develop SSR markers for <em xmlns="" class="EmphasisTypeItalic">Clematis</em>. A total of 7532 SSRs were recognized from 6337 unigenes. These SSRs could be divided into five categories based on the number of bases from two to six, di-nucleotides and tri-nucleotides being the most abundant types, which is in concord with previous studies [<span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>, <span class="CitationRef"><a href="#CR37" data-toggle="citation">37</a></span>]. Furthermore, AG/CT and AAG/CTT were the most abundant motif typesamong-nucleotides and tri-nucleotides, similarly as previously reported [<span class="CitationRef"><a href="#CR38" data-toggle="citation">38</a></span>]. Tetr-nucleotide and penta-nucleotide motifs of SSRs are usually less polymorphism in coding sequences region. Nevertheless, the two types were generally included in the list of screening primers [<span class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>-<span class="CitationRef"><a href="#CR31" data-toggle="citation">31</a></span>]. In the present study, SSR primers of <em xmlns="" class="EmphasisTypeItalic">Clematis</em> were screened for the first time. In consideration of the comprehensiveness of the experiment, a total of 210 primer pairs from all five SSR types (including tetr-nucleotide and penta- nucleotide motifs) were randomly chosen. We successfully developed 52 SSR markers, which gave clear amplification products and showed good transferability among different species of <em xmlns="" class="EmphasisTypeItalic">Clematis</em>. Within the genus, 19 markers were polymorphic, and all except one of these polymorphic loci showed the presence of a high level of polymorphism with PIC values above 0.60. Several studies have indicated that tetra-nucleotide SSRs have high polymorphism rates [<span class="CitationRef"><a href="#CR39" data-toggle="citation">39</a></span>]. In the present study, both two tetra-nucleotide SSRs showed high polymorphism, of then‘4–11’was the most informative tetra-nucleotide SSR with the PIC value of 0.88. Hexa-nucleotides (9, 56.25%) were the most abundant type among the 19 polymorphic SSRs, followed by tri-nucleotides (4, 21.05%) and di-nucleotides (3, 15.79%). This observation was similar to the result of SSR development in the rubber tree [<span class="CitationRef"><a href="#CR40" data-toggle="citation">40</a></span>]. All amplified fragments resulting from the 19 primer pairs were around the expected size and showing that the detected polymorphism resulted from the variation in the number of SSR repeats.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par31" class="Para">Microsatellite markers have been proved to be effective tool in the evaluation of interspecific genetic diversity and phylogenetic relationship. The phylogenetic tree constructed based on SSR data showed that <em xmlns="" class="EmphasisTypeItalic">C. brevicaudata</em> and <em xmlns="" class="EmphasisTypeItalic">C. apiifolia</em> belong to the same group; <em xmlns="" class="EmphasisTypeItalic">C. uncinata</em> and <em xmlns="" class="EmphasisTypeItalic">C.finetiana</em> are closely related. These results are similar to those of Xie et al., who used nuclear ITS and three plastid regions in the phylogentic analyses of <em xmlns="" class="EmphasisTypeItalic">Clematis</em> [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>]. This result demonstrated the effectiveness in analyzing <em xmlns="" class="EmphasisTypeItalic">Clematis</em> genetic relationships and confirming the potential value of the transcriptome database for the development of new SSR markers. The newly developed polymorphic SSR markers can be applied into population genetic and phylogenetic studies, species and hybrid identification, and possibly also to marker-assisted breeding in <em xmlns="" class="EmphasisTypeItalic">Clematis</em>.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec17" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Conclusion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Conclusion">Conclusion</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Conclusion" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par32" class="Para">The present study reported the functional characterization of transcriptome sequences of <em xmlns="" class="EmphasisTypeItalic">Clematis</em> and development of new SSR markers, which are applicable to many kinds of studies in <em xmlns="" class="EmphasisTypeItalic">Clematis</em>. To the best of our knowledge, this is the first report on the development and use of any kind microsatellite markers in the genus <em xmlns="" class="EmphasisTypeItalic">Clematis</em>. They can be used in further studies on the genetic diversity, population genetics and phylogeography of <em xmlns="" class="EmphasisTypeItalic">Clematis</em>, and they can assist the breeding of new ornamental cultivars.</p></div></section>
          <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AbbreviationGroup Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Abbreviations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Abbreviations">Abbreviations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Abbreviations"><div class="DefinitionList">
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>COG:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par4" class="Para">Clusters of Orthologous Groups</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>GO:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par5" class="Para">Gene Ontology</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>KEGG:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par6" class="Para">Kyoto Encyclopedia of Genes and Genomes</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>Nr:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par7" class="Para">Non-redundant</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>SSR:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par8" class="Para">Simple sequence repeat marker</p>
                  </dd></dl>
              </div></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments"><h3 xmlns="" class="Heading">Acknowledgements</h3><p xmlns="" class="SimplePara">We would like to thank Fuqiang Cui, Helena Korpelainen, Chao Zhang, Bin Dong and Zaikang Tong for their suggestions and commenting on the paper.</p><div id="FPar1" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Funding</h3>
                <p id="Par33" class="Para">This work was supported by Projects for floriculture special breeding of the Zhejiang province (2012C12909–4) and Major science projects for agriculture (floriculture) new variety breeding of the Zhejiang province (2016C02056–13-4).</p>
              </div><div id="FPar2" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Availability of data and materials</h3>
                <p id="Par34" class="Para">The datasets used and analyzed during the current study are available from the corresponding author on reasonable request.</p>
              </div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="FormalPara FormalParaRenderingStyle1"><h3 xmlns="" class="Heading">Authors’ contributions</h3><p xmlns="" class="SimplePara">L-ZG, S-YB and J-MC developed the project and designed the experiments. Y-Y and C-WC performed the experiments. L-ZG, S-WL and S-YM wrote and revised the manuscript. All authors discussed the results and commented on the manuscript. All authors read and approved the final manuscript.</p></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Ethics">
              <div id="FPar3" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Ethics approval and consent to participate</h3>
                <p id="Par35" class="Para">Not applicable.</p>
              </div>
              <div id="FPar4" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Competing interests</h3>
                <p id="Par36" class="Para">The authors declare that they have no competing interests.</p>
              </div>
              <div id="FPar5" class="FormalPara FormalParaRenderingStyle1">
                <h3 xmlns="" class="Heading">Publisher’s Note</h3>
                <p id="Par37" class="Para">Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
              </div>
            </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="License">
                <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Open Access</strong>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/4.0/"><span class="RefSource">http://creativecommons.org/licenses/by/4.0/</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/publicdomain/zero/1.0/"><span class="RefSource">http://creativecommons.org/publicdomain/zero/1.0/</span></a></span>) applies to the data made available in this article, unless otherwise stated.</p>
              </div><aside class="Appendix" id="App1">
              
                <h3 class="Heading">Additional files</h3><div id="widgetContainer" data-test="figshare-container"></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par38" class="Para">
                  <div class="MediaObject" id="MOESM1">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0060-x/MediaObjects/41065_2018_60_MOESM1_ESM.xlsx"><span class="caption-number">Additional file 1:</span></a>
                          <span xmlns="" class="SimplePara">Sampling information of the six Clematis species. (XLSX 10 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM2">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0060-x/MediaObjects/41065_2018_60_MOESM2_ESM.xlsx"><span class="caption-number">Additional file 2:</span></a>
                          <span xmlns="" class="SimplePara">Used primers for qRT-PCR experiments. (XLSX 12 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM3">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0060-x/MediaObjects/41065_2018_60_MOESM3_ESM.xlsx"><span class="caption-number">Additional file 3:</span></a>
                          <span xmlns="" class="SimplePara">Details of 210 selected SSR markers used for polymorphism validation. (XLSX 34 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM4">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0060-x/MediaObjects/41065_2018_60_MOESM4_ESM.xlsx"><span class="caption-number">Additional file 4:</span></a>
                          <span xmlns="" class="SimplePara">The KEGG pathway annotations. (XLSX 10 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM5">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0060-x/MediaObjects/41065_2018_60_MOESM5_ESM.xlsx"><span class="caption-number">Additional file 5:</span></a>
                          <span xmlns="" class="SimplePara">The frequency of classified SSR repeat types. (XLSX 24 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM6">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0060-x/MediaObjects/41065_2018_60_MOESM6_ESM.xlsx"><span class="caption-number">Additional file 6:</span></a>
                          <span xmlns="" class="SimplePara">Detected allele sizes of 19 SSR loci in different Clematis species. (XLSX 12 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM7">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0060-x/MediaObjects/41065_2018_60_MOESM7_ESM.png"><span class="caption-number">Additional file 7:</span></a>
                          <span xmlns="" class="SimplePara">The product sizes of the amplified locus ‘6-73’ in 4 Clematis samples determined by Qsep 100 DNA Analyzer. (PNG 136 kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM8">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs41065-018-0060-x/MediaObjects/41065_2018_60_MOESM8_ESM.xlsx"><span class="caption-number">Additional file 8:</span></a>
                          <span xmlns="" class="SimplePara">Characteristics of 19 polymorphic SSR loci. (XLSX 11 kb)</span>
                        </div></div>
                  </div>
                </div>
              
            </aside></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authors’ Affiliations</a></h2><div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1"><span class="AffiliationNumber">(1)</span><div class="AffiliationText">College of Landscape Architecture, Nanjing Forestry University, Nanjing, 210037, Jiangsu, People’s Republic of China</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff2"><span class="AffiliationNumber">(2)</span><div class="AffiliationText">College of Landscape Architecture, Zhejiang A &amp; F University, Hangzhou, 311300&nbsp;Zhejiang, People’s Republic of China</div></div></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper"><li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wang WT, Bartholomew B. Clematis. In: Wu Z-Y, Raven P, editors. Flora of China, vol. 6. Beijing: Science Press, St. Louis; 2001. p. 97–165.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Clematis&amp;author=WT.%20Wang&amp;author=B.%20Bartholomew&amp;pages=97-165&amp;publication_year=2001"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hao DC, Gu XJ, Xiao PG, Peng Y. Chemical and biological research of Clematis medicinal resources. Chin Sci Bull. 2013;58(10):1120–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11434-012-5628-7"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Chemical%20and%20biological%20research%20of%20Clematis%20medicinal%20resources&amp;author=DC.%20Hao&amp;author=XJ.%20Gu&amp;author=PG.%20Xiao&amp;author=Y.%20Peng&amp;journal=Chin%20Sci%20Bull&amp;volume=58&amp;issue=10&amp;pages=1120-1129&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wang WT, Li LQ. A new system of classification of the genus Clematis (Ranunculaceae). Acta Phytotaxon Sin. 2005;43(5):431–88.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20new%20system%20of%20classification%20of%20the%20genus%20Clematis%20%28Ranunculaceae%29&amp;author=WT.%20Wang&amp;author=LQ.%20Li&amp;journal=Acta%20Phytotaxon%20Sin&amp;volume=43&amp;issue=5&amp;pages=431-488&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sheng L, Ji K, Yu L. Karyotype analysis on 11 species of the genus Clematis. Braz J Bot. 2014;37(4):601–8.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s40415-014-0099-5"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Karyotype%20analysis%20on%2011%20species%20of%20the%20genus%20Clematis&amp;author=L.%20Sheng&amp;author=K.%20Ji&amp;author=L.%20Yu&amp;journal=Braz%20J%20Bot&amp;volume=37&amp;issue=4&amp;pages=601-608&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">He YX, Li L, Zhang K, Liu ZR. Cytotoxic triterpene saponins from Clematis mandshurica. J Asian Nat Prod Res. 2011;13(12):1104–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1080/10286020.2011.618453"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22115034"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Cytotoxic%20triterpene%20saponins%20from%20Clematis%20mandshurica&amp;author=YX.%20He&amp;author=L.%20Li&amp;author=K.%20Zhang&amp;author=ZR.%20Liu&amp;journal=J%20Asian%20Nat%20Prod%20Res&amp;volume=13&amp;issue=12&amp;pages=1104-1109&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Pedroni MJ, Sondgeroth KS, Gallegolopez GM, Echaide I, Lau AO. Comparative transcriptome analysis of geographically distinct virulent and attenuated Babesia bovis strains reveals similar gene expression changes through attenuation. BMC Genomics. 2013;4(1):763.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2164-14-763"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparative%20transcriptome%20analysis%20of%20geographically%20distinct%20virulent%20and%20attenuated%20Babesia%20bovis%20strains%20reveals%20similar%20gene%20expression%20changes%20through%20attenuation&amp;author=MJ.%20Pedroni&amp;author=KS.%20Sondgeroth&amp;author=GM.%20Gallegolopez&amp;author=I.%20Echaide&amp;author=AO.%20Lau&amp;journal=BMC%20Genomics&amp;volume=4&amp;issue=1&amp;pages=763&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Rai HS, Mock KE, Richardson BA, Cronn RC, Hayden KJ, Wright JW, Knaus BJ, Wolf PG. Transcriptome characterization and detection of gene expression differences in aspen (Populus tremuloides ). Tree Genet Genome. 2013;9(4):1031–41.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11295-013-0615-y"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Transcriptome%20characterization%20and%20detection%20of%20gene%20expression%20differences%20in%20aspen%20%28Populus%20tremuloides%20%29&amp;author=HS.%20Rai&amp;author=KE.%20Mock&amp;author=BA.%20Richardson&amp;author=RC.%20Cronn&amp;author=KJ.%20Hayden&amp;author=JW.%20Wright&amp;author=BJ.%20Knaus&amp;author=PG.%20Wolf&amp;journal=Tree%20Genet%20Genome&amp;volume=9&amp;issue=4&amp;pages=1031-1041&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yoo MJ, Wendel JF. Comparative evolutionary and developmental dynamics of the cotton ( Gossypium hirsutum ) fiber transcriptome. PLoS Genet. 2014;10(1):e1004073.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pgen.1004073"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24391525"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3879233"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparative%20evolutionary%20and%20developmental%20dynamics%20of%20the%20cotton%20%28%20Gossypium%20hirsutum%20%29%20fiber%20transcriptome&amp;author=MJ.%20Yoo&amp;author=JF.%20Wendel&amp;journal=PLoS%20Genet&amp;volume=10&amp;issue=1&amp;pages=e1004073&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ma H, Jiang W, Liu P, Feng N, Ma Q, Ma C, Li S, et al. Identification of transcriptome-derived microsatellite markers and their association with the growth performance of the mud crab (Scylla paramamosain). PLoS One. 2014;9(2):e89134.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0089134"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24551232"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3923883"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Identification%20of%20transcriptome-derived%20microsatellite%20markers%20and%20their%20association%20with%20the%20growth%20performance%20of%20the%20mud%20crab%20%28Scylla%20paramamosain%29&amp;author=H.%20Ma&amp;author=W.%20Jiang&amp;author=P.%20Liu&amp;author=N.%20Feng&amp;author=Q.%20Ma&amp;author=C.%20Ma&amp;author=S.%20Li&amp;journal=PLoS%20One&amp;volume=9&amp;issue=2&amp;pages=e89134&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Varshney RK, Sigmund R, Börner A, Korzun V, Stein N, Sorrells ME, Langridge P, Graner A. Interspecific transferability and comparative mapping of barley EST-SSR markers in wheat, rye and rice. Plant Sci. 2005;168(1):195–202.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.plantsci.2004.08.001"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Interspecific%20transferability%20and%20comparative%20mapping%20of%20barley%20EST-SSR%20markers%20in%20wheat%2C%20rye%20and%20rice&amp;author=RK.%20Varshney&amp;author=R.%20Sigmund&amp;author=A.%20B%C3%B6rner&amp;author=V.%20Korzun&amp;author=N.%20Stein&amp;author=ME.%20Sorrells&amp;author=P.%20Langridge&amp;author=A.%20Graner&amp;journal=Plant%20Sci&amp;volume=168&amp;issue=1&amp;pages=195-202&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gardner N, Hokanson SC. Intersimple sequence repeat fingerprinting and genetic variation in a collection of Clematis cultivars and commercial germplasm. J Am Soc Hortic Sci. 2005;40(7):1982–7.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Intersimple%20sequence%20repeat%20fingerprinting%20and%20genetic%20variation%20in%20a%20collection%20of%20Clematis%20cultivars%20and%20commercial%20germplasm&amp;author=N.%20Gardner&amp;author=SC.%20Hokanson&amp;journal=J%20Am%20Soc%20Hortic%20Sci&amp;volume=40&amp;issue=7&amp;pages=1982-1987&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Tao Y, Wang LY, Roh MS. Confirmation of Clematis hybrids using molecular markers. Sci Hortic-Amsterdam. 2010;125(2):136–45.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.scienta.2010.03.005"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Confirmation%20of%20Clematis%20hybrids%20using%20molecular%20markers&amp;author=Y.%20Tao&amp;author=LY.%20Wang&amp;author=MS.%20Roh&amp;journal=Sci%20Hortic-Amsterdam&amp;volume=125&amp;issue=2&amp;pages=136-145&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Xie L, Wen J, Li LQ. Phylogenetic analyses of Clematis (Ranunculaceae) based on sequences of nuclear ribosomal ITS and three plastid regions. Syst Bot. 2011;36(4):907–21.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1600/036364411X604921"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Phylogenetic%20analyses%20of%20Clematis%20%28Ranunculaceae%29%20based%20on%20sequences%20of%20nuclear%20ribosomal%20ITS%20and%20three%20plastid%20regions&amp;author=L.%20Xie&amp;author=J.%20Wen&amp;author=LQ.%20Li&amp;journal=Syst%20Bot&amp;volume=36&amp;issue=4&amp;pages=907-921&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Xiao LI, Wang P, Zhao L. rDNA-ITS sequence analysis of the traditional mongolian medicinal plants in Clematis L. J Inner Mongolia Medical College. 2012;34(4):325–30.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=rDNA-ITS%20sequence%20analysis%20of%20the%20traditional%20mongolian%20medicinal%20plants%20in%20Clematis%20L&amp;author=LI.%20Xiao&amp;author=P.%20Wang&amp;author=L.%20Zhao&amp;journal=J%20Inner%20Mongolia%20Medical%20College&amp;volume=34&amp;issue=4&amp;pages=325-330&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Slomba JM, Garey JR, Essig FB. The actin I intron—a phylogenetically informative DNA region in Clematis (Ranunculaceae). Sida Contributions to Botany. 2004;21(2):879–86.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20actin%20I%20intron%E2%80%94a%20phylogenetically%20informative%20DNA%20region%20in%20Clematis%20%28Ranunculaceae%29&amp;author=JM.%20Slomba&amp;author=JR.%20Garey&amp;author=FB.%20Essig&amp;journal=Sida%20Contributions%20to%20Botany&amp;volume=21&amp;issue=2&amp;pages=879-886&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yan SX, Liu HJ, Lin LL, Liao S, Jin YL, Pei LY, Xie L. Taxonomic status of Clematis acerifolia var. elobata, based on molecular evidence. Phytotaxa. 2016;268(3):209.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.11646/phytotaxa.268.3.5"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Taxonomic%20status%20of%20Clematis%20acerifolia%20var.%20elobata%2C%20based%20on%20molecular%20evidence&amp;author=SX.%20Yan&amp;author=HJ.%20Liu&amp;author=LL.%20Lin&amp;author=S.%20Liao&amp;author=YL.%20Jin&amp;author=LY.%20Pei&amp;author=L.%20Xie&amp;journal=Phytotaxa&amp;volume=268&amp;issue=3&amp;pages=209&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gao L, Ma Y, Wang P, Wang S, Yang R, Wang Q, Li L, et al. Transcriptome profiling of Clematis apiifolia: insights into heat-stress responses. DNA Cell Biol. 2017;36(11):938–46.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1089/dna.2017.3850"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28945464"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Transcriptome%20profiling%20of%20Clematis%20apiifolia%3A%20insights%20into%20heat-stress%20responses&amp;author=L.%20Gao&amp;author=Y.%20Ma&amp;author=P.%20Wang&amp;author=S.%20Wang&amp;author=R.%20Yang&amp;author=Q.%20Wang&amp;author=L.%20Li&amp;journal=DNA%20Cell%20Biol&amp;volume=36&amp;issue=11&amp;pages=938-946&amp;publication_year=2017"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Grabherr MG, Haas BJ, Yassour M, Levin JZ, Thompson DA, Amit I, Adiconis X, et al. Full-length transcriptome assembly from rna-seq data without a reference genome. Nat Biotechnol. 2011;29(7):644.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nbt.1883"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21572440"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3571712"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Full-length%20transcriptome%20assembly%20from%20rna-seq%20data%20without%20a%20reference%20genome&amp;author=MG.%20Grabherr&amp;author=BJ.%20Haas&amp;author=M.%20Yassour&amp;author=JZ.%20Levin&amp;author=DA.%20Thompson&amp;author=I.%20Amit&amp;author=X.%20Adiconis&amp;journal=Nat%20Biotechnol&amp;volume=29&amp;issue=7&amp;pages=644&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gao CX, Yang BX, Zhang DD, Chen M, Tian JK. Enhanced metabolic process to indole alkaloids in <em class="EmphasisTypeItalic">Clematis terniflora</em> DC after exposure to high level of UV-B irradiation followed by the dark. BMC Plant Biol. 2016;16:231.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/s12870-016-0920-3"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27776479"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5078895"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Enhanced%20metabolic%20process%20to%20indole%20alkaloids%20in%20Clematis%20terniflora%20DC%20after%20exposure%20to%20high%20level%20of%20UV-B%20irradiation%20followed%20by%20the%20dark&amp;author=CX.%20Gao&amp;author=BX.%20Yang&amp;author=DD.%20Zhang&amp;author=M.%20Chen&amp;author=JK.%20Tian&amp;journal=BMC%20Plant%20Biol&amp;volume=16&amp;pages=231&amp;publication_year=2016"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Amirkhanian V, Tsai E. Use of a pen-shaped capillary gel electrophoresis cartridge for cost-effective DNA fragment analysis. Am Lab. 2012;44(7):16–7.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Use%20of%20a%20pen-shaped%20capillary%20gel%20electrophoresis%20cartridge%20for%20cost-effective%20DNA%20fragment%20analysis&amp;author=V.%20Amirkhanian&amp;author=E.%20Tsai&amp;journal=Am%20Lab&amp;volume=44&amp;issue=7&amp;pages=16-17&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yeh FC, Boyle TJB. Population genetic analysis of co-dominant and dominant markers and quantitative traits. Belg J Bot. 1997;129:157.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Population%20genetic%20analysis%20of%20co-dominant%20and%20dominant%20markers%20and%20quantitative%20traits&amp;author=FC.%20Yeh&amp;author=TJB.%20Boyle&amp;journal=Belg%20J%20Bot&amp;volume=129&amp;pages=157&amp;publication_year=1997"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Liu K, Muse SV. Power marker: an integrated analysis environment for genetic marker analysis. Bioinformatics. 2005;21(9):2128–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/bti282"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15705655"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Power%20marker%3A%20an%20integrated%20analysis%20environment%20for%20genetic%20marker%20analysis&amp;author=K.%20Liu&amp;author=SV.%20Muse&amp;journal=Bioinformatics&amp;volume=21&amp;issue=9&amp;pages=2128-2129&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Nei M, Li WH. Mathematical model for studying genetic variation in terms of restriction endonucleases. Proc Natl Acad Sci U S A. 1979;76(10):5269–73.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.76.10.5269"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=291943"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC413122"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Mathematical%20model%20for%20studying%20genetic%20variation%20in%20terms%20of%20restriction%20endonucleases&amp;author=M.%20Nei&amp;author=WH.%20Li&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;volume=76&amp;issue=10&amp;pages=5269-5273&amp;publication_year=1979"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR24" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wang W, Lu AM, Ren Y, Endress ME, Chen ZD. Phylogeny and classification of Ranunculales: evidence from four molecular loci and morphological data. Perspect Plant Ecol. 2009;11(2):81–110.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.ppees.2009.01.001"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Phylogeny%20and%20classification%20of%20Ranunculales%3A%20evidence%20from%20four%20molecular%20loci%20and%20morphological%20data&amp;author=W.%20Wang&amp;author=AM.%20Lu&amp;author=Y.%20Ren&amp;author=ME.%20Endress&amp;author=ZD.%20Chen&amp;journal=Perspect%20Plant%20Ecol&amp;volume=11&amp;issue=2&amp;pages=81-110&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR25" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wang W, Xie L. A revision of Clematis sect. Tubulosae (Ranunculaceae). Acta Phytotaxon Sin. 2007;45(4):425–57.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1360/aps06114"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20revision%20of%20Clematis%20sect.%20Tubulosae%20%28Ranunculaceae%29&amp;author=W.%20Wang&amp;author=L.%20Xie&amp;journal=Acta%20Phytotaxon%20Sin.&amp;volume=45&amp;issue=4&amp;pages=425-457&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR26" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sorkheh K, Dehkordi MK, Ercisli S, Hegedus A, Halász J. Comparison of traditional and new generation DNA markers declares high genetic diversity and differentiated population structure of wild almond species. Sci Rep. 2017;7(1):1–17.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/s41598-017-06084-4"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparison%20of%20traditional%20and%20new%20generation%20DNA%20markers%20declares%20high%20genetic%20diversity%20and%20differentiated%20population%20structure%20of%20wild%20almond%20species&amp;author=K.%20Sorkheh&amp;author=MK.%20Dehkordi&amp;author=S.%20Ercisli&amp;author=A.%20Hegedus&amp;author=J.%20Hal%C3%A1sz&amp;journal=Sci%20Rep&amp;volume=7&amp;issue=1&amp;pages=1-17&amp;publication_year=2017"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR27" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Jiang M, Zhou YQ, Li RR. ITS sequence analysis of eight medicinal plants in Clematis L. Chin Tradit Herbal Drugs. 2011;42(9):1802–6.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=ITS%20sequence%20analysis%20of%20eight%20medicinal%20plants%20in%20Clematis%20L&amp;author=M.%20Jiang&amp;author=YQ.%20Zhou&amp;author=RR.%20Li&amp;journal=Chin%20Tradit%20Herbal%20Drugs&amp;volume=42&amp;issue=9&amp;pages=1802-1806&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR28" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wang L, Du HY, Li TZ, Wuyun TN. De novo transcriptome sequencing and identification of genes related to salt stress in Eucommia ulmoides Oliver. Trees. 2017;32(1):151–63.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s00468-017-1620-9"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=De%20novo%20transcriptome%20sequencing%20and%20identification%20of%20genes%20related%20to%20salt%20stress%20in%20Eucommia%20ulmoides%20Oliver&amp;author=L.%20Wang&amp;author=HY.%20Du&amp;author=TZ.%20Li&amp;author=TN.%20Wuyun&amp;journal=Trees&amp;volume=32&amp;issue=1&amp;pages=151-163&amp;publication_year=2017"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR29" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Souframanien J, Reddy KS. De novo assembly, characterization of immature seed transcriptome and development of genic-SSR markers in black gram [<em class="EmphasisTypeItalic">Vigna mungo</em> (L.) Hepper]. Plos One. 2015;10(6):e0128748.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0128748"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26042595"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4456365"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%2C%20characterization%20of%20immature%20seed%20transcriptome%20and%20development%20of%20genic-SSR%20markers%20in%20black%20gram%20%5BVigna%20mungo%20%28L.%29%20Hepper%5D&amp;author=J.%20Souframanien&amp;author=KS.%20Reddy&amp;journal=Plos%20One&amp;volume=10&amp;issue=6&amp;pages=e0128748&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR30" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Dong S, Liu Y, Niu J, Ning Y, Lin S, Zhang Z. De novo transcriptome analysis of the Siberian apricot (Prunus sibirica L.) and search for potential SSR markers by 454 pyrosequencing. Gene. 2014;544(2):220.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.gene.2014.04.031"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24746601"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=De%20novo%20transcriptome%20analysis%20of%20the%20Siberian%20apricot%20%28Prunus%20sibirica%20L.%29%20and%20search%20for%20potential%20SSR%20markers%20by%20454%20pyrosequencing&amp;author=S.%20Dong&amp;author=Y.%20Liu&amp;author=J.%20Niu&amp;author=Y.%20Ning&amp;author=S.%20Lin&amp;author=Z.%20Zhang&amp;journal=Gene&amp;volume=544&amp;issue=2&amp;pages=220&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR31" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhang LS, Yang XN, Qi XN, Guo CH, Jing ZB. Characterizing the transcriptome and microsatellite markers for almond (Amygdalus communis L.) using the illumina sequencing platform. Hereditas. 2018;155(1):14.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/s41065-017-0049-x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29075165"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Characterizing%20the%20transcriptome%20and%20microsatellite%20markers%20for%20almond%20%28Amygdalus%20communis%20L.%29%20using%20the%20illumina%20sequencing%20platform&amp;author=LS.%20Zhang&amp;author=XN.%20Yang&amp;author=XN.%20Qi&amp;author=CH.%20Guo&amp;author=ZB.%20Jing&amp;journal=Hereditas&amp;volume=155&amp;issue=1&amp;pages=14&amp;publication_year=2018"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR32" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wu T, Luo S, Wang R, Zhong Y, Xu X, Lin Y, He X, et al. The first illumina-based de novo transcriptome sequencing and analysis of pumpkin (Cucurbita moschata Duch.) and SSR marker development. Mol Breed. 2014;34(3):1437–47.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11032-014-0128-x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20first%20illumina-based%20de%20novo%20transcriptome%20sequencing%20and%20analysis%20of%20pumpkin%20%28Cucurbita%20moschata%20Duch.%29%20and%20SSR%20marker%20development&amp;author=T.%20Wu&amp;author=S.%20Luo&amp;author=R.%20Wang&amp;author=Y.%20Zhong&amp;author=X.%20Xu&amp;author=Y.%20Lin&amp;author=X.%20He&amp;journal=Mol%20Breed&amp;volume=34&amp;issue=3&amp;pages=1437-1447&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR33" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wang J, Chen Z, Jin S, Hu Z, Huang Y, Diao Y. Development and characterization of simple sequence repeat (SSR) markers based on a full-length cDNA library of napier grass (Pennisetum purpureum Schum). Genes &amp; Genom. 2017;3:1–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Development%20and%20characterization%20of%20simple%20sequence%20repeat%20%28SSR%29%20markers%20based%20on%20a%20full-length%20cDNA%20library%20of%20napier%20grass%20%28Pennisetum%20purpureum%20Schum%29&amp;author=J.%20Wang&amp;author=Z.%20Chen&amp;author=S.%20Jin&amp;author=Z.%20Hu&amp;author=Y.%20Huang&amp;author=Y.%20Diao&amp;journal=Genes%20%26%20Genom&amp;volume=3&amp;pages=1-9&amp;publication_year=2017"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR34" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yue X, Liu G, Zong Y, Teng Y, Cai D. Development of genic SSR markers from transcriptome sequencing of pear buds. J Zhejiang Univ Sci B. 2014;5(4):303–12.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1631/jzus.B1300240"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Development%20of%20genic%20SSR%20markers%20from%20transcriptome%20sequencing%20of%20pear%20buds&amp;author=X.%20Yue&amp;author=G.%20Liu&amp;author=Y.%20Zong&amp;author=Y.%20Teng&amp;author=D.%20Cai&amp;journal=J%20Zhejiang%20Univ%20Sci%20B&amp;volume=5&amp;issue=4&amp;pages=303-312&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR35" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yang Y, Jing ZB, Ruan XF, Cheng JM. Development of simple sequence repeat markers in persimmon (Diospyros L) and their potential use in related species. Genet Mol Res. 2015;14(1):609.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.4238/2015.January.30.2"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25729996"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Development%20of%20simple%20sequence%20repeat%20markers%20in%20persimmon%20%28Diospyros%20L%29%20and%20their%20potential%20use%20in%20related%20species&amp;author=Y.%20Yang&amp;author=ZB.%20Jing&amp;author=XF.%20Ruan&amp;author=JM.%20Cheng&amp;journal=Genet%20Mol%20Res&amp;volume=14&amp;issue=1&amp;pages=609&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR36" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Miikeda O, Kita K, Handa T, Yukawa T. Phylogenetic relationships of Clematis (Ranunculaceae) based on chloroplast and nuclear DNA sequences. Bot J Linn Soc. 2006;152(2):153–68.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1095-8339.2006.00551.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Phylogenetic%20relationships%20of%20Clematis%20%28Ranunculaceae%29%20based%20on%20chloroplast%20and%20nuclear%20DNA%20sequences&amp;author=O.%20Miikeda&amp;author=K.%20Kita&amp;author=T.%20Handa&amp;author=T.%20Yukawa&amp;journal=Bot%20J%20Linn%20Soc&amp;volume=152&amp;issue=2&amp;pages=153-168&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR37" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li Y, Xu C, Lin X, Cui B, Wu R, Pang X. De novo assembly and characterization of the fruit transcriptome of Chinese jujube (<em class="EmphasisTypeItalic">Ziziphus jujuba</em> Mill.) using 454 pyrosequencing and the development of novel tri-nucleotide SSR markers. Plos One. 2014;9(9):e106438.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0106438"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25184704"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4153635"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20and%20characterization%20of%20the%20fruit%20transcriptome%20of%20Chinese%20jujube%20%28Ziziphus%20jujuba%20Mill.%29%20using%20454%20pyrosequencing%20and%20the%20development%20of%20novel%20tri-nucleotide%20SSR%20markers&amp;author=Y.%20Li&amp;author=C.%20Xu&amp;author=X.%20Lin&amp;author=B.%20Cui&amp;author=R.%20Wu&amp;author=X.%20Pang&amp;journal=Plos%20One&amp;volume=9&amp;issue=9&amp;pages=e106438&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR38" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Huang LL, Yang X, Sun P, Wen T, Hu SQ. The first illumina-based De novo transcriptome sequencing and analysis of safflower flowers. PLoS One. 2012;7(6):e38653.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0038653"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20first%20illumina-based%20De%20novo%20transcriptome%20sequencing%20and%20analysis%20of%20safflower%20flowers&amp;author=LL.%20Huang&amp;author=X.%20Yang&amp;author=P.%20Sun&amp;author=T.%20Wen&amp;author=SQ.%20Hu&amp;journal=PLoS%20One&amp;volume=7&amp;issue=6&amp;pages=e38653&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR39" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Chen H, Liu L, Wang L, Wang S, Somta P, Cheng X. Development and validation of EST-SSR markers from the transcriptome of adzuki bean (Vigna angularis). PLoS One. 2015;10(7):e0131939.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0131939"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26146990"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4492930"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Development%20and%20validation%20of%20EST-SSR%20markers%20from%20the%20transcriptome%20of%20adzuki%20bean%20%28Vigna%20angularis%29&amp;author=H.%20Chen&amp;author=L.%20Liu&amp;author=L.%20Wang&amp;author=S.%20Wang&amp;author=P.%20Somta&amp;author=X.%20Cheng&amp;journal=PLoS%20One&amp;volume=10&amp;issue=7&amp;pages=e0131939&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR40" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li D, Zhi D, Bi Q, Liu X, Men Z. De novo assembly and characterization of bark transcriptome using illumina sequencing and development of EST-SSR markers in rubber tree (<em class="EmphasisTypeItalic">Hevea brasiliensis</em> Muell. Arg.). Bmc Genomics. 2012;13(1):192.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2164-13-192"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22607098"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3431226"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20and%20characterization%20of%20bark%20transcriptome%20using%20illumina%20sequencing%20and%20development%20of%20EST-SSR%20markers%20in%20rubber%20tree%20%28Hevea%20brasiliensis%20Muell.%20Arg.%29&amp;author=D.%20Li&amp;author=D.%20Zhi&amp;author=Q.%20Bi&amp;author=X.%20Liu&amp;author=Z.%20Men&amp;journal=Bmc%20Genomics&amp;volume=13&amp;issue=1&amp;pages=192&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li></ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#TheAuthors" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="TheAuthors">Copyright</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="TheAuthors"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">©&nbsp;The Author(s)&nbsp;2018</div></div></section></div>
                        


                    </article>
                </main>

                <div class="c-content-layout__side u-text-sm">
                    <aside>
                        <div class="p-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="//hereditasjournal.biomedcentral.com/track/pdf/10.1186/s41065-018-0060-x" data-track="click" data-track-category="PDF Download"
            data-track-action="Click download article PDF"
            data-track-label="10.1186/s41065-018-0060-x">Download PDF</a>
        </p>
    
                            
                            
    <button type="button"
       class="c-button c-button--secondary c-button--full-width c-button--expandable"
       data-toggle="collapse"
       data-target="export-citations"
       data-test="export-citations-button"
       data-sticky-update="true"
       aria-controls="export-citations"
       aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/s41065-018-0060-x"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0060-x?format=refman&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/s41065-018-0060-x"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0060-x?format=refman&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/s41065-018-0060-x"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0060-x?format=refman&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/s41065-018-0060-x"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0060-x?format=endnote&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/s41065-018-0060-x"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0060-x?format=endnote&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/s41065-018-0060-x"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0060-x?format=endnote&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/s41065-018-0060-x"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0060-x?format=bibtex&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/s41065-018-0060-x"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0060-x?format=bibtex&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/s41065-018-0060-x"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s41065-018-0060-x?format=bibtex&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
    </div>

                        </div>
                        
    
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
                                <div id="article-access-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/access-count/10.1186/s41065-018-0060-x #accessCount-container"></div>
                                <div id="citation-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/citation-count/10.1186/s41065-018-0060-x #count-container"></div>
                                <div id="altmetric-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/altmetric/10.1186/s41065-018-0060-x #altmetric-container"></div>
                            </ul>
                        </div>
                        
    <div id="sociaMediaShare" class="p-separator u-js-shown">
        <h3 class="p-section-title">Share this article</h3>
        <ul id="shareButtons" class="c-social-links">
            <li class="c-social-links__item">
                <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s41065-018-0060-x">
                    <span class="u-visually-hidden">Share on Twitter</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-twitter"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s41065-018-0060-x">
                    <span class="u-visually-hidden">Share on Facebook</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-facebook"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click"
                        data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s41065-018-0060-x">
                    <span class="u-visually-hidden">Share on LinkedIn</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-linkedin"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s41065-018-0060-x">
                    <span class="u-visually-hidden">Share on Weibo</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-weibo"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnGooglePlus" class="js-btnShareOnGplus" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Google+" data-track-label="10.1186/s41065-018-0060-x">
                    <span class="u-visually-hidden">Share on Google Plus</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-gplus"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click"
                        data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s41065-018-0060-x">
                    <span class="u-visually-hidden">Share on Reddit</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-reddit"></use>
                    </svg>
                </button>
            </li>
        </ul>
    </div>
 
                        
    
        <div class="p-separator u-js-shown">
            <h3 class="p-section-title">See updates</h3>
            <button class="c-crossmark-button" data-target="crossmark" data-test="crossmark" data-track="click" data-track-category="Article Page" data-track-action="Click CrossMark" data-track-label="10.1186/s41065-018-0060-x">
                <img id="crossmark-icon" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" alt="Crossmark logo" />
            </button>
        </div>
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
                                <li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/s41065-018-0060-x" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=738&doi=10.1186%2Fs41065-018-0060-x&volume=155&issue=1&title=Characterization+of+new+microsatellite+markers+based+on+the+transcriptome+sequencing+of+Clematis+finetiana&author_name=Zhigao+Liu&start_page=1&end_page=9">Order reprint</a>

                                </li>
                            </ul>
                        </div>
                    </aside>
                    
    <div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">Hereditas</span>
                        </h4>
                        <p class="c-journal-footer__issn">ISSN: 1601-5223</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
                                
                                    <li class="c-journal-footer__contact-item">Submission enquiries: <a href="https://www.editorialmanager.com/here/" target="_blank">Access here and click Contact Us</a></li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a></li>
                                
                            </ul>
                        </div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class='tracker' style='display:none' src='/track/article/10.1186/s41065-018-0060-x' alt=""/>

         
    <footer>
        <div class="ctx-publisher-footer">
    <div class="c-navbar">
        <div class="u-container">
            <div class="c-navbar__logo u-margin-right-xl">
                <a href="https://www.biomedcentral.com">
                    <img src=/static/images/bmc/logos/logo-bmc-aj-594e03e20d.svg alt="BMC logo" data-test="navbar-logo-footer">

                </a>
            </div>
            <div class="c-navbar__content">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


            </div>
        </div>
    </div>
    <div class="c-publisher-footer" data-test="publisher-footer">
        <div class="u-container">
            
            <div class="c-flex c-flex--wrap c-flex--space-between" role="navigation" data-test="publisher-footer-menu">
                <div class="c-flex" aria-label="Publisher footer links">
                    
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/login">Receive BMC newsletters</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/account">Manage article alerts</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/policies">Policies</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/accessibility">Accessibility</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/press-centre">Press center</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/contact-us">Contact us</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="//www.biomedcentral.com/about/jobs">Careers</a>
                                    </li>
                                
                            </ul>
                        
                    
                </div>
                <div class="u-margin-bottom-lg">
                    <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                    <ul class="c-flex u-reset-list" data-test="footer-social-links">
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://twitter.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Twitter link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Twitter page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-twitter-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://www.facebook.com/BioMedCentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Facebook link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Facebook page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-facebook-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://plus.google.com/106226337984152901734/"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Google Plus link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Google Plus page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-gplus-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="http://www.weibo.com/biomedcentral"
                                   class="u-text-inherit"
                                   data-track="click"
                                   data-track-action="Click Weibo link"
                                   data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Weibo page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-weibo-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </div>
            <p class="u-reset-margin">
                By using this website, you agree to our
                <a class="u-text-inherit" href="//www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
                <a class="u-text-inherit" href="//www.biomedcentral.com/privacy-statement">Privacy
                    statement</a> and
                <a class="u-text-inherit" href="//www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
                
                    <a class="optanon-toggle-display u-text-inherit" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
                
            </p>
        </div>
    </div>
</div>


        <div class="c-sn-footer c-sn-footer--neutral">
    <div class="u-container">
        <img src=/static/images/bmc/logos/logo-springernature-8bd4e8d82f.svg class="c-sn-footer__logo" alt="Springer Nature Logo" itemprop="logo" >
        <p data-test="copyright">&#169; 2018 BioMed Central Ltd unless otherwise stated. Part of
            <a href="http://www.springernature.com"
               data-track="click"
               data-track-action="Click Springer Nature link"
               data-track-category="Corporate footer"
               itemscope itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>

    </div>

    
    <script type="text/javascript" src="//cdn.pbgrd.com/bmc.js" async="async"></script>
    <noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>
    <!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker -->

<script id="springerlink-event-tracker" src="https://event-tracker.springernature.com/dist/eventTracker.js"></script>
<script type="text/javascript">
    (function(){
        var dataLayer = window.dataLayer[0];
        var doi = dataLayer.content.article.doi;
        var imprint = dataLayer.content.contentInfo.imprint;
        var bpids = dataLayer.session.authentication.authenticationID;
        var publisher;
        var pageType = dataLayer.page.category.pageType;
        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }
        var eventObject = {'content_type' : pageType,'doi' : doi};

        if (imprint === 'BioMed Central') { publisher = 'BMC' }
        else if (imprint === 'SpringerOpen') { publisher = 'SpringerOpen' }
        else { throw new Error('Event Tracker Error: The publisher has an unexpected value') }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }
        new EventTracker({'platform':publisher}).sendEvent('display', eventObject);
    })();
</script>


    </body>
</html>

